{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Python adaption from DYLAN"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "usage: map_on_oasis.py [-h] [-i INPUT] [-b BATCH_NAME] [-a ALIGNER]\n",
      "                       [-g GENOME] [-n NUM_CORES] [-p]\n",
      "map_on_oasis.py: error: unrecognized arguments: -f /run/user/513359/jupyter/kernel-f0157aea-acb5-4115-8264-0aae4457976f.json\n"
     ]
    },
    {
     "ename": "SystemExit",
     "evalue": "2",
     "output_type": "error",
     "traceback": [
      "An exception has occurred, use %tb to see the full traceback.\n",
      "\u001b[0;31mSystemExit\u001b[0m\u001b[0;31m:\u001b[0m 2\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/gpfs/data01/glasslab/home/iholtman/anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:2889: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.\n",
      "  warn(\"To exit: use 'exit', 'quit', or Ctrl-D.\", stacklevel=1)\n"
     ]
    }
   ],
   "source": [
    "#!/home/phage/anaconda2/bin/python\n",
    "import argparse\n",
    "import os\n",
    "import shutil\n",
    "import random\n",
    "import collections\n",
    "import subprocess\n",
    "\n",
    "BOWTIE2_OPTIONS = '-t --very-sensitive'\n",
    "DEFAULT_CORES = 8\n",
    "\n",
    "GLASS_DATA_PATH='/projects/ps-glasslab-data/scratch/phage'\n",
    "# MAPPING_SCRIPTS_PATH='/projects/ps-glasslab-bioinformatics/glassutils/mapping_scripts'\n",
    "BOWTIE2_INDEX_PATH='/projects/ps-glasslab-bioinformatics/software/bowtie2/indexes'\n",
    "STAR_PATH='/projects/ps-glasslab-bioinformatics/software/STAR'\n",
    "BOWTIE2_PATH='/projects/glass-group/bioinformatics/bowtie2/bowtie2'\n",
    "HOMER_PATH='/projects/glass-group/bioinformatics/homer/bin'\n",
    "TSCC_SCRATCH_PATH='/oasis/tscc/scratch/phage/aligning'\n",
    "\n",
    "\n",
    "BOWTIE2_SE_TEMPLATE = \"\"\"#!/bin/bash\n",
    "#PBS -q hotel\n",
    "#PBS -N {fname_prefix}\n",
    "#PBS -l nodes=1:ppn={num_cores}\n",
    "#PBS -l walltime=1:00:00\n",
    "#PBS -o {tscc_output_path}/{fname_prefix}_pbs_output.txt\n",
    "#PBS -e {tscc_output_path}/{fname_prefix}_pbs_error.txt\n",
    "#PBS -M dskola@ucsd.edu\n",
    "#PBS -V\n",
    "#PBS -m abe\n",
    "#PBS -A glass-group\n",
    "{bowtie2_path} {bowtie2_options} -q -p {num_cores} -x {bowtie2_index_path}/{genome} -U <(zcat {input_filepath}) > {tscc_output_path}/{sam_filename} 2> {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log \n",
    "{homer_path}/makeTagDirectory {tscc_output_path}/tag_directories/{fname_prefix}_with_M -genome {genome} -checkGC {tscc_output_path}/{sam_filename} -format sam\n",
    "rm {tscc_output_path}/tag_directories/{fname_prefix}_with_M/chrM.tags.tsv\n",
    "{homer_path}/makeTagDirectory {tscc_output_path}/tag_directories/{fname_prefix} -d {tscc_output_path}/tag_directories/{fname_prefix}_with_M\n",
    "cp {tscc_output_path}/tag_directories/{fname_prefix}_with_M/tagInfo.txt {tscc_output_path}/tag_directories/{fname_prefix}/tagInfo_with_M.txt\n",
    "rm -rf {tscc_output_path}/tag_directories/{fname_prefix}_with_M\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log {glass_data_output_path}/log_files\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log {tscc_output_path}/tag_directories/{fname_prefix}/\n",
    "cp -r {tscc_output_path}/tag_directories/{fname_prefix} {glass_data_output_path}/tag_directories\"\"\"\n",
    "\n",
    "BOWTIE2_PE_TEMPLATE = \"\"\"#!/bin/bash\n",
    "#PBS -q hotel\n",
    "#PBS -N {fname_prefix}\n",
    "#PBS -l nodes=1:ppn={num_cores}\n",
    "#PBS -l walltime=1:00:00\n",
    "#PBS -o {tscc_output_path}/{fname_prefix}_pbs_output.txt\n",
    "#PBS -e {tscc_output_path}/{fname_prefix}_pbs_error.txt\n",
    "#PBS -M dskola@ucsd.edu\n",
    "#PBS -V\n",
    "#PBS -m abe\n",
    "#PBS -A glass-group\n",
    "{bowtie2_path} {bowtie2_options} -q -p {num_cores} -x {bowtie2_index_path}/{genome} -1 <(zcat {input_1_filepath}) -2 <(zcat {input_2_filepath}) > {tscc_output_path}/{sam_filename} 2> {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log \n",
    "{homer_path}/makeTagDirectory {tscc_output_path}/tag_directories/{fname_prefix}_with_M -genome {genome} -checkGC {tscc_output_path}/{sam_filename} -format sam\n",
    "rm {tscc_output_path}/tag_directories/{fname_prefix}_with_M/chrM.tags.tsv\n",
    "{homer_path}/makeTagDirectory {tscc_output_path}/tag_directories/{fname_prefix} -d {tscc_output_path}/tag_directories/{fname_prefix}_with_M\n",
    "cp {tscc_output_path}/tag_directories/{fname_prefix}_with_M/tagInfo.txt {tscc_output_path}/tag_directories/{fname_prefix}/tagInfo_with_M.txt\n",
    "rm -rf {tscc_output_path}/tag_directories/{fname_prefix}_with_M\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log {glass_data_output_path}/log_files\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.bowtie2.log {tscc_output_path}/tag_directories/{fname_prefix}/\n",
    "cp -r {tscc_output_path}/tag_directories/{fname_prefix} {glass_data_output_path}/tag_directories\"\"\"\n",
    "\n",
    "STAR_TEMPLATE= \"\"\"#!/bin/bash\n",
    "#PBS -q hotel\n",
    "#PBS -N {fname_prefix}\n",
    "#PBS -l nodes=1:ppn={num_cores}\n",
    "#PBS -l walltime=3:00:00\n",
    "#PBS -o {tscc_output_path}/{fname_prefix}_pbs_output.txt\n",
    "#PBS -e {tscc_output_path}/{fname_prefix}_pbs_error.txt\n",
    "#PBS -M dskola@ucsd.edu\n",
    "#PBS -V\n",
    "#PBS -m abe\n",
    "#PBS -A glass-group\n",
    "mkdir {tscc_output_path}/{fname_prefix}\n",
    "{star_path}/STAR --genomeDir {star_path}/genomes/{genome} --readFilesCommand zcat --readFilesIn {input_filepath} --outFileNamePrefix {tscc_output_path}/{fname_prefix}/ --runThreadN {num_cores}\n",
    "mv /{tscc_output_path}/{fname_prefix}/Aligned.out.sam {tscc_output_path}/{fname_prefix}/{fname_prefix}.{genome}.star.sam\n",
    "mv {tscc_output_path}/{fname_prefix}/Log.final.out {tscc_output_path}/log_files/{fname_prefix}.{genome}.star.log\n",
    "{homer_path}/makeTagDirectory {tscc_output_path}/tag_directories/{fname_prefix} -genome hg38 -checkGC {tscc_output_path}/{fname_prefix}/{fname_prefix}.{genome}.star.sam -format sam -flip\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.star.log {glass_data_output_path}/log_files\n",
    "cp {tscc_output_path}/log_files/{fname_prefix}.{genome}.star.log {tscc_output_path}/tag_directories/{fname_prefix}\n",
    "cp -r {tscc_output_path}/tag_directories/{fname_prefix} {glass_data_output_path}/tag_directories\"\"\"\n",
    "\n",
    "def make_uuid():\n",
    "    return '{}_{}'.format(os.environ['USER'], int(random.random() * 10000))\n",
    "\n",
    "\n",
    "def establish_path(directory):\n",
    "    if not os.path.exists(directory):\n",
    "        os.makedirs(directory)\n",
    "\n",
    "\n",
    "def process_all(batch_name, input_folder, aligner, num_cores, genome, paired_end=False):\n",
    "    input_folder = os.path.abspath(input_folder)\n",
    "    input_filenames = sorted([f for f in os.listdir(input_folder) if f.endswith('.fastq.gz') and os.path.isfile(os.path.join(input_folder, f))])\n",
    "\n",
    "    input_files = collections.defaultdict(lambda: {})\n",
    "    for input_filename in input_filenames:\n",
    "        input_filepath = os.path.join(input_folder, input_filename)\n",
    "\n",
    "        fname_prefix = input_filename.split('.')[0]\n",
    "#   print('path: {} prefix: {}'.format(input_filepath, fname_prefix))\n",
    "        if fname_prefix.endswith('_1'):\n",
    "            fname_prefix = fname_prefix[:-2]\n",
    "            input_files[fname_prefix]['pe_1'] = input_filepath\n",
    "        elif fname_prefix.endswith('_2'):\n",
    "            fname_prefix = fname_prefix[:-2]\n",
    "            input_files[fname_prefix]['pe_2'] = input_filepath \n",
    "        else:\n",
    "            input_files[fname_prefix]['se'] = input_filepath \n",
    "    #print(input_files) \n",
    "    # QC\n",
    "    for fname_prefix in input_files:\n",
    "        if 'se' in input_files[fname_prefix]:\n",
    "            assert len(input_files[fname_prefix]) == 1\n",
    "        if 'pe_1' in input_files[fname_prefix]:\n",
    "            assert 'pe_2' in input_files[fname_prefix] and len(input_files[fname_prefix]) == 2\n",
    "        if 'pe_2' in input_files[fname_prefix]:\n",
    "            assert 'pe_1' in input_files[fname_prefix] and len(input_files[fname_prefix]) == 2, input_files[fname_prefix]\n",
    "\n",
    "    print('Aligning {} with {} ({} cores)'.format(', '.join(input_files), aligner, num_cores ))\n",
    "    \n",
    "    # make various subfolders\n",
    "    tscc_output_path = os.path.join(TSCC_SCRATCH_PATH, batch_name)\n",
    "    glass_data_output_path = os.path.join(GLASS_DATA_PATH, batch_name)\n",
    "    print('Output path on TSCC: {}'.format(tscc_output_path))\n",
    "    print('Output path on epiglass: {}'.format(glass_data_output_path))\n",
    "\n",
    "    establish_path(os.path.join(tscc_output_path, 'tag_directories'))\n",
    "    establish_path(os.path.join(tscc_output_path, 'log_files'))\n",
    "    \n",
    "    establish_path(os.path.join(glass_data_output_path, 'tag_directories'))\n",
    "    establish_path(os.path.join(glass_data_output_path, 'log_files'))\n",
    "    \n",
    "    \n",
    "    # generate and submit scripts\n",
    "    for fname_prefix, input_filepaths in input_files.items():\n",
    "        \n",
    "        sam_filename = fname_prefix + '.sam'\n",
    "        script_filename = os.path.join(tscc_output_path, '{}.pbs.sh'.format(fname_prefix))\n",
    "        if aligner == 'bowtie2':\n",
    "            generate_bowtie2_script(input_filepaths=input_filepaths, fname_prefix=fname_prefix,\n",
    "                                    sam_filename=sam_filename, genome=genome, tscc_output_path=tscc_output_path,\n",
    "                                    glass_data_output_path=glass_data_output_path, num_cores=num_cores,\n",
    "                                    script_filename=script_filename)\n",
    "\n",
    "        \n",
    "        elif aligner == 'star':\n",
    "            generate_star_script(input_filepaths=input_filepaths, fname_prefix=fname_prefix,\n",
    "                                    sam_filename=sam_filename, genome=genome, tscc_output_path=tscc_output_path,\n",
    "                                    glass_data_output_path=glass_data_output_path, num_cores=num_cores,\n",
    "                                    script_filename=script_filename)\n",
    "                     \n",
    "        print('Submitting job for {}'.format(fname_prefix))\n",
    "        cmd_line = ['qsub', script_filename]\n",
    "        output = subprocess.check_output(cmd_line, stderr=subprocess.STDOUT)\n",
    "        print(output)\n",
    "   \n",
    "\n",
    "def generate_star_script(input_filepaths, fname_prefix, sam_filename, genome, tscc_output_path, glass_data_output_path, num_cores, script_filename):\n",
    "    if 'pe_1' in input_filepaths:\n",
    "        input_substring = '{} {}'.format(input_filepaths['pe_1'], input_filepaths['pe_2'])\n",
    "    else:\n",
    "        input_substring = input_filepaths['se']\n",
    "\n",
    "    script_string = STAR_TEMPLATE.format(input_filepath=input_substring, fname_prefix=fname_prefix, sam_filename=sam_filename, genome=genome, glass_data_output_path=glass_data_output_path, tscc_output_path=tscc_output_path, num_cores=num_cores, homer_path=HOMER_PATH, star_path=STAR_PATH)\n",
    "    with open(script_filename, 'wt') as script_file:\n",
    "        script_file.write(script_string)\n",
    "\n",
    "\n",
    "def generate_bowtie2_script(input_filepaths, fname_prefix, sam_filename, genome, tscc_output_path, glass_data_output_path, num_cores, script_filename):\n",
    "    if 'pe_1' in input_filepaths:\n",
    "        script_string = BOWTIE2_PE_TEMPLATE.format(input_1_filepath=input_filepaths['pe_1'], \n",
    "                                                input_2_filepath=input_filepaths['pe_2'],\n",
    "                                                fname_prefix=fname_prefix,\n",
    "                                                sam_filename=sam_filename,\n",
    "                                                genome=genome,\n",
    "                                                glass_data_output_path=glass_data_output_path,\n",
    "                                                tscc_output_path=tscc_output_path,\n",
    "                                                num_cores=num_cores,\n",
    "                                                homer_path=HOMER_PATH,\n",
    "                                                bowtie2_path=BOWTIE2_PATH,\n",
    "                                                bowtie2_index_path=BOWTIE2_INDEX_PATH, \n",
    "                                                bowtie2_options=BOWTIE2_OPTIONS)\n",
    "    else:\n",
    "        script_string = BOWTIE2_SE_TEMPLATE.format(input_filepath=input_filepaths['se'], fname_prefix=fname_prefix, sam_filename=sam_filename, genome=genome, glass_data_output_path=glass_data_output_path,\n",
    "                                                tscc_output_path=tscc_output_path, num_cores=num_cores, homer_path=HOMER_PATH, bowtie2_path=BOWTIE2_PATH,\n",
    "                                                bowtie2_index_path=BOWTIE2_INDEX_PATH, bowtie2_options=BOWTIE2_OPTIONS)\n",
    "    with open(script_filename, 'wt') as script_file:\n",
    "        script_file.write(script_string)\n",
    "\n",
    "\n",
    "def main():\n",
    "    argparser = argparse.ArgumentParser(prog='map_on_oasis.py')\n",
    "    argparser.add_argument('-i', '--input', type=str, help='Folder containing gzipped fastq files to process')\n",
    "    argparser.add_argument('-b', '--batch-name', type=str, help='Name of batch job')\n",
    "    argparser.add_argument('-a', '--aligner', type=str, help='Which aligner to use. Valid options are \\'bowtie2\\' and \\'star\\'')\n",
    "    argparser.add_argument('-g', '--genome', type=str, help='Which genome to use.')\n",
    "    argparser.add_argument('-n', '--num-cores', type=int, default=DEFAULT_CORES, help='How many processors to request')\n",
    "    argparser.add_argument('-p', '--paired_end', action='store_true', help='Set to true if dealing with paired end sequences')  \n",
    " \n",
    "    args = argparser.parse_args()\n",
    "    assert args.aligner in ('star', 'bowtie2')\n",
    "    \n",
    "    process_all(batch_name = args.batch_name, input_folder = args.input, aligner = args.aligner, num_cores = args.num_cores, genome=args.genome, paired_end=args.paired_end) \n",
    "\n",
    "\n",
    "if __name__ == '__main__':\n",
    "    main()\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### SCRIPT FROM JENHAN: - SHELL script"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#! /bin/bash\n",
    "################################################################################\n",
    "\n",
    "### SUMMARY OF FUNCTIONS ###\n",
    "# One stop destination for all your mapping needs. This script, which is meant\n",
    "# to be executed on the TSCC, will create qsub scripts (and execute them) to\n",
    "# perform the following:\n",
    "# 1. Copy raw data (fastq.gz) from the Glassome to the TSCC (without using qsub)\n",
    "#    into an automatically generated directory on the Oasis file system\n",
    "# 2. Decompress the files if necessary\n",
    "# 3. Map the fastq files to a reference genome, producing a *.sam file\n",
    "#    * using STAR or Bowtie2\n",
    "# 4. Creates tag directories\n",
    "# 5. Calculates the PBC coefficient - which involves removing all non-uniquely \n",
    "#    mapped reads, duplicate reads, and calculating the pileups\n",
    "#    * read more about the PBC coefficient here: \n",
    "#      http://genome.ucsc.edu/ENCODE/qualityMetrics.html\n",
    "# 6. Moves all files from the TSCC back to Glassome\n",
    "# 7. Cleans up if neccessary\n",
    "# 8. Produces a summary of executed jobs - namely which ones failed\n",
    "###\n",
    "\n",
    "\n",
    "### OPTIONS AND ARGUMENTS ###\n",
    "# -l map files that are on the TSCC already\n",
    "# -t generate qsub scripts but do not execute them\n",
    "### \n",
    "\n",
    "### set default options ###\n",
    "\n",
    "map_local_files=false\n",
    "testing=false\n",
    "map_only=false\n",
    "no_emails=false\n",
    "paired=false\n",
    "copy_sam=false\n",
    "glassome_path='/projects/ps-glasslab-data/'\n",
    "mappingScripts_path='/projects/ps-glasslab-bioinformatics/glassutils/mapping_scripts/'\n",
    "scripts_path='/projects/ps-glasslab-bioinformatics/glassutils/scripts/'\n",
    "bowtie_index_path='/projects/ps-glasslab-bioinformatics/software/bowtie2/indexes/'\n",
    "star_path='/projects/ps-glasslab-bioinformatics/software/STAR/'\n",
    "bowtie_path='/projects/glass-group/bioinformatics/bowtie2'\n",
    "homer_path='/projects/glass-group/bioinformatics/homer/bin/'\n",
    "flip=true\n",
    "\n",
    "# check number of arguments\n",
    "if [ $# -lt 4 ] \n",
    "then\n",
    "    echo \"Usage: \"\n",
    "    echo \"map_on_oasis.sh <experiment type (atac|chip|rna)> <genome> \\\n",
    "<email> <input file directory> [optional arguments]\"\n",
    "    echo \"Options:\n",
    "-l    map files already on tscc or are already copied over\n",
    "-t    generate qsub scripts but do not execute them\n",
    "-m    only map files - do not create tag directories\n",
    "-e    do not send email notifications\n",
    "-s    copy sam files to Glassome\n",
    "-p    input data is paired end\n",
    "-f    do not use the -flip option when making RNA tag directories\"\n",
    "    exit 1\n",
    "fi\n",
    "\n",
    "### parse the input ###\n",
    "\n",
    "OPTIND=5\n",
    "while getopts \"ltmepsf\" option ; do # set $o to the next passed option\n",
    "    case \"$option\" in  \n",
    "    l)  \n",
    "       map_local_files=true \n",
    "    ;;  \n",
    "    t)  \n",
    "        testing=true\n",
    "    ;;  \n",
    "    m)  \n",
    "        map_only=true\n",
    "    ;;  \n",
    "    e)  \n",
    "        no_emails=true\n",
    "    ;;  \n",
    "    p)  \n",
    "        paired=true\n",
    "    ;;  \n",
    "    s)  \n",
    "        copy_sam=true\n",
    "    ;;  \n",
    "    f)  \n",
    "        flip=false\n",
    "    ;;  \n",
    "    esac\n",
    "done\n",
    "\n",
    "experimentType=$1\n",
    "genome=$2\n",
    "email=$3\n",
    "inputDirectory=$4\n",
    "###\n",
    "\n",
    "echo \"Beginning processing for $experimentType exeriments.\"\n",
    "echo \"Data contained in $inputDirectory will be mapped to the $genome genome\"\n",
    "if $paired\n",
    "then\n",
    "    echo \"Paired end option specified. This script is designed to work with\n",
    "Illumina paired end reads only\"\n",
    "fi\n",
    "\n",
    "if $no_emails\n",
    "then\n",
    "    echo \"Email notifications have been disabled\"\n",
    "else\n",
    "    echo \"Email notifications will be sent to $email\"\n",
    "fi\n",
    "\n",
    "if $map_only\n",
    "then\n",
    "    echo \"You are using the map only option - tag directories won't be created\n",
    " but sam files will be created on TSCC. Specify the -s option to copy sam files\n",
    "to Glassome\"\n",
    "fi\n",
    "\n",
    "if $copy_sam\n",
    "then\n",
    "    echo \"Sam files will be copied to Glassome\"\n",
    "fi\n",
    "\n",
    "if $testing\n",
    "then\n",
    "    echo \"Testing option enabled - qsub scripts will not be activated\"\n",
    "fi\n",
    "\n",
    "### check arguments ###\n",
    "\n",
    "#if [ $fileSource == \"glassome\" ]\n",
    "if ! $map_local_files\n",
    "then\n",
    "    inputDirectory=${inputDirectory#*gpfs/data01/glasslab/}\n",
    "    inputDirectory=$(readlink -fm ${glassome_path}/${inputDirectory/data//})\n",
    "else\n",
    "    if [[ $inputDirectory == \"/oasis/tscc/scratch/\"* ]]\n",
    "    then\n",
    "        inputDirectory=$(readlink -fm $inputDirectory)\n",
    "    else\n",
    "        inputDirectory=$(readlink -fm ${glassome_path}/${inputDirectory/data//})\n",
    "    fi\n",
    "fi\n",
    "\n",
    "# check that experiment type is rna (for RNA-seq) or chip (for ChIP-seq and etc)\n",
    "if [ ! $experimentType == \"rna\" ] && [ ! $experimentType == \"chip\" ] && [ ! $experimentType == \"atac\" ]\n",
    "then\n",
    "    echo \"Error! valid choices for experiment type are 'atac', 'chip' or rna'\"\n",
    "    exit 1\n",
    "fi\n",
    "\n",
    "# check that the input directory exists on specified fileSource machine\n",
    "if $map_local_files\n",
    "then\n",
    "    if [ ! -d $inputDirectory ]\n",
    "    then\n",
    "        echo \"Error! $inputDirectory cannot be found on the TSCC - try removing\\\n",
    " the -l option\" \n",
    "        exit 1\n",
    "    fi    \n",
    "else\n",
    "    if [ ! -d $inputDirectory ]\n",
    "    then\n",
    "        echo \"Error! $inputDirectory cannot be found on the Glassome - try \\\n",
    "using the -l option\" \n",
    "        exit 1\n",
    "    fi    \n",
    "fi\n",
    "\n",
    "###\n",
    "\n",
    "# create directory where data will be stored on glassome\n",
    "glassomeOutputDirectory=\"/projects/ps-glasslab-data/scratch/$USER/${inputDirectory##*/}\"\n",
    "if [ ! -d $glassomeOutputDirectory ]\n",
    "then\n",
    "    mkdir -p $glassomeOutputDirectory\n",
    "else\n",
    "    read -p \"This script will copy output files to $glassomeOutputDirectory, \\\n",
    "which already exists! Would you like to delete it. \\\n",
    "Enter y for yes and n for no [yn]?\" -n 1 -r \n",
    "    echo\n",
    "    if [[ $REPLY =~ ^[Yy]$ ]]\n",
    "    then\n",
    "        rm -rf $glassomeOutputDirectory\n",
    "        mkdir -p $glassomeOutputDirectory\n",
    "    fi\n",
    "fi\n",
    "\n",
    "#### copy files to oasis ###\n",
    "\n",
    "if ! [[ $inputDirectory == \"/oasis/tscc/scratch/\"* ]]\n",
    "then\n",
    "    outputDirectory=\"/oasis/tscc/scratch/$USER/${inputDirectory##*/}\"\n",
    "    if [ -d $outputDirectory ]\n",
    "    then\n",
    "        read -p \"$outputDirectory already exists on tscc! \\\n",
    "Would you like to delete it and recopy files?\\\n",
    " Enter y for yes and n for no [yn]\" -n 1 -r\n",
    "        echo\n",
    "        if [[ $REPLY =~ ^[Yy]$ ]]\n",
    "        then\n",
    "            echo \"removing $outputDirectory\"\n",
    "            rm -rf $outputDirectory\n",
    "            echo \"Copying files from $inputDirectory to $outputDirectory\"\n",
    "            scp -r $inputDirectory $outputDirectory\n",
    "        else\n",
    "            echo \"Files in $inputDirectory won't be copied to $outputDirectory\"\n",
    "        fi\n",
    "    else\n",
    "        echo \"Copying files from $inputDirectory to $outputDirectory\"\n",
    "        scp -r $inputDirectory $outputDirectory\n",
    "    fi\n",
    "else\n",
    "    outputDirectory=$inputDirectory\n",
    "fi\n",
    "\n",
    "# create separate directory for each sample\n",
    "if [ $(ls $outputDirectory/*fastq.gz| wc -l) -gt 0 ]\n",
    "then\n",
    "    for f in $outputDirectory/*fastq.gz;\n",
    "    do\n",
    "        dirname=${f%_S[0-9][0-9]*_L0*.fastq.gz}\n",
    "        dirname=${dirname%_S[0-9][0-9]*.fastq.gz}\n",
    "        dirname=${dirname%_S[0-9]*.fastq.gz}\n",
    "        dirname=${dirname%.fastq.gz} # for older data without lane number\n",
    "        if [ ! -d $dirname ]\n",
    "        then\n",
    "            mkdir $dirname\n",
    "        fi\n",
    "        mv $f $dirname\n",
    "    done\n",
    "fi\n",
    "\n",
    "\n",
    "if [ $(ls $outputDirectory/*fastq| wc -l) -gt 0 ]\n",
    "then\n",
    "    for f in $outputDirectory/*fastq;\n",
    "    do\n",
    "        dirname=${f%_S[0-9][0-9]*_L0*.fastq}\n",
    "        dirname=${dirname%_S[0-9][0-9]*.fastq}\n",
    "        dirname=${dirname%_S[0-9]*.fastq}\n",
    "        dirname=${dirname%.fastq} # for older data without lane number\n",
    "        if [ ! -d $dirname ]\n",
    "        then\n",
    "            mkdir $dirname\n",
    "        fi\n",
    "        mv $f $dirname\n",
    "    done\n",
    "fi\n",
    "\n",
    "\n",
    "if [ $(ls $outputDirectory/*sra| wc -l) -gt 0 ]\n",
    "then\n",
    "    for f in $outputDirectory/*sra;\n",
    "    do\n",
    "        dirname=${f%_S[0-9][0-9]*_L0*.sra}\n",
    "        dirname=${dirname%_S[0-9][0-9]*.sra}\n",
    "        dirname=${dirname%_S[0-9]*.sra}\n",
    "        dirname=${dirname%.sra} # for older data without lane number\n",
    "        if [ ! -d $dirname ]\n",
    "        then\n",
    "            mkdir $dirname\n",
    "        fi\n",
    "        mv $f $dirname\n",
    "    done\n",
    "fi\n",
    "\n",
    "### decompress files\n",
    "\n",
    "echo \"Decompressing raw data (fastq.gz files)\"\n",
    "\n",
    "# find fastq.gz files\n",
    "compressedDirs=()\n",
    "echo $outputDirectory\n",
    "compressedPaths_1=( $(find $outputDirectory -path \"*fastq.gz\" -type f) )\n",
    "compressedPaths_2=( $(find $outputDirectory -path \"*sra\" -type f) )\n",
    "compressedPaths_3=( $(find $outputDirectory -path \"*fastq\" -type f) )\n",
    "compressedPaths=( ${compressedPaths_1[@]} ${compressedPaths_2[@]} ${compressedPaths_3[@]})\n",
    "for f in ${compressedPaths[*]}\n",
    "do\n",
    "    # remove file name to get sample directory\n",
    "    compressedDir=${f%/*gz}\n",
    "    compressedDir=${compressedDir%/*sra}\n",
    "    compressedDir=${compressedDir%/*fastq}\n",
    "    # append sample directory to list\n",
    "    compressedDirs[${#compressedDirs[*]}]=$compressedDir\n",
    "done\n",
    "\n",
    "# filter out duplicated directories\n",
    "sampleDirs=$(echo \"${compressedDirs[@]}\" | tr ' ' '\\n' | sort -u | tr '\\n' ' ')\n",
    "\n",
    "\n",
    "for sample_dir in ${sampleDirs[*]}\n",
    "do\n",
    "    bname=`basename $sample_dir`\n",
    "    echo \"Decompressing $bname\"\n",
    "\n",
    "    # If there is exactly one fastq file, use that; otherwise...\n",
    "    if $paired\n",
    "    then\n",
    "        # for paired end sequence\n",
    "        if ls $sample_dir/*.fastq &> /dev/null; then\n",
    "            if [ `ls -l $sample_dir/*.fastq | wc -l` -ne 2 ]; then\n",
    "                # for first set of reads\n",
    "                cat $sample_dir/*R1*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                rm $sample_dir/*R1*.fastq\n",
    "                mv $sample_dir/$bname.fastq_joined $sample_dir/${bname}_R1.fastq\n",
    "                # for second set of reads\n",
    "                cat $sample_dir/*R2*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                rm $sample_dir/*R2*.fastq\n",
    "                mv $sample_dir/$bname.fastq_joined $sample_dir/${bname}_R2.fastq\n",
    "            fi  \n",
    "            # else, only one .fastq; will be used.\n",
    "        else\n",
    "            # If there are any .sra files, dump to .fastq\n",
    "            if ls $sample_dir/*.sra &> /dev/null; then \n",
    "                for sra in $sample_dir/*.sra\n",
    "                    do  \n",
    "                        current_dir=`pwd`\n",
    "                        # CD in so that fastq-dump works correctly\n",
    "                        cd $sample_dir\n",
    "                        fastq-dump $sra\n",
    "                        rm $sra\n",
    "                        cd $current_dir\n",
    "                    done\n",
    "                # Then compile all the .fastq\n",
    "                if [ `ls -l $sample_dir/*.fastq | wc -l` -ne 2 ]; then\n",
    "                    # for first set of reads\n",
    "                    cat $sample_dir/*R1*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                    rm $sample_dir/*R1*.fastq\n",
    "                    mv $sample_dir/$bname.fastq_joined $sample_dir/${bname}_R1.fastq\n",
    "                    # for second set of reads\n",
    "                    cat $sample_dir/*R2*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                    rm $sample_dir/*R2*.fastq\n",
    "                    mv $sample_dir/$bname.fastq_joined $sample_dir/${bname}_R2.fastq\n",
    "                else\n",
    "                    # Rename singular dumped sra file\n",
    "                    mv $sample_dir/*R1*.fastq $sample_dir/${bname}_R1.fastq\n",
    "                    mv $sample_dir/*R2*.fastq $sample_dir/${bname}_R2.fastq\n",
    "                fi  \n",
    "            fi  \n",
    "            # Make single file, unzipping simultaneously if they are zipped\n",
    "            if ls $sample_dir/*.gz &> /dev/null; then \n",
    "                # for first set of reads\n",
    "                zcat $sample_dir/*R1*.gz > $sample_dir/${bname}_R1.fastq\n",
    "                # for second set of reads\n",
    "                zcat $sample_dir/*R2*.gz > $sample_dir/${bname}_R2.fastq\n",
    "            fi  \n",
    "        fi  \n",
    "    else\n",
    "        # for single end sequencing\n",
    "        if ls $sample_dir/*.fastq &> /dev/null; then\n",
    "            if [ `ls -l $sample_dir/*.fastq | wc -l` -ne 1 ]; then\n",
    "                cat $sample_dir/*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                rm $sample_dir/*.fastq\n",
    "                mv $sample_dir/$bname.fastq_joined $sample_dir/$bname.fastq\n",
    "            fi  \n",
    "            # else, only one .fastq; will be used.\n",
    "        else\n",
    "            # If there are any .sra files, dump to .fastq\n",
    "            if ls $sample_dir/*.sra &> /dev/null; then \n",
    "                for sra in $sample_dir/*.sra\n",
    "                    do  \n",
    "                        current_dir=`pwd`\n",
    "                        # CD in so that fastq-dump works correctly\n",
    "                        cd $sample_dir\n",
    "                        fastq-dump $sra\n",
    "                        rm $sra\n",
    "                        cd $current_dir\n",
    "                    done\n",
    "                # Then compile all the .fastq\n",
    "                if [ `ls -l $sample_dir/*.fastq | wc -l` -ne 1 ]; then\n",
    "                    cat $sample_dir/*.fastq > $sample_dir/$bname.fastq_joined\n",
    "                    rm $sample_dir/*.fastq\n",
    "                    mv $sample_dir/$bname.fastq_joined $sample_dir/$bname.fastq\n",
    "                else\n",
    "                    # Rename singular dumped sra file\n",
    "                    mv $sample_dir/*.fastq $sample_dir/$bname.fastq\n",
    "                fi  \n",
    "            fi  \n",
    "            # Make single file, unzipping simultaneously if they are zipped\n",
    "            if ls $sample_dir/*.gz &> /dev/null; then \n",
    "                zcat $sample_dir/*.gz > $sample_dir/$bname.fastq\n",
    "            fi  \n",
    "        fi  \n",
    "    fi\n",
    "done\n",
    "\n",
    "# create output directories\n",
    "\n",
    "# make directory for tag directories on Glassome\n",
    "if [ ! -d $glassomeOutputDirectory/tag_directories ]\n",
    "then\n",
    "    mkdir -p $glassomeOutputDirectory/tag_directories\n",
    "fi\n",
    "\n",
    "# make directory for log files on Glassome\n",
    "if [ ! -d $glassomeOutputDirectory/log_files ]\n",
    "then\n",
    "    mkdir -p $glassomeOutputDirectory/log_files\n",
    "fi\n",
    "\n",
    "if [ ! -d $outputDirectory/qsub_scripts ]\n",
    "then\n",
    "    mkdir $outputDirectory/qsub_scripts\n",
    "else\n",
    "    # delete existing scripts\n",
    "    if [ $(ls $outputDirectory/qsub_scripts/* |wc -l) -ne 0 ]\n",
    "    then\n",
    "        rm $outputDirectory/qsub_scripts/*\n",
    "    fi\n",
    "fi\n",
    "\n",
    "# make directory for sam files\n",
    "if [ ! -d $outputDirectory/sam_files ]\n",
    "then\n",
    "    mkdir $outputDirectory/sam_files\n",
    "fi\n",
    "\n",
    "# make directory for tag directories on tscc\n",
    "if [ ! -d $outputDirectory/tag_directories ]\n",
    "then\n",
    "    mkdir $outputDirectory/tag_directories\n",
    "fi\n",
    "\n",
    "# make directory for log files on tscc\n",
    "if [ ! -d $outputDirectory/log_files ]\n",
    "then\n",
    "    mkdir $outputDirectory/log_files\n",
    "fi\n",
    "\n",
    "# make scratch directory for pbc calculation\n",
    "#if [ ! -d $outputDirectory/pbc ]\n",
    "#then\n",
    "#    mkdir $outputDirectory/pbc\n",
    "#fi\n",
    "\n",
    "### generate qsub scripts ###\n",
    "\n",
    "# generate a UUID for this set of jobs\n",
    "uuid=$[ 1 + $[ RANDOM % 10000 ]] # generate a random number between 0 and 10000\n",
    "uuid=${USER}_${uuid}\n",
    "\n",
    "# find directory where script is located\n",
    "codebase=$( cd \"$( dirname \"${BASH_SOURCE[0]}\" )\" && pwd ) \n",
    "\n",
    "# generate script to map fastq to genome\n",
    "\n",
    "# find all fastq files\n",
    "echo \"generating scripts\"\n",
    "for sampleDir in ${sampleDirs[*]}\n",
    "    do\n",
    "    if $paired\n",
    "    then\n",
    "        fastqFiles=\"$(readlink -fm $sampleDir/*R1*fastq) $(readlink -fm $sampleDir/*R2*fastq)\"\n",
    "        fastqFile=$(readlink -fm $sampleDir/*R1*fastq)\n",
    "        currentDirectory=${fastqFile%/*R1*fastq}\n",
    "        sampleName=${fastqFile%_R1.fastq}\n",
    "    else\n",
    "        fastqFile=$(readlink -fm $sampleDir/*fastq)\n",
    "        fastqFiles=$fastqFile\n",
    "        currentDirectory=${fastqFile%/*fastq}\n",
    "        sampleName=${fastqFile%.fastq}\n",
    "    fi\n",
    "    sampleName=${sampleName##/*/} # remove preceding file path\n",
    "    sampleName=${sampleName#Sample_} # remove \"Sample_\" from file names\"\n",
    "\n",
    "    samName=\"\"\n",
    "    logName=\"\"\n",
    "\n",
    "    # map file\n",
    "    ### ChIP-seq ###\n",
    "    if [ $experimentType == \"chip\" ]\n",
    "    then\n",
    "        samName=\"${sampleName}.${genome}.bowtie2.sam\" # change extension to sam\n",
    "        logName=\"${sampleName}.${genome}.bowtie2.log\" # remove path preceding file name\n",
    "        if [ -f $outputDirectory/log_files/$logName ]\n",
    "        then\n",
    "            rm $outputDirectory/log_files/$logName\n",
    "        fi\n",
    "\n",
    "        # execute bowtie\n",
    "        if $paired\n",
    "        then\n",
    "            fastqFiles=\"-1 ${fastqFiles/ / -2 }\"\n",
    "        fi\n",
    "        command=\"$bowtie_path/bowtie2 \\\n",
    "-p 8 \\\n",
    "-x $bowtie_index_path/$genome \\\n",
    "$fastqFiles \\\n",
    "> $outputDirectory/sam_files/$samName \\\n",
    "2> $outputDirectory/log_files/$logName \\n\"\n",
    "\n",
    "        # create tag directory\n",
    "        if ! $map_only\n",
    "        then\n",
    "            command+=\"$homer_path/makeTagDirectory \\\n",
    "$outputDirectory/tag_directories/$sampleName \\\n",
    "-genome $genome \\\n",
    "-checkGC $outputDirectory/sam_files/$samName \"\n",
    "            if $paired\n",
    "            then\n",
    "                command+=\"-sspe \"\n",
    "            fi\n",
    "            command+=\"-format sam \\n\"\n",
    "        fi\n",
    "\n",
    "    ### RNA-seq ###\n",
    "    elif [ $experimentType == \"rna\" ]\n",
    "    then\n",
    "        samName=\"${sampleName}.${genome}.star.sam\" # change extension to sam\n",
    "        logName=\"${sampleName}.${genome}.star.log\" # remove path preceding file name\n",
    "        if [ -f $outputDirectory/log_files/$logName ]\n",
    "        then\n",
    "            rm $outputDirectory/log_files/$logName\n",
    "        fi\n",
    "        # execute star\n",
    "        command=\"$star_path/STAR \\\n",
    "--genomeDir $star_path/genomes/$genome \\\n",
    "--readFilesIn $fastqFiles \\\n",
    "--outFileNamePrefix $currentDirectory/ \\\n",
    "--runThreadN 8\\n\"\n",
    "        # rename aligned file\n",
    "        command+=\"mv $currentDirectory/Aligned.out.sam \\\n",
    "$outputDirectory/sam_files/$samName\\n\"\n",
    "        # rename log file\n",
    "        command+=\"mv $currentDirectory/Log.final.out \\\n",
    "$outputDirectory/log_files/$logName\\n\"\n",
    "        # create tag directory\n",
    "        if ! $map_only\n",
    "        then\n",
    "            command+=\"$homer_path/makeTagDirectory \\\n",
    "$outputDirectory/tag_directories/${sampleName} \\\n",
    "-genome $genome \\\n",
    "-checkGC $outputDirectory/sam_files/$samName \"\n",
    "            if $paired\n",
    "            then\n",
    "                command+=\"-sspe \"\n",
    "            fi\n",
    "            if $flip         \n",
    "            then\n",
    "                command+=\"-format sam -flip\\n\"\n",
    "            else\n",
    "                command+=\"-format sam\\n\"\n",
    "            fi\n",
    "        fi\n",
    "    ### ATAC-seq ###\n",
    "    elif [ $experimentType == \"atac\" ]\n",
    "    then\n",
    "        samName=\"${sampleName}.${genome}.bowtie2.sam\" # change extension to sam\n",
    "        logName=\"${sampleName}.${genome}.bowtie2.log\" # remove path preceding file name\n",
    "        if [ -f $outputDirectory/log_files/$logName ]\n",
    "        then\n",
    "            rm $outputDirectory/log_files/$logName\n",
    "        fi\n",
    "        # execute bowtie\n",
    "        if $paired\n",
    "        then\n",
    "            fastqFiles=\"-1 ${fastqFiles/ / -2 }\"\n",
    "        fi\n",
    "        command=\"$bowtie_path/bowtie2 \\\n",
    "-p 8 \\\n",
    "-x $bowtie_index_path/$genome \\\n",
    "$fastqFiles \\\n",
    "> $outputDirectory/sam_files/$samName \\\n",
    "2> $outputDirectory/log_files/$logName \\n\"\n",
    "        # create tag directory\n",
    "        if ! $map_only\n",
    "        then\n",
    "        command+=\"$homer_path/makeTagDirectory \\\n",
    "$outputDirectory/tag_directories/${sampleName}_with_M \\\n",
    "-genome $genome \\\n",
    "-checkGC $outputDirectory/sam_files/$samName \"\n",
    "        if $paired\n",
    "        then\n",
    "            command+=\"-sspe \"\n",
    "        fi\n",
    "        command+=\"-format sam\\n\"\n",
    "        # remove contaminating tags from chromosome M\n",
    "        command+=\"rm $outputDirectory/tag_directories/${sampleName}_with_M/chrM.tags.tsv\\n\"\n",
    "        # remake tag directory\n",
    "        command+=\"$homer_path/makeTagDirectory \\\n",
    "$outputDirectory/tag_directories/${sampleName} -d \\\n",
    "$outputDirectory/tag_directories/${sampleName}_with_M\\n\"\n",
    "        # copy original tag info file\n",
    "        command+=\"mv \\\n",
    "$outputDirectory/tag_directories/${sampleName}_with_M/tagInfo.txt \\\n",
    "$outputDirectory/tag_directories/${sampleName}/tagInfo_with_M.txt\\n\"\n",
    "        # remove original tag directory\n",
    "        command+=\"rm -rf $outputDirectory/tag_directories/${sampleName}_with_M\\n\"\n",
    "        fi\n",
    "\n",
    "    else\n",
    "        echo \"Error! valid choices for experiment type are atac, chip or rna\"\n",
    "        exit 1\n",
    "    fi\n",
    "\n",
    "    # calculate PBC coefficient\n",
    "\n",
    "#    if [ \"$experimentType\" == \"atac\" ] || [ \"$experimentType\" == \"chip\" ]\n",
    "#    then\n",
    "#        uniqueFile=$outputDirectory/pbc/${sampleName}.unique.bam\n",
    "#        sortedFile=$outputDirectory/pbc/${sampleName}.sorted\n",
    "#        pileupFile=$outputDirectory/pbc/${sampleName}.pileup\n",
    "#\n",
    "#        command+=\"samtools view -Sbq 1 $outputDirectory/sam_files/$samName > \\\n",
    "#    $uniqueFile\\n\"\n",
    "#        command+=\"samtools sort $uniqueFile $sortedFile\\n\"\n",
    "#        command+=\"samtools mpileup ${sortedFile}.bam > $pileupFile\\n\"\n",
    "#        command+=\"PBC=\\$(awk 'BEGIN {N1=0;ND=0} {if(\\$4==1){N1+=1} ND+=1} END{print N1/ND}' ${pileupFile})\\n\"\n",
    "#        command+=\"echo -e \\\"PBC    \\$PBC\\\" >>$outputDirectory/log_files/$logName\\n\" #\"\n",
    "#    fi\n",
    "\n",
    "    \n",
    "    # copy files to Glassome scratch directory\n",
    "    # copy log file\n",
    "    command+=\"cp $outputDirectory/log_files/$logName \\\n",
    "$glassomeOutputDirectory/log_files/\\n\"\n",
    "    if ! $map_only\n",
    "    then\n",
    "        # copy log file to tag directory\n",
    "        command+=\"cp $outputDirectory/log_files/$logName \\\n",
    "$outputDirectory/tag_directories/$sampleName/\\n\"\n",
    "        command+=\"cp -r $outputDirectory/tag_directories/$sampleName \\\n",
    "$glassomeOutputDirectory/tag_directories\\n\"\n",
    "    fi\n",
    "\n",
    "    if $copy_sam\n",
    "    then\n",
    "        command+=\"cp $outputDirectory/sam_files/$samName \\\n",
    "${glassomeOutputDirectory}\\n\"\n",
    "    fi\n",
    "\n",
    "    \n",
    "\n",
    "\n",
    "    # create qsub script\n",
    "    if $no_emails\n",
    "    then\n",
    "    echo -e \"#!/bin/bash\n",
    "#PBS -q hotel\n",
    "#PBS -N ${sampleName}\n",
    "#PBS -l nodes=1:ppn=8\n",
    "#PBS -l walltime=2:00:00\n",
    "#PBS -o $outputDirectory/qsub_scripts/${sampleName}_torque_output.txt\n",
    "#PBS -e $outputDirectory/qsub_scripts/${sampleName}_torque_error.txt\n",
    "#PBS -m n \n",
    "#PBS -V\n",
    "#PBS -A glass-group\n",
    "$command\" > $outputDirectory/qsub_scripts/${sampleName}.torque.sh\n",
    "    else\n",
    "    echo -e \"#!/bin/bash\n",
    "#PBS -q hotel\n",
    "#PBS -N ${sampleName}\n",
    "#PBS -l nodes=1:ppn=8\n",
    "#PBS -l walltime=2:00:00\n",
    "#PBS -o $outputDirectory/qsub_scripts/${sampleName}_torque_output.txt\n",
    "#PBS -e $outputDirectory/qsub_scripts/${sampleName}_torque_error.txt\n",
    "#PBS -M $email\n",
    "#PBS -V\n",
    "#PBS -m abe\n",
    "#PBS -A glass-group\n",
    "$command\" > $outputDirectory/qsub_scripts/${sampleName}.torque.sh\n",
    "    fi\n",
    "    \n",
    "\n",
    "    # submit script\n",
    "    if ! $testing\n",
    "        then\n",
    "        echo \"Submitting job for $sampleName\"\n",
    "        qsub $outputDirectory/qsub_scripts/${sampleName}.torque.sh\n",
    "        chmod a+r $outputDirectory/qsub_scripts/*\n",
    "    fi\n",
    "done\n",
    "\n",
    "### LICENSE STATEMENT ###\n",
    "# Copyright (c) 2015, Jenhan Tao \n",
    "# All rights reserved. \n",
    "# \n",
    "# Redistribution and use in source and binary forms, with or without \n",
    "# modification, are permitted provided that the following conditions are met: \n",
    "#\n",
    "# * Redistributions of source code must retain the above copyright notice, \n",
    "#   this list of conditions and the following disclaimer. \n",
    "# * Redistributions in binary form must reproduce the above copyright \n",
    "#   notice, this list of conditions and the following disclaimer in the \n",
    "#   documentation and/or other materials provided with the distribution. \n",
    "# * Neither the name of UC San Diego nor the names of its contributors may \n",
    "#   be used to endorse or promote products derived from this software \n",
    "#   without specific prior written permission. \n",
    "#\n",
    "# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS \"AS IS\" \n",
    "# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE \n",
    "# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE \n",
    "# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE \n",
    "# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR \n",
    "# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF \n",
    "# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS \n",
    "# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN \n",
    "# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) \n",
    "# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE \n",
    "# POSSIBILITY OF SUCH DAMAGE. \n",
    "###\n",
    "###############################################################################"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
