{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import glob\n",
    "import re\n",
    "import subprocess"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02', 'DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02', 'DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23', 'DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23', 'DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02', 'DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23', 'DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23', 'DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02']\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02 /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722438855\n",
      "\tEstimated average read density = 0.003353 per bp\n",
      "\tTotal Tags = 9127543.0\n",
      "\tTotal Positions = 7081108\n",
      "\tAverage tag length = 47.7\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.289\n",
      "\tFragment Length Estimate: 147\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 3.3\n",
      "\t\tDiff strand fold enrichment: 3.3\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02 /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722439920\n",
      "\tEstimated average read density = 0.003351 per bp\n",
      "\tTotal Tags = 9123334.0\n",
      "\tTotal Positions = 6819896\n",
      "\tAverage tag length = 48.0\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.338\n",
      "\tFragment Length Estimate: 149\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 3.0\n",
      "\t\tDiff strand fold enrichment: 3.0\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23 /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722435497\n",
      "\tEstimated average read density = 0.001460 per bp\n",
      "\tTotal Tags = 3974667.0\n",
      "\tTotal Positions = 2665677\n",
      "\tAverage tag length = 48.2\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.491\n",
      "\tFragment Length Estimate: 118\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 4.1\n",
      "\t\tDiff strand fold enrichment: 4.2\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23 /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722378552\n",
      "\tEstimated average read density = 0.000994 per bp\n",
      "\tTotal Tags = 2706558.0\n",
      "\tTotal Positions = 1191896\n",
      "\tAverage tag length = 47.3\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 2.271\n",
      "\tFragment Length Estimate: 117\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 4.2\n",
      "\t\tDiff strand fold enrichment: 4.4\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02 /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722442133\n",
      "\tEstimated average read density = 0.002859 per bp\n",
      "\tTotal Tags = 7784474.0\n",
      "\tTotal Positions = 5130241\n",
      "\tAverage tag length = 47.4\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.517\n",
      "\tFragment Length Estimate: 280\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 2.5\n",
      "\t\tDiff strand fold enrichment: 2.6\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23 /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Nx_PND16_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722408481\n",
      "\tEstimated average read density = 0.001387 per bp\n",
      "\tTotal Tags = 3774950.0\n",
      "\tTotal Positions = 1491777\n",
      "\tAverage tag length = 48.0\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 2.531\n",
      "\tFragment Length Estimate: 119\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 3.8\n",
      "\t\tDiff strand fold enrichment: 3.9\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23 /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_MGretina_RNA_Tn5_Hprx_PND15_ES_16_09_23.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722425674\n",
      "\tEstimated average read density = 0.001427 per bp\n",
      "\tTotal Tags = 3886185.0\n",
      "\tTotal Positions = 2626225\n",
      "\tAverage tag length = 48.5\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.480\n",
      "\tFragment Length Estimate: 121\n",
      "\tPeak Width Estimate: 7\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 3.9\n",
      "\t\tDiff strand fold enrichment: 4.0\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n",
      "makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02 /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tWill parse file: /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\n",
      "\tCreating directory: /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02 and removing existing *.tags.tsv\n",
      "\n",
      "\tReading alignment file /data/scratch/iholtman/DBA_Retina/DBA_EndoRet_RNA_Tn5_Hprx_ES_16_09_02.dba2j.star_shifted_from_DBA2J.sam\n",
      "\tGuessing (by default) that your alignment file is sam format\n",
      "\n",
      "\tOptimizing tag files...\n",
      "\tEstimated genome size = 2722432910\n",
      "\tEstimated average read density = 0.002835 per bp\n",
      "\tTotal Tags = 7718634.0\n",
      "\tTotal Positions = 5568876\n",
      "\tAverage tag length = 47.2\n",
      "\tMedian tags per position = 1 (ideal: 1)\n",
      "\tAverage tags per position = 1.386\n",
      "\tFragment Length Estimate: 263\n",
      "\tPeak Width Estimate: 0\n",
      "\t\t!!! No reliable estimate for peak size\n",
      "\t\tSetting Peak width estimate to be equal to fragment length estimate\n",
      "\tAutocorrelation quality control metrics:\n",
      "\t\tSame strand fold enrichment: 2.9\n",
      "\t\tDiff strand fold enrichment: 3.0\n",
      "\t\tSame / Diff fold enrichment: 1.0\n",
      "\n",
      "\t\tGuessing sample is ChIP-Seq or unstranded RNA-Seq - autocorrelation looks good.\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "files = glob.glob('/data/scratch/iholtman/DBA_Retina/*.*star_shifted_from_DBA2J.sam')\n",
    "#print(files)\n",
    "\n",
    "shortened = []\n",
    "new_shortened = []\n",
    "\n",
    "for file in files: \n",
    "    shortened.append(re.sub('/data/scratch/iholtman/DBA_Retina/', '', file))\n",
    "\n",
    "for file in shortened: \n",
    "    new_shortened.append(re.sub('.dba2j.star_shifted_from_DBA2J.sam', '', file))\n",
    "\n",
    "print(new_shortened)\n",
    "\n",
    "\n",
    "for file in new_shortened: \n",
    "    code = 'makeTagDirectory /data/scratch/iholtman/Project_Retina_hyperoxia/DBA/' + file + ' ' + '/data/scratch/iholtman/DBA_Retina/' + file + '.dba2j.star_shifted_from_DBA2J.sam'\n",
    "    print(code)\n",
    "    print(subprocess.check_output(code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python [conda env:IRH-python3]",
   "language": "python",
   "name": "conda-env-IRH-python3-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
