{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import re\n",
    "import math, random\n",
    "import numpy,scipy,pandas\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn\n",
    "import itertools\n",
    "import subprocess\n",
    "\n",
    "# from pgtools import myplots\n",
    "from pgtools import toolbox, myplots\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('/home/dskola/workspace/expression_modeling/')\n",
    "import rnatools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz'], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz'], [], [], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz'], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz'], ['/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz'], ['/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H1_DGO_15_10_30_S64_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E3_DGO_15_10_30_S69_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E1_DGO_15_10_30_S67_L006_R1_001.fastq'], ['/data/archive/15-12-27-glass/Project_David/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq'], ['/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04_S51_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04_S54_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_lL34_TGFBR1antagonist_A8301_1uM_24h_rep1_160201_DGO_16_04_04_S47_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_rep1_160201_DGO_16_04_04_S46_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001_2.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_rep1_160201_DGO_16_04_04_S45_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04_S55_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04_S50_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFB1_160201_DGO_16_04_04_S56_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001.fastq.gz'], ['/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_Salk_151209_3_2_DGO_16_05_02_GTGAAA_S7_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_IL34_Salk_1512_09_5_1_DGO_16_05_02_GTTTCG_S9_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_IL34_Salk_1512093_1_DGO_16_05_02_GTCCGC_S6_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_GMCSF_StemPro_IL34_Salk_151209_4_1_DGO_16_05_02_GTGGCC_S8_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz'], ['/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz'], ['/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq'], ['/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq'], ['/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz']]\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "import os\n",
    "    \n",
    "fastq_dir = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/fastq_directories.csv')\n",
    "#print(fastq_dir[\"Dirs\"])\n",
    "\n",
    "files = []\n",
    "for dir in fastq_dir[\"Dirs\"]:\n",
    "    #os.listdir(dir + \"*.fastq\" )\n",
    "    dir_fastq = glob.glob(dir +\"*.fastq\")\n",
    "    dir_fastq_gz = glob.glob(dir +\"*.fastq.gz\")\n",
    "    files.append(dir_fastq or dir_fastq_gz)\n",
    "print(files)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H1_DGO_15_10_30_S64_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E3_DGO_15_10_30_S69_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E1_DGO_15_10_30_S67_L006_R1_001.fastq', '/data/archive/15-12-27-glass/Project_David/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04_S51_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04_S54_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_lL34_TGFBR1antagonist_A8301_1uM_24h_rep1_160201_DGO_16_04_04_S47_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_rep1_160201_DGO_16_04_04_S46_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001_2.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_rep1_160201_DGO_16_04_04_S45_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04_S55_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04_S50_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFB1_160201_DGO_16_04_04_S56_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_Salk_151209_3_2_DGO_16_05_02_GTGAAA_S7_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_IL34_Salk_1512_09_5_1_DGO_16_05_02_GTTTCG_S9_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_IL34_Salk_1512093_1_DGO_16_05_02_GTCCGC_S6_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_GMCSF_StemPro_IL34_Salk_151209_4_1_DGO_16_05_02_GTGGCC_S8_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz', '/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz']\n"
     ]
    }
   ],
   "source": [
    "### One list with all fastq-files  \n",
    "fastqs = []\n",
    "for file in files:\n",
    "    for fastq in file:    \n",
    "        fastqs.append(fastq)\n",
    "\n",
    "print(fastqs)\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: [/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq, ...]\n",
      "\n",
      "[149 rows x 0 columns]\n"
     ]
    }
   ],
   "source": [
    "folders = pandas.DataFrame([], index=fastqs)\n",
    "folders.index.name='fastq'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "print(folders)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{0, 2, 3, 4, 5, 6, 7, 8, 10, 11, 14, 15, 16, 17, 18, 19, 21}\n",
      "{0, 2, 3, 4, 5, 6, 7, 8, 10, 11, 12, 14, 15, 16, 17, 18, 19, 21, 23, 24, 26, 27, 29, 31}\n",
      "fastq\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz                          5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz     5\n",
      "                                                                                                                                                                                                                ..\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz                                          0\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz                                                             15\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz       17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz        17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz                         17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz                                                                              4\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz                                                                              5\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz                                                                              7\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz                                                                    8\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz                                                                     8\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz                                                                 14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz                                                                 14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz                                                                 18\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz                                                             18\n",
      "/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq                                                                   3\n",
      "/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq                                                                19\n",
      "/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq                                                                  21\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz                                                                             23\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz                                                                                24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz                                                                                26\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz                                                                                27\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz                                                                              29\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz                                                                                      31\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz                                                               23\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz                                                              24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz                                                               24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz                                                             31\n",
      "Name: patients, dtype: int64\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/gpfs/data01/glasslab/home/iholtman/anaconda3/lib/python3.5/site-packages/ipykernel/__main__.py:9: DeprecationWarning: bad escape \\P\n"
     ]
    }
   ],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\P(?P<patient>[0-9]{1,2})\\_\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "    \n",
    "folders['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patients\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "\n",
    "#folders[\"fastq_folder\"] = folders[\"fastq_path\"]\n",
    "print(set(folders[\"patient\"]))\n",
    "print(set(folders[\"patients\"]))\n",
    "print(folders[\"patients\"])\n",
    "\n",
    "#OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "#folders.to_csv(os.path.join(OUTPUT_PATH, 'ALL_fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cat</th>\n",
       "      <th>patient</th>\n",
       "      <th>patientz</th>\n",
       "      <th>patients</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq</th>\n",
       "      <td>False</td>\n",
       "      <td>11</td>\n",
       "      <td>0</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq</th>\n",
       "      <td>False</td>\n",
       "      <td>11</td>\n",
       "      <td>0</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                      cat  patient  patientz  \\\n",
       "fastq                                                                          \n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11         0   \n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11         0   \n",
       "\n",
       "                                                    patients  \n",
       "fastq                                                         \n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...        11  \n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...        11  "
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "folders[folders[\"patient\"] == 11]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['in_vitro'] = [i.lower().find('in_vitro') > -1 for i in folders.index]\n",
    "folders['tgfb'] = [i.lower().find('tgf') > -1 for i in folders.index]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#fastq_mapping = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/human_rna_tag_fastq_mapping.csv', index_col=0)\n",
    "#print(fastq_mapping[\"fastq_path\"])\n",
    "\n",
    "#OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#tag_mapping = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/human_rna_tag_fastq_mapping.csv', index_col=0).fastq_path.to_dict()\n",
    "#print(tag_mapping)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#folders = pandas.DataFrame([], index=tag_mapping)\n",
    "#folders.index.name='Sample'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "#print(folders)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Annotate with patient info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "         Broad_Diagnosis                                          Pathology  \\\n",
      "Patient                                                                       \n",
      "1                  Tumor                                       Glioblastoma   \n",
      "2                  Tumor                       Atypical rhaboid brain tumor   \n",
      "3                  Tumor                                  High grade glioma   \n",
      "4               Epilepsy            Epilepsy - cortical dysplasia, grade IA   \n",
      "5               Epilepsy                      Epilepsy - negative pathology   \n",
      "6                  Tumor             Cerebellar pylotic astrocytoma grade I   \n",
      "7               Epilepsy  Epilepsy - neurocytic ganglioglioma with sever...   \n",
      "8               Epilepsy  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "9                  Tumor              Anaplastic ependymoma - WHO grade III   \n",
      "10                 Tumor              Anaplastic ependymoma - WHO grade III   \n",
      "11              Epilepsy                      Epilepsy - negative pathology   \n",
      "12                 Tumor                       Atypical rhaboid brain tumor   \n",
      "13              Epilepsy                        Epilepsy - ischemic changes   \n",
      "14              Epilepsy      Epilepsy - Focal cortical dysplasia, type IIB   \n",
      "15                 Tumor  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "16                 Tumor     Pilocytic astrocytoma with anaplastic features   \n",
      "17              Epilepsy               Epilepsy - neuronal loss and gliosis   \n",
      "18                 Tumor                Anaplastic medulloblastoma - WHO IV   \n",
      "19              Epilepsy                       Epilepsy - pathology pending   \n",
      "20                 Tumor                              Glioblastoma - WHO IV   \n",
      "21              Epilepsy      Epilepsy - focal cortical dysplasia, type IIA   \n",
      "22                 Tumor                                      Pineoblastoma   \n",
      "23              Epilepsy            Epilepsy - posterior pial angiomatosis    \n",
      "24                 Tumor         Juvenile Pilocytic Atrocytoma, WHO grade 1   \n",
      "25                 Tumor         Juvenile Pilocytic Atrocytoma, WHO grade 1   \n",
      "26                 Tumor  Dysembryoplastic neuroepithelial tumor, WHO gr...   \n",
      "27                 Tumor                         Embryonic choriocarcinoma    \n",
      "28                 Tumor                                         Ependymoma   \n",
      "29              Aneurysm                                           Aneurysm   \n",
      "30                 Tumor                                    Medullablastoma   \n",
      "31                   NaN                                                NaN   \n",
      "\n",
      "                Age  Sex Ethnicity Date (mm-dd-yy)  \\\n",
      "Patient                                              \n",
      "1          9.000000    M         C         3/11/15   \n",
      "2          1.000000    F         C         3/25/15   \n",
      "3          4.000000    M       His          4/6/15   \n",
      "4          0.660000    F         C         4/20/15   \n",
      "5          7.000000    F       His         5/26/15   \n",
      "6          3.000000    F         C         5/28/15   \n",
      "7         13.000000    F         C         5/28/15   \n",
      "8         17.000000    M       His         6/22/15   \n",
      "9         11.000000    M         C         7/20/15   \n",
      "10        13.000000    F       His         8/19/15   \n",
      "11        15.000000    F       His         8/27/15   \n",
      "12         1.166667    M       His        11/25/15   \n",
      "13        13.000000    F    Asian         12/14/15   \n",
      "14         1.000000    F       His         1/12/16   \n",
      "15        11.000000    F         C         1/19/16   \n",
      "16        13.000000    F       His         1/25/16   \n",
      "17         2.000000    F         C          2/1/16   \n",
      "18         5.000000    M       His         2/19/16   \n",
      "19         1.666667    M         C         4/26/16   \n",
      "20         5.000000    M       His         4/27/16   \n",
      "21         9.000000    F         C          6/6/16   \n",
      "22        12.000000    F     Asian         7/29/16   \n",
      "23         4.000000    M       His          8/8/16   \n",
      "24         8.000000    M       His         8/10/16   \n",
      "25         5.000000    M         C         8/15/16   \n",
      "26        16.000000    M       His         8/15/16   \n",
      "27         3.000000    M     AfrAm         8/29/16   \n",
      "28              NaN  NaN       NaN         9/18/16   \n",
      "29        16.000000    F       NaN         9/19/16   \n",
      "30              NaN  NaN       NaN         9/19/16   \n",
      "31              NaN  NaN       NaN         9/27/16   \n",
      "\n",
      "                                         Medication regimen  \n",
      "Patient                                                      \n",
      "1                                                      None  \n",
      "2                            Steroids, Keppra/Phenobarbitol  \n",
      "3                           Temozolomide, Bevaclizumab, XRT  \n",
      "4               Klonopin, Keppra, Trileptal, Phenobarbital   \n",
      "5                                    Keppra, Depakote, Onfi  \n",
      "6                                                 Steroids   \n",
      "7                                                 Phenytoin  \n",
      "8                                Klonopin, Lamictal, Keppra  \n",
      "9                                         Radiation therapy  \n",
      "10                                                     None  \n",
      "11                                                Trileptal  \n",
      "12        Methotrexate, Vincristine, Cisplatin, Cyclopho...  \n",
      "13                              Topamax, Vimpat, Zonisamide  \n",
      "14                            Vimpat, Phenobarbital, Keppra  \n",
      "15                                                  Keppra   \n",
      "16                                                     None  \n",
      "17                                        Clobazam, Topamax  \n",
      "18                                                 Decadron  \n",
      "19                                                   Keppra  \n",
      "20        Temozolomide, Bevaclizumab, XRT, Pablociclib, ...  \n",
      "21                                          Topamax, Vimpat  \n",
      "22                                                 Decadron  \n",
      "23                                           Keppra, Banzal  \n",
      "24                                                 Decadron  \n",
      "25                                                 Decadron  \n",
      "26                                             Lamotrigine   \n",
      "27                                                 Decadron  \n",
      "28                                                      NaN  \n",
      "29                                                      NaN  \n",
      "30                                                      NaN  \n",
      "31                                                      NaN  \n"
     ]
    }
   ],
   "source": [
    "patient_table = pandas.read_csv('/gpfs/data01/glasslab/home/iholtman/Human_microglia_python_tutorial/patient_info.csv', index_col=0)\n",
    "print(patient_table)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                      cat  patient  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        3   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "...                                                   ...      ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        8   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        8   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...   True       10   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        4   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...  False       19   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...  False       21   \n",
      "\n",
      "                                                   fastq_folder  \\\n",
      "fastq                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "...                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          NaN   \n",
      "\n",
      "                                                   Broad_Diagnosis  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "...                                                            ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...        Epilepsy   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...        Epilepsy   \n",
      "\n",
      "                                                                                            Pathology  \\\n",
      "fastq                                                                                                   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "...                                                                                               ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                      Epilepsy - negative pathology   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...     Pilocytic astrocytoma with anaplastic features   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...            Epilepsy - cortical dysplasia, grade IA   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       Epilepsy - pathology pending   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...      Epilepsy - focal cortical dysplasia, type IIA   \n",
      "\n",
      "                                                          Age Sex Ethnicity  \\\n",
      "fastq                                                                         \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "...                                                       ...  ..       ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  17.000000   M       His   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  17.000000   M       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  15.000000   F       His   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  15.000000   F       His   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  13.000000   F       His   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   0.660000   F         C   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...   1.666667   M         C   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...   9.000000   F         C   \n",
      "\n",
      "                                                   Date (mm-dd-yy)  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "...                                                            ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         6/22/15   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         6/22/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         8/27/15   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         8/27/15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/25/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         4/20/15   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         4/26/16   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          6/6/16   \n",
      "\n",
      "                                                                             Medication regimen  \\\n",
      "fastq                                                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "...                                                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                   Klonopin, Lamictal, Keppra   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                   Klonopin, Lamictal, Keppra   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                                    Trileptal   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                                    Trileptal   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                         None   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Klonopin, Keppra, Trileptal, Phenobarbital    \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                                       Keppra   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                              Topamax, Vimpat   \n",
      "\n",
      "                                                   sample_id Region  \\\n",
      "fastq                                                                 \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "...                                                      ...    ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...       NaN    NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...       NaN    NaN   \n",
      "\n",
      "                                                   Tissue resected (in gram)  \\\n",
      "fastq                                                                          \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "...                                                                      ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                       NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       NaN   \n",
      "\n",
      "                                                    Cells isolated (in million)  \\\n",
      "fastq                                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "...                                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                          NaN   \n",
      "\n",
      "                                                   Region Full tissue_type  \\\n",
      "fastq                                                                        \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "...                                                        ...         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN         NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN         NaN   \n",
      "\n",
      "                                                   sample_date  patient_age  \n",
      "fastq                                                                        \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "...                                                        ...          ...  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN          NaN  \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN          NaN  \n",
      "\n",
      "[124 rows x 18 columns]\n"
     ]
    }
   ],
   "source": [
    "annotated_samples = pandas.merge(folders, patient_table, left_on='patient', right_index=True)\n",
    "\n",
    "sample_info = pandas.read_csv('/home/dskola/human/sample_info.csv')\n",
    "sample_info.drop([8,17,21], axis=0, inplace=True)\n",
    "i = annotated_samples.index\n",
    "annotated_samples = annotated_samples.merge(sample_info, how='left', left_on='patient', right_on='Patient')\n",
    "annotated_samples.drop('Patient', axis=1, inplace=True)\n",
    "annotated_samples.index = i\n",
    "print(annotated_samples)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#annotated_samples = pandas.merge(folders, patient_table, left_on='patient', right_index=True)\n",
    "#print(annotated_samples)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "annotated_samples.to_csv(os.path.join(OUTPUT_PATH, 'fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "annotated_samples.to_csv(os.path.join(OUTPUT_PATH, 'ALL_fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Concetated FASTQ-files processing according to GATK best practices "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: [/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz]\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "import os\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/\"\n",
    "files= glob.glob(dir +\"*.fastq.gz\")\n",
    "\n",
    "folders = pandas.DataFrame([], index=files)\n",
    "folders.index.name='fastq'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "print(folders)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                     cat  patient\n",
      "fastq                                                            \n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       16\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       19\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       21\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        2\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        3\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        6\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       10\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       15\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True       17\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        5\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        7\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  True        8\n"
     ]
    }
   ],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "folders['patient'] = patients\n",
    "\n",
    "\n",
    "\n",
    "print(folders)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz 16\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz 19\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz 21\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz 2\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz 3\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz 6\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz 10\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz 15\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz 17\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz 5\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz 7\n",
      "/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz 8\n"
     ]
    }
   ],
   "source": [
    "###There were accidently two P10 fastq files! The first one is the smallest, and hence least important one.  \n",
    "## \n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(index, row['patient'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 232,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' CreateSequenceDictionary R= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta' O= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.dict'\n"
     ]
    }
   ],
   "source": [
    "##Createdictionary #File is generated \n",
    "\n",
    "#http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz is that the same refernence as used here? \n",
    "\n",
    "terminal_process = \"java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' CreateSequenceDictionary R= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta' O= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.dict'\"\n",
    "\n",
    "print(terminal_process) \n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##File is generated already! \n",
    "terminal_process = \"samtools faidx human_g1k_v37.fasta\"\n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Step 1: already happened too! \n",
    "genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/\"\n",
    "terminal_process = \"'STAR --runMode genomeGenerate --genomeDir' genome_dir '--genomeFastaFiles human_g1k_v37.fasta --runThreadN 32'\"\n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P16 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P19 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P21 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P10 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P2 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P3 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P6 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P10 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P15 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P17 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P5 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P7 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P8 --readFilesCommand zcat\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##STEP 2 -first pass Alignment \n",
    "genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ \"\n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = \"STAR --genomeDir \" + genome_dir + \"--readFilesIn \" + index + \" --runThreadN 32 --outFileNamePrefix \" + aligned_dir + \"P\" + str(row['patient']) + \" --readFilesCommand zcat\"\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "    print(terminal_code)\n",
    "    print(\"\\n\")\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 274,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P16SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P16\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P19SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P19\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P21SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P21\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P10SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P2SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P2\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P3SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P3\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P6SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P6\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P10.fastq.gz P10\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P10SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P15.fastq.gz P15\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P15SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P15\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P17.fastq.gz P17\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P17SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P17\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P5.fastq.gz P5\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P5SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P5\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P7.fastq.gz P7\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P7SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P7\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P8.fastq.gz P8\n",
      "STAR --runMode genomeGenerate --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/  --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta     --sjdbFileChrStartEnd /data/scratch/iholtman/human_microglia_genotyping/step2/P8SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step3/P8\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#STEP 3 - update the reference genome - not working! \n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "reference_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ \"\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "\n",
    "    terminal_code = \"STAR --runMode genomeGenerate --genomeDir \" + genome_dir + \" --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "    --sjdbFileChrStartEnd \" + aligned_dir + \"P\" + str(row['patient']) + \"SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix \" \\\n",
    "    + reference_dir + \"P\" + str(row['patient'])\n",
    "                                              \n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "    print(\"\\n\") "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 291,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P16\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P19\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P21\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P2\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P3\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P6\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P10.fastq.gz P10\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P15.fastq.gz P15\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P15\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P17.fastq.gz P17\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P17\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P5.fastq.gz P5\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P5\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P7.fastq.gz P7\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P7\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P8.fastq.gz P8\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P8\n"
     ]
    }
   ],
   "source": [
    "#Make sub dirs for step 3\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    genome_dir=\"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "    terminal_code = \"mkdir \" + genome_dir + \"P\" + str(row['patient'])\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "Nov 29 22:25:52 ..... started STAR run\n",
      "Nov 29 22:25:52 ... starting to generate Genome files\n",
      "Nov 29 22:26:46 ... starting to sort Suffix Array. This may take a long time...\n",
      "Nov 29 22:27:00 ... sorting Suffix Array chunks and saving them to disk...\n",
      "Nov 29 22:34:34 ... loading chunks from disk, packing SA...\n",
      "Nov 29 22:35:29 ... finished generating suffix array\n",
      "Nov 29 22:35:29 ... generating Suffix Array index\n",
      "Nov 29 22:38:27 ... completed Suffix Array index\n",
      "Nov 29 22:38:27 ..... inserting junctions into the genome indices\n",
      "Nov 29 22:39:46 ... writing Genome to disk ...\n",
      "Nov 29 22:39:48 ... writing Suffix Array to disk ...\n",
      "Nov 29 22:40:07 ... writing SAindex to disk\n",
      "Nov 29 22:40:08 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "Nov 29 22:40:08 ..... started STAR run\n",
      "Nov 29 22:40:08 ... starting to generate Genome files\n",
      "Nov 29 22:41:04 ... starting to sort Suffix Array. This may take a long time...\n",
      "Nov 29 22:41:18 ... sorting Suffix Array chunks and saving them to disk...\n",
      "Nov 29 22:49:11 ... loading chunks from disk, packing SA...\n",
      "Nov 29 22:50:06 ... finished generating suffix array\n",
      "Nov 29 22:50:06 ... generating Suffix Array index\n",
      "Nov 29 22:53:03 ... completed Suffix Array index\n",
      "Nov 29 22:53:04 ..... inserting junctions into the genome indices\n",
      "Nov 29 22:54:27 ... writing Genome to disk ...\n",
      "Nov 29 22:54:30 ... writing Suffix Array to disk ...\n",
      "Nov 29 22:54:59 ... writing SAindex to disk\n",
      "Nov 29 22:55:00 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n"
     ]
    }
   ],
   "source": [
    "#STEP 3 - update the reference genome\n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "genome_dir=\"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "\n",
    "    terminal_code = \"STAR --runMode genomeGenerate --genomeDir \" + genome_dir + \"P\" + str(row['patient']) + \" --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "    --sjdbFileChrStartEnd \" + aligned_dir + \"P\" + str(row['patient']) + \"SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix \" \\\n",
    "    + \"P\" + str(row['patient'])\n",
    "                                              \n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")                                       \n",
    "                  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/data/scratch/iholtman/human_microglia_genotyping/step3/P8\n"
     ]
    }
   ],
   "source": [
    "print(reference_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "Nov 30 09:48:53 ..... started STAR run\n",
      "Nov 30 09:48:53 ..... loading genome\n",
      "Nov 30 09:49:05 ..... started mapping\n",
      "Nov 30 09:49:55 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "Nov 30 09:49:55 ..... started STAR run\n",
      "Nov 30 09:49:55 ..... loading genome\n",
      "Nov 30 09:50:07 ..... started mapping\n",
      "Nov 30 09:51:00 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "Nov 30 09:51:01 ..... started STAR run\n",
      "Nov 30 09:51:01 ..... loading genome\n",
      "Nov 30 09:51:30 ..... started mapping\n",
      "Nov 30 09:52:28 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "Nov 30 09:52:28 ..... started STAR run\n",
      "Nov 30 09:52:28 ..... loading genome\n",
      "Nov 30 09:52:55 ..... started mapping\n",
      "Nov 30 09:53:37 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "Nov 30 09:53:37 ..... started STAR run\n",
      "Nov 30 09:53:37 ..... loading genome\n",
      "Nov 30 09:54:07 ..... started mapping\n",
      "Nov 30 09:54:56 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "Nov 30 09:54:56 ..... started STAR run\n",
      "Nov 30 09:54:56 ..... loading genome\n",
      "Nov 30 09:55:25 ..... started mapping\n",
      "Nov 30 09:56:10 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "Nov 30 09:56:10 ..... started STAR run\n",
      "Nov 30 09:56:10 ..... loading genome\n",
      "Nov 30 09:56:39 ..... started mapping\n",
      "Nov 30 09:57:28 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "Nov 30 09:57:28 ..... started STAR run\n",
      "Nov 30 09:57:29 ..... loading genome\n",
      "Nov 30 09:57:56 ..... started mapping\n",
      "Nov 30 10:00:21 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "Nov 30 10:00:21 ..... started STAR run\n",
      "Nov 30 10:00:21 ..... loading genome\n",
      "Nov 30 10:00:50 ..... started mapping\n",
      "Nov 30 10:03:34 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "Nov 30 10:03:34 ..... started STAR run\n",
      "Nov 30 10:03:34 ..... loading genome\n",
      "Nov 30 10:04:02 ..... started mapping\n",
      "Nov 30 10:13:25 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "Nov 30 10:13:25 ..... started STAR run\n",
      "Nov 30 10:13:25 ..... loading genome\n",
      "Nov 30 10:13:51 ..... started mapping\n",
      "Nov 30 10:17:57 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "Nov 30 10:17:57 ..... started STAR run\n",
      "Nov 30 10:17:57 ..... loading genome\n",
      "Nov 30 10:18:24 ..... started mapping\n",
      "Nov 30 10:21:51 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "Nov 30 10:21:51 ..... started STAR run\n",
      "Nov 30 10:21:51 ..... loading genome\n",
      "Nov 30 10:22:08 ..... started mapping\n",
      "Nov 30 10:26:37 ..... finished successfully\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#####STEP 4 realign with updated reference genome \n",
    "final_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step4/\"\n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "reference_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "#genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ \"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    reference_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step3/\" + \"P\" + str(row['patient'])\n",
    "    terminal_code = \"STAR --genomeDir \" + reference_dir + \" --readFilesIn \" + index \\\n",
    "    + \" --runThreadN 32 --outFileNamePrefix \" + final_dir + \"P\" + str(row['patient']) + \" --readFilesCommand zcat\"\n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    \n",
    "    print(\"\\n\")\n",
    "    \n",
    "    \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P16Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P16rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 10:44:54 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P16Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P16rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 10:44:54 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 10:44:54\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 10:44:59\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 6:2,893,992\n",
      "INFO\t2016-11-30 10:45:05\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 16:16,199,846\n",
      "INFO\t2016-11-30 10:45:10\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 20:35,408,705\n",
      "INFO\t2016-11-30 10:45:15\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 4:99,997,870\n",
      "INFO\t2016-11-30 10:45:20\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:    4s.  Last read position: 2:43,451,014\n",
      "INFO\t2016-11-30 10:45:25\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:    4s.  Last read position: 3:151,056,379\n",
      "INFO\t2016-11-30 10:45:31\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 18:48,405,292\n",
      "INFO\t2016-11-30 10:45:37\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    5s.  Last read position: 3:152,176,739\n",
      "INFO\t2016-11-30 10:45:42\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:47s.  Time for last 1,000,000:    4s.  Last read position: 19:6,853,965\n",
      "INFO\t2016-11-30 10:45:46\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:    4s.  Last read position: 7:72,433,454\n",
      "INFO\t2016-11-30 10:45:53\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    6s.  Last read position: 3:195,599,211\n",
      "INFO\t2016-11-30 10:45:58\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    4s.  Last read position: 16:73,093,637\n",
      "INFO\t2016-11-30 10:46:03\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    4s.  Last read position: 17:47,447,015\n",
      "INFO\t2016-11-30 10:46:08\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    5s.  Last read position: 2:24,947,491\n",
      "INFO\t2016-11-30 10:46:13\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:    4s.  Last read position: 14:77,248,034\n",
      "INFO\t2016-11-30 10:46:18\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:23s.  Time for last 1,000,000:    4s.  Last read position: 19:45,976,855\n",
      "INFO\t2016-11-30 10:46:23\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:28s.  Time for last 1,000,000:    4s.  Last read position: 16:74,409,065\n",
      "INFO\t2016-11-30 10:46:27\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    4s.  Last read position: 15:35,652,584\n",
      "INFO\t2016-11-30 10:46:32\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:37s.  Time for last 1,000,000:    4s.  Last read position: 4:39,459,249\n",
      "INFO\t2016-11-30 10:46:38\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:43s.  Time for last 1,000,000:    5s.  Last read position: 14:105,527,261\n",
      "INFO\t2016-11-30 10:46:42\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    4s.  Last read position: 2:32,141,593\n",
      "INFO\t2016-11-30 10:46:47\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:52s.  Time for last 1,000,000:    4s.  Last read position: 2:207,942,781\n",
      "INFO\t2016-11-30 10:46:53\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    5s.  Last read position: 16:11,642,891\n",
      "INFO\t2016-11-30 10:46:58\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    4s.  Last read position: 16:22,511,418\n",
      "INFO\t2016-11-30 10:47:02\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:07s.  Time for last 1,000,000:    4s.  Last read position: 7:7,612,474\n",
      "INFO\t2016-11-30 10:47:11\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    9s.  Last read position: 10:43,062,554\n",
      "INFO\t2016-11-30 10:47:16\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    4s.  Last read position: 11:90,694,591\n",
      "INFO\t2016-11-30 10:47:21\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    4s.  Last read position: 7:126,984,799\n",
      "INFO\t2016-11-30 10:47:26\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    4s.  Last read position: 12:11,026,818\n",
      "INFO\t2016-11-30 10:47:30\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:35s.  Time for last 1,000,000:    4s.  Last read position: 1:146,744,566\n",
      "INFO\t2016-11-30 10:47:35\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:40s.  Time for last 1,000,000:    4s.  Last read position: 3:69,154,060\n",
      "INFO\t2016-11-30 10:47:40\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:45s.  Time for last 1,000,000:    4s.  Last read position: 7:98,015,148\n",
      "INFO\t2016-11-30 10:47:44\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:49s.  Time for last 1,000,000:    4s.  Last read position: 21:44,454,798\n",
      "INFO\t2016-11-30 10:47:49\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:54s.  Time for last 1,000,000:    4s.  Last read position: 14:70,235,520\n",
      "[Wed Nov 30 10:50:16 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.36 minutes.\n",
      "Runtime.totalMemory()=11685330944\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P19Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P19rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 10:50:16 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P19Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P19rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 10:50:16 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 10:50:16\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 10:50:21\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:169,050\n",
      "INFO\t2016-11-30 10:50:26\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 7:128,298,327\n",
      "INFO\t2016-11-30 10:50:31\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    5s.  Last read position: 11:63,978,602\n",
      "INFO\t2016-11-30 10:50:36\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 15:84,983,197\n",
      "INFO\t2016-11-30 10:50:41\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    4s.  Last read position: 1:144,519,671\n",
      "INFO\t2016-11-30 10:50:46\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:    4s.  Last read position: 11:1,494,088\n",
      "INFO\t2016-11-30 10:50:51\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    5s.  Last read position: 10:20,131,310\n",
      "INFO\t2016-11-30 10:50:57\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    6s.  Last read position: 12:105,578,868\n",
      "INFO\t2016-11-30 10:51:02\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    4s.  Last read position: X:3,545,146\n",
      "INFO\t2016-11-30 10:51:07\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:50s.  Time for last 1,000,000:    4s.  Last read position: 6:157,451,022\n",
      "INFO\t2016-11-30 10:51:15\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    7s.  Last read position: 1:3,736,488\n",
      "INFO\t2016-11-30 10:51:20\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    4s.  Last read position: 11:67,944,397\n",
      "INFO\t2016-11-30 10:51:25\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    4s.  Last read position: 9:93,646,342\n",
      "INFO\t2016-11-30 10:51:30\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    5s.  Last read position: 15:34,629,569\n",
      "INFO\t2016-11-30 10:51:35\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:    5s.  Last read position: 1:17,973,819\n",
      "INFO\t2016-11-30 10:51:40\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:23s.  Time for last 1,000,000:    5s.  Last read position: 11:117,059,919\n",
      "INFO\t2016-11-30 10:51:45\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    5s.  Last read position: X:135,955,690\n",
      "INFO\t2016-11-30 10:51:50\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:34s.  Time for last 1,000,000:    4s.  Last read position: 19:40,478,278\n",
      "INFO\t2016-11-30 10:51:55\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    5s.  Last read position: 5:142,653,574\n",
      "INFO\t2016-11-30 10:52:01\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    6s.  Last read position: 12:25,253,022\n",
      "INFO\t2016-11-30 10:52:07\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    5s.  Last read position: 19:6,686,198\n",
      "INFO\t2016-11-30 10:52:12\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    5s.  Last read position: 7:75,943,507\n",
      "INFO\t2016-11-30 10:52:19\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    7s.  Last read position: 9:125,139,027\n",
      "INFO\t2016-11-30 10:52:25\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:08s.  Time for last 1,000,000:    5s.  Last read position: 7:111,670,800\n",
      "INFO\t2016-11-30 10:52:30\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:13s.  Time for last 1,000,000:    5s.  Last read position: 5:14,833,037\n",
      "INFO\t2016-11-30 10:52:39\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:22s.  Time for last 1,000,000:    8s.  Last read position: 10:23,740,542\n",
      "INFO\t2016-11-30 10:52:44\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:28s.  Time for last 1,000,000:    5s.  Last read position: MT:11,274\n",
      "INFO\t2016-11-30 10:52:50\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:33s.  Time for last 1,000,000:    5s.  Last read position: 5:149,434,889\n",
      "INFO\t2016-11-30 10:52:54\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:38s.  Time for last 1,000,000:    4s.  Last read position: 1:664,382\n",
      "INFO\t2016-11-30 10:52:59\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    4s.  Last read position: 10:119,049,008\n",
      "INFO\t2016-11-30 10:53:03\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:47s.  Time for last 1,000,000:    4s.  Last read position: 2:24,011,410\n",
      "INFO\t2016-11-30 10:53:08\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:51s.  Time for last 1,000,000:    4s.  Last read position: 11:73,192,383\n",
      "INFO\t2016-11-30 10:53:13\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:56s.  Time for last 1,000,000:    4s.  Last read position: 19:12,059,246\n",
      "INFO\t2016-11-30 10:53:17\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:03:00s.  Time for last 1,000,000:    4s.  Last read position: 7:30,424,516\n",
      "[Wed Nov 30 10:55:36 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.33 minutes.\n",
      "Runtime.totalMemory()=11686379520\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P21Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P21rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 10:55:36 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P21Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P21rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 10:55:36 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 10:55:37\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 10:55:41\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 4:68,380,085\n",
      "INFO\t2016-11-30 10:55:46\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 1:121,171,222\n",
      "INFO\t2016-11-30 10:55:52\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 15:40,594,117\n",
      "INFO\t2016-11-30 10:55:57\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: X:15,776,274\n",
      "INFO\t2016-11-30 10:56:02\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:    4s.  Last read position: 5:138,861,447\n",
      "INFO\t2016-11-30 10:56:07\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:    5s.  Last read position: 17:21,731,168\n",
      "INFO\t2016-11-30 10:56:12\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    5s.  Last read position: 16:26,143,052\n",
      "INFO\t2016-11-30 10:56:19\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    6s.  Last read position: 12:116,675,395\n",
      "INFO\t2016-11-30 10:56:24\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    5s.  Last read position: 13:52,362,123\n",
      "INFO\t2016-11-30 10:56:29\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    5s.  Last read position: X:100,609,058\n",
      "INFO\t2016-11-30 10:56:36\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    7s.  Last read position: 6:126,243,569\n",
      "INFO\t2016-11-30 10:56:41\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    4s.  Last read position: 11:48,081,696\n",
      "INFO\t2016-11-30 10:56:46\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    4s.  Last read position: 11:74,916,131\n",
      "INFO\t2016-11-30 10:56:51\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:14s.  Time for last 1,000,000:    5s.  Last read position: 1:566,580\n",
      "INFO\t2016-11-30 10:56:56\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    4s.  Last read position: 19:11,489,158\n",
      "INFO\t2016-11-30 10:57:01\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:24s.  Time for last 1,000,000:    4s.  Last read position: 3:143,387,779\n",
      "INFO\t2016-11-30 10:57:07\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    5s.  Last read position: 1:149,391,758\n",
      "INFO\t2016-11-30 10:57:12\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    5s.  Last read position: 9:96,238,607\n",
      "INFO\t2016-11-30 10:57:17\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:40s.  Time for last 1,000,000:    5s.  Last read position: 14:69,256,692\n",
      "INFO\t2016-11-30 10:57:23\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:46s.  Time for last 1,000,000:    6s.  Last read position: 15:42,470,812\n",
      "INFO\t2016-11-30 10:57:28\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:51s.  Time for last 1,000,000:    5s.  Last read position: 18:52,903,280\n",
      "INFO\t2016-11-30 10:57:33\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:56s.  Time for last 1,000,000:    4s.  Last read position: 5:69,777,186\n",
      "INFO\t2016-11-30 10:57:38\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    4s.  Last read position: 10:100,177,422\n",
      "INFO\t2016-11-30 10:57:43\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    4s.  Last read position: 12:78,469,241\n",
      "INFO\t2016-11-30 10:57:48\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 8:130,950,201\n",
      "INFO\t2016-11-30 10:57:53\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    4s.  Last read position: 5:172,513,545\n",
      "INFO\t2016-11-30 10:57:58\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    5s.  Last read position: 12:48,525,114\n",
      "INFO\t2016-11-30 10:58:03\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    4s.  Last read position: 9:33,326,000\n",
      "INFO\t2016-11-30 10:58:08\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    5s.  Last read position: 15:52,884,449\n",
      "INFO\t2016-11-30 10:58:14\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:37s.  Time for last 1,000,000:    6s.  Last read position: 1:114,524,411\n",
      "INFO\t2016-11-30 10:58:19\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    5s.  Last read position: 11:71,938,213\n",
      "INFO\t2016-11-30 10:58:24\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:47s.  Time for last 1,000,000:    5s.  Last read position: 17:8,053,824\n",
      "INFO\t2016-11-30 10:58:31\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:54s.  Time for last 1,000,000:    6s.  Last read position: 12:53,886,055\n",
      "INFO\t2016-11-30 10:58:36\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:59s.  Time for last 1,000,000:    4s.  Last read position: 3:65,853,309\n",
      "INFO\t2016-11-30 10:58:41\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:04s.  Time for last 1,000,000:    5s.  Last read position: 19:6,713,474\n",
      "INFO\t2016-11-30 10:58:46\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:09s.  Time for last 1,000,000:    5s.  Last read position: 22:17,661,627\n",
      "INFO\t2016-11-30 10:58:51\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:14s.  Time for last 1,000,000:    5s.  Last read position: 1:233,135,033\n",
      "[Wed Nov 30 11:01:36 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.99 minutes.\n",
      "Runtime.totalMemory()=11720458240\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P10Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:01:36 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P10Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:01:36 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:01:36\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:01:41\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 11:125,475,543\n",
      "INFO\t2016-11-30 11:01:46\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 10:18,196,642\n",
      "INFO\t2016-11-30 11:01:51\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 19:9,861,249\n",
      "INFO\t2016-11-30 11:01:57\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 19:1,985,843\n",
      "INFO\t2016-11-30 11:02:04\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:    7s.  Last read position: 11:113,973,023\n",
      "INFO\t2016-11-30 11:02:09\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 1:569,352\n",
      "INFO\t2016-11-30 11:02:15\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:38s.  Time for last 1,000,000:    5s.  Last read position: 16:16,367,437\n",
      "INFO\t2016-11-30 11:02:20\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:44s.  Time for last 1,000,000:    5s.  Last read position: 12:1,897,250\n",
      "INFO\t2016-11-30 11:02:25\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:49s.  Time for last 1,000,000:    5s.  Last read position: 2:242,035,140\n",
      "INFO\t2016-11-30 11:02:30\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:54s.  Time for last 1,000,000:    5s.  Last read position: 19:54,968,043\n",
      "INFO\t2016-11-30 11:02:36\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    5s.  Last read position: 9:68,428,086\n",
      "INFO\t2016-11-30 11:02:41\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:05s.  Time for last 1,000,000:    5s.  Last read position: 6:4,006,927\n",
      "INFO\t2016-11-30 11:02:46\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:10s.  Time for last 1,000,000:    5s.  Last read position: 1:43,306,950\n",
      "INFO\t2016-11-30 11:02:53\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:16s.  Time for last 1,000,000:    6s.  Last read position: 7:133,076,214\n",
      "INFO\t2016-11-30 11:02:58\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:21s.  Time for last 1,000,000:    5s.  Last read position: 1:198,668,598\n",
      "INFO\t2016-11-30 11:03:03\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:26s.  Time for last 1,000,000:    5s.  Last read position: 12:1,746,484\n",
      "INFO\t2016-11-30 11:03:10\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:33s.  Time for last 1,000,000:    7s.  Last read position: 1:8,536,527\n",
      "INFO\t2016-11-30 11:03:15\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    5s.  Last read position: 2:234,097,710\n",
      "INFO\t2016-11-30 11:03:20\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    5s.  Last read position: 6:134,493,157\n",
      "INFO\t2016-11-30 11:03:27\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    6s.  Last read position: 5:158,793,886\n",
      "INFO\t2016-11-30 11:03:32\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    5s.  Last read position: 2:114,175,825\n",
      "INFO\t2016-11-30 11:03:37\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    5s.  Last read position: 16:85,952,107\n",
      "INFO\t2016-11-30 11:03:42\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    5s.  Last read position: 17:63,012,157\n",
      "INFO\t2016-11-30 11:03:47\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 20:31,042,643\n",
      "INFO\t2016-11-30 11:03:53\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    5s.  Last read position: 12:100,009,721\n",
      "INFO\t2016-11-30 11:03:58\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    5s.  Last read position: 1:100,507,493\n",
      "INFO\t2016-11-30 11:04:03\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    5s.  Last read position: 7:66,151,453\n",
      "INFO\t2016-11-30 11:04:08\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    5s.  Last read position: 7:2,701,663\n",
      "INFO\t2016-11-30 11:04:13\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:36s.  Time for last 1,000,000:    5s.  Last read position: 19:2,072,545\n",
      "INFO\t2016-11-30 11:04:18\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:41s.  Time for last 1,000,000:    5s.  Last read position: 5:177,037,158\n",
      "INFO\t2016-11-30 11:04:23\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:46s.  Time for last 1,000,000:    5s.  Last read position: 3:70,503,744\n",
      "INFO\t2016-11-30 11:04:28\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:51s.  Time for last 1,000,000:    5s.  Last read position: 11:65,686,039\n",
      "INFO\t2016-11-30 11:04:33\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:56s.  Time for last 1,000,000:    5s.  Last read position: 13:100,028,468\n",
      "INFO\t2016-11-30 11:04:38\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:03:01s.  Time for last 1,000,000:    5s.  Last read position: 2:101,901,028\n",
      "INFO\t2016-11-30 11:04:43\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:07s.  Time for last 1,000,000:    5s.  Last read position: 17:37,617,955\n",
      "INFO\t2016-11-30 11:04:48\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:12s.  Time for last 1,000,000:    5s.  Last read position: 1:149,369,771\n",
      "INFO\t2016-11-30 11:04:53\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:17s.  Time for last 1,000,000:    5s.  Last read position: 2:59,361,893\n",
      "INFO\t2016-11-30 11:04:59\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:22s.  Time for last 1,000,000:    5s.  Last read position: 18:7,639,518\n",
      "INFO\t2016-11-30 11:05:04\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:27s.  Time for last 1,000,000:    5s.  Last read position: 14:77,269,719\n",
      "INFO\t2016-11-30 11:05:09\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:32s.  Time for last 1,000,000:    4s.  Last read position: 10:70,102,379\n",
      "INFO\t2016-11-30 11:05:15\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:38s.  Time for last 1,000,000:    6s.  Last read position: MT:9,033\n",
      "INFO\t2016-11-30 11:05:20\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:43s.  Time for last 1,000,000:    4s.  Last read position: 16:70,760,786\n",
      "INFO\t2016-11-30 11:05:25\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:48s.  Time for last 1,000,000:    5s.  Last read position: 1:33,400,176\n",
      "INFO\t2016-11-30 11:05:31\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:03:55s.  Time for last 1,000,000:    6s.  Last read position: 4:8,226,935\n",
      "INFO\t2016-11-30 11:05:36\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:04:00s.  Time for last 1,000,000:    5s.  Last read position: 3:170,110,118\n",
      "INFO\t2016-11-30 11:05:41\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:04:05s.  Time for last 1,000,000:    5s.  Last read position: 1:40,037,860\n",
      "INFO\t2016-11-30 11:05:48\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:12s.  Time for last 1,000,000:    6s.  Last read position: 20:3,853,222\n",
      "INFO\t2016-11-30 11:05:53\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:17s.  Time for last 1,000,000:    4s.  Last read position: 5:171,584,869\n",
      "INFO\t2016-11-30 11:05:58\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:22s.  Time for last 1,000,000:    4s.  Last read position: 2:242,281,645\n",
      "INFO\t2016-11-30 11:06:07\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:30s.  Time for last 1,000,000:    8s.  Last read position: 5:70,413,380\n",
      "INFO\t2016-11-30 11:06:12\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:35s.  Time for last 1,000,000:    5s.  Last read position: 11:58,978,423\n",
      "INFO\t2016-11-30 11:06:17\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:40s.  Time for last 1,000,000:    5s.  Last read position: 12:125,397,476\n",
      "INFO\t2016-11-30 11:06:23\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:47s.  Time for last 1,000,000:    6s.  Last read position: 16:22,485,504\n",
      "INFO\t2016-11-30 11:06:28\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:52s.  Time for last 1,000,000:    5s.  Last read position: 9:70,845,640\n",
      "INFO\t2016-11-30 11:06:33\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:57s.  Time for last 1,000,000:    5s.  Last read position: 1:121,130,570\n",
      "INFO\t2016-11-30 11:06:38\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:05:02s.  Time for last 1,000,000:    5s.  Last read position: 6:87,929,110\n",
      "INFO\t2016-11-30 11:06:43\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:05:07s.  Time for last 1,000,000:    5s.  Last read position: X:218,246\n",
      "INFO\t2016-11-30 11:06:49\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:05:12s.  Time for last 1,000,000:    5s.  Last read position: 1:206,082,735\n",
      "INFO\t2016-11-30 11:06:54\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:17s.  Time for last 1,000,000:    5s.  Last read position: 12:73,171,932\n",
      "INFO\t2016-11-30 11:06:59\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:22s.  Time for last 1,000,000:    5s.  Last read position: 16:22,545,189\n",
      "INFO\t2016-11-30 11:07:04\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:27s.  Time for last 1,000,000:    5s.  Last read position: 12:104,746,425\n",
      "INFO\t2016-11-30 11:07:09\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:32s.  Time for last 1,000,000:    5s.  Last read position: X:77,084,478\n",
      "INFO\t2016-11-30 11:07:14\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:37s.  Time for last 1,000,000:    5s.  Last read position: 20:44,052,556\n",
      "INFO\t2016-11-30 11:07:19\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:42s.  Time for last 1,000,000:    4s.  Last read position: 5:156,513,890\n",
      "INFO\t2016-11-30 11:07:24\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:47s.  Time for last 1,000,000:    5s.  Last read position: 6:146,273,594\n",
      "INFO\t2016-11-30 11:07:29\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:53s.  Time for last 1,000,000:    5s.  Last read position: 19:41,083,326\n",
      "INFO\t2016-11-30 11:07:34\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:05:58s.  Time for last 1,000,000:    5s.  Last read position: 6:33,037,425\n",
      "INFO\t2016-11-30 11:07:39\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:06:03s.  Time for last 1,000,000:    5s.  Last read position: 16:21,884,835\n",
      "INFO\t2016-11-30 11:07:44\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:06:08s.  Time for last 1,000,000:    5s.  Last read position: 1:566,577\n",
      "INFO\t2016-11-30 11:07:50\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:13s.  Time for last 1,000,000:    5s.  Last read position: 3:195,971,791\n",
      "INFO\t2016-11-30 11:07:54\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:18s.  Time for last 1,000,000:    4s.  Last read position: 20:57,485,899\n",
      "INFO\t2016-11-30 11:07:59\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    4s.  Last read position: 19:6,680,199\n",
      "INFO\t2016-11-30 11:08:04\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:28s.  Time for last 1,000,000:    5s.  Last read position: 19:54,024,688\n",
      "INFO\t2016-11-30 11:08:09\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:33s.  Time for last 1,000,000:    4s.  Last read position: X:73,041,313\n",
      "INFO\t2016-11-30 11:08:15\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:38s.  Time for last 1,000,000:    5s.  Last read position: 9:136,405,742\n",
      "INFO\t2016-11-30 11:08:20\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:43s.  Time for last 1,000,000:    5s.  Last read position: 17:29,109,207\n",
      "INFO\t2016-11-30 11:08:25\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:48s.  Time for last 1,000,000:    5s.  Last read position: 5:130,766,871\n",
      "INFO\t2016-11-30 11:08:30\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:06:54s.  Time for last 1,000,000:    5s.  Last read position: 13:45,913,689\n",
      "INFO\t2016-11-30 11:08:35\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:06:59s.  Time for last 1,000,000:    4s.  Last read position: 4:9,876,703\n",
      "INFO\t2016-11-30 11:08:40\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:07:04s.  Time for last 1,000,000:    4s.  Last read position: X:80,547,665\n",
      "INFO\t2016-11-30 11:08:46\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:07:09s.  Time for last 1,000,000:    5s.  Last read position: 19:36,841,958\n",
      "INFO\t2016-11-30 11:08:51\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:14s.  Time for last 1,000,000:    4s.  Last read position: 7:139,735,680\n",
      "INFO\t2016-11-30 11:08:56\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:19s.  Time for last 1,000,000:    4s.  Last read position: 3:186,295,123\n",
      "INFO\t2016-11-30 11:09:02\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:25s.  Time for last 1,000,000:    5s.  Last read position: 19:11,433,462\n",
      "INFO\t2016-11-30 11:09:07\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:30s.  Time for last 1,000,000:    4s.  Last read position: 16:29,422,316\n",
      "INFO\t2016-11-30 11:09:11\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:35s.  Time for last 1,000,000:    4s.  Last read position: 5:134,686,349\n",
      "INFO\t2016-11-30 11:09:19\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:43s.  Time for last 1,000,000:    8s.  Last read position: 2:169,714,439\n",
      "INFO\t2016-11-30 11:09:24\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:48s.  Time for last 1,000,000:    4s.  Last read position: 10:51,851,833\n",
      "INFO\t2016-11-30 11:09:29\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:53s.  Time for last 1,000,000:    4s.  Last read position: 1:184,061,906\n",
      "INFO\t2016-11-30 11:09:38\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:08:01s.  Time for last 1,000,000:    8s.  Last read position: 1:11,191,997\n",
      "INFO\t2016-11-30 11:09:42\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:08:06s.  Time for last 1,000,000:    4s.  Last read position: 1:145,498,545\n",
      "INFO\t2016-11-30 11:09:47\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:08:11s.  Time for last 1,000,000:    4s.  Last read position: 15:102,507,552\n",
      "INFO\t2016-11-30 11:09:53\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:16s.  Time for last 1,000,000:    5s.  Last read position: 11:41,995,821\n",
      "INFO\t2016-11-30 11:09:58\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:21s.  Time for last 1,000,000:    4s.  Last read position: 9:139,906,561\n",
      "INFO\t2016-11-30 11:10:03\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:26s.  Time for last 1,000,000:    4s.  Last read position: 2:37,076,107\n",
      "INFO\t2016-11-30 11:10:08\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:31s.  Time for last 1,000,000:    5s.  Last read position: 12:26,107,552\n",
      "INFO\t2016-11-30 11:10:13\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:36s.  Time for last 1,000,000:    5s.  Last read position: 15:100,252,914\n",
      "INFO\t2016-11-30 11:10:18\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:41s.  Time for last 1,000,000:    5s.  Last read position: 12:120,636,757\n",
      "INFO\t2016-11-30 11:10:23\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:46s.  Time for last 1,000,000:    5s.  Last read position: 1:65,370,866\n",
      "INFO\t2016-11-30 11:10:28\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:51s.  Time for last 1,000,000:    4s.  Last read position: 16:25,835,782\n",
      "INFO\t2016-11-30 11:10:33\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:08:56s.  Time for last 1,000,000:    5s.  Last read position: 20:5,527,361\n",
      "INFO\t2016-11-30 11:10:38\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:09:01s.  Time for last 1,000,000:    5s.  Last read position: 7:45,097,184\n",
      "[Wed Nov 30 11:17:43 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 16.12 minutes.\n",
      "Runtime.totalMemory()=10822877184\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P2Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P2rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:17:44 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P2Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P2rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:17:44 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:17:44\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:17:49\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 6:31,322,014\n",
      "INFO\t2016-11-30 11:17:54\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 17:46,063,544\n",
      "INFO\t2016-11-30 11:18:00\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 6:50,880,321\n",
      "INFO\t2016-11-30 11:18:05\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:21s.  Time for last 1,000,000:    5s.  Last read position: 11:122,622,482\n",
      "INFO\t2016-11-30 11:18:10\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:    4s.  Last read position: 6:29,913,373\n",
      "INFO\t2016-11-30 11:18:17\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    6s.  Last read position: 9:135,896,170\n",
      "INFO\t2016-11-30 11:18:22\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:38s.  Time for last 1,000,000:    5s.  Last read position: 5:95,873,027\n",
      "INFO\t2016-11-30 11:18:28\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:    5s.  Last read position: 16:2,129,574\n",
      "INFO\t2016-11-30 11:18:33\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    4s.  Last read position: 11:46,450,332\n",
      "INFO\t2016-11-30 11:18:38\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:    5s.  Last read position: MT:3,135\n",
      "INFO\t2016-11-30 11:18:44\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    5s.  Last read position: 10:26,940,153\n",
      "INFO\t2016-11-30 11:18:49\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    5s.  Last read position: GL000220.1:117,710\n",
      "INFO\t2016-11-30 11:18:54\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    4s.  Last read position: 19:1,036,520\n",
      "INFO\t2016-11-30 11:19:01\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:17s.  Time for last 1,000,000:    7s.  Last read position: 16:72,832,318\n",
      "INFO\t2016-11-30 11:19:06\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    5s.  Last read position: 19:2,513,714\n",
      "INFO\t2016-11-30 11:19:11\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:27s.  Time for last 1,000,000:    5s.  Last read position: 3:123,634,124\n",
      "INFO\t2016-11-30 11:19:16\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    5s.  Last read position: 10:73,578,389\n",
      "INFO\t2016-11-30 11:19:21\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:37s.  Time for last 1,000,000:    4s.  Last read position: 15:76,519,728\n",
      "INFO\t2016-11-30 11:19:26\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:42s.  Time for last 1,000,000:    4s.  Last read position: 16:24,583,486\n",
      "INFO\t2016-11-30 11:19:31\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    4s.  Last read position: 22:29,729,179\n",
      "INFO\t2016-11-30 11:19:36\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:51s.  Time for last 1,000,000:    4s.  Last read position: 1:149,762,990\n",
      "INFO\t2016-11-30 11:19:41\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:56s.  Time for last 1,000,000:    4s.  Last read position: MT:7,437\n",
      "INFO\t2016-11-30 11:19:46\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    5s.  Last read position: 9:95,884,335\n",
      "INFO\t2016-11-30 11:19:51\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    4s.  Last read position: 2:90,060,731\n",
      "INFO\t2016-11-30 11:19:56\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 16:69,931,919\n",
      "INFO\t2016-11-30 11:20:01\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:17s.  Time for last 1,000,000:    5s.  Last read position: 7:56,172,112\n",
      "INFO\t2016-11-30 11:20:06\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    4s.  Last read position: 12:54,943,067\n",
      "INFO\t2016-11-30 11:20:11\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    4s.  Last read position: 9:36,205,411\n",
      "INFO\t2016-11-30 11:20:16\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:32s.  Time for last 1,000,000:    5s.  Last read position: 9:136,031,500\n",
      "INFO\t2016-11-30 11:20:21\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:37s.  Time for last 1,000,000:    5s.  Last read position: 20:35,545,358\n",
      "INFO\t2016-11-30 11:20:26\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    4s.  Last read position: 1:144,618,167\n",
      "INFO\t2016-11-30 11:20:33\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:48s.  Time for last 1,000,000:    6s.  Last read position: 5:148,207,014\n",
      "INFO\t2016-11-30 11:20:37\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:53s.  Time for last 1,000,000:    4s.  Last read position: 8:38,273,431\n",
      "INFO\t2016-11-30 11:20:42\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:58s.  Time for last 1,000,000:    4s.  Last read position: 4:113,092,597\n",
      "INFO\t2016-11-30 11:20:48\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:04s.  Time for last 1,000,000:    6s.  Last read position: 9:134,350,294\n",
      "[Wed Nov 30 11:23:04 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.34 minutes.\n",
      "Runtime.totalMemory()=11238113280\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P3Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P3rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:23:04 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P3Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P3rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:23:04 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:23:04\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:23:09\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 5:150,566,318\n",
      "INFO\t2016-11-30 11:23:14\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 11:1,775,342\n",
      "INFO\t2016-11-30 11:23:19\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    5s.  Last read position: 1:161,511,582\n",
      "INFO\t2016-11-30 11:23:24\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 7:36,633,975\n",
      "INFO\t2016-11-30 11:23:29\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:    4s.  Last read position: 19:42,044,551\n",
      "INFO\t2016-11-30 11:23:36\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    6s.  Last read position: 1:245,017,544\n",
      "INFO\t2016-11-30 11:23:41\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 8:56,924,454\n",
      "INFO\t2016-11-30 11:23:47\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    5s.  Last read position: 2:47,485,947\n",
      "INFO\t2016-11-30 11:23:52\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:47s.  Time for last 1,000,000:    4s.  Last read position: 15:45,007,856\n",
      "INFO\t2016-11-30 11:23:57\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:    4s.  Last read position: 14:64,153,340\n",
      "INFO\t2016-11-30 11:24:03\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    6s.  Last read position: 11:74,862,241\n",
      "INFO\t2016-11-30 11:24:08\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    4s.  Last read position: 7:155,573,348\n",
      "INFO\t2016-11-30 11:24:13\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    4s.  Last read position: 4:138,872,928\n",
      "INFO\t2016-11-30 11:24:20\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    6s.  Last read position: 3:52,557,458\n",
      "INFO\t2016-11-30 11:24:25\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:20s.  Time for last 1,000,000:    5s.  Last read position: 2:99,237,673\n",
      "INFO\t2016-11-30 11:24:30\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:25s.  Time for last 1,000,000:    4s.  Last read position: 9:464,347\n",
      "INFO\t2016-11-30 11:24:35\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    5s.  Last read position: 10:46,272,830\n",
      "INFO\t2016-11-30 11:24:40\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    5s.  Last read position: 11:34,460,558\n",
      "INFO\t2016-11-30 11:24:45\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:40s.  Time for last 1,000,000:    5s.  Last read position: 9:35,752,812\n",
      "INFO\t2016-11-30 11:24:50\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    5s.  Last read position: 11:60,160,409\n",
      "INFO\t2016-11-30 11:24:55\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,772\n",
      "INFO\t2016-11-30 11:25:00\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    4s.  Last read position: 17:76,127,701\n",
      "INFO\t2016-11-30 11:25:05\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:    4s.  Last read position: 1:569,559\n",
      "INFO\t2016-11-30 11:25:10\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:05s.  Time for last 1,000,000:    4s.  Last read position: 11:33,729,656\n",
      "INFO\t2016-11-30 11:25:15\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:10s.  Time for last 1,000,000:    4s.  Last read position: 4:187,007,347\n",
      "INFO\t2016-11-30 11:25:20\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:15s.  Time for last 1,000,000:    5s.  Last read position: 14:103,592,908\n",
      "INFO\t2016-11-30 11:25:25\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:20s.  Time for last 1,000,000:    4s.  Last read position: 6:35,190,642\n",
      "INFO\t2016-11-30 11:25:30\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:25s.  Time for last 1,000,000:    4s.  Last read position: 8:81,892,681\n",
      "INFO\t2016-11-30 11:25:37\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:32s.  Time for last 1,000,000:    6s.  Last read position: 20:29,653,530\n",
      "INFO\t2016-11-30 11:25:42\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:37s.  Time for last 1,000,000:    5s.  Last read position: 1:327,543\n",
      "INFO\t2016-11-30 11:25:47\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    5s.  Last read position: 21:9,827,369\n",
      "[Wed Nov 30 11:28:07 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.04 minutes.\n",
      "Runtime.totalMemory()=11738808320\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P6Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P6rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:28:07 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P6Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P6rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:28:07 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:28:07\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:28:12\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 14:105,878,224\n",
      "INFO\t2016-11-30 11:28:17\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 8:11,700,752\n",
      "INFO\t2016-11-30 11:28:22\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 11:67,812,919\n",
      "INFO\t2016-11-30 11:28:28\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 19:7,700,472\n",
      "INFO\t2016-11-30 11:28:33\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:    5s.  Last read position: 19:41,774,143\n",
      "INFO\t2016-11-30 11:28:39\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    5s.  Last read position: 7:100,781,196\n",
      "INFO\t2016-11-30 11:28:44\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 9:82,305,006\n",
      "INFO\t2016-11-30 11:28:50\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    5s.  Last read position: 3:11,613,912\n",
      "INFO\t2016-11-30 11:28:55\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:47s.  Time for last 1,000,000:    5s.  Last read position: 19:62,874\n",
      "INFO\t2016-11-30 11:29:00\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:    5s.  Last read position: 8:11,712,304\n",
      "INFO\t2016-11-30 11:29:07\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    6s.  Last read position: 5:180,218,147\n",
      "INFO\t2016-11-30 11:29:12\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    5s.  Last read position: 9:124,065,203\n",
      "INFO\t2016-11-30 11:29:17\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    5s.  Last read position: 1:2,161,833\n",
      "INFO\t2016-11-30 11:29:24\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:16s.  Time for last 1,000,000:    7s.  Last read position: 19:17,164,858\n",
      "INFO\t2016-11-30 11:29:29\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    5s.  Last read position: X:117,900,496\n",
      "INFO\t2016-11-30 11:29:35\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:27s.  Time for last 1,000,000:    5s.  Last read position: 1:92,450,862\n",
      "INFO\t2016-11-30 11:29:40\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    5s.  Last read position: 15:41,154,836\n",
      "INFO\t2016-11-30 11:29:45\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:37s.  Time for last 1,000,000:    5s.  Last read position: 9:135,896,231\n",
      "INFO\t2016-11-30 11:29:50\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:42s.  Time for last 1,000,000:    5s.  Last read position: 6:167,356,518\n",
      "INFO\t2016-11-30 11:29:55\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:48s.  Time for last 1,000,000:    5s.  Last read position: X:74,332,748\n",
      "INFO\t2016-11-30 11:30:01\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:53s.  Time for last 1,000,000:    5s.  Last read position: 10:103,261,070\n",
      "INFO\t2016-11-30 11:30:06\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    5s.  Last read position: 6:122,768,074\n",
      "INFO\t2016-11-30 11:30:11\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    5s.  Last read position: 7:26,235,517\n",
      "INFO\t2016-11-30 11:30:16\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:09s.  Time for last 1,000,000:    5s.  Last read position: 7:128,291,692\n",
      "INFO\t2016-11-30 11:30:21\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:14s.  Time for last 1,000,000:    5s.  Last read position: 8:133,144,492\n",
      "INFO\t2016-11-30 11:30:28\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:20s.  Time for last 1,000,000:    6s.  Last read position: 18:45,395,686\n",
      "INFO\t2016-11-30 11:30:33\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    5s.  Last read position: 17:914,049\n",
      "INFO\t2016-11-30 11:30:39\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    5s.  Last read position: 6:149,705,760\n",
      "INFO\t2016-11-30 11:30:46\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:38s.  Time for last 1,000,000:    6s.  Last read position: 6:32,412,619\n",
      "INFO\t2016-11-30 11:30:51\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:43s.  Time for last 1,000,000:    5s.  Last read position: 5:179,593,702\n",
      "INFO\t2016-11-30 11:30:56\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:48s.  Time for last 1,000,000:    5s.  Last read position: 17:30,195,117\n",
      "INFO\t2016-11-30 11:31:03\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:55s.  Time for last 1,000,000:    6s.  Last read position: 6:74,227,434\n",
      "INFO\t2016-11-30 11:31:08\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:03:00s.  Time for last 1,000,000:    5s.  Last read position: 1:158,145,678\n",
      "[Wed Nov 30 11:33:54 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 5.77 minutes.\n",
      "Runtime.totalMemory()=11249123328\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P10Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:33:54 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P10Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:33:54 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:33:54\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:33:59\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 11:125,475,543\n",
      "INFO\t2016-11-30 11:34:05\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: 10:18,196,642\n",
      "INFO\t2016-11-30 11:34:10\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 19:9,861,249\n",
      "INFO\t2016-11-30 11:34:15\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:21s.  Time for last 1,000,000:    5s.  Last read position: 19:1,985,843\n",
      "INFO\t2016-11-30 11:34:21\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:    5s.  Last read position: 11:113,973,023\n",
      "INFO\t2016-11-30 11:34:26\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    5s.  Last read position: 1:569,352\n",
      "INFO\t2016-11-30 11:34:32\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:37s.  Time for last 1,000,000:    5s.  Last read position: 16:16,367,437\n",
      "INFO\t2016-11-30 11:34:38\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:    5s.  Last read position: 12:1,897,250\n",
      "INFO\t2016-11-30 11:34:43\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:    5s.  Last read position: 2:242,035,140\n",
      "INFO\t2016-11-30 11:34:48\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:    5s.  Last read position: 19:54,968,043\n",
      "INFO\t2016-11-30 11:34:55\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:01:00s.  Time for last 1,000,000:    6s.  Last read position: 9:68,428,086\n",
      "INFO\t2016-11-30 11:35:00\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:05s.  Time for last 1,000,000:    5s.  Last read position: 6:4,006,927\n",
      "INFO\t2016-11-30 11:35:05\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:11s.  Time for last 1,000,000:    5s.  Last read position: 1:43,306,950\n",
      "INFO\t2016-11-30 11:35:14\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    8s.  Last read position: 7:133,076,214\n",
      "INFO\t2016-11-30 11:35:19\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:24s.  Time for last 1,000,000:    5s.  Last read position: 1:198,668,598\n",
      "INFO\t2016-11-30 11:35:24\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    5s.  Last read position: 12:1,746,484\n",
      "INFO\t2016-11-30 11:35:29\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    5s.  Last read position: 1:8,536,527\n",
      "INFO\t2016-11-30 11:35:34\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:40s.  Time for last 1,000,000:    5s.  Last read position: 2:234,097,710\n",
      "INFO\t2016-11-30 11:35:39\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    5s.  Last read position: 6:134,493,157\n",
      "INFO\t2016-11-30 11:35:45\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    5s.  Last read position: 5:158,793,886\n",
      "INFO\t2016-11-30 11:35:50\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    5s.  Last read position: 2:114,175,825\n",
      "INFO\t2016-11-30 11:35:55\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:    5s.  Last read position: 16:85,952,107\n",
      "INFO\t2016-11-30 11:36:01\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    6s.  Last read position: 17:63,012,157\n",
      "INFO\t2016-11-30 11:36:06\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 20:31,042,643\n",
      "INFO\t2016-11-30 11:36:11\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    4s.  Last read position: 12:100,009,721\n",
      "INFO\t2016-11-30 11:36:17\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:22s.  Time for last 1,000,000:    6s.  Last read position: 1:100,507,493\n",
      "INFO\t2016-11-30 11:36:22\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:27s.  Time for last 1,000,000:    5s.  Last read position: 7:66,151,453\n",
      "INFO\t2016-11-30 11:36:27\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:32s.  Time for last 1,000,000:    4s.  Last read position: 7:2,701,663\n",
      "INFO\t2016-11-30 11:36:34\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:39s.  Time for last 1,000,000:    6s.  Last read position: 19:2,072,545\n",
      "INFO\t2016-11-30 11:36:39\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:44s.  Time for last 1,000,000:    5s.  Last read position: 5:177,037,158\n",
      "INFO\t2016-11-30 11:36:44\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:49s.  Time for last 1,000,000:    4s.  Last read position: 3:70,503,744\n",
      "INFO\t2016-11-30 11:36:53\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:58s.  Time for last 1,000,000:    9s.  Last read position: 11:65,686,039\n",
      "INFO\t2016-11-30 11:36:58\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:03:03s.  Time for last 1,000,000:    5s.  Last read position: 13:100,028,468\n",
      "INFO\t2016-11-30 11:37:03\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:03:09s.  Time for last 1,000,000:    5s.  Last read position: 2:101,901,028\n",
      "INFO\t2016-11-30 11:37:09\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:14s.  Time for last 1,000,000:    5s.  Last read position: 17:37,617,955\n",
      "INFO\t2016-11-30 11:37:14\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:19s.  Time for last 1,000,000:    5s.  Last read position: 1:149,369,771\n",
      "INFO\t2016-11-30 11:37:19\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:24s.  Time for last 1,000,000:    5s.  Last read position: 2:59,361,893\n",
      "INFO\t2016-11-30 11:37:24\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:29s.  Time for last 1,000,000:    5s.  Last read position: 18:7,639,518\n",
      "INFO\t2016-11-30 11:37:29\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:34s.  Time for last 1,000,000:    5s.  Last read position: 14:77,269,719\n",
      "INFO\t2016-11-30 11:37:34\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:39s.  Time for last 1,000,000:    5s.  Last read position: 10:70,102,379\n",
      "INFO\t2016-11-30 11:37:39\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:44s.  Time for last 1,000,000:    5s.  Last read position: MT:9,033\n",
      "INFO\t2016-11-30 11:37:44\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:50s.  Time for last 1,000,000:    5s.  Last read position: 16:70,760,786\n",
      "INFO\t2016-11-30 11:37:50\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:55s.  Time for last 1,000,000:    5s.  Last read position: 1:33,400,176\n",
      "INFO\t2016-11-30 11:37:55\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:04:00s.  Time for last 1,000,000:    5s.  Last read position: 4:8,226,935\n",
      "INFO\t2016-11-30 11:38:00\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:04:05s.  Time for last 1,000,000:    5s.  Last read position: 3:170,110,118\n",
      "INFO\t2016-11-30 11:38:05\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:04:10s.  Time for last 1,000,000:    5s.  Last read position: 1:40,037,860\n",
      "INFO\t2016-11-30 11:38:10\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:15s.  Time for last 1,000,000:    5s.  Last read position: 20:3,853,222\n",
      "INFO\t2016-11-30 11:38:16\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    5s.  Last read position: 5:171,584,869\n",
      "INFO\t2016-11-30 11:38:21\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:26s.  Time for last 1,000,000:    5s.  Last read position: 2:242,281,645\n",
      "INFO\t2016-11-30 11:38:26\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:31s.  Time for last 1,000,000:    5s.  Last read position: 5:70,413,380\n",
      "INFO\t2016-11-30 11:38:32\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:37s.  Time for last 1,000,000:    6s.  Last read position: 11:58,978,423\n",
      "INFO\t2016-11-30 11:38:37\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:42s.  Time for last 1,000,000:    5s.  Last read position: 12:125,397,476\n",
      "INFO\t2016-11-30 11:38:42\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:47s.  Time for last 1,000,000:    5s.  Last read position: 16:22,485,504\n",
      "INFO\t2016-11-30 11:38:48\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:53s.  Time for last 1,000,000:    6s.  Last read position: 9:70,845,640\n",
      "INFO\t2016-11-30 11:38:53\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:58s.  Time for last 1,000,000:    4s.  Last read position: 1:121,130,570\n",
      "INFO\t2016-11-30 11:38:58\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:05:03s.  Time for last 1,000,000:    4s.  Last read position: 6:87,929,110\n",
      "INFO\t2016-11-30 11:39:06\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:05:11s.  Time for last 1,000,000:    7s.  Last read position: X:218,246\n",
      "INFO\t2016-11-30 11:39:11\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:05:16s.  Time for last 1,000,000:    5s.  Last read position: 1:206,082,735\n",
      "INFO\t2016-11-30 11:39:16\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:21s.  Time for last 1,000,000:    5s.  Last read position: 12:73,171,932\n",
      "INFO\t2016-11-30 11:39:21\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:26s.  Time for last 1,000,000:    5s.  Last read position: 16:22,545,189\n",
      "INFO\t2016-11-30 11:39:26\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:31s.  Time for last 1,000,000:    5s.  Last read position: 12:104,746,425\n",
      "INFO\t2016-11-30 11:39:31\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:36s.  Time for last 1,000,000:    5s.  Last read position: X:77,084,478\n",
      "INFO\t2016-11-30 11:39:37\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:42s.  Time for last 1,000,000:    5s.  Last read position: 20:44,052,556\n",
      "INFO\t2016-11-30 11:39:42\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:47s.  Time for last 1,000,000:    5s.  Last read position: 5:156,513,890\n",
      "INFO\t2016-11-30 11:39:47\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:52s.  Time for last 1,000,000:    4s.  Last read position: 6:146,273,594\n",
      "INFO\t2016-11-30 11:39:52\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:57s.  Time for last 1,000,000:    5s.  Last read position: 19:41,083,326\n",
      "INFO\t2016-11-30 11:39:57\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:06:02s.  Time for last 1,000,000:    4s.  Last read position: 6:33,037,425\n",
      "INFO\t2016-11-30 11:40:02\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:06:07s.  Time for last 1,000,000:    5s.  Last read position: 16:21,884,835\n",
      "INFO\t2016-11-30 11:40:07\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:06:13s.  Time for last 1,000,000:    5s.  Last read position: 1:566,577\n",
      "INFO\t2016-11-30 11:40:13\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:18s.  Time for last 1,000,000:    5s.  Last read position: 3:195,971,791\n",
      "INFO\t2016-11-30 11:40:18\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    5s.  Last read position: 20:57,485,899\n",
      "INFO\t2016-11-30 11:40:25\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:30s.  Time for last 1,000,000:    7s.  Last read position: 19:6,680,199\n",
      "INFO\t2016-11-30 11:40:30\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:35s.  Time for last 1,000,000:    5s.  Last read position: 19:54,024,688\n",
      "INFO\t2016-11-30 11:40:35\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:40s.  Time for last 1,000,000:    5s.  Last read position: X:73,041,313\n",
      "INFO\t2016-11-30 11:40:42\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:47s.  Time for last 1,000,000:    6s.  Last read position: 9:136,405,742\n",
      "INFO\t2016-11-30 11:40:47\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:52s.  Time for last 1,000,000:    5s.  Last read position: 17:29,109,207\n",
      "INFO\t2016-11-30 11:40:52\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:57s.  Time for last 1,000,000:    5s.  Last read position: 5:130,766,871\n",
      "INFO\t2016-11-30 11:40:57\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:07:03s.  Time for last 1,000,000:    5s.  Last read position: 13:45,913,689\n",
      "INFO\t2016-11-30 11:41:03\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:07:08s.  Time for last 1,000,000:    5s.  Last read position: 4:9,876,703\n",
      "INFO\t2016-11-30 11:41:08\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:07:13s.  Time for last 1,000,000:    4s.  Last read position: X:80,547,665\n",
      "INFO\t2016-11-30 11:41:13\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:07:18s.  Time for last 1,000,000:    4s.  Last read position: 19:36,841,958\n",
      "INFO\t2016-11-30 11:41:18\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:23s.  Time for last 1,000,000:    5s.  Last read position: 7:139,735,680\n",
      "INFO\t2016-11-30 11:41:23\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:28s.  Time for last 1,000,000:    4s.  Last read position: 3:186,295,123\n",
      "INFO\t2016-11-30 11:41:28\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:33s.  Time for last 1,000,000:    4s.  Last read position: 19:11,433,462\n",
      "INFO\t2016-11-30 11:41:33\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:38s.  Time for last 1,000,000:    5s.  Last read position: 16:29,422,316\n",
      "INFO\t2016-11-30 11:41:38\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:43s.  Time for last 1,000,000:    4s.  Last read position: 5:134,686,349\n",
      "INFO\t2016-11-30 11:41:43\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:48s.  Time for last 1,000,000:    5s.  Last read position: 2:169,714,439\n",
      "INFO\t2016-11-30 11:41:49\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:54s.  Time for last 1,000,000:    5s.  Last read position: 10:51,851,833\n",
      "INFO\t2016-11-30 11:41:54\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:59s.  Time for last 1,000,000:    5s.  Last read position: 1:184,061,906\n",
      "INFO\t2016-11-30 11:41:59\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:08:04s.  Time for last 1,000,000:    5s.  Last read position: 1:11,191,997\n",
      "INFO\t2016-11-30 11:42:04\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:08:10s.  Time for last 1,000,000:    5s.  Last read position: 1:145,498,545\n",
      "INFO\t2016-11-30 11:42:10\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:08:15s.  Time for last 1,000,000:    5s.  Last read position: 15:102,507,552\n",
      "INFO\t2016-11-30 11:42:15\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:20s.  Time for last 1,000,000:    5s.  Last read position: 11:41,995,821\n",
      "INFO\t2016-11-30 11:42:23\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:28s.  Time for last 1,000,000:    7s.  Last read position: 9:139,906,561\n",
      "INFO\t2016-11-30 11:42:27\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:32s.  Time for last 1,000,000:    4s.  Last read position: 2:37,076,107\n",
      "INFO\t2016-11-30 11:42:33\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:38s.  Time for last 1,000,000:    5s.  Last read position: 12:26,107,552\n",
      "INFO\t2016-11-30 11:42:39\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:45s.  Time for last 1,000,000:    6s.  Last read position: 15:100,252,914\n",
      "INFO\t2016-11-30 11:42:44\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:50s.  Time for last 1,000,000:    5s.  Last read position: 12:120,636,757\n",
      "INFO\t2016-11-30 11:42:49\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:54s.  Time for last 1,000,000:    4s.  Last read position: 1:65,370,866\n",
      "INFO\t2016-11-30 11:42:54\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:59s.  Time for last 1,000,000:    4s.  Last read position: 16:25,835,782\n",
      "INFO\t2016-11-30 11:42:59\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:09:04s.  Time for last 1,000,000:    5s.  Last read position: 20:5,527,361\n",
      "INFO\t2016-11-30 11:43:04\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:09:10s.  Time for last 1,000,000:    5s.  Last read position: 7:45,097,184\n",
      "[Wed Nov 30 11:50:25 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 16.52 minutes.\n",
      "Runtime.totalMemory()=11684806656\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P15Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P15rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 11:50:26 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P15Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P15rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 11:50:26 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 11:50:26\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 11:50:30\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:23,467,011\n",
      "INFO\t2016-11-30 11:50:36\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 14:24,473,890\n",
      "INFO\t2016-11-30 11:50:41\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    5s.  Last read position: 9:70,842,879\n",
      "INFO\t2016-11-30 11:50:46\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    5s.  Last read position: 11:67,120,284\n",
      "INFO\t2016-11-30 11:50:51\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:25s.  Time for last 1,000,000:    4s.  Last read position: 13:41,326,637\n",
      "INFO\t2016-11-30 11:50:56\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:    4s.  Last read position: 14:105,912,680\n",
      "INFO\t2016-11-30 11:51:01\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    5s.  Last read position: GL000220.1:113,557\n",
      "INFO\t2016-11-30 11:51:07\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    5s.  Last read position: 11:121,476,237\n",
      "INFO\t2016-11-30 11:51:12\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,772\n",
      "INFO\t2016-11-30 11:51:17\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    4s.  Last read position: 3:182,981,417\n",
      "INFO\t2016-11-30 11:51:25\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    7s.  Last read position: 17:41,948,516\n",
      "INFO\t2016-11-30 11:51:30\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    5s.  Last read position: 12:111,113,682\n",
      "INFO\t2016-11-30 11:51:35\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    5s.  Last read position: 18:11,642,513\n",
      "INFO\t2016-11-30 11:51:40\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:14s.  Time for last 1,000,000:    5s.  Last read position: 7:44,807,546\n",
      "INFO\t2016-11-30 11:51:45\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    4s.  Last read position: 11:7,454,062\n",
      "INFO\t2016-11-30 11:51:50\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:24s.  Time for last 1,000,000:    5s.  Last read position: 2:127,815,144\n",
      "INFO\t2016-11-30 11:51:56\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    5s.  Last read position: 5:149,473,991\n",
      "INFO\t2016-11-30 11:52:01\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:34s.  Time for last 1,000,000:    4s.  Last read position: 19:10,228,081\n",
      "INFO\t2016-11-30 11:52:06\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    5s.  Last read position: 13:31,728,749\n",
      "INFO\t2016-11-30 11:52:12\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    5s.  Last read position: 8:95,949,575\n",
      "INFO\t2016-11-30 11:52:17\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    5s.  Last read position: 1:120,399,797\n",
      "INFO\t2016-11-30 11:52:22\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:56s.  Time for last 1,000,000:    5s.  Last read position: 16:47,661,105\n",
      "INFO\t2016-11-30 11:52:27\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    5s.  Last read position: 14:89,977,908\n",
      "INFO\t2016-11-30 11:52:32\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    5s.  Last read position: 7:141,469,025\n",
      "INFO\t2016-11-30 11:52:37\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    4s.  Last read position: 9:132,057,188\n",
      "INFO\t2016-11-30 11:52:42\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    4s.  Last read position: 12:10,317,582\n",
      "INFO\t2016-11-30 11:52:47\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    5s.  Last read position: 2:69,012,324\n",
      "INFO\t2016-11-30 11:52:53\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:    5s.  Last read position: 8:24,198,904\n",
      "INFO\t2016-11-30 11:52:58\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    4s.  Last read position: 3:197,917,188\n",
      "INFO\t2016-11-30 11:53:03\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:37s.  Time for last 1,000,000:    5s.  Last read position: 2:152,651,726\n",
      "INFO\t2016-11-30 11:53:08\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    4s.  Last read position: 1:247,364,442\n",
      "INFO\t2016-11-30 11:53:13\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:47s.  Time for last 1,000,000:    4s.  Last read position: 9:124,043,983\n",
      "INFO\t2016-11-30 11:53:21\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:55s.  Time for last 1,000,000:    8s.  Last read position: 1:206,626,936\n",
      "INFO\t2016-11-30 11:53:27\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:03:00s.  Time for last 1,000,000:    5s.  Last read position: 8:17,874,889\n",
      "INFO\t2016-11-30 11:53:32\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:05s.  Time for last 1,000,000:    4s.  Last read position: 8:95,789,967\n",
      "INFO\t2016-11-30 11:53:37\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:10s.  Time for last 1,000,000:    5s.  Last read position: 4:154,628,974\n",
      "INFO\t2016-11-30 11:53:42\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:15s.  Time for last 1,000,000:    4s.  Last read position: 7:6,786,828\n",
      "INFO\t2016-11-30 11:53:47\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:20s.  Time for last 1,000,000:    4s.  Last read position: 12:9,224,912\n",
      "INFO\t2016-11-30 11:53:52\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:26s.  Time for last 1,000,000:    5s.  Last read position: 11:57,581,234\n",
      "INFO\t2016-11-30 11:53:57\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:31s.  Time for last 1,000,000:    4s.  Last read position: 14:77,101,855\n",
      "INFO\t2016-11-30 11:54:02\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:36s.  Time for last 1,000,000:    4s.  Last read position: 9:5,066,429\n",
      "INFO\t2016-11-30 11:54:08\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:42s.  Time for last 1,000,000:    6s.  Last read position: 1:182,352,563\n",
      "INFO\t2016-11-30 11:54:13\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:47s.  Time for last 1,000,000:    4s.  Last read position: 5:133,914,468\n",
      "INFO\t2016-11-30 11:54:18\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:03:52s.  Time for last 1,000,000:    4s.  Last read position: 10:13,687,581\n",
      "INFO\t2016-11-30 11:54:24\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:03:58s.  Time for last 1,000,000:    6s.  Last read position: 19:49,468,823\n",
      "INFO\t2016-11-30 11:54:29\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:04:03s.  Time for last 1,000,000:    4s.  Last read position: 14:23,314,461\n",
      "INFO\t2016-11-30 11:54:34\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:08s.  Time for last 1,000,000:    5s.  Last read position: X:153,581,139\n",
      "INFO\t2016-11-30 11:54:39\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:13s.  Time for last 1,000,000:    5s.  Last read position: 1:21,071,007\n",
      "INFO\t2016-11-30 11:54:44\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:18s.  Time for last 1,000,000:    4s.  Last read position: 2:225,717,669\n",
      "INFO\t2016-11-30 11:54:49\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:23s.  Time for last 1,000,000:    4s.  Last read position: 20:35,835,825\n",
      "INFO\t2016-11-30 11:54:54\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:28s.  Time for last 1,000,000:    4s.  Last read position: 1:150,704,135\n",
      "INFO\t2016-11-30 11:54:59\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:33s.  Time for last 1,000,000:    4s.  Last read position: 3:46,245,155\n",
      "INFO\t2016-11-30 11:55:04\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:38s.  Time for last 1,000,000:    4s.  Last read position: 1:161,092,929\n",
      "INFO\t2016-11-30 11:55:09\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:43s.  Time for last 1,000,000:    5s.  Last read position: 5:140,011,864\n",
      "INFO\t2016-11-30 11:55:15\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:48s.  Time for last 1,000,000:    5s.  Last read position: 13:95,818,461\n",
      "INFO\t2016-11-30 11:55:20\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:04:54s.  Time for last 1,000,000:    5s.  Last read position: 2:217,059,674\n",
      "INFO\t2016-11-30 11:55:25\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:04:58s.  Time for last 1,000,000:    4s.  Last read position: 7:23,300,739\n",
      "INFO\t2016-11-30 11:55:32\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:05:06s.  Time for last 1,000,000:    7s.  Last read position: 8:61,533,896\n",
      "INFO\t2016-11-30 11:55:37\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:10s.  Time for last 1,000,000:    4s.  Last read position: 12:2,874,344\n",
      "INFO\t2016-11-30 11:55:42\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:15s.  Time for last 1,000,000:    4s.  Last read position: 4:88,899,912\n",
      "INFO\t2016-11-30 11:55:50\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:24s.  Time for last 1,000,000:    8s.  Last read position: 17:44,127,901\n",
      "INFO\t2016-11-30 11:55:55\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:29s.  Time for last 1,000,000:    4s.  Last read position: 20:57,576,617\n",
      "INFO\t2016-11-30 11:56:00\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:34s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,968\n",
      "INFO\t2016-11-30 11:56:05\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:39s.  Time for last 1,000,000:    5s.  Last read position: 17:37,034,383\n",
      "INFO\t2016-11-30 11:56:10\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:44s.  Time for last 1,000,000:    4s.  Last read position: 1:148,251,301\n",
      "INFO\t2016-11-30 11:56:15\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:49s.  Time for last 1,000,000:    5s.  Last read position: 21:34,900,572\n",
      "INFO\t2016-11-30 11:56:20\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:05:54s.  Time for last 1,000,000:    5s.  Last read position: 17:43,110,831\n",
      "INFO\t2016-11-30 11:56:25\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:05:59s.  Time for last 1,000,000:    4s.  Last read position: 10:6,262,623\n",
      "INFO\t2016-11-30 11:56:30\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:06:04s.  Time for last 1,000,000:    4s.  Last read position: 17:46,136,248\n",
      "INFO\t2016-11-30 11:56:36\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:10s.  Time for last 1,000,000:    6s.  Last read position: 6:70,500,315\n",
      "INFO\t2016-11-30 11:56:41\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:15s.  Time for last 1,000,000:    4s.  Last read position: 16:66,597,037\n",
      "INFO\t2016-11-30 11:56:46\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:20s.  Time for last 1,000,000:    4s.  Last read position: 16:21,652,541\n",
      "INFO\t2016-11-30 11:56:52\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:26s.  Time for last 1,000,000:    6s.  Last read position: 17:2,575,580\n",
      "INFO\t2016-11-30 11:56:57\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:31s.  Time for last 1,000,000:    4s.  Last read position: 10:104,573,701\n",
      "INFO\t2016-11-30 11:57:02\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:36s.  Time for last 1,000,000:    4s.  Last read position: 7:72,466,335\n",
      "INFO\t2016-11-30 11:57:09\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:43s.  Time for last 1,000,000:    7s.  Last read position: 10:13,849,308\n",
      "INFO\t2016-11-30 11:57:14\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:48s.  Time for last 1,000,000:    4s.  Last read position: 9:33,481,888\n",
      "INFO\t2016-11-30 11:57:19\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:06:53s.  Time for last 1,000,000:    4s.  Last read position: 17:26,960,736\n",
      "INFO\t2016-11-30 11:57:24\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:06:58s.  Time for last 1,000,000:    4s.  Last read position: 5:149,452,936\n",
      "INFO\t2016-11-30 11:57:29\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:07:03s.  Time for last 1,000,000:    5s.  Last read position: 8:61,779,065\n",
      "INFO\t2016-11-30 11:57:34\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:07:08s.  Time for last 1,000,000:    4s.  Last read position: 15:86,050,261\n",
      "INFO\t2016-11-30 11:57:39\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:13s.  Time for last 1,000,000:    4s.  Last read position: 17:66,594,584\n",
      "INFO\t2016-11-30 11:57:44\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:18s.  Time for last 1,000,000:    5s.  Last read position: 1:121,147,052\n",
      "INFO\t2016-11-30 11:57:49\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:23s.  Time for last 1,000,000:    4s.  Last read position: 4:39,122,833\n",
      "INFO\t2016-11-30 11:57:54\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:27s.  Time for last 1,000,000:    4s.  Last read position: 5:70,281,442\n",
      "INFO\t2016-11-30 11:57:59\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:33s.  Time for last 1,000,000:    5s.  Last read position: 10:72,326,459\n",
      "INFO\t2016-11-30 11:58:04\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:37s.  Time for last 1,000,000:    4s.  Last read position: 11:121,446,955\n",
      "INFO\t2016-11-30 11:58:09\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:42s.  Time for last 1,000,000:    4s.  Last read position: 5:56,546,876\n",
      "INFO\t2016-11-30 11:58:15\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:49s.  Time for last 1,000,000:    6s.  Last read position: 1:53,379,445\n",
      "INFO\t2016-11-30 11:58:20\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:07:54s.  Time for last 1,000,000:    4s.  Last read position: 10:118,761,814\n",
      "INFO\t2016-11-30 11:58:25\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:07:59s.  Time for last 1,000,000:    4s.  Last read position: 4:74,055,712\n",
      "INFO\t2016-11-30 11:58:32\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:08:06s.  Time for last 1,000,000:    6s.  Last read position: 1:104,161,673\n",
      "INFO\t2016-11-30 11:58:37\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:11s.  Time for last 1,000,000:    4s.  Last read position: 2:144,321,230\n",
      "INFO\t2016-11-30 11:58:42\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:15s.  Time for last 1,000,000:    4s.  Last read position: 4:56,265,638\n",
      "INFO\t2016-11-30 11:58:47\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:20s.  Time for last 1,000,000:    4s.  Last read position: 1:121,133,562\n",
      "INFO\t2016-11-30 11:58:51\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:25s.  Time for last 1,000,000:    4s.  Last read position: 17:25,636,075\n",
      "INFO\t2016-11-30 11:58:56\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:30s.  Time for last 1,000,000:    4s.  Last read position: 19:1,598,135\n",
      "INFO\t2016-11-30 11:59:01\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:35s.  Time for last 1,000,000:    4s.  Last read position: 12:111,085,634\n",
      "INFO\t2016-11-30 11:59:06\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:40s.  Time for last 1,000,000:    4s.  Last read position: Y:8,979,507\n",
      "INFO\t2016-11-30 11:59:11\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:45s.  Time for last 1,000,000:    4s.  Last read position: 17:48,218,165\n",
      "INFO\t2016-11-30 11:59:16\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:08:50s.  Time for last 1,000,000:    5s.  Last read position: 6:114,178,718\n",
      "INFO\t2016-11-30 11:59:21\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:08:55s.  Time for last 1,000,000:    5s.  Last read position: 6:31,797,588\n",
      "INFO\t2016-11-30 11:59:26\tAddOrReplaceReadGroups\tProcessed   103,000,000 records.  Elapsed time: 00:09:00s.  Time for last 1,000,000:    4s.  Last read position: 4:141,931,049\n",
      "INFO\t2016-11-30 11:59:33\tAddOrReplaceReadGroups\tProcessed   104,000,000 records.  Elapsed time: 00:09:06s.  Time for last 1,000,000:    6s.  Last read position: 11:102,289,714\n",
      "INFO\t2016-11-30 11:59:38\tAddOrReplaceReadGroups\tProcessed   105,000,000 records.  Elapsed time: 00:09:11s.  Time for last 1,000,000:    5s.  Last read position: 10:129,883,018\n",
      "INFO\t2016-11-30 11:59:43\tAddOrReplaceReadGroups\tProcessed   106,000,000 records.  Elapsed time: 00:09:16s.  Time for last 1,000,000:    4s.  Last read position: 19:40,386,107\n",
      "INFO\t2016-11-30 11:59:49\tAddOrReplaceReadGroups\tProcessed   107,000,000 records.  Elapsed time: 00:09:23s.  Time for last 1,000,000:    6s.  Last read position: 7:128,292,537\n",
      "INFO\t2016-11-30 11:59:54\tAddOrReplaceReadGroups\tProcessed   108,000,000 records.  Elapsed time: 00:09:27s.  Time for last 1,000,000:    4s.  Last read position: 8:74,621,317\n",
      "INFO\t2016-11-30 11:59:58\tAddOrReplaceReadGroups\tProcessed   109,000,000 records.  Elapsed time: 00:09:32s.  Time for last 1,000,000:    4s.  Last read position: 20:30,143,105\n",
      "INFO\t2016-11-30 12:00:03\tAddOrReplaceReadGroups\tProcessed   110,000,000 records.  Elapsed time: 00:09:37s.  Time for last 1,000,000:    5s.  Last read position: 2:219,257,506\n",
      "INFO\t2016-11-30 12:00:10\tAddOrReplaceReadGroups\tProcessed   111,000,000 records.  Elapsed time: 00:09:44s.  Time for last 1,000,000:    6s.  Last read position: 15:74,837,476\n",
      "INFO\t2016-11-30 12:00:15\tAddOrReplaceReadGroups\tProcessed   112,000,000 records.  Elapsed time: 00:09:49s.  Time for last 1,000,000:    4s.  Last read position: 1:33,496,797\n",
      "INFO\t2016-11-30 12:00:20\tAddOrReplaceReadGroups\tProcessed   113,000,000 records.  Elapsed time: 00:09:53s.  Time for last 1,000,000:    4s.  Last read position: 22:17,583,070\n",
      "INFO\t2016-11-30 12:00:25\tAddOrReplaceReadGroups\tProcessed   114,000,000 records.  Elapsed time: 00:09:58s.  Time for last 1,000,000:    5s.  Last read position: 7:7,612,698\n",
      "INFO\t2016-11-30 12:00:29\tAddOrReplaceReadGroups\tProcessed   115,000,000 records.  Elapsed time: 00:10:03s.  Time for last 1,000,000:    4s.  Last read position: 22:38,202,576\n",
      "INFO\t2016-11-30 12:00:34\tAddOrReplaceReadGroups\tProcessed   116,000,000 records.  Elapsed time: 00:10:08s.  Time for last 1,000,000:    4s.  Last read position: 12:54,936,690\n",
      "INFO\t2016-11-30 12:00:39\tAddOrReplaceReadGroups\tProcessed   117,000,000 records.  Elapsed time: 00:10:12s.  Time for last 1,000,000:    4s.  Last read position: 15:51,741,056\n",
      "INFO\t2016-11-30 12:00:44\tAddOrReplaceReadGroups\tProcessed   118,000,000 records.  Elapsed time: 00:10:18s.  Time for last 1,000,000:    5s.  Last read position: X:48,674,457\n",
      "INFO\t2016-11-30 12:00:49\tAddOrReplaceReadGroups\tProcessed   119,000,000 records.  Elapsed time: 00:10:23s.  Time for last 1,000,000:    5s.  Last read position: 12:125,397,342\n",
      "INFO\t2016-11-30 12:00:55\tAddOrReplaceReadGroups\tProcessed   120,000,000 records.  Elapsed time: 00:10:29s.  Time for last 1,000,000:    5s.  Last read position: 3:39,306,143\n",
      "INFO\t2016-11-30 12:01:00\tAddOrReplaceReadGroups\tProcessed   121,000,000 records.  Elapsed time: 00:10:34s.  Time for last 1,000,000:    5s.  Last read position: 3:121,371,378\n",
      "INFO\t2016-11-30 12:01:05\tAddOrReplaceReadGroups\tProcessed   122,000,000 records.  Elapsed time: 00:10:39s.  Time for last 1,000,000:    5s.  Last read position: 2:26,492,848\n",
      "INFO\t2016-11-30 12:01:11\tAddOrReplaceReadGroups\tProcessed   123,000,000 records.  Elapsed time: 00:10:45s.  Time for last 1,000,000:    5s.  Last read position: 12:89,760,492\n",
      "[Wed Nov 30 12:09:23 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 18.96 minutes.\n",
      "Runtime.totalMemory()=10331619328\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P17Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P17rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 12:09:24 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P17Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P17rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 12:09:24 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 12:09:24\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 12:09:28\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 7:72,413,903\n",
      "INFO\t2016-11-30 12:09:34\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: MT:8,755\n",
      "INFO\t2016-11-30 12:09:41\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    6s.  Last read position: 1:569,297\n",
      "INFO\t2016-11-30 12:09:46\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:22s.  Time for last 1,000,000:    5s.  Last read position: 3:96,336,779\n",
      "INFO\t2016-11-30 12:09:51\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:    4s.  Last read position: 15:79,189,505\n",
      "INFO\t2016-11-30 12:09:56\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 3:114,057,697\n",
      "INFO\t2016-11-30 12:10:02\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:37s.  Time for last 1,000,000:    5s.  Last read position: 1:186,934,583\n",
      "INFO\t2016-11-30 12:10:08\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:44s.  Time for last 1,000,000:    6s.  Last read position: 13:46,926,427\n",
      "INFO\t2016-11-30 12:10:13\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:49s.  Time for last 1,000,000:    5s.  Last read position: 14:54,642,956\n",
      "INFO\t2016-11-30 12:10:18\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:54s.  Time for last 1,000,000:    4s.  Last read position: 19:47,998,837\n",
      "INFO\t2016-11-30 12:10:24\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    5s.  Last read position: 19:3,178,653\n",
      "INFO\t2016-11-30 12:10:29\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    4s.  Last read position: 1:109,780,556\n",
      "INFO\t2016-11-30 12:10:34\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    5s.  Last read position: X:40,985,896\n",
      "INFO\t2016-11-30 12:10:39\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    5s.  Last read position: 12:122,622,985\n",
      "INFO\t2016-11-30 12:10:44\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:19s.  Time for last 1,000,000:    4s.  Last read position: 16:3,786,110\n",
      "INFO\t2016-11-30 12:10:49\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:25s.  Time for last 1,000,000:    5s.  Last read position: GL000220.1:109,684\n",
      "INFO\t2016-11-30 12:10:54\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    4s.  Last read position: 5:76,176,977\n",
      "INFO\t2016-11-30 12:10:59\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    4s.  Last read position: 19:200,465\n",
      "INFO\t2016-11-30 12:11:04\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:40s.  Time for last 1,000,000:    4s.  Last read position: 5:138,891,910\n",
      "INFO\t2016-11-30 12:11:10\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    5s.  Last read position: 8:142,185,453\n",
      "INFO\t2016-11-30 12:11:15\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    4s.  Last read position: 19:46,024,313\n",
      "INFO\t2016-11-30 12:11:19\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    4s.  Last read position: 10:81,272,548\n",
      "INFO\t2016-11-30 12:11:25\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    6s.  Last read position: 19:45,411,113\n",
      "INFO\t2016-11-30 12:11:30\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    4s.  Last read position: 11:48,152,189\n",
      "INFO\t2016-11-30 12:11:35\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 14:66,367,552\n",
      "INFO\t2016-11-30 12:11:43\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:18s.  Time for last 1,000,000:    7s.  Last read position: 14:50,346,219\n",
      "INFO\t2016-11-30 12:11:47\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:23s.  Time for last 1,000,000:    4s.  Last read position: 1:212,911,767\n",
      "INFO\t2016-11-30 12:11:53\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:28s.  Time for last 1,000,000:    5s.  Last read position: 14:102,481,556\n",
      "INFO\t2016-11-30 12:11:58\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:33s.  Time for last 1,000,000:    5s.  Last read position: 12:123,375,063\n",
      "INFO\t2016-11-30 12:12:03\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:38s.  Time for last 1,000,000:    4s.  Last read position: 19:13,231,501\n",
      "INFO\t2016-11-30 12:12:08\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:43s.  Time for last 1,000,000:    4s.  Last read position: 13:41,750,978\n",
      "INFO\t2016-11-30 12:12:13\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:48s.  Time for last 1,000,000:    5s.  Last read position: 9:107,549,842\n",
      "INFO\t2016-11-30 12:12:18\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:54s.  Time for last 1,000,000:    5s.  Last read position: 4:85,035,008\n",
      "INFO\t2016-11-30 12:12:23\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:59s.  Time for last 1,000,000:    5s.  Last read position: 1:79,093,986\n",
      "INFO\t2016-11-30 12:12:28\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:04s.  Time for last 1,000,000:    4s.  Last read position: 16:89,351,770\n",
      "INFO\t2016-11-30 12:12:34\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:10s.  Time for last 1,000,000:    5s.  Last read position: 2:42,559,202\n",
      "INFO\t2016-11-30 12:12:39\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:14s.  Time for last 1,000,000:    4s.  Last read position: 1:225,589,625\n",
      "INFO\t2016-11-30 12:12:44\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:19s.  Time for last 1,000,000:    4s.  Last read position: 17:65,717,595\n",
      "INFO\t2016-11-30 12:12:50\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:26s.  Time for last 1,000,000:    6s.  Last read position: 22:29,686,223\n",
      "INFO\t2016-11-30 12:12:55\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:30s.  Time for last 1,000,000:    4s.  Last read position: 2:170,773,805\n",
      "INFO\t2016-11-30 12:13:00\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:35s.  Time for last 1,000,000:    4s.  Last read position: 16:72,822,727\n",
      "INFO\t2016-11-30 12:13:09\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:45s.  Time for last 1,000,000:    9s.  Last read position: 7:63,814,713\n",
      "INFO\t2016-11-30 12:13:14\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:50s.  Time for last 1,000,000:    4s.  Last read position: 16:9,678,271\n",
      "INFO\t2016-11-30 12:13:19\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:03:55s.  Time for last 1,000,000:    4s.  Last read position: 8:100,753,763\n",
      "INFO\t2016-11-30 12:13:28\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:04:04s.  Time for last 1,000,000:    9s.  Last read position: 15:50,184,371\n",
      "INFO\t2016-11-30 12:13:35\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:04:10s.  Time for last 1,000,000:    6s.  Last read position: 9:124,055,137\n",
      "INFO\t2016-11-30 12:13:40\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:16s.  Time for last 1,000,000:    5s.  Last read position: 13:111,557,826\n",
      "INFO\t2016-11-30 12:13:45\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    4s.  Last read position: 17:4,380,379\n",
      "INFO\t2016-11-30 12:13:50\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:26s.  Time for last 1,000,000:    4s.  Last read position: 6:149,700,810\n",
      "INFO\t2016-11-30 12:13:55\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:30s.  Time for last 1,000,000:    4s.  Last read position: 1:230,209,556\n",
      "INFO\t2016-11-30 12:14:00\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:35s.  Time for last 1,000,000:    4s.  Last read position: 17:17,488,831\n",
      "INFO\t2016-11-30 12:14:05\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:40s.  Time for last 1,000,000:    4s.  Last read position: 19:197,666\n",
      "INFO\t2016-11-30 12:14:09\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:45s.  Time for last 1,000,000:    4s.  Last read position: 8:28,047,280\n",
      "INFO\t2016-11-30 12:14:14\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:50s.  Time for last 1,000,000:    4s.  Last read position: 6:37,002,243\n",
      "INFO\t2016-11-30 12:14:19\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:55s.  Time for last 1,000,000:    4s.  Last read position: 1:48,780,460\n",
      "INFO\t2016-11-30 12:14:24\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:05:00s.  Time for last 1,000,000:    4s.  Last read position: X:64,954,650\n",
      "INFO\t2016-11-30 12:14:29\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:05:05s.  Time for last 1,000,000:    4s.  Last read position: 4:120,214,231\n",
      "INFO\t2016-11-30 12:14:34\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:05:10s.  Time for last 1,000,000:    4s.  Last read position: 11:73,115,881\n",
      "INFO\t2016-11-30 12:14:39\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:15s.  Time for last 1,000,000:    4s.  Last read position: 21:44,330,093\n",
      "INFO\t2016-11-30 12:14:44\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:20s.  Time for last 1,000,000:    5s.  Last read position: 15:22,976,563\n",
      "INFO\t2016-11-30 12:14:49\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:25s.  Time for last 1,000,000:    5s.  Last read position: 10:26,774,606\n",
      "INFO\t2016-11-30 12:14:54\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:30s.  Time for last 1,000,000:    4s.  Last read position: 20:2,637,103\n",
      "INFO\t2016-11-30 12:15:00\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:36s.  Time for last 1,000,000:    5s.  Last read position: 5:148,883,497\n",
      "INFO\t2016-11-30 12:15:05\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:41s.  Time for last 1,000,000:    5s.  Last read position: 3:142,168,161\n",
      "INFO\t2016-11-30 12:15:10\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:46s.  Time for last 1,000,000:    5s.  Last read position: 1:112,020,169\n",
      "INFO\t2016-11-30 12:15:16\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:52s.  Time for last 1,000,000:    6s.  Last read position: 9:120,474,723\n",
      "INFO\t2016-11-30 12:15:21\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:05:57s.  Time for last 1,000,000:    4s.  Last read position: 19:6,682,171\n",
      "INFO\t2016-11-30 12:15:26\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:06:02s.  Time for last 1,000,000:    4s.  Last read position: 14:53,226,128\n",
      "INFO\t2016-11-30 12:15:33\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:06:08s.  Time for last 1,000,000:    6s.  Last read position: 6:109,703,428\n",
      "INFO\t2016-11-30 12:15:38\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:13s.  Time for last 1,000,000:    4s.  Last read position: 19:13,050,934\n",
      "INFO\t2016-11-30 12:15:43\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:18s.  Time for last 1,000,000:    4s.  Last read position: 5:149,456,893\n",
      "INFO\t2016-11-30 12:15:47\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    4s.  Last read position: 10:95,718,651\n",
      "INFO\t2016-11-30 12:15:52\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:28s.  Time for last 1,000,000:    4s.  Last read position: 3:31,621,492\n",
      "INFO\t2016-11-30 12:15:57\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:33s.  Time for last 1,000,000:    4s.  Last read position: 14:90,756,639\n",
      "INFO\t2016-11-30 12:16:02\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:38s.  Time for last 1,000,000:    4s.  Last read position: 3:151,066,183\n",
      "INFO\t2016-11-30 12:16:07\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:42s.  Time for last 1,000,000:    4s.  Last read position: 21:42,780,692\n",
      "INFO\t2016-11-30 12:16:12\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:47s.  Time for last 1,000,000:    4s.  Last read position: 11:65,732,676\n",
      "INFO\t2016-11-30 12:16:17\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:06:52s.  Time for last 1,000,000:    4s.  Last read position: X:153,220,602\n",
      "INFO\t2016-11-30 12:16:22\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:06:57s.  Time for last 1,000,000:    4s.  Last read position: 6:31,784,800\n",
      "INFO\t2016-11-30 12:16:26\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:07:02s.  Time for last 1,000,000:    4s.  Last read position: 1:224,377,924\n",
      "INFO\t2016-11-30 12:16:31\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:07:07s.  Time for last 1,000,000:    4s.  Last read position: 11:57,174,827\n",
      "INFO\t2016-11-30 12:16:36\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:12s.  Time for last 1,000,000:    5s.  Last read position: X:67,752,127\n",
      "INFO\t2016-11-30 12:16:41\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:17s.  Time for last 1,000,000:    4s.  Last read position: 5:77,783,764\n",
      "INFO\t2016-11-30 12:16:46\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:22s.  Time for last 1,000,000:    4s.  Last read position: 7:5,568,312\n",
      "INFO\t2016-11-30 12:16:51\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:27s.  Time for last 1,000,000:    5s.  Last read position: 17:74,344,644\n",
      "INFO\t2016-11-30 12:16:56\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:32s.  Time for last 1,000,000:    4s.  Last read position: 9:103,105,505\n",
      "INFO\t2016-11-30 12:17:01\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:37s.  Time for last 1,000,000:    4s.  Last read position: 17:7,749,556\n",
      "INFO\t2016-11-30 12:17:09\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:44s.  Time for last 1,000,000:    7s.  Last read position: 7:30,661,952\n",
      "INFO\t2016-11-30 12:17:14\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:49s.  Time for last 1,000,000:    4s.  Last read position: 12:64,538,849\n",
      "INFO\t2016-11-30 12:17:19\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:07:54s.  Time for last 1,000,000:    4s.  Last read position: 19:36,640,513\n",
      "INFO\t2016-11-30 12:17:27\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:08:03s.  Time for last 1,000,000:    8s.  Last read position: 9:6,756,488\n",
      "INFO\t2016-11-30 12:17:32\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:08:07s.  Time for last 1,000,000:    4s.  Last read position: 18:9,255,468\n",
      "INFO\t2016-11-30 12:17:37\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:12s.  Time for last 1,000,000:    4s.  Last read position: 10:75,545,624\n",
      "INFO\t2016-11-30 12:17:41\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:17s.  Time for last 1,000,000:    4s.  Last read position: 14:64,380,383\n",
      "INFO\t2016-11-30 12:17:46\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:22s.  Time for last 1,000,000:    4s.  Last read position: 4:88,901,212\n",
      "INFO\t2016-11-30 12:17:51\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:27s.  Time for last 1,000,000:    4s.  Last read position: 16:68,224,897\n",
      "INFO\t2016-11-30 12:17:56\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:32s.  Time for last 1,000,000:    5s.  Last read position: 1:38,462,794\n",
      "INFO\t2016-11-30 12:18:01\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:37s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,858\n",
      "INFO\t2016-11-30 12:18:06\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:42s.  Time for last 1,000,000:    4s.  Last read position: 3:50,049,852\n",
      "INFO\t2016-11-30 12:18:11\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:46s.  Time for last 1,000,000:    4s.  Last read position: 1:22,974,107\n",
      "INFO\t2016-11-30 12:18:16\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:08:51s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,894\n",
      "INFO\t2016-11-30 12:18:20\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:08:56s.  Time for last 1,000,000:    4s.  Last read position: 11:64,973,408\n",
      "INFO\t2016-11-30 12:18:26\tAddOrReplaceReadGroups\tProcessed   103,000,000 records.  Elapsed time: 00:09:01s.  Time for last 1,000,000:    5s.  Last read position: 13:23,270,104\n",
      "INFO\t2016-11-30 12:18:31\tAddOrReplaceReadGroups\tProcessed   104,000,000 records.  Elapsed time: 00:09:06s.  Time for last 1,000,000:    4s.  Last read position: MT:11,838\n",
      "INFO\t2016-11-30 12:18:35\tAddOrReplaceReadGroups\tProcessed   105,000,000 records.  Elapsed time: 00:09:11s.  Time for last 1,000,000:    4s.  Last read position: X:24,096,389\n",
      "INFO\t2016-11-30 12:18:41\tAddOrReplaceReadGroups\tProcessed   106,000,000 records.  Elapsed time: 00:09:17s.  Time for last 1,000,000:    5s.  Last read position: 1:1,596,164\n",
      "INFO\t2016-11-30 12:18:46\tAddOrReplaceReadGroups\tProcessed   107,000,000 records.  Elapsed time: 00:09:21s.  Time for last 1,000,000:    4s.  Last read position: X:147,134,150\n",
      "INFO\t2016-11-30 12:18:51\tAddOrReplaceReadGroups\tProcessed   108,000,000 records.  Elapsed time: 00:09:26s.  Time for last 1,000,000:    4s.  Last read position: 8:82,434,228\n",
      "INFO\t2016-11-30 12:18:57\tAddOrReplaceReadGroups\tProcessed   109,000,000 records.  Elapsed time: 00:09:32s.  Time for last 1,000,000:    6s.  Last read position: 11:61,732,108\n",
      "INFO\t2016-11-30 12:19:01\tAddOrReplaceReadGroups\tProcessed   110,000,000 records.  Elapsed time: 00:09:37s.  Time for last 1,000,000:    4s.  Last read position: 5:17,353,877\n",
      "INFO\t2016-11-30 12:19:06\tAddOrReplaceReadGroups\tProcessed   111,000,000 records.  Elapsed time: 00:09:42s.  Time for last 1,000,000:    4s.  Last read position: 6:30,711,821\n",
      "INFO\t2016-11-30 12:19:13\tAddOrReplaceReadGroups\tProcessed   112,000,000 records.  Elapsed time: 00:09:48s.  Time for last 1,000,000:    6s.  Last read position: 20:33,724,593\n",
      "INFO\t2016-11-30 12:19:18\tAddOrReplaceReadGroups\tProcessed   113,000,000 records.  Elapsed time: 00:09:53s.  Time for last 1,000,000:    4s.  Last read position: 12:26,564,277\n",
      "INFO\t2016-11-30 12:19:22\tAddOrReplaceReadGroups\tProcessed   114,000,000 records.  Elapsed time: 00:09:58s.  Time for last 1,000,000:    4s.  Last read position: 19:50,381,758\n",
      "INFO\t2016-11-30 12:19:27\tAddOrReplaceReadGroups\tProcessed   115,000,000 records.  Elapsed time: 00:10:03s.  Time for last 1,000,000:    4s.  Last read position: 17:78,355,521\n",
      "INFO\t2016-11-30 12:19:32\tAddOrReplaceReadGroups\tProcessed   116,000,000 records.  Elapsed time: 00:10:08s.  Time for last 1,000,000:    4s.  Last read position: 7:74,701,531\n",
      "INFO\t2016-11-30 12:19:37\tAddOrReplaceReadGroups\tProcessed   117,000,000 records.  Elapsed time: 00:10:13s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,304\n",
      "INFO\t2016-11-30 12:19:42\tAddOrReplaceReadGroups\tProcessed   118,000,000 records.  Elapsed time: 00:10:17s.  Time for last 1,000,000:    4s.  Last read position: 1:1,691,676\n",
      "INFO\t2016-11-30 12:19:47\tAddOrReplaceReadGroups\tProcessed   119,000,000 records.  Elapsed time: 00:10:22s.  Time for last 1,000,000:    4s.  Last read position: 4:185,678,015\n",
      "INFO\t2016-11-30 12:19:51\tAddOrReplaceReadGroups\tProcessed   120,000,000 records.  Elapsed time: 00:10:27s.  Time for last 1,000,000:    4s.  Last read position: 9:15,527,611\n",
      "INFO\t2016-11-30 12:19:56\tAddOrReplaceReadGroups\tProcessed   121,000,000 records.  Elapsed time: 00:10:32s.  Time for last 1,000,000:    4s.  Last read position: 5:74,889,805\n",
      "INFO\t2016-11-30 12:20:01\tAddOrReplaceReadGroups\tProcessed   122,000,000 records.  Elapsed time: 00:10:37s.  Time for last 1,000,000:    5s.  Last read position: 15:51,350,281\n",
      "INFO\t2016-11-30 12:20:06\tAddOrReplaceReadGroups\tProcessed   123,000,000 records.  Elapsed time: 00:10:42s.  Time for last 1,000,000:    4s.  Last read position: 18:11,894,005\n",
      "INFO\t2016-11-30 12:20:11\tAddOrReplaceReadGroups\tProcessed   124,000,000 records.  Elapsed time: 00:10:47s.  Time for last 1,000,000:    4s.  Last read position: 3:146,245,928\n",
      "INFO\t2016-11-30 12:20:17\tAddOrReplaceReadGroups\tProcessed   125,000,000 records.  Elapsed time: 00:10:52s.  Time for last 1,000,000:    5s.  Last read position: 5:108,671,801\n",
      "INFO\t2016-11-30 12:20:21\tAddOrReplaceReadGroups\tProcessed   126,000,000 records.  Elapsed time: 00:10:57s.  Time for last 1,000,000:    4s.  Last read position: 4:77,968,897\n",
      "INFO\t2016-11-30 12:20:26\tAddOrReplaceReadGroups\tProcessed   127,000,000 records.  Elapsed time: 00:11:02s.  Time for last 1,000,000:    4s.  Last read position: 21:45,472,353\n",
      "INFO\t2016-11-30 12:20:32\tAddOrReplaceReadGroups\tProcessed   128,000,000 records.  Elapsed time: 00:11:08s.  Time for last 1,000,000:    6s.  Last read position: 13:31,338,152\n",
      "INFO\t2016-11-30 12:20:37\tAddOrReplaceReadGroups\tProcessed   129,000,000 records.  Elapsed time: 00:11:13s.  Time for last 1,000,000:    4s.  Last read position: X:23,802,923\n",
      "INFO\t2016-11-30 12:20:42\tAddOrReplaceReadGroups\tProcessed   130,000,000 records.  Elapsed time: 00:11:18s.  Time for last 1,000,000:    4s.  Last read position: 4:185,309,110\n",
      "INFO\t2016-11-30 12:20:49\tAddOrReplaceReadGroups\tProcessed   131,000,000 records.  Elapsed time: 00:11:25s.  Time for last 1,000,000:    7s.  Last read position: 1:31,344,447\n",
      "INFO\t2016-11-30 12:20:54\tAddOrReplaceReadGroups\tProcessed   132,000,000 records.  Elapsed time: 00:11:30s.  Time for last 1,000,000:    4s.  Last read position: 17:62,500,149\n",
      "INFO\t2016-11-30 12:20:59\tAddOrReplaceReadGroups\tProcessed   133,000,000 records.  Elapsed time: 00:11:34s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,806\n",
      "INFO\t2016-11-30 12:21:07\tAddOrReplaceReadGroups\tProcessed   134,000,000 records.  Elapsed time: 00:11:43s.  Time for last 1,000,000:    8s.  Last read position: 11:82,878,253\n",
      "INFO\t2016-11-30 12:21:12\tAddOrReplaceReadGroups\tProcessed   135,000,000 records.  Elapsed time: 00:11:48s.  Time for last 1,000,000:    4s.  Last read position: 2:178,085,125\n",
      "INFO\t2016-11-30 12:21:17\tAddOrReplaceReadGroups\tProcessed   136,000,000 records.  Elapsed time: 00:11:52s.  Time for last 1,000,000:    4s.  Last read position: 6:160,103,114\n",
      "INFO\t2016-11-30 12:21:21\tAddOrReplaceReadGroups\tProcessed   137,000,000 records.  Elapsed time: 00:11:57s.  Time for last 1,000,000:    4s.  Last read position: 14:32,583,157\n",
      "INFO\t2016-11-30 12:21:26\tAddOrReplaceReadGroups\tProcessed   138,000,000 records.  Elapsed time: 00:12:02s.  Time for last 1,000,000:    4s.  Last read position: 2:65,482,792\n",
      "INFO\t2016-11-30 12:21:31\tAddOrReplaceReadGroups\tProcessed   139,000,000 records.  Elapsed time: 00:12:07s.  Time for last 1,000,000:    4s.  Last read position: 8:11,704,615\n",
      "INFO\t2016-11-30 12:21:36\tAddOrReplaceReadGroups\tProcessed   140,000,000 records.  Elapsed time: 00:12:12s.  Time for last 1,000,000:    4s.  Last read position: 13:73,284,294\n",
      "INFO\t2016-11-30 12:21:41\tAddOrReplaceReadGroups\tProcessed   141,000,000 records.  Elapsed time: 00:12:16s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,580\n",
      "INFO\t2016-11-30 12:21:46\tAddOrReplaceReadGroups\tProcessed   142,000,000 records.  Elapsed time: 00:12:21s.  Time for last 1,000,000:    4s.  Last read position: 9:35,704,789\n",
      "INFO\t2016-11-30 12:21:50\tAddOrReplaceReadGroups\tProcessed   143,000,000 records.  Elapsed time: 00:12:26s.  Time for last 1,000,000:    4s.  Last read position: 19:11,031,763\n",
      "INFO\t2016-11-30 12:21:55\tAddOrReplaceReadGroups\tProcessed   144,000,000 records.  Elapsed time: 00:12:31s.  Time for last 1,000,000:    4s.  Last read position: 16:31,393,934\n",
      "INFO\t2016-11-30 12:22:00\tAddOrReplaceReadGroups\tProcessed   145,000,000 records.  Elapsed time: 00:12:36s.  Time for last 1,000,000:    4s.  Last read position: 11:1,774,458\n",
      "INFO\t2016-11-30 12:22:05\tAddOrReplaceReadGroups\tProcessed   146,000,000 records.  Elapsed time: 00:12:41s.  Time for last 1,000,000:    5s.  Last read position: 7:128,231,917\n",
      "INFO\t2016-11-30 12:22:10\tAddOrReplaceReadGroups\tProcessed   147,000,000 records.  Elapsed time: 00:12:46s.  Time for last 1,000,000:    4s.  Last read position: 9:19,378,380\n",
      "INFO\t2016-11-30 12:22:15\tAddOrReplaceReadGroups\tProcessed   148,000,000 records.  Elapsed time: 00:12:51s.  Time for last 1,000,000:    4s.  Last read position: 10:95,089,539\n",
      "INFO\t2016-11-30 12:22:21\tAddOrReplaceReadGroups\tProcessed   149,000,000 records.  Elapsed time: 00:12:57s.  Time for last 1,000,000:    6s.  Last read position: 14:57,675,187\n",
      "INFO\t2016-11-30 12:22:26\tAddOrReplaceReadGroups\tProcessed   150,000,000 records.  Elapsed time: 00:13:02s.  Time for last 1,000,000:    4s.  Last read position: 19:1,106,686\n",
      "INFO\t2016-11-30 12:22:31\tAddOrReplaceReadGroups\tProcessed   151,000,000 records.  Elapsed time: 00:13:07s.  Time for last 1,000,000:    4s.  Last read position: 8:19,700,379\n",
      "INFO\t2016-11-30 12:22:38\tAddOrReplaceReadGroups\tProcessed   152,000,000 records.  Elapsed time: 00:13:14s.  Time for last 1,000,000:    6s.  Last read position: 4:88,902,785\n",
      "INFO\t2016-11-30 12:22:43\tAddOrReplaceReadGroups\tProcessed   153,000,000 records.  Elapsed time: 00:13:18s.  Time for last 1,000,000:    4s.  Last read position: 8:20,078,754\n",
      "INFO\t2016-11-30 12:22:48\tAddOrReplaceReadGroups\tProcessed   154,000,000 records.  Elapsed time: 00:13:23s.  Time for last 1,000,000:    4s.  Last read position: 10:105,761,935\n",
      "INFO\t2016-11-30 12:22:55\tAddOrReplaceReadGroups\tProcessed   155,000,000 records.  Elapsed time: 00:13:31s.  Time for last 1,000,000:    7s.  Last read position: 9:19,700\n",
      "INFO\t2016-11-30 12:23:00\tAddOrReplaceReadGroups\tProcessed   156,000,000 records.  Elapsed time: 00:13:36s.  Time for last 1,000,000:    4s.  Last read position: 4:88,901,558\n",
      "INFO\t2016-11-30 12:23:05\tAddOrReplaceReadGroups\tProcessed   157,000,000 records.  Elapsed time: 00:13:41s.  Time for last 1,000,000:    4s.  Last read position: 3:180,685,956\n",
      "INFO\t2016-11-30 12:23:10\tAddOrReplaceReadGroups\tProcessed   158,000,000 records.  Elapsed time: 00:13:45s.  Time for last 1,000,000:    4s.  Last read position: 2:36,777,705\n",
      "INFO\t2016-11-30 12:23:15\tAddOrReplaceReadGroups\tProcessed   159,000,000 records.  Elapsed time: 00:13:50s.  Time for last 1,000,000:    4s.  Last read position: 20:30,253,075\n",
      "INFO\t2016-11-30 12:23:20\tAddOrReplaceReadGroups\tProcessed   160,000,000 records.  Elapsed time: 00:13:55s.  Time for last 1,000,000:    4s.  Last read position: 3:136,665,522\n",
      "INFO\t2016-11-30 12:23:24\tAddOrReplaceReadGroups\tProcessed   161,000,000 records.  Elapsed time: 00:14:00s.  Time for last 1,000,000:    4s.  Last read position: 10:73,578,800\n",
      "INFO\t2016-11-30 12:23:29\tAddOrReplaceReadGroups\tProcessed   162,000,000 records.  Elapsed time: 00:14:05s.  Time for last 1,000,000:    4s.  Last read position: 18:60,244,366\n",
      "INFO\t2016-11-30 12:23:34\tAddOrReplaceReadGroups\tProcessed   163,000,000 records.  Elapsed time: 00:14:10s.  Time for last 1,000,000:    4s.  Last read position: 11:77,445,513\n",
      "INFO\t2016-11-30 12:23:39\tAddOrReplaceReadGroups\tProcessed   164,000,000 records.  Elapsed time: 00:14:15s.  Time for last 1,000,000:    4s.  Last read position: MT:7,879\n",
      "INFO\t2016-11-30 12:23:44\tAddOrReplaceReadGroups\tProcessed   165,000,000 records.  Elapsed time: 00:14:20s.  Time for last 1,000,000:    4s.  Last read position: 19:49,468,809\n",
      "INFO\t2016-11-30 12:23:49\tAddOrReplaceReadGroups\tProcessed   166,000,000 records.  Elapsed time: 00:14:25s.  Time for last 1,000,000:    4s.  Last read position: 3:124,482,415\n",
      "INFO\t2016-11-30 12:23:54\tAddOrReplaceReadGroups\tProcessed   167,000,000 records.  Elapsed time: 00:14:29s.  Time for last 1,000,000:    4s.  Last read position: 9:123,550,270\n",
      "INFO\t2016-11-30 12:23:59\tAddOrReplaceReadGroups\tProcessed   168,000,000 records.  Elapsed time: 00:14:35s.  Time for last 1,000,000:    5s.  Last read position: 1:160,062,195\n",
      "INFO\t2016-11-30 12:24:04\tAddOrReplaceReadGroups\tProcessed   169,000,000 records.  Elapsed time: 00:14:39s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,811\n",
      "INFO\t2016-11-30 12:24:09\tAddOrReplaceReadGroups\tProcessed   170,000,000 records.  Elapsed time: 00:14:44s.  Time for last 1,000,000:    4s.  Last read position: 17:78,341,883\n",
      "INFO\t2016-11-30 12:24:16\tAddOrReplaceReadGroups\tProcessed   171,000,000 records.  Elapsed time: 00:14:52s.  Time for last 1,000,000:    7s.  Last read position: 2:131,414,659\n",
      "INFO\t2016-11-30 12:24:21\tAddOrReplaceReadGroups\tProcessed   172,000,000 records.  Elapsed time: 00:14:57s.  Time for last 1,000,000:    4s.  Last read position: 12:56,511,299\n",
      "INFO\t2016-11-30 12:24:26\tAddOrReplaceReadGroups\tProcessed   173,000,000 records.  Elapsed time: 00:15:01s.  Time for last 1,000,000:    4s.  Last read position: 3:49,052,950\n",
      "INFO\t2016-11-30 12:24:32\tAddOrReplaceReadGroups\tProcessed   174,000,000 records.  Elapsed time: 00:15:08s.  Time for last 1,000,000:    6s.  Last read position: 7:48,128,419\n",
      "INFO\t2016-11-30 12:24:37\tAddOrReplaceReadGroups\tProcessed   175,000,000 records.  Elapsed time: 00:15:13s.  Time for last 1,000,000:    4s.  Last read position: 17:74,730,360\n",
      "INFO\t2016-11-30 12:24:42\tAddOrReplaceReadGroups\tProcessed   176,000,000 records.  Elapsed time: 00:15:18s.  Time for last 1,000,000:    4s.  Last read position: 7:32,055\n",
      "INFO\t2016-11-30 12:24:49\tAddOrReplaceReadGroups\tProcessed   177,000,000 records.  Elapsed time: 00:15:24s.  Time for last 1,000,000:    6s.  Last read position: 2:26,416,589\n",
      "INFO\t2016-11-30 12:24:53\tAddOrReplaceReadGroups\tProcessed   178,000,000 records.  Elapsed time: 00:15:29s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,054\n",
      "INFO\t2016-11-30 12:24:58\tAddOrReplaceReadGroups\tProcessed   179,000,000 records.  Elapsed time: 00:15:34s.  Time for last 1,000,000:    4s.  Last read position: 9:124,086,855\n",
      "INFO\t2016-11-30 12:25:03\tAddOrReplaceReadGroups\tProcessed   180,000,000 records.  Elapsed time: 00:15:39s.  Time for last 1,000,000:    4s.  Last read position: 3:33,184,004\n",
      "INFO\t2016-11-30 12:25:08\tAddOrReplaceReadGroups\tProcessed   181,000,000 records.  Elapsed time: 00:15:44s.  Time for last 1,000,000:    4s.  Last read position: 11:67,796,585\n",
      "INFO\t2016-11-30 12:25:13\tAddOrReplaceReadGroups\tProcessed   182,000,000 records.  Elapsed time: 00:15:49s.  Time for last 1,000,000:    4s.  Last read position: 3:127,410,013\n",
      "INFO\t2016-11-30 12:25:18\tAddOrReplaceReadGroups\tProcessed   183,000,000 records.  Elapsed time: 00:15:53s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,147\n",
      "INFO\t2016-11-30 12:25:23\tAddOrReplaceReadGroups\tProcessed   184,000,000 records.  Elapsed time: 00:15:58s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,863\n",
      "INFO\t2016-11-30 12:25:27\tAddOrReplaceReadGroups\tProcessed   185,000,000 records.  Elapsed time: 00:16:03s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,914\n",
      "INFO\t2016-11-30 12:25:32\tAddOrReplaceReadGroups\tProcessed   186,000,000 records.  Elapsed time: 00:16:08s.  Time for last 1,000,000:    4s.  Last read position: 12:51,392,975\n",
      "INFO\t2016-11-30 12:25:37\tAddOrReplaceReadGroups\tProcessed   187,000,000 records.  Elapsed time: 00:16:13s.  Time for last 1,000,000:    4s.  Last read position: 11:65,622,458\n",
      "INFO\t2016-11-30 12:25:42\tAddOrReplaceReadGroups\tProcessed   188,000,000 records.  Elapsed time: 00:16:18s.  Time for last 1,000,000:    4s.  Last read position: 15:69,745,307\n",
      "INFO\t2016-11-30 12:25:47\tAddOrReplaceReadGroups\tProcessed   189,000,000 records.  Elapsed time: 00:16:23s.  Time for last 1,000,000:    5s.  Last read position: 8:145,650,416\n",
      "INFO\t2016-11-30 12:25:52\tAddOrReplaceReadGroups\tProcessed   190,000,000 records.  Elapsed time: 00:16:28s.  Time for last 1,000,000:    4s.  Last read position: 6:50,880,318\n",
      "INFO\t2016-11-30 12:25:57\tAddOrReplaceReadGroups\tProcessed   191,000,000 records.  Elapsed time: 00:16:33s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,805\n",
      "INFO\t2016-11-30 12:26:03\tAddOrReplaceReadGroups\tProcessed   192,000,000 records.  Elapsed time: 00:16:39s.  Time for last 1,000,000:    5s.  Last read position: 6:116,360,635\n",
      "INFO\t2016-11-30 12:26:08\tAddOrReplaceReadGroups\tProcessed   193,000,000 records.  Elapsed time: 00:16:44s.  Time for last 1,000,000:    4s.  Last read position: 9:130,616,797\n",
      "INFO\t2016-11-30 12:26:13\tAddOrReplaceReadGroups\tProcessed   194,000,000 records.  Elapsed time: 00:16:49s.  Time for last 1,000,000:    4s.  Last read position: 6:134,494,666\n",
      "INFO\t2016-11-30 12:26:19\tAddOrReplaceReadGroups\tProcessed   195,000,000 records.  Elapsed time: 00:16:55s.  Time for last 1,000,000:    6s.  Last read position: 20:4,004,629\n",
      "INFO\t2016-11-30 12:26:24\tAddOrReplaceReadGroups\tProcessed   196,000,000 records.  Elapsed time: 00:17:00s.  Time for last 1,000,000:    4s.  Last read position: 2:27,615,827\n",
      "INFO\t2016-11-30 12:26:29\tAddOrReplaceReadGroups\tProcessed   197,000,000 records.  Elapsed time: 00:17:05s.  Time for last 1,000,000:    4s.  Last read position: 15:90,610,943\n",
      "INFO\t2016-11-30 12:26:36\tAddOrReplaceReadGroups\tProcessed   198,000,000 records.  Elapsed time: 00:17:11s.  Time for last 1,000,000:    6s.  Last read position: 19:8,615,552\n",
      "INFO\t2016-11-30 12:26:41\tAddOrReplaceReadGroups\tProcessed   199,000,000 records.  Elapsed time: 00:17:16s.  Time for last 1,000,000:    4s.  Last read position: 5:77,081,501\n",
      "INFO\t2016-11-30 12:26:46\tAddOrReplaceReadGroups\tProcessed   200,000,000 records.  Elapsed time: 00:17:21s.  Time for last 1,000,000:    4s.  Last read position: 21:46,307,132\n",
      "INFO\t2016-11-30 12:26:53\tAddOrReplaceReadGroups\tProcessed   201,000,000 records.  Elapsed time: 00:17:29s.  Time for last 1,000,000:    7s.  Last read position: 12:65,109,660\n",
      "INFO\t2016-11-30 12:26:58\tAddOrReplaceReadGroups\tProcessed   202,000,000 records.  Elapsed time: 00:17:34s.  Time for last 1,000,000:    4s.  Last read position: 19:49,470,055\n",
      "INFO\t2016-11-30 12:27:03\tAddOrReplaceReadGroups\tProcessed   203,000,000 records.  Elapsed time: 00:17:39s.  Time for last 1,000,000:    4s.  Last read position: 2:38,793,118\n",
      "INFO\t2016-11-30 12:27:08\tAddOrReplaceReadGroups\tProcessed   204,000,000 records.  Elapsed time: 00:17:44s.  Time for last 1,000,000:    4s.  Last read position: 7:100,735,703\n",
      "INFO\t2016-11-30 12:27:13\tAddOrReplaceReadGroups\tProcessed   205,000,000 records.  Elapsed time: 00:17:49s.  Time for last 1,000,000:    5s.  Last read position: 15:52,161,444\n",
      "INFO\t2016-11-30 12:27:18\tAddOrReplaceReadGroups\tProcessed   206,000,000 records.  Elapsed time: 00:17:54s.  Time for last 1,000,000:    4s.  Last read position: GL000220.1:159,974\n",
      "INFO\t2016-11-30 12:27:23\tAddOrReplaceReadGroups\tProcessed   207,000,000 records.  Elapsed time: 00:17:58s.  Time for last 1,000,000:    4s.  Last read position: 3:49,358,704\n",
      "INFO\t2016-11-30 12:27:28\tAddOrReplaceReadGroups\tProcessed   208,000,000 records.  Elapsed time: 00:18:03s.  Time for last 1,000,000:    4s.  Last read position: 16:18,544,344\n",
      "INFO\t2016-11-30 12:27:33\tAddOrReplaceReadGroups\tProcessed   209,000,000 records.  Elapsed time: 00:18:08s.  Time for last 1,000,000:    4s.  Last read position: 15:75,644,674\n",
      "INFO\t2016-11-30 12:27:37\tAddOrReplaceReadGroups\tProcessed   210,000,000 records.  Elapsed time: 00:18:13s.  Time for last 1,000,000:    4s.  Last read position: 10:106,027,028\n",
      "INFO\t2016-11-30 12:27:42\tAddOrReplaceReadGroups\tProcessed   211,000,000 records.  Elapsed time: 00:18:18s.  Time for last 1,000,000:    4s.  Last read position: 20:44,641,893\n",
      "INFO\t2016-11-30 12:27:47\tAddOrReplaceReadGroups\tProcessed   212,000,000 records.  Elapsed time: 00:18:23s.  Time for last 1,000,000:    4s.  Last read position: 12:1,018,357\n",
      "INFO\t2016-11-30 12:27:52\tAddOrReplaceReadGroups\tProcessed   213,000,000 records.  Elapsed time: 00:18:28s.  Time for last 1,000,000:    4s.  Last read position: 8:116,631,578\n",
      "INFO\t2016-11-30 12:27:57\tAddOrReplaceReadGroups\tProcessed   214,000,000 records.  Elapsed time: 00:18:33s.  Time for last 1,000,000:    5s.  Last read position: X:153,594,553\n",
      "INFO\t2016-11-30 12:28:02\tAddOrReplaceReadGroups\tProcessed   215,000,000 records.  Elapsed time: 00:18:38s.  Time for last 1,000,000:    4s.  Last read position: 4:88,901,559\n",
      "INFO\t2016-11-30 12:28:07\tAddOrReplaceReadGroups\tProcessed   216,000,000 records.  Elapsed time: 00:18:43s.  Time for last 1,000,000:    4s.  Last read position: 1:15,756,115\n",
      "INFO\t2016-11-30 12:28:13\tAddOrReplaceReadGroups\tProcessed   217,000,000 records.  Elapsed time: 00:18:48s.  Time for last 1,000,000:    5s.  Last read position: 11:77,445,630\n",
      "INFO\t2016-11-30 12:28:17\tAddOrReplaceReadGroups\tProcessed   218,000,000 records.  Elapsed time: 00:18:53s.  Time for last 1,000,000:    4s.  Last read position: 8:17,869,188\n",
      "INFO\t2016-11-30 12:28:23\tAddOrReplaceReadGroups\tProcessed   219,000,000 records.  Elapsed time: 00:18:58s.  Time for last 1,000,000:    5s.  Last read position: 7:5,567,076\n",
      "INFO\t2016-11-30 12:28:29\tAddOrReplaceReadGroups\tProcessed   220,000,000 records.  Elapsed time: 00:19:04s.  Time for last 1,000,000:    6s.  Last read position: 9:35,971,782\n",
      "INFO\t2016-11-30 12:28:33\tAddOrReplaceReadGroups\tProcessed   221,000,000 records.  Elapsed time: 00:19:09s.  Time for last 1,000,000:    4s.  Last read position: 12:104,370,759\n",
      "INFO\t2016-11-30 12:28:38\tAddOrReplaceReadGroups\tProcessed   222,000,000 records.  Elapsed time: 00:19:14s.  Time for last 1,000,000:    4s.  Last read position: 11:62,125,690\n",
      "INFO\t2016-11-30 12:28:44\tAddOrReplaceReadGroups\tProcessed   223,000,000 records.  Elapsed time: 00:19:20s.  Time for last 1,000,000:    5s.  Last read position: 12:131,357,439\n",
      "INFO\t2016-11-30 12:28:49\tAddOrReplaceReadGroups\tProcessed   224,000,000 records.  Elapsed time: 00:19:24s.  Time for last 1,000,000:    4s.  Last read position: 19:40,357,672\n",
      "INFO\t2016-11-30 12:28:54\tAddOrReplaceReadGroups\tProcessed   225,000,000 records.  Elapsed time: 00:19:29s.  Time for last 1,000,000:    4s.  Last read position: 1:53,407,507\n",
      "INFO\t2016-11-30 12:28:58\tAddOrReplaceReadGroups\tProcessed   226,000,000 records.  Elapsed time: 00:19:34s.  Time for last 1,000,000:    4s.  Last read position: 20:61,479,507\n",
      "INFO\t2016-11-30 12:29:03\tAddOrReplaceReadGroups\tProcessed   227,000,000 records.  Elapsed time: 00:19:39s.  Time for last 1,000,000:    4s.  Last read position: 11:47,491,373\n",
      "INFO\t2016-11-30 12:29:08\tAddOrReplaceReadGroups\tProcessed   228,000,000 records.  Elapsed time: 00:19:44s.  Time for last 1,000,000:    4s.  Last read position: 11:61,724,397\n",
      "INFO\t2016-11-30 12:29:13\tAddOrReplaceReadGroups\tProcessed   229,000,000 records.  Elapsed time: 00:19:49s.  Time for last 1,000,000:    4s.  Last read position: MT:5,262\n",
      "INFO\t2016-11-30 12:29:18\tAddOrReplaceReadGroups\tProcessed   230,000,000 records.  Elapsed time: 00:19:54s.  Time for last 1,000,000:    4s.  Last read position: 11:61,734,936\n",
      "INFO\t2016-11-30 12:29:23\tAddOrReplaceReadGroups\tProcessed   231,000,000 records.  Elapsed time: 00:19:58s.  Time for last 1,000,000:    4s.  Last read position: 10:6,275,258\n",
      "INFO\t2016-11-30 12:29:28\tAddOrReplaceReadGroups\tProcessed   232,000,000 records.  Elapsed time: 00:20:04s.  Time for last 1,000,000:    5s.  Last read position: 1:567,496\n",
      "INFO\t2016-11-30 12:29:33\tAddOrReplaceReadGroups\tProcessed   233,000,000 records.  Elapsed time: 00:20:08s.  Time for last 1,000,000:    4s.  Last read position: X:119,581,814\n",
      "INFO\t2016-11-30 12:29:38\tAddOrReplaceReadGroups\tProcessed   234,000,000 records.  Elapsed time: 00:20:13s.  Time for last 1,000,000:    4s.  Last read position: 1:228,286,512\n",
      "INFO\t2016-11-30 12:29:43\tAddOrReplaceReadGroups\tProcessed   235,000,000 records.  Elapsed time: 00:20:18s.  Time for last 1,000,000:    5s.  Last read position: X:44,654,078\n",
      "INFO\t2016-11-30 12:29:48\tAddOrReplaceReadGroups\tProcessed   236,000,000 records.  Elapsed time: 00:20:23s.  Time for last 1,000,000:    4s.  Last read position: 1:59,248,308\n",
      "INFO\t2016-11-30 12:29:52\tAddOrReplaceReadGroups\tProcessed   237,000,000 records.  Elapsed time: 00:20:28s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,676\n",
      "INFO\t2016-11-30 12:29:59\tAddOrReplaceReadGroups\tProcessed   238,000,000 records.  Elapsed time: 00:20:34s.  Time for last 1,000,000:    6s.  Last read position: 1:36,846,251\n",
      "INFO\t2016-11-30 12:30:03\tAddOrReplaceReadGroups\tProcessed   239,000,000 records.  Elapsed time: 00:20:39s.  Time for last 1,000,000:    4s.  Last read position: 7:2,696,166\n",
      "INFO\t2016-11-30 12:30:08\tAddOrReplaceReadGroups\tProcessed   240,000,000 records.  Elapsed time: 00:20:44s.  Time for last 1,000,000:    4s.  Last read position: 22:36,662,935\n",
      "INFO\t2016-11-30 12:30:15\tAddOrReplaceReadGroups\tProcessed   241,000,000 records.  Elapsed time: 00:20:50s.  Time for last 1,000,000:    6s.  Last read position: 1:160,515,932\n",
      "INFO\t2016-11-30 12:30:19\tAddOrReplaceReadGroups\tProcessed   242,000,000 records.  Elapsed time: 00:20:55s.  Time for last 1,000,000:    4s.  Last read position: 1:17,739,239\n",
      "INFO\t2016-11-30 12:30:24\tAddOrReplaceReadGroups\tProcessed   243,000,000 records.  Elapsed time: 00:21:00s.  Time for last 1,000,000:    4s.  Last read position: 10:99,321,177\n",
      "INFO\t2016-11-30 12:30:31\tAddOrReplaceReadGroups\tProcessed   244,000,000 records.  Elapsed time: 00:21:06s.  Time for last 1,000,000:    6s.  Last read position: 16:2,988,335\n",
      "INFO\t2016-11-30 12:30:35\tAddOrReplaceReadGroups\tProcessed   245,000,000 records.  Elapsed time: 00:21:11s.  Time for last 1,000,000:    4s.  Last read position: 15:79,189,883\n",
      "INFO\t2016-11-30 12:30:40\tAddOrReplaceReadGroups\tProcessed   246,000,000 records.  Elapsed time: 00:21:16s.  Time for last 1,000,000:    4s.  Last read position: 5:17,354,127\n",
      "INFO\t2016-11-30 12:30:45\tAddOrReplaceReadGroups\tProcessed   247,000,000 records.  Elapsed time: 00:21:21s.  Time for last 1,000,000:    4s.  Last read position: 3:51,426,557\n",
      "INFO\t2016-11-30 12:30:50\tAddOrReplaceReadGroups\tProcessed   248,000,000 records.  Elapsed time: 00:21:25s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,156\n",
      "INFO\t2016-11-30 12:30:55\tAddOrReplaceReadGroups\tProcessed   249,000,000 records.  Elapsed time: 00:21:30s.  Time for last 1,000,000:    4s.  Last read position: 21:34,823,058\n",
      "INFO\t2016-11-30 12:30:59\tAddOrReplaceReadGroups\tProcessed   250,000,000 records.  Elapsed time: 00:21:35s.  Time for last 1,000,000:    4s.  Last read position: 1:155,874,556\n",
      "INFO\t2016-11-30 12:31:04\tAddOrReplaceReadGroups\tProcessed   251,000,000 records.  Elapsed time: 00:21:40s.  Time for last 1,000,000:    4s.  Last read position: 11:65,292,690\n",
      "INFO\t2016-11-30 12:31:09\tAddOrReplaceReadGroups\tProcessed   252,000,000 records.  Elapsed time: 00:21:45s.  Time for last 1,000,000:    4s.  Last read position: 20:4,004,755\n",
      "INFO\t2016-11-30 12:31:14\tAddOrReplaceReadGroups\tProcessed   253,000,000 records.  Elapsed time: 00:21:49s.  Time for last 1,000,000:    4s.  Last read position: 5:179,147,455\n",
      "INFO\t2016-11-30 12:31:19\tAddOrReplaceReadGroups\tProcessed   254,000,000 records.  Elapsed time: 00:21:54s.  Time for last 1,000,000:    4s.  Last read position: 16:74,499,603\n",
      "INFO\t2016-11-30 12:31:23\tAddOrReplaceReadGroups\tProcessed   255,000,000 records.  Elapsed time: 00:21:59s.  Time for last 1,000,000:    4s.  Last read position: 10:51,532,251\n",
      "INFO\t2016-11-30 12:31:28\tAddOrReplaceReadGroups\tProcessed   256,000,000 records.  Elapsed time: 00:22:04s.  Time for last 1,000,000:    4s.  Last read position: 18:12,359,944\n",
      "INFO\t2016-11-30 12:31:33\tAddOrReplaceReadGroups\tProcessed   257,000,000 records.  Elapsed time: 00:22:09s.  Time for last 1,000,000:    4s.  Last read position: 2:144,193,202\n",
      "INFO\t2016-11-30 12:31:38\tAddOrReplaceReadGroups\tProcessed   258,000,000 records.  Elapsed time: 00:22:14s.  Time for last 1,000,000:    4s.  Last read position: 2:27,616,032\n",
      "INFO\t2016-11-30 12:31:43\tAddOrReplaceReadGroups\tProcessed   259,000,000 records.  Elapsed time: 00:22:19s.  Time for last 1,000,000:    4s.  Last read position: 22:43,245,448\n",
      "INFO\t2016-11-30 12:31:48\tAddOrReplaceReadGroups\tProcessed   260,000,000 records.  Elapsed time: 00:22:24s.  Time for last 1,000,000:    5s.  Last read position: 19:40,386,650\n",
      "INFO\t2016-11-30 12:31:53\tAddOrReplaceReadGroups\tProcessed   261,000,000 records.  Elapsed time: 00:22:29s.  Time for last 1,000,000:    4s.  Last read position: 17:73,773,874\n",
      "INFO\t2016-11-30 12:31:58\tAddOrReplaceReadGroups\tProcessed   262,000,000 records.  Elapsed time: 00:22:33s.  Time for last 1,000,000:    4s.  Last read position: 16:18,549,866\n",
      "INFO\t2016-11-30 12:32:03\tAddOrReplaceReadGroups\tProcessed   263,000,000 records.  Elapsed time: 00:22:39s.  Time for last 1,000,000:    5s.  Last read position: 11:63,276,318\n",
      "INFO\t2016-11-30 12:32:08\tAddOrReplaceReadGroups\tProcessed   264,000,000 records.  Elapsed time: 00:22:44s.  Time for last 1,000,000:    4s.  Last read position: 8:101,721,825\n",
      "INFO\t2016-11-30 12:32:13\tAddOrReplaceReadGroups\tProcessed   265,000,000 records.  Elapsed time: 00:22:49s.  Time for last 1,000,000:    4s.  Last read position: 19:18,391,278\n",
      "INFO\t2016-11-30 12:32:20\tAddOrReplaceReadGroups\tProcessed   266,000,000 records.  Elapsed time: 00:22:55s.  Time for last 1,000,000:    6s.  Last read position: 2:242,292,514\n",
      "INFO\t2016-11-30 12:32:24\tAddOrReplaceReadGroups\tProcessed   267,000,000 records.  Elapsed time: 00:23:00s.  Time for last 1,000,000:    4s.  Last read position: 4:88,898,247\n",
      "INFO\t2016-11-30 12:32:29\tAddOrReplaceReadGroups\tProcessed   268,000,000 records.  Elapsed time: 00:23:05s.  Time for last 1,000,000:    4s.  Last read position: 7:44,839,334\n",
      "INFO\t2016-11-30 12:32:36\tAddOrReplaceReadGroups\tProcessed   269,000,000 records.  Elapsed time: 00:23:12s.  Time for last 1,000,000:    6s.  Last read position: 4:13,603,754\n",
      "INFO\t2016-11-30 12:32:41\tAddOrReplaceReadGroups\tProcessed   270,000,000 records.  Elapsed time: 00:23:17s.  Time for last 1,000,000:    4s.  Last read position: 5:179,143,197\n",
      "INFO\t2016-11-30 12:32:46\tAddOrReplaceReadGroups\tProcessed   271,000,000 records.  Elapsed time: 00:23:21s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,662\n",
      "INFO\t2016-11-30 12:32:51\tAddOrReplaceReadGroups\tProcessed   272,000,000 records.  Elapsed time: 00:23:27s.  Time for last 1,000,000:    5s.  Last read position: 11:72,396,273\n",
      "INFO\t2016-11-30 12:32:56\tAddOrReplaceReadGroups\tProcessed   273,000,000 records.  Elapsed time: 00:23:32s.  Time for last 1,000,000:    4s.  Last read position: GL000220.1:109,328\n",
      "INFO\t2016-11-30 12:33:01\tAddOrReplaceReadGroups\tProcessed   274,000,000 records.  Elapsed time: 00:23:37s.  Time for last 1,000,000:    4s.  Last read position: 20:57,478,795\n",
      "INFO\t2016-11-30 12:33:06\tAddOrReplaceReadGroups\tProcessed   275,000,000 records.  Elapsed time: 00:23:41s.  Time for last 1,000,000:    4s.  Last read position: 9:130,922,759\n",
      "INFO\t2016-11-30 12:33:11\tAddOrReplaceReadGroups\tProcessed   276,000,000 records.  Elapsed time: 00:23:46s.  Time for last 1,000,000:    4s.  Last read position: 6:44,198,548\n",
      "INFO\t2016-11-30 12:33:16\tAddOrReplaceReadGroups\tProcessed   277,000,000 records.  Elapsed time: 00:23:51s.  Time for last 1,000,000:    5s.  Last read position: 15:83,205,524\n",
      "INFO\t2016-11-30 12:33:20\tAddOrReplaceReadGroups\tProcessed   278,000,000 records.  Elapsed time: 00:23:56s.  Time for last 1,000,000:    4s.  Last read position: 1:183,524,911\n",
      "INFO\t2016-11-30 12:33:25\tAddOrReplaceReadGroups\tProcessed   279,000,000 records.  Elapsed time: 00:24:01s.  Time for last 1,000,000:    4s.  Last read position: 19:6,829,813\n",
      "INFO\t2016-11-30 12:33:30\tAddOrReplaceReadGroups\tProcessed   280,000,000 records.  Elapsed time: 00:24:06s.  Time for last 1,000,000:    5s.  Last read position: 13:98,119,565\n",
      "INFO\t2016-11-30 12:33:35\tAddOrReplaceReadGroups\tProcessed   281,000,000 records.  Elapsed time: 00:24:11s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,759\n",
      "INFO\t2016-11-30 12:33:40\tAddOrReplaceReadGroups\tProcessed   282,000,000 records.  Elapsed time: 00:24:16s.  Time for last 1,000,000:    4s.  Last read position: 2:181,737,467\n",
      "INFO\t2016-11-30 12:33:45\tAddOrReplaceReadGroups\tProcessed   283,000,000 records.  Elapsed time: 00:24:21s.  Time for last 1,000,000:    5s.  Last read position: 17:59,968,904\n",
      "INFO\t2016-11-30 12:33:50\tAddOrReplaceReadGroups\tProcessed   284,000,000 records.  Elapsed time: 00:24:26s.  Time for last 1,000,000:    4s.  Last read position: 3:176,741,889\n",
      "INFO\t2016-11-30 12:33:55\tAddOrReplaceReadGroups\tProcessed   285,000,000 records.  Elapsed time: 00:24:30s.  Time for last 1,000,000:    4s.  Last read position: X:71,492,503\n",
      "INFO\t2016-11-30 12:33:59\tAddOrReplaceReadGroups\tProcessed   286,000,000 records.  Elapsed time: 00:24:35s.  Time for last 1,000,000:    4s.  Last read position: 12:66,622,089\n",
      "INFO\t2016-11-30 12:34:05\tAddOrReplaceReadGroups\tProcessed   287,000,000 records.  Elapsed time: 00:24:41s.  Time for last 1,000,000:    5s.  Last read position: 6:30,692,403\n",
      "INFO\t2016-11-30 12:34:10\tAddOrReplaceReadGroups\tProcessed   288,000,000 records.  Elapsed time: 00:24:46s.  Time for last 1,000,000:    4s.  Last read position: 1:230,415,437\n",
      "INFO\t2016-11-30 12:34:15\tAddOrReplaceReadGroups\tProcessed   289,000,000 records.  Elapsed time: 00:24:50s.  Time for last 1,000,000:    4s.  Last read position: 1:65,344,786\n",
      "INFO\t2016-11-30 12:34:20\tAddOrReplaceReadGroups\tProcessed   290,000,000 records.  Elapsed time: 00:24:56s.  Time for last 1,000,000:    5s.  Last read position: 5:137,843,116\n",
      "INFO\t2016-11-30 12:34:25\tAddOrReplaceReadGroups\tProcessed   291,000,000 records.  Elapsed time: 00:25:01s.  Time for last 1,000,000:    4s.  Last read position: 14:23,315,644\n",
      "INFO\t2016-11-30 12:34:30\tAddOrReplaceReadGroups\tProcessed   292,000,000 records.  Elapsed time: 00:25:05s.  Time for last 1,000,000:    4s.  Last read position: X:147,133,789\n",
      "INFO\t2016-11-30 12:34:36\tAddOrReplaceReadGroups\tProcessed   293,000,000 records.  Elapsed time: 00:25:12s.  Time for last 1,000,000:    6s.  Last read position: 7:44,841,036\n",
      "INFO\t2016-11-30 12:34:41\tAddOrReplaceReadGroups\tProcessed   294,000,000 records.  Elapsed time: 00:25:16s.  Time for last 1,000,000:    4s.  Last read position: 12:6,929,175\n",
      "INFO\t2016-11-30 12:34:46\tAddOrReplaceReadGroups\tProcessed   295,000,000 records.  Elapsed time: 00:25:21s.  Time for last 1,000,000:    4s.  Last read position: X:47,058,932\n",
      "INFO\t2016-11-30 12:34:54\tAddOrReplaceReadGroups\tProcessed   296,000,000 records.  Elapsed time: 00:25:30s.  Time for last 1,000,000:    8s.  Last read position: X:12,937,965\n",
      "INFO\t2016-11-30 12:34:59\tAddOrReplaceReadGroups\tProcessed   297,000,000 records.  Elapsed time: 00:25:34s.  Time for last 1,000,000:    4s.  Last read position: 11:18,422,396\n",
      "INFO\t2016-11-30 12:35:03\tAddOrReplaceReadGroups\tProcessed   298,000,000 records.  Elapsed time: 00:25:39s.  Time for last 1,000,000:    4s.  Last read position: 3:119,196,274\n",
      "INFO\t2016-11-30 12:35:08\tAddOrReplaceReadGroups\tProcessed   299,000,000 records.  Elapsed time: 00:25:44s.  Time for last 1,000,000:    4s.  Last read position: 2:241,517,230\n",
      "INFO\t2016-11-30 12:35:13\tAddOrReplaceReadGroups\tProcessed   300,000,000 records.  Elapsed time: 00:25:48s.  Time for last 1,000,000:    4s.  Last read position: 4:56,290,807\n",
      "INFO\t2016-11-30 12:35:18\tAddOrReplaceReadGroups\tProcessed   301,000,000 records.  Elapsed time: 00:25:53s.  Time for last 1,000,000:    4s.  Last read position: 12:971,308\n",
      "INFO\t2016-11-30 12:35:22\tAddOrReplaceReadGroups\tProcessed   302,000,000 records.  Elapsed time: 00:25:58s.  Time for last 1,000,000:    4s.  Last read position: 17:1,247,886\n",
      "INFO\t2016-11-30 12:35:27\tAddOrReplaceReadGroups\tProcessed   303,000,000 records.  Elapsed time: 00:26:03s.  Time for last 1,000,000:    4s.  Last read position: 5:95,233,978\n",
      "INFO\t2016-11-30 12:35:32\tAddOrReplaceReadGroups\tProcessed   304,000,000 records.  Elapsed time: 00:26:08s.  Time for last 1,000,000:    4s.  Last read position: 7:5,569,183\n",
      "INFO\t2016-11-30 12:35:37\tAddOrReplaceReadGroups\tProcessed   305,000,000 records.  Elapsed time: 00:26:12s.  Time for last 1,000,000:    4s.  Last read position: 6:50,880,597\n",
      "INFO\t2016-11-30 12:35:42\tAddOrReplaceReadGroups\tProcessed   306,000,000 records.  Elapsed time: 00:26:17s.  Time for last 1,000,000:    4s.  Last read position: 2:27,615,838\n",
      "INFO\t2016-11-30 12:35:47\tAddOrReplaceReadGroups\tProcessed   307,000,000 records.  Elapsed time: 00:26:22s.  Time for last 1,000,000:    4s.  Last read position: 22:44,343,489\n",
      "INFO\t2016-11-30 12:35:51\tAddOrReplaceReadGroups\tProcessed   308,000,000 records.  Elapsed time: 00:26:27s.  Time for last 1,000,000:    4s.  Last read position: 1:19,666,105\n",
      "INFO\t2016-11-30 12:35:57\tAddOrReplaceReadGroups\tProcessed   309,000,000 records.  Elapsed time: 00:26:33s.  Time for last 1,000,000:    5s.  Last read position: 19:36,395,422\n",
      "INFO\t2016-11-30 12:36:02\tAddOrReplaceReadGroups\tProcessed   310,000,000 records.  Elapsed time: 00:26:37s.  Time for last 1,000,000:    4s.  Last read position: 18:72,187,875\n",
      "INFO\t2016-11-30 12:36:07\tAddOrReplaceReadGroups\tProcessed   311,000,000 records.  Elapsed time: 00:26:42s.  Time for last 1,000,000:    4s.  Last read position: 8:141,535,926\n",
      "INFO\t2016-11-30 12:36:12\tAddOrReplaceReadGroups\tProcessed   312,000,000 records.  Elapsed time: 00:26:48s.  Time for last 1,000,000:    5s.  Last read position: 20:36,158,870\n",
      "INFO\t2016-11-30 12:36:17\tAddOrReplaceReadGroups\tProcessed   313,000,000 records.  Elapsed time: 00:26:52s.  Time for last 1,000,000:    4s.  Last read position: 6:30,229,254\n",
      "INFO\t2016-11-30 12:36:22\tAddOrReplaceReadGroups\tProcessed   314,000,000 records.  Elapsed time: 00:26:57s.  Time for last 1,000,000:    4s.  Last read position: 16:1,412,103\n",
      "INFO\t2016-11-30 12:36:27\tAddOrReplaceReadGroups\tProcessed   315,000,000 records.  Elapsed time: 00:27:03s.  Time for last 1,000,000:    5s.  Last read position: 19:55,175,927\n",
      "INFO\t2016-11-30 12:36:32\tAddOrReplaceReadGroups\tProcessed   316,000,000 records.  Elapsed time: 00:27:08s.  Time for last 1,000,000:    4s.  Last read position: 16:56,973,812\n",
      "INFO\t2016-11-30 12:36:37\tAddOrReplaceReadGroups\tProcessed   317,000,000 records.  Elapsed time: 00:27:13s.  Time for last 1,000,000:    4s.  Last read position: 13:113,885,161\n",
      "INFO\t2016-11-30 12:36:43\tAddOrReplaceReadGroups\tProcessed   318,000,000 records.  Elapsed time: 00:27:19s.  Time for last 1,000,000:    6s.  Last read position: X:153,582,330\n",
      "INFO\t2016-11-30 12:36:48\tAddOrReplaceReadGroups\tProcessed   319,000,000 records.  Elapsed time: 00:27:24s.  Time for last 1,000,000:    4s.  Last read position: X:153,697,019\n",
      "INFO\t2016-11-30 12:36:53\tAddOrReplaceReadGroups\tProcessed   320,000,000 records.  Elapsed time: 00:27:29s.  Time for last 1,000,000:    4s.  Last read position: X:119,571,429\n",
      "INFO\t2016-11-30 12:37:03\tAddOrReplaceReadGroups\tProcessed   321,000,000 records.  Elapsed time: 00:27:38s.  Time for last 1,000,000:    9s.  Last read position: 21:38,529,096\n",
      "INFO\t2016-11-30 12:37:07\tAddOrReplaceReadGroups\tProcessed   322,000,000 records.  Elapsed time: 00:27:43s.  Time for last 1,000,000:    4s.  Last read position: 2:132,021,463\n",
      "INFO\t2016-11-30 12:37:12\tAddOrReplaceReadGroups\tProcessed   323,000,000 records.  Elapsed time: 00:27:48s.  Time for last 1,000,000:    4s.  Last read position: 12:120,651,686\n",
      "INFO\t2016-11-30 12:37:17\tAddOrReplaceReadGroups\tProcessed   324,000,000 records.  Elapsed time: 00:27:53s.  Time for last 1,000,000:    4s.  Last read position: 8:38,709,749\n",
      "INFO\t2016-11-30 12:37:22\tAddOrReplaceReadGroups\tProcessed   325,000,000 records.  Elapsed time: 00:27:57s.  Time for last 1,000,000:    4s.  Last read position: 7:148,981,286\n",
      "INFO\t2016-11-30 12:37:26\tAddOrReplaceReadGroups\tProcessed   326,000,000 records.  Elapsed time: 00:28:02s.  Time for last 1,000,000:    4s.  Last read position: 18:57,006,007\n",
      "INFO\t2016-11-30 12:37:31\tAddOrReplaceReadGroups\tProcessed   327,000,000 records.  Elapsed time: 00:28:07s.  Time for last 1,000,000:    4s.  Last read position: 5:43,496,294\n",
      "INFO\t2016-11-30 12:37:36\tAddOrReplaceReadGroups\tProcessed   328,000,000 records.  Elapsed time: 00:28:12s.  Time for last 1,000,000:    4s.  Last read position: 3:38,184,076\n",
      "INFO\t2016-11-30 12:37:41\tAddOrReplaceReadGroups\tProcessed   329,000,000 records.  Elapsed time: 00:28:16s.  Time for last 1,000,000:    4s.  Last read position: 17:45,387,747\n",
      "INFO\t2016-11-30 12:37:46\tAddOrReplaceReadGroups\tProcessed   330,000,000 records.  Elapsed time: 00:28:21s.  Time for last 1,000,000:    4s.  Last read position: 3:143,271,259\n",
      "INFO\t2016-11-30 12:37:50\tAddOrReplaceReadGroups\tProcessed   331,000,000 records.  Elapsed time: 00:28:26s.  Time for last 1,000,000:    4s.  Last read position: 13:23,270,053\n",
      "INFO\t2016-11-30 12:37:55\tAddOrReplaceReadGroups\tProcessed   332,000,000 records.  Elapsed time: 00:28:31s.  Time for last 1,000,000:    4s.  Last read position: 14:99,969,060\n",
      "INFO\t2016-11-30 12:38:00\tAddOrReplaceReadGroups\tProcessed   333,000,000 records.  Elapsed time: 00:28:36s.  Time for last 1,000,000:    4s.  Last read position: 15:28,908,527\n",
      "INFO\t2016-11-30 12:38:05\tAddOrReplaceReadGroups\tProcessed   334,000,000 records.  Elapsed time: 00:28:40s.  Time for last 1,000,000:    4s.  Last read position: MT:1,241\n",
      "INFO\t2016-11-30 12:38:09\tAddOrReplaceReadGroups\tProcessed   335,000,000 records.  Elapsed time: 00:28:45s.  Time for last 1,000,000:    4s.  Last read position: 1:568,522\n",
      "INFO\t2016-11-30 12:38:15\tAddOrReplaceReadGroups\tProcessed   336,000,000 records.  Elapsed time: 00:28:51s.  Time for last 1,000,000:    5s.  Last read position: 8:11,702,597\n",
      "INFO\t2016-11-30 12:38:20\tAddOrReplaceReadGroups\tProcessed   337,000,000 records.  Elapsed time: 00:28:56s.  Time for last 1,000,000:    4s.  Last read position: 7:137,613,003\n",
      "INFO\t2016-11-30 12:38:24\tAddOrReplaceReadGroups\tProcessed   338,000,000 records.  Elapsed time: 00:29:00s.  Time for last 1,000,000:    4s.  Last read position: 15:64,418,392\n",
      "INFO\t2016-11-30 12:38:30\tAddOrReplaceReadGroups\tProcessed   339,000,000 records.  Elapsed time: 00:29:05s.  Time for last 1,000,000:    5s.  Last read position: 14:24,409,718\n",
      "INFO\t2016-11-30 12:38:34\tAddOrReplaceReadGroups\tProcessed   340,000,000 records.  Elapsed time: 00:29:10s.  Time for last 1,000,000:    4s.  Last read position: 22:38,013,651\n",
      "INFO\t2016-11-30 12:38:39\tAddOrReplaceReadGroups\tProcessed   341,000,000 records.  Elapsed time: 00:29:15s.  Time for last 1,000,000:    4s.  Last read position: 2:26,414,491\n",
      "INFO\t2016-11-30 12:38:46\tAddOrReplaceReadGroups\tProcessed   342,000,000 records.  Elapsed time: 00:29:22s.  Time for last 1,000,000:    6s.  Last read position: 4:79,101,807\n",
      "INFO\t2016-11-30 12:38:51\tAddOrReplaceReadGroups\tProcessed   343,000,000 records.  Elapsed time: 00:29:26s.  Time for last 1,000,000:    4s.  Last read position: 11:3,714,573\n",
      "INFO\t2016-11-30 12:38:55\tAddOrReplaceReadGroups\tProcessed   344,000,000 records.  Elapsed time: 00:29:31s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,077\n",
      "INFO\t2016-11-30 12:39:04\tAddOrReplaceReadGroups\tProcessed   345,000,000 records.  Elapsed time: 00:29:40s.  Time for last 1,000,000:    8s.  Last read position: 12:57,594,497\n",
      "INFO\t2016-11-30 12:39:09\tAddOrReplaceReadGroups\tProcessed   346,000,000 records.  Elapsed time: 00:29:44s.  Time for last 1,000,000:    4s.  Last read position: 19:17,942,158\n",
      "INFO\t2016-11-30 12:39:13\tAddOrReplaceReadGroups\tProcessed   347,000,000 records.  Elapsed time: 00:29:49s.  Time for last 1,000,000:    4s.  Last read position: 6:36,567,402\n",
      "INFO\t2016-11-30 12:39:18\tAddOrReplaceReadGroups\tProcessed   348,000,000 records.  Elapsed time: 00:29:54s.  Time for last 1,000,000:    4s.  Last read position: 11:1,774,663\n",
      "INFO\t2016-11-30 12:39:23\tAddOrReplaceReadGroups\tProcessed   349,000,000 records.  Elapsed time: 00:29:59s.  Time for last 1,000,000:    4s.  Last read position: X:48,887,651\n",
      "INFO\t2016-11-30 12:39:28\tAddOrReplaceReadGroups\tProcessed   350,000,000 records.  Elapsed time: 00:30:03s.  Time for last 1,000,000:    4s.  Last read position: 19:45,412,056\n",
      "INFO\t2016-11-30 12:39:32\tAddOrReplaceReadGroups\tProcessed   351,000,000 records.  Elapsed time: 00:30:08s.  Time for last 1,000,000:    4s.  Last read position: 9:5,786,394\n",
      "INFO\t2016-11-30 12:39:37\tAddOrReplaceReadGroups\tProcessed   352,000,000 records.  Elapsed time: 00:30:13s.  Time for last 1,000,000:    4s.  Last read position: 11:110,118,503\n",
      "INFO\t2016-11-30 12:39:42\tAddOrReplaceReadGroups\tProcessed   353,000,000 records.  Elapsed time: 00:30:18s.  Time for last 1,000,000:    4s.  Last read position: 2:101,893,550\n",
      "INFO\t2016-11-30 12:39:47\tAddOrReplaceReadGroups\tProcessed   354,000,000 records.  Elapsed time: 00:30:22s.  Time for last 1,000,000:    4s.  Last read position: 10:51,590,512\n",
      "INFO\t2016-11-30 12:39:52\tAddOrReplaceReadGroups\tProcessed   355,000,000 records.  Elapsed time: 00:30:27s.  Time for last 1,000,000:    4s.  Last read position: 2:74,721,506\n",
      "INFO\t2016-11-30 12:39:57\tAddOrReplaceReadGroups\tProcessed   356,000,000 records.  Elapsed time: 00:30:32s.  Time for last 1,000,000:    4s.  Last read position: 11:65,302,786\n",
      "INFO\t2016-11-30 12:40:02\tAddOrReplaceReadGroups\tProcessed   357,000,000 records.  Elapsed time: 00:30:37s.  Time for last 1,000,000:    4s.  Last read position: 19:17,267,497\n",
      "INFO\t2016-11-30 12:40:07\tAddOrReplaceReadGroups\tProcessed   358,000,000 records.  Elapsed time: 00:30:43s.  Time for last 1,000,000:    5s.  Last read position: 10:73,576,165\n",
      "INFO\t2016-11-30 12:40:12\tAddOrReplaceReadGroups\tProcessed   359,000,000 records.  Elapsed time: 00:30:47s.  Time for last 1,000,000:    4s.  Last read position: 17:6,905,105\n",
      "INFO\t2016-11-30 12:40:16\tAddOrReplaceReadGroups\tProcessed   360,000,000 records.  Elapsed time: 00:30:52s.  Time for last 1,000,000:    4s.  Last read position: 8:11,702,495\n",
      "INFO\t2016-11-30 12:40:22\tAddOrReplaceReadGroups\tProcessed   361,000,000 records.  Elapsed time: 00:30:58s.  Time for last 1,000,000:    6s.  Last read position: 19:40,384,915\n",
      "INFO\t2016-11-30 12:40:28\tAddOrReplaceReadGroups\tProcessed   362,000,000 records.  Elapsed time: 00:31:04s.  Time for last 1,000,000:    5s.  Last read position: 1:228,823,202\n",
      "INFO\t2016-11-30 12:40:33\tAddOrReplaceReadGroups\tProcessed   363,000,000 records.  Elapsed time: 00:31:09s.  Time for last 1,000,000:    5s.  Last read position: 13:23,270,266\n",
      "INFO\t2016-11-30 12:40:40\tAddOrReplaceReadGroups\tProcessed   364,000,000 records.  Elapsed time: 00:31:16s.  Time for last 1,000,000:    6s.  Last read position: 3:188,657,846\n",
      "INFO\t2016-11-30 12:40:45\tAddOrReplaceReadGroups\tProcessed   365,000,000 records.  Elapsed time: 00:31:20s.  Time for last 1,000,000:    4s.  Last read position: 6:31,765,476\n",
      "INFO\t2016-11-30 12:40:50\tAddOrReplaceReadGroups\tProcessed   366,000,000 records.  Elapsed time: 00:31:25s.  Time for last 1,000,000:    4s.  Last read position: X:46,146,858\n",
      "INFO\t2016-11-30 12:40:57\tAddOrReplaceReadGroups\tProcessed   367,000,000 records.  Elapsed time: 00:31:33s.  Time for last 1,000,000:    7s.  Last read position: 5:180,217,834\n",
      "INFO\t2016-11-30 12:41:02\tAddOrReplaceReadGroups\tProcessed   368,000,000 records.  Elapsed time: 00:31:38s.  Time for last 1,000,000:    4s.  Last read position: 4:88,904,237\n",
      "INFO\t2016-11-30 12:41:07\tAddOrReplaceReadGroups\tProcessed   369,000,000 records.  Elapsed time: 00:31:42s.  Time for last 1,000,000:    4s.  Last read position: 12:72,030,780\n",
      "INFO\t2016-11-30 12:41:14\tAddOrReplaceReadGroups\tProcessed   370,000,000 records.  Elapsed time: 00:31:49s.  Time for last 1,000,000:    7s.  Last read position: 7:105,891,313\n",
      "INFO\t2016-11-30 12:41:19\tAddOrReplaceReadGroups\tProcessed   371,000,000 records.  Elapsed time: 00:31:54s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,783\n",
      "INFO\t2016-11-30 12:41:23\tAddOrReplaceReadGroups\tProcessed   372,000,000 records.  Elapsed time: 00:31:59s.  Time for last 1,000,000:    4s.  Last read position: 8:71,486,276\n",
      "INFO\t2016-11-30 12:41:28\tAddOrReplaceReadGroups\tProcessed   373,000,000 records.  Elapsed time: 00:32:04s.  Time for last 1,000,000:    4s.  Last read position: 8:82,195,671\n",
      "INFO\t2016-11-30 12:41:33\tAddOrReplaceReadGroups\tProcessed   374,000,000 records.  Elapsed time: 00:32:09s.  Time for last 1,000,000:    4s.  Last read position: 17:63,996,917\n",
      "INFO\t2016-11-30 12:41:38\tAddOrReplaceReadGroups\tProcessed   375,000,000 records.  Elapsed time: 00:32:14s.  Time for last 1,000,000:    4s.  Last read position: 13:23,270,282\n",
      "INFO\t2016-11-30 12:41:43\tAddOrReplaceReadGroups\tProcessed   376,000,000 records.  Elapsed time: 00:32:19s.  Time for last 1,000,000:    4s.  Last read position: 19:44,160,691\n",
      "INFO\t2016-11-30 12:41:48\tAddOrReplaceReadGroups\tProcessed   377,000,000 records.  Elapsed time: 00:32:24s.  Time for last 1,000,000:    4s.  Last read position: X:41,205,803\n",
      "INFO\t2016-11-30 12:41:53\tAddOrReplaceReadGroups\tProcessed   378,000,000 records.  Elapsed time: 00:32:28s.  Time for last 1,000,000:    4s.  Last read position: 16:55,563,806\n",
      "INFO\t2016-11-30 12:41:58\tAddOrReplaceReadGroups\tProcessed   379,000,000 records.  Elapsed time: 00:32:33s.  Time for last 1,000,000:    4s.  Last read position: 5:17,354,178\n",
      "INFO\t2016-11-30 12:42:02\tAddOrReplaceReadGroups\tProcessed   380,000,000 records.  Elapsed time: 00:32:38s.  Time for last 1,000,000:    4s.  Last read position: 17:17,119,715\n",
      "INFO\t2016-11-30 12:42:07\tAddOrReplaceReadGroups\tProcessed   381,000,000 records.  Elapsed time: 00:32:43s.  Time for last 1,000,000:    4s.  Last read position: 17:76,698,642\n",
      "INFO\t2016-11-30 12:42:12\tAddOrReplaceReadGroups\tProcessed   382,000,000 records.  Elapsed time: 00:32:48s.  Time for last 1,000,000:    5s.  Last read position: 4:88,902,922\n",
      "INFO\t2016-11-30 12:42:17\tAddOrReplaceReadGroups\tProcessed   383,000,000 records.  Elapsed time: 00:32:53s.  Time for last 1,000,000:    4s.  Last read position: 11:77,445,519\n",
      "INFO\t2016-11-30 12:42:22\tAddOrReplaceReadGroups\tProcessed   384,000,000 records.  Elapsed time: 00:32:58s.  Time for last 1,000,000:    4s.  Last read position: 3:10,157,454\n",
      "INFO\t2016-11-30 12:42:27\tAddOrReplaceReadGroups\tProcessed   385,000,000 records.  Elapsed time: 00:33:03s.  Time for last 1,000,000:    5s.  Last read position: 12:21,796,970\n",
      "INFO\t2016-11-30 12:42:32\tAddOrReplaceReadGroups\tProcessed   386,000,000 records.  Elapsed time: 00:33:08s.  Time for last 1,000,000:    4s.  Last read position: 6:145,981,813\n",
      "INFO\t2016-11-30 12:42:37\tAddOrReplaceReadGroups\tProcessed   387,000,000 records.  Elapsed time: 00:33:13s.  Time for last 1,000,000:    4s.  Last read position: 1:6,156,810\n",
      "INFO\t2016-11-30 12:42:43\tAddOrReplaceReadGroups\tProcessed   388,000,000 records.  Elapsed time: 00:33:19s.  Time for last 1,000,000:    6s.  Last read position: 4:88,902,838\n",
      "INFO\t2016-11-30 12:42:48\tAddOrReplaceReadGroups\tProcessed   389,000,000 records.  Elapsed time: 00:33:23s.  Time for last 1,000,000:    4s.  Last read position: 14:23,316,543\n",
      "INFO\t2016-11-30 12:42:53\tAddOrReplaceReadGroups\tProcessed   390,000,000 records.  Elapsed time: 00:33:28s.  Time for last 1,000,000:    4s.  Last read position: 10:11,312,659\n",
      "INFO\t2016-11-30 12:43:00\tAddOrReplaceReadGroups\tProcessed   391,000,000 records.  Elapsed time: 00:33:35s.  Time for last 1,000,000:    6s.  Last read position: 12:49,522,432\n",
      "INFO\t2016-11-30 12:43:04\tAddOrReplaceReadGroups\tProcessed   392,000,000 records.  Elapsed time: 00:33:40s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,632\n",
      "INFO\t2016-11-30 12:43:09\tAddOrReplaceReadGroups\tProcessed   393,000,000 records.  Elapsed time: 00:33:45s.  Time for last 1,000,000:    4s.  Last read position: 3:75,671,327\n",
      "INFO\t2016-11-30 12:43:15\tAddOrReplaceReadGroups\tProcessed   394,000,000 records.  Elapsed time: 00:33:51s.  Time for last 1,000,000:    6s.  Last read position: 20:57,570,445\n",
      "INFO\t2016-11-30 12:43:20\tAddOrReplaceReadGroups\tProcessed   395,000,000 records.  Elapsed time: 00:33:56s.  Time for last 1,000,000:    4s.  Last read position: 17:59,975,039\n",
      "INFO\t2016-11-30 12:43:25\tAddOrReplaceReadGroups\tProcessed   396,000,000 records.  Elapsed time: 00:34:01s.  Time for last 1,000,000:    4s.  Last read position: 7:44,841,061\n",
      "INFO\t2016-11-30 12:43:30\tAddOrReplaceReadGroups\tProcessed   397,000,000 records.  Elapsed time: 00:34:05s.  Time for last 1,000,000:    4s.  Last read position: 9:139,399,896\n",
      "INFO\t2016-11-30 12:43:35\tAddOrReplaceReadGroups\tProcessed   398,000,000 records.  Elapsed time: 00:34:10s.  Time for last 1,000,000:    4s.  Last read position: 1:182,352,695\n",
      "INFO\t2016-11-30 12:43:39\tAddOrReplaceReadGroups\tProcessed   399,000,000 records.  Elapsed time: 00:34:15s.  Time for last 1,000,000:    4s.  Last read position: Y:22,894,145\n",
      "INFO\t2016-11-30 12:43:44\tAddOrReplaceReadGroups\tProcessed   400,000,000 records.  Elapsed time: 00:34:20s.  Time for last 1,000,000:    4s.  Last read position: 16:67,672,019\n",
      "INFO\t2016-11-30 12:43:49\tAddOrReplaceReadGroups\tProcessed   401,000,000 records.  Elapsed time: 00:34:25s.  Time for last 1,000,000:    4s.  Last read position: 6:119,499,878\n",
      "INFO\t2016-11-30 12:43:54\tAddOrReplaceReadGroups\tProcessed   402,000,000 records.  Elapsed time: 00:34:29s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,144\n",
      "INFO\t2016-11-30 12:43:58\tAddOrReplaceReadGroups\tProcessed   403,000,000 records.  Elapsed time: 00:34:34s.  Time for last 1,000,000:    4s.  Last read position: 15:73,996,764\n",
      "INFO\t2016-11-30 12:44:03\tAddOrReplaceReadGroups\tProcessed   404,000,000 records.  Elapsed time: 00:34:39s.  Time for last 1,000,000:    4s.  Last read position: 11:64,977,372\n",
      "INFO\t2016-11-30 12:44:08\tAddOrReplaceReadGroups\tProcessed   405,000,000 records.  Elapsed time: 00:34:43s.  Time for last 1,000,000:    4s.  Last read position: 11:60,064,780\n",
      "INFO\t2016-11-30 12:44:12\tAddOrReplaceReadGroups\tProcessed   406,000,000 records.  Elapsed time: 00:34:48s.  Time for last 1,000,000:    4s.  Last read position: 15:72,491,567\n",
      "INFO\t2016-11-30 12:44:17\tAddOrReplaceReadGroups\tProcessed   407,000,000 records.  Elapsed time: 00:34:53s.  Time for last 1,000,000:    4s.  Last read position: 3:27,416,271\n",
      "INFO\t2016-11-30 12:44:22\tAddOrReplaceReadGroups\tProcessed   408,000,000 records.  Elapsed time: 00:34:58s.  Time for last 1,000,000:    4s.  Last read position: 20:19,982,377\n",
      "INFO\t2016-11-30 12:44:27\tAddOrReplaceReadGroups\tProcessed   409,000,000 records.  Elapsed time: 00:35:02s.  Time for last 1,000,000:    4s.  Last read position: 8:17,922,038\n",
      "INFO\t2016-11-30 12:44:32\tAddOrReplaceReadGroups\tProcessed   410,000,000 records.  Elapsed time: 00:35:08s.  Time for last 1,000,000:    5s.  Last read position: 17:65,191,851\n",
      "INFO\t2016-11-30 12:44:37\tAddOrReplaceReadGroups\tProcessed   411,000,000 records.  Elapsed time: 00:35:13s.  Time for last 1,000,000:    4s.  Last read position: 12:38,721,147\n",
      "INFO\t2016-11-30 12:44:42\tAddOrReplaceReadGroups\tProcessed   412,000,000 records.  Elapsed time: 00:35:18s.  Time for last 1,000,000:    4s.  Last read position: 22:40,723,276\n",
      "INFO\t2016-11-30 12:44:48\tAddOrReplaceReadGroups\tProcessed   413,000,000 records.  Elapsed time: 00:35:23s.  Time for last 1,000,000:    5s.  Last read position: 11:61,732,267\n",
      "INFO\t2016-11-30 12:44:52\tAddOrReplaceReadGroups\tProcessed   414,000,000 records.  Elapsed time: 00:35:28s.  Time for last 1,000,000:    4s.  Last read position: 11:72,410,149\n",
      "INFO\t2016-11-30 12:44:57\tAddOrReplaceReadGroups\tProcessed   415,000,000 records.  Elapsed time: 00:35:32s.  Time for last 1,000,000:    4s.  Last read position: 12:80,168,113\n",
      "INFO\t2016-11-30 12:45:03\tAddOrReplaceReadGroups\tProcessed   416,000,000 records.  Elapsed time: 00:35:39s.  Time for last 1,000,000:    6s.  Last read position: 8:20,067,930\n",
      "INFO\t2016-11-30 12:45:08\tAddOrReplaceReadGroups\tProcessed   417,000,000 records.  Elapsed time: 00:35:43s.  Time for last 1,000,000:    4s.  Last read position: 15:45,007,877\n",
      "INFO\t2016-11-30 12:45:12\tAddOrReplaceReadGroups\tProcessed   418,000,000 records.  Elapsed time: 00:35:48s.  Time for last 1,000,000:    4s.  Last read position: 10:114,545,770\n",
      "INFO\t2016-11-30 12:45:17\tAddOrReplaceReadGroups\tProcessed   419,000,000 records.  Elapsed time: 00:35:52s.  Time for last 1,000,000:    4s.  Last read position: 13:25,671,390\n",
      "INFO\t2016-11-30 12:45:21\tAddOrReplaceReadGroups\tProcessed   420,000,000 records.  Elapsed time: 00:35:57s.  Time for last 1,000,000:    4s.  Last read position: 5:95,278,707\n",
      "INFO\t2016-11-30 12:45:26\tAddOrReplaceReadGroups\tProcessed   421,000,000 records.  Elapsed time: 00:36:02s.  Time for last 1,000,000:    4s.  Last read position: 19:38,874,379\n",
      "INFO\t2016-11-30 12:45:31\tAddOrReplaceReadGroups\tProcessed   422,000,000 records.  Elapsed time: 00:36:06s.  Time for last 1,000,000:    4s.  Last read position: 12:7,090,756\n",
      "INFO\t2016-11-30 12:45:36\tAddOrReplaceReadGroups\tProcessed   423,000,000 records.  Elapsed time: 00:36:11s.  Time for last 1,000,000:    4s.  Last read position: 7:128,595,330\n",
      "INFO\t2016-11-30 12:45:40\tAddOrReplaceReadGroups\tProcessed   424,000,000 records.  Elapsed time: 00:36:16s.  Time for last 1,000,000:    4s.  Last read position: 1:567,558\n",
      "INFO\t2016-11-30 12:45:45\tAddOrReplaceReadGroups\tProcessed   425,000,000 records.  Elapsed time: 00:36:21s.  Time for last 1,000,000:    4s.  Last read position: 6:144,184,772\n",
      "INFO\t2016-11-30 12:45:50\tAddOrReplaceReadGroups\tProcessed   426,000,000 records.  Elapsed time: 00:36:26s.  Time for last 1,000,000:    4s.  Last read position: 9:135,894,999\n",
      "INFO\t2016-11-30 12:45:55\tAddOrReplaceReadGroups\tProcessed   427,000,000 records.  Elapsed time: 00:36:31s.  Time for last 1,000,000:    4s.  Last read position: 22:29,538,808\n",
      "INFO\t2016-11-30 12:46:00\tAddOrReplaceReadGroups\tProcessed   428,000,000 records.  Elapsed time: 00:36:36s.  Time for last 1,000,000:    5s.  Last read position: 19:11,304,058\n",
      "INFO\t2016-11-30 12:46:05\tAddOrReplaceReadGroups\tProcessed   429,000,000 records.  Elapsed time: 00:36:41s.  Time for last 1,000,000:    4s.  Last read position: 11:72,397,206\n",
      "INFO\t2016-11-30 12:46:10\tAddOrReplaceReadGroups\tProcessed   430,000,000 records.  Elapsed time: 00:36:45s.  Time for last 1,000,000:    4s.  Last read position: 1:568,257\n",
      "INFO\t2016-11-30 12:46:15\tAddOrReplaceReadGroups\tProcessed   431,000,000 records.  Elapsed time: 00:36:51s.  Time for last 1,000,000:    5s.  Last read position: 19:52,327,386\n",
      "INFO\t2016-11-30 12:46:20\tAddOrReplaceReadGroups\tProcessed   432,000,000 records.  Elapsed time: 00:36:56s.  Time for last 1,000,000:    4s.  Last read position: 2:207,944,076\n",
      "INFO\t2016-11-30 12:46:25\tAddOrReplaceReadGroups\tProcessed   433,000,000 records.  Elapsed time: 00:37:00s.  Time for last 1,000,000:    4s.  Last read position: 17:2,583,476\n",
      "INFO\t2016-11-30 12:46:31\tAddOrReplaceReadGroups\tProcessed   434,000,000 records.  Elapsed time: 00:37:07s.  Time for last 1,000,000:    6s.  Last read position: 19:40,399,654\n",
      "INFO\t2016-11-30 12:46:36\tAddOrReplaceReadGroups\tProcessed   435,000,000 records.  Elapsed time: 00:37:11s.  Time for last 1,000,000:    4s.  Last read position: 6:53,380,558\n",
      "INFO\t2016-11-30 12:46:40\tAddOrReplaceReadGroups\tProcessed   436,000,000 records.  Elapsed time: 00:37:16s.  Time for last 1,000,000:    4s.  Last read position: 8:29,921,122\n",
      "INFO\t2016-11-30 12:46:47\tAddOrReplaceReadGroups\tProcessed   437,000,000 records.  Elapsed time: 00:37:23s.  Time for last 1,000,000:    6s.  Last read position: 8:11,703,279\n",
      "INFO\t2016-11-30 12:46:52\tAddOrReplaceReadGroups\tProcessed   438,000,000 records.  Elapsed time: 00:37:27s.  Time for last 1,000,000:    4s.  Last read position: 2:26,164,888\n",
      "INFO\t2016-11-30 12:46:56\tAddOrReplaceReadGroups\tProcessed   439,000,000 records.  Elapsed time: 00:37:32s.  Time for last 1,000,000:    4s.  Last read position: 1:52,271,188\n",
      "INFO\t2016-11-30 12:47:01\tAddOrReplaceReadGroups\tProcessed   440,000,000 records.  Elapsed time: 00:37:37s.  Time for last 1,000,000:    4s.  Last read position: 11:61,734,803\n",
      "INFO\t2016-11-30 12:47:06\tAddOrReplaceReadGroups\tProcessed   441,000,000 records.  Elapsed time: 00:37:41s.  Time for last 1,000,000:    4s.  Last read position: 18:19,231,281\n",
      "INFO\t2016-11-30 12:47:10\tAddOrReplaceReadGroups\tProcessed   442,000,000 records.  Elapsed time: 00:37:46s.  Time for last 1,000,000:    4s.  Last read position: 16:70,037,954\n",
      "[Wed Nov 30 13:15:45 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 66.36 minutes.\n",
      "Runtime.totalMemory()=3813146624\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P5Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P5rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 13:15:46 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P5Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P5rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 13:15:46 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 13:15:46\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 13:15:50\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 19:6,684,437\n",
      "INFO\t2016-11-30 13:15:55\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    4s.  Last read position: 20:47,253,004\n",
      "INFO\t2016-11-30 13:16:00\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    4s.  Last read position: 15:28,561,117\n",
      "INFO\t2016-11-30 13:16:05\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    5s.  Last read position: 16:89,353,439\n",
      "INFO\t2016-11-30 13:16:10\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    4s.  Last read position: 1:212,788,180\n",
      "INFO\t2016-11-30 13:16:16\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:    5s.  Last read position: 21:27,607,564\n",
      "INFO\t2016-11-30 13:16:21\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    5s.  Last read position: 4:154,607,195\n",
      "INFO\t2016-11-30 13:16:26\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    5s.  Last read position: 16:81,702,687\n",
      "INFO\t2016-11-30 13:16:31\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:45s.  Time for last 1,000,000:    4s.  Last read position: 11:117,071,071\n",
      "INFO\t2016-11-30 13:16:36\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:49s.  Time for last 1,000,000:    4s.  Last read position: 1:206,516,222\n",
      "INFO\t2016-11-30 13:16:42\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    6s.  Last read position: 7:909,068\n",
      "INFO\t2016-11-30 13:16:47\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    4s.  Last read position: 6:35,494,302\n",
      "INFO\t2016-11-30 13:16:52\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:05s.  Time for last 1,000,000:    4s.  Last read position: 17:1,044,397\n",
      "INFO\t2016-11-30 13:16:58\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:12s.  Time for last 1,000,000:    6s.  Last read position: 12:121,655,070\n",
      "INFO\t2016-11-30 13:17:03\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:17s.  Time for last 1,000,000:    4s.  Last read position: 3:42,252,415\n",
      "INFO\t2016-11-30 13:17:08\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    4s.  Last read position: 10:26,851,287\n",
      "INFO\t2016-11-30 13:17:13\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:26s.  Time for last 1,000,000:    4s.  Last read position: 5:137,803,440\n",
      "INFO\t2016-11-30 13:17:17\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:31s.  Time for last 1,000,000:    4s.  Last read position: 7:5,567,303\n",
      "INFO\t2016-11-30 13:17:22\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    4s.  Last read position: 11:71,808,937\n",
      "INFO\t2016-11-30 13:17:27\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:41s.  Time for last 1,000,000:    4s.  Last read position: 1:153,963,663\n",
      "INFO\t2016-11-30 13:17:32\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:46s.  Time for last 1,000,000:    4s.  Last read position: 16:15,024,523\n",
      "INFO\t2016-11-30 13:17:37\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    4s.  Last read position: 6:31,238,150\n",
      "INFO\t2016-11-30 13:17:42\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    5s.  Last read position: 17:34,538,870\n",
      "INFO\t2016-11-30 13:17:46\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:    4s.  Last read position: 1:42,965,559\n",
      "INFO\t2016-11-30 13:17:51\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:05s.  Time for last 1,000,000:    4s.  Last read position: 14:75,912,118\n",
      "INFO\t2016-11-30 13:17:57\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:    5s.  Last read position: 19:5,661,594\n",
      "INFO\t2016-11-30 13:18:02\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    4s.  Last read position: 19:56,190,184\n",
      "INFO\t2016-11-30 13:18:07\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    4s.  Last read position: 1:16,945,345\n",
      "INFO\t2016-11-30 13:18:13\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:27s.  Time for last 1,000,000:    6s.  Last read position: 12:92,873,531\n",
      "INFO\t2016-11-30 13:18:18\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    4s.  Last read position: 19:49,120,580\n",
      "INFO\t2016-11-30 13:18:22\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:36s.  Time for last 1,000,000:    4s.  Last read position: 4:88,904,165\n",
      "INFO\t2016-11-30 13:18:29\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:43s.  Time for last 1,000,000:    6s.  Last read position: 4:88,904,038\n",
      "INFO\t2016-11-30 13:18:34\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:48s.  Time for last 1,000,000:    4s.  Last read position: 15:60,641,347\n",
      "INFO\t2016-11-30 13:18:38\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:52s.  Time for last 1,000,000:    4s.  Last read position: 2:181,737,665\n",
      "INFO\t2016-11-30 13:18:43\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:02:57s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,873\n",
      "INFO\t2016-11-30 13:18:48\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:02s.  Time for last 1,000,000:    4s.  Last read position: 5:177,482,904\n",
      "INFO\t2016-11-30 13:18:53\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:07s.  Time for last 1,000,000:    4s.  Last read position: 3:5,220,745\n",
      "INFO\t2016-11-30 13:18:58\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:11s.  Time for last 1,000,000:    4s.  Last read position: 9:131,939,403\n",
      "INFO\t2016-11-30 13:19:03\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:16s.  Time for last 1,000,000:    5s.  Last read position: 10:73,576,727\n",
      "INFO\t2016-11-30 13:19:07\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:21s.  Time for last 1,000,000:    4s.  Last read position: 19:40,382,381\n",
      "INFO\t2016-11-30 13:19:12\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:26s.  Time for last 1,000,000:    4s.  Last read position: 12:49,666,379\n",
      "INFO\t2016-11-30 13:19:17\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:31s.  Time for last 1,000,000:    4s.  Last read position: 17:42,429,573\n",
      "INFO\t2016-11-30 13:19:22\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:36s.  Time for last 1,000,000:    4s.  Last read position: X:23,855,301\n",
      "INFO\t2016-11-30 13:19:26\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:03:40s.  Time for last 1,000,000:    4s.  Last read position: 4:88,904,306\n",
      "INFO\t2016-11-30 13:19:31\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:03:45s.  Time for last 1,000,000:    4s.  Last read position: 5:17,353,873\n",
      "INFO\t2016-11-30 13:19:36\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:03:50s.  Time for last 1,000,000:    4s.  Last read position: 1:96,913,832\n",
      "INFO\t2016-11-30 13:19:41\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:03:55s.  Time for last 1,000,000:    4s.  Last read position: 6:36,457,822\n",
      "INFO\t2016-11-30 13:19:47\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:00s.  Time for last 1,000,000:    5s.  Last read position: 6:44,221,461\n",
      "INFO\t2016-11-30 13:19:51\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:05s.  Time for last 1,000,000:    4s.  Last read position: 1:45,976,957\n",
      "INFO\t2016-11-30 13:19:56\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:10s.  Time for last 1,000,000:    4s.  Last read position: 12:49,522,013\n",
      "INFO\t2016-11-30 13:20:02\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:16s.  Time for last 1,000,000:    5s.  Last read position: 7:5,566,884\n",
      "INFO\t2016-11-30 13:20:07\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    4s.  Last read position: 1:24,021,235\n",
      "INFO\t2016-11-30 13:20:12\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:26s.  Time for last 1,000,000:    4s.  Last read position: 19:40,382,350\n",
      "INFO\t2016-11-30 13:20:18\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:32s.  Time for last 1,000,000:    6s.  Last read position: 9:80,537,129\n",
      "INFO\t2016-11-30 13:20:23\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:37s.  Time for last 1,000,000:    4s.  Last read position: 19:30,447,650\n",
      "INFO\t2016-11-30 13:20:28\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:04:42s.  Time for last 1,000,000:    4s.  Last read position: 19:3,976,397\n",
      "INFO\t2016-11-30 13:20:37\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:04:51s.  Time for last 1,000,000:    9s.  Last read position: 4:113,486,246\n",
      "INFO\t2016-11-30 13:20:42\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:04:56s.  Time for last 1,000,000:    4s.  Last read position: 19:42,373,777\n",
      "INFO\t2016-11-30 13:20:47\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:01s.  Time for last 1,000,000:    4s.  Last read position: 9:126,143,239\n",
      "INFO\t2016-11-30 13:20:52\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:06s.  Time for last 1,000,000:    4s.  Last read position: 4:88,904,284\n",
      "INFO\t2016-11-30 13:20:57\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:10s.  Time for last 1,000,000:    4s.  Last read position: 17:38,286,375\n",
      "INFO\t2016-11-30 13:21:02\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:15s.  Time for last 1,000,000:    4s.  Last read position: X:147,134,320\n",
      "INFO\t2016-11-30 13:21:07\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:20s.  Time for last 1,000,000:    5s.  Last read position: 13:48,835,378\n",
      "INFO\t2016-11-30 13:21:12\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:25s.  Time for last 1,000,000:    5s.  Last read position: 2:71,300,711\n",
      "INFO\t2016-11-30 13:21:16\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:30s.  Time for last 1,000,000:    4s.  Last read position: 6:56,869,468\n",
      "INFO\t2016-11-30 13:21:22\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:35s.  Time for last 1,000,000:    5s.  Last read position: 13:113,963,984\n",
      "INFO\t2016-11-30 13:21:26\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:05:40s.  Time for last 1,000,000:    4s.  Last read position: 12:56,553,840\n",
      "INFO\t2016-11-30 13:21:31\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:05:45s.  Time for last 1,000,000:    5s.  Last read position: 5:134,678,992\n",
      "INFO\t2016-11-30 13:21:37\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:05:51s.  Time for last 1,000,000:    5s.  Last read position: 1:228,823,238\n",
      "INFO\t2016-11-30 13:21:42\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:05:56s.  Time for last 1,000,000:    4s.  Last read position: 16:2,733,329\n",
      "INFO\t2016-11-30 13:21:47\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:00s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,806\n",
      "INFO\t2016-11-30 13:21:53\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:07s.  Time for last 1,000,000:    6s.  Last read position: 14:102,478,411\n",
      "INFO\t2016-11-30 13:21:58\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:12s.  Time for last 1,000,000:    4s.  Last read position: 12:49,522,612\n",
      "INFO\t2016-11-30 13:22:03\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:17s.  Time for last 1,000,000:    5s.  Last read position: 7:5,566,847\n",
      "INFO\t2016-11-30 13:22:09\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    5s.  Last read position: 7:5,567,679\n",
      "INFO\t2016-11-30 13:22:14\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:28s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,883\n",
      "INFO\t2016-11-30 13:22:19\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:33s.  Time for last 1,000,000:    4s.  Last read position: 10:32,340,199\n",
      "INFO\t2016-11-30 13:22:25\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:06:39s.  Time for last 1,000,000:    5s.  Last read position: 14:51,376,770\n",
      "INFO\t2016-11-30 13:22:30\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:06:43s.  Time for last 1,000,000:    4s.  Last read position: 1:31,206,371\n",
      "INFO\t2016-11-30 13:22:34\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:06:48s.  Time for last 1,000,000:    4s.  Last read position: 22:38,075,750\n",
      "INFO\t2016-11-30 13:22:41\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:06:55s.  Time for last 1,000,000:    6s.  Last read position: 3:116,746,104\n",
      "INFO\t2016-11-30 13:22:46\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:00s.  Time for last 1,000,000:    4s.  Last read position: 7:134,674,208\n",
      "INFO\t2016-11-30 13:22:51\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:05s.  Time for last 1,000,000:    4s.  Last read position: 19:40,370,206\n",
      "INFO\t2016-11-30 13:22:56\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:10s.  Time for last 1,000,000:    4s.  Last read position: 8:17,918,949\n",
      "INFO\t2016-11-30 13:23:01\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:15s.  Time for last 1,000,000:    4s.  Last read position: 14:24,661,449\n",
      "INFO\t2016-11-30 13:23:06\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:20s.  Time for last 1,000,000:    4s.  Last read position: 3:128,483,400\n",
      "INFO\t2016-11-30 13:23:11\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:25s.  Time for last 1,000,000:    4s.  Last read position: 2:209,100,998\n",
      "INFO\t2016-11-30 13:23:16\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:30s.  Time for last 1,000,000:    5s.  Last read position: 9:130,167,806\n",
      "INFO\t2016-11-30 13:23:21\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:35s.  Time for last 1,000,000:    4s.  Last read position: 1:567,124\n",
      "INFO\t2016-11-30 13:23:26\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:07:40s.  Time for last 1,000,000:    4s.  Last read position: 11:61,734,925\n",
      "INFO\t2016-11-30 13:23:31\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:07:45s.  Time for last 1,000,000:    4s.  Last read position: 11:60,159,655\n",
      "INFO\t2016-11-30 13:23:36\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:07:50s.  Time for last 1,000,000:    4s.  Last read position: 19:55,086,720\n",
      "INFO\t2016-11-30 13:23:41\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:07:54s.  Time for last 1,000,000:    4s.  Last read position: 22:39,619,796\n",
      "INFO\t2016-11-30 13:23:46\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:07:59s.  Time for last 1,000,000:    4s.  Last read position: 10:99,211,042\n",
      "INFO\t2016-11-30 13:23:51\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:04s.  Time for last 1,000,000:    4s.  Last read position: 17:27,889,861\n",
      "INFO\t2016-11-30 13:23:56\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:09s.  Time for last 1,000,000:    4s.  Last read position: 1:569,646\n",
      "INFO\t2016-11-30 13:24:01\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:14s.  Time for last 1,000,000:    5s.  Last read position: 21:45,175,916\n",
      "INFO\t2016-11-30 13:24:06\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:19s.  Time for last 1,000,000:    5s.  Last read position: 6:50,880,590\n",
      "INFO\t2016-11-30 13:24:11\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:24s.  Time for last 1,000,000:    4s.  Last read position: X:40,465,766\n",
      "INFO\t2016-11-30 13:24:16\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:30s.  Time for last 1,000,000:    5s.  Last read position: 15:40,594,351\n",
      "INFO\t2016-11-30 13:24:21\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:08:35s.  Time for last 1,000,000:    4s.  Last read position: 8:82,434,034\n",
      "INFO\t2016-11-30 13:24:26\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:08:40s.  Time for last 1,000,000:    4s.  Last read position: 6:3,730,433\n",
      "INFO\t2016-11-30 13:24:32\tAddOrReplaceReadGroups\tProcessed   103,000,000 records.  Elapsed time: 00:08:45s.  Time for last 1,000,000:    5s.  Last read position: 12:10,311,930\n",
      "INFO\t2016-11-30 13:24:37\tAddOrReplaceReadGroups\tProcessed   104,000,000 records.  Elapsed time: 00:08:50s.  Time for last 1,000,000:    4s.  Last read position: 12:32,863,848\n",
      "INFO\t2016-11-30 13:24:41\tAddOrReplaceReadGroups\tProcessed   105,000,000 records.  Elapsed time: 00:08:55s.  Time for last 1,000,000:    4s.  Last read position: 4:88,903,767\n",
      "INFO\t2016-11-30 13:24:48\tAddOrReplaceReadGroups\tProcessed   106,000,000 records.  Elapsed time: 00:09:01s.  Time for last 1,000,000:    6s.  Last read position: 22:50,356,925\n",
      "INFO\t2016-11-30 13:24:53\tAddOrReplaceReadGroups\tProcessed   107,000,000 records.  Elapsed time: 00:09:06s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,703\n",
      "INFO\t2016-11-30 13:24:57\tAddOrReplaceReadGroups\tProcessed   108,000,000 records.  Elapsed time: 00:09:11s.  Time for last 1,000,000:    4s.  Last read position: 7:5,568,901\n",
      "INFO\t2016-11-30 13:25:05\tAddOrReplaceReadGroups\tProcessed   109,000,000 records.  Elapsed time: 00:09:19s.  Time for last 1,000,000:    7s.  Last read position: 20:17,601,361\n",
      "INFO\t2016-11-30 13:25:10\tAddOrReplaceReadGroups\tProcessed   110,000,000 records.  Elapsed time: 00:09:24s.  Time for last 1,000,000:    4s.  Last read position: 4:88,899,122\n",
      "INFO\t2016-11-30 13:25:15\tAddOrReplaceReadGroups\tProcessed   111,000,000 records.  Elapsed time: 00:09:29s.  Time for last 1,000,000:    4s.  Last read position: 2:112,780,775\n",
      "INFO\t2016-11-30 13:25:24\tAddOrReplaceReadGroups\tProcessed   112,000,000 records.  Elapsed time: 00:09:38s.  Time for last 1,000,000:    8s.  Last read position: 11:66,039,718\n",
      "INFO\t2016-11-30 13:25:29\tAddOrReplaceReadGroups\tProcessed   113,000,000 records.  Elapsed time: 00:09:42s.  Time for last 1,000,000:    4s.  Last read position: 22:18,075,203\n",
      "INFO\t2016-11-30 13:25:34\tAddOrReplaceReadGroups\tProcessed   114,000,000 records.  Elapsed time: 00:09:47s.  Time for last 1,000,000:    4s.  Last read position: 19:55,143,873\n",
      "INFO\t2016-11-30 13:25:38\tAddOrReplaceReadGroups\tProcessed   115,000,000 records.  Elapsed time: 00:09:52s.  Time for last 1,000,000:    4s.  Last read position: 1:569,913\n",
      "INFO\t2016-11-30 13:25:43\tAddOrReplaceReadGroups\tProcessed   116,000,000 records.  Elapsed time: 00:09:57s.  Time for last 1,000,000:    4s.  Last read position: 17:76,109,085\n",
      "INFO\t2016-11-30 13:25:48\tAddOrReplaceReadGroups\tProcessed   117,000,000 records.  Elapsed time: 00:10:02s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,182\n",
      "INFO\t2016-11-30 13:25:53\tAddOrReplaceReadGroups\tProcessed   118,000,000 records.  Elapsed time: 00:10:07s.  Time for last 1,000,000:    4s.  Last read position: 19:1,077,927\n",
      "INFO\t2016-11-30 13:25:58\tAddOrReplaceReadGroups\tProcessed   119,000,000 records.  Elapsed time: 00:10:12s.  Time for last 1,000,000:    4s.  Last read position: MT:670\n",
      "INFO\t2016-11-30 13:26:03\tAddOrReplaceReadGroups\tProcessed   120,000,000 records.  Elapsed time: 00:10:17s.  Time for last 1,000,000:    4s.  Last read position: 9:139,978,568\n",
      "INFO\t2016-11-30 13:26:08\tAddOrReplaceReadGroups\tProcessed   121,000,000 records.  Elapsed time: 00:10:22s.  Time for last 1,000,000:    5s.  Last read position: 10:104,574,960\n",
      "INFO\t2016-11-30 13:26:13\tAddOrReplaceReadGroups\tProcessed   122,000,000 records.  Elapsed time: 00:10:27s.  Time for last 1,000,000:    4s.  Last read position: 1:228,823,255\n",
      "INFO\t2016-11-30 13:26:18\tAddOrReplaceReadGroups\tProcessed   123,000,000 records.  Elapsed time: 00:10:32s.  Time for last 1,000,000:    4s.  Last read position: 10:88,827,869\n",
      "INFO\t2016-11-30 13:26:23\tAddOrReplaceReadGroups\tProcessed   124,000,000 records.  Elapsed time: 00:10:37s.  Time for last 1,000,000:    4s.  Last read position: 18:43,667,002\n",
      "INFO\t2016-11-30 13:26:28\tAddOrReplaceReadGroups\tProcessed   125,000,000 records.  Elapsed time: 00:10:42s.  Time for last 1,000,000:    4s.  Last read position: 1:568,782\n",
      "INFO\t2016-11-30 13:26:33\tAddOrReplaceReadGroups\tProcessed   126,000,000 records.  Elapsed time: 00:10:47s.  Time for last 1,000,000:    4s.  Last read position: MT:6,545\n",
      "INFO\t2016-11-30 13:26:38\tAddOrReplaceReadGroups\tProcessed   127,000,000 records.  Elapsed time: 00:10:52s.  Time for last 1,000,000:    5s.  Last read position: 16:81,453,090\n",
      "INFO\t2016-11-30 13:26:43\tAddOrReplaceReadGroups\tProcessed   128,000,000 records.  Elapsed time: 00:10:57s.  Time for last 1,000,000:    4s.  Last read position: MT:7,922\n",
      "INFO\t2016-11-30 13:26:48\tAddOrReplaceReadGroups\tProcessed   129,000,000 records.  Elapsed time: 00:11:02s.  Time for last 1,000,000:    4s.  Last read position: 1:146,334,179\n",
      "INFO\t2016-11-30 13:26:53\tAddOrReplaceReadGroups\tProcessed   130,000,000 records.  Elapsed time: 00:11:07s.  Time for last 1,000,000:    5s.  Last read position: 1:564,534\n",
      "INFO\t2016-11-30 13:26:58\tAddOrReplaceReadGroups\tProcessed   131,000,000 records.  Elapsed time: 00:11:12s.  Time for last 1,000,000:    4s.  Last read position: 18:74,692,102\n",
      "INFO\t2016-11-30 13:27:03\tAddOrReplaceReadGroups\tProcessed   132,000,000 records.  Elapsed time: 00:11:17s.  Time for last 1,000,000:    4s.  Last read position: 17:76,849,965\n",
      "INFO\t2016-11-30 13:27:10\tAddOrReplaceReadGroups\tProcessed   133,000,000 records.  Elapsed time: 00:11:23s.  Time for last 1,000,000:    6s.  Last read position: 3:37,317,772\n",
      "INFO\t2016-11-30 13:27:14\tAddOrReplaceReadGroups\tProcessed   134,000,000 records.  Elapsed time: 00:11:28s.  Time for last 1,000,000:    4s.  Last read position: 1:569,516\n",
      "INFO\t2016-11-30 13:27:19\tAddOrReplaceReadGroups\tProcessed   135,000,000 records.  Elapsed time: 00:11:33s.  Time for last 1,000,000:    4s.  Last read position: MT:6,626\n",
      "INFO\t2016-11-30 13:27:27\tAddOrReplaceReadGroups\tProcessed   136,000,000 records.  Elapsed time: 00:11:41s.  Time for last 1,000,000:    7s.  Last read position: MT:6,543\n",
      "INFO\t2016-11-30 13:27:32\tAddOrReplaceReadGroups\tProcessed   137,000,000 records.  Elapsed time: 00:11:46s.  Time for last 1,000,000:    4s.  Last read position: 2:161,280,719\n",
      "INFO\t2016-11-30 13:27:37\tAddOrReplaceReadGroups\tProcessed   138,000,000 records.  Elapsed time: 00:11:51s.  Time for last 1,000,000:    5s.  Last read position: 7:148,489,065\n",
      "INFO\t2016-11-30 13:27:46\tAddOrReplaceReadGroups\tProcessed   139,000,000 records.  Elapsed time: 00:11:59s.  Time for last 1,000,000:    8s.  Last read position: 1:567,284\n",
      "INFO\t2016-11-30 13:27:50\tAddOrReplaceReadGroups\tProcessed   140,000,000 records.  Elapsed time: 00:12:04s.  Time for last 1,000,000:    4s.  Last read position: 16:30,199,282\n",
      "INFO\t2016-11-30 13:27:55\tAddOrReplaceReadGroups\tProcessed   141,000,000 records.  Elapsed time: 00:12:09s.  Time for last 1,000,000:    4s.  Last read position: MT:4,997\n",
      "INFO\t2016-11-30 13:28:00\tAddOrReplaceReadGroups\tProcessed   142,000,000 records.  Elapsed time: 00:12:14s.  Time for last 1,000,000:    4s.  Last read position: 14:77,248,856\n",
      "INFO\t2016-11-30 13:28:05\tAddOrReplaceReadGroups\tProcessed   143,000,000 records.  Elapsed time: 00:12:19s.  Time for last 1,000,000:    5s.  Last read position: 1:109,817,439\n",
      "INFO\t2016-11-30 13:28:10\tAddOrReplaceReadGroups\tProcessed   144,000,000 records.  Elapsed time: 00:12:24s.  Time for last 1,000,000:    4s.  Last read position: 1:567,528\n",
      "INFO\t2016-11-30 13:28:15\tAddOrReplaceReadGroups\tProcessed   145,000,000 records.  Elapsed time: 00:12:29s.  Time for last 1,000,000:    4s.  Last read position: MT:9,130\n",
      "INFO\t2016-11-30 13:28:20\tAddOrReplaceReadGroups\tProcessed   146,000,000 records.  Elapsed time: 00:12:34s.  Time for last 1,000,000:    4s.  Last read position: MT:7,347\n",
      "INFO\t2016-11-30 13:28:25\tAddOrReplaceReadGroups\tProcessed   147,000,000 records.  Elapsed time: 00:12:38s.  Time for last 1,000,000:    4s.  Last read position: X:153,073,495\n",
      "INFO\t2016-11-30 13:28:29\tAddOrReplaceReadGroups\tProcessed   148,000,000 records.  Elapsed time: 00:12:43s.  Time for last 1,000,000:    4s.  Last read position: MT:1,864\n",
      "INFO\t2016-11-30 13:28:34\tAddOrReplaceReadGroups\tProcessed   149,000,000 records.  Elapsed time: 00:12:48s.  Time for last 1,000,000:    4s.  Last read position: MT:11,965\n",
      "INFO\t2016-11-30 13:28:39\tAddOrReplaceReadGroups\tProcessed   150,000,000 records.  Elapsed time: 00:12:53s.  Time for last 1,000,000:    4s.  Last read position: 11:124,790,335\n",
      "INFO\t2016-11-30 13:28:44\tAddOrReplaceReadGroups\tProcessed   151,000,000 records.  Elapsed time: 00:12:58s.  Time for last 1,000,000:    4s.  Last read position: 5:71,504,528\n",
      "INFO\t2016-11-30 13:28:49\tAddOrReplaceReadGroups\tProcessed   152,000,000 records.  Elapsed time: 00:13:03s.  Time for last 1,000,000:    5s.  Last read position: MT:15,836\n",
      "INFO\t2016-11-30 13:28:54\tAddOrReplaceReadGroups\tProcessed   153,000,000 records.  Elapsed time: 00:13:08s.  Time for last 1,000,000:    4s.  Last read position: 1:564,478\n",
      "INFO\t2016-11-30 13:28:59\tAddOrReplaceReadGroups\tProcessed   154,000,000 records.  Elapsed time: 00:13:13s.  Time for last 1,000,000:    4s.  Last read position: 9:86,821,440\n",
      "INFO\t2016-11-30 13:29:05\tAddOrReplaceReadGroups\tProcessed   155,000,000 records.  Elapsed time: 00:13:18s.  Time for last 1,000,000:    5s.  Last read position: MT:6,988\n",
      "INFO\t2016-11-30 13:29:09\tAddOrReplaceReadGroups\tProcessed   156,000,000 records.  Elapsed time: 00:13:23s.  Time for last 1,000,000:    4s.  Last read position: MT:9,812\n",
      "INFO\t2016-11-30 13:29:14\tAddOrReplaceReadGroups\tProcessed   157,000,000 records.  Elapsed time: 00:13:28s.  Time for last 1,000,000:    4s.  Last read position: 19:49,468,836\n",
      "INFO\t2016-11-30 13:29:21\tAddOrReplaceReadGroups\tProcessed   158,000,000 records.  Elapsed time: 00:13:35s.  Time for last 1,000,000:    7s.  Last read position: 11:10,529,888\n",
      "INFO\t2016-11-30 13:29:26\tAddOrReplaceReadGroups\tProcessed   159,000,000 records.  Elapsed time: 00:13:40s.  Time for last 1,000,000:    4s.  Last read position: 16:30,995,810\n",
      "INFO\t2016-11-30 13:29:31\tAddOrReplaceReadGroups\tProcessed   160,000,000 records.  Elapsed time: 00:13:45s.  Time for last 1,000,000:    4s.  Last read position: 1:567,490\n",
      "INFO\t2016-11-30 13:29:37\tAddOrReplaceReadGroups\tProcessed   161,000,000 records.  Elapsed time: 00:13:51s.  Time for last 1,000,000:    6s.  Last read position: 17:7,096,366\n",
      "INFO\t2016-11-30 13:29:42\tAddOrReplaceReadGroups\tProcessed   162,000,000 records.  Elapsed time: 00:13:55s.  Time for last 1,000,000:    4s.  Last read position: 1:568,497\n",
      "INFO\t2016-11-30 13:29:46\tAddOrReplaceReadGroups\tProcessed   163,000,000 records.  Elapsed time: 00:14:00s.  Time for last 1,000,000:    4s.  Last read position: 1:38,010,500\n",
      "INFO\t2016-11-30 13:29:51\tAddOrReplaceReadGroups\tProcessed   164,000,000 records.  Elapsed time: 00:14:05s.  Time for last 1,000,000:    4s.  Last read position: 16:226,772\n",
      "INFO\t2016-11-30 13:29:56\tAddOrReplaceReadGroups\tProcessed   165,000,000 records.  Elapsed time: 00:14:09s.  Time for last 1,000,000:    4s.  Last read position: 5:145,967,605\n",
      "INFO\t2016-11-30 13:30:00\tAddOrReplaceReadGroups\tProcessed   166,000,000 records.  Elapsed time: 00:14:14s.  Time for last 1,000,000:    4s.  Last read position: 4:24,529,119\n",
      "INFO\t2016-11-30 13:30:05\tAddOrReplaceReadGroups\tProcessed   167,000,000 records.  Elapsed time: 00:14:19s.  Time for last 1,000,000:    4s.  Last read position: MT:2,974\n",
      "INFO\t2016-11-30 13:30:09\tAddOrReplaceReadGroups\tProcessed   168,000,000 records.  Elapsed time: 00:14:23s.  Time for last 1,000,000:    4s.  Last read position: MT:5,255\n",
      "INFO\t2016-11-30 13:30:14\tAddOrReplaceReadGroups\tProcessed   169,000,000 records.  Elapsed time: 00:14:28s.  Time for last 1,000,000:    4s.  Last read position: 19:39,336,706\n",
      "INFO\t2016-11-30 13:30:19\tAddOrReplaceReadGroups\tProcessed   170,000,000 records.  Elapsed time: 00:14:33s.  Time for last 1,000,000:    5s.  Last read position: MT:2,677\n",
      "INFO\t2016-11-30 13:30:24\tAddOrReplaceReadGroups\tProcessed   171,000,000 records.  Elapsed time: 00:14:38s.  Time for last 1,000,000:    4s.  Last read position: MT:6,952\n",
      "INFO\t2016-11-30 13:30:28\tAddOrReplaceReadGroups\tProcessed   172,000,000 records.  Elapsed time: 00:14:42s.  Time for last 1,000,000:    4s.  Last read position: MT:2,624\n",
      "INFO\t2016-11-30 13:30:34\tAddOrReplaceReadGroups\tProcessed   173,000,000 records.  Elapsed time: 00:14:48s.  Time for last 1,000,000:    5s.  Last read position: 1:225,684,009\n",
      "INFO\t2016-11-30 13:30:39\tAddOrReplaceReadGroups\tProcessed   174,000,000 records.  Elapsed time: 00:14:53s.  Time for last 1,000,000:    4s.  Last read position: MT:8,670\n",
      "INFO\t2016-11-30 13:30:44\tAddOrReplaceReadGroups\tProcessed   175,000,000 records.  Elapsed time: 00:14:58s.  Time for last 1,000,000:    4s.  Last read position: 12:20,704,444\n",
      "INFO\t2016-11-30 13:30:52\tAddOrReplaceReadGroups\tProcessed   176,000,000 records.  Elapsed time: 00:15:06s.  Time for last 1,000,000:    8s.  Last read position: 17:73,847,669\n",
      "INFO\t2016-11-30 13:30:57\tAddOrReplaceReadGroups\tProcessed   177,000,000 records.  Elapsed time: 00:15:11s.  Time for last 1,000,000:    4s.  Last read position: MT:9,694\n",
      "INFO\t2016-11-30 13:31:02\tAddOrReplaceReadGroups\tProcessed   178,000,000 records.  Elapsed time: 00:15:15s.  Time for last 1,000,000:    4s.  Last read position: 18:74,691,477\n",
      "INFO\t2016-11-30 13:31:06\tAddOrReplaceReadGroups\tProcessed   179,000,000 records.  Elapsed time: 00:15:20s.  Time for last 1,000,000:    4s.  Last read position: 19:1,257,601\n",
      "[Wed Nov 30 13:42:03 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 26.30 minutes.\n",
      "Runtime.totalMemory()=10744758272\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P7Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P7rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 13:42:04 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P7Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P7rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 13:42:04 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 13:42:04\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 13:42:08\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 16:21,877,420\n",
      "INFO\t2016-11-30 13:42:14\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 12:113,730,771\n",
      "INFO\t2016-11-30 13:42:19\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:14s.  Time for last 1,000,000:    4s.  Last read position: 1:27,731,008\n",
      "INFO\t2016-11-30 13:42:24\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    5s.  Last read position: 10:103,558,817\n",
      "INFO\t2016-11-30 13:42:29\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    4s.  Last read position: 12:27,133,564\n",
      "INFO\t2016-11-30 13:42:36\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    6s.  Last read position: 11:66,434,022\n",
      "INFO\t2016-11-30 13:42:41\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    5s.  Last read position: 21:32,494,038\n",
      "INFO\t2016-11-30 13:42:46\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    5s.  Last read position: GL000221.1:61,789\n",
      "INFO\t2016-11-30 13:42:51\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:47s.  Time for last 1,000,000:    4s.  Last read position: 12:125,396,562\n",
      "INFO\t2016-11-30 13:42:56\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    4s.  Last read position: 9:70,447,727\n",
      "INFO\t2016-11-30 13:43:03\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:00:58s.  Time for last 1,000,000:    7s.  Last read position: 3:195,595,153\n",
      "INFO\t2016-11-30 13:43:08\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    4s.  Last read position: 19:10,465,247\n",
      "INFO\t2016-11-30 13:43:13\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    4s.  Last read position: 1:156,278,891\n",
      "INFO\t2016-11-30 13:43:19\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    6s.  Last read position: 15:66,793,750\n",
      "INFO\t2016-11-30 13:43:24\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:20s.  Time for last 1,000,000:    4s.  Last read position: 12:50,038,115\n",
      "INFO\t2016-11-30 13:43:29\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:25s.  Time for last 1,000,000:    4s.  Last read position: 6:31,784,797\n",
      "INFO\t2016-11-30 13:43:34\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    4s.  Last read position: 17:60,770,288\n",
      "INFO\t2016-11-30 13:43:39\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:34s.  Time for last 1,000,000:    4s.  Last read position: 19:57,185,085\n",
      "INFO\t2016-11-30 13:43:44\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    4s.  Last read position: 6:137,518,942\n",
      "INFO\t2016-11-30 13:43:49\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    5s.  Last read position: 17:34,538,606\n",
      "INFO\t2016-11-30 13:43:54\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    5s.  Last read position: 9:135,895,748\n",
      "INFO\t2016-11-30 13:43:59\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:01:54s.  Time for last 1,000,000:    4s.  Last read position: 13:25,671,361\n",
      "INFO\t2016-11-30 13:44:04\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:01:59s.  Time for last 1,000,000:    4s.  Last read position: 19:56,180,102\n",
      "INFO\t2016-11-30 13:44:09\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:04s.  Time for last 1,000,000:    4s.  Last read position: 1:27,272,696\n",
      "INFO\t2016-11-30 13:44:14\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:09s.  Time for last 1,000,000:    4s.  Last read position: MT:11,281\n",
      "INFO\t2016-11-30 13:44:19\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:15s.  Time for last 1,000,000:    5s.  Last read position: 6:31,795,957\n",
      "INFO\t2016-11-30 13:44:24\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:20s.  Time for last 1,000,000:    4s.  Last read position: 21:16,339,507\n",
      "INFO\t2016-11-30 13:44:29\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:25s.  Time for last 1,000,000:    4s.  Last read position: 8:16,035,452\n",
      "INFO\t2016-11-30 13:44:35\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:30s.  Time for last 1,000,000:    5s.  Last read position: 4:88,898,059\n",
      "INFO\t2016-11-30 13:44:40\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:35s.  Time for last 1,000,000:    4s.  Last read position: 11:61,235,411\n",
      "INFO\t2016-11-30 13:44:45\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:40s.  Time for last 1,000,000:    4s.  Last read position: 6:31,554,870\n",
      "INFO\t2016-11-30 13:44:52\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:47s.  Time for last 1,000,000:    7s.  Last read position: 13:45,914,161\n",
      "INFO\t2016-11-30 13:44:57\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:02:52s.  Time for last 1,000,000:    4s.  Last read position: 19:49,469,854\n",
      "INFO\t2016-11-30 13:45:02\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:02:57s.  Time for last 1,000,000:    4s.  Last read position: 1:45,243,369\n",
      "INFO\t2016-11-30 13:45:08\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:04s.  Time for last 1,000,000:    6s.  Last read position: 12:56,112,937\n",
      "INFO\t2016-11-30 13:45:13\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:09s.  Time for last 1,000,000:    4s.  Last read position: 3:121,213,220\n",
      "INFO\t2016-11-30 13:45:18\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:14s.  Time for last 1,000,000:    4s.  Last read position: 16:67,116,205\n",
      "INFO\t2016-11-30 13:45:23\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:19s.  Time for last 1,000,000:    4s.  Last read position: 15:64,993,844\n",
      "INFO\t2016-11-30 13:45:28\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:24s.  Time for last 1,000,000:    5s.  Last read position: 22:44,583,711\n",
      "INFO\t2016-11-30 13:45:33\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:29s.  Time for last 1,000,000:    4s.  Last read position: 5:78,580,639\n",
      "INFO\t2016-11-30 13:45:38\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:34s.  Time for last 1,000,000:    4s.  Last read position: 17:32,582,372\n",
      "INFO\t2016-11-30 13:45:43\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:39s.  Time for last 1,000,000:    5s.  Last read position: 6:6,248,586\n",
      "INFO\t2016-11-30 13:45:48\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:44s.  Time for last 1,000,000:    4s.  Last read position: 9:127,634,095\n",
      "INFO\t2016-11-30 13:45:53\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:03:48s.  Time for last 1,000,000:    4s.  Last read position: 19:12,575,145\n",
      "INFO\t2016-11-30 13:45:58\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:03:53s.  Time for last 1,000,000:    4s.  Last read position: 6:74,227,539\n",
      "INFO\t2016-11-30 13:46:03\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:03:58s.  Time for last 1,000,000:    5s.  Last read position: 19:40,370,291\n",
      "INFO\t2016-11-30 13:46:08\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:03s.  Time for last 1,000,000:    4s.  Last read position: 4:88,902,660\n",
      "INFO\t2016-11-30 13:46:13\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:08s.  Time for last 1,000,000:    5s.  Last read position: X:71,495,455\n",
      "INFO\t2016-11-30 13:46:18\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:13s.  Time for last 1,000,000:    4s.  Last read position: 7:2,703,227\n",
      "INFO\t2016-11-30 13:46:23\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:18s.  Time for last 1,000,000:    4s.  Last read position: 4:91,760,019\n",
      "INFO\t2016-11-30 13:46:28\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:23s.  Time for last 1,000,000:    4s.  Last read position: 20:25,290,101\n",
      "INFO\t2016-11-30 13:46:33\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:28s.  Time for last 1,000,000:    4s.  Last read position: 22:20,106,654\n",
      "INFO\t2016-11-30 13:46:37\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:33s.  Time for last 1,000,000:    4s.  Last read position: 12:49,331,842\n",
      "INFO\t2016-11-30 13:46:43\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:04:38s.  Time for last 1,000,000:    5s.  Last read position: MT:7,022\n",
      "INFO\t2016-11-30 13:46:48\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:04:43s.  Time for last 1,000,000:    4s.  Last read position: 7:5,569,173\n",
      "INFO\t2016-11-30 13:46:53\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:04:48s.  Time for last 1,000,000:    4s.  Last read position: 2:74,783,605\n",
      "INFO\t2016-11-30 13:46:58\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:04:53s.  Time for last 1,000,000:    5s.  Last read position: MT:6,876\n",
      "INFO\t2016-11-30 13:47:03\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:04:58s.  Time for last 1,000,000:    4s.  Last read position: 3:57,570,140\n",
      "INFO\t2016-11-30 13:47:07\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:03s.  Time for last 1,000,000:    4s.  Last read position: 12:104,659,187\n",
      "INFO\t2016-11-30 13:47:14\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:09s.  Time for last 1,000,000:    6s.  Last read position: 6:30,458,969\n",
      "INFO\t2016-11-30 13:47:19\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:14s.  Time for last 1,000,000:    4s.  Last read position: 8:24,170,920\n",
      "INFO\t2016-11-30 13:47:23\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:19s.  Time for last 1,000,000:    4s.  Last read position: 1:46,775,579\n",
      "INFO\t2016-11-30 13:47:31\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:26s.  Time for last 1,000,000:    7s.  Last read position: 19:40,434,058\n",
      "INFO\t2016-11-30 13:47:36\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:31s.  Time for last 1,000,000:    4s.  Last read position: 6:50,880,577\n",
      "INFO\t2016-11-30 13:47:40\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:36s.  Time for last 1,000,000:    4s.  Last read position: 22:41,612,235\n",
      "INFO\t2016-11-30 13:47:48\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:05:44s.  Time for last 1,000,000:    7s.  Last read position: MT:15,682\n",
      "INFO\t2016-11-30 13:47:53\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:05:49s.  Time for last 1,000,000:    4s.  Last read position: 14:95,193,004\n",
      "INFO\t2016-11-30 13:47:58\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:05:53s.  Time for last 1,000,000:    4s.  Last read position: 12:50,152,194\n",
      "INFO\t2016-11-30 13:48:03\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:05:58s.  Time for last 1,000,000:    4s.  Last read position: 19:51,655,311\n",
      "INFO\t2016-11-30 13:48:08\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:03s.  Time for last 1,000,000:    4s.  Last read position: 16:84,599,857\n",
      "INFO\t2016-11-30 13:48:12\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:08s.  Time for last 1,000,000:    4s.  Last read position: 6:50,880,324\n",
      "INFO\t2016-11-30 13:48:17\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:13s.  Time for last 1,000,000:    4s.  Last read position: 16:15,223,739\n",
      "INFO\t2016-11-30 13:48:22\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:18s.  Time for last 1,000,000:    4s.  Last read position: 7:72,422,788\n",
      "INFO\t2016-11-30 13:48:27\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    4s.  Last read position: 8:11,702,560\n",
      "INFO\t2016-11-30 13:48:32\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:28s.  Time for last 1,000,000:    4s.  Last read position: 4:88,898,074\n",
      "INFO\t2016-11-30 13:48:37\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:32s.  Time for last 1,000,000:    4s.  Last read position: X:153,664,553\n",
      "INFO\t2016-11-30 13:48:42\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:06:37s.  Time for last 1,000,000:    4s.  Last read position: 3:98,503,835\n",
      "INFO\t2016-11-30 13:48:47\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:06:42s.  Time for last 1,000,000:    5s.  Last read position: 1:8,924,113\n",
      "INFO\t2016-11-30 13:48:52\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:06:47s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,459\n",
      "INFO\t2016-11-30 13:48:57\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:06:52s.  Time for last 1,000,000:    4s.  Last read position: 6:26,393,632\n",
      "INFO\t2016-11-30 13:49:01\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:06:57s.  Time for last 1,000,000:    4s.  Last read position: MT:131\n",
      "INFO\t2016-11-30 13:49:06\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:02s.  Time for last 1,000,000:    4s.  Last read position: 3:158,547,020\n",
      "INFO\t2016-11-30 13:49:11\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:07s.  Time for last 1,000,000:    4s.  Last read position: 1:113,245,216\n",
      "INFO\t2016-11-30 13:49:16\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:12s.  Time for last 1,000,000:    4s.  Last read position: 17:7,212,951\n",
      "INFO\t2016-11-30 13:49:21\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:17s.  Time for last 1,000,000:    4s.  Last read position: 7:5,567,447\n",
      "INFO\t2016-11-30 13:49:26\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:21s.  Time for last 1,000,000:    4s.  Last read position: 2:127,314,938\n",
      "INFO\t2016-11-30 13:49:31\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:27s.  Time for last 1,000,000:    5s.  Last read position: 4:88,904,202\n",
      "INFO\t2016-11-30 13:49:36\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:31s.  Time for last 1,000,000:    4s.  Last read position: 13:48,985,422\n",
      "INFO\t2016-11-30 13:49:41\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:07:36s.  Time for last 1,000,000:    4s.  Last read position: 1:28,149,012\n",
      "INFO\t2016-11-30 13:49:47\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:07:43s.  Time for last 1,000,000:    6s.  Last read position: 1:43,162,578\n",
      "INFO\t2016-11-30 13:49:52\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:07:47s.  Time for last 1,000,000:    4s.  Last read position: 4:89,602,447\n",
      "INFO\t2016-11-30 13:49:57\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:07:52s.  Time for last 1,000,000:    4s.  Last read position: 1:47,052,814\n",
      "INFO\t2016-11-30 13:50:04\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:00s.  Time for last 1,000,000:    7s.  Last read position: MT:747\n",
      "INFO\t2016-11-30 13:50:09\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:05s.  Time for last 1,000,000:    4s.  Last read position: 8:142,202,375\n",
      "INFO\t2016-11-30 13:50:14\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:10s.  Time for last 1,000,000:    4s.  Last read position: MT:14,872\n",
      "INFO\t2016-11-30 13:50:23\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:18s.  Time for last 1,000,000:    8s.  Last read position: 1:567,637\n",
      "INFO\t2016-11-30 13:50:27\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:23s.  Time for last 1,000,000:    4s.  Last read position: MT:7,235\n",
      "INFO\t2016-11-30 13:50:32\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:28s.  Time for last 1,000,000:    4s.  Last read position: 1:567,993\n",
      "INFO\t2016-11-30 13:50:37\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:33s.  Time for last 1,000,000:    4s.  Last read position: 6:44,221,283\n",
      "INFO\t2016-11-30 13:50:42\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:38s.  Time for last 1,000,000:    4s.  Last read position: 19:12,805,439\n",
      "INFO\t2016-11-30 13:50:47\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:08:42s.  Time for last 1,000,000:    4s.  Last read position: 16:28,622,456\n",
      "INFO\t2016-11-30 13:50:52\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:08:47s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,237\n",
      "INFO\t2016-11-30 13:50:57\tAddOrReplaceReadGroups\tProcessed   103,000,000 records.  Elapsed time: 00:08:52s.  Time for last 1,000,000:    4s.  Last read position: MT:9,051\n",
      "INFO\t2016-11-30 13:51:02\tAddOrReplaceReadGroups\tProcessed   104,000,000 records.  Elapsed time: 00:08:57s.  Time for last 1,000,000:    4s.  Last read position: 1:7,829,710\n",
      "INFO\t2016-11-30 13:51:06\tAddOrReplaceReadGroups\tProcessed   105,000,000 records.  Elapsed time: 00:09:02s.  Time for last 1,000,000:    4s.  Last read position: MT:7,134\n",
      "INFO\t2016-11-30 13:51:11\tAddOrReplaceReadGroups\tProcessed   106,000,000 records.  Elapsed time: 00:09:07s.  Time for last 1,000,000:    5s.  Last read position: MT:5,979\n",
      "INFO\t2016-11-30 13:51:16\tAddOrReplaceReadGroups\tProcessed   107,000,000 records.  Elapsed time: 00:09:12s.  Time for last 1,000,000:    4s.  Last read position: MT:9,347\n",
      "INFO\t2016-11-30 13:51:21\tAddOrReplaceReadGroups\tProcessed   108,000,000 records.  Elapsed time: 00:09:17s.  Time for last 1,000,000:    4s.  Last read position: MT:6,592\n",
      "INFO\t2016-11-30 13:51:28\tAddOrReplaceReadGroups\tProcessed   109,000,000 records.  Elapsed time: 00:09:24s.  Time for last 1,000,000:    6s.  Last read position: 12:1,889,744\n",
      "INFO\t2016-11-30 13:51:33\tAddOrReplaceReadGroups\tProcessed   110,000,000 records.  Elapsed time: 00:09:28s.  Time for last 1,000,000:    4s.  Last read position: MT:14,087\n",
      "INFO\t2016-11-30 13:51:38\tAddOrReplaceReadGroups\tProcessed   111,000,000 records.  Elapsed time: 00:09:33s.  Time for last 1,000,000:    4s.  Last read position: MT:14,559\n",
      "INFO\t2016-11-30 13:51:45\tAddOrReplaceReadGroups\tProcessed   112,000,000 records.  Elapsed time: 00:09:40s.  Time for last 1,000,000:    6s.  Last read position: 3:141,089,981\n",
      "INFO\t2016-11-30 13:51:49\tAddOrReplaceReadGroups\tProcessed   113,000,000 records.  Elapsed time: 00:09:45s.  Time for last 1,000,000:    4s.  Last read position: 5:43,527,497\n",
      "INFO\t2016-11-30 13:51:54\tAddOrReplaceReadGroups\tProcessed   114,000,000 records.  Elapsed time: 00:09:50s.  Time for last 1,000,000:    4s.  Last read position: 11:47,491,393\n",
      "INFO\t2016-11-30 13:52:01\tAddOrReplaceReadGroups\tProcessed   115,000,000 records.  Elapsed time: 00:09:57s.  Time for last 1,000,000:    7s.  Last read position: 7:94,296,461\n",
      "INFO\t2016-11-30 13:52:06\tAddOrReplaceReadGroups\tProcessed   116,000,000 records.  Elapsed time: 00:10:02s.  Time for last 1,000,000:    4s.  Last read position: MT:6,078\n",
      "INFO\t2016-11-30 13:52:11\tAddOrReplaceReadGroups\tProcessed   117,000,000 records.  Elapsed time: 00:10:07s.  Time for last 1,000,000:    4s.  Last read position: 2:198,244,638\n",
      "INFO\t2016-11-30 13:52:16\tAddOrReplaceReadGroups\tProcessed   118,000,000 records.  Elapsed time: 00:10:11s.  Time for last 1,000,000:    4s.  Last read position: 19:3,296,890\n",
      "INFO\t2016-11-30 13:52:21\tAddOrReplaceReadGroups\tProcessed   119,000,000 records.  Elapsed time: 00:10:16s.  Time for last 1,000,000:    4s.  Last read position: 19:6,440,733\n",
      "INFO\t2016-11-30 13:52:26\tAddOrReplaceReadGroups\tProcessed   120,000,000 records.  Elapsed time: 00:10:21s.  Time for last 1,000,000:    4s.  Last read position: 13:78,912,140\n",
      "INFO\t2016-11-30 13:52:30\tAddOrReplaceReadGroups\tProcessed   121,000,000 records.  Elapsed time: 00:10:26s.  Time for last 1,000,000:    4s.  Last read position: 11:35,642,157\n",
      "INFO\t2016-11-30 13:52:35\tAddOrReplaceReadGroups\tProcessed   122,000,000 records.  Elapsed time: 00:10:31s.  Time for last 1,000,000:    4s.  Last read position: 1:1,717,637\n",
      "INFO\t2016-11-30 13:52:40\tAddOrReplaceReadGroups\tProcessed   123,000,000 records.  Elapsed time: 00:10:36s.  Time for last 1,000,000:    4s.  Last read position: 5:71,491,067\n",
      "INFO\t2016-11-30 13:52:45\tAddOrReplaceReadGroups\tProcessed   124,000,000 records.  Elapsed time: 00:10:41s.  Time for last 1,000,000:    4s.  Last read position: 19:17,688,883\n",
      "INFO\t2016-11-30 13:52:50\tAddOrReplaceReadGroups\tProcessed   125,000,000 records.  Elapsed time: 00:10:45s.  Time for last 1,000,000:    4s.  Last read position: 11:134,117,053\n",
      "INFO\t2016-11-30 13:52:54\tAddOrReplaceReadGroups\tProcessed   126,000,000 records.  Elapsed time: 00:10:50s.  Time for last 1,000,000:    4s.  Last read position: 16:30,080,261\n",
      "INFO\t2016-11-30 13:53:00\tAddOrReplaceReadGroups\tProcessed   127,000,000 records.  Elapsed time: 00:10:55s.  Time for last 1,000,000:    5s.  Last read position: 5:6,492,354\n",
      "INFO\t2016-11-30 13:53:05\tAddOrReplaceReadGroups\tProcessed   128,000,000 records.  Elapsed time: 00:11:00s.  Time for last 1,000,000:    4s.  Last read position: X:125,606,476\n",
      "INFO\t2016-11-30 13:53:09\tAddOrReplaceReadGroups\tProcessed   129,000,000 records.  Elapsed time: 00:11:05s.  Time for last 1,000,000:    4s.  Last read position: MT:7,239\n",
      "INFO\t2016-11-30 13:53:15\tAddOrReplaceReadGroups\tProcessed   130,000,000 records.  Elapsed time: 00:11:11s.  Time for last 1,000,000:    5s.  Last read position: 6:33,269,020\n",
      "INFO\t2016-11-30 13:53:20\tAddOrReplaceReadGroups\tProcessed   131,000,000 records.  Elapsed time: 00:11:15s.  Time for last 1,000,000:    4s.  Last read position: 11:125,351,435\n",
      "INFO\t2016-11-30 13:53:24\tAddOrReplaceReadGroups\tProcessed   132,000,000 records.  Elapsed time: 00:11:20s.  Time for last 1,000,000:    4s.  Last read position: 2:47,387,913\n",
      "INFO\t2016-11-30 13:53:31\tAddOrReplaceReadGroups\tProcessed   133,000,000 records.  Elapsed time: 00:11:27s.  Time for last 1,000,000:    6s.  Last read position: MT:6,556\n",
      "INFO\t2016-11-30 13:53:36\tAddOrReplaceReadGroups\tProcessed   134,000,000 records.  Elapsed time: 00:11:31s.  Time for last 1,000,000:    4s.  Last read position: 11:60,698,048\n",
      "INFO\t2016-11-30 13:53:41\tAddOrReplaceReadGroups\tProcessed   135,000,000 records.  Elapsed time: 00:11:36s.  Time for last 1,000,000:    4s.  Last read position: 8:17,605,553\n",
      "INFO\t2016-11-30 13:53:48\tAddOrReplaceReadGroups\tProcessed   136,000,000 records.  Elapsed time: 00:11:43s.  Time for last 1,000,000:    7s.  Last read position: 1:20,809,425\n",
      "INFO\t2016-11-30 13:53:52\tAddOrReplaceReadGroups\tProcessed   137,000,000 records.  Elapsed time: 00:11:48s.  Time for last 1,000,000:    4s.  Last read position: 16:87,731,643\n",
      "INFO\t2016-11-30 13:53:57\tAddOrReplaceReadGroups\tProcessed   138,000,000 records.  Elapsed time: 00:11:52s.  Time for last 1,000,000:    4s.  Last read position: 1:567,502\n",
      "INFO\t2016-11-30 13:54:01\tAddOrReplaceReadGroups\tProcessed   139,000,000 records.  Elapsed time: 00:11:57s.  Time for last 1,000,000:    4s.  Last read position: 4:6,114,518\n",
      "INFO\t2016-11-30 13:54:06\tAddOrReplaceReadGroups\tProcessed   140,000,000 records.  Elapsed time: 00:12:02s.  Time for last 1,000,000:    4s.  Last read position: 6:24,977,011\n",
      "INFO\t2016-11-30 13:54:11\tAddOrReplaceReadGroups\tProcessed   141,000,000 records.  Elapsed time: 00:12:06s.  Time for last 1,000,000:    4s.  Last read position: 1:569,412\n",
      "INFO\t2016-11-30 13:54:15\tAddOrReplaceReadGroups\tProcessed   142,000,000 records.  Elapsed time: 00:12:11s.  Time for last 1,000,000:    4s.  Last read position: 2:31,484,434\n",
      "INFO\t2016-11-30 13:54:20\tAddOrReplaceReadGroups\tProcessed   143,000,000 records.  Elapsed time: 00:12:16s.  Time for last 1,000,000:    4s.  Last read position: 19:39,888,710\n",
      "INFO\t2016-11-30 13:54:25\tAddOrReplaceReadGroups\tProcessed   144,000,000 records.  Elapsed time: 00:12:20s.  Time for last 1,000,000:    4s.  Last read position: 5:150,030,418\n",
      "INFO\t2016-11-30 13:54:29\tAddOrReplaceReadGroups\tProcessed   145,000,000 records.  Elapsed time: 00:12:25s.  Time for last 1,000,000:    4s.  Last read position: 9:73,213,427\n",
      "INFO\t2016-11-30 13:54:34\tAddOrReplaceReadGroups\tProcessed   146,000,000 records.  Elapsed time: 00:12:30s.  Time for last 1,000,000:    4s.  Last read position: 9:34,398,588\n",
      "INFO\t2016-11-30 13:54:39\tAddOrReplaceReadGroups\tProcessed   147,000,000 records.  Elapsed time: 00:12:34s.  Time for last 1,000,000:    4s.  Last read position: 1:569,648\n",
      "INFO\t2016-11-30 13:54:43\tAddOrReplaceReadGroups\tProcessed   148,000,000 records.  Elapsed time: 00:12:39s.  Time for last 1,000,000:    4s.  Last read position: 14:103,809,188\n",
      "INFO\t2016-11-30 13:54:49\tAddOrReplaceReadGroups\tProcessed   149,000,000 records.  Elapsed time: 00:12:45s.  Time for last 1,000,000:    5s.  Last read position: 6:122,765,186\n",
      "INFO\t2016-11-30 13:54:54\tAddOrReplaceReadGroups\tProcessed   150,000,000 records.  Elapsed time: 00:12:49s.  Time for last 1,000,000:    4s.  Last read position: 1:569,348\n",
      "INFO\t2016-11-30 13:54:58\tAddOrReplaceReadGroups\tProcessed   151,000,000 records.  Elapsed time: 00:12:54s.  Time for last 1,000,000:    4s.  Last read position: 3:96,336,934\n",
      "INFO\t2016-11-30 13:55:04\tAddOrReplaceReadGroups\tProcessed   152,000,000 records.  Elapsed time: 00:13:00s.  Time for last 1,000,000:    5s.  Last read position: MT:7,211\n",
      "INFO\t2016-11-30 13:55:09\tAddOrReplaceReadGroups\tProcessed   153,000,000 records.  Elapsed time: 00:13:04s.  Time for last 1,000,000:    4s.  Last read position: 1:174,964,057\n",
      "[Wed Nov 30 14:04:26 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 22.37 minutes.\n",
      "Runtime.totalMemory()=10777264128\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar AddOrReplaceReadGroups I=/data/scratch/iholtman/human_microglia_genotyping/step4/P8Aligned.out.sam O=/data/scratch/iholtman/human_microglia_genotyping/step5/P8rg_added_sorted.bam SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\n",
      "[Wed Nov 30 14:04:26 PST 2016] picard.sam.AddOrReplaceReadGroups INPUT=/data/scratch/iholtman/human_microglia_genotyping/step4/P8Aligned.out.sam OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P8rg_added_sorted.bam SORT_ORDER=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample    VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 14:04:26 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 14:04:26\tAddOrReplaceReadGroups\tCreated read group ID=ILLUMINA PL=ILLUMINA LB=ILLUMINA SM=sample\n",
      "\n",
      "INFO\t2016-11-30 14:04:31\tAddOrReplaceReadGroups\tProcessed     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 20:32,881,947\n",
      "INFO\t2016-11-30 14:04:37\tAddOrReplaceReadGroups\tProcessed     2,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    5s.  Last read position: X:12,995,131\n",
      "INFO\t2016-11-30 14:04:42\tAddOrReplaceReadGroups\tProcessed     3,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 4:183,812,203\n",
      "INFO\t2016-11-30 14:04:48\tAddOrReplaceReadGroups\tProcessed     4,000,000 records.  Elapsed time: 00:00:21s.  Time for last 1,000,000:    5s.  Last read position: 5:114,726,696\n",
      "INFO\t2016-11-30 14:04:55\tAddOrReplaceReadGroups\tProcessed     5,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:    6s.  Last read position: 3:12,881,661\n",
      "INFO\t2016-11-30 14:05:00\tAddOrReplaceReadGroups\tProcessed     6,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    5s.  Last read position: 5:140,011,590\n",
      "INFO\t2016-11-30 14:05:06\tAddOrReplaceReadGroups\tProcessed     7,000,000 records.  Elapsed time: 00:00:39s.  Time for last 1,000,000:    5s.  Last read position: 6:160,101,113\n",
      "INFO\t2016-11-30 14:05:11\tAddOrReplaceReadGroups\tProcessed     8,000,000 records.  Elapsed time: 00:00:45s.  Time for last 1,000,000:    5s.  Last read position: 19:49,469,855\n",
      "INFO\t2016-11-30 14:05:17\tAddOrReplaceReadGroups\tProcessed     9,000,000 records.  Elapsed time: 00:00:50s.  Time for last 1,000,000:    5s.  Last read position: 3:39,450,144\n",
      "INFO\t2016-11-30 14:05:22\tAddOrReplaceReadGroups\tProcessed    10,000,000 records.  Elapsed time: 00:00:55s.  Time for last 1,000,000:    5s.  Last read position: 4:88,903,732\n",
      "INFO\t2016-11-30 14:05:28\tAddOrReplaceReadGroups\tProcessed    11,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    6s.  Last read position: 1:33,475,912\n",
      "INFO\t2016-11-30 14:05:33\tAddOrReplaceReadGroups\tProcessed    12,000,000 records.  Elapsed time: 00:01:07s.  Time for last 1,000,000:    5s.  Last read position: 11:118,888,131\n",
      "INFO\t2016-11-30 14:05:39\tAddOrReplaceReadGroups\tProcessed    13,000,000 records.  Elapsed time: 00:01:12s.  Time for last 1,000,000:    5s.  Last read position: X:1,429,634\n",
      "INFO\t2016-11-30 14:05:45\tAddOrReplaceReadGroups\tProcessed    14,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:    6s.  Last read position: 2:42,530,277\n",
      "INFO\t2016-11-30 14:05:50\tAddOrReplaceReadGroups\tProcessed    15,000,000 records.  Elapsed time: 00:01:23s.  Time for last 1,000,000:    5s.  Last read position: 4:88,901,205\n",
      "INFO\t2016-11-30 14:05:55\tAddOrReplaceReadGroups\tProcessed    16,000,000 records.  Elapsed time: 00:01:28s.  Time for last 1,000,000:    5s.  Last read position: 18:48,439,262\n",
      "INFO\t2016-11-30 14:06:03\tAddOrReplaceReadGroups\tProcessed    17,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    7s.  Last read position: 11:61,732,264\n",
      "INFO\t2016-11-30 14:06:08\tAddOrReplaceReadGroups\tProcessed    18,000,000 records.  Elapsed time: 00:01:41s.  Time for last 1,000,000:    5s.  Last read position: 16:70,736,156\n",
      "INFO\t2016-11-30 14:06:13\tAddOrReplaceReadGroups\tProcessed    19,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    5s.  Last read position: 19:49,468,674\n",
      "INFO\t2016-11-30 14:06:21\tAddOrReplaceReadGroups\tProcessed    20,000,000 records.  Elapsed time: 00:01:54s.  Time for last 1,000,000:    7s.  Last read position: 2:229,889,518\n",
      "INFO\t2016-11-30 14:06:26\tAddOrReplaceReadGroups\tProcessed    21,000,000 records.  Elapsed time: 00:01:59s.  Time for last 1,000,000:    5s.  Last read position: 7:12,276,728\n",
      "INFO\t2016-11-30 14:06:31\tAddOrReplaceReadGroups\tProcessed    22,000,000 records.  Elapsed time: 00:02:04s.  Time for last 1,000,000:    5s.  Last read position: 1:228,823,113\n",
      "INFO\t2016-11-30 14:06:37\tAddOrReplaceReadGroups\tProcessed    23,000,000 records.  Elapsed time: 00:02:10s.  Time for last 1,000,000:    5s.  Last read position: 11:64,802,369\n",
      "INFO\t2016-11-30 14:06:42\tAddOrReplaceReadGroups\tProcessed    24,000,000 records.  Elapsed time: 00:02:15s.  Time for last 1,000,000:    5s.  Last read position: 1:245,019,209\n",
      "INFO\t2016-11-30 14:06:47\tAddOrReplaceReadGroups\tProcessed    25,000,000 records.  Elapsed time: 00:02:20s.  Time for last 1,000,000:    5s.  Last read position: 10:17,279,231\n",
      "INFO\t2016-11-30 14:06:52\tAddOrReplaceReadGroups\tProcessed    26,000,000 records.  Elapsed time: 00:02:25s.  Time for last 1,000,000:    5s.  Last read position: 17:79,479,137\n",
      "INFO\t2016-11-30 14:06:58\tAddOrReplaceReadGroups\tProcessed    27,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    5s.  Last read position: 12:9,264,982\n",
      "INFO\t2016-11-30 14:07:03\tAddOrReplaceReadGroups\tProcessed    28,000,000 records.  Elapsed time: 00:02:36s.  Time for last 1,000,000:    5s.  Last read position: 11:27,603,311\n",
      "INFO\t2016-11-30 14:07:08\tAddOrReplaceReadGroups\tProcessed    29,000,000 records.  Elapsed time: 00:02:41s.  Time for last 1,000,000:    5s.  Last read position: MT:4,012\n",
      "INFO\t2016-11-30 14:07:14\tAddOrReplaceReadGroups\tProcessed    30,000,000 records.  Elapsed time: 00:02:47s.  Time for last 1,000,000:    5s.  Last read position: 16:30,080,922\n",
      "INFO\t2016-11-30 14:07:19\tAddOrReplaceReadGroups\tProcessed    31,000,000 records.  Elapsed time: 00:02:52s.  Time for last 1,000,000:    5s.  Last read position: 3:27,676,117\n",
      "INFO\t2016-11-30 14:07:24\tAddOrReplaceReadGroups\tProcessed    32,000,000 records.  Elapsed time: 00:02:57s.  Time for last 1,000,000:    5s.  Last read position: X:153,006,101\n",
      "INFO\t2016-11-30 14:07:29\tAddOrReplaceReadGroups\tProcessed    33,000,000 records.  Elapsed time: 00:03:02s.  Time for last 1,000,000:    5s.  Last read position: 2:74,732,229\n",
      "INFO\t2016-11-30 14:07:34\tAddOrReplaceReadGroups\tProcessed    34,000,000 records.  Elapsed time: 00:03:07s.  Time for last 1,000,000:    5s.  Last read position: 6:160,101,642\n",
      "INFO\t2016-11-30 14:07:40\tAddOrReplaceReadGroups\tProcessed    35,000,000 records.  Elapsed time: 00:03:13s.  Time for last 1,000,000:    5s.  Last read position: X:65,241,697\n",
      "INFO\t2016-11-30 14:07:45\tAddOrReplaceReadGroups\tProcessed    36,000,000 records.  Elapsed time: 00:03:18s.  Time for last 1,000,000:    5s.  Last read position: 11:61,732,361\n",
      "INFO\t2016-11-30 14:07:50\tAddOrReplaceReadGroups\tProcessed    37,000,000 records.  Elapsed time: 00:03:23s.  Time for last 1,000,000:    5s.  Last read position: 11:61,404,933\n",
      "INFO\t2016-11-30 14:07:56\tAddOrReplaceReadGroups\tProcessed    38,000,000 records.  Elapsed time: 00:03:29s.  Time for last 1,000,000:    6s.  Last read position: 18:9,583,168\n",
      "INFO\t2016-11-30 14:08:01\tAddOrReplaceReadGroups\tProcessed    39,000,000 records.  Elapsed time: 00:03:34s.  Time for last 1,000,000:    5s.  Last read position: 19:13,065,040\n",
      "INFO\t2016-11-30 14:08:06\tAddOrReplaceReadGroups\tProcessed    40,000,000 records.  Elapsed time: 00:03:40s.  Time for last 1,000,000:    5s.  Last read position: 17:7,464,659\n",
      "INFO\t2016-11-30 14:08:12\tAddOrReplaceReadGroups\tProcessed    41,000,000 records.  Elapsed time: 00:03:45s.  Time for last 1,000,000:    5s.  Last read position: 13:24,874,670\n",
      "INFO\t2016-11-30 14:08:17\tAddOrReplaceReadGroups\tProcessed    42,000,000 records.  Elapsed time: 00:03:50s.  Time for last 1,000,000:    5s.  Last read position: 11:1,774,346\n",
      "INFO\t2016-11-30 14:08:22\tAddOrReplaceReadGroups\tProcessed    43,000,000 records.  Elapsed time: 00:03:55s.  Time for last 1,000,000:    5s.  Last read position: 1:145,442,443\n",
      "INFO\t2016-11-30 14:08:29\tAddOrReplaceReadGroups\tProcessed    44,000,000 records.  Elapsed time: 00:04:02s.  Time for last 1,000,000:    6s.  Last read position: 11:64,888,264\n",
      "INFO\t2016-11-30 14:08:34\tAddOrReplaceReadGroups\tProcessed    45,000,000 records.  Elapsed time: 00:04:07s.  Time for last 1,000,000:    5s.  Last read position: 7:27,872,163\n",
      "INFO\t2016-11-30 14:08:40\tAddOrReplaceReadGroups\tProcessed    46,000,000 records.  Elapsed time: 00:04:13s.  Time for last 1,000,000:    5s.  Last read position: 4:57,368,080\n",
      "INFO\t2016-11-30 14:08:46\tAddOrReplaceReadGroups\tProcessed    47,000,000 records.  Elapsed time: 00:04:19s.  Time for last 1,000,000:    6s.  Last read position: 1:12,249,824\n",
      "INFO\t2016-11-30 14:08:51\tAddOrReplaceReadGroups\tProcessed    48,000,000 records.  Elapsed time: 00:04:24s.  Time for last 1,000,000:    5s.  Last read position: 19:10,820,687\n",
      "INFO\t2016-11-30 14:08:56\tAddOrReplaceReadGroups\tProcessed    49,000,000 records.  Elapsed time: 00:04:29s.  Time for last 1,000,000:    5s.  Last read position: 13:41,507,852\n",
      "INFO\t2016-11-30 14:09:03\tAddOrReplaceReadGroups\tProcessed    50,000,000 records.  Elapsed time: 00:04:36s.  Time for last 1,000,000:    6s.  Last read position: 9:19,378,382\n",
      "INFO\t2016-11-30 14:09:08\tAddOrReplaceReadGroups\tProcessed    51,000,000 records.  Elapsed time: 00:04:41s.  Time for last 1,000,000:    4s.  Last read position: 5:179,307,801\n",
      "INFO\t2016-11-30 14:09:13\tAddOrReplaceReadGroups\tProcessed    52,000,000 records.  Elapsed time: 00:04:46s.  Time for last 1,000,000:    5s.  Last read position: 6:2,889,213\n",
      "INFO\t2016-11-30 14:09:22\tAddOrReplaceReadGroups\tProcessed    53,000,000 records.  Elapsed time: 00:04:55s.  Time for last 1,000,000:    9s.  Last read position: 13:23,270,055\n",
      "INFO\t2016-11-30 14:09:27\tAddOrReplaceReadGroups\tProcessed    54,000,000 records.  Elapsed time: 00:05:00s.  Time for last 1,000,000:    4s.  Last read position: 7:128,505,768\n",
      "INFO\t2016-11-30 14:09:32\tAddOrReplaceReadGroups\tProcessed    55,000,000 records.  Elapsed time: 00:05:05s.  Time for last 1,000,000:    4s.  Last read position: 3:122,419,620\n",
      "INFO\t2016-11-30 14:09:37\tAddOrReplaceReadGroups\tProcessed    56,000,000 records.  Elapsed time: 00:05:10s.  Time for last 1,000,000:    5s.  Last read position: 10:135,170,223\n",
      "INFO\t2016-11-30 14:09:43\tAddOrReplaceReadGroups\tProcessed    57,000,000 records.  Elapsed time: 00:05:16s.  Time for last 1,000,000:    5s.  Last read position: 5:17,354,155\n",
      "INFO\t2016-11-30 14:09:48\tAddOrReplaceReadGroups\tProcessed    58,000,000 records.  Elapsed time: 00:05:21s.  Time for last 1,000,000:    5s.  Last read position: 6:160,103,126\n",
      "INFO\t2016-11-30 14:09:53\tAddOrReplaceReadGroups\tProcessed    59,000,000 records.  Elapsed time: 00:05:26s.  Time for last 1,000,000:    5s.  Last read position: 1:159,889,545\n",
      "INFO\t2016-11-30 14:09:59\tAddOrReplaceReadGroups\tProcessed    60,000,000 records.  Elapsed time: 00:05:32s.  Time for last 1,000,000:    5s.  Last read position: 6:35,362,911\n",
      "INFO\t2016-11-30 14:10:04\tAddOrReplaceReadGroups\tProcessed    61,000,000 records.  Elapsed time: 00:05:37s.  Time for last 1,000,000:    5s.  Last read position: 19:10,677,520\n",
      "INFO\t2016-11-30 14:10:09\tAddOrReplaceReadGroups\tProcessed    62,000,000 records.  Elapsed time: 00:05:42s.  Time for last 1,000,000:    5s.  Last read position: 12:66,646,370\n",
      "INFO\t2016-11-30 14:10:14\tAddOrReplaceReadGroups\tProcessed    63,000,000 records.  Elapsed time: 00:05:47s.  Time for last 1,000,000:    5s.  Last read position: X:153,577,023\n",
      "INFO\t2016-11-30 14:10:20\tAddOrReplaceReadGroups\tProcessed    64,000,000 records.  Elapsed time: 00:05:53s.  Time for last 1,000,000:    5s.  Last read position: 15:75,761,210\n",
      "INFO\t2016-11-30 14:10:25\tAddOrReplaceReadGroups\tProcessed    65,000,000 records.  Elapsed time: 00:05:58s.  Time for last 1,000,000:    5s.  Last read position: 17:5,111,393\n",
      "INFO\t2016-11-30 14:10:30\tAddOrReplaceReadGroups\tProcessed    66,000,000 records.  Elapsed time: 00:06:03s.  Time for last 1,000,000:    5s.  Last read position: X:119,575,684\n",
      "INFO\t2016-11-30 14:10:35\tAddOrReplaceReadGroups\tProcessed    67,000,000 records.  Elapsed time: 00:06:08s.  Time for last 1,000,000:    5s.  Last read position: X:23,851,536\n",
      "INFO\t2016-11-30 14:10:40\tAddOrReplaceReadGroups\tProcessed    68,000,000 records.  Elapsed time: 00:06:13s.  Time for last 1,000,000:    5s.  Last read position: 3:52,546,921\n",
      "INFO\t2016-11-30 14:10:46\tAddOrReplaceReadGroups\tProcessed    69,000,000 records.  Elapsed time: 00:06:19s.  Time for last 1,000,000:    5s.  Last read position: 17:36,923,178\n",
      "INFO\t2016-11-30 14:10:51\tAddOrReplaceReadGroups\tProcessed    70,000,000 records.  Elapsed time: 00:06:24s.  Time for last 1,000,000:    5s.  Last read position: 17:1,431,893\n",
      "INFO\t2016-11-30 14:10:56\tAddOrReplaceReadGroups\tProcessed    71,000,000 records.  Elapsed time: 00:06:29s.  Time for last 1,000,000:    5s.  Last read position: 4:48,422,302\n",
      "INFO\t2016-11-30 14:11:01\tAddOrReplaceReadGroups\tProcessed    72,000,000 records.  Elapsed time: 00:06:34s.  Time for last 1,000,000:    5s.  Last read position: 20:23,356,334\n",
      "INFO\t2016-11-30 14:11:06\tAddOrReplaceReadGroups\tProcessed    73,000,000 records.  Elapsed time: 00:06:39s.  Time for last 1,000,000:    5s.  Last read position: 1:985,834\n",
      "INFO\t2016-11-30 14:11:11\tAddOrReplaceReadGroups\tProcessed    74,000,000 records.  Elapsed time: 00:06:44s.  Time for last 1,000,000:    5s.  Last read position: 21:46,311,850\n",
      "INFO\t2016-11-30 14:11:16\tAddOrReplaceReadGroups\tProcessed    75,000,000 records.  Elapsed time: 00:06:49s.  Time for last 1,000,000:    5s.  Last read position: 13:23,270,512\n",
      "INFO\t2016-11-30 14:11:22\tAddOrReplaceReadGroups\tProcessed    76,000,000 records.  Elapsed time: 00:06:55s.  Time for last 1,000,000:    5s.  Last read position: 1:205,683,108\n",
      "INFO\t2016-11-30 14:11:26\tAddOrReplaceReadGroups\tProcessed    77,000,000 records.  Elapsed time: 00:07:00s.  Time for last 1,000,000:    4s.  Last read position: 8:82,434,185\n",
      "INFO\t2016-11-30 14:11:32\tAddOrReplaceReadGroups\tProcessed    78,000,000 records.  Elapsed time: 00:07:05s.  Time for last 1,000,000:    5s.  Last read position: 4:176,561,343\n",
      "INFO\t2016-11-30 14:11:37\tAddOrReplaceReadGroups\tProcessed    79,000,000 records.  Elapsed time: 00:07:10s.  Time for last 1,000,000:    4s.  Last read position: 17:35,500,209\n",
      "INFO\t2016-11-30 14:11:42\tAddOrReplaceReadGroups\tProcessed    80,000,000 records.  Elapsed time: 00:07:15s.  Time for last 1,000,000:    4s.  Last read position: 19:41,197,986\n",
      "INFO\t2016-11-30 14:11:47\tAddOrReplaceReadGroups\tProcessed    81,000,000 records.  Elapsed time: 00:07:20s.  Time for last 1,000,000:    5s.  Last read position: 4:178,580,886\n",
      "INFO\t2016-11-30 14:11:52\tAddOrReplaceReadGroups\tProcessed    82,000,000 records.  Elapsed time: 00:07:25s.  Time for last 1,000,000:    4s.  Last read position: 16:18,440,104\n",
      "INFO\t2016-11-30 14:11:57\tAddOrReplaceReadGroups\tProcessed    83,000,000 records.  Elapsed time: 00:07:30s.  Time for last 1,000,000:    4s.  Last read position: 11:1,774,190\n",
      "INFO\t2016-11-30 14:12:02\tAddOrReplaceReadGroups\tProcessed    84,000,000 records.  Elapsed time: 00:07:35s.  Time for last 1,000,000:    5s.  Last read position: 7:32,884,739\n",
      "INFO\t2016-11-30 14:12:07\tAddOrReplaceReadGroups\tProcessed    85,000,000 records.  Elapsed time: 00:07:40s.  Time for last 1,000,000:    4s.  Last read position: 16:18,445,737\n",
      "INFO\t2016-11-30 14:12:12\tAddOrReplaceReadGroups\tProcessed    86,000,000 records.  Elapsed time: 00:07:45s.  Time for last 1,000,000:    4s.  Last read position: 11:10,530,469\n",
      "INFO\t2016-11-30 14:12:18\tAddOrReplaceReadGroups\tProcessed    87,000,000 records.  Elapsed time: 00:07:51s.  Time for last 1,000,000:    6s.  Last read position: 13:100,803,247\n",
      "INFO\t2016-11-30 14:12:23\tAddOrReplaceReadGroups\tProcessed    88,000,000 records.  Elapsed time: 00:07:56s.  Time for last 1,000,000:    4s.  Last read position: 5:71,490,990\n",
      "INFO\t2016-11-30 14:12:28\tAddOrReplaceReadGroups\tProcessed    89,000,000 records.  Elapsed time: 00:08:01s.  Time for last 1,000,000:    4s.  Last read position: 16:5,619,442\n",
      "INFO\t2016-11-30 14:12:32\tAddOrReplaceReadGroups\tProcessed    90,000,000 records.  Elapsed time: 00:08:05s.  Time for last 1,000,000:    4s.  Last read position: 8:104,342,116\n",
      "INFO\t2016-11-30 14:12:37\tAddOrReplaceReadGroups\tProcessed    91,000,000 records.  Elapsed time: 00:08:10s.  Time for last 1,000,000:    4s.  Last read position: MT:12,036\n",
      "INFO\t2016-11-30 14:12:42\tAddOrReplaceReadGroups\tProcessed    92,000,000 records.  Elapsed time: 00:08:15s.  Time for last 1,000,000:    4s.  Last read position: 19:34,884,663\n",
      "INFO\t2016-11-30 14:12:47\tAddOrReplaceReadGroups\tProcessed    93,000,000 records.  Elapsed time: 00:08:20s.  Time for last 1,000,000:    4s.  Last read position: 5:79,946,699\n",
      "INFO\t2016-11-30 14:12:52\tAddOrReplaceReadGroups\tProcessed    94,000,000 records.  Elapsed time: 00:08:25s.  Time for last 1,000,000:    4s.  Last read position: 12:41,366,075\n",
      "INFO\t2016-11-30 14:12:57\tAddOrReplaceReadGroups\tProcessed    95,000,000 records.  Elapsed time: 00:08:30s.  Time for last 1,000,000:    4s.  Last read position: 9:23,690,260\n",
      "INFO\t2016-11-30 14:13:02\tAddOrReplaceReadGroups\tProcessed    96,000,000 records.  Elapsed time: 00:08:35s.  Time for last 1,000,000:    5s.  Last read position: 21:34,921,799\n",
      "INFO\t2016-11-30 14:13:07\tAddOrReplaceReadGroups\tProcessed    97,000,000 records.  Elapsed time: 00:08:40s.  Time for last 1,000,000:    4s.  Last read position: 21:41,239,430\n",
      "INFO\t2016-11-30 14:13:11\tAddOrReplaceReadGroups\tProcessed    98,000,000 records.  Elapsed time: 00:08:45s.  Time for last 1,000,000:    4s.  Last read position: GL000220.1:117,228\n",
      "INFO\t2016-11-30 14:13:17\tAddOrReplaceReadGroups\tProcessed    99,000,000 records.  Elapsed time: 00:08:50s.  Time for last 1,000,000:    5s.  Last read position: 18:23,596,378\n",
      "INFO\t2016-11-30 14:13:22\tAddOrReplaceReadGroups\tProcessed   100,000,000 records.  Elapsed time: 00:08:55s.  Time for last 1,000,000:    4s.  Last read position: 11:10,530,450\n",
      "INFO\t2016-11-30 14:13:27\tAddOrReplaceReadGroups\tProcessed   101,000,000 records.  Elapsed time: 00:09:00s.  Time for last 1,000,000:    4s.  Last read position: X:16,863,047\n",
      "INFO\t2016-11-30 14:13:34\tAddOrReplaceReadGroups\tProcessed   102,000,000 records.  Elapsed time: 00:09:07s.  Time for last 1,000,000:    7s.  Last read position: 1:45,495,943\n",
      "INFO\t2016-11-30 14:13:39\tAddOrReplaceReadGroups\tProcessed   103,000,000 records.  Elapsed time: 00:09:12s.  Time for last 1,000,000:    4s.  Last read position: 10:51,771,751\n",
      "INFO\t2016-11-30 14:13:44\tAddOrReplaceReadGroups\tProcessed   104,000,000 records.  Elapsed time: 00:09:17s.  Time for last 1,000,000:    4s.  Last read position: 8:67,430,239\n",
      "INFO\t2016-11-30 14:13:49\tAddOrReplaceReadGroups\tProcessed   105,000,000 records.  Elapsed time: 00:09:23s.  Time for last 1,000,000:    5s.  Last read position: MT:11,998\n",
      "INFO\t2016-11-30 14:13:54\tAddOrReplaceReadGroups\tProcessed   106,000,000 records.  Elapsed time: 00:09:27s.  Time for last 1,000,000:    4s.  Last read position: X:48,693,844\n",
      "INFO\t2016-11-30 14:13:59\tAddOrReplaceReadGroups\tProcessed   107,000,000 records.  Elapsed time: 00:09:32s.  Time for last 1,000,000:    4s.  Last read position: 18:24,432,817\n",
      "INFO\t2016-11-30 14:14:04\tAddOrReplaceReadGroups\tProcessed   108,000,000 records.  Elapsed time: 00:09:37s.  Time for last 1,000,000:    4s.  Last read position: 12:4,406,620\n",
      "INFO\t2016-11-30 14:14:09\tAddOrReplaceReadGroups\tProcessed   109,000,000 records.  Elapsed time: 00:09:42s.  Time for last 1,000,000:    4s.  Last read position: 17:2,283,868\n",
      "INFO\t2016-11-30 14:14:14\tAddOrReplaceReadGroups\tProcessed   110,000,000 records.  Elapsed time: 00:09:47s.  Time for last 1,000,000:    4s.  Last read position: MT:12,002\n",
      "INFO\t2016-11-30 14:14:18\tAddOrReplaceReadGroups\tProcessed   111,000,000 records.  Elapsed time: 00:09:51s.  Time for last 1,000,000:    4s.  Last read position: 1:569,292\n",
      "INFO\t2016-11-30 14:14:23\tAddOrReplaceReadGroups\tProcessed   112,000,000 records.  Elapsed time: 00:09:56s.  Time for last 1,000,000:    4s.  Last read position: 5:134,260,926\n",
      "INFO\t2016-11-30 14:14:28\tAddOrReplaceReadGroups\tProcessed   113,000,000 records.  Elapsed time: 00:10:01s.  Time for last 1,000,000:    4s.  Last read position: MT:10,290\n",
      "INFO\t2016-11-30 14:14:33\tAddOrReplaceReadGroups\tProcessed   114,000,000 records.  Elapsed time: 00:10:06s.  Time for last 1,000,000:    4s.  Last read position: 2:15,768,575\n",
      "INFO\t2016-11-30 14:14:38\tAddOrReplaceReadGroups\tProcessed   115,000,000 records.  Elapsed time: 00:10:11s.  Time for last 1,000,000:    4s.  Last read position: 4:71,685,705\n",
      "INFO\t2016-11-30 14:14:43\tAddOrReplaceReadGroups\tProcessed   116,000,000 records.  Elapsed time: 00:10:16s.  Time for last 1,000,000:    4s.  Last read position: 22:51,222,409\n",
      "INFO\t2016-11-30 14:14:47\tAddOrReplaceReadGroups\tProcessed   117,000,000 records.  Elapsed time: 00:10:20s.  Time for last 1,000,000:    4s.  Last read position: 16:16,441,283\n",
      "INFO\t2016-11-30 14:14:53\tAddOrReplaceReadGroups\tProcessed   118,000,000 records.  Elapsed time: 00:10:26s.  Time for last 1,000,000:    5s.  Last read position: X:153,031,087\n",
      "INFO\t2016-11-30 14:14:58\tAddOrReplaceReadGroups\tProcessed   119,000,000 records.  Elapsed time: 00:10:31s.  Time for last 1,000,000:    4s.  Last read position: 2:55,200,228\n",
      "INFO\t2016-11-30 14:15:02\tAddOrReplaceReadGroups\tProcessed   120,000,000 records.  Elapsed time: 00:10:36s.  Time for last 1,000,000:    4s.  Last read position: 1:26,802,351\n",
      "INFO\t2016-11-30 14:15:09\tAddOrReplaceReadGroups\tProcessed   121,000,000 records.  Elapsed time: 00:10:43s.  Time for last 1,000,000:    6s.  Last read position: 1:565,730\n",
      "INFO\t2016-11-30 14:15:14\tAddOrReplaceReadGroups\tProcessed   122,000,000 records.  Elapsed time: 00:10:47s.  Time for last 1,000,000:    4s.  Last read position: 12:120,876,220\n",
      "INFO\t2016-11-30 14:15:19\tAddOrReplaceReadGroups\tProcessed   123,000,000 records.  Elapsed time: 00:10:52s.  Time for last 1,000,000:    4s.  Last read position: 19:47,113,569\n",
      "INFO\t2016-11-30 14:15:25\tAddOrReplaceReadGroups\tProcessed   124,000,000 records.  Elapsed time: 00:10:58s.  Time for last 1,000,000:    5s.  Last read position: 12:49,522,439\n",
      "INFO\t2016-11-30 14:15:30\tAddOrReplaceReadGroups\tProcessed   125,000,000 records.  Elapsed time: 00:11:03s.  Time for last 1,000,000:    4s.  Last read position: 18:7,037,859\n",
      "INFO\t2016-11-30 14:15:34\tAddOrReplaceReadGroups\tProcessed   126,000,000 records.  Elapsed time: 00:11:08s.  Time for last 1,000,000:    4s.  Last read position: 12:56,599,358\n",
      "INFO\t2016-11-30 14:15:43\tAddOrReplaceReadGroups\tProcessed   127,000,000 records.  Elapsed time: 00:11:17s.  Time for last 1,000,000:    8s.  Last read position: MT:9,136\n",
      "INFO\t2016-11-30 14:15:48\tAddOrReplaceReadGroups\tProcessed   128,000,000 records.  Elapsed time: 00:11:21s.  Time for last 1,000,000:    4s.  Last read position: 1:40,537,855\n",
      "INFO\t2016-11-30 14:15:53\tAddOrReplaceReadGroups\tProcessed   129,000,000 records.  Elapsed time: 00:11:26s.  Time for last 1,000,000:    4s.  Last read position: MT:5,361\n",
      "INFO\t2016-11-30 14:16:00\tAddOrReplaceReadGroups\tProcessed   130,000,000 records.  Elapsed time: 00:11:33s.  Time for last 1,000,000:    6s.  Last read position: 1:567,278\n",
      "INFO\t2016-11-30 14:16:04\tAddOrReplaceReadGroups\tProcessed   131,000,000 records.  Elapsed time: 00:11:38s.  Time for last 1,000,000:    4s.  Last read position: MT:2,491\n",
      "INFO\t2016-11-30 14:16:09\tAddOrReplaceReadGroups\tProcessed   132,000,000 records.  Elapsed time: 00:11:42s.  Time for last 1,000,000:    4s.  Last read position: MT:2,497\n",
      "INFO\t2016-11-30 14:16:14\tAddOrReplaceReadGroups\tProcessed   133,000,000 records.  Elapsed time: 00:11:47s.  Time for last 1,000,000:    4s.  Last read position: 1:44,134,827\n",
      "INFO\t2016-11-30 14:16:19\tAddOrReplaceReadGroups\tProcessed   134,000,000 records.  Elapsed time: 00:11:52s.  Time for last 1,000,000:    4s.  Last read position: MT:11,724\n",
      "INFO\t2016-11-30 14:16:24\tAddOrReplaceReadGroups\tProcessed   135,000,000 records.  Elapsed time: 00:11:57s.  Time for last 1,000,000:    4s.  Last read position: MT:1,200\n",
      "INFO\t2016-11-30 14:16:29\tAddOrReplaceReadGroups\tProcessed   136,000,000 records.  Elapsed time: 00:12:02s.  Time for last 1,000,000:    4s.  Last read position: 11:61,732,262\n",
      "INFO\t2016-11-30 14:16:34\tAddOrReplaceReadGroups\tProcessed   137,000,000 records.  Elapsed time: 00:12:07s.  Time for last 1,000,000:    4s.  Last read position: MT:7,148\n",
      "INFO\t2016-11-30 14:16:38\tAddOrReplaceReadGroups\tProcessed   138,000,000 records.  Elapsed time: 00:12:11s.  Time for last 1,000,000:    4s.  Last read position: 17:73,841,934\n",
      "INFO\t2016-11-30 14:16:43\tAddOrReplaceReadGroups\tProcessed   139,000,000 records.  Elapsed time: 00:12:16s.  Time for last 1,000,000:    4s.  Last read position: MT:7,301\n",
      "INFO\t2016-11-30 14:16:48\tAddOrReplaceReadGroups\tProcessed   140,000,000 records.  Elapsed time: 00:12:21s.  Time for last 1,000,000:    4s.  Last read position: 4:176,556,174\n",
      "INFO\t2016-11-30 14:16:53\tAddOrReplaceReadGroups\tProcessed   141,000,000 records.  Elapsed time: 00:12:26s.  Time for last 1,000,000:    4s.  Last read position: 5:79,946,699\n",
      "INFO\t2016-11-30 14:16:58\tAddOrReplaceReadGroups\tProcessed   142,000,000 records.  Elapsed time: 00:12:31s.  Time for last 1,000,000:    4s.  Last read position: MT:15,452\n",
      "INFO\t2016-11-30 14:17:03\tAddOrReplaceReadGroups\tProcessed   143,000,000 records.  Elapsed time: 00:12:36s.  Time for last 1,000,000:    4s.  Last read position: MT:8,028\n",
      "INFO\t2016-11-30 14:17:07\tAddOrReplaceReadGroups\tProcessed   144,000,000 records.  Elapsed time: 00:12:40s.  Time for last 1,000,000:    4s.  Last read position: 9:33,289,741\n",
      "INFO\t2016-11-30 14:17:12\tAddOrReplaceReadGroups\tProcessed   145,000,000 records.  Elapsed time: 00:12:45s.  Time for last 1,000,000:    4s.  Last read position: MT:11,281\n",
      "INFO\t2016-11-30 14:17:17\tAddOrReplaceReadGroups\tProcessed   146,000,000 records.  Elapsed time: 00:12:50s.  Time for last 1,000,000:    4s.  Last read position: 15:75,950,220\n",
      "INFO\t2016-11-30 14:17:22\tAddOrReplaceReadGroups\tProcessed   147,000,000 records.  Elapsed time: 00:12:55s.  Time for last 1,000,000:    4s.  Last read position: 1:565,805\n",
      "INFO\t2016-11-30 14:17:27\tAddOrReplaceReadGroups\tProcessed   148,000,000 records.  Elapsed time: 00:13:00s.  Time for last 1,000,000:    5s.  Last read position: 18:54,695,579\n",
      "INFO\t2016-11-30 14:17:32\tAddOrReplaceReadGroups\tProcessed   149,000,000 records.  Elapsed time: 00:13:05s.  Time for last 1,000,000:    4s.  Last read position: MT:11,946\n",
      "INFO\t2016-11-30 14:17:37\tAddOrReplaceReadGroups\tProcessed   150,000,000 records.  Elapsed time: 00:13:10s.  Time for last 1,000,000:    4s.  Last read position: 5:68,660,820\n",
      "INFO\t2016-11-30 14:17:42\tAddOrReplaceReadGroups\tProcessed   151,000,000 records.  Elapsed time: 00:13:15s.  Time for last 1,000,000:    5s.  Last read position: MT:6,416\n",
      "INFO\t2016-11-30 14:17:47\tAddOrReplaceReadGroups\tProcessed   152,000,000 records.  Elapsed time: 00:13:20s.  Time for last 1,000,000:    4s.  Last read position: 3:33,188,224\n",
      "INFO\t2016-11-30 14:17:52\tAddOrReplaceReadGroups\tProcessed   153,000,000 records.  Elapsed time: 00:13:25s.  Time for last 1,000,000:    4s.  Last read position: MT:716\n",
      "INFO\t2016-11-30 14:17:58\tAddOrReplaceReadGroups\tProcessed   154,000,000 records.  Elapsed time: 00:13:31s.  Time for last 1,000,000:    6s.  Last read position: 2:55,199,423\n",
      "INFO\t2016-11-30 14:18:03\tAddOrReplaceReadGroups\tProcessed   155,000,000 records.  Elapsed time: 00:13:36s.  Time for last 1,000,000:    4s.  Last read position: MT:7,983\n",
      "INFO\t2016-11-30 14:18:07\tAddOrReplaceReadGroups\tProcessed   156,000,000 records.  Elapsed time: 00:13:41s.  Time for last 1,000,000:    4s.  Last read position: 1:220,316,418\n",
      "INFO\t2016-11-30 14:18:15\tAddOrReplaceReadGroups\tProcessed   157,000,000 records.  Elapsed time: 00:13:48s.  Time for last 1,000,000:    7s.  Last read position: 14:21,557,104\n",
      "INFO\t2016-11-30 14:18:20\tAddOrReplaceReadGroups\tProcessed   158,000,000 records.  Elapsed time: 00:13:53s.  Time for last 1,000,000:    4s.  Last read position: 2:36,782,040\n",
      "INFO\t2016-11-30 14:18:25\tAddOrReplaceReadGroups\tProcessed   159,000,000 records.  Elapsed time: 00:13:58s.  Time for last 1,000,000:    4s.  Last read position: 16:21,880,685\n",
      "INFO\t2016-11-30 14:18:29\tAddOrReplaceReadGroups\tProcessed   160,000,000 records.  Elapsed time: 00:14:03s.  Time for last 1,000,000:    4s.  Last read position: MT:2,884\n",
      "INFO\t2016-11-30 14:18:34\tAddOrReplaceReadGroups\tProcessed   161,000,000 records.  Elapsed time: 00:14:07s.  Time for last 1,000,000:    4s.  Last read position: 1:226,626,330\n",
      "INFO\t2016-11-30 14:18:39\tAddOrReplaceReadGroups\tProcessed   162,000,000 records.  Elapsed time: 00:14:12s.  Time for last 1,000,000:    4s.  Last read position: 1:567,490\n",
      "INFO\t2016-11-30 14:18:44\tAddOrReplaceReadGroups\tProcessed   163,000,000 records.  Elapsed time: 00:14:17s.  Time for last 1,000,000:    4s.  Last read position: 9:87,430,346\n",
      "INFO\t2016-11-30 14:18:49\tAddOrReplaceReadGroups\tProcessed   164,000,000 records.  Elapsed time: 00:14:22s.  Time for last 1,000,000:    4s.  Last read position: 1:110,566,121\n",
      "INFO\t2016-11-30 14:18:53\tAddOrReplaceReadGroups\tProcessed   165,000,000 records.  Elapsed time: 00:14:27s.  Time for last 1,000,000:    4s.  Last read position: MT:7,421\n",
      "INFO\t2016-11-30 14:18:58\tAddOrReplaceReadGroups\tProcessed   166,000,000 records.  Elapsed time: 00:14:31s.  Time for last 1,000,000:    4s.  Last read position: MT:7,280\n",
      "INFO\t2016-11-30 14:19:03\tAddOrReplaceReadGroups\tProcessed   167,000,000 records.  Elapsed time: 00:14:36s.  Time for last 1,000,000:    4s.  Last read position: MT:7,113\n",
      "INFO\t2016-11-30 14:19:08\tAddOrReplaceReadGroups\tProcessed   168,000,000 records.  Elapsed time: 00:14:41s.  Time for last 1,000,000:    4s.  Last read position: MT:11,121\n",
      "INFO\t2016-11-30 14:19:12\tAddOrReplaceReadGroups\tProcessed   169,000,000 records.  Elapsed time: 00:14:45s.  Time for last 1,000,000:    4s.  Last read position: MT:8,719\n",
      "INFO\t2016-11-30 14:19:18\tAddOrReplaceReadGroups\tProcessed   170,000,000 records.  Elapsed time: 00:14:51s.  Time for last 1,000,000:    5s.  Last read position: 2:96,521,531\n",
      "INFO\t2016-11-30 14:19:22\tAddOrReplaceReadGroups\tProcessed   171,000,000 records.  Elapsed time: 00:14:55s.  Time for last 1,000,000:    4s.  Last read position: 6:9,661,436\n",
      "INFO\t2016-11-30 14:19:27\tAddOrReplaceReadGroups\tProcessed   172,000,000 records.  Elapsed time: 00:15:00s.  Time for last 1,000,000:    4s.  Last read position: 11:61,734,809\n",
      "INFO\t2016-11-30 14:19:33\tAddOrReplaceReadGroups\tProcessed   173,000,000 records.  Elapsed time: 00:15:06s.  Time for last 1,000,000:    5s.  Last read position: MT:2,243\n",
      "INFO\t2016-11-30 14:19:37\tAddOrReplaceReadGroups\tProcessed   174,000,000 records.  Elapsed time: 00:15:10s.  Time for last 1,000,000:    4s.  Last read position: 17:36,686,475\n",
      "INFO\t2016-11-30 14:19:42\tAddOrReplaceReadGroups\tProcessed   175,000,000 records.  Elapsed time: 00:15:15s.  Time for last 1,000,000:    4s.  Last read position: 20:49,457,206\n",
      "INFO\t2016-11-30 14:19:49\tAddOrReplaceReadGroups\tProcessed   176,000,000 records.  Elapsed time: 00:15:22s.  Time for last 1,000,000:    6s.  Last read position: 1:566,577\n",
      "INFO\t2016-11-30 14:19:53\tAddOrReplaceReadGroups\tProcessed   177,000,000 records.  Elapsed time: 00:15:26s.  Time for last 1,000,000:    4s.  Last read position: 5:108,670,629\n",
      "INFO\t2016-11-30 14:19:58\tAddOrReplaceReadGroups\tProcessed   178,000,000 records.  Elapsed time: 00:15:31s.  Time for last 1,000,000:    4s.  Last read position: MT:10,702\n",
      "INFO\t2016-11-30 14:20:05\tAddOrReplaceReadGroups\tProcessed   179,000,000 records.  Elapsed time: 00:15:38s.  Time for last 1,000,000:    6s.  Last read position: MT:959\n",
      "INFO\t2016-11-30 14:20:09\tAddOrReplaceReadGroups\tProcessed   180,000,000 records.  Elapsed time: 00:15:42s.  Time for last 1,000,000:    4s.  Last read position: 3:125,879,798\n",
      "INFO\t2016-11-30 14:20:14\tAddOrReplaceReadGroups\tProcessed   181,000,000 records.  Elapsed time: 00:15:47s.  Time for last 1,000,000:    4s.  Last read position: 11:10,820,859\n",
      "INFO\t2016-11-30 14:20:19\tAddOrReplaceReadGroups\tProcessed   182,000,000 records.  Elapsed time: 00:15:52s.  Time for last 1,000,000:    4s.  Last read position: MT:6,294\n",
      "INFO\t2016-11-30 14:20:23\tAddOrReplaceReadGroups\tProcessed   183,000,000 records.  Elapsed time: 00:15:56s.  Time for last 1,000,000:    4s.  Last read position: MT:1,940\n",
      "INFO\t2016-11-30 14:20:28\tAddOrReplaceReadGroups\tProcessed   184,000,000 records.  Elapsed time: 00:16:01s.  Time for last 1,000,000:    4s.  Last read position: 5:134,259,859\n",
      "[Wed Nov 30 14:33:29 PST 2016] picard.sam.AddOrReplaceReadGroups done. Elapsed time: 29.04 minutes.\n",
      "Runtime.totalMemory()=9510584320\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##Step 5 add readgroups \n",
    "#Add readgroups and markduplicates \n",
    "##77bp read length Next-seq and 50bp is hiseq. For time being everything is set to HISEQ! \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "PICARD=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar\" \n",
    "\n",
    "####\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = PICARD + \" AddOrReplaceReadGroups I=\" + final_dir + \"P\" + str(row['patient']) + \"Aligned.out.sam\" \\\n",
    "    + \" O=\" + step5_dir + \"P\" + str(row['patient']) + \"rg_added_sorted.bam \" + \"SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=sample\"  \n",
    "\n",
    "    print(terminal_code)\n",
    "    #print(\"\\n\")\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P16rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P16dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P16output.metrics\n",
      "[Wed Nov 30 14:37:01 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P16rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P16dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P16output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 14:37:01 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 14:37:01\tMarkDuplicates\tStart of doWork freeMemory: 2042777032; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:37:01\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 14:37:01\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 14:37:06\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:36,755,209\n",
      "INFO\t2016-11-30 14:37:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:09\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:08s.  Time for last 1,000,000:    3s.  Last read position: 1:121,133,284\n",
      "INFO\t2016-11-30 14:37:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:11\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    1s.  Last read position: 1:198,236,098\n",
      "INFO\t2016-11-30 14:37:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:13\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:12s.  Time for last 1,000,000:    2s.  Last read position: 2:43,452,072\n",
      "INFO\t2016-11-30 14:37:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:18\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    4s.  Last read position: 2:152,102,353\n",
      "INFO\t2016-11-30 14:37:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:32\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:30s.  Time for last 1,000,000:   14s.  Last read position: 3:38,056,866\n",
      "INFO\t2016-11-30 14:37:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:34\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    1s.  Last read position: 3:151,056,291\n",
      "INFO\t2016-11-30 14:37:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:45\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:   11s.  Last read position: 4:79,788,693\n",
      "INFO\t2016-11-30 14:37:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:46\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:45s.  Time for last 1,000,000:    1s.  Last read position: 5:67,510,905\n",
      "INFO\t2016-11-30 14:37:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:37:48\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    1s.  Last read position: 5:149,781,347\n",
      "INFO\t2016-11-30 14:37:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:01\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:   12s.  Last read position: 6:31,797,762\n",
      "INFO\t2016-11-30 14:38:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:02\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    1s.  Last read position: 6:151,203,982\n",
      "INFO\t2016-11-30 14:38:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:24\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:01:23s.  Time for last 1,000,000:   21s.  Last read position: 7:73,028,917\n",
      "INFO\t2016-11-30 14:38:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:26\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:24s.  Time for last 1,000,000:    1s.  Last read position: 7:154,796,074\n",
      "INFO\t2016-11-30 14:38:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:27\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:26s.  Time for last 1,000,000:    1s.  Last read position: 8:141,540,483\n",
      "INFO\t2016-11-30 14:38:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:29\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:27s.  Time for last 1,000,000:    1s.  Last read position: 9:128,586,304\n",
      "INFO\t2016-11-30 14:38:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:36\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    7s.  Last read position: 10:46,961,175\n",
      "INFO\t2016-11-30 14:38:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:38\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    1s.  Last read position: 10:126,328,491\n",
      "INFO\t2016-11-30 14:38:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:39\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:38s.  Time for last 1,000,000:    1s.  Last read position: 11:67,052,988\n",
      "INFO\t2016-11-30 14:38:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:41\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    1s.  Last read position: 12:6,728,444\n",
      "INFO\t2016-11-30 14:38:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:55\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:53s.  Time for last 1,000,000:   14s.  Last read position: 12:69,137,106\n",
      "INFO\t2016-11-30 14:38:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:57\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    1s.  Last read position: 13:43,667,035\n",
      "INFO\t2016-11-30 14:38:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:38:58\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:57s.  Time for last 1,000,000:    1s.  Last read position: 14:73,569,653\n",
      "INFO\t2016-11-30 14:38:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:00\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    1s.  Last read position: 15:50,651,891\n",
      "INFO\t2016-11-30 14:39:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:02\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:    1s.  Last read position: 16:11,021,674\n",
      "INFO\t2016-11-30 14:39:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:14\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:02:12s.  Time for last 1,000,000:   12s.  Last read position: 16:31,504,768\n",
      "INFO\t2016-11-30 14:39:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:15\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:02:14s.  Time for last 1,000,000:    1s.  Last read position: 17:7,126,145\n",
      "INFO\t2016-11-30 14:39:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:17\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:02:15s.  Time for last 1,000,000:    1s.  Last read position: 17:48,818,475\n",
      "INFO\t2016-11-30 14:39:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:19\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:02:17s.  Time for last 1,000,000:    1s.  Last read position: 18:48,602,506\n",
      "INFO\t2016-11-30 14:39:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:20\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:19s.  Time for last 1,000,000:    1s.  Last read position: 19:17,675,159\n",
      "INFO\t2016-11-30 14:39:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:40\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:39s.  Time for last 1,000,000:   19s.  Last read position: 19:58,849,270\n",
      "INFO\t2016-11-30 14:39:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:42\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:40s.  Time for last 1,000,000:    1s.  Last read position: 21:44,151,627\n",
      "INFO\t2016-11-30 14:39:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:43\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    1s.  Last read position: X:13,396,899\n",
      "INFO\t2016-11-30 14:39:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:45\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:43s.  Time for last 1,000,000:    1s.  Last read position: MT:3,588\n",
      "INFO\t2016-11-30 14:39:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:39:46\tMarkDuplicates\tRead 34555611 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 14:40:02\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10367749840; totalMemory: 13594787840; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:40:02\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 14:40:02\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:40:02\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:40:06\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 14:40:07\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 6242766672; totalMemory: 13594787840; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:40:07\tMarkDuplicates\tMarking 19117175 records as duplicates.\n",
      "INFO\t2016-11-30 14:40:07\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 14:40:07\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 14:41:04\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:57s.  Time for last 10,000,000:   57s.  Last read position: 5:149,781,347\n",
      "INFO\t2016-11-30 14:41:59\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:52s.  Time for last 10,000,000:   55s.  Last read position: 12:6,728,444\n",
      "INFO\t2016-11-30 14:42:53\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:45s.  Time for last 10,000,000:   53s.  Last read position: 19:17,675,159\n",
      "INFO\t2016-11-30 14:43:18\tMarkDuplicates\tBefore output close freeMemory: 20535132872; totalMemory: 20728774656; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:43:18\tMarkDuplicates\tAfter output close freeMemory: 20147680928; totalMemory: 20341325824; maxMemory: 28631367680\n",
      "[Wed Nov 30 14:43:18 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 6.29 minutes.\n",
      "Runtime.totalMemory()=20341325824\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P19rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P19dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P19output.metrics\n",
      "[Wed Nov 30 14:43:19 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P19rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P19dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P19output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 14:43:19 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 14:43:19\tMarkDuplicates\tStart of doWork freeMemory: 2042777032; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:43:19\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 14:43:19\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 14:43:23\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:33,805,531\n",
      "INFO\t2016-11-30 14:43:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:31\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    7s.  Last read position: 1:120,934,545\n",
      "INFO\t2016-11-30 14:43:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:32\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:13s.  Time for last 1,000,000:    1s.  Last read position: 1:192,545,470\n",
      "INFO\t2016-11-30 14:43:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:35\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    3s.  Last read position: 2:46,818,461\n",
      "INFO\t2016-11-30 14:43:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:40\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:20s.  Time for last 1,000,000:    4s.  Last read position: 2:157,184,322\n",
      "INFO\t2016-11-30 14:43:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:42\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:22s.  Time for last 1,000,000:    1s.  Last read position: 3:41,062,714\n",
      "INFO\t2016-11-30 14:43:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:43:48\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:28s.  Time for last 1,000,000:    5s.  Last read position: 3:169,687,497\n",
      "INFO\t2016-11-30 14:43:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:00\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:   12s.  Last read position: 4:87,967,315\n",
      "INFO\t2016-11-30 14:44:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:01\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    1s.  Last read position: 5:70,406,446\n",
      "INFO\t2016-11-30 14:44:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:03\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:    1s.  Last read position: 5:177,577,630\n",
      "INFO\t2016-11-30 14:44:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:15\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:   12s.  Last read position: 6:32,411,588\n",
      "INFO\t2016-11-30 14:44:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:17\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:57s.  Time for last 1,000,000:    1s.  Last read position: 6:163,827,692\n",
      "INFO\t2016-11-30 14:44:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:19\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:59s.  Time for last 1,000,000:    1s.  Last read position: 7:75,042,357\n",
      "INFO\t2016-11-30 14:44:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:20\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    1s.  Last read position: 8:11,710,854\n",
      "INFO\t2016-11-30 14:44:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:30\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:10s.  Time for last 1,000,000:    9s.  Last read position: 9:6,758,090\n",
      "INFO\t2016-11-30 14:44:30\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:31\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:12s.  Time for last 1,000,000:    1s.  Last read position: 9:136,024,842\n",
      "INFO\t2016-11-30 14:44:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:33\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    1s.  Last read position: 10:64,573,162\n",
      "INFO\t2016-11-30 14:44:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:34\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:15s.  Time for last 1,000,000:    1s.  Last read position: 11:7,836,591\n",
      "INFO\t2016-11-30 14:44:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:54\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:   20s.  Last read position: 11:74,906,614\n",
      "INFO\t2016-11-30 14:44:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:56\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    1s.  Last read position: 12:25,242,899\n",
      "INFO\t2016-11-30 14:44:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:57\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:38s.  Time for last 1,000,000:    1s.  Last read position: 12:110,017,606\n",
      "INFO\t2016-11-30 14:44:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:44:59\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:39s.  Time for last 1,000,000:    1s.  Last read position: 13:92,487,823\n",
      "INFO\t2016-11-30 14:44:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:04\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    5s.  Last read position: 14:100,607,491\n",
      "INFO\t2016-11-30 14:45:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:05\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:46s.  Time for last 1,000,000:    1s.  Last read position: 15:81,595,591\n",
      "INFO\t2016-11-30 14:45:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:07\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    1s.  Last read position: 16:22,535,252\n",
      "INFO\t2016-11-30 14:45:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:08\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:49s.  Time for last 1,000,000:    1s.  Last read position: 16:85,951,122\n",
      "INFO\t2016-11-30 14:45:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:10\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    1s.  Last read position: 17:35,838,255\n",
      "INFO\t2016-11-30 14:45:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:11\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:01:52s.  Time for last 1,000,000:    1s.  Last read position: 17:79,782,483\n",
      "INFO\t2016-11-30 14:45:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:31\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:02:11s.  Time for last 1,000,000:   19s.  Last read position: 19:6,714,063\n",
      "INFO\t2016-11-30 14:45:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:32\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:13s.  Time for last 1,000,000:    1s.  Last read position: 19:44,159,161\n",
      "INFO\t2016-11-30 14:45:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:34\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:14s.  Time for last 1,000,000:    1s.  Last read position: 20:43,135,586\n",
      "INFO\t2016-11-30 14:45:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:35\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    1s.  Last read position: 22:33,659,776\n",
      "INFO\t2016-11-30 14:45:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:37\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:17s.  Time for last 1,000,000:    1s.  Last read position: X:84,576,053\n",
      "INFO\t2016-11-30 14:45:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:38\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:19s.  Time for last 1,000,000:    1s.  Last read position: GL000220.1:160,214\n",
      "INFO\t2016-11-30 14:45:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:45:38\tMarkDuplicates\tRead 34058495 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 14:45:58\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10162712904; totalMemory: 13341556736; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:45:58\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 14:45:59\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:45:59\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:46:02\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 14:46:03\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 5990803576; totalMemory: 13341556736; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:46:03\tMarkDuplicates\tMarking 15490867 records as duplicates.\n",
      "INFO\t2016-11-30 14:46:03\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 14:46:03\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 14:47:02\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:59s.  Time for last 10,000,000:   59s.  Last read position: 5:177,577,630\n",
      "INFO\t2016-11-30 14:47:59\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:56s.  Time for last 10,000,000:   56s.  Last read position: 12:25,242,899\n",
      "INFO\t2016-11-30 14:48:57\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:54s.  Time for last 10,000,000:   57s.  Last read position: 19:44,159,161\n",
      "INFO\t2016-11-30 14:49:21\tMarkDuplicates\tBefore output close freeMemory: 19874158936; totalMemory: 20066074624; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:49:21\tMarkDuplicates\tAfter output close freeMemory: 20285559744; totalMemory: 20477640704; maxMemory: 28631367680\n",
      "[Wed Nov 30 14:49:21 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 6.04 minutes.\n",
      "Runtime.totalMemory()=20477640704\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P21rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P21dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P21output.metrics\n",
      "[Wed Nov 30 14:49:22 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P21rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P21dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P21output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 14:49:22 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 14:49:22\tMarkDuplicates\tStart of doWork freeMemory: 2042777032; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:49:22\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 14:49:22\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 14:49:27\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:31,211,857\n",
      "INFO\t2016-11-30 14:49:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:31\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:08s.  Time for last 1,000,000:    3s.  Last read position: 1:111,678,714\n",
      "INFO\t2016-11-30 14:49:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:32\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    1s.  Last read position: 1:161,928,804\n",
      "INFO\t2016-11-30 14:49:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:38\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 1:245,025,717\n",
      "INFO\t2016-11-30 14:49:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:42\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    4s.  Last read position: 2:106,497,939\n",
      "INFO\t2016-11-30 14:49:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:45\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:22s.  Time for last 1,000,000:    2s.  Last read position: 2:225,407,075\n",
      "INFO\t2016-11-30 14:49:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:46\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    1s.  Last read position: 3:96,336,237\n",
      "INFO\t2016-11-30 14:49:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:57\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:   11s.  Last read position: 3:197,784,663\n",
      "INFO\t2016-11-30 14:49:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:49:59\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    1s.  Last read position: 4:119,400,075\n",
      "INFO\t2016-11-30 14:49:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:00\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:38s.  Time for last 1,000,000:    1s.  Last read position: 5:69,725,045\n",
      "INFO\t2016-11-30 14:50:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:23\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:01:00s.  Time for last 1,000,000:   22s.  Last read position: 5:149,786,871\n",
      "INFO\t2016-11-30 14:50:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:24\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:01:02s.  Time for last 1,000,000:    1s.  Last read position: 6:31,785,047\n",
      "INFO\t2016-11-30 14:50:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:26\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    1s.  Last read position: 6:109,826,620\n",
      "INFO\t2016-11-30 14:50:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:39\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:17s.  Time for last 1,000,000:   13s.  Last read position: 7:26,235,469\n",
      "INFO\t2016-11-30 14:50:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:41\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:    1s.  Last read position: 7:102,227,380\n",
      "INFO\t2016-11-30 14:50:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:42\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:20s.  Time for last 1,000,000:    1s.  Last read position: 8:37,699,003\n",
      "INFO\t2016-11-30 14:50:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:44\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    1s.  Last read position: 9:35,068,609\n",
      "INFO\t2016-11-30 14:50:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:54\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:31s.  Time for last 1,000,000:    9s.  Last read position: 9:135,895,226\n",
      "INFO\t2016-11-30 14:50:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:55\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:33s.  Time for last 1,000,000:    1s.  Last read position: 10:47,133,786\n",
      "INFO\t2016-11-30 14:50:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:57\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:34s.  Time for last 1,000,000:    1s.  Last read position: 10:121,427,940\n",
      "INFO\t2016-11-30 14:50:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:50:58\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:36s.  Time for last 1,000,000:    1s.  Last read position: 11:65,243,173\n",
      "INFO\t2016-11-30 14:50:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:19\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:56s.  Time for last 1,000,000:   20s.  Last read position: 12:86,326\n",
      "INFO\t2016-11-30 14:51:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:20\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    1s.  Last read position: 12:57,557,691\n",
      "INFO\t2016-11-30 14:51:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:21\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:59s.  Time for last 1,000,000:    1s.  Last read position: 12:133,397,908\n",
      "INFO\t2016-11-30 14:51:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:23\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    1s.  Last read position: 14:45,716,175\n",
      "INFO\t2016-11-30 14:51:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:25\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:02:02s.  Time for last 1,000,000:    1s.  Last read position: 15:30,383,081\n",
      "INFO\t2016-11-30 14:51:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:37\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:02:15s.  Time for last 1,000,000:   12s.  Last read position: 15:101,603,358\n",
      "INFO\t2016-11-30 14:51:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:38\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:02:16s.  Time for last 1,000,000:    1s.  Last read position: 16:29,526,721\n",
      "INFO\t2016-11-30 14:51:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:40\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:02:18s.  Time for last 1,000,000:    1s.  Last read position: 17:951,350\n",
      "INFO\t2016-11-30 14:51:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:41\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:19s.  Time for last 1,000,000:    1s.  Last read position: 17:41,154,726\n",
      "INFO\t2016-11-30 14:51:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:43\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:21s.  Time for last 1,000,000:    1s.  Last read position: 18:9,736,422\n",
      "INFO\t2016-11-30 14:51:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:44\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:22s.  Time for last 1,000,000:    1s.  Last read position: 19:8,590,400\n",
      "INFO\t2016-11-30 14:51:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:50\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:27s.  Time for last 1,000,000:    5s.  Last read position: 19:45,977,486\n",
      "INFO\t2016-11-30 14:51:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:51\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:29s.  Time for last 1,000,000:    1s.  Last read position: 20:46,208,660\n",
      "INFO\t2016-11-30 14:51:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:52\tMarkDuplicates\tRead    35,000,000 records.  Elapsed time: 00:02:30s.  Time for last 1,000,000:    1s.  Last read position: 22:32,185,255\n",
      "INFO\t2016-11-30 14:51:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:54\tMarkDuplicates\tRead    36,000,000 records.  Elapsed time: 00:02:32s.  Time for last 1,000,000:    1s.  Last read position: X:71,563,515\n",
      "INFO\t2016-11-30 14:51:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:55\tMarkDuplicates\tRead    37,000,000 records.  Elapsed time: 00:02:33s.  Time for last 1,000,000:    1s.  Last read position: GL000220.1:106,992\n",
      "INFO\t2016-11-30 14:51:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:51:56\tMarkDuplicates\tRead 37368051 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 14:52:10\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10005633368; totalMemory: 13360955392; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:52:10\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 14:52:10\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:52:10\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 14:52:13\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 14:52:14\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 6009546168; totalMemory: 13360955392; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:52:14\tMarkDuplicates\tMarking 16834565 records as duplicates.\n",
      "INFO\t2016-11-30 14:52:14\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 14:52:14\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 14:53:13\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:58s.  Time for last 10,000,000:   58s.  Last read position: 5:69,725,045\n",
      "INFO\t2016-11-30 14:54:12\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:57s.  Time for last 10,000,000:   59s.  Last read position: 10:121,427,940\n",
      "INFO\t2016-11-30 14:55:11\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:56s.  Time for last 10,000,000:   58s.  Last read position: 17:41,154,726\n",
      "INFO\t2016-11-30 14:55:54\tMarkDuplicates\tBefore output close freeMemory: 20499150000; totalMemory: 20691550208; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:55:54\tMarkDuplicates\tAfter output close freeMemory: 20077619336; totalMemory: 20270022656; maxMemory: 28631367680\n",
      "[Wed Nov 30 14:55:54 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 6.54 minutes.\n",
      "Runtime.totalMemory()=20270022656\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P10output.metrics\n",
      "[Wed Nov 30 14:55:55 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P10output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 14:55:55 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 14:55:55\tMarkDuplicates\tStart of doWork freeMemory: 2042777032; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 14:55:55\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 14:55:55\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 14:55:59\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:03s.  Time for last 1,000,000:    3s.  Last read position: 1:9,638,447\n",
      "INFO\t2016-11-30 14:55:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:04\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    5s.  Last read position: 1:23,103,433\n",
      "INFO\t2016-11-30 14:56:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:06\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    1s.  Last read position: 1:36,932,400\n",
      "INFO\t2016-11-30 14:56:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:11\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 1:53,147,827\n",
      "INFO\t2016-11-30 14:56:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:17\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:22s.  Time for last 1,000,000:    6s.  Last read position: 1:104,076,387\n",
      "INFO\t2016-11-30 14:56:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:23\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:27s.  Time for last 1,000,000:    5s.  Last read position: 1:121,121,490\n",
      "INFO\t2016-11-30 14:56:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:24\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:29s.  Time for last 1,000,000:    1s.  Last read position: 1:149,754,961\n",
      "INFO\t2016-11-30 14:56:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:29\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    5s.  Last read position: 1:161,483,955\n",
      "INFO\t2016-11-30 14:56:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:31\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    1s.  Last read position: 1:193,063,734\n",
      "INFO\t2016-11-30 14:56:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:32\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:37s.  Time for last 1,000,000:    1s.  Last read position: 1:212,867,462\n",
      "INFO\t2016-11-30 14:56:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:49\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:   16s.  Last read position: 1:247,353,214\n",
      "INFO\t2016-11-30 14:56:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:50\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:55s.  Time for last 1,000,000:    1s.  Last read position: 2:43,777,400\n",
      "INFO\t2016-11-30 14:56:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:56:52\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    1s.  Last read position: 2:75,102,076\n",
      "INFO\t2016-11-30 14:56:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:05\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:10s.  Time for last 1,000,000:   13s.  Last read position: 2:112,559,696\n",
      "INFO\t2016-11-30 14:57:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:07\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:12s.  Time for last 1,000,000:    1s.  Last read position: 2:153,300,275\n",
      "INFO\t2016-11-30 14:57:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:08\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:13s.  Time for last 1,000,000:    1s.  Last read position: 2:203,576,479\n",
      "INFO\t2016-11-30 14:57:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:10\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:14s.  Time for last 1,000,000:    1s.  Last read position: 2:242,203,961\n",
      "INFO\t2016-11-30 14:57:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:26\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:   15s.  Last read position: 3:40,803,108\n",
      "INFO\t2016-11-30 14:57:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:27\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    1s.  Last read position: 3:57,313,999\n",
      "INFO\t2016-11-30 14:57:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:29\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:33s.  Time for last 1,000,000:    1s.  Last read position: 3:128,838,067\n",
      "INFO\t2016-11-30 14:57:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:30\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    1s.  Last read position: 3:179,116,354\n",
      "INFO\t2016-11-30 14:57:30\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:38\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:42s.  Time for last 1,000,000:    7s.  Last read position: 3:197,946,855\n",
      "INFO\t2016-11-30 14:57:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:39\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    1s.  Last read position: 4:38,756,580\n",
      "INFO\t2016-11-30 14:57:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:41\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    1s.  Last read position: 4:88,897,955\n",
      "INFO\t2016-11-30 14:57:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:42\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    1s.  Last read position: 4:144,469,088\n",
      "INFO\t2016-11-30 14:57:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:44\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:49s.  Time for last 1,000,000:    1s.  Last read position: 5:17,270,237\n",
      "INFO\t2016-11-30 14:57:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:46\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:50s.  Time for last 1,000,000:    1s.  Last read position: 5:64,873,820\n",
      "INFO\t2016-11-30 14:57:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:53\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:01:58s.  Time for last 1,000,000:    7s.  Last read position: 5:90,667,475\n",
      "INFO\t2016-11-30 14:57:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:55\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:    1s.  Last read position: 5:140,011,632\n",
      "INFO\t2016-11-30 14:57:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:56\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:01s.  Time for last 1,000,000:    1s.  Last read position: 5:157,214,336\n",
      "INFO\t2016-11-30 14:57:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:58\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    1s.  Last read position: 6:2,887,768\n",
      "INFO\t2016-11-30 14:57:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:57:59\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:04s.  Time for last 1,000,000:    1s.  Last read position: 6:31,584,205\n",
      "INFO\t2016-11-30 14:57:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:01\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    1s.  Last read position: 6:32,973,462\n",
      "INFO\t2016-11-30 14:58:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:22\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:26s.  Time for last 1,000,000:   20s.  Last read position: 6:42,923,454\n",
      "INFO\t2016-11-30 14:58:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:23\tMarkDuplicates\tRead    35,000,000 records.  Elapsed time: 00:02:28s.  Time for last 1,000,000:    1s.  Last read position: 6:108,881,209\n",
      "INFO\t2016-11-30 14:58:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:25\tMarkDuplicates\tRead    36,000,000 records.  Elapsed time: 00:02:29s.  Time for last 1,000,000:    1s.  Last read position: 6:144,169,988\n",
      "INFO\t2016-11-30 14:58:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:26\tMarkDuplicates\tRead    37,000,000 records.  Elapsed time: 00:02:31s.  Time for last 1,000,000:    1s.  Last read position: 7:2,685,261\n",
      "INFO\t2016-11-30 14:58:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:28\tMarkDuplicates\tRead    38,000,000 records.  Elapsed time: 00:02:32s.  Time for last 1,000,000:    1s.  Last read position: 7:26,240,234\n",
      "INFO\t2016-11-30 14:58:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:29\tMarkDuplicates\tRead    39,000,000 records.  Elapsed time: 00:02:34s.  Time for last 1,000,000:    1s.  Last read position: 7:66,009,371\n",
      "INFO\t2016-11-30 14:58:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:50\tMarkDuplicates\tRead    40,000,000 records.  Elapsed time: 00:02:55s.  Time for last 1,000,000:   21s.  Last read position: 7:95,217,600\n",
      "INFO\t2016-11-30 14:58:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:52\tMarkDuplicates\tRead    41,000,000 records.  Elapsed time: 00:02:56s.  Time for last 1,000,000:    1s.  Last read position: 7:114,294,021\n",
      "INFO\t2016-11-30 14:58:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:53\tMarkDuplicates\tRead    42,000,000 records.  Elapsed time: 00:02:58s.  Time for last 1,000,000:    1s.  Last read position: 7:150,069,288\n",
      "INFO\t2016-11-30 14:58:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:55\tMarkDuplicates\tRead    43,000,000 records.  Elapsed time: 00:02:59s.  Time for last 1,000,000:    1s.  Last read position: 8:22,876,584\n",
      "INFO\t2016-11-30 14:58:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:56\tMarkDuplicates\tRead    44,000,000 records.  Elapsed time: 00:03:01s.  Time for last 1,000,000:    1s.  Last read position: 8:67,545,967\n",
      "INFO\t2016-11-30 14:58:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:58:58\tMarkDuplicates\tRead    45,000,000 records.  Elapsed time: 00:03:03s.  Time for last 1,000,000:    1s.  Last read position: 8:134,059,308\n",
      "INFO\t2016-11-30 14:58:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:15\tMarkDuplicates\tRead    46,000,000 records.  Elapsed time: 00:03:19s.  Time for last 1,000,000:   16s.  Last read position: 9:21,512,314\n",
      "INFO\t2016-11-30 14:59:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:16\tMarkDuplicates\tRead    47,000,000 records.  Elapsed time: 00:03:21s.  Time for last 1,000,000:    1s.  Last read position: 9:71,590,443\n",
      "INFO\t2016-11-30 14:59:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:18\tMarkDuplicates\tRead    48,000,000 records.  Elapsed time: 00:03:22s.  Time for last 1,000,000:    1s.  Last read position: 9:120,481,070\n",
      "INFO\t2016-11-30 14:59:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:19\tMarkDuplicates\tRead    49,000,000 records.  Elapsed time: 00:03:24s.  Time for last 1,000,000:    1s.  Last read position: 9:135,895,890\n",
      "INFO\t2016-11-30 14:59:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:21\tMarkDuplicates\tRead    50,000,000 records.  Elapsed time: 00:03:25s.  Time for last 1,000,000:    1s.  Last read position: 10:7,844,324\n",
      "INFO\t2016-11-30 14:59:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:22\tMarkDuplicates\tRead    51,000,000 records.  Elapsed time: 00:03:27s.  Time for last 1,000,000:    1s.  Last read position: 10:22,618,812\n",
      "INFO\t2016-11-30 14:59:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:24\tMarkDuplicates\tRead    52,000,000 records.  Elapsed time: 00:03:28s.  Time for last 1,000,000:    1s.  Last read position: 10:50,174,630\n",
      "INFO\t2016-11-30 14:59:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:46\tMarkDuplicates\tRead    53,000,000 records.  Elapsed time: 00:03:51s.  Time for last 1,000,000:   22s.  Last read position: 10:75,464,701\n",
      "INFO\t2016-11-30 14:59:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:48\tMarkDuplicates\tRead    54,000,000 records.  Elapsed time: 00:03:53s.  Time for last 1,000,000:    1s.  Last read position: 10:102,258,934\n",
      "INFO\t2016-11-30 14:59:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:49\tMarkDuplicates\tRead    55,000,000 records.  Elapsed time: 00:03:54s.  Time for last 1,000,000:    1s.  Last read position: 11:499,887\n",
      "INFO\t2016-11-30 14:59:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:51\tMarkDuplicates\tRead    56,000,000 records.  Elapsed time: 00:03:56s.  Time for last 1,000,000:    1s.  Last read position: 11:33,376,486\n",
      "INFO\t2016-11-30 14:59:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:53\tMarkDuplicates\tRead    57,000,000 records.  Elapsed time: 00:03:57s.  Time for last 1,000,000:    1s.  Last read position: 11:64,538,708\n",
      "INFO\t2016-11-30 14:59:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 14:59:54\tMarkDuplicates\tRead    58,000,000 records.  Elapsed time: 00:03:59s.  Time for last 1,000,000:    1s.  Last read position: 11:71,727,074\n",
      "INFO\t2016-11-30 14:59:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:15\tMarkDuplicates\tRead    59,000,000 records.  Elapsed time: 00:04:19s.  Time for last 1,000,000:   20s.  Last read position: 11:102,268,515\n",
      "INFO\t2016-11-30 15:00:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:16\tMarkDuplicates\tRead    60,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    1s.  Last read position: 12:80,753\n",
      "INFO\t2016-11-30 15:00:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:18\tMarkDuplicates\tRead    61,000,000 records.  Elapsed time: 00:04:23s.  Time for last 1,000,000:    1s.  Last read position: 12:15,095,427\n",
      "INFO\t2016-11-30 15:00:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:19\tMarkDuplicates\tRead    62,000,000 records.  Elapsed time: 00:04:24s.  Time for last 1,000,000:    1s.  Last read position: 12:50,095,718\n",
      "INFO\t2016-11-30 15:00:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:21\tMarkDuplicates\tRead    63,000,000 records.  Elapsed time: 00:04:26s.  Time for last 1,000,000:    1s.  Last read position: 12:66,542,138\n",
      "INFO\t2016-11-30 15:00:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:22\tMarkDuplicates\tRead    64,000,000 records.  Elapsed time: 00:04:27s.  Time for last 1,000,000:    1s.  Last read position: 12:108,988,143\n",
      "INFO\t2016-11-30 15:00:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:34\tMarkDuplicates\tRead    65,000,000 records.  Elapsed time: 00:04:39s.  Time for last 1,000,000:   11s.  Last read position: 12:125,267,897\n",
      "INFO\t2016-11-30 15:00:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:36\tMarkDuplicates\tRead    66,000,000 records.  Elapsed time: 00:04:41s.  Time for last 1,000,000:    1s.  Last read position: 13:42,877,252\n",
      "INFO\t2016-11-30 15:00:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:37\tMarkDuplicates\tRead    67,000,000 records.  Elapsed time: 00:04:42s.  Time for last 1,000,000:    1s.  Last read position: 13:100,026,695\n",
      "INFO\t2016-11-30 15:00:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:39\tMarkDuplicates\tRead    68,000,000 records.  Elapsed time: 00:04:44s.  Time for last 1,000,000:    1s.  Last read position: 14:24,903,066\n",
      "INFO\t2016-11-30 15:00:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:41\tMarkDuplicates\tRead    69,000,000 records.  Elapsed time: 00:04:45s.  Time for last 1,000,000:    1s.  Last read position: 14:75,190,712\n",
      "INFO\t2016-11-30 15:00:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:00:42\tMarkDuplicates\tRead    70,000,000 records.  Elapsed time: 00:04:47s.  Time for last 1,000,000:    1s.  Last read position: 14:100,604,464\n",
      "INFO\t2016-11-30 15:00:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:06\tMarkDuplicates\tRead    71,000,000 records.  Elapsed time: 00:05:10s.  Time for last 1,000,000:   23s.  Last read position: 15:31,666,246\n",
      "INFO\t2016-11-30 15:01:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:07\tMarkDuplicates\tRead    72,000,000 records.  Elapsed time: 00:05:12s.  Time for last 1,000,000:    1s.  Last read position: 15:59,495,303\n",
      "INFO\t2016-11-30 15:01:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:09\tMarkDuplicates\tRead    73,000,000 records.  Elapsed time: 00:05:14s.  Time for last 1,000,000:    1s.  Last read position: 15:83,054,555\n",
      "INFO\t2016-11-30 15:01:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:10\tMarkDuplicates\tRead    74,000,000 records.  Elapsed time: 00:05:15s.  Time for last 1,000,000:    1s.  Last read position: 16:2,650,200\n",
      "INFO\t2016-11-30 15:01:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:12\tMarkDuplicates\tRead    75,000,000 records.  Elapsed time: 00:05:16s.  Time for last 1,000,000:    1s.  Last read position: 16:18,525,685\n",
      "INFO\t2016-11-30 15:01:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:13\tMarkDuplicates\tRead    76,000,000 records.  Elapsed time: 00:05:18s.  Time for last 1,000,000:    1s.  Last read position: 16:24,227,411\n",
      "INFO\t2016-11-30 15:01:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:14\tMarkDuplicates\tRead    77,000,000 records.  Elapsed time: 00:05:19s.  Time for last 1,000,000:    1s.  Last read position: 16:30,292,712\n",
      "INFO\t2016-11-30 15:01:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:35\tMarkDuplicates\tRead    78,000,000 records.  Elapsed time: 00:05:40s.  Time for last 1,000,000:   20s.  Last read position: 16:67,264,865\n",
      "INFO\t2016-11-30 15:01:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:36\tMarkDuplicates\tRead    79,000,000 records.  Elapsed time: 00:05:41s.  Time for last 1,000,000:    1s.  Last read position: 16:88,784,270\n",
      "INFO\t2016-11-30 15:01:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:38\tMarkDuplicates\tRead    80,000,000 records.  Elapsed time: 00:05:43s.  Time for last 1,000,000:    1s.  Last read position: 17:4,721,854\n",
      "INFO\t2016-11-30 15:01:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:39\tMarkDuplicates\tRead    81,000,000 records.  Elapsed time: 00:05:44s.  Time for last 1,000,000:    1s.  Last read position: 17:20,315,825\n",
      "INFO\t2016-11-30 15:01:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:41\tMarkDuplicates\tRead    82,000,000 records.  Elapsed time: 00:05:46s.  Time for last 1,000,000:    1s.  Last read position: 17:36,884,949\n",
      "INFO\t2016-11-30 15:01:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:42\tMarkDuplicates\tRead    83,000,000 records.  Elapsed time: 00:05:47s.  Time for last 1,000,000:    1s.  Last read position: 17:56,076,005\n",
      "INFO\t2016-11-30 15:01:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:49\tMarkDuplicates\tRead    84,000,000 records.  Elapsed time: 00:05:54s.  Time for last 1,000,000:    6s.  Last read position: 17:74,472,771\n",
      "INFO\t2016-11-30 15:01:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:51\tMarkDuplicates\tRead    85,000,000 records.  Elapsed time: 00:05:55s.  Time for last 1,000,000:    1s.  Last read position: 18:7,119,505\n",
      "INFO\t2016-11-30 15:01:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:52\tMarkDuplicates\tRead    86,000,000 records.  Elapsed time: 00:05:57s.  Time for last 1,000,000:    1s.  Last read position: 18:67,808,631\n",
      "INFO\t2016-11-30 15:01:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:54\tMarkDuplicates\tRead    87,000,000 records.  Elapsed time: 00:05:58s.  Time for last 1,000,000:    1s.  Last read position: 19:5,106,969\n",
      "INFO\t2016-11-30 15:01:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:55\tMarkDuplicates\tRead    88,000,000 records.  Elapsed time: 00:06:00s.  Time for last 1,000,000:    1s.  Last read position: 19:10,906,663\n",
      "INFO\t2016-11-30 15:01:55\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:01:57\tMarkDuplicates\tRead    89,000,000 records.  Elapsed time: 00:06:01s.  Time for last 1,000,000:    1s.  Last read position: 19:18,970,507\n",
      "INFO\t2016-11-30 15:01:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:10\tMarkDuplicates\tRead    90,000,000 records.  Elapsed time: 00:06:15s.  Time for last 1,000,000:   13s.  Last read position: 19:41,838,110\n",
      "INFO\t2016-11-30 15:02:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:12\tMarkDuplicates\tRead    91,000,000 records.  Elapsed time: 00:06:16s.  Time for last 1,000,000:    1s.  Last read position: 19:51,919,281\n",
      "INFO\t2016-11-30 15:02:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:13\tMarkDuplicates\tRead    92,000,000 records.  Elapsed time: 00:06:18s.  Time for last 1,000,000:    1s.  Last read position: 20:5,690,004\n",
      "INFO\t2016-11-30 15:02:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:15\tMarkDuplicates\tRead    93,000,000 records.  Elapsed time: 00:06:19s.  Time for last 1,000,000:    1s.  Last read position: 20:42,088,866\n",
      "INFO\t2016-11-30 15:02:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:16\tMarkDuplicates\tRead    94,000,000 records.  Elapsed time: 00:06:21s.  Time for last 1,000,000:    1s.  Last read position: 21:15,858,348\n",
      "INFO\t2016-11-30 15:02:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:18\tMarkDuplicates\tRead    95,000,000 records.  Elapsed time: 00:06:23s.  Time for last 1,000,000:    1s.  Last read position: 21:46,929,653\n",
      "INFO\t2016-11-30 15:02:18\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:32\tMarkDuplicates\tRead    96,000,000 records.  Elapsed time: 00:06:37s.  Time for last 1,000,000:   13s.  Last read position: 22:32,079,256\n",
      "INFO\t2016-11-30 15:02:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:33\tMarkDuplicates\tRead    97,000,000 records.  Elapsed time: 00:06:38s.  Time for last 1,000,000:    1s.  Last read position: 22:47,563,146\n",
      "INFO\t2016-11-30 15:02:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:35\tMarkDuplicates\tRead    98,000,000 records.  Elapsed time: 00:06:39s.  Time for last 1,000,000:    1s.  Last read position: X:34,888,392\n",
      "INFO\t2016-11-30 15:02:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:36\tMarkDuplicates\tRead    99,000,000 records.  Elapsed time: 00:06:41s.  Time for last 1,000,000:    1s.  Last read position: X:73,062,855\n",
      "INFO\t2016-11-30 15:02:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:38\tMarkDuplicates\tRead   100,000,000 records.  Elapsed time: 00:06:43s.  Time for last 1,000,000:    1s.  Last read position: X:135,943,890\n",
      "INFO\t2016-11-30 15:02:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:02:39\tMarkDuplicates\tRead   101,000,000 records.  Elapsed time: 00:06:44s.  Time for last 1,000,000:    1s.  Last read position: MT:9,634\n",
      "INFO\t2016-11-30 15:02:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:03:21\tMarkDuplicates\tRead   102,000,000 records.  Elapsed time: 00:07:26s.  Time for last 1,000,000:   41s.  Last read position: GL000220.1:118,036\n",
      "INFO\t2016-11-30 15:03:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:03:22\tMarkDuplicates\tRead 102634504 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 15:03:40\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 11598163784; totalMemory: 19080413184; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:03:40\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 15:03:40\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:03:40\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:03:50\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 15:03:55\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 18263245984; totalMemory: 25609371648; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:03:55\tMarkDuplicates\tMarking 61264506 records as duplicates.\n",
      "INFO\t2016-11-30 15:03:55\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 15:03:55\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 15:04:50\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:55s.  Time for last 10,000,000:   55s.  Last read position: 1:212,867,462\n",
      "INFO\t2016-11-30 15:05:47\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:52s.  Time for last 10,000,000:   57s.  Last read position: 3:128,838,067\n",
      "INFO\t2016-11-30 15:06:41\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:46s.  Time for last 10,000,000:   53s.  Last read position: 5:157,214,336\n",
      "INFO\t2016-11-30 15:07:34\tMarkDuplicates\tWritten    40,000,000 records.  Elapsed time: 00:03:39s.  Time for last 10,000,000:   52s.  Last read position: 7:95,217,600\n",
      "INFO\t2016-11-30 15:08:27\tMarkDuplicates\tWritten    50,000,000 records.  Elapsed time: 00:04:32s.  Time for last 10,000,000:   53s.  Last read position: 10:7,844,324\n",
      "INFO\t2016-11-30 15:09:18\tMarkDuplicates\tWritten    60,000,000 records.  Elapsed time: 00:05:23s.  Time for last 10,000,000:   51s.  Last read position: 12:80,753\n",
      "INFO\t2016-11-30 15:10:10\tMarkDuplicates\tWritten    70,000,000 records.  Elapsed time: 00:06:15s.  Time for last 10,000,000:   52s.  Last read position: 14:100,604,464\n",
      "INFO\t2016-11-30 15:11:01\tMarkDuplicates\tWritten    80,000,000 records.  Elapsed time: 00:07:06s.  Time for last 10,000,000:   50s.  Last read position: 17:4,721,854\n",
      "INFO\t2016-11-30 15:11:52\tMarkDuplicates\tWritten    90,000,000 records.  Elapsed time: 00:07:57s.  Time for last 10,000,000:   51s.  Last read position: 19:41,838,110\n",
      "INFO\t2016-11-30 15:12:46\tMarkDuplicates\tWritten   100,000,000 records.  Elapsed time: 00:08:51s.  Time for last 10,000,000:   54s.  Last read position: X:135,943,890\n",
      "INFO\t2016-11-30 15:12:59\tMarkDuplicates\tBefore output close freeMemory: 25863645160; totalMemory: 26062880768; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:12:59\tMarkDuplicates\tAfter output close freeMemory: 26181884864; totalMemory: 26381123584; maxMemory: 28631367680\n",
      "[Wed Nov 30 15:12:59 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 17.07 minutes.\n",
      "Runtime.totalMemory()=26381123584\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P2rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P2dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P2output.metrics\n",
      "[Wed Nov 30 15:12:59 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P2rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P2dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P2output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 15:12:59 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 15:12:59\tMarkDuplicates\tStart of doWork freeMemory: 2042777056; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:12:59\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 15:12:59\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 15:13:04\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:31,211,857\n",
      "INFO\t2016-11-30 15:13:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:09\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    4s.  Last read position: 1:114,911,213\n",
      "INFO\t2016-11-30 15:13:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:11\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    1s.  Last read position: 1:179,813,069\n",
      "INFO\t2016-11-30 15:13:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:15\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    4s.  Last read position: 2:55,237,227\n",
      "INFO\t2016-11-30 15:13:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:23\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:22s.  Time for last 1,000,000:    7s.  Last read position: 2:179,306,338\n",
      "INFO\t2016-11-30 15:13:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:24\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    1s.  Last read position: 3:49,753,417\n",
      "INFO\t2016-11-30 15:13:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:31\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:31s.  Time for last 1,000,000:    6s.  Last read position: 3:186,795,411\n",
      "INFO\t2016-11-30 15:13:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:33\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    1s.  Last read position: 4:89,182,401\n",
      "INFO\t2016-11-30 15:13:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:40\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:40s.  Time for last 1,000,000:    6s.  Last read position: 5:72,893,917\n",
      "INFO\t2016-11-30 15:13:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:41\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:41s.  Time for last 1,000,000:    1s.  Last read position: 5:176,777,576\n",
      "INFO\t2016-11-30 15:13:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:51\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    9s.  Last read position: 6:33,036,917\n",
      "INFO\t2016-11-30 15:13:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:52\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:52s.  Time for last 1,000,000:    1s.  Last read position: 6:160,101,292\n",
      "INFO\t2016-11-30 15:13:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:54\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:54s.  Time for last 1,000,000:    1s.  Last read position: 7:74,285,509\n",
      "INFO\t2016-11-30 15:13:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:13:56\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    1s.  Last read position: 8:22,481,845\n",
      "INFO\t2016-11-30 15:13:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:04\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    8s.  Last read position: 9:42,712,222\n",
      "INFO\t2016-11-30 15:14:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:06\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:06s.  Time for last 1,000,000:    1s.  Last read position: 10:7,455,349\n",
      "INFO\t2016-11-30 15:14:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:07\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:07s.  Time for last 1,000,000:    1s.  Last read position: 10:98,281,798\n",
      "INFO\t2016-11-30 15:14:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:09\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:09s.  Time for last 1,000,000:    1s.  Last read position: 11:60,154,800\n",
      "INFO\t2016-11-30 15:14:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:28\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:28s.  Time for last 1,000,000:   18s.  Last read position: 11:118,257,111\n",
      "INFO\t2016-11-30 15:14:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:29\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    1s.  Last read position: 12:50,036,039\n",
      "INFO\t2016-11-30 15:14:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:31\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    1s.  Last read position: 12:121,768,403\n",
      "INFO\t2016-11-30 15:14:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:32\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    1s.  Last read position: 14:25,735,564\n",
      "INFO\t2016-11-30 15:14:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:41\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:41s.  Time for last 1,000,000:    9s.  Last read position: 15:40,618,586\n",
      "INFO\t2016-11-30 15:14:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:43\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:42s.  Time for last 1,000,000:    1s.  Last read position: 16:2,818,221\n",
      "INFO\t2016-11-30 15:14:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:44\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    1s.  Last read position: 16:68,331,197\n",
      "INFO\t2016-11-30 15:14:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:45\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    1s.  Last read position: 17:26,890,876\n",
      "INFO\t2016-11-30 15:14:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:46\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:46s.  Time for last 1,000,000:    1s.  Last read position: 17:72,840,258\n",
      "INFO\t2016-11-30 15:14:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:48\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:01:48s.  Time for last 1,000,000:    1s.  Last read position: 19:4,021,251\n",
      "INFO\t2016-11-30 15:14:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:14:50\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:01:49s.  Time for last 1,000,000:    1s.  Last read position: 19:19,764,612\n",
      "INFO\t2016-11-30 15:14:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:00\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:   10s.  Last read position: 19:50,023,673\n",
      "INFO\t2016-11-30 15:15:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:02\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:02s.  Time for last 1,000,000:    1s.  Last read position: 20:49,456,834\n",
      "INFO\t2016-11-30 15:15:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:03\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    1s.  Last read position: 22:38,476,047\n",
      "INFO\t2016-11-30 15:15:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:04\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:04s.  Time for last 1,000,000:    1s.  Last read position: X:73,174,149\n",
      "INFO\t2016-11-30 15:15:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:06\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    1s.  Last read position: MT:10,679\n",
      "INFO\t2016-11-30 15:15:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:07\tMarkDuplicates\tRead    35,000,000 records.  Elapsed time: 00:02:07s.  Time for last 1,000,000:    1s.  Last read position: GL000195.1:149,908\n",
      "INFO\t2016-11-30 15:15:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:15:07\tMarkDuplicates\tRead 35009859 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 15:15:24\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10141337024; totalMemory: 13329498112; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:15:24\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 15:15:24\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:15:24\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:15:28\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 15:15:29\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 5979304376; totalMemory: 13329498112; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:15:29\tMarkDuplicates\tMarking 19579075 records as duplicates.\n",
      "INFO\t2016-11-30 15:15:29\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 15:15:29\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 15:16:20\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:50s.  Time for last 10,000,000:   50s.  Last read position: 5:176,777,576\n",
      "INFO\t2016-11-30 15:17:13\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:43s.  Time for last 10,000,000:   52s.  Last read position: 12:50,036,039\n",
      "INFO\t2016-11-30 15:18:04\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:35s.  Time for last 10,000,000:   51s.  Last read position: 19:50,023,673\n",
      "INFO\t2016-11-30 15:18:29\tMarkDuplicates\tBefore output close freeMemory: 19850324856; totalMemory: 20041957376; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:18:29\tMarkDuplicates\tAfter output close freeMemory: 20268037352; totalMemory: 20459814912; maxMemory: 28631367680\n",
      "[Wed Nov 30 15:18:29 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 5.49 minutes.\n",
      "Runtime.totalMemory()=20459814912\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P3rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P3dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P3output.metrics\n",
      "[Wed Nov 30 15:18:29 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P3rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P3dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P3output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 15:18:29 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 15:18:29\tMarkDuplicates\tStart of doWork freeMemory: 2042777056; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:18:29\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 15:18:29\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 15:18:34\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:31,206,410\n",
      "INFO\t2016-11-30 15:18:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:38\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:08s.  Time for last 1,000,000:    4s.  Last read position: 1:143,162,910\n",
      "INFO\t2016-11-30 15:18:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:40\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:10s.  Time for last 1,000,000:    1s.  Last read position: 1:182,353,621\n",
      "INFO\t2016-11-30 15:18:40\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:46\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:16s.  Time for last 1,000,000:    5s.  Last read position: 2:64,819,475\n",
      "INFO\t2016-11-30 15:18:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:50\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    3s.  Last read position: 2:209,101,345\n",
      "INFO\t2016-11-30 15:18:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:51\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:21s.  Time for last 1,000,000:    1s.  Last read position: 3:73,113,213\n",
      "INFO\t2016-11-30 15:18:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:18:56\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:26s.  Time for last 1,000,000:    4s.  Last read position: 4:26,422,276\n",
      "INFO\t2016-11-30 15:18:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:02\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:32s.  Time for last 1,000,000:    5s.  Last read position: 5:32,354,775\n",
      "INFO\t2016-11-30 15:19:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:03\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    1s.  Last read position: 5:149,781,213\n",
      "INFO\t2016-11-30 15:19:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:05\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    1s.  Last read position: 6:31,238,872\n",
      "INFO\t2016-11-30 15:19:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:19\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:48s.  Time for last 1,000,000:   13s.  Last read position: 6:89,791,836\n",
      "INFO\t2016-11-30 15:19:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:20\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:50s.  Time for last 1,000,000:    1s.  Last read position: 7:38,366,802\n",
      "INFO\t2016-11-30 15:19:20\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:22\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:51s.  Time for last 1,000,000:    1s.  Last read position: 8:11,700,675\n",
      "INFO\t2016-11-30 15:19:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:23\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:    1s.  Last read position: 9:34,398,892\n",
      "INFO\t2016-11-30 15:19:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:31\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:    7s.  Last read position: 10:13,701,385\n",
      "INFO\t2016-11-30 15:19:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:32\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:02s.  Time for last 1,000,000:    1s.  Last read position: 10:103,922,955\n",
      "INFO\t2016-11-30 15:19:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:34\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    1s.  Last read position: 11:63,971,291\n",
      "INFO\t2016-11-30 15:19:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:36\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:05s.  Time for last 1,000,000:    1s.  Last read position: 12:6,573,703\n",
      "INFO\t2016-11-30 15:19:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:48\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:   12s.  Last read position: 12:65,141,642\n",
      "INFO\t2016-11-30 15:19:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:50\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:20s.  Time for last 1,000,000:    1s.  Last read position: 13:48,835,821\n",
      "INFO\t2016-11-30 15:19:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:51\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:21s.  Time for last 1,000,000:    1s.  Last read position: 14:97,031,435\n",
      "INFO\t2016-11-30 15:19:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:53\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:23s.  Time for last 1,000,000:    1s.  Last read position: 15:86,292,503\n",
      "INFO\t2016-11-30 15:19:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:57\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:27s.  Time for last 1,000,000:    4s.  Last read position: 16:61,089,732\n",
      "INFO\t2016-11-30 15:19:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:19:59\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    1s.  Last read position: 17:29,311,662\n",
      "INFO\t2016-11-30 15:19:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:01\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    1s.  Last read position: 17:78,355,125\n",
      "INFO\t2016-11-30 15:20:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:02\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    1s.  Last read position: 19:6,719,379\n",
      "INFO\t2016-11-30 15:20:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:03\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:33s.  Time for last 1,000,000:    1s.  Last read position: 19:41,092,821\n",
      "INFO\t2016-11-30 15:20:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:05\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    1s.  Last read position: 20:23,063,455\n",
      "INFO\t2016-11-30 15:20:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:24\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:01:54s.  Time for last 1,000,000:   18s.  Last read position: 22:24,515,557\n",
      "INFO\t2016-11-30 15:20:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:25\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:01:55s.  Time for last 1,000,000:    1s.  Last read position: X:64,958,456\n",
      "INFO\t2016-11-30 15:20:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:26\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:01:56s.  Time for last 1,000,000:    1s.  Last read position: MT:11,014\n",
      "INFO\t2016-11-30 15:20:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:20:27\tMarkDuplicates\tRead 31438960 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 15:20:42\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10405418544; totalMemory: 13421248512; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:20:42\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 15:20:42\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:20:42\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:20:45\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 15:20:46\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 6069320616; totalMemory: 13421248512; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:20:46\tMarkDuplicates\tMarking 14827648 records as duplicates.\n",
      "INFO\t2016-11-30 15:20:46\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 15:20:46\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 15:21:42\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:55s.  Time for last 10,000,000:   55s.  Last read position: 6:31,238,872\n",
      "INFO\t2016-11-30 15:22:36\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:49s.  Time for last 10,000,000:   54s.  Last read position: 13:48,835,821\n",
      "INFO\t2016-11-30 15:23:31\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:44s.  Time for last 10,000,000:   54s.  Last read position: X:64,958,456\n",
      "INFO\t2016-11-30 15:23:38\tMarkDuplicates\tBefore output close freeMemory: 20362450792; totalMemory: 20554186752; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:23:38\tMarkDuplicates\tAfter output close freeMemory: 19966085952; totalMemory: 20157825024; maxMemory: 28631367680\n",
      "[Wed Nov 30 15:23:38 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 5.14 minutes.\n",
      "Runtime.totalMemory()=20157825024\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P6rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P6dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P6output.metrics\n",
      "[Wed Nov 30 15:23:38 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P6rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P6dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P6output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 15:23:38 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 15:23:38\tMarkDuplicates\tStart of doWork freeMemory: 2042777056; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:23:38\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 15:23:38\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 15:23:44\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:05s.  Time for last 1,000,000:    5s.  Last read position: 1:36,932,176\n",
      "INFO\t2016-11-30 15:23:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:23:51\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:11s.  Time for last 1,000,000:    6s.  Last read position: 1:121,133,154\n",
      "INFO\t2016-11-30 15:23:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:23:52\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:13s.  Time for last 1,000,000:    1s.  Last read position: 1:192,546,489\n",
      "INFO\t2016-11-30 15:23:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:23:58\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    5s.  Last read position: 2:44,428,629\n",
      "INFO\t2016-11-30 15:23:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:03\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:24s.  Time for last 1,000,000:    4s.  Last read position: 2:144,716,605\n",
      "INFO\t2016-11-30 15:24:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:13\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:   10s.  Last read position: 3:37,365,844\n",
      "INFO\t2016-11-30 15:24:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:15\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:36s.  Time for last 1,000,000:    1s.  Last read position: 3:158,520,012\n",
      "INFO\t2016-11-30 15:24:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:21\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:42s.  Time for last 1,000,000:    6s.  Last read position: 4:83,348,611\n",
      "INFO\t2016-11-30 15:24:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:23\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:44s.  Time for last 1,000,000:    1s.  Last read position: 5:40,707,791\n",
      "INFO\t2016-11-30 15:24:23\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:25\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:45s.  Time for last 1,000,000:    1s.  Last read position: 5:149,312,137\n",
      "INFO\t2016-11-30 15:24:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:32\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:53s.  Time for last 1,000,000:    7s.  Last read position: 6:31,796,766\n",
      "INFO\t2016-11-30 15:24:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:34\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:55s.  Time for last 1,000,000:    1s.  Last read position: 6:137,519,736\n",
      "INFO\t2016-11-30 15:24:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:35\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:56s.  Time for last 1,000,000:    1s.  Last read position: 7:64,145,135\n",
      "INFO\t2016-11-30 15:24:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:42\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    6s.  Last read position: 7:150,270,449\n",
      "INFO\t2016-11-30 15:24:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:43\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    1s.  Last read position: 8:135,505,025\n",
      "INFO\t2016-11-30 15:24:43\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:45\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:06s.  Time for last 1,000,000:    1s.  Last read position: 9:124,047,914\n",
      "INFO\t2016-11-30 15:24:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:24:47\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:08s.  Time for last 1,000,000:    1s.  Last read position: 10:46,887,841\n",
      "INFO\t2016-11-30 15:24:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:05\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:25s.  Time for last 1,000,000:   17s.  Last read position: 11:631,811\n",
      "INFO\t2016-11-30 15:25:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:06\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:27s.  Time for last 1,000,000:    1s.  Last read position: 11:74,876,947\n",
      "INFO\t2016-11-30 15:25:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:08\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    1s.  Last read position: 12:26,275,703\n",
      "INFO\t2016-11-30 15:25:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:10\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:30s.  Time for last 1,000,000:    1s.  Last read position: 12:110,378,103\n",
      "INFO\t2016-11-30 15:25:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:19\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:40s.  Time for last 1,000,000:    9s.  Last read position: 13:113,342,451\n",
      "INFO\t2016-11-30 15:25:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:21\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:42s.  Time for last 1,000,000:    1s.  Last read position: 15:23,201,580\n",
      "INFO\t2016-11-30 15:25:21\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:22\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:43s.  Time for last 1,000,000:    1s.  Last read position: 15:101,567,932\n",
      "INFO\t2016-11-30 15:25:22\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:24\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    1s.  Last read position: 16:29,526,348\n",
      "INFO\t2016-11-30 15:25:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:25\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:46s.  Time for last 1,000,000:    1s.  Last read position: 17:3,916,784\n",
      "INFO\t2016-11-30 15:25:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:27\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:48s.  Time for last 1,000,000:    1s.  Last read position: 17:45,745,727\n",
      "INFO\t2016-11-30 15:25:27\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:39\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:02:00s.  Time for last 1,000,000:   12s.  Last read position: 18:59,925,785\n",
      "INFO\t2016-11-30 15:25:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:41\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:02:02s.  Time for last 1,000,000:    1s.  Last read position: 19:21,901,047\n",
      "INFO\t2016-11-30 15:25:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:42\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    1s.  Last read position: 20:23,348,249\n",
      "INFO\t2016-11-30 15:25:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:44\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:02:05s.  Time for last 1,000,000:    1s.  Last read position: 22:21,065,080\n",
      "INFO\t2016-11-30 15:25:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:45\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    1s.  Last read position: X:48,932,387\n",
      "INFO\t2016-11-30 15:25:45\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:47\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:02:08s.  Time for last 1,000,000:    1s.  Last read position: MT:11,141\n",
      "INFO\t2016-11-30 15:25:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:25:47\tMarkDuplicates\tRead 33346929 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 15:25:59\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 10186838280; totalMemory: 13335789568; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:25:59\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 15:26:00\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:26:00\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:26:03\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 15:26:04\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 5985192856; totalMemory: 13335789568; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:26:04\tMarkDuplicates\tMarking 15392165 records as duplicates.\n",
      "INFO\t2016-11-30 15:26:04\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 15:26:04\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 15:27:07\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:01:03s.  Time for last 10,000,000:   63s.  Last read position: 5:149,312,137\n",
      "INFO\t2016-11-30 15:28:08\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:02:04s.  Time for last 10,000,000:   60s.  Last read position: 12:26,275,703\n",
      "INFO\t2016-11-30 15:29:08\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:03:03s.  Time for last 10,000,000:   59s.  Last read position: 20:23,348,249\n",
      "INFO\t2016-11-30 15:29:28\tMarkDuplicates\tBefore output close freeMemory: 20297827928; totalMemory: 20488126464; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:29:28\tMarkDuplicates\tAfter output close freeMemory: 19856374320; totalMemory: 20046675968; maxMemory: 28631367680\n",
      "[Wed Nov 30 15:29:28 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 5.83 minutes.\n",
      "Runtime.totalMemory()=20046675968\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P10output.metrics\n",
      "[Wed Nov 30 15:29:29 PST 2016] picard.sam.markduplicates.MarkDuplicates INPUT=[/data/scratch/iholtman/human_microglia_genotyping/step5/P10rg_added_sorted.bam] OUTPUT=/data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam METRICS_FILE=/data/scratch/iholtman/human_microglia_genotyping/step5/P10output.metrics VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag REMOVE_DUPLICATES=false ASSUME_SORTED=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Nov 30 15:29:29 PST 2016] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2016-11-30 15:29:29\tMarkDuplicates\tStart of doWork freeMemory: 2042777032; totalMemory: 2058354688; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:29:29\tMarkDuplicates\tReading input file and constructing read end information.\n",
      "INFO\t2016-11-30 15:29:29\tMarkDuplicates\tWill retain up to 110120644 data points before spilling to disk.\n",
      "INFO\t2016-11-30 15:29:33\tMarkDuplicates\tRead     1,000,000 records.  Elapsed time: 00:00:04s.  Time for last 1,000,000:    4s.  Last read position: 1:9,638,447\n",
      "INFO\t2016-11-30 15:29:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:36\tMarkDuplicates\tRead     2,000,000 records.  Elapsed time: 00:00:06s.  Time for last 1,000,000:    2s.  Last read position: 1:23,103,433\n",
      "INFO\t2016-11-30 15:29:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:39\tMarkDuplicates\tRead     3,000,000 records.  Elapsed time: 00:00:09s.  Time for last 1,000,000:    3s.  Last read position: 1:36,932,400\n",
      "INFO\t2016-11-30 15:29:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:44\tMarkDuplicates\tRead     4,000,000 records.  Elapsed time: 00:00:15s.  Time for last 1,000,000:    5s.  Last read position: 1:53,147,827\n",
      "INFO\t2016-11-30 15:29:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:48\tMarkDuplicates\tRead     5,000,000 records.  Elapsed time: 00:00:19s.  Time for last 1,000,000:    4s.  Last read position: 1:104,076,387\n",
      "INFO\t2016-11-30 15:29:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:51\tMarkDuplicates\tRead     6,000,000 records.  Elapsed time: 00:00:21s.  Time for last 1,000,000:    2s.  Last read position: 1:121,121,490\n",
      "INFO\t2016-11-30 15:29:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:29:52\tMarkDuplicates\tRead     7,000,000 records.  Elapsed time: 00:00:23s.  Time for last 1,000,000:    1s.  Last read position: 1:149,754,961\n",
      "INFO\t2016-11-30 15:29:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:02\tMarkDuplicates\tRead     8,000,000 records.  Elapsed time: 00:00:33s.  Time for last 1,000,000:    9s.  Last read position: 1:161,483,955\n",
      "INFO\t2016-11-30 15:30:02\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:03\tMarkDuplicates\tRead     9,000,000 records.  Elapsed time: 00:00:34s.  Time for last 1,000,000:    1s.  Last read position: 1:193,063,734\n",
      "INFO\t2016-11-30 15:30:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:05\tMarkDuplicates\tRead    10,000,000 records.  Elapsed time: 00:00:35s.  Time for last 1,000,000:    1s.  Last read position: 1:212,867,462\n",
      "INFO\t2016-11-30 15:30:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:12\tMarkDuplicates\tRead    11,000,000 records.  Elapsed time: 00:00:43s.  Time for last 1,000,000:    7s.  Last read position: 1:247,353,214\n",
      "INFO\t2016-11-30 15:30:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:14\tMarkDuplicates\tRead    12,000,000 records.  Elapsed time: 00:00:44s.  Time for last 1,000,000:    1s.  Last read position: 2:43,777,400\n",
      "INFO\t2016-11-30 15:30:14\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:15\tMarkDuplicates\tRead    13,000,000 records.  Elapsed time: 00:00:46s.  Time for last 1,000,000:    1s.  Last read position: 2:75,102,076\n",
      "INFO\t2016-11-30 15:30:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:31\tMarkDuplicates\tRead    14,000,000 records.  Elapsed time: 00:01:01s.  Time for last 1,000,000:   15s.  Last read position: 2:112,559,696\n",
      "INFO\t2016-11-30 15:30:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:32\tMarkDuplicates\tRead    15,000,000 records.  Elapsed time: 00:01:03s.  Time for last 1,000,000:    1s.  Last read position: 2:153,300,275\n",
      "INFO\t2016-11-30 15:30:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:33\tMarkDuplicates\tRead    16,000,000 records.  Elapsed time: 00:01:04s.  Time for last 1,000,000:    1s.  Last read position: 2:203,576,479\n",
      "INFO\t2016-11-30 15:30:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:35\tMarkDuplicates\tRead    17,000,000 records.  Elapsed time: 00:01:05s.  Time for last 1,000,000:    1s.  Last read position: 2:242,203,961\n",
      "INFO\t2016-11-30 15:30:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:48\tMarkDuplicates\tRead    18,000,000 records.  Elapsed time: 00:01:18s.  Time for last 1,000,000:   12s.  Last read position: 3:40,803,108\n",
      "INFO\t2016-11-30 15:30:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:49\tMarkDuplicates\tRead    19,000,000 records.  Elapsed time: 00:01:20s.  Time for last 1,000,000:    1s.  Last read position: 3:57,313,999\n",
      "INFO\t2016-11-30 15:30:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:50\tMarkDuplicates\tRead    20,000,000 records.  Elapsed time: 00:01:21s.  Time for last 1,000,000:    1s.  Last read position: 3:128,838,067\n",
      "INFO\t2016-11-30 15:30:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:52\tMarkDuplicates\tRead    21,000,000 records.  Elapsed time: 00:01:22s.  Time for last 1,000,000:    1s.  Last read position: 3:179,116,354\n",
      "INFO\t2016-11-30 15:30:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:57\tMarkDuplicates\tRead    22,000,000 records.  Elapsed time: 00:01:28s.  Time for last 1,000,000:    5s.  Last read position: 3:197,946,855\n",
      "INFO\t2016-11-30 15:30:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:30:59\tMarkDuplicates\tRead    23,000,000 records.  Elapsed time: 00:01:29s.  Time for last 1,000,000:    1s.  Last read position: 4:38,756,580\n",
      "INFO\t2016-11-30 15:30:59\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:00\tMarkDuplicates\tRead    24,000,000 records.  Elapsed time: 00:01:31s.  Time for last 1,000,000:    1s.  Last read position: 4:88,897,955\n",
      "INFO\t2016-11-30 15:31:00\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:01\tMarkDuplicates\tRead    25,000,000 records.  Elapsed time: 00:01:32s.  Time for last 1,000,000:    1s.  Last read position: 4:144,469,088\n",
      "INFO\t2016-11-30 15:31:01\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:03\tMarkDuplicates\tRead    26,000,000 records.  Elapsed time: 00:01:33s.  Time for last 1,000,000:    1s.  Last read position: 5:17,270,237\n",
      "INFO\t2016-11-30 15:31:03\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:04\tMarkDuplicates\tRead    27,000,000 records.  Elapsed time: 00:01:35s.  Time for last 1,000,000:    1s.  Last read position: 5:64,873,820\n",
      "INFO\t2016-11-30 15:31:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:12\tMarkDuplicates\tRead    28,000,000 records.  Elapsed time: 00:01:43s.  Time for last 1,000,000:    7s.  Last read position: 5:90,667,475\n",
      "INFO\t2016-11-30 15:31:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:13\tMarkDuplicates\tRead    29,000,000 records.  Elapsed time: 00:01:44s.  Time for last 1,000,000:    1s.  Last read position: 5:140,011,632\n",
      "INFO\t2016-11-30 15:31:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:15\tMarkDuplicates\tRead    30,000,000 records.  Elapsed time: 00:01:45s.  Time for last 1,000,000:    1s.  Last read position: 5:157,214,336\n",
      "INFO\t2016-11-30 15:31:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:16\tMarkDuplicates\tRead    31,000,000 records.  Elapsed time: 00:01:47s.  Time for last 1,000,000:    1s.  Last read position: 6:2,887,768\n",
      "INFO\t2016-11-30 15:31:16\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:17\tMarkDuplicates\tRead    32,000,000 records.  Elapsed time: 00:01:48s.  Time for last 1,000,000:    1s.  Last read position: 6:31,584,205\n",
      "INFO\t2016-11-30 15:31:17\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:19\tMarkDuplicates\tRead    33,000,000 records.  Elapsed time: 00:01:49s.  Time for last 1,000,000:    1s.  Last read position: 6:32,973,462\n",
      "INFO\t2016-11-30 15:31:19\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:31\tMarkDuplicates\tRead    34,000,000 records.  Elapsed time: 00:02:02s.  Time for last 1,000,000:   12s.  Last read position: 6:42,923,454\n",
      "INFO\t2016-11-30 15:31:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:33\tMarkDuplicates\tRead    35,000,000 records.  Elapsed time: 00:02:03s.  Time for last 1,000,000:    1s.  Last read position: 6:108,881,209\n",
      "INFO\t2016-11-30 15:31:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:34\tMarkDuplicates\tRead    36,000,000 records.  Elapsed time: 00:02:05s.  Time for last 1,000,000:    1s.  Last read position: 6:144,169,988\n",
      "INFO\t2016-11-30 15:31:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:35\tMarkDuplicates\tRead    37,000,000 records.  Elapsed time: 00:02:06s.  Time for last 1,000,000:    1s.  Last read position: 7:2,685,261\n",
      "INFO\t2016-11-30 15:31:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:37\tMarkDuplicates\tRead    38,000,000 records.  Elapsed time: 00:02:07s.  Time for last 1,000,000:    1s.  Last read position: 7:26,240,234\n",
      "INFO\t2016-11-30 15:31:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:31:38\tMarkDuplicates\tRead    39,000,000 records.  Elapsed time: 00:02:09s.  Time for last 1,000,000:    1s.  Last read position: 7:66,009,371\n",
      "INFO\t2016-11-30 15:31:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:05\tMarkDuplicates\tRead    40,000,000 records.  Elapsed time: 00:02:36s.  Time for last 1,000,000:   27s.  Last read position: 7:95,217,600\n",
      "INFO\t2016-11-30 15:32:05\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:07\tMarkDuplicates\tRead    41,000,000 records.  Elapsed time: 00:02:37s.  Time for last 1,000,000:    1s.  Last read position: 7:114,294,021\n",
      "INFO\t2016-11-30 15:32:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:08\tMarkDuplicates\tRead    42,000,000 records.  Elapsed time: 00:02:39s.  Time for last 1,000,000:    1s.  Last read position: 7:150,069,288\n",
      "INFO\t2016-11-30 15:32:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:09\tMarkDuplicates\tRead    43,000,000 records.  Elapsed time: 00:02:40s.  Time for last 1,000,000:    1s.  Last read position: 8:22,876,584\n",
      "INFO\t2016-11-30 15:32:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:11\tMarkDuplicates\tRead    44,000,000 records.  Elapsed time: 00:02:42s.  Time for last 1,000,000:    1s.  Last read position: 8:67,545,967\n",
      "INFO\t2016-11-30 15:32:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:12\tMarkDuplicates\tRead    45,000,000 records.  Elapsed time: 00:02:43s.  Time for last 1,000,000:    1s.  Last read position: 8:134,059,308\n",
      "INFO\t2016-11-30 15:32:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:35\tMarkDuplicates\tRead    46,000,000 records.  Elapsed time: 00:03:06s.  Time for last 1,000,000:   23s.  Last read position: 9:21,512,314\n",
      "INFO\t2016-11-30 15:32:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:37\tMarkDuplicates\tRead    47,000,000 records.  Elapsed time: 00:03:07s.  Time for last 1,000,000:    1s.  Last read position: 9:71,590,443\n",
      "INFO\t2016-11-30 15:32:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:38\tMarkDuplicates\tRead    48,000,000 records.  Elapsed time: 00:03:09s.  Time for last 1,000,000:    1s.  Last read position: 9:120,481,070\n",
      "INFO\t2016-11-30 15:32:38\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:39\tMarkDuplicates\tRead    49,000,000 records.  Elapsed time: 00:03:10s.  Time for last 1,000,000:    1s.  Last read position: 9:135,895,890\n",
      "INFO\t2016-11-30 15:32:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:41\tMarkDuplicates\tRead    50,000,000 records.  Elapsed time: 00:03:11s.  Time for last 1,000,000:    1s.  Last read position: 10:7,844,324\n",
      "INFO\t2016-11-30 15:32:41\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:42\tMarkDuplicates\tRead    51,000,000 records.  Elapsed time: 00:03:13s.  Time for last 1,000,000:    1s.  Last read position: 10:22,618,812\n",
      "INFO\t2016-11-30 15:32:42\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:44\tMarkDuplicates\tRead    52,000,000 records.  Elapsed time: 00:03:15s.  Time for last 1,000,000:    1s.  Last read position: 10:50,174,630\n",
      "INFO\t2016-11-30 15:32:44\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:52\tMarkDuplicates\tRead    53,000,000 records.  Elapsed time: 00:03:22s.  Time for last 1,000,000:    7s.  Last read position: 10:75,464,701\n",
      "INFO\t2016-11-30 15:32:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:53\tMarkDuplicates\tRead    54,000,000 records.  Elapsed time: 00:03:24s.  Time for last 1,000,000:    1s.  Last read position: 10:102,258,934\n",
      "INFO\t2016-11-30 15:32:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:54\tMarkDuplicates\tRead    55,000,000 records.  Elapsed time: 00:03:25s.  Time for last 1,000,000:    1s.  Last read position: 11:499,887\n",
      "INFO\t2016-11-30 15:32:54\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:56\tMarkDuplicates\tRead    56,000,000 records.  Elapsed time: 00:03:26s.  Time for last 1,000,000:    1s.  Last read position: 11:33,376,486\n",
      "INFO\t2016-11-30 15:32:56\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:57\tMarkDuplicates\tRead    57,000,000 records.  Elapsed time: 00:03:28s.  Time for last 1,000,000:    1s.  Last read position: 11:64,538,708\n",
      "INFO\t2016-11-30 15:32:57\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:32:58\tMarkDuplicates\tRead    58,000,000 records.  Elapsed time: 00:03:29s.  Time for last 1,000,000:    1s.  Last read position: 11:71,727,074\n",
      "INFO\t2016-11-30 15:32:58\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:30\tMarkDuplicates\tRead    59,000,000 records.  Elapsed time: 00:04:01s.  Time for last 1,000,000:   31s.  Last read position: 11:102,268,515\n",
      "INFO\t2016-11-30 15:33:30\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:31\tMarkDuplicates\tRead    60,000,000 records.  Elapsed time: 00:04:02s.  Time for last 1,000,000:    1s.  Last read position: 12:80,753\n",
      "INFO\t2016-11-30 15:33:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:33\tMarkDuplicates\tRead    61,000,000 records.  Elapsed time: 00:04:03s.  Time for last 1,000,000:    1s.  Last read position: 12:15,095,427\n",
      "INFO\t2016-11-30 15:33:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:34\tMarkDuplicates\tRead    62,000,000 records.  Elapsed time: 00:04:05s.  Time for last 1,000,000:    1s.  Last read position: 12:50,095,718\n",
      "INFO\t2016-11-30 15:33:34\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:35\tMarkDuplicates\tRead    63,000,000 records.  Elapsed time: 00:04:06s.  Time for last 1,000,000:    1s.  Last read position: 12:66,542,138\n",
      "INFO\t2016-11-30 15:33:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:37\tMarkDuplicates\tRead    64,000,000 records.  Elapsed time: 00:04:07s.  Time for last 1,000,000:    1s.  Last read position: 12:108,988,143\n",
      "INFO\t2016-11-30 15:33:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:47\tMarkDuplicates\tRead    65,000,000 records.  Elapsed time: 00:04:17s.  Time for last 1,000,000:    9s.  Last read position: 12:125,267,897\n",
      "INFO\t2016-11-30 15:33:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:48\tMarkDuplicates\tRead    66,000,000 records.  Elapsed time: 00:04:18s.  Time for last 1,000,000:    1s.  Last read position: 13:42,877,252\n",
      "INFO\t2016-11-30 15:33:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:49\tMarkDuplicates\tRead    67,000,000 records.  Elapsed time: 00:04:20s.  Time for last 1,000,000:    1s.  Last read position: 13:100,026,695\n",
      "INFO\t2016-11-30 15:33:49\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:50\tMarkDuplicates\tRead    68,000,000 records.  Elapsed time: 00:04:21s.  Time for last 1,000,000:    1s.  Last read position: 14:24,903,066\n",
      "INFO\t2016-11-30 15:33:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:52\tMarkDuplicates\tRead    69,000,000 records.  Elapsed time: 00:04:22s.  Time for last 1,000,000:    1s.  Last read position: 14:75,190,712\n",
      "INFO\t2016-11-30 15:33:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:33:53\tMarkDuplicates\tRead    70,000,000 records.  Elapsed time: 00:04:24s.  Time for last 1,000,000:    1s.  Last read position: 14:100,604,464\n",
      "INFO\t2016-11-30 15:33:53\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:04\tMarkDuplicates\tRead    71,000,000 records.  Elapsed time: 00:04:35s.  Time for last 1,000,000:   11s.  Last read position: 15:31,666,246\n",
      "INFO\t2016-11-30 15:34:04\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:06\tMarkDuplicates\tRead    72,000,000 records.  Elapsed time: 00:04:36s.  Time for last 1,000,000:    1s.  Last read position: 15:59,495,303\n",
      "INFO\t2016-11-30 15:34:06\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:07\tMarkDuplicates\tRead    73,000,000 records.  Elapsed time: 00:04:38s.  Time for last 1,000,000:    1s.  Last read position: 15:83,054,555\n",
      "INFO\t2016-11-30 15:34:07\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:08\tMarkDuplicates\tRead    74,000,000 records.  Elapsed time: 00:04:39s.  Time for last 1,000,000:    1s.  Last read position: 16:2,650,200\n",
      "INFO\t2016-11-30 15:34:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:10\tMarkDuplicates\tRead    75,000,000 records.  Elapsed time: 00:04:40s.  Time for last 1,000,000:    1s.  Last read position: 16:18,525,685\n",
      "INFO\t2016-11-30 15:34:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:11\tMarkDuplicates\tRead    76,000,000 records.  Elapsed time: 00:04:41s.  Time for last 1,000,000:    1s.  Last read position: 16:24,227,411\n",
      "INFO\t2016-11-30 15:34:11\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:12\tMarkDuplicates\tRead    77,000,000 records.  Elapsed time: 00:04:43s.  Time for last 1,000,000:    1s.  Last read position: 16:30,292,712\n",
      "INFO\t2016-11-30 15:34:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:32\tMarkDuplicates\tRead    78,000,000 records.  Elapsed time: 00:05:03s.  Time for last 1,000,000:   20s.  Last read position: 16:67,264,865\n",
      "INFO\t2016-11-30 15:34:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:33\tMarkDuplicates\tRead    79,000,000 records.  Elapsed time: 00:05:04s.  Time for last 1,000,000:    1s.  Last read position: 16:88,784,270\n",
      "INFO\t2016-11-30 15:34:33\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:35\tMarkDuplicates\tRead    80,000,000 records.  Elapsed time: 00:05:05s.  Time for last 1,000,000:    1s.  Last read position: 17:4,721,854\n",
      "INFO\t2016-11-30 15:34:35\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:36\tMarkDuplicates\tRead    81,000,000 records.  Elapsed time: 00:05:07s.  Time for last 1,000,000:    1s.  Last read position: 17:20,315,825\n",
      "INFO\t2016-11-30 15:34:36\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:37\tMarkDuplicates\tRead    82,000,000 records.  Elapsed time: 00:05:08s.  Time for last 1,000,000:    1s.  Last read position: 17:36,884,949\n",
      "INFO\t2016-11-30 15:34:37\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:39\tMarkDuplicates\tRead    83,000,000 records.  Elapsed time: 00:05:09s.  Time for last 1,000,000:    1s.  Last read position: 17:56,076,005\n",
      "INFO\t2016-11-30 15:34:39\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:46\tMarkDuplicates\tRead    84,000,000 records.  Elapsed time: 00:05:16s.  Time for last 1,000,000:    6s.  Last read position: 17:74,472,771\n",
      "INFO\t2016-11-30 15:34:46\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:47\tMarkDuplicates\tRead    85,000,000 records.  Elapsed time: 00:05:18s.  Time for last 1,000,000:    1s.  Last read position: 18:7,119,505\n",
      "INFO\t2016-11-30 15:34:47\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:48\tMarkDuplicates\tRead    86,000,000 records.  Elapsed time: 00:05:19s.  Time for last 1,000,000:    1s.  Last read position: 18:67,808,631\n",
      "INFO\t2016-11-30 15:34:48\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:50\tMarkDuplicates\tRead    87,000,000 records.  Elapsed time: 00:05:20s.  Time for last 1,000,000:    1s.  Last read position: 19:5,106,969\n",
      "INFO\t2016-11-30 15:34:50\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:51\tMarkDuplicates\tRead    88,000,000 records.  Elapsed time: 00:05:22s.  Time for last 1,000,000:    1s.  Last read position: 19:10,906,663\n",
      "INFO\t2016-11-30 15:34:51\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:34:52\tMarkDuplicates\tRead    89,000,000 records.  Elapsed time: 00:05:23s.  Time for last 1,000,000:    1s.  Last read position: 19:18,970,507\n",
      "INFO\t2016-11-30 15:34:52\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:08\tMarkDuplicates\tRead    90,000,000 records.  Elapsed time: 00:05:38s.  Time for last 1,000,000:   15s.  Last read position: 19:41,838,110\n",
      "INFO\t2016-11-30 15:35:08\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:09\tMarkDuplicates\tRead    91,000,000 records.  Elapsed time: 00:05:40s.  Time for last 1,000,000:    1s.  Last read position: 19:51,919,281\n",
      "INFO\t2016-11-30 15:35:09\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:10\tMarkDuplicates\tRead    92,000,000 records.  Elapsed time: 00:05:41s.  Time for last 1,000,000:    1s.  Last read position: 20:5,690,004\n",
      "INFO\t2016-11-30 15:35:10\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:12\tMarkDuplicates\tRead    93,000,000 records.  Elapsed time: 00:05:43s.  Time for last 1,000,000:    1s.  Last read position: 20:42,088,866\n",
      "INFO\t2016-11-30 15:35:12\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:13\tMarkDuplicates\tRead    94,000,000 records.  Elapsed time: 00:05:44s.  Time for last 1,000,000:    1s.  Last read position: 21:15,858,348\n",
      "INFO\t2016-11-30 15:35:13\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:15\tMarkDuplicates\tRead    95,000,000 records.  Elapsed time: 00:05:46s.  Time for last 1,000,000:    1s.  Last read position: 21:46,929,653\n",
      "INFO\t2016-11-30 15:35:15\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:25\tMarkDuplicates\tRead    96,000,000 records.  Elapsed time: 00:05:56s.  Time for last 1,000,000:   10s.  Last read position: 22:32,079,256\n",
      "INFO\t2016-11-30 15:35:25\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:26\tMarkDuplicates\tRead    97,000,000 records.  Elapsed time: 00:05:57s.  Time for last 1,000,000:    1s.  Last read position: 22:47,563,146\n",
      "INFO\t2016-11-30 15:35:26\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:28\tMarkDuplicates\tRead    98,000,000 records.  Elapsed time: 00:05:59s.  Time for last 1,000,000:    1s.  Last read position: X:34,888,392\n",
      "INFO\t2016-11-30 15:35:28\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:29\tMarkDuplicates\tRead    99,000,000 records.  Elapsed time: 00:06:00s.  Time for last 1,000,000:    1s.  Last read position: X:73,062,855\n",
      "INFO\t2016-11-30 15:35:29\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:31\tMarkDuplicates\tRead   100,000,000 records.  Elapsed time: 00:06:02s.  Time for last 1,000,000:    1s.  Last read position: X:135,943,890\n",
      "INFO\t2016-11-30 15:35:31\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:35:32\tMarkDuplicates\tRead   101,000,000 records.  Elapsed time: 00:06:03s.  Time for last 1,000,000:    1s.  Last read position: MT:9,634\n",
      "INFO\t2016-11-30 15:35:32\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:36:24\tMarkDuplicates\tRead   102,000,000 records.  Elapsed time: 00:06:55s.  Time for last 1,000,000:   52s.  Last read position: GL000220.1:118,036\n",
      "INFO\t2016-11-30 15:36:24\tMarkDuplicates\tTracking 0 as yet unmatched pairs. 0 records in RAM.\n",
      "INFO\t2016-11-30 15:36:25\tMarkDuplicates\tRead 102634504 records. 0 pairs never matched.\n",
      "INFO\t2016-11-30 15:36:42\tMarkDuplicates\tAfter buildSortedReadEndLists freeMemory: 11621949616; totalMemory: 19104006144; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:36:42\tMarkDuplicates\tWill retain up to 894730240 duplicate indices before spilling to disk.\n",
      "INFO\t2016-11-30 15:36:43\tMarkDuplicates\tTraversing read pair information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:36:43\tMarkDuplicates\tTraversing fragment information and detecting duplicates.\n",
      "INFO\t2016-11-30 15:36:52\tMarkDuplicates\tSorting list of duplicate records.\n",
      "INFO\t2016-11-30 15:36:56\tMarkDuplicates\tAfter generateDuplicateIndexes freeMemory: 18287329944; totalMemory: 25633488896; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:36:56\tMarkDuplicates\tMarking 61264506 records as duplicates.\n",
      "INFO\t2016-11-30 15:36:56\tMarkDuplicates\tFound 0 optical duplicate clusters.\n",
      "INFO\t2016-11-30 15:36:56\tMarkDuplicates\tReads are assumed to be ordered by: coordinate\n",
      "INFO\t2016-11-30 15:37:50\tMarkDuplicates\tWritten    10,000,000 records.  Elapsed time: 00:00:52s.  Time for last 10,000,000:   52s.  Last read position: 1:212,867,462\n",
      "INFO\t2016-11-30 15:38:43\tMarkDuplicates\tWritten    20,000,000 records.  Elapsed time: 00:01:46s.  Time for last 10,000,000:   53s.  Last read position: 3:128,838,067\n",
      "INFO\t2016-11-30 15:39:36\tMarkDuplicates\tWritten    30,000,000 records.  Elapsed time: 00:02:39s.  Time for last 10,000,000:   53s.  Last read position: 5:157,214,336\n",
      "INFO\t2016-11-30 15:40:30\tMarkDuplicates\tWritten    40,000,000 records.  Elapsed time: 00:03:33s.  Time for last 10,000,000:   54s.  Last read position: 7:95,217,600\n",
      "INFO\t2016-11-30 15:41:23\tMarkDuplicates\tWritten    50,000,000 records.  Elapsed time: 00:04:26s.  Time for last 10,000,000:   52s.  Last read position: 10:7,844,324\n",
      "INFO\t2016-11-30 15:42:14\tMarkDuplicates\tWritten    60,000,000 records.  Elapsed time: 00:05:17s.  Time for last 10,000,000:   51s.  Last read position: 12:80,753\n",
      "INFO\t2016-11-30 15:43:05\tMarkDuplicates\tWritten    70,000,000 records.  Elapsed time: 00:06:08s.  Time for last 10,000,000:   51s.  Last read position: 14:100,604,464\n",
      "INFO\t2016-11-30 15:43:56\tMarkDuplicates\tWritten    80,000,000 records.  Elapsed time: 00:06:59s.  Time for last 10,000,000:   50s.  Last read position: 17:4,721,854\n",
      "INFO\t2016-11-30 15:44:48\tMarkDuplicates\tWritten    90,000,000 records.  Elapsed time: 00:07:51s.  Time for last 10,000,000:   51s.  Last read position: 19:41,838,110\n",
      "INFO\t2016-11-30 15:45:39\tMarkDuplicates\tWritten   100,000,000 records.  Elapsed time: 00:08:42s.  Time for last 10,000,000:   51s.  Last read position: X:135,943,890\n",
      "INFO\t2016-11-30 15:45:52\tMarkDuplicates\tBefore output close freeMemory: 26297615280; totalMemory: 26500136960; maxMemory: 28631367680\n",
      "INFO\t2016-11-30 15:45:52\tMarkDuplicates\tAfter output close freeMemory: 25978845064; totalMemory: 26181369856; maxMemory: 28631367680\n",
      "[Wed Nov 30 15:45:52 PST 2016] picard.sam.markduplicates.MarkDuplicates done. Elapsed time: 16.39 minutes.\n",
      "Runtime.totalMemory()=26181369856\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar MarkDuplicates I=/data/scratch/iholtman/human_microglia_genotyping/step5/P15rg_added_sorted.bam O= /data/scratch/iholtman/human_microglia_genotyping/step5/P15dedupped.bam CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M= /data/scratch/iholtman/human_microglia_genotyping/step5/P15output.metrics\n"
     ]
    }
   ],
   "source": [
    "##Step 5: Markduplicates! \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "PICARD=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar\" \n",
    "\n",
    "####\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = PICARD + \" MarkDuplicates I=\" + step5_dir + \"P\" + str(row['patient']) + \"rg_added_sorted.bam \" + \\\n",
    "    \"O= \" + step5_dir + \"P\" + str(row['patient']) + \"dedupped.bam \" + \"CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT \" + \\\n",
    "    \"M= \" + step5_dir + \"P\" + str(row['patient']) + \"output.metrics\" \n",
    "    \n",
    "\n",
    "    print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    #print(\"\\n\")\n",
    "     \n",
    "##\n",
    "##Original \n",
    "##java -jar picard.jar MarkDuplicates I=rg_added_sorted.bam O=dedupped.bam  CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M=output.metrics "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P16dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P19dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P21dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P21split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P2dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P3dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P6dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P15dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P17dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P5dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P7dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P8dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###split exons \n",
    "##Question: Is this function referring to the original fasta-file?? \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "step6_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step6/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "    \" -I \" + step5_dir + \"P\" + str(row['patient']) + \"dedupped.bam\" \\\n",
    "    \" -o \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \"\n",
    "\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "    \n",
    "###Original \n",
    "##java -jar GenomeAnalysisTK.jar -T SplitNCigarReads -R ref.fasta -I dedupped.bam -o split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n"
     ]
    }
   ],
   "source": [
    "###Variant calling \n",
    "step7_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step7/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "    \" -I \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0\" \\\n",
    "    \" -o \" + step7_dir + \"P\" + str(row['patient']) + \"_output.vcf\"\n",
    "\n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "\n",
    "\n",
    "##GVCF\n",
    "#java -jar GenomeAnalysisTK.jar \\\n",
    "#    -R reference.fasta \\\n",
    "#     -T HaplotypeCaller \\\n",
    "#     -I sample1.bam \\\n",
    "#     [--dbsnp dbSNP.vcf] \\\n",
    "#     -stand_call_conf 20 \\\n",
    "#     -stand_emit_conf 20 \\\n",
    "#     -o output.raw.snps.indels.vcf\n",
    " \n",
    "    \n",
    "##original \n",
    "##java -jar GenomeAnalysisTK.jar -T HaplotypeCaller -R ref.fasta -I input.bam -dontUseSoftClippedBases \n",
    "#-stand_call_conf 20.0 -stand_emit_conf 20.0 -o output.vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n",
      "INFO  18:05:48,122 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  18:05:48,124 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  18:05:48,124 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  18:05:48,124 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  18:05:48,124 HelpFormatter - [Sat Dec 03 18:05:48 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  18:05:48,124 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  18:05:48,127 HelpFormatter - Program Args: -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 --emitRefConfidence GVCF -o /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf \n",
      "INFO  18:05:48,137 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  18:05:48,137 HelpFormatter - Date/Time: 2016/12/03 18:05:48 \n",
      "INFO  18:05:48,138 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  18:05:48,138 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  18:05:48,161 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  18:05:48,275 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 500 \n",
      "INFO  18:05:48,281 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  18:05:48,307 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.02 \n",
      "INFO  18:05:48,317 HCMappingQualityFilter - Filtering out reads with MAPQ < 20 \n",
      "INFO  18:05:48,373 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  18:05:48,499 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  18:05:48,499 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  18:05:48,500 ProgressMeter -                 |      processed |    time |         per 1M |           |   total | remaining \n",
      "INFO  18:05:48,500 ProgressMeter -        Location | active regions | elapsed | active regions | completed | runtime |   runtime \n",
      "INFO  18:05:48,501 HaplotypeCaller - Standard Emitting and Calling confidence set to 0.0 for reference-model confidence output \n",
      "INFO  18:05:48,501 HaplotypeCaller - All sites annotated with PLs forced to true for reference-model confidence output \n",
      "WARN  18:05:48,538 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "INFO  18:05:48,592 HaplotypeCaller - Using global mismapping rate of 45 => -4.5 in log10 likelihood units \n",
      "Using AVX accelerated implementation of PairHMM\n",
      "INFO  18:05:49,750 VectorLoglessPairHMM - libVectorLoglessPairHMM unpacked successfully from GATK jar file \n",
      "INFO  18:05:49,750 VectorLoglessPairHMM - Using vectorized implementation of PairHMM \n",
      "WARN  18:05:49,795 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper \n",
      "INFO  18:06:18,505 ProgressMeter -      1:12808374              0.0    30.0 s           49.6 w        0.4%     2.0 h       2.0 h \n",
      "WARN  18:06:23,702 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "WARN  18:06:23,703 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "INFO  18:06:48,507 ProgressMeter -      1:27075394              0.0    60.0 s           99.2 w        0.9%   114.6 m     113.6 m \n",
      "INFO  18:07:18,509 ProgressMeter -      1:41172211              0.0    90.0 s          148.8 w        1.3%   113.0 m     111.5 m \n",
      "INFO  18:07:48,510 ProgressMeter -      1:61018363              0.0   120.0 s          198.4 w        2.0%   101.7 m      99.7 m \n",
      "INFO  18:08:18,512 ProgressMeter -      1:87328753              0.0     2.5 m          248.0 w        2.8%    88.8 m      86.3 m \n",
      "INFO  18:08:48,513 ProgressMeter -     1:110219074              0.0     3.0 m          297.6 w        3.6%    84.4 m      81.4 m \n",
      "INFO  18:09:18,515 ProgressMeter -     1:134216672              0.0     3.5 m          347.2 w        4.3%    80.9 m      77.4 m \n",
      "INFO  18:09:48,517 ProgressMeter -     1:154807385              0.0     4.0 m          396.9 w        5.0%    80.1 m      76.1 m \n",
      "INFO  18:10:18,518 ProgressMeter -     1:173449157              0.0     4.5 m          446.5 w        5.6%    80.5 m      76.0 m \n",
      "INFO  18:10:48,519 ProgressMeter -     1:198774669              0.0     5.0 m          496.1 w        6.4%    78.0 m      73.0 m \n",
      "INFO  18:11:18,521 ProgressMeter -     1:221795260              0.0     5.5 m          545.7 w        7.2%    76.9 m      71.4 m \n",
      "INFO  18:11:48,522 ProgressMeter -     1:240463649              0.0     6.0 m          595.3 w        7.8%    77.4 m      71.4 m \n",
      "INFO  18:12:18,524 ProgressMeter -      2:15642093     2.49250621E8     6.5 m            1.0 s        8.5%    76.1 m      69.6 m \n",
      "INFO  18:12:48,525 ProgressMeter -      2:37340558     2.49250621E8     7.0 m            1.0 s        9.2%    75.8 m      68.8 m \n",
      "INFO  18:13:18,526 ProgressMeter -      2:58450602     2.49250621E8     7.5 m            1.0 s        9.9%    75.6 m      68.1 m \n",
      "INFO  18:13:48,527 ProgressMeter -      2:76583056     2.49250621E8     8.0 m            1.0 s       10.5%    76.2 m      68.2 m \n",
      "INFO  18:14:18,529 ProgressMeter -     2:102029246     2.49250621E8     8.5 m            2.0 s       11.3%    75.1 m      66.6 m \n",
      "INFO  18:14:48,530 ProgressMeter -     2:128096761     2.49250621E8     9.0 m            2.0 s       12.2%    74.0 m      65.0 m \n",
      "INFO  18:15:18,532 ProgressMeter -     2:152862389     2.49250621E8     9.5 m            2.0 s       13.0%    73.3 m      63.8 m \n",
      "INFO  18:15:48,534 ProgressMeter -     2:178105255     2.49250621E8    10.0 m            2.0 s       13.8%    72.6 m      62.6 m \n",
      "INFO  18:16:18,546 ProgressMeter -     2:202373983     2.49250621E8    10.5 m            2.0 s       14.6%    72.1 m      61.6 m \n",
      "INFO  18:16:48,547 ProgressMeter -     2:225655472     2.49250621E8    11.0 m            2.0 s       15.3%    71.8 m      60.8 m \n",
      "INFO  18:17:18,549 ProgressMeter -       3:3199249     4.92449994E8    11.5 m            1.0 s       16.0%    72.0 m      60.5 m \n",
      "INFO  18:17:48,551 ProgressMeter -      3:25003321     4.92449994E8    12.0 m            1.0 s       16.7%    71.9 m      59.9 m \n",
      "INFO  18:18:18,552 ProgressMeter -      3:46431619     4.92449994E8    12.5 m            1.0 s       17.4%    72.0 m      59.5 m \n",
      "INFO  18:18:48,554 ProgressMeter -      3:63296966     4.92449994E8    13.0 m            1.0 s       17.9%    72.6 m      59.6 m \n",
      "INFO  18:19:18,556 ProgressMeter -      3:96260655     4.92449994E8    13.5 m            1.0 s       19.0%    71.1 m      57.6 m \n",
      "INFO  18:19:48,558 ProgressMeter -     3:122235588     4.92449994E8    14.0 m            1.0 s       19.8%    70.6 m      56.6 m \n",
      "INFO  18:20:18,560 ProgressMeter -     3:142405476     4.92449994E8    14.5 m            1.0 s       20.5%    70.8 m      56.3 m \n",
      "INFO  18:20:48,562 ProgressMeter -     3:168914105     4.92449994E8    15.0 m            1.0 s       21.3%    70.4 m      55.4 m \n",
      "INFO  18:21:18,564 ProgressMeter -     3:189790961     4.92449994E8    15.5 m            1.0 s       22.0%    70.5 m      55.0 m \n",
      "INFO  18:21:48,565 ProgressMeter -       4:7038495     6.90472424E8    16.0 m            1.0 s       22.5%    71.2 m      55.2 m \n",
      "INFO  18:22:18,567 ProgressMeter -      4:35647253     6.90472424E8    16.5 m            1.0 s       23.4%    70.5 m      54.0 m \n",
      "INFO  18:22:48,569 ProgressMeter -      4:57311604     6.90472424E8    17.0 m            1.0 s       24.1%    70.5 m      53.5 m \n",
      "INFO  18:23:18,572 ProgressMeter -      4:84526132     6.90472424E8    17.5 m            1.0 s       25.0%    70.0 m      52.5 m \n",
      "INFO  18:23:48,574 ProgressMeter -     4:106697514     6.90472424E8    18.0 m            1.0 s       25.7%    70.0 m      52.0 m \n",
      "INFO  18:24:18,576 ProgressMeter -     4:135849594     6.90472424E8    18.5 m            1.0 s       26.6%    69.4 m      50.9 m \n",
      "INFO  18:24:48,579 ProgressMeter -     4:161074537     6.90472424E8    19.0 m            1.0 s       27.5%    69.2 m      50.2 m \n",
      "INFO  18:25:18,580 ProgressMeter -     4:191154276     6.90472424E8    19.5 m            1.0 s       28.4%    68.6 m      49.1 m \n",
      "INFO  18:25:48,582 ProgressMeter -      5:31570470       8.816267E8    20.0 m            1.0 s       29.4%    67.9 m      47.9 m \n",
      "INFO  18:26:18,584 ProgressMeter -      5:56012935       8.816267E8    20.5 m            1.0 s       30.2%    67.8 m      47.3 m \n",
      "INFO  18:26:48,585 ProgressMeter -      5:79414999       8.816267E8    21.0 m            1.0 s       31.0%    67.8 m      46.8 m \n",
      "INFO  18:27:18,587 ProgressMeter -     5:107403912       8.816267E8    21.5 m            1.0 s       31.9%    67.4 m      45.9 m \n",
      "INFO  18:27:48,589 ProgressMeter -     5:132437098       8.816267E8    22.0 m            1.0 s       32.7%    67.3 m      45.3 m \n",
      "INFO  18:28:18,591 ProgressMeter -     5:151120324       8.816267E8    22.5 m            1.0 s       33.3%    67.6 m      45.1 m \n",
      "INFO  18:28:48,593 ProgressMeter -     5:175982021       8.816267E8    23.0 m            1.0 s       34.1%    67.5 m      44.5 m \n",
      "INFO  18:29:18,594 ProgressMeter -      6:10615631     1.06254196E9    23.5 m            1.0 s       34.6%    67.9 m      44.4 m \n",
      "INFO  18:29:48,597 ProgressMeter -      6:31168843     1.06254196E9    24.0 m            1.0 s       35.3%    68.1 m      44.1 m \n",
      "INFO  18:30:18,599 ProgressMeter -      6:35434327     1.06254196E9    24.5 m            1.0 s       35.4%    69.2 m      44.7 m \n",
      "INFO  18:30:48,600 ProgressMeter -      6:55537929     1.06254196E9    25.0 m            1.0 s       36.0%    69.4 m      44.4 m \n",
      "INFO  18:31:18,602 ProgressMeter -      6:83823533     1.06254196E9    25.5 m            1.0 s       37.0%    69.0 m      43.5 m \n",
      "INFO  18:31:48,604 ProgressMeter -     6:109572431     1.06254196E9    26.0 m            1.0 s       37.8%    68.8 m      42.8 m \n",
      "INFO  18:32:18,606 ProgressMeter -     6:134391146     1.06254196E9    26.5 m            1.0 s       38.6%    68.7 m      42.2 m \n",
      "INFO  18:32:48,608 ProgressMeter -     6:157337625     1.06254196E9    27.0 m            1.0 s       39.3%    68.7 m      41.7 m \n",
      "INFO  18:33:18,609 ProgressMeter -       7:4506166    1.233657027E9    27.5 m            1.0 s       39.9%    68.9 m      41.4 m \n",
      "INFO  18:33:48,610 ProgressMeter -      7:26490844    1.233657027E9    28.0 m            1.0 s       40.6%    68.9 m      40.9 m \n",
      "INFO  18:34:18,612 ProgressMeter -      7:44584894    1.233657027E9    28.5 m            1.0 s       41.2%    69.2 m      40.7 m \n",
      "INFO  18:34:48,613 ProgressMeter -      7:69498736    1.233657027E9    29.0 m            1.0 s       42.0%    69.0 m      40.0 m \n",
      "INFO  18:35:18,615 ProgressMeter -      7:94046337    1.233657027E9    29.5 m            1.0 s       42.8%    68.9 m      39.4 m \n",
      "INFO  18:35:48,616 ProgressMeter -     7:111763821    1.233657027E9    30.0 m            1.0 s       43.4%    69.2 m      39.2 m \n",
      "INFO  18:36:18,618 ProgressMeter -     7:134839039    1.233657027E9    30.5 m            1.0 s       44.1%    69.1 m      38.6 m \n",
      "INFO  18:36:48,620 ProgressMeter -     7:155222529    1.233657027E9    31.0 m            1.0 s       44.8%    69.2 m      38.2 m \n",
      "INFO  18:37:18,630 ProgressMeter -      8:17777722     1.39279569E9    31.5 m            1.0 s       45.5%    69.3 m      37.8 m \n",
      "INFO  18:37:48,631 ProgressMeter -      8:38830922     1.39279569E9    32.0 m            1.0 s       46.2%    69.3 m      37.3 m \n",
      "INFO  18:38:18,633 ProgressMeter -      8:65258352     1.39279569E9    32.5 m            1.0 s       47.0%    69.1 m      36.6 m \n",
      "INFO  18:38:48,634 ProgressMeter -      8:94816998     1.39279569E9    33.0 m            1.0 s       48.0%    68.8 m      35.8 m \n",
      "INFO  18:39:18,648 ProgressMeter -     8:121397191     1.39279569E9    33.5 m            1.0 s       48.8%    68.6 m      35.1 m \n",
      "INFO  18:39:48,650 ProgressMeter -     8:143726062     1.39279569E9    34.0 m            1.0 s       49.5%    68.6 m      34.6 m \n",
      "INFO  18:40:18,651 ProgressMeter -      9:18898940    1.539159712E9    34.5 m            1.0 s       50.2%    68.7 m      34.2 m \n",
      "INFO  18:40:48,653 ProgressMeter -      9:45962026    1.539159712E9    35.0 m            1.0 s       51.1%    68.5 m      33.5 m \n",
      "INFO  18:41:18,654 ProgressMeter -      9:75163846    1.539159712E9    35.5 m            1.0 s       52.0%    68.2 m      32.7 m \n",
      "INFO  18:41:48,655 ProgressMeter -      9:98767945    1.539159712E9    36.0 m            1.0 s       52.8%    68.2 m      32.2 m \n",
      "INFO  18:42:18,657 ProgressMeter -     9:121384754    1.539159712E9    36.5 m            1.0 s       53.5%    68.2 m      31.7 m \n",
      "INFO  18:42:48,658 ProgressMeter -     9:134709040    1.539159712E9    37.0 m            1.0 s       54.0%    68.6 m      31.6 m \n",
      "INFO  18:43:18,705 ProgressMeter -      10:8429316    1.680373143E9    37.5 m            1.0 s       54.4%    68.9 m      31.4 m \n",
      "INFO  18:43:48,707 ProgressMeter -     10:24705477    1.680373143E9    38.0 m            1.0 s       55.0%    69.1 m      31.1 m \n",
      "INFO  18:44:18,708 ProgressMeter -     10:46614330    1.680373143E9    38.5 m            1.0 s       55.7%    69.1 m      30.6 m \n",
      "INFO  18:44:48,709 ProgressMeter -     10:71270235    1.680373143E9    39.0 m            1.0 s       56.5%    69.1 m      30.1 m \n",
      "INFO  18:45:18,711 ProgressMeter -     10:88964868    1.680373143E9    39.5 m            1.0 s       57.0%    69.2 m      29.7 m \n",
      "INFO  18:45:48,712 ProgressMeter -    10:104513787    1.680373143E9    40.0 m            1.0 s       57.5%    69.5 m      29.5 m \n",
      "INFO  18:46:18,713 ProgressMeter -    10:127126403    1.680373143E9    40.5 m            1.0 s       58.3%    69.5 m      29.0 m \n",
      "INFO  18:46:48,715 ProgressMeter -      11:9966181     1.81590789E9    41.0 m            1.0 s       58.9%    69.7 m      28.7 m \n",
      "INFO  18:47:18,716 ProgressMeter -     11:35010445     1.81590789E9    41.5 m            1.0 s       59.7%    69.5 m      28.0 m \n",
      "INFO  18:47:48,717 ProgressMeter -     11:60073069     1.81590789E9    42.0 m            1.0 s       60.5%    69.4 m      27.4 m \n",
      "INFO  18:48:18,718 ProgressMeter -     11:72625760     1.81590789E9    42.5 m            1.0 s       60.9%    69.8 m      27.3 m \n",
      "INFO  18:48:48,721 ProgressMeter -     11:92059755     1.81590789E9    43.0 m            1.0 s       61.5%    69.9 m      26.9 m \n",
      "INFO  18:49:18,723 ProgressMeter -    11:116959959     1.81590789E9    43.5 m            1.0 s       62.3%    69.8 m      26.3 m \n",
      "INFO  18:49:48,725 ProgressMeter -       12:156440    1.950914406E9    44.0 m            1.0 s       62.9%    70.0 m      26.0 m \n",
      "INFO  18:50:18,726 ProgressMeter -     12:17899259    1.950914406E9    44.5 m            1.0 s       63.5%    70.1 m      25.6 m \n",
      "INFO  18:50:48,728 ProgressMeter -     12:42480367    1.950914406E9    45.0 m            1.0 s       64.3%    70.0 m      25.0 m \n",
      "INFO  18:51:18,730 ProgressMeter -     12:57574546    1.950914406E9    45.5 m            1.0 s       64.8%    70.3 m      24.8 m \n",
      "INFO  18:51:48,731 ProgressMeter -     12:78543368    1.950914406E9    46.0 m            1.0 s       65.4%    70.3 m      24.3 m \n",
      "INFO  18:52:18,733 ProgressMeter -    12:104939703    1.950914406E9    46.5 m            1.0 s       66.3%    70.2 m      23.7 m \n",
      "INFO  18:52:48,735 ProgressMeter -    12:121571219    1.950914406E9    47.0 m            1.0 s       66.8%    70.3 m      23.3 m \n",
      "INFO  18:53:18,806 ProgressMeter -     13:10966001    2.084766301E9    47.5 m            1.0 s       67.6%    70.3 m      22.8 m \n",
      "INFO  18:53:48,808 ProgressMeter -     13:35236248    2.084766301E9    48.0 m            1.0 s       68.3%    70.2 m      22.2 m \n",
      "INFO  18:54:18,809 ProgressMeter -     13:54067036    2.084766301E9    48.5 m            1.0 s       69.0%    70.3 m      21.8 m \n",
      "INFO  18:54:48,811 ProgressMeter -     13:86695306    2.084766301E9    49.0 m            1.0 s       70.0%    70.0 m      21.0 m \n",
      "INFO  18:55:18,814 ProgressMeter -    13:112407786    2.084766301E9    49.5 m            1.0 s       70.8%    69.9 m      20.4 m \n",
      "INFO  18:55:48,816 ProgressMeter -     14:23562797    2.199936179E9    50.0 m            1.0 s       71.7%    69.8 m      19.8 m \n",
      "INFO  18:56:18,818 ProgressMeter -     14:50793309    2.199936179E9    50.5 m            1.0 s       72.6%    69.6 m      19.1 m \n",
      "INFO  18:56:48,819 ProgressMeter -     14:71040609    2.199936179E9    51.0 m            1.0 s       73.2%    69.7 m      18.7 m \n",
      "INFO  18:57:18,821 ProgressMeter -     14:90582887    2.199936179E9    51.5 m            1.0 s       73.8%    69.7 m      18.2 m \n",
      "INFO  18:57:48,822 ProgressMeter -      15:2761601    2.307285719E9    52.0 m            1.0 s       74.5%    69.8 m      17.8 m \n",
      "INFO  18:58:18,823 ProgressMeter -     15:34618289    2.307285719E9    52.5 m            1.0 s       75.5%    69.5 m      17.0 m \n",
      "INFO  18:58:48,825 ProgressMeter -     15:52931918    2.307285719E9    53.0 m            1.0 s       76.1%    69.7 m      16.7 m \n",
      "INFO  18:59:18,826 ProgressMeter -     15:70386554    2.307285719E9    53.5 m            1.0 s       76.7%    69.8 m      16.3 m \n",
      "INFO  18:59:48,827 ProgressMeter -     15:90347155    2.307285719E9    54.0 m            1.0 s       77.3%    69.9 m      15.9 m \n",
      "INFO  19:00:18,828 ProgressMeter -      16:4319136    2.409817111E9    54.5 m            1.0 s       77.8%    70.0 m      15.5 m \n",
      "INFO  19:00:48,830 ProgressMeter -     16:23508028    2.409817111E9    55.0 m            1.0 s       78.4%    70.1 m      15.1 m \n",
      "INFO  19:01:18,831 ProgressMeter -     16:46020519    2.409817111E9    55.5 m            1.0 s       79.2%    70.1 m      14.6 m \n",
      "INFO  19:01:48,832 ProgressMeter -     16:68235188    2.409817111E9    56.0 m            1.0 s       79.9%    70.1 m      14.1 m \n",
      "INFO  19:02:18,833 ProgressMeter -     16:85198134    2.409817111E9    56.5 m            1.0 s       80.4%    70.2 m      13.7 m \n",
      "INFO  19:02:48,834 ProgressMeter -      17:4930925    2.500171864E9    57.0 m            1.0 s       80.8%    70.6 m      13.6 m \n",
      "INFO  19:03:18,836 ProgressMeter -     17:21699639    2.500171864E9    57.5 m            1.0 s       81.3%    70.7 m      13.2 m \n",
      "INFO  19:03:48,837 ProgressMeter -     17:37570952    2.500171864E9    58.0 m            1.0 s       81.8%    70.9 m      12.9 m \n",
      "INFO  19:04:18,839 ProgressMeter -     17:52688423    2.500171864E9    58.5 m            1.0 s       82.3%    71.1 m      12.6 m \n",
      "INFO  19:04:48,840 ProgressMeter -     17:71181784    2.500171864E9    59.0 m            1.0 s       82.9%    71.2 m      12.2 m \n",
      "INFO  19:05:18,841 ProgressMeter -      18:2145294    2.581367074E9    59.5 m            1.0 s       83.3%    71.4 m      11.9 m \n",
      "INFO  19:05:48,843 ProgressMeter -     18:24948543    2.581367074E9    60.0 m            1.0 s       84.0%    71.4 m      11.4 m \n",
      "INFO  19:06:18,844 ProgressMeter -     18:54286671    2.581367074E9    60.5 m            1.0 s       85.0%    71.2 m      10.7 m \n",
      "INFO  19:06:48,845 ProgressMeter -      19:1117625    2.659444322E9    61.0 m            1.0 s       85.8%    71.1 m      10.1 m \n",
      "INFO  19:07:18,847 ProgressMeter -     19:10474478    2.659444322E9    61.5 m            1.0 s       86.1%    71.4 m       9.9 m \n",
      "INFO  19:07:48,848 ProgressMeter -     19:20716972    2.659444322E9    62.0 m            1.0 s       86.4%    71.8 m       9.8 m \n",
      "INFO  19:08:18,850 ProgressMeter -     19:39909583    2.659444322E9    62.5 m            1.0 s       87.0%    71.8 m       9.3 m \n",
      "INFO  19:08:48,851 ProgressMeter -     19:51949583    2.659444322E9    63.0 m            1.0 s       87.4%    72.1 m       9.1 m \n",
      "INFO  19:09:18,853 ProgressMeter -      20:5666566    2.718573305E9    63.5 m            1.0 s       87.8%    72.3 m       8.8 m \n",
      "INFO  19:09:48,854 ProgressMeter -     20:30768691    2.718573305E9    64.0 m            1.0 s       88.6%    72.2 m       8.2 m \n",
      "INFO  19:10:18,855 ProgressMeter -     20:47727595    2.718573305E9    64.5 m            1.0 s       89.2%    72.3 m       7.8 m \n",
      "INFO  19:10:48,857 ProgressMeter -      21:9538232    2.781598825E9    65.0 m            1.0 s       90.0%    72.2 m       7.2 m \n",
      "INFO  19:11:18,858 ProgressMeter -     21:36722581    2.781598825E9    65.5 m            1.0 s       90.9%    72.1 m       6.6 m \n",
      "INFO  19:11:48,859 ProgressMeter -     22:10088511     2.82972872E9    66.0 m            1.0 s       91.6%    72.1 m       6.1 m \n",
      "INFO  19:12:18,860 ProgressMeter -     22:29948801     2.82972872E9    66.5 m            1.0 s       92.2%    72.1 m       5.6 m \n",
      "INFO  19:12:58,861 ProgressMeter -     22:50179358     2.82972872E9    67.2 m            1.0 s       92.8%    72.3 m       5.2 m \n",
      "INFO  19:13:28,862 ProgressMeter -      X:20218034    2.881033286E9    67.7 m            1.0 s       93.5%    72.3 m       4.7 m \n",
      "INFO  19:13:58,864 ProgressMeter -      X:49011289    2.881033286E9    68.2 m            1.0 s       94.5%    72.2 m       4.0 m \n",
      "INFO  19:14:28,865 ProgressMeter -      X:74645767    2.881033286E9    68.7 m            1.0 s       95.3%    72.1 m       3.4 m \n",
      "INFO  19:14:58,866 ProgressMeter -     X:106886148    2.881033286E9    69.2 m            1.0 s       96.3%    71.8 m       2.6 m \n",
      "INFO  19:15:28,868 ProgressMeter -     X:134442015    2.881033286E9    69.7 m            1.0 s       97.2%    71.7 m     119.0 s \n",
      "INFO  19:15:58,869 ProgressMeter -       Y:9827893    3.036303846E9    70.2 m            1.0 s       98.2%    71.4 m      76.0 s \n",
      "INFO  19:16:28,870 ProgressMeter -      Y:50004903    3.036303846E9    70.7 m            1.0 s       99.5%    71.0 m      21.0 s \n",
      "INFO  19:16:44,739 VectorLoglessPairHMM - Time spent in setup for JNI call : 0.140808086 \n",
      "INFO  19:16:44,739 PairHMM - Total compute time in PairHMM computeLikelihoods() : 17.217465988 \n",
      "INFO  19:16:44,739 HaplotypeCaller - Ran local assembly on 11114648 active regions \n",
      "INFO  19:16:44,755 ProgressMeter -            done    3.101804739E9    70.9 m            1.0 s      100.0%    70.9 m       0.0 s \n",
      "INFO  19:16:44,756 ProgressMeter - Total runtime 4256.26 secs, 70.94 min, 1.18 hours \n",
      "INFO  19:16:44,756 MicroScheduler - 26017422 reads were filtered out during the traversal out of approximately 36601691 total reads (71.08%) \n",
      "INFO  19:16:44,756 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  19:16:44,756 MicroScheduler -   -> 18931737 reads (51.72% of total) failing DuplicateReadFilter \n",
      "INFO  19:16:44,756 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  19:16:44,756 MicroScheduler -   -> 7085685 reads (19.36% of total) failing HCMappingQualityFilter \n",
      "INFO  19:16:44,757 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  19:16:44,757 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  19:16:44,757 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  19:16:44,757 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were 4 WARN messages, the first 4 are repeated below.\n",
      "WARN  18:05:48,538 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "WARN  18:05:49,795 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper \n",
      "WARN  18:06:23,702 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "WARN  18:06:23,703 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz sample P19\n",
      "INFO  19:16:46,051 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  19:16:46,052 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  19:16:46,052 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  19:16:46,053 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  19:16:46,053 HelpFormatter - [Sat Dec 03 19:16:46 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  19:16:46,053 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  19:16:46,055 HelpFormatter - Program Args: -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 --emitRefConfidence GVCF -o /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf \n",
      "INFO  19:16:46,065 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  19:16:46,065 HelpFormatter - Date/Time: 2016/12/03 19:16:46 \n",
      "INFO  19:16:46,066 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  19:16:46,066 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  19:16:46,088 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  19:16:46,202 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 500 \n",
      "INFO  19:16:46,209 SAMDataSource$SAMReaders - Initializing SAMRecords in serial \n",
      "INFO  19:16:46,233 SAMDataSource$SAMReaders - Done initializing BAM readers: total time 0.02 \n",
      "INFO  19:16:46,244 HCMappingQualityFilter - Filtering out reads with MAPQ < 20 \n",
      "INFO  19:16:46,302 GenomeAnalysisEngine - Preparing for traversal over 1 BAM files \n",
      "INFO  19:16:46,432 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  19:16:46,432 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  19:16:46,433 ProgressMeter -                 |      processed |    time |         per 1M |           |   total | remaining \n",
      "INFO  19:16:46,433 ProgressMeter -        Location | active regions | elapsed | active regions | completed | runtime |   runtime \n",
      "INFO  19:16:46,434 HaplotypeCaller - Standard Emitting and Calling confidence set to 0.0 for reference-model confidence output \n",
      "INFO  19:16:46,434 HaplotypeCaller - All sites annotated with PLs forced to true for reference-model confidence output \n",
      "WARN  19:16:46,474 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "INFO  19:16:46,525 HaplotypeCaller - Using global mismapping rate of 45 => -4.5 in log10 likelihood units \n",
      "Using AVX accelerated implementation of PairHMM\n",
      "INFO  19:16:47,733 VectorLoglessPairHMM - libVectorLoglessPairHMM unpacked successfully from GATK jar file \n",
      "INFO  19:16:47,733 VectorLoglessPairHMM - Using vectorized implementation of PairHMM \n",
      "WARN  19:16:47,784 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper \n",
      "INFO  19:17:16,438 ProgressMeter -      1:11889371              0.0    30.0 s           49.6 w        0.4%     2.2 h       2.2 h \n",
      "INFO  19:17:46,440 ProgressMeter -      1:23627073              0.0    60.0 s           99.2 w        0.8%     2.2 h       2.2 h \n",
      "WARN  19:17:49,165 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "WARN  19:17:49,165 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "INFO  19:18:16,441 ProgressMeter -      1:36672876              0.0    90.0 s          148.8 w        1.2%     2.1 h       2.1 h \n",
      "INFO  19:18:46,443 ProgressMeter -      1:50741501              0.0   120.0 s          198.4 w        1.6%     2.0 h       2.0 h \n",
      "INFO  19:19:16,444 ProgressMeter -      1:74110454              0.0     2.5 m          248.0 w        2.4%   104.6 m     102.1 m \n",
      "INFO  19:19:46,446 ProgressMeter -      1:95445316              0.0     3.0 m          297.6 w        3.1%    97.5 m      94.5 m \n",
      "INFO  19:20:16,447 ProgressMeter -     1:115257556              0.0     3.5 m          347.2 w        3.7%    94.2 m      90.7 m \n",
      "INFO  19:20:46,449 ProgressMeter -     1:145839787              0.0     4.0 m          396.9 w        4.7%    85.1 m      81.1 m \n",
      "INFO  19:21:16,450 ProgressMeter -     1:159422700              0.0     4.5 m          446.5 w        5.1%    87.6 m      83.1 m \n",
      "INFO  19:21:46,451 ProgressMeter -     1:178573142              0.0     5.0 m          496.1 w        5.8%    86.8 m      81.8 m \n",
      "INFO  19:22:16,453 ProgressMeter -     1:202501100              0.0     5.5 m          545.7 w        6.5%    84.2 m      78.7 m \n",
      "INFO  19:22:46,456 ProgressMeter -     1:222934468              0.0     6.0 m          595.3 w        7.2%    83.5 m      77.5 m \n",
      "INFO  19:23:16,457 ProgressMeter -     1:237030775              0.0     6.5 m          644.9 w        7.6%    85.1 m      78.6 m \n",
      "INFO  19:23:46,459 ProgressMeter -       2:9193593     2.49250621E8     7.0 m            1.0 s        8.3%    84.0 m      77.0 m \n",
      "INFO  19:24:16,460 ProgressMeter -      2:28958197     2.49250621E8     7.5 m            1.0 s        9.0%    83.6 m      76.1 m \n",
      "INFO  19:24:46,461 ProgressMeter -      2:48033077     2.49250621E8     8.0 m            1.0 s        9.6%    83.5 m      75.5 m \n",
      "INFO  19:25:16,462 ProgressMeter -      2:66769186     2.49250621E8     8.5 m            2.0 s       10.2%    83.4 m      74.9 m \n",
      "INFO  19:25:46,463 ProgressMeter -      2:87161258     2.49250621E8     9.0 m            2.0 s       10.8%    83.0 m      74.0 m \n",
      "INFO  19:26:16,465 ProgressMeter -     2:109806556     2.49250621E8     9.5 m            2.0 s       11.6%    82.1 m      72.6 m \n",
      "INFO  19:26:46,466 ProgressMeter -     2:130915653     2.49250621E8    10.0 m            2.0 s       12.3%    81.6 m      71.6 m \n",
      "INFO  19:27:16,468 ProgressMeter -     2:153920312     2.49250621E8    10.5 m            2.0 s       13.0%    80.8 m      70.3 m \n",
      "INFO  19:27:46,469 ProgressMeter -     2:178152014     2.49250621E8    11.0 m            2.0 s       13.8%    79.8 m      68.8 m \n",
      "INFO  19:28:16,470 ProgressMeter -     2:201678356     2.49250621E8    11.5 m            2.0 s       14.5%    79.1 m      67.6 m \n",
      "INFO  19:28:46,471 ProgressMeter -     2:222161004     2.49250621E8    12.0 m            2.0 s       15.2%    79.0 m      67.0 m \n",
      "INFO  19:29:16,472 ProgressMeter -     2:242027103     2.49250621E8    12.5 m            3.0 s       15.8%    78.9 m      66.4 m \n",
      "INFO  19:29:46,474 ProgressMeter -      3:17119576     4.92449994E8    13.0 m            1.0 s       16.4%    79.1 m      66.1 m \n",
      "INFO  19:30:16,475 ProgressMeter -      3:39274319     4.92449994E8    13.5 m            1.0 s       17.1%    78.8 m      65.3 m \n",
      "INFO  19:30:46,476 ProgressMeter -      3:52554218     4.92449994E8    14.0 m            1.0 s       17.6%    79.7 m      65.7 m \n",
      "INFO  19:31:16,477 ProgressMeter -      3:76083835     4.92449994E8    14.5 m            1.0 s       18.3%    79.1 m      64.6 m \n",
      "INFO  19:31:46,479 ProgressMeter -     3:108619491     4.92449994E8    15.0 m            1.0 s       19.4%    77.4 m      62.4 m \n",
      "INFO  19:32:16,480 ProgressMeter -     3:128805335     4.92449994E8    15.5 m            1.0 s       20.0%    77.4 m      61.9 m \n",
      "INFO  19:32:46,482 ProgressMeter -     3:149704510     4.92449994E8    16.0 m            1.0 s       20.7%    77.3 m      61.3 m \n",
      "INFO  19:33:16,484 ProgressMeter -     3:174163235     4.92449994E8    16.5 m            2.0 s       21.5%    76.8 m      60.3 m \n",
      "INFO  19:33:46,485 ProgressMeter -     3:193237786     4.92449994E8    17.0 m            2.0 s       22.1%    76.9 m      59.9 m \n",
      "INFO  19:34:16,487 ProgressMeter -       4:7073681     6.90472424E8    17.5 m            1.0 s       22.5%    77.8 m      60.3 m \n",
      "INFO  19:34:46,488 ProgressMeter -      4:33039934     6.90472424E8    18.0 m            1.0 s       23.3%    77.2 m      59.2 m \n",
      "INFO  19:35:16,490 ProgressMeter -      4:54864306     6.90472424E8    18.5 m            1.0 s       24.0%    77.0 m      58.5 m \n",
      "INFO  19:35:46,492 ProgressMeter -      4:83215753     6.90472424E8    19.0 m            1.0 s       24.9%    76.2 m      57.2 m \n",
      "INFO  19:36:16,494 ProgressMeter -     4:103574889     6.90472424E8    19.5 m            1.0 s       25.6%    76.2 m      56.7 m \n",
      "INFO  19:36:46,495 ProgressMeter -     4:128908328     6.90472424E8    20.0 m            1.0 s       26.4%    75.7 m      55.7 m \n",
      "INFO  19:37:16,497 ProgressMeter -     4:155533827     6.90472424E8    20.5 m            1.0 s       27.3%    75.2 m      54.7 m \n",
      "INFO  19:37:46,498 ProgressMeter -     4:184775361     6.90472424E8    21.0 m            1.0 s       28.2%    74.4 m      53.4 m \n",
      "INFO  19:38:16,500 ProgressMeter -      5:19822156       8.816267E8    21.5 m            1.0 s       29.1%    74.0 m      52.5 m \n",
      "INFO  19:38:46,501 ProgressMeter -      5:47048990       8.816267E8    22.0 m            1.0 s       29.9%    73.5 m      51.5 m \n",
      "INFO  19:39:16,502 ProgressMeter -      5:71015120       8.816267E8    22.5 m            1.0 s       30.7%    73.3 m      50.8 m \n",
      "INFO  19:39:46,504 ProgressMeter -      5:93016183       8.816267E8    23.0 m            1.0 s       31.4%    73.2 m      50.2 m \n",
      "INFO  19:40:16,505 ProgressMeter -     5:118866689       8.816267E8    23.5 m            1.0 s       32.3%    72.9 m      49.4 m \n",
      "INFO  19:40:46,511 ProgressMeter -     5:139676805       8.816267E8    24.0 m            1.0 s       32.9%    72.9 m      48.9 m \n",
      "INFO  19:41:16,512 ProgressMeter -     5:159135571       8.816267E8    24.5 m            1.0 s       33.6%    73.0 m      48.5 m \n",
      "INFO  19:41:46,514 ProgressMeter -     5:179563676       8.816267E8    25.0 m            1.0 s       34.2%    73.1 m      48.1 m \n",
      "INFO  19:42:16,515 ProgressMeter -      6:15361057     1.06254196E9    25.5 m            1.0 s       34.8%    73.4 m      47.9 m \n",
      "INFO  19:42:46,516 ProgressMeter -      6:31318916     1.06254196E9    26.0 m            1.0 s       35.3%    73.7 m      47.7 m \n",
      "INFO  19:43:16,518 ProgressMeter -      6:41568150     1.06254196E9    26.5 m            1.0 s       35.6%    74.4 m      47.9 m \n",
      "INFO  19:43:46,519 ProgressMeter -      6:66212869     1.06254196E9    27.0 m            1.0 s       36.4%    74.2 m      47.2 m \n",
      "INFO  19:44:16,521 ProgressMeter -      6:93437318     1.06254196E9    27.5 m            1.0 s       37.3%    73.8 m      46.3 m \n",
      "INFO  19:44:46,522 ProgressMeter -     6:116167998     1.06254196E9    28.0 m            1.0 s       38.0%    73.7 m      45.7 m \n",
      "INFO  19:45:16,523 ProgressMeter -     6:139932690     1.06254196E9    28.5 m            1.0 s       38.8%    73.5 m      45.0 m \n",
      "INFO  19:45:46,525 ProgressMeter -     6:160209280     1.06254196E9    29.0 m            1.0 s       39.4%    73.6 m      44.6 m \n",
      "INFO  19:46:16,527 ProgressMeter -       7:5451706    1.233657027E9    29.5 m            1.0 s       39.9%    73.8 m      44.3 m \n",
      "INFO  19:46:46,528 ProgressMeter -      7:26757495    1.233657027E9    30.0 m            1.0 s       40.6%    73.8 m      43.8 m \n",
      "INFO  19:47:16,530 ProgressMeter -      7:44107820    1.233657027E9    30.5 m            1.0 s       41.2%    74.0 m      43.5 m \n",
      "INFO  19:47:46,531 ProgressMeter -      7:67108823    1.233657027E9    31.0 m            1.0 s       41.9%    73.9 m      42.9 m \n",
      "INFO  19:48:16,533 ProgressMeter -      7:91976011    1.233657027E9    31.5 m            1.0 s       42.7%    73.7 m      42.2 m \n",
      "INFO  19:48:46,534 ProgressMeter -     7:109368359    1.233657027E9    32.0 m            1.0 s       43.3%    73.9 m      41.9 m \n",
      "INFO  19:49:16,535 ProgressMeter -     7:130037611    1.233657027E9    32.5 m            1.0 s       44.0%    73.9 m      41.4 m \n",
      "INFO  19:49:46,537 ProgressMeter -     7:149488072    1.233657027E9    33.0 m            1.0 s       44.6%    74.0 m      41.0 m \n",
      "INFO  19:50:16,538 ProgressMeter -      8:11701641     1.39279569E9    33.5 m            1.0 s       45.3%    74.0 m      40.5 m \n",
      "INFO  19:50:46,540 ProgressMeter -      8:29138941     1.39279569E9    34.0 m            1.0 s       45.8%    74.2 m      40.2 m \n",
      "INFO  19:51:16,541 ProgressMeter -      8:53432213     1.39279569E9    34.5 m            1.0 s       46.6%    74.0 m      39.5 m \n",
      "INFO  19:51:46,542 ProgressMeter -      8:78953383     1.39279569E9    35.0 m            1.0 s       47.4%    73.8 m      38.8 m \n",
      "INFO  19:52:16,543 ProgressMeter -     8:104810316     1.39279569E9    35.5 m            1.0 s       48.3%    73.5 m      38.0 m \n",
      "INFO  19:52:46,545 ProgressMeter -     8:131195053     1.39279569E9    36.0 m            1.0 s       49.1%    73.3 m      37.3 m \n",
      "INFO  19:53:16,546 ProgressMeter -        9:683854    1.539159712E9    36.5 m            1.0 s       49.6%    73.5 m      37.0 m \n",
      "INFO  19:53:46,548 ProgressMeter -      9:26351116    1.539159712E9    37.0 m            1.0 s       50.5%    73.3 m      36.3 m \n",
      "INFO  19:54:16,549 ProgressMeter -      9:52723995    1.539159712E9    37.5 m            1.0 s       51.3%    73.1 m      35.6 m \n",
      "INFO  19:54:46,551 ProgressMeter -      9:75988920    1.539159712E9    38.0 m            1.0 s       52.1%    73.0 m      35.0 m \n",
      "INFO  19:55:16,552 ProgressMeter -      9:96403901    1.539159712E9    38.5 m            1.0 s       52.7%    73.0 m      34.5 m \n",
      "INFO  19:55:46,553 ProgressMeter -     9:116915833    1.539159712E9    39.0 m            1.0 s       53.4%    73.0 m      34.0 m \n",
      "INFO  19:56:16,555 ProgressMeter -     9:132397373    1.539159712E9    39.5 m            1.0 s       53.9%    73.3 m      33.8 m \n",
      "INFO  19:56:46,556 ProgressMeter -      10:4243673    1.680373143E9    40.0 m            1.0 s       54.3%    73.7 m      33.7 m \n",
      "INFO  19:57:16,557 ProgressMeter -     10:15092608    1.680373143E9    40.5 m            1.0 s       54.7%    74.1 m      33.6 m \n",
      "INFO  19:57:46,559 ProgressMeter -     10:32072133    1.680373143E9    41.0 m            1.0 s       55.2%    74.3 m      33.3 m \n",
      "INFO  19:58:16,560 ProgressMeter -     10:53977946    1.680373143E9    41.5 m            1.0 s       55.9%    74.2 m      32.7 m \n",
      "INFO  19:58:46,561 ProgressMeter -     10:74475876    1.680373143E9    42.0 m            1.0 s       56.6%    74.2 m      32.2 m \n",
      "INFO  19:59:16,563 ProgressMeter -     10:91723708    1.680373143E9    42.5 m            1.0 s       57.1%    74.4 m      31.9 m \n",
      "INFO  19:59:46,565 ProgressMeter -    10:104890805    1.680373143E9    43.0 m            1.0 s       57.6%    74.7 m      31.7 m \n",
      "INFO  20:00:16,566 ProgressMeter -    10:126620062    1.680373143E9    43.5 m            1.0 s       58.3%    74.7 m      31.2 m \n",
      "INFO  20:00:46,568 ProgressMeter -      11:8682232     1.81590789E9    44.0 m            1.0 s       58.8%    74.8 m      30.8 m \n",
      "INFO  20:01:16,569 ProgressMeter -     11:31466105     1.81590789E9    44.5 m            1.0 s       59.6%    74.7 m      30.2 m \n",
      "INFO  20:01:46,570 ProgressMeter -     11:53573041     1.81590789E9    45.0 m            1.0 s       60.3%    74.7 m      29.7 m \n",
      "INFO  20:02:16,572 ProgressMeter -     11:67817974     1.81590789E9    45.5 m            1.0 s       60.7%    74.9 m      29.4 m \n",
      "INFO  20:02:46,573 ProgressMeter -     11:83807332     1.81590789E9    46.0 m            1.0 s       61.2%    75.1 m      29.1 m \n",
      "INFO  20:03:16,574 ProgressMeter -    11:108015081     1.81590789E9    46.5 m            1.0 s       62.0%    75.0 m      28.5 m \n",
      "INFO  20:03:46,576 ProgressMeter -    11:126005121     1.81590789E9    47.0 m            1.0 s       62.6%    75.1 m      28.1 m \n",
      "INFO  20:04:16,577 ProgressMeter -      12:9261922    1.950914406E9    47.5 m            1.0 s       63.2%    75.2 m      27.7 m \n",
      "INFO  20:04:46,578 ProgressMeter -     12:26963699    1.950914406E9    48.0 m            1.0 s       63.8%    75.3 m      27.3 m \n",
      "INFO  20:05:16,580 ProgressMeter -     12:49535920    1.950914406E9    48.5 m            1.0 s       64.5%    75.2 m      26.7 m \n",
      "INFO  20:05:46,591 ProgressMeter -     12:63735490    1.950914406E9    49.0 m            1.0 s       65.0%    75.4 m      26.4 m \n",
      "INFO  20:06:16,594 ProgressMeter -     12:85791131    1.950914406E9    49.5 m            1.0 s       65.7%    75.4 m      25.9 m \n",
      "INFO  20:06:46,596 ProgressMeter -    12:105817897    1.950914406E9    50.0 m            1.0 s       66.3%    75.4 m      25.4 m \n",
      "INFO  20:07:16,598 ProgressMeter -    12:121306933    1.950914406E9    50.5 m            1.0 s       66.8%    75.6 m      25.1 m \n",
      "INFO  20:07:46,599 ProgressMeter -      13:4786301    2.084766301E9    51.0 m            1.0 s       67.4%    75.7 m      24.7 m \n",
      "INFO  20:08:16,600 ProgressMeter -     13:32927601    2.084766301E9    51.5 m            1.0 s       68.3%    75.4 m      23.9 m \n",
      "INFO  20:08:46,601 ProgressMeter -     13:51361594    2.084766301E9    52.0 m            1.0 s       68.9%    75.5 m      23.5 m \n",
      "INFO  20:09:16,603 ProgressMeter -     13:79892673    2.084766301E9    52.5 m            1.0 s       69.8%    75.2 m      22.7 m \n",
      "INFO  20:09:46,604 ProgressMeter -    13:108986097    2.084766301E9    53.0 m            1.0 s       70.7%    74.9 m      21.9 m \n",
      "INFO  20:10:16,606 ProgressMeter -     14:18895633    2.199936179E9    53.5 m            1.0 s       71.5%    74.8 m      21.3 m \n",
      "INFO  20:10:46,607 ProgressMeter -     14:40732618    2.199936179E9    54.0 m            1.0 s       72.2%    74.8 m      20.8 m \n",
      "INFO  20:11:16,608 ProgressMeter -     14:64320650    2.199936179E9    54.5 m            1.0 s       73.0%    74.7 m      20.2 m \n",
      "INFO  20:11:46,609 ProgressMeter -     14:80174512    2.199936179E9    55.0 m            1.0 s       73.5%    74.8 m      19.8 m \n",
      "INFO  20:12:16,611 ProgressMeter -    14:100374566    2.199936179E9    55.5 m            1.0 s       74.2%    74.8 m      19.3 m \n",
      "INFO  20:12:46,612 ProgressMeter -     15:20266993    2.307285719E9    56.0 m            1.0 s       75.0%    74.6 m      18.6 m \n",
      "INFO  20:13:16,626 ProgressMeter -     15:42322766    2.307285719E9    56.5 m            1.0 s       75.7%    74.6 m      18.1 m \n",
      "INFO  20:13:46,627 ProgressMeter -     15:59529205    2.307285719E9    57.0 m            1.0 s       76.3%    74.7 m      17.7 m \n",
      "INFO  20:14:16,628 ProgressMeter -     15:74577125    2.307285719E9    57.5 m            1.0 s       76.8%    74.9 m      17.4 m \n",
      "INFO  20:14:46,629 ProgressMeter -     15:91874078    2.307285719E9    58.0 m            1.0 s       77.3%    75.0 m      17.0 m \n",
      "INFO  20:15:16,630 ProgressMeter -      16:3795075    2.409817111E9    58.5 m            1.0 s       77.8%    75.2 m      16.7 m \n",
      "INFO  20:15:46,632 ProgressMeter -     16:21643915    2.409817111E9    59.0 m            1.0 s       78.4%    75.3 m      16.3 m \n",
      "INFO  20:16:16,633 ProgressMeter -     16:37955593    2.409817111E9    59.5 m            1.0 s       78.9%    75.4 m      15.9 m \n",
      "INFO  20:16:46,634 ProgressMeter -     16:63381311    2.409817111E9    60.0 m            1.0 s       79.7%    75.3 m      15.3 m \n",
      "INFO  20:17:16,635 ProgressMeter -     16:79229884    2.409817111E9    60.5 m            1.0 s       80.2%    75.4 m      14.9 m \n",
      "INFO  20:17:46,637 ProgressMeter -      17:1032105    2.500171864E9    61.0 m            1.0 s       80.6%    75.6 m      14.6 m \n",
      "INFO  20:18:16,638 ProgressMeter -     17:10576512    2.500171864E9    61.5 m            1.0 s       80.9%    76.0 m      14.5 m \n",
      "INFO  20:18:46,639 ProgressMeter -     17:28224277    2.500171864E9    62.0 m            1.0 s       81.5%    76.1 m      14.1 m \n",
      "INFO  20:19:16,641 ProgressMeter -     17:41231123    2.500171864E9    62.5 m            1.0 s       81.9%    76.3 m      13.8 m \n",
      "INFO  20:19:46,642 ProgressMeter -     17:56686790    2.500171864E9    63.0 m            1.0 s       82.4%    76.4 m      13.4 m \n",
      "INFO  20:20:16,644 ProgressMeter -     17:72466750    2.500171864E9    63.5 m            1.0 s       82.9%    76.6 m      13.1 m \n",
      "INFO  20:20:46,645 ProgressMeter -       18:700362    2.581367074E9    64.0 m            1.0 s       83.2%    76.9 m      12.9 m \n",
      "INFO  20:21:16,647 ProgressMeter -     18:21055898    2.581367074E9    64.5 m            1.0 s       83.9%    76.9 m      12.4 m \n",
      "INFO  20:21:46,648 ProgressMeter -     18:47013237    2.581367074E9    65.0 m            1.0 s       84.7%    76.7 m      11.7 m \n",
      "INFO  20:22:16,649 ProgressMeter -     18:72707608    2.581367074E9    65.5 m            1.0 s       85.6%    76.5 m      11.0 m \n",
      "INFO  20:22:46,651 ProgressMeter -      19:5139389    2.659444322E9    66.0 m            1.0 s       85.9%    76.8 m      10.8 m \n",
      "INFO  20:23:16,652 ProgressMeter -     19:14077240    2.659444322E9    66.5 m            1.0 s       86.2%    77.2 m      10.7 m \n",
      "INFO  20:23:46,654 ProgressMeter -     19:26847188    2.659444322E9    67.0 m            1.0 s       86.6%    77.4 m      10.4 m \n",
      "INFO  20:24:16,655 ProgressMeter -     19:41842837    2.659444322E9    67.5 m            1.0 s       87.1%    77.5 m      10.0 m \n",
      "INFO  20:24:46,657 ProgressMeter -     19:52482742    2.659444322E9    68.0 m            1.0 s       87.4%    77.8 m       9.8 m \n",
      "INFO  20:25:16,658 ProgressMeter -      20:3930808    2.718573305E9    68.5 m            1.0 s       87.8%    78.0 m       9.5 m \n",
      "INFO  20:25:46,660 ProgressMeter -     20:25369240    2.718573305E9    69.0 m            1.0 s       88.5%    78.0 m       9.0 m \n",
      "INFO  20:26:16,661 ProgressMeter -     20:42846039    2.718573305E9    69.5 m            1.0 s       89.0%    78.1 m       8.6 m \n",
      "INFO  20:26:46,662 ProgressMeter -     20:60882651    2.718573305E9    70.0 m            1.0 s       89.6%    78.1 m       8.1 m \n",
      "INFO  20:27:16,664 ProgressMeter -     21:27067332    2.781598825E9    70.5 m            1.0 s       90.5%    77.9 m       7.4 m \n",
      "INFO  20:27:46,665 ProgressMeter -     21:44285422    2.781598825E9    71.0 m            1.0 s       91.1%    77.9 m       6.9 m \n",
      "INFO  20:28:16,666 ProgressMeter -     22:18853698     2.82972872E9    71.5 m            1.0 s       91.8%    77.9 m       6.4 m \n",
      "INFO  20:28:46,667 ProgressMeter -     22:32388139     2.82972872E9    72.0 m            1.0 s       92.3%    78.0 m       6.0 m \n",
      "INFO  20:29:16,669 ProgressMeter -     22:45573735     2.82972872E9    72.5 m            1.0 s       92.7%    78.2 m       5.7 m \n",
      "INFO  20:29:46,671 ProgressMeter -      X:10510606    2.881033286E9    73.0 m            1.0 s       93.2%    78.3 m       5.3 m \n",
      "INFO  20:30:16,672 ProgressMeter -      X:38240041    2.881033286E9    73.5 m            1.0 s       94.1%    78.1 m       4.6 m \n",
      "INFO  20:30:46,673 ProgressMeter -      X:63051278    2.881033286E9    74.0 m            1.0 s       94.9%    78.0 m       4.0 m \n",
      "INFO  20:31:16,674 ProgressMeter -      X:88239049    2.881033286E9    74.5 m            1.0 s       95.7%    77.8 m       3.3 m \n",
      "INFO  20:31:46,676 ProgressMeter -     X:117671368    2.881033286E9    75.0 m            1.0 s       96.7%    77.6 m       2.6 m \n",
      "INFO  20:32:16,677 ProgressMeter -     X:143488669    2.881033286E9    75.5 m            1.0 s       97.5%    77.4 m     115.0 s \n",
      "INFO  20:32:46,678 ProgressMeter -      Y:16955464    3.036303846E9    76.0 m            1.0 s       98.4%    77.2 m      72.0 s \n",
      "INFO  20:33:16,680 ProgressMeter -      Y:55645469    3.036303846E9    76.5 m            1.0 s       99.7%    76.7 m      14.0 s \n",
      "INFO  20:33:29,746 VectorLoglessPairHMM - Time spent in setup for JNI call : 0.18252035300000002 \n",
      "INFO  20:33:29,746 PairHMM - Total compute time in PairHMM computeLikelihoods() : 25.849576465000002 \n",
      "INFO  20:33:29,746 HaplotypeCaller - Ran local assembly on 11228139 active regions \n",
      "INFO  20:33:29,770 ProgressMeter -            done    3.101804739E9    76.7 m            1.0 s      100.0%    76.7 m       0.0 s \n",
      "INFO  20:33:29,770 ProgressMeter - Total runtime 4603.34 secs, 76.72 min, 1.28 hours \n",
      "INFO  20:33:29,770 MicroScheduler - 22805735 reads were filtered out during the traversal out of approximately 36292737 total reads (62.84%) \n",
      "INFO  20:33:29,770 MicroScheduler -   -> 0 reads (0.00% of total) failing BadCigarFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 15346570 reads (42.29% of total) failing DuplicateReadFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 0 reads (0.00% of total) failing FailsVendorQualityCheckFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 7459165 reads (20.55% of total) failing HCMappingQualityFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 0 reads (0.00% of total) failing MalformedReadFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 0 reads (0.00% of total) failing MappingQualityUnavailableFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 0 reads (0.00% of total) failing NotPrimaryAlignmentFilter \n",
      "INFO  20:33:29,771 MicroScheduler -   -> 0 reads (0.00% of total) failing UnmappedReadFilter \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were 4 WARN messages, the first 4 are repeated below.\n",
      "WARN  19:16:46,474 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "WARN  19:16:47,784 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper \n",
      "WARN  19:17:49,165 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "WARN  19:17:49,165 AnnotationUtils - Annotation will not be calculated, genotype is not called \n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz sample P21\n"
     ]
    }
   ],
   "source": [
    "###Variant calling with G-VCF functionality \n",
    "##Suggesties van NIEK! \n",
    "#Misschien geen cut-offs doen hier? Maar bij de volgende stap? \n",
    "\n",
    "\n",
    "step7_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step7/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "    \" -I \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF \" \\\n",
    "    \" -o \" + step7_dir + \"P\" + str(row['patient']) + \"_output.g.vcf\"\n",
    "\n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E1_DGO_15_10_30_S67_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E3_DGO_15_10_30_S69_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H1_DGO_15_10_30_S64_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq sample P11\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P11split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P11_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq sample P11\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P11split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P11_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/15-12-27-glass/Project_David/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04_S50_L004_R1_001.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P15_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P16_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001_2.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04_S54_L004_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04_S51_L004_R1_001.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P15_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04_S55_L004_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFB1_160201_DGO_16_04_04_S56_L004_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_rep1_160201_DGO_16_04_04_S46_L003_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_rep1_160201_DGO_16_04_04_S45_L003_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_lL34_TGFBR1antagonist_A8301_1uM_24h_rep1_160201_DGO_16_04_04_S47_L003_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_GMCSF_StemPro_IL34_Salk_151209_4_1_DGO_16_05_02_GTGGCC_S8_L001_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_IL34_Salk_1512093_1_DGO_16_05_02_GTCCGC_S6_L001_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_Salk_151209_3_2_DGO_16_05_02_GTGAAA_S7_L001_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_IL34_Salk_1512_09_5_1_DGO_16_05_02_GTTTCG_S9_L001_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P15_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz sample P4\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P4split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P4_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq sample P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P19_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq sample P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P21split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P21_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz sample P0\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P0split.bam -dontUseSoftClippedBases --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P0_no_filter_output.g.vcf\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###Variant calling with G-VCF functionality \n",
    "##Suggesties van NIEK! \n",
    "#Zonder cut-offs! \n",
    "\n",
    "step6_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step6/\"\n",
    "step7_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step7/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "    \" -I \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -dontUseSoftClippedBases --emitRefConfidence GVCF \" \\\n",
    "    \" -o \" + step7_dir + \"P\" + str(row['patient']) + \"_no_filter_output.g.vcf\"\n",
    "\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "INFO  14:46:49,349 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  14:46:49,351 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  14:46:49,351 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  14:46:49,351 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  14:46:49,351 HelpFormatter - [Sun Dec 04 14:46:49 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  14:46:49,352 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  14:46:49,354 HelpFormatter - Program Args: -T GenotypeGVCFs -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -o /data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf -log sample.genotypegvcfs.log -stand_call_conf 20.0 -stand_emit_conf 20.0 -V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf -V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf \n",
      "INFO  14:46:49,363 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  14:46:49,364 HelpFormatter - Date/Time: 2016/12/04 14:46:49 \n",
      "INFO  14:46:49,364 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  14:46:49,365 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  14:46:49,404 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  14:46:49,509 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  14:46:49,731 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  14:46:49,735 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  14:46:49,736 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  14:46:49,736 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  14:46:49,737 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  14:46:49,841 StrandBiasTest - StrandBiasBySample annotation exists in input VCF header. Attempting to use StrandBiasBySample values to calculate strand bias annotation values. If no sample has the SB genotype annotation, annotation may still fail. \n",
      "WARN  14:46:49,841 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "WARN  14:46:49,841 StrandBiasTest - StrandBiasBySample annotation exists in input VCF header. Attempting to use StrandBiasBySample values to calculate strand bias annotation values. If no sample has the SB genotype annotation, annotation may still fail. \n",
      "INFO  14:46:49,842 GenotypeGVCFs - Notice that the -ploidy parameter is ignored in GenotypeGVCFs tool as this is automatically determined by the input variant files \n",
      "WARN  14:46:50,138 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper, not org.broadinstitute.gatk.tools.walkers.variantutils.GenotypeGVCFs \n",
      "INFO  14:47:19,740 ProgressMeter -       1:3999901   3000000.0    30.0 s      10.0 s        0.1%     6.5 h       6.5 h \n",
      "INFO  14:47:49,742 ProgressMeter -       1:8517601   8000000.0    60.0 s       7.0 s        0.3%     6.1 h       6.1 h \n",
      "INFO  14:48:19,744 ProgressMeter -      1:11742801       1.1E7    90.0 s       8.0 s        0.4%     6.6 h       6.6 h \n",
      "INFO  14:48:49,746 ProgressMeter -      1:14746101       1.4E7   120.0 s       8.0 s        0.5%     7.0 h       7.0 h \n",
      "INFO  14:49:19,748 ProgressMeter -      1:16999901       1.6E7     2.5 m       9.0 s        0.5%     7.6 h       7.6 h \n",
      "INFO  14:49:49,751 ProgressMeter -      1:19726801       1.9E7     3.0 m       9.0 s        0.6%     7.9 h       7.8 h \n",
      "INFO  14:50:19,752 ProgressMeter -      1:21879101       2.1E7     3.5 m      10.0 s        0.7%     8.3 h       8.2 h \n",
      "INFO  14:50:49,754 ProgressMeter -      1:23999901       2.3E7     4.0 m      10.0 s        0.8%     8.6 h       8.5 h \n",
      "INFO  14:51:19,756 ProgressMeter -      1:25999901       2.5E7     4.5 m      10.0 s        0.8%     8.9 h       8.9 h \n",
      "INFO  14:51:49,758 ProgressMeter -      1:27999901       2.7E7     5.0 m      11.0 s        0.9%     9.2 h       9.1 h \n",
      "INFO  14:52:19,760 ProgressMeter -      1:29999901       2.9E7     5.5 m      11.0 s        1.0%     9.5 h       9.4 h \n",
      "INFO  14:52:49,763 ProgressMeter -      1:31999901       3.1E7     6.0 m      11.0 s        1.0%     9.7 h       9.6 h \n",
      "INFO  14:53:19,765 ProgressMeter -      1:33999901       3.3E7     6.5 m      11.0 s        1.1%     9.9 h       9.8 h \n",
      "INFO  14:53:49,767 ProgressMeter -      1:35999901       3.5E7     7.0 m      12.0 s        1.2%    10.1 h       9.9 h \n",
      "INFO  14:54:19,769 ProgressMeter -      1:37999901       3.7E7     7.5 m      12.0 s        1.2%    10.2 h      10.1 h \n",
      "INFO  14:54:49,771 ProgressMeter -      1:39999901       3.9E7     8.0 m      12.0 s        1.3%    10.3 h      10.2 h \n",
      "INFO  14:55:19,774 ProgressMeter -      1:41999901       4.1E7     8.5 m      12.0 s        1.4%    10.5 h      10.3 h \n",
      "INFO  14:55:49,775 ProgressMeter -      1:43999901       4.3E7     9.0 m      12.0 s        1.4%    10.6 h      10.4 h \n",
      "INFO  14:56:19,777 ProgressMeter -      1:45999901       4.5E7     9.5 m      12.0 s        1.5%    10.7 h      10.5 h \n",
      "INFO  14:56:49,779 ProgressMeter -      1:47783301       4.7E7    10.0 m      12.0 s        1.5%    10.8 h      10.7 h \n",
      "INFO  14:57:19,781 ProgressMeter -      1:49999901       4.9E7    10.5 m      12.0 s        1.6%    10.9 h      10.7 h \n",
      "INFO  14:57:49,783 ProgressMeter -      1:51999901       5.1E7    11.0 m      12.0 s        1.7%    10.9 h      10.8 h \n",
      "INFO  14:58:19,785 ProgressMeter -      1:54318201       5.4E7    11.5 m      12.0 s        1.8%    10.9 h      10.8 h \n",
      "INFO  14:58:49,786 ProgressMeter -      1:56999901       5.6E7    12.0 m      12.0 s        1.8%    10.9 h      10.7 h \n",
      "INFO  14:59:19,787 ProgressMeter -      1:59999901       5.9E7    12.5 m      12.0 s        1.9%    10.8 h      10.6 h \n",
      "INFO  14:59:49,789 ProgressMeter -      1:62999901       6.2E7    13.0 m      12.0 s        2.0%    10.7 h      10.5 h \n",
      "INFO  15:00:19,790 ProgressMeter -      1:65999901       6.5E7    13.5 m      12.0 s        2.1%    10.6 h      10.3 h \n",
      "INFO  15:00:49,792 ProgressMeter -      1:70138901       7.0E7    14.0 m      12.0 s        2.3%    10.3 h      10.1 h \n",
      "INFO  15:01:19,794 ProgressMeter -      1:74999901       7.4E7    14.5 m      11.0 s        2.4%    10.0 h       9.8 h \n",
      "INFO  15:01:49,795 ProgressMeter -      1:78999901       7.8E7    15.0 m      11.0 s        2.5%     9.8 h       9.6 h \n",
      "INFO  15:02:19,796 ProgressMeter -      1:84218101       8.4E7    15.5 m      11.0 s        2.7%     9.5 h       9.3 h \n",
      "INFO  15:02:49,798 ProgressMeter -      1:88999901       8.8E7    16.0 m      10.0 s        2.9%     9.3 h       9.0 h \n",
      "INFO  15:03:19,799 ProgressMeter -      1:92999901       9.2E7    16.5 m      10.0 s        3.0%     9.2 h       8.9 h \n",
      "INFO  15:03:49,801 ProgressMeter -      1:96999901       9.6E7    17.0 m      10.0 s        3.1%     9.1 h       8.8 h \n",
      "INFO  15:04:19,802 ProgressMeter -     1:100999901       1.0E8    17.5 m      10.0 s        3.3%     9.0 h       8.7 h \n",
      "INFO  15:04:49,809 ProgressMeter -     1:105900601      1.05E8    18.0 m      10.0 s        3.4%     8.8 h       8.5 h \n",
      "INFO  15:05:19,811 ProgressMeter -     1:109999901      1.09E8    18.5 m      10.0 s        3.5%     8.7 h       8.4 h \n",
      "INFO  15:05:49,812 ProgressMeter -     1:112999901      1.12E8    19.0 m      10.0 s        3.6%     8.7 h       8.4 h \n",
      "INFO  15:06:19,813 ProgressMeter -     1:115999901      1.15E8    19.5 m      10.0 s        3.7%     8.7 h       8.4 h \n",
      "INFO  15:06:49,815 ProgressMeter -     1:119645001      1.19E8    20.0 m      10.0 s        3.9%     8.6 h       8.3 h \n",
      "INFO  15:07:19,816 ProgressMeter -     1:122999901      1.22E8    20.5 m      10.0 s        4.0%     8.6 h       8.3 h \n",
      "INFO  15:07:49,818 ProgressMeter -     1:126999901      1.26E8    21.0 m      10.0 s        4.1%     8.5 h       8.2 h \n",
      "INFO  15:08:19,820 ProgressMeter -     1:130999901       1.3E8    21.5 m       9.0 s        4.2%     8.5 h       8.1 h \n",
      "INFO  15:08:49,821 ProgressMeter -     1:136999901      1.36E8    22.0 m       9.0 s        4.4%     8.3 h       7.9 h \n",
      "INFO  15:09:19,822 ProgressMeter -     1:145290201      1.45E8    22.5 m       9.0 s        4.7%     8.0 h       7.6 h \n",
      "INFO  15:09:49,824 ProgressMeter -     1:150587601       1.5E8    23.0 m       9.0 s        4.9%     7.9 h       7.5 h \n",
      "INFO  15:10:19,825 ProgressMeter -     1:153999901      1.53E8    23.5 m       9.0 s        5.0%     7.9 h       7.5 h \n",
      "INFO  15:10:49,827 ProgressMeter -     1:156145601      1.56E8    24.0 m       9.0 s        5.0%     7.9 h       7.5 h \n",
      "INFO  15:11:19,828 ProgressMeter -     1:158999901      1.58E8    24.5 m       9.0 s        5.1%     8.0 h       7.6 h \n",
      "INFO  15:11:49,829 ProgressMeter -     1:161437601      1.61E8    25.0 m       9.0 s        5.2%     8.0 h       7.6 h \n",
      "INFO  15:12:19,831 ProgressMeter -     1:163999901      1.63E8    25.5 m       9.0 s        5.3%     8.0 h       7.6 h \n",
      "INFO  15:12:49,832 ProgressMeter -     1:166298301      1.66E8    26.0 m       9.0 s        5.4%     8.1 h       7.6 h \n",
      "INFO  15:13:19,834 ProgressMeter -     1:168999901      1.68E8    26.5 m       9.0 s        5.4%     8.1 h       7.7 h \n",
      "INFO  15:13:49,835 ProgressMeter -     1:170999901       1.7E8    27.0 m       9.0 s        5.5%     8.2 h       7.7 h \n",
      "INFO  15:14:19,836 ProgressMeter -     1:173972301      1.73E8    27.5 m       9.0 s        5.6%     8.2 h       7.7 h \n",
      "INFO  15:14:49,838 ProgressMeter -     1:176211501      1.76E8    28.0 m       9.0 s        5.7%     8.2 h       7.7 h \n",
      "INFO  15:15:19,839 ProgressMeter -     1:179737601      1.79E8    28.5 m       9.0 s        5.8%     8.2 h       7.7 h \n",
      "INFO  15:15:49,840 ProgressMeter -     1:182778401      1.82E8    29.0 m       9.0 s        5.9%     8.2 h       7.7 h \n",
      "INFO  15:16:19,841 ProgressMeter -     1:185999901      1.85E8    29.5 m       9.0 s        6.0%     8.2 h       7.7 h \n",
      "INFO  15:16:49,843 ProgressMeter -     1:188999901      1.88E8    30.0 m       9.0 s        6.1%     8.2 h       7.7 h \n",
      "INFO  15:17:19,844 ProgressMeter -     1:193063101      1.93E8    30.5 m       9.0 s        6.2%     8.2 h       7.7 h \n",
      "INFO  15:17:49,846 ProgressMeter -     1:197789901      1.97E8    31.0 m       9.0 s        6.4%     8.1 h       7.6 h \n",
      "INFO  15:18:19,847 ProgressMeter -     1:201963701      2.01E8    31.5 m       9.0 s        6.5%     8.1 h       7.5 h \n",
      "INFO  15:18:49,848 ProgressMeter -     1:205580501      2.05E8    32.0 m       9.0 s        6.6%     8.0 h       7.5 h \n",
      "INFO  15:19:19,850 ProgressMeter -     1:208999901      2.08E8    32.5 m       9.0 s        6.7%     8.0 h       7.5 h \n",
      "INFO  15:19:49,851 ProgressMeter -     1:212999901      2.12E8    33.0 m       9.0 s        6.9%     8.0 h       7.5 h \n",
      "INFO  15:20:19,852 ProgressMeter -     1:216256301      2.16E8    33.5 m       9.0 s        7.0%     8.0 h       7.4 h \n",
      "INFO  15:20:49,860 ProgressMeter -     1:219999901      2.19E8    34.0 m       9.0 s        7.1%     8.0 h       7.4 h \n",
      "INFO  15:21:19,861 ProgressMeter -     1:223303501      2.23E8    34.5 m       9.0 s        7.2%     8.0 h       7.4 h \n",
      "INFO  15:21:49,863 ProgressMeter -     1:226800801      2.26E8    35.0 m       9.0 s        7.3%     8.0 h       7.4 h \n",
      "INFO  15:22:19,864 ProgressMeter -     1:229999901      2.29E8    35.5 m       9.0 s        7.4%     8.0 h       7.4 h \n",
      "INFO  15:22:49,865 ProgressMeter -     1:232999901      2.32E8    36.0 m       9.0 s        7.5%     8.0 h       7.4 h \n",
      "INFO  15:23:19,866 ProgressMeter -     1:235999901      2.35E8    36.5 m       9.0 s        7.6%     8.0 h       7.4 h \n",
      "INFO  15:23:49,867 ProgressMeter -     1:238999901      2.38E8    37.0 m       9.0 s        7.7%     8.0 h       7.4 h \n",
      "INFO  15:24:19,869 ProgressMeter -     1:242543401      2.42E8    37.5 m       9.0 s        7.8%     8.0 h       7.4 h \n",
      "INFO  15:24:49,870 ProgressMeter -     1:245252201      2.45E8    38.0 m       9.0 s        7.9%     8.0 h       7.4 h \n",
      "INFO  15:25:19,871 ProgressMeter -     1:248103401      2.48E8    38.5 m       9.0 s        8.0%     8.0 h       7.4 h \n",
      "INFO  15:25:49,873 ProgressMeter -       2:5693801   2.54250621E8    39.0 m       9.0 s        8.2%     7.9 h       7.3 h \n",
      "INFO  15:26:19,874 ProgressMeter -      2:12632401   2.61250621E8    39.5 m       9.0 s        8.4%     7.8 h       7.1 h \n",
      "INFO  15:26:49,875 ProgressMeter -      2:20818801   2.69250621E8    40.0 m       8.0 s        8.7%     7.7 h       7.0 h \n",
      "INFO  15:27:19,876 ProgressMeter -      2:26604001   2.75250621E8    40.5 m       8.0 s        8.9%     7.6 h       6.9 h \n",
      "INFO  15:27:49,882 ProgressMeter -      2:31817501   2.80250621E8    41.0 m       8.0 s        9.1%     7.5 h       6.9 h \n",
      "INFO  15:28:19,883 ProgressMeter -      2:37999901   2.86250621E8    41.5 m       8.0 s        9.3%     7.5 h       6.8 h \n",
      "INFO  15:28:49,885 ProgressMeter -      2:43496201   2.92250621E8    42.0 m       8.0 s        9.4%     7.4 h       6.7 h \n",
      "INFO  15:29:19,886 ProgressMeter -      2:49207601   2.98250621E8    42.5 m       8.0 s        9.6%     7.4 h       6.7 h \n",
      "INFO  15:29:49,888 ProgressMeter -      2:56122301   3.05250621E8    43.0 m       8.0 s        9.8%     7.3 h       6.6 h \n",
      "INFO  15:30:19,889 ProgressMeter -      2:62577501   3.11250621E8    43.5 m       8.0 s       10.1%     7.2 h       6.5 h \n",
      "INFO  15:30:49,891 ProgressMeter -      2:67999901   3.16250621E8    44.0 m       8.0 s       10.2%     7.2 h       6.4 h \n",
      "INFO  15:31:19,892 ProgressMeter -      2:72908601   3.21250621E8    44.5 m       8.0 s       10.4%     7.1 h       6.4 h \n",
      "INFO  15:31:49,893 ProgressMeter -      2:78190401   3.27250621E8    45.0 m       8.0 s       10.6%     7.1 h       6.4 h \n",
      "INFO  15:32:19,894 ProgressMeter -      2:86331301   3.35250621E8    45.5 m       8.0 s       10.8%     7.0 h       6.3 h \n",
      "INFO  15:32:49,895 ProgressMeter -      2:94229801   3.43250621E8    46.0 m       8.0 s       11.1%     6.9 h       6.2 h \n",
      "INFO  15:33:19,897 ProgressMeter -      2:99999901   3.48250621E8    46.5 m       8.0 s       11.3%     6.9 h       6.1 h \n",
      "INFO  15:33:49,898 ProgressMeter -     2:106024301   3.55250621E8    47.0 m       7.0 s       11.5%     6.8 h       6.1 h \n",
      "INFO  15:34:19,899 ProgressMeter -     2:112538101   3.61250621E8    47.5 m       7.0 s       11.7%     6.8 h       6.0 h \n",
      "INFO  15:34:49,901 ProgressMeter -     2:118605701   3.67250621E8    48.0 m       7.0 s       11.9%     6.7 h       5.9 h \n",
      "INFO  15:35:19,902 ProgressMeter -     2:126045401   3.75250621E8    48.5 m       7.0 s       12.1%     6.7 h       5.9 h \n",
      "INFO  15:35:49,903 ProgressMeter -     2:132067101   3.81250621E8    49.0 m       7.0 s       12.3%     6.6 h       5.8 h \n",
      "INFO  15:36:19,904 ProgressMeter -     2:138730401   3.87250621E8    49.5 m       7.0 s       12.5%     6.6 h       5.8 h \n",
      "INFO  15:36:49,906 ProgressMeter -     2:146252901   3.95250621E8    50.0 m       7.0 s       12.8%     6.5 h       5.7 h \n",
      "INFO  15:37:19,907 ProgressMeter -     2:153356501   4.02250621E8    50.5 m       7.0 s       13.0%     6.5 h       5.6 h \n",
      "INFO  15:37:49,909 ProgressMeter -     2:160420401   4.09250621E8    51.0 m       7.0 s       13.2%     6.4 h       5.6 h \n",
      "INFO  15:38:19,910 ProgressMeter -     2:167765401   4.16250621E8    51.5 m       7.0 s       13.4%     6.4 h       5.5 h \n",
      "INFO  15:38:49,911 ProgressMeter -     2:174394001   4.23250621E8    52.0 m       7.0 s       13.7%     6.3 h       5.5 h \n",
      "INFO  15:39:19,912 ProgressMeter -     2:180654101   4.29250621E8    52.5 m       7.0 s       13.9%     6.3 h       5.4 h \n",
      "INFO  15:39:49,914 ProgressMeter -     2:188443001   4.37250621E8    53.0 m       7.0 s       14.1%     6.3 h       5.4 h \n",
      "INFO  15:40:19,916 ProgressMeter -     2:195696601   4.44250621E8    53.5 m       7.0 s       14.3%     6.2 h       5.3 h \n",
      "INFO  15:40:49,917 ProgressMeter -     2:201931101   4.50250621E8    54.0 m       7.0 s       14.5%     6.2 h       5.3 h \n",
      "INFO  15:41:19,918 ProgressMeter -     2:207001101   4.56250621E8    54.5 m       7.0 s       14.7%     6.2 h       5.3 h \n",
      "INFO  15:41:49,947 ProgressMeter -     2:213867101   4.62250621E8    55.0 m       7.0 s       14.9%     6.1 h       5.2 h \n",
      "INFO  15:42:19,948 ProgressMeter -     2:220344801   4.69250621E8    55.5 m       7.0 s       15.1%     6.1 h       5.2 h \n",
      "INFO  15:42:49,949 ProgressMeter -     2:227384501   4.76250621E8    56.0 m       7.0 s       15.4%     6.1 h       5.1 h \n",
      "INFO  15:43:19,951 ProgressMeter -     2:232999901   4.81250621E8    56.5 m       7.0 s       15.5%     6.1 h       5.1 h \n",
      "INFO  15:43:49,952 ProgressMeter -     2:238999901   4.87250621E8    57.0 m       7.0 s       15.7%     6.0 h       5.1 h \n",
      "INFO  15:44:19,953 ProgressMeter -       3:1079101   4.93449994E8    57.5 m       6.0 s       15.9%     6.0 h       5.1 h \n",
      "INFO  15:44:49,954 ProgressMeter -       3:9000301   5.01449994E8    58.0 m       6.0 s       16.2%     6.0 h       5.0 h \n",
      "INFO  15:45:19,955 ProgressMeter -      3:13070501   5.05449994E8    58.5 m       6.0 s       16.3%     6.0 h       5.0 h \n",
      "INFO  15:45:49,956 ProgressMeter -      3:18902001   5.10449994E8    59.0 m       6.0 s       16.5%     6.0 h       5.0 h \n",
      "INFO  15:46:19,958 ProgressMeter -      3:26185801   5.18449994E8    59.5 m       6.0 s       16.7%     5.9 h       4.9 h \n",
      "INFO  15:46:49,959 ProgressMeter -      3:33110301   5.25449994E8    60.0 m       6.0 s       16.9%     5.9 h       4.9 h \n",
      "INFO  15:47:19,968 ProgressMeter -      3:39137801   5.31449994E8    60.5 m       6.0 s       17.1%     5.9 h       4.9 h \n",
      "INFO  15:47:49,969 ProgressMeter -      3:44491701   5.36449994E8    61.0 m       6.0 s       17.3%     5.9 h       4.9 h \n",
      "INFO  15:48:19,970 ProgressMeter -      3:48972001   5.40449994E8    61.5 m       6.0 s       17.5%     5.9 h       4.8 h \n",
      "INFO  15:48:49,971 ProgressMeter -      3:52240601   5.44449994E8    62.0 m       6.0 s       17.6%     5.9 h       4.9 h \n",
      "INFO  15:49:19,972 ProgressMeter -      3:56712701   5.48449994E8    62.5 m       6.0 s       17.7%     5.9 h       4.8 h \n",
      "INFO  15:49:49,973 ProgressMeter -      3:63510001   5.55449994E8    63.0 m       6.0 s       17.9%     5.9 h       4.8 h \n",
      "INFO  15:50:19,975 ProgressMeter -      3:70560101   5.62449994E8    63.5 m       6.0 s       18.2%     5.8 h       4.8 h \n",
      "INFO  15:50:49,976 ProgressMeter -      3:79016601   5.71449994E8    64.0 m       6.0 s       18.4%     5.8 h       4.7 h \n",
      "INFO  15:51:19,977 ProgressMeter -      3:88190901   5.80449994E8    64.5 m       6.0 s       18.7%     5.7 h       4.7 h \n",
      "INFO  15:51:49,978 ProgressMeter -      3:97591301   5.89449994E8    65.0 m       6.0 s       19.0%     5.7 h       4.6 h \n",
      "INFO  15:52:19,979 ProgressMeter -     3:104548801   5.96449994E8    65.5 m       6.0 s       19.2%     5.7 h       4.6 h \n",
      "INFO  15:52:49,981 ProgressMeter -     3:112699601   6.04449994E8    66.0 m       6.0 s       19.5%     5.6 h       4.5 h \n",
      "INFO  15:53:19,982 ProgressMeter -     3:119255901   6.11449994E8    66.5 m       6.0 s       19.7%     5.6 h       4.5 h \n",
      "INFO  15:53:49,983 ProgressMeter -     3:123999901   6.15449994E8    67.0 m       6.0 s       19.9%     5.6 h       4.5 h \n",
      "INFO  15:54:19,984 ProgressMeter -     3:128999901   6.20449994E8    67.5 m       6.0 s       20.0%     5.6 h       4.5 h \n",
      "INFO  15:54:49,985 ProgressMeter -     3:134285901   6.26449994E8    68.0 m       6.0 s       20.2%     5.6 h       4.5 h \n",
      "INFO  15:55:19,992 ProgressMeter -     3:140925301   6.32449994E8    68.5 m       6.0 s       20.4%     5.6 h       4.4 h \n",
      "INFO  15:55:49,992 ProgressMeter -     3:146980101   6.38449994E8    69.0 m       6.0 s       20.6%     5.6 h       4.4 h \n",
      "INFO  15:56:19,994 ProgressMeter -     3:152999901   6.44449994E8    69.5 m       6.0 s       20.8%     5.6 h       4.4 h \n",
      "INFO  15:56:49,995 ProgressMeter -     3:159999901   6.51449994E8    70.0 m       6.0 s       21.0%     5.5 h       4.4 h \n",
      "INFO  15:57:19,996 ProgressMeter -     3:168369701   6.60449994E8    70.5 m       6.0 s       21.3%     5.5 h       4.3 h \n",
      "INFO  15:57:49,997 ProgressMeter -     3:175106301   6.67449994E8    71.0 m       6.0 s       21.5%     5.5 h       4.3 h \n",
      "INFO  15:58:19,998 ProgressMeter -     3:181986901   6.73449994E8    71.5 m       6.0 s       21.7%     5.5 h       4.3 h \n",
      "INFO  15:58:50,000 ProgressMeter -     3:186370201   6.78449994E8    72.0 m       6.0 s       21.9%     5.5 h       4.3 h \n",
      "INFO  15:59:20,001 ProgressMeter -     3:191724801   6.83449994E8    72.5 m       6.0 s       22.1%     5.5 h       4.3 h \n",
      "INFO  15:59:50,002 ProgressMeter -     3:196999901   6.88449994E8    73.0 m       6.0 s       22.2%     5.5 h       4.3 h \n",
      "INFO  16:00:20,003 ProgressMeter -       4:2346901   6.92472424E8    73.5 m       6.0 s       22.3%     5.5 h       4.3 h \n",
      "INFO  16:00:50,004 ProgressMeter -       4:6884201   6.96472424E8    74.0 m       6.0 s       22.5%     5.5 h       4.3 h \n",
      "INFO  16:01:20,005 ProgressMeter -      4:12694401   7.02472424E8    74.5 m       6.0 s       22.7%     5.5 h       4.2 h \n",
      "INFO  16:01:50,007 ProgressMeter -      4:19382301   7.09472424E8    75.0 m       6.0 s       22.9%     5.5 h       4.2 h \n",
      "INFO  16:02:20,008 ProgressMeter -      4:26878501   7.16472424E8    75.5 m       6.0 s       23.1%     5.4 h       4.2 h \n",
      "INFO  16:02:50,009 ProgressMeter -      4:35908901   7.25472424E8    76.0 m       6.0 s       23.4%     5.4 h       4.1 h \n",
      "INFO  16:03:20,010 ProgressMeter -      4:40542901   7.30472424E8    76.5 m       6.0 s       23.6%     5.4 h       4.1 h \n",
      "INFO  16:03:50,012 ProgressMeter -      4:47559701   7.37472424E8    77.0 m       6.0 s       23.8%     5.4 h       4.1 h \n",
      "INFO  16:04:20,014 ProgressMeter -      4:54999901   7.44472424E8    77.5 m       6.0 s       24.0%     5.4 h       4.1 h \n",
      "INFO  16:04:50,015 ProgressMeter -      4:62352701   7.52472424E8    78.0 m       6.0 s       24.3%     5.4 h       4.1 h \n",
      "INFO  16:05:20,017 ProgressMeter -      4:70759501   7.60472424E8    78.5 m       6.0 s       24.5%     5.3 h       4.0 h \n",
      "INFO  16:05:50,019 ProgressMeter -      4:77738001   7.67472424E8    79.0 m       6.0 s       24.8%     5.3 h       4.0 h \n",
      "INFO  16:06:20,021 ProgressMeter -      4:84100201   7.74472424E8    79.5 m       6.0 s       25.0%     5.3 h       4.0 h \n",
      "INFO  16:06:50,022 ProgressMeter -      4:88999901   7.78472424E8    80.0 m       6.0 s       25.1%     5.3 h       4.0 h \n",
      "INFO  16:07:20,025 ProgressMeter -      4:95705401   7.85472424E8    80.5 m       6.0 s       25.3%     5.3 h       4.0 h \n",
      "INFO  16:07:50,026 ProgressMeter -     4:103433801   7.93472424E8    81.0 m       6.0 s       25.6%     5.3 h       3.9 h \n",
      "INFO  16:08:20,032 ProgressMeter -     4:110236501   8.00472424E8    81.5 m       6.0 s       25.8%     5.3 h       3.9 h \n",
      "INFO  16:08:50,034 ProgressMeter -     4:117659501   8.07472424E8    82.0 m       6.0 s       26.1%     5.2 h       3.9 h \n",
      "INFO  16:09:20,035 ProgressMeter -     4:124761201   8.14472424E8    82.5 m       6.0 s       26.3%     5.2 h       3.9 h \n",
      "INFO  16:09:50,036 ProgressMeter -     4:132828501   8.22472424E8    83.0 m       6.0 s       26.5%     5.2 h       3.8 h \n",
      "INFO  16:10:20,037 ProgressMeter -     4:140999901   8.30472424E8    83.5 m       6.0 s       26.8%     5.2 h       3.8 h \n",
      "INFO  16:10:50,038 ProgressMeter -     4:147999901   8.37472424E8    84.0 m       6.0 s       27.0%     5.2 h       3.8 h \n",
      "INFO  16:11:20,040 ProgressMeter -     4:154999901   8.44472424E8    84.5 m       6.0 s       27.3%     5.2 h       3.8 h \n",
      "INFO  16:11:50,046 ProgressMeter -     4:161937401   8.51472424E8    85.0 m       5.0 s       27.5%     5.2 h       3.7 h \n",
      "INFO  16:12:20,048 ProgressMeter -     4:169513901   8.59472424E8    85.5 m       5.0 s       27.7%     5.1 h       3.7 h \n",
      "INFO  16:12:50,049 ProgressMeter -     4:177231001   8.67472424E8    86.0 m       5.0 s       28.0%     5.1 h       3.7 h \n",
      "INFO  16:13:20,050 ProgressMeter -     4:185621701   8.75472424E8    86.5 m       5.0 s       28.2%     5.1 h       3.7 h \n",
      "INFO  16:13:50,051 ProgressMeter -       5:1112901   8.826267E8    87.0 m       5.0 s       28.5%     5.1 h       3.6 h \n",
      "INFO  16:14:20,052 ProgressMeter -       5:8966201   8.896267E8    87.5 m       5.0 s       28.7%     5.1 h       3.6 h \n",
      "INFO  16:14:50,054 ProgressMeter -      5:16375501   8.976267E8    88.0 m       5.0 s       29.0%     5.1 h       3.6 h \n",
      "INFO  16:15:20,055 ProgressMeter -      5:25139301   9.066267E8    88.5 m       5.0 s       29.2%     5.0 h       3.6 h \n",
      "INFO  16:15:50,057 ProgressMeter -      5:33981701   9.146267E8    89.0 m       5.0 s       29.5%     5.0 h       3.5 h \n",
      "INFO  16:16:20,058 ProgressMeter -      5:39933901   9.206267E8    89.5 m       5.0 s       29.7%     5.0 h       3.5 h \n",
      "INFO  16:16:50,059 ProgressMeter -      5:46690401   9.276267E8    90.0 m       5.0 s       29.9%     5.0 h       3.5 h \n",
      "INFO  16:17:20,060 ProgressMeter -      5:54999901   9.356267E8    90.5 m       5.0 s       30.2%     5.0 h       3.5 h \n",
      "INFO  16:17:50,061 ProgressMeter -      5:61876601   9.426267E8    91.0 m       5.0 s       30.4%     5.0 h       3.5 h \n",
      "INFO  16:18:20,062 ProgressMeter -      5:68865501   9.496267E8    91.5 m       5.0 s       30.6%     5.0 h       3.5 h \n",
      "INFO  16:18:50,063 ProgressMeter -      5:75999901   9.566267E8    92.0 m       5.0 s       30.9%     5.0 h       3.4 h \n",
      "INFO  16:19:20,064 ProgressMeter -      5:81061701   9.626267E8    92.5 m       5.0 s       31.0%     5.0 h       3.4 h \n",
      "INFO  16:19:50,065 ProgressMeter -      5:88292201   9.696267E8    93.0 m       5.0 s       31.3%     5.0 h       3.4 h \n",
      "INFO  16:20:20,066 ProgressMeter -      5:95343101   9.766267E8    93.5 m       5.0 s       31.5%     4.9 h       3.4 h \n",
      "INFO  16:20:50,068 ProgressMeter -     5:102601601   9.836267E8    94.0 m       5.0 s       31.7%     4.9 h       3.4 h \n",
      "INFO  16:21:20,069 ProgressMeter -     5:110847201   9.916267E8    94.5 m       5.0 s       32.0%     4.9 h       3.3 h \n",
      "INFO  16:21:50,070 ProgressMeter -     5:117874101   9.986267E8    95.0 m       5.0 s       32.2%     4.9 h       3.3 h \n",
      "INFO  16:22:20,071 ProgressMeter -     5:125901101   1.0066267E9    95.5 m       5.0 s       32.5%     4.9 h       3.3 h \n",
      "INFO  16:22:50,072 ProgressMeter -     5:132212201   1.0136267E9    96.0 m       5.0 s       32.7%     4.9 h       3.3 h \n",
      "INFO  16:23:30,073 ProgressMeter -     5:139099001   1.0206267E9    96.7 m       5.0 s       32.9%     4.9 h       3.3 h \n",
      "INFO  16:24:10,074 ProgressMeter -     5:145853101   1.0266267E9    97.3 m       5.0 s       33.1%     4.9 h       3.3 h \n",
      "INFO  16:24:40,075 ProgressMeter -     5:151259301   1.0326267E9    97.8 m       5.0 s       33.3%     4.9 h       3.3 h \n",
      "INFO  16:25:10,076 ProgressMeter -     5:158313501   1.0396267E9    98.3 m       5.0 s       33.5%     4.9 h       3.2 h \n",
      "INFO  16:25:40,077 ProgressMeter -     5:166075401   1.0476267E9    98.8 m       5.0 s       33.8%     4.9 h       3.2 h \n",
      "INFO  16:26:10,079 ProgressMeter -     5:172414101   1.0536267E9    99.3 m       5.0 s       34.0%     4.9 h       3.2 h \n",
      "INFO  16:26:40,080 ProgressMeter -     5:177999901   1.0586267E9    99.8 m       5.0 s       34.2%     4.9 h       3.2 h \n",
      "INFO  16:27:20,081 ProgressMeter -       6:3729101   1.06554196E9   100.5 m       5.0 s       34.4%     4.9 h       3.2 h \n",
      "INFO  16:27:50,082 ProgressMeter -       6:9503001   1.07154196E9   101.0 m       5.0 s       34.6%     4.9 h       3.2 h \n",
      "INFO  16:28:20,083 ProgressMeter -      6:15965801   1.07754196E9   101.5 m       5.0 s       34.8%     4.9 h       3.2 h \n",
      "INFO  16:28:50,084 ProgressMeter -      6:22700701   1.08454196E9   102.0 m       5.0 s       35.0%     4.9 h       3.2 h \n",
      "INFO  16:29:20,085 ProgressMeter -      6:28999901   1.09054196E9   102.5 m       5.0 s       35.2%     4.9 h       3.1 h \n",
      "WARN  16:29:40,895 ExactAFCalculator - This tool is currently set to genotype at most 6 alternate alleles in a given context, but the context at 6: 31797923 has 7 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument. Unless the DEBUG logging level is used, this warning message is output just once per run and further warnings are suppressed. \n",
      "INFO  16:29:50,086 ProgressMeter -      6:32867101   1.09454196E9   103.0 m       5.0 s       35.3%     4.9 h       3.1 h \n",
      "INFO  16:30:20,087 ProgressMeter -      6:35999901   1.09754196E9   103.5 m       5.0 s       35.4%     4.9 h       3.1 h \n",
      "INFO  16:30:50,088 ProgressMeter -      6:40446501   1.10254196E9   104.0 m       5.0 s       35.6%     4.9 h       3.1 h \n",
      "INFO  16:31:20,090 ProgressMeter -      6:44827701   1.10654196E9   104.5 m       5.0 s       35.7%     4.9 h       3.1 h \n",
      "INFO  16:31:50,091 ProgressMeter -      6:51916801   1.11354196E9   105.0 m       5.0 s       35.9%     4.9 h       3.1 h \n",
      "INFO  16:32:20,092 ProgressMeter -      6:59347801   1.12154196E9   105.5 m       5.0 s       36.2%     4.9 h       3.1 h \n",
      "INFO  16:32:50,093 ProgressMeter -      6:68956401   1.13054196E9   106.0 m       5.0 s       36.5%     4.8 h       3.1 h \n",
      "INFO  16:33:20,095 ProgressMeter -      6:76369001   1.13854196E9   106.5 m       5.0 s       36.7%     4.8 h       3.1 h \n",
      "INFO  16:33:50,096 ProgressMeter -      6:84102201   1.14654196E9   107.0 m       5.0 s       37.0%     4.8 h       3.0 h \n",
      "INFO  16:34:20,097 ProgressMeter -      6:90999901   1.15254196E9   107.5 m       5.0 s       37.2%     4.8 h       3.0 h \n",
      "INFO  16:34:50,098 ProgressMeter -      6:99563801   1.16154196E9   108.0 m       5.0 s       37.5%     4.8 h       3.0 h \n",
      "INFO  16:35:20,100 ProgressMeter -     6:106606001   1.16854196E9   108.5 m       5.0 s       37.7%     4.8 h       3.0 h \n",
      "INFO  16:35:50,106 ProgressMeter -     6:111999901   1.17354196E9   109.0 m       5.0 s       37.9%     4.8 h       3.0 h \n",
      "INFO  16:36:20,108 ProgressMeter -     6:119164601   1.18154196E9   109.5 m       5.0 s       38.1%     4.8 h       3.0 h \n",
      "INFO  16:36:50,109 ProgressMeter -     6:127030001   1.18954196E9   110.0 m       5.0 s       38.4%     4.8 h       2.9 h \n",
      "INFO  16:37:20,110 ProgressMeter -     6:134004601   1.19654196E9   110.5 m       5.0 s       38.6%     4.8 h       2.9 h \n",
      "INFO  16:37:50,111 ProgressMeter -     6:139752401   1.20154196E9   111.0 m       5.0 s       38.8%     4.8 h       2.9 h \n",
      "INFO  16:38:20,112 ProgressMeter -     6:146482301   1.20854196E9   111.5 m       5.0 s       39.0%     4.8 h       2.9 h \n",
      "INFO  16:38:50,113 ProgressMeter -     6:152762801   1.21454196E9   112.0 m       5.0 s       39.2%     4.8 h       2.9 h \n",
      "INFO  16:39:30,114 ProgressMeter -     6:160443301   1.22254196E9   112.7 m       5.0 s       39.4%     4.8 h       2.9 h \n",
      "INFO  16:40:00,115 ProgressMeter -     6:166999901   1.22854196E9   113.2 m       5.0 s       39.6%     4.8 h       2.9 h \n",
      "INFO  16:40:30,116 ProgressMeter -       7:1573001   1.234657027E9   113.7 m       5.0 s       39.8%     4.8 h       2.9 h \n",
      "INFO  16:41:00,121 ProgressMeter -       7:5745501   1.238657027E9   114.2 m       5.0 s       40.0%     4.8 h       2.9 h \n",
      "INFO  16:41:30,123 ProgressMeter -      7:10749501   1.243657027E9   114.7 m       5.0 s       40.1%     4.8 h       2.9 h \n",
      "INFO  16:42:00,124 ProgressMeter -      7:18780201   1.251657027E9   115.2 m       5.0 s       40.4%     4.8 h       2.8 h \n",
      "INFO  16:42:30,125 ProgressMeter -      7:25654901   1.258657027E9   115.7 m       5.0 s       40.6%     4.7 h       2.8 h \n",
      "INFO  16:43:00,126 ProgressMeter -      7:30612701   1.263657027E9   116.2 m       5.0 s       40.8%     4.8 h       2.8 h \n",
      "INFO  16:43:30,128 ProgressMeter -      7:36666001   1.269657027E9   116.7 m       5.0 s       41.0%     4.7 h       2.8 h \n",
      "INFO  16:44:00,131 ProgressMeter -      7:42733901   1.275657027E9   117.2 m       5.0 s       41.1%     4.7 h       2.8 h \n",
      "INFO  16:44:30,132 ProgressMeter -      7:48067201   1.281657027E9   117.7 m       5.0 s       41.3%     4.7 h       2.8 h \n",
      "INFO  16:45:00,133 ProgressMeter -      7:56131701   1.289657027E9   118.2 m       5.0 s       41.6%     4.7 h       2.8 h \n",
      "INFO  16:45:30,135 ProgressMeter -      7:64856901   1.297657027E9   118.7 m       5.0 s       41.9%     4.7 h       2.7 h \n",
      "INFO  16:46:10,136 ProgressMeter -      7:72765701   1.305657027E9   119.3 m       5.0 s       42.1%     4.7 h       2.7 h \n",
      "INFO  16:46:40,137 ProgressMeter -      7:77497201   1.310657027E9   119.8 m       5.0 s       42.3%     4.7 h       2.7 h \n",
      "INFO  16:47:10,138 ProgressMeter -      7:85896701   1.318657027E9     2.0 h       5.0 s       42.5%     4.7 h       2.7 h \n",
      "INFO  16:47:40,140 ProgressMeter -      7:92914101   1.325657027E9     2.0 h       5.0 s       42.8%     4.7 h       2.7 h \n",
      "INFO  16:48:10,141 ProgressMeter -      7:99001801   1.332657027E9     2.0 h       5.0 s       43.0%     4.7 h       2.7 h \n",
      "INFO  16:48:40,142 ProgressMeter -     7:102781501   1.335657027E9     2.0 h       5.0 s       43.1%     4.7 h       2.7 h \n",
      "INFO  16:49:10,143 ProgressMeter -     7:108425601   1.341657027E9     2.0 h       5.0 s       43.3%     4.7 h       2.7 h \n",
      "INFO  16:49:40,144 ProgressMeter -     7:114294801   1.347657027E9     2.0 h       5.0 s       43.5%     4.7 h       2.7 h \n",
      "INFO  16:50:10,146 ProgressMeter -     7:121716501   1.354657027E9     2.1 h       5.0 s       43.7%     4.7 h       2.6 h \n",
      "INFO  16:50:40,147 ProgressMeter -     7:128999901   1.361657027E9     2.1 h       5.0 s       43.9%     4.7 h       2.6 h \n",
      "INFO  16:51:10,148 ProgressMeter -     7:134213401   1.367657027E9     2.1 h       5.0 s       44.1%     4.7 h       2.6 h \n",
      "INFO  16:51:40,149 ProgressMeter -     7:139970201   1.372657027E9     2.1 h       5.0 s       44.3%     4.7 h       2.6 h \n",
      "INFO  16:52:10,150 ProgressMeter -     7:145923901   1.378657027E9     2.1 h       5.0 s       44.5%     4.7 h       2.6 h \n",
      "INFO  16:52:40,151 ProgressMeter -     7:151524801   1.384657027E9     2.1 h       5.0 s       44.7%     4.7 h       2.6 h \n",
      "INFO  16:53:10,152 ProgressMeter -     7:157602101   1.390657027E9     2.1 h       5.0 s       44.9%     4.7 h       2.6 h \n",
      "INFO  16:53:40,154 ProgressMeter -       8:5763101   1.39779569E9     2.1 h       5.0 s       45.1%     4.7 h       2.6 h \n",
      "INFO  16:54:10,154 ProgressMeter -      8:12606101   1.40479569E9     2.1 h       5.0 s       45.3%     4.7 h       2.6 h \n",
      "INFO  16:54:40,155 ProgressMeter -      8:18999901   1.41079569E9     2.1 h       5.0 s       45.5%     4.7 h       2.6 h \n",
      "INFO  16:55:10,157 ProgressMeter -      8:23968301   1.41579569E9     2.1 h       5.0 s       45.7%     4.7 h       2.5 h \n",
      "INFO  16:55:40,158 ProgressMeter -      8:29713401   1.42179569E9     2.1 h       5.0 s       45.9%     4.7 h       2.5 h \n",
      "INFO  16:56:10,159 ProgressMeter -      8:37362001   1.42979569E9     2.2 h       5.0 s       46.1%     4.7 h       2.5 h \n",
      "INFO  16:56:40,160 ProgressMeter -      8:42921001   1.43479569E9     2.2 h       5.0 s       46.3%     4.7 h       2.5 h \n",
      "INFO  16:57:10,161 ProgressMeter -      8:50369501   1.44279569E9     2.2 h       5.0 s       46.5%     4.7 h       2.5 h \n",
      "INFO  16:57:40,162 ProgressMeter -      8:57327801   1.44979569E9     2.2 h       5.0 s       46.8%     4.7 h       2.5 h \n",
      "INFO  16:58:10,163 ProgressMeter -      8:64113901   1.45679569E9     2.2 h       5.0 s       47.0%     4.7 h       2.5 h \n",
      "INFO  16:58:40,164 ProgressMeter -      8:71360901   1.46379569E9     2.2 h       5.0 s       47.2%     4.7 h       2.5 h \n",
      "INFO  16:59:10,165 ProgressMeter -      8:79226601   1.47179569E9     2.2 h       5.0 s       47.5%     4.6 h       2.4 h \n",
      "INFO  16:59:40,167 ProgressMeter -      8:87165101   1.47979569E9     2.2 h       5.0 s       47.7%     4.6 h       2.4 h \n",
      "INFO  17:00:10,167 ProgressMeter -      8:95094501   1.48779569E9     2.2 h       5.0 s       48.0%     4.6 h       2.4 h \n",
      "INFO  17:00:40,168 ProgressMeter -     8:101317101   1.49379569E9     2.2 h       5.0 s       48.2%     4.6 h       2.4 h \n",
      "INFO  17:01:10,170 ProgressMeter -     8:107565201   1.49979569E9     2.2 h       5.0 s       48.4%     4.6 h       2.4 h \n",
      "INFO  17:01:50,171 ProgressMeter -     8:118914801   1.51079569E9     2.3 h       5.0 s       48.7%     4.6 h       2.4 h \n",
      "INFO  17:02:20,172 ProgressMeter -     8:125842601   1.51779569E9     2.3 h       5.0 s       49.0%     4.6 h       2.4 h \n",
      "INFO  17:02:50,173 ProgressMeter -     8:132023601   1.52479569E9     2.3 h       5.0 s       49.2%     4.6 h       2.3 h \n",
      "INFO  17:03:20,174 ProgressMeter -     8:137926701   1.52979569E9     2.3 h       5.0 s       49.3%     4.6 h       2.3 h \n",
      "INFO  17:03:50,175 ProgressMeter -     8:144351601   1.53679569E9     2.3 h       5.0 s       49.6%     4.6 h       2.3 h \n",
      "INFO  17:04:20,176 ProgressMeter -       9:2192501   1.541159712E9     2.3 h       5.0 s       49.7%     4.6 h       2.3 h \n",
      "INFO  17:04:50,177 ProgressMeter -       9:7058501   1.546159712E9     2.3 h       5.0 s       49.8%     4.6 h       2.3 h \n",
      "INFO  17:05:20,178 ProgressMeter -      9:12999901   1.551159712E9     2.3 h       5.0 s       50.0%     4.6 h       2.3 h \n",
      "INFO  17:05:50,179 ProgressMeter -      9:17999901   1.556159712E9     2.3 h       5.0 s       50.2%     4.6 h       2.3 h \n",
      "INFO  17:06:20,180 ProgressMeter -      9:22821101   1.561159712E9     2.3 h       5.0 s       50.4%     4.6 h       2.3 h \n",
      "INFO  17:06:50,181 ProgressMeter -      9:28663701   1.567159712E9     2.3 h       5.0 s       50.5%     4.6 h       2.3 h \n",
      "INFO  17:07:20,182 ProgressMeter -      9:33999901   1.572159712E9     2.3 h       5.0 s       50.7%     4.6 h       2.3 h \n",
      "INFO  17:07:50,183 ProgressMeter -      9:37937401   1.576159712E9     2.4 h       5.0 s       50.8%     4.6 h       2.3 h \n",
      "INFO  17:08:20,185 ProgressMeter -      9:42731201   1.581159712E9     2.4 h       5.0 s       51.0%     4.6 h       2.3 h \n",
      "INFO  17:08:50,185 ProgressMeter -      9:47999901   1.586159712E9     2.4 h       5.0 s       51.2%     4.6 h       2.3 h \n",
      "INFO  17:09:20,186 ProgressMeter -      9:53999901   1.592159712E9     2.4 h       5.0 s       51.4%     4.6 h       2.2 h \n",
      "INFO  17:09:50,188 ProgressMeter -      9:64257401   1.603159712E9     2.4 h       5.0 s       51.7%     4.6 h       2.2 h \n",
      "INFO  17:10:30,189 ProgressMeter -      9:74299901   1.613159712E9     2.4 h       5.0 s       52.0%     4.6 h       2.2 h \n",
      "INFO  17:11:00,190 ProgressMeter -      9:79999901   1.618159712E9     2.4 h       5.0 s       52.2%     4.6 h       2.2 h \n",
      "INFO  17:11:30,191 ProgressMeter -      9:85347801   1.624159712E9     2.4 h       5.0 s       52.4%     4.6 h       2.2 h \n",
      "INFO  17:12:00,192 ProgressMeter -      9:90169301   1.629159712E9     2.4 h       5.0 s       52.5%     4.6 h       2.2 h \n",
      "INFO  17:12:30,193 ProgressMeter -      9:94903201   1.633159712E9     2.4 h       5.0 s       52.7%     4.6 h       2.2 h \n",
      "INFO  17:13:00,194 ProgressMeter -      9:98276301   1.637159712E9     2.4 h       5.0 s       52.8%     4.6 h       2.2 h \n",
      "INFO  17:13:30,195 ProgressMeter -     9:101987501   1.640159712E9     2.4 h       5.0 s       52.9%     4.6 h       2.2 h \n",
      "INFO  17:14:00,196 ProgressMeter -     9:104999901   1.643159712E9     2.5 h       5.0 s       53.0%     4.6 h       2.2 h \n",
      "INFO  17:14:30,197 ProgressMeter -     9:108999901   1.647159712E9     2.5 h       5.0 s       53.1%     4.6 h       2.2 h \n",
      "INFO  17:15:00,198 ProgressMeter -     9:112999901   1.651159712E9     2.5 h       5.0 s       53.3%     4.6 h       2.2 h \n",
      "INFO  17:15:30,199 ProgressMeter -     9:116042301   1.655159712E9     2.5 h       5.0 s       53.4%     4.6 h       2.2 h \n",
      "INFO  17:16:00,200 ProgressMeter -     9:120177201   1.659159712E9     2.5 h       5.0 s       53.5%     4.6 h       2.2 h \n",
      "INFO  17:16:30,202 ProgressMeter -     9:124184201   1.663159712E9     2.5 h       5.0 s       53.6%     4.7 h       2.2 h \n",
      "INFO  17:17:00,203 ProgressMeter -     9:127615501   1.666159712E9     2.5 h       5.0 s       53.7%     4.7 h       2.2 h \n",
      "INFO  17:17:30,204 ProgressMeter -     9:130505701   1.669159712E9     2.5 h       5.0 s       53.8%     4.7 h       2.2 h \n",
      "INFO  17:18:00,205 ProgressMeter -     9:132863601   1.671159712E9     2.5 h       5.0 s       53.9%     4.7 h       2.2 h \n",
      "INFO  17:18:30,206 ProgressMeter -     9:134999901   1.673159712E9     2.5 h       5.0 s       54.0%     4.7 h       2.2 h \n",
      "INFO  17:19:00,207 ProgressMeter -     9:137052401   1.676159712E9     2.5 h       5.0 s       54.0%     4.7 h       2.2 h \n",
      "INFO  17:19:30,208 ProgressMeter -     9:139752901   1.678159712E9     2.5 h       5.0 s       54.1%     4.7 h       2.2 h \n",
      "INFO  17:20:00,209 ProgressMeter -       10:294301   1.680373143E9     2.6 h       5.0 s       54.2%     4.7 h       2.2 h \n",
      "INFO  17:20:30,210 ProgressMeter -      10:6347301   1.686373143E9     2.6 h       5.0 s       54.4%     4.7 h       2.1 h \n",
      "INFO  17:21:00,211 ProgressMeter -     10:11371901   1.691373143E9     2.6 h       5.0 s       54.5%     4.7 h       2.1 h \n",
      "INFO  17:21:30,212 ProgressMeter -     10:15174501   1.695373143E9     2.6 h       5.0 s       54.7%     4.7 h       2.1 h \n",
      "INFO  17:22:00,213 ProgressMeter -     10:20871901   1.700373143E9     2.6 h       5.0 s       54.8%     4.7 h       2.1 h \n",
      "INFO  17:22:30,214 ProgressMeter -     10:26783201   1.706373143E9     2.6 h       5.0 s       55.0%     4.7 h       2.1 h \n",
      "INFO  17:23:00,215 ProgressMeter -     10:32003301   1.712373143E9     2.6 h       5.0 s       55.2%     4.7 h       2.1 h \n",
      "INFO  17:23:30,216 ProgressMeter -     10:38245001   1.718373143E9     2.6 h       5.0 s       55.4%     4.7 h       2.1 h \n",
      "INFO  17:24:00,217 ProgressMeter -     10:45999901   1.725373143E9     2.6 h       5.0 s       55.7%     4.7 h       2.1 h \n",
      "INFO  17:24:30,218 ProgressMeter -     10:52135501   1.732373143E9     2.6 h       5.0 s       55.9%     4.7 h       2.1 h \n",
      "INFO  17:25:00,219 ProgressMeter -     10:60001501   1.740373143E9     2.6 h       5.0 s       56.1%     4.7 h       2.1 h \n",
      "INFO  17:25:30,220 ProgressMeter -     10:67841901   1.747373143E9     2.6 h       5.0 s       56.4%     4.7 h       2.0 h \n",
      "INFO  17:26:00,221 ProgressMeter -     10:73115901   1.753373143E9     2.7 h       5.0 s       56.5%     4.7 h       2.0 h \n",
      "INFO  17:26:30,222 ProgressMeter -     10:77109201   1.757373143E9     2.7 h       5.0 s       56.7%     4.7 h       2.0 h \n",
      "INFO  17:27:00,223 ProgressMeter -     10:81988501   1.761373143E9     2.7 h       5.0 s       56.8%     4.7 h       2.0 h \n",
      "INFO  17:27:30,224 ProgressMeter -     10:89221801   1.769373143E9     2.7 h       5.0 s       57.1%     4.7 h       2.0 h \n",
      "INFO  17:28:00,225 ProgressMeter -     10:94071201   1.774373143E9     2.7 h       5.0 s       57.2%     4.7 h       2.0 h \n",
      "INFO  17:28:30,226 ProgressMeter -     10:98859501   1.778373143E9     2.7 h       5.0 s       57.4%     4.7 h       2.0 h \n",
      "INFO  17:29:00,228 ProgressMeter -    10:103607101   1.783373143E9     2.7 h       5.0 s       57.5%     4.7 h     119.8 m \n",
      "INFO  17:29:30,228 ProgressMeter -    10:108182501   1.788373143E9     2.7 h       5.0 s       57.7%     4.7 h     119.4 m \n",
      "INFO  17:30:00,229 ProgressMeter -    10:115807801   1.795373143E9     2.7 h       5.0 s       57.9%     4.7 h     118.6 m \n",
      "INFO  17:30:30,230 ProgressMeter -    10:121990101   1.801373143E9     2.7 h       5.0 s       58.1%     4.7 h     118.0 m \n",
      "INFO  17:31:00,231 ProgressMeter -    10:127025901   1.807373143E9     2.7 h       5.0 s       58.3%     4.7 h     117.6 m \n",
      "INFO  17:31:30,233 ProgressMeter -    10:134094701   1.814373143E9     2.7 h       5.0 s       58.5%     4.7 h     116.8 m \n",
      "INFO  17:32:00,234 ProgressMeter -      11:2999901   1.81790789E9     2.8 h       5.0 s       58.6%     4.7 h     116.5 m \n",
      "INFO  17:32:30,241 ProgressMeter -      11:7999901   1.82290789E9     2.8 h       5.0 s       58.8%     4.7 h     116.1 m \n",
      "INFO  17:33:00,242 ProgressMeter -     11:12940601   1.82790789E9     2.8 h       5.0 s       59.0%     4.7 h     115.7 m \n",
      "INFO  17:33:30,243 ProgressMeter -     11:18576001   1.83390789E9     2.8 h       5.0 s       59.1%     4.7 h     115.1 m \n",
      "INFO  17:34:00,244 ProgressMeter -     11:26714901   1.84190789E9     2.8 h       5.0 s       59.4%     4.7 h     114.2 m \n",
      "INFO  17:34:30,245 ProgressMeter -     11:33908501   1.84890789E9     2.8 h       5.0 s       59.6%     4.7 h     113.5 m \n",
      "INFO  17:35:00,246 ProgressMeter -     11:41342501   1.85690789E9     2.8 h       5.0 s       59.9%     4.7 h     112.7 m \n",
      "INFO  17:35:30,247 ProgressMeter -     11:47366901   1.86290789E9     2.8 h       5.0 s       60.1%     4.7 h     112.1 m \n",
      "INFO  17:36:00,248 ProgressMeter -     11:53650401   1.86890789E9     2.8 h       5.0 s       60.3%     4.7 h     111.5 m \n",
      "INFO  17:36:30,250 ProgressMeter -     11:60770601   1.87590789E9     2.8 h       5.0 s       60.5%     4.7 h     110.8 m \n",
      "INFO  17:37:00,251 ProgressMeter -     11:64627201   1.87990789E9     2.8 h       5.0 s       60.6%     4.7 h     110.5 m \n",
      "INFO  17:37:30,252 ProgressMeter -     11:67271301   1.88290789E9     2.8 h       5.0 s       60.7%     4.7 h     110.4 m \n",
      "INFO  17:38:00,378 ProgressMeter -     11:71743801   1.88690789E9     2.9 h       5.0 s       60.9%     4.7 h     110.1 m \n",
      "INFO  17:38:30,379 ProgressMeter -     11:75579001   1.89090789E9     2.9 h       5.0 s       61.0%     4.7 h     109.8 m \n",
      "INFO  17:39:00,380 ProgressMeter -     11:80404801   1.89590789E9     2.9 h       5.0 s       61.1%     4.7 h     109.4 m \n",
      "INFO  17:39:30,381 ProgressMeter -     11:87347201   1.90290789E9     2.9 h       5.0 s       61.4%     4.7 h     108.7 m \n",
      "INFO  17:40:00,383 ProgressMeter -     11:94777801   1.90990789E9     2.9 h       5.0 s       61.6%     4.7 h     108.0 m \n",
      "INFO  17:40:30,383 ProgressMeter -    11:101779101   1.91690789E9     2.9 h       5.0 s       61.8%     4.7 h     107.2 m \n",
      "INFO  17:41:00,384 ProgressMeter -    11:108944301   1.92390789E9     2.9 h       5.0 s       62.1%     4.7 h     106.5 m \n",
      "INFO  17:41:30,385 ProgressMeter -    11:115057701   1.93090789E9     2.9 h       5.0 s       62.3%     4.7 h     105.9 m \n",
      "INFO  17:42:00,386 ProgressMeter -    11:119999901   1.93490789E9     2.9 h       5.0 s       62.4%     4.7 h     105.5 m \n",
      "INFO  17:42:30,387 ProgressMeter -    11:125308201   1.94090789E9     2.9 h       5.0 s       62.6%     4.7 h     105.0 m \n",
      "INFO  17:43:00,388 ProgressMeter -    11:131108601   1.94690789E9     2.9 h       5.0 s       62.8%     4.7 h     104.5 m \n",
      "INFO  17:43:30,389 ProgressMeter -      12:2770701   1.952914406E9     2.9 h       5.0 s       63.0%     4.7 h     103.8 m \n",
      "INFO  17:44:00,390 ProgressMeter -      12:7735701   1.957914406E9     3.0 h       5.0 s       63.1%     4.7 h     103.4 m \n",
      "INFO  17:44:30,391 ProgressMeter -     12:12316701   1.962914406E9     3.0 h       5.0 s       63.3%     4.7 h     103.0 m \n",
      "INFO  17:45:00,392 ProgressMeter -     12:18638201   1.968914406E9     3.0 h       5.0 s       63.5%     4.7 h     102.4 m \n",
      "INFO  17:45:30,393 ProgressMeter -     12:26106101   1.976914406E9     3.0 h       5.0 s       63.7%     4.7 h     101.6 m \n",
      "INFO  17:46:00,394 ProgressMeter -     12:31412901   1.981914406E9     3.0 h       5.0 s       63.9%     4.7 h     101.2 m \n",
      "INFO  17:46:30,395 ProgressMeter -     12:39111401   1.989914406E9     3.0 h       5.0 s       64.2%     4.7 h     100.4 m \n",
      "INFO  17:47:00,396 ProgressMeter -     12:46632501   1.996914406E9     3.0 h       5.0 s       64.4%     4.7 h      99.6 m \n",
      "INFO  17:47:30,397 ProgressMeter -     12:50999901   2.000914406E9     3.0 h       5.0 s       64.5%     4.7 h      99.3 m \n",
      "INFO  17:48:00,398 ProgressMeter -     12:54999901   2.004914406E9     3.0 h       5.0 s       64.7%     4.7 h      99.0 m \n",
      "INFO  17:48:30,400 ProgressMeter -     12:58906101   2.008914406E9     3.0 h       5.0 s       64.8%     4.7 h      98.7 m \n",
      "INFO  17:49:00,401 ProgressMeter -     12:65196901   2.015914406E9     3.0 h       5.0 s       65.0%     4.7 h      98.1 m \n",
      "INFO  17:49:30,405 ProgressMeter -     12:70927201   2.020914406E9     3.0 h       5.0 s       65.2%     4.7 h      97.6 m \n",
      "INFO  17:50:00,406 ProgressMeter -     12:77679501   2.027914406E9     3.1 h       5.0 s       65.4%     4.7 h      96.9 m \n",
      "INFO  17:50:30,407 ProgressMeter -     12:85261301   2.035914406E9     3.1 h       5.0 s       65.6%     4.7 h      96.1 m \n",
      "INFO  17:51:00,408 ProgressMeter -     12:93416401   2.043914406E9     3.1 h       5.0 s       65.9%     4.7 h      95.3 m \n",
      "INFO  17:51:30,409 ProgressMeter -     12:99290101   2.049914406E9     3.1 h       5.0 s       66.1%     4.7 h      94.7 m \n",
      "INFO  17:52:00,410 ProgressMeter -    12:105311401   2.055914406E9     3.1 h       5.0 s       66.3%     4.7 h      94.2 m \n",
      "INFO  17:52:30,411 ProgressMeter -    12:110120001   2.060914406E9     3.1 h       5.0 s       66.4%     4.7 h      93.8 m \n",
      "INFO  17:53:00,412 ProgressMeter -    12:113820501   2.063914406E9     3.1 h       5.0 s       66.6%     4.7 h      93.5 m \n",
      "INFO  17:53:30,413 ProgressMeter -    12:119772201   2.069914406E9     3.1 h       5.0 s       66.8%     4.7 h      93.0 m \n",
      "INFO  17:54:00,414 ProgressMeter -    12:123776101   2.073914406E9     3.1 h       5.0 s       66.9%     4.7 h      92.7 m \n",
      "INFO  17:54:30,415 ProgressMeter -    12:128675901   2.078914406E9     3.1 h       5.0 s       67.0%     4.7 h      92.2 m \n",
      "INFO  17:55:00,416 ProgressMeter -      13:2144401   2.086766301E9     3.1 h       5.0 s       67.3%     4.7 h      91.5 m \n",
      "INFO  17:55:30,417 ProgressMeter -     13:12988601   2.096766301E9     3.1 h       5.0 s       67.6%     4.6 h      90.3 m \n",
      "INFO  17:56:00,418 ProgressMeter -     13:21919901   2.105766301E9     3.2 h       5.0 s       67.9%     4.6 h      89.4 m \n",
      "INFO  17:56:30,420 ProgressMeter -     13:27986901   2.111766301E9     3.2 h       5.0 s       68.1%     4.6 h      88.8 m \n",
      "INFO  17:57:00,421 ProgressMeter -     13:32524601   2.116766301E9     3.2 h       5.0 s       68.3%     4.6 h      88.4 m \n",
      "INFO  17:57:30,423 ProgressMeter -     13:36999901   2.120766301E9     3.2 h       5.0 s       68.4%     4.6 h      88.1 m \n",
      "INFO  17:58:00,424 ProgressMeter -     13:41165601   2.125766301E9     3.2 h       5.0 s       68.5%     4.6 h      87.8 m \n",
      "INFO  17:58:30,425 ProgressMeter -     13:45009101   2.129766301E9     3.2 h       5.0 s       68.7%     4.7 h      87.5 m \n",
      "INFO  17:59:00,426 ProgressMeter -     13:48999901   2.132766301E9     3.2 h       5.0 s       68.8%     4.7 h      87.2 m \n",
      "INFO  17:59:30,427 ProgressMeter -     13:51999901   2.135766301E9     3.2 h       5.0 s       68.9%     4.7 h      87.0 m \n",
      "INFO  18:00:00,428 ProgressMeter -     13:55193801   2.139766301E9     3.2 h       5.0 s       69.0%     4.7 h      86.8 m \n",
      "INFO  18:00:30,429 ProgressMeter -     13:59066301   2.143766301E9     3.2 h       5.0 s       69.1%     4.7 h      86.5 m \n",
      "INFO  18:01:00,430 ProgressMeter -     13:63675001   2.147766301E9     3.2 h       5.0 s       69.3%     4.7 h      86.2 m \n",
      "INFO  18:01:30,431 ProgressMeter -     13:68379001   2.152766301E9     3.2 h       5.0 s       69.4%     4.7 h      85.8 m \n",
      "INFO  18:02:00,432 ProgressMeter -     13:75888301   2.159766301E9     3.3 h       5.0 s       69.7%     4.7 h      85.0 m \n",
      "INFO  18:02:30,434 ProgressMeter -     13:81846901   2.165766301E9     3.3 h       5.0 s       69.9%     4.7 h      84.5 m \n",
      "INFO  18:03:10,435 ProgressMeter -     13:90999901   2.174766301E9     3.3 h       5.0 s       70.1%     4.7 h      83.6 m \n",
      "INFO  18:03:50,436 ProgressMeter -     13:98999901   2.182766301E9     3.3 h       5.0 s       70.4%     4.7 h      82.8 m \n",
      "INFO  18:04:30,437 ProgressMeter -    13:106438201   2.190766301E9     3.3 h       5.0 s       70.6%     4.7 h      82.1 m \n",
      "INFO  18:05:10,438 ProgressMeter -    13:112999901   2.196766301E9     3.3 h       5.0 s       70.9%     4.7 h      81.6 m \n",
      "INFO  18:05:40,439 ProgressMeter -      14:1690801   2.200936179E9     3.3 h       5.0 s       71.0%     4.7 h      81.3 m \n",
      "INFO  18:06:10,440 ProgressMeter -     14:12656201   2.211936179E9     3.3 h       5.0 s       71.3%     4.7 h      80.1 m \n",
      "INFO  18:06:40,441 ProgressMeter -     14:21860401   2.220936179E9     3.3 h       5.0 s       71.6%     4.6 h      79.1 m \n",
      "INFO  18:07:10,442 ProgressMeter -     14:26117801   2.225936179E9     3.3 h       5.0 s       71.8%     4.7 h      78.8 m \n",
      "INFO  18:07:40,443 ProgressMeter -     14:31391801   2.230936179E9     3.3 h       5.0 s       71.9%     4.7 h      78.3 m \n",
      "INFO  18:08:10,444 ProgressMeter -     14:35507101   2.234936179E9     3.4 h       5.0 s       72.1%     4.7 h      78.0 m \n",
      "INFO  18:08:40,445 ProgressMeter -     14:39505501   2.238936179E9     3.4 h       5.0 s       72.2%     4.7 h      77.7 m \n",
      "INFO  18:09:10,447 ProgressMeter -     14:43999901   2.242936179E9     3.4 h       5.0 s       72.3%     4.7 h      77.4 m \n",
      "INFO  18:09:50,447 ProgressMeter -     14:51070601   2.250936179E9     3.4 h       5.0 s       72.6%     4.7 h      76.7 m \n",
      "INFO  18:10:20,448 ProgressMeter -     14:55778001   2.254936179E9     3.4 h       5.0 s       72.7%     4.7 h      76.3 m \n",
      "INFO  18:10:50,450 ProgressMeter -     14:59999901   2.258936179E9     3.4 h       5.0 s       72.9%     4.7 h      76.0 m \n",
      "INFO  18:11:30,451 ProgressMeter -     14:64999901   2.263936179E9     3.4 h       5.0 s       73.0%     4.7 h      75.6 m \n",
      "INFO  18:12:00,452 ProgressMeter -     14:68549501   2.267936179E9     3.4 h       5.0 s       73.1%     4.7 h      75.4 m \n",
      "INFO  18:12:30,453 ProgressMeter -     14:71954101   2.270936179E9     3.4 h       5.0 s       73.2%     4.7 h      75.1 m \n",
      "INFO  18:13:00,454 ProgressMeter -     14:74999901   2.273936179E9     3.4 h       5.0 s       73.3%     4.7 h      74.9 m \n",
      "INFO  18:13:30,455 ProgressMeter -     14:77253201   2.276936179E9     3.4 h       5.0 s       73.4%     4.7 h      74.8 m \n",
      "INFO  18:14:00,456 ProgressMeter -     14:79999901   2.278936179E9     3.5 h       5.0 s       73.5%     4.7 h      74.7 m \n",
      "INFO  18:14:30,457 ProgressMeter -     14:82999901   2.281936179E9     3.5 h       5.0 s       73.6%     4.7 h      74.5 m \n",
      "INFO  18:15:00,458 ProgressMeter -     14:86970901   2.285936179E9     3.5 h       5.0 s       73.7%     4.7 h      74.2 m \n",
      "INFO  18:15:30,459 ProgressMeter -     14:89997501   2.288936179E9     3.5 h       5.0 s       73.8%     4.7 h      74.0 m \n",
      "INFO  18:16:00,460 ProgressMeter -     14:92524101   2.291936179E9     3.5 h       5.0 s       73.9%     4.7 h      73.8 m \n",
      "INFO  18:16:30,461 ProgressMeter -     14:95055801   2.294936179E9     3.5 h       5.0 s       74.0%     4.7 h      73.7 m \n",
      "INFO  18:17:00,462 ProgressMeter -     14:98999901   2.297936179E9     3.5 h       5.0 s       74.1%     4.7 h      73.4 m \n",
      "INFO  18:17:30,463 ProgressMeter -    14:102555101   2.301936179E9     3.5 h       5.0 s       74.2%     4.7 h      73.1 m \n",
      "INFO  18:18:00,464 ProgressMeter -    14:104999901   2.303936179E9     3.5 h       5.0 s       74.3%     4.7 h      73.0 m \n",
      "INFO  18:18:30,465 ProgressMeter -      15:1916801   2.308285719E9     3.5 h       5.0 s       74.4%     4.7 h      72.7 m \n",
      "INFO  18:19:00,467 ProgressMeter -     15:12879001   2.319285719E9     3.5 h       5.0 s       74.8%     4.7 h      71.5 m \n",
      "INFO  18:19:30,468 ProgressMeter -     15:22967801   2.329285719E9     3.5 h       5.0 s       75.1%     4.7 h      70.4 m \n",
      "INFO  18:20:00,469 ProgressMeter -     15:29288101   2.336285719E9     3.6 h       5.0 s       75.3%     4.7 h      69.8 m \n",
      "INFO  18:20:30,470 ProgressMeter -     15:34651401   2.341285719E9     3.6 h       5.0 s       75.5%     4.7 h      69.3 m \n",
      "INFO  18:21:00,471 ProgressMeter -     15:39583201   2.346285719E9     3.6 h       5.0 s       75.7%     4.7 h      68.9 m \n",
      "INFO  18:21:30,472 ProgressMeter -     15:42984001   2.349285719E9     3.6 h       5.0 s       75.8%     4.7 h      68.6 m \n",
      "INFO  18:22:00,474 ProgressMeter -     15:45808701   2.352285719E9     3.6 h       5.0 s       75.9%     4.7 h      68.5 m \n",
      "INFO  18:22:30,475 ProgressMeter -     15:48999901   2.355285719E9     3.6 h       5.0 s       76.0%     4.7 h      68.2 m \n",
      "INFO  18:23:10,476 ProgressMeter -     15:52999901   2.359285719E9     3.6 h       5.0 s       76.1%     4.7 h      68.0 m \n",
      "INFO  18:23:40,477 ProgressMeter -     15:55999901   2.362285719E9     3.6 h       5.0 s       76.2%     4.7 h      67.8 m \n",
      "INFO  18:24:10,479 ProgressMeter -     15:58999901   2.365285719E9     3.6 h       5.0 s       76.3%     4.7 h      67.6 m \n",
      "INFO  18:24:40,480 ProgressMeter -     15:61167501   2.368285719E9     3.6 h       5.0 s       76.4%     4.8 h      67.4 m \n",
      "INFO  18:25:10,481 ProgressMeter -     15:63999901   2.370285719E9     3.6 h       5.0 s       76.4%     4.8 h      67.3 m \n",
      "INFO  18:25:40,482 ProgressMeter -     15:66999901   2.373285719E9     3.6 h       5.0 s       76.5%     4.8 h      67.1 m \n",
      "INFO  18:26:10,483 ProgressMeter -     15:69026301   2.376285719E9     3.7 h       5.0 s       76.6%     4.8 h      67.0 m \n",
      "INFO  18:26:40,484 ProgressMeter -     15:71999901   2.378285719E9     3.7 h       5.0 s       76.7%     4.8 h      66.8 m \n",
      "INFO  18:27:10,485 ProgressMeter -     15:74907701   2.381285719E9     3.7 h       5.0 s       76.8%     4.8 h      66.6 m \n",
      "INFO  18:27:40,486 ProgressMeter -     15:76999901   2.383285719E9     3.7 h       5.0 s       76.9%     4.8 h      66.5 m \n",
      "INFO  18:28:10,487 ProgressMeter -     15:79180401   2.386285719E9     3.7 h       5.0 s       76.9%     4.8 h      66.3 m \n",
      "INFO  18:28:40,488 ProgressMeter -     15:81999901   2.388285719E9     3.7 h       5.0 s       77.0%     4.8 h      66.2 m \n",
      "INFO  18:29:10,490 ProgressMeter -     15:84545501   2.391285719E9     3.7 h       5.0 s       77.1%     4.8 h      66.0 m \n",
      "INFO  18:29:40,490 ProgressMeter -     15:86999901   2.393285719E9     3.7 h       5.0 s       77.2%     4.8 h      65.8 m \n",
      "INFO  18:30:10,491 ProgressMeter -     15:90274401   2.397285719E9     3.7 h       5.0 s       77.3%     4.8 h      65.6 m \n",
      "INFO  18:30:40,493 ProgressMeter -     15:92999901   2.399285719E9     3.7 h       5.0 s       77.4%     4.8 h      65.4 m \n",
      "INFO  18:31:10,494 ProgressMeter -     15:96671801   2.403285719E9     3.7 h       5.0 s       77.5%     4.8 h      65.1 m \n",
      "INFO  18:31:40,495 ProgressMeter -    15:100266401   2.407285719E9     3.7 h       5.0 s       77.6%     4.8 h      64.8 m \n",
      "INFO  18:32:20,496 ProgressMeter -      16:2225501   2.411817111E9     3.8 h       5.0 s       77.8%     4.8 h      64.5 m \n",
      "INFO  18:32:50,497 ProgressMeter -      16:5228001   2.414817111E9     3.8 h       5.0 s       77.9%     4.8 h      64.3 m \n",
      "INFO  18:33:20,498 ProgressMeter -      16:8999901   2.417817111E9     3.8 h       5.0 s       78.0%     4.8 h      64.0 m \n",
      "INFO  18:33:50,505 ProgressMeter -     16:12018101   2.421817111E9     3.8 h       5.0 s       78.1%     4.8 h      63.7 m \n",
      "INFO  18:34:20,506 ProgressMeter -     16:15999901   2.424817111E9     3.8 h       5.0 s       78.2%     4.8 h      63.4 m \n",
      "INFO  18:34:50,507 ProgressMeter -     16:20434401   2.429817111E9     3.8 h       5.0 s       78.3%     4.9 h      63.0 m \n",
      "INFO  18:35:20,508 ProgressMeter -     16:24226701   2.433817111E9     3.8 h       5.0 s       78.5%     4.9 h      62.7 m \n",
      "INFO  18:35:50,509 ProgressMeter -     16:27999901   2.436817111E9     3.8 h       5.0 s       78.6%     4.9 h      62.4 m \n",
      "INFO  18:36:20,510 ProgressMeter -     16:30939801   2.439817111E9     3.8 h       5.0 s       78.7%     4.9 h      62.2 m \n",
      "INFO  18:36:50,511 ProgressMeter -     16:33999901   2.442817111E9     3.8 h       5.0 s       78.8%     4.9 h      61.9 m \n",
      "INFO  18:37:20,512 ProgressMeter -     16:37999901   2.446817111E9     3.8 h       5.0 s       78.9%     4.9 h      61.6 m \n",
      "INFO  18:37:50,513 ProgressMeter -     16:45182101   2.454817111E9     3.9 h       5.0 s       79.1%     4.9 h      60.9 m \n",
      "INFO  18:38:20,514 ProgressMeter -     16:51580901   2.460817111E9     3.9 h       5.0 s       79.4%     4.9 h      60.2 m \n",
      "INFO  18:38:50,515 ProgressMeter -     16:56907501   2.465817111E9     3.9 h       5.0 s       79.5%     4.9 h      59.7 m \n",
      "INFO  18:39:20,516 ProgressMeter -     16:60999901   2.469817111E9     3.9 h       5.0 s       79.7%     4.9 h      59.4 m \n",
      "INFO  18:39:50,517 ProgressMeter -     16:66646401   2.475817111E9     3.9 h       5.0 s       79.8%     4.9 h      58.8 m \n",
      "INFO  18:40:20,518 ProgressMeter -     16:69960901   2.478817111E9     3.9 h       5.0 s       79.9%     4.9 h      58.6 m \n",
      "INFO  18:41:00,519 ProgressMeter -     16:74399401   2.483817111E9     3.9 h       5.0 s       80.1%     4.9 h      58.2 m \n",
      "INFO  18:41:30,520 ProgressMeter -     16:77999901   2.486817111E9     3.9 h       5.0 s       80.2%     4.9 h      57.9 m \n",
      "INFO  18:42:00,521 ProgressMeter -     16:81668801   2.490817111E9     3.9 h       5.0 s       80.3%     4.9 h      57.6 m \n",
      "INFO  18:42:30,522 ProgressMeter -     16:85748901   2.494817111E9     3.9 h       5.0 s       80.5%     4.9 h      57.2 m \n",
      "INFO  18:43:00,523 ProgressMeter -     16:89353401   2.498817111E9     3.9 h       5.0 s       80.6%     4.9 h      56.9 m \n",
      "INFO  18:43:30,524 ProgressMeter -      17:1999901   2.501171864E9     3.9 h       5.0 s       80.7%     4.9 h      56.7 m \n",
      "INFO  18:44:00,525 ProgressMeter -      17:5113901   2.505171864E9     4.0 h       5.0 s       80.8%     4.9 h      56.5 m \n",
      "INFO  18:44:30,526 ProgressMeter -      17:8843501   2.508171864E9     4.0 h       5.0 s       80.9%     4.9 h      56.2 m \n",
      "INFO  18:45:00,527 ProgressMeter -     17:15105501   2.515171864E9     4.0 h       5.0 s       81.1%     4.9 h      55.5 m \n",
      "INFO  18:45:30,528 ProgressMeter -     17:19816101   2.519171864E9     4.0 h       5.0 s       81.2%     4.9 h      55.1 m \n",
      "INFO  18:46:00,529 ProgressMeter -     17:26597101   2.526171864E9     4.0 h       5.0 s       81.5%     4.9 h      54.4 m \n",
      "INFO  18:46:30,530 ProgressMeter -     17:29999901   2.529171864E9     4.0 h       5.0 s       81.6%     4.9 h      54.1 m \n",
      "INFO  18:47:00,532 ProgressMeter -     17:34866201   2.534171864E9     4.0 h       5.0 s       81.7%     4.9 h      53.7 m \n",
      "INFO  18:47:30,532 ProgressMeter -     17:39085701   2.539171864E9     4.0 h       5.0 s       81.9%     4.9 h      53.3 m \n",
      "INFO  18:48:00,533 ProgressMeter -     17:42999901   2.542171864E9     4.0 h       5.0 s       82.0%     4.9 h      53.0 m \n",
      "INFO  18:48:30,534 ProgressMeter -     17:46888001   2.546171864E9     4.0 h       5.0 s       82.1%     4.9 h      52.6 m \n",
      "INFO  18:49:00,535 ProgressMeter -     17:52167501   2.552171864E9     4.0 h       5.0 s       82.3%     4.9 h      52.1 m \n",
      "INFO  18:49:30,537 ProgressMeter -     17:57999901   2.557171864E9     4.0 h       5.0 s       82.5%     4.9 h      51.6 m \n",
      "INFO  18:50:00,538 ProgressMeter -     17:62624401   2.562171864E9     4.1 h       5.0 s       82.6%     4.9 h      51.1 m \n",
      "INFO  18:50:30,539 ProgressMeter -     17:67097401   2.567171864E9     4.1 h       5.0 s       82.8%     4.9 h      50.7 m \n",
      "INFO  18:51:10,545 ProgressMeter -     17:74457901   2.574171864E9     4.1 h       5.0 s       83.0%     4.9 h      50.0 m \n",
      "INFO  18:51:40,546 ProgressMeter -     17:78176401   2.578171864E9     4.1 h       5.0 s       83.1%     4.9 h      49.7 m \n",
      "INFO  18:52:10,547 ProgressMeter -       18:248601   2.581367074E9     4.1 h       5.0 s       83.2%     4.9 h      49.4 m \n",
      "INFO  18:52:40,548 ProgressMeter -      18:7021101   2.588367074E9     4.1 h       5.0 s       83.4%     4.9 h      48.8 m \n",
      "INFO  18:53:10,549 ProgressMeter -     18:12999901   2.593367074E9     4.1 h       5.0 s       83.6%     4.9 h      48.2 m \n",
      "INFO  18:53:40,550 ProgressMeter -     18:19323701   2.600367074E9     4.1 h       5.0 s       83.8%     4.9 h      47.6 m \n",
      "INFO  18:54:10,551 ProgressMeter -     18:26592401   2.607367074E9     4.1 h       5.0 s       84.1%     4.9 h      46.8 m \n",
      "INFO  18:54:40,552 ProgressMeter -     18:33939201   2.614367074E9     4.1 h       5.0 s       84.3%     4.9 h      46.1 m \n",
      "INFO  18:55:10,553 ProgressMeter -     18:42405701   2.623367074E9     4.1 h       5.0 s       84.6%     4.9 h      45.2 m \n",
      "INFO  18:55:40,554 ProgressMeter -     18:48705301   2.629367074E9     4.1 h       5.0 s       84.8%     4.9 h      44.6 m \n",
      "INFO  18:56:10,555 ProgressMeter -     18:56133401   2.637367074E9     4.2 h       5.0 s       85.0%     4.9 h      43.9 m \n",
      "INFO  18:56:40,556 ProgressMeter -     18:63217201   2.644367074E9     4.2 h       5.0 s       85.3%     4.9 h      43.2 m \n",
      "INFO  18:57:10,557 ProgressMeter -     18:71290901   2.652367074E9     4.2 h       5.0 s       85.5%     4.9 h      42.4 m \n",
      "INFO  18:57:40,558 ProgressMeter -     18:77999901   2.658367074E9     4.2 h       5.0 s       85.7%     4.9 h      41.7 m \n",
      "INFO  18:58:10,559 ProgressMeter -      19:3233901   2.662444322E9     4.2 h       5.0 s       85.8%     4.9 h      41.4 m \n",
      "INFO  18:58:40,560 ProgressMeter -      19:6122501   2.665444322E9     4.2 h       5.0 s       85.9%     4.9 h      41.2 m \n",
      "INFO  18:59:10,561 ProgressMeter -      19:9791801   2.668444322E9     4.2 h       5.0 s       86.1%     4.9 h      40.9 m \n",
      "INFO  18:59:40,568 ProgressMeter -     19:12650101   2.671444322E9     4.2 h       5.0 s       86.1%     4.9 h      40.7 m \n",
      "INFO  19:00:10,569 ProgressMeter -     19:15772101   2.674444322E9     4.2 h       5.0 s       86.2%     4.9 h      40.4 m \n",
      "INFO  19:00:40,570 ProgressMeter -     19:18959701   2.677444322E9     4.2 h       5.0 s       86.3%     4.9 h      40.1 m \n",
      "INFO  19:01:10,572 ProgressMeter -     19:22468901   2.681444322E9     4.2 h       5.0 s       86.5%     4.9 h      39.8 m \n",
      "INFO  19:01:50,573 ProgressMeter -     19:33063201   2.692444322E9     4.3 h       5.0 s       86.8%     4.9 h      38.8 m \n",
      "INFO  19:02:20,574 ProgressMeter -     19:37697301   2.696444322E9     4.3 h       5.0 s       87.0%     4.9 h      38.3 m \n",
      "INFO  19:02:50,575 ProgressMeter -     19:40999901   2.699444322E9     4.3 h       5.0 s       87.1%     4.9 h      38.0 m \n",
      "INFO  19:03:20,576 ProgressMeter -     19:44599201   2.703444322E9     4.3 h       5.0 s       87.2%     4.9 h      37.7 m \n",
      "INFO  19:03:50,577 ProgressMeter -     19:47999901   2.706444322E9     4.3 h       5.0 s       87.3%     4.9 h      37.4 m \n",
      "INFO  19:04:20,578 ProgressMeter -     19:51109401   2.710444322E9     4.3 h       5.0 s       87.4%     4.9 h      37.2 m \n",
      "INFO  19:04:50,579 ProgressMeter -     19:54852501   2.713444322E9     4.3 h       5.0 s       87.5%     4.9 h      36.8 m \n",
      "INFO  19:05:20,580 ProgressMeter -     19:58417501   2.717444322E9     4.3 h       5.0 s       87.6%     4.9 h      36.5 m \n",
      "INFO  19:05:50,585 ProgressMeter -      20:3627301   2.721573305E9     4.3 h       5.0 s       87.8%     4.9 h      36.1 m \n",
      "INFO  19:06:20,586 ProgressMeter -      20:8916501   2.726573305E9     4.3 h       5.0 s       87.9%     4.9 h      35.6 m \n",
      "INFO  19:06:50,587 ProgressMeter -     20:16999901   2.734573305E9     4.3 h       5.0 s       88.2%     4.9 h      34.8 m \n",
      "INFO  19:07:20,588 ProgressMeter -     20:23479401   2.741573305E9     4.3 h       5.0 s       88.4%     4.9 h      34.2 m \n",
      "INFO  19:07:50,589 ProgressMeter -     20:30265501   2.748573305E9     4.4 h       5.0 s       88.6%     4.9 h      33.5 m \n",
      "INFO  19:08:20,590 ProgressMeter -     20:34265501   2.752573305E9     4.4 h       5.0 s       88.7%     4.9 h      33.1 m \n",
      "INFO  19:08:50,591 ProgressMeter -     20:38750401   2.756573305E9     4.4 h       5.0 s       88.9%     4.9 h      32.7 m \n",
      "INFO  19:09:20,598 ProgressMeter -     20:44996901   2.762573305E9     4.4 h       5.0 s       89.1%     4.9 h      32.1 m \n",
      "INFO  19:09:50,600 ProgressMeter -     20:49535401   2.767573305E9     4.4 h       5.0 s       89.2%     4.9 h      31.7 m \n",
      "INFO  19:10:20,600 ProgressMeter -     20:56513201   2.774573305E9     4.4 h       5.0 s       89.5%     4.9 h      31.0 m \n",
      "INFO  19:10:50,625 ProgressMeter -     20:61907801   2.779573305E9     4.4 h       5.0 s       89.6%     4.9 h      30.5 m \n",
      "INFO  19:11:20,626 ProgressMeter -      21:6922201   2.787598825E9     4.4 h       5.0 s       89.9%     4.9 h      29.7 m \n",
      "INFO  19:11:50,627 ProgressMeter -     21:16364101   2.797598825E9     4.4 h       5.0 s       90.2%     4.9 h      28.8 m \n",
      "INFO  19:12:20,629 ProgressMeter -     21:23758301   2.804598825E9     4.4 h       5.0 s       90.4%     4.9 h      28.1 m \n",
      "INFO  19:13:00,630 ProgressMeter -     21:32999901   2.813598825E9     4.4 h       5.0 s       90.7%     4.9 h      27.2 m \n",
      "INFO  19:13:30,631 ProgressMeter -     21:36999901   2.817598825E9     4.4 h       5.0 s       90.9%     4.9 h      26.8 m \n",
      "INFO  19:14:00,632 ProgressMeter -     21:40999901   2.821598825E9     4.5 h       5.0 s       91.0%     4.9 h      26.4 m \n",
      "INFO  19:14:30,633 ProgressMeter -     21:44842101   2.825598825E9     4.5 h       5.0 s       91.1%     4.9 h      26.1 m \n",
      "INFO  19:15:00,634 ProgressMeter -     21:47856801   2.828598825E9     4.5 h       5.0 s       91.2%     4.9 h      25.8 m \n",
      "INFO  19:15:30,635 ProgressMeter -      22:8399001   2.83772872E9     4.5 h       5.0 s       91.5%     4.9 h      25.0 m \n",
      "INFO  19:16:00,636 ProgressMeter -     22:18053401   2.84772872E9     4.5 h       5.0 s       91.8%     4.9 h      24.0 m \n",
      "INFO  19:16:30,637 ProgressMeter -     22:22274301   2.85172872E9     4.5 h       5.0 s       91.9%     4.9 h      23.6 m \n",
      "INFO  19:17:00,638 ProgressMeter -     22:25999901   2.85472872E9     4.5 h       5.0 s       92.1%     4.9 h      23.3 m \n",
      "INFO  19:17:30,639 ProgressMeter -     22:29703701   2.85872872E9     4.5 h       5.0 s       92.2%     4.9 h      22.9 m \n",
      "INFO  19:18:00,640 ProgressMeter -     22:31999901   2.86072872E9     4.5 h       5.0 s       92.3%     4.9 h      22.7 m \n",
      "INFO  19:18:30,641 ProgressMeter -     22:34999901   2.86372872E9     4.5 h       5.0 s       92.4%     4.9 h      22.5 m \n",
      "INFO  19:19:00,642 ProgressMeter -     22:37999901   2.86672872E9     4.5 h       5.0 s       92.5%     4.9 h      22.2 m \n",
      "INFO  19:19:30,643 ProgressMeter -     22:39999901   2.86872872E9     4.5 h       5.0 s       92.5%     4.9 h      22.1 m \n",
      "INFO  19:20:00,644 ProgressMeter -     22:42341201   2.87172872E9     4.6 h       5.0 s       92.6%     4.9 h      21.9 m \n",
      "INFO  19:20:30,645 ProgressMeter -     22:44999901   2.87372872E9     4.6 h       5.0 s       92.7%     4.9 h      21.6 m \n",
      "INFO  19:21:00,647 ProgressMeter -     22:46999901   2.87572872E9     4.6 h       5.0 s       92.7%     4.9 h      21.5 m \n",
      "INFO  19:21:30,648 ProgressMeter -     22:49991101   2.87872872E9     4.6 h       5.0 s       92.8%     4.9 h      21.2 m \n",
      "INFO  19:22:00,649 ProgressMeter -       X:1418201   2.882033286E9     4.6 h       5.0 s       92.9%     4.9 h      20.9 m \n",
      "INFO  19:22:30,650 ProgressMeter -       X:7999901   2.888033286E9     4.6 h       5.0 s       93.1%     4.9 h      20.3 m \n",
      "INFO  19:23:00,651 ProgressMeter -      X:15488801   2.896033286E9     4.6 h       5.0 s       93.4%     4.9 h      19.6 m \n",
      "INFO  19:23:30,652 ProgressMeter -      X:21779901   2.902033286E9     4.6 h       5.0 s       93.6%     4.9 h      19.0 m \n",
      "INFO  19:24:00,653 ProgressMeter -      X:29597501   2.910033286E9     4.6 h       5.0 s       93.8%     4.9 h      18.2 m \n",
      "INFO  19:24:30,654 ProgressMeter -      X:38326101   2.919033286E9     4.6 h       5.0 s       94.1%     4.9 h      17.4 m \n",
      "INFO  19:25:00,655 ProgressMeter -      X:45625501   2.926033286E9     4.6 h       5.0 s       94.4%     4.9 h      16.6 m \n",
      "INFO  19:25:30,656 ProgressMeter -      X:51341501   2.932033286E9     4.6 h       5.0 s       94.5%     4.9 h      16.1 m \n",
      "INFO  19:26:00,657 ProgressMeter -      X:57777001   2.938033286E9     4.7 h       5.0 s       94.7%     4.9 h      15.5 m \n",
      "INFO  19:26:30,658 ProgressMeter -      X:66392401   2.947033286E9     4.7 h       5.0 s       95.0%     4.9 h      14.6 m \n",
      "INFO  19:27:00,659 ProgressMeter -      X:72999901   2.953033286E9     4.7 h       5.0 s       95.2%     4.9 h      14.0 m \n",
      "INFO  19:27:30,660 ProgressMeter -      X:79100401   2.960033286E9     4.7 h       5.0 s       95.4%     4.9 h      13.4 m \n",
      "INFO  19:28:00,661 ProgressMeter -      X:87810001   2.968033286E9     4.7 h       5.0 s       95.7%     4.9 h      12.6 m \n",
      "INFO  19:28:30,662 ProgressMeter -      X:96999901   2.977033286E9     4.7 h       5.0 s       96.0%     4.9 h      11.7 m \n",
      "INFO  19:29:00,663 ProgressMeter -     X:104125801   2.985033286E9     4.7 h       5.0 s       96.2%     4.9 h      11.0 m \n",
      "INFO  19:29:30,664 ProgressMeter -     X:111999901   2.992033286E9     4.7 h       5.0 s       96.5%     4.9 h      10.3 m \n",
      "INFO  19:30:00,665 ProgressMeter -     X:119059201   3.000033286E9     4.7 h       5.0 s       96.7%     4.9 h       9.6 m \n",
      "INFO  19:30:30,666 ProgressMeter -     X:126851301   3.007033286E9     4.7 h       5.0 s       97.0%     4.9 h       8.9 m \n",
      "INFO  19:31:00,667 ProgressMeter -     X:134023501   3.015033286E9     4.7 h       5.0 s       97.2%     4.9 h       8.2 m \n",
      "INFO  19:31:30,668 ProgressMeter -     X:141581201   3.022033286E9     4.7 h       5.0 s       97.4%     4.9 h       7.5 m \n",
      "INFO  19:32:00,669 ProgressMeter -     X:150404901   3.031033286E9     4.8 h       5.0 s       97.7%     4.9 h       6.6 m \n",
      "INFO  19:32:30,670 ProgressMeter -       Y:1288601   3.037303846E9     4.8 h       5.0 s       97.9%     4.9 h       6.0 m \n",
      "INFO  19:33:00,677 ProgressMeter -      Y:11234401   3.047303846E9     4.8 h       5.0 s       98.3%     4.9 h       5.1 m \n",
      "INFO  19:33:30,679 ProgressMeter -      Y:20775601   3.056303846E9     4.8 h       5.0 s       98.6%     4.8 h       4.2 m \n",
      "INFO  19:34:00,680 ProgressMeter -      Y:29999901   3.065303846E9     4.8 h       5.0 s       98.9%     4.8 h       3.3 m \n",
      "INFO  19:34:30,681 ProgressMeter -      Y:40065101   3.076303846E9     4.8 h       5.0 s       99.2%     4.8 h       2.4 m \n",
      "INFO  19:35:00,683 ProgressMeter -      Y:50532101   3.086303846E9     4.8 h       5.0 s       99.5%     4.8 h      83.0 s \n",
      "INFO  19:35:40,684 ProgressMeter - GL000212.1:98001   3.100104058E9     4.8 h       5.0 s       99.9%     4.8 h       8.0 s \n",
      "INFO  19:35:45,810 ProgressMeter -            done   3.101804739E9     4.8 h       5.0 s      100.0%     4.8 h       0.0 s \n",
      "INFO  19:35:45,810 ProgressMeter - Total runtime 17336.07 secs, 288.93 min, 4.82 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were 5 WARN messages, the first 5 are repeated below.\n",
      "WARN  14:46:49,841 StrandBiasTest - StrandBiasBySample annotation exists in input VCF header. Attempting to use StrandBiasBySample values to calculate strand bias annotation values. If no sample has the SB genotype annotation, annotation may still fail. \n",
      "WARN  14:46:49,841 InbreedingCoeff - Annotation will not be calculated. InbreedingCoeff requires at least 10 unrelated samples. \n",
      "WARN  14:46:49,841 StrandBiasTest - StrandBiasBySample annotation exists in input VCF header. Attempting to use StrandBiasBySample values to calculate strand bias annotation values. If no sample has the SB genotype annotation, annotation may still fail. \n",
      "WARN  14:46:50,138 HaplotypeScore - Annotation will not be calculated, must be called from UnifiedGenotyper, not org.broadinstitute.gatk.tools.walkers.variantutils.GenotypeGVCFs \n",
      "WARN  16:29:40,895 ExactAFCalculator - This tool is currently set to genotype at most 6 alternate alleles in a given context, but the context at 6: 31797923 has 7 alternate alleles so only the top alleles will be used; see the --max_alternate_alleles argument. Unless the DEBUG logging level is used, this warning message is output just once per run and further warnings are suppressed. \n",
      "------------------------------------------------------------------------------------------\n",
      "\n"
     ]
    }
   ],
   "source": [
    "######\n",
    "#Combine all separate g.vcf files into a single VCF file _pijplieding van Niek! \n",
    "\n",
    "step8_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step8/\"\n",
    "\n",
    "\n",
    "VCF_Files = glob.glob(step7_dir + \"*.g.vcf\")\n",
    "#print(mergeVCF + \" \" + str(VCF_Files) + \" > \" + step1_dir + \"Human_MG_merged_VCF.vcf\")\n",
    "\n",
    "terminal_code = GATK + \" -T GenotypeGVCFs -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -o \" + step8_dir + \"all.sample.vcf -log sample.genotypegvcfs.log  -stand_call_conf 20.0 -stand_emit_conf 20.0 \" \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf \"  +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf \"  +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf \" \n",
    "    \n",
    "#print(terminal_code)    \n",
    "print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "######FAKE STEP 8 in order to get to 20+ samples necessary for conform-gt \n",
    "\n",
    "step8_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step8/\"\n",
    "\n",
    "\n",
    "VCF_Files = glob.glob(step7_dir + \"*.g.vcf\")\n",
    "#print(mergeVCF + \" \" + str(VCF_Files) + \" > \" + step1_dir + \"Human_MG_merged_VCF.vcf\")\n",
    "\n",
    "terminal_code = GATK + \" -T GenotypeGVCFs -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -o \" + step8_dir + \"FAKE_joint_calls_all.sample.vcf \" \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf \"  +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf \"  +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf \" +\\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf \" \n",
    "\n",
    "    \n",
    "#print(terminal_code)    \n",
    "print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P16filtered_output.vcf\n",
      "INFO  16:15:18,437 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:18,439 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:18,439 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:18,439 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:18,439 HelpFormatter - [Sat Dec 03 16:15:18 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:18,440 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:18,442 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P16filtered_output.vcf \n",
      "INFO  16:15:18,452 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:18,453 HelpFormatter - Date/Time: 2016/12/03 16:15:18 \n",
      "INFO  16:15:18,453 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:18,453 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:18,480 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:18,595 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:18,715 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:18,720 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:18,721 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:18,721 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:18,722 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:15:21,030 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:15:23,923 ProgressMeter -            done     62287.0     5.0 s      83.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  16:15:23,923 ProgressMeter - Total runtime 5.20 secs, 0.09 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz sample P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P19filtered_output.vcf\n",
      "INFO  16:15:25,163 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:25,164 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:25,165 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:25,165 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:25,165 HelpFormatter - [Sat Dec 03 16:15:25 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:25,165 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:25,168 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P19filtered_output.vcf \n",
      "INFO  16:15:25,176 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:25,177 HelpFormatter - Date/Time: 2016/12/03 16:15:25 \n",
      "INFO  16:15:25,177 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:25,178 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:25,204 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:25,311 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:25,427 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:25,431 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:25,432 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:25,432 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:25,432 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:15:30,714 ProgressMeter -            done     74591.0     5.0 s      70.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  16:15:30,715 ProgressMeter - Total runtime 5.28 secs, 0.09 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz sample P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P21filtered_output.vcf\n",
      "INFO  16:15:31,970 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:31,972 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:31,972 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:31,972 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:31,972 HelpFormatter - [Sat Dec 03 16:15:31 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:31,972 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:31,975 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P21filtered_output.vcf \n",
      "INFO  16:15:31,983 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:31,983 HelpFormatter - Date/Time: 2016/12/03 16:15:31 \n",
      "INFO  16:15:31,984 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:31,984 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:32,009 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:32,128 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:32,246 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:32,251 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:32,251 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:32,252 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:32,252 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:15:37,719 ProgressMeter -            done     83176.0     5.0 s      65.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  16:15:37,721 ProgressMeter - Total runtime 5.47 secs, 0.09 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P2filtered_output.vcf\n",
      "INFO  16:15:39,000 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:39,001 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:39,001 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:39,002 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:39,002 HelpFormatter - [Sat Dec 03 16:15:38 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:39,002 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:39,004 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P2filtered_output.vcf \n",
      "INFO  16:15:39,013 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:39,014 HelpFormatter - Date/Time: 2016/12/03 16:15:38 \n",
      "INFO  16:15:39,014 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:39,015 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:39,045 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:39,158 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:39,266 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:39,270 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:39,271 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:39,271 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:39,271 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:15:43,952 ProgressMeter -            done     45432.0     4.0 s     103.0 s       99.9%     4.0 s       0.0 s \n",
      "INFO  16:15:43,954 ProgressMeter - Total runtime 4.68 secs, 0.08 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P3filtered_output.vcf\n",
      "INFO  16:15:45,229 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:45,230 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:45,231 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:45,231 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:45,231 HelpFormatter - [Sat Dec 03 16:15:45 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:45,231 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:45,234 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P3filtered_output.vcf \n",
      "INFO  16:15:45,241 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:45,241 HelpFormatter - Date/Time: 2016/12/03 16:15:45 \n",
      "INFO  16:15:45,241 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:45,242 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:45,264 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:45,365 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:45,477 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:45,482 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:45,482 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:45,482 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:45,482 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:15:47,773 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:15:50,620 ProgressMeter -            done     58926.0     5.0 s      87.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  16:15:50,620 ProgressMeter - Total runtime 5.14 secs, 0.09 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P6filtered_output.vcf\n",
      "INFO  16:15:51,885 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:51,887 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:51,887 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:51,887 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:51,887 HelpFormatter - [Sat Dec 03 16:15:51 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:51,887 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:51,890 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P6filtered_output.vcf \n",
      "INFO  16:15:51,898 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:51,899 HelpFormatter - Date/Time: 2016/12/03 16:15:51 \n",
      "INFO  16:15:51,899 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:51,899 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:51,924 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:52,027 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:52,153 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:52,158 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:52,158 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:52,158 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:52,159 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "INFO  16:15:57,488 ProgressMeter -            done     71622.0     5.0 s      74.0 s      100.0%     5.0 s       0.0 s \n",
      "INFO  16:15:57,489 ProgressMeter - Total runtime 5.33 secs, 0.09 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P10filtered_output.vcf\n",
      "INFO  16:15:58,696 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:58,698 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:15:58,698 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:15:58,698 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:15:58,698 HelpFormatter - [Sat Dec 03 16:15:58 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:15:58,699 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:15:58,701 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P10filtered_output.vcf \n",
      "INFO  16:15:58,710 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:15:58,710 HelpFormatter - Date/Time: 2016/12/03 16:15:58 \n",
      "INFO  16:15:58,711 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:58,711 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:15:58,739 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:15:58,853 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:15:58,985 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:15:58,990 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:15:58,990 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:15:58,991 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:15:58,991 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:15:59,750 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:06,317 ProgressMeter -            done    192396.0     7.0 s      38.0 s      100.0%     7.0 s       0.0 s \n",
      "INFO  16:16:06,317 ProgressMeter - Total runtime 7.33 secs, 0.12 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P15filtered_output.vcf\n",
      "INFO  16:16:07,526 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:07,527 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:16:07,527 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:16:07,528 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:16:07,528 HelpFormatter - [Sat Dec 03 16:16:07 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:16:07,528 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:16:07,530 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P15filtered_output.vcf \n",
      "INFO  16:16:07,538 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:16:07,539 HelpFormatter - Date/Time: 2016/12/03 16:16:07 \n",
      "INFO  16:16:07,540 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:07,540 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:07,569 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:16:07,682 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:16:07,809 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:16:07,813 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:16:07,813 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:16:07,814 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:16:07,814 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:16:11,810 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:14,568 ProgressMeter -            done    154624.0     6.0 s      43.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  16:16:14,568 ProgressMeter - Total runtime 6.76 secs, 0.11 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P17filtered_output.vcf\n",
      "INFO  16:16:15,916 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:15,917 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:16:15,917 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:16:15,918 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:16:15,918 HelpFormatter - [Sat Dec 03 16:16:15 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:16:15,918 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:16:15,920 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P17filtered_output.vcf \n",
      "INFO  16:16:15,929 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:16:15,930 HelpFormatter - Date/Time: 2016/12/03 16:16:15 \n",
      "INFO  16:16:15,930 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:15,931 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:15,953 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:16:16,069 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:16:16,201 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:16:16,206 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:16:16,206 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:16:16,206 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:16:16,206 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:16:17,937 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:18,105 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:20,722 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:24,425 ProgressMeter -            done    252373.0     8.0 s      32.0 s      100.0%     8.0 s       0.0 s \n",
      "INFO  16:16:24,425 ProgressMeter - Total runtime 8.22 secs, 0.14 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P5filtered_output.vcf\n",
      "INFO  16:16:25,664 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:25,666 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:16:25,666 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:16:25,666 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:16:25,666 HelpFormatter - [Sat Dec 03 16:16:25 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:16:25,666 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:16:25,669 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P5filtered_output.vcf \n",
      "INFO  16:16:25,676 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:16:25,677 HelpFormatter - Date/Time: 2016/12/03 16:16:25 \n",
      "INFO  16:16:25,677 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:25,678 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:25,705 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:16:25,824 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:16:25,947 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:16:25,951 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:16:25,951 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:16:25,952 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:16:25,952 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:16:28,704 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:28,739 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:28,740 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:29,672 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:30,095 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:32,418 ProgressMeter -            done    105685.0     6.0 s      61.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  16:16:32,419 ProgressMeter - Total runtime 6.47 secs, 0.11 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P7filtered_output.vcf\n",
      "INFO  16:16:33,715 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:33,717 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:16:33,717 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:16:33,717 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:16:33,718 HelpFormatter - [Sat Dec 03 16:16:33 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:16:33,718 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:16:33,720 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P7filtered_output.vcf \n",
      "INFO  16:16:33,729 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:16:33,730 HelpFormatter - Date/Time: 2016/12/03 16:16:33 \n",
      "INFO  16:16:33,730 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:33,730 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:33,757 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:16:33,870 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:16:33,993 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:16:33,998 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:16:33,998 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:16:33,998 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:16:33,998 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:16:35,453 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:36,881 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:40,796 ProgressMeter -            done    129503.0     6.0 s      52.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  16:16:40,797 ProgressMeter - Total runtime 6.80 secs, 0.11 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.vcf -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' -o /data/scratch/iholtman/human_microglia_genotyping/step8/P8filtered_output.vcf\n",
      "INFO  16:16:42,046 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:42,048 HelpFormatter - The Genome Analysis Toolkit (GATK) v3.6-0-g89b7209, Compiled 2016/06/01 22:27:29 \n",
      "INFO  16:16:42,048 HelpFormatter - Copyright (c) 2010-2016 The Broad Institute \n",
      "INFO  16:16:42,048 HelpFormatter - For support and documentation go to https://www.broadinstitute.org/gatk \n",
      "INFO  16:16:42,049 HelpFormatter - [Sat Dec 03 16:16:42 PST 2016] Executing on Linux 3.10.0-327.36.1.el7.x86_64 amd64 \n",
      "INFO  16:16:42,049 HelpFormatter - OpenJDK 64-Bit Server VM 1.8.0_102-b14 JdkDeflater \n",
      "INFO  16:16:42,051 HelpFormatter - Program Args: -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.vcf -window 35 -cluster 3 -filterName FS -filter FS > 30.0 -filterName QD -filter QD < 2.0 -o /data/scratch/iholtman/human_microglia_genotyping/step8/P8filtered_output.vcf \n",
      "INFO  16:16:42,060 HelpFormatter - Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.1.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14. \n",
      "INFO  16:16:42,061 HelpFormatter - Date/Time: 2016/12/03 16:16:42 \n",
      "INFO  16:16:42,061 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:42,061 HelpFormatter - ---------------------------------------------------------------------------------- \n",
      "INFO  16:16:42,088 GenomeAnalysisEngine - Strictness is SILENT \n",
      "INFO  16:16:42,202 GenomeAnalysisEngine - Downsampling Settings: Method: BY_SAMPLE, Target Coverage: 1000 \n",
      "INFO  16:16:42,317 GenomeAnalysisEngine - Preparing for traversal \n",
      "INFO  16:16:42,322 GenomeAnalysisEngine - Done preparing for traversal \n",
      "INFO  16:16:42,322 ProgressMeter - [INITIALIZATION COMPLETE; STARTING PROCESSING] \n",
      "INFO  16:16:42,323 ProgressMeter -                 | processed |    time |    per 1M |           |   total | remaining \n",
      "INFO  16:16:42,323 ProgressMeter -        Location |     sites | elapsed |     sites | completed | runtime |   runtime \n",
      "WARN  16:16:45,854 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "WARN  16:16:46,194 Interpreter - ![0,2]: 'QD < 2.0;' undefined variable QD \n",
      "INFO  16:16:49,279 ProgressMeter -            done    120287.0     6.0 s      57.0 s      100.0%     6.0 s       0.0 s \n",
      "INFO  16:16:49,279 ProgressMeter - Total runtime 6.96 secs, 0.12 min, 0.00 hours \n",
      "------------------------------------------------------------------------------------------\n",
      "Done. There were no warn messages.\n",
      "------------------------------------------------------------------------------------------\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###Variant filtering \n",
    "step8_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step8/\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "    \" -V \" + step7_dir + \"P\" + str(row['patient']) + \"_output.vcf\" + \\\n",
    "    \" -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0'\" \\\n",
    "    \" -o \" + step8_dir + \"P\" + str(row['patient']) + \"filtered_output.vcf\"\n",
    "\n",
    "    print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    #print(\"\\n\")\n",
    "\n",
    "\n",
    "####original script \n",
    "#java -jar GenomeAnalysisTK.jar -T VariantFiltration -R hg_19.fasta -V input.vcf -window 35 -cluster 3 \n",
    "#-filterName FS -filter \"FS > 30.0\" -filterName QD -filter \"QD < 2.0\" -o output.vcf \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##In de handleiding worden nog een aantal zaken besproken, zoals \"mergeGvcf\" en \"genotypeGvcf\". Als je de \n",
    "#haplotype caller in gvcf modus draait wordt voor alle varianten en reference sites oa de coverage informatie\n",
    "#bewaard. Als je meerdere samples hebt kun je de individuele samples/calls met elkaar mergen, waarbij de \"mergeGvcf\" \n",
    "#stap rekening houdt met de coverage e.d. van de referentie sites. Vervolgens kan de \"genotypeGvcf\" stap deze \n",
    "#informatie gebruiken om optimale genotype calls te maken met meerdere samples. Gedurende deze laatste methode wordt \n",
    "#ook rekening gehouden met de haplotype informatie van de andere samples in je cohort om de callset te verbeteren.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####Base recalibration - recommended but not specified \n",
    "\n",
    "\n",
    "#\\java -jar $GATK \\\n",
    "#    -T BaseRecalibrator \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I $sFILE'split.bam' \\\n",
    "#    -L 20 \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/dbsnp_138.b37.vcf \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/Mills_and_1000G_gold_standard.indels.b37.vcf \\\n",
    "#    -o $sFILE'recal_data.table'\n",
    "\n",
    "#java -jar $GATK \\\n",
    "#    -T BaseRecalibrator \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I $sFILE'split.bam' \\\n",
    "#    -L 20 \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/dbsnp_138.b37.vcf \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/Mills_and_1000G_gold_standard.indels.b37.vcf \\\n",
    "#    -BQSR $sFILE'recal_data.table' \\\n",
    "#    -o $sFILE'post_recal_data.table'\n",
    "\n",
    "    #MAKE plots before and after \n",
    "#java -jar $GATK \\\n",
    "#    -T AnalyzeCovariates \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -L 20 \\\n",
    "#    -before $sFILE'recal_data.table' \\\n",
    "#    -after $sFILE'post_recal_data.table' \\\n",
    "#    -plots $sFILE'recalibration_plots.pdf'\n",
    "\n",
    "#intergate into new BAM-files \n",
    "#java -jar $GATK \\\n",
    "#    -T PrintReads \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I split.bam \\\n",
    "#    -L 20 \\\n",
    "##    -BQSR $sFILE'recal_data.table' \\\n",
    "#    -o $sFILE'recal_reads.bam'    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "####TXT conversion files for BEAGLE \n",
    "##https://faculty.washington.edu/browning/beagle_utilities/utilities.html\n",
    "##vcf2beagle.jar\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####################\n",
    "### in Loop without basecalibration\n",
    "#FILES=$(cat /gpfs/data01/glasslab/home/iholtman/Molgenis/second_pass/LIST_FILES)\n",
    "#for P in $FILES\n",
    "##do\n",
    "#java -jar $GATK -T HaplotypeCaller \\\n",
    "# -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "# -I $P'_split.bam' \\\n",
    "# -dontUseSoftClippedBases \\\n",
    "## -stand_call_conf 20.0 \\\n",
    "# -stand_emit_conf 20.0 \\\n",
    "#--emitRefConfidence GVCF \\\n",
    "# -o $P'_Genotype_callign_output_no_calibration.g.vcf'\n",
    "#done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "####\n",
    "#Combine all separate g.vcf files into a single VCF file \n",
    "#Exclude P9 \n",
    "\n",
    "#java -jar $GATK \\\n",
    "# -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "# -T  GenotypeGVCFs \\\n",
    "#-V P10_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P16_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P2_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P6_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P11_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P17_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P3_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P7_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P12_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P19_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P4_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P8_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P15_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P21_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "#-V P5_Genotype_callign_output_no_calibration.g.vcf \\\n",
    "##-o  all.sample_without_P9.vcf \\\n",
    "#-log sample.genotypegvcfs.log \\\n",
    "#-stand_call_conf 20.0 \\\n",
    "#-stand_emit_conf 20.0 \\\n",
    "#-nt 15\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##vcf2beagle.jar\n",
    "#https://faculty.washington.edu/browning/beagle_utilities/utilities.html#vcf2beagle explained here \n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##Beagle:: \n",
    "\n",
    "######Step 1: MERGE VCF files into a single VCF\n",
    "## BV door: \n",
    "##https://faculty.washington.edu/browning/beagle_utilities/utilities.html#mergevcf\n",
    "\n",
    "#Step2 Conform-gt to make the VCF files similar to the reference VCF (from 1000G?) \n",
    "## vThe reference and target VCF files are each required to have data for at least 20 individuals so that the \n",
    "#statistical tests performed by the conform-gt program give reasonable results.\n",
    "\n",
    "##/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase1_vcf\n",
    "##chr1.1kg.ref.phase1_release_v3.20101123.vcf.gz       \n",
    "\n",
    "## Step3 BEAGLE imputation and phasing ! \n",
    "\n",
    "#####INformation \n",
    "#https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263197/ \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/mergevcf.jar ['/data/scratch/iholtman/human_microglia_genotyping/step8/P7filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P19filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P2filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P6filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P3filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P10filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P5filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P16filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P21filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P17filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P8filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P15filtered_output.vcf'] > /data/scratch/iholtman/human_microglia_genotyping/imputation/step1/Human_MG_merged_VCF.vcf\n"
     ]
    }
   ],
   "source": [
    "###\n",
    "step1_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step1/\"\n",
    "\n",
    "\n",
    "#/gpfs/data01/glasslab/home/iholtman/software/pgtools-0.3.0/pgtools/toolbox.py\n",
    "mergeVCF = \"java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/mergevcf.jar\"\n",
    "\n",
    "step8_dir\n",
    "\n",
    "VCF_Files = glob.glob(step8_dir + \"*.vcf\")\n",
    "print(mergeVCF + \" \" + str(VCF_Files) + \" > \" + step1_dir + \"Human_MG_merged_VCF.vcf\")\n",
    "\n",
    "###### \n",
    "## \n",
    "#usage: java -jar mergevcf.jar [chrom] [vcf 1] [vcf 2] ... > [out vcf]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr1.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/step8/FAKE_all.sample.vcf chrom=1 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr1_conformed\n"
     ]
    }
   ],
   "source": [
    "###STEP2 \n",
    "step2_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/\"\n",
    "\n",
    "##Conform-gt \n",
    "##Make variables from files \n",
    "#VCF_file = \"/data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf\"\n",
    "VCF_file = \"/data/scratch/iholtman/human_microglia_genotyping/step8/FAKE_all.sample.vcf\"\n",
    "conform_GT =\"java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar \" \n",
    "ref_chr1 = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr1.1kg.phase3.v5.vcf.gz\"\n",
    "\n",
    "##Command:\n",
    "\n",
    "terminal_code = conform_GT + \" ref=\" + ref_chr1 + \" gt=\" + VCF_file + \" chrom=1\" + \" out=\" + step2_dir + \"chr1_conformed\"\n",
    "print(terminal_code)\n",
    "\n",
    "#####\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###STEP2 ---- TEST DATA! \n",
    "\n",
    "\"/gpfs/data01/glasslab/home/iholtman/software/beagle/test/phase1_integrated_calls.20101123.ALL.panel\"\n",
    "#grep -v EUR phase1_integrated_calls.20101123.ALL.panel | cut -f1 > non.eur.excl\n",
    "\n",
    "\n",
    "##Conform-gt \n",
    "#Files: \n",
    "\"/data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf\"\n",
    "\"/gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar\"\n",
    "\n",
    "##Make variables \n",
    "VCF_file = \"/data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf\"\n",
    "conform_GT =\"java 1kg.ref.phase1_release_v3.20101123.vcf.gz-jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar\" \n",
    "ref_chr1 = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase1_vcf/chr1.1kg.ref.phase1_release_v3.20101123.vcf.gz\"\n",
    "##Command:\n",
    "\n",
    "terminal_code = conform_GT + \" ref=\" + ref_chr1 + \" gt=\" + VCF_file + \" chrom=2\" + \" out=\" + step2_dir + \"chr2_conformed\"\n",
    "print(terminal_code)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## BEAGLE tutorial \n",
    "## https://faculty.washington.edu/browning/beagle/beagle_4.1_03May16.pdf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "cd ~/software/beagle/test/\n",
    "\n",
    "###\"Example 1a: phasing the reference panel\"\n",
    "java -Xmx1000m -jar beagle.jar unphased=ref.bgl.gz missing=N out=ex1a lowmem=true gprobs=false verbose=true\n",
    "\n",
    "\n",
    "#\"Example 1b: phasing the reference panel AND perform fastIBD analysis\"\n",
    "#\"  -- setting ibdscale parameter to sqrt[ (# samples) / 100 ]  (current best practice)\"\n",
    "java -Xmx1000m -jar beagle.jar unphased=ref.bgl.gz missing=N out=ex1b lowmem=true gprobs=false verbose=true fastibd=true ibdscale=2.3\n",
    "\n",
    "\n",
    "##\"Example 2: imputation of ungenotyped markers using a phased reference panel\"\n",
    "java -Xmx1000m -jar beagle.jar markers=ref.markers phased=ex1b.ref.bgl.gz.phased.gz unphased=sparse.target.bgl.gz missing=N out=ex2 lowmem=true verbose=true\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#!/bin/bash\n",
    "if [ ! -f beagle.27Jul16.86a.jar ]; then\n",
    "  echo\n",
    "  echo \"Downloading beagle.27Jul16.86a.jar\"\n",
    "  wget http://faculty.washington.edu/browning/beagle/beagle.27Jul16.86a.jar\n",
    "fi\n",
    "\n",
    "if [ ! -f bref.27Jul16.86a.jar ]; then\n",
    "  echo\n",
    "  echo \"Downloading bref.27Jul16.86a.jar\"\n",
    "  wget http://faculty.washington.edu/browning/beagle/bref.27Jul16.86a.jar\n",
    "fi\n",
    "\n",
    "echo\n",
    "\n",
    "if [ ! -f test.27Jul16.86a.vcf.gz ]; then\n",
    "    echo\n",
    "    echo \"*** Downloading some 1000 Genomes Project data to file: test.27Jul16.86a.vcf.gz ***\"\n",
    "    wget http://faculty.washington.edu/browning/beagle/test.27Jul16.86a.vcf.gz\n",
    "fi\n",
    "\n",
    "echo\n",
    "echo \"*** Creating test files: ref.27Jul16.86a.vcf.gz target.27Jul16.86a.vcf.gz ***\"\n",
    "echo\n",
    "zcat test.27Jul16.86a.vcf.gz | cut -f1-190 | tr '/' '|' | gzip > ref.27Jul16.86a.vcf.gz\n",
    "zcat test.27Jul16.86a.vcf.gz | cut -f1-9,191-200 | gzip > target.27Jul16.86a.vcf.gz\n",
    "\n",
    "\n",
    "echo\n",
    "echo \"*** Running test analysis with \\\"gt=\\\" argument ***\"\n",
    "echo\n",
    "java -jar beagle.27Jul16.86a.jar gt=test.27Jul16.86a.vcf.gz out=out.gt\n",
    "\n",
    "echo\n",
    "echo \"*** Running test analysis with \\\"gl=\\\" argument ***\"\n",
    "echo\n",
    "java -jar beagle.27Jul16.86a.jar gl=test.27Jul16.86a.vcf.gz out=out.gl\n",
    "\n",
    "echo\n",
    "echo \"*** Running test analysis with \\\"ref=\\\" argument ***\"\n",
    "echo\n",
    "java -jar beagle.27Jul16.86a.jar ref=ref.27Jul16.86a.vcf.gz gt=target.27Jul16.86a.vcf.gz out=out.ref\n",
    "\n",
    "echo\n",
    "echo \"*** Making \\\"bref\\\" file ***\"\n",
    "echo\n",
    "java -jar bref.27Jul16.86a.jar ref.27Jul16.86a.vcf.gz\n",
    "\n",
    "echo\n",
    "echo \"*** Running test analysis with \\\"bref=\\\" argument ***\"\n",
    "echo\n",
    "java -jar beagle.27Jul16.86a.jar ref=ref.27Jul16.86a.bref gt=target.27Jul16.86a.vcf.gz out=out.bref\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##################\n",
    "###Current location for reference 1000G VCF files: \n",
    "1000G_DIR = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase1_vcf/\"\n",
    "\n",
    "##VCF files need to be made similar using conform-GT \n",
    "#https://faculty.washington.edu/browning/conform-gt.html\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##VCF manipulation\n",
    "with PLINK-SEQ, VCFTools,  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "# Create \"de-identified\" matrices of TPM and read counts for submission"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Find and copy / rename fastq files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Samples missing FASTQ paths:\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15 (human_rna_10_days_TGFB_individual_5)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Sorted_RNA_PolyA_P24_Culture_24h_IL34_160810_DGO_16_10_28 (human_rna_24_hours_noTGFB_individual_24)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P12_151125_DGO_15_12_14 (human_rna_ex_vivo_noTGFB_individual_12)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15 (human_rna_10_days_noTGFB_individual_7)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P8_150622_Parietal_DGO_15_08_21 (human_rna_ex_vivo_noTGFB_individual_8_parietal)\n",
      "\t\tSample appears to have 8 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15 (human_rna_ex_vivo_noTGFB_individual_3_surgery_1)\n",
      "\t\tSample appears to have 8 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample P14_brain_RNA_Brain_Grey_JoS_16_04_27_ATTCCT (human_rna_wholebrain_individual_14)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23 (human_rna_ex_vivo_noTGFB_individual_10)\n",
      "\t\tSample appears to have 8 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15 (human_rna_ex_vivo_noTGFB_individual_5)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT (human_rna_wholebrain_individual_7)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15 (human_rna_ex_vivo_noTGFB_individual_7)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Sorted_RNA_PolyA_P23_Culture_6h_IL34_160808_DGO_16_10_28 (human_rna_6_hours_noTGFB_individual_23)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Sorted_RNA_PolyA_P24_Culture_6h_IL34_160810_DGO_16_10_28 (human_rna_6_hours_noTGFB_individual_24)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG (human_rna_wholebrain_individual_4)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04 (human_rna_ex_vivo_noTGFB_individual_15)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P18_exvivo_JoS_16_04_27 (human_rna_ex_vivo_noTGFB_individual_18)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21 (human_rna_7_days_noTGFB_individual_8)\n",
      "\t\tSample appears to have 8 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15 (human_rna_ex_vivo_noTGFB_individual_2)\n",
      "\t\tSample appears to have 9 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04 (human_rna_7_days_noTGFB_individual_15)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04 (human_rna_ex_vivo_noTGFB_individual_16)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04 (human_rna_ex_vivo_noTGFB_individual_17)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15 (human_rna_10_days_TGFB_individual_7)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70 (human_rna_ex_vivo_noTGFB_individual_11)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25 (human_rna_ex_vivo_noTGFB_individual_21)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15 (human_rna_10_days_noTGFB_individual_5)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Brain_Sorted_RNA_PolyA_P3_Surgery2_Culture_24h_IL34_Vehicle_24h_160427_DGO_16_05_12 (human_rna_24_hours_noTGFB_individual_3)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04 (human_rna_7_days_noTGFB_individual_17)\n",
      "\t\tSample appears to be a single uncompressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT (human_rna_wholebrain_individual_8)\n",
      "\t\tSample appears to be a single compressed file\n",
      "\t\tNo need to copy\n",
      "\tProcessing sample Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21 (human_rna_ex_vivo_noTGFB_individual_8_occipital)\n",
      "\t\tSample appears to have 7 lane files\n",
      "\t\tConcatenating lane file /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21_TTAGGC_L003_R1_001.fastq.gz ...done.\n",
      "\t\tConcatenating lane file /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21_TTAGGC_L003_R1_003.fastq.gz ...done.\n",
      "\t\tConcatenating lane file /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_150622_Occipital_DGO_15_08_21_TTAGGC_L003_R1_005.fastq.gz ..."
     ]
    }
   ],
   "source": [
    "import os\n",
    "import shutil\n",
    "import subprocess\n",
    "\n",
    "fastq_mapping = pandas.read_csv('/home/dskola/human/dbgap_submission/human_rna_tag_fastq_mapping.csv', index_col=0).fastq_path.to_dict()\n",
    "DESTINATION_FOLDER='/home/dskola/human/dbgap_submission/submission_folder/'\n",
    "\n",
    "missing_samples=set(name_mapping.keys()).difference(fastq_mapping.keys())\n",
    "print('\\nSamples missing FASTQ paths:\\n')\n",
    "print('\\n'.join(missing_samples))\n",
    "print('\\n')\n",
    "open('/home/dskola/human/dbgap_submission/missing_rna.txt', 'wt').write('\\n'.join(missing_samples))\n",
    "for tag_name, simple_name in name_mapping.items():\n",
    "    if tag_name in fastq_mapping:        \n",
    "        fastq_path = fastq_mapping[tag_name]\n",
    "        print('\\tProcessing sample {} ({})'.format(tag_name, simple_name))\n",
    "        \n",
    "        destination_path = os.path.join(DESTINATION_FOLDER, '{}.fastq.gz'.format(simple_name))\n",
    "        \n",
    "        if fastq_path.endswith('.fastq.gz'):\n",
    "            print('\\t\\tSample appears to be a single compressed file')\n",
    "            need_copy = True\n",
    "            if os.path.isfile(destination_path):\n",
    "                source_size = os.stat(fastq_path)[6]\n",
    "                dest_size = os.stat(destination_path)[6]\n",
    "                if source_size == dest_size:\n",
    "                    need_copy = False\n",
    "            if need_copy:\n",
    "                print('\\t\\tCopying...', end=\"\", flush=True)\n",
    "                shutil.copyfile(fastq_path, destination_path)\n",
    "                print('done.')\n",
    "            else:\n",
    "                print('\\t\\tNo need to copy')\n",
    "        elif fastq_path.endswith('.fastq'):\n",
    "            print('\\t\\tSample appears to be a single uncompressed file')\n",
    "            need_copy = True\n",
    "            if os.path.isfile(destination_path):\n",
    "                need_copy = False\n",
    "            if need_copy:\n",
    "                cmd_line = ['cat', fastq_path, '|', 'gzip', '-c', '-9', '>', destination_path]\n",
    "                print('\\t\\tCopying and gzipping...', end=\"\", flush=True)\n",
    "                output = subprocess.check_output(' '.join(cmd_line), shell=True, stderr=subprocess.STDOUT)\n",
    "                print('done.')\n",
    "            else:\n",
    "                print('\\t\\tNo need to copy')\n",
    "        else:\n",
    "            # concatenate all subfiles\n",
    "            subfiles = [os.path.join(fastq_path,f) for f in os.listdir(fastq_path) if f.endswith('.fastq.gz') or f.endswith('.fastq')]\n",
    "            if len(subfiles) > 0:\n",
    "                print('\\t\\tSample appears to have {} lane files'.format(len(subfiles)))\n",
    "                need_copy = True\n",
    "            else:\n",
    "                raise Exception('No lane subfiles found in path:\\n{}'.format(fastq_path))\n",
    "            if os.path.isfile(destination_path):\n",
    "                dest_size = os.stat(destination_path)[6]\n",
    "                if dest_size > 10000:\n",
    "                    need_copy = False\n",
    "            if need_copy:\n",
    "                destination_tmp_file = destination_path + '.tmp'\n",
    "                for fname in subfiles:\n",
    "                    \n",
    "                    if fname.endswith('.fastq'):\n",
    "                        cmd_line = ['cat', fname, '>>', destination_tmp_file]\n",
    "                    elif fname.endswith('.fastq.gz'):\n",
    "                        cmd_line = ['zcat', fname, '>>', destination_tmp_file]\n",
    "#                     print(' '.join(cmd_line))\n",
    "                    print('\\t\\tConcatenating lane file {} ...'.format(fname), end=\"\", flush=True)\n",
    "\n",
    "                    output = subprocess.check_output(' '.join(cmd_line), shell=True, stderr=subprocess.STDOUT)\n",
    "                    print('done.')\n",
    "\n",
    "                cmd_line = ['cat', destination_tmp_file, '|', 'gzip', '-c', '-9', '>', destination_path]                \n",
    "#                 print(' '.join(cmd_line))\n",
    "\n",
    "                print('\\t\\tGzipping concatenated file...', end=\"\", flush=True)               \n",
    "\n",
    "                output = subprocess.check_output(' '.join(cmd_line), shell=True, stderr=subprocess.STDOUT)\n",
    "                os.remove(destination_tmp_file)\n",
    "                print('done.')\n",
    "            else:\n",
    "                print('\\t\\tNo need to copy')\n",
    "        \n",
    "    else:\n",
    "        print('\\n!!No FASTQ path found for sample {} ({})!!\\n'.format(tag_name, simple_name))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Find read lengths"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tag_lengths = {}\n",
    "for tag_folder, simple_name in name_mapping.items():\n",
    "    tag_path=os.path.join(annotated_samples.loc[tag_folder, 'path'], 'tagInfo.txt')\n",
    "    with open(tag_path, 'rt') as tag_info:\n",
    "#         print(tag_info.read())\n",
    "        for line in tag_info:\n",
    "            if line.startswith('averageTagLength'):\n",
    "                tag_lengths[simple_name] = toolbox.iround(line.split('=')[-1])\n",
    "                if tag_lengths[simple_name] == 76:\n",
    "                    tag_lengths[simple_name] = 77\n",
    "                    \n",
    "pandas.Series(tag_lengths).sort_index().to_csv('/home/dskola/mouse/geo_submission/mouse_rna_read_lengths.csv')"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
