{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##############IMport 0,1,2 genotype file in python and make heatmap\n",
    "####"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import pandas\n",
    "import numpy\n",
    "import seaborn\n",
    "import matplotlib.pyplot as plt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "####Read in 012 -genotype file\n",
    "data = pandas.read_table(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/genotype_converted_to_012_format_shortened_copy_chr1.raw\", sep = \" \")\n",
    "#data = pandas.read_table(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/genotype_converted_to_012_format_chr1.raw\", sep = \" \")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                               WGS_P15 WGS_P17 WGS_P4 WGS_P5  \\\n",
      "1:10146_AC                                           0     NaN    NaN      1   \n",
      "1:10146_HET                                          0     NaN    NaN      1   \n",
      "rs367896724_AC                                     NaN     NaN    NaN    NaN   \n",
      "rs367896724_HET                                    NaN     NaN    NaN    NaN   \n",
      "rs540431307_0                                      NaN     NaN    NaN    NaN   \n",
      "rs540431307_HET                                    NaN     NaN    NaN    NaN   \n",
      "rs555500075_0                                      NaN     NaN    NaN    NaN   \n",
      "rs555500075_HET                                    NaN     NaN    NaN    NaN   \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC     NaN     NaN      1    NaN   \n",
      "1:10403_HET                                        NaN     NaN      1    NaN   \n",
      "\n",
      "                                               WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "1:10146_AC                                        NaN    NaN     NaN     NaN   \n",
      "1:10146_HET                                       NaN    NaN     NaN     NaN   \n",
      "rs367896724_AC                                    NaN    NaN       0       0   \n",
      "rs367896724_HET                                   NaN    NaN       0       0   \n",
      "rs540431307_0                                     NaN    NaN       0       0   \n",
      "rs540431307_HET                                   NaN    NaN       0       0   \n",
      "rs555500075_0                                     NaN    NaN       0       0   \n",
      "rs555500075_HET                                   NaN    NaN       0       0   \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC    NaN      1     NaN     NaN   \n",
      "1:10403_HET                                       NaN      1     NaN     NaN   \n",
      "\n",
      "                                               imp_P12 imp_P14  ...   imp_P26  \\\n",
      "1:10146_AC                                         NaN     NaN  ...       NaN   \n",
      "1:10146_HET                                        NaN     NaN  ...       NaN   \n",
      "rs367896724_AC                                       0       0  ...         0   \n",
      "rs367896724_HET                                      0       0  ...         0   \n",
      "rs540431307_0                                        0       0  ...         0   \n",
      "rs540431307_HET                                      0       0  ...         0   \n",
      "rs555500075_0                                        0       0  ...         0   \n",
      "rs555500075_HET                                      0       0  ...         0   \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC     NaN     NaN  ...       NaN   \n",
      "1:10403_HET                                        NaN     NaN  ...       NaN   \n",
      "\n",
      "                                               imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "1:10146_AC                                         NaN     NaN    NaN     NaN   \n",
      "1:10146_HET                                        NaN     NaN    NaN     NaN   \n",
      "rs367896724_AC                                       0       0      0       0   \n",
      "rs367896724_HET                                      0       0      0       0   \n",
      "rs540431307_0                                        0       0      0       0   \n",
      "rs540431307_HET                                      0       0      0       0   \n",
      "rs555500075_0                                        0       0      0       0   \n",
      "rs555500075_HET                                      0       0      0       0   \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC     NaN     NaN    NaN     NaN   \n",
      "1:10403_HET                                        NaN     NaN    NaN     NaN   \n",
      "\n",
      "                                               imp_P4 imp_P5 imp_P6 imp_P7  \\\n",
      "1:10146_AC                                        NaN    NaN    NaN    NaN   \n",
      "1:10146_HET                                       NaN    NaN    NaN    NaN   \n",
      "rs367896724_AC                                      0      0      0      0   \n",
      "rs367896724_HET                                     0      0      0      0   \n",
      "rs540431307_0                                       0      0      0      0   \n",
      "rs540431307_HET                                     0      0      0      0   \n",
      "rs555500075_0                                       0      0      0      0   \n",
      "rs555500075_HET                                     0      0      0      0   \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC    NaN    NaN    NaN    NaN   \n",
      "1:10403_HET                                       NaN    NaN    NaN    NaN   \n",
      "\n",
      "                                               imp_P8  \n",
      "1:10146_AC                                        NaN  \n",
      "1:10146_HET                                       NaN  \n",
      "rs367896724_AC                                      0  \n",
      "rs367896724_HET                                     0  \n",
      "rs540431307_0                                       0  \n",
      "rs540431307_HET                                     0  \n",
      "rs555500075_0                                       0  \n",
      "rs555500075_HET                                     0  \n",
      "1:10403_ACCCTAACCCTAACCCTAACCCTAACCCTAACCCTAAC    NaN  \n",
      "1:10403_HET                                       NaN  \n",
      "\n",
      "[10 rows x 29 columns]\n"
     ]
    }
   ],
   "source": [
    "### Transform in numpy object, transform and convert to pandas\n",
    "table = data.as_matrix()\n",
    "genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "pd_genotype = pd_genotype[6:]\n",
    "\n",
    "print(pd_genotype[0:10])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs62635286_HET        1       1      1      1      1      1       0       0   \n",
      "rs200579949_G         1       1      1      1      1      1       0       0   \n",
      "rs200579949_HET       1       1      1      1      1      1       0       0   \n",
      "rs75454623_G          1       1      1      1      1      1       1       1   \n",
      "rs75454623_HET        1       1      1      1      1      1       1       1   \n",
      "rs78601809_T          1       1      1      1      1      1       1       1   \n",
      "rs78601809_HET        1       1      1      1      1      1       1       1   \n",
      "rs2691277_0           0       0      0      0      0      0       0       0   \n",
      "rs2691277_HET         0       0      0      0      0      0       0       0   \n",
      "\n",
      "                imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs62635286_HET        0       0  ...         0       0       0      0       0   \n",
      "rs200579949_G         0       0  ...         0       0       0      0       0   \n",
      "rs200579949_HET       0       0  ...         0       0       0      0       0   \n",
      "rs75454623_G          1       1  ...         1       2       1      1       0   \n",
      "rs75454623_HET        1       1  ...         1       0       1      1       0   \n",
      "rs78601809_T          0       1  ...         0       0       0      1       1   \n",
      "rs78601809_HET        0       1  ...         0       0       0      1       1   \n",
      "rs2691277_0           0       0  ...         0       0       0      0       0   \n",
      "rs2691277_HET         0       0  ...         0       0       0      0       0   \n",
      "\n",
      "                imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs62635286_HET       0      0      0      0      0  \n",
      "rs200579949_G        0      0      0      0      0  \n",
      "rs200579949_HET      0      0      0      0      0  \n",
      "rs75454623_G         1      1      1      1      1  \n",
      "rs75454623_HET       1      1      1      1      1  \n",
      "rs78601809_T         1      1      0      0      0  \n",
      "rs78601809_HET       1      1      0      0      0  \n",
      "rs2691277_0          0      0      0      0      0  \n",
      "rs2691277_HET        0      0      0      0      0  \n",
      "\n",
      "[9 rows x 29 columns]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(1396, 29)"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "##Remove NA values \n",
    "pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "print(pd_genotype[1:10])\n",
    "pd_genotype.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##Calculate correlations \n",
    "plot_data = pd_genotype.astype(float).corr()\n",
    "#print(plot_data)\n",
    "\n",
    "PLOT_CMAP = 'YlOrRd'\n",
    "VMIN = 0\n",
    "VMAX = 1\n",
    "fig_size=(12,12)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##PLot\n",
    "\n",
    "cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "_=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "#plot_data.to_csv(fig_path('{}.csv'.format(fig_name)))\n",
    "\n",
    "#for fig_ext in FIG_EXTS:\n",
    "#plt.savefig(dpi=PNG_DPI, bbox_inches='tight')\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS_chr1.pdf',  bbox_inches='tight')\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS_chr1.png',  bbox_inches='tight')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import pickle\n",
    "\n",
    "##Safe object \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr1_shortened_pickle', 'wb') as f:\n",
    "    \n",
    "    # Pickle the 'data' dictionary using the highest protocol available.\n",
    "    pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##Load object \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr1_shortened_pickle', 'rb') as f:\n",
    "    data = pickle.load(f)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                 WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs62635286_G           1       1      1      1      1      1       0       0   \n",
      "rs62635286_HET         1       1      1      1      1      1       0       0   \n",
      "rs200579949_G          1       1      1      1      1      1       0       0   \n",
      "rs200579949_HET        1       1      1      1      1      1       0       0   \n",
      "rs75454623_G           1       1      1      1      1      1       1       1   \n",
      "rs75454623_HET         1       1      1      1      1      1       1       1   \n",
      "rs78601809_T           1       1      1      1      1      1       1       1   \n",
      "rs78601809_HET         1       1      1      1      1      1       1       1   \n",
      "rs2691277_0            0       0      0      0      0      0       0       0   \n",
      "rs2691277_HET          0       0      0      0      0      0       0       0   \n",
      "rs3091274_0            0       0      0      0      0      0       0       0   \n",
      "rs3091274_HET          0       0      0      0      0      0       0       0   \n",
      "rs377161483_AATG       1       1      1      1      0      1       1       1   \n",
      "rs377161483_HET        1       1      1      1      0      1       1       1   \n",
      "rs62642131_A           1       0      0      0      1      0       0       0   \n",
      "rs62642131_HET         1       0      0      0      1      0       0       0   \n",
      "rs1856864_C            1       1      1      1      1      1       1       1   \n",
      "rs1856864_HET          1       1      1      1      1      1       1       1   \n",
      "rs1578391_T            1       1      1      1      1      1       1       1   \n",
      "rs1578391_HET          1       1      1      1      1      1       1       1   \n",
      "rs6594029_T            1       1      1      1      1      1       0       0   \n",
      "rs6594029_HET          1       1      1      1      1      1       0       0   \n",
      "rs3021087_G            1       1      1      1      1      1       0       0   \n",
      "rs3021087_HET          1       1      1      1      1      1       0       0   \n",
      "rs6650105_A            1       1      1      1      1      1       2       2   \n",
      "rs6650105_HET          1       1      1      1      1      1       0       0   \n",
      "rs78732431_T           1       1      1      1      1      1       0       0   \n",
      "rs78732431_HET         1       1      1      1      1      1       0       0   \n",
      "rs530588710_C          1       1      1      1      1      1       0       0   \n",
      "rs530588710_HET        1       1      1      1      1      1       0       0   \n",
      "...                  ...     ...    ...    ...    ...    ...     ...     ...   \n",
      "rs61776804_G           1       1      0      0      1      0       1       1   \n",
      "rs61776804_HET         1       1      0      0      1      0       1       1   \n",
      "rs2076329_C            1       1      2      2      1      2       0       0   \n",
      "rs2076329_HET          1       1      0      0      1      0       0       0   \n",
      "rs1059822_G            1       1      0      0      1      0       1       1   \n",
      "rs1059822_HET          1       1      0      0      1      0       1       1   \n",
      "rs9439480_G            1       1      2      2      1      2       1       1   \n",
      "rs9439480_HET          1       1      0      0      1      0       1       1   \n",
      "rs17162876_G           1       1      0      0      1      0       1       1   \n",
      "rs17162876_HET         1       1      0      0      1      0       1       1   \n",
      "rs138369165_C          1       1      0      0      1      1       1       1   \n",
      "rs138369165_HET        1       1      0      0      1      1       1       1   \n",
      "rs149613000_C          1       1      2      2      1      2       1       1   \n",
      "rs149613000_HET        1       1      0      0      1      0       1       1   \n",
      "rs74378830_C           1       1      1      2      1      2       0       0   \n",
      "rs74378830_HET         1       1      1      0      1      0       0       0   \n",
      "rs61776853_A           0       0      0      0      0      0       0       0   \n",
      "rs61776853_HET         0       0      0      0      0      0       0       0   \n",
      "rs61776857_T           0       0      0      1      0      1       0       0   \n",
      "rs61776857_HET         0       0      0      1      0      1       0       0   \n",
      "rs111515312_0          0       0      0      0      0      0       0       0   \n",
      "rs111515312_HET        0       0      0      0      0      0       0       0   \n",
      "rs74045988_T           1       1      1      1      1      1       0       0   \n",
      "rs74045988_HET         1       1      1      1      1      1       0       0   \n",
      "rs4648784_G            1       1      1      1      1      1       0       0   \n",
      "rs4648784_HET          1       1      1      1      1      1       0       0   \n",
      "rs57481478_ATTT        1       1      0      1      0      1       0       0   \n",
      "rs57481478_HET         1       1      0      1      0      1       0       0   \n",
      "rs1061884_T            1       1      1      1      1      1       1       1   \n",
      "rs1061884_HET          1       1      1      1      1      1       1       1   \n",
      "\n",
      "                 imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3  \\\n",
      "rs62635286_G           0       0  ...         0       0       0      0   \n",
      "rs62635286_HET         0       0  ...         0       0       0      0   \n",
      "rs200579949_G          0       0  ...         0       0       0      0   \n",
      "rs200579949_HET        0       0  ...         0       0       0      0   \n",
      "rs75454623_G           1       1  ...         1       2       1      1   \n",
      "rs75454623_HET         1       1  ...         1       0       1      1   \n",
      "rs78601809_T           0       1  ...         0       0       0      1   \n",
      "rs78601809_HET         0       1  ...         0       0       0      1   \n",
      "rs2691277_0            0       0  ...         0       0       0      0   \n",
      "rs2691277_HET          0       0  ...         0       0       0      0   \n",
      "rs3091274_0            0       0  ...         0       0       0      0   \n",
      "rs3091274_HET          0       0  ...         0       0       0      0   \n",
      "rs377161483_AATG       0       1  ...         0       0       0      1   \n",
      "rs377161483_HET        0       1  ...         0       0       0      1   \n",
      "rs62642131_A           0       0  ...         0       0       0      0   \n",
      "rs62642131_HET         0       0  ...         0       0       0      0   \n",
      "rs1856864_C            1       1  ...         0       0       1      0   \n",
      "rs1856864_HET          1       1  ...         0       0       1      0   \n",
      "rs1578391_T            1       1  ...         0       2       1      1   \n",
      "rs1578391_HET          1       1  ...         0       0       1      1   \n",
      "rs6594029_T            1       1  ...         0       0       1      0   \n",
      "rs6594029_HET          1       1  ...         0       0       1      0   \n",
      "rs3021087_G            0       0  ...         0       0       1      0   \n",
      "rs3021087_HET          0       0  ...         0       0       1      0   \n",
      "rs6650105_A            1       1  ...         1       2       1      1   \n",
      "rs6650105_HET          1       1  ...         1       0       1      1   \n",
      "rs78732431_T           0       0  ...         0       0       0      0   \n",
      "rs78732431_HET         0       0  ...         0       0       0      0   \n",
      "rs530588710_C          0       1  ...         0       0       0      0   \n",
      "rs530588710_HET        0       1  ...         0       0       0      0   \n",
      "...                  ...     ...  ...       ...     ...     ...    ...   \n",
      "rs61776804_G           2       1  ...         1       0       2      1   \n",
      "rs61776804_HET         0       1  ...         1       0       0      1   \n",
      "rs2076329_C            0       2  ...         1       2       0      1   \n",
      "rs2076329_HET          0       0  ...         1       0       0      1   \n",
      "rs1059822_G            2       0  ...         1       0       2      1   \n",
      "rs1059822_HET          0       0  ...         1       0       0      1   \n",
      "rs9439480_G            0       1  ...         0       0       0      0   \n",
      "rs9439480_HET          0       1  ...         0       0       0      0   \n",
      "rs17162876_G           0       0  ...         1       2       0      0   \n",
      "rs17162876_HET         0       0  ...         1       0       0      0   \n",
      "rs138369165_C          2       1  ...         1       0       2      1   \n",
      "rs138369165_HET        0       1  ...         1       0       0      1   \n",
      "rs149613000_C          0       1  ...         1       0       0      1   \n",
      "rs149613000_HET        0       1  ...         1       0       0      1   \n",
      "rs74378830_C           0       0  ...         1       1       0      0   \n",
      "rs74378830_HET         0       0  ...         1       1       0      0   \n",
      "rs61776853_A           0       0  ...         0       0       0      0   \n",
      "rs61776853_HET         0       0  ...         0       0       0      0   \n",
      "rs61776857_T           0       1  ...         0       0       1      1   \n",
      "rs61776857_HET         0       1  ...         0       0       1      1   \n",
      "rs111515312_0          0       0  ...         0       0       0      0   \n",
      "rs111515312_HET        0       0  ...         0       0       0      0   \n",
      "rs74045988_T           0       0  ...         0       0       0      0   \n",
      "rs74045988_HET         0       0  ...         0       0       0      0   \n",
      "rs4648784_G            0       1  ...         0       1       1      1   \n",
      "rs4648784_HET          0       1  ...         0       1       1      1   \n",
      "rs57481478_ATTT        0       0  ...         2       0       1      1   \n",
      "rs57481478_HET         0       0  ...         0       0       1      1   \n",
      "rs1061884_T            1       0  ...         1       1       0      0   \n",
      "rs1061884_HET          1       0  ...         1       1       0      0   \n",
      "\n",
      "                 imp_P31 imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs62635286_G           0      0      0      0      0      0  \n",
      "rs62635286_HET         0      0      0      0      0      0  \n",
      "rs200579949_G          0      0      0      0      0      0  \n",
      "rs200579949_HET        0      0      0      0      0      0  \n",
      "rs75454623_G           0      1      1      1      1      1  \n",
      "rs75454623_HET         0      1      1      1      1      1  \n",
      "rs78601809_T           1      1      1      0      0      0  \n",
      "rs78601809_HET         1      1      1      0      0      0  \n",
      "rs2691277_0            0      0      0      0      0      0  \n",
      "rs2691277_HET          0      0      0      0      0      0  \n",
      "rs3091274_0            0      0      0      0      0      0  \n",
      "rs3091274_HET          0      0      0      0      0      0  \n",
      "rs377161483_AATG       1      1      1      0      0      0  \n",
      "rs377161483_HET        1      1      1      0      0      0  \n",
      "rs62642131_A           0      0      0      0      0      0  \n",
      "rs62642131_HET         0      0      0      0      0      0  \n",
      "rs1856864_C            0      1      1      0      0      0  \n",
      "rs1856864_HET          0      1      1      0      0      0  \n",
      "rs1578391_T            0      1      1      1      1      0  \n",
      "rs1578391_HET          0      1      1      1      1      0  \n",
      "rs6594029_T            0      1      1      0      0      0  \n",
      "rs6594029_HET          0      1      1      0      0      0  \n",
      "rs3021087_G            0      0      0      0      0      0  \n",
      "rs3021087_HET          0      0      0      0      0      0  \n",
      "rs6650105_A            0      1      1      1      0      0  \n",
      "rs6650105_HET          0      1      1      1      0      0  \n",
      "rs78732431_T           0      0      0      0      0      0  \n",
      "rs78732431_HET         0      0      0      0      0      0  \n",
      "rs530588710_C          0      0      0      0      0      0  \n",
      "rs530588710_HET        0      0      0      0      0      0  \n",
      "...                  ...    ...    ...    ...    ...    ...  \n",
      "rs61776804_G           1      0      0      1      1      0  \n",
      "rs61776804_HET         1      0      0      1      1      0  \n",
      "rs2076329_C            1      2      2      0      0      2  \n",
      "rs2076329_HET          1      0      0      0      0      0  \n",
      "rs1059822_G            0      0      0      1      1      0  \n",
      "rs1059822_HET          0      0      0      1      1      0  \n",
      "rs9439480_G            1      1      1      1      0      1  \n",
      "rs9439480_HET          1      1      1      1      0      1  \n",
      "rs17162876_G           0      0      2      0      0      2  \n",
      "rs17162876_HET         0      0      0      0      0      0  \n",
      "rs138369165_C          1      0      0      1      1      0  \n",
      "rs138369165_HET        1      0      0      1      1      0  \n",
      "rs149613000_C          1      1      2      1      1      2  \n",
      "rs149613000_HET        1      1      0      1      1      0  \n",
      "rs74378830_C           1      2      1      0      0      2  \n",
      "rs74378830_HET         1      0      1      0      0      0  \n",
      "rs61776853_A           0      0      0      0      0      0  \n",
      "rs61776853_HET         0      0      0      0      0      0  \n",
      "rs61776857_T           0      1      2      1      1      2  \n",
      "rs61776857_HET         0      1      0      1      1      0  \n",
      "rs111515312_0          0      0      0      0      0      0  \n",
      "rs111515312_HET        0      0      0      0      0      0  \n",
      "rs74045988_T           0      0      0      0      0      0  \n",
      "rs74045988_HET         0      0      0      0      0      0  \n",
      "rs4648784_G            0      1      1      1      0      1  \n",
      "rs4648784_HET          0      1      1      1      0      1  \n",
      "rs57481478_ATTT        0      0      2      1      1      1  \n",
      "rs57481478_HET         0      0      0      1      1      1  \n",
      "rs1061884_T            1      1      1      0      1      1  \n",
      "rs1061884_HET          1      1      1      0      1      1  \n",
      "\n",
      "[1396 rows x 29 columns]\n"
     ]
    }
   ],
   "source": [
    "print(pd_genotype)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_1_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr1.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr1.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr1\n",
      "plink --file concatenated_imputation_and_WGS_output_chr1_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr1\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_2_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr2.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr2.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_2_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr2.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr2.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr2\n",
      "plink --file concatenated_imputation_and_WGS_output_chr2_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr2\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_3_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr3.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr3.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_3_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr3.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr3.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr3\n",
      "plink --file concatenated_imputation_and_WGS_output_chr3_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr3\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_4_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr4.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr4.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_4_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr4.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr4.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr4\n",
      "plink --file concatenated_imputation_and_WGS_output_chr4_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr4\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_5_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr5.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr5.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_5_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr5.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr5.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr5\n",
      "plink --file concatenated_imputation_and_WGS_output_chr5_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr5\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_6_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr6.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr6.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_6_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr6.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr6.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr6\n",
      "plink --file concatenated_imputation_and_WGS_output_chr6_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr6\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_7_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr7.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr7.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_7_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr7.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr7.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr7\n",
      "plink --file concatenated_imputation_and_WGS_output_chr7_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr7\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_8_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr8.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr8.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_8_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr8.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr8.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr8\n",
      "plink --file concatenated_imputation_and_WGS_output_chr8_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr8\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_9_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr9.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr9.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_9_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr9.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr9.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr9\n",
      "plink --file concatenated_imputation_and_WGS_output_chr9_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr9\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_10_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr10.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr10.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr10.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr10.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr10\n",
      "plink --file concatenated_imputation_and_WGS_output_chr10_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr10\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_11_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr11.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr11.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_11_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr11.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr11.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr11\n",
      "plink --file concatenated_imputation_and_WGS_output_chr11_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr11\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_12_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr12.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr12.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_12_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr12.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr12.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr12\n",
      "plink --file concatenated_imputation_and_WGS_output_chr12_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr12\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_13_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr13.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr13.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_13_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr13.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr13.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr13\n",
      "plink --file concatenated_imputation_and_WGS_output_chr13_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr13\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_14_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr14.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr14.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_14_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr14.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr14.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr14\n",
      "plink --file concatenated_imputation_and_WGS_output_chr14_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr14\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_15_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr15.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr15.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_15_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr15.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr15.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr15\n",
      "plink --file concatenated_imputation_and_WGS_output_chr15_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr15\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_16_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr16.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr16.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_16_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr16.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr16.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr16\n",
      "plink --file concatenated_imputation_and_WGS_output_chr16_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr16\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_17_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr17.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr17.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_17_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr17.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr17.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr17\n",
      "plink --file concatenated_imputation_and_WGS_output_chr17_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr17\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_18_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr18.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr18.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_18_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr18.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr18.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr18\n",
      "plink --file concatenated_imputation_and_WGS_output_chr18_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr18\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_19_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr19.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr19.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_19_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr19.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr19.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr19\n",
      "plink --file concatenated_imputation_and_WGS_output_chr19_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr19\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_20_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr20.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr20.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_20_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr20.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr20.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr20\n",
      "plink --file concatenated_imputation_and_WGS_output_chr20_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr20\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_21_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr21.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr21.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_21_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr21.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr21.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr21\n",
      "plink --file concatenated_imputation_and_WGS_output_chr21_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr21\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf chr_22_imputed_phased_v5a.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr22.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr22.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_22_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr22.vcf.gz\n",
      "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr22.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr22\n",
      "plink --file concatenated_imputation_and_WGS_output_chr22_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr22\n"
     ]
    }
   ],
   "source": [
    "######Prepare Each chromosome genotype file separetely: WGS and imputation, index, and merge \n",
    "nrs =list(range(22))\n",
    "nnrs=list()\n",
    "\n",
    "for nr in nrs:\n",
    "    print(\"\\n\")\n",
    "    print(\"export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\")\n",
    "    print(\"tabix -p vcf chr_\" + str(nr +1) + \"_imputed_phased_v5a.vcf.gz\")\n",
    "    print(\"tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr\" + str(nr +1) + \".vcf.gz\")\n",
    "    print(\"vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr\" + str(nr+1) + \".recode.vcf.gz \" \\\n",
    "    \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr+1) +\"_imputed_phased_v5a.vcf.gz\" \\\n",
    "    \"| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz\")\n",
    "    print(\"vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr\" + str(nr+1))\n",
    "    print(\"plink --file concatenated_imputation_and_WGS_output_chr\" + str(nr+1) +\"_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr\" + str(nr+1))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##########################################\n",
    "####################Read whole genome! \n",
    "data = pandas.read_table(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/genotype_converted_to_012_format.raw\", sep = \" \")\n",
    "\n",
    "##Transform in numpy object, transform and convert to pandas\n",
    "table = data.as_matrix()\n",
    "genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "pd_genotype = pd_genotype[6:]\n",
    "\n",
    "#print(pd_genotype[0:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Select only even rows (because they are duplicate)\n",
    "pd_genotype =pd_genotype[::2]\n",
    "pd_genotype.shape\n",
    "\n",
    "print(pd_genotype[1:5])\n",
    "\n",
    "##Safe object as pickle \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/whole_genome_pickle', 'wb') as f:\n",
    "    pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####################################\n",
    "##Remove NA values \n",
    "pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "#print(pd_genotype[1:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Calculate correlations \n",
    "plot_data = pd_genotype.astype(float).corr()\n",
    "#print(plot_data)\n",
    "\n",
    "PLOT_CMAP = 'YlOrRd'\n",
    "VMIN = 0\n",
    "VMAX = 1\n",
    "fig_size=(12,12)\n",
    "\n",
    "##PLot and write to files\n",
    "\n",
    "cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "_=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS.pdf',  bbox_inches='tight')\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS.png',  bbox_inches='tight')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "[Wed Jan 11 17:21:26 PST 2017] picard.vcf.GenotypeConcordance TRUTH_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz CALL_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz OUTPUT=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P15_truth_WGS_P15.vcf TRUTH_SAMPLE=WGS_P15 CALL_SAMPLE=WGS_P15    INTERSECT_INTERVALS=true MIN_GQ=0 MIN_DP=0 OUTPUT_ALL_ROWS=false USE_VCF_INDEX=false MISSING_SITES_HOM_REF=false VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Wed Jan 11 17:21:26 PST 2017] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.3.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2017-01-11 17:21:26\tGenotypeConcordance\tStarting iteration over variants.\n",
      "INFO\t2017-01-11 17:21:27\tGenotypeConcordance\tchecked        10,000 variants.  Elapsed time: 00:00:00s.  Time for last 10,000:    0s.  Last read position: 1:1,120,957\n",
      "INFO\t2017-01-11 17:21:27\tGenotypeConcordance\tchecked        20,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:1,707,743\n",
      "INFO\t2017-01-11 17:21:27\tGenotypeConcordance\tchecked        30,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:2,389,563\n",
      "INFO\t2017-01-11 17:21:28\tGenotypeConcordance\tchecked        40,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,017,290\n",
      "INFO\t2017-01-11 17:21:28\tGenotypeConcordance\tchecked        50,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,627,524\n",
      "INFO\t2017-01-11 17:21:29\tGenotypeConcordance\tchecked        60,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:4,394,403\n",
      "INFO\t2017-01-11 17:21:29\tGenotypeConcordance\tchecked        70,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,073,378\n",
      "INFO\t2017-01-11 17:21:29\tGenotypeConcordance\tchecked        80,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,732,295\n",
      "INFO\t2017-01-11 17:21:29\tGenotypeConcordance\tchecked        90,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:6,438,378\n",
      "INFO\t2017-01-11 17:21:30\tGenotypeConcordance\tchecked       100,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:7,315,516\n",
      "INFO\t2017-01-11 17:21:30\tGenotypeConcordance\tchecked       110,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:8,086,334\n",
      "INFO\t2017-01-11 17:21:30\tGenotypeConcordance\tchecked       120,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:8,968,709\n",
      "INFO\t2017-01-11 17:21:31\tGenotypeConcordance\tchecked       130,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:9,716,019\n",
      "INFO\t2017-01-11 17:21:31\tGenotypeConcordance\tchecked       140,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:10,623,841\n",
      "INFO\t2017-01-11 17:21:31\tGenotypeConcordance\tchecked       150,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:11,403,951\n",
      "INFO\t2017-01-11 17:21:32\tGenotypeConcordance\tchecked       160,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:12,131,083\n",
      "INFO\t2017-01-11 17:21:32\tGenotypeConcordance\tchecked       170,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:12,899,984\n",
      "INFO\t2017-01-11 17:21:32\tGenotypeConcordance\tchecked       180,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:14,034,254\n",
      "INFO\t2017-01-11 17:21:32\tGenotypeConcordance\tchecked       190,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:14,782,380\n",
      "INFO\t2017-01-11 17:21:33\tGenotypeConcordance\tchecked       200,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:15,518,433\n",
      "INFO\t2017-01-11 17:21:33\tGenotypeConcordance\tchecked       210,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:16,317,389\n",
      "INFO\t2017-01-11 17:21:33\tGenotypeConcordance\tchecked       220,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:16,940,194\n",
      "INFO\t2017-01-11 17:21:34\tGenotypeConcordance\tchecked       230,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:17,543,832\n",
      "INFO\t2017-01-11 17:21:34\tGenotypeConcordance\tchecked       240,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:18,280,366\n",
      "INFO\t2017-01-11 17:21:34\tGenotypeConcordance\tchecked       250,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:19,026,057\n",
      "INFO\t2017-01-11 17:21:35\tGenotypeConcordance\tchecked       260,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:19,777,837\n",
      "INFO\t2017-01-11 17:21:35\tGenotypeConcordance\tchecked       270,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:20,592,124\n",
      "INFO\t2017-01-11 17:21:35\tGenotypeConcordance\tchecked       280,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:21,534,929\n",
      "INFO\t2017-01-11 17:21:36\tGenotypeConcordance\tchecked       290,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:22,326,328\n",
      "INFO\t2017-01-11 17:21:36\tGenotypeConcordance\tchecked       300,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:23,116,975\n",
      "INFO\t2017-01-11 17:21:36\tGenotypeConcordance\tchecked       310,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:24,100,808\n",
      "INFO\t2017-01-11 17:21:37\tGenotypeConcordance\tchecked       320,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:25,015,530\n",
      "INFO\t2017-01-11 17:21:37\tGenotypeConcordance\tchecked       330,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:25,907,167\n",
      "INFO\t2017-01-11 17:21:37\tGenotypeConcordance\tchecked       340,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:26,844,752\n",
      "INFO\t2017-01-11 17:21:37\tGenotypeConcordance\tchecked       350,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:27,980,714\n",
      "INFO\t2017-01-11 17:21:38\tGenotypeConcordance\tchecked       360,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:28,981,150\n",
      "INFO\t2017-01-11 17:21:38\tGenotypeConcordance\tchecked       370,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:30,094,819\n",
      "INFO\t2017-01-11 17:21:38\tGenotypeConcordance\tchecked       380,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:30,842,172\n",
      "INFO\t2017-01-11 17:21:39\tGenotypeConcordance\tchecked       390,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:31,587,628\n",
      "INFO\t2017-01-11 17:21:39\tGenotypeConcordance\tchecked       400,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:32,447,963\n",
      "INFO\t2017-01-11 17:21:39\tGenotypeConcordance\tchecked       410,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:33,489,498\n",
      "INFO\t2017-01-11 17:21:40\tGenotypeConcordance\tchecked       420,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:34,445,892\n",
      "INFO\t2017-01-11 17:21:40\tGenotypeConcordance\tchecked       430,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:35,297,017\n",
      "INFO\t2017-01-11 17:21:40\tGenotypeConcordance\tchecked       440,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:36,349,560\n",
      "INFO\t2017-01-11 17:21:40\tGenotypeConcordance\tchecked       450,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:37,300,320\n",
      "INFO\t2017-01-11 17:21:41\tGenotypeConcordance\tchecked       460,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:38,174,415\n",
      "INFO\t2017-01-11 17:21:41\tGenotypeConcordance\tchecked       470,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:39,100,863\n",
      "INFO\t2017-01-11 17:21:41\tGenotypeConcordance\tchecked       480,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:40,075,458\n",
      "INFO\t2017-01-11 17:21:42\tGenotypeConcordance\tchecked       490,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:40,968,201\n",
      "INFO\t2017-01-11 17:21:42\tGenotypeConcordance\tchecked       500,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:41,904,021\n",
      "INFO\t2017-01-11 17:21:42\tGenotypeConcordance\tchecked       510,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:42,843,316\n",
      "INFO\t2017-01-11 17:21:42\tGenotypeConcordance\tchecked       520,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:43,718,158\n",
      "INFO\t2017-01-11 17:21:43\tGenotypeConcordance\tchecked       530,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:44,706,113\n",
      "INFO\t2017-01-11 17:21:43\tGenotypeConcordance\tchecked       540,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:45,691,503\n",
      "INFO\t2017-01-11 17:21:43\tGenotypeConcordance\tchecked       550,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:46,757,990\n",
      "INFO\t2017-01-11 17:21:43\tGenotypeConcordance\tchecked       560,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:47,610,065\n",
      "INFO\t2017-01-11 17:21:44\tGenotypeConcordance\tchecked       570,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:48,529,955\n",
      "INFO\t2017-01-11 17:21:44\tGenotypeConcordance\tchecked       580,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:49,495,669\n",
      "INFO\t2017-01-11 17:21:44\tGenotypeConcordance\tchecked       590,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:50,815,397\n",
      "INFO\t2017-01-11 17:21:45\tGenotypeConcordance\tchecked       600,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:51,923,014\n",
      "INFO\t2017-01-11 17:21:45\tGenotypeConcordance\tchecked       610,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:52,935,423\n",
      "INFO\t2017-01-11 17:21:45\tGenotypeConcordance\tchecked       620,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:53,802,466\n",
      "INFO\t2017-01-11 17:21:45\tGenotypeConcordance\tchecked       630,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:54,708,123\n",
      "INFO\t2017-01-11 17:21:46\tGenotypeConcordance\tchecked       640,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:55,450,942\n",
      "INFO\t2017-01-11 17:21:46\tGenotypeConcordance\tchecked       650,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:56,258,074\n",
      "INFO\t2017-01-11 17:21:46\tGenotypeConcordance\tchecked       660,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:57,130,566\n",
      "INFO\t2017-01-11 17:21:47\tGenotypeConcordance\tchecked       670,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:57,986,610\n",
      "INFO\t2017-01-11 17:21:47\tGenotypeConcordance\tchecked       680,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:58,874,734\n",
      "INFO\t2017-01-11 17:21:47\tGenotypeConcordance\tchecked       690,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:59,704,642\n",
      "INFO\t2017-01-11 17:21:47\tGenotypeConcordance\tchecked       700,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:60,635,069\n",
      "INFO\t2017-01-11 17:21:48\tGenotypeConcordance\tchecked       710,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:61,525,763\n",
      "INFO\t2017-01-11 17:21:48\tGenotypeConcordance\tchecked       720,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:62,435,181\n",
      "INFO\t2017-01-11 17:21:48\tGenotypeConcordance\tchecked       730,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:63,297,853\n",
      "INFO\t2017-01-11 17:21:49\tGenotypeConcordance\tchecked       740,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:64,149,558\n",
      "INFO\t2017-01-11 17:21:49\tGenotypeConcordance\tchecked       750,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:65,061,144\n",
      "INFO\t2017-01-11 17:21:49\tGenotypeConcordance\tchecked       760,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:66,100,450\n",
      "INFO\t2017-01-11 17:21:50\tGenotypeConcordance\tchecked       770,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:67,014,855\n",
      "INFO\t2017-01-11 17:21:50\tGenotypeConcordance\tchecked       780,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:67,947,783\n",
      "INFO\t2017-01-11 17:21:50\tGenotypeConcordance\tchecked       790,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:68,799,916\n",
      "INFO\t2017-01-11 17:21:50\tGenotypeConcordance\tchecked       800,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:69,777,695\n",
      "INFO\t2017-01-11 17:21:51\tGenotypeConcordance\tchecked       810,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:70,764,703\n",
      "INFO\t2017-01-11 17:21:51\tGenotypeConcordance\tchecked       820,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:71,722,094\n",
      "INFO\t2017-01-11 17:21:51\tGenotypeConcordance\tchecked       830,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:72,751,572\n",
      "INFO\t2017-01-11 17:21:51\tGenotypeConcordance\tchecked       840,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:73,686,618\n",
      "INFO\t2017-01-11 17:21:52\tGenotypeConcordance\tchecked       850,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:74,633,797\n",
      "INFO\t2017-01-11 17:21:52\tGenotypeConcordance\tchecked       860,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:75,588,924\n",
      "INFO\t2017-01-11 17:21:52\tGenotypeConcordance\tchecked       870,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:76,479,278\n",
      "INFO\t2017-01-11 17:21:53\tGenotypeConcordance\tchecked       880,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:77,378,755\n",
      "INFO\t2017-01-11 17:21:53\tGenotypeConcordance\tchecked       890,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:78,442,124\n",
      "INFO\t2017-01-11 17:21:53\tGenotypeConcordance\tchecked       900,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:79,343,051\n",
      "INFO\t2017-01-11 17:21:53\tGenotypeConcordance\tchecked       910,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:80,107,710\n",
      "INFO\t2017-01-11 17:21:54\tGenotypeConcordance\tchecked       920,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:80,894,520\n",
      "INFO\t2017-01-11 17:21:54\tGenotypeConcordance\tchecked       930,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:81,661,481\n",
      "INFO\t2017-01-11 17:21:54\tGenotypeConcordance\tchecked       940,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:82,483,449\n",
      "INFO\t2017-01-11 17:21:55\tGenotypeConcordance\tchecked       950,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:83,299,949\n",
      "INFO\t2017-01-11 17:21:55\tGenotypeConcordance\tchecked       960,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:84,203,620\n",
      "INFO\t2017-01-11 17:21:55\tGenotypeConcordance\tchecked       970,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:85,182,774\n",
      "INFO\t2017-01-11 17:21:55\tGenotypeConcordance\tchecked       980,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:86,067,680\n",
      "INFO\t2017-01-11 17:21:56\tGenotypeConcordance\tchecked       990,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:86,962,209\n",
      "INFO\t2017-01-11 17:21:56\tGenotypeConcordance\tchecked     1,000,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:87,883,535\n",
      "INFO\t2017-01-11 17:21:56\tGenotypeConcordance\tchecked     1,010,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:88,852,286\n",
      "INFO\t2017-01-11 17:21:57\tGenotypeConcordance\tchecked     1,020,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:89,803,108\n",
      "INFO\t2017-01-11 17:21:57\tGenotypeConcordance\tchecked     1,030,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:90,776,039\n",
      "INFO\t2017-01-11 17:21:57\tGenotypeConcordance\tchecked     1,040,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:91,725,653\n",
      "INFO\t2017-01-11 17:21:57\tGenotypeConcordance\tchecked     1,050,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:92,586,331\n",
      "INFO\t2017-01-11 17:21:58\tGenotypeConcordance\tchecked     1,060,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:93,572,071\n",
      "INFO\t2017-01-11 17:21:58\tGenotypeConcordance\tchecked     1,070,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:94,567,490\n",
      "INFO\t2017-01-11 17:21:58\tGenotypeConcordance\tchecked     1,080,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:95,464,821\n",
      "INFO\t2017-01-11 17:21:59\tGenotypeConcordance\tchecked     1,090,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:96,459,037\n",
      "INFO\t2017-01-11 17:21:59\tGenotypeConcordance\tchecked     1,100,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:97,413,779\n",
      "INFO\t2017-01-11 17:21:59\tGenotypeConcordance\tchecked     1,110,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:98,403,989\n",
      "INFO\t2017-01-11 17:21:59\tGenotypeConcordance\tchecked     1,120,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:99,373,916\n",
      "INFO\t2017-01-11 17:22:00\tGenotypeConcordance\tchecked     1,130,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:100,226,223\n",
      "INFO\t2017-01-11 17:22:00\tGenotypeConcordance\tchecked     1,140,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:101,205,712\n",
      "INFO\t2017-01-11 17:22:00\tGenotypeConcordance\tchecked     1,150,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:102,069,120\n",
      "INFO\t2017-01-11 17:22:00\tGenotypeConcordance\tchecked     1,160,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:102,856,121\n",
      "INFO\t2017-01-11 17:22:01\tGenotypeConcordance\tchecked     1,170,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:103,594,920\n",
      "INFO\t2017-01-11 17:22:01\tGenotypeConcordance\tchecked     1,180,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:104,787,833\n",
      "INFO\t2017-01-11 17:22:01\tGenotypeConcordance\tchecked     1,190,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:105,436,080\n",
      "INFO\t2017-01-11 17:22:02\tGenotypeConcordance\tchecked     1,200,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:106,102,802\n",
      "INFO\t2017-01-11 17:22:02\tGenotypeConcordance\tchecked     1,210,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:106,860,402\n",
      "INFO\t2017-01-11 17:22:02\tGenotypeConcordance\tchecked     1,220,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:107,709,523\n",
      "INFO\t2017-01-11 17:22:02\tGenotypeConcordance\tchecked     1,230,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:108,611,935\n",
      "INFO\t2017-01-11 17:22:03\tGenotypeConcordance\tchecked     1,240,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:109,661,045\n",
      "INFO\t2017-01-11 17:22:03\tGenotypeConcordance\tchecked     1,250,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:110,640,998\n",
      "INFO\t2017-01-11 17:22:03\tGenotypeConcordance\tchecked     1,260,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:111,585,494\n",
      "INFO\t2017-01-11 17:22:04\tGenotypeConcordance\tchecked     1,270,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:112,468,814\n",
      "INFO\t2017-01-11 17:22:04\tGenotypeConcordance\tchecked     1,280,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:113,310,684\n",
      "INFO\t2017-01-11 17:22:04\tGenotypeConcordance\tchecked     1,290,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:114,333,600\n",
      "INFO\t2017-01-11 17:22:04\tGenotypeConcordance\tchecked     1,300,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:115,277,766\n",
      "INFO\t2017-01-11 17:22:05\tGenotypeConcordance\tchecked     1,310,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:116,173,981\n",
      "INFO\t2017-01-11 17:22:05\tGenotypeConcordance\tchecked     1,320,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:117,088,239\n",
      "INFO\t2017-01-11 17:22:05\tGenotypeConcordance\tchecked     1,330,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:118,014,306\n",
      "INFO\t2017-01-11 17:22:05\tGenotypeConcordance\tchecked     1,340,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:118,986,025\n",
      "INFO\t2017-01-11 17:22:06\tGenotypeConcordance\tchecked     1,350,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:119,895,482\n",
      "INFO\t2017-01-11 17:22:06\tGenotypeConcordance\tchecked     1,360,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:120,855,863\n",
      "INFO\t2017-01-11 17:22:06\tGenotypeConcordance\tchecked     1,370,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:142,560,186\n",
      "INFO\t2017-01-11 17:22:07\tGenotypeConcordance\tchecked     1,380,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:143,134,694\n",
      "INFO\t2017-01-11 17:22:07\tGenotypeConcordance\tchecked     1,390,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:143,518,251\n",
      "INFO\t2017-01-11 17:22:07\tGenotypeConcordance\tchecked     1,400,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:144,956,013\n",
      "INFO\t2017-01-11 17:22:07\tGenotypeConcordance\tchecked     1,410,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:145,706,219\n",
      "INFO\t2017-01-11 17:22:08\tGenotypeConcordance\tchecked     1,420,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:146,994,229\n",
      "INFO\t2017-01-11 17:22:08\tGenotypeConcordance\tchecked     1,430,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:148,558,499\n",
      "INFO\t2017-01-11 17:22:08\tGenotypeConcordance\tchecked     1,440,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:149,287,681\n",
      "INFO\t2017-01-11 17:22:09\tGenotypeConcordance\tchecked     1,450,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:150,398,945\n",
      "INFO\t2017-01-11 17:22:09\tGenotypeConcordance\tchecked     1,460,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:151,459,516\n",
      "INFO\t2017-01-11 17:22:09\tGenotypeConcordance\tchecked     1,470,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:152,433,928\n",
      "INFO\t2017-01-11 17:22:09\tGenotypeConcordance\tchecked     1,480,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:153,175,210\n",
      "INFO\t2017-01-11 17:22:10\tGenotypeConcordance\tchecked     1,490,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:154,116,128\n",
      "INFO\t2017-01-11 17:22:10\tGenotypeConcordance\tchecked     1,500,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:155,169,355\n",
      "INFO\t2017-01-11 17:22:10\tGenotypeConcordance\tchecked     1,510,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:156,316,245\n",
      "INFO\t2017-01-11 17:22:11\tGenotypeConcordance\tchecked     1,520,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:157,227,945\n",
      "INFO\t2017-01-11 17:22:11\tGenotypeConcordance\tchecked     1,530,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:158,040,895\n",
      "INFO\t2017-01-11 17:22:11\tGenotypeConcordance\tchecked     1,540,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:158,772,702\n",
      "INFO\t2017-01-11 17:22:12\tGenotypeConcordance\tchecked     1,550,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:159,678,816\n",
      "INFO\t2017-01-11 17:22:12\tGenotypeConcordance\tchecked     1,560,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:160,585,017\n",
      "INFO\t2017-01-11 17:22:12\tGenotypeConcordance\tchecked     1,570,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:161,463,235\n",
      "INFO\t2017-01-11 17:22:12\tGenotypeConcordance\tchecked     1,580,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:162,347,238\n",
      "INFO\t2017-01-11 17:22:13\tGenotypeConcordance\tchecked     1,590,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:163,202,408\n",
      "INFO\t2017-01-11 17:22:13\tGenotypeConcordance\tchecked     1,600,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:164,043,656\n",
      "INFO\t2017-01-11 17:22:13\tGenotypeConcordance\tchecked     1,610,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:164,898,936\n",
      "INFO\t2017-01-11 17:22:14\tGenotypeConcordance\tchecked     1,620,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:165,684,677\n",
      "INFO\t2017-01-11 17:22:14\tGenotypeConcordance\tchecked     1,630,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:166,521,779\n",
      "INFO\t2017-01-11 17:22:14\tGenotypeConcordance\tchecked     1,640,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:167,398,628\n",
      "INFO\t2017-01-11 17:22:14\tGenotypeConcordance\tchecked     1,650,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:168,184,589\n",
      "INFO\t2017-01-11 17:22:15\tGenotypeConcordance\tchecked     1,660,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:168,994,546\n",
      "INFO\t2017-01-11 17:22:15\tGenotypeConcordance\tchecked     1,670,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:169,850,842\n",
      "INFO\t2017-01-11 17:22:15\tGenotypeConcordance\tchecked     1,680,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:170,782,473\n",
      "INFO\t2017-01-11 17:22:16\tGenotypeConcordance\tchecked     1,690,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:171,615,936\n",
      "INFO\t2017-01-11 17:22:16\tGenotypeConcordance\tchecked     1,700,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:172,529,121\n",
      "INFO\t2017-01-11 17:22:16\tGenotypeConcordance\tchecked     1,710,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:173,534,759\n",
      "INFO\t2017-01-11 17:22:16\tGenotypeConcordance\tchecked     1,720,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:174,599,343\n",
      "INFO\t2017-01-11 17:22:17\tGenotypeConcordance\tchecked     1,730,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:175,533,921\n",
      "INFO\t2017-01-11 17:22:17\tGenotypeConcordance\tchecked     1,740,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:176,392,987\n",
      "INFO\t2017-01-11 17:22:17\tGenotypeConcordance\tchecked     1,750,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:177,478,762\n",
      "INFO\t2017-01-11 17:22:17\tGenotypeConcordance\tchecked     1,760,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:178,444,857\n",
      "INFO\t2017-01-11 17:22:18\tGenotypeConcordance\tchecked     1,770,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:179,335,138\n",
      "INFO\t2017-01-11 17:22:18\tGenotypeConcordance\tchecked     1,780,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:180,187,922\n",
      "INFO\t2017-01-11 17:22:18\tGenotypeConcordance\tchecked     1,790,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:181,078,749\n",
      "INFO\t2017-01-11 17:22:19\tGenotypeConcordance\tchecked     1,800,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:181,947,148\n",
      "INFO\t2017-01-11 17:22:19\tGenotypeConcordance\tchecked     1,810,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:182,845,174\n",
      "INFO\t2017-01-11 17:22:19\tGenotypeConcordance\tchecked     1,820,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:183,791,551\n",
      "INFO\t2017-01-11 17:22:19\tGenotypeConcordance\tchecked     1,830,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:184,717,573\n",
      "INFO\t2017-01-11 17:22:20\tGenotypeConcordance\tchecked     1,840,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:185,649,535\n",
      "INFO\t2017-01-11 17:22:20\tGenotypeConcordance\tchecked     1,850,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:186,580,648\n",
      "INFO\t2017-01-11 17:22:20\tGenotypeConcordance\tchecked     1,860,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:187,490,339\n",
      "INFO\t2017-01-11 17:22:20\tGenotypeConcordance\tchecked     1,870,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:188,255,718\n",
      "INFO\t2017-01-11 17:22:21\tGenotypeConcordance\tchecked     1,880,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:188,965,252\n",
      "INFO\t2017-01-11 17:22:21\tGenotypeConcordance\tchecked     1,890,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:189,713,805\n",
      "INFO\t2017-01-11 17:22:21\tGenotypeConcordance\tchecked     1,900,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:190,541,681\n",
      "INFO\t2017-01-11 17:22:22\tGenotypeConcordance\tchecked     1,910,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:191,390,229\n",
      "INFO\t2017-01-11 17:22:22\tGenotypeConcordance\tchecked     1,920,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:192,341,209\n",
      "INFO\t2017-01-11 17:22:22\tGenotypeConcordance\tchecked     1,930,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:193,353,750\n",
      "INFO\t2017-01-11 17:22:22\tGenotypeConcordance\tchecked     1,940,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:194,158,102\n",
      "INFO\t2017-01-11 17:22:23\tGenotypeConcordance\tchecked     1,950,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:194,936,207\n",
      "INFO\t2017-01-11 17:22:23\tGenotypeConcordance\tchecked     1,960,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:195,802,260\n",
      "INFO\t2017-01-11 17:22:23\tGenotypeConcordance\tchecked     1,970,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:196,705,654\n",
      "INFO\t2017-01-11 17:22:23\tGenotypeConcordance\tchecked     1,980,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:197,591,397\n",
      "INFO\t2017-01-11 17:22:24\tGenotypeConcordance\tchecked     1,990,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:198,546,499\n",
      "INFO\t2017-01-11 17:22:24\tGenotypeConcordance\tchecked     2,000,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:199,523,656\n",
      "INFO\t2017-01-11 17:22:24\tGenotypeConcordance\tchecked     2,010,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:200,392,412\n",
      "INFO\t2017-01-11 17:22:25\tGenotypeConcordance\tchecked     2,020,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:201,243,556\n",
      "INFO\t2017-01-11 17:22:25\tGenotypeConcordance\tchecked     2,030,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:202,089,610\n",
      "INFO\t2017-01-11 17:22:25\tGenotypeConcordance\tchecked     2,040,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:203,062,824\n",
      "INFO\t2017-01-11 17:22:25\tGenotypeConcordance\tchecked     2,050,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:203,890,477\n",
      "INFO\t2017-01-11 17:22:26\tGenotypeConcordance\tchecked     2,060,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:204,741,992\n",
      "INFO\t2017-01-11 17:22:26\tGenotypeConcordance\tchecked     2,070,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:205,615,287\n",
      "INFO\t2017-01-11 17:22:26\tGenotypeConcordance\tchecked     2,080,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:206,882,135\n",
      "INFO\t2017-01-11 17:22:26\tGenotypeConcordance\tchecked     2,090,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:207,908,172\n",
      "INFO\t2017-01-11 17:22:27\tGenotypeConcordance\tchecked     2,100,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:208,782,188\n",
      "INFO\t2017-01-11 17:22:27\tGenotypeConcordance\tchecked     2,110,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:209,634,321\n",
      "INFO\t2017-01-11 17:22:27\tGenotypeConcordance\tchecked     2,120,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:210,528,934\n",
      "INFO\t2017-01-11 17:22:28\tGenotypeConcordance\tchecked     2,130,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:211,348,493\n",
      "INFO\t2017-01-11 17:22:28\tGenotypeConcordance\tchecked     2,140,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:212,233,950\n",
      "INFO\t2017-01-11 17:22:28\tGenotypeConcordance\tchecked     2,150,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:213,102,148\n",
      "INFO\t2017-01-11 17:22:28\tGenotypeConcordance\tchecked     2,160,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:214,041,894\n",
      "INFO\t2017-01-11 17:22:29\tGenotypeConcordance\tchecked     2,170,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:214,885,561\n",
      "INFO\t2017-01-11 17:22:29\tGenotypeConcordance\tchecked     2,180,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:215,832,378\n",
      "INFO\t2017-01-11 17:22:29\tGenotypeConcordance\tchecked     2,190,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:216,662,472\n",
      "INFO\t2017-01-11 17:22:30\tGenotypeConcordance\tchecked     2,200,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:217,442,851\n",
      "INFO\t2017-01-11 17:22:30\tGenotypeConcordance\tchecked     2,210,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:218,264,979\n",
      "INFO\t2017-01-11 17:22:30\tGenotypeConcordance\tchecked     2,220,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:219,125,281\n",
      "INFO\t2017-01-11 17:22:31\tGenotypeConcordance\tchecked     2,230,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:220,050,341\n",
      "INFO\t2017-01-11 17:22:31\tGenotypeConcordance\tchecked     2,240,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:220,962,453\n",
      "INFO\t2017-01-11 17:22:31\tGenotypeConcordance\tchecked     2,250,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:221,845,715\n",
      "INFO\t2017-01-11 17:22:31\tGenotypeConcordance\tchecked     2,260,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:222,741,944\n",
      "INFO\t2017-01-11 17:22:32\tGenotypeConcordance\tchecked     2,270,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:223,647,626\n",
      "INFO\t2017-01-11 17:22:32\tGenotypeConcordance\tchecked     2,280,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:224,520,873\n",
      "INFO\t2017-01-11 17:22:32\tGenotypeConcordance\tchecked     2,290,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:225,434,050\n",
      "INFO\t2017-01-11 17:22:33\tGenotypeConcordance\tchecked     2,300,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:226,371,672\n",
      "INFO\t2017-01-11 17:22:33\tGenotypeConcordance\tchecked     2,310,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:227,320,600\n",
      "INFO\t2017-01-11 17:22:33\tGenotypeConcordance\tchecked     2,320,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:228,129,263\n",
      "INFO\t2017-01-11 17:22:33\tGenotypeConcordance\tchecked     2,330,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:229,027,774\n",
      "INFO\t2017-01-11 17:22:34\tGenotypeConcordance\tchecked     2,340,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:229,841,108\n",
      "INFO\t2017-01-11 17:22:34\tGenotypeConcordance\tchecked     2,350,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:230,622,939\n",
      "INFO\t2017-01-11 17:22:34\tGenotypeConcordance\tchecked     2,360,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:231,468,446\n",
      "INFO\t2017-01-11 17:22:34\tGenotypeConcordance\tchecked     2,370,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:232,321,862\n",
      "INFO\t2017-01-11 17:22:35\tGenotypeConcordance\tchecked     2,380,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:233,153,225\n",
      "INFO\t2017-01-11 17:22:35\tGenotypeConcordance\tchecked     2,390,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:233,956,282\n",
      "INFO\t2017-01-11 17:22:35\tGenotypeConcordance\tchecked     2,400,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:234,728,108\n",
      "INFO\t2017-01-11 17:22:36\tGenotypeConcordance\tchecked     2,410,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:235,631,568\n",
      "INFO\t2017-01-11 17:22:36\tGenotypeConcordance\tchecked     2,420,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:236,469,498\n",
      "INFO\t2017-01-11 17:22:36\tGenotypeConcordance\tchecked     2,430,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:237,172,962\n",
      "INFO\t2017-01-11 17:22:36\tGenotypeConcordance\tchecked     2,440,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:237,962,535\n",
      "INFO\t2017-01-11 17:22:37\tGenotypeConcordance\tchecked     2,450,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:238,728,187\n",
      "INFO\t2017-01-11 17:22:37\tGenotypeConcordance\tchecked     2,460,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:239,511,263\n",
      "INFO\t2017-01-11 17:22:37\tGenotypeConcordance\tchecked     2,470,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:240,311,555\n",
      "INFO\t2017-01-11 17:22:38\tGenotypeConcordance\tchecked     2,480,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:241,083,566\n",
      "INFO\t2017-01-11 17:22:38\tGenotypeConcordance\tchecked     2,490,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:241,845,577\n",
      "INFO\t2017-01-11 17:22:38\tGenotypeConcordance\tchecked     2,500,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:242,532,846\n",
      "INFO\t2017-01-11 17:22:38\tGenotypeConcordance\tchecked     2,510,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:243,261,977\n",
      "INFO\t2017-01-11 17:22:39\tGenotypeConcordance\tchecked     2,520,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:244,311,942\n",
      "INFO\t2017-01-11 17:22:39\tGenotypeConcordance\tchecked     2,530,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:245,077,322\n",
      "INFO\t2017-01-11 17:22:39\tGenotypeConcordance\tchecked     2,540,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:245,800,205\n",
      "INFO\t2017-01-11 17:22:40\tGenotypeConcordance\tchecked     2,550,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:246,454,550\n",
      "INFO\t2017-01-11 17:22:40\tGenotypeConcordance\tchecked     2,560,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:247,179,859\n",
      "INFO\t2017-01-11 17:22:40\tGenotypeConcordance\tchecked     2,570,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:247,891,094\n",
      "INFO\t2017-01-11 17:22:41\tGenotypeConcordance\tchecked     2,580,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:248,544,206\n",
      "INFO\t2017-01-11 17:22:41\tGenotypeConcordance\tchecked     2,590,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:94,559\n",
      "INFO\t2017-01-11 17:22:41\tGenotypeConcordance\tchecked     2,600,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:837,910\n",
      "INFO\t2017-01-11 17:22:41\tGenotypeConcordance\tchecked     2,610,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:1,455,313\n",
      "INFO\t2017-01-11 17:22:42\tGenotypeConcordance\tchecked     2,620,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:2,113,647\n",
      "INFO\t2017-01-11 17:22:42\tGenotypeConcordance\tchecked     2,630,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:2,833,582\n",
      "INFO\t2017-01-11 17:22:42\tGenotypeConcordance\tchecked     2,640,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:3,609,598\n",
      "INFO\t2017-01-11 17:22:43\tGenotypeConcordance\tchecked     2,650,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:4,320,485\n",
      "INFO\t2017-01-11 17:22:43\tGenotypeConcordance\tchecked     2,660,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:5,168,050\n",
      "INFO\t2017-01-11 17:22:43\tGenotypeConcordance\tchecked     2,670,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:5,929,456\n",
      "INFO\t2017-01-11 17:22:43\tGenotypeConcordance\tchecked     2,680,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:6,651,520\n",
      "INFO\t2017-01-11 17:22:44\tGenotypeConcordance\tchecked     2,690,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:7,475,285\n",
      "INFO\t2017-01-11 17:22:44\tGenotypeConcordance\tchecked     2,700,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:8,297,461\n",
      "INFO\t2017-01-11 17:22:44\tGenotypeConcordance\tchecked     2,710,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:9,191,332\n",
      "INFO\t2017-01-11 17:22:44\tGenotypeConcordance\tchecked     2,720,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:9,995,071\n",
      "INFO\t2017-01-11 17:22:45\tGenotypeConcordance\tchecked     2,730,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:10,710,145\n",
      "INFO\t2017-01-11 17:22:45\tGenotypeConcordance\tchecked     2,740,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:11,442,442\n",
      "INFO\t2017-01-11 17:22:45\tGenotypeConcordance\tchecked     2,750,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:12,208,518\n",
      "INFO\t2017-01-11 17:22:46\tGenotypeConcordance\tchecked     2,760,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:13,082,247\n",
      "INFO\t2017-01-11 17:22:46\tGenotypeConcordance\tchecked     2,770,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:13,861,615\n",
      "INFO\t2017-01-11 17:22:46\tGenotypeConcordance\tchecked     2,780,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:14,651,618\n",
      "INFO\t2017-01-11 17:22:46\tGenotypeConcordance\tchecked     2,790,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:15,509,146\n",
      "INFO\t2017-01-11 17:22:47\tGenotypeConcordance\tchecked     2,800,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:16,274,500\n",
      "INFO\t2017-01-11 17:22:47\tGenotypeConcordance\tchecked     2,810,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:17,168,614\n",
      "INFO\t2017-01-11 17:22:47\tGenotypeConcordance\tchecked     2,820,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:18,102,927\n",
      "INFO\t2017-01-11 17:22:48\tGenotypeConcordance\tchecked     2,830,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:18,909,666\n",
      "INFO\t2017-01-11 17:22:48\tGenotypeConcordance\tchecked     2,840,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:19,850,364\n",
      "INFO\t2017-01-11 17:22:48\tGenotypeConcordance\tchecked     2,850,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:20,700,647\n",
      "INFO\t2017-01-11 17:22:48\tGenotypeConcordance\tchecked     2,860,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:21,512,069\n",
      "INFO\t2017-01-11 17:22:49\tGenotypeConcordance\tchecked     2,870,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:22,460,536\n",
      "INFO\t2017-01-11 17:22:49\tGenotypeConcordance\tchecked     2,880,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:23,331,614\n",
      "INFO\t2017-01-11 17:22:49\tGenotypeConcordance\tchecked     2,890,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:24,242,812\n",
      "INFO\t2017-01-11 17:22:50\tGenotypeConcordance\tchecked     2,900,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:25,284,933\n",
      "INFO\t2017-01-11 17:22:50\tGenotypeConcordance\tchecked     2,910,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:26,304,419\n",
      "INFO\t2017-01-11 17:22:50\tGenotypeConcordance\tchecked     2,920,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:27,213,937\n",
      "INFO\t2017-01-11 17:22:50\tGenotypeConcordance\tchecked     2,930,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:28,384,132\n",
      "INFO\t2017-01-11 17:22:51\tGenotypeConcordance\tchecked     2,940,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:29,271,357\n",
      "INFO\t2017-01-11 17:22:51\tGenotypeConcordance\tchecked     2,950,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:30,051,408\n",
      "INFO\t2017-01-11 17:22:51\tGenotypeConcordance\tchecked     2,960,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:30,862,245\n",
      "INFO\t2017-01-11 17:22:51\tGenotypeConcordance\tchecked     2,970,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:31,586,861\n",
      "INFO\t2017-01-11 17:22:52\tGenotypeConcordance\tchecked     2,980,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:32,555,031\n",
      "INFO\t2017-01-11 17:22:52\tGenotypeConcordance\tchecked     2,990,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:33,390,728\n",
      "INFO\t2017-01-11 17:22:52\tGenotypeConcordance\tchecked     3,000,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:34,029,312\n",
      "INFO\t2017-01-11 17:22:53\tGenotypeConcordance\tchecked     3,010,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:34,626,685\n",
      "INFO\t2017-01-11 17:22:53\tGenotypeConcordance\tchecked     3,020,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:35,300,605\n",
      "INFO\t2017-01-11 17:22:53\tGenotypeConcordance\tchecked     3,030,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:35,994,111\n",
      "INFO\t2017-01-11 17:22:53\tGenotypeConcordance\tchecked     3,040,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:36,699,168\n",
      "INFO\t2017-01-11 17:22:54\tGenotypeConcordance\tchecked     3,050,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:37,453,581\n",
      "INFO\t2017-01-11 17:22:54\tGenotypeConcordance\tchecked     3,060,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:38,024,047\n",
      "INFO\t2017-01-11 17:22:54\tGenotypeConcordance\tchecked     3,070,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:38,691,187\n",
      "INFO\t2017-01-11 17:22:54\tGenotypeConcordance\tchecked     3,080,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:39,523,829\n",
      "INFO\t2017-01-11 17:22:55\tGenotypeConcordance\tchecked     3,090,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:40,295,627\n",
      "INFO\t2017-01-11 17:22:55\tGenotypeConcordance\tchecked     3,100,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:41,048,757\n",
      "INFO\t2017-01-11 17:22:55\tGenotypeConcordance\tchecked     3,110,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:41,786,707\n",
      "INFO\t2017-01-11 17:22:56\tGenotypeConcordance\tchecked     3,120,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:42,459,288\n",
      "INFO\t2017-01-11 17:22:56\tGenotypeConcordance\tchecked     3,130,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:43,269,737\n",
      "INFO\t2017-01-11 17:22:56\tGenotypeConcordance\tchecked     3,140,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:44,027,827\n",
      "INFO\t2017-01-11 17:22:56\tGenotypeConcordance\tchecked     3,150,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:44,763,797\n",
      "INFO\t2017-01-11 17:22:57\tGenotypeConcordance\tchecked     3,160,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:45,546,836\n",
      "INFO\t2017-01-11 17:22:57\tGenotypeConcordance\tchecked     3,170,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:46,363,699\n",
      "INFO\t2017-01-11 17:22:57\tGenotypeConcordance\tchecked     3,180,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:47,118,531\n",
      "INFO\t2017-01-11 17:22:58\tGenotypeConcordance\tchecked     3,190,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:47,813,742\n",
      "INFO\t2017-01-11 17:22:58\tGenotypeConcordance\tchecked     3,200,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:48,663,410\n",
      "INFO\t2017-01-11 17:22:58\tGenotypeConcordance\tchecked     3,210,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:49,407,380\n",
      "INFO\t2017-01-11 17:22:58\tGenotypeConcordance\tchecked     3,220,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:50,178,059\n",
      "INFO\t2017-01-11 17:22:59\tGenotypeConcordance\tchecked     3,230,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:50,987,496\n",
      "INFO\t2017-01-11 17:22:59\tGenotypeConcordance\tchecked     3,240,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:51,778,185\n",
      "INFO\t2017-01-11 17:22:59\tGenotypeConcordance\tchecked     3,250,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:52,480,847\n",
      "INFO\t2017-01-11 17:22:59\tGenotypeConcordance\tchecked     3,260,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:53,100,764\n",
      "INFO\t2017-01-11 17:23:00\tGenotypeConcordance\tchecked     3,270,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:53,780,128\n",
      "INFO\t2017-01-11 17:23:00\tGenotypeConcordance\tchecked     3,280,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:54,605,148\n",
      "INFO\t2017-01-11 17:23:00\tGenotypeConcordance\tchecked     3,290,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:55,413,056\n",
      "INFO\t2017-01-11 17:23:00\tGenotypeConcordance\tchecked     3,300,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:56,271,006\n",
      "INFO\t2017-01-11 17:23:01\tGenotypeConcordance\tchecked     3,310,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:56,996,573\n",
      "INFO\t2017-01-11 17:23:01\tGenotypeConcordance\tchecked     3,320,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:57,677,079\n",
      "INFO\t2017-01-11 17:23:01\tGenotypeConcordance\tchecked     3,330,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:58,541,238\n",
      "INFO\t2017-01-11 17:23:02\tGenotypeConcordance\tchecked     3,340,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:59,526,280\n",
      "INFO\t2017-01-11 17:23:02\tGenotypeConcordance\tchecked     3,350,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:60,364,993\n",
      "INFO\t2017-01-11 17:23:02\tGenotypeConcordance\tchecked     3,360,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:61,323,655\n",
      "INFO\t2017-01-11 17:23:02\tGenotypeConcordance\tchecked     3,370,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:62,260,132\n",
      "INFO\t2017-01-11 17:23:03\tGenotypeConcordance\tchecked     3,380,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:63,330,513\n",
      "INFO\t2017-01-11 17:23:03\tGenotypeConcordance\tchecked     3,390,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:64,467,184\n",
      "INFO\t2017-01-11 17:23:03\tGenotypeConcordance\tchecked     3,400,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:65,346,733\n",
      "INFO\t2017-01-11 17:23:03\tGenotypeConcordance\tchecked     3,410,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:66,185,353\n",
      "INFO\t2017-01-11 17:23:04\tGenotypeConcordance\tchecked     3,420,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:67,072,585\n",
      "INFO\t2017-01-11 17:23:04\tGenotypeConcordance\tchecked     3,430,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:67,925,384\n",
      "INFO\t2017-01-11 17:23:04\tGenotypeConcordance\tchecked     3,440,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:68,716,263\n",
      "INFO\t2017-01-11 17:23:05\tGenotypeConcordance\tchecked     3,450,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:69,552,521\n",
      "INFO\t2017-01-11 17:23:05\tGenotypeConcordance\tchecked     3,460,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:70,460,722\n",
      "INFO\t2017-01-11 17:23:05\tGenotypeConcordance\tchecked     3,470,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:71,249,509\n",
      "INFO\t2017-01-11 17:23:05\tGenotypeConcordance\tchecked     3,480,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:72,040,506\n",
      "INFO\t2017-01-11 17:23:06\tGenotypeConcordance\tchecked     3,490,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:73,127,224\n",
      "INFO\t2017-01-11 17:23:06\tGenotypeConcordance\tchecked     3,500,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:74,131,066\n",
      "INFO\t2017-01-11 17:23:06\tGenotypeConcordance\tchecked     3,510,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:75,111,166\n",
      "INFO\t2017-01-11 17:23:06\tGenotypeConcordance\tchecked     3,520,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:75,909,079\n",
      "INFO\t2017-01-11 17:23:07\tGenotypeConcordance\tchecked     3,530,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:76,669,171\n",
      "INFO\t2017-01-11 17:23:07\tGenotypeConcordance\tchecked     3,540,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:77,310,158\n",
      "INFO\t2017-01-11 17:23:07\tGenotypeConcordance\tchecked     3,550,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:78,049,271\n",
      "INFO\t2017-01-11 17:23:08\tGenotypeConcordance\tchecked     3,560,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:78,848,211\n",
      "INFO\t2017-01-11 17:23:08\tGenotypeConcordance\tchecked     3,570,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:79,617,030\n",
      "INFO\t2017-01-11 17:23:08\tGenotypeConcordance\tchecked     3,580,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:80,391,778\n",
      "INFO\t2017-01-11 17:23:09\tGenotypeConcordance\tchecked     3,590,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:81,215,144\n",
      "INFO\t2017-01-11 17:23:09\tGenotypeConcordance\tchecked     3,600,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:82,049,582\n",
      "INFO\t2017-01-11 17:23:09\tGenotypeConcordance\tchecked     3,610,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:82,955,443\n",
      "INFO\t2017-01-11 17:23:09\tGenotypeConcordance\tchecked     3,620,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:83,808,468\n",
      "INFO\t2017-01-11 17:23:10\tGenotypeConcordance\tchecked     3,630,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:84,673,058\n",
      "INFO\t2017-01-11 17:23:10\tGenotypeConcordance\tchecked     3,640,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:85,615,083\n",
      "INFO\t2017-01-11 17:23:10\tGenotypeConcordance\tchecked     3,650,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:86,501,141\n",
      "INFO\t2017-01-11 17:23:11\tGenotypeConcordance\tchecked     3,660,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:87,574,999\n",
      "INFO\t2017-01-11 17:23:11\tGenotypeConcordance\tchecked     3,670,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:88,668,496\n",
      "INFO\t2017-01-11 17:23:11\tGenotypeConcordance\tchecked     3,680,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:89,449,714\n",
      "INFO\t2017-01-11 17:23:11\tGenotypeConcordance\tchecked     3,690,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:91,634,435\n",
      "INFO\t2017-01-11 17:23:12\tGenotypeConcordance\tchecked     3,700,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:92,132,836\n",
      "INFO\t2017-01-11 17:23:12\tGenotypeConcordance\tchecked     3,710,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:95,489,168\n",
      "INFO\t2017-01-11 17:23:12\tGenotypeConcordance\tchecked     3,720,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:96,762,284\n",
      "INFO\t2017-01-11 17:23:13\tGenotypeConcordance\tchecked     3,730,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:97,863,112\n",
      "INFO\t2017-01-11 17:23:13\tGenotypeConcordance\tchecked     3,740,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:99,055,792\n",
      "INFO\t2017-01-11 17:23:13\tGenotypeConcordance\tchecked     3,750,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:100,134,730\n",
      "INFO\t2017-01-11 17:23:13\tGenotypeConcordance\tchecked     3,760,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:101,070,493\n",
      "INFO\t2017-01-11 17:23:14\tGenotypeConcordance\tchecked     3,770,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:101,865,270\n",
      "INFO\t2017-01-11 17:23:14\tGenotypeConcordance\tchecked     3,780,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:102,727,198\n",
      "INFO\t2017-01-11 17:23:14\tGenotypeConcordance\tchecked     3,790,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:103,625,469\n",
      "INFO\t2017-01-11 17:23:14\tGenotypeConcordance\tchecked     3,800,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:104,550,916\n",
      "INFO\t2017-01-11 17:23:15\tGenotypeConcordance\tchecked     3,810,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:105,447,029\n",
      "INFO\t2017-01-11 17:23:15\tGenotypeConcordance\tchecked     3,820,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:106,265,647\n",
      "INFO\t2017-01-11 17:23:15\tGenotypeConcordance\tchecked     3,830,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:107,073,736\n",
      "INFO\t2017-01-11 17:23:16\tGenotypeConcordance\tchecked     3,840,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:107,908,532\n",
      "INFO\t2017-01-11 17:23:16\tGenotypeConcordance\tchecked     3,850,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:108,821,619\n",
      "INFO\t2017-01-11 17:23:16\tGenotypeConcordance\tchecked     3,860,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:109,685,534\n",
      "INFO\t2017-01-11 17:23:16\tGenotypeConcordance\tchecked     3,870,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:111,052,113\n",
      "INFO\t2017-01-11 17:23:17\tGenotypeConcordance\tchecked     3,880,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:112,344,713\n",
      "INFO\t2017-01-11 17:23:17\tGenotypeConcordance\tchecked     3,890,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:113,309,969\n",
      "INFO\t2017-01-11 17:23:17\tGenotypeConcordance\tchecked     3,900,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:114,178,591\n",
      "INFO\t2017-01-11 17:23:18\tGenotypeConcordance\tchecked     3,910,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:115,045,896\n",
      "INFO\t2017-01-11 17:23:18\tGenotypeConcordance\tchecked     3,920,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:115,933,697\n",
      "INFO\t2017-01-11 17:23:18\tGenotypeConcordance\tchecked     3,930,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:116,831,339\n",
      "INFO\t2017-01-11 17:23:18\tGenotypeConcordance\tchecked     3,940,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:117,629,795\n",
      "INFO\t2017-01-11 17:23:19\tGenotypeConcordance\tchecked     3,950,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:118,364,176\n",
      "INFO\t2017-01-11 17:23:19\tGenotypeConcordance\tchecked     3,960,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:119,197,552\n",
      "INFO\t2017-01-11 17:23:19\tGenotypeConcordance\tchecked     3,970,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:120,018,077\n",
      "INFO\t2017-01-11 17:23:20\tGenotypeConcordance\tchecked     3,980,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:121,022,354\n",
      "INFO\t2017-01-11 17:23:20\tGenotypeConcordance\tchecked     3,990,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:121,816,314\n",
      "INFO\t2017-01-11 17:23:20\tGenotypeConcordance\tchecked     4,000,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:122,669,673\n",
      "INFO\t2017-01-11 17:23:20\tGenotypeConcordance\tchecked     4,010,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:123,573,226\n",
      "INFO\t2017-01-11 17:23:21\tGenotypeConcordance\tchecked     4,020,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:124,315,714\n",
      "INFO\t2017-01-11 17:23:21\tGenotypeConcordance\tchecked     4,030,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:125,130,263\n",
      "INFO\t2017-01-11 17:23:21\tGenotypeConcordance\tchecked     4,040,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:125,965,376\n",
      "INFO\t2017-01-11 17:23:21\tGenotypeConcordance\tchecked     4,050,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:126,776,164\n",
      "INFO\t2017-01-11 17:23:22\tGenotypeConcordance\tchecked     4,060,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:127,580,979\n",
      "INFO\t2017-01-11 17:23:22\tGenotypeConcordance\tchecked     4,070,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:128,388,902\n",
      "INFO\t2017-01-11 17:23:22\tGenotypeConcordance\tchecked     4,080,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:129,225,828\n",
      "INFO\t2017-01-11 17:23:23\tGenotypeConcordance\tchecked     4,090,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:130,003,278\n",
      "INFO\t2017-01-11 17:23:23\tGenotypeConcordance\tchecked     4,100,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:130,795,561\n",
      "INFO\t2017-01-11 17:23:23\tGenotypeConcordance\tchecked     4,110,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:131,710,030\n",
      "INFO\t2017-01-11 17:23:23\tGenotypeConcordance\tchecked     4,120,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:132,424,600\n",
      "INFO\t2017-01-11 17:23:24\tGenotypeConcordance\tchecked     4,130,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:132,978,093\n",
      "INFO\t2017-01-11 17:23:24\tGenotypeConcordance\tchecked     4,140,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:133,345,134\n",
      "INFO\t2017-01-11 17:23:24\tGenotypeConcordance\tchecked     4,150,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:134,247,340\n",
      "INFO\t2017-01-11 17:23:24\tGenotypeConcordance\tchecked     4,160,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:135,059,317\n",
      "INFO\t2017-01-11 17:23:25\tGenotypeConcordance\tchecked     4,170,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:136,005,966\n",
      "INFO\t2017-01-11 17:23:25\tGenotypeConcordance\tchecked     4,180,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:137,028,225\n",
      "INFO\t2017-01-11 17:23:25\tGenotypeConcordance\tchecked     4,190,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:137,826,312\n",
      "INFO\t2017-01-11 17:23:26\tGenotypeConcordance\tchecked     4,200,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:138,609,631\n",
      "INFO\t2017-01-11 17:23:26\tGenotypeConcordance\tchecked     4,210,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:139,499,493\n",
      "INFO\t2017-01-11 17:23:26\tGenotypeConcordance\tchecked     4,220,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:140,328,107\n",
      "INFO\t2017-01-11 17:23:26\tGenotypeConcordance\tchecked     4,230,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:141,130,695\n",
      "INFO\t2017-01-11 17:23:27\tGenotypeConcordance\tchecked     4,240,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:141,872,447\n",
      "INFO\t2017-01-11 17:23:27\tGenotypeConcordance\tchecked     4,250,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:142,585,797\n",
      "INFO\t2017-01-11 17:23:27\tGenotypeConcordance\tchecked     4,260,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:143,476,813\n",
      "INFO\t2017-01-11 17:23:28\tGenotypeConcordance\tchecked     4,270,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:144,483,428\n",
      "INFO\t2017-01-11 17:23:28\tGenotypeConcordance\tchecked     4,280,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:145,702,623\n",
      "INFO\t2017-01-11 17:23:28\tGenotypeConcordance\tchecked     4,290,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:146,688,483\n",
      "INFO\t2017-01-11 17:23:28\tGenotypeConcordance\tchecked     4,300,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:147,757,946\n",
      "INFO\t2017-01-11 17:23:29\tGenotypeConcordance\tchecked     4,310,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:148,865,167\n",
      "INFO\t2017-01-11 17:23:29\tGenotypeConcordance\tchecked     4,320,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:149,967,939\n",
      "INFO\t2017-01-11 17:23:29\tGenotypeConcordance\tchecked     4,330,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:150,885,168\n",
      "INFO\t2017-01-11 17:23:30\tGenotypeConcordance\tchecked     4,340,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:151,825,296\n",
      "INFO\t2017-01-11 17:23:30\tGenotypeConcordance\tchecked     4,350,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:152,751,141\n",
      "INFO\t2017-01-11 17:23:30\tGenotypeConcordance\tchecked     4,360,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:153,695,805\n",
      "INFO\t2017-01-11 17:23:30\tGenotypeConcordance\tchecked     4,370,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:154,529,360\n",
      "INFO\t2017-01-11 17:23:31\tGenotypeConcordance\tchecked     4,380,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:155,443,439\n",
      "INFO\t2017-01-11 17:23:31\tGenotypeConcordance\tchecked     4,390,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:156,394,351\n",
      "INFO\t2017-01-11 17:23:31\tGenotypeConcordance\tchecked     4,400,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:157,529,424\n",
      "INFO\t2017-01-11 17:23:31\tGenotypeConcordance\tchecked     4,410,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:158,604,128\n",
      "INFO\t2017-01-11 17:23:32\tGenotypeConcordance\tchecked     4,420,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:159,476,445\n",
      "INFO\t2017-01-11 17:23:32\tGenotypeConcordance\tchecked     4,430,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:160,325,118\n",
      "INFO\t2017-01-11 17:23:32\tGenotypeConcordance\tchecked     4,440,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:161,357,258\n",
      "INFO\t2017-01-11 17:23:33\tGenotypeConcordance\tchecked     4,450,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:162,321,436\n",
      "INFO\t2017-01-11 17:23:33\tGenotypeConcordance\tchecked     4,460,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:163,400,015\n",
      "INFO\t2017-01-11 17:23:33\tGenotypeConcordance\tchecked     4,470,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:164,328,393\n",
      "INFO\t2017-01-11 17:23:33\tGenotypeConcordance\tchecked     4,480,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:165,295,123\n",
      "INFO\t2017-01-11 17:23:34\tGenotypeConcordance\tchecked     4,490,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:166,285,342\n",
      "INFO\t2017-01-11 17:23:34\tGenotypeConcordance\tchecked     4,500,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:167,191,274\n",
      "INFO\t2017-01-11 17:23:34\tGenotypeConcordance\tchecked     4,510,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:167,990,754\n",
      "INFO\t2017-01-11 17:23:34\tGenotypeConcordance\tchecked     4,520,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:168,825,793\n",
      "INFO\t2017-01-11 17:23:35\tGenotypeConcordance\tchecked     4,530,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:169,689,042\n",
      "INFO\t2017-01-11 17:23:35\tGenotypeConcordance\tchecked     4,540,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:170,536,190\n",
      "INFO\t2017-01-11 17:23:35\tGenotypeConcordance\tchecked     4,550,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:171,334,858\n",
      "INFO\t2017-01-11 17:23:36\tGenotypeConcordance\tchecked     4,560,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:172,242,147\n",
      "INFO\t2017-01-11 17:23:36\tGenotypeConcordance\tchecked     4,570,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:173,138,239\n",
      "INFO\t2017-01-11 17:23:36\tGenotypeConcordance\tchecked     4,580,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:173,969,044\n",
      "INFO\t2017-01-11 17:23:37\tGenotypeConcordance\tchecked     4,590,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:174,823,687\n",
      "INFO\t2017-01-11 17:23:37\tGenotypeConcordance\tchecked     4,600,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:175,815,803\n",
      "INFO\t2017-01-11 17:23:38\tGenotypeConcordance\tchecked     4,610,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:176,740,621\n",
      "INFO\t2017-01-11 17:23:38\tGenotypeConcordance\tchecked     4,620,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:177,642,869\n",
      "INFO\t2017-01-11 17:23:38\tGenotypeConcordance\tchecked     4,630,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:178,545,934\n",
      "INFO\t2017-01-11 17:23:39\tGenotypeConcordance\tchecked     4,640,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:179,541,612\n",
      "INFO\t2017-01-11 17:23:39\tGenotypeConcordance\tchecked     4,650,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:180,421,557\n",
      "INFO\t2017-01-11 17:23:39\tGenotypeConcordance\tchecked     4,660,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:181,281,745\n",
      "INFO\t2017-01-11 17:23:40\tGenotypeConcordance\tchecked     4,670,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:182,215,203\n",
      "INFO\t2017-01-11 17:23:40\tGenotypeConcordance\tchecked     4,680,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:183,182,776\n",
      "INFO\t2017-01-11 17:23:40\tGenotypeConcordance\tchecked     4,690,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:184,083,649\n",
      "INFO\t2017-01-11 17:23:41\tGenotypeConcordance\tchecked     4,700,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:184,938,143\n",
      "INFO\t2017-01-11 17:23:41\tGenotypeConcordance\tchecked     4,710,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:185,881,295\n",
      "INFO\t2017-01-11 17:23:41\tGenotypeConcordance\tchecked     4,720,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:186,928,089\n",
      "INFO\t2017-01-11 17:23:41\tGenotypeConcordance\tchecked     4,730,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:187,898,064\n",
      "INFO\t2017-01-11 17:23:42\tGenotypeConcordance\tchecked     4,740,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:188,787,991\n",
      "INFO\t2017-01-11 17:23:42\tGenotypeConcordance\tchecked     4,750,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:189,778,384\n",
      "INFO\t2017-01-11 17:23:42\tGenotypeConcordance\tchecked     4,760,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:190,783,398\n",
      "INFO\t2017-01-11 17:23:43\tGenotypeConcordance\tchecked     4,770,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:191,766,693\n",
      "INFO\t2017-01-11 17:23:43\tGenotypeConcordance\tchecked     4,780,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:192,696,437\n",
      "INFO\t2017-01-11 17:23:43\tGenotypeConcordance\tchecked     4,790,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:193,648,710\n",
      "INFO\t2017-01-11 17:23:43\tGenotypeConcordance\tchecked     4,800,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:194,694,625\n",
      "INFO\t2017-01-11 17:23:44\tGenotypeConcordance\tchecked     4,810,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:195,615,120\n",
      "INFO\t2017-01-11 17:23:44\tGenotypeConcordance\tchecked     4,820,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:196,443,136\n",
      "INFO\t2017-01-11 17:23:44\tGenotypeConcordance\tchecked     4,830,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:197,361,687\n",
      "INFO\t2017-01-11 17:23:44\tGenotypeConcordance\tchecked     4,840,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:198,398,651\n",
      "INFO\t2017-01-11 17:23:45\tGenotypeConcordance\tchecked     4,850,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:199,395,615\n",
      "INFO\t2017-01-11 17:23:45\tGenotypeConcordance\tchecked     4,860,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:200,486,639\n",
      "INFO\t2017-01-11 17:23:45\tGenotypeConcordance\tchecked     4,870,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:201,601,564\n",
      "INFO\t2017-01-11 17:23:46\tGenotypeConcordance\tchecked     4,880,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:202,646,115\n",
      "INFO\t2017-01-11 17:23:46\tGenotypeConcordance\tchecked     4,890,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:203,697,319\n",
      "INFO\t2017-01-11 17:23:46\tGenotypeConcordance\tchecked     4,900,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:204,828,870\n",
      "INFO\t2017-01-11 17:23:46\tGenotypeConcordance\tchecked     4,910,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:205,748,607\n",
      "INFO\t2017-01-11 17:23:47\tGenotypeConcordance\tchecked     4,920,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:206,669,887\n",
      "INFO\t2017-01-11 17:23:47\tGenotypeConcordance\tchecked     4,930,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:207,607,797\n",
      "INFO\t2017-01-11 17:23:47\tGenotypeConcordance\tchecked     4,940,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:208,565,664\n",
      "INFO\t2017-01-11 17:23:47\tGenotypeConcordance\tchecked     4,950,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:209,454,495\n",
      "INFO\t2017-01-11 17:23:48\tGenotypeConcordance\tchecked     4,960,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:210,203,064\n",
      "INFO\t2017-01-11 17:23:48\tGenotypeConcordance\tchecked     4,970,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:211,270,752\n",
      "INFO\t2017-01-11 17:23:48\tGenotypeConcordance\tchecked     4,980,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:212,081,721\n",
      "INFO\t2017-01-11 17:23:49\tGenotypeConcordance\tchecked     4,990,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:212,838,216\n",
      "INFO\t2017-01-11 17:23:49\tGenotypeConcordance\tchecked     5,000,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:213,680,033\n",
      "INFO\t2017-01-11 17:23:49\tGenotypeConcordance\tchecked     5,010,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:214,658,378\n",
      "INFO\t2017-01-11 17:23:49\tGenotypeConcordance\tchecked     5,020,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:215,532,581\n",
      "INFO\t2017-01-11 17:23:50\tGenotypeConcordance\tchecked     5,030,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:216,350,538\n",
      "INFO\t2017-01-11 17:23:50\tGenotypeConcordance\tchecked     5,040,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:217,243,369\n",
      "INFO\t2017-01-11 17:23:50\tGenotypeConcordance\tchecked     5,050,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:218,084,158\n",
      "INFO\t2017-01-11 17:23:51\tGenotypeConcordance\tchecked     5,060,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:218,924,659\n",
      "INFO\t2017-01-11 17:23:51\tGenotypeConcordance\tchecked     5,070,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:219,941,063\n",
      "INFO\t2017-01-11 17:23:51\tGenotypeConcordance\tchecked     5,080,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:220,840,025\n",
      "INFO\t2017-01-11 17:23:51\tGenotypeConcordance\tchecked     5,090,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:221,681,143\n",
      "INFO\t2017-01-11 17:23:52\tGenotypeConcordance\tchecked     5,100,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:222,534,932\n",
      "INFO\t2017-01-11 17:23:52\tGenotypeConcordance\tchecked     5,110,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:223,449,802\n",
      "INFO\t2017-01-11 17:23:52\tGenotypeConcordance\tchecked     5,120,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:224,278,085\n",
      "INFO\t2017-01-11 17:23:53\tGenotypeConcordance\tchecked     5,130,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:225,150,894\n",
      "INFO\t2017-01-11 17:23:53\tGenotypeConcordance\tchecked     5,140,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:226,017,466\n",
      "INFO\t2017-01-11 17:23:53\tGenotypeConcordance\tchecked     5,150,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:226,935,636\n",
      "INFO\t2017-01-11 17:23:53\tGenotypeConcordance\tchecked     5,160,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:227,877,295\n",
      "INFO\t2017-01-11 17:23:54\tGenotypeConcordance\tchecked     5,170,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:228,617,113\n",
      "INFO\t2017-01-11 17:23:54\tGenotypeConcordance\tchecked     5,180,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:229,390,956\n",
      "INFO\t2017-01-11 17:23:54\tGenotypeConcordance\tchecked     5,190,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:230,252,758\n",
      "INFO\t2017-01-11 17:23:55\tGenotypeConcordance\tchecked     5,200,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:231,125,026\n",
      "INFO\t2017-01-11 17:23:55\tGenotypeConcordance\tchecked     5,210,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:231,888,425\n",
      "INFO\t2017-01-11 17:23:55\tGenotypeConcordance\tchecked     5,220,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:232,711,377\n",
      "INFO\t2017-01-11 17:23:55\tGenotypeConcordance\tchecked     5,230,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:233,577,038\n",
      "INFO\t2017-01-11 17:23:56\tGenotypeConcordance\tchecked     5,240,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:234,410,059\n",
      "INFO\t2017-01-11 17:23:56\tGenotypeConcordance\tchecked     5,250,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:235,214,315\n",
      "INFO\t2017-01-11 17:23:56\tGenotypeConcordance\tchecked     5,260,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:235,916,669\n",
      "INFO\t2017-01-11 17:23:56\tGenotypeConcordance\tchecked     5,270,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:236,628,437\n",
      "INFO\t2017-01-11 17:23:57\tGenotypeConcordance\tchecked     5,280,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:237,398,313\n",
      "INFO\t2017-01-11 17:23:57\tGenotypeConcordance\tchecked     5,290,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:238,096,357\n",
      "INFO\t2017-01-11 17:23:57\tGenotypeConcordance\tchecked     5,300,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:238,800,921\n",
      "INFO\t2017-01-11 17:23:57\tGenotypeConcordance\tchecked     5,310,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:239,612,662\n",
      "INFO\t2017-01-11 17:23:58\tGenotypeConcordance\tchecked     5,320,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:240,365,866\n",
      "INFO\t2017-01-11 17:23:58\tGenotypeConcordance\tchecked     5,330,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:241,064,471\n",
      "INFO\t2017-01-11 17:23:58\tGenotypeConcordance\tchecked     5,340,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:241,753,985\n",
      "INFO\t2017-01-11 17:23:59\tGenotypeConcordance\tchecked     5,350,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:242,538,598\n",
      "INFO\t2017-01-11 17:23:59\tGenotypeConcordance\tchecked     5,360,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:142,304\n",
      "INFO\t2017-01-11 17:23:59\tGenotypeConcordance\tchecked     5,370,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:753,272\n",
      "INFO\t2017-01-11 17:23:59\tGenotypeConcordance\tchecked     5,380,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:1,398,990\n",
      "INFO\t2017-01-11 17:24:00\tGenotypeConcordance\tchecked     5,390,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:1,921,909\n",
      "INFO\t2017-01-11 17:24:00\tGenotypeConcordance\tchecked     5,400,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:2,542,326\n",
      "INFO\t2017-01-11 17:24:00\tGenotypeConcordance\tchecked     5,410,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:3,164,512\n",
      "INFO\t2017-01-11 17:24:01\tGenotypeConcordance\tchecked     5,420,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:3,786,649\n",
      "INFO\t2017-01-11 17:24:01\tGenotypeConcordance\tchecked     5,430,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:4,431,070\n",
      "INFO\t2017-01-11 17:24:01\tGenotypeConcordance\tchecked     5,440,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:5,157,196\n",
      "INFO\t2017-01-11 17:24:02\tGenotypeConcordance\tchecked     5,450,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:5,767,097\n",
      "INFO\t2017-01-11 17:24:02\tGenotypeConcordance\tchecked     5,460,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:6,374,405\n",
      "INFO\t2017-01-11 17:24:02\tGenotypeConcordance\tchecked     5,470,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:7,054,619\n",
      "INFO\t2017-01-11 17:24:02\tGenotypeConcordance\tchecked     5,480,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:7,744,875\n",
      "INFO\t2017-01-11 17:24:03\tGenotypeConcordance\tchecked     5,490,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:8,432,535\n",
      "INFO\t2017-01-11 17:24:03\tGenotypeConcordance\tchecked     5,500,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:9,164,409\n",
      "INFO\t2017-01-11 17:24:03\tGenotypeConcordance\tchecked     5,510,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:10,058,314\n",
      "INFO\t2017-01-11 17:24:04\tGenotypeConcordance\tchecked     5,520,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:10,833,472\n",
      "INFO\t2017-01-11 17:24:04\tGenotypeConcordance\tchecked     5,530,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:11,671,483\n",
      "INFO\t2017-01-11 17:24:04\tGenotypeConcordance\tchecked     5,540,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:12,522,664\n",
      "INFO\t2017-01-11 17:24:04\tGenotypeConcordance\tchecked     5,550,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:13,286,504\n",
      "INFO\t2017-01-11 17:24:05\tGenotypeConcordance\tchecked     5,560,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:14,043,326\n",
      "INFO\t2017-01-11 17:24:05\tGenotypeConcordance\tchecked     5,570,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:14,841,664\n",
      "INFO\t2017-01-11 17:24:05\tGenotypeConcordance\tchecked     5,580,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:15,660,580\n",
      "INFO\t2017-01-11 17:24:06\tGenotypeConcordance\tchecked     5,590,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:16,498,472\n",
      "INFO\t2017-01-11 17:24:06\tGenotypeConcordance\tchecked     5,600,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:17,382,833\n",
      "INFO\t2017-01-11 17:24:06\tGenotypeConcordance\tchecked     5,610,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:18,430,301\n",
      "INFO\t2017-01-11 17:24:06\tGenotypeConcordance\tchecked     5,620,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:19,332,760\n",
      "INFO\t2017-01-11 17:24:07\tGenotypeConcordance\tchecked     5,630,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:20,227,639\n",
      "INFO\t2017-01-11 17:24:07\tGenotypeConcordance\tchecked     5,640,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:21,044,936\n",
      "INFO\t2017-01-11 17:24:07\tGenotypeConcordance\tchecked     5,650,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:21,791,930\n",
      "INFO\t2017-01-11 17:24:07\tGenotypeConcordance\tchecked     5,660,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:22,325,974\n",
      "INFO\t2017-01-11 17:24:08\tGenotypeConcordance\tchecked     5,670,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:23,034,968\n",
      "INFO\t2017-01-11 17:24:08\tGenotypeConcordance\tchecked     5,680,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:23,834,246\n",
      "INFO\t2017-01-11 17:24:08\tGenotypeConcordance\tchecked     5,690,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:24,648,750\n",
      "INFO\t2017-01-11 17:24:09\tGenotypeConcordance\tchecked     5,700,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:25,317,279\n",
      "INFO\t2017-01-11 17:24:09\tGenotypeConcordance\tchecked     5,710,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:26,164,092\n",
      "INFO\t2017-01-11 17:24:09\tGenotypeConcordance\tchecked     5,720,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:26,927,337\n",
      "INFO\t2017-01-11 17:24:09\tGenotypeConcordance\tchecked     5,730,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:27,753,599\n",
      "INFO\t2017-01-11 17:24:10\tGenotypeConcordance\tchecked     5,740,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:28,662,924\n",
      "INFO\t2017-01-11 17:24:10\tGenotypeConcordance\tchecked     5,750,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:29,472,792\n",
      "INFO\t2017-01-11 17:24:10\tGenotypeConcordance\tchecked     5,760,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:30,244,931\n",
      "INFO\t2017-01-11 17:24:10\tGenotypeConcordance\tchecked     5,770,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:31,034,548\n",
      "INFO\t2017-01-11 17:24:11\tGenotypeConcordance\tchecked     5,780,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:31,892,448\n",
      "INFO\t2017-01-11 17:24:11\tGenotypeConcordance\tchecked     5,790,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:32,663,921\n",
      "INFO\t2017-01-11 17:24:11\tGenotypeConcordance\tchecked     5,800,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:33,525,814\n",
      "INFO\t2017-01-11 17:24:12\tGenotypeConcordance\tchecked     5,810,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:34,520,755\n",
      "INFO\t2017-01-11 17:24:12\tGenotypeConcordance\tchecked     5,820,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:35,395,255\n",
      "INFO\t2017-01-11 17:24:12\tGenotypeConcordance\tchecked     5,830,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:36,280,768\n",
      "INFO\t2017-01-11 17:24:12\tGenotypeConcordance\tchecked     5,840,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:37,206,578\n",
      "INFO\t2017-01-11 17:24:13\tGenotypeConcordance\tchecked     5,850,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:38,189,828\n",
      "INFO\t2017-01-11 17:24:13\tGenotypeConcordance\tchecked     5,860,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:39,147,202\n",
      "INFO\t2017-01-11 17:24:13\tGenotypeConcordance\tchecked     5,870,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:40,005,584\n",
      "INFO\t2017-01-11 17:24:14\tGenotypeConcordance\tchecked     5,880,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:40,977,031\n",
      "INFO\t2017-01-11 17:24:14\tGenotypeConcordance\tchecked     5,890,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:41,817,711\n",
      "INFO\t2017-01-11 17:24:14\tGenotypeConcordance\tchecked     5,900,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:42,586,839\n",
      "INFO\t2017-01-11 17:24:14\tGenotypeConcordance\tchecked     5,910,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:43,567,237\n",
      "INFO\t2017-01-11 17:24:15\tGenotypeConcordance\tchecked     5,920,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:44,592,561\n",
      "INFO\t2017-01-11 17:24:15\tGenotypeConcordance\tchecked     5,930,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:45,523,962\n",
      "INFO\t2017-01-11 17:24:15\tGenotypeConcordance\tchecked     5,940,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:46,472,977\n",
      "INFO\t2017-01-11 17:24:16\tGenotypeConcordance\tchecked     5,950,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:47,594,105\n",
      "INFO\t2017-01-11 17:24:16\tGenotypeConcordance\tchecked     5,960,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:48,890,681\n",
      "INFO\t2017-01-11 17:24:16\tGenotypeConcordance\tchecked     5,970,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:50,240,933\n",
      "INFO\t2017-01-11 17:24:16\tGenotypeConcordance\tchecked     5,980,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:51,543,025\n",
      "INFO\t2017-01-11 17:24:17\tGenotypeConcordance\tchecked     5,990,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:52,807,018\n",
      "INFO\t2017-01-11 17:24:17\tGenotypeConcordance\tchecked     6,000,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:53,745,766\n",
      "INFO\t2017-01-11 17:24:17\tGenotypeConcordance\tchecked     6,010,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:54,552,208\n",
      "INFO\t2017-01-11 17:24:17\tGenotypeConcordance\tchecked     6,020,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:55,346,193\n",
      "INFO\t2017-01-11 17:24:18\tGenotypeConcordance\tchecked     6,030,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:56,245,895\n",
      "INFO\t2017-01-11 17:24:18\tGenotypeConcordance\tchecked     6,040,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:57,068,327\n",
      "INFO\t2017-01-11 17:24:18\tGenotypeConcordance\tchecked     6,050,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:58,022,175\n",
      "INFO\t2017-01-11 17:24:19\tGenotypeConcordance\tchecked     6,060,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:58,915,946\n",
      "INFO\t2017-01-11 17:24:19\tGenotypeConcordance\tchecked     6,070,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:59,744,362\n",
      "INFO\t2017-01-11 17:24:19\tGenotypeConcordance\tchecked     6,080,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:60,367,056\n",
      "INFO\t2017-01-11 17:24:19\tGenotypeConcordance\tchecked     6,090,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:61,074,082\n",
      "INFO\t2017-01-11 17:24:20\tGenotypeConcordance\tchecked     6,100,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:61,866,807\n",
      "INFO\t2017-01-11 17:24:20\tGenotypeConcordance\tchecked     6,110,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:62,711,417\n",
      "INFO\t2017-01-11 17:24:20\tGenotypeConcordance\tchecked     6,120,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:63,449,344\n",
      "INFO\t2017-01-11 17:24:21\tGenotypeConcordance\tchecked     6,130,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:64,345,206\n",
      "INFO\t2017-01-11 17:24:21\tGenotypeConcordance\tchecked     6,140,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:65,099,686\n",
      "INFO\t2017-01-11 17:24:21\tGenotypeConcordance\tchecked     6,150,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:65,870,455\n",
      "INFO\t2017-01-11 17:24:21\tGenotypeConcordance\tchecked     6,160,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:66,760,847\n",
      "INFO\t2017-01-11 17:24:22\tGenotypeConcordance\tchecked     6,170,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:67,476,577\n",
      "INFO\t2017-01-11 17:24:22\tGenotypeConcordance\tchecked     6,180,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:68,340,541\n",
      "INFO\t2017-01-11 17:24:22\tGenotypeConcordance\tchecked     6,190,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:69,163,405\n",
      "INFO\t2017-01-11 17:24:22\tGenotypeConcordance\tchecked     6,200,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:69,956,607\n",
      "INFO\t2017-01-11 17:24:23\tGenotypeConcordance\tchecked     6,210,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:70,920,041\n",
      "INFO\t2017-01-11 17:24:23\tGenotypeConcordance\tchecked     6,220,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:71,898,456\n",
      "INFO\t2017-01-11 17:24:23\tGenotypeConcordance\tchecked     6,230,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:72,736,441\n",
      "INFO\t2017-01-11 17:24:24\tGenotypeConcordance\tchecked     6,240,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:73,520,153\n",
      "INFO\t2017-01-11 17:24:24\tGenotypeConcordance\tchecked     6,250,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:74,323,700\n",
      "INFO\t2017-01-11 17:24:24\tGenotypeConcordance\tchecked     6,260,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:75,158,571\n",
      "INFO\t2017-01-11 17:24:24\tGenotypeConcordance\tchecked     6,270,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:75,694,903\n",
      "INFO\t2017-01-11 17:24:25\tGenotypeConcordance\tchecked     6,280,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:76,169,067\n",
      "INFO\t2017-01-11 17:24:25\tGenotypeConcordance\tchecked     6,290,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:76,998,027\n",
      "INFO\t2017-01-11 17:24:25\tGenotypeConcordance\tchecked     6,300,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:77,832,564\n",
      "INFO\t2017-01-11 17:24:26\tGenotypeConcordance\tchecked     6,310,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:78,713,353\n",
      "INFO\t2017-01-11 17:24:26\tGenotypeConcordance\tchecked     6,320,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:79,716,784\n",
      "INFO\t2017-01-11 17:24:26\tGenotypeConcordance\tchecked     6,330,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:80,551,644\n",
      "INFO\t2017-01-11 17:24:27\tGenotypeConcordance\tchecked     6,340,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:81,508,145\n",
      "INFO\t2017-01-11 17:24:27\tGenotypeConcordance\tchecked     6,350,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:82,510,638\n",
      "INFO\t2017-01-11 17:24:27\tGenotypeConcordance\tchecked     6,360,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:83,454,824\n",
      "INFO\t2017-01-11 17:24:27\tGenotypeConcordance\tchecked     6,370,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:84,375,440\n",
      "INFO\t2017-01-11 17:24:28\tGenotypeConcordance\tchecked     6,380,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:85,236,039\n",
      "INFO\t2017-01-11 17:24:28\tGenotypeConcordance\tchecked     6,390,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:86,119,954\n",
      "INFO\t2017-01-11 17:24:28\tGenotypeConcordance\tchecked     6,400,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:86,997,521\n",
      "INFO\t2017-01-11 17:24:29\tGenotypeConcordance\tchecked     6,410,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:87,868,712\n",
      "INFO\t2017-01-11 17:24:29\tGenotypeConcordance\tchecked     6,420,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:88,716,678\n",
      "INFO\t2017-01-11 17:24:29\tGenotypeConcordance\tchecked     6,430,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:89,612,355\n",
      "INFO\t2017-01-11 17:24:29\tGenotypeConcordance\tchecked     6,440,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:90,430,115\n",
      "INFO\t2017-01-11 17:24:30\tGenotypeConcordance\tchecked     6,450,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:94,327,307\n",
      "INFO\t2017-01-11 17:24:30\tGenotypeConcordance\tchecked     6,460,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:95,106,745\n",
      "INFO\t2017-01-11 17:24:30\tGenotypeConcordance\tchecked     6,470,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:95,861,359\n",
      "INFO\t2017-01-11 17:24:31\tGenotypeConcordance\tchecked     6,480,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:96,699,514\n",
      "INFO\t2017-01-11 17:24:31\tGenotypeConcordance\tchecked     6,490,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:97,764,431\n",
      "INFO\t2017-01-11 17:24:31\tGenotypeConcordance\tchecked     6,500,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:98,394,062\n",
      "INFO\t2017-01-11 17:24:31\tGenotypeConcordance\tchecked     6,510,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:99,150,890\n",
      "INFO\t2017-01-11 17:24:32\tGenotypeConcordance\tchecked     6,520,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:100,192,167\n",
      "INFO\t2017-01-11 17:24:32\tGenotypeConcordance\tchecked     6,530,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:101,042,354\n",
      "INFO\t2017-01-11 17:24:32\tGenotypeConcordance\tchecked     6,540,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:102,053,029\n",
      "INFO\t2017-01-11 17:24:33\tGenotypeConcordance\tchecked     6,550,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:102,939,521\n",
      "INFO\t2017-01-11 17:24:33\tGenotypeConcordance\tchecked     6,560,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:103,694,562\n",
      "INFO\t2017-01-11 17:24:33\tGenotypeConcordance\tchecked     6,570,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:104,515,697\n",
      "INFO\t2017-01-11 17:24:33\tGenotypeConcordance\tchecked     6,580,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:105,348,141\n",
      "INFO\t2017-01-11 17:24:34\tGenotypeConcordance\tchecked     6,590,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:106,153,752\n",
      "INFO\t2017-01-11 17:24:34\tGenotypeConcordance\tchecked     6,600,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:107,027,393\n",
      "INFO\t2017-01-11 17:24:34\tGenotypeConcordance\tchecked     6,610,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:108,030,714\n",
      "INFO\t2017-01-11 17:24:34\tGenotypeConcordance\tchecked     6,620,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:108,975,423\n",
      "INFO\t2017-01-11 17:24:35\tGenotypeConcordance\tchecked     6,630,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:109,796,515\n",
      "INFO\t2017-01-11 17:24:35\tGenotypeConcordance\tchecked     6,640,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:110,729,247\n",
      "INFO\t2017-01-11 17:24:35\tGenotypeConcordance\tchecked     6,650,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:111,657,902\n",
      "INFO\t2017-01-11 17:24:36\tGenotypeConcordance\tchecked     6,660,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:112,508,344\n",
      "INFO\t2017-01-11 17:24:36\tGenotypeConcordance\tchecked     6,670,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:113,334,752\n",
      "INFO\t2017-01-11 17:24:36\tGenotypeConcordance\tchecked     6,680,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:114,394,131\n",
      "INFO\t2017-01-11 17:24:36\tGenotypeConcordance\tchecked     6,690,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:115,515,376\n",
      "INFO\t2017-01-11 17:24:37\tGenotypeConcordance\tchecked     6,700,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:116,424,880\n",
      "INFO\t2017-01-11 17:24:37\tGenotypeConcordance\tchecked     6,710,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:117,241,077\n",
      "INFO\t2017-01-11 17:24:37\tGenotypeConcordance\tchecked     6,720,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:118,082,137\n",
      "INFO\t2017-01-11 17:24:38\tGenotypeConcordance\tchecked     6,730,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:118,972,605\n",
      "INFO\t2017-01-11 17:24:38\tGenotypeConcordance\tchecked     6,740,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:119,878,441\n",
      "INFO\t2017-01-11 17:24:38\tGenotypeConcordance\tchecked     6,750,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:120,792,187\n",
      "INFO\t2017-01-11 17:24:38\tGenotypeConcordance\tchecked     6,760,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:121,807,420\n",
      "INFO\t2017-01-11 17:24:39\tGenotypeConcordance\tchecked     6,770,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:122,764,377\n",
      "INFO\t2017-01-11 17:24:39\tGenotypeConcordance\tchecked     6,780,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:123,678,926\n",
      "INFO\t2017-01-11 17:24:39\tGenotypeConcordance\tchecked     6,790,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:124,600,003\n",
      "INFO\t2017-01-11 17:24:40\tGenotypeConcordance\tchecked     6,800,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:125,454,681\n",
      "INFO\t2017-01-11 17:24:40\tGenotypeConcordance\tchecked     6,810,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:126,016,956\n",
      "INFO\t2017-01-11 17:24:40\tGenotypeConcordance\tchecked     6,820,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:126,896,517\n",
      "INFO\t2017-01-11 17:24:40\tGenotypeConcordance\tchecked     6,830,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:127,779,944\n",
      "INFO\t2017-01-11 17:24:41\tGenotypeConcordance\tchecked     6,840,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:128,694,269\n",
      "INFO\t2017-01-11 17:24:41\tGenotypeConcordance\tchecked     6,850,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:129,642,923\n",
      "INFO\t2017-01-11 17:24:41\tGenotypeConcordance\tchecked     6,860,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:130,278,086\n",
      "INFO\t2017-01-11 17:24:41\tGenotypeConcordance\tchecked     6,870,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:131,283,892\n",
      "INFO\t2017-01-11 17:24:42\tGenotypeConcordance\tchecked     6,880,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:132,072,419\n",
      "INFO\t2017-01-11 17:24:42\tGenotypeConcordance\tchecked     6,890,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:132,944,181\n",
      "INFO\t2017-01-11 17:24:42\tGenotypeConcordance\tchecked     6,900,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:133,776,063\n",
      "INFO\t2017-01-11 17:24:43\tGenotypeConcordance\tchecked     6,910,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:134,677,613\n",
      "INFO\t2017-01-11 17:24:43\tGenotypeConcordance\tchecked     6,920,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:135,647,261\n",
      "INFO\t2017-01-11 17:24:43\tGenotypeConcordance\tchecked     6,930,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:136,701,538\n",
      "INFO\t2017-01-11 17:24:43\tGenotypeConcordance\tchecked     6,940,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:137,668,783\n",
      "INFO\t2017-01-11 17:24:44\tGenotypeConcordance\tchecked     6,950,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:138,676,755\n",
      "INFO\t2017-01-11 17:24:44\tGenotypeConcordance\tchecked     6,960,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:139,651,255\n",
      "INFO\t2017-01-11 17:24:44\tGenotypeConcordance\tchecked     6,970,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:140,523,699\n",
      "INFO\t2017-01-11 17:24:44\tGenotypeConcordance\tchecked     6,980,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:141,412,883\n",
      "INFO\t2017-01-11 17:24:45\tGenotypeConcordance\tchecked     6,990,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:142,335,410\n",
      "INFO\t2017-01-11 17:24:45\tGenotypeConcordance\tchecked     7,000,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:143,238,874\n",
      "INFO\t2017-01-11 17:24:45\tGenotypeConcordance\tchecked     7,010,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:144,113,799\n",
      "INFO\t2017-01-11 17:24:45\tGenotypeConcordance\tchecked     7,020,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:144,923,706\n",
      "INFO\t2017-01-11 17:24:46\tGenotypeConcordance\tchecked     7,030,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:145,686,843\n",
      "INFO\t2017-01-11 17:24:46\tGenotypeConcordance\tchecked     7,040,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:146,478,254\n",
      "INFO\t2017-01-11 17:24:46\tGenotypeConcordance\tchecked     7,050,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:147,498,662\n",
      "INFO\t2017-01-11 17:24:47\tGenotypeConcordance\tchecked     7,060,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:148,355,413\n",
      "INFO\t2017-01-11 17:24:47\tGenotypeConcordance\tchecked     7,070,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:149,199,064\n",
      "INFO\t2017-01-11 17:24:47\tGenotypeConcordance\tchecked     7,080,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:150,148,092\n",
      "INFO\t2017-01-11 17:24:47\tGenotypeConcordance\tchecked     7,090,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:151,097,654\n",
      "INFO\t2017-01-11 17:24:48\tGenotypeConcordance\tchecked     7,100,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:151,986,427\n",
      "INFO\t2017-01-11 17:24:48\tGenotypeConcordance\tchecked     7,110,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:152,886,681\n",
      "INFO\t2017-01-11 17:24:48\tGenotypeConcordance\tchecked     7,120,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:153,699,976\n",
      "INFO\t2017-01-11 17:24:49\tGenotypeConcordance\tchecked     7,130,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:154,593,619\n",
      "INFO\t2017-01-11 17:24:49\tGenotypeConcordance\tchecked     7,140,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:155,486,703\n",
      "INFO\t2017-01-11 17:24:49\tGenotypeConcordance\tchecked     7,150,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:156,390,308\n",
      "INFO\t2017-01-11 17:24:49\tGenotypeConcordance\tchecked     7,160,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:157,358,048\n",
      "INFO\t2017-01-11 17:24:50\tGenotypeConcordance\tchecked     7,170,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:158,271,690\n",
      "INFO\t2017-01-11 17:24:50\tGenotypeConcordance\tchecked     7,180,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:159,142,234\n",
      "INFO\t2017-01-11 17:24:50\tGenotypeConcordance\tchecked     7,190,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:159,977,751\n",
      "INFO\t2017-01-11 17:24:51\tGenotypeConcordance\tchecked     7,200,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:160,895,762\n",
      "INFO\t2017-01-11 17:24:51\tGenotypeConcordance\tchecked     7,210,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:161,729,127\n",
      "INFO\t2017-01-11 17:24:51\tGenotypeConcordance\tchecked     7,220,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:162,494,864\n",
      "INFO\t2017-01-11 17:24:51\tGenotypeConcordance\tchecked     7,230,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:163,367,841\n",
      "INFO\t2017-01-11 17:24:52\tGenotypeConcordance\tchecked     7,240,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:164,200,352\n",
      "INFO\t2017-01-11 17:24:52\tGenotypeConcordance\tchecked     7,250,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:165,126,400\n",
      "INFO\t2017-01-11 17:24:52\tGenotypeConcordance\tchecked     7,260,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:165,943,024\n",
      "INFO\t2017-01-11 17:24:53\tGenotypeConcordance\tchecked     7,270,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:166,736,793\n",
      "INFO\t2017-01-11 17:24:53\tGenotypeConcordance\tchecked     7,280,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:167,666,443\n",
      "INFO\t2017-01-11 17:24:53\tGenotypeConcordance\tchecked     7,290,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:168,628,196\n",
      "INFO\t2017-01-11 17:24:53\tGenotypeConcordance\tchecked     7,300,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:169,554,129\n",
      "INFO\t2017-01-11 17:24:54\tGenotypeConcordance\tchecked     7,310,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:170,420,463\n",
      "INFO\t2017-01-11 17:24:54\tGenotypeConcordance\tchecked     7,320,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:171,326,315\n",
      "INFO\t2017-01-11 17:24:54\tGenotypeConcordance\tchecked     7,330,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:172,239,988\n",
      "INFO\t2017-01-11 17:24:55\tGenotypeConcordance\tchecked     7,340,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:173,149,550\n",
      "INFO\t2017-01-11 17:24:55\tGenotypeConcordance\tchecked     7,350,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:174,049,118\n",
      "INFO\t2017-01-11 17:24:55\tGenotypeConcordance\tchecked     7,360,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:174,838,151\n",
      "INFO\t2017-01-11 17:24:55\tGenotypeConcordance\tchecked     7,370,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:175,559,562\n",
      "INFO\t2017-01-11 17:24:56\tGenotypeConcordance\tchecked     7,380,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:176,421,790\n",
      "INFO\t2017-01-11 17:24:56\tGenotypeConcordance\tchecked     7,390,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:177,215,858\n",
      "INFO\t2017-01-11 17:24:56\tGenotypeConcordance\tchecked     7,400,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:178,145,346\n",
      "INFO\t2017-01-11 17:24:56\tGenotypeConcordance\tchecked     7,410,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:179,073,852\n",
      "INFO\t2017-01-11 17:24:57\tGenotypeConcordance\tchecked     7,420,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:180,029,935\n",
      "INFO\t2017-01-11 17:24:57\tGenotypeConcordance\tchecked     7,430,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:181,106,125\n",
      "INFO\t2017-01-11 17:24:57\tGenotypeConcordance\tchecked     7,440,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:182,153,831\n",
      "INFO\t2017-01-11 17:24:58\tGenotypeConcordance\tchecked     7,450,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:183,126,506\n",
      "INFO\t2017-01-11 17:24:58\tGenotypeConcordance\tchecked     7,460,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:183,985,669\n",
      "INFO\t2017-01-11 17:24:58\tGenotypeConcordance\tchecked     7,470,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:184,834,329\n",
      "INFO\t2017-01-11 17:24:58\tGenotypeConcordance\tchecked     7,480,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:185,793,273\n",
      "INFO\t2017-01-11 17:24:59\tGenotypeConcordance\tchecked     7,490,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:186,628,163\n",
      "INFO\t2017-01-11 17:24:59\tGenotypeConcordance\tchecked     7,500,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:187,405,595\n",
      "INFO\t2017-01-11 17:24:59\tGenotypeConcordance\tchecked     7,510,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:188,229,424\n",
      "INFO\t2017-01-11 17:25:00\tGenotypeConcordance\tchecked     7,520,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:189,133,827\n",
      "INFO\t2017-01-11 17:25:00\tGenotypeConcordance\tchecked     7,530,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:189,871,351\n",
      "INFO\t2017-01-11 17:25:00\tGenotypeConcordance\tchecked     7,540,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:190,595,591\n",
      "INFO\t2017-01-11 17:25:00\tGenotypeConcordance\tchecked     7,550,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:191,343,146\n",
      "INFO\t2017-01-11 17:25:01\tGenotypeConcordance\tchecked     7,560,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:192,081,284\n",
      "INFO\t2017-01-11 17:25:01\tGenotypeConcordance\tchecked     7,570,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:192,852,448\n",
      "INFO\t2017-01-11 17:25:01\tGenotypeConcordance\tchecked     7,580,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:193,718,465\n",
      "INFO\t2017-01-11 17:25:02\tGenotypeConcordance\tchecked     7,590,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:194,398,645\n",
      "INFO\t2017-01-11 17:25:02\tGenotypeConcordance\tchecked     7,600,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:195,076,877\n",
      "INFO\t2017-01-11 17:25:02\tGenotypeConcordance\tchecked     7,610,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:195,597,503\n",
      "INFO\t2017-01-11 17:25:03\tGenotypeConcordance\tchecked     7,620,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 3:196,255,340\n",
      "INFO\t2017-01-11 17:25:03\tGenotypeConcordance\tchecked     7,630,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 3:196,988,678\n",
      "INFO\t2017-01-11 17:25:03\tGenotypeConcordance\tchecked     7,640,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 3:197,702,371\n",
      "INFO\t2017-01-11 17:25:04\tGenotypeConcordance\tchecked     7,650,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 4:542,898\n",
      "INFO\t2017-01-11 17:25:04\tGenotypeConcordance\tchecked     7,660,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:1,221,441\n",
      "INFO\t2017-01-11 17:25:04\tGenotypeConcordance\tchecked     7,670,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:1,910,811\n",
      "INFO\t2017-01-11 17:25:04\tGenotypeConcordance\tchecked     7,680,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:2,749,343\n",
      "INFO\t2017-01-11 17:25:05\tGenotypeConcordance\tchecked     7,690,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:3,514,344\n",
      "INFO\t2017-01-11 17:25:05\tGenotypeConcordance\tchecked     7,700,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:3,929,865\n",
      "INFO\t2017-01-11 17:25:05\tGenotypeConcordance\tchecked     7,710,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:4,344,979\n",
      "INFO\t2017-01-11 17:25:05\tGenotypeConcordance\tchecked     7,720,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:5,089,303\n",
      "INFO\t2017-01-11 17:25:06\tGenotypeConcordance\tchecked     7,730,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:5,669,247\n",
      "INFO\t2017-01-11 17:25:06\tGenotypeConcordance\tchecked     7,740,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:6,298,705\n",
      "INFO\t2017-01-11 17:25:06\tGenotypeConcordance\tchecked     7,750,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:6,957,332\n",
      "INFO\t2017-01-11 17:25:07\tGenotypeConcordance\tchecked     7,760,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:7,535,219\n",
      "INFO\t2017-01-11 17:25:07\tGenotypeConcordance\tchecked     7,770,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:8,114,300\n",
      "INFO\t2017-01-11 17:25:07\tGenotypeConcordance\tchecked     7,780,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:8,722,998\n",
      "INFO\t2017-01-11 17:25:07\tGenotypeConcordance\tchecked     7,790,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:9,183,124\n",
      "INFO\t2017-01-11 17:25:08\tGenotypeConcordance\tchecked     7,800,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:9,669,609\n",
      "INFO\t2017-01-11 17:25:08\tGenotypeConcordance\tchecked     7,810,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:10,210,906\n",
      "INFO\t2017-01-11 17:25:08\tGenotypeConcordance\tchecked     7,820,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:10,942,705\n",
      "INFO\t2017-01-11 17:25:09\tGenotypeConcordance\tchecked     7,830,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:11,650,710\n",
      "INFO\t2017-01-11 17:25:09\tGenotypeConcordance\tchecked     7,840,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:12,318,791\n",
      "INFO\t2017-01-11 17:25:09\tGenotypeConcordance\tchecked     7,850,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:13,018,888\n",
      "INFO\t2017-01-11 17:25:09\tGenotypeConcordance\tchecked     7,860,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:13,864,746\n",
      "INFO\t2017-01-11 17:25:10\tGenotypeConcordance\tchecked     7,870,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:14,624,305\n",
      "INFO\t2017-01-11 17:25:10\tGenotypeConcordance\tchecked     7,880,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:15,457,861\n",
      "INFO\t2017-01-11 17:25:10\tGenotypeConcordance\tchecked     7,890,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:16,377,284\n",
      "INFO\t2017-01-11 17:25:10\tGenotypeConcordance\tchecked     7,900,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:17,240,787\n",
      "INFO\t2017-01-11 17:25:11\tGenotypeConcordance\tchecked     7,910,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:18,080,821\n",
      "INFO\t2017-01-11 17:25:11\tGenotypeConcordance\tchecked     7,920,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:18,918,814\n",
      "INFO\t2017-01-11 17:25:11\tGenotypeConcordance\tchecked     7,930,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:19,736,258\n",
      "INFO\t2017-01-11 17:25:12\tGenotypeConcordance\tchecked     7,940,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:20,560,883\n",
      "INFO\t2017-01-11 17:25:12\tGenotypeConcordance\tchecked     7,950,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:21,283,211\n",
      "INFO\t2017-01-11 17:25:12\tGenotypeConcordance\tchecked     7,960,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:22,057,888\n",
      "INFO\t2017-01-11 17:25:12\tGenotypeConcordance\tchecked     7,970,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:22,800,848\n",
      "INFO\t2017-01-11 17:25:13\tGenotypeConcordance\tchecked     7,980,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:23,656,138\n",
      "INFO\t2017-01-11 17:25:13\tGenotypeConcordance\tchecked     7,990,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:24,464,042\n",
      "INFO\t2017-01-11 17:25:13\tGenotypeConcordance\tchecked     8,000,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:25,290,880\n",
      "INFO\t2017-01-11 17:25:13\tGenotypeConcordance\tchecked     8,010,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:26,074,011\n",
      "INFO\t2017-01-11 17:25:14\tGenotypeConcordance\tchecked     8,020,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:26,996,783\n",
      "INFO\t2017-01-11 17:25:14\tGenotypeConcordance\tchecked     8,030,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:27,795,241\n",
      "INFO\t2017-01-11 17:25:14\tGenotypeConcordance\tchecked     8,040,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:28,570,198\n",
      "INFO\t2017-01-11 17:25:15\tGenotypeConcordance\tchecked     8,050,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:29,369,334\n",
      "INFO\t2017-01-11 17:25:15\tGenotypeConcordance\tchecked     8,060,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:30,159,982\n",
      "INFO\t2017-01-11 17:25:15\tGenotypeConcordance\tchecked     8,070,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:30,953,996\n",
      "INFO\t2017-01-11 17:25:15\tGenotypeConcordance\tchecked     8,080,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:31,734,106\n",
      "INFO\t2017-01-11 17:25:16\tGenotypeConcordance\tchecked     8,090,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:32,484,424\n",
      "INFO\t2017-01-11 17:25:16\tGenotypeConcordance\tchecked     8,100,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:33,203,384\n",
      "INFO\t2017-01-11 17:25:16\tGenotypeConcordance\tchecked     8,110,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:34,051,125\n",
      "INFO\t2017-01-11 17:25:17\tGenotypeConcordance\tchecked     8,120,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:34,917,773\n",
      "INFO\t2017-01-11 17:25:17\tGenotypeConcordance\tchecked     8,130,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:35,642,436\n",
      "INFO\t2017-01-11 17:25:17\tGenotypeConcordance\tchecked     8,140,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:36,438,525\n",
      "INFO\t2017-01-11 17:25:17\tGenotypeConcordance\tchecked     8,150,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:37,172,706\n",
      "INFO\t2017-01-11 17:25:18\tGenotypeConcordance\tchecked     8,160,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:37,925,177\n",
      "INFO\t2017-01-11 17:25:18\tGenotypeConcordance\tchecked     8,170,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:38,750,185\n",
      "INFO\t2017-01-11 17:25:18\tGenotypeConcordance\tchecked     8,180,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:39,591,005\n",
      "INFO\t2017-01-11 17:25:19\tGenotypeConcordance\tchecked     8,190,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:40,407,664\n",
      "INFO\t2017-01-11 17:25:19\tGenotypeConcordance\tchecked     8,200,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:41,208,436\n",
      "INFO\t2017-01-11 17:25:19\tGenotypeConcordance\tchecked     8,210,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:42,117,633\n",
      "INFO\t2017-01-11 17:25:19\tGenotypeConcordance\tchecked     8,220,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:42,908,795\n",
      "INFO\t2017-01-11 17:25:20\tGenotypeConcordance\tchecked     8,230,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:43,683,418\n",
      "INFO\t2017-01-11 17:25:20\tGenotypeConcordance\tchecked     8,240,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:44,537,786\n",
      "INFO\t2017-01-11 17:25:20\tGenotypeConcordance\tchecked     8,250,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:45,281,592\n",
      "INFO\t2017-01-11 17:25:20\tGenotypeConcordance\tchecked     8,260,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:46,140,691\n",
      "INFO\t2017-01-11 17:25:21\tGenotypeConcordance\tchecked     8,270,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:47,056,073\n",
      "INFO\t2017-01-11 17:25:21\tGenotypeConcordance\tchecked     8,280,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:48,060,169\n",
      "INFO\t2017-01-11 17:25:21\tGenotypeConcordance\tchecked     8,290,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:49,095,993\n",
      "INFO\t2017-01-11 17:25:22\tGenotypeConcordance\tchecked     8,300,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:49,334,997\n",
      "INFO\t2017-01-11 17:25:22\tGenotypeConcordance\tchecked     8,310,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:52,851,830\n",
      "INFO\t2017-01-11 17:25:22\tGenotypeConcordance\tchecked     8,320,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:53,712,121\n",
      "INFO\t2017-01-11 17:25:22\tGenotypeConcordance\tchecked     8,330,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:54,633,846\n",
      "INFO\t2017-01-11 17:25:23\tGenotypeConcordance\tchecked     8,340,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:55,490,518\n",
      "INFO\t2017-01-11 17:25:23\tGenotypeConcordance\tchecked     8,350,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:56,300,941\n",
      "INFO\t2017-01-11 17:25:23\tGenotypeConcordance\tchecked     8,360,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:57,171,987\n",
      "INFO\t2017-01-11 17:25:23\tGenotypeConcordance\tchecked     8,370,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:57,934,235\n",
      "INFO\t2017-01-11 17:25:24\tGenotypeConcordance\tchecked     8,380,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:58,608,254\n",
      "INFO\t2017-01-11 17:25:24\tGenotypeConcordance\tchecked     8,390,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:59,261,192\n",
      "INFO\t2017-01-11 17:25:24\tGenotypeConcordance\tchecked     8,400,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:60,033,417\n",
      "INFO\t2017-01-11 17:25:25\tGenotypeConcordance\tchecked     8,410,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:60,690,523\n",
      "INFO\t2017-01-11 17:25:25\tGenotypeConcordance\tchecked     8,420,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:61,375,698\n",
      "INFO\t2017-01-11 17:25:25\tGenotypeConcordance\tchecked     8,430,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:62,195,940\n",
      "INFO\t2017-01-11 17:25:25\tGenotypeConcordance\tchecked     8,440,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:63,135,706\n",
      "INFO\t2017-01-11 17:25:26\tGenotypeConcordance\tchecked     8,450,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:63,808,535\n",
      "INFO\t2017-01-11 17:25:26\tGenotypeConcordance\tchecked     8,460,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:64,456,321\n",
      "INFO\t2017-01-11 17:25:26\tGenotypeConcordance\tchecked     8,470,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:65,227,122\n",
      "INFO\t2017-01-11 17:25:27\tGenotypeConcordance\tchecked     8,480,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:65,930,371\n",
      "INFO\t2017-01-11 17:25:27\tGenotypeConcordance\tchecked     8,490,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:66,696,287\n",
      "INFO\t2017-01-11 17:25:27\tGenotypeConcordance\tchecked     8,500,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:67,443,752\n",
      "INFO\t2017-01-11 17:25:28\tGenotypeConcordance\tchecked     8,510,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:68,269,826\n",
      "INFO\t2017-01-11 17:25:28\tGenotypeConcordance\tchecked     8,520,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:69,141,149\n",
      "INFO\t2017-01-11 17:25:28\tGenotypeConcordance\tchecked     8,530,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:69,867,584\n",
      "INFO\t2017-01-11 17:25:28\tGenotypeConcordance\tchecked     8,540,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:70,460,328\n",
      "INFO\t2017-01-11 17:25:29\tGenotypeConcordance\tchecked     8,550,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:71,210,727\n",
      "INFO\t2017-01-11 17:25:29\tGenotypeConcordance\tchecked     8,560,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:71,994,066\n",
      "INFO\t2017-01-11 17:25:29\tGenotypeConcordance\tchecked     8,570,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:72,868,259\n",
      "INFO\t2017-01-11 17:25:30\tGenotypeConcordance\tchecked     8,580,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:73,633,538\n",
      "INFO\t2017-01-11 17:25:30\tGenotypeConcordance\tchecked     8,590,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:74,539,627\n",
      "INFO\t2017-01-11 17:25:30\tGenotypeConcordance\tchecked     8,600,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:75,534,895\n",
      "INFO\t2017-01-11 17:25:30\tGenotypeConcordance\tchecked     8,610,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:76,302,332\n",
      "INFO\t2017-01-11 17:25:31\tGenotypeConcordance\tchecked     8,620,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:77,132,071\n",
      "INFO\t2017-01-11 17:25:31\tGenotypeConcordance\tchecked     8,630,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:78,001,508\n",
      "INFO\t2017-01-11 17:25:31\tGenotypeConcordance\tchecked     8,640,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:78,806,713\n",
      "INFO\t2017-01-11 17:25:32\tGenotypeConcordance\tchecked     8,650,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:79,650,815\n",
      "INFO\t2017-01-11 17:25:32\tGenotypeConcordance\tchecked     8,660,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:80,643,067\n",
      "INFO\t2017-01-11 17:25:32\tGenotypeConcordance\tchecked     8,670,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:81,589,438\n",
      "INFO\t2017-01-11 17:25:32\tGenotypeConcordance\tchecked     8,680,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:82,495,893\n",
      "INFO\t2017-01-11 17:25:33\tGenotypeConcordance\tchecked     8,690,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:83,369,296\n",
      "INFO\t2017-01-11 17:25:33\tGenotypeConcordance\tchecked     8,700,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:84,240,409\n",
      "INFO\t2017-01-11 17:25:33\tGenotypeConcordance\tchecked     8,710,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:85,074,102\n",
      "INFO\t2017-01-11 17:25:34\tGenotypeConcordance\tchecked     8,720,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:86,099,883\n",
      "INFO\t2017-01-11 17:25:34\tGenotypeConcordance\tchecked     8,730,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:86,933,202\n",
      "INFO\t2017-01-11 17:25:34\tGenotypeConcordance\tchecked     8,740,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:87,940,940\n",
      "INFO\t2017-01-11 17:25:34\tGenotypeConcordance\tchecked     8,750,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:88,751,096\n",
      "INFO\t2017-01-11 17:25:35\tGenotypeConcordance\tchecked     8,760,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:89,665,891\n",
      "INFO\t2017-01-11 17:25:35\tGenotypeConcordance\tchecked     8,770,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:90,545,192\n",
      "INFO\t2017-01-11 17:25:35\tGenotypeConcordance\tchecked     8,780,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:91,403,851\n",
      "INFO\t2017-01-11 17:25:36\tGenotypeConcordance\tchecked     8,790,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:92,197,449\n",
      "INFO\t2017-01-11 17:25:36\tGenotypeConcordance\tchecked     8,800,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:93,013,683\n",
      "INFO\t2017-01-11 17:25:36\tGenotypeConcordance\tchecked     8,810,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:93,999,179\n",
      "INFO\t2017-01-11 17:25:36\tGenotypeConcordance\tchecked     8,820,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:94,932,155\n",
      "INFO\t2017-01-11 17:25:37\tGenotypeConcordance\tchecked     8,830,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:95,783,955\n",
      "INFO\t2017-01-11 17:25:37\tGenotypeConcordance\tchecked     8,840,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:96,588,172\n",
      "INFO\t2017-01-11 17:25:37\tGenotypeConcordance\tchecked     8,850,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:97,439,543\n",
      "INFO\t2017-01-11 17:25:38\tGenotypeConcordance\tchecked     8,860,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:98,277,522\n",
      "INFO\t2017-01-11 17:25:38\tGenotypeConcordance\tchecked     8,870,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:99,144,321\n",
      "INFO\t2017-01-11 17:25:38\tGenotypeConcordance\tchecked     8,880,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:100,042,242\n",
      "INFO\t2017-01-11 17:25:38\tGenotypeConcordance\tchecked     8,890,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:101,020,755\n",
      "INFO\t2017-01-11 17:25:39\tGenotypeConcordance\tchecked     8,900,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:101,871,546\n",
      "INFO\t2017-01-11 17:25:39\tGenotypeConcordance\tchecked     8,910,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:102,722,852\n",
      "INFO\t2017-01-11 17:25:39\tGenotypeConcordance\tchecked     8,920,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:103,633,275\n",
      "INFO\t2017-01-11 17:25:39\tGenotypeConcordance\tchecked     8,930,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:104,514,952\n",
      "INFO\t2017-01-11 17:25:40\tGenotypeConcordance\tchecked     8,940,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:105,414,718\n",
      "INFO\t2017-01-11 17:25:40\tGenotypeConcordance\tchecked     8,950,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:106,367,210\n",
      "INFO\t2017-01-11 17:25:40\tGenotypeConcordance\tchecked     8,960,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:107,350,635\n",
      "INFO\t2017-01-11 17:25:41\tGenotypeConcordance\tchecked     8,970,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:108,330,475\n",
      "INFO\t2017-01-11 17:25:41\tGenotypeConcordance\tchecked     8,980,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:109,301,111\n",
      "INFO\t2017-01-11 17:25:41\tGenotypeConcordance\tchecked     8,990,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:110,144,593\n",
      "INFO\t2017-01-11 17:25:41\tGenotypeConcordance\tchecked     9,000,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:111,005,409\n",
      "INFO\t2017-01-11 17:25:42\tGenotypeConcordance\tchecked     9,010,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:111,858,679\n",
      "INFO\t2017-01-11 17:25:42\tGenotypeConcordance\tchecked     9,020,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:112,800,228\n",
      "INFO\t2017-01-11 17:25:42\tGenotypeConcordance\tchecked     9,030,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:113,797,055\n",
      "INFO\t2017-01-11 17:25:43\tGenotypeConcordance\tchecked     9,040,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:114,704,051\n",
      "INFO\t2017-01-11 17:25:43\tGenotypeConcordance\tchecked     9,050,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:115,488,127\n",
      "INFO\t2017-01-11 17:25:43\tGenotypeConcordance\tchecked     9,060,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:116,329,991\n",
      "INFO\t2017-01-11 17:25:43\tGenotypeConcordance\tchecked     9,070,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:117,110,376\n",
      "INFO\t2017-01-11 17:25:44\tGenotypeConcordance\tchecked     9,080,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:117,894,668\n",
      "INFO\t2017-01-11 17:25:44\tGenotypeConcordance\tchecked     9,090,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:118,721,958\n",
      "INFO\t2017-01-11 17:25:44\tGenotypeConcordance\tchecked     9,100,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:119,668,186\n",
      "INFO\t2017-01-11 17:25:45\tGenotypeConcordance\tchecked     9,110,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:120,459,548\n",
      "INFO\t2017-01-11 17:25:45\tGenotypeConcordance\tchecked     9,120,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:121,345,846\n",
      "INFO\t2017-01-11 17:25:45\tGenotypeConcordance\tchecked     9,130,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:122,164,895\n",
      "INFO\t2017-01-11 17:25:46\tGenotypeConcordance\tchecked     9,140,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:123,034,210\n",
      "INFO\t2017-01-11 17:25:46\tGenotypeConcordance\tchecked     9,150,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:123,983,264\n",
      "INFO\t2017-01-11 17:25:46\tGenotypeConcordance\tchecked     9,160,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:124,821,664\n",
      "INFO\t2017-01-11 17:25:46\tGenotypeConcordance\tchecked     9,170,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:125,713,066\n",
      "INFO\t2017-01-11 17:25:47\tGenotypeConcordance\tchecked     9,180,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:126,556,132\n",
      "INFO\t2017-01-11 17:25:47\tGenotypeConcordance\tchecked     9,190,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:127,370,617\n",
      "INFO\t2017-01-11 17:25:47\tGenotypeConcordance\tchecked     9,200,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:128,314,339\n",
      "INFO\t2017-01-11 17:25:47\tGenotypeConcordance\tchecked     9,210,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:129,423,388\n",
      "INFO\t2017-01-11 17:25:48\tGenotypeConcordance\tchecked     9,220,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:130,270,267\n",
      "INFO\t2017-01-11 17:25:48\tGenotypeConcordance\tchecked     9,230,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:131,095,087\n",
      "INFO\t2017-01-11 17:25:48\tGenotypeConcordance\tchecked     9,240,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:131,872,320\n",
      "INFO\t2017-01-11 17:25:49\tGenotypeConcordance\tchecked     9,250,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:132,725,160\n",
      "INFO\t2017-01-11 17:25:49\tGenotypeConcordance\tchecked     9,260,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:133,576,536\n",
      "INFO\t2017-01-11 17:25:49\tGenotypeConcordance\tchecked     9,270,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:134,425,481\n",
      "INFO\t2017-01-11 17:25:49\tGenotypeConcordance\tchecked     9,280,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:135,351,753\n",
      "INFO\t2017-01-11 17:25:50\tGenotypeConcordance\tchecked     9,290,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:136,111,663\n",
      "INFO\t2017-01-11 17:25:50\tGenotypeConcordance\tchecked     9,300,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:136,937,648\n",
      "INFO\t2017-01-11 17:25:50\tGenotypeConcordance\tchecked     9,310,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:137,816,744\n",
      "INFO\t2017-01-11 17:25:51\tGenotypeConcordance\tchecked     9,320,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:138,608,230\n",
      "INFO\t2017-01-11 17:25:51\tGenotypeConcordance\tchecked     9,330,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:139,477,559\n",
      "INFO\t2017-01-11 17:25:51\tGenotypeConcordance\tchecked     9,340,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:140,354,073\n",
      "INFO\t2017-01-11 17:25:51\tGenotypeConcordance\tchecked     9,350,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:141,301,420\n",
      "INFO\t2017-01-11 17:25:52\tGenotypeConcordance\tchecked     9,360,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:142,199,705\n",
      "INFO\t2017-01-11 17:25:52\tGenotypeConcordance\tchecked     9,370,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:143,041,723\n",
      "INFO\t2017-01-11 17:25:52\tGenotypeConcordance\tchecked     9,380,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:143,950,762\n",
      "INFO\t2017-01-11 17:25:53\tGenotypeConcordance\tchecked     9,390,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:144,818,822\n",
      "INFO\t2017-01-11 17:25:53\tGenotypeConcordance\tchecked     9,400,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:145,562,738\n",
      "INFO\t2017-01-11 17:25:53\tGenotypeConcordance\tchecked     9,410,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:146,602,653\n",
      "INFO\t2017-01-11 17:25:54\tGenotypeConcordance\tchecked     9,420,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:147,514,783\n",
      "INFO\t2017-01-11 17:25:54\tGenotypeConcordance\tchecked     9,430,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:148,503,146\n",
      "INFO\t2017-01-11 17:25:54\tGenotypeConcordance\tchecked     9,440,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:149,458,917\n",
      "INFO\t2017-01-11 17:25:54\tGenotypeConcordance\tchecked     9,450,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:150,443,928\n",
      "INFO\t2017-01-11 17:25:55\tGenotypeConcordance\tchecked     9,460,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:151,399,246\n",
      "INFO\t2017-01-11 17:25:55\tGenotypeConcordance\tchecked     9,470,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:152,493,890\n",
      "INFO\t2017-01-11 17:25:55\tGenotypeConcordance\tchecked     9,480,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:153,527,808\n",
      "INFO\t2017-01-11 17:25:56\tGenotypeConcordance\tchecked     9,490,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:154,394,503\n",
      "INFO\t2017-01-11 17:25:56\tGenotypeConcordance\tchecked     9,500,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:155,262,227\n",
      "INFO\t2017-01-11 17:25:56\tGenotypeConcordance\tchecked     9,510,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:156,167,667\n",
      "INFO\t2017-01-11 17:25:56\tGenotypeConcordance\tchecked     9,520,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:157,000,262\n",
      "INFO\t2017-01-11 17:25:57\tGenotypeConcordance\tchecked     9,530,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:157,797,969\n",
      "INFO\t2017-01-11 17:25:57\tGenotypeConcordance\tchecked     9,540,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:158,668,868\n",
      "INFO\t2017-01-11 17:25:57\tGenotypeConcordance\tchecked     9,550,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:159,641,244\n",
      "INFO\t2017-01-11 17:25:58\tGenotypeConcordance\tchecked     9,560,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:160,534,399\n",
      "INFO\t2017-01-11 17:25:58\tGenotypeConcordance\tchecked     9,570,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:161,344,495\n",
      "INFO\t2017-01-11 17:25:58\tGenotypeConcordance\tchecked     9,580,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:162,066,509\n",
      "INFO\t2017-01-11 17:25:58\tGenotypeConcordance\tchecked     9,590,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:162,836,241\n",
      "INFO\t2017-01-11 17:25:59\tGenotypeConcordance\tchecked     9,600,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:163,537,134\n",
      "INFO\t2017-01-11 17:25:59\tGenotypeConcordance\tchecked     9,610,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:164,384,855\n",
      "INFO\t2017-01-11 17:25:59\tGenotypeConcordance\tchecked     9,620,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:165,170,243\n",
      "INFO\t2017-01-11 17:26:00\tGenotypeConcordance\tchecked     9,630,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:165,900,979\n",
      "INFO\t2017-01-11 17:26:00\tGenotypeConcordance\tchecked     9,640,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:166,681,527\n",
      "INFO\t2017-01-11 17:26:00\tGenotypeConcordance\tchecked     9,650,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:167,434,458\n",
      "INFO\t2017-01-11 17:26:00\tGenotypeConcordance\tchecked     9,660,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:168,273,946\n",
      "INFO\t2017-01-11 17:26:01\tGenotypeConcordance\tchecked     9,670,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:169,121,588\n",
      "INFO\t2017-01-11 17:26:01\tGenotypeConcordance\tchecked     9,680,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:169,996,302\n",
      "INFO\t2017-01-11 17:26:01\tGenotypeConcordance\tchecked     9,690,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:170,914,622\n",
      "INFO\t2017-01-11 17:26:02\tGenotypeConcordance\tchecked     9,700,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:171,679,226\n",
      "INFO\t2017-01-11 17:26:02\tGenotypeConcordance\tchecked     9,710,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:172,489,942\n",
      "INFO\t2017-01-11 17:26:02\tGenotypeConcordance\tchecked     9,720,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:173,298,069\n",
      "INFO\t2017-01-11 17:26:02\tGenotypeConcordance\tchecked     9,730,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:174,216,526\n",
      "INFO\t2017-01-11 17:26:03\tGenotypeConcordance\tchecked     9,740,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:175,061,519\n",
      "INFO\t2017-01-11 17:26:03\tGenotypeConcordance\tchecked     9,750,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:175,887,670\n",
      "INFO\t2017-01-11 17:26:03\tGenotypeConcordance\tchecked     9,760,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:176,732,480\n",
      "INFO\t2017-01-11 17:26:04\tGenotypeConcordance\tchecked     9,770,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:177,532,499\n",
      "INFO\t2017-01-11 17:26:04\tGenotypeConcordance\tchecked     9,780,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:178,354,599\n",
      "INFO\t2017-01-11 17:26:04\tGenotypeConcordance\tchecked     9,790,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:179,136,091\n",
      "INFO\t2017-01-11 17:26:05\tGenotypeConcordance\tchecked     9,800,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:179,816,709\n",
      "INFO\t2017-01-11 17:26:05\tGenotypeConcordance\tchecked     9,810,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:180,507,487\n",
      "INFO\t2017-01-11 17:26:05\tGenotypeConcordance\tchecked     9,820,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:181,226,698\n",
      "INFO\t2017-01-11 17:26:05\tGenotypeConcordance\tchecked     9,830,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:182,002,228\n",
      "INFO\t2017-01-11 17:26:06\tGenotypeConcordance\tchecked     9,840,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:182,680,609\n",
      "INFO\t2017-01-11 17:26:06\tGenotypeConcordance\tchecked     9,850,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:183,458,715\n",
      "INFO\t2017-01-11 17:26:06\tGenotypeConcordance\tchecked     9,860,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:184,257,215\n",
      "INFO\t2017-01-11 17:26:07\tGenotypeConcordance\tchecked     9,870,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:184,913,872\n",
      "INFO\t2017-01-11 17:26:07\tGenotypeConcordance\tchecked     9,880,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:185,656,390\n",
      "INFO\t2017-01-11 17:26:07\tGenotypeConcordance\tchecked     9,890,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:186,418,492\n",
      "INFO\t2017-01-11 17:26:08\tGenotypeConcordance\tchecked     9,900,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:187,118,832\n",
      "INFO\t2017-01-11 17:26:08\tGenotypeConcordance\tchecked     9,910,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:187,835,603\n",
      "INFO\t2017-01-11 17:26:08\tGenotypeConcordance\tchecked     9,920,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:188,626,838\n",
      "INFO\t2017-01-11 17:26:09\tGenotypeConcordance\tchecked     9,930,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:189,251,223\n",
      "INFO\t2017-01-11 17:26:09\tGenotypeConcordance\tchecked     9,940,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:189,869,211\n",
      "INFO\t2017-01-11 17:26:09\tGenotypeConcordance\tchecked     9,950,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:190,520,824\n",
      "INFO\t2017-01-11 17:26:09\tGenotypeConcordance\tchecked     9,960,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:190,852,643\n",
      "INFO\t2017-01-11 17:26:10\tGenotypeConcordance\tchecked     9,970,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:442,904\n",
      "INFO\t2017-01-11 17:26:10\tGenotypeConcordance\tchecked     9,980,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:1,044,904\n",
      "INFO\t2017-01-11 17:26:10\tGenotypeConcordance\tchecked     9,990,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:1,634,161\n",
      "INFO\t2017-01-11 17:26:11\tGenotypeConcordance\tchecked    10,000,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:2,262,683\n",
      "INFO\t2017-01-11 17:26:11\tGenotypeConcordance\tchecked    10,010,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:2,886,975\n",
      "INFO\t2017-01-11 17:26:11\tGenotypeConcordance\tchecked    10,020,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:3,529,660\n",
      "INFO\t2017-01-11 17:26:12\tGenotypeConcordance\tchecked    10,030,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:4,250,000\n",
      "INFO\t2017-01-11 17:26:12\tGenotypeConcordance\tchecked    10,040,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:5,018,551\n",
      "INFO\t2017-01-11 17:26:12\tGenotypeConcordance\tchecked    10,050,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:5,742,391\n",
      "INFO\t2017-01-11 17:26:12\tGenotypeConcordance\tchecked    10,060,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:6,491,204\n",
      "INFO\t2017-01-11 17:26:13\tGenotypeConcordance\tchecked    10,070,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:7,260,750\n",
      "INFO\t2017-01-11 17:26:13\tGenotypeConcordance\tchecked    10,080,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:8,036,175\n",
      "INFO\t2017-01-11 17:26:13\tGenotypeConcordance\tchecked    10,090,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:8,744,233\n",
      "INFO\t2017-01-11 17:26:14\tGenotypeConcordance\tchecked    10,100,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:9,551,780\n",
      "INFO\t2017-01-11 17:26:14\tGenotypeConcordance\tchecked    10,110,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:10,332,965\n",
      "INFO\t2017-01-11 17:26:14\tGenotypeConcordance\tchecked    10,120,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:11,152,355\n",
      "INFO\t2017-01-11 17:26:14\tGenotypeConcordance\tchecked    10,130,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:11,996,952\n",
      "INFO\t2017-01-11 17:26:15\tGenotypeConcordance\tchecked    10,140,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:12,884,007\n",
      "INFO\t2017-01-11 17:26:15\tGenotypeConcordance\tchecked    10,150,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:13,652,373\n",
      "INFO\t2017-01-11 17:26:15\tGenotypeConcordance\tchecked    10,160,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:14,483,612\n",
      "INFO\t2017-01-11 17:26:16\tGenotypeConcordance\tchecked    10,170,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:15,332,828\n",
      "INFO\t2017-01-11 17:26:16\tGenotypeConcordance\tchecked    10,180,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:16,297,063\n",
      "INFO\t2017-01-11 17:26:16\tGenotypeConcordance\tchecked    10,190,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:17,033,036\n",
      "INFO\t2017-01-11 17:26:16\tGenotypeConcordance\tchecked    10,200,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:17,886,282\n",
      "INFO\t2017-01-11 17:26:17\tGenotypeConcordance\tchecked    10,210,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:18,640,846\n",
      "INFO\t2017-01-11 17:26:17\tGenotypeConcordance\tchecked    10,220,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:19,470,753\n",
      "INFO\t2017-01-11 17:26:17\tGenotypeConcordance\tchecked    10,230,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:20,340,926\n",
      "INFO\t2017-01-11 17:26:18\tGenotypeConcordance\tchecked    10,240,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:21,188,773\n",
      "INFO\t2017-01-11 17:26:18\tGenotypeConcordance\tchecked    10,250,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:21,934,011\n",
      "INFO\t2017-01-11 17:26:18\tGenotypeConcordance\tchecked    10,260,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:22,796,407\n",
      "INFO\t2017-01-11 17:26:19\tGenotypeConcordance\tchecked    10,270,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:23,614,535\n",
      "INFO\t2017-01-11 17:26:19\tGenotypeConcordance\tchecked    10,280,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:24,402,069\n",
      "INFO\t2017-01-11 17:26:19\tGenotypeConcordance\tchecked    10,290,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:25,260,064\n",
      "INFO\t2017-01-11 17:26:19\tGenotypeConcordance\tchecked    10,300,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:26,005,870\n",
      "INFO\t2017-01-11 17:26:20\tGenotypeConcordance\tchecked    10,310,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:26,737,371\n",
      "INFO\t2017-01-11 17:26:20\tGenotypeConcordance\tchecked    10,320,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:27,561,458\n",
      "INFO\t2017-01-11 17:26:20\tGenotypeConcordance\tchecked    10,330,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:28,376,064\n",
      "INFO\t2017-01-11 17:26:21\tGenotypeConcordance\tchecked    10,340,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:29,221,051\n",
      "INFO\t2017-01-11 17:26:21\tGenotypeConcordance\tchecked    10,350,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:29,857,854\n",
      "INFO\t2017-01-11 17:26:21\tGenotypeConcordance\tchecked    10,360,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:30,635,630\n",
      "INFO\t2017-01-11 17:26:21\tGenotypeConcordance\tchecked    10,370,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:31,486,791\n",
      "INFO\t2017-01-11 17:26:22\tGenotypeConcordance\tchecked    10,380,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:32,247,129\n",
      "INFO\t2017-01-11 17:26:22\tGenotypeConcordance\tchecked    10,390,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:33,084,945\n",
      "INFO\t2017-01-11 17:26:22\tGenotypeConcordance\tchecked    10,400,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:33,869,728\n",
      "INFO\t2017-01-11 17:26:23\tGenotypeConcordance\tchecked    10,410,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:34,702,457\n",
      "INFO\t2017-01-11 17:26:23\tGenotypeConcordance\tchecked    10,420,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:35,540,135\n",
      "INFO\t2017-01-11 17:26:23\tGenotypeConcordance\tchecked    10,430,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:36,414,401\n",
      "INFO\t2017-01-11 17:26:24\tGenotypeConcordance\tchecked    10,440,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:37,442,135\n",
      "INFO\t2017-01-11 17:26:24\tGenotypeConcordance\tchecked    10,450,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:38,297,409\n",
      "INFO\t2017-01-11 17:26:24\tGenotypeConcordance\tchecked    10,460,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:39,155,400\n",
      "INFO\t2017-01-11 17:26:24\tGenotypeConcordance\tchecked    10,470,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:39,942,034\n",
      "INFO\t2017-01-11 17:26:25\tGenotypeConcordance\tchecked    10,480,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:40,734,183\n",
      "INFO\t2017-01-11 17:26:25\tGenotypeConcordance\tchecked    10,490,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:41,527,256\n",
      "INFO\t2017-01-11 17:26:25\tGenotypeConcordance\tchecked    10,500,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:42,356,099\n",
      "INFO\t2017-01-11 17:26:26\tGenotypeConcordance\tchecked    10,510,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:43,198,241\n",
      "INFO\t2017-01-11 17:26:26\tGenotypeConcordance\tchecked    10,520,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:44,041,626\n",
      "INFO\t2017-01-11 17:26:26\tGenotypeConcordance\tchecked    10,530,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:44,986,828\n",
      "INFO\t2017-01-11 17:26:26\tGenotypeConcordance\tchecked    10,540,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:45,891,794\n",
      "INFO\t2017-01-11 17:26:27\tGenotypeConcordance\tchecked    10,550,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:46,363,228\n",
      "INFO\t2017-01-11 17:26:27\tGenotypeConcordance\tchecked    10,560,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:49,878,993\n",
      "INFO\t2017-01-11 17:26:27\tGenotypeConcordance\tchecked    10,570,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:50,776,123\n",
      "INFO\t2017-01-11 17:26:27\tGenotypeConcordance\tchecked    10,580,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:51,592,067\n",
      "INFO\t2017-01-11 17:26:28\tGenotypeConcordance\tchecked    10,590,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:52,430,327\n",
      "INFO\t2017-01-11 17:26:28\tGenotypeConcordance\tchecked    10,600,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:53,268,199\n",
      "INFO\t2017-01-11 17:26:28\tGenotypeConcordance\tchecked    10,610,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:54,083,713\n",
      "INFO\t2017-01-11 17:26:29\tGenotypeConcordance\tchecked    10,620,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:54,969,713\n",
      "INFO\t2017-01-11 17:26:29\tGenotypeConcordance\tchecked    10,630,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:55,801,586\n",
      "INFO\t2017-01-11 17:26:29\tGenotypeConcordance\tchecked    10,640,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:56,717,667\n",
      "INFO\t2017-01-11 17:26:29\tGenotypeConcordance\tchecked    10,650,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:57,576,673\n",
      "INFO\t2017-01-11 17:26:30\tGenotypeConcordance\tchecked    10,660,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:58,424,097\n",
      "INFO\t2017-01-11 17:26:30\tGenotypeConcordance\tchecked    10,670,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:59,399,042\n",
      "INFO\t2017-01-11 17:26:30\tGenotypeConcordance\tchecked    10,680,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:60,467,949\n",
      "INFO\t2017-01-11 17:26:31\tGenotypeConcordance\tchecked    10,690,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:61,486,338\n",
      "INFO\t2017-01-11 17:26:31\tGenotypeConcordance\tchecked    10,700,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:62,380,641\n",
      "INFO\t2017-01-11 17:26:31\tGenotypeConcordance\tchecked    10,710,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:63,252,049\n",
      "INFO\t2017-01-11 17:26:31\tGenotypeConcordance\tchecked    10,720,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:64,263,903\n",
      "INFO\t2017-01-11 17:26:32\tGenotypeConcordance\tchecked    10,730,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:65,223,029\n",
      "INFO\t2017-01-11 17:26:32\tGenotypeConcordance\tchecked    10,740,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:66,081,464\n",
      "INFO\t2017-01-11 17:26:32\tGenotypeConcordance\tchecked    10,750,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:66,990,048\n",
      "INFO\t2017-01-11 17:26:33\tGenotypeConcordance\tchecked    10,760,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:67,866,560\n",
      "INFO\t2017-01-11 17:26:33\tGenotypeConcordance\tchecked    10,770,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:68,829,079\n",
      "INFO\t2017-01-11 17:26:33\tGenotypeConcordance\tchecked    10,780,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:71,324,354\n",
      "INFO\t2017-01-11 17:26:33\tGenotypeConcordance\tchecked    10,790,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:72,356,452\n",
      "INFO\t2017-01-11 17:26:34\tGenotypeConcordance\tchecked    10,800,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:73,255,812\n",
      "INFO\t2017-01-11 17:26:34\tGenotypeConcordance\tchecked    10,810,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:74,074,589\n",
      "INFO\t2017-01-11 17:26:34\tGenotypeConcordance\tchecked    10,820,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:75,057,430\n",
      "INFO\t2017-01-11 17:26:34\tGenotypeConcordance\tchecked    10,830,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:75,792,550\n",
      "INFO\t2017-01-11 17:26:35\tGenotypeConcordance\tchecked    10,840,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:76,665,184\n",
      "INFO\t2017-01-11 17:26:35\tGenotypeConcordance\tchecked    10,850,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:77,570,082\n",
      "INFO\t2017-01-11 17:26:35\tGenotypeConcordance\tchecked    10,860,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:78,435,578\n",
      "INFO\t2017-01-11 17:26:36\tGenotypeConcordance\tchecked    10,870,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:79,384,952\n",
      "INFO\t2017-01-11 17:26:36\tGenotypeConcordance\tchecked    10,880,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 5:80,183,772\n",
      "INFO\t2017-01-11 17:26:36\tGenotypeConcordance\tchecked    10,890,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 5:81,119,661\n",
      "INFO\t2017-01-11 17:26:37\tGenotypeConcordance\tchecked    10,900,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 5:82,100,752\n",
      "INFO\t2017-01-11 17:26:37\tGenotypeConcordance\tchecked    10,910,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 5:82,988,485\n",
      "INFO\t2017-01-11 17:26:37\tGenotypeConcordance\tchecked    10,920,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 5:83,977,669\n",
      "INFO\t2017-01-11 17:26:37\tGenotypeConcordance\tchecked    10,930,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 5:84,755,439\n",
      "INFO\t2017-01-11 17:26:38\tGenotypeConcordance\tchecked    10,940,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 5:85,562,014\n",
      "INFO\t2017-01-11 17:26:38\tGenotypeConcordance\tchecked    10,950,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 5:86,454,768\n",
      "INFO\t2017-01-11 17:26:38\tGenotypeConcordance\tchecked    10,960,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 5:87,609,719\n",
      "INFO\t2017-01-11 17:26:38\tGenotypeConcordance\tchecked    10,970,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 5:88,875,426\n",
      "INFO\t2017-01-11 17:26:39\tGenotypeConcordance\tchecked    10,980,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:89,930,666\n",
      "INFO\t2017-01-11 17:26:39\tGenotypeConcordance\tchecked    10,990,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:90,893,636\n",
      "INFO\t2017-01-11 17:26:39\tGenotypeConcordance\tchecked    11,000,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:92,005,876\n",
      "INFO\t2017-01-11 17:26:40\tGenotypeConcordance\tchecked    11,010,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:93,183,020\n",
      "INFO\t2017-01-11 17:26:40\tGenotypeConcordance\tchecked    11,020,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:94,366,450\n",
      "INFO\t2017-01-11 17:26:40\tGenotypeConcordance\tchecked    11,030,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:95,348,201\n",
      "INFO\t2017-01-11 17:26:40\tGenotypeConcordance\tchecked    11,040,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:96,243,650\n",
      "INFO\t2017-01-11 17:26:41\tGenotypeConcordance\tchecked    11,050,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:97,060,608\n",
      "INFO\t2017-01-11 17:26:41\tGenotypeConcordance\tchecked    11,060,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:97,797,639\n",
      "INFO\t2017-01-11 17:26:41\tGenotypeConcordance\tchecked    11,070,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:98,679,017\n",
      "INFO\t2017-01-11 17:26:42\tGenotypeConcordance\tchecked    11,080,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:99,450,876\n",
      "INFO\t2017-01-11 17:26:42\tGenotypeConcordance\tchecked    11,090,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:100,271,134\n",
      "INFO\t2017-01-11 17:26:42\tGenotypeConcordance\tchecked    11,100,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:101,117,693\n",
      "INFO\t2017-01-11 17:26:42\tGenotypeConcordance\tchecked    11,110,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:101,923,665\n",
      "INFO\t2017-01-11 17:26:43\tGenotypeConcordance\tchecked    11,120,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:102,855,869\n",
      "INFO\t2017-01-11 17:26:43\tGenotypeConcordance\tchecked    11,130,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:103,673,009\n",
      "INFO\t2017-01-11 17:26:43\tGenotypeConcordance\tchecked    11,140,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:104,490,778\n",
      "INFO\t2017-01-11 17:26:44\tGenotypeConcordance\tchecked    11,150,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:105,387,654\n",
      "INFO\t2017-01-11 17:26:44\tGenotypeConcordance\tchecked    11,160,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 5:106,232,456\n",
      "INFO\t2017-01-11 17:26:44\tGenotypeConcordance\tchecked    11,170,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 5:107,063,746\n",
      "INFO\t2017-01-11 17:26:45\tGenotypeConcordance\tchecked    11,180,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 5:107,893,632\n",
      "INFO\t2017-01-11 17:26:45\tGenotypeConcordance\tchecked    11,190,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:108,656,067\n",
      "INFO\t2017-01-11 17:26:45\tGenotypeConcordance\tchecked    11,200,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:109,449,361\n",
      "INFO\t2017-01-11 17:26:45\tGenotypeConcordance\tchecked    11,210,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:110,188,988\n",
      "INFO\t2017-01-11 17:26:46\tGenotypeConcordance\tchecked    11,220,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:110,982,398\n",
      "INFO\t2017-01-11 17:26:46\tGenotypeConcordance\tchecked    11,230,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 5:111,799,267\n",
      "INFO\t2017-01-11 17:26:47\tGenotypeConcordance\tchecked    11,240,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 5:112,500,404\n",
      "INFO\t2017-01-11 17:26:47\tGenotypeConcordance\tchecked    11,250,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:113,282,113\n",
      "INFO\t2017-01-11 17:26:47\tGenotypeConcordance\tchecked    11,260,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:114,043,417\n",
      "INFO\t2017-01-11 17:26:47\tGenotypeConcordance\tchecked    11,270,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:114,747,126\n",
      "INFO\t2017-01-11 17:26:48\tGenotypeConcordance\tchecked    11,280,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:115,475,502\n",
      "INFO\t2017-01-11 17:26:48\tGenotypeConcordance\tchecked    11,290,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:116,128,151\n",
      "INFO\t2017-01-11 17:26:48\tGenotypeConcordance\tchecked    11,300,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:116,860,242\n",
      "INFO\t2017-01-11 17:26:49\tGenotypeConcordance\tchecked    11,310,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:117,586,651\n",
      "INFO\t2017-01-11 17:26:49\tGenotypeConcordance\tchecked    11,320,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:118,268,018\n",
      "INFO\t2017-01-11 17:26:49\tGenotypeConcordance\tchecked    11,330,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:119,055,044\n",
      "INFO\t2017-01-11 17:26:50\tGenotypeConcordance\tchecked    11,340,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:119,804,529\n",
      "INFO\t2017-01-11 17:26:50\tGenotypeConcordance\tchecked    11,350,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:120,537,042\n",
      "INFO\t2017-01-11 17:26:50\tGenotypeConcordance\tchecked    11,360,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:121,238,944\n",
      "INFO\t2017-01-11 17:26:50\tGenotypeConcordance\tchecked    11,370,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:122,087,919\n",
      "INFO\t2017-01-11 17:26:51\tGenotypeConcordance\tchecked    11,380,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:122,999,952\n",
      "INFO\t2017-01-11 17:26:51\tGenotypeConcordance\tchecked    11,390,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:123,816,401\n",
      "INFO\t2017-01-11 17:26:51\tGenotypeConcordance\tchecked    11,400,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:124,590,123\n",
      "INFO\t2017-01-11 17:26:52\tGenotypeConcordance\tchecked    11,410,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:125,350,827\n",
      "INFO\t2017-01-11 17:26:52\tGenotypeConcordance\tchecked    11,420,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:126,106,785\n",
      "INFO\t2017-01-11 17:26:52\tGenotypeConcordance\tchecked    11,430,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:126,973,564\n",
      "INFO\t2017-01-11 17:26:52\tGenotypeConcordance\tchecked    11,440,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:127,962,811\n",
      "INFO\t2017-01-11 17:26:53\tGenotypeConcordance\tchecked    11,450,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:128,755,827\n",
      "INFO\t2017-01-11 17:26:53\tGenotypeConcordance\tchecked    11,460,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:129,618,090\n",
      "INFO\t2017-01-11 17:26:53\tGenotypeConcordance\tchecked    11,470,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:130,538,004\n",
      "INFO\t2017-01-11 17:26:54\tGenotypeConcordance\tchecked    11,480,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:131,488,880\n",
      "INFO\t2017-01-11 17:26:54\tGenotypeConcordance\tchecked    11,490,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:132,444,721\n",
      "INFO\t2017-01-11 17:26:54\tGenotypeConcordance\tchecked    11,500,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:133,310,944\n",
      "INFO\t2017-01-11 17:26:54\tGenotypeConcordance\tchecked    11,510,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:134,343,130\n",
      "INFO\t2017-01-11 17:26:55\tGenotypeConcordance\tchecked    11,520,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:135,213,719\n",
      "INFO\t2017-01-11 17:26:55\tGenotypeConcordance\tchecked    11,530,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:136,146,592\n",
      "INFO\t2017-01-11 17:26:55\tGenotypeConcordance\tchecked    11,540,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:137,147,168\n",
      "INFO\t2017-01-11 17:26:55\tGenotypeConcordance\tchecked    11,550,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:138,262,670\n",
      "INFO\t2017-01-11 17:26:56\tGenotypeConcordance\tchecked    11,560,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:139,523,692\n",
      "INFO\t2017-01-11 17:26:56\tGenotypeConcordance\tchecked    11,570,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:140,578,250\n",
      "INFO\t2017-01-11 17:26:56\tGenotypeConcordance\tchecked    11,580,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:141,506,555\n",
      "INFO\t2017-01-11 17:26:57\tGenotypeConcordance\tchecked    11,590,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:142,349,030\n",
      "INFO\t2017-01-11 17:26:57\tGenotypeConcordance\tchecked    11,600,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:143,268,544\n",
      "INFO\t2017-01-11 17:26:57\tGenotypeConcordance\tchecked    11,610,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:144,098,567\n",
      "INFO\t2017-01-11 17:26:58\tGenotypeConcordance\tchecked    11,620,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:144,968,443\n",
      "INFO\t2017-01-11 17:26:58\tGenotypeConcordance\tchecked    11,630,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:145,792,218\n",
      "INFO\t2017-01-11 17:26:58\tGenotypeConcordance\tchecked    11,640,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:146,672,934\n",
      "INFO\t2017-01-11 17:26:58\tGenotypeConcordance\tchecked    11,650,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:147,516,195\n",
      "INFO\t2017-01-11 17:26:59\tGenotypeConcordance\tchecked    11,660,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:148,382,327\n",
      "INFO\t2017-01-11 17:26:59\tGenotypeConcordance\tchecked    11,670,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:149,254,197\n",
      "INFO\t2017-01-11 17:26:59\tGenotypeConcordance\tchecked    11,680,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:150,126,266\n",
      "INFO\t2017-01-11 17:27:00\tGenotypeConcordance\tchecked    11,690,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:150,904,341\n",
      "INFO\t2017-01-11 17:27:00\tGenotypeConcordance\tchecked    11,700,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:151,776,870\n",
      "INFO\t2017-01-11 17:27:00\tGenotypeConcordance\tchecked    11,710,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:152,662,803\n",
      "INFO\t2017-01-11 17:27:01\tGenotypeConcordance\tchecked    11,720,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:153,584,135\n",
      "INFO\t2017-01-11 17:27:01\tGenotypeConcordance\tchecked    11,730,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:154,491,541\n",
      "INFO\t2017-01-11 17:27:01\tGenotypeConcordance\tchecked    11,740,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:155,395,740\n",
      "INFO\t2017-01-11 17:27:01\tGenotypeConcordance\tchecked    11,750,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:156,258,787\n",
      "INFO\t2017-01-11 17:27:02\tGenotypeConcordance\tchecked    11,760,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:157,082,025\n",
      "INFO\t2017-01-11 17:27:02\tGenotypeConcordance\tchecked    11,770,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:157,906,224\n",
      "INFO\t2017-01-11 17:27:02\tGenotypeConcordance\tchecked    11,780,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:158,930,967\n",
      "INFO\t2017-01-11 17:27:02\tGenotypeConcordance\tchecked    11,790,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:159,764,701\n",
      "INFO\t2017-01-11 17:27:03\tGenotypeConcordance\tchecked    11,800,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:160,558,193\n",
      "INFO\t2017-01-11 17:27:03\tGenotypeConcordance\tchecked    11,810,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:161,447,890\n",
      "INFO\t2017-01-11 17:27:03\tGenotypeConcordance\tchecked    11,820,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:162,290,454\n",
      "INFO\t2017-01-11 17:27:04\tGenotypeConcordance\tchecked    11,830,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:163,154,418\n",
      "INFO\t2017-01-11 17:27:04\tGenotypeConcordance\tchecked    11,840,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:164,060,480\n",
      "INFO\t2017-01-11 17:27:04\tGenotypeConcordance\tchecked    11,850,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:164,962,496\n",
      "INFO\t2017-01-11 17:27:04\tGenotypeConcordance\tchecked    11,860,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:165,828,151\n",
      "INFO\t2017-01-11 17:27:05\tGenotypeConcordance\tchecked    11,870,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:166,698,017\n",
      "INFO\t2017-01-11 17:27:05\tGenotypeConcordance\tchecked    11,880,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:167,624,491\n",
      "INFO\t2017-01-11 17:27:05\tGenotypeConcordance\tchecked    11,890,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:168,444,499\n",
      "INFO\t2017-01-11 17:27:06\tGenotypeConcordance\tchecked    11,900,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:169,285,912\n",
      "INFO\t2017-01-11 17:27:06\tGenotypeConcordance\tchecked    11,910,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:170,131,207\n",
      "INFO\t2017-01-11 17:27:06\tGenotypeConcordance\tchecked    11,920,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:170,979,510\n",
      "INFO\t2017-01-11 17:27:06\tGenotypeConcordance\tchecked    11,930,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:171,797,320\n",
      "INFO\t2017-01-11 17:27:07\tGenotypeConcordance\tchecked    11,940,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:172,625,758\n",
      "INFO\t2017-01-11 17:27:07\tGenotypeConcordance\tchecked    11,950,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:173,447,445\n",
      "INFO\t2017-01-11 17:27:07\tGenotypeConcordance\tchecked    11,960,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:174,222,149\n",
      "INFO\t2017-01-11 17:27:07\tGenotypeConcordance\tchecked    11,970,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:174,974,131\n",
      "INFO\t2017-01-11 17:27:08\tGenotypeConcordance\tchecked    11,980,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:175,879,296\n",
      "INFO\t2017-01-11 17:27:08\tGenotypeConcordance\tchecked    11,990,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:176,742,298\n",
      "INFO\t2017-01-11 17:27:08\tGenotypeConcordance\tchecked    12,000,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:177,618,851\n",
      "INFO\t2017-01-11 17:27:09\tGenotypeConcordance\tchecked    12,010,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:178,260,360\n",
      "INFO\t2017-01-11 17:27:09\tGenotypeConcordance\tchecked    12,020,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:178,906,447\n",
      "INFO\t2017-01-11 17:27:09\tGenotypeConcordance\tchecked    12,030,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:179,670,271\n",
      "INFO\t2017-01-11 17:27:09\tGenotypeConcordance\tchecked    12,040,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:180,340,541\n",
      "INFO\t2017-01-11 17:27:10\tGenotypeConcordance\tchecked    12,050,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:471,112\n",
      "INFO\t2017-01-11 17:27:10\tGenotypeConcordance\tchecked    12,060,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:1,216,201\n",
      "INFO\t2017-01-11 17:27:10\tGenotypeConcordance\tchecked    12,070,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:1,981,340\n",
      "INFO\t2017-01-11 17:27:11\tGenotypeConcordance\tchecked    12,080,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:2,796,358\n",
      "INFO\t2017-01-11 17:27:11\tGenotypeConcordance\tchecked    12,090,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:3,464,174\n",
      "INFO\t2017-01-11 17:27:11\tGenotypeConcordance\tchecked    12,100,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:4,194,676\n",
      "INFO\t2017-01-11 17:27:11\tGenotypeConcordance\tchecked    12,110,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:4,932,311\n",
      "INFO\t2017-01-11 17:27:12\tGenotypeConcordance\tchecked    12,120,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:5,684,333\n",
      "INFO\t2017-01-11 17:27:12\tGenotypeConcordance\tchecked    12,130,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:6,454,886\n",
      "INFO\t2017-01-11 17:27:12\tGenotypeConcordance\tchecked    12,140,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:7,153,646\n",
      "INFO\t2017-01-11 17:27:13\tGenotypeConcordance\tchecked    12,150,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:7,938,293\n",
      "INFO\t2017-01-11 17:27:13\tGenotypeConcordance\tchecked    12,160,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:8,668,730\n",
      "INFO\t2017-01-11 17:27:13\tGenotypeConcordance\tchecked    12,170,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:9,507,852\n",
      "INFO\t2017-01-11 17:27:13\tGenotypeConcordance\tchecked    12,180,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:10,343,891\n",
      "INFO\t2017-01-11 17:27:14\tGenotypeConcordance\tchecked    12,190,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:11,086,696\n",
      "INFO\t2017-01-11 17:27:14\tGenotypeConcordance\tchecked    12,200,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:11,900,987\n",
      "INFO\t2017-01-11 17:27:14\tGenotypeConcordance\tchecked    12,210,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:12,756,394\n",
      "INFO\t2017-01-11 17:27:15\tGenotypeConcordance\tchecked    12,220,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:13,609,053\n",
      "INFO\t2017-01-11 17:27:15\tGenotypeConcordance\tchecked    12,230,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:14,445,739\n",
      "INFO\t2017-01-11 17:27:15\tGenotypeConcordance\tchecked    12,240,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:15,240,194\n",
      "INFO\t2017-01-11 17:27:15\tGenotypeConcordance\tchecked    12,250,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:16,087,931\n",
      "INFO\t2017-01-11 17:27:16\tGenotypeConcordance\tchecked    12,260,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:16,966,904\n",
      "INFO\t2017-01-11 17:27:16\tGenotypeConcordance\tchecked    12,270,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:17,801,284\n",
      "INFO\t2017-01-11 17:27:16\tGenotypeConcordance\tchecked    12,280,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:18,601,148\n",
      "INFO\t2017-01-11 17:27:17\tGenotypeConcordance\tchecked    12,290,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:19,431,136\n",
      "INFO\t2017-01-11 17:27:17\tGenotypeConcordance\tchecked    12,300,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:20,212,662\n",
      "INFO\t2017-01-11 17:27:17\tGenotypeConcordance\tchecked    12,310,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:21,045,170\n",
      "INFO\t2017-01-11 17:27:17\tGenotypeConcordance\tchecked    12,320,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:21,904,782\n",
      "INFO\t2017-01-11 17:27:18\tGenotypeConcordance\tchecked    12,330,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:22,668,647\n",
      "INFO\t2017-01-11 17:27:18\tGenotypeConcordance\tchecked    12,340,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:23,489,057\n",
      "INFO\t2017-01-11 17:27:18\tGenotypeConcordance\tchecked    12,350,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:24,097,567\n",
      "INFO\t2017-01-11 17:27:19\tGenotypeConcordance\tchecked    12,360,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:24,872,685\n",
      "INFO\t2017-01-11 17:27:19\tGenotypeConcordance\tchecked    12,370,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:25,649,715\n",
      "INFO\t2017-01-11 17:27:19\tGenotypeConcordance\tchecked    12,380,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:26,534,616\n",
      "INFO\t2017-01-11 17:27:19\tGenotypeConcordance\tchecked    12,390,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:27,470,810\n",
      "INFO\t2017-01-11 17:27:20\tGenotypeConcordance\tchecked    12,400,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:28,462,083\n",
      "INFO\t2017-01-11 17:27:20\tGenotypeConcordance\tchecked    12,410,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:29,286,670\n",
      "INFO\t2017-01-11 17:27:20\tGenotypeConcordance\tchecked    12,420,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:29,811,022\n",
      "INFO\t2017-01-11 17:27:21\tGenotypeConcordance\tchecked    12,430,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:30,044,935\n",
      "INFO\t2017-01-11 17:27:21\tGenotypeConcordance\tchecked    12,440,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:30,757,813\n",
      "INFO\t2017-01-11 17:27:21\tGenotypeConcordance\tchecked    12,450,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:31,234,693\n",
      "INFO\t2017-01-11 17:27:22\tGenotypeConcordance\tchecked    12,460,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:31,480,632\n",
      "INFO\t2017-01-11 17:27:22\tGenotypeConcordance\tchecked    12,470,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:32,360,464\n",
      "INFO\t2017-01-11 17:27:22\tGenotypeConcordance\tchecked    12,480,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:32,505,266\n",
      "INFO\t2017-01-11 17:27:23\tGenotypeConcordance\tchecked    12,490,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:32,589,327\n",
      "INFO\t2017-01-11 17:27:23\tGenotypeConcordance\tchecked    12,500,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:32,728,842\n",
      "INFO\t2017-01-11 17:27:23\tGenotypeConcordance\tchecked    12,510,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:33,226,633\n",
      "INFO\t2017-01-11 17:27:24\tGenotypeConcordance\tchecked    12,520,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:33,970,014\n",
      "INFO\t2017-01-11 17:27:24\tGenotypeConcordance\tchecked    12,530,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:34,715,186\n",
      "INFO\t2017-01-11 17:27:24\tGenotypeConcordance\tchecked    12,540,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:35,664,772\n",
      "INFO\t2017-01-11 17:27:25\tGenotypeConcordance\tchecked    12,550,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:36,544,329\n",
      "INFO\t2017-01-11 17:27:25\tGenotypeConcordance\tchecked    12,560,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:37,320,844\n",
      "INFO\t2017-01-11 17:27:25\tGenotypeConcordance\tchecked    12,570,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:38,169,864\n",
      "INFO\t2017-01-11 17:27:25\tGenotypeConcordance\tchecked    12,580,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:39,033,063\n",
      "INFO\t2017-01-11 17:27:26\tGenotypeConcordance\tchecked    12,590,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:39,853,946\n",
      "INFO\t2017-01-11 17:27:26\tGenotypeConcordance\tchecked    12,600,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:40,640,259\n",
      "INFO\t2017-01-11 17:27:26\tGenotypeConcordance\tchecked    12,610,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:41,449,935\n",
      "INFO\t2017-01-11 17:27:26\tGenotypeConcordance\tchecked    12,620,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:42,247,863\n",
      "INFO\t2017-01-11 17:27:27\tGenotypeConcordance\tchecked    12,630,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:43,186,138\n",
      "INFO\t2017-01-11 17:27:27\tGenotypeConcordance\tchecked    12,640,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:44,098,694\n",
      "INFO\t2017-01-11 17:27:27\tGenotypeConcordance\tchecked    12,650,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:44,909,518\n",
      "INFO\t2017-01-11 17:27:28\tGenotypeConcordance\tchecked    12,660,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:45,768,698\n",
      "INFO\t2017-01-11 17:27:28\tGenotypeConcordance\tchecked    12,670,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 6:46,675,025\n",
      "INFO\t2017-01-11 17:27:28\tGenotypeConcordance\tchecked    12,680,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 6:47,479,657\n",
      "INFO\t2017-01-11 17:27:28\tGenotypeConcordance\tchecked    12,690,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 6:48,305,445\n",
      "INFO\t2017-01-11 17:27:29\tGenotypeConcordance\tchecked    12,700,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 6:49,088,350\n",
      "INFO\t2017-01-11 17:27:29\tGenotypeConcordance\tchecked    12,710,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 6:49,927,623\n",
      "INFO\t2017-01-11 17:27:29\tGenotypeConcordance\tchecked    12,720,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 6:50,844,870\n",
      "INFO\t2017-01-11 17:27:30\tGenotypeConcordance\tchecked    12,730,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 6:51,746,771\n",
      "INFO\t2017-01-11 17:27:30\tGenotypeConcordance\tchecked    12,740,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 6:52,615,360\n",
      "INFO\t2017-01-11 17:27:30\tGenotypeConcordance\tchecked    12,750,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 6:53,412,381\n",
      "INFO\t2017-01-11 17:27:30\tGenotypeConcordance\tchecked    12,760,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 6:54,258,078\n",
      "INFO\t2017-01-11 17:27:31\tGenotypeConcordance\tchecked    12,770,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 6:55,013,303\n",
      "INFO\t2017-01-11 17:27:31\tGenotypeConcordance\tchecked    12,780,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 6:55,786,412\n",
      "INFO\t2017-01-11 17:27:31\tGenotypeConcordance\tchecked    12,790,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 6:56,665,182\n",
      "INFO\t2017-01-11 17:27:32\tGenotypeConcordance\tchecked    12,800,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 6:57,371,946\n",
      "INFO\t2017-01-11 17:27:32\tGenotypeConcordance\tchecked    12,810,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 6:57,588,249\n",
      "INFO\t2017-01-11 17:27:32\tGenotypeConcordance\tchecked    12,820,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 6:58,370,815\n",
      "INFO\t2017-01-11 17:27:32\tGenotypeConcordance\tchecked    12,830,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 6:62,054,774\n",
      "INFO\t2017-01-11 17:27:33\tGenotypeConcordance\tchecked    12,840,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 6:62,838,574\n",
      "INFO\t2017-01-11 17:27:33\tGenotypeConcordance\tchecked    12,850,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 6:63,638,516\n",
      "INFO\t2017-01-11 17:27:33\tGenotypeConcordance\tchecked    12,860,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 6:64,544,537\n",
      "INFO\t2017-01-11 17:27:33\tGenotypeConcordance\tchecked    12,870,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 6:65,351,867\n",
      "INFO\t2017-01-11 17:27:34\tGenotypeConcordance\tchecked    12,880,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 6:66,133,980\n",
      "INFO\t2017-01-11 17:27:34\tGenotypeConcordance\tchecked    12,890,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 6:66,889,399\n",
      "INFO\t2017-01-11 17:27:34\tGenotypeConcordance\tchecked    12,900,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 6:67,550,407\n",
      "INFO\t2017-01-11 17:27:35\tGenotypeConcordance\tchecked    12,910,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 6:68,223,010\n",
      "INFO\t2017-01-11 17:27:35\tGenotypeConcordance\tchecked    12,920,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 6:69,012,763\n",
      "INFO\t2017-01-11 17:27:35\tGenotypeConcordance\tchecked    12,930,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 6:69,849,256\n",
      "INFO\t2017-01-11 17:27:35\tGenotypeConcordance\tchecked    12,940,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 6:70,734,005\n",
      "INFO\t2017-01-11 17:27:36\tGenotypeConcordance\tchecked    12,950,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 6:71,606,550\n",
      "INFO\t2017-01-11 17:27:36\tGenotypeConcordance\tchecked    12,960,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 6:72,399,994\n",
      "INFO\t2017-01-11 17:27:36\tGenotypeConcordance\tchecked    12,970,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 6:73,293,710\n",
      "INFO\t2017-01-11 17:27:37\tGenotypeConcordance\tchecked    12,980,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 6:74,213,881\n",
      "INFO\t2017-01-11 17:27:37\tGenotypeConcordance\tchecked    12,990,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 6:74,971,740\n",
      "INFO\t2017-01-11 17:27:37\tGenotypeConcordance\tchecked    13,000,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 6:75,761,832\n",
      "INFO\t2017-01-11 17:27:37\tGenotypeConcordance\tchecked    13,010,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 6:76,803,226\n",
      "INFO\t2017-01-11 17:27:38\tGenotypeConcordance\tchecked    13,020,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 6:77,503,299\n",
      "INFO\t2017-01-11 17:27:38\tGenotypeConcordance\tchecked    13,030,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 6:78,320,930\n",
      "INFO\t2017-01-11 17:27:38\tGenotypeConcordance\tchecked    13,040,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 6:79,068,557\n",
      "INFO\t2017-01-11 17:27:38\tGenotypeConcordance\tchecked    13,050,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 6:79,779,767\n",
      "INFO\t2017-01-11 17:27:39\tGenotypeConcordance\tchecked    13,060,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 6:80,635,130\n",
      "INFO\t2017-01-11 17:27:39\tGenotypeConcordance\tchecked    13,070,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 6:81,409,273\n",
      "INFO\t2017-01-11 17:27:39\tGenotypeConcordance\tchecked    13,080,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 6:82,095,610\n",
      "INFO\t2017-01-11 17:27:40\tGenotypeConcordance\tchecked    13,090,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 6:82,957,593\n",
      "INFO\t2017-01-11 17:27:40\tGenotypeConcordance\tchecked    13,100,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 6:83,896,510\n",
      "INFO\t2017-01-11 17:27:40\tGenotypeConcordance\tchecked    13,110,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 6:84,871,065\n",
      "INFO\t2017-01-11 17:27:40\tGenotypeConcordance\tchecked    13,120,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 6:85,734,809\n",
      "INFO\t2017-01-11 17:27:41\tGenotypeConcordance\tchecked    13,130,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 6:86,585,114\n",
      "INFO\t2017-01-11 17:27:41\tGenotypeConcordance\tchecked    13,140,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 6:87,317,765\n",
      "INFO\t2017-01-11 17:27:41\tGenotypeConcordance\tchecked    13,150,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 6:88,120,947\n",
      "INFO\t2017-01-11 17:27:42\tGenotypeConcordance\tchecked    13,160,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 6:89,002,133\n",
      "INFO\t2017-01-11 17:27:42\tGenotypeConcordance\tchecked    13,170,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 6:89,839,906\n",
      "INFO\t2017-01-11 17:27:42\tGenotypeConcordance\tchecked    13,180,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 6:90,755,352\n",
      "INFO\t2017-01-11 17:27:42\tGenotypeConcordance\tchecked    13,190,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 6:91,720,488\n",
      "INFO\t2017-01-11 17:27:43\tGenotypeConcordance\tchecked    13,200,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 6:92,584,877\n",
      "INFO\t2017-01-11 17:27:43\tGenotypeConcordance\tchecked    13,210,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 6:93,440,311\n",
      "INFO\t2017-01-11 17:27:43\tGenotypeConcordance\tchecked    13,220,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 6:94,258,257\n",
      "INFO\t2017-01-11 17:27:44\tGenotypeConcordance\tchecked    13,230,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:95,081,286\n",
      "INFO\t2017-01-11 17:27:44\tGenotypeConcordance\tchecked    13,240,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:96,019,311\n",
      "INFO\t2017-01-11 17:27:44\tGenotypeConcordance\tchecked    13,250,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:96,896,078\n",
      "INFO\t2017-01-11 17:27:45\tGenotypeConcordance\tchecked    13,260,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:97,869,464\n",
      "INFO\t2017-01-11 17:27:45\tGenotypeConcordance\tchecked    13,270,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:98,981,180\n",
      "INFO\t2017-01-11 17:27:45\tGenotypeConcordance\tchecked    13,280,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:99,984,803\n",
      "INFO\t2017-01-11 17:27:45\tGenotypeConcordance\tchecked    13,290,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:100,886,666\n",
      "INFO\t2017-01-11 17:27:46\tGenotypeConcordance\tchecked    13,300,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:101,813,169\n",
      "INFO\t2017-01-11 17:27:46\tGenotypeConcordance\tchecked    13,310,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:102,654,831\n",
      "INFO\t2017-01-11 17:27:46\tGenotypeConcordance\tchecked    13,320,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:103,468,653\n",
      "INFO\t2017-01-11 17:27:46\tGenotypeConcordance\tchecked    13,330,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:104,242,548\n",
      "INFO\t2017-01-11 17:27:47\tGenotypeConcordance\tchecked    13,340,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:105,116,200\n",
      "INFO\t2017-01-11 17:27:47\tGenotypeConcordance\tchecked    13,350,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:106,070,640\n",
      "INFO\t2017-01-11 17:27:47\tGenotypeConcordance\tchecked    13,360,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:106,972,698\n",
      "INFO\t2017-01-11 17:27:47\tGenotypeConcordance\tchecked    13,370,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:107,786,703\n",
      "INFO\t2017-01-11 17:27:48\tGenotypeConcordance\tchecked    13,380,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:108,721,321\n",
      "INFO\t2017-01-11 17:27:48\tGenotypeConcordance\tchecked    13,390,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:109,633,189\n",
      "INFO\t2017-01-11 17:27:48\tGenotypeConcordance\tchecked    13,400,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:110,571,347\n",
      "INFO\t2017-01-11 17:27:49\tGenotypeConcordance\tchecked    13,410,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:111,464,713\n",
      "INFO\t2017-01-11 17:27:49\tGenotypeConcordance\tchecked    13,420,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:112,501,468\n",
      "INFO\t2017-01-11 17:27:49\tGenotypeConcordance\tchecked    13,430,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:113,456,066\n",
      "INFO\t2017-01-11 17:27:49\tGenotypeConcordance\tchecked    13,440,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:114,326,693\n",
      "INFO\t2017-01-11 17:27:50\tGenotypeConcordance\tchecked    13,450,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:115,295,446\n",
      "INFO\t2017-01-11 17:27:50\tGenotypeConcordance\tchecked    13,460,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:116,100,131\n",
      "INFO\t2017-01-11 17:27:50\tGenotypeConcordance\tchecked    13,470,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:117,081,206\n",
      "INFO\t2017-01-11 17:27:51\tGenotypeConcordance\tchecked    13,480,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:118,004,817\n",
      "INFO\t2017-01-11 17:27:51\tGenotypeConcordance\tchecked    13,490,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:118,942,300\n",
      "INFO\t2017-01-11 17:27:51\tGenotypeConcordance\tchecked    13,500,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:119,829,445\n",
      "INFO\t2017-01-11 17:27:51\tGenotypeConcordance\tchecked    13,510,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:120,701,960\n",
      "INFO\t2017-01-11 17:27:52\tGenotypeConcordance\tchecked    13,520,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:121,490,824\n",
      "INFO\t2017-01-11 17:27:52\tGenotypeConcordance\tchecked    13,530,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:122,349,231\n",
      "INFO\t2017-01-11 17:27:52\tGenotypeConcordance\tchecked    13,540,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:123,265,483\n",
      "INFO\t2017-01-11 17:27:53\tGenotypeConcordance\tchecked    13,550,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:124,115,011\n",
      "INFO\t2017-01-11 17:27:53\tGenotypeConcordance\tchecked    13,560,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:124,972,844\n",
      "INFO\t2017-01-11 17:27:53\tGenotypeConcordance\tchecked    13,570,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:125,885,633\n",
      "INFO\t2017-01-11 17:27:54\tGenotypeConcordance\tchecked    13,580,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:127,043,239\n",
      "INFO\t2017-01-11 17:27:54\tGenotypeConcordance\tchecked    13,590,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:128,081,304\n",
      "INFO\t2017-01-11 17:27:54\tGenotypeConcordance\tchecked    13,600,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:129,067,102\n",
      "INFO\t2017-01-11 17:27:54\tGenotypeConcordance\tchecked    13,610,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:129,947,080\n",
      "INFO\t2017-01-11 17:27:55\tGenotypeConcordance\tchecked    13,620,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:130,779,148\n",
      "INFO\t2017-01-11 17:27:55\tGenotypeConcordance\tchecked    13,630,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:131,625,710\n",
      "INFO\t2017-01-11 17:27:55\tGenotypeConcordance\tchecked    13,640,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:132,485,007\n",
      "INFO\t2017-01-11 17:27:56\tGenotypeConcordance\tchecked    13,650,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:133,256,704\n",
      "INFO\t2017-01-11 17:27:56\tGenotypeConcordance\tchecked    13,660,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:134,153,208\n",
      "INFO\t2017-01-11 17:27:56\tGenotypeConcordance\tchecked    13,670,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:135,013,843\n",
      "INFO\t2017-01-11 17:27:56\tGenotypeConcordance\tchecked    13,680,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:135,935,894\n",
      "INFO\t2017-01-11 17:27:57\tGenotypeConcordance\tchecked    13,690,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:136,923,799\n",
      "INFO\t2017-01-11 17:27:57\tGenotypeConcordance\tchecked    13,700,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 6:137,801,790\n",
      "INFO\t2017-01-11 17:27:57\tGenotypeConcordance\tchecked    13,710,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 6:138,616,627\n",
      "INFO\t2017-01-11 17:27:57\tGenotypeConcordance\tchecked    13,720,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 6:139,414,505\n",
      "INFO\t2017-01-11 17:27:58\tGenotypeConcordance\tchecked    13,730,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:140,281,071\n",
      "INFO\t2017-01-11 17:27:58\tGenotypeConcordance\tchecked    13,740,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:141,397,909\n",
      "INFO\t2017-01-11 17:27:58\tGenotypeConcordance\tchecked    13,750,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:142,274,618\n",
      "INFO\t2017-01-11 17:27:59\tGenotypeConcordance\tchecked    13,760,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:143,170,566\n",
      "INFO\t2017-01-11 17:27:59\tGenotypeConcordance\tchecked    13,770,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:144,101,343\n",
      "INFO\t2017-01-11 17:27:59\tGenotypeConcordance\tchecked    13,780,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:145,007,723\n",
      "INFO\t2017-01-11 17:28:00\tGenotypeConcordance\tchecked    13,790,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:145,868,680\n",
      "INFO\t2017-01-11 17:28:00\tGenotypeConcordance\tchecked    13,800,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:146,881,529\n",
      "INFO\t2017-01-11 17:28:00\tGenotypeConcordance\tchecked    13,810,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:147,852,871\n",
      "INFO\t2017-01-11 17:28:00\tGenotypeConcordance\tchecked    13,820,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:148,620,015\n",
      "INFO\t2017-01-11 17:28:01\tGenotypeConcordance\tchecked    13,830,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:149,407,904\n",
      "INFO\t2017-01-11 17:28:01\tGenotypeConcordance\tchecked    13,840,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:150,269,249\n",
      "INFO\t2017-01-11 17:28:01\tGenotypeConcordance\tchecked    13,850,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:150,949,223\n",
      "INFO\t2017-01-11 17:28:01\tGenotypeConcordance\tchecked    13,860,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:151,665,650\n",
      "INFO\t2017-01-11 17:28:02\tGenotypeConcordance\tchecked    13,870,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:152,510,429\n",
      "INFO\t2017-01-11 17:28:02\tGenotypeConcordance\tchecked    13,880,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:153,351,923\n",
      "INFO\t2017-01-11 17:28:02\tGenotypeConcordance\tchecked    13,890,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:154,108,443\n",
      "INFO\t2017-01-11 17:28:02\tGenotypeConcordance\tchecked    13,900,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:154,853,986\n",
      "INFO\t2017-01-11 17:28:03\tGenotypeConcordance\tchecked    13,910,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:155,633,969\n",
      "INFO\t2017-01-11 17:28:03\tGenotypeConcordance\tchecked    13,920,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:156,463,383\n",
      "INFO\t2017-01-11 17:28:03\tGenotypeConcordance\tchecked    13,930,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:157,314,040\n",
      "INFO\t2017-01-11 17:28:04\tGenotypeConcordance\tchecked    13,940,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:158,239,938\n",
      "INFO\t2017-01-11 17:28:04\tGenotypeConcordance\tchecked    13,950,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:159,029,041\n",
      "INFO\t2017-01-11 17:28:04\tGenotypeConcordance\tchecked    13,960,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:159,805,647\n",
      "INFO\t2017-01-11 17:28:04\tGenotypeConcordance\tchecked    13,970,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:160,579,368\n",
      "INFO\t2017-01-11 17:28:05\tGenotypeConcordance\tchecked    13,980,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:161,354,194\n",
      "INFO\t2017-01-11 17:28:05\tGenotypeConcordance\tchecked    13,990,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:162,128,543\n",
      "INFO\t2017-01-11 17:28:05\tGenotypeConcordance\tchecked    14,000,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:162,832,167\n",
      "INFO\t2017-01-11 17:28:06\tGenotypeConcordance\tchecked    14,010,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:163,627,243\n",
      "INFO\t2017-01-11 17:28:06\tGenotypeConcordance\tchecked    14,020,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:164,409,147\n",
      "INFO\t2017-01-11 17:28:06\tGenotypeConcordance\tchecked    14,030,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:165,190,113\n",
      "INFO\t2017-01-11 17:28:06\tGenotypeConcordance\tchecked    14,040,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:165,944,280\n",
      "INFO\t2017-01-11 17:28:07\tGenotypeConcordance\tchecked    14,050,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:166,693,445\n",
      "INFO\t2017-01-11 17:28:07\tGenotypeConcordance\tchecked    14,060,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:167,378,619\n",
      "INFO\t2017-01-11 17:28:07\tGenotypeConcordance\tchecked    14,070,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:168,086,374\n",
      "INFO\t2017-01-11 17:28:08\tGenotypeConcordance\tchecked    14,080,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:168,746,649\n",
      "INFO\t2017-01-11 17:28:08\tGenotypeConcordance\tchecked    14,090,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:169,354,072\n",
      "INFO\t2017-01-11 17:28:08\tGenotypeConcordance\tchecked    14,100,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:170,046,595\n",
      "INFO\t2017-01-11 17:28:08\tGenotypeConcordance\tchecked    14,110,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:170,767,224\n",
      "INFO\t2017-01-11 17:28:09\tGenotypeConcordance\tchecked    14,120,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:416,852\n",
      "INFO\t2017-01-11 17:28:09\tGenotypeConcordance\tchecked    14,130,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:960,050\n",
      "INFO\t2017-01-11 17:28:09\tGenotypeConcordance\tchecked    14,140,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:1,514,209\n",
      "INFO\t2017-01-11 17:28:10\tGenotypeConcordance\tchecked    14,150,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:2,120,371\n",
      "INFO\t2017-01-11 17:28:10\tGenotypeConcordance\tchecked    14,160,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:2,745,489\n",
      "INFO\t2017-01-11 17:28:10\tGenotypeConcordance\tchecked    14,170,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:3,292,730\n",
      "INFO\t2017-01-11 17:28:10\tGenotypeConcordance\tchecked    14,180,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:3,856,970\n",
      "INFO\t2017-01-11 17:28:11\tGenotypeConcordance\tchecked    14,190,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:4,419,504\n",
      "INFO\t2017-01-11 17:28:11\tGenotypeConcordance\tchecked    14,200,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:4,986,942\n",
      "INFO\t2017-01-11 17:28:11\tGenotypeConcordance\tchecked    14,210,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:5,584,173\n",
      "INFO\t2017-01-11 17:28:12\tGenotypeConcordance\tchecked    14,220,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:6,254,993\n",
      "INFO\t2017-01-11 17:28:12\tGenotypeConcordance\tchecked    14,230,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:6,924,262\n",
      "INFO\t2017-01-11 17:28:12\tGenotypeConcordance\tchecked    14,240,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:7,442,180\n",
      "INFO\t2017-01-11 17:28:12\tGenotypeConcordance\tchecked    14,250,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:8,157,016\n",
      "INFO\t2017-01-11 17:28:13\tGenotypeConcordance\tchecked    14,260,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:8,827,383\n",
      "INFO\t2017-01-11 17:28:13\tGenotypeConcordance\tchecked    14,270,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:9,435,883\n",
      "INFO\t2017-01-11 17:28:13\tGenotypeConcordance\tchecked    14,280,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:10,073,714\n",
      "INFO\t2017-01-11 17:28:14\tGenotypeConcordance\tchecked    14,290,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:10,759,632\n",
      "INFO\t2017-01-11 17:28:14\tGenotypeConcordance\tchecked    14,300,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:11,517,514\n",
      "INFO\t2017-01-11 17:28:14\tGenotypeConcordance\tchecked    14,310,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:12,190,315\n",
      "INFO\t2017-01-11 17:28:14\tGenotypeConcordance\tchecked    14,320,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:12,799,881\n",
      "INFO\t2017-01-11 17:28:15\tGenotypeConcordance\tchecked    14,330,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:13,434,529\n",
      "INFO\t2017-01-11 17:28:15\tGenotypeConcordance\tchecked    14,340,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:14,125,432\n",
      "INFO\t2017-01-11 17:28:15\tGenotypeConcordance\tchecked    14,350,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:14,847,293\n",
      "INFO\t2017-01-11 17:28:15\tGenotypeConcordance\tchecked    14,360,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:15,501,400\n",
      "INFO\t2017-01-11 17:28:16\tGenotypeConcordance\tchecked    14,370,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 7:16,237,550\n",
      "INFO\t2017-01-11 17:28:16\tGenotypeConcordance\tchecked    14,380,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 7:16,956,718\n",
      "INFO\t2017-01-11 17:28:16\tGenotypeConcordance\tchecked    14,390,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 7:17,732,291\n",
      "INFO\t2017-01-11 17:28:16\tGenotypeConcordance\tchecked    14,400,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 7:18,627,588\n",
      "INFO\t2017-01-11 17:28:17\tGenotypeConcordance\tchecked    14,410,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 7:19,379,028\n",
      "INFO\t2017-01-11 17:28:17\tGenotypeConcordance\tchecked    14,420,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 7:20,038,602\n",
      "INFO\t2017-01-11 17:28:17\tGenotypeConcordance\tchecked    14,430,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 7:20,777,031\n",
      "INFO\t2017-01-11 17:28:18\tGenotypeConcordance\tchecked    14,440,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 7:21,582,964\n",
      "INFO\t2017-01-11 17:28:18\tGenotypeConcordance\tchecked    14,450,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 7:22,259,212\n",
      "INFO\t2017-01-11 17:28:18\tGenotypeConcordance\tchecked    14,460,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 7:23,070,387\n",
      "INFO\t2017-01-11 17:28:18\tGenotypeConcordance\tchecked    14,470,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 7:23,915,676\n",
      "INFO\t2017-01-11 17:28:19\tGenotypeConcordance\tchecked    14,480,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 7:24,700,907\n",
      "INFO\t2017-01-11 17:28:19\tGenotypeConcordance\tchecked    14,490,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 7:25,483,161\n",
      "INFO\t2017-01-11 17:28:19\tGenotypeConcordance\tchecked    14,500,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 7:26,381,207\n",
      "INFO\t2017-01-11 17:28:19\tGenotypeConcordance\tchecked    14,510,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 7:27,325,930\n",
      "INFO\t2017-01-11 17:28:20\tGenotypeConcordance\tchecked    14,520,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 7:28,190,727\n",
      "INFO\t2017-01-11 17:28:20\tGenotypeConcordance\tchecked    14,530,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 7:29,024,764\n",
      "INFO\t2017-01-11 17:28:20\tGenotypeConcordance\tchecked    14,540,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 7:29,837,093\n",
      "INFO\t2017-01-11 17:28:21\tGenotypeConcordance\tchecked    14,550,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 7:30,802,303\n",
      "INFO\t2017-01-11 17:28:21\tGenotypeConcordance\tchecked    14,560,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 7:31,654,462\n",
      "INFO\t2017-01-11 17:28:21\tGenotypeConcordance\tchecked    14,570,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 7:32,429,979\n",
      "INFO\t2017-01-11 17:28:22\tGenotypeConcordance\tchecked    14,580,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 7:33,219,952\n",
      "INFO\t2017-01-11 17:28:22\tGenotypeConcordance\tchecked    14,590,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 7:34,117,565\n",
      "INFO\t2017-01-11 17:28:22\tGenotypeConcordance\tchecked    14,600,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 7:34,992,376\n",
      "INFO\t2017-01-11 17:28:22\tGenotypeConcordance\tchecked    14,610,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 7:35,913,034\n",
      "INFO\t2017-01-11 17:28:23\tGenotypeConcordance\tchecked    14,620,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 7:36,742,034\n",
      "INFO\t2017-01-11 17:28:23\tGenotypeConcordance\tchecked    14,630,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 7:37,602,360\n",
      "INFO\t2017-01-11 17:28:23\tGenotypeConcordance\tchecked    14,640,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 7:38,385,712\n",
      "INFO\t2017-01-11 17:28:23\tGenotypeConcordance\tchecked    14,650,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 7:39,249,358\n",
      "INFO\t2017-01-11 17:28:24\tGenotypeConcordance\tchecked    14,660,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 7:40,150,910\n",
      "INFO\t2017-01-11 17:28:24\tGenotypeConcordance\tchecked    14,670,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 7:41,139,004\n",
      "INFO\t2017-01-11 17:28:24\tGenotypeConcordance\tchecked    14,680,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 7:42,001,171\n",
      "INFO\t2017-01-11 17:28:25\tGenotypeConcordance\tchecked    14,690,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 7:42,843,325\n",
      "INFO\t2017-01-11 17:28:25\tGenotypeConcordance\tchecked    14,700,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 7:43,703,622\n",
      "INFO\t2017-01-11 17:28:25\tGenotypeConcordance\tchecked    14,710,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 7:44,664,806\n",
      "INFO\t2017-01-11 17:28:25\tGenotypeConcordance\tchecked    14,720,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 7:45,516,136\n",
      "INFO\t2017-01-11 17:28:26\tGenotypeConcordance\tchecked    14,730,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 7:46,323,938\n",
      "INFO\t2017-01-11 17:28:26\tGenotypeConcordance\tchecked    14,740,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 7:47,144,689\n",
      "INFO\t2017-01-11 17:28:26\tGenotypeConcordance\tchecked    14,750,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 7:47,915,917\n",
      "INFO\t2017-01-11 17:28:27\tGenotypeConcordance\tchecked    14,760,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 7:48,772,811\n",
      "INFO\t2017-01-11 17:28:27\tGenotypeConcordance\tchecked    14,770,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 7:49,547,057\n",
      "INFO\t2017-01-11 17:28:27\tGenotypeConcordance\tchecked    14,780,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 7:50,478,573\n",
      "INFO\t2017-01-11 17:28:27\tGenotypeConcordance\tchecked    14,790,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 7:51,332,935\n",
      "INFO\t2017-01-11 17:28:28\tGenotypeConcordance\tchecked    14,800,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 7:52,054,153\n",
      "INFO\t2017-01-11 17:28:28\tGenotypeConcordance\tchecked    14,810,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 7:52,737,397\n",
      "INFO\t2017-01-11 17:28:28\tGenotypeConcordance\tchecked    14,820,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 7:53,356,363\n",
      "INFO\t2017-01-11 17:28:29\tGenotypeConcordance\tchecked    14,830,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 7:54,109,581\n",
      "INFO\t2017-01-11 17:28:29\tGenotypeConcordance\tchecked    14,840,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 7:54,792,121\n",
      "INFO\t2017-01-11 17:28:29\tGenotypeConcordance\tchecked    14,850,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 7:55,595,270\n",
      "INFO\t2017-01-11 17:28:29\tGenotypeConcordance\tchecked    14,860,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 7:56,389,623\n",
      "INFO\t2017-01-11 17:28:30\tGenotypeConcordance\tchecked    14,870,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 7:57,064,567\n",
      "INFO\t2017-01-11 17:28:30\tGenotypeConcordance\tchecked    14,880,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 7:57,640,840\n",
      "INFO\t2017-01-11 17:28:30\tGenotypeConcordance\tchecked    14,890,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 7:57,927,767\n",
      "INFO\t2017-01-11 17:28:31\tGenotypeConcordance\tchecked    14,900,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 7:61,554,595\n",
      "INFO\t2017-01-11 17:28:31\tGenotypeConcordance\tchecked    14,910,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 7:61,823,904\n",
      "INFO\t2017-01-11 17:28:31\tGenotypeConcordance\tchecked    14,920,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 7:62,189,119\n",
      "INFO\t2017-01-11 17:28:31\tGenotypeConcordance\tchecked    14,930,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 7:62,828,260\n",
      "INFO\t2017-01-11 17:28:32\tGenotypeConcordance\tchecked    14,940,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 7:63,624,322\n",
      "INFO\t2017-01-11 17:28:32\tGenotypeConcordance\tchecked    14,950,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 7:64,427,308\n",
      "INFO\t2017-01-11 17:28:32\tGenotypeConcordance\tchecked    14,960,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 7:65,192,146\n",
      "INFO\t2017-01-11 17:28:33\tGenotypeConcordance\tchecked    14,970,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 7:66,033,033\n",
      "INFO\t2017-01-11 17:28:33\tGenotypeConcordance\tchecked    14,980,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 7:66,825,558\n",
      "INFO\t2017-01-11 17:28:33\tGenotypeConcordance\tchecked    14,990,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 7:67,532,341\n",
      "INFO\t2017-01-11 17:28:33\tGenotypeConcordance\tchecked    15,000,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 7:68,156,564\n",
      "INFO\t2017-01-11 17:28:34\tGenotypeConcordance\tchecked    15,010,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 7:68,889,003\n",
      "INFO\t2017-01-11 17:28:34\tGenotypeConcordance\tchecked    15,020,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 7:70,083,260\n",
      "INFO\t2017-01-11 17:28:34\tGenotypeConcordance\tchecked    15,030,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 7:70,842,337\n",
      "INFO\t2017-01-11 17:28:34\tGenotypeConcordance\tchecked    15,040,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 7:71,592,894\n",
      "INFO\t2017-01-11 17:28:35\tGenotypeConcordance\tchecked    15,050,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 7:72,403,209\n",
      "INFO\t2017-01-11 17:28:35\tGenotypeConcordance\tchecked    15,060,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 7:73,641,497\n",
      "INFO\t2017-01-11 17:28:35\tGenotypeConcordance\tchecked    15,070,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 7:75,243,991\n",
      "INFO\t2017-01-11 17:28:36\tGenotypeConcordance\tchecked    15,080,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 7:76,024,084\n",
      "INFO\t2017-01-11 17:28:36\tGenotypeConcordance\tchecked    15,090,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 7:76,897,826\n",
      "INFO\t2017-01-11 17:28:36\tGenotypeConcordance\tchecked    15,100,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 7:77,825,224\n",
      "INFO\t2017-01-11 17:28:37\tGenotypeConcordance\tchecked    15,110,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 7:78,599,311\n",
      "INFO\t2017-01-11 17:28:37\tGenotypeConcordance\tchecked    15,120,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 7:79,434,835\n",
      "INFO\t2017-01-11 17:28:37\tGenotypeConcordance\tchecked    15,130,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 7:80,311,647\n",
      "INFO\t2017-01-11 17:28:37\tGenotypeConcordance\tchecked    15,140,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 7:81,081,130\n",
      "INFO\t2017-01-11 17:28:38\tGenotypeConcordance\tchecked    15,150,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 7:81,985,187\n",
      "INFO\t2017-01-11 17:28:38\tGenotypeConcordance\tchecked    15,160,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 7:82,869,581\n",
      "INFO\t2017-01-11 17:28:38\tGenotypeConcordance\tchecked    15,170,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 7:83,679,299\n",
      "INFO\t2017-01-11 17:28:39\tGenotypeConcordance\tchecked    15,180,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 7:84,560,389\n",
      "INFO\t2017-01-11 17:28:39\tGenotypeConcordance\tchecked    15,190,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 7:85,456,390\n",
      "INFO\t2017-01-11 17:28:39\tGenotypeConcordance\tchecked    15,200,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 7:86,335,723\n",
      "INFO\t2017-01-11 17:28:39\tGenotypeConcordance\tchecked    15,210,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 7:87,331,322\n",
      "INFO\t2017-01-11 17:28:40\tGenotypeConcordance\tchecked    15,220,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 7:88,354,527\n",
      "INFO\t2017-01-11 17:28:40\tGenotypeConcordance\tchecked    15,230,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 7:89,166,525\n",
      "INFO\t2017-01-11 17:28:40\tGenotypeConcordance\tchecked    15,240,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 7:90,019,464\n",
      "INFO\t2017-01-11 17:28:41\tGenotypeConcordance\tchecked    15,250,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 7:90,969,510\n",
      "INFO\t2017-01-11 17:28:41\tGenotypeConcordance\tchecked    15,260,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 7:92,014,645\n",
      "INFO\t2017-01-11 17:28:41\tGenotypeConcordance\tchecked    15,270,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 7:93,133,758\n",
      "INFO\t2017-01-11 17:28:41\tGenotypeConcordance\tchecked    15,280,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 7:94,119,404\n",
      "INFO\t2017-01-11 17:28:42\tGenotypeConcordance\tchecked    15,290,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 7:95,175,661\n",
      "INFO\t2017-01-11 17:28:42\tGenotypeConcordance\tchecked    15,300,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 7:96,145,105\n",
      "INFO\t2017-01-11 17:28:42\tGenotypeConcordance\tchecked    15,310,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 7:97,060,570\n",
      "INFO\t2017-01-11 17:28:42\tGenotypeConcordance\tchecked    15,320,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 7:97,832,154\n",
      "INFO\t2017-01-11 17:28:43\tGenotypeConcordance\tchecked    15,330,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 7:98,660,529\n",
      "INFO\t2017-01-11 17:28:43\tGenotypeConcordance\tchecked    15,340,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 7:99,705,796\n",
      "INFO\t2017-01-11 17:28:43\tGenotypeConcordance\tchecked    15,350,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 7:100,635,956\n",
      "INFO\t2017-01-11 17:28:44\tGenotypeConcordance\tchecked    15,360,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 7:101,335,614\n",
      "INFO\t2017-01-11 17:28:44\tGenotypeConcordance\tchecked    15,370,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 7:102,296,892\n",
      "INFO\t2017-01-11 17:28:44\tGenotypeConcordance\tchecked    15,380,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 7:103,245,417\n",
      "INFO\t2017-01-11 17:28:44\tGenotypeConcordance\tchecked    15,390,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 7:103,976,997\n",
      "INFO\t2017-01-11 17:28:45\tGenotypeConcordance\tchecked    15,400,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:104,845,980\n",
      "INFO\t2017-01-11 17:28:45\tGenotypeConcordance\tchecked    15,410,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:105,730,603\n",
      "INFO\t2017-01-11 17:28:45\tGenotypeConcordance\tchecked    15,420,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:106,629,378\n",
      "INFO\t2017-01-11 17:28:45\tGenotypeConcordance\tchecked    15,430,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:107,551,530\n",
      "INFO\t2017-01-11 17:28:46\tGenotypeConcordance\tchecked    15,440,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:108,456,043\n",
      "INFO\t2017-01-11 17:28:46\tGenotypeConcordance\tchecked    15,450,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:109,264,452\n",
      "INFO\t2017-01-11 17:28:46\tGenotypeConcordance\tchecked    15,460,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:110,063,445\n",
      "INFO\t2017-01-11 17:28:47\tGenotypeConcordance\tchecked    15,470,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:111,002,121\n",
      "INFO\t2017-01-11 17:28:47\tGenotypeConcordance\tchecked    15,480,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:111,945,260\n",
      "INFO\t2017-01-11 17:28:47\tGenotypeConcordance\tchecked    15,490,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:112,942,522\n",
      "INFO\t2017-01-11 17:28:47\tGenotypeConcordance\tchecked    15,500,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:113,946,693\n",
      "INFO\t2017-01-11 17:28:48\tGenotypeConcordance\tchecked    15,510,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:114,991,272\n",
      "INFO\t2017-01-11 17:28:48\tGenotypeConcordance\tchecked    15,520,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:115,993,469\n",
      "INFO\t2017-01-11 17:28:48\tGenotypeConcordance\tchecked    15,530,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:117,162,954\n",
      "INFO\t2017-01-11 17:28:48\tGenotypeConcordance\tchecked    15,540,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:118,225,978\n",
      "INFO\t2017-01-11 17:28:49\tGenotypeConcordance\tchecked    15,550,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:119,161,984\n",
      "INFO\t2017-01-11 17:28:49\tGenotypeConcordance\tchecked    15,560,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:120,124,719\n",
      "INFO\t2017-01-11 17:28:49\tGenotypeConcordance\tchecked    15,570,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:121,138,156\n",
      "INFO\t2017-01-11 17:28:49\tGenotypeConcordance\tchecked    15,580,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:122,155,821\n",
      "INFO\t2017-01-11 17:28:50\tGenotypeConcordance\tchecked    15,590,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:123,089,364\n",
      "INFO\t2017-01-11 17:28:50\tGenotypeConcordance\tchecked    15,600,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:124,085,851\n",
      "INFO\t2017-01-11 17:28:50\tGenotypeConcordance\tchecked    15,610,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:124,959,006\n",
      "INFO\t2017-01-11 17:28:51\tGenotypeConcordance\tchecked    15,620,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:125,705,705\n",
      "INFO\t2017-01-11 17:28:51\tGenotypeConcordance\tchecked    15,630,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:126,592,772\n",
      "INFO\t2017-01-11 17:28:51\tGenotypeConcordance\tchecked    15,640,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:127,644,243\n",
      "INFO\t2017-01-11 17:28:51\tGenotypeConcordance\tchecked    15,650,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:128,514,177\n",
      "INFO\t2017-01-11 17:28:52\tGenotypeConcordance\tchecked    15,660,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:129,452,834\n",
      "INFO\t2017-01-11 17:28:52\tGenotypeConcordance\tchecked    15,670,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:130,495,991\n",
      "INFO\t2017-01-11 17:28:52\tGenotypeConcordance\tchecked    15,680,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:131,385,570\n",
      "INFO\t2017-01-11 17:28:52\tGenotypeConcordance\tchecked    15,690,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:132,186,073\n",
      "INFO\t2017-01-11 17:28:53\tGenotypeConcordance\tchecked    15,700,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:133,237,686\n",
      "INFO\t2017-01-11 17:28:53\tGenotypeConcordance\tchecked    15,710,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:134,202,002\n",
      "INFO\t2017-01-11 17:28:53\tGenotypeConcordance\tchecked    15,720,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:135,123,402\n",
      "INFO\t2017-01-11 17:28:54\tGenotypeConcordance\tchecked    15,730,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:135,979,376\n",
      "INFO\t2017-01-11 17:28:54\tGenotypeConcordance\tchecked    15,740,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:136,861,576\n",
      "INFO\t2017-01-11 17:28:54\tGenotypeConcordance\tchecked    15,750,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:137,783,240\n",
      "INFO\t2017-01-11 17:28:54\tGenotypeConcordance\tchecked    15,760,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:138,477,526\n",
      "INFO\t2017-01-11 17:28:55\tGenotypeConcordance\tchecked    15,770,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:139,321,855\n",
      "INFO\t2017-01-11 17:28:55\tGenotypeConcordance\tchecked    15,780,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:140,190,337\n",
      "INFO\t2017-01-11 17:28:55\tGenotypeConcordance\tchecked    15,790,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:141,169,727\n",
      "INFO\t2017-01-11 17:28:55\tGenotypeConcordance\tchecked    15,800,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:141,992,116\n",
      "INFO\t2017-01-11 17:28:56\tGenotypeConcordance\tchecked    15,810,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:142,474,879\n",
      "INFO\t2017-01-11 17:28:56\tGenotypeConcordance\tchecked    15,820,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:143,632,984\n",
      "INFO\t2017-01-11 17:28:56\tGenotypeConcordance\tchecked    15,830,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:144,622,552\n",
      "INFO\t2017-01-11 17:28:57\tGenotypeConcordance\tchecked    15,840,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:145,434,874\n",
      "INFO\t2017-01-11 17:28:57\tGenotypeConcordance\tchecked    15,850,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:146,201,148\n",
      "INFO\t2017-01-11 17:28:57\tGenotypeConcordance\tchecked    15,860,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:146,982,163\n",
      "INFO\t2017-01-11 17:28:57\tGenotypeConcordance\tchecked    15,870,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:147,645,125\n",
      "INFO\t2017-01-11 17:28:58\tGenotypeConcordance\tchecked    15,880,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:148,373,150\n",
      "INFO\t2017-01-11 17:28:58\tGenotypeConcordance\tchecked    15,890,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:149,183,459\n",
      "INFO\t2017-01-11 17:28:58\tGenotypeConcordance\tchecked    15,900,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:149,895,465\n",
      "INFO\t2017-01-11 17:28:58\tGenotypeConcordance\tchecked    15,910,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:150,602,428\n",
      "INFO\t2017-01-11 17:28:59\tGenotypeConcordance\tchecked    15,920,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:151,425,178\n",
      "INFO\t2017-01-11 17:28:59\tGenotypeConcordance\tchecked    15,930,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:152,113,680\n",
      "INFO\t2017-01-11 17:28:59\tGenotypeConcordance\tchecked    15,940,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:152,801,860\n",
      "INFO\t2017-01-11 17:29:00\tGenotypeConcordance\tchecked    15,950,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:153,503,092\n",
      "INFO\t2017-01-11 17:29:00\tGenotypeConcordance\tchecked    15,960,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:154,175,478\n",
      "INFO\t2017-01-11 17:29:00\tGenotypeConcordance\tchecked    15,970,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:154,951,511\n",
      "INFO\t2017-01-11 17:29:00\tGenotypeConcordance\tchecked    15,980,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:155,624,450\n",
      "INFO\t2017-01-11 17:29:01\tGenotypeConcordance\tchecked    15,990,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:156,286,725\n",
      "INFO\t2017-01-11 17:29:01\tGenotypeConcordance\tchecked    16,000,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:157,030,216\n",
      "INFO\t2017-01-11 17:29:01\tGenotypeConcordance\tchecked    16,010,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:157,624,989\n",
      "INFO\t2017-01-11 17:29:01\tGenotypeConcordance\tchecked    16,020,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:158,128,863\n",
      "INFO\t2017-01-11 17:29:02\tGenotypeConcordance\tchecked    16,030,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 7:158,765,311\n",
      "INFO\t2017-01-11 17:29:02\tGenotypeConcordance\tchecked    16,040,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 8:307,858\n",
      "INFO\t2017-01-11 17:29:02\tGenotypeConcordance\tchecked    16,050,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 8:855,881\n",
      "INFO\t2017-01-11 17:29:03\tGenotypeConcordance\tchecked    16,060,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 8:1,317,099\n",
      "INFO\t2017-01-11 17:29:03\tGenotypeConcordance\tchecked    16,070,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 8:1,817,602\n",
      "INFO\t2017-01-11 17:29:03\tGenotypeConcordance\tchecked    16,080,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 8:2,253,551\n",
      "INFO\t2017-01-11 17:29:03\tGenotypeConcordance\tchecked    16,090,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 8:2,747,624\n",
      "INFO\t2017-01-11 17:29:04\tGenotypeConcordance\tchecked    16,100,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 8:3,171,301\n",
      "INFO\t2017-01-11 17:29:04\tGenotypeConcordance\tchecked    16,110,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 8:3,511,936\n",
      "INFO\t2017-01-11 17:29:04\tGenotypeConcordance\tchecked    16,120,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 8:3,815,701\n",
      "INFO\t2017-01-11 17:29:05\tGenotypeConcordance\tchecked    16,130,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 8:4,109,451\n",
      "INFO\t2017-01-11 17:29:05\tGenotypeConcordance\tchecked    16,140,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 8:4,448,182\n",
      "INFO\t2017-01-11 17:29:05\tGenotypeConcordance\tchecked    16,150,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 8:4,824,267\n",
      "INFO\t2017-01-11 17:29:05\tGenotypeConcordance\tchecked    16,160,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 8:5,230,310\n",
      "INFO\t2017-01-11 17:29:06\tGenotypeConcordance\tchecked    16,170,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 8:5,629,968\n",
      "INFO\t2017-01-11 17:29:06\tGenotypeConcordance\tchecked    16,180,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 8:6,035,203\n",
      "INFO\t2017-01-11 17:29:06\tGenotypeConcordance\tchecked    16,190,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 8:6,450,224\n",
      "INFO\t2017-01-11 17:29:06\tGenotypeConcordance\tchecked    16,200,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 8:6,933,197\n",
      "INFO\t2017-01-11 17:29:07\tGenotypeConcordance\tchecked    16,210,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 8:7,915,309\n",
      "INFO\t2017-01-11 17:29:07\tGenotypeConcordance\tchecked    16,220,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 8:8,528,908\n",
      "INFO\t2017-01-11 17:29:07\tGenotypeConcordance\tchecked    16,230,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 8:9,103,724\n",
      "INFO\t2017-01-11 17:29:08\tGenotypeConcordance\tchecked    16,240,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 8:9,715,476\n",
      "INFO\t2017-01-11 17:29:08\tGenotypeConcordance\tchecked    16,250,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 8:10,349,672\n",
      "INFO\t2017-01-11 17:29:08\tGenotypeConcordance\tchecked    16,260,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 8:10,957,243\n",
      "INFO\t2017-01-11 17:29:08\tGenotypeConcordance\tchecked    16,270,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 8:11,529,409\n",
      "INFO\t2017-01-11 17:29:09\tGenotypeConcordance\tchecked    16,280,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 8:12,075,226\n",
      "INFO\t2017-01-11 17:29:09\tGenotypeConcordance\tchecked    16,290,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 8:12,675,621\n",
      "INFO\t2017-01-11 17:29:09\tGenotypeConcordance\tchecked    16,300,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 8:13,212,734\n",
      "INFO\t2017-01-11 17:29:10\tGenotypeConcordance\tchecked    16,310,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 8:13,687,093\n",
      "INFO\t2017-01-11 17:29:10\tGenotypeConcordance\tchecked    16,320,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 8:14,169,711\n",
      "INFO\t2017-01-11 17:29:10\tGenotypeConcordance\tchecked    16,330,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 8:14,669,987\n",
      "INFO\t2017-01-11 17:29:11\tGenotypeConcordance\tchecked    16,340,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 8:15,240,776\n",
      "INFO\t2017-01-11 17:29:11\tGenotypeConcordance\tchecked    16,350,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 8:15,745,075\n",
      "INFO\t2017-01-11 17:29:11\tGenotypeConcordance\tchecked    16,360,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 8:16,315,830\n",
      "INFO\t2017-01-11 17:29:11\tGenotypeConcordance\tchecked    16,370,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 8:16,852,283\n",
      "INFO\t2017-01-11 17:29:12\tGenotypeConcordance\tchecked    16,380,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 8:17,469,422\n",
      "INFO\t2017-01-11 17:29:12\tGenotypeConcordance\tchecked    16,390,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 8:18,004,224\n",
      "INFO\t2017-01-11 17:29:12\tGenotypeConcordance\tchecked    16,400,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 8:18,577,420\n",
      "INFO\t2017-01-11 17:29:13\tGenotypeConcordance\tchecked    16,410,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 8:19,170,003\n",
      "INFO\t2017-01-11 17:29:13\tGenotypeConcordance\tchecked    16,420,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 8:19,839,175\n",
      "INFO\t2017-01-11 17:29:13\tGenotypeConcordance\tchecked    16,430,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 8:20,533,362\n",
      "INFO\t2017-01-11 17:29:14\tGenotypeConcordance\tchecked    16,440,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 8:21,223,083\n",
      "INFO\t2017-01-11 17:29:14\tGenotypeConcordance\tchecked    16,450,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 8:22,029,129\n",
      "INFO\t2017-01-11 17:29:14\tGenotypeConcordance\tchecked    16,460,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 8:22,836,272\n",
      "INFO\t2017-01-11 17:29:14\tGenotypeConcordance\tchecked    16,470,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 8:23,540,354\n",
      "INFO\t2017-01-11 17:29:15\tGenotypeConcordance\tchecked    16,480,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 8:24,382,034\n",
      "INFO\t2017-01-11 17:29:15\tGenotypeConcordance\tchecked    16,490,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 8:25,187,933\n",
      "INFO\t2017-01-11 17:29:15\tGenotypeConcordance\tchecked    16,500,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 8:26,024,889\n",
      "INFO\t2017-01-11 17:29:15\tGenotypeConcordance\tchecked    16,510,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 8:26,816,678\n",
      "INFO\t2017-01-11 17:29:16\tGenotypeConcordance\tchecked    16,520,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 8:27,635,009\n",
      "INFO\t2017-01-11 17:29:16\tGenotypeConcordance\tchecked    16,530,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 8:28,461,311\n",
      "INFO\t2017-01-11 17:29:16\tGenotypeConcordance\tchecked    16,540,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 8:29,366,588\n",
      "INFO\t2017-01-11 17:29:17\tGenotypeConcordance\tchecked    16,550,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 8:30,207,421\n",
      "INFO\t2017-01-11 17:29:17\tGenotypeConcordance\tchecked    16,560,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 8:31,123,251\n",
      "INFO\t2017-01-11 17:29:17\tGenotypeConcordance\tchecked    16,570,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 8:31,992,258\n",
      "INFO\t2017-01-11 17:29:17\tGenotypeConcordance\tchecked    16,580,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 8:32,882,217\n",
      "INFO\t2017-01-11 17:29:18\tGenotypeConcordance\tchecked    16,590,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 8:33,780,637\n",
      "INFO\t2017-01-11 17:29:18\tGenotypeConcordance\tchecked    16,600,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 8:34,663,135\n",
      "INFO\t2017-01-11 17:29:18\tGenotypeConcordance\tchecked    16,610,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 8:35,648,562\n",
      "INFO\t2017-01-11 17:29:18\tGenotypeConcordance\tchecked    16,620,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 8:36,675,564\n",
      "INFO\t2017-01-11 17:29:19\tGenotypeConcordance\tchecked    16,630,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 8:37,647,063\n",
      "INFO\t2017-01-11 17:29:19\tGenotypeConcordance\tchecked    16,640,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 8:38,658,984\n",
      "INFO\t2017-01-11 17:29:19\tGenotypeConcordance\tchecked    16,650,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 8:39,627,891\n",
      "INFO\t2017-01-11 17:29:19\tGenotypeConcordance\tchecked    16,660,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 8:40,375,243\n",
      "INFO\t2017-01-11 17:29:20\tGenotypeConcordance\tchecked    16,670,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 8:41,149,507\n",
      "INFO\t2017-01-11 17:29:20\tGenotypeConcordance\tchecked    16,680,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 8:42,013,150\n",
      "INFO\t2017-01-11 17:29:20\tGenotypeConcordance\tchecked    16,690,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 8:43,006,010\n",
      "INFO\t2017-01-11 17:29:20\tGenotypeConcordance\tchecked    16,700,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 8:43,684,593\n",
      "INFO\t2017-01-11 17:29:21\tGenotypeConcordance\tchecked    16,710,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 8:47,287,215\n",
      "INFO\t2017-01-11 17:29:21\tGenotypeConcordance\tchecked    16,720,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 8:48,163,682\n",
      "INFO\t2017-01-11 17:29:21\tGenotypeConcordance\tchecked    16,730,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 8:49,357,151\n",
      "INFO\t2017-01-11 17:29:22\tGenotypeConcordance\tchecked    16,740,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 8:50,307,281\n",
      "INFO\t2017-01-11 17:29:22\tGenotypeConcordance\tchecked    16,750,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 8:51,141,137\n",
      "INFO\t2017-01-11 17:29:22\tGenotypeConcordance\tchecked    16,760,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 8:51,883,407\n",
      "INFO\t2017-01-11 17:29:22\tGenotypeConcordance\tchecked    16,770,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 8:52,626,791\n",
      "INFO\t2017-01-11 17:29:23\tGenotypeConcordance\tchecked    16,780,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 8:53,530,970\n",
      "INFO\t2017-01-11 17:29:23\tGenotypeConcordance\tchecked    16,790,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 8:54,364,075\n",
      "INFO\t2017-01-11 17:29:23\tGenotypeConcordance\tchecked    16,800,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 8:55,217,867\n",
      "INFO\t2017-01-11 17:29:23\tGenotypeConcordance\tchecked    16,810,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 8:56,085,891\n",
      "INFO\t2017-01-11 17:29:24\tGenotypeConcordance\tchecked    16,820,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 8:56,904,454\n",
      "INFO\t2017-01-11 17:29:24\tGenotypeConcordance\tchecked    16,830,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 8:57,723,177\n",
      "INFO\t2017-01-11 17:29:24\tGenotypeConcordance\tchecked    16,840,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 8:58,436,870\n",
      "INFO\t2017-01-11 17:29:25\tGenotypeConcordance\tchecked    16,850,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 8:59,289,763\n",
      "INFO\t2017-01-11 17:29:25\tGenotypeConcordance\tchecked    16,860,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 8:60,198,604\n",
      "INFO\t2017-01-11 17:29:25\tGenotypeConcordance\tchecked    16,870,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 8:61,122,265\n",
      "INFO\t2017-01-11 17:29:25\tGenotypeConcordance\tchecked    16,880,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 8:62,067,486\n",
      "INFO\t2017-01-11 17:29:26\tGenotypeConcordance\tchecked    16,890,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 8:62,924,308\n",
      "INFO\t2017-01-11 17:29:26\tGenotypeConcordance\tchecked    16,900,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 8:63,713,933\n",
      "INFO\t2017-01-11 17:29:26\tGenotypeConcordance\tchecked    16,910,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 8:64,609,056\n",
      "INFO\t2017-01-11 17:29:26\tGenotypeConcordance\tchecked    16,920,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 8:65,738,190\n",
      "INFO\t2017-01-11 17:29:27\tGenotypeConcordance\tchecked    16,930,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 8:66,669,635\n",
      "INFO\t2017-01-11 17:29:27\tGenotypeConcordance\tchecked    16,940,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 8:67,635,455\n",
      "INFO\t2017-01-11 17:29:27\tGenotypeConcordance\tchecked    16,950,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 8:68,673,649\n",
      "INFO\t2017-01-11 17:29:27\tGenotypeConcordance\tchecked    16,960,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 8:69,559,617\n",
      "INFO\t2017-01-11 17:29:28\tGenotypeConcordance\tchecked    16,970,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 8:70,375,937\n",
      "INFO\t2017-01-11 17:29:28\tGenotypeConcordance\tchecked    16,980,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 8:71,248,222\n",
      "INFO\t2017-01-11 17:29:28\tGenotypeConcordance\tchecked    16,990,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 8:72,204,352\n",
      "INFO\t2017-01-11 17:29:29\tGenotypeConcordance\tchecked    17,000,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 8:73,068,636\n",
      "INFO\t2017-01-11 17:29:29\tGenotypeConcordance\tchecked    17,010,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 8:73,929,721\n",
      "INFO\t2017-01-11 17:29:29\tGenotypeConcordance\tchecked    17,020,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 8:74,777,699\n",
      "INFO\t2017-01-11 17:29:29\tGenotypeConcordance\tchecked    17,030,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 8:75,663,682\n",
      "INFO\t2017-01-11 17:29:30\tGenotypeConcordance\tchecked    17,040,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 8:76,499,018\n",
      "INFO\t2017-01-11 17:29:30\tGenotypeConcordance\tchecked    17,050,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 8:77,432,409\n",
      "INFO\t2017-01-11 17:29:30\tGenotypeConcordance\tchecked    17,060,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 8:78,380,536\n",
      "INFO\t2017-01-11 17:29:31\tGenotypeConcordance\tchecked    17,070,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 8:79,277,609\n",
      "INFO\t2017-01-11 17:29:31\tGenotypeConcordance\tchecked    17,080,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 8:80,246,971\n",
      "INFO\t2017-01-11 17:29:31\tGenotypeConcordance\tchecked    17,090,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 8:81,172,159\n",
      "INFO\t2017-01-11 17:29:31\tGenotypeConcordance\tchecked    17,100,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 8:82,116,596\n",
      "INFO\t2017-01-11 17:29:32\tGenotypeConcordance\tchecked    17,110,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 8:82,996,542\n",
      "INFO\t2017-01-11 17:29:32\tGenotypeConcordance\tchecked    17,120,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 8:83,820,378\n",
      "INFO\t2017-01-11 17:29:32\tGenotypeConcordance\tchecked    17,130,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 8:84,599,247\n",
      "INFO\t2017-01-11 17:29:32\tGenotypeConcordance\tchecked    17,140,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 8:85,364,470\n",
      "INFO\t2017-01-11 17:29:33\tGenotypeConcordance\tchecked    17,150,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 8:86,343,937\n",
      "INFO\t2017-01-11 17:29:33\tGenotypeConcordance\tchecked    17,160,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:87,518,101\n",
      "INFO\t2017-01-11 17:29:33\tGenotypeConcordance\tchecked    17,170,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:88,328,054\n",
      "INFO\t2017-01-11 17:29:33\tGenotypeConcordance\tchecked    17,180,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:89,157,257\n",
      "INFO\t2017-01-11 17:29:34\tGenotypeConcordance\tchecked    17,190,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:90,035,634\n",
      "INFO\t2017-01-11 17:29:34\tGenotypeConcordance\tchecked    17,200,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:90,869,755\n",
      "INFO\t2017-01-11 17:29:34\tGenotypeConcordance\tchecked    17,210,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:91,788,105\n",
      "INFO\t2017-01-11 17:29:34\tGenotypeConcordance\tchecked    17,220,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:92,842,791\n",
      "INFO\t2017-01-11 17:29:35\tGenotypeConcordance\tchecked    17,230,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:93,935,665\n",
      "INFO\t2017-01-11 17:29:35\tGenotypeConcordance\tchecked    17,240,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:94,962,637\n",
      "INFO\t2017-01-11 17:29:35\tGenotypeConcordance\tchecked    17,250,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:95,818,155\n",
      "INFO\t2017-01-11 17:29:36\tGenotypeConcordance\tchecked    17,260,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:96,698,676\n",
      "INFO\t2017-01-11 17:29:36\tGenotypeConcordance\tchecked    17,270,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:97,563,575\n",
      "INFO\t2017-01-11 17:29:36\tGenotypeConcordance\tchecked    17,280,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:98,498,193\n",
      "INFO\t2017-01-11 17:29:36\tGenotypeConcordance\tchecked    17,290,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:99,296,821\n",
      "INFO\t2017-01-11 17:29:37\tGenotypeConcordance\tchecked    17,300,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:100,503,115\n",
      "INFO\t2017-01-11 17:29:37\tGenotypeConcordance\tchecked    17,310,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:101,577,545\n",
      "INFO\t2017-01-11 17:29:37\tGenotypeConcordance\tchecked    17,320,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:102,406,084\n",
      "INFO\t2017-01-11 17:29:37\tGenotypeConcordance\tchecked    17,330,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:103,258,448\n",
      "INFO\t2017-01-11 17:29:38\tGenotypeConcordance\tchecked    17,340,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:104,124,353\n",
      "INFO\t2017-01-11 17:29:38\tGenotypeConcordance\tchecked    17,350,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:105,018,863\n",
      "INFO\t2017-01-11 17:29:38\tGenotypeConcordance\tchecked    17,360,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:105,916,115\n",
      "INFO\t2017-01-11 17:29:39\tGenotypeConcordance\tchecked    17,370,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:106,883,102\n",
      "INFO\t2017-01-11 17:29:39\tGenotypeConcordance\tchecked    17,380,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:107,849,895\n",
      "INFO\t2017-01-11 17:29:39\tGenotypeConcordance\tchecked    17,390,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:108,706,623\n",
      "INFO\t2017-01-11 17:29:39\tGenotypeConcordance\tchecked    17,400,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:109,608,623\n",
      "INFO\t2017-01-11 17:29:40\tGenotypeConcordance\tchecked    17,410,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:110,573,770\n",
      "INFO\t2017-01-11 17:29:40\tGenotypeConcordance\tchecked    17,420,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:111,622,857\n",
      "INFO\t2017-01-11 17:29:40\tGenotypeConcordance\tchecked    17,430,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:112,537,538\n",
      "INFO\t2017-01-11 17:29:40\tGenotypeConcordance\tchecked    17,440,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:113,450,623\n",
      "INFO\t2017-01-11 17:29:41\tGenotypeConcordance\tchecked    17,450,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:114,440,909\n",
      "INFO\t2017-01-11 17:29:41\tGenotypeConcordance\tchecked    17,460,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:115,245,185\n",
      "INFO\t2017-01-11 17:29:41\tGenotypeConcordance\tchecked    17,470,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:116,080,876\n",
      "INFO\t2017-01-11 17:29:42\tGenotypeConcordance\tchecked    17,480,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:117,160,596\n",
      "INFO\t2017-01-11 17:29:42\tGenotypeConcordance\tchecked    17,490,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:118,146,283\n",
      "INFO\t2017-01-11 17:29:42\tGenotypeConcordance\tchecked    17,500,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:119,097,287\n",
      "INFO\t2017-01-11 17:29:42\tGenotypeConcordance\tchecked    17,510,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:119,972,696\n",
      "INFO\t2017-01-11 17:29:43\tGenotypeConcordance\tchecked    17,520,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:120,860,318\n",
      "INFO\t2017-01-11 17:29:43\tGenotypeConcordance\tchecked    17,530,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:121,777,896\n",
      "INFO\t2017-01-11 17:29:43\tGenotypeConcordance\tchecked    17,540,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:122,583,991\n",
      "INFO\t2017-01-11 17:29:44\tGenotypeConcordance\tchecked    17,550,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:123,466,306\n",
      "INFO\t2017-01-11 17:29:44\tGenotypeConcordance\tchecked    17,560,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:124,350,791\n",
      "INFO\t2017-01-11 17:29:44\tGenotypeConcordance\tchecked    17,570,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:125,229,684\n",
      "INFO\t2017-01-11 17:29:44\tGenotypeConcordance\tchecked    17,580,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:126,009,595\n",
      "INFO\t2017-01-11 17:29:45\tGenotypeConcordance\tchecked    17,590,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:126,859,771\n",
      "INFO\t2017-01-11 17:29:45\tGenotypeConcordance\tchecked    17,600,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:127,651,334\n",
      "INFO\t2017-01-11 17:29:45\tGenotypeConcordance\tchecked    17,610,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:128,446,680\n",
      "INFO\t2017-01-11 17:29:45\tGenotypeConcordance\tchecked    17,620,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:129,223,877\n",
      "INFO\t2017-01-11 17:29:46\tGenotypeConcordance\tchecked    17,630,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:130,025,628\n",
      "INFO\t2017-01-11 17:29:46\tGenotypeConcordance\tchecked    17,640,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:130,877,547\n",
      "INFO\t2017-01-11 17:29:46\tGenotypeConcordance\tchecked    17,650,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:131,731,502\n",
      "INFO\t2017-01-11 17:29:46\tGenotypeConcordance\tchecked    17,660,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:132,508,007\n",
      "INFO\t2017-01-11 17:29:47\tGenotypeConcordance\tchecked    17,670,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:133,264,699\n",
      "INFO\t2017-01-11 17:29:47\tGenotypeConcordance\tchecked    17,680,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:134,161,488\n",
      "INFO\t2017-01-11 17:29:47\tGenotypeConcordance\tchecked    17,690,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:134,885,329\n",
      "INFO\t2017-01-11 17:29:48\tGenotypeConcordance\tchecked    17,700,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:135,641,806\n",
      "INFO\t2017-01-11 17:29:48\tGenotypeConcordance\tchecked    17,710,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:136,456,909\n",
      "INFO\t2017-01-11 17:29:48\tGenotypeConcordance\tchecked    17,720,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:137,291,187\n",
      "INFO\t2017-01-11 17:29:48\tGenotypeConcordance\tchecked    17,730,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:138,078,078\n",
      "INFO\t2017-01-11 17:29:49\tGenotypeConcordance\tchecked    17,740,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:138,839,468\n",
      "INFO\t2017-01-11 17:29:49\tGenotypeConcordance\tchecked    17,750,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:139,625,123\n",
      "INFO\t2017-01-11 17:29:49\tGenotypeConcordance\tchecked    17,760,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:140,335,008\n",
      "INFO\t2017-01-11 17:29:50\tGenotypeConcordance\tchecked    17,770,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:141,074,664\n",
      "INFO\t2017-01-11 17:29:50\tGenotypeConcordance\tchecked    17,780,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:141,946,507\n",
      "INFO\t2017-01-11 17:29:50\tGenotypeConcordance\tchecked    17,790,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:142,636,650\n",
      "INFO\t2017-01-11 17:29:50\tGenotypeConcordance\tchecked    17,800,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:143,364,390\n",
      "INFO\t2017-01-11 17:29:51\tGenotypeConcordance\tchecked    17,810,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:144,058,475\n",
      "INFO\t2017-01-11 17:29:51\tGenotypeConcordance\tchecked    17,820,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:144,717,757\n",
      "INFO\t2017-01-11 17:29:51\tGenotypeConcordance\tchecked    17,830,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:145,752,130\n",
      "INFO\t2017-01-11 17:29:51\tGenotypeConcordance\tchecked    17,840,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 9:210,882\n",
      "INFO\t2017-01-11 17:29:52\tGenotypeConcordance\tchecked    17,850,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 9:765,689\n",
      "INFO\t2017-01-11 17:29:52\tGenotypeConcordance\tchecked    17,860,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 9:1,289,862\n",
      "INFO\t2017-01-11 17:29:52\tGenotypeConcordance\tchecked    17,870,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 9:1,806,787\n",
      "INFO\t2017-01-11 17:29:53\tGenotypeConcordance\tchecked    17,880,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 9:2,402,859\n",
      "INFO\t2017-01-11 17:29:53\tGenotypeConcordance\tchecked    17,890,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 9:3,039,034\n",
      "INFO\t2017-01-11 17:29:53\tGenotypeConcordance\tchecked    17,900,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 9:3,780,497\n",
      "INFO\t2017-01-11 17:29:53\tGenotypeConcordance\tchecked    17,910,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 9:4,393,680\n",
      "INFO\t2017-01-11 17:29:54\tGenotypeConcordance\tchecked    17,920,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 9:4,990,416\n",
      "INFO\t2017-01-11 17:29:54\tGenotypeConcordance\tchecked    17,930,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 9:5,720,296\n",
      "INFO\t2017-01-11 17:29:54\tGenotypeConcordance\tchecked    17,940,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 9:6,577,741\n",
      "INFO\t2017-01-11 17:29:55\tGenotypeConcordance\tchecked    17,950,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 9:7,147,976\n",
      "INFO\t2017-01-11 17:29:55\tGenotypeConcordance\tchecked    17,960,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 9:7,727,080\n",
      "INFO\t2017-01-11 17:29:55\tGenotypeConcordance\tchecked    17,970,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 9:8,371,681\n",
      "INFO\t2017-01-11 17:29:56\tGenotypeConcordance\tchecked    17,980,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 9:8,979,212\n",
      "INFO\t2017-01-11 17:29:56\tGenotypeConcordance\tchecked    17,990,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 9:9,634,771\n",
      "INFO\t2017-01-11 17:29:56\tGenotypeConcordance\tchecked    18,000,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 9:10,207,866\n",
      "INFO\t2017-01-11 17:29:56\tGenotypeConcordance\tchecked    18,010,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 9:10,835,964\n",
      "INFO\t2017-01-11 17:29:57\tGenotypeConcordance\tchecked    18,020,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 9:11,426,814\n",
      "INFO\t2017-01-11 17:29:57\tGenotypeConcordance\tchecked    18,030,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 9:12,019,745\n",
      "INFO\t2017-01-11 17:29:57\tGenotypeConcordance\tchecked    18,040,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 9:12,693,777\n",
      "INFO\t2017-01-11 17:29:57\tGenotypeConcordance\tchecked    18,050,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 9:13,505,677\n",
      "INFO\t2017-01-11 17:29:58\tGenotypeConcordance\tchecked    18,060,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 9:14,182,713\n",
      "INFO\t2017-01-11 17:29:58\tGenotypeConcordance\tchecked    18,070,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 9:14,907,332\n",
      "INFO\t2017-01-11 17:29:58\tGenotypeConcordance\tchecked    18,080,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 9:15,578,240\n",
      "INFO\t2017-01-11 17:29:59\tGenotypeConcordance\tchecked    18,090,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 9:16,299,388\n",
      "INFO\t2017-01-11 17:29:59\tGenotypeConcordance\tchecked    18,100,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 9:16,974,010\n",
      "INFO\t2017-01-11 17:29:59\tGenotypeConcordance\tchecked    18,110,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 9:17,652,187\n",
      "INFO\t2017-01-11 17:30:00\tGenotypeConcordance\tchecked    18,120,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 9:18,313,213\n",
      "INFO\t2017-01-11 17:30:00\tGenotypeConcordance\tchecked    18,130,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 9:18,999,766\n",
      "INFO\t2017-01-11 17:30:00\tGenotypeConcordance\tchecked    18,140,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 9:19,706,618\n",
      "INFO\t2017-01-11 17:30:00\tGenotypeConcordance\tchecked    18,150,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 9:20,477,167\n",
      "INFO\t2017-01-11 17:30:01\tGenotypeConcordance\tchecked    18,160,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 9:21,292,255\n",
      "INFO\t2017-01-11 17:30:01\tGenotypeConcordance\tchecked    18,170,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 9:22,088,832\n",
      "INFO\t2017-01-11 17:30:01\tGenotypeConcordance\tchecked    18,180,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 9:22,841,222\n",
      "INFO\t2017-01-11 17:30:01\tGenotypeConcordance\tchecked    18,190,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 9:23,504,185\n",
      "INFO\t2017-01-11 17:30:02\tGenotypeConcordance\tchecked    18,200,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 9:24,301,818\n",
      "INFO\t2017-01-11 17:30:02\tGenotypeConcordance\tchecked    18,210,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 9:25,042,178\n",
      "INFO\t2017-01-11 17:30:02\tGenotypeConcordance\tchecked    18,220,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 9:25,687,819\n",
      "INFO\t2017-01-11 17:30:02\tGenotypeConcordance\tchecked    18,230,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 9:26,465,331\n",
      "INFO\t2017-01-11 17:30:03\tGenotypeConcordance\tchecked    18,240,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 9:27,199,009\n",
      "INFO\t2017-01-11 17:30:03\tGenotypeConcordance\tchecked    18,250,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 9:27,937,370\n",
      "INFO\t2017-01-11 17:30:03\tGenotypeConcordance\tchecked    18,260,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 9:28,639,917\n",
      "INFO\t2017-01-11 17:30:04\tGenotypeConcordance\tchecked    18,270,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 9:29,409,616\n",
      "INFO\t2017-01-11 17:30:04\tGenotypeConcordance\tchecked    18,280,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 9:30,137,652\n",
      "INFO\t2017-01-11 17:30:04\tGenotypeConcordance\tchecked    18,290,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 9:30,938,150\n",
      "INFO\t2017-01-11 17:30:04\tGenotypeConcordance\tchecked    18,300,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 9:31,622,953\n",
      "INFO\t2017-01-11 17:30:05\tGenotypeConcordance\tchecked    18,310,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 9:32,324,971\n",
      "INFO\t2017-01-11 17:30:05\tGenotypeConcordance\tchecked    18,320,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 9:33,170,547\n",
      "INFO\t2017-01-11 17:30:05\tGenotypeConcordance\tchecked    18,330,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 9:34,027,736\n",
      "INFO\t2017-01-11 17:30:05\tGenotypeConcordance\tchecked    18,340,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 9:35,037,954\n",
      "INFO\t2017-01-11 17:30:06\tGenotypeConcordance\tchecked    18,350,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 9:36,104,753\n",
      "INFO\t2017-01-11 17:30:06\tGenotypeConcordance\tchecked    18,360,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 9:36,965,295\n",
      "INFO\t2017-01-11 17:30:06\tGenotypeConcordance\tchecked    18,370,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 9:37,889,110\n",
      "INFO\t2017-01-11 17:30:07\tGenotypeConcordance\tchecked    18,380,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 9:38,676,375\n",
      "INFO\t2017-01-11 17:30:07\tGenotypeConcordance\tchecked    18,390,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 9:40,646,116\n",
      "INFO\t2017-01-11 17:30:07\tGenotypeConcordance\tchecked    18,400,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 9:43,437,884\n",
      "INFO\t2017-01-11 17:30:08\tGenotypeConcordance\tchecked    18,410,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 9:44,761,195\n",
      "INFO\t2017-01-11 17:30:08\tGenotypeConcordance\tchecked    18,420,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 9:46,141,478\n",
      "INFO\t2017-01-11 17:30:08\tGenotypeConcordance\tchecked    18,430,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 9:66,478,266\n",
      "INFO\t2017-01-11 17:30:09\tGenotypeConcordance\tchecked    18,440,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 9:67,302,206\n",
      "INFO\t2017-01-11 17:30:09\tGenotypeConcordance\tchecked    18,450,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 9:68,406,490\n",
      "INFO\t2017-01-11 17:30:09\tGenotypeConcordance\tchecked    18,460,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 9:68,990,079\n",
      "INFO\t2017-01-11 17:30:09\tGenotypeConcordance\tchecked    18,470,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 9:69,784,342\n",
      "INFO\t2017-01-11 17:30:10\tGenotypeConcordance\tchecked    18,480,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 9:70,944,657\n",
      "INFO\t2017-01-11 17:30:10\tGenotypeConcordance\tchecked    18,490,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 9:71,842,391\n",
      "INFO\t2017-01-11 17:30:10\tGenotypeConcordance\tchecked    18,500,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 9:72,743,932\n",
      "INFO\t2017-01-11 17:30:11\tGenotypeConcordance\tchecked    18,510,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 9:73,649,941\n",
      "INFO\t2017-01-11 17:30:11\tGenotypeConcordance\tchecked    18,520,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 9:74,555,136\n",
      "INFO\t2017-01-11 17:30:11\tGenotypeConcordance\tchecked    18,530,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 9:75,492,246\n",
      "INFO\t2017-01-11 17:30:11\tGenotypeConcordance\tchecked    18,540,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 9:76,491,744\n",
      "INFO\t2017-01-11 17:30:12\tGenotypeConcordance\tchecked    18,550,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 9:77,464,267\n",
      "INFO\t2017-01-11 17:30:12\tGenotypeConcordance\tchecked    18,560,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 9:78,294,951\n",
      "INFO\t2017-01-11 17:30:12\tGenotypeConcordance\tchecked    18,570,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 9:79,099,199\n",
      "INFO\t2017-01-11 17:30:13\tGenotypeConcordance\tchecked    18,580,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 9:79,849,874\n",
      "INFO\t2017-01-11 17:30:13\tGenotypeConcordance\tchecked    18,590,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 9:80,741,971\n",
      "INFO\t2017-01-11 17:30:13\tGenotypeConcordance\tchecked    18,600,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 9:81,539,515\n",
      "INFO\t2017-01-11 17:30:14\tGenotypeConcordance\tchecked    18,610,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 9:82,385,885\n",
      "INFO\t2017-01-11 17:30:14\tGenotypeConcordance\tchecked    18,620,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 9:83,179,764\n",
      "INFO\t2017-01-11 17:30:14\tGenotypeConcordance\tchecked    18,630,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 9:83,895,853\n",
      "INFO\t2017-01-11 17:30:14\tGenotypeConcordance\tchecked    18,640,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 9:84,900,687\n",
      "INFO\t2017-01-11 17:30:15\tGenotypeConcordance\tchecked    18,650,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 9:85,847,317\n",
      "INFO\t2017-01-11 17:30:15\tGenotypeConcordance\tchecked    18,660,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 9:86,738,065\n",
      "INFO\t2017-01-11 17:30:15\tGenotypeConcordance\tchecked    18,670,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 9:87,579,283\n",
      "INFO\t2017-01-11 17:30:16\tGenotypeConcordance\tchecked    18,680,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 9:88,487,791\n",
      "INFO\t2017-01-11 17:30:16\tGenotypeConcordance\tchecked    18,690,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 9:89,340,620\n",
      "INFO\t2017-01-11 17:30:16\tGenotypeConcordance\tchecked    18,700,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 9:90,141,514\n",
      "INFO\t2017-01-11 17:30:17\tGenotypeConcordance\tchecked    18,710,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 9:90,871,647\n",
      "INFO\t2017-01-11 17:30:17\tGenotypeConcordance\tchecked    18,720,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 9:91,590,324\n",
      "INFO\t2017-01-11 17:30:17\tGenotypeConcordance\tchecked    18,730,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 9:92,495,150\n",
      "INFO\t2017-01-11 17:30:17\tGenotypeConcordance\tchecked    18,740,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 9:93,451,481\n",
      "INFO\t2017-01-11 17:30:18\tGenotypeConcordance\tchecked    18,750,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 9:94,326,169\n",
      "INFO\t2017-01-11 17:30:18\tGenotypeConcordance\tchecked    18,760,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 9:95,210,570\n",
      "INFO\t2017-01-11 17:30:18\tGenotypeConcordance\tchecked    18,770,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 9:96,114,712\n",
      "INFO\t2017-01-11 17:30:19\tGenotypeConcordance\tchecked    18,780,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 9:97,010,498\n",
      "INFO\t2017-01-11 17:30:19\tGenotypeConcordance\tchecked    18,790,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 9:97,898,118\n",
      "INFO\t2017-01-11 17:30:19\tGenotypeConcordance\tchecked    18,800,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 9:98,839,797\n",
      "INFO\t2017-01-11 17:30:19\tGenotypeConcordance\tchecked    18,810,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 9:99,739,358\n",
      "INFO\t2017-01-11 17:30:20\tGenotypeConcordance\tchecked    18,820,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 9:100,704,638\n",
      "INFO\t2017-01-11 17:30:20\tGenotypeConcordance\tchecked    18,830,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:101,574,873\n",
      "INFO\t2017-01-11 17:30:20\tGenotypeConcordance\tchecked    18,840,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:102,651,712\n",
      "INFO\t2017-01-11 17:30:20\tGenotypeConcordance\tchecked    18,850,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:103,704,965\n",
      "INFO\t2017-01-11 17:30:21\tGenotypeConcordance\tchecked    18,860,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:104,605,646\n",
      "INFO\t2017-01-11 17:30:21\tGenotypeConcordance\tchecked    18,870,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:105,307,422\n",
      "INFO\t2017-01-11 17:30:21\tGenotypeConcordance\tchecked    18,880,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:106,024,523\n",
      "INFO\t2017-01-11 17:30:21\tGenotypeConcordance\tchecked    18,890,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:106,760,737\n",
      "INFO\t2017-01-11 17:30:22\tGenotypeConcordance\tchecked    18,900,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:107,525,317\n",
      "INFO\t2017-01-11 17:30:22\tGenotypeConcordance\tchecked    18,910,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:108,293,284\n",
      "INFO\t2017-01-11 17:30:22\tGenotypeConcordance\tchecked    18,920,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:109,106,490\n",
      "INFO\t2017-01-11 17:30:22\tGenotypeConcordance\tchecked    18,930,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:110,051,413\n",
      "INFO\t2017-01-11 17:30:23\tGenotypeConcordance\tchecked    18,940,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:110,801,732\n",
      "INFO\t2017-01-11 17:30:23\tGenotypeConcordance\tchecked    18,950,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:111,541,925\n",
      "INFO\t2017-01-11 17:30:23\tGenotypeConcordance\tchecked    18,960,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:112,419,640\n",
      "INFO\t2017-01-11 17:30:24\tGenotypeConcordance\tchecked    18,970,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:113,197,163\n",
      "INFO\t2017-01-11 17:30:24\tGenotypeConcordance\tchecked    18,980,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:113,996,326\n",
      "INFO\t2017-01-11 17:30:24\tGenotypeConcordance\tchecked    18,990,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:114,858,970\n",
      "INFO\t2017-01-11 17:30:24\tGenotypeConcordance\tchecked    19,000,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:115,728,200\n",
      "INFO\t2017-01-11 17:30:25\tGenotypeConcordance\tchecked    19,010,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:116,692,868\n",
      "INFO\t2017-01-11 17:30:25\tGenotypeConcordance\tchecked    19,020,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:117,505,696\n",
      "INFO\t2017-01-11 17:30:25\tGenotypeConcordance\tchecked    19,030,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:118,393,276\n",
      "INFO\t2017-01-11 17:30:26\tGenotypeConcordance\tchecked    19,040,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:119,383,881\n",
      "INFO\t2017-01-11 17:30:26\tGenotypeConcordance\tchecked    19,050,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:120,234,468\n",
      "INFO\t2017-01-11 17:30:26\tGenotypeConcordance\tchecked    19,060,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:121,146,490\n",
      "INFO\t2017-01-11 17:30:26\tGenotypeConcordance\tchecked    19,070,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:122,051,697\n",
      "INFO\t2017-01-11 17:30:27\tGenotypeConcordance\tchecked    19,080,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:123,018,415\n",
      "INFO\t2017-01-11 17:30:27\tGenotypeConcordance\tchecked    19,090,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:124,093,813\n",
      "INFO\t2017-01-11 17:30:27\tGenotypeConcordance\tchecked    19,100,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:124,977,324\n",
      "INFO\t2017-01-11 17:30:28\tGenotypeConcordance\tchecked    19,110,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:125,921,216\n",
      "INFO\t2017-01-11 17:30:28\tGenotypeConcordance\tchecked    19,120,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:126,913,488\n",
      "INFO\t2017-01-11 17:30:28\tGenotypeConcordance\tchecked    19,130,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:127,809,938\n",
      "INFO\t2017-01-11 17:30:28\tGenotypeConcordance\tchecked    19,140,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:128,791,885\n",
      "INFO\t2017-01-11 17:30:29\tGenotypeConcordance\tchecked    19,150,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:129,631,774\n",
      "INFO\t2017-01-11 17:30:29\tGenotypeConcordance\tchecked    19,160,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:130,680,447\n",
      "INFO\t2017-01-11 17:30:29\tGenotypeConcordance\tchecked    19,170,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:131,549,221\n",
      "INFO\t2017-01-11 17:30:30\tGenotypeConcordance\tchecked    19,180,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:132,249,683\n",
      "INFO\t2017-01-11 17:30:30\tGenotypeConcordance\tchecked    19,190,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:132,983,263\n",
      "INFO\t2017-01-11 17:30:30\tGenotypeConcordance\tchecked    19,200,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:133,887,443\n",
      "INFO\t2017-01-11 17:30:30\tGenotypeConcordance\tchecked    19,210,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:134,700,370\n",
      "INFO\t2017-01-11 17:30:31\tGenotypeConcordance\tchecked    19,220,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:135,558,065\n",
      "INFO\t2017-01-11 17:30:31\tGenotypeConcordance\tchecked    19,230,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:136,293,373\n",
      "INFO\t2017-01-11 17:30:31\tGenotypeConcordance\tchecked    19,240,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:136,949,180\n",
      "INFO\t2017-01-11 17:30:32\tGenotypeConcordance\tchecked    19,250,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:137,627,492\n",
      "INFO\t2017-01-11 17:30:32\tGenotypeConcordance\tchecked    19,260,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:138,204,537\n",
      "INFO\t2017-01-11 17:30:32\tGenotypeConcordance\tchecked    19,270,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:138,839,570\n",
      "INFO\t2017-01-11 17:30:32\tGenotypeConcordance\tchecked    19,280,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:139,567,127\n",
      "INFO\t2017-01-11 17:30:33\tGenotypeConcordance\tchecked    19,290,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:140,296,257\n",
      "INFO\t2017-01-11 17:30:33\tGenotypeConcordance\tchecked    19,300,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 9:141,081,573\n",
      "INFO\t2017-01-11 17:30:33\tGenotypeConcordance\tchecked    19,310,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 10:725,681\n",
      "INFO\t2017-01-11 17:30:33\tGenotypeConcordance\tchecked    19,320,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 10:1,415,982\n",
      "INFO\t2017-01-11 17:30:34\tGenotypeConcordance\tchecked    19,330,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 10:2,048,779\n",
      "INFO\t2017-01-11 17:30:34\tGenotypeConcordance\tchecked    19,340,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 10:2,669,744\n",
      "INFO\t2017-01-11 17:30:34\tGenotypeConcordance\tchecked    19,350,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 10:3,237,782\n",
      "INFO\t2017-01-11 17:30:34\tGenotypeConcordance\tchecked    19,360,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 10:3,972,634\n",
      "INFO\t2017-01-11 17:30:35\tGenotypeConcordance\tchecked    19,370,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 10:4,660,761\n",
      "INFO\t2017-01-11 17:30:35\tGenotypeConcordance\tchecked    19,380,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 10:5,265,216\n",
      "INFO\t2017-01-11 17:30:35\tGenotypeConcordance\tchecked    19,390,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 10:5,931,011\n",
      "INFO\t2017-01-11 17:30:36\tGenotypeConcordance\tchecked    19,400,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 10:6,645,899\n",
      "INFO\t2017-01-11 17:30:36\tGenotypeConcordance\tchecked    19,410,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 10:7,286,305\n",
      "INFO\t2017-01-11 17:30:36\tGenotypeConcordance\tchecked    19,420,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 10:8,018,107\n",
      "INFO\t2017-01-11 17:30:36\tGenotypeConcordance\tchecked    19,430,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 10:8,728,517\n",
      "INFO\t2017-01-11 17:30:37\tGenotypeConcordance\tchecked    19,440,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 10:9,561,105\n",
      "INFO\t2017-01-11 17:30:37\tGenotypeConcordance\tchecked    19,450,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 10:10,380,004\n",
      "INFO\t2017-01-11 17:30:37\tGenotypeConcordance\tchecked    19,460,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 10:11,181,950\n",
      "INFO\t2017-01-11 17:30:37\tGenotypeConcordance\tchecked    19,470,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 10:11,933,218\n",
      "INFO\t2017-01-11 17:30:38\tGenotypeConcordance\tchecked    19,480,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 10:12,616,887\n",
      "INFO\t2017-01-11 17:30:38\tGenotypeConcordance\tchecked    19,490,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 10:13,283,711\n",
      "INFO\t2017-01-11 17:30:38\tGenotypeConcordance\tchecked    19,500,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 10:13,966,471\n",
      "INFO\t2017-01-11 17:30:39\tGenotypeConcordance\tchecked    19,510,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 10:14,669,633\n",
      "INFO\t2017-01-11 17:30:39\tGenotypeConcordance\tchecked    19,520,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 10:15,373,886\n",
      "INFO\t2017-01-11 17:30:39\tGenotypeConcordance\tchecked    19,530,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 10:16,116,605\n",
      "INFO\t2017-01-11 17:30:39\tGenotypeConcordance\tchecked    19,540,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 10:16,809,597\n",
      "INFO\t2017-01-11 17:30:40\tGenotypeConcordance\tchecked    19,550,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 10:17,531,922\n",
      "INFO\t2017-01-11 17:30:40\tGenotypeConcordance\tchecked    19,560,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 10:18,586,608\n",
      "INFO\t2017-01-11 17:30:40\tGenotypeConcordance\tchecked    19,570,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 10:19,287,256\n",
      "INFO\t2017-01-11 17:30:41\tGenotypeConcordance\tchecked    19,580,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 10:19,941,090\n",
      "INFO\t2017-01-11 17:30:41\tGenotypeConcordance\tchecked    19,590,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 10:20,677,088\n",
      "INFO\t2017-01-11 17:30:41\tGenotypeConcordance\tchecked    19,600,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 10:21,386,755\n",
      "INFO\t2017-01-11 17:30:41\tGenotypeConcordance\tchecked    19,610,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 10:22,397,086\n",
      "INFO\t2017-01-11 17:30:42\tGenotypeConcordance\tchecked    19,620,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 10:23,293,921\n",
      "INFO\t2017-01-11 17:30:42\tGenotypeConcordance\tchecked    19,630,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 10:24,079,364\n",
      "INFO\t2017-01-11 17:30:42\tGenotypeConcordance\tchecked    19,640,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 10:24,880,195\n",
      "INFO\t2017-01-11 17:30:43\tGenotypeConcordance\tchecked    19,650,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 10:25,728,766\n",
      "INFO\t2017-01-11 17:30:43\tGenotypeConcordance\tchecked    19,660,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 10:26,497,157\n",
      "INFO\t2017-01-11 17:30:43\tGenotypeConcordance\tchecked    19,670,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 10:27,176,737\n",
      "INFO\t2017-01-11 17:30:43\tGenotypeConcordance\tchecked    19,680,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 10:27,899,408\n",
      "INFO\t2017-01-11 17:30:44\tGenotypeConcordance\tchecked    19,690,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 10:28,727,624\n",
      "INFO\t2017-01-11 17:30:44\tGenotypeConcordance\tchecked    19,700,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 10:29,464,255\n",
      "INFO\t2017-01-11 17:30:44\tGenotypeConcordance\tchecked    19,710,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 10:30,175,131\n",
      "INFO\t2017-01-11 17:30:45\tGenotypeConcordance\tchecked    19,720,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 10:30,892,831\n",
      "INFO\t2017-01-11 17:30:45\tGenotypeConcordance\tchecked    19,730,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 10:31,680,578\n",
      "INFO\t2017-01-11 17:30:45\tGenotypeConcordance\tchecked    19,740,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 10:32,538,430\n",
      "INFO\t2017-01-11 17:30:45\tGenotypeConcordance\tchecked    19,750,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 10:33,387,550\n",
      "INFO\t2017-01-11 17:30:46\tGenotypeConcordance\tchecked    19,760,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 10:34,202,005\n",
      "INFO\t2017-01-11 17:30:46\tGenotypeConcordance\tchecked    19,770,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 10:35,061,590\n",
      "INFO\t2017-01-11 17:30:46\tGenotypeConcordance\tchecked    19,780,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 10:35,984,031\n",
      "INFO\t2017-01-11 17:30:47\tGenotypeConcordance\tchecked    19,790,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 10:36,784,026\n",
      "INFO\t2017-01-11 17:30:47\tGenotypeConcordance\tchecked    19,800,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 10:37,586,233\n",
      "INFO\t2017-01-11 17:30:47\tGenotypeConcordance\tchecked    19,810,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 10:38,474,106\n",
      "INFO\t2017-01-11 17:30:47\tGenotypeConcordance\tchecked    19,820,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 10:38,982,460\n",
      "INFO\t2017-01-11 17:30:48\tGenotypeConcordance\tchecked    19,830,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 10:42,383,853\n",
      "INFO\t2017-01-11 17:30:48\tGenotypeConcordance\tchecked    19,840,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 10:42,630,572\n",
      "INFO\t2017-01-11 17:30:48\tGenotypeConcordance\tchecked    19,850,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 10:43,024,580\n",
      "INFO\t2017-01-11 17:30:49\tGenotypeConcordance\tchecked    19,860,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 10:43,820,851\n",
      "INFO\t2017-01-11 17:30:49\tGenotypeConcordance\tchecked    19,870,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 10:44,616,224\n",
      "INFO\t2017-01-11 17:30:49\tGenotypeConcordance\tchecked    19,880,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 10:45,375,110\n",
      "INFO\t2017-01-11 17:30:49\tGenotypeConcordance\tchecked    19,890,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 10:46,186,800\n",
      "INFO\t2017-01-11 17:30:50\tGenotypeConcordance\tchecked    19,900,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 10:47,698,150\n",
      "INFO\t2017-01-11 17:30:50\tGenotypeConcordance\tchecked    19,910,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 10:49,572,482\n",
      "INFO\t2017-01-11 17:30:50\tGenotypeConcordance\tchecked    19,920,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 10:50,385,880\n",
      "INFO\t2017-01-11 17:30:51\tGenotypeConcordance\tchecked    19,930,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 10:51,504,074\n",
      "INFO\t2017-01-11 17:30:51\tGenotypeConcordance\tchecked    19,940,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 10:52,590,708\n",
      "INFO\t2017-01-11 17:30:51\tGenotypeConcordance\tchecked    19,950,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 10:53,444,658\n",
      "INFO\t2017-01-11 17:30:51\tGenotypeConcordance\tchecked    19,960,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 10:54,219,820\n",
      "INFO\t2017-01-11 17:30:52\tGenotypeConcordance\tchecked    19,970,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 10:55,000,909\n",
      "INFO\t2017-01-11 17:30:52\tGenotypeConcordance\tchecked    19,980,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 10:55,728,299\n",
      "INFO\t2017-01-11 17:30:52\tGenotypeConcordance\tchecked    19,990,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 10:56,417,474\n",
      "INFO\t2017-01-11 17:30:52\tGenotypeConcordance\tchecked    20,000,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 10:57,209,676\n",
      "INFO\t2017-01-11 17:30:53\tGenotypeConcordance\tchecked    20,010,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 10:58,005,755\n",
      "INFO\t2017-01-11 17:30:53\tGenotypeConcordance\tchecked    20,020,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 10:58,784,814\n",
      "INFO\t2017-01-11 17:30:53\tGenotypeConcordance\tchecked    20,030,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 10:59,483,570\n",
      "INFO\t2017-01-11 17:30:54\tGenotypeConcordance\tchecked    20,040,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 10:60,314,183\n",
      "INFO\t2017-01-11 17:30:54\tGenotypeConcordance\tchecked    20,050,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 10:61,311,243\n",
      "INFO\t2017-01-11 17:30:54\tGenotypeConcordance\tchecked    20,060,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 10:62,136,212\n",
      "INFO\t2017-01-11 17:30:54\tGenotypeConcordance\tchecked    20,070,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 10:63,002,267\n",
      "INFO\t2017-01-11 17:30:55\tGenotypeConcordance\tchecked    20,080,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 10:63,918,915\n",
      "INFO\t2017-01-11 17:30:55\tGenotypeConcordance\tchecked    20,090,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 10:64,810,727\n",
      "INFO\t2017-01-11 17:30:55\tGenotypeConcordance\tchecked    20,100,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 10:65,743,503\n",
      "INFO\t2017-01-11 17:30:55\tGenotypeConcordance\tchecked    20,110,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 10:66,613,822\n",
      "INFO\t2017-01-11 17:30:56\tGenotypeConcordance\tchecked    20,120,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 10:67,321,688\n",
      "INFO\t2017-01-11 17:30:56\tGenotypeConcordance\tchecked    20,130,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 10:68,085,353\n",
      "INFO\t2017-01-11 17:30:56\tGenotypeConcordance\tchecked    20,140,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 10:68,788,942\n",
      "INFO\t2017-01-11 17:30:56\tGenotypeConcordance\tchecked    20,150,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 10:69,661,777\n",
      "INFO\t2017-01-11 17:30:57\tGenotypeConcordance\tchecked    20,160,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 10:70,491,316\n",
      "INFO\t2017-01-11 17:30:57\tGenotypeConcordance\tchecked    20,170,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 10:71,268,224\n",
      "INFO\t2017-01-11 17:30:57\tGenotypeConcordance\tchecked    20,180,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 10:72,058,768\n",
      "INFO\t2017-01-11 17:30:58\tGenotypeConcordance\tchecked    20,190,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 10:72,788,149\n",
      "INFO\t2017-01-11 17:30:58\tGenotypeConcordance\tchecked    20,200,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 10:73,578,704\n",
      "INFO\t2017-01-11 17:30:58\tGenotypeConcordance\tchecked    20,210,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 10:74,476,968\n",
      "INFO\t2017-01-11 17:30:58\tGenotypeConcordance\tchecked    20,220,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 10:75,568,839\n",
      "INFO\t2017-01-11 17:30:59\tGenotypeConcordance\tchecked    20,230,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:76,679,726\n",
      "INFO\t2017-01-11 17:30:59\tGenotypeConcordance\tchecked    20,240,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:77,737,701\n",
      "INFO\t2017-01-11 17:30:59\tGenotypeConcordance\tchecked    20,250,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:78,671,629\n",
      "INFO\t2017-01-11 17:31:00\tGenotypeConcordance\tchecked    20,260,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:79,600,256\n",
      "INFO\t2017-01-11 17:31:00\tGenotypeConcordance\tchecked    20,270,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:80,410,447\n",
      "INFO\t2017-01-11 17:31:00\tGenotypeConcordance\tchecked    20,280,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:81,229,468\n",
      "INFO\t2017-01-11 17:31:00\tGenotypeConcordance\tchecked    20,290,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:81,924,530\n",
      "INFO\t2017-01-11 17:31:01\tGenotypeConcordance\tchecked    20,300,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:82,720,794\n",
      "INFO\t2017-01-11 17:31:01\tGenotypeConcordance\tchecked    20,310,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:83,496,501\n",
      "INFO\t2017-01-11 17:31:01\tGenotypeConcordance\tchecked    20,320,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:84,358,581\n",
      "INFO\t2017-01-11 17:31:01\tGenotypeConcordance\tchecked    20,330,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:85,176,752\n",
      "INFO\t2017-01-11 17:31:02\tGenotypeConcordance\tchecked    20,340,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:85,947,354\n",
      "INFO\t2017-01-11 17:31:02\tGenotypeConcordance\tchecked    20,350,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:86,800,415\n",
      "INFO\t2017-01-11 17:31:02\tGenotypeConcordance\tchecked    20,360,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:87,535,013\n",
      "INFO\t2017-01-11 17:31:03\tGenotypeConcordance\tchecked    20,370,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:88,409,087\n",
      "INFO\t2017-01-11 17:31:03\tGenotypeConcordance\tchecked    20,380,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:89,361,512\n",
      "INFO\t2017-01-11 17:31:03\tGenotypeConcordance\tchecked    20,390,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:90,361,234\n",
      "INFO\t2017-01-11 17:31:04\tGenotypeConcordance\tchecked    20,400,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:91,291,104\n",
      "INFO\t2017-01-11 17:31:04\tGenotypeConcordance\tchecked    20,410,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:92,100,943\n",
      "INFO\t2017-01-11 17:31:04\tGenotypeConcordance\tchecked    20,420,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:92,980,926\n",
      "INFO\t2017-01-11 17:31:04\tGenotypeConcordance\tchecked    20,430,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:94,056,078\n",
      "INFO\t2017-01-11 17:31:05\tGenotypeConcordance\tchecked    20,440,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:95,024,704\n",
      "INFO\t2017-01-11 17:31:05\tGenotypeConcordance\tchecked    20,450,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:95,859,119\n",
      "INFO\t2017-01-11 17:31:05\tGenotypeConcordance\tchecked    20,460,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:96,752,130\n",
      "INFO\t2017-01-11 17:31:05\tGenotypeConcordance\tchecked    20,470,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:97,720,824\n",
      "INFO\t2017-01-11 17:31:06\tGenotypeConcordance\tchecked    20,480,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:98,703,175\n",
      "INFO\t2017-01-11 17:31:06\tGenotypeConcordance\tchecked    20,490,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:99,636,526\n",
      "INFO\t2017-01-11 17:31:06\tGenotypeConcordance\tchecked    20,500,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:100,526,502\n",
      "INFO\t2017-01-11 17:31:07\tGenotypeConcordance\tchecked    20,510,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:101,550,981\n",
      "INFO\t2017-01-11 17:31:07\tGenotypeConcordance\tchecked    20,520,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:102,454,375\n",
      "INFO\t2017-01-11 17:31:07\tGenotypeConcordance\tchecked    20,530,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:103,536,044\n",
      "INFO\t2017-01-11 17:31:07\tGenotypeConcordance\tchecked    20,540,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:104,765,420\n",
      "INFO\t2017-01-11 17:31:08\tGenotypeConcordance\tchecked    20,550,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:105,816,331\n",
      "INFO\t2017-01-11 17:31:08\tGenotypeConcordance\tchecked    20,560,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:106,773,878\n",
      "INFO\t2017-01-11 17:31:08\tGenotypeConcordance\tchecked    20,570,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:107,683,576\n",
      "INFO\t2017-01-11 17:31:09\tGenotypeConcordance\tchecked    20,580,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:108,541,839\n",
      "INFO\t2017-01-11 17:31:09\tGenotypeConcordance\tchecked    20,590,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:109,395,501\n",
      "INFO\t2017-01-11 17:31:09\tGenotypeConcordance\tchecked    20,600,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:110,150,313\n",
      "INFO\t2017-01-11 17:31:09\tGenotypeConcordance\tchecked    20,610,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:110,892,470\n",
      "INFO\t2017-01-11 17:31:10\tGenotypeConcordance\tchecked    20,620,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:111,688,998\n",
      "INFO\t2017-01-11 17:31:10\tGenotypeConcordance\tchecked    20,630,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:112,566,850\n",
      "INFO\t2017-01-11 17:31:10\tGenotypeConcordance\tchecked    20,640,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:113,370,069\n",
      "INFO\t2017-01-11 17:31:11\tGenotypeConcordance\tchecked    20,650,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:114,153,072\n",
      "INFO\t2017-01-11 17:31:11\tGenotypeConcordance\tchecked    20,660,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:115,110,707\n",
      "INFO\t2017-01-11 17:31:11\tGenotypeConcordance\tchecked    20,670,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:115,958,189\n",
      "INFO\t2017-01-11 17:31:11\tGenotypeConcordance\tchecked    20,680,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:116,837,002\n",
      "INFO\t2017-01-11 17:31:12\tGenotypeConcordance\tchecked    20,690,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:117,678,091\n",
      "INFO\t2017-01-11 17:31:12\tGenotypeConcordance\tchecked    20,700,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:118,540,025\n",
      "INFO\t2017-01-11 17:31:12\tGenotypeConcordance\tchecked    20,710,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:119,483,071\n",
      "INFO\t2017-01-11 17:31:13\tGenotypeConcordance\tchecked    20,720,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:120,415,610\n",
      "INFO\t2017-01-11 17:31:13\tGenotypeConcordance\tchecked    20,730,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:121,225,680\n",
      "INFO\t2017-01-11 17:31:13\tGenotypeConcordance\tchecked    20,740,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:122,033,585\n",
      "INFO\t2017-01-11 17:31:14\tGenotypeConcordance\tchecked    20,750,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:122,813,199\n",
      "INFO\t2017-01-11 17:31:14\tGenotypeConcordance\tchecked    20,760,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:123,582,994\n",
      "INFO\t2017-01-11 17:31:14\tGenotypeConcordance\tchecked    20,770,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:124,379,794\n",
      "INFO\t2017-01-11 17:31:14\tGenotypeConcordance\tchecked    20,780,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:125,264,951\n",
      "INFO\t2017-01-11 17:31:15\tGenotypeConcordance\tchecked    20,790,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:126,042,780\n",
      "INFO\t2017-01-11 17:31:15\tGenotypeConcordance\tchecked    20,800,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:126,795,646\n",
      "INFO\t2017-01-11 17:31:15\tGenotypeConcordance\tchecked    20,810,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:127,594,582\n",
      "INFO\t2017-01-11 17:31:16\tGenotypeConcordance\tchecked    20,820,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:128,338,095\n",
      "INFO\t2017-01-11 17:31:16\tGenotypeConcordance\tchecked    20,830,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:129,233,829\n",
      "INFO\t2017-01-11 17:31:16\tGenotypeConcordance\tchecked    20,840,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:129,926,565\n",
      "INFO\t2017-01-11 17:31:16\tGenotypeConcordance\tchecked    20,850,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:130,678,444\n",
      "INFO\t2017-01-11 17:31:17\tGenotypeConcordance\tchecked    20,860,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:131,414,478\n",
      "INFO\t2017-01-11 17:31:17\tGenotypeConcordance\tchecked    20,870,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:132,193,733\n",
      "INFO\t2017-01-11 17:31:17\tGenotypeConcordance\tchecked    20,880,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:132,832,275\n",
      "INFO\t2017-01-11 17:31:18\tGenotypeConcordance\tchecked    20,890,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:133,508,735\n",
      "INFO\t2017-01-11 17:31:18\tGenotypeConcordance\tchecked    20,900,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:134,177,217\n",
      "INFO\t2017-01-11 17:31:18\tGenotypeConcordance\tchecked    20,910,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:134,818,963\n",
      "INFO\t2017-01-11 17:31:18\tGenotypeConcordance\tchecked    20,920,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:135,444,061\n",
      "INFO\t2017-01-11 17:31:19\tGenotypeConcordance\tchecked    20,930,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 11:624,995\n",
      "INFO\t2017-01-11 17:31:19\tGenotypeConcordance\tchecked    20,940,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 11:1,344,447\n",
      "INFO\t2017-01-11 17:31:19\tGenotypeConcordance\tchecked    20,950,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 11:2,026,711\n",
      "INFO\t2017-01-11 17:31:20\tGenotypeConcordance\tchecked    20,960,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 11:2,748,347\n",
      "INFO\t2017-01-11 17:31:20\tGenotypeConcordance\tchecked    20,970,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 11:3,467,305\n",
      "INFO\t2017-01-11 17:31:20\tGenotypeConcordance\tchecked    20,980,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 11:4,122,736\n",
      "INFO\t2017-01-11 17:31:20\tGenotypeConcordance\tchecked    20,990,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 11:4,587,367\n",
      "INFO\t2017-01-11 17:31:21\tGenotypeConcordance\tchecked    21,000,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 11:5,234,024\n",
      "INFO\t2017-01-11 17:31:21\tGenotypeConcordance\tchecked    21,010,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 11:5,826,843\n",
      "INFO\t2017-01-11 17:31:21\tGenotypeConcordance\tchecked    21,020,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 11:6,615,123\n",
      "INFO\t2017-01-11 17:31:22\tGenotypeConcordance\tchecked    21,030,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 11:7,390,425\n",
      "INFO\t2017-01-11 17:31:22\tGenotypeConcordance\tchecked    21,040,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 11:8,045,050\n",
      "INFO\t2017-01-11 17:31:22\tGenotypeConcordance\tchecked    21,050,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 11:8,861,669\n",
      "INFO\t2017-01-11 17:31:23\tGenotypeConcordance\tchecked    21,060,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 11:9,729,626\n",
      "INFO\t2017-01-11 17:31:23\tGenotypeConcordance\tchecked    21,070,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 11:10,567,802\n",
      "INFO\t2017-01-11 17:31:23\tGenotypeConcordance\tchecked    21,080,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 11:11,327,295\n",
      "INFO\t2017-01-11 17:31:23\tGenotypeConcordance\tchecked    21,090,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 11:12,127,181\n",
      "INFO\t2017-01-11 17:31:24\tGenotypeConcordance\tchecked    21,100,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 11:12,931,825\n",
      "INFO\t2017-01-11 17:31:24\tGenotypeConcordance\tchecked    21,110,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 11:13,771,997\n",
      "INFO\t2017-01-11 17:31:24\tGenotypeConcordance\tchecked    21,120,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 11:14,772,334\n",
      "INFO\t2017-01-11 17:31:24\tGenotypeConcordance\tchecked    21,130,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 11:15,733,176\n",
      "INFO\t2017-01-11 17:31:25\tGenotypeConcordance\tchecked    21,140,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 11:16,814,479\n",
      "INFO\t2017-01-11 17:31:25\tGenotypeConcordance\tchecked    21,150,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 11:17,679,578\n",
      "INFO\t2017-01-11 17:31:25\tGenotypeConcordance\tchecked    21,160,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 11:18,604,396\n",
      "INFO\t2017-01-11 17:31:26\tGenotypeConcordance\tchecked    21,170,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 11:19,418,971\n",
      "INFO\t2017-01-11 17:31:26\tGenotypeConcordance\tchecked    21,180,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 11:20,193,304\n",
      "INFO\t2017-01-11 17:31:26\tGenotypeConcordance\tchecked    21,190,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 11:20,971,402\n",
      "INFO\t2017-01-11 17:31:27\tGenotypeConcordance\tchecked    21,200,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 11:21,694,637\n",
      "INFO\t2017-01-11 17:31:27\tGenotypeConcordance\tchecked    21,210,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 11:22,426,734\n",
      "INFO\t2017-01-11 17:31:27\tGenotypeConcordance\tchecked    21,220,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 11:23,165,535\n",
      "INFO\t2017-01-11 17:31:27\tGenotypeConcordance\tchecked    21,230,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 11:23,854,110\n",
      "INFO\t2017-01-11 17:31:28\tGenotypeConcordance\tchecked    21,240,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 11:24,528,308\n",
      "INFO\t2017-01-11 17:31:28\tGenotypeConcordance\tchecked    21,250,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 11:25,194,744\n",
      "INFO\t2017-01-11 17:31:28\tGenotypeConcordance\tchecked    21,260,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 11:25,850,138\n",
      "INFO\t2017-01-11 17:31:29\tGenotypeConcordance\tchecked    21,270,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 11:26,530,867\n",
      "INFO\t2017-01-11 17:31:29\tGenotypeConcordance\tchecked    21,280,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 11:27,426,514\n",
      "INFO\t2017-01-11 17:31:29\tGenotypeConcordance\tchecked    21,290,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 11:28,639,341\n",
      "INFO\t2017-01-11 17:31:30\tGenotypeConcordance\tchecked    21,300,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 11:29,531,037\n",
      "INFO\t2017-01-11 17:31:30\tGenotypeConcordance\tchecked    21,310,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 11:30,400,212\n",
      "INFO\t2017-01-11 17:31:30\tGenotypeConcordance\tchecked    21,320,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 11:31,531,318\n",
      "INFO\t2017-01-11 17:31:30\tGenotypeConcordance\tchecked    21,330,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 11:32,485,030\n",
      "INFO\t2017-01-11 17:31:31\tGenotypeConcordance\tchecked    21,340,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 11:33,451,507\n",
      "INFO\t2017-01-11 17:31:31\tGenotypeConcordance\tchecked    21,350,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 11:34,306,865\n",
      "INFO\t2017-01-11 17:31:31\tGenotypeConcordance\tchecked    21,360,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 11:35,143,574\n",
      "INFO\t2017-01-11 17:31:32\tGenotypeConcordance\tchecked    21,370,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 11:36,008,210\n",
      "INFO\t2017-01-11 17:31:32\tGenotypeConcordance\tchecked    21,380,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 11:36,806,048\n",
      "INFO\t2017-01-11 17:31:32\tGenotypeConcordance\tchecked    21,390,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 11:37,632,706\n",
      "INFO\t2017-01-11 17:31:32\tGenotypeConcordance\tchecked    21,400,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 11:38,411,042\n",
      "INFO\t2017-01-11 17:31:33\tGenotypeConcordance\tchecked    21,410,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 11:39,225,437\n",
      "INFO\t2017-01-11 17:31:33\tGenotypeConcordance\tchecked    21,420,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 11:39,957,989\n",
      "INFO\t2017-01-11 17:31:33\tGenotypeConcordance\tchecked    21,430,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 11:40,812,047\n",
      "INFO\t2017-01-11 17:31:34\tGenotypeConcordance\tchecked    21,440,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 11:41,734,059\n",
      "INFO\t2017-01-11 17:31:34\tGenotypeConcordance\tchecked    21,450,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 11:42,461,347\n",
      "INFO\t2017-01-11 17:31:34\tGenotypeConcordance\tchecked    21,460,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 11:43,167,637\n",
      "INFO\t2017-01-11 17:31:34\tGenotypeConcordance\tchecked    21,470,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 11:44,075,440\n",
      "INFO\t2017-01-11 17:31:35\tGenotypeConcordance\tchecked    21,480,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 11:44,906,045\n",
      "INFO\t2017-01-11 17:31:35\tGenotypeConcordance\tchecked    21,490,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 11:45,781,007\n",
      "INFO\t2017-01-11 17:31:35\tGenotypeConcordance\tchecked    21,500,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 11:46,960,059\n",
      "INFO\t2017-01-11 17:31:35\tGenotypeConcordance\tchecked    21,510,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 11:48,082,064\n",
      "INFO\t2017-01-11 17:31:36\tGenotypeConcordance\tchecked    21,520,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 11:48,625,640\n",
      "INFO\t2017-01-11 17:31:36\tGenotypeConcordance\tchecked    21,530,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 11:49,199,515\n",
      "INFO\t2017-01-11 17:31:36\tGenotypeConcordance\tchecked    21,540,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 11:50,022,940\n",
      "INFO\t2017-01-11 17:31:37\tGenotypeConcordance\tchecked    21,550,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 11:50,640,670\n",
      "INFO\t2017-01-11 17:31:37\tGenotypeConcordance\tchecked    21,560,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 11:51,409,683\n",
      "INFO\t2017-01-11 17:31:37\tGenotypeConcordance\tchecked    21,570,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 11:54,978,599\n",
      "INFO\t2017-01-11 17:31:37\tGenotypeConcordance\tchecked    21,580,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 11:55,451,187\n",
      "INFO\t2017-01-11 17:31:38\tGenotypeConcordance\tchecked    21,590,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 11:56,124,183\n",
      "INFO\t2017-01-11 17:31:38\tGenotypeConcordance\tchecked    21,600,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 11:56,844,128\n",
      "INFO\t2017-01-11 17:31:38\tGenotypeConcordance\tchecked    21,610,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 11:57,873,300\n",
      "INFO\t2017-01-11 17:31:39\tGenotypeConcordance\tchecked    21,620,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 11:58,771,287\n",
      "INFO\t2017-01-11 17:31:39\tGenotypeConcordance\tchecked    21,630,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 11:59,659,528\n",
      "INFO\t2017-01-11 17:31:39\tGenotypeConcordance\tchecked    21,640,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 11:60,481,211\n",
      "INFO\t2017-01-11 17:31:39\tGenotypeConcordance\tchecked    21,650,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 11:61,456,576\n",
      "INFO\t2017-01-11 17:31:40\tGenotypeConcordance\tchecked    21,660,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 11:62,329,292\n",
      "INFO\t2017-01-11 17:31:40\tGenotypeConcordance\tchecked    21,670,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 11:63,182,615\n",
      "INFO\t2017-01-11 17:31:40\tGenotypeConcordance\tchecked    21,680,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 11:64,147,001\n",
      "INFO\t2017-01-11 17:31:41\tGenotypeConcordance\tchecked    21,690,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 11:65,167,904\n",
      "INFO\t2017-01-11 17:31:41\tGenotypeConcordance\tchecked    21,700,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 11:66,202,257\n",
      "INFO\t2017-01-11 17:31:41\tGenotypeConcordance\tchecked    21,710,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 11:67,375,162\n",
      "INFO\t2017-01-11 17:31:41\tGenotypeConcordance\tchecked    21,720,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 11:67,977,596\n",
      "INFO\t2017-01-11 17:31:42\tGenotypeConcordance\tchecked    21,730,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 11:68,825,475\n",
      "INFO\t2017-01-11 17:31:42\tGenotypeConcordance\tchecked    21,740,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 11:69,611,475\n",
      "INFO\t2017-01-11 17:31:42\tGenotypeConcordance\tchecked    21,750,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 11:70,426,352\n",
      "INFO\t2017-01-11 17:31:42\tGenotypeConcordance\tchecked    21,760,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 11:71,175,220\n",
      "INFO\t2017-01-11 17:31:43\tGenotypeConcordance\tchecked    21,770,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 11:71,772,465\n",
      "INFO\t2017-01-11 17:31:43\tGenotypeConcordance\tchecked    21,780,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 11:72,817,255\n",
      "INFO\t2017-01-11 17:31:43\tGenotypeConcordance\tchecked    21,790,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 11:73,735,375\n",
      "INFO\t2017-01-11 17:31:44\tGenotypeConcordance\tchecked    21,800,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 11:74,681,953\n",
      "INFO\t2017-01-11 17:31:44\tGenotypeConcordance\tchecked    21,810,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 11:75,672,886\n",
      "INFO\t2017-01-11 17:31:44\tGenotypeConcordance\tchecked    21,820,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 11:76,541,459\n",
      "INFO\t2017-01-11 17:31:44\tGenotypeConcordance\tchecked    21,830,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 11:77,481,797\n",
      "INFO\t2017-01-11 17:31:45\tGenotypeConcordance\tchecked    21,840,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:78,304,659\n",
      "INFO\t2017-01-11 17:31:45\tGenotypeConcordance\tchecked    21,850,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:79,144,077\n",
      "INFO\t2017-01-11 17:31:45\tGenotypeConcordance\tchecked    21,860,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:79,970,467\n",
      "INFO\t2017-01-11 17:31:46\tGenotypeConcordance\tchecked    21,870,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:80,775,922\n",
      "INFO\t2017-01-11 17:31:46\tGenotypeConcordance\tchecked    21,880,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:81,487,576\n",
      "INFO\t2017-01-11 17:31:46\tGenotypeConcordance\tchecked    21,890,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:82,314,055\n",
      "INFO\t2017-01-11 17:31:47\tGenotypeConcordance\tchecked    21,900,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:83,210,558\n",
      "INFO\t2017-01-11 17:31:47\tGenotypeConcordance\tchecked    21,910,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:84,029,214\n",
      "INFO\t2017-01-11 17:31:47\tGenotypeConcordance\tchecked    21,920,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:84,987,313\n",
      "INFO\t2017-01-11 17:31:48\tGenotypeConcordance\tchecked    21,930,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:85,902,035\n",
      "INFO\t2017-01-11 17:31:48\tGenotypeConcordance\tchecked    21,940,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:86,729,527\n",
      "INFO\t2017-01-11 17:31:48\tGenotypeConcordance\tchecked    21,950,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:87,474,301\n",
      "INFO\t2017-01-11 17:31:49\tGenotypeConcordance\tchecked    21,960,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:88,276,892\n",
      "INFO\t2017-01-11 17:31:49\tGenotypeConcordance\tchecked    21,970,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:89,087,643\n",
      "INFO\t2017-01-11 17:31:49\tGenotypeConcordance\tchecked    21,980,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:90,086,953\n",
      "INFO\t2017-01-11 17:31:49\tGenotypeConcordance\tchecked    21,990,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:90,839,732\n",
      "INFO\t2017-01-11 17:31:50\tGenotypeConcordance\tchecked    22,000,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:91,582,982\n",
      "INFO\t2017-01-11 17:31:50\tGenotypeConcordance\tchecked    22,010,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:92,434,078\n",
      "INFO\t2017-01-11 17:31:50\tGenotypeConcordance\tchecked    22,020,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:93,377,875\n",
      "INFO\t2017-01-11 17:31:51\tGenotypeConcordance\tchecked    22,030,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:94,158,534\n",
      "INFO\t2017-01-11 17:31:51\tGenotypeConcordance\tchecked    22,040,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:95,045,372\n",
      "INFO\t2017-01-11 17:31:51\tGenotypeConcordance\tchecked    22,050,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:95,923,127\n",
      "INFO\t2017-01-11 17:31:51\tGenotypeConcordance\tchecked    22,060,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:96,851,454\n",
      "INFO\t2017-01-11 17:31:52\tGenotypeConcordance\tchecked    22,070,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:97,651,857\n",
      "INFO\t2017-01-11 17:31:52\tGenotypeConcordance\tchecked    22,080,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:98,369,187\n",
      "INFO\t2017-01-11 17:31:52\tGenotypeConcordance\tchecked    22,090,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:99,042,908\n",
      "INFO\t2017-01-11 17:31:53\tGenotypeConcordance\tchecked    22,100,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:99,724,188\n",
      "INFO\t2017-01-11 17:31:53\tGenotypeConcordance\tchecked    22,110,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:100,476,938\n",
      "INFO\t2017-01-11 17:31:53\tGenotypeConcordance\tchecked    22,120,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:101,298,056\n",
      "INFO\t2017-01-11 17:31:54\tGenotypeConcordance\tchecked    22,130,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:102,112,044\n",
      "INFO\t2017-01-11 17:31:54\tGenotypeConcordance\tchecked    22,140,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:102,951,503\n",
      "INFO\t2017-01-11 17:31:54\tGenotypeConcordance\tchecked    22,150,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:103,766,176\n",
      "INFO\t2017-01-11 17:31:54\tGenotypeConcordance\tchecked    22,160,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:104,605,464\n",
      "INFO\t2017-01-11 17:31:55\tGenotypeConcordance\tchecked    22,170,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:105,453,453\n",
      "INFO\t2017-01-11 17:31:55\tGenotypeConcordance\tchecked    22,180,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:106,460,523\n",
      "INFO\t2017-01-11 17:31:55\tGenotypeConcordance\tchecked    22,190,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:107,338,209\n",
      "INFO\t2017-01-11 17:31:56\tGenotypeConcordance\tchecked    22,200,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:108,204,808\n",
      "INFO\t2017-01-11 17:31:56\tGenotypeConcordance\tchecked    22,210,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:109,098,928\n",
      "INFO\t2017-01-11 17:31:56\tGenotypeConcordance\tchecked    22,220,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:110,059,060\n",
      "INFO\t2017-01-11 17:31:56\tGenotypeConcordance\tchecked    22,230,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:110,934,323\n",
      "INFO\t2017-01-11 17:31:57\tGenotypeConcordance\tchecked    22,240,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:111,928,226\n",
      "INFO\t2017-01-11 17:31:57\tGenotypeConcordance\tchecked    22,250,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:112,789,916\n",
      "INFO\t2017-01-11 17:31:57\tGenotypeConcordance\tchecked    22,260,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:113,734,365\n",
      "INFO\t2017-01-11 17:31:58\tGenotypeConcordance\tchecked    22,270,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:114,509,142\n",
      "INFO\t2017-01-11 17:31:58\tGenotypeConcordance\tchecked    22,280,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:115,438,165\n",
      "INFO\t2017-01-11 17:31:58\tGenotypeConcordance\tchecked    22,290,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:116,307,582\n",
      "INFO\t2017-01-11 17:31:58\tGenotypeConcordance\tchecked    22,300,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:117,095,280\n",
      "INFO\t2017-01-11 17:31:59\tGenotypeConcordance\tchecked    22,310,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:117,920,100\n",
      "INFO\t2017-01-11 17:31:59\tGenotypeConcordance\tchecked    22,320,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:118,836,281\n",
      "INFO\t2017-01-11 17:31:59\tGenotypeConcordance\tchecked    22,330,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:119,664,304\n",
      "INFO\t2017-01-11 17:31:59\tGenotypeConcordance\tchecked    22,340,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:120,517,905\n",
      "INFO\t2017-01-11 17:32:00\tGenotypeConcordance\tchecked    22,350,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:121,416,589\n",
      "INFO\t2017-01-11 17:32:00\tGenotypeConcordance\tchecked    22,360,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:122,367,529\n",
      "INFO\t2017-01-11 17:32:00\tGenotypeConcordance\tchecked    22,370,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:123,124,838\n",
      "INFO\t2017-01-11 17:32:01\tGenotypeConcordance\tchecked    22,380,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:123,896,733\n",
      "INFO\t2017-01-11 17:32:01\tGenotypeConcordance\tchecked    22,390,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:124,713,023\n",
      "INFO\t2017-01-11 17:32:01\tGenotypeConcordance\tchecked    22,400,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:125,607,198\n",
      "INFO\t2017-01-11 17:32:01\tGenotypeConcordance\tchecked    22,410,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:126,448,571\n",
      "INFO\t2017-01-11 17:32:02\tGenotypeConcordance\tchecked    22,420,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:127,318,546\n",
      "INFO\t2017-01-11 17:32:02\tGenotypeConcordance\tchecked    22,430,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:128,178,450\n",
      "INFO\t2017-01-11 17:32:02\tGenotypeConcordance\tchecked    22,440,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:129,092,082\n",
      "INFO\t2017-01-11 17:32:03\tGenotypeConcordance\tchecked    22,450,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:129,895,088\n",
      "INFO\t2017-01-11 17:32:03\tGenotypeConcordance\tchecked    22,460,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:130,701,645\n",
      "INFO\t2017-01-11 17:32:03\tGenotypeConcordance\tchecked    22,470,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:131,435,048\n",
      "INFO\t2017-01-11 17:32:03\tGenotypeConcordance\tchecked    22,480,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:132,194,934\n",
      "INFO\t2017-01-11 17:32:04\tGenotypeConcordance\tchecked    22,490,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:132,985,714\n",
      "INFO\t2017-01-11 17:32:04\tGenotypeConcordance\tchecked    22,500,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:133,804,152\n",
      "INFO\t2017-01-11 17:32:04\tGenotypeConcordance\tchecked    22,510,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:134,541,267\n",
      "INFO\t2017-01-11 17:32:05\tGenotypeConcordance\tchecked    22,520,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 12:313,627\n",
      "INFO\t2017-01-11 17:32:05\tGenotypeConcordance\tchecked    22,530,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 12:1,097,096\n",
      "INFO\t2017-01-11 17:32:05\tGenotypeConcordance\tchecked    22,540,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 12:1,851,862\n",
      "INFO\t2017-01-11 17:32:05\tGenotypeConcordance\tchecked    22,550,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 12:2,680,852\n",
      "INFO\t2017-01-11 17:32:06\tGenotypeConcordance\tchecked    22,560,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 12:3,393,512\n",
      "INFO\t2017-01-11 17:32:06\tGenotypeConcordance\tchecked    22,570,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 12:4,206,830\n",
      "INFO\t2017-01-11 17:32:06\tGenotypeConcordance\tchecked    22,580,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 12:5,061,819\n",
      "INFO\t2017-01-11 17:32:07\tGenotypeConcordance\tchecked    22,590,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 12:5,919,979\n",
      "INFO\t2017-01-11 17:32:07\tGenotypeConcordance\tchecked    22,600,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 12:6,635,274\n",
      "INFO\t2017-01-11 17:32:07\tGenotypeConcordance\tchecked    22,610,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 12:7,572,005\n",
      "INFO\t2017-01-11 17:32:08\tGenotypeConcordance\tchecked    22,620,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 12:8,272,350\n",
      "INFO\t2017-01-11 17:32:08\tGenotypeConcordance\tchecked    22,630,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 12:8,851,166\n",
      "INFO\t2017-01-11 17:32:08\tGenotypeConcordance\tchecked    22,640,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 12:9,609,298\n",
      "INFO\t2017-01-11 17:32:09\tGenotypeConcordance\tchecked    22,650,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 12:10,398,906\n",
      "INFO\t2017-01-11 17:32:09\tGenotypeConcordance\tchecked    22,660,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 12:11,170,074\n",
      "INFO\t2017-01-11 17:32:09\tGenotypeConcordance\tchecked    22,670,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 12:11,717,489\n",
      "INFO\t2017-01-11 17:32:09\tGenotypeConcordance\tchecked    22,680,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 12:12,516,393\n",
      "INFO\t2017-01-11 17:32:10\tGenotypeConcordance\tchecked    22,690,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 12:13,370,829\n",
      "INFO\t2017-01-11 17:32:10\tGenotypeConcordance\tchecked    22,700,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 12:14,231,402\n",
      "INFO\t2017-01-11 17:32:10\tGenotypeConcordance\tchecked    22,710,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 12:15,091,702\n",
      "INFO\t2017-01-11 17:32:11\tGenotypeConcordance\tchecked    22,720,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 12:16,045,476\n",
      "INFO\t2017-01-11 17:32:11\tGenotypeConcordance\tchecked    22,730,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 12:17,041,838\n",
      "INFO\t2017-01-11 17:32:11\tGenotypeConcordance\tchecked    22,740,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 12:17,925,474\n",
      "INFO\t2017-01-11 17:32:12\tGenotypeConcordance\tchecked    22,750,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 12:18,742,458\n",
      "INFO\t2017-01-11 17:32:12\tGenotypeConcordance\tchecked    22,760,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 12:19,581,726\n",
      "INFO\t2017-01-11 17:32:12\tGenotypeConcordance\tchecked    22,770,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 12:20,343,474\n",
      "INFO\t2017-01-11 17:32:12\tGenotypeConcordance\tchecked    22,780,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 12:21,125,365\n",
      "INFO\t2017-01-11 17:32:13\tGenotypeConcordance\tchecked    22,790,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 12:21,909,730\n",
      "INFO\t2017-01-11 17:32:13\tGenotypeConcordance\tchecked    22,800,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 12:22,894,851\n",
      "INFO\t2017-01-11 17:32:13\tGenotypeConcordance\tchecked    22,810,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 12:23,803,265\n",
      "INFO\t2017-01-11 17:32:13\tGenotypeConcordance\tchecked    22,820,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 12:24,673,740\n",
      "INFO\t2017-01-11 17:32:14\tGenotypeConcordance\tchecked    22,830,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 12:25,517,238\n",
      "INFO\t2017-01-11 17:32:14\tGenotypeConcordance\tchecked    22,840,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 12:26,311,517\n",
      "INFO\t2017-01-11 17:32:14\tGenotypeConcordance\tchecked    22,850,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 12:27,206,030\n",
      "INFO\t2017-01-11 17:32:15\tGenotypeConcordance\tchecked    22,860,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 12:28,101,641\n",
      "INFO\t2017-01-11 17:32:15\tGenotypeConcordance\tchecked    22,870,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 12:29,041,646\n",
      "INFO\t2017-01-11 17:32:15\tGenotypeConcordance\tchecked    22,880,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 12:29,854,972\n",
      "INFO\t2017-01-11 17:32:15\tGenotypeConcordance\tchecked    22,890,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 12:30,534,912\n",
      "INFO\t2017-01-11 17:32:16\tGenotypeConcordance\tchecked    22,900,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 12:31,302,583\n",
      "INFO\t2017-01-11 17:32:16\tGenotypeConcordance\tchecked    22,910,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 12:32,062,348\n",
      "INFO\t2017-01-11 17:32:16\tGenotypeConcordance\tchecked    22,920,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 12:32,801,194\n",
      "INFO\t2017-01-11 17:32:17\tGenotypeConcordance\tchecked    22,930,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 12:33,534,368\n",
      "INFO\t2017-01-11 17:32:17\tGenotypeConcordance\tchecked    22,940,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 12:34,307,426\n",
      "INFO\t2017-01-11 17:32:17\tGenotypeConcordance\tchecked    22,950,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 12:34,791,956\n",
      "INFO\t2017-01-11 17:32:18\tGenotypeConcordance\tchecked    22,960,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 12:38,159,731\n",
      "INFO\t2017-01-11 17:32:18\tGenotypeConcordance\tchecked    22,970,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 12:38,595,450\n",
      "INFO\t2017-01-11 17:32:18\tGenotypeConcordance\tchecked    22,980,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 12:39,346,678\n",
      "INFO\t2017-01-11 17:32:18\tGenotypeConcordance\tchecked    22,990,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 12:40,200,736\n",
      "INFO\t2017-01-11 17:32:19\tGenotypeConcordance\tchecked    23,000,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 12:41,034,018\n",
      "INFO\t2017-01-11 17:32:19\tGenotypeConcordance\tchecked    23,010,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 12:41,844,645\n",
      "INFO\t2017-01-11 17:32:19\tGenotypeConcordance\tchecked    23,020,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 12:42,595,907\n",
      "INFO\t2017-01-11 17:32:20\tGenotypeConcordance\tchecked    23,030,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 12:43,439,383\n",
      "INFO\t2017-01-11 17:32:20\tGenotypeConcordance\tchecked    23,040,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 12:44,318,922\n",
      "INFO\t2017-01-11 17:32:20\tGenotypeConcordance\tchecked    23,050,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 12:45,220,052\n",
      "INFO\t2017-01-11 17:32:20\tGenotypeConcordance\tchecked    23,060,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 12:46,088,790\n",
      "INFO\t2017-01-11 17:32:21\tGenotypeConcordance\tchecked    23,070,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 12:47,018,302\n",
      "INFO\t2017-01-11 17:32:21\tGenotypeConcordance\tchecked    23,080,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 12:47,829,588\n",
      "INFO\t2017-01-11 17:32:21\tGenotypeConcordance\tchecked    23,090,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 12:48,658,456\n",
      "INFO\t2017-01-11 17:32:22\tGenotypeConcordance\tchecked    23,100,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 12:49,640,962\n",
      "INFO\t2017-01-11 17:32:22\tGenotypeConcordance\tchecked    23,110,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 12:50,652,083\n",
      "INFO\t2017-01-11 17:32:22\tGenotypeConcordance\tchecked    23,120,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 12:51,659,071\n",
      "INFO\t2017-01-11 17:32:22\tGenotypeConcordance\tchecked    23,130,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 12:52,602,013\n",
      "INFO\t2017-01-11 17:32:23\tGenotypeConcordance\tchecked    23,140,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 12:53,331,970\n",
      "INFO\t2017-01-11 17:32:23\tGenotypeConcordance\tchecked    23,150,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 12:54,302,849\n",
      "INFO\t2017-01-11 17:32:23\tGenotypeConcordance\tchecked    23,160,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 12:55,219,254\n",
      "INFO\t2017-01-11 17:32:24\tGenotypeConcordance\tchecked    23,170,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 12:55,933,792\n",
      "INFO\t2017-01-11 17:32:24\tGenotypeConcordance\tchecked    23,180,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 12:56,962,625\n",
      "INFO\t2017-01-11 17:32:24\tGenotypeConcordance\tchecked    23,190,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 12:58,042,281\n",
      "INFO\t2017-01-11 17:32:24\tGenotypeConcordance\tchecked    23,200,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 12:58,952,712\n",
      "INFO\t2017-01-11 17:32:25\tGenotypeConcordance\tchecked    23,210,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 12:59,842,989\n",
      "INFO\t2017-01-11 17:32:25\tGenotypeConcordance\tchecked    23,220,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 12:60,725,868\n",
      "INFO\t2017-01-11 17:32:25\tGenotypeConcordance\tchecked    23,230,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 12:61,498,477\n",
      "INFO\t2017-01-11 17:32:26\tGenotypeConcordance\tchecked    23,240,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 12:62,310,217\n",
      "INFO\t2017-01-11 17:32:26\tGenotypeConcordance\tchecked    23,250,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 12:63,265,984\n",
      "INFO\t2017-01-11 17:32:26\tGenotypeConcordance\tchecked    23,260,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 12:64,067,829\n",
      "INFO\t2017-01-11 17:32:26\tGenotypeConcordance\tchecked    23,270,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 12:64,998,755\n",
      "INFO\t2017-01-11 17:32:27\tGenotypeConcordance\tchecked    23,280,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 12:66,061,422\n",
      "INFO\t2017-01-11 17:32:27\tGenotypeConcordance\tchecked    23,290,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 12:66,983,046\n",
      "INFO\t2017-01-11 17:32:27\tGenotypeConcordance\tchecked    23,300,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 12:67,878,789\n",
      "INFO\t2017-01-11 17:32:28\tGenotypeConcordance\tchecked    23,310,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 12:68,778,009\n",
      "INFO\t2017-01-11 17:32:28\tGenotypeConcordance\tchecked    23,320,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 12:69,634,706\n",
      "INFO\t2017-01-11 17:32:28\tGenotypeConcordance\tchecked    23,330,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 12:70,481,733\n",
      "INFO\t2017-01-11 17:32:28\tGenotypeConcordance\tchecked    23,340,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 12:71,351,875\n",
      "INFO\t2017-01-11 17:32:29\tGenotypeConcordance\tchecked    23,350,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 12:72,245,731\n",
      "INFO\t2017-01-11 17:32:29\tGenotypeConcordance\tchecked    23,360,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 12:73,218,766\n",
      "INFO\t2017-01-11 17:32:29\tGenotypeConcordance\tchecked    23,370,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 12:74,069,440\n",
      "INFO\t2017-01-11 17:32:29\tGenotypeConcordance\tchecked    23,380,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 12:74,989,531\n",
      "INFO\t2017-01-11 17:32:30\tGenotypeConcordance\tchecked    23,390,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 12:75,880,173\n",
      "INFO\t2017-01-11 17:32:30\tGenotypeConcordance\tchecked    23,400,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 12:76,720,599\n",
      "INFO\t2017-01-11 17:32:30\tGenotypeConcordance\tchecked    23,410,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 12:77,652,904\n",
      "INFO\t2017-01-11 17:32:31\tGenotypeConcordance\tchecked    23,420,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 12:78,531,976\n",
      "INFO\t2017-01-11 17:32:31\tGenotypeConcordance\tchecked    23,430,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 12:79,403,944\n",
      "INFO\t2017-01-11 17:32:31\tGenotypeConcordance\tchecked    23,440,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 12:80,478,326\n",
      "INFO\t2017-01-11 17:32:31\tGenotypeConcordance\tchecked    23,450,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 12:81,409,164\n",
      "INFO\t2017-01-11 17:32:32\tGenotypeConcordance\tchecked    23,460,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 12:82,386,288\n",
      "INFO\t2017-01-11 17:32:32\tGenotypeConcordance\tchecked    23,470,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:83,310,584\n",
      "INFO\t2017-01-11 17:32:32\tGenotypeConcordance\tchecked    23,480,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:84,095,819\n",
      "INFO\t2017-01-11 17:32:33\tGenotypeConcordance\tchecked    23,490,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:84,945,384\n",
      "INFO\t2017-01-11 17:32:33\tGenotypeConcordance\tchecked    23,500,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:85,972,453\n",
      "INFO\t2017-01-11 17:32:33\tGenotypeConcordance\tchecked    23,510,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:86,849,237\n",
      "INFO\t2017-01-11 17:32:33\tGenotypeConcordance\tchecked    23,520,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:87,742,759\n",
      "INFO\t2017-01-11 17:32:34\tGenotypeConcordance\tchecked    23,530,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:88,774,181\n",
      "INFO\t2017-01-11 17:32:34\tGenotypeConcordance\tchecked    23,540,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:89,904,596\n",
      "INFO\t2017-01-11 17:32:34\tGenotypeConcordance\tchecked    23,550,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:90,989,605\n",
      "INFO\t2017-01-11 17:32:34\tGenotypeConcordance\tchecked    23,560,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:91,964,219\n",
      "INFO\t2017-01-11 17:32:35\tGenotypeConcordance\tchecked    23,570,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:92,862,368\n",
      "INFO\t2017-01-11 17:32:35\tGenotypeConcordance\tchecked    23,580,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:93,735,865\n",
      "INFO\t2017-01-11 17:32:35\tGenotypeConcordance\tchecked    23,590,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:94,580,465\n",
      "INFO\t2017-01-11 17:32:36\tGenotypeConcordance\tchecked    23,600,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:95,398,440\n",
      "INFO\t2017-01-11 17:32:36\tGenotypeConcordance\tchecked    23,610,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:96,216,288\n",
      "INFO\t2017-01-11 17:32:36\tGenotypeConcordance\tchecked    23,620,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:97,086,351\n",
      "INFO\t2017-01-11 17:32:37\tGenotypeConcordance\tchecked    23,630,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:97,956,918\n",
      "INFO\t2017-01-11 17:32:37\tGenotypeConcordance\tchecked    23,640,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:98,835,746\n",
      "INFO\t2017-01-11 17:32:37\tGenotypeConcordance\tchecked    23,650,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:99,799,739\n",
      "INFO\t2017-01-11 17:32:37\tGenotypeConcordance\tchecked    23,660,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:100,671,902\n",
      "INFO\t2017-01-11 17:32:38\tGenotypeConcordance\tchecked    23,670,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:101,581,368\n",
      "INFO\t2017-01-11 17:32:38\tGenotypeConcordance\tchecked    23,680,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:102,429,347\n",
      "INFO\t2017-01-11 17:32:38\tGenotypeConcordance\tchecked    23,690,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:103,419,959\n",
      "INFO\t2017-01-11 17:32:39\tGenotypeConcordance\tchecked    23,700,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:104,310,945\n",
      "INFO\t2017-01-11 17:32:39\tGenotypeConcordance\tchecked    23,710,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:105,172,880\n",
      "INFO\t2017-01-11 17:32:39\tGenotypeConcordance\tchecked    23,720,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:106,030,833\n",
      "INFO\t2017-01-11 17:32:39\tGenotypeConcordance\tchecked    23,730,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:106,871,530\n",
      "INFO\t2017-01-11 17:32:40\tGenotypeConcordance\tchecked    23,740,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:107,859,928\n",
      "INFO\t2017-01-11 17:32:40\tGenotypeConcordance\tchecked    23,750,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:108,704,733\n",
      "INFO\t2017-01-11 17:32:40\tGenotypeConcordance\tchecked    23,760,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:109,650,050\n",
      "INFO\t2017-01-11 17:32:41\tGenotypeConcordance\tchecked    23,770,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:110,582,833\n",
      "INFO\t2017-01-11 17:32:41\tGenotypeConcordance\tchecked    23,780,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:111,636,580\n",
      "INFO\t2017-01-11 17:32:41\tGenotypeConcordance\tchecked    23,790,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:112,926,772\n",
      "INFO\t2017-01-11 17:32:41\tGenotypeConcordance\tchecked    23,800,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:113,898,522\n",
      "INFO\t2017-01-11 17:32:42\tGenotypeConcordance\tchecked    23,810,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:114,655,737\n",
      "INFO\t2017-01-11 17:32:42\tGenotypeConcordance\tchecked    23,820,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:115,410,225\n",
      "INFO\t2017-01-11 17:32:42\tGenotypeConcordance\tchecked    23,830,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:116,185,637\n",
      "INFO\t2017-01-11 17:32:42\tGenotypeConcordance\tchecked    23,840,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:117,089,737\n",
      "INFO\t2017-01-11 17:32:43\tGenotypeConcordance\tchecked    23,850,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:117,926,094\n",
      "INFO\t2017-01-11 17:32:43\tGenotypeConcordance\tchecked    23,860,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:118,752,878\n",
      "INFO\t2017-01-11 17:32:43\tGenotypeConcordance\tchecked    23,870,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:119,577,532\n",
      "INFO\t2017-01-11 17:32:44\tGenotypeConcordance\tchecked    23,880,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:120,492,514\n",
      "INFO\t2017-01-11 17:32:44\tGenotypeConcordance\tchecked    23,890,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:121,395,940\n",
      "INFO\t2017-01-11 17:32:44\tGenotypeConcordance\tchecked    23,900,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:122,225,369\n",
      "INFO\t2017-01-11 17:32:45\tGenotypeConcordance\tchecked    23,910,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:123,183,113\n",
      "INFO\t2017-01-11 17:32:45\tGenotypeConcordance\tchecked    23,920,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:124,154,840\n",
      "INFO\t2017-01-11 17:32:45\tGenotypeConcordance\tchecked    23,930,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:124,965,596\n",
      "INFO\t2017-01-11 17:32:45\tGenotypeConcordance\tchecked    23,940,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:125,679,919\n",
      "INFO\t2017-01-11 17:32:46\tGenotypeConcordance\tchecked    23,950,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:126,466,142\n",
      "INFO\t2017-01-11 17:32:46\tGenotypeConcordance\tchecked    23,960,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:127,177,853\n",
      "INFO\t2017-01-11 17:32:46\tGenotypeConcordance\tchecked    23,970,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:127,868,209\n",
      "INFO\t2017-01-11 17:32:46\tGenotypeConcordance\tchecked    23,980,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:128,567,404\n",
      "INFO\t2017-01-11 17:32:47\tGenotypeConcordance\tchecked    23,990,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:129,219,398\n",
      "INFO\t2017-01-11 17:32:47\tGenotypeConcordance\tchecked    24,000,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:129,879,302\n",
      "INFO\t2017-01-11 17:32:47\tGenotypeConcordance\tchecked    24,010,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:130,567,576\n",
      "INFO\t2017-01-11 17:32:48\tGenotypeConcordance\tchecked    24,020,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:131,228,645\n",
      "INFO\t2017-01-11 17:32:48\tGenotypeConcordance\tchecked    24,030,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:131,861,871\n",
      "INFO\t2017-01-11 17:32:48\tGenotypeConcordance\tchecked    24,040,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:132,604,096\n",
      "INFO\t2017-01-11 17:32:48\tGenotypeConcordance\tchecked    24,050,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:133,251,435\n",
      "INFO\t2017-01-11 17:32:49\tGenotypeConcordance\tchecked    24,060,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 13:19,304,599\n",
      "INFO\t2017-01-11 17:32:49\tGenotypeConcordance\tchecked    24,070,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 13:19,762,694\n",
      "INFO\t2017-01-11 17:32:49\tGenotypeConcordance\tchecked    24,080,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 13:20,474,044\n",
      "INFO\t2017-01-11 17:32:50\tGenotypeConcordance\tchecked    24,090,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 13:21,260,126\n",
      "INFO\t2017-01-11 17:32:50\tGenotypeConcordance\tchecked    24,100,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 13:22,023,707\n",
      "INFO\t2017-01-11 17:32:50\tGenotypeConcordance\tchecked    24,110,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 13:22,778,689\n",
      "INFO\t2017-01-11 17:32:51\tGenotypeConcordance\tchecked    24,120,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 13:23,490,255\n",
      "INFO\t2017-01-11 17:32:51\tGenotypeConcordance\tchecked    24,130,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 13:24,110,002\n",
      "INFO\t2017-01-11 17:32:51\tGenotypeConcordance\tchecked    24,140,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 13:24,796,423\n",
      "INFO\t2017-01-11 17:32:52\tGenotypeConcordance\tchecked    24,150,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 13:25,465,752\n",
      "INFO\t2017-01-11 17:32:52\tGenotypeConcordance\tchecked    24,160,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 13:26,218,409\n",
      "INFO\t2017-01-11 17:32:52\tGenotypeConcordance\tchecked    24,170,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 13:26,976,591\n",
      "INFO\t2017-01-11 17:32:53\tGenotypeConcordance\tchecked    24,180,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 13:27,717,165\n",
      "INFO\t2017-01-11 17:32:53\tGenotypeConcordance\tchecked    24,190,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 13:28,488,883\n",
      "INFO\t2017-01-11 17:32:53\tGenotypeConcordance\tchecked    24,200,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 13:29,306,662\n",
      "INFO\t2017-01-11 17:32:53\tGenotypeConcordance\tchecked    24,210,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 13:30,084,970\n",
      "INFO\t2017-01-11 17:32:54\tGenotypeConcordance\tchecked    24,220,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 13:30,871,177\n",
      "INFO\t2017-01-11 17:32:54\tGenotypeConcordance\tchecked    24,230,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 13:31,714,045\n",
      "INFO\t2017-01-11 17:32:54\tGenotypeConcordance\tchecked    24,240,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 13:32,533,595\n",
      "INFO\t2017-01-11 17:32:55\tGenotypeConcordance\tchecked    24,250,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 13:33,477,112\n",
      "INFO\t2017-01-11 17:32:55\tGenotypeConcordance\tchecked    24,260,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 13:34,290,654\n",
      "INFO\t2017-01-11 17:32:55\tGenotypeConcordance\tchecked    24,270,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 13:35,142,002\n",
      "INFO\t2017-01-11 17:32:56\tGenotypeConcordance\tchecked    24,280,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 13:36,131,705\n",
      "INFO\t2017-01-11 17:32:56\tGenotypeConcordance\tchecked    24,290,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 13:36,999,746\n",
      "INFO\t2017-01-11 17:32:56\tGenotypeConcordance\tchecked    24,300,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 13:37,905,116\n",
      "INFO\t2017-01-11 17:32:56\tGenotypeConcordance\tchecked    24,310,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 13:38,687,006\n",
      "INFO\t2017-01-11 17:32:57\tGenotypeConcordance\tchecked    24,320,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 13:39,515,037\n",
      "INFO\t2017-01-11 17:32:57\tGenotypeConcordance\tchecked    24,330,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 13:40,312,048\n",
      "INFO\t2017-01-11 17:32:57\tGenotypeConcordance\tchecked    24,340,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 13:41,140,781\n",
      "INFO\t2017-01-11 17:32:58\tGenotypeConcordance\tchecked    24,350,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 13:42,029,712\n",
      "INFO\t2017-01-11 17:32:58\tGenotypeConcordance\tchecked    24,360,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 13:42,911,957\n",
      "INFO\t2017-01-11 17:32:58\tGenotypeConcordance\tchecked    24,370,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 13:43,729,296\n",
      "INFO\t2017-01-11 17:32:59\tGenotypeConcordance\tchecked    24,380,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 13:44,669,703\n",
      "INFO\t2017-01-11 17:32:59\tGenotypeConcordance\tchecked    24,390,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 13:45,663,718\n",
      "INFO\t2017-01-11 17:32:59\tGenotypeConcordance\tchecked    24,400,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 13:46,510,533\n",
      "INFO\t2017-01-11 17:32:59\tGenotypeConcordance\tchecked    24,410,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 13:47,405,736\n",
      "INFO\t2017-01-11 17:33:00\tGenotypeConcordance\tchecked    24,420,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 13:48,262,266\n",
      "INFO\t2017-01-11 17:33:00\tGenotypeConcordance\tchecked    24,430,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 13:49,145,424\n",
      "INFO\t2017-01-11 17:33:00\tGenotypeConcordance\tchecked    24,440,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 13:50,088,357\n",
      "INFO\t2017-01-11 17:33:01\tGenotypeConcordance\tchecked    24,450,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 13:51,105,300\n",
      "INFO\t2017-01-11 17:33:01\tGenotypeConcordance\tchecked    24,460,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 13:52,065,039\n",
      "INFO\t2017-01-11 17:33:01\tGenotypeConcordance\tchecked    24,470,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 13:53,059,356\n",
      "INFO\t2017-01-11 17:33:01\tGenotypeConcordance\tchecked    24,480,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 13:53,973,902\n",
      "INFO\t2017-01-11 17:33:02\tGenotypeConcordance\tchecked    24,490,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 13:54,971,013\n",
      "INFO\t2017-01-11 17:33:02\tGenotypeConcordance\tchecked    24,500,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 13:55,867,116\n",
      "INFO\t2017-01-11 17:33:02\tGenotypeConcordance\tchecked    24,510,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 13:56,783,486\n",
      "INFO\t2017-01-11 17:33:02\tGenotypeConcordance\tchecked    24,520,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 13:57,999,028\n",
      "INFO\t2017-01-11 17:33:03\tGenotypeConcordance\tchecked    24,530,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 13:59,108,671\n",
      "INFO\t2017-01-11 17:33:03\tGenotypeConcordance\tchecked    24,540,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 13:60,016,892\n",
      "INFO\t2017-01-11 17:33:03\tGenotypeConcordance\tchecked    24,550,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 13:61,003,976\n",
      "INFO\t2017-01-11 17:33:04\tGenotypeConcordance\tchecked    24,560,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 13:61,802,659\n",
      "INFO\t2017-01-11 17:33:04\tGenotypeConcordance\tchecked    24,570,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 13:62,598,484\n",
      "INFO\t2017-01-11 17:33:04\tGenotypeConcordance\tchecked    24,580,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 13:63,452,736\n",
      "INFO\t2017-01-11 17:33:04\tGenotypeConcordance\tchecked    24,590,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 13:64,125,797\n",
      "INFO\t2017-01-11 17:33:05\tGenotypeConcordance\tchecked    24,600,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 13:64,878,263\n",
      "INFO\t2017-01-11 17:33:05\tGenotypeConcordance\tchecked    24,610,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 13:65,774,822\n",
      "INFO\t2017-01-11 17:33:05\tGenotypeConcordance\tchecked    24,620,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 13:66,668,526\n",
      "INFO\t2017-01-11 17:33:06\tGenotypeConcordance\tchecked    24,630,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 13:67,589,688\n",
      "INFO\t2017-01-11 17:33:06\tGenotypeConcordance\tchecked    24,640,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 13:68,476,463\n",
      "INFO\t2017-01-11 17:33:06\tGenotypeConcordance\tchecked    24,650,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 13:69,344,408\n",
      "INFO\t2017-01-11 17:33:06\tGenotypeConcordance\tchecked    24,660,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 13:70,123,444\n",
      "INFO\t2017-01-11 17:33:07\tGenotypeConcordance\tchecked    24,670,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 13:70,875,105\n",
      "INFO\t2017-01-11 17:33:07\tGenotypeConcordance\tchecked    24,680,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 13:71,646,397\n",
      "INFO\t2017-01-11 17:33:07\tGenotypeConcordance\tchecked    24,690,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 13:72,550,851\n",
      "INFO\t2017-01-11 17:33:08\tGenotypeConcordance\tchecked    24,700,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 13:73,365,211\n",
      "INFO\t2017-01-11 17:33:08\tGenotypeConcordance\tchecked    24,710,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 13:74,216,457\n",
      "INFO\t2017-01-11 17:33:08\tGenotypeConcordance\tchecked    24,720,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 13:75,062,769\n",
      "INFO\t2017-01-11 17:33:08\tGenotypeConcordance\tchecked    24,730,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 13:75,922,295\n",
      "INFO\t2017-01-11 17:33:09\tGenotypeConcordance\tchecked    24,740,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 13:76,749,475\n",
      "INFO\t2017-01-11 17:33:09\tGenotypeConcordance\tchecked    24,750,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 13:77,592,009\n",
      "INFO\t2017-01-11 17:33:09\tGenotypeConcordance\tchecked    24,760,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 13:78,602,863\n",
      "INFO\t2017-01-11 17:33:10\tGenotypeConcordance\tchecked    24,770,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 13:79,430,982\n",
      "INFO\t2017-01-11 17:33:10\tGenotypeConcordance\tchecked    24,780,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 13:80,380,090\n",
      "INFO\t2017-01-11 17:33:10\tGenotypeConcordance\tchecked    24,790,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 13:81,290,711\n",
      "INFO\t2017-01-11 17:33:11\tGenotypeConcordance\tchecked    24,800,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 13:82,174,716\n",
      "INFO\t2017-01-11 17:33:11\tGenotypeConcordance\tchecked    24,810,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 13:83,086,550\n",
      "INFO\t2017-01-11 17:33:11\tGenotypeConcordance\tchecked    24,820,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 13:83,903,522\n",
      "INFO\t2017-01-11 17:33:11\tGenotypeConcordance\tchecked    24,830,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 13:84,728,732\n",
      "INFO\t2017-01-11 17:33:12\tGenotypeConcordance\tchecked    24,840,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 13:85,558,486\n",
      "INFO\t2017-01-11 17:33:12\tGenotypeConcordance\tchecked    24,850,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 13:86,333,194\n",
      "INFO\t2017-01-11 17:33:12\tGenotypeConcordance\tchecked    24,860,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 13:87,268,660\n",
      "INFO\t2017-01-11 17:33:13\tGenotypeConcordance\tchecked    24,870,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 13:88,145,991\n",
      "INFO\t2017-01-11 17:33:13\tGenotypeConcordance\tchecked    24,880,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 13:89,056,014\n",
      "INFO\t2017-01-11 17:33:13\tGenotypeConcordance\tchecked    24,890,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 13:89,983,106\n",
      "INFO\t2017-01-11 17:33:14\tGenotypeConcordance\tchecked    24,900,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 13:90,769,532\n",
      "INFO\t2017-01-11 17:33:14\tGenotypeConcordance\tchecked    24,910,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 13:91,614,449\n",
      "INFO\t2017-01-11 17:33:14\tGenotypeConcordance\tchecked    24,920,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 13:92,416,361\n",
      "INFO\t2017-01-11 17:33:14\tGenotypeConcordance\tchecked    24,930,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 13:93,247,541\n",
      "INFO\t2017-01-11 17:33:15\tGenotypeConcordance\tchecked    24,940,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 13:94,106,204\n",
      "INFO\t2017-01-11 17:33:15\tGenotypeConcordance\tchecked    24,950,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 13:94,992,827\n",
      "INFO\t2017-01-11 17:33:15\tGenotypeConcordance\tchecked    24,960,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 13:95,842,518\n",
      "INFO\t2017-01-11 17:33:16\tGenotypeConcordance\tchecked    24,970,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 13:96,724,928\n",
      "INFO\t2017-01-11 17:33:16\tGenotypeConcordance\tchecked    24,980,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 13:97,744,284\n",
      "INFO\t2017-01-11 17:33:16\tGenotypeConcordance\tchecked    24,990,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 13:98,678,011\n",
      "INFO\t2017-01-11 17:33:16\tGenotypeConcordance\tchecked    25,000,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 13:99,392,176\n",
      "INFO\t2017-01-11 17:33:17\tGenotypeConcordance\tchecked    25,010,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:100,239,768\n",
      "INFO\t2017-01-11 17:33:17\tGenotypeConcordance\tchecked    25,020,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:101,123,315\n",
      "INFO\t2017-01-11 17:33:17\tGenotypeConcordance\tchecked    25,030,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:101,962,466\n",
      "INFO\t2017-01-11 17:33:18\tGenotypeConcordance\tchecked    25,040,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:102,781,655\n",
      "INFO\t2017-01-11 17:33:18\tGenotypeConcordance\tchecked    25,050,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:103,642,152\n",
      "INFO\t2017-01-11 17:33:18\tGenotypeConcordance\tchecked    25,060,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:104,384,890\n",
      "INFO\t2017-01-11 17:33:18\tGenotypeConcordance\tchecked    25,070,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:105,118,015\n",
      "INFO\t2017-01-11 17:33:19\tGenotypeConcordance\tchecked    25,080,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:105,846,909\n",
      "INFO\t2017-01-11 17:33:19\tGenotypeConcordance\tchecked    25,090,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:106,559,339\n",
      "INFO\t2017-01-11 17:33:19\tGenotypeConcordance\tchecked    25,100,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:107,386,589\n",
      "INFO\t2017-01-11 17:33:20\tGenotypeConcordance\tchecked    25,110,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:108,174,428\n",
      "INFO\t2017-01-11 17:33:20\tGenotypeConcordance\tchecked    25,120,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:108,953,991\n",
      "INFO\t2017-01-11 17:33:20\tGenotypeConcordance\tchecked    25,130,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:109,669,163\n",
      "INFO\t2017-01-11 17:33:20\tGenotypeConcordance\tchecked    25,140,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:110,420,075\n",
      "INFO\t2017-01-11 17:33:21\tGenotypeConcordance\tchecked    25,150,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:111,132,820\n",
      "INFO\t2017-01-11 17:33:21\tGenotypeConcordance\tchecked    25,160,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:111,873,880\n",
      "INFO\t2017-01-11 17:33:21\tGenotypeConcordance\tchecked    25,170,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:112,661,858\n",
      "INFO\t2017-01-11 17:33:22\tGenotypeConcordance\tchecked    25,180,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:113,345,185\n",
      "INFO\t2017-01-11 17:33:22\tGenotypeConcordance\tchecked    25,190,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:114,011,135\n",
      "INFO\t2017-01-11 17:33:22\tGenotypeConcordance\tchecked    25,200,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:114,803,398\n",
      "INFO\t2017-01-11 17:33:22\tGenotypeConcordance\tchecked    25,210,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 14:19,265,633\n",
      "INFO\t2017-01-11 17:33:23\tGenotypeConcordance\tchecked    25,220,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 14:20,118,185\n",
      "INFO\t2017-01-11 17:33:23\tGenotypeConcordance\tchecked    25,230,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 14:20,713,788\n",
      "INFO\t2017-01-11 17:33:23\tGenotypeConcordance\tchecked    25,240,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 14:21,426,155\n",
      "INFO\t2017-01-11 17:33:24\tGenotypeConcordance\tchecked    25,250,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 14:22,226,035\n",
      "INFO\t2017-01-11 17:33:24\tGenotypeConcordance\tchecked    25,260,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 14:22,921,280\n",
      "INFO\t2017-01-11 17:33:24\tGenotypeConcordance\tchecked    25,270,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 14:23,718,040\n",
      "INFO\t2017-01-11 17:33:25\tGenotypeConcordance\tchecked    25,280,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 14:24,525,345\n",
      "INFO\t2017-01-11 17:33:25\tGenotypeConcordance\tchecked    25,290,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 14:25,424,736\n",
      "INFO\t2017-01-11 17:33:25\tGenotypeConcordance\tchecked    25,300,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 14:26,226,897\n",
      "INFO\t2017-01-11 17:33:26\tGenotypeConcordance\tchecked    25,310,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 14:27,142,849\n",
      "INFO\t2017-01-11 17:33:26\tGenotypeConcordance\tchecked    25,320,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 14:28,086,730\n",
      "INFO\t2017-01-11 17:33:26\tGenotypeConcordance\tchecked    25,330,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 14:28,882,960\n",
      "INFO\t2017-01-11 17:33:26\tGenotypeConcordance\tchecked    25,340,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 14:29,843,244\n",
      "INFO\t2017-01-11 17:33:27\tGenotypeConcordance\tchecked    25,350,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 14:30,803,579\n",
      "INFO\t2017-01-11 17:33:27\tGenotypeConcordance\tchecked    25,360,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 14:31,698,980\n",
      "INFO\t2017-01-11 17:33:27\tGenotypeConcordance\tchecked    25,370,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 14:32,508,962\n",
      "INFO\t2017-01-11 17:33:28\tGenotypeConcordance\tchecked    25,380,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 14:33,352,908\n",
      "INFO\t2017-01-11 17:33:28\tGenotypeConcordance\tchecked    25,390,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 14:34,186,012\n",
      "INFO\t2017-01-11 17:33:28\tGenotypeConcordance\tchecked    25,400,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 14:34,953,955\n",
      "INFO\t2017-01-11 17:33:29\tGenotypeConcordance\tchecked    25,410,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 14:35,801,851\n",
      "INFO\t2017-01-11 17:33:29\tGenotypeConcordance\tchecked    25,420,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 14:36,737,697\n",
      "INFO\t2017-01-11 17:33:29\tGenotypeConcordance\tchecked    25,430,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 14:37,651,543\n",
      "INFO\t2017-01-11 17:33:30\tGenotypeConcordance\tchecked    25,440,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 14:38,640,848\n",
      "INFO\t2017-01-11 17:33:30\tGenotypeConcordance\tchecked    25,450,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 14:39,535,695\n",
      "INFO\t2017-01-11 17:33:30\tGenotypeConcordance\tchecked    25,460,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 14:40,367,185\n",
      "INFO\t2017-01-11 17:33:31\tGenotypeConcordance\tchecked    25,470,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 14:41,101,973\n",
      "INFO\t2017-01-11 17:33:31\tGenotypeConcordance\tchecked    25,480,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 14:41,865,711\n",
      "INFO\t2017-01-11 17:33:31\tGenotypeConcordance\tchecked    25,490,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 14:42,694,914\n",
      "INFO\t2017-01-11 17:33:32\tGenotypeConcordance\tchecked    25,500,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 14:43,435,164\n",
      "INFO\t2017-01-11 17:33:32\tGenotypeConcordance\tchecked    25,510,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 14:44,159,974\n",
      "INFO\t2017-01-11 17:33:32\tGenotypeConcordance\tchecked    25,520,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 14:44,993,347\n",
      "INFO\t2017-01-11 17:33:32\tGenotypeConcordance\tchecked    25,530,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 14:45,986,130\n",
      "INFO\t2017-01-11 17:33:33\tGenotypeConcordance\tchecked    25,540,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 14:46,753,003\n",
      "INFO\t2017-01-11 17:33:33\tGenotypeConcordance\tchecked    25,550,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 14:47,608,303\n",
      "INFO\t2017-01-11 17:33:33\tGenotypeConcordance\tchecked    25,560,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 14:48,415,645\n",
      "INFO\t2017-01-11 17:33:34\tGenotypeConcordance\tchecked    25,570,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 14:49,196,486\n",
      "INFO\t2017-01-11 17:33:34\tGenotypeConcordance\tchecked    25,580,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 14:50,051,751\n",
      "INFO\t2017-01-11 17:33:34\tGenotypeConcordance\tchecked    25,590,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 14:50,956,974\n",
      "INFO\t2017-01-11 17:33:34\tGenotypeConcordance\tchecked    25,600,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 14:51,819,420\n",
      "INFO\t2017-01-11 17:33:35\tGenotypeConcordance\tchecked    25,610,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 14:52,626,461\n",
      "INFO\t2017-01-11 17:33:35\tGenotypeConcordance\tchecked    25,620,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 14:53,481,995\n",
      "INFO\t2017-01-11 17:33:35\tGenotypeConcordance\tchecked    25,630,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 14:54,322,781\n",
      "INFO\t2017-01-11 17:33:36\tGenotypeConcordance\tchecked    25,640,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 14:55,256,021\n",
      "INFO\t2017-01-11 17:33:36\tGenotypeConcordance\tchecked    25,650,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 14:56,071,798\n",
      "INFO\t2017-01-11 17:33:36\tGenotypeConcordance\tchecked    25,660,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 14:56,868,003\n",
      "INFO\t2017-01-11 17:33:36\tGenotypeConcordance\tchecked    25,670,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 14:57,692,107\n",
      "INFO\t2017-01-11 17:33:37\tGenotypeConcordance\tchecked    25,680,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 14:58,514,790\n",
      "INFO\t2017-01-11 17:33:37\tGenotypeConcordance\tchecked    25,690,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 14:59,519,143\n",
      "INFO\t2017-01-11 17:33:37\tGenotypeConcordance\tchecked    25,700,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 14:60,460,445\n",
      "INFO\t2017-01-11 17:33:37\tGenotypeConcordance\tchecked    25,710,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 14:61,510,680\n",
      "INFO\t2017-01-11 17:33:38\tGenotypeConcordance\tchecked    25,720,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 14:62,358,047\n",
      "INFO\t2017-01-11 17:33:38\tGenotypeConcordance\tchecked    25,730,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 14:63,140,769\n",
      "INFO\t2017-01-11 17:33:38\tGenotypeConcordance\tchecked    25,740,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 14:64,010,783\n",
      "INFO\t2017-01-11 17:33:39\tGenotypeConcordance\tchecked    25,750,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 14:64,914,537\n",
      "INFO\t2017-01-11 17:33:39\tGenotypeConcordance\tchecked    25,760,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 14:65,804,664\n",
      "INFO\t2017-01-11 17:33:39\tGenotypeConcordance\tchecked    25,770,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 14:66,697,945\n",
      "INFO\t2017-01-11 17:33:39\tGenotypeConcordance\tchecked    25,780,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 14:67,693,839\n",
      "INFO\t2017-01-11 17:33:40\tGenotypeConcordance\tchecked    25,790,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 14:68,685,460\n",
      "INFO\t2017-01-11 17:33:40\tGenotypeConcordance\tchecked    25,800,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 14:69,593,656\n",
      "INFO\t2017-01-11 17:33:40\tGenotypeConcordance\tchecked    25,810,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 14:70,479,078\n",
      "INFO\t2017-01-11 17:33:40\tGenotypeConcordance\tchecked    25,820,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 14:71,301,755\n",
      "INFO\t2017-01-11 17:33:41\tGenotypeConcordance\tchecked    25,830,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 14:72,309,303\n",
      "INFO\t2017-01-11 17:33:41\tGenotypeConcordance\tchecked    25,840,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 14:73,076,504\n",
      "INFO\t2017-01-11 17:33:41\tGenotypeConcordance\tchecked    25,850,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 14:73,967,895\n",
      "INFO\t2017-01-11 17:33:42\tGenotypeConcordance\tchecked    25,860,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 14:74,749,606\n",
      "INFO\t2017-01-11 17:33:42\tGenotypeConcordance\tchecked    25,870,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 14:75,724,473\n",
      "INFO\t2017-01-11 17:33:42\tGenotypeConcordance\tchecked    25,880,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 14:76,659,723\n",
      "INFO\t2017-01-11 17:33:42\tGenotypeConcordance\tchecked    25,890,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 14:77,436,787\n",
      "INFO\t2017-01-11 17:33:43\tGenotypeConcordance\tchecked    25,900,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 14:78,239,259\n",
      "INFO\t2017-01-11 17:33:43\tGenotypeConcordance\tchecked    25,910,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 14:79,104,652\n",
      "INFO\t2017-01-11 17:33:43\tGenotypeConcordance\tchecked    25,920,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 14:80,016,502\n",
      "INFO\t2017-01-11 17:33:44\tGenotypeConcordance\tchecked    25,930,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 14:80,931,119\n",
      "INFO\t2017-01-11 17:33:44\tGenotypeConcordance\tchecked    25,940,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 14:81,836,996\n",
      "INFO\t2017-01-11 17:33:44\tGenotypeConcordance\tchecked    25,950,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 14:82,673,878\n",
      "INFO\t2017-01-11 17:33:45\tGenotypeConcordance\tchecked    25,960,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 14:83,528,049\n",
      "INFO\t2017-01-11 17:33:45\tGenotypeConcordance\tchecked    25,970,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 14:84,402,471\n",
      "INFO\t2017-01-11 17:33:45\tGenotypeConcordance\tchecked    25,980,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 14:85,234,218\n",
      "INFO\t2017-01-11 17:33:45\tGenotypeConcordance\tchecked    25,990,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 14:86,098,183\n",
      "INFO\t2017-01-11 17:33:46\tGenotypeConcordance\tchecked    26,000,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 14:86,839,203\n",
      "INFO\t2017-01-11 17:33:46\tGenotypeConcordance\tchecked    26,010,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 14:87,705,702\n",
      "INFO\t2017-01-11 17:33:46\tGenotypeConcordance\tchecked    26,020,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 14:88,489,630\n",
      "INFO\t2017-01-11 17:33:47\tGenotypeConcordance\tchecked    26,030,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 14:89,384,118\n",
      "INFO\t2017-01-11 17:33:47\tGenotypeConcordance\tchecked    26,040,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 14:90,169,202\n",
      "INFO\t2017-01-11 17:33:47\tGenotypeConcordance\tchecked    26,050,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 14:90,998,817\n",
      "INFO\t2017-01-11 17:33:47\tGenotypeConcordance\tchecked    26,060,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 14:91,845,588\n",
      "INFO\t2017-01-11 17:33:48\tGenotypeConcordance\tchecked    26,070,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 14:92,692,952\n",
      "INFO\t2017-01-11 17:33:48\tGenotypeConcordance\tchecked    26,080,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 14:93,461,185\n",
      "INFO\t2017-01-11 17:33:48\tGenotypeConcordance\tchecked    26,090,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 14:94,338,490\n",
      "INFO\t2017-01-11 17:33:48\tGenotypeConcordance\tchecked    26,100,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 14:95,099,611\n",
      "INFO\t2017-01-11 17:33:49\tGenotypeConcordance\tchecked    26,110,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 14:95,831,101\n",
      "INFO\t2017-01-11 17:33:49\tGenotypeConcordance\tchecked    26,120,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 14:96,525,596\n",
      "INFO\t2017-01-11 17:33:49\tGenotypeConcordance\tchecked    26,130,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 14:97,350,173\n",
      "INFO\t2017-01-11 17:33:50\tGenotypeConcordance\tchecked    26,140,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 14:98,181,180\n",
      "INFO\t2017-01-11 17:33:50\tGenotypeConcordance\tchecked    26,150,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 14:98,901,493\n",
      "INFO\t2017-01-11 17:33:50\tGenotypeConcordance\tchecked    26,160,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 14:99,647,028\n",
      "INFO\t2017-01-11 17:33:51\tGenotypeConcordance\tchecked    26,170,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 14:100,483,129\n",
      "INFO\t2017-01-11 17:33:51\tGenotypeConcordance\tchecked    26,180,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 14:101,306,685\n",
      "INFO\t2017-01-11 17:33:51\tGenotypeConcordance\tchecked    26,190,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 14:102,126,242\n",
      "INFO\t2017-01-11 17:33:51\tGenotypeConcordance\tchecked    26,200,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 14:102,987,286\n",
      "INFO\t2017-01-11 17:33:52\tGenotypeConcordance\tchecked    26,210,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 14:103,772,717\n",
      "INFO\t2017-01-11 17:33:52\tGenotypeConcordance\tchecked    26,220,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:104,587,869\n",
      "INFO\t2017-01-11 17:33:52\tGenotypeConcordance\tchecked    26,230,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:105,236,672\n",
      "INFO\t2017-01-11 17:33:52\tGenotypeConcordance\tchecked    26,240,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:105,978,010\n",
      "INFO\t2017-01-11 17:33:53\tGenotypeConcordance\tchecked    26,250,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:106,728,317\n",
      "INFO\t2017-01-11 17:33:53\tGenotypeConcordance\tchecked    26,260,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 15:20,019,922\n",
      "INFO\t2017-01-11 17:33:53\tGenotypeConcordance\tchecked    26,270,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 15:20,492,132\n",
      "INFO\t2017-01-11 17:33:54\tGenotypeConcordance\tchecked    26,280,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 15:21,887,545\n",
      "INFO\t2017-01-11 17:33:54\tGenotypeConcordance\tchecked    26,290,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 15:22,576,643\n",
      "INFO\t2017-01-11 17:33:54\tGenotypeConcordance\tchecked    26,300,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 15:23,506,879\n",
      "INFO\t2017-01-11 17:33:54\tGenotypeConcordance\tchecked    26,310,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 15:24,320,566\n",
      "INFO\t2017-01-11 17:33:55\tGenotypeConcordance\tchecked    26,320,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 15:24,758,125\n",
      "INFO\t2017-01-11 17:33:55\tGenotypeConcordance\tchecked    26,330,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 15:25,391,168\n",
      "INFO\t2017-01-11 17:33:55\tGenotypeConcordance\tchecked    26,340,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 15:26,145,292\n",
      "INFO\t2017-01-11 17:33:56\tGenotypeConcordance\tchecked    26,350,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 15:26,880,157\n",
      "INFO\t2017-01-11 17:33:56\tGenotypeConcordance\tchecked    26,360,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 15:27,682,851\n",
      "INFO\t2017-01-11 17:33:56\tGenotypeConcordance\tchecked    26,370,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 15:28,481,743\n",
      "INFO\t2017-01-11 17:33:56\tGenotypeConcordance\tchecked    26,380,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 15:29,454,669\n",
      "INFO\t2017-01-11 17:33:57\tGenotypeConcordance\tchecked    26,390,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 15:30,269,669\n",
      "INFO\t2017-01-11 17:33:57\tGenotypeConcordance\tchecked    26,400,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 15:31,427,069\n",
      "INFO\t2017-01-11 17:33:57\tGenotypeConcordance\tchecked    26,410,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 15:32,255,382\n",
      "INFO\t2017-01-11 17:33:58\tGenotypeConcordance\tchecked    26,420,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 15:33,259,773\n",
      "INFO\t2017-01-11 17:33:58\tGenotypeConcordance\tchecked    26,430,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 15:33,950,236\n",
      "INFO\t2017-01-11 17:33:58\tGenotypeConcordance\tchecked    26,440,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 15:34,689,957\n",
      "INFO\t2017-01-11 17:33:58\tGenotypeConcordance\tchecked    26,450,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 15:35,516,664\n",
      "INFO\t2017-01-11 17:33:59\tGenotypeConcordance\tchecked    26,460,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 15:36,481,940\n",
      "INFO\t2017-01-11 17:33:59\tGenotypeConcordance\tchecked    26,470,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 15:37,395,212\n",
      "INFO\t2017-01-11 17:33:59\tGenotypeConcordance\tchecked    26,480,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 15:38,231,284\n",
      "INFO\t2017-01-11 17:34:00\tGenotypeConcordance\tchecked    26,490,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 15:39,119,986\n",
      "INFO\t2017-01-11 17:34:00\tGenotypeConcordance\tchecked    26,500,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 15:39,845,432\n",
      "INFO\t2017-01-11 17:34:00\tGenotypeConcordance\tchecked    26,510,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 15:40,747,604\n",
      "INFO\t2017-01-11 17:34:00\tGenotypeConcordance\tchecked    26,520,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 15:41,674,132\n",
      "INFO\t2017-01-11 17:34:01\tGenotypeConcordance\tchecked    26,530,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 15:42,589,729\n",
      "INFO\t2017-01-11 17:34:01\tGenotypeConcordance\tchecked    26,540,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 15:43,677,568\n",
      "INFO\t2017-01-11 17:34:01\tGenotypeConcordance\tchecked    26,550,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 15:44,858,941\n",
      "INFO\t2017-01-11 17:34:02\tGenotypeConcordance\tchecked    26,560,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 15:45,749,030\n",
      "INFO\t2017-01-11 17:34:02\tGenotypeConcordance\tchecked    26,570,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 15:46,574,981\n",
      "INFO\t2017-01-11 17:34:02\tGenotypeConcordance\tchecked    26,580,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 15:47,420,677\n",
      "INFO\t2017-01-11 17:34:03\tGenotypeConcordance\tchecked    26,590,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 15:48,354,993\n",
      "INFO\t2017-01-11 17:34:03\tGenotypeConcordance\tchecked    26,600,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 15:49,460,678\n",
      "INFO\t2017-01-11 17:34:03\tGenotypeConcordance\tchecked    26,610,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 15:50,437,483\n",
      "INFO\t2017-01-11 17:34:04\tGenotypeConcordance\tchecked    26,620,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 15:51,287,916\n",
      "INFO\t2017-01-11 17:34:04\tGenotypeConcordance\tchecked    26,630,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 15:52,259,467\n",
      "INFO\t2017-01-11 17:34:04\tGenotypeConcordance\tchecked    26,640,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 15:53,111,692\n",
      "INFO\t2017-01-11 17:34:04\tGenotypeConcordance\tchecked    26,650,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 15:53,876,637\n",
      "INFO\t2017-01-11 17:34:05\tGenotypeConcordance\tchecked    26,660,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 15:54,578,397\n",
      "INFO\t2017-01-11 17:34:05\tGenotypeConcordance\tchecked    26,670,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 15:55,257,869\n",
      "INFO\t2017-01-11 17:34:05\tGenotypeConcordance\tchecked    26,680,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 15:56,040,341\n",
      "INFO\t2017-01-11 17:34:06\tGenotypeConcordance\tchecked    26,690,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 15:56,866,113\n",
      "INFO\t2017-01-11 17:34:06\tGenotypeConcordance\tchecked    26,700,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 15:57,762,045\n",
      "INFO\t2017-01-11 17:34:06\tGenotypeConcordance\tchecked    26,710,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 15:58,473,395\n",
      "INFO\t2017-01-11 17:34:07\tGenotypeConcordance\tchecked    26,720,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 15:59,270,634\n",
      "INFO\t2017-01-11 17:34:07\tGenotypeConcordance\tchecked    26,730,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 15:60,013,216\n",
      "INFO\t2017-01-11 17:34:07\tGenotypeConcordance\tchecked    26,740,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 15:60,779,014\n",
      "INFO\t2017-01-11 17:34:07\tGenotypeConcordance\tchecked    26,750,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 15:61,639,757\n",
      "INFO\t2017-01-11 17:34:08\tGenotypeConcordance\tchecked    26,760,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 15:62,471,025\n",
      "INFO\t2017-01-11 17:34:08\tGenotypeConcordance\tchecked    26,770,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 15:63,249,167\n",
      "INFO\t2017-01-11 17:34:08\tGenotypeConcordance\tchecked    26,780,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 15:64,141,036\n",
      "INFO\t2017-01-11 17:34:09\tGenotypeConcordance\tchecked    26,790,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 15:65,114,039\n",
      "INFO\t2017-01-11 17:34:09\tGenotypeConcordance\tchecked    26,800,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 15:66,020,432\n",
      "INFO\t2017-01-11 17:34:09\tGenotypeConcordance\tchecked    26,810,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 15:66,861,241\n",
      "INFO\t2017-01-11 17:34:09\tGenotypeConcordance\tchecked    26,820,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 15:67,714,571\n",
      "INFO\t2017-01-11 17:34:10\tGenotypeConcordance\tchecked    26,830,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 15:68,671,717\n",
      "INFO\t2017-01-11 17:34:10\tGenotypeConcordance\tchecked    26,840,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 15:69,559,255\n",
      "INFO\t2017-01-11 17:34:10\tGenotypeConcordance\tchecked    26,850,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 15:70,435,947\n",
      "INFO\t2017-01-11 17:34:11\tGenotypeConcordance\tchecked    26,860,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 15:71,437,333\n",
      "INFO\t2017-01-11 17:34:11\tGenotypeConcordance\tchecked    26,870,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 15:72,287,902\n",
      "INFO\t2017-01-11 17:34:11\tGenotypeConcordance\tchecked    26,880,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 15:73,395,364\n",
      "INFO\t2017-01-11 17:34:11\tGenotypeConcordance\tchecked    26,890,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 15:74,357,244\n",
      "INFO\t2017-01-11 17:34:12\tGenotypeConcordance\tchecked    26,900,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 15:75,449,828\n",
      "INFO\t2017-01-11 17:34:12\tGenotypeConcordance\tchecked    26,910,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 15:76,482,077\n",
      "INFO\t2017-01-11 17:34:12\tGenotypeConcordance\tchecked    26,920,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 15:77,484,477\n",
      "INFO\t2017-01-11 17:34:12\tGenotypeConcordance\tchecked    26,930,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 15:78,275,500\n",
      "INFO\t2017-01-11 17:34:13\tGenotypeConcordance\tchecked    26,940,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 15:79,099,537\n",
      "INFO\t2017-01-11 17:34:13\tGenotypeConcordance\tchecked    26,950,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 15:79,864,072\n",
      "INFO\t2017-01-11 17:34:13\tGenotypeConcordance\tchecked    26,960,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 15:80,624,809\n",
      "INFO\t2017-01-11 17:34:14\tGenotypeConcordance\tchecked    26,970,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 15:81,466,197\n",
      "INFO\t2017-01-11 17:34:14\tGenotypeConcordance\tchecked    26,980,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 15:82,366,461\n",
      "INFO\t2017-01-11 17:34:14\tGenotypeConcordance\tchecked    26,990,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 15:83,940,733\n",
      "INFO\t2017-01-11 17:34:15\tGenotypeConcordance\tchecked    27,000,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 15:84,956,557\n",
      "INFO\t2017-01-11 17:34:15\tGenotypeConcordance\tchecked    27,010,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 15:85,985,755\n",
      "INFO\t2017-01-11 17:34:15\tGenotypeConcordance\tchecked    27,020,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 15:86,721,032\n",
      "INFO\t2017-01-11 17:34:16\tGenotypeConcordance\tchecked    27,030,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 15:87,402,722\n",
      "INFO\t2017-01-11 17:34:16\tGenotypeConcordance\tchecked    27,040,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 15:88,075,564\n",
      "INFO\t2017-01-11 17:34:16\tGenotypeConcordance\tchecked    27,050,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 15:88,878,123\n",
      "INFO\t2017-01-11 17:34:16\tGenotypeConcordance\tchecked    27,060,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 15:89,588,694\n",
      "INFO\t2017-01-11 17:34:17\tGenotypeConcordance\tchecked    27,070,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 15:90,347,814\n",
      "INFO\t2017-01-11 17:34:17\tGenotypeConcordance\tchecked    27,080,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 15:91,122,156\n",
      "INFO\t2017-01-11 17:34:17\tGenotypeConcordance\tchecked    27,090,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 15:91,860,260\n",
      "INFO\t2017-01-11 17:34:18\tGenotypeConcordance\tchecked    27,100,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 15:92,609,246\n",
      "INFO\t2017-01-11 17:34:18\tGenotypeConcordance\tchecked    27,110,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 15:93,310,705\n",
      "INFO\t2017-01-11 17:34:18\tGenotypeConcordance\tchecked    27,120,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 15:93,972,367\n",
      "INFO\t2017-01-11 17:34:18\tGenotypeConcordance\tchecked    27,130,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 15:94,735,128\n",
      "INFO\t2017-01-11 17:34:19\tGenotypeConcordance\tchecked    27,140,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 15:95,504,426\n",
      "INFO\t2017-01-11 17:34:19\tGenotypeConcordance\tchecked    27,150,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 15:96,372,089\n",
      "INFO\t2017-01-11 17:34:19\tGenotypeConcordance\tchecked    27,160,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 15:97,247,665\n",
      "INFO\t2017-01-11 17:34:20\tGenotypeConcordance\tchecked    27,170,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 15:98,033,244\n",
      "INFO\t2017-01-11 17:34:20\tGenotypeConcordance\tchecked    27,180,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 15:98,729,165\n",
      "INFO\t2017-01-11 17:34:20\tGenotypeConcordance\tchecked    27,190,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 15:99,450,259\n",
      "INFO\t2017-01-11 17:34:20\tGenotypeConcordance\tchecked    27,200,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 15:100,260,501\n",
      "INFO\t2017-01-11 17:34:21\tGenotypeConcordance\tchecked    27,210,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 15:100,826,957\n",
      "INFO\t2017-01-11 17:34:21\tGenotypeConcordance\tchecked    27,220,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 15:101,499,504\n",
      "INFO\t2017-01-11 17:34:21\tGenotypeConcordance\tchecked    27,230,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 15:102,117,423\n",
      "INFO\t2017-01-11 17:34:22\tGenotypeConcordance\tchecked    27,240,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 16:370,074\n",
      "INFO\t2017-01-11 17:34:22\tGenotypeConcordance\tchecked    27,250,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 16:941,628\n",
      "INFO\t2017-01-11 17:34:22\tGenotypeConcordance\tchecked    27,260,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 16:1,448,751\n",
      "INFO\t2017-01-11 17:34:22\tGenotypeConcordance\tchecked    27,270,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 16:2,117,260\n",
      "INFO\t2017-01-11 17:34:23\tGenotypeConcordance\tchecked    27,280,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 16:2,970,999\n",
      "INFO\t2017-01-11 17:34:23\tGenotypeConcordance\tchecked    27,290,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 16:3,651,424\n",
      "INFO\t2017-01-11 17:34:23\tGenotypeConcordance\tchecked    27,300,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 16:4,358,039\n",
      "INFO\t2017-01-11 17:34:24\tGenotypeConcordance\tchecked    27,310,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 16:5,035,619\n",
      "INFO\t2017-01-11 17:34:24\tGenotypeConcordance\tchecked    27,320,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 16:5,556,417\n",
      "INFO\t2017-01-11 17:34:24\tGenotypeConcordance\tchecked    27,330,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 16:6,039,092\n",
      "INFO\t2017-01-11 17:34:25\tGenotypeConcordance\tchecked    27,340,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 16:6,565,511\n",
      "INFO\t2017-01-11 17:34:25\tGenotypeConcordance\tchecked    27,350,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 16:6,938,233\n",
      "INFO\t2017-01-11 17:34:25\tGenotypeConcordance\tchecked    27,360,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 16:7,277,983\n",
      "INFO\t2017-01-11 17:34:25\tGenotypeConcordance\tchecked    27,370,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 16:7,774,406\n",
      "INFO\t2017-01-11 17:34:26\tGenotypeConcordance\tchecked    27,380,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 16:8,292,694\n",
      "INFO\t2017-01-11 17:34:26\tGenotypeConcordance\tchecked    27,390,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 16:8,826,064\n",
      "INFO\t2017-01-11 17:34:26\tGenotypeConcordance\tchecked    27,400,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 16:9,385,760\n",
      "INFO\t2017-01-11 17:34:27\tGenotypeConcordance\tchecked    27,410,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 16:10,073,295\n",
      "INFO\t2017-01-11 17:34:27\tGenotypeConcordance\tchecked    27,420,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 16:10,700,928\n",
      "INFO\t2017-01-11 17:34:27\tGenotypeConcordance\tchecked    27,430,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 16:11,369,940\n",
      "INFO\t2017-01-11 17:34:27\tGenotypeConcordance\tchecked    27,440,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 16:11,985,410\n",
      "INFO\t2017-01-11 17:34:28\tGenotypeConcordance\tchecked    27,450,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 16:12,631,768\n",
      "INFO\t2017-01-11 17:34:28\tGenotypeConcordance\tchecked    27,460,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 16:13,091,656\n",
      "INFO\t2017-01-11 17:34:28\tGenotypeConcordance\tchecked    27,470,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 16:13,787,547\n",
      "INFO\t2017-01-11 17:34:28\tGenotypeConcordance\tchecked    27,480,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 16:14,624,301\n",
      "INFO\t2017-01-11 17:34:29\tGenotypeConcordance\tchecked    27,490,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 16:15,823,169\n",
      "INFO\t2017-01-11 17:34:29\tGenotypeConcordance\tchecked    27,500,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 16:16,738,140\n",
      "INFO\t2017-01-11 17:34:29\tGenotypeConcordance\tchecked    27,510,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 16:17,595,028\n",
      "INFO\t2017-01-11 17:34:30\tGenotypeConcordance\tchecked    27,520,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 16:18,383,160\n",
      "INFO\t2017-01-11 17:34:30\tGenotypeConcordance\tchecked    27,530,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 16:19,408,637\n",
      "INFO\t2017-01-11 17:34:30\tGenotypeConcordance\tchecked    27,540,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 16:20,258,451\n",
      "INFO\t2017-01-11 17:34:30\tGenotypeConcordance\tchecked    27,550,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 16:21,163,690\n",
      "INFO\t2017-01-11 17:34:31\tGenotypeConcordance\tchecked    27,560,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 16:22,574,512\n",
      "INFO\t2017-01-11 17:34:31\tGenotypeConcordance\tchecked    27,570,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 16:23,488,031\n",
      "INFO\t2017-01-11 17:34:31\tGenotypeConcordance\tchecked    27,580,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 16:24,351,328\n",
      "INFO\t2017-01-11 17:34:32\tGenotypeConcordance\tchecked    27,590,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 16:25,166,621\n",
      "INFO\t2017-01-11 17:34:32\tGenotypeConcordance\tchecked    27,600,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 16:25,976,419\n",
      "INFO\t2017-01-11 17:34:32\tGenotypeConcordance\tchecked    27,610,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 16:26,731,700\n",
      "INFO\t2017-01-11 17:34:33\tGenotypeConcordance\tchecked    27,620,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 16:27,461,941\n",
      "INFO\t2017-01-11 17:34:33\tGenotypeConcordance\tchecked    27,630,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 16:28,490,200\n",
      "INFO\t2017-01-11 17:34:33\tGenotypeConcordance\tchecked    27,640,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 16:29,589,556\n",
      "INFO\t2017-01-11 17:34:33\tGenotypeConcordance\tchecked    27,650,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 16:30,930,224\n",
      "INFO\t2017-01-11 17:34:34\tGenotypeConcordance\tchecked    27,660,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 16:32,034,627\n",
      "INFO\t2017-01-11 17:34:34\tGenotypeConcordance\tchecked    27,670,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 16:32,491,794\n",
      "INFO\t2017-01-11 17:34:34\tGenotypeConcordance\tchecked    27,680,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 16:33,049,162\n",
      "INFO\t2017-01-11 17:34:35\tGenotypeConcordance\tchecked    27,690,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 16:33,664,589\n",
      "INFO\t2017-01-11 17:34:35\tGenotypeConcordance\tchecked    27,700,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 16:33,986,076\n",
      "INFO\t2017-01-11 17:34:35\tGenotypeConcordance\tchecked    27,710,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 16:34,850,281\n",
      "INFO\t2017-01-11 17:34:36\tGenotypeConcordance\tchecked    27,720,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 16:46,477,222\n",
      "INFO\t2017-01-11 17:34:36\tGenotypeConcordance\tchecked    27,730,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 16:47,784,203\n",
      "INFO\t2017-01-11 17:34:36\tGenotypeConcordance\tchecked    27,740,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 16:48,834,737\n",
      "INFO\t2017-01-11 17:34:37\tGenotypeConcordance\tchecked    27,750,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 16:49,721,942\n",
      "INFO\t2017-01-11 17:34:37\tGenotypeConcordance\tchecked    27,760,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 16:50,636,647\n",
      "INFO\t2017-01-11 17:34:37\tGenotypeConcordance\tchecked    27,770,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 16:51,510,060\n",
      "INFO\t2017-01-11 17:34:38\tGenotypeConcordance\tchecked    27,780,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 16:52,342,550\n",
      "INFO\t2017-01-11 17:34:38\tGenotypeConcordance\tchecked    27,790,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 16:53,166,187\n",
      "INFO\t2017-01-11 17:34:38\tGenotypeConcordance\tchecked    27,800,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 16:54,126,444\n",
      "INFO\t2017-01-11 17:34:38\tGenotypeConcordance\tchecked    27,810,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 16:54,886,126\n",
      "INFO\t2017-01-11 17:34:39\tGenotypeConcordance\tchecked    27,820,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 16:55,658,747\n",
      "INFO\t2017-01-11 17:34:39\tGenotypeConcordance\tchecked    27,830,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 16:56,440,037\n",
      "INFO\t2017-01-11 17:34:39\tGenotypeConcordance\tchecked    27,840,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 16:57,251,503\n",
      "INFO\t2017-01-11 17:34:40\tGenotypeConcordance\tchecked    27,850,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 16:58,044,594\n",
      "INFO\t2017-01-11 17:34:40\tGenotypeConcordance\tchecked    27,860,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 16:58,837,767\n",
      "INFO\t2017-01-11 17:34:40\tGenotypeConcordance\tchecked    27,870,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 16:59,710,112\n",
      "INFO\t2017-01-11 17:34:41\tGenotypeConcordance\tchecked    27,880,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 16:60,598,004\n",
      "INFO\t2017-01-11 17:34:41\tGenotypeConcordance\tchecked    27,890,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 16:61,588,986\n",
      "INFO\t2017-01-11 17:34:41\tGenotypeConcordance\tchecked    27,900,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 16:62,507,747\n",
      "INFO\t2017-01-11 17:34:41\tGenotypeConcordance\tchecked    27,910,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 16:63,375,745\n",
      "INFO\t2017-01-11 17:34:42\tGenotypeConcordance\tchecked    27,920,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 16:64,296,779\n",
      "INFO\t2017-01-11 17:34:42\tGenotypeConcordance\tchecked    27,930,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 16:65,173,715\n",
      "INFO\t2017-01-11 17:34:42\tGenotypeConcordance\tchecked    27,940,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 16:66,016,188\n",
      "INFO\t2017-01-11 17:34:42\tGenotypeConcordance\tchecked    27,950,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 16:67,048,064\n",
      "INFO\t2017-01-11 17:34:43\tGenotypeConcordance\tchecked    27,960,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 16:68,343,793\n",
      "INFO\t2017-01-11 17:34:43\tGenotypeConcordance\tchecked    27,970,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 16:69,356,819\n",
      "INFO\t2017-01-11 17:34:43\tGenotypeConcordance\tchecked    27,980,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 16:70,296,899\n",
      "INFO\t2017-01-11 17:34:43\tGenotypeConcordance\tchecked    27,990,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 16:71,335,258\n",
      "INFO\t2017-01-11 17:34:44\tGenotypeConcordance\tchecked    28,000,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 16:72,355,139\n",
      "INFO\t2017-01-11 17:34:44\tGenotypeConcordance\tchecked    28,010,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 16:73,332,747\n",
      "INFO\t2017-01-11 17:34:44\tGenotypeConcordance\tchecked    28,020,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 16:74,102,050\n",
      "INFO\t2017-01-11 17:34:45\tGenotypeConcordance\tchecked    28,030,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 16:74,877,135\n",
      "INFO\t2017-01-11 17:34:45\tGenotypeConcordance\tchecked    28,040,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 16:75,537,283\n",
      "INFO\t2017-01-11 17:34:45\tGenotypeConcordance\tchecked    28,050,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 16:76,276,902\n",
      "INFO\t2017-01-11 17:34:45\tGenotypeConcordance\tchecked    28,060,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 16:76,930,088\n",
      "INFO\t2017-01-11 17:34:46\tGenotypeConcordance\tchecked    28,070,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 16:77,545,392\n",
      "INFO\t2017-01-11 17:34:46\tGenotypeConcordance\tchecked    28,080,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 16:78,123,591\n",
      "INFO\t2017-01-11 17:34:46\tGenotypeConcordance\tchecked    28,090,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 16:78,581,022\n",
      "INFO\t2017-01-11 17:34:46\tGenotypeConcordance\tchecked    28,100,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 16:78,986,468\n",
      "INFO\t2017-01-11 17:34:47\tGenotypeConcordance\tchecked    28,110,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 16:79,484,853\n",
      "INFO\t2017-01-11 17:34:47\tGenotypeConcordance\tchecked    28,120,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 16:80,027,021\n",
      "INFO\t2017-01-11 17:34:47\tGenotypeConcordance\tchecked    28,130,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 16:80,603,749\n",
      "INFO\t2017-01-11 17:34:48\tGenotypeConcordance\tchecked    28,140,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 16:81,170,121\n",
      "INFO\t2017-01-11 17:34:48\tGenotypeConcordance\tchecked    28,150,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 16:81,751,183\n",
      "INFO\t2017-01-11 17:34:48\tGenotypeConcordance\tchecked    28,160,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 16:82,280,060\n",
      "INFO\t2017-01-11 17:34:48\tGenotypeConcordance\tchecked    28,170,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 16:82,818,499\n",
      "INFO\t2017-01-11 17:34:49\tGenotypeConcordance\tchecked    28,180,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 16:83,322,894\n",
      "INFO\t2017-01-11 17:34:49\tGenotypeConcordance\tchecked    28,190,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 16:83,875,618\n",
      "INFO\t2017-01-11 17:34:49\tGenotypeConcordance\tchecked    28,200,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 16:84,347,646\n",
      "INFO\t2017-01-11 17:34:50\tGenotypeConcordance\tchecked    28,210,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 16:84,838,816\n",
      "INFO\t2017-01-11 17:34:50\tGenotypeConcordance\tchecked    28,220,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 16:85,394,614\n",
      "INFO\t2017-01-11 17:34:50\tGenotypeConcordance\tchecked    28,230,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 16:85,985,535\n",
      "INFO\t2017-01-11 17:34:51\tGenotypeConcordance\tchecked    28,240,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 16:86,487,208\n",
      "INFO\t2017-01-11 17:34:51\tGenotypeConcordance\tchecked    28,250,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 16:87,021,168\n",
      "INFO\t2017-01-11 17:34:51\tGenotypeConcordance\tchecked    28,260,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 16:87,626,154\n",
      "INFO\t2017-01-11 17:34:51\tGenotypeConcordance\tchecked    28,270,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 16:88,131,535\n",
      "INFO\t2017-01-11 17:34:52\tGenotypeConcordance\tchecked    28,280,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 16:88,699,386\n",
      "INFO\t2017-01-11 17:34:52\tGenotypeConcordance\tchecked    28,290,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 16:89,206,841\n",
      "INFO\t2017-01-11 17:34:52\tGenotypeConcordance\tchecked    28,300,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 16:89,801,920\n",
      "INFO\t2017-01-11 17:34:53\tGenotypeConcordance\tchecked    28,310,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 17:168,015\n",
      "INFO\t2017-01-11 17:34:53\tGenotypeConcordance\tchecked    28,320,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 17:853,031\n",
      "INFO\t2017-01-11 17:34:53\tGenotypeConcordance\tchecked    28,330,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 17:1,421,182\n",
      "INFO\t2017-01-11 17:34:53\tGenotypeConcordance\tchecked    28,340,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 17:2,203,408\n",
      "INFO\t2017-01-11 17:34:54\tGenotypeConcordance\tchecked    28,350,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 17:2,946,054\n",
      "INFO\t2017-01-11 17:34:54\tGenotypeConcordance\tchecked    28,360,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 17:3,637,453\n",
      "INFO\t2017-01-11 17:34:54\tGenotypeConcordance\tchecked    28,370,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 17:4,371,113\n",
      "INFO\t2017-01-11 17:34:55\tGenotypeConcordance\tchecked    28,380,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 17:5,140,515\n",
      "INFO\t2017-01-11 17:34:55\tGenotypeConcordance\tchecked    28,390,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 17:5,868,104\n",
      "INFO\t2017-01-11 17:34:55\tGenotypeConcordance\tchecked    28,400,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 17:6,594,343\n",
      "INFO\t2017-01-11 17:34:56\tGenotypeConcordance\tchecked    28,410,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 17:7,394,337\n",
      "INFO\t2017-01-11 17:34:56\tGenotypeConcordance\tchecked    28,420,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 17:8,258,055\n",
      "INFO\t2017-01-11 17:34:56\tGenotypeConcordance\tchecked    28,430,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 17:9,083,200\n",
      "INFO\t2017-01-11 17:34:56\tGenotypeConcordance\tchecked    28,440,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 17:9,796,111\n",
      "INFO\t2017-01-11 17:34:57\tGenotypeConcordance\tchecked    28,450,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 17:10,568,838\n",
      "INFO\t2017-01-11 17:34:57\tGenotypeConcordance\tchecked    28,460,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 17:11,343,068\n",
      "INFO\t2017-01-11 17:34:57\tGenotypeConcordance\tchecked    28,470,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 17:12,170,535\n",
      "INFO\t2017-01-11 17:34:58\tGenotypeConcordance\tchecked    28,480,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 17:12,925,095\n",
      "INFO\t2017-01-11 17:34:58\tGenotypeConcordance\tchecked    28,490,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 17:13,609,487\n",
      "INFO\t2017-01-11 17:34:58\tGenotypeConcordance\tchecked    28,500,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 17:14,344,552\n",
      "INFO\t2017-01-11 17:34:58\tGenotypeConcordance\tchecked    28,510,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 17:14,992,958\n",
      "INFO\t2017-01-11 17:34:59\tGenotypeConcordance\tchecked    28,520,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 17:15,755,824\n",
      "INFO\t2017-01-11 17:34:59\tGenotypeConcordance\tchecked    28,530,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 17:16,621,175\n",
      "INFO\t2017-01-11 17:34:59\tGenotypeConcordance\tchecked    28,540,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 17:17,467,357\n",
      "INFO\t2017-01-11 17:35:00\tGenotypeConcordance\tchecked    28,550,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 17:18,466,336\n",
      "INFO\t2017-01-11 17:35:00\tGenotypeConcordance\tchecked    28,560,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 17:19,626,546\n",
      "INFO\t2017-01-11 17:35:00\tGenotypeConcordance\tchecked    28,570,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 17:20,540,503\n",
      "INFO\t2017-01-11 17:35:00\tGenotypeConcordance\tchecked    28,580,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 17:21,223,950\n",
      "INFO\t2017-01-11 17:35:01\tGenotypeConcordance\tchecked    28,590,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 17:21,666,993\n",
      "INFO\t2017-01-11 17:35:01\tGenotypeConcordance\tchecked    28,600,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 17:25,266,624\n",
      "INFO\t2017-01-11 17:35:01\tGenotypeConcordance\tchecked    28,610,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 17:25,762,907\n",
      "INFO\t2017-01-11 17:35:02\tGenotypeConcordance\tchecked    28,620,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 17:26,821,799\n",
      "INFO\t2017-01-11 17:35:02\tGenotypeConcordance\tchecked    28,630,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 17:27,981,897\n",
      "INFO\t2017-01-11 17:35:02\tGenotypeConcordance\tchecked    28,640,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 17:29,076,862\n",
      "INFO\t2017-01-11 17:35:02\tGenotypeConcordance\tchecked    28,650,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 17:29,989,696\n",
      "INFO\t2017-01-11 17:35:03\tGenotypeConcordance\tchecked    28,660,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 17:30,869,423\n",
      "INFO\t2017-01-11 17:35:03\tGenotypeConcordance\tchecked    28,670,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 17:31,773,020\n",
      "INFO\t2017-01-11 17:35:03\tGenotypeConcordance\tchecked    28,680,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 17:32,580,820\n",
      "INFO\t2017-01-11 17:35:03\tGenotypeConcordance\tchecked    28,690,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 17:33,488,856\n",
      "INFO\t2017-01-11 17:35:04\tGenotypeConcordance\tchecked    28,700,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 17:34,363,703\n",
      "INFO\t2017-01-11 17:35:04\tGenotypeConcordance\tchecked    28,710,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 17:35,372,194\n",
      "INFO\t2017-01-11 17:35:04\tGenotypeConcordance\tchecked    28,720,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 17:36,387,469\n",
      "INFO\t2017-01-11 17:35:05\tGenotypeConcordance\tchecked    28,730,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 17:37,268,133\n",
      "INFO\t2017-01-11 17:35:05\tGenotypeConcordance\tchecked    28,740,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 17:38,235,129\n",
      "INFO\t2017-01-11 17:35:05\tGenotypeConcordance\tchecked    28,750,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 17:39,187,406\n",
      "INFO\t2017-01-11 17:35:05\tGenotypeConcordance\tchecked    28,760,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 17:39,924,576\n",
      "INFO\t2017-01-11 17:35:06\tGenotypeConcordance\tchecked    28,770,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 17:41,028,959\n",
      "INFO\t2017-01-11 17:35:06\tGenotypeConcordance\tchecked    28,780,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 17:41,981,789\n",
      "INFO\t2017-01-11 17:35:06\tGenotypeConcordance\tchecked    28,790,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 17:42,964,475\n",
      "INFO\t2017-01-11 17:35:06\tGenotypeConcordance\tchecked    28,800,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 17:43,868,739\n",
      "INFO\t2017-01-11 17:35:07\tGenotypeConcordance\tchecked    28,810,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 17:44,970,433\n",
      "INFO\t2017-01-11 17:35:07\tGenotypeConcordance\tchecked    28,820,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 17:45,937,923\n",
      "INFO\t2017-01-11 17:35:07\tGenotypeConcordance\tchecked    28,830,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 17:46,977,468\n",
      "INFO\t2017-01-11 17:35:08\tGenotypeConcordance\tchecked    28,840,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 17:47,944,720\n",
      "INFO\t2017-01-11 17:35:08\tGenotypeConcordance\tchecked    28,850,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 17:48,803,520\n",
      "INFO\t2017-01-11 17:35:08\tGenotypeConcordance\tchecked    28,860,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 17:49,713,867\n",
      "INFO\t2017-01-11 17:35:08\tGenotypeConcordance\tchecked    28,870,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 17:50,649,633\n",
      "INFO\t2017-01-11 17:35:09\tGenotypeConcordance\tchecked    28,880,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 17:51,487,974\n",
      "INFO\t2017-01-11 17:35:09\tGenotypeConcordance\tchecked    28,890,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 17:52,341,783\n",
      "INFO\t2017-01-11 17:35:09\tGenotypeConcordance\tchecked    28,900,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 17:53,216,997\n",
      "INFO\t2017-01-11 17:35:10\tGenotypeConcordance\tchecked    28,910,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 17:54,121,113\n",
      "INFO\t2017-01-11 17:35:10\tGenotypeConcordance\tchecked    28,920,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 17:55,063,133\n",
      "INFO\t2017-01-11 17:35:10\tGenotypeConcordance\tchecked    28,930,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 17:55,897,724\n",
      "INFO\t2017-01-11 17:35:10\tGenotypeConcordance\tchecked    28,940,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 17:57,028,805\n",
      "INFO\t2017-01-11 17:35:11\tGenotypeConcordance\tchecked    28,950,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 17:58,212,750\n",
      "INFO\t2017-01-11 17:35:11\tGenotypeConcordance\tchecked    28,960,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 17:59,291,729\n",
      "INFO\t2017-01-11 17:35:11\tGenotypeConcordance\tchecked    28,970,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 17:60,282,418\n",
      "INFO\t2017-01-11 17:35:12\tGenotypeConcordance\tchecked    28,980,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 17:61,244,869\n",
      "INFO\t2017-01-11 17:35:12\tGenotypeConcordance\tchecked    28,990,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 17:62,167,506\n",
      "INFO\t2017-01-11 17:35:12\tGenotypeConcordance\tchecked    29,000,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 17:63,216,128\n",
      "INFO\t2017-01-11 17:35:12\tGenotypeConcordance\tchecked    29,010,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 17:64,023,781\n",
      "INFO\t2017-01-11 17:35:13\tGenotypeConcordance\tchecked    29,020,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 17:64,831,185\n",
      "INFO\t2017-01-11 17:35:13\tGenotypeConcordance\tchecked    29,030,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 17:65,704,455\n",
      "INFO\t2017-01-11 17:35:13\tGenotypeConcordance\tchecked    29,040,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 17:66,485,484\n",
      "INFO\t2017-01-11 17:35:14\tGenotypeConcordance\tchecked    29,050,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 17:67,385,862\n",
      "INFO\t2017-01-11 17:35:14\tGenotypeConcordance\tchecked    29,060,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 17:68,331,568\n",
      "INFO\t2017-01-11 17:35:14\tGenotypeConcordance\tchecked    29,070,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 17:69,174,196\n",
      "INFO\t2017-01-11 17:35:14\tGenotypeConcordance\tchecked    29,080,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 17:69,994,450\n",
      "INFO\t2017-01-11 17:35:15\tGenotypeConcordance\tchecked    29,090,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 17:70,796,942\n",
      "INFO\t2017-01-11 17:35:15\tGenotypeConcordance\tchecked    29,100,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 17:71,524,987\n",
      "INFO\t2017-01-11 17:35:15\tGenotypeConcordance\tchecked    29,110,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 17:72,237,435\n",
      "INFO\t2017-01-11 17:35:15\tGenotypeConcordance\tchecked    29,120,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 17:73,028,078\n",
      "INFO\t2017-01-11 17:35:16\tGenotypeConcordance\tchecked    29,130,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 17:73,857,022\n",
      "INFO\t2017-01-11 17:35:16\tGenotypeConcordance\tchecked    29,140,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 17:74,639,811\n",
      "INFO\t2017-01-11 17:35:16\tGenotypeConcordance\tchecked    29,150,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 17:75,331,648\n",
      "INFO\t2017-01-11 17:35:17\tGenotypeConcordance\tchecked    29,160,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 17:76,015,286\n",
      "INFO\t2017-01-11 17:35:17\tGenotypeConcordance\tchecked    29,170,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 17:76,639,316\n",
      "INFO\t2017-01-11 17:35:17\tGenotypeConcordance\tchecked    29,180,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 17:77,313,109\n",
      "INFO\t2017-01-11 17:35:17\tGenotypeConcordance\tchecked    29,190,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 17:78,104,137\n",
      "INFO\t2017-01-11 17:35:18\tGenotypeConcordance\tchecked    29,200,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 17:78,819,267\n",
      "INFO\t2017-01-11 17:35:18\tGenotypeConcordance\tchecked    29,210,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 17:79,439,361\n",
      "INFO\t2017-01-11 17:35:18\tGenotypeConcordance\tchecked    29,220,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 17:80,220,622\n",
      "INFO\t2017-01-11 17:35:18\tGenotypeConcordance\tchecked    29,230,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 17:80,874,543\n",
      "INFO\t2017-01-11 17:35:19\tGenotypeConcordance\tchecked    29,240,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 18:343,344\n",
      "INFO\t2017-01-11 17:35:19\tGenotypeConcordance\tchecked    29,250,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 18:1,070,098\n",
      "INFO\t2017-01-11 17:35:19\tGenotypeConcordance\tchecked    29,260,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 18:1,865,620\n",
      "INFO\t2017-01-11 17:35:20\tGenotypeConcordance\tchecked    29,270,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 18:2,609,293\n",
      "INFO\t2017-01-11 17:35:20\tGenotypeConcordance\tchecked    29,280,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 18:3,329,118\n",
      "INFO\t2017-01-11 17:35:20\tGenotypeConcordance\tchecked    29,290,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 18:4,089,873\n",
      "INFO\t2017-01-11 17:35:20\tGenotypeConcordance\tchecked    29,300,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 18:4,870,485\n",
      "INFO\t2017-01-11 17:35:21\tGenotypeConcordance\tchecked    29,310,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 18:5,647,385\n",
      "INFO\t2017-01-11 17:35:21\tGenotypeConcordance\tchecked    29,320,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 18:6,472,983\n",
      "INFO\t2017-01-11 17:35:21\tGenotypeConcordance\tchecked    29,330,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 18:7,193,858\n",
      "INFO\t2017-01-11 17:35:21\tGenotypeConcordance\tchecked    29,340,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 18:8,007,349\n",
      "INFO\t2017-01-11 17:35:22\tGenotypeConcordance\tchecked    29,350,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 18:8,753,976\n",
      "INFO\t2017-01-11 17:35:22\tGenotypeConcordance\tchecked    29,360,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 18:9,539,701\n",
      "INFO\t2017-01-11 17:35:22\tGenotypeConcordance\tchecked    29,370,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 18:10,290,966\n",
      "INFO\t2017-01-11 17:35:23\tGenotypeConcordance\tchecked    29,380,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 18:10,963,030\n",
      "INFO\t2017-01-11 17:35:23\tGenotypeConcordance\tchecked    29,390,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 18:11,679,246\n",
      "INFO\t2017-01-11 17:35:23\tGenotypeConcordance\tchecked    29,400,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 18:12,387,659\n",
      "INFO\t2017-01-11 17:35:23\tGenotypeConcordance\tchecked    29,410,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 18:13,217,897\n",
      "INFO\t2017-01-11 17:35:24\tGenotypeConcordance\tchecked    29,420,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 18:14,010,580\n",
      "INFO\t2017-01-11 17:35:24\tGenotypeConcordance\tchecked    29,430,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 18:14,539,793\n",
      "INFO\t2017-01-11 17:35:24\tGenotypeConcordance\tchecked    29,440,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 18:15,203,733\n",
      "INFO\t2017-01-11 17:35:25\tGenotypeConcordance\tchecked    29,450,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 18:18,973,557\n",
      "INFO\t2017-01-11 17:35:25\tGenotypeConcordance\tchecked    29,460,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 18:20,028,670\n",
      "INFO\t2017-01-11 17:35:25\tGenotypeConcordance\tchecked    29,470,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 18:20,946,105\n",
      "INFO\t2017-01-11 17:35:25\tGenotypeConcordance\tchecked    29,480,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 18:21,934,157\n",
      "INFO\t2017-01-11 17:35:26\tGenotypeConcordance\tchecked    29,490,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 18:22,747,289\n",
      "INFO\t2017-01-11 17:35:26\tGenotypeConcordance\tchecked    29,500,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 18:23,633,640\n",
      "INFO\t2017-01-11 17:35:26\tGenotypeConcordance\tchecked    29,510,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 18:24,508,415\n",
      "INFO\t2017-01-11 17:35:26\tGenotypeConcordance\tchecked    29,520,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 18:25,468,124\n",
      "INFO\t2017-01-11 17:35:27\tGenotypeConcordance\tchecked    29,530,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 18:26,487,449\n",
      "INFO\t2017-01-11 17:35:27\tGenotypeConcordance\tchecked    29,540,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 18:27,351,117\n",
      "INFO\t2017-01-11 17:35:27\tGenotypeConcordance\tchecked    29,550,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 18:28,160,788\n",
      "INFO\t2017-01-11 17:35:28\tGenotypeConcordance\tchecked    29,560,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 18:28,984,706\n",
      "INFO\t2017-01-11 17:35:28\tGenotypeConcordance\tchecked    29,570,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 18:29,811,919\n",
      "INFO\t2017-01-11 17:35:28\tGenotypeConcordance\tchecked    29,580,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 18:30,842,774\n",
      "INFO\t2017-01-11 17:35:28\tGenotypeConcordance\tchecked    29,590,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 18:31,891,898\n",
      "INFO\t2017-01-11 17:35:29\tGenotypeConcordance\tchecked    29,600,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 18:32,871,256\n",
      "INFO\t2017-01-11 17:35:29\tGenotypeConcordance\tchecked    29,610,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 18:33,759,114\n",
      "INFO\t2017-01-11 17:35:29\tGenotypeConcordance\tchecked    29,620,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 18:34,809,470\n",
      "INFO\t2017-01-11 17:35:30\tGenotypeConcordance\tchecked    29,630,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 18:35,764,592\n",
      "INFO\t2017-01-11 17:35:30\tGenotypeConcordance\tchecked    29,640,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 18:36,704,958\n",
      "INFO\t2017-01-11 17:35:30\tGenotypeConcordance\tchecked    29,650,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 18:37,727,036\n",
      "INFO\t2017-01-11 17:35:30\tGenotypeConcordance\tchecked    29,660,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 18:38,594,224\n",
      "INFO\t2017-01-11 17:35:31\tGenotypeConcordance\tchecked    29,670,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 18:39,510,120\n",
      "INFO\t2017-01-11 17:35:31\tGenotypeConcordance\tchecked    29,680,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 18:40,425,012\n",
      "INFO\t2017-01-11 17:35:31\tGenotypeConcordance\tchecked    29,690,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 18:41,342,975\n",
      "INFO\t2017-01-11 17:35:32\tGenotypeConcordance\tchecked    29,700,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 18:42,285,729\n",
      "INFO\t2017-01-11 17:35:32\tGenotypeConcordance\tchecked    29,710,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 18:43,228,567\n",
      "INFO\t2017-01-11 17:35:32\tGenotypeConcordance\tchecked    29,720,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 18:44,072,052\n",
      "INFO\t2017-01-11 17:35:32\tGenotypeConcordance\tchecked    29,730,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 18:44,993,364\n",
      "INFO\t2017-01-11 17:35:33\tGenotypeConcordance\tchecked    29,740,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 18:45,858,505\n",
      "INFO\t2017-01-11 17:35:33\tGenotypeConcordance\tchecked    29,750,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 18:46,697,891\n",
      "INFO\t2017-01-11 17:35:33\tGenotypeConcordance\tchecked    29,760,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 18:47,504,347\n",
      "INFO\t2017-01-11 17:35:34\tGenotypeConcordance\tchecked    29,770,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 18:48,306,936\n",
      "INFO\t2017-01-11 17:35:34\tGenotypeConcordance\tchecked    29,780,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 18:49,182,209\n",
      "INFO\t2017-01-11 17:35:34\tGenotypeConcordance\tchecked    29,790,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 18:50,005,954\n",
      "INFO\t2017-01-11 17:35:34\tGenotypeConcordance\tchecked    29,800,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 18:50,738,582\n",
      "INFO\t2017-01-11 17:35:35\tGenotypeConcordance\tchecked    29,810,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 18:51,505,483\n",
      "INFO\t2017-01-11 17:35:35\tGenotypeConcordance\tchecked    29,820,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 18:52,532,162\n",
      "INFO\t2017-01-11 17:35:35\tGenotypeConcordance\tchecked    29,830,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 18:53,642,248\n",
      "INFO\t2017-01-11 17:35:36\tGenotypeConcordance\tchecked    29,840,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 18:54,590,715\n",
      "INFO\t2017-01-11 17:35:36\tGenotypeConcordance\tchecked    29,850,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 18:55,411,554\n",
      "INFO\t2017-01-11 17:35:36\tGenotypeConcordance\tchecked    29,860,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 18:56,203,252\n",
      "INFO\t2017-01-11 17:35:36\tGenotypeConcordance\tchecked    29,870,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 18:57,014,381\n",
      "INFO\t2017-01-11 17:35:37\tGenotypeConcordance\tchecked    29,880,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 18:57,725,641\n",
      "INFO\t2017-01-11 17:35:37\tGenotypeConcordance\tchecked    29,890,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 18:58,511,600\n",
      "INFO\t2017-01-11 17:35:37\tGenotypeConcordance\tchecked    29,900,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 18:59,396,194\n",
      "INFO\t2017-01-11 17:35:37\tGenotypeConcordance\tchecked    29,910,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 18:60,281,207\n",
      "INFO\t2017-01-11 17:35:38\tGenotypeConcordance\tchecked    29,920,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 18:61,218,269\n",
      "INFO\t2017-01-11 17:35:38\tGenotypeConcordance\tchecked    29,930,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 18:62,044,045\n",
      "INFO\t2017-01-11 17:35:38\tGenotypeConcordance\tchecked    29,940,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 18:62,834,950\n",
      "INFO\t2017-01-11 17:35:39\tGenotypeConcordance\tchecked    29,950,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 18:63,613,087\n",
      "INFO\t2017-01-11 17:35:39\tGenotypeConcordance\tchecked    29,960,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 18:64,401,957\n",
      "INFO\t2017-01-11 17:35:39\tGenotypeConcordance\tchecked    29,970,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 18:65,212,889\n",
      "INFO\t2017-01-11 17:35:39\tGenotypeConcordance\tchecked    29,980,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 18:65,938,039\n",
      "INFO\t2017-01-11 17:35:40\tGenotypeConcordance\tchecked    29,990,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 18:66,728,756\n",
      "INFO\t2017-01-11 17:35:40\tGenotypeConcordance\tchecked    30,000,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 18:67,523,260\n",
      "INFO\t2017-01-11 17:35:40\tGenotypeConcordance\tchecked    30,010,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 18:68,372,694\n",
      "INFO\t2017-01-11 17:35:41\tGenotypeConcordance\tchecked    30,020,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 18:69,159,389\n",
      "INFO\t2017-01-11 17:35:41\tGenotypeConcordance\tchecked    30,030,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 18:69,913,590\n",
      "INFO\t2017-01-11 17:35:41\tGenotypeConcordance\tchecked    30,040,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 18:70,641,009\n",
      "INFO\t2017-01-11 17:35:41\tGenotypeConcordance\tchecked    30,050,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 18:71,351,521\n",
      "INFO\t2017-01-11 17:35:42\tGenotypeConcordance\tchecked    30,060,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 18:72,108,494\n",
      "INFO\t2017-01-11 17:35:42\tGenotypeConcordance\tchecked    30,070,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 18:73,031,691\n",
      "INFO\t2017-01-11 17:35:42\tGenotypeConcordance\tchecked    30,080,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 18:73,760,114\n",
      "INFO\t2017-01-11 17:35:43\tGenotypeConcordance\tchecked    30,090,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 18:74,457,620\n",
      "INFO\t2017-01-11 17:35:43\tGenotypeConcordance\tchecked    30,100,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 18:75,134,510\n",
      "INFO\t2017-01-11 17:35:43\tGenotypeConcordance\tchecked    30,110,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 18:75,869,682\n",
      "INFO\t2017-01-11 17:35:44\tGenotypeConcordance\tchecked    30,120,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 18:76,457,627\n",
      "INFO\t2017-01-11 17:35:44\tGenotypeConcordance\tchecked    30,130,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 18:77,169,216\n",
      "INFO\t2017-01-11 17:35:44\tGenotypeConcordance\tchecked    30,140,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 18:77,790,689\n",
      "INFO\t2017-01-11 17:35:44\tGenotypeConcordance\tchecked    30,150,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 19:597,941\n",
      "INFO\t2017-01-11 17:35:45\tGenotypeConcordance\tchecked    30,160,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 19:1,139,469\n",
      "INFO\t2017-01-11 17:35:45\tGenotypeConcordance\tchecked    30,170,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 19:1,819,007\n",
      "INFO\t2017-01-11 17:35:45\tGenotypeConcordance\tchecked    30,180,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 19:2,481,012\n",
      "INFO\t2017-01-11 17:35:46\tGenotypeConcordance\tchecked    30,190,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 19:3,080,333\n",
      "INFO\t2017-01-11 17:35:46\tGenotypeConcordance\tchecked    30,200,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 19:3,742,710\n",
      "INFO\t2017-01-11 17:35:46\tGenotypeConcordance\tchecked    30,210,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 19:4,451,560\n",
      "INFO\t2017-01-11 17:35:46\tGenotypeConcordance\tchecked    30,220,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 19:5,123,070\n",
      "INFO\t2017-01-11 17:35:47\tGenotypeConcordance\tchecked    30,230,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 19:5,858,371\n",
      "INFO\t2017-01-11 17:35:47\tGenotypeConcordance\tchecked    30,240,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 19:6,597,715\n",
      "INFO\t2017-01-11 17:35:47\tGenotypeConcordance\tchecked    30,250,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 19:7,251,423\n",
      "INFO\t2017-01-11 17:35:47\tGenotypeConcordance\tchecked    30,260,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 19:7,999,362\n",
      "INFO\t2017-01-11 17:35:48\tGenotypeConcordance\tchecked    30,270,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 19:8,755,058\n",
      "INFO\t2017-01-11 17:35:48\tGenotypeConcordance\tchecked    30,280,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 19:9,402,550\n",
      "INFO\t2017-01-11 17:35:48\tGenotypeConcordance\tchecked    30,290,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 19:10,158,124\n",
      "INFO\t2017-01-11 17:35:49\tGenotypeConcordance\tchecked    30,300,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 19:11,026,045\n",
      "INFO\t2017-01-11 17:35:49\tGenotypeConcordance\tchecked    30,310,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 19:11,817,808\n",
      "INFO\t2017-01-11 17:35:49\tGenotypeConcordance\tchecked    30,320,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 19:12,669,438\n",
      "INFO\t2017-01-11 17:35:50\tGenotypeConcordance\tchecked    30,330,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 19:13,579,418\n",
      "INFO\t2017-01-11 17:35:50\tGenotypeConcordance\tchecked    30,340,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 19:14,379,965\n",
      "INFO\t2017-01-11 17:35:50\tGenotypeConcordance\tchecked    30,350,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 19:15,084,896\n",
      "INFO\t2017-01-11 17:35:50\tGenotypeConcordance\tchecked    30,360,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 19:15,835,206\n",
      "INFO\t2017-01-11 17:35:51\tGenotypeConcordance\tchecked    30,370,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 19:16,527,917\n",
      "INFO\t2017-01-11 17:35:51\tGenotypeConcordance\tchecked    30,380,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 19:17,324,751\n",
      "INFO\t2017-01-11 17:35:51\tGenotypeConcordance\tchecked    30,390,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 19:18,073,775\n",
      "INFO\t2017-01-11 17:35:51\tGenotypeConcordance\tchecked    30,400,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 19:18,929,561\n",
      "INFO\t2017-01-11 17:35:52\tGenotypeConcordance\tchecked    30,410,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 19:19,891,687\n",
      "INFO\t2017-01-11 17:35:52\tGenotypeConcordance\tchecked    30,420,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 19:20,688,283\n",
      "INFO\t2017-01-11 17:35:52\tGenotypeConcordance\tchecked    30,430,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 19:21,320,316\n",
      "INFO\t2017-01-11 17:35:53\tGenotypeConcordance\tchecked    30,440,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 19:21,957,865\n",
      "INFO\t2017-01-11 17:35:53\tGenotypeConcordance\tchecked    30,450,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 19:22,646,934\n",
      "INFO\t2017-01-11 17:35:53\tGenotypeConcordance\tchecked    30,460,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 19:23,344,209\n",
      "INFO\t2017-01-11 17:35:54\tGenotypeConcordance\tchecked    30,470,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 19:24,030,473\n",
      "INFO\t2017-01-11 17:35:54\tGenotypeConcordance\tchecked    30,480,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 19:27,733,063\n",
      "INFO\t2017-01-11 17:35:54\tGenotypeConcordance\tchecked    30,490,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 19:28,165,191\n",
      "INFO\t2017-01-11 17:35:54\tGenotypeConcordance\tchecked    30,500,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 19:28,897,587\n",
      "INFO\t2017-01-11 17:35:55\tGenotypeConcordance\tchecked    30,510,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 19:29,567,472\n",
      "INFO\t2017-01-11 17:35:55\tGenotypeConcordance\tchecked    30,520,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 19:30,274,191\n",
      "INFO\t2017-01-11 17:35:55\tGenotypeConcordance\tchecked    30,530,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 19:31,140,007\n",
      "INFO\t2017-01-11 17:35:55\tGenotypeConcordance\tchecked    30,540,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 19:32,050,829\n",
      "INFO\t2017-01-11 17:35:56\tGenotypeConcordance\tchecked    30,550,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 19:32,951,841\n",
      "INFO\t2017-01-11 17:35:56\tGenotypeConcordance\tchecked    30,560,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 19:33,728,968\n",
      "INFO\t2017-01-11 17:35:56\tGenotypeConcordance\tchecked    30,570,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 19:34,514,806\n",
      "INFO\t2017-01-11 17:35:56\tGenotypeConcordance\tchecked    30,580,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 19:35,372,415\n",
      "INFO\t2017-01-11 17:35:57\tGenotypeConcordance\tchecked    30,590,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 19:36,188,129\n",
      "INFO\t2017-01-11 17:35:57\tGenotypeConcordance\tchecked    30,600,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 19:37,050,707\n",
      "INFO\t2017-01-11 17:35:57\tGenotypeConcordance\tchecked    30,610,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 19:38,073,098\n",
      "INFO\t2017-01-11 17:35:58\tGenotypeConcordance\tchecked    30,620,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 19:39,000,236\n",
      "INFO\t2017-01-11 17:35:58\tGenotypeConcordance\tchecked    30,630,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 19:39,738,666\n",
      "INFO\t2017-01-11 17:35:58\tGenotypeConcordance\tchecked    30,640,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 19:40,575,008\n",
      "INFO\t2017-01-11 17:35:58\tGenotypeConcordance\tchecked    30,650,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 19:41,438,722\n",
      "INFO\t2017-01-11 17:35:59\tGenotypeConcordance\tchecked    30,660,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 19:42,165,671\n",
      "INFO\t2017-01-11 17:35:59\tGenotypeConcordance\tchecked    30,670,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 19:43,266,640\n",
      "INFO\t2017-01-11 17:35:59\tGenotypeConcordance\tchecked    30,680,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 19:43,697,726\n",
      "INFO\t2017-01-11 17:35:59\tGenotypeConcordance\tchecked    30,690,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 19:44,428,911\n",
      "INFO\t2017-01-11 17:36:00\tGenotypeConcordance\tchecked    30,700,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 19:45,216,026\n",
      "INFO\t2017-01-11 17:36:00\tGenotypeConcordance\tchecked    30,710,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 19:45,974,668\n",
      "INFO\t2017-01-11 17:36:00\tGenotypeConcordance\tchecked    30,720,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 19:46,785,021\n",
      "INFO\t2017-01-11 17:36:01\tGenotypeConcordance\tchecked    30,730,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 19:47,580,641\n",
      "INFO\t2017-01-11 17:36:01\tGenotypeConcordance\tchecked    30,740,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 19:48,381,365\n",
      "INFO\t2017-01-11 17:36:01\tGenotypeConcordance\tchecked    30,750,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 19:49,114,433\n",
      "INFO\t2017-01-11 17:36:01\tGenotypeConcordance\tchecked    30,760,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 19:49,808,849\n",
      "INFO\t2017-01-11 17:36:02\tGenotypeConcordance\tchecked    30,770,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 19:50,593,858\n",
      "INFO\t2017-01-11 17:36:02\tGenotypeConcordance\tchecked    30,780,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 19:51,321,827\n",
      "INFO\t2017-01-11 17:36:02\tGenotypeConcordance\tchecked    30,790,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 19:52,041,801\n",
      "INFO\t2017-01-11 17:36:03\tGenotypeConcordance\tchecked    30,800,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 19:52,798,735\n",
      "INFO\t2017-01-11 17:36:03\tGenotypeConcordance\tchecked    30,810,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 19:53,359,005\n",
      "INFO\t2017-01-11 17:36:03\tGenotypeConcordance\tchecked    30,820,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 19:53,959,745\n",
      "INFO\t2017-01-11 17:36:04\tGenotypeConcordance\tchecked    30,830,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 19:54,536,734\n",
      "INFO\t2017-01-11 17:36:04\tGenotypeConcordance\tchecked    30,840,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 19:55,098,680\n",
      "INFO\t2017-01-11 17:36:04\tGenotypeConcordance\tchecked    30,850,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 19:55,623,635\n",
      "INFO\t2017-01-11 17:36:04\tGenotypeConcordance\tchecked    30,860,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 19:56,298,003\n",
      "INFO\t2017-01-11 17:36:05\tGenotypeConcordance\tchecked    30,870,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 19:56,863,145\n",
      "INFO\t2017-01-11 17:36:05\tGenotypeConcordance\tchecked    30,880,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 19:57,660,692\n",
      "INFO\t2017-01-11 17:36:05\tGenotypeConcordance\tchecked    30,890,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 19:58,503,369\n",
      "INFO\t2017-01-11 17:36:06\tGenotypeConcordance\tchecked    30,900,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 20:223,652\n",
      "INFO\t2017-01-11 17:36:06\tGenotypeConcordance\tchecked    30,910,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 20:923,176\n",
      "INFO\t2017-01-11 17:36:06\tGenotypeConcordance\tchecked    30,920,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 20:1,796,112\n",
      "INFO\t2017-01-11 17:36:07\tGenotypeConcordance\tchecked    30,930,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 20:2,579,107\n",
      "INFO\t2017-01-11 17:36:07\tGenotypeConcordance\tchecked    30,940,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 20:3,409,428\n",
      "INFO\t2017-01-11 17:36:07\tGenotypeConcordance\tchecked    30,950,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 20:4,222,977\n",
      "INFO\t2017-01-11 17:36:07\tGenotypeConcordance\tchecked    30,960,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 20:4,996,506\n",
      "INFO\t2017-01-11 17:36:08\tGenotypeConcordance\tchecked    30,970,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 20:5,730,566\n",
      "INFO\t2017-01-11 17:36:08\tGenotypeConcordance\tchecked    30,980,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 20:6,491,219\n",
      "INFO\t2017-01-11 17:36:08\tGenotypeConcordance\tchecked    30,990,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 20:7,293,914\n",
      "INFO\t2017-01-11 17:36:09\tGenotypeConcordance\tchecked    31,000,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 20:8,106,464\n",
      "INFO\t2017-01-11 17:36:09\tGenotypeConcordance\tchecked    31,010,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 20:8,981,641\n",
      "INFO\t2017-01-11 17:36:09\tGenotypeConcordance\tchecked    31,020,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 20:9,917,409\n",
      "INFO\t2017-01-11 17:36:09\tGenotypeConcordance\tchecked    31,030,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 20:10,823,171\n",
      "INFO\t2017-01-11 17:36:10\tGenotypeConcordance\tchecked    31,040,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 20:11,704,160\n",
      "INFO\t2017-01-11 17:36:10\tGenotypeConcordance\tchecked    31,050,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 20:12,497,212\n",
      "INFO\t2017-01-11 17:36:10\tGenotypeConcordance\tchecked    31,060,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 20:13,294,098\n",
      "INFO\t2017-01-11 17:36:11\tGenotypeConcordance\tchecked    31,070,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 20:14,314,039\n",
      "INFO\t2017-01-11 17:36:11\tGenotypeConcordance\tchecked    31,080,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 20:15,236,000\n",
      "INFO\t2017-01-11 17:36:11\tGenotypeConcordance\tchecked    31,090,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 20:16,034,829\n",
      "INFO\t2017-01-11 17:36:12\tGenotypeConcordance\tchecked    31,100,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 20:16,828,207\n",
      "INFO\t2017-01-11 17:36:12\tGenotypeConcordance\tchecked    31,110,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 20:17,629,161\n",
      "INFO\t2017-01-11 17:36:12\tGenotypeConcordance\tchecked    31,120,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 20:18,408,862\n",
      "INFO\t2017-01-11 17:36:12\tGenotypeConcordance\tchecked    31,130,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 20:19,227,230\n",
      "INFO\t2017-01-11 17:36:13\tGenotypeConcordance\tchecked    31,140,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 20:20,041,868\n",
      "INFO\t2017-01-11 17:36:13\tGenotypeConcordance\tchecked    31,150,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 20:21,016,811\n",
      "INFO\t2017-01-11 17:36:13\tGenotypeConcordance\tchecked    31,160,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 20:22,030,882\n",
      "INFO\t2017-01-11 17:36:14\tGenotypeConcordance\tchecked    31,170,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 20:22,828,996\n",
      "INFO\t2017-01-11 17:36:14\tGenotypeConcordance\tchecked    31,180,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 20:23,627,632\n",
      "INFO\t2017-01-11 17:36:14\tGenotypeConcordance\tchecked    31,190,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 20:24,237,109\n",
      "INFO\t2017-01-11 17:36:14\tGenotypeConcordance\tchecked    31,200,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 20:24,984,699\n",
      "INFO\t2017-01-11 17:36:15\tGenotypeConcordance\tchecked    31,210,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 20:25,764,616\n",
      "INFO\t2017-01-11 17:36:15\tGenotypeConcordance\tchecked    31,220,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 20:26,138,588\n",
      "INFO\t2017-01-11 17:36:15\tGenotypeConcordance\tchecked    31,230,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 20:29,512,482\n",
      "INFO\t2017-01-11 17:36:16\tGenotypeConcordance\tchecked    31,240,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 20:29,913,360\n",
      "INFO\t2017-01-11 17:36:16\tGenotypeConcordance\tchecked    31,250,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 20:30,972,774\n",
      "INFO\t2017-01-11 17:36:16\tGenotypeConcordance\tchecked    31,260,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 20:31,863,599\n",
      "INFO\t2017-01-11 17:36:16\tGenotypeConcordance\tchecked    31,270,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 20:32,938,216\n",
      "INFO\t2017-01-11 17:36:17\tGenotypeConcordance\tchecked    31,280,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 20:34,033,548\n",
      "INFO\t2017-01-11 17:36:17\tGenotypeConcordance\tchecked    31,290,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 20:35,192,121\n",
      "INFO\t2017-01-11 17:36:17\tGenotypeConcordance\tchecked    31,300,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 20:36,185,592\n",
      "INFO\t2017-01-11 17:36:18\tGenotypeConcordance\tchecked    31,310,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 20:37,053,089\n",
      "INFO\t2017-01-11 17:36:18\tGenotypeConcordance\tchecked    31,320,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 20:38,026,027\n",
      "INFO\t2017-01-11 17:36:18\tGenotypeConcordance\tchecked    31,330,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 20:38,994,449\n",
      "INFO\t2017-01-11 17:36:18\tGenotypeConcordance\tchecked    31,340,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 20:39,966,631\n",
      "INFO\t2017-01-11 17:36:19\tGenotypeConcordance\tchecked    31,350,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 20:40,879,061\n",
      "INFO\t2017-01-11 17:36:19\tGenotypeConcordance\tchecked    31,360,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 20:41,702,152\n",
      "INFO\t2017-01-11 17:36:19\tGenotypeConcordance\tchecked    31,370,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 20:42,567,251\n",
      "INFO\t2017-01-11 17:36:19\tGenotypeConcordance\tchecked    31,380,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 20:43,473,176\n",
      "INFO\t2017-01-11 17:36:20\tGenotypeConcordance\tchecked    31,390,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 20:44,361,076\n",
      "INFO\t2017-01-11 17:36:20\tGenotypeConcordance\tchecked    31,400,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 20:45,253,237\n",
      "INFO\t2017-01-11 17:36:20\tGenotypeConcordance\tchecked    31,410,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 20:46,051,930\n",
      "INFO\t2017-01-11 17:36:21\tGenotypeConcordance\tchecked    31,420,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 20:46,833,219\n",
      "INFO\t2017-01-11 17:36:21\tGenotypeConcordance\tchecked    31,430,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 20:47,596,482\n",
      "INFO\t2017-01-11 17:36:21\tGenotypeConcordance\tchecked    31,440,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 20:48,491,474\n",
      "INFO\t2017-01-11 17:36:21\tGenotypeConcordance\tchecked    31,450,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 20:49,333,954\n",
      "INFO\t2017-01-11 17:36:22\tGenotypeConcordance\tchecked    31,460,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 20:50,169,212\n",
      "INFO\t2017-01-11 17:36:22\tGenotypeConcordance\tchecked    31,470,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 20:50,947,016\n",
      "INFO\t2017-01-11 17:36:22\tGenotypeConcordance\tchecked    31,480,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 20:51,863,191\n",
      "INFO\t2017-01-11 17:36:23\tGenotypeConcordance\tchecked    31,490,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 20:52,656,660\n",
      "INFO\t2017-01-11 17:36:23\tGenotypeConcordance\tchecked    31,500,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 20:53,431,548\n",
      "INFO\t2017-01-11 17:36:23\tGenotypeConcordance\tchecked    31,510,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 20:54,194,776\n",
      "INFO\t2017-01-11 17:36:23\tGenotypeConcordance\tchecked    31,520,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 20:54,929,764\n",
      "INFO\t2017-01-11 17:36:24\tGenotypeConcordance\tchecked    31,530,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 20:55,668,324\n",
      "INFO\t2017-01-11 17:36:24\tGenotypeConcordance\tchecked    31,540,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 20:56,432,447\n",
      "INFO\t2017-01-11 17:36:24\tGenotypeConcordance\tchecked    31,550,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 20:57,224,982\n",
      "INFO\t2017-01-11 17:36:24\tGenotypeConcordance\tchecked    31,560,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 20:58,106,502\n",
      "INFO\t2017-01-11 17:36:25\tGenotypeConcordance\tchecked    31,570,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 20:58,889,224\n",
      "INFO\t2017-01-11 17:36:25\tGenotypeConcordance\tchecked    31,580,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 20:59,632,459\n",
      "INFO\t2017-01-11 17:36:25\tGenotypeConcordance\tchecked    31,590,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 20:60,284,329\n",
      "INFO\t2017-01-11 17:36:26\tGenotypeConcordance\tchecked    31,600,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 20:60,939,284\n",
      "INFO\t2017-01-11 17:36:26\tGenotypeConcordance\tchecked    31,610,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 20:61,662,583\n",
      "INFO\t2017-01-11 17:36:26\tGenotypeConcordance\tchecked    31,620,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 20:62,298,541\n",
      "INFO\t2017-01-11 17:36:26\tGenotypeConcordance\tchecked    31,630,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 21:9,481,371\n",
      "INFO\t2017-01-11 17:36:27\tGenotypeConcordance\tchecked    31,640,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 21:9,955,419\n",
      "INFO\t2017-01-11 17:36:27\tGenotypeConcordance\tchecked    31,650,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 21:10,706,664\n",
      "INFO\t2017-01-11 17:36:27\tGenotypeConcordance\tchecked    31,660,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 21:10,868,983\n",
      "INFO\t2017-01-11 17:36:28\tGenotypeConcordance\tchecked    31,670,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 21:14,361,793\n",
      "INFO\t2017-01-11 17:36:28\tGenotypeConcordance\tchecked    31,680,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 21:15,218,151\n",
      "INFO\t2017-01-11 17:36:28\tGenotypeConcordance\tchecked    31,690,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 21:15,738,677\n",
      "INFO\t2017-01-11 17:36:28\tGenotypeConcordance\tchecked    31,700,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 21:16,535,727\n",
      "INFO\t2017-01-11 17:36:29\tGenotypeConcordance\tchecked    31,710,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 21:17,393,193\n",
      "INFO\t2017-01-11 17:36:29\tGenotypeConcordance\tchecked    31,720,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 21:18,190,322\n",
      "INFO\t2017-01-11 17:36:29\tGenotypeConcordance\tchecked    31,730,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 21:18,817,058\n",
      "INFO\t2017-01-11 17:36:30\tGenotypeConcordance\tchecked    31,740,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 21:19,594,702\n",
      "INFO\t2017-01-11 17:36:30\tGenotypeConcordance\tchecked    31,750,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 21:20,348,469\n",
      "INFO\t2017-01-11 17:36:30\tGenotypeConcordance\tchecked    31,760,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 21:21,104,416\n",
      "INFO\t2017-01-11 17:36:30\tGenotypeConcordance\tchecked    31,770,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 21:21,823,904\n",
      "INFO\t2017-01-11 17:36:31\tGenotypeConcordance\tchecked    31,780,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 21:22,478,483\n",
      "INFO\t2017-01-11 17:36:31\tGenotypeConcordance\tchecked    31,790,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 21:23,231,909\n",
      "INFO\t2017-01-11 17:36:31\tGenotypeConcordance\tchecked    31,800,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 21:24,000,969\n",
      "INFO\t2017-01-11 17:36:32\tGenotypeConcordance\tchecked    31,810,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 21:24,713,951\n",
      "INFO\t2017-01-11 17:36:32\tGenotypeConcordance\tchecked    31,820,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 21:25,364,632\n",
      "INFO\t2017-01-11 17:36:32\tGenotypeConcordance\tchecked    31,830,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 21:26,050,834\n",
      "INFO\t2017-01-11 17:36:32\tGenotypeConcordance\tchecked    31,840,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 21:26,930,332\n",
      "INFO\t2017-01-11 17:36:33\tGenotypeConcordance\tchecked    31,850,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 21:27,738,422\n",
      "INFO\t2017-01-11 17:36:33\tGenotypeConcordance\tchecked    31,860,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 21:28,560,262\n",
      "INFO\t2017-01-11 17:36:33\tGenotypeConcordance\tchecked    31,870,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 21:29,329,556\n",
      "INFO\t2017-01-11 17:36:33\tGenotypeConcordance\tchecked    31,880,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 21:30,094,309\n",
      "INFO\t2017-01-11 17:36:34\tGenotypeConcordance\tchecked    31,890,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 21:30,946,049\n",
      "INFO\t2017-01-11 17:36:34\tGenotypeConcordance\tchecked    31,900,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 21:31,740,553\n",
      "INFO\t2017-01-11 17:36:34\tGenotypeConcordance\tchecked    31,910,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 21:32,569,974\n",
      "INFO\t2017-01-11 17:36:35\tGenotypeConcordance\tchecked    31,920,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 21:33,365,595\n",
      "INFO\t2017-01-11 17:36:35\tGenotypeConcordance\tchecked    31,930,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 21:34,237,858\n",
      "INFO\t2017-01-11 17:36:35\tGenotypeConcordance\tchecked    31,940,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 21:35,132,301\n",
      "INFO\t2017-01-11 17:36:35\tGenotypeConcordance\tchecked    31,950,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 21:35,965,934\n",
      "INFO\t2017-01-11 17:36:36\tGenotypeConcordance\tchecked    31,960,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 21:36,777,485\n",
      "INFO\t2017-01-11 17:36:36\tGenotypeConcordance\tchecked    31,970,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 21:37,622,270\n",
      "INFO\t2017-01-11 17:36:36\tGenotypeConcordance\tchecked    31,980,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 21:38,400,659\n",
      "INFO\t2017-01-11 17:36:36\tGenotypeConcordance\tchecked    31,990,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 21:39,250,687\n",
      "INFO\t2017-01-11 17:36:37\tGenotypeConcordance\tchecked    32,000,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 21:40,142,443\n",
      "INFO\t2017-01-11 17:36:37\tGenotypeConcordance\tchecked    32,010,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 21:40,926,096\n",
      "INFO\t2017-01-11 17:36:37\tGenotypeConcordance\tchecked    32,020,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 21:41,637,410\n",
      "INFO\t2017-01-11 17:36:38\tGenotypeConcordance\tchecked    32,030,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 21:42,356,760\n",
      "INFO\t2017-01-11 17:36:38\tGenotypeConcordance\tchecked    32,040,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 21:43,145,704\n",
      "INFO\t2017-01-11 17:36:38\tGenotypeConcordance\tchecked    32,050,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 21:43,798,797\n",
      "INFO\t2017-01-11 17:36:38\tGenotypeConcordance\tchecked    32,060,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 21:44,466,913\n",
      "INFO\t2017-01-11 17:36:39\tGenotypeConcordance\tchecked    32,070,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 21:45,271,831\n",
      "INFO\t2017-01-11 17:36:39\tGenotypeConcordance\tchecked    32,080,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 21:45,959,452\n",
      "INFO\t2017-01-11 17:36:39\tGenotypeConcordance\tchecked    32,090,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 21:46,567,946\n",
      "INFO\t2017-01-11 17:36:40\tGenotypeConcordance\tchecked    32,100,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 21:47,259,186\n",
      "INFO\t2017-01-11 17:36:40\tGenotypeConcordance\tchecked    32,110,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 21:47,920,912\n",
      "INFO\t2017-01-11 17:36:40\tGenotypeConcordance\tchecked    32,120,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 22:16,853,596\n",
      "INFO\t2017-01-11 17:36:40\tGenotypeConcordance\tchecked    32,130,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 22:17,318,864\n",
      "INFO\t2017-01-11 17:36:41\tGenotypeConcordance\tchecked    32,140,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 22:17,958,427\n",
      "INFO\t2017-01-11 17:36:41\tGenotypeConcordance\tchecked    32,150,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 22:18,639,178\n",
      "INFO\t2017-01-11 17:36:41\tGenotypeConcordance\tchecked    32,160,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 22:19,410,846\n",
      "INFO\t2017-01-11 17:36:42\tGenotypeConcordance\tchecked    32,170,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 22:20,177,320\n",
      "INFO\t2017-01-11 17:36:42\tGenotypeConcordance\tchecked    32,180,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 22:21,158,858\n",
      "INFO\t2017-01-11 17:36:42\tGenotypeConcordance\tchecked    32,190,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 22:22,082,603\n",
      "INFO\t2017-01-11 17:36:42\tGenotypeConcordance\tchecked    32,200,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 22:22,714,989\n",
      "INFO\t2017-01-11 17:36:43\tGenotypeConcordance\tchecked    32,210,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 22:23,298,763\n",
      "INFO\t2017-01-11 17:36:43\tGenotypeConcordance\tchecked    32,220,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 22:23,940,670\n",
      "INFO\t2017-01-11 17:36:43\tGenotypeConcordance\tchecked    32,230,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 22:24,702,549\n",
      "INFO\t2017-01-11 17:36:44\tGenotypeConcordance\tchecked    32,240,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 22:25,465,840\n",
      "INFO\t2017-01-11 17:36:44\tGenotypeConcordance\tchecked    32,250,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 22:26,130,523\n",
      "INFO\t2017-01-11 17:36:44\tGenotypeConcordance\tchecked    32,260,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 22:26,904,992\n",
      "INFO\t2017-01-11 17:36:44\tGenotypeConcordance\tchecked    32,270,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 22:27,646,849\n",
      "INFO\t2017-01-11 17:36:45\tGenotypeConcordance\tchecked    32,280,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 22:28,487,368\n",
      "INFO\t2017-01-11 17:36:45\tGenotypeConcordance\tchecked    32,290,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 22:29,552,332\n",
      "INFO\t2017-01-11 17:36:45\tGenotypeConcordance\tchecked    32,300,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 22:30,454,598\n",
      "INFO\t2017-01-11 17:36:46\tGenotypeConcordance\tchecked    32,310,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 22:31,414,524\n",
      "INFO\t2017-01-11 17:36:46\tGenotypeConcordance\tchecked    32,320,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 22:32,373,174\n",
      "INFO\t2017-01-11 17:36:46\tGenotypeConcordance\tchecked    32,330,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 22:33,106,989\n",
      "INFO\t2017-01-11 17:36:47\tGenotypeConcordance\tchecked    32,340,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 22:33,924,594\n",
      "INFO\t2017-01-11 17:36:47\tGenotypeConcordance\tchecked    32,350,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 22:34,726,062\n",
      "INFO\t2017-01-11 17:36:47\tGenotypeConcordance\tchecked    32,360,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 22:35,410,604\n",
      "INFO\t2017-01-11 17:36:47\tGenotypeConcordance\tchecked    32,370,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 22:36,266,441\n",
      "INFO\t2017-01-11 17:36:48\tGenotypeConcordance\tchecked    32,380,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 22:37,087,302\n",
      "INFO\t2017-01-11 17:36:48\tGenotypeConcordance\tchecked    32,390,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 22:37,833,010\n",
      "INFO\t2017-01-11 17:36:48\tGenotypeConcordance\tchecked    32,400,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 22:38,681,603\n",
      "INFO\t2017-01-11 17:36:49\tGenotypeConcordance\tchecked    32,410,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 22:39,538,816\n",
      "INFO\t2017-01-11 17:36:49\tGenotypeConcordance\tchecked    32,420,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 22:40,431,574\n",
      "INFO\t2017-01-11 17:36:49\tGenotypeConcordance\tchecked    32,430,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 22:41,534,436\n",
      "INFO\t2017-01-11 17:36:49\tGenotypeConcordance\tchecked    32,440,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 22:42,541,285\n",
      "INFO\t2017-01-11 17:36:50\tGenotypeConcordance\tchecked    32,450,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 22:43,264,421\n",
      "INFO\t2017-01-11 17:36:50\tGenotypeConcordance\tchecked    32,460,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 22:43,974,763\n",
      "INFO\t2017-01-11 17:36:50\tGenotypeConcordance\tchecked    32,470,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 22:44,739,132\n",
      "INFO\t2017-01-11 17:36:51\tGenotypeConcordance\tchecked    32,480,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 22:45,333,014\n",
      "INFO\t2017-01-11 17:36:51\tGenotypeConcordance\tchecked    32,490,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 22:46,056,236\n",
      "INFO\t2017-01-11 17:36:51\tGenotypeConcordance\tchecked    32,500,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 22:46,789,230\n",
      "INFO\t2017-01-11 17:36:51\tGenotypeConcordance\tchecked    32,510,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 22:47,424,320\n",
      "INFO\t2017-01-11 17:36:52\tGenotypeConcordance\tchecked    32,520,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 22:48,039,140\n",
      "INFO\t2017-01-11 17:36:52\tGenotypeConcordance\tchecked    32,530,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 22:48,719,417\n",
      "INFO\t2017-01-11 17:36:52\tGenotypeConcordance\tchecked    32,540,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 22:49,244,940\n",
      "INFO\t2017-01-11 17:36:52\tGenotypeConcordance\tchecked    32,550,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 22:49,709,319\n",
      "INFO\t2017-01-11 17:36:53\tGenotypeConcordance\tchecked    32,560,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: 22:50,325,778\n",
      "INFO\t2017-01-11 17:36:53\tGenotypeConcordance\tchecked    32,570,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: 22:51,047,884\n",
      "INFO\t2017-01-11 17:36:53\tGenotypeConcordance\tchecked    32,580,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: X:935,063\n",
      "INFO\t2017-01-11 17:36:54\tGenotypeConcordance\tchecked    32,590,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: X:2,361,712\n",
      "INFO\t2017-01-11 17:36:54\tGenotypeConcordance\tchecked    32,600,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: X:5,125,475\n",
      "INFO\t2017-01-11 17:36:54\tGenotypeConcordance\tchecked    32,610,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: X:9,429,617\n",
      "INFO\t2017-01-11 17:36:55\tGenotypeConcordance\tchecked    32,620,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: X:14,769,205\n",
      "INFO\t2017-01-11 17:36:55\tGenotypeConcordance\tchecked    32,630,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: X:20,705,016\n",
      "INFO\t2017-01-11 17:36:55\tGenotypeConcordance\tchecked    32,640,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: X:25,016,033\n",
      "INFO\t2017-01-11 17:36:55\tGenotypeConcordance\tchecked    32,650,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: X:29,596,864\n",
      "INFO\t2017-01-11 17:36:56\tGenotypeConcordance\tchecked    32,660,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: X:33,419,349\n",
      "INFO\t2017-01-11 17:36:56\tGenotypeConcordance\tchecked    32,670,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: X:38,417,258\n",
      "INFO\t2017-01-11 17:36:56\tGenotypeConcordance\tchecked    32,680,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: X:44,067,038\n",
      "INFO\t2017-01-11 17:36:57\tGenotypeConcordance\tchecked    32,690,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: X:49,731,605\n",
      "INFO\t2017-01-11 17:36:57\tGenotypeConcordance\tchecked    32,700,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: X:56,420,605\n",
      "INFO\t2017-01-11 17:36:57\tGenotypeConcordance\tchecked    32,710,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: X:66,541,098\n",
      "INFO\t2017-01-11 17:36:57\tGenotypeConcordance\tchecked    32,720,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: X:71,953,657\n",
      "INFO\t2017-01-11 17:36:58\tGenotypeConcordance\tchecked    32,730,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: X:77,354,225\n",
      "INFO\t2017-01-11 17:36:58\tGenotypeConcordance\tchecked    32,740,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: X:82,045,482\n",
      "INFO\t2017-01-11 17:36:58\tGenotypeConcordance\tchecked    32,750,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: X:86,374,697\n",
      "INFO\t2017-01-11 17:36:59\tGenotypeConcordance\tchecked    32,760,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: X:89,567,544\n",
      "INFO\t2017-01-11 17:36:59\tGenotypeConcordance\tchecked    32,770,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: X:93,423,195\n",
      "INFO\t2017-01-11 17:36:59\tGenotypeConcordance\tchecked    32,780,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: X:99,281,815\n",
      "INFO\t2017-01-11 17:36:59\tGenotypeConcordance\tchecked    32,790,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: X:106,502,505\n",
      "INFO\t2017-01-11 17:37:00\tGenotypeConcordance\tchecked    32,800,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: X:113,702,053\n",
      "INFO\t2017-01-11 17:37:00\tGenotypeConcordance\tchecked    32,810,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: X:117,536,606\n",
      "INFO\t2017-01-11 17:37:00\tGenotypeConcordance\tchecked    32,820,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: X:122,065,109\n",
      "INFO\t2017-01-11 17:37:01\tGenotypeConcordance\tchecked    32,830,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: X:126,859,678\n",
      "INFO\t2017-01-11 17:37:01\tGenotypeConcordance\tchecked    32,840,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: X:133,271,121\n",
      "INFO\t2017-01-11 17:37:01\tGenotypeConcordance\tchecked    32,850,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: X:139,596,676\n",
      "INFO\t2017-01-11 17:37:01\tGenotypeConcordance\tchecked    32,860,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: X:143,268,766\n",
      "INFO\t2017-01-11 17:37:02\tGenotypeConcordance\tchecked    32,870,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: X:147,221,049\n",
      "INFO\t2017-01-11 17:37:02\tGenotypeConcordance\tchecked    32,880,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: X:152,563,618\n",
      "INFO\t2017-01-11 17:37:02\tGenotypeConcordance\tchecked    32,890,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: Y:9,950,470\n",
      "INFO\t2017-01-11 17:37:03\tGenotypeConcordance\tchecked    32,900,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: Y:13,275,853\n",
      "INFO\t2017-01-11 17:37:03\tGenotypeConcordance\tchecked    32,910,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: Y:13,489,619\n",
      "INFO\t2017-01-11 17:37:03\tGenotypeConcordance\tchecked    32,920,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: Y:28,795,146\n",
      "INFO\t2017-01-11 17:37:04\tGenotypeConcordance\tchecked    32,930,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: GL000235.1:27,259\n",
      "INFO\t2017-01-11 17:37:04\tGenotypeConcordance\tchecked    32,940,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: GL000236.1:20,854\n",
      "INFO\t2017-01-11 17:37:04\tGenotypeConcordance\tchecked    32,950,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: GL000208.1:75,947\n",
      "INFO\t2017-01-11 17:37:05\tGenotypeConcordance\tchecked    32,960,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: GL000209.1:97,874\n",
      "INFO\t2017-01-11 17:37:05\tGenotypeConcordance\tchecked    32,970,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: GL000211.1:121,478\n",
      "INFO\t2017-01-11 17:37:05\tGenotypeConcordance\tchecked    32,980,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: GL000217.1:134,987\n",
      "INFO\t2017-01-11 17:37:05\tGenotypeConcordance\tchecked    32,990,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: GL000216.1:170,739\n",
      "INFO\t2017-01-11 17:37:06\tGenotypeConcordance\tchecked    33,000,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: GL000219.1:126,521\n",
      "INFO\t2017-01-11 17:37:06\tGenotypeConcordance\tchecked    33,010,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: GL000195.1:67,098\n",
      "INFO\t2017-01-11 17:37:06\tGenotypeConcordance\tchecked    33,020,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: GL000193.1:26,272\n",
      "INFO\t2017-01-11 17:37:07\tGenotypeConcordance\tchecked    33,030,000 variants.  Elapsed time: 00:15:41s.  Time for last 10,000:    0s.  Last read position: GL000225.1:33,627\n",
      "INFO\t2017-01-11 17:37:07\tGenotypeConcordance\tchecked    33,040,000 variants.  Elapsed time: 00:15:41s.  Time for last 10,000:    0s.  Last read position: GL000225.1:131,216\n",
      "INFO\t2017-01-11 17:37:07\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MIXED MIXED 163\n",
      "INFO\t2017-01-11 17:37:07\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: NO_VARIATION NO_VARIATION 30093676\n",
      "INFO\t2017-01-11 17:37:07\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP MNP 6265\n",
      "[Wed Jan 11 17:37:07 PST 2017] picard.vcf.GenotypeConcordance done. Elapsed time: 15.69 minutes.\n",
      "Runtime.totalMemory()=2623537152\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "############Concordance table met Picard \n",
    "import subprocess\n",
    "\n",
    "samples = [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"]\n",
    "\n",
    "for file_x in samples:\n",
    "    for file_y in samples:\n",
    "        print(\"\\n\")\n",
    "\n",
    "        terminal_code = (\"java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance \" + \\\n",
    "        \" CALL_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz \" + \\\n",
    "        \" CALL_SAMPLE= \" + file_x + \\\n",
    "        \" O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_\" + file_x + \"_truth_\" + file_y + \".vcf \" + \\\n",
    "        \" TRUTH_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz \" + \\\n",
    "        \" TRUTH_SAMPLE= \" + file_y )\n",
    "    \n",
    "        #print(terminal_code)\n",
    "        print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## Change the names of the header in terminal\n",
    "bcftools reheader merged_RNA_imputed_and_WGS_data.vcf.gz -s samples -o merged_RNA_imputed_and_WGS_data_new_header.vcf.gz\n",
    "\n",
    "bcftools query -l merged_RNA_imputed_and_WGS_data.vcf.gz\n",
    "bcftools query -l merged_RNA_imputed_and_WGS_data_new_header.vcf.gz\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr1.log ]\n",
      "Analysis started: Thu Jan 12 16:21:55 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr1\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr1\n",
      "\n",
      "2524436 (of 2524436) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr1.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr1.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr1.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 1:4372921 ]\n",
      "Duplicate marker name found: [ 1:4533115 ]\n",
      "Duplicate marker name found: [ 1:4533115 ]\n",
      "Duplicate marker name found: [ 1:5669414 ]\n",
      "Duplicate marker name found: [ 1:5669414 ]\n",
      "Duplicate marker name found: [ 1:7996478 ]\n",
      "Duplicate marker name found: [ 1:9359527 ]\n",
      "Duplicate marker name found: [ 1:9556654 ]\n",
      "Duplicate marker name found: [ 1:9979415 ]\n",
      "Duplicate marker name found: [ 1:9979415 ]\n",
      "Duplicate marker name found: [ 1:10162034 ]\n",
      "Duplicate marker name found: [ 1:11912243 ]\n",
      "Duplicate marker name found: [ 1:11913674 ]\n",
      "Duplicate marker name found: [ 1:11913674 ]\n",
      "Duplicate marker name found: [ 1:11913674 ]\n",
      "Duplicate marker name found: [ 1:12671939 ]\n",
      "Duplicate marker name found: [ 1:14804774 ]\n",
      "Duplicate marker name found: [ 1:15199495 ]\n",
      "Duplicate marker name found: [ 1:15747004 ]\n",
      "Duplicate marker name found: [ 1:15747004 ]\n",
      "Duplicate marker name found: [ 1:16492210 ]\n",
      "Duplicate marker name found: [ 1:16637469 ]\n",
      "Duplicate marker name found: [ 1:17938472 ]\n",
      "Duplicate marker name found: [ 1:18260397 ]\n",
      "Duplicate marker name found: [ 1:20956324 ]\n",
      "Duplicate marker name found: [ 1:21093443 ]\n",
      "Duplicate marker name found: [ 1:21775951 ]\n",
      "Duplicate marker name found: [ 1:21775951 ]\n",
      "Duplicate marker name found: [ 1:23793666 ]\n",
      "Duplicate marker name found: [ 1:24614120 ]\n",
      "Duplicate marker name found: [ 1:25220269 ]\n",
      "Duplicate marker name found: [ 1:25220269 ]\n",
      "Duplicate marker name found: [ 1:25472163 ]\n",
      "Duplicate marker name found: [ 1:26044490 ]\n",
      "Duplicate marker name found: [ 1:26468959 ]\n",
      "Duplicate marker name found: [ 1:28329983 ]\n",
      "Duplicate marker name found: [ 1:28470877 ]\n",
      "Duplicate marker name found: [ 1:28801451 ]\n",
      "Duplicate marker name found: [ 1:30385925 ]\n",
      "Duplicate marker name found: [ 1:30385925 ]\n",
      "Duplicate marker name found: [ 1:32468548 ]\n",
      "Duplicate marker name found: [ 1:32778683 ]\n",
      "Duplicate marker name found: [ 1:35360966 ]\n",
      "Duplicate marker name found: [ 1:36273771 ]\n",
      "Duplicate marker name found: [ 1:36485361 ]\n",
      "Duplicate marker name found: [ 1:41004497 ]\n",
      "Duplicate marker name found: [ 1:41621179 ]\n",
      "Duplicate marker name found: [ 1:42344529 ]\n",
      "Duplicate marker name found: [ 1:44553779 ]\n",
      "Duplicate marker name found: [ 1:47106307 ]\n",
      "Duplicate marker name found: [ 1:47801810 ]\n",
      "Duplicate marker name found: [ 1:47801810 ]\n",
      "Duplicate marker name found: [ 1:51645901 ]\n",
      "Duplicate marker name found: [ 1:52362869 ]\n",
      "Duplicate marker name found: [ 1:52484076 ]\n",
      "Duplicate marker name found: [ 1:54649648 ]\n",
      "Duplicate marker name found: [ 1:54867057 ]\n",
      "Duplicate marker name found: [ 1:55174090 ]\n",
      "Duplicate marker name found: [ 1:55174090 ]\n",
      "Duplicate marker name found: [ 1:55421117 ]\n",
      "Duplicate marker name found: [ 1:56800346 ]\n",
      "Duplicate marker name found: [ 1:57376738 ]\n",
      "Duplicate marker name found: [ 1:57983860 ]\n",
      "Duplicate marker name found: [ 1:58217506 ]\n",
      "Duplicate marker name found: [ 1:58941149 ]\n",
      "Duplicate marker name found: [ 1:59002734 ]\n",
      "Duplicate marker name found: [ 1:60267845 ]\n",
      "Duplicate marker name found: [ 1:60267845 ]\n",
      "Duplicate marker name found: [ 1:61924937 ]\n",
      "Duplicate marker name found: [ 1:67334756 ]\n",
      "Duplicate marker name found: [ 1:67334756 ]\n",
      "Duplicate marker name found: [ 1:67807901 ]\n",
      "Duplicate marker name found: [ 1:70355521 ]\n",
      "Duplicate marker name found: [ 1:70904523 ]\n",
      "Duplicate marker name found: [ 1:70904523 ]\n",
      "Duplicate marker name found: [ 1:71039227 ]\n",
      "Duplicate marker name found: [ 1:72571227 ]\n",
      "Duplicate marker name found: [ 1:73219534 ]\n",
      "Duplicate marker name found: [ 1:73614459 ]\n",
      "Duplicate marker name found: [ 1:73614459 ]\n",
      "Duplicate marker name found: [ 1:74821288 ]\n",
      "Duplicate marker name found: [ 1:76947824 ]\n",
      "Duplicate marker name found: [ 1:76947824 ]\n",
      "Duplicate marker name found: [ 1:82641386 ]\n",
      "Duplicate marker name found: [ 1:82757440 ]\n",
      "Duplicate marker name found: [ 1:83113053 ]\n",
      "Duplicate marker name found: [ 1:85225916 ]\n",
      "Duplicate marker name found: [ 1:85225916 ]\n",
      "Duplicate marker name found: [ 1:85225916 ]\n",
      "Duplicate marker name found: [ 1:85397712 ]\n",
      "Duplicate marker name found: [ 1:87286529 ]\n",
      "Duplicate marker name found: [ 1:87333173 ]\n",
      "Duplicate marker name found: [ 1:88342048 ]\n",
      "Duplicate marker name found: [ 1:89258088 ]\n",
      "Duplicate marker name found: [ 1:90000073 ]\n",
      "Duplicate marker name found: [ 1:90000073 ]\n",
      "Duplicate marker name found: [ 1:90270903 ]\n",
      "Duplicate marker name found: [ 1:90270903 ]\n",
      "Duplicate marker name found: [ 1:92875659 ]\n",
      "Duplicate marker name found: [ 1:93254033 ]\n",
      "Duplicate marker name found: [ 1:94125609 ]\n",
      "Duplicate marker name found: [ 1:94345343 ]\n",
      "Duplicate marker name found: [ 1:94515383 ]\n",
      "Duplicate marker name found: [ 1:94515383 ]\n",
      "Duplicate marker name found: [ 1:94872586 ]\n",
      "Duplicate marker name found: [ 1:95819759 ]\n",
      "Duplicate marker name found: [ 1:98668405 ]\n",
      "Duplicate marker name found: [ 1:98668405 ]\n",
      "Duplicate marker name found: [ 1:99541424 ]\n",
      "Duplicate marker name found: [ 1:99667133 ]\n",
      "Duplicate marker name found: [ 1:99667133 ]\n",
      "Duplicate marker name found: [ 1:105163778 ]\n",
      "Duplicate marker name found: [ 1:105163778 ]\n",
      "Duplicate marker name found: [ 1:105181107 ]\n",
      "Duplicate marker name found: [ 1:105397460 ]\n",
      "Duplicate marker name found: [ 1:109025352 ]\n",
      "Duplicate marker name found: [ 1:109787579 ]\n",
      "Duplicate marker name found: [ 1:110131508 ]\n",
      "Duplicate marker name found: [ 1:110447824 ]\n",
      "Duplicate marker name found: [ 1:110447824 ]\n",
      "Duplicate marker name found: [ 1:110877470 ]\n",
      "Duplicate marker name found: [ 1:111429680 ]\n",
      "Duplicate marker name found: [ 1:112161442 ]\n",
      "Duplicate marker name found: [ 1:114353006 ]\n",
      "Duplicate marker name found: [ 1:114825452 ]\n",
      "Duplicate marker name found: [ 1:114825452 ]\n",
      "Duplicate marker name found: [ 1:114986007 ]\n",
      "Duplicate marker name found: [ 1:115072202 ]\n",
      "Duplicate marker name found: [ 1:115112889 ]\n",
      "Duplicate marker name found: [ 1:115910207 ]\n",
      "Duplicate marker name found: [ 1:116089843 ]\n",
      "Duplicate marker name found: [ 1:117573001 ]\n",
      "Duplicate marker name found: [ 1:146508195 ]\n",
      "Duplicate marker name found: [ 1:147324681 ]\n",
      "Duplicate marker name found: [ 1:150180839 ]\n",
      "Duplicate marker name found: [ 1:150180839 ]\n",
      "Duplicate marker name found: [ 1:150256412 ]\n",
      "Duplicate marker name found: [ 1:150434938 ]\n",
      "Duplicate marker name found: [ 1:150648462 ]\n",
      "Duplicate marker name found: [ 1:150748134 ]\n",
      "Duplicate marker name found: [ 1:151136354 ]\n",
      "Duplicate marker name found: [ 1:151136354 ]\n",
      "Duplicate marker name found: [ 1:151136354 ]\n",
      "Duplicate marker name found: [ 1:153137633 ]\n",
      "Duplicate marker name found: [ 1:153285078 ]\n",
      "Duplicate marker name found: [ 1:153285078 ]\n",
      "Duplicate marker name found: [ 1:154392987 ]\n",
      "Duplicate marker name found: [ 1:155127504 ]\n",
      "Duplicate marker name found: [ 1:155755890 ]\n",
      "Duplicate marker name found: [ 1:156193461 ]\n",
      "Duplicate marker name found: [ 1:156193461 ]\n",
      "Duplicate marker name found: [ 1:157177383 ]\n",
      "Duplicate marker name found: [ 1:157177383 ]\n",
      "Duplicate marker name found: [ 1:160073671 ]\n",
      "Duplicate marker name found: [ 1:160073671 ]\n",
      "Duplicate marker name found: [ 1:160397344 ]\n",
      "Duplicate marker name found: [ 1:160397344 ]\n",
      "Duplicate marker name found: [ 1:161037360 ]\n",
      "Duplicate marker name found: [ 1:162978188 ]\n",
      "Duplicate marker name found: [ 1:162978188 ]\n",
      "Duplicate marker name found: [ 1:162978188 ]\n",
      "Duplicate marker name found: [ 1:163363718 ]\n",
      "Duplicate marker name found: [ 1:163363718 ]\n",
      "Duplicate marker name found: [ 1:163770731 ]\n",
      "Duplicate marker name found: [ 1:167172918 ]\n",
      "Duplicate marker name found: [ 1:168902072 ]\n",
      "Duplicate marker name found: [ 1:170420792 ]\n",
      "Duplicate marker name found: [ 1:171287455 ]\n",
      "Duplicate marker name found: [ 1:171287455 ]\n",
      "Duplicate marker name found: [ 1:171287455 ]\n",
      "Duplicate marker name found: [ 1:171647627 ]\n",
      "Duplicate marker name found: [ 1:171647627 ]\n",
      "Duplicate marker name found: [ 1:172280452 ]\n",
      "Duplicate marker name found: [ 1:174631544 ]\n",
      "Duplicate marker name found: [ 1:174651409 ]\n",
      "Duplicate marker name found: [ 1:176062974 ]\n",
      "Duplicate marker name found: [ 1:176948822 ]\n",
      "Duplicate marker name found: [ 1:177235257 ]\n",
      "Duplicate marker name found: [ 1:177517823 ]\n",
      "Duplicate marker name found: [ 1:178117568 ]\n",
      "Duplicate marker name found: [ 1:178434937 ]\n",
      "Duplicate marker name found: [ 1:178469116 ]\n",
      "Duplicate marker name found: [ 1:178469116 ]\n",
      "Duplicate marker name found: [ 1:179201808 ]\n",
      "Duplicate marker name found: [ 1:179201808 ]\n",
      "Duplicate marker name found: [ 1:179765093 ]\n",
      "Duplicate marker name found: [ 1:181359438 ]\n",
      "Duplicate marker name found: [ 1:184350703 ]\n",
      "Duplicate marker name found: [ 1:184350703 ]\n",
      "Duplicate marker name found: [ 1:184637788 ]\n",
      "Duplicate marker name found: [ 1:184880195 ]\n",
      "Duplicate marker name found: [ 1:184929755 ]\n",
      "Duplicate marker name found: [ 1:184929755 ]\n",
      "Duplicate marker name found: [ 1:184929755 ]\n",
      "Duplicate marker name found: [ 1:185649218 ]\n",
      "Duplicate marker name found: [ 1:185649218 ]\n",
      "Duplicate marker name found: [ 1:185989369 ]\n",
      "Duplicate marker name found: [ 1:186617298 ]\n",
      "Duplicate marker name found: [ 1:189098850 ]\n",
      "Duplicate marker name found: [ 1:191332307 ]\n",
      "Duplicate marker name found: [ 1:192441626 ]\n",
      "Duplicate marker name found: [ 1:193770768 ]\n",
      "Duplicate marker name found: [ 1:193786297 ]\n",
      "Duplicate marker name found: [ 1:194948015 ]\n",
      "Duplicate marker name found: [ 1:194948015 ]\n",
      "Duplicate marker name found: [ 1:194948015 ]\n",
      "Duplicate marker name found: [ 1:195272696 ]\n",
      "Duplicate marker name found: [ 1:195497863 ]\n",
      "Duplicate marker name found: [ 1:200119169 ]\n",
      "Duplicate marker name found: [ 1:202692989 ]\n",
      "Duplicate marker name found: [ 1:203484206 ]\n",
      "Duplicate marker name found: [ 1:208587104 ]\n",
      "Duplicate marker name found: [ 1:209025912 ]\n",
      "Duplicate marker name found: [ 1:209025912 ]\n",
      "Duplicate marker name found: [ 1:209521798 ]\n",
      "Duplicate marker name found: [ 1:209902708 ]\n",
      "Duplicate marker name found: [ 1:209902708 ]\n",
      "Duplicate marker name found: [ 1:211269389 ]\n",
      "Duplicate marker name found: [ 1:211352710 ]\n",
      "Duplicate marker name found: [ 1:211773331 ]\n",
      "Duplicate marker name found: [ 1:211791937 ]\n",
      "Duplicate marker name found: [ 1:212219266 ]\n",
      "Duplicate marker name found: [ 1:212219266 ]\n",
      "Duplicate marker name found: [ 1:212797111 ]\n",
      "Duplicate marker name found: [ 1:213664887 ]\n",
      "Duplicate marker name found: [ 1:213815172 ]\n",
      "Duplicate marker name found: [ 1:213815172 ]\n",
      "Duplicate marker name found: [ 1:214749047 ]\n",
      "Duplicate marker name found: [ 1:214749047 ]\n",
      "Duplicate marker name found: [ 1:215735840 ]\n",
      "Duplicate marker name found: [ 1:215804569 ]\n",
      "Duplicate marker name found: [ 1:215804569 ]\n",
      "Duplicate marker name found: [ 1:216259061 ]\n",
      "Duplicate marker name found: [ 1:218436679 ]\n",
      "Duplicate marker name found: [ 1:219613304 ]\n",
      "Duplicate marker name found: [ 1:220982833 ]\n",
      "Duplicate marker name found: [ 1:221137773 ]\n",
      "Duplicate marker name found: [ 1:223039665 ]\n",
      "Duplicate marker name found: [ 1:223225693 ]\n",
      "Duplicate marker name found: [ 1:226193307 ]\n",
      "Duplicate marker name found: [ 1:226758934 ]\n",
      "Duplicate marker name found: [ 1:229346559 ]\n",
      "Duplicate marker name found: [ 1:231422894 ]\n",
      "Duplicate marker name found: [ 1:231422894 ]\n",
      "Duplicate marker name found: [ 1:231720148 ]\n",
      "Duplicate marker name found: [ 1:231720148 ]\n",
      "Duplicate marker name found: [ 1:231720148 ]\n",
      "Duplicate marker name found: [ 1:233764506 ]\n",
      "Duplicate marker name found: [ 1:234455452 ]\n",
      "Duplicate marker name found: [ 1:234455452 ]\n",
      "Duplicate marker name found: [ 1:234519319 ]\n",
      "Duplicate marker name found: [ 1:234519319 ]\n",
      "Duplicate marker name found: [ 1:235620423 ]\n",
      "Duplicate marker name found: [ 1:236197722 ]\n",
      "Duplicate marker name found: [ 1:236921896 ]\n",
      "Duplicate marker name found: [ 1:237563337 ]\n",
      "Duplicate marker name found: [ 1:237563337 ]\n",
      "Duplicate marker name found: [ 1:238393639 ]\n",
      "Duplicate marker name found: [ 1:238537500 ]\n",
      "Duplicate marker name found: [ 1:238537500 ]\n",
      "Duplicate marker name found: [ 1:238537500 ]\n",
      "Duplicate marker name found: [ 1:238601745 ]\n",
      "Duplicate marker name found: [ 1:238992448 ]\n",
      "Duplicate marker name found: [ 1:240880920 ]\n",
      "Duplicate marker name found: [ 1:242048514 ]\n",
      "Duplicate marker name found: [ 1:242048514 ]\n",
      "Duplicate marker name found: [ 1:242048514 ]\n",
      "Duplicate marker name found: [ 1:242843543 ]\n",
      "Duplicate marker name found: [ 1:245831998 ]\n",
      "Duplicate marker name found: [ 1:248147531 ]\n",
      "Duplicate marker name found: [ 1:248147531 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2524436 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.778147\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2524436 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr1.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:22:40 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr2.log ]\n",
      "Analysis started: Thu Jan 12 16:22:41 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr2\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr2\n",
      "\n",
      "2702779 (of 2702779) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr2.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr2.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr2.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 2:1548796 ]\n",
      "Duplicate marker name found: [ 2:6324566 ]\n",
      "Duplicate marker name found: [ 2:7245913 ]\n",
      "Duplicate marker name found: [ 2:8334822 ]\n",
      "Duplicate marker name found: [ 2:9576903 ]\n",
      "Duplicate marker name found: [ 2:9576903 ]\n",
      "Duplicate marker name found: [ 2:9667412 ]\n",
      "Duplicate marker name found: [ 2:12846546 ]\n",
      "Duplicate marker name found: [ 2:13221067 ]\n",
      "Duplicate marker name found: [ 2:13221067 ]\n",
      "Duplicate marker name found: [ 2:13437884 ]\n",
      "Duplicate marker name found: [ 2:13956611 ]\n",
      "Duplicate marker name found: [ 2:15215884 ]\n",
      "Duplicate marker name found: [ 2:15215884 ]\n",
      "Duplicate marker name found: [ 2:15614673 ]\n",
      "Duplicate marker name found: [ 2:15614673 ]\n",
      "Duplicate marker name found: [ 2:15614673 ]\n",
      "Duplicate marker name found: [ 2:15899420 ]\n",
      "Duplicate marker name found: [ 2:15899420 ]\n",
      "Duplicate marker name found: [ 2:17088973 ]\n",
      "Duplicate marker name found: [ 2:18450387 ]\n",
      "Duplicate marker name found: [ 2:18450387 ]\n",
      "Duplicate marker name found: [ 2:18696677 ]\n",
      "Duplicate marker name found: [ 2:21884270 ]\n",
      "Duplicate marker name found: [ 2:22566618 ]\n",
      "Duplicate marker name found: [ 2:23141292 ]\n",
      "Duplicate marker name found: [ 2:23407784 ]\n",
      "Duplicate marker name found: [ 2:24214276 ]\n",
      "Duplicate marker name found: [ 2:25914544 ]\n",
      "Duplicate marker name found: [ 2:26078751 ]\n",
      "Duplicate marker name found: [ 2:26078751 ]\n",
      "Duplicate marker name found: [ 2:26285548 ]\n",
      "Duplicate marker name found: [ 2:28068394 ]\n",
      "Duplicate marker name found: [ 2:28068394 ]\n",
      "Duplicate marker name found: [ 2:29821231 ]\n",
      "Duplicate marker name found: [ 2:30022372 ]\n",
      "Duplicate marker name found: [ 2:30298817 ]\n",
      "Duplicate marker name found: [ 2:31734145 ]\n",
      "Duplicate marker name found: [ 2:31734145 ]\n",
      "Duplicate marker name found: [ 2:32819769 ]\n",
      "Duplicate marker name found: [ 2:32933745 ]\n",
      "Duplicate marker name found: [ 2:33469927 ]\n",
      "Duplicate marker name found: [ 2:33763570 ]\n",
      "Duplicate marker name found: [ 2:33965166 ]\n",
      "Duplicate marker name found: [ 2:33965166 ]\n",
      "Duplicate marker name found: [ 2:35742208 ]\n",
      "Duplicate marker name found: [ 2:35742208 ]\n",
      "Duplicate marker name found: [ 2:37057065 ]\n",
      "Duplicate marker name found: [ 2:37585581 ]\n",
      "Duplicate marker name found: [ 2:38799354 ]\n",
      "Duplicate marker name found: [ 2:38994448 ]\n",
      "Duplicate marker name found: [ 2:39054275 ]\n",
      "Duplicate marker name found: [ 2:41263368 ]\n",
      "Duplicate marker name found: [ 2:45125506 ]\n",
      "Duplicate marker name found: [ 2:46476079 ]\n",
      "Duplicate marker name found: [ 2:50013381 ]\n",
      "Duplicate marker name found: [ 2:50081705 ]\n",
      "Duplicate marker name found: [ 2:50081705 ]\n",
      "Duplicate marker name found: [ 2:51051439 ]\n",
      "Duplicate marker name found: [ 2:51051439 ]\n",
      "Duplicate marker name found: [ 2:51808786 ]\n",
      "Duplicate marker name found: [ 2:51808786 ]\n",
      "Duplicate marker name found: [ 2:51884264 ]\n",
      "Duplicate marker name found: [ 2:53755377 ]\n",
      "Duplicate marker name found: [ 2:53881300 ]\n",
      "Duplicate marker name found: [ 2:54246746 ]\n",
      "Duplicate marker name found: [ 2:54270209 ]\n",
      "Duplicate marker name found: [ 2:55739523 ]\n",
      "Duplicate marker name found: [ 2:55936439 ]\n",
      "Duplicate marker name found: [ 2:56480154 ]\n",
      "Duplicate marker name found: [ 2:56607764 ]\n",
      "Duplicate marker name found: [ 2:58012684 ]\n",
      "Duplicate marker name found: [ 2:59439241 ]\n",
      "Duplicate marker name found: [ 2:59874098 ]\n",
      "Duplicate marker name found: [ 2:60980266 ]\n",
      "Duplicate marker name found: [ 2:61974804 ]\n",
      "Duplicate marker name found: [ 2:61990610 ]\n",
      "Duplicate marker name found: [ 2:62024509 ]\n",
      "Duplicate marker name found: [ 2:64011705 ]\n",
      "Duplicate marker name found: [ 2:66296821 ]\n",
      "Duplicate marker name found: [ 2:66779160 ]\n",
      "Duplicate marker name found: [ 2:66779160 ]\n",
      "Duplicate marker name found: [ 2:69830872 ]\n",
      "Duplicate marker name found: [ 2:70012683 ]\n",
      "Duplicate marker name found: [ 2:70012683 ]\n",
      "Duplicate marker name found: [ 2:70012683 ]\n",
      "Duplicate marker name found: [ 2:70245280 ]\n",
      "Duplicate marker name found: [ 2:70281708 ]\n",
      "Duplicate marker name found: [ 2:72450814 ]\n",
      "Duplicate marker name found: [ 2:72450814 ]\n",
      "Duplicate marker name found: [ 2:72590726 ]\n",
      "Duplicate marker name found: [ 2:72590726 ]\n",
      "Duplicate marker name found: [ 2:73343224 ]\n",
      "Duplicate marker name found: [ 2:74496505 ]\n",
      "Duplicate marker name found: [ 2:75109732 ]\n",
      "Duplicate marker name found: [ 2:75551203 ]\n",
      "Duplicate marker name found: [ 2:76464439 ]\n",
      "Duplicate marker name found: [ 2:77378243 ]\n",
      "Duplicate marker name found: [ 2:78291450 ]\n",
      "Duplicate marker name found: [ 2:79612621 ]\n",
      "Duplicate marker name found: [ 2:79612621 ]\n",
      "Duplicate marker name found: [ 2:84034466 ]\n",
      "Duplicate marker name found: [ 2:84242364 ]\n",
      "Duplicate marker name found: [ 2:84242364 ]\n",
      "Duplicate marker name found: [ 2:85225691 ]\n",
      "Duplicate marker name found: [ 2:86228871 ]\n",
      "Duplicate marker name found: [ 2:86803596 ]\n",
      "Duplicate marker name found: [ 2:88665165 ]\n",
      "Duplicate marker name found: [ 2:96006086 ]\n",
      "Duplicate marker name found: [ 2:97521709 ]\n",
      "Duplicate marker name found: [ 2:100235355 ]\n",
      "Duplicate marker name found: [ 2:100235355 ]\n",
      "Duplicate marker name found: [ 2:100523306 ]\n",
      "Duplicate marker name found: [ 2:100554894 ]\n",
      "Duplicate marker name found: [ 2:100643178 ]\n",
      "Duplicate marker name found: [ 2:100838205 ]\n",
      "Duplicate marker name found: [ 2:100838205 ]\n",
      "Duplicate marker name found: [ 2:101033150 ]\n",
      "Duplicate marker name found: [ 2:101033150 ]\n",
      "Duplicate marker name found: [ 2:101777256 ]\n",
      "Duplicate marker name found: [ 2:102248988 ]\n",
      "Duplicate marker name found: [ 2:103115766 ]\n",
      "Duplicate marker name found: [ 2:103389398 ]\n",
      "Duplicate marker name found: [ 2:103684869 ]\n",
      "Duplicate marker name found: [ 2:104486169 ]\n",
      "Duplicate marker name found: [ 2:104486169 ]\n",
      "Duplicate marker name found: [ 2:104716606 ]\n",
      "Duplicate marker name found: [ 2:109663952 ]\n",
      "Duplicate marker name found: [ 2:110854287 ]\n",
      "Duplicate marker name found: [ 2:112451667 ]\n",
      "Duplicate marker name found: [ 2:112669574 ]\n",
      "Duplicate marker name found: [ 2:114439276 ]\n",
      "Duplicate marker name found: [ 2:114439276 ]\n",
      "Duplicate marker name found: [ 2:114439276 ]\n",
      "Duplicate marker name found: [ 2:116607618 ]\n",
      "Duplicate marker name found: [ 2:117962796 ]\n",
      "Duplicate marker name found: [ 2:119717694 ]\n",
      "Duplicate marker name found: [ 2:120100327 ]\n",
      "Duplicate marker name found: [ 2:120235013 ]\n",
      "Duplicate marker name found: [ 2:122422836 ]\n",
      "Duplicate marker name found: [ 2:122439595 ]\n",
      "Duplicate marker name found: [ 2:123341078 ]\n",
      "Duplicate marker name found: [ 2:124688094 ]\n",
      "Duplicate marker name found: [ 2:127982285 ]\n",
      "Duplicate marker name found: [ 2:129641719 ]\n",
      "Duplicate marker name found: [ 2:130390517 ]\n",
      "Duplicate marker name found: [ 2:134843695 ]\n",
      "Duplicate marker name found: [ 2:135590359 ]\n",
      "Duplicate marker name found: [ 2:135839711 ]\n",
      "Duplicate marker name found: [ 2:140870229 ]\n",
      "Duplicate marker name found: [ 2:143263287 ]\n",
      "Duplicate marker name found: [ 2:144122581 ]\n",
      "Duplicate marker name found: [ 2:148084527 ]\n",
      "Duplicate marker name found: [ 2:148298476 ]\n",
      "Duplicate marker name found: [ 2:149599704 ]\n",
      "Duplicate marker name found: [ 2:149599704 ]\n",
      "Duplicate marker name found: [ 2:151222752 ]\n",
      "Duplicate marker name found: [ 2:151347136 ]\n",
      "Duplicate marker name found: [ 2:152150935 ]\n",
      "Duplicate marker name found: [ 2:153471255 ]\n",
      "Duplicate marker name found: [ 2:153896534 ]\n",
      "Duplicate marker name found: [ 2:157575593 ]\n",
      "Duplicate marker name found: [ 2:158534420 ]\n",
      "Duplicate marker name found: [ 2:158548904 ]\n",
      "Duplicate marker name found: [ 2:159687981 ]\n",
      "Duplicate marker name found: [ 2:163050782 ]\n",
      "Duplicate marker name found: [ 2:163050782 ]\n",
      "Duplicate marker name found: [ 2:163545653 ]\n",
      "Duplicate marker name found: [ 2:164519835 ]\n",
      "Duplicate marker name found: [ 2:165034530 ]\n",
      "Duplicate marker name found: [ 2:165297965 ]\n",
      "Duplicate marker name found: [ 2:166713168 ]\n",
      "Duplicate marker name found: [ 2:166713168 ]\n",
      "Duplicate marker name found: [ 2:167527474 ]\n",
      "Duplicate marker name found: [ 2:169450036 ]\n",
      "Duplicate marker name found: [ 2:169681278 ]\n",
      "Duplicate marker name found: [ 2:172490240 ]\n",
      "Duplicate marker name found: [ 2:172490240 ]\n",
      "Duplicate marker name found: [ 2:177267741 ]\n",
      "Duplicate marker name found: [ 2:178636729 ]\n",
      "Duplicate marker name found: [ 2:179292512 ]\n",
      "Duplicate marker name found: [ 2:182889584 ]\n",
      "Duplicate marker name found: [ 2:185091439 ]\n",
      "Duplicate marker name found: [ 2:185091439 ]\n",
      "Duplicate marker name found: [ 2:185091439 ]\n",
      "Duplicate marker name found: [ 2:185259443 ]\n",
      "Duplicate marker name found: [ 2:185259443 ]\n",
      "Duplicate marker name found: [ 2:186118815 ]\n",
      "Duplicate marker name found: [ 2:188743308 ]\n",
      "Duplicate marker name found: [ 2:190267513 ]\n",
      "Duplicate marker name found: [ 2:192613530 ]\n",
      "Duplicate marker name found: [ 2:192613530 ]\n",
      "Duplicate marker name found: [ 2:192620855 ]\n",
      "Duplicate marker name found: [ 2:192620855 ]\n",
      "Duplicate marker name found: [ 2:192788681 ]\n",
      "Duplicate marker name found: [ 2:194014901 ]\n",
      "Duplicate marker name found: [ 2:196008460 ]\n",
      "Duplicate marker name found: [ 2:196276510 ]\n",
      "Duplicate marker name found: [ 2:197152726 ]\n",
      "Duplicate marker name found: [ 2:198063523 ]\n",
      "Duplicate marker name found: [ 2:199212087 ]\n",
      "Duplicate marker name found: [ 2:201191625 ]\n",
      "Duplicate marker name found: [ 2:202370991 ]\n",
      "Duplicate marker name found: [ 2:205443839 ]\n",
      "Duplicate marker name found: [ 2:205821239 ]\n",
      "Duplicate marker name found: [ 2:207338296 ]\n",
      "Duplicate marker name found: [ 2:208963606 ]\n",
      "Duplicate marker name found: [ 2:209506287 ]\n",
      "Duplicate marker name found: [ 2:209582221 ]\n",
      "Duplicate marker name found: [ 2:210944869 ]\n",
      "Duplicate marker name found: [ 2:211390530 ]\n",
      "Duplicate marker name found: [ 2:212788437 ]\n",
      "Duplicate marker name found: [ 2:213262216 ]\n",
      "Duplicate marker name found: [ 2:213287224 ]\n",
      "Duplicate marker name found: [ 2:213287224 ]\n",
      "Duplicate marker name found: [ 2:214305280 ]\n",
      "Duplicate marker name found: [ 2:214305280 ]\n",
      "Duplicate marker name found: [ 2:214841543 ]\n",
      "Duplicate marker name found: [ 2:216376343 ]\n",
      "Duplicate marker name found: [ 2:216738516 ]\n",
      "Duplicate marker name found: [ 2:216891076 ]\n",
      "Duplicate marker name found: [ 2:217911189 ]\n",
      "Duplicate marker name found: [ 2:219224862 ]\n",
      "Duplicate marker name found: [ 2:223912605 ]\n",
      "Duplicate marker name found: [ 2:224422863 ]\n",
      "Duplicate marker name found: [ 2:224429104 ]\n",
      "Duplicate marker name found: [ 2:224501585 ]\n",
      "Duplicate marker name found: [ 2:224501585 ]\n",
      "Duplicate marker name found: [ 2:225522522 ]\n",
      "Duplicate marker name found: [ 2:225695637 ]\n",
      "Duplicate marker name found: [ 2:225695637 ]\n",
      "Duplicate marker name found: [ 2:226736129 ]\n",
      "Duplicate marker name found: [ 2:226736129 ]\n",
      "Duplicate marker name found: [ 2:227762704 ]\n",
      "Duplicate marker name found: [ 2:227838941 ]\n",
      "Duplicate marker name found: [ 2:229373268 ]\n",
      "Duplicate marker name found: [ 2:232556650 ]\n",
      "Duplicate marker name found: [ 2:234661392 ]\n",
      "Duplicate marker name found: [ 2:235209114 ]\n",
      "Duplicate marker name found: [ 2:235209114 ]\n",
      "Duplicate marker name found: [ 2:236636945 ]\n",
      "Duplicate marker name found: [ 2:236921408 ]\n",
      "Duplicate marker name found: [ 2:237338841 ]\n",
      "Duplicate marker name found: [ 2:240681066 ]\n",
      "Duplicate marker name found: [ 2:241341294 ]\n",
      "Duplicate marker name found: [ 2:241341294 ]\n",
      "Duplicate marker name found: [ 2:242666700 ]\n",
      "Duplicate marker name found: [ 2:242666700 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2702779 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.784703\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2702779 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr2.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:23:30 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr3.log ]\n",
      "Analysis started: Thu Jan 12 16:23:31 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr3\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr3\n",
      "\n",
      "2228228 (of 2228228) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr3.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr3.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr3.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 3:154855 ]\n",
      "Duplicate marker name found: [ 3:652154 ]\n",
      "Duplicate marker name found: [ 3:652154 ]\n",
      "Duplicate marker name found: [ 3:865946 ]\n",
      "Duplicate marker name found: [ 3:1687407 ]\n",
      "Duplicate marker name found: [ 3:5700939 ]\n",
      "Duplicate marker name found: [ 3:6775450 ]\n",
      "Duplicate marker name found: [ 3:9205960 ]\n",
      "Duplicate marker name found: [ 3:9205960 ]\n",
      "Duplicate marker name found: [ 3:9755756 ]\n",
      "Duplicate marker name found: [ 3:15456550 ]\n",
      "Duplicate marker name found: [ 3:17412560 ]\n",
      "Duplicate marker name found: [ 3:17575381 ]\n",
      "Duplicate marker name found: [ 3:17575381 ]\n",
      "Duplicate marker name found: [ 3:17575381 ]\n",
      "Duplicate marker name found: [ 3:19329064 ]\n",
      "Duplicate marker name found: [ 3:19329064 ]\n",
      "Duplicate marker name found: [ 3:20071540 ]\n",
      "Duplicate marker name found: [ 3:20074801 ]\n",
      "Duplicate marker name found: [ 3:22058427 ]\n",
      "Duplicate marker name found: [ 3:22058427 ]\n",
      "Duplicate marker name found: [ 3:22595426 ]\n",
      "Duplicate marker name found: [ 3:24424709 ]\n",
      "Duplicate marker name found: [ 3:26356299 ]\n",
      "Duplicate marker name found: [ 3:26356299 ]\n",
      "Duplicate marker name found: [ 3:27622920 ]\n",
      "Duplicate marker name found: [ 3:28167934 ]\n",
      "Duplicate marker name found: [ 3:30767976 ]\n",
      "Duplicate marker name found: [ 3:31054686 ]\n",
      "Duplicate marker name found: [ 3:34507804 ]\n",
      "Duplicate marker name found: [ 3:34507804 ]\n",
      "Duplicate marker name found: [ 3:36358690 ]\n",
      "Duplicate marker name found: [ 3:36358690 ]\n",
      "Duplicate marker name found: [ 3:36358690 ]\n",
      "Duplicate marker name found: [ 3:36497252 ]\n",
      "Duplicate marker name found: [ 3:37256652 ]\n",
      "Duplicate marker name found: [ 3:37256652 ]\n",
      "Duplicate marker name found: [ 3:37256652 ]\n",
      "Duplicate marker name found: [ 3:39469685 ]\n",
      "Duplicate marker name found: [ 3:42468436 ]\n",
      "Duplicate marker name found: [ 3:44653251 ]\n",
      "Duplicate marker name found: [ 3:45219349 ]\n",
      "Duplicate marker name found: [ 3:45219349 ]\n",
      "Duplicate marker name found: [ 3:45243215 ]\n",
      "Duplicate marker name found: [ 3:45440958 ]\n",
      "Duplicate marker name found: [ 3:45625489 ]\n",
      "Duplicate marker name found: [ 3:48039197 ]\n",
      "Duplicate marker name found: [ 3:48039197 ]\n",
      "Duplicate marker name found: [ 3:50254188 ]\n",
      "Duplicate marker name found: [ 3:50254188 ]\n",
      "Duplicate marker name found: [ 3:52609242 ]\n",
      "Duplicate marker name found: [ 3:52609242 ]\n",
      "Duplicate marker name found: [ 3:52824284 ]\n",
      "Duplicate marker name found: [ 3:52824284 ]\n",
      "Duplicate marker name found: [ 3:53576607 ]\n",
      "Duplicate marker name found: [ 3:54022528 ]\n",
      "Duplicate marker name found: [ 3:54022528 ]\n",
      "Duplicate marker name found: [ 3:55690126 ]\n",
      "Duplicate marker name found: [ 3:57620927 ]\n",
      "Duplicate marker name found: [ 3:62392810 ]\n",
      "Duplicate marker name found: [ 3:63995579 ]\n",
      "Duplicate marker name found: [ 3:64526474 ]\n",
      "Duplicate marker name found: [ 3:64890566 ]\n",
      "Duplicate marker name found: [ 3:65607592 ]\n",
      "Duplicate marker name found: [ 3:65607592 ]\n",
      "Duplicate marker name found: [ 3:65607592 ]\n",
      "Duplicate marker name found: [ 3:65607592 ]\n",
      "Duplicate marker name found: [ 3:66865963 ]\n",
      "Duplicate marker name found: [ 3:66865963 ]\n",
      "Duplicate marker name found: [ 3:67532606 ]\n",
      "Duplicate marker name found: [ 3:68510433 ]\n",
      "Duplicate marker name found: [ 3:68995447 ]\n",
      "Duplicate marker name found: [ 3:71203247 ]\n",
      "Duplicate marker name found: [ 3:71528830 ]\n",
      "Duplicate marker name found: [ 3:73239455 ]\n",
      "Duplicate marker name found: [ 3:74904974 ]\n",
      "Duplicate marker name found: [ 3:75571194 ]\n",
      "Duplicate marker name found: [ 3:76130988 ]\n",
      "Duplicate marker name found: [ 3:76569924 ]\n",
      "Duplicate marker name found: [ 3:77123888 ]\n",
      "Duplicate marker name found: [ 3:80193917 ]\n",
      "Duplicate marker name found: [ 3:80193917 ]\n",
      "Duplicate marker name found: [ 3:80680004 ]\n",
      "Duplicate marker name found: [ 3:81967495 ]\n",
      "Duplicate marker name found: [ 3:82224983 ]\n",
      "Duplicate marker name found: [ 3:82988805 ]\n",
      "Duplicate marker name found: [ 3:85065942 ]\n",
      "Duplicate marker name found: [ 3:85826394 ]\n",
      "Duplicate marker name found: [ 3:86998976 ]\n",
      "Duplicate marker name found: [ 3:89365627 ]\n",
      "Duplicate marker name found: [ 3:94998614 ]\n",
      "Duplicate marker name found: [ 3:95640937 ]\n",
      "Duplicate marker name found: [ 3:95640937 ]\n",
      "Duplicate marker name found: [ 3:95640937 ]\n",
      "Duplicate marker name found: [ 3:98563187 ]\n",
      "Duplicate marker name found: [ 3:98563187 ]\n",
      "Duplicate marker name found: [ 3:98563187 ]\n",
      "Duplicate marker name found: [ 3:98697788 ]\n",
      "Duplicate marker name found: [ 3:99954300 ]\n",
      "Duplicate marker name found: [ 3:100834358 ]\n",
      "Duplicate marker name found: [ 3:103044340 ]\n",
      "Duplicate marker name found: [ 3:103385068 ]\n",
      "Duplicate marker name found: [ 3:103402028 ]\n",
      "Duplicate marker name found: [ 3:103560061 ]\n",
      "Duplicate marker name found: [ 3:106919571 ]\n",
      "Duplicate marker name found: [ 3:107043413 ]\n",
      "Duplicate marker name found: [ 3:107179786 ]\n",
      "Duplicate marker name found: [ 3:107179786 ]\n",
      "Duplicate marker name found: [ 3:107179786 ]\n",
      "Duplicate marker name found: [ 3:108255114 ]\n",
      "Duplicate marker name found: [ 3:109201054 ]\n",
      "Duplicate marker name found: [ 3:111732646 ]\n",
      "Duplicate marker name found: [ 3:114754391 ]\n",
      "Duplicate marker name found: [ 3:115039723 ]\n",
      "Duplicate marker name found: [ 3:115039723 ]\n",
      "Duplicate marker name found: [ 3:116513023 ]\n",
      "Duplicate marker name found: [ 3:116979531 ]\n",
      "Duplicate marker name found: [ 3:117715300 ]\n",
      "Duplicate marker name found: [ 3:118814893 ]\n",
      "Duplicate marker name found: [ 3:119813231 ]\n",
      "Duplicate marker name found: [ 3:120824242 ]\n",
      "Duplicate marker name found: [ 3:120824242 ]\n",
      "Duplicate marker name found: [ 3:120824242 ]\n",
      "Duplicate marker name found: [ 3:122478032 ]\n",
      "Duplicate marker name found: [ 3:122504687 ]\n",
      "Duplicate marker name found: [ 3:122504687 ]\n",
      "Duplicate marker name found: [ 3:123330286 ]\n",
      "Duplicate marker name found: [ 3:123552875 ]\n",
      "Duplicate marker name found: [ 3:124825200 ]\n",
      "Duplicate marker name found: [ 3:125198400 ]\n",
      "Duplicate marker name found: [ 3:126018048 ]\n",
      "Duplicate marker name found: [ 3:126617366 ]\n",
      "Duplicate marker name found: [ 3:126836255 ]\n",
      "Duplicate marker name found: [ 3:127895824 ]\n",
      "Duplicate marker name found: [ 3:128142321 ]\n",
      "Duplicate marker name found: [ 3:129052507 ]\n",
      "Duplicate marker name found: [ 3:129052507 ]\n",
      "Duplicate marker name found: [ 3:129057299 ]\n",
      "Duplicate marker name found: [ 3:129057299 ]\n",
      "Duplicate marker name found: [ 3:129135261 ]\n",
      "Duplicate marker name found: [ 3:130958658 ]\n",
      "Duplicate marker name found: [ 3:130958658 ]\n",
      "Duplicate marker name found: [ 3:132307036 ]\n",
      "Duplicate marker name found: [ 3:132307036 ]\n",
      "Duplicate marker name found: [ 3:133252523 ]\n",
      "Duplicate marker name found: [ 3:134116229 ]\n",
      "Duplicate marker name found: [ 3:134116229 ]\n",
      "Duplicate marker name found: [ 3:135742057 ]\n",
      "Duplicate marker name found: [ 3:139368164 ]\n",
      "Duplicate marker name found: [ 3:139790674 ]\n",
      "Duplicate marker name found: [ 3:142715853 ]\n",
      "Duplicate marker name found: [ 3:144299007 ]\n",
      "Duplicate marker name found: [ 3:144349883 ]\n",
      "Duplicate marker name found: [ 3:144428168 ]\n",
      "Duplicate marker name found: [ 3:144428168 ]\n",
      "Duplicate marker name found: [ 3:146892480 ]\n",
      "Duplicate marker name found: [ 3:147232226 ]\n",
      "Duplicate marker name found: [ 3:147232226 ]\n",
      "Duplicate marker name found: [ 3:147275226 ]\n",
      "Duplicate marker name found: [ 3:147538419 ]\n",
      "Duplicate marker name found: [ 3:148170767 ]\n",
      "Duplicate marker name found: [ 3:149145497 ]\n",
      "Duplicate marker name found: [ 3:149265731 ]\n",
      "Duplicate marker name found: [ 3:149420251 ]\n",
      "Duplicate marker name found: [ 3:149776936 ]\n",
      "Duplicate marker name found: [ 3:150250895 ]\n",
      "Duplicate marker name found: [ 3:151648071 ]\n",
      "Duplicate marker name found: [ 3:154381852 ]\n",
      "Duplicate marker name found: [ 3:158886756 ]\n",
      "Duplicate marker name found: [ 3:159429731 ]\n",
      "Duplicate marker name found: [ 3:159729165 ]\n",
      "Duplicate marker name found: [ 3:160400227 ]\n",
      "Duplicate marker name found: [ 3:160400227 ]\n",
      "Duplicate marker name found: [ 3:160400227 ]\n",
      "Duplicate marker name found: [ 3:161024440 ]\n",
      "Duplicate marker name found: [ 3:161682565 ]\n",
      "Duplicate marker name found: [ 3:162945491 ]\n",
      "Duplicate marker name found: [ 3:164625152 ]\n",
      "Duplicate marker name found: [ 3:164625152 ]\n",
      "Duplicate marker name found: [ 3:165119100 ]\n",
      "Duplicate marker name found: [ 3:167525922 ]\n",
      "Duplicate marker name found: [ 3:167755239 ]\n",
      "Duplicate marker name found: [ 3:169899928 ]\n",
      "Duplicate marker name found: [ 3:170030049 ]\n",
      "Duplicate marker name found: [ 3:170030049 ]\n",
      "Duplicate marker name found: [ 3:171042888 ]\n",
      "Duplicate marker name found: [ 3:171491532 ]\n",
      "Duplicate marker name found: [ 3:174589689 ]\n",
      "Duplicate marker name found: [ 3:175380934 ]\n",
      "Duplicate marker name found: [ 3:175480871 ]\n",
      "Duplicate marker name found: [ 3:175887195 ]\n",
      "Duplicate marker name found: [ 3:176937837 ]\n",
      "Duplicate marker name found: [ 3:177286477 ]\n",
      "Duplicate marker name found: [ 3:177537763 ]\n",
      "Duplicate marker name found: [ 3:178807294 ]\n",
      "Duplicate marker name found: [ 3:180840548 ]\n",
      "Duplicate marker name found: [ 3:181355628 ]\n",
      "Duplicate marker name found: [ 3:181355628 ]\n",
      "Duplicate marker name found: [ 3:182476393 ]\n",
      "Duplicate marker name found: [ 3:183308877 ]\n",
      "Duplicate marker name found: [ 3:183943004 ]\n",
      "Duplicate marker name found: [ 3:184548242 ]\n",
      "Duplicate marker name found: [ 3:186975151 ]\n",
      "Duplicate marker name found: [ 3:187485915 ]\n",
      "Duplicate marker name found: [ 3:187706680 ]\n",
      "Duplicate marker name found: [ 3:191984587 ]\n",
      "Duplicate marker name found: [ 3:192726128 ]\n",
      "Duplicate marker name found: [ 3:195903235 ]\n",
      "Duplicate marker name found: [ 3:197225052 ]\n",
      "Duplicate marker name found: [ 3:197225052 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2228228 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.791165\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2228228 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr3.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:24:20 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr4.log ]\n",
      "Analysis started: Thu Jan 12 16:24:21 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr4\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr4\n",
      "\n",
      "2264691 (of 2264691) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr4.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr4.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr4.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 4:245113 ]\n",
      "Duplicate marker name found: [ 4:6425627 ]\n",
      "Duplicate marker name found: [ 4:7094075 ]\n",
      "Duplicate marker name found: [ 4:8305627 ]\n",
      "Duplicate marker name found: [ 4:8395217 ]\n",
      "Duplicate marker name found: [ 4:8395217 ]\n",
      "Duplicate marker name found: [ 4:9561165 ]\n",
      "Duplicate marker name found: [ 4:10805030 ]\n",
      "Duplicate marker name found: [ 4:11800378 ]\n",
      "Duplicate marker name found: [ 4:12777815 ]\n",
      "Duplicate marker name found: [ 4:12777815 ]\n",
      "Duplicate marker name found: [ 4:13129913 ]\n",
      "Duplicate marker name found: [ 4:14075727 ]\n",
      "Duplicate marker name found: [ 4:15464905 ]\n",
      "Duplicate marker name found: [ 4:15773630 ]\n",
      "Duplicate marker name found: [ 4:15966104 ]\n",
      "Duplicate marker name found: [ 4:17178146 ]\n",
      "Duplicate marker name found: [ 4:17447905 ]\n",
      "Duplicate marker name found: [ 4:17860352 ]\n",
      "Duplicate marker name found: [ 4:17860352 ]\n",
      "Duplicate marker name found: [ 4:19835611 ]\n",
      "Duplicate marker name found: [ 4:21229293 ]\n",
      "Duplicate marker name found: [ 4:22289902 ]\n",
      "Duplicate marker name found: [ 4:24114903 ]\n",
      "Duplicate marker name found: [ 4:25691056 ]\n",
      "Duplicate marker name found: [ 4:27806162 ]\n",
      "Duplicate marker name found: [ 4:28382931 ]\n",
      "Duplicate marker name found: [ 4:28599935 ]\n",
      "Duplicate marker name found: [ 4:28924579 ]\n",
      "Duplicate marker name found: [ 4:28924579 ]\n",
      "Duplicate marker name found: [ 4:31410946 ]\n",
      "Duplicate marker name found: [ 4:31410946 ]\n",
      "Duplicate marker name found: [ 4:31970833 ]\n",
      "Duplicate marker name found: [ 4:32360352 ]\n",
      "Duplicate marker name found: [ 4:32360352 ]\n",
      "Duplicate marker name found: [ 4:34084742 ]\n",
      "Duplicate marker name found: [ 4:34316653 ]\n",
      "Duplicate marker name found: [ 4:34316653 ]\n",
      "Duplicate marker name found: [ 4:34931998 ]\n",
      "Duplicate marker name found: [ 4:35627165 ]\n",
      "Duplicate marker name found: [ 4:36344705 ]\n",
      "Duplicate marker name found: [ 4:38780050 ]\n",
      "Duplicate marker name found: [ 4:38780050 ]\n",
      "Duplicate marker name found: [ 4:38949111 ]\n",
      "Duplicate marker name found: [ 4:39418924 ]\n",
      "Duplicate marker name found: [ 4:40398868 ]\n",
      "Duplicate marker name found: [ 4:41040230 ]\n",
      "Duplicate marker name found: [ 4:41252038 ]\n",
      "Duplicate marker name found: [ 4:42197566 ]\n",
      "Duplicate marker name found: [ 4:42717115 ]\n",
      "Duplicate marker name found: [ 4:42717115 ]\n",
      "Duplicate marker name found: [ 4:44783937 ]\n",
      "Duplicate marker name found: [ 4:46315584 ]\n",
      "Duplicate marker name found: [ 4:54685460 ]\n",
      "Duplicate marker name found: [ 4:55716525 ]\n",
      "Duplicate marker name found: [ 4:56557084 ]\n",
      "Duplicate marker name found: [ 4:57494868 ]\n",
      "Duplicate marker name found: [ 4:58347946 ]\n",
      "Duplicate marker name found: [ 4:59069493 ]\n",
      "Duplicate marker name found: [ 4:59069493 ]\n",
      "Duplicate marker name found: [ 4:59382842 ]\n",
      "Duplicate marker name found: [ 4:61300143 ]\n",
      "Duplicate marker name found: [ 4:61300143 ]\n",
      "Duplicate marker name found: [ 4:62066219 ]\n",
      "Duplicate marker name found: [ 4:62072703 ]\n",
      "Duplicate marker name found: [ 4:62674450 ]\n",
      "Duplicate marker name found: [ 4:63639441 ]\n",
      "Duplicate marker name found: [ 4:67901011 ]\n",
      "Duplicate marker name found: [ 4:69781004 ]\n",
      "Duplicate marker name found: [ 4:74116405 ]\n",
      "Duplicate marker name found: [ 4:75124106 ]\n",
      "Duplicate marker name found: [ 4:79389639 ]\n",
      "Duplicate marker name found: [ 4:79389639 ]\n",
      "Duplicate marker name found: [ 4:79514779 ]\n",
      "Duplicate marker name found: [ 4:79514779 ]\n",
      "Duplicate marker name found: [ 4:79514779 ]\n",
      "Duplicate marker name found: [ 4:80295155 ]\n",
      "Duplicate marker name found: [ 4:82012427 ]\n",
      "Duplicate marker name found: [ 4:82247802 ]\n",
      "Duplicate marker name found: [ 4:82247802 ]\n",
      "Duplicate marker name found: [ 4:82247802 ]\n",
      "Duplicate marker name found: [ 4:83001640 ]\n",
      "Duplicate marker name found: [ 4:83001640 ]\n",
      "Duplicate marker name found: [ 4:83001640 ]\n",
      "Duplicate marker name found: [ 4:83975495 ]\n",
      "Duplicate marker name found: [ 4:84713025 ]\n",
      "Duplicate marker name found: [ 4:86056824 ]\n",
      "Duplicate marker name found: [ 4:86118128 ]\n",
      "Duplicate marker name found: [ 4:87437791 ]\n",
      "Duplicate marker name found: [ 4:87745311 ]\n",
      "Duplicate marker name found: [ 4:88129443 ]\n",
      "Duplicate marker name found: [ 4:89151130 ]\n",
      "Duplicate marker name found: [ 4:89151130 ]\n",
      "Duplicate marker name found: [ 4:91359925 ]\n",
      "Duplicate marker name found: [ 4:94282618 ]\n",
      "Duplicate marker name found: [ 4:94752556 ]\n",
      "Duplicate marker name found: [ 4:97221117 ]\n",
      "Duplicate marker name found: [ 4:98174526 ]\n",
      "Duplicate marker name found: [ 4:98174526 ]\n",
      "Duplicate marker name found: [ 4:98236420 ]\n",
      "Duplicate marker name found: [ 4:99714448 ]\n",
      "Duplicate marker name found: [ 4:99817346 ]\n",
      "Duplicate marker name found: [ 4:99858857 ]\n",
      "Duplicate marker name found: [ 4:100569711 ]\n",
      "Duplicate marker name found: [ 4:101853032 ]\n",
      "Duplicate marker name found: [ 4:108006901 ]\n",
      "Duplicate marker name found: [ 4:108006901 ]\n",
      "Duplicate marker name found: [ 4:109676977 ]\n",
      "Duplicate marker name found: [ 4:110962228 ]\n",
      "Duplicate marker name found: [ 4:111097307 ]\n",
      "Duplicate marker name found: [ 4:113361968 ]\n",
      "Duplicate marker name found: [ 4:113361968 ]\n",
      "Duplicate marker name found: [ 4:113361968 ]\n",
      "Duplicate marker name found: [ 4:113656492 ]\n",
      "Duplicate marker name found: [ 4:115359551 ]\n",
      "Duplicate marker name found: [ 4:117685816 ]\n",
      "Duplicate marker name found: [ 4:119300025 ]\n",
      "Duplicate marker name found: [ 4:119300025 ]\n",
      "Duplicate marker name found: [ 4:119302060 ]\n",
      "Duplicate marker name found: [ 4:119302060 ]\n",
      "Duplicate marker name found: [ 4:119842934 ]\n",
      "Duplicate marker name found: [ 4:119842934 ]\n",
      "Duplicate marker name found: [ 4:119842934 ]\n",
      "Duplicate marker name found: [ 4:120186825 ]\n",
      "Duplicate marker name found: [ 4:120186825 ]\n",
      "Duplicate marker name found: [ 4:120186825 ]\n",
      "Duplicate marker name found: [ 4:121130255 ]\n",
      "Duplicate marker name found: [ 4:121518068 ]\n",
      "Duplicate marker name found: [ 4:121518068 ]\n",
      "Duplicate marker name found: [ 4:125338129 ]\n",
      "Duplicate marker name found: [ 4:126071369 ]\n",
      "Duplicate marker name found: [ 4:128800358 ]\n",
      "Duplicate marker name found: [ 4:129375090 ]\n",
      "Duplicate marker name found: [ 4:129836869 ]\n",
      "Duplicate marker name found: [ 4:131201409 ]\n",
      "Duplicate marker name found: [ 4:131201409 ]\n",
      "Duplicate marker name found: [ 4:139919752 ]\n",
      "Duplicate marker name found: [ 4:140348922 ]\n",
      "Duplicate marker name found: [ 4:140410732 ]\n",
      "Duplicate marker name found: [ 4:140991269 ]\n",
      "Duplicate marker name found: [ 4:141066705 ]\n",
      "Duplicate marker name found: [ 4:142106021 ]\n",
      "Duplicate marker name found: [ 4:143540027 ]\n",
      "Duplicate marker name found: [ 4:143758312 ]\n",
      "Duplicate marker name found: [ 4:143956648 ]\n",
      "Duplicate marker name found: [ 4:146486469 ]\n",
      "Duplicate marker name found: [ 4:146995145 ]\n",
      "Duplicate marker name found: [ 4:148144358 ]\n",
      "Duplicate marker name found: [ 4:148144358 ]\n",
      "Duplicate marker name found: [ 4:149803966 ]\n",
      "Duplicate marker name found: [ 4:151310505 ]\n",
      "Duplicate marker name found: [ 4:151994839 ]\n",
      "Duplicate marker name found: [ 4:152165351 ]\n",
      "Duplicate marker name found: [ 4:152815466 ]\n",
      "Duplicate marker name found: [ 4:152815466 ]\n",
      "Duplicate marker name found: [ 4:153255204 ]\n",
      "Duplicate marker name found: [ 4:153616465 ]\n",
      "Duplicate marker name found: [ 4:153950952 ]\n",
      "Duplicate marker name found: [ 4:155136103 ]\n",
      "Duplicate marker name found: [ 4:155258884 ]\n",
      "Duplicate marker name found: [ 4:156953158 ]\n",
      "Duplicate marker name found: [ 4:156953158 ]\n",
      "Duplicate marker name found: [ 4:158765447 ]\n",
      "Duplicate marker name found: [ 4:159750863 ]\n",
      "Duplicate marker name found: [ 4:159984702 ]\n",
      "Duplicate marker name found: [ 4:161792433 ]\n",
      "Duplicate marker name found: [ 4:164501214 ]\n",
      "Duplicate marker name found: [ 4:164966761 ]\n",
      "Duplicate marker name found: [ 4:165341131 ]\n",
      "Duplicate marker name found: [ 4:166588202 ]\n",
      "Duplicate marker name found: [ 4:169221731 ]\n",
      "Duplicate marker name found: [ 4:169221731 ]\n",
      "Duplicate marker name found: [ 4:169221731 ]\n",
      "Duplicate marker name found: [ 4:172641498 ]\n",
      "Duplicate marker name found: [ 4:172641498 ]\n",
      "Duplicate marker name found: [ 4:172641498 ]\n",
      "Duplicate marker name found: [ 4:173917314 ]\n",
      "Duplicate marker name found: [ 4:177440762 ]\n",
      "Duplicate marker name found: [ 4:178765048 ]\n",
      "Duplicate marker name found: [ 4:179807399 ]\n",
      "Duplicate marker name found: [ 4:181899810 ]\n",
      "Duplicate marker name found: [ 4:184307880 ]\n",
      "Duplicate marker name found: [ 4:186049193 ]\n",
      "Duplicate marker name found: [ 4:186049193 ]\n",
      "Duplicate marker name found: [ 4:189392684 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2264691 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.78997\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2264691 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr4.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:25:04 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr5.log ]\n",
      "Analysis started: Thu Jan 12 16:25:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr5\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr5\n",
      "\n",
      "2031572 (of 2031572) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr5.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr5.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr5.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 5:316807 ]\n",
      "Duplicate marker name found: [ 5:4282449 ]\n",
      "Duplicate marker name found: [ 5:6107863 ]\n",
      "Duplicate marker name found: [ 5:6388272 ]\n",
      "Duplicate marker name found: [ 5:8962076 ]\n",
      "Duplicate marker name found: [ 5:9049263 ]\n",
      "Duplicate marker name found: [ 5:9049263 ]\n",
      "Duplicate marker name found: [ 5:9049263 ]\n",
      "Duplicate marker name found: [ 5:10743036 ]\n",
      "Duplicate marker name found: [ 5:13825455 ]\n",
      "Duplicate marker name found: [ 5:14893075 ]\n",
      "Duplicate marker name found: [ 5:14893075 ]\n",
      "Duplicate marker name found: [ 5:14893438 ]\n",
      "Duplicate marker name found: [ 5:15103574 ]\n",
      "Duplicate marker name found: [ 5:15884358 ]\n",
      "Duplicate marker name found: [ 5:16109212 ]\n",
      "Duplicate marker name found: [ 5:16356086 ]\n",
      "Duplicate marker name found: [ 5:16356086 ]\n",
      "Duplicate marker name found: [ 5:17121656 ]\n",
      "Duplicate marker name found: [ 5:18844328 ]\n",
      "Duplicate marker name found: [ 5:20373751 ]\n",
      "Duplicate marker name found: [ 5:20899721 ]\n",
      "Duplicate marker name found: [ 5:21831176 ]\n",
      "Duplicate marker name found: [ 5:23863527 ]\n",
      "Duplicate marker name found: [ 5:24077505 ]\n",
      "Duplicate marker name found: [ 5:26067659 ]\n",
      "Duplicate marker name found: [ 5:27426142 ]\n",
      "Duplicate marker name found: [ 5:28576137 ]\n",
      "Duplicate marker name found: [ 5:28576137 ]\n",
      "Duplicate marker name found: [ 5:28591556 ]\n",
      "Duplicate marker name found: [ 5:29097976 ]\n",
      "Duplicate marker name found: [ 5:32217109 ]\n",
      "Duplicate marker name found: [ 5:32217109 ]\n",
      "Duplicate marker name found: [ 5:35975764 ]\n",
      "Duplicate marker name found: [ 5:37258001 ]\n",
      "Duplicate marker name found: [ 5:40599806 ]\n",
      "Duplicate marker name found: [ 5:40599806 ]\n",
      "Duplicate marker name found: [ 5:42532450 ]\n",
      "Duplicate marker name found: [ 5:42532450 ]\n",
      "Duplicate marker name found: [ 5:43392208 ]\n",
      "Duplicate marker name found: [ 5:44561710 ]\n",
      "Duplicate marker name found: [ 5:44561710 ]\n",
      "Duplicate marker name found: [ 5:44561710 ]\n",
      "Duplicate marker name found: [ 5:45293581 ]\n",
      "Duplicate marker name found: [ 5:46104908 ]\n",
      "Duplicate marker name found: [ 5:51388412 ]\n",
      "Duplicate marker name found: [ 5:52931017 ]\n",
      "Duplicate marker name found: [ 5:53872601 ]\n",
      "Duplicate marker name found: [ 5:56970497 ]\n",
      "Duplicate marker name found: [ 5:58429810 ]\n",
      "Duplicate marker name found: [ 5:58429810 ]\n",
      "Duplicate marker name found: [ 5:61195468 ]\n",
      "Duplicate marker name found: [ 5:65210459 ]\n",
      "Duplicate marker name found: [ 5:65712198 ]\n",
      "Duplicate marker name found: [ 5:65895746 ]\n",
      "Duplicate marker name found: [ 5:66026579 ]\n",
      "Duplicate marker name found: [ 5:66520551 ]\n",
      "Duplicate marker name found: [ 5:66520551 ]\n",
      "Duplicate marker name found: [ 5:66520551 ]\n",
      "Duplicate marker name found: [ 5:68435835 ]\n",
      "Duplicate marker name found: [ 5:68761799 ]\n",
      "Duplicate marker name found: [ 5:70690814 ]\n",
      "Duplicate marker name found: [ 5:71822353 ]\n",
      "Duplicate marker name found: [ 5:72999472 ]\n",
      "Duplicate marker name found: [ 5:74840466 ]\n",
      "Duplicate marker name found: [ 5:75025380 ]\n",
      "Duplicate marker name found: [ 5:76291180 ]\n",
      "Duplicate marker name found: [ 5:76316976 ]\n",
      "Duplicate marker name found: [ 5:78299770 ]\n",
      "Duplicate marker name found: [ 5:79074634 ]\n",
      "Duplicate marker name found: [ 5:79918127 ]\n",
      "Duplicate marker name found: [ 5:81172257 ]\n",
      "Duplicate marker name found: [ 5:81172257 ]\n",
      "Duplicate marker name found: [ 5:81774446 ]\n",
      "Duplicate marker name found: [ 5:83301897 ]\n",
      "Duplicate marker name found: [ 5:83563955 ]\n",
      "Duplicate marker name found: [ 5:85030863 ]\n",
      "Duplicate marker name found: [ 5:89283696 ]\n",
      "Duplicate marker name found: [ 5:89771303 ]\n",
      "Duplicate marker name found: [ 5:90411764 ]\n",
      "Duplicate marker name found: [ 5:93462948 ]\n",
      "Duplicate marker name found: [ 5:94710466 ]\n",
      "Duplicate marker name found: [ 5:95099506 ]\n",
      "Duplicate marker name found: [ 5:95287084 ]\n",
      "Duplicate marker name found: [ 5:96930719 ]\n",
      "Duplicate marker name found: [ 5:98296015 ]\n",
      "Duplicate marker name found: [ 5:98324037 ]\n",
      "Duplicate marker name found: [ 5:99055224 ]\n",
      "Duplicate marker name found: [ 5:99055224 ]\n",
      "Duplicate marker name found: [ 5:103210852 ]\n",
      "Duplicate marker name found: [ 5:105653672 ]\n",
      "Duplicate marker name found: [ 5:105697329 ]\n",
      "Duplicate marker name found: [ 5:106230698 ]\n",
      "Duplicate marker name found: [ 5:106230698 ]\n",
      "Duplicate marker name found: [ 5:107282156 ]\n",
      "Duplicate marker name found: [ 5:107282156 ]\n",
      "Duplicate marker name found: [ 5:111207543 ]\n",
      "Duplicate marker name found: [ 5:116189516 ]\n",
      "Duplicate marker name found: [ 5:117184214 ]\n",
      "Duplicate marker name found: [ 5:117184214 ]\n",
      "Duplicate marker name found: [ 5:117225937 ]\n",
      "Duplicate marker name found: [ 5:118294155 ]\n",
      "Duplicate marker name found: [ 5:120275167 ]\n",
      "Duplicate marker name found: [ 5:122998748 ]\n",
      "Duplicate marker name found: [ 5:122998748 ]\n",
      "Duplicate marker name found: [ 5:124131483 ]\n",
      "Duplicate marker name found: [ 5:124131483 ]\n",
      "Duplicate marker name found: [ 5:124131483 ]\n",
      "Duplicate marker name found: [ 5:124131483 ]\n",
      "Duplicate marker name found: [ 5:124336672 ]\n",
      "Duplicate marker name found: [ 5:124440449 ]\n",
      "Duplicate marker name found: [ 5:125910390 ]\n",
      "Duplicate marker name found: [ 5:126035577 ]\n",
      "Duplicate marker name found: [ 5:126129571 ]\n",
      "Duplicate marker name found: [ 5:126145378 ]\n",
      "Duplicate marker name found: [ 5:126791415 ]\n",
      "Duplicate marker name found: [ 5:127881977 ]\n",
      "Duplicate marker name found: [ 5:127881977 ]\n",
      "Duplicate marker name found: [ 5:130745872 ]\n",
      "Duplicate marker name found: [ 5:132802949 ]\n",
      "Duplicate marker name found: [ 5:134024357 ]\n",
      "Duplicate marker name found: [ 5:134024357 ]\n",
      "Duplicate marker name found: [ 5:136099466 ]\n",
      "Duplicate marker name found: [ 5:136785416 ]\n",
      "Duplicate marker name found: [ 5:137570428 ]\n",
      "Duplicate marker name found: [ 5:137570428 ]\n",
      "Duplicate marker name found: [ 5:138592693 ]\n",
      "Duplicate marker name found: [ 5:138647829 ]\n",
      "Duplicate marker name found: [ 5:139225589 ]\n",
      "Duplicate marker name found: [ 5:139426902 ]\n",
      "Duplicate marker name found: [ 5:139686076 ]\n",
      "Duplicate marker name found: [ 5:140304667 ]\n",
      "Duplicate marker name found: [ 5:141930261 ]\n",
      "Duplicate marker name found: [ 5:141930261 ]\n",
      "Duplicate marker name found: [ 5:145309623 ]\n",
      "Duplicate marker name found: [ 5:145572221 ]\n",
      "Duplicate marker name found: [ 5:147004527 ]\n",
      "Duplicate marker name found: [ 5:147004527 ]\n",
      "Duplicate marker name found: [ 5:148245303 ]\n",
      "Duplicate marker name found: [ 5:148967761 ]\n",
      "Duplicate marker name found: [ 5:149787303 ]\n",
      "Duplicate marker name found: [ 5:149854116 ]\n",
      "Duplicate marker name found: [ 5:149854116 ]\n",
      "Duplicate marker name found: [ 5:150616944 ]\n",
      "Duplicate marker name found: [ 5:151607294 ]\n",
      "Duplicate marker name found: [ 5:153462718 ]\n",
      "Duplicate marker name found: [ 5:153462718 ]\n",
      "Duplicate marker name found: [ 5:153880781 ]\n",
      "Duplicate marker name found: [ 5:156415886 ]\n",
      "Duplicate marker name found: [ 5:156415886 ]\n",
      "Duplicate marker name found: [ 5:156550955 ]\n",
      "Duplicate marker name found: [ 5:156550955 ]\n",
      "Duplicate marker name found: [ 5:156550955 ]\n",
      "Duplicate marker name found: [ 5:156550956 ]\n",
      "Duplicate marker name found: [ 5:157365766 ]\n",
      "Duplicate marker name found: [ 5:160077125 ]\n",
      "Duplicate marker name found: [ 5:163449884 ]\n",
      "Duplicate marker name found: [ 5:163449884 ]\n",
      "Duplicate marker name found: [ 5:164140691 ]\n",
      "Duplicate marker name found: [ 5:164350349 ]\n",
      "Duplicate marker name found: [ 5:164794002 ]\n",
      "Duplicate marker name found: [ 5:166005138 ]\n",
      "Duplicate marker name found: [ 5:166846134 ]\n",
      "Duplicate marker name found: [ 5:166846134 ]\n",
      "Duplicate marker name found: [ 5:167334237 ]\n",
      "Duplicate marker name found: [ 5:168487395 ]\n",
      "Duplicate marker name found: [ 5:168692160 ]\n",
      "Duplicate marker name found: [ 5:169201897 ]\n",
      "Duplicate marker name found: [ 5:170295421 ]\n",
      "Duplicate marker name found: [ 5:170295421 ]\n",
      "Duplicate marker name found: [ 5:170452589 ]\n",
      "Duplicate marker name found: [ 5:171226946 ]\n",
      "Duplicate marker name found: [ 5:171226946 ]\n",
      "Duplicate marker name found: [ 5:171429418 ]\n",
      "Duplicate marker name found: [ 5:171429418 ]\n",
      "Duplicate marker name found: [ 5:171548714 ]\n",
      "Duplicate marker name found: [ 5:171993669 ]\n",
      "Duplicate marker name found: [ 5:171993669 ]\n",
      "Duplicate marker name found: [ 5:174242400 ]\n",
      "Duplicate marker name found: [ 5:174242400 ]\n",
      "Duplicate marker name found: [ 5:176119177 ]\n",
      "Duplicate marker name found: [ 5:177488789 ]\n",
      "Duplicate marker name found: [ 5:177536822 ]\n",
      "Duplicate marker name found: [ 5:177602869 ]\n",
      "Duplicate marker name found: [ 5:177843090 ]\n",
      "Duplicate marker name found: [ 5:178450107 ]\n",
      "Duplicate marker name found: [ 5:178450107 ]\n",
      "Duplicate marker name found: [ 5:178450107 ]\n",
      "Duplicate marker name found: [ 5:178718731 ]\n",
      "Duplicate marker name found: [ 5:180122048 ]\n",
      "Duplicate marker name found: [ 5:180601902 ]\n",
      "Duplicate marker name found: [ 5:180700015 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2031572 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.788272\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2031572 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr5.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:25:52 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr6.log ]\n",
      "Analysis started: Thu Jan 12 16:25:53 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr6\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr6\n",
      "\n",
      "2015265 (of 2015265) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr6.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr6.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr6.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 6:2125509 ]\n",
      "Duplicate marker name found: [ 6:2670602 ]\n",
      "Duplicate marker name found: [ 6:3904251 ]\n",
      "Duplicate marker name found: [ 6:5641516 ]\n",
      "Duplicate marker name found: [ 6:6665281 ]\n",
      "Duplicate marker name found: [ 6:7107906 ]\n",
      "Duplicate marker name found: [ 6:7328165 ]\n",
      "Duplicate marker name found: [ 6:8287132 ]\n",
      "Duplicate marker name found: [ 6:11275798 ]\n",
      "Duplicate marker name found: [ 6:11277175 ]\n",
      "Duplicate marker name found: [ 6:11616106 ]\n",
      "Duplicate marker name found: [ 6:11691294 ]\n",
      "Duplicate marker name found: [ 6:12774612 ]\n",
      "Duplicate marker name found: [ 6:13213320 ]\n",
      "Duplicate marker name found: [ 6:14402762 ]\n",
      "Duplicate marker name found: [ 6:14610230 ]\n",
      "Duplicate marker name found: [ 6:14990215 ]\n",
      "Duplicate marker name found: [ 6:16817423 ]\n",
      "Duplicate marker name found: [ 6:16871829 ]\n",
      "Duplicate marker name found: [ 6:17518133 ]\n",
      "Duplicate marker name found: [ 6:17808617 ]\n",
      "Duplicate marker name found: [ 6:18233985 ]\n",
      "Duplicate marker name found: [ 6:18290860 ]\n",
      "Duplicate marker name found: [ 6:18290860 ]\n",
      "Duplicate marker name found: [ 6:20226561 ]\n",
      "Duplicate marker name found: [ 6:22267429 ]\n",
      "Duplicate marker name found: [ 6:22267429 ]\n",
      "Duplicate marker name found: [ 6:22352034 ]\n",
      "Duplicate marker name found: [ 6:25256588 ]\n",
      "Duplicate marker name found: [ 6:25256588 ]\n",
      "Duplicate marker name found: [ 6:25996157 ]\n",
      "Duplicate marker name found: [ 6:28939819 ]\n",
      "Duplicate marker name found: [ 6:34138530 ]\n",
      "Duplicate marker name found: [ 6:34881396 ]\n",
      "Duplicate marker name found: [ 6:34881396 ]\n",
      "Duplicate marker name found: [ 6:36027440 ]\n",
      "Duplicate marker name found: [ 6:36327228 ]\n",
      "Duplicate marker name found: [ 6:36814598 ]\n",
      "Duplicate marker name found: [ 6:36814598 ]\n",
      "Duplicate marker name found: [ 6:38617125 ]\n",
      "Duplicate marker name found: [ 6:39026418 ]\n",
      "Duplicate marker name found: [ 6:39808005 ]\n",
      "Duplicate marker name found: [ 6:41262687 ]\n",
      "Duplicate marker name found: [ 6:43257642 ]\n",
      "Duplicate marker name found: [ 6:43442320 ]\n",
      "Duplicate marker name found: [ 6:43442320 ]\n",
      "Duplicate marker name found: [ 6:43788089 ]\n",
      "Duplicate marker name found: [ 6:43788089 ]\n",
      "Duplicate marker name found: [ 6:43788089 ]\n",
      "Duplicate marker name found: [ 6:43788089 ]\n",
      "Duplicate marker name found: [ 6:44744183 ]\n",
      "Duplicate marker name found: [ 6:44884709 ]\n",
      "Duplicate marker name found: [ 6:44884709 ]\n",
      "Duplicate marker name found: [ 6:45461467 ]\n",
      "Duplicate marker name found: [ 6:47170498 ]\n",
      "Duplicate marker name found: [ 6:47170498 ]\n",
      "Duplicate marker name found: [ 6:47970490 ]\n",
      "Duplicate marker name found: [ 6:47970490 ]\n",
      "Duplicate marker name found: [ 6:48447029 ]\n",
      "Duplicate marker name found: [ 6:49092756 ]\n",
      "Duplicate marker name found: [ 6:49092756 ]\n",
      "Duplicate marker name found: [ 6:50362806 ]\n",
      "Duplicate marker name found: [ 6:51871169 ]\n",
      "Duplicate marker name found: [ 6:53326520 ]\n",
      "Duplicate marker name found: [ 6:54286667 ]\n",
      "Duplicate marker name found: [ 6:54286667 ]\n",
      "Duplicate marker name found: [ 6:54286667 ]\n",
      "Duplicate marker name found: [ 6:54999009 ]\n",
      "Duplicate marker name found: [ 6:55706398 ]\n",
      "Duplicate marker name found: [ 6:55890232 ]\n",
      "Duplicate marker name found: [ 6:55890232 ]\n",
      "Duplicate marker name found: [ 6:55890232 ]\n",
      "Duplicate marker name found: [ 6:56285153 ]\n",
      "Duplicate marker name found: [ 6:62405127 ]\n",
      "Duplicate marker name found: [ 6:62433061 ]\n",
      "Duplicate marker name found: [ 6:64337605 ]\n",
      "Duplicate marker name found: [ 6:64988610 ]\n",
      "Duplicate marker name found: [ 6:67981961 ]\n",
      "Duplicate marker name found: [ 6:72108211 ]\n",
      "Duplicate marker name found: [ 6:72174978 ]\n",
      "Duplicate marker name found: [ 6:72174978 ]\n",
      "Duplicate marker name found: [ 6:72174978 ]\n",
      "Duplicate marker name found: [ 6:74042126 ]\n",
      "Duplicate marker name found: [ 6:74315353 ]\n",
      "Duplicate marker name found: [ 6:76679448 ]\n",
      "Duplicate marker name found: [ 6:77464685 ]\n",
      "Duplicate marker name found: [ 6:79734523 ]\n",
      "Duplicate marker name found: [ 6:81268964 ]\n",
      "Duplicate marker name found: [ 6:81981502 ]\n",
      "Duplicate marker name found: [ 6:82200372 ]\n",
      "Duplicate marker name found: [ 6:82200372 ]\n",
      "Duplicate marker name found: [ 6:82200372 ]\n",
      "Duplicate marker name found: [ 6:83275229 ]\n",
      "Duplicate marker name found: [ 6:84489130 ]\n",
      "Duplicate marker name found: [ 6:86513420 ]\n",
      "Duplicate marker name found: [ 6:88962789 ]\n",
      "Duplicate marker name found: [ 6:91709132 ]\n",
      "Duplicate marker name found: [ 6:95541653 ]\n",
      "Duplicate marker name found: [ 6:95541653 ]\n",
      "Duplicate marker name found: [ 6:96580801 ]\n",
      "Duplicate marker name found: [ 6:101593872 ]\n",
      "Duplicate marker name found: [ 6:101622290 ]\n",
      "Duplicate marker name found: [ 6:102577256 ]\n",
      "Duplicate marker name found: [ 6:102577256 ]\n",
      "Duplicate marker name found: [ 6:103904041 ]\n",
      "Duplicate marker name found: [ 6:104262823 ]\n",
      "Duplicate marker name found: [ 6:104262823 ]\n",
      "Duplicate marker name found: [ 6:106396234 ]\n",
      "Duplicate marker name found: [ 6:107206000 ]\n",
      "Duplicate marker name found: [ 6:107234538 ]\n",
      "Duplicate marker name found: [ 6:107234538 ]\n",
      "Duplicate marker name found: [ 6:107348342 ]\n",
      "Duplicate marker name found: [ 6:108100384 ]\n",
      "Duplicate marker name found: [ 6:108465620 ]\n",
      "Duplicate marker name found: [ 6:108465620 ]\n",
      "Duplicate marker name found: [ 6:110454166 ]\n",
      "Duplicate marker name found: [ 6:110454166 ]\n",
      "Duplicate marker name found: [ 6:112386766 ]\n",
      "Duplicate marker name found: [ 6:112638757 ]\n",
      "Duplicate marker name found: [ 6:114663584 ]\n",
      "Duplicate marker name found: [ 6:114964009 ]\n",
      "Duplicate marker name found: [ 6:119679689 ]\n",
      "Duplicate marker name found: [ 6:120838752 ]\n",
      "Duplicate marker name found: [ 6:122947218 ]\n",
      "Duplicate marker name found: [ 6:122947218 ]\n",
      "Duplicate marker name found: [ 6:122947218 ]\n",
      "Duplicate marker name found: [ 6:123775035 ]\n",
      "Duplicate marker name found: [ 6:123775035 ]\n",
      "Duplicate marker name found: [ 6:123775035 ]\n",
      "Duplicate marker name found: [ 6:124351186 ]\n",
      "Duplicate marker name found: [ 6:124386358 ]\n",
      "Duplicate marker name found: [ 6:124386358 ]\n",
      "Duplicate marker name found: [ 6:125545616 ]\n",
      "Duplicate marker name found: [ 6:125545616 ]\n",
      "Duplicate marker name found: [ 6:125545616 ]\n",
      "Duplicate marker name found: [ 6:127969323 ]\n",
      "Duplicate marker name found: [ 6:128253544 ]\n",
      "Duplicate marker name found: [ 6:128253544 ]\n",
      "Duplicate marker name found: [ 6:130885421 ]\n",
      "Duplicate marker name found: [ 6:131686195 ]\n",
      "Duplicate marker name found: [ 6:131774945 ]\n",
      "Duplicate marker name found: [ 6:132145582 ]\n",
      "Duplicate marker name found: [ 6:134443456 ]\n",
      "Duplicate marker name found: [ 6:134945519 ]\n",
      "Duplicate marker name found: [ 6:135217184 ]\n",
      "Duplicate marker name found: [ 6:135217184 ]\n",
      "Duplicate marker name found: [ 6:135613247 ]\n",
      "Duplicate marker name found: [ 6:135613247 ]\n",
      "Duplicate marker name found: [ 6:138887536 ]\n",
      "Duplicate marker name found: [ 6:139096073 ]\n",
      "Duplicate marker name found: [ 6:139125719 ]\n",
      "Duplicate marker name found: [ 6:139125719 ]\n",
      "Duplicate marker name found: [ 6:139125719 ]\n",
      "Duplicate marker name found: [ 6:139575452 ]\n",
      "Duplicate marker name found: [ 6:139575452 ]\n",
      "Duplicate marker name found: [ 6:139575452 ]\n",
      "Duplicate marker name found: [ 6:139923398 ]\n",
      "Duplicate marker name found: [ 6:139923398 ]\n",
      "Duplicate marker name found: [ 6:139923398 ]\n",
      "Duplicate marker name found: [ 6:140697513 ]\n",
      "Duplicate marker name found: [ 6:141426558 ]\n",
      "Duplicate marker name found: [ 6:143859230 ]\n",
      "Duplicate marker name found: [ 6:144387381 ]\n",
      "Duplicate marker name found: [ 6:144490800 ]\n",
      "Duplicate marker name found: [ 6:144575368 ]\n",
      "Duplicate marker name found: [ 6:144674980 ]\n",
      "Duplicate marker name found: [ 6:144894834 ]\n",
      "Duplicate marker name found: [ 6:144894834 ]\n",
      "Duplicate marker name found: [ 6:146195345 ]\n",
      "Duplicate marker name found: [ 6:146195345 ]\n",
      "Duplicate marker name found: [ 6:146195345 ]\n",
      "Duplicate marker name found: [ 6:147317691 ]\n",
      "Duplicate marker name found: [ 6:147317691 ]\n",
      "Duplicate marker name found: [ 6:147317691 ]\n",
      "Duplicate marker name found: [ 6:147317691 ]\n",
      "Duplicate marker name found: [ 6:149863149 ]\n",
      "Duplicate marker name found: [ 6:151397786 ]\n",
      "Duplicate marker name found: [ 6:151520661 ]\n",
      "Duplicate marker name found: [ 6:151520661 ]\n",
      "Duplicate marker name found: [ 6:151520661 ]\n",
      "Duplicate marker name found: [ 6:151799508 ]\n",
      "Duplicate marker name found: [ 6:151799508 ]\n",
      "Duplicate marker name found: [ 6:152434709 ]\n",
      "Duplicate marker name found: [ 6:152434709 ]\n",
      "Duplicate marker name found: [ 6:153108138 ]\n",
      "Duplicate marker name found: [ 6:153979618 ]\n",
      "Duplicate marker name found: [ 6:153979618 ]\n",
      "Duplicate marker name found: [ 6:153979618 ]\n",
      "Duplicate marker name found: [ 6:153979618 ]\n",
      "Duplicate marker name found: [ 6:154925890 ]\n",
      "Duplicate marker name found: [ 6:154977968 ]\n",
      "Duplicate marker name found: [ 6:158314640 ]\n",
      "Duplicate marker name found: [ 6:158314640 ]\n",
      "Duplicate marker name found: [ 6:159272320 ]\n",
      "Duplicate marker name found: [ 6:159889918 ]\n",
      "Duplicate marker name found: [ 6:159931835 ]\n",
      "Duplicate marker name found: [ 6:160192098 ]\n",
      "Duplicate marker name found: [ 6:160192098 ]\n",
      "Duplicate marker name found: [ 6:161993692 ]\n",
      "Duplicate marker name found: [ 6:161993692 ]\n",
      "Duplicate marker name found: [ 6:162023136 ]\n",
      "Duplicate marker name found: [ 6:162023136 ]\n",
      "Duplicate marker name found: [ 6:162023136 ]\n",
      "Duplicate marker name found: [ 6:162910109 ]\n",
      "Duplicate marker name found: [ 6:164507204 ]\n",
      "Duplicate marker name found: [ 6:166209061 ]\n",
      "Duplicate marker name found: [ 6:170102316 ]\n",
      "Duplicate marker name found: [ 6:170161769 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 2015265 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.78744\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 2015265 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr6.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:26:31 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr7.log ]\n",
      "Analysis started: Thu Jan 12 16:26:31 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr7\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr7\n",
      "\n",
      "1875213 (of 1875213) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr7.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr7.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr7.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 7:3569180 ]\n",
      "Duplicate marker name found: [ 7:4188687 ]\n",
      "Duplicate marker name found: [ 7:5179328 ]\n",
      "Duplicate marker name found: [ 7:5715270 ]\n",
      "Duplicate marker name found: [ 7:6119090 ]\n",
      "Duplicate marker name found: [ 7:6335539 ]\n",
      "Duplicate marker name found: [ 7:6335539 ]\n",
      "Duplicate marker name found: [ 7:10373035 ]\n",
      "Duplicate marker name found: [ 7:11975263 ]\n",
      "Duplicate marker name found: [ 7:12960096 ]\n",
      "Duplicate marker name found: [ 7:13359794 ]\n",
      "Duplicate marker name found: [ 7:13359794 ]\n",
      "Duplicate marker name found: [ 7:13359794 ]\n",
      "Duplicate marker name found: [ 7:13607932 ]\n",
      "Duplicate marker name found: [ 7:14825764 ]\n",
      "Duplicate marker name found: [ 7:14825764 ]\n",
      "Duplicate marker name found: [ 7:15806745 ]\n",
      "Duplicate marker name found: [ 7:15806745 ]\n",
      "Duplicate marker name found: [ 7:15918884 ]\n",
      "Duplicate marker name found: [ 7:17106614 ]\n",
      "Duplicate marker name found: [ 7:18234561 ]\n",
      "Duplicate marker name found: [ 7:24263850 ]\n",
      "Duplicate marker name found: [ 7:24263850 ]\n",
      "Duplicate marker name found: [ 7:24805909 ]\n",
      "Duplicate marker name found: [ 7:25133714 ]\n",
      "Duplicate marker name found: [ 7:26158639 ]\n",
      "Duplicate marker name found: [ 7:29934380 ]\n",
      "Duplicate marker name found: [ 7:30253445 ]\n",
      "Duplicate marker name found: [ 7:30253445 ]\n",
      "Duplicate marker name found: [ 7:31192641 ]\n",
      "Duplicate marker name found: [ 7:31209962 ]\n",
      "Duplicate marker name found: [ 7:31489119 ]\n",
      "Duplicate marker name found: [ 7:31966514 ]\n",
      "Duplicate marker name found: [ 7:31978843 ]\n",
      "Duplicate marker name found: [ 7:32148045 ]\n",
      "Duplicate marker name found: [ 7:32484650 ]\n",
      "Duplicate marker name found: [ 7:34480605 ]\n",
      "Duplicate marker name found: [ 7:36971675 ]\n",
      "Duplicate marker name found: [ 7:37318809 ]\n",
      "Duplicate marker name found: [ 7:37345049 ]\n",
      "Duplicate marker name found: [ 7:38602734 ]\n",
      "Duplicate marker name found: [ 7:38602734 ]\n",
      "Duplicate marker name found: [ 7:39534751 ]\n",
      "Duplicate marker name found: [ 7:40531591 ]\n",
      "Duplicate marker name found: [ 7:40531591 ]\n",
      "Duplicate marker name found: [ 7:40531591 ]\n",
      "Duplicate marker name found: [ 7:41519284 ]\n",
      "Duplicate marker name found: [ 7:41621265 ]\n",
      "Duplicate marker name found: [ 7:42171673 ]\n",
      "Duplicate marker name found: [ 7:43345184 ]\n",
      "Duplicate marker name found: [ 7:43485753 ]\n",
      "Duplicate marker name found: [ 7:43485753 ]\n",
      "Duplicate marker name found: [ 7:51982291 ]\n",
      "Duplicate marker name found: [ 7:56324816 ]\n",
      "Duplicate marker name found: [ 7:56828527 ]\n",
      "Duplicate marker name found: [ 7:56828527 ]\n",
      "Duplicate marker name found: [ 7:57297149 ]\n",
      "Duplicate marker name found: [ 7:57391837 ]\n",
      "Duplicate marker name found: [ 7:57391837 ]\n",
      "Duplicate marker name found: [ 7:66231697 ]\n",
      "Duplicate marker name found: [ 7:67552849 ]\n",
      "Duplicate marker name found: [ 7:68137205 ]\n",
      "Duplicate marker name found: [ 7:70937671 ]\n",
      "Duplicate marker name found: [ 7:71700260 ]\n",
      "Duplicate marker name found: [ 7:76865385 ]\n",
      "Duplicate marker name found: [ 7:77151958 ]\n",
      "Duplicate marker name found: [ 7:79100750 ]\n",
      "Duplicate marker name found: [ 7:80586907 ]\n",
      "Duplicate marker name found: [ 7:80586907 ]\n",
      "Duplicate marker name found: [ 7:81201745 ]\n",
      "Duplicate marker name found: [ 7:82378536 ]\n",
      "Duplicate marker name found: [ 7:82378536 ]\n",
      "Duplicate marker name found: [ 7:85977701 ]\n",
      "Duplicate marker name found: [ 7:88796758 ]\n",
      "Duplicate marker name found: [ 7:90065320 ]\n",
      "Duplicate marker name found: [ 7:90294597 ]\n",
      "Duplicate marker name found: [ 7:90771114 ]\n",
      "Duplicate marker name found: [ 7:90771114 ]\n",
      "Duplicate marker name found: [ 7:91112194 ]\n",
      "Duplicate marker name found: [ 7:91779013 ]\n",
      "Duplicate marker name found: [ 7:92531225 ]\n",
      "Duplicate marker name found: [ 7:92531225 ]\n",
      "Duplicate marker name found: [ 7:93605656 ]\n",
      "Duplicate marker name found: [ 7:95659290 ]\n",
      "Duplicate marker name found: [ 7:97653968 ]\n",
      "Duplicate marker name found: [ 7:97653968 ]\n",
      "Duplicate marker name found: [ 7:98931627 ]\n",
      "Duplicate marker name found: [ 7:98951080 ]\n",
      "Duplicate marker name found: [ 7:99031674 ]\n",
      "Duplicate marker name found: [ 7:99339624 ]\n",
      "Duplicate marker name found: [ 7:99511293 ]\n",
      "Duplicate marker name found: [ 7:100129946 ]\n",
      "Duplicate marker name found: [ 7:100323154 ]\n",
      "Duplicate marker name found: [ 7:100372542 ]\n",
      "Duplicate marker name found: [ 7:100809067 ]\n",
      "Duplicate marker name found: [ 7:101255577 ]\n",
      "Duplicate marker name found: [ 7:108564402 ]\n",
      "Duplicate marker name found: [ 7:108833047 ]\n",
      "Duplicate marker name found: [ 7:108833047 ]\n",
      "Duplicate marker name found: [ 7:109192933 ]\n",
      "Duplicate marker name found: [ 7:109192933 ]\n",
      "Duplicate marker name found: [ 7:112161017 ]\n",
      "Duplicate marker name found: [ 7:112161017 ]\n",
      "Duplicate marker name found: [ 7:113955665 ]\n",
      "Duplicate marker name found: [ 7:115746426 ]\n",
      "Duplicate marker name found: [ 7:117252288 ]\n",
      "Duplicate marker name found: [ 7:117515035 ]\n",
      "Duplicate marker name found: [ 7:118246015 ]\n",
      "Duplicate marker name found: [ 7:121221499 ]\n",
      "Duplicate marker name found: [ 7:121514093 ]\n",
      "Duplicate marker name found: [ 7:121691864 ]\n",
      "Duplicate marker name found: [ 7:123229355 ]\n",
      "Duplicate marker name found: [ 7:124450773 ]\n",
      "Duplicate marker name found: [ 7:126194471 ]\n",
      "Duplicate marker name found: [ 7:127765815 ]\n",
      "Duplicate marker name found: [ 7:127765815 ]\n",
      "Duplicate marker name found: [ 7:127825806 ]\n",
      "Duplicate marker name found: [ 7:129190050 ]\n",
      "Duplicate marker name found: [ 7:129194579 ]\n",
      "Duplicate marker name found: [ 7:129642308 ]\n",
      "Duplicate marker name found: [ 7:130750283 ]\n",
      "Duplicate marker name found: [ 7:130835244 ]\n",
      "Duplicate marker name found: [ 7:131857517 ]\n",
      "Duplicate marker name found: [ 7:131996908 ]\n",
      "Duplicate marker name found: [ 7:131996908 ]\n",
      "Duplicate marker name found: [ 7:131996908 ]\n",
      "Duplicate marker name found: [ 7:132093207 ]\n",
      "Duplicate marker name found: [ 7:132093207 ]\n",
      "Duplicate marker name found: [ 7:132966043 ]\n",
      "Duplicate marker name found: [ 7:133458217 ]\n",
      "Duplicate marker name found: [ 7:133504257 ]\n",
      "Duplicate marker name found: [ 7:137431299 ]\n",
      "Duplicate marker name found: [ 7:137958268 ]\n",
      "Duplicate marker name found: [ 7:138235063 ]\n",
      "Duplicate marker name found: [ 7:138365316 ]\n",
      "Duplicate marker name found: [ 7:138365316 ]\n",
      "Duplicate marker name found: [ 7:138933756 ]\n",
      "Duplicate marker name found: [ 7:138933756 ]\n",
      "Duplicate marker name found: [ 7:140125085 ]\n",
      "Duplicate marker name found: [ 7:140148518 ]\n",
      "Duplicate marker name found: [ 7:140370027 ]\n",
      "Duplicate marker name found: [ 7:140395029 ]\n",
      "Duplicate marker name found: [ 7:140395029 ]\n",
      "Duplicate marker name found: [ 7:142562051 ]\n",
      "Duplicate marker name found: [ 7:142562051 ]\n",
      "Duplicate marker name found: [ 7:143059858 ]\n",
      "Duplicate marker name found: [ 7:146093800 ]\n",
      "Duplicate marker name found: [ 7:146227057 ]\n",
      "Duplicate marker name found: [ 7:146227057 ]\n",
      "Duplicate marker name found: [ 7:146809920 ]\n",
      "Duplicate marker name found: [ 7:147782786 ]\n",
      "Duplicate marker name found: [ 7:151315539 ]\n",
      "Duplicate marker name found: [ 7:151668180 ]\n",
      "Duplicate marker name found: [ 7:151744772 ]\n",
      "Duplicate marker name found: [ 7:152416418 ]\n",
      "Duplicate marker name found: [ 7:152629852 ]\n",
      "Duplicate marker name found: [ 7:152629852 ]\n",
      "Duplicate marker name found: [ 7:153895391 ]\n",
      "Duplicate marker name found: [ 7:153939466 ]\n",
      "Duplicate marker name found: [ 7:153960764 ]\n",
      "Duplicate marker name found: [ 7:154176035 ]\n",
      "Duplicate marker name found: [ 7:155638800 ]\n",
      "Duplicate marker name found: [ 7:155638800 ]\n",
      "Duplicate marker name found: [ 7:155989126 ]\n",
      "Duplicate marker name found: [ 7:155989126 ]\n",
      "Duplicate marker name found: [ 7:156817609 ]\n",
      "Duplicate marker name found: [ 7:156817609 ]\n",
      "Duplicate marker name found: [ 7:156817609 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1875213 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.776574\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1875213 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr7.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:27:05 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr8.log ]\n",
      "Analysis started: Thu Jan 12 16:27:05 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr8\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr8\n",
      "\n",
      "1757348 (of 1757348) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr8.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr8.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr8.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 8:356754 ]\n",
      "Duplicate marker name found: [ 8:449225 ]\n",
      "Duplicate marker name found: [ 8:688674 ]\n",
      "Duplicate marker name found: [ 8:3221034 ]\n",
      "Duplicate marker name found: [ 8:4219487 ]\n",
      "Duplicate marker name found: [ 8:4671718 ]\n",
      "Duplicate marker name found: [ 8:5067030 ]\n",
      "Duplicate marker name found: [ 8:5281069 ]\n",
      "Duplicate marker name found: [ 8:6083444 ]\n",
      "Duplicate marker name found: [ 8:8881917 ]\n",
      "Duplicate marker name found: [ 8:9788919 ]\n",
      "Duplicate marker name found: [ 8:13482577 ]\n",
      "Duplicate marker name found: [ 8:14753946 ]\n",
      "Duplicate marker name found: [ 8:14753946 ]\n",
      "Duplicate marker name found: [ 8:15196047 ]\n",
      "Duplicate marker name found: [ 8:15196047 ]\n",
      "Duplicate marker name found: [ 8:18739717 ]\n",
      "Duplicate marker name found: [ 8:19624009 ]\n",
      "Duplicate marker name found: [ 8:21463390 ]\n",
      "Duplicate marker name found: [ 8:22774158 ]\n",
      "Duplicate marker name found: [ 8:23969019 ]\n",
      "Duplicate marker name found: [ 8:23969019 ]\n",
      "Duplicate marker name found: [ 8:23969019 ]\n",
      "Duplicate marker name found: [ 8:23969019 ]\n",
      "Duplicate marker name found: [ 8:25011257 ]\n",
      "Duplicate marker name found: [ 8:25193005 ]\n",
      "Duplicate marker name found: [ 8:26019753 ]\n",
      "Duplicate marker name found: [ 8:26063335 ]\n",
      "Duplicate marker name found: [ 8:26063335 ]\n",
      "Duplicate marker name found: [ 8:27044315 ]\n",
      "Duplicate marker name found: [ 8:28514400 ]\n",
      "Duplicate marker name found: [ 8:28559134 ]\n",
      "Duplicate marker name found: [ 8:29325914 ]\n",
      "Duplicate marker name found: [ 8:29325914 ]\n",
      "Duplicate marker name found: [ 8:29325914 ]\n",
      "Duplicate marker name found: [ 8:29702540 ]\n",
      "Duplicate marker name found: [ 8:29958365 ]\n",
      "Duplicate marker name found: [ 8:29958365 ]\n",
      "Duplicate marker name found: [ 8:30043618 ]\n",
      "Duplicate marker name found: [ 8:30063476 ]\n",
      "Duplicate marker name found: [ 8:30166755 ]\n",
      "Duplicate marker name found: [ 8:30994352 ]\n",
      "Duplicate marker name found: [ 8:32150623 ]\n",
      "Duplicate marker name found: [ 8:32595943 ]\n",
      "Duplicate marker name found: [ 8:32595943 ]\n",
      "Duplicate marker name found: [ 8:34562292 ]\n",
      "Duplicate marker name found: [ 8:35568686 ]\n",
      "Duplicate marker name found: [ 8:37651826 ]\n",
      "Duplicate marker name found: [ 8:37666076 ]\n",
      "Duplicate marker name found: [ 8:37679934 ]\n",
      "Duplicate marker name found: [ 8:38156048 ]\n",
      "Duplicate marker name found: [ 8:38246720 ]\n",
      "Duplicate marker name found: [ 8:40038021 ]\n",
      "Duplicate marker name found: [ 8:40038021 ]\n",
      "Duplicate marker name found: [ 8:40818508 ]\n",
      "Duplicate marker name found: [ 8:40957251 ]\n",
      "Duplicate marker name found: [ 8:41072452 ]\n",
      "Duplicate marker name found: [ 8:41847046 ]\n",
      "Duplicate marker name found: [ 8:42074293 ]\n",
      "Duplicate marker name found: [ 8:42119944 ]\n",
      "Duplicate marker name found: [ 8:47643683 ]\n",
      "Duplicate marker name found: [ 8:47661806 ]\n",
      "Duplicate marker name found: [ 8:50133330 ]\n",
      "Duplicate marker name found: [ 8:52463646 ]\n",
      "Duplicate marker name found: [ 8:52494053 ]\n",
      "Duplicate marker name found: [ 8:53110449 ]\n",
      "Duplicate marker name found: [ 8:53727187 ]\n",
      "Duplicate marker name found: [ 8:54412071 ]\n",
      "Duplicate marker name found: [ 8:54412071 ]\n",
      "Duplicate marker name found: [ 8:55237048 ]\n",
      "Duplicate marker name found: [ 8:56121763 ]\n",
      "Duplicate marker name found: [ 8:59377368 ]\n",
      "Duplicate marker name found: [ 8:61273451 ]\n",
      "Duplicate marker name found: [ 8:62826554 ]\n",
      "Duplicate marker name found: [ 8:62968456 ]\n",
      "Duplicate marker name found: [ 8:67072591 ]\n",
      "Duplicate marker name found: [ 8:67072591 ]\n",
      "Duplicate marker name found: [ 8:67072591 ]\n",
      "Duplicate marker name found: [ 8:67661120 ]\n",
      "Duplicate marker name found: [ 8:69428855 ]\n",
      "Duplicate marker name found: [ 8:70418470 ]\n",
      "Duplicate marker name found: [ 8:76546678 ]\n",
      "Duplicate marker name found: [ 8:77615198 ]\n",
      "Duplicate marker name found: [ 8:77615198 ]\n",
      "Duplicate marker name found: [ 8:79286266 ]\n",
      "Duplicate marker name found: [ 8:80648589 ]\n",
      "Duplicate marker name found: [ 8:80890050 ]\n",
      "Duplicate marker name found: [ 8:81415733 ]\n",
      "Duplicate marker name found: [ 8:81885175 ]\n",
      "Duplicate marker name found: [ 8:85292292 ]\n",
      "Duplicate marker name found: [ 8:85292292 ]\n",
      "Duplicate marker name found: [ 8:85544818 ]\n",
      "Duplicate marker name found: [ 8:86535441 ]\n",
      "Duplicate marker name found: [ 8:86866703 ]\n",
      "Duplicate marker name found: [ 8:86866703 ]\n",
      "Duplicate marker name found: [ 8:86970850 ]\n",
      "Duplicate marker name found: [ 8:87094836 ]\n",
      "Duplicate marker name found: [ 8:89978408 ]\n",
      "Duplicate marker name found: [ 8:89978408 ]\n",
      "Duplicate marker name found: [ 8:90944106 ]\n",
      "Duplicate marker name found: [ 8:90944106 ]\n",
      "Duplicate marker name found: [ 8:91483960 ]\n",
      "Duplicate marker name found: [ 8:91483960 ]\n",
      "Duplicate marker name found: [ 8:93050274 ]\n",
      "Duplicate marker name found: [ 8:93050274 ]\n",
      "Duplicate marker name found: [ 8:93486264 ]\n",
      "Duplicate marker name found: [ 8:93749344 ]\n",
      "Duplicate marker name found: [ 8:96419181 ]\n",
      "Duplicate marker name found: [ 8:100228717 ]\n",
      "Duplicate marker name found: [ 8:101372430 ]\n",
      "Duplicate marker name found: [ 8:101777734 ]\n",
      "Duplicate marker name found: [ 8:101777734 ]\n",
      "Duplicate marker name found: [ 8:102663032 ]\n",
      "Duplicate marker name found: [ 8:102980068 ]\n",
      "Duplicate marker name found: [ 8:103462024 ]\n",
      "Duplicate marker name found: [ 8:103462024 ]\n",
      "Duplicate marker name found: [ 8:103462024 ]\n",
      "Duplicate marker name found: [ 8:104130593 ]\n",
      "Duplicate marker name found: [ 8:104130593 ]\n",
      "Duplicate marker name found: [ 8:104612487 ]\n",
      "Duplicate marker name found: [ 8:104680881 ]\n",
      "Duplicate marker name found: [ 8:104825617 ]\n",
      "Duplicate marker name found: [ 8:105561166 ]\n",
      "Duplicate marker name found: [ 8:107502571 ]\n",
      "Duplicate marker name found: [ 8:107618516 ]\n",
      "Duplicate marker name found: [ 8:107618516 ]\n",
      "Duplicate marker name found: [ 8:108018455 ]\n",
      "Duplicate marker name found: [ 8:108813514 ]\n",
      "Duplicate marker name found: [ 8:110537039 ]\n",
      "Duplicate marker name found: [ 8:110818648 ]\n",
      "Duplicate marker name found: [ 8:111550872 ]\n",
      "Duplicate marker name found: [ 8:112462587 ]\n",
      "Duplicate marker name found: [ 8:113148233 ]\n",
      "Duplicate marker name found: [ 8:114880733 ]\n",
      "Duplicate marker name found: [ 8:114880733 ]\n",
      "Duplicate marker name found: [ 8:115183750 ]\n",
      "Duplicate marker name found: [ 8:116365483 ]\n",
      "Duplicate marker name found: [ 8:116471551 ]\n",
      "Duplicate marker name found: [ 8:116471551 ]\n",
      "Duplicate marker name found: [ 8:116471551 ]\n",
      "Duplicate marker name found: [ 8:117345750 ]\n",
      "Duplicate marker name found: [ 8:117933468 ]\n",
      "Duplicate marker name found: [ 8:121392416 ]\n",
      "Duplicate marker name found: [ 8:121606293 ]\n",
      "Duplicate marker name found: [ 8:122309853 ]\n",
      "Duplicate marker name found: [ 8:122309853 ]\n",
      "Duplicate marker name found: [ 8:122534169 ]\n",
      "Duplicate marker name found: [ 8:122534169 ]\n",
      "Duplicate marker name found: [ 8:122534169 ]\n",
      "Duplicate marker name found: [ 8:122538341 ]\n",
      "Duplicate marker name found: [ 8:124393738 ]\n",
      "Duplicate marker name found: [ 8:124393738 ]\n",
      "Duplicate marker name found: [ 8:124393738 ]\n",
      "Duplicate marker name found: [ 8:126274338 ]\n",
      "Duplicate marker name found: [ 8:129155276 ]\n",
      "Duplicate marker name found: [ 8:129155276 ]\n",
      "Duplicate marker name found: [ 8:132641577 ]\n",
      "Duplicate marker name found: [ 8:132641577 ]\n",
      "Duplicate marker name found: [ 8:133369254 ]\n",
      "Duplicate marker name found: [ 8:133369254 ]\n",
      "Duplicate marker name found: [ 8:133703370 ]\n",
      "Duplicate marker name found: [ 8:133999966 ]\n",
      "Duplicate marker name found: [ 8:134078013 ]\n",
      "Duplicate marker name found: [ 8:134226750 ]\n",
      "Duplicate marker name found: [ 8:134226750 ]\n",
      "Duplicate marker name found: [ 8:134226750 ]\n",
      "Duplicate marker name found: [ 8:134819548 ]\n",
      "Duplicate marker name found: [ 8:136647614 ]\n",
      "Duplicate marker name found: [ 8:138581809 ]\n",
      "Duplicate marker name found: [ 8:142467264 ]\n",
      "Duplicate marker name found: [ 8:142467264 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1757348 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.789505\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1757348 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr8.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:27:36 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr9.log ]\n",
      "Analysis started: Thu Jan 12 16:27:36 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr9\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr9\n",
      "\n",
      "1427317 (of 1427317) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr9.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr9.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr9.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 9:1905444 ]\n",
      "Duplicate marker name found: [ 9:2032717 ]\n",
      "Duplicate marker name found: [ 9:2730448 ]\n",
      "Duplicate marker name found: [ 9:3232038 ]\n",
      "Duplicate marker name found: [ 9:4430028 ]\n",
      "Duplicate marker name found: [ 9:9787772 ]\n",
      "Duplicate marker name found: [ 9:9787772 ]\n",
      "Duplicate marker name found: [ 9:10158003 ]\n",
      "Duplicate marker name found: [ 9:11149313 ]\n",
      "Duplicate marker name found: [ 9:13453273 ]\n",
      "Duplicate marker name found: [ 9:15248600 ]\n",
      "Duplicate marker name found: [ 9:16987064 ]\n",
      "Duplicate marker name found: [ 9:18465734 ]\n",
      "Duplicate marker name found: [ 9:20256919 ]\n",
      "Duplicate marker name found: [ 9:21337315 ]\n",
      "Duplicate marker name found: [ 9:23001647 ]\n",
      "Duplicate marker name found: [ 9:23001647 ]\n",
      "Duplicate marker name found: [ 9:23001647 ]\n",
      "Duplicate marker name found: [ 9:24301221 ]\n",
      "Duplicate marker name found: [ 9:24342116 ]\n",
      "Duplicate marker name found: [ 9:24342116 ]\n",
      "Duplicate marker name found: [ 9:25862981 ]\n",
      "Duplicate marker name found: [ 9:26372394 ]\n",
      "Duplicate marker name found: [ 9:26871912 ]\n",
      "Duplicate marker name found: [ 9:28531046 ]\n",
      "Duplicate marker name found: [ 9:28531046 ]\n",
      "Duplicate marker name found: [ 9:28588596 ]\n",
      "Duplicate marker name found: [ 9:28685975 ]\n",
      "Duplicate marker name found: [ 9:30873512 ]\n",
      "Duplicate marker name found: [ 9:32018354 ]\n",
      "Duplicate marker name found: [ 9:35794831 ]\n",
      "Duplicate marker name found: [ 9:36028127 ]\n",
      "Duplicate marker name found: [ 9:36318972 ]\n",
      "Duplicate marker name found: [ 9:36563082 ]\n",
      "Duplicate marker name found: [ 9:38465702 ]\n",
      "Duplicate marker name found: [ 9:72215722 ]\n",
      "Duplicate marker name found: [ 9:72430893 ]\n",
      "Duplicate marker name found: [ 9:72430893 ]\n",
      "Duplicate marker name found: [ 9:72430893 ]\n",
      "Duplicate marker name found: [ 9:73187030 ]\n",
      "Duplicate marker name found: [ 9:73187030 ]\n",
      "Duplicate marker name found: [ 9:74206634 ]\n",
      "Duplicate marker name found: [ 9:75039509 ]\n",
      "Duplicate marker name found: [ 9:75463775 ]\n",
      "Duplicate marker name found: [ 9:76481989 ]\n",
      "Duplicate marker name found: [ 9:76481989 ]\n",
      "Duplicate marker name found: [ 9:78523731 ]\n",
      "Duplicate marker name found: [ 9:78690140 ]\n",
      "Duplicate marker name found: [ 9:79659402 ]\n",
      "Duplicate marker name found: [ 9:79662631 ]\n",
      "Duplicate marker name found: [ 9:79662631 ]\n",
      "Duplicate marker name found: [ 9:80114766 ]\n",
      "Duplicate marker name found: [ 9:83622023 ]\n",
      "Duplicate marker name found: [ 9:84315577 ]\n",
      "Duplicate marker name found: [ 9:84315577 ]\n",
      "Duplicate marker name found: [ 9:86910501 ]\n",
      "Duplicate marker name found: [ 9:86931917 ]\n",
      "Duplicate marker name found: [ 9:86931917 ]\n",
      "Duplicate marker name found: [ 9:87754143 ]\n",
      "Duplicate marker name found: [ 9:87754143 ]\n",
      "Duplicate marker name found: [ 9:87976860 ]\n",
      "Duplicate marker name found: [ 9:88830752 ]\n",
      "Duplicate marker name found: [ 9:88830752 ]\n",
      "Duplicate marker name found: [ 9:89582475 ]\n",
      "Duplicate marker name found: [ 9:90412036 ]\n",
      "Duplicate marker name found: [ 9:93242630 ]\n",
      "Duplicate marker name found: [ 9:93637522 ]\n",
      "Duplicate marker name found: [ 9:93817879 ]\n",
      "Duplicate marker name found: [ 9:93817879 ]\n",
      "Duplicate marker name found: [ 9:97455804 ]\n",
      "Duplicate marker name found: [ 9:97455804 ]\n",
      "Duplicate marker name found: [ 9:98522983 ]\n",
      "Duplicate marker name found: [ 9:99202640 ]\n",
      "Duplicate marker name found: [ 9:99202640 ]\n",
      "Duplicate marker name found: [ 9:99202640 ]\n",
      "Duplicate marker name found: [ 9:99202640 ]\n",
      "Duplicate marker name found: [ 9:99202640 ]\n",
      "Duplicate marker name found: [ 9:102645940 ]\n",
      "Duplicate marker name found: [ 9:102645940 ]\n",
      "Duplicate marker name found: [ 9:102915287 ]\n",
      "Duplicate marker name found: [ 9:105967597 ]\n",
      "Duplicate marker name found: [ 9:106085015 ]\n",
      "Duplicate marker name found: [ 9:106085015 ]\n",
      "Duplicate marker name found: [ 9:106085015 ]\n",
      "Duplicate marker name found: [ 9:107974474 ]\n",
      "Duplicate marker name found: [ 9:109786999 ]\n",
      "Duplicate marker name found: [ 9:109786999 ]\n",
      "Duplicate marker name found: [ 9:110288168 ]\n",
      "Duplicate marker name found: [ 9:110288168 ]\n",
      "Duplicate marker name found: [ 9:112858706 ]\n",
      "Duplicate marker name found: [ 9:112919670 ]\n",
      "Duplicate marker name found: [ 9:112919670 ]\n",
      "Duplicate marker name found: [ 9:112919670 ]\n",
      "Duplicate marker name found: [ 9:113338507 ]\n",
      "Duplicate marker name found: [ 9:114668703 ]\n",
      "Duplicate marker name found: [ 9:115105009 ]\n",
      "Duplicate marker name found: [ 9:116428427 ]\n",
      "Duplicate marker name found: [ 9:116428427 ]\n",
      "Duplicate marker name found: [ 9:117371785 ]\n",
      "Duplicate marker name found: [ 9:118865862 ]\n",
      "Duplicate marker name found: [ 9:119418152 ]\n",
      "Duplicate marker name found: [ 9:119595219 ]\n",
      "Duplicate marker name found: [ 9:119933443 ]\n",
      "Duplicate marker name found: [ 9:120719099 ]\n",
      "Duplicate marker name found: [ 9:124437155 ]\n",
      "Duplicate marker name found: [ 9:125601187 ]\n",
      "Duplicate marker name found: [ 9:125601187 ]\n",
      "Duplicate marker name found: [ 9:125743303 ]\n",
      "Duplicate marker name found: [ 9:125811569 ]\n",
      "Duplicate marker name found: [ 9:126078159 ]\n",
      "Duplicate marker name found: [ 9:127880791 ]\n",
      "Duplicate marker name found: [ 9:129329119 ]\n",
      "Duplicate marker name found: [ 9:130768150 ]\n",
      "Duplicate marker name found: [ 9:130806739 ]\n",
      "Duplicate marker name found: [ 9:131385016 ]\n",
      "Duplicate marker name found: [ 9:132173496 ]\n",
      "Duplicate marker name found: [ 9:133907989 ]\n",
      "Duplicate marker name found: [ 9:136140145 ]\n",
      "Duplicate marker name found: [ 9:136140145 ]\n",
      "Duplicate marker name found: [ 9:136140145 ]\n",
      "Duplicate marker name found: [ 9:136366046 ]\n",
      "Duplicate marker name found: [ 9:137102224 ]\n",
      "Duplicate marker name found: [ 9:140492380 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1427317 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.762695\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1427317 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr9.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:28:02 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr10.log ]\n",
      "Analysis started: Thu Jan 12 16:28:02 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr10\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr10\n",
      "\n",
      "1580642 (of 1580642) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr10.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr10.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr10.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 10:1291668 ]\n",
      "Duplicate marker name found: [ 10:3261403 ]\n",
      "Duplicate marker name found: [ 10:4776861 ]\n",
      "Duplicate marker name found: [ 10:4817583 ]\n",
      "Duplicate marker name found: [ 10:5429149 ]\n",
      "Duplicate marker name found: [ 10:6378293 ]\n",
      "Duplicate marker name found: [ 10:7481021 ]\n",
      "Duplicate marker name found: [ 10:7481021 ]\n",
      "Duplicate marker name found: [ 10:7840159 ]\n",
      "Duplicate marker name found: [ 10:8025807 ]\n",
      "Duplicate marker name found: [ 10:8526820 ]\n",
      "Duplicate marker name found: [ 10:10585721 ]\n",
      "Duplicate marker name found: [ 10:12088822 ]\n",
      "Duplicate marker name found: [ 10:12088822 ]\n",
      "Duplicate marker name found: [ 10:12467263 ]\n",
      "Duplicate marker name found: [ 10:12546224 ]\n",
      "Duplicate marker name found: [ 10:12546224 ]\n",
      "Duplicate marker name found: [ 10:12546224 ]\n",
      "Duplicate marker name found: [ 10:14618528 ]\n",
      "Duplicate marker name found: [ 10:14618528 ]\n",
      "Duplicate marker name found: [ 10:15409198 ]\n",
      "Duplicate marker name found: [ 10:16579070 ]\n",
      "Duplicate marker name found: [ 10:18457426 ]\n",
      "Duplicate marker name found: [ 10:18910966 ]\n",
      "Duplicate marker name found: [ 10:19642797 ]\n",
      "Duplicate marker name found: [ 10:19885668 ]\n",
      "Duplicate marker name found: [ 10:21312625 ]\n",
      "Duplicate marker name found: [ 10:23107885 ]\n",
      "Duplicate marker name found: [ 10:24085269 ]\n",
      "Duplicate marker name found: [ 10:24119750 ]\n",
      "Duplicate marker name found: [ 10:24119750 ]\n",
      "Duplicate marker name found: [ 10:24289998 ]\n",
      "Duplicate marker name found: [ 10:24388498 ]\n",
      "Duplicate marker name found: [ 10:24882519 ]\n",
      "Duplicate marker name found: [ 10:26530898 ]\n",
      "Duplicate marker name found: [ 10:26530898 ]\n",
      "Duplicate marker name found: [ 10:26684527 ]\n",
      "Duplicate marker name found: [ 10:26684527 ]\n",
      "Duplicate marker name found: [ 10:28755228 ]\n",
      "Duplicate marker name found: [ 10:29951707 ]\n",
      "Duplicate marker name found: [ 10:31819525 ]\n",
      "Duplicate marker name found: [ 10:31819525 ]\n",
      "Duplicate marker name found: [ 10:32081517 ]\n",
      "Duplicate marker name found: [ 10:32439281 ]\n",
      "Duplicate marker name found: [ 10:32543688 ]\n",
      "Duplicate marker name found: [ 10:32898957 ]\n",
      "Duplicate marker name found: [ 10:33344161 ]\n",
      "Duplicate marker name found: [ 10:33574761 ]\n",
      "Duplicate marker name found: [ 10:34237840 ]\n",
      "Duplicate marker name found: [ 10:34392723 ]\n",
      "Duplicate marker name found: [ 10:34392723 ]\n",
      "Duplicate marker name found: [ 10:35115427 ]\n",
      "Duplicate marker name found: [ 10:35519924 ]\n",
      "Duplicate marker name found: [ 10:35519924 ]\n",
      "Duplicate marker name found: [ 10:35607154 ]\n",
      "Duplicate marker name found: [ 10:38062916 ]\n",
      "Duplicate marker name found: [ 10:38062916 ]\n",
      "Duplicate marker name found: [ 10:38787027 ]\n",
      "Duplicate marker name found: [ 10:43018850 ]\n",
      "Duplicate marker name found: [ 10:43018850 ]\n",
      "Duplicate marker name found: [ 10:43021113 ]\n",
      "Duplicate marker name found: [ 10:43785522 ]\n",
      "Duplicate marker name found: [ 10:44117993 ]\n",
      "Duplicate marker name found: [ 10:46047288 ]\n",
      "Duplicate marker name found: [ 10:46047288 ]\n",
      "Duplicate marker name found: [ 10:46092918 ]\n",
      "Duplicate marker name found: [ 10:50217647 ]\n",
      "Duplicate marker name found: [ 10:50217647 ]\n",
      "Duplicate marker name found: [ 10:52090053 ]\n",
      "Duplicate marker name found: [ 10:52498935 ]\n",
      "Duplicate marker name found: [ 10:53175670 ]\n",
      "Duplicate marker name found: [ 10:54483097 ]\n",
      "Duplicate marker name found: [ 10:56394922 ]\n",
      "Duplicate marker name found: [ 10:57188326 ]\n",
      "Duplicate marker name found: [ 10:57600901 ]\n",
      "Duplicate marker name found: [ 10:58136753 ]\n",
      "Duplicate marker name found: [ 10:60194743 ]\n",
      "Duplicate marker name found: [ 10:61453686 ]\n",
      "Duplicate marker name found: [ 10:62522856 ]\n",
      "Duplicate marker name found: [ 10:63644470 ]\n",
      "Duplicate marker name found: [ 10:63644470 ]\n",
      "Duplicate marker name found: [ 10:64199931 ]\n",
      "Duplicate marker name found: [ 10:64852735 ]\n",
      "Duplicate marker name found: [ 10:64852735 ]\n",
      "Duplicate marker name found: [ 10:64879766 ]\n",
      "Duplicate marker name found: [ 10:65590618 ]\n",
      "Duplicate marker name found: [ 10:65978111 ]\n",
      "Duplicate marker name found: [ 10:66059371 ]\n",
      "Duplicate marker name found: [ 10:66059371 ]\n",
      "Duplicate marker name found: [ 10:66059371 ]\n",
      "Duplicate marker name found: [ 10:67623528 ]\n",
      "Duplicate marker name found: [ 10:68141314 ]\n",
      "Duplicate marker name found: [ 10:69995766 ]\n",
      "Duplicate marker name found: [ 10:71540717 ]\n",
      "Duplicate marker name found: [ 10:71892520 ]\n",
      "Duplicate marker name found: [ 10:72255767 ]\n",
      "Duplicate marker name found: [ 10:73032634 ]\n",
      "Duplicate marker name found: [ 10:73957664 ]\n",
      "Duplicate marker name found: [ 10:73957664 ]\n",
      "Duplicate marker name found: [ 10:74386440 ]\n",
      "Duplicate marker name found: [ 10:74386440 ]\n",
      "Duplicate marker name found: [ 10:75703860 ]\n",
      "Duplicate marker name found: [ 10:76935273 ]\n",
      "Duplicate marker name found: [ 10:78307252 ]\n",
      "Duplicate marker name found: [ 10:78839127 ]\n",
      "Duplicate marker name found: [ 10:78839127 ]\n",
      "Duplicate marker name found: [ 10:78848186 ]\n",
      "Duplicate marker name found: [ 10:78943820 ]\n",
      "Duplicate marker name found: [ 10:78943820 ]\n",
      "Duplicate marker name found: [ 10:78943820 ]\n",
      "Duplicate marker name found: [ 10:78943820 ]\n",
      "Duplicate marker name found: [ 10:79184264 ]\n",
      "Duplicate marker name found: [ 10:80387179 ]\n",
      "Duplicate marker name found: [ 10:80530855 ]\n",
      "Duplicate marker name found: [ 10:80651006 ]\n",
      "Duplicate marker name found: [ 10:81093876 ]\n",
      "Duplicate marker name found: [ 10:82901122 ]\n",
      "Duplicate marker name found: [ 10:83037957 ]\n",
      "Duplicate marker name found: [ 10:83054135 ]\n",
      "Duplicate marker name found: [ 10:86577705 ]\n",
      "Duplicate marker name found: [ 10:87572487 ]\n",
      "Duplicate marker name found: [ 10:88663358 ]\n",
      "Duplicate marker name found: [ 10:89388930 ]\n",
      "Duplicate marker name found: [ 10:91765840 ]\n",
      "Duplicate marker name found: [ 10:92101093 ]\n",
      "Duplicate marker name found: [ 10:94123310 ]\n",
      "Duplicate marker name found: [ 10:94123310 ]\n",
      "Duplicate marker name found: [ 10:94289983 ]\n",
      "Duplicate marker name found: [ 10:94744756 ]\n",
      "Duplicate marker name found: [ 10:95982088 ]\n",
      "Duplicate marker name found: [ 10:97351586 ]\n",
      "Duplicate marker name found: [ 10:97351586 ]\n",
      "Duplicate marker name found: [ 10:97934495 ]\n",
      "Duplicate marker name found: [ 10:97934495 ]\n",
      "Duplicate marker name found: [ 10:101108647 ]\n",
      "Duplicate marker name found: [ 10:101167540 ]\n",
      "Duplicate marker name found: [ 10:101882666 ]\n",
      "Duplicate marker name found: [ 10:102154403 ]\n",
      "Duplicate marker name found: [ 10:102387183 ]\n",
      "Duplicate marker name found: [ 10:102387183 ]\n",
      "Duplicate marker name found: [ 10:103643743 ]\n",
      "Duplicate marker name found: [ 10:103837724 ]\n",
      "Duplicate marker name found: [ 10:104039887 ]\n",
      "Duplicate marker name found: [ 10:104564717 ]\n",
      "Duplicate marker name found: [ 10:105279008 ]\n",
      "Duplicate marker name found: [ 10:106199462 ]\n",
      "Duplicate marker name found: [ 10:106199462 ]\n",
      "Duplicate marker name found: [ 10:106199462 ]\n",
      "Duplicate marker name found: [ 10:106666885 ]\n",
      "Duplicate marker name found: [ 10:106666885 ]\n",
      "Duplicate marker name found: [ 10:106717312 ]\n",
      "Duplicate marker name found: [ 10:106717312 ]\n",
      "Duplicate marker name found: [ 10:106717312 ]\n",
      "Duplicate marker name found: [ 10:108119666 ]\n",
      "Duplicate marker name found: [ 10:110256736 ]\n",
      "Duplicate marker name found: [ 10:112225282 ]\n",
      "Duplicate marker name found: [ 10:112508423 ]\n",
      "Duplicate marker name found: [ 10:113167131 ]\n",
      "Duplicate marker name found: [ 10:114676626 ]\n",
      "Duplicate marker name found: [ 10:114796424 ]\n",
      "Duplicate marker name found: [ 10:117442179 ]\n",
      "Duplicate marker name found: [ 10:117442179 ]\n",
      "Duplicate marker name found: [ 10:117451233 ]\n",
      "Duplicate marker name found: [ 10:119126920 ]\n",
      "Duplicate marker name found: [ 10:121678110 ]\n",
      "Duplicate marker name found: [ 10:123033831 ]\n",
      "Duplicate marker name found: [ 10:123272061 ]\n",
      "Duplicate marker name found: [ 10:123272066 ]\n",
      "Duplicate marker name found: [ 10:124689715 ]\n",
      "Duplicate marker name found: [ 10:124993351 ]\n",
      "Duplicate marker name found: [ 10:125497787 ]\n",
      "Duplicate marker name found: [ 10:126054959 ]\n",
      "Duplicate marker name found: [ 10:127470993 ]\n",
      "Duplicate marker name found: [ 10:128159146 ]\n",
      "Duplicate marker name found: [ 10:128159146 ]\n",
      "Duplicate marker name found: [ 10:128159146 ]\n",
      "Duplicate marker name found: [ 10:128557609 ]\n",
      "Duplicate marker name found: [ 10:129409100 ]\n",
      "Duplicate marker name found: [ 10:130424083 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1580642 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.7834\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1580642 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr10.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:28:33 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr11.log ]\n",
      "Analysis started: Thu Jan 12 16:28:33 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr11\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr11\n",
      "\n",
      "1555173 (of 1555173) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr11.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr11.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr11.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 11:3773748 ]\n",
      "Duplicate marker name found: [ 11:3871137 ]\n",
      "Duplicate marker name found: [ 11:6442887 ]\n",
      "Duplicate marker name found: [ 11:7264277 ]\n",
      "Duplicate marker name found: [ 11:7576016 ]\n",
      "Duplicate marker name found: [ 11:7742544 ]\n",
      "Duplicate marker name found: [ 11:9488435 ]\n",
      "Duplicate marker name found: [ 11:9732543 ]\n",
      "Duplicate marker name found: [ 11:9944856 ]\n",
      "Duplicate marker name found: [ 11:11608536 ]\n",
      "Duplicate marker name found: [ 11:11732811 ]\n",
      "Duplicate marker name found: [ 11:12663289 ]\n",
      "Duplicate marker name found: [ 11:12991083 ]\n",
      "Duplicate marker name found: [ 11:12991083 ]\n",
      "Duplicate marker name found: [ 11:16879255 ]\n",
      "Duplicate marker name found: [ 11:20028221 ]\n",
      "Duplicate marker name found: [ 11:20028221 ]\n",
      "Duplicate marker name found: [ 11:20322946 ]\n",
      "Duplicate marker name found: [ 11:21746889 ]\n",
      "Duplicate marker name found: [ 11:21746889 ]\n",
      "Duplicate marker name found: [ 11:26335418 ]\n",
      "Duplicate marker name found: [ 11:26340691 ]\n",
      "Duplicate marker name found: [ 11:26722053 ]\n",
      "Duplicate marker name found: [ 11:26722053 ]\n",
      "Duplicate marker name found: [ 11:28688650 ]\n",
      "Duplicate marker name found: [ 11:31520913 ]\n",
      "Duplicate marker name found: [ 11:31520913 ]\n",
      "Duplicate marker name found: [ 11:37749516 ]\n",
      "Duplicate marker name found: [ 11:37749516 ]\n",
      "Duplicate marker name found: [ 11:38052299 ]\n",
      "Duplicate marker name found: [ 11:38810620 ]\n",
      "Duplicate marker name found: [ 11:40650030 ]\n",
      "Duplicate marker name found: [ 11:41265721 ]\n",
      "Duplicate marker name found: [ 11:42829629 ]\n",
      "Duplicate marker name found: [ 11:43339539 ]\n",
      "Duplicate marker name found: [ 11:43394032 ]\n",
      "Duplicate marker name found: [ 11:43394032 ]\n",
      "Duplicate marker name found: [ 11:44093698 ]\n",
      "Duplicate marker name found: [ 11:44496427 ]\n",
      "Duplicate marker name found: [ 11:44496427 ]\n",
      "Duplicate marker name found: [ 11:44580402 ]\n",
      "Duplicate marker name found: [ 11:45158715 ]\n",
      "Duplicate marker name found: [ 11:45158715 ]\n",
      "Duplicate marker name found: [ 11:46331806 ]\n",
      "Duplicate marker name found: [ 11:46629849 ]\n",
      "Duplicate marker name found: [ 11:47167451 ]\n",
      "Duplicate marker name found: [ 11:47503759 ]\n",
      "Duplicate marker name found: [ 11:49219992 ]\n",
      "Duplicate marker name found: [ 11:50127429 ]\n",
      "Duplicate marker name found: [ 11:56744629 ]\n",
      "Duplicate marker name found: [ 11:57169631 ]\n",
      "Duplicate marker name found: [ 11:58100117 ]\n",
      "Duplicate marker name found: [ 11:60778650 ]\n",
      "Duplicate marker name found: [ 11:60778650 ]\n",
      "Duplicate marker name found: [ 11:60778650 ]\n",
      "Duplicate marker name found: [ 11:60778650 ]\n",
      "Duplicate marker name found: [ 11:60778650 ]\n",
      "Duplicate marker name found: [ 11:61398166 ]\n",
      "Duplicate marker name found: [ 11:61436460 ]\n",
      "Duplicate marker name found: [ 11:62855843 ]\n",
      "Duplicate marker name found: [ 11:62855843 ]\n",
      "Duplicate marker name found: [ 11:64930402 ]\n",
      "Duplicate marker name found: [ 11:64930402 ]\n",
      "Duplicate marker name found: [ 11:65933610 ]\n",
      "Duplicate marker name found: [ 11:66584626 ]\n",
      "Duplicate marker name found: [ 11:67868071 ]\n",
      "Duplicate marker name found: [ 11:68449322 ]\n",
      "Duplicate marker name found: [ 11:68604318 ]\n",
      "Duplicate marker name found: [ 11:69030248 ]\n",
      "Duplicate marker name found: [ 11:69030248 ]\n",
      "Duplicate marker name found: [ 11:69419824 ]\n",
      "Duplicate marker name found: [ 11:69419824 ]\n",
      "Duplicate marker name found: [ 11:69419824 ]\n",
      "Duplicate marker name found: [ 11:70295833 ]\n",
      "Duplicate marker name found: [ 11:71700116 ]\n",
      "Duplicate marker name found: [ 11:72972921 ]\n",
      "Duplicate marker name found: [ 11:73862350 ]\n",
      "Duplicate marker name found: [ 11:74961228 ]\n",
      "Duplicate marker name found: [ 11:75834802 ]\n",
      "Duplicate marker name found: [ 11:76615885 ]\n",
      "Duplicate marker name found: [ 11:78123077 ]\n",
      "Duplicate marker name found: [ 11:78321121 ]\n",
      "Duplicate marker name found: [ 11:78409449 ]\n",
      "Duplicate marker name found: [ 11:78896354 ]\n",
      "Duplicate marker name found: [ 11:79282759 ]\n",
      "Duplicate marker name found: [ 11:80200249 ]\n",
      "Duplicate marker name found: [ 11:80671239 ]\n",
      "Duplicate marker name found: [ 11:84152175 ]\n",
      "Duplicate marker name found: [ 11:84152175 ]\n",
      "Duplicate marker name found: [ 11:84507882 ]\n",
      "Duplicate marker name found: [ 11:84507882 ]\n",
      "Duplicate marker name found: [ 11:86860074 ]\n",
      "Duplicate marker name found: [ 11:87382197 ]\n",
      "Duplicate marker name found: [ 11:89403730 ]\n",
      "Duplicate marker name found: [ 11:90380304 ]\n",
      "Duplicate marker name found: [ 11:91387024 ]\n",
      "Duplicate marker name found: [ 11:91387024 ]\n",
      "Duplicate marker name found: [ 11:91387024 ]\n",
      "Duplicate marker name found: [ 11:93113553 ]\n",
      "Duplicate marker name found: [ 11:93336278 ]\n",
      "Duplicate marker name found: [ 11:96532499 ]\n",
      "Duplicate marker name found: [ 11:98233341 ]\n",
      "Duplicate marker name found: [ 11:100971336 ]\n",
      "Duplicate marker name found: [ 11:101076591 ]\n",
      "Duplicate marker name found: [ 11:101098694 ]\n",
      "Duplicate marker name found: [ 11:101274413 ]\n",
      "Duplicate marker name found: [ 11:102001128 ]\n",
      "Duplicate marker name found: [ 11:103821299 ]\n",
      "Duplicate marker name found: [ 11:103821299 ]\n",
      "Duplicate marker name found: [ 11:103821299 ]\n",
      "Duplicate marker name found: [ 11:103880157 ]\n",
      "Duplicate marker name found: [ 11:103880157 ]\n",
      "Duplicate marker name found: [ 11:105294825 ]\n",
      "Duplicate marker name found: [ 11:105294825 ]\n",
      "Duplicate marker name found: [ 11:105294825 ]\n",
      "Duplicate marker name found: [ 11:108348876 ]\n",
      "Duplicate marker name found: [ 11:110180665 ]\n",
      "Duplicate marker name found: [ 11:110519071 ]\n",
      "Duplicate marker name found: [ 11:111434220 ]\n",
      "Duplicate marker name found: [ 11:112827863 ]\n",
      "Duplicate marker name found: [ 11:113577990 ]\n",
      "Duplicate marker name found: [ 11:114823147 ]\n",
      "Duplicate marker name found: [ 11:114823147 ]\n",
      "Duplicate marker name found: [ 11:117806732 ]\n",
      "Duplicate marker name found: [ 11:117996357 ]\n",
      "Duplicate marker name found: [ 11:118831245 ]\n",
      "Duplicate marker name found: [ 11:118831245 ]\n",
      "Duplicate marker name found: [ 11:119442830 ]\n",
      "Duplicate marker name found: [ 11:122253993 ]\n",
      "Duplicate marker name found: [ 11:124299779 ]\n",
      "Duplicate marker name found: [ 11:124299779 ]\n",
      "Duplicate marker name found: [ 11:124299779 ]\n",
      "Duplicate marker name found: [ 11:124606099 ]\n",
      "Duplicate marker name found: [ 11:124606099 ]\n",
      "Duplicate marker name found: [ 11:124928920 ]\n",
      "Duplicate marker name found: [ 11:125259092 ]\n",
      "Duplicate marker name found: [ 11:125264855 ]\n",
      "Duplicate marker name found: [ 11:125373022 ]\n",
      "Duplicate marker name found: [ 11:125373022 ]\n",
      "Duplicate marker name found: [ 11:126121868 ]\n",
      "Duplicate marker name found: [ 11:126203760 ]\n",
      "Duplicate marker name found: [ 11:128102394 ]\n",
      "Duplicate marker name found: [ 11:130149555 ]\n",
      "Duplicate marker name found: [ 11:130149555 ]\n",
      "Duplicate marker name found: [ 11:130194874 ]\n",
      "Duplicate marker name found: [ 11:130891329 ]\n",
      "Duplicate marker name found: [ 11:132529599 ]\n",
      "Duplicate marker name found: [ 11:133535631 ]\n",
      "Duplicate marker name found: [ 11:134047777 ]\n",
      "Duplicate marker name found: [ 11:134047777 ]\n",
      "Duplicate marker name found: [ 11:134884960 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1555173 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.789996\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1555173 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr11.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:29:02 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr12.log ]\n",
      "Analysis started: Thu Jan 12 16:29:03 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr12\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr12\n",
      "\n",
      "1499483 (of 1499483) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr12.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr12.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr12.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 12:527847 ]\n",
      "Duplicate marker name found: [ 12:1370832 ]\n",
      "Duplicate marker name found: [ 12:1402619 ]\n",
      "Duplicate marker name found: [ 12:2007843 ]\n",
      "Duplicate marker name found: [ 12:2007843 ]\n",
      "Duplicate marker name found: [ 12:2007843 ]\n",
      "Duplicate marker name found: [ 12:2226436 ]\n",
      "Duplicate marker name found: [ 12:2959819 ]\n",
      "Duplicate marker name found: [ 12:5752229 ]\n",
      "Duplicate marker name found: [ 12:10326647 ]\n",
      "Duplicate marker name found: [ 12:10948470 ]\n",
      "Duplicate marker name found: [ 12:11425449 ]\n",
      "Duplicate marker name found: [ 12:11710407 ]\n",
      "Duplicate marker name found: [ 12:12150541 ]\n",
      "Duplicate marker name found: [ 12:12150541 ]\n",
      "Duplicate marker name found: [ 12:13153388 ]\n",
      "Duplicate marker name found: [ 12:13927913 ]\n",
      "Duplicate marker name found: [ 12:14021389 ]\n",
      "Duplicate marker name found: [ 12:14021389 ]\n",
      "Duplicate marker name found: [ 12:14700114 ]\n",
      "Duplicate marker name found: [ 12:16884326 ]\n",
      "Duplicate marker name found: [ 12:16884326 ]\n",
      "Duplicate marker name found: [ 12:18569185 ]\n",
      "Duplicate marker name found: [ 12:18700578 ]\n",
      "Duplicate marker name found: [ 12:18721318 ]\n",
      "Duplicate marker name found: [ 12:19316603 ]\n",
      "Duplicate marker name found: [ 12:20202178 ]\n",
      "Duplicate marker name found: [ 12:21745074 ]\n",
      "Duplicate marker name found: [ 12:22564788 ]\n",
      "Duplicate marker name found: [ 12:23495323 ]\n",
      "Duplicate marker name found: [ 12:23639780 ]\n",
      "Duplicate marker name found: [ 12:24819376 ]\n",
      "Duplicate marker name found: [ 12:25847900 ]\n",
      "Duplicate marker name found: [ 12:27004882 ]\n",
      "Duplicate marker name found: [ 12:27250649 ]\n",
      "Duplicate marker name found: [ 12:27250649 ]\n",
      "Duplicate marker name found: [ 12:33422768 ]\n",
      "Duplicate marker name found: [ 12:40500860 ]\n",
      "Duplicate marker name found: [ 12:41021005 ]\n",
      "Duplicate marker name found: [ 12:43691484 ]\n",
      "Duplicate marker name found: [ 12:45958804 ]\n",
      "Duplicate marker name found: [ 12:46123448 ]\n",
      "Duplicate marker name found: [ 12:47400249 ]\n",
      "Duplicate marker name found: [ 12:47400249 ]\n",
      "Duplicate marker name found: [ 12:47400249 ]\n",
      "Duplicate marker name found: [ 12:47569334 ]\n",
      "Duplicate marker name found: [ 12:48260780 ]\n",
      "Duplicate marker name found: [ 12:48616439 ]\n",
      "Duplicate marker name found: [ 12:49640922 ]\n",
      "Duplicate marker name found: [ 12:50480262 ]\n",
      "Duplicate marker name found: [ 12:51203602 ]\n",
      "Duplicate marker name found: [ 12:51570368 ]\n",
      "Duplicate marker name found: [ 12:51864663 ]\n",
      "Duplicate marker name found: [ 12:52097021 ]\n",
      "Duplicate marker name found: [ 12:54082177 ]\n",
      "Duplicate marker name found: [ 12:54270898 ]\n",
      "Duplicate marker name found: [ 12:54395213 ]\n",
      "Duplicate marker name found: [ 12:54776949 ]\n",
      "Duplicate marker name found: [ 12:55918162 ]\n",
      "Duplicate marker name found: [ 12:56455856 ]\n",
      "Duplicate marker name found: [ 12:57318213 ]\n",
      "Duplicate marker name found: [ 12:57509545 ]\n",
      "Duplicate marker name found: [ 12:57665220 ]\n",
      "Duplicate marker name found: [ 12:59605202 ]\n",
      "Duplicate marker name found: [ 12:59605202 ]\n",
      "Duplicate marker name found: [ 12:59956463 ]\n",
      "Duplicate marker name found: [ 12:60100143 ]\n",
      "Duplicate marker name found: [ 12:60226467 ]\n",
      "Duplicate marker name found: [ 12:62220119 ]\n",
      "Duplicate marker name found: [ 12:62220119 ]\n",
      "Duplicate marker name found: [ 12:62682996 ]\n",
      "Duplicate marker name found: [ 12:62682996 ]\n",
      "Duplicate marker name found: [ 12:65038479 ]\n",
      "Duplicate marker name found: [ 12:65927139 ]\n",
      "Duplicate marker name found: [ 12:66804422 ]\n",
      "Duplicate marker name found: [ 12:67215780 ]\n",
      "Duplicate marker name found: [ 12:68809604 ]\n",
      "Duplicate marker name found: [ 12:69398326 ]\n",
      "Duplicate marker name found: [ 12:69840204 ]\n",
      "Duplicate marker name found: [ 12:70064544 ]\n",
      "Duplicate marker name found: [ 12:72405953 ]\n",
      "Duplicate marker name found: [ 12:72405953 ]\n",
      "Duplicate marker name found: [ 12:75222052 ]\n",
      "Duplicate marker name found: [ 12:77188997 ]\n",
      "Duplicate marker name found: [ 12:77188997 ]\n",
      "Duplicate marker name found: [ 12:79174377 ]\n",
      "Duplicate marker name found: [ 12:79325245 ]\n",
      "Duplicate marker name found: [ 12:80650276 ]\n",
      "Duplicate marker name found: [ 12:81087546 ]\n",
      "Duplicate marker name found: [ 12:81221163 ]\n",
      "Duplicate marker name found: [ 12:81621177 ]\n",
      "Duplicate marker name found: [ 12:83572743 ]\n",
      "Duplicate marker name found: [ 12:83572743 ]\n",
      "Duplicate marker name found: [ 12:84938300 ]\n",
      "Duplicate marker name found: [ 12:84938300 ]\n",
      "Duplicate marker name found: [ 12:86432318 ]\n",
      "Duplicate marker name found: [ 12:86432318 ]\n",
      "Duplicate marker name found: [ 12:87029434 ]\n",
      "Duplicate marker name found: [ 12:87029434 ]\n",
      "Duplicate marker name found: [ 12:87029434 ]\n",
      "Duplicate marker name found: [ 12:87724975 ]\n",
      "Duplicate marker name found: [ 12:89208684 ]\n",
      "Duplicate marker name found: [ 12:90888273 ]\n",
      "Duplicate marker name found: [ 12:93367291 ]\n",
      "Duplicate marker name found: [ 12:93650762 ]\n",
      "Duplicate marker name found: [ 12:94488513 ]\n",
      "Duplicate marker name found: [ 12:95870290 ]\n",
      "Duplicate marker name found: [ 12:97952866 ]\n",
      "Duplicate marker name found: [ 12:97952866 ]\n",
      "Duplicate marker name found: [ 12:98932980 ]\n",
      "Duplicate marker name found: [ 12:98932980 ]\n",
      "Duplicate marker name found: [ 12:99272152 ]\n",
      "Duplicate marker name found: [ 12:99272152 ]\n",
      "Duplicate marker name found: [ 12:99865426 ]\n",
      "Duplicate marker name found: [ 12:99865426 ]\n",
      "Duplicate marker name found: [ 12:101060952 ]\n",
      "Duplicate marker name found: [ 12:102041354 ]\n",
      "Duplicate marker name found: [ 12:102798854 ]\n",
      "Duplicate marker name found: [ 12:104586232 ]\n",
      "Duplicate marker name found: [ 12:104885221 ]\n",
      "Duplicate marker name found: [ 12:105242674 ]\n",
      "Duplicate marker name found: [ 12:105242674 ]\n",
      "Duplicate marker name found: [ 12:106553919 ]\n",
      "Duplicate marker name found: [ 12:106675944 ]\n",
      "Duplicate marker name found: [ 12:110557895 ]\n",
      "Duplicate marker name found: [ 12:110957999 ]\n",
      "Duplicate marker name found: [ 12:110957999 ]\n",
      "Duplicate marker name found: [ 12:111173869 ]\n",
      "Duplicate marker name found: [ 12:111301067 ]\n",
      "Duplicate marker name found: [ 12:111301067 ]\n",
      "Duplicate marker name found: [ 12:111579031 ]\n",
      "Duplicate marker name found: [ 12:111824299 ]\n",
      "Duplicate marker name found: [ 12:112038739 ]\n",
      "Duplicate marker name found: [ 12:112971809 ]\n",
      "Duplicate marker name found: [ 12:112971809 ]\n",
      "Duplicate marker name found: [ 12:113518394 ]\n",
      "Duplicate marker name found: [ 12:113566777 ]\n",
      "Duplicate marker name found: [ 12:113566777 ]\n",
      "Duplicate marker name found: [ 12:113657598 ]\n",
      "Duplicate marker name found: [ 12:113657598 ]\n",
      "Duplicate marker name found: [ 12:114887979 ]\n",
      "Duplicate marker name found: [ 12:114887979 ]\n",
      "Duplicate marker name found: [ 12:114887979 ]\n",
      "Duplicate marker name found: [ 12:115315423 ]\n",
      "Duplicate marker name found: [ 12:115998618 ]\n",
      "Duplicate marker name found: [ 12:116322528 ]\n",
      "Duplicate marker name found: [ 12:116875480 ]\n",
      "Duplicate marker name found: [ 12:116875480 ]\n",
      "Duplicate marker name found: [ 12:117639242 ]\n",
      "Duplicate marker name found: [ 12:117639242 ]\n",
      "Duplicate marker name found: [ 12:117762520 ]\n",
      "Duplicate marker name found: [ 12:117762520 ]\n",
      "Duplicate marker name found: [ 12:119642191 ]\n",
      "Duplicate marker name found: [ 12:119642191 ]\n",
      "Duplicate marker name found: [ 12:119820414 ]\n",
      "Duplicate marker name found: [ 12:119896535 ]\n",
      "Duplicate marker name found: [ 12:119896535 ]\n",
      "Duplicate marker name found: [ 12:120005471 ]\n",
      "Duplicate marker name found: [ 12:120232074 ]\n",
      "Duplicate marker name found: [ 12:120232074 ]\n",
      "Duplicate marker name found: [ 12:121585402 ]\n",
      "Duplicate marker name found: [ 12:121791238 ]\n",
      "Duplicate marker name found: [ 12:123435788 ]\n",
      "Duplicate marker name found: [ 12:123517024 ]\n",
      "Duplicate marker name found: [ 12:123517024 ]\n",
      "Duplicate marker name found: [ 12:123985990 ]\n",
      "Duplicate marker name found: [ 12:124056732 ]\n",
      "Duplicate marker name found: [ 12:124056732 ]\n",
      "Duplicate marker name found: [ 12:125192677 ]\n",
      "Duplicate marker name found: [ 12:125274923 ]\n",
      "Duplicate marker name found: [ 12:125274923 ]\n",
      "Duplicate marker name found: [ 12:126125475 ]\n",
      "Duplicate marker name found: [ 12:127097050 ]\n",
      "Duplicate marker name found: [ 12:127097050 ]\n",
      "Duplicate marker name found: [ 12:127487494 ]\n",
      "Duplicate marker name found: [ 12:128283121 ]\n",
      "Duplicate marker name found: [ 12:128283121 ]\n",
      "Duplicate marker name found: [ 12:128283121 ]\n",
      "Duplicate marker name found: [ 12:129616155 ]\n",
      "Duplicate marker name found: [ 12:130056976 ]\n",
      "Duplicate marker name found: [ 12:130751291 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1499483 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.789156\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1499483 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr12.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:29:31 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr13.log ]\n",
      "Analysis started: Thu Jan 12 16:29:32 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr13\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr13\n",
      "\n",
      "1120110 (of 1120110) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr13.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr13.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr13.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 13:19455915 ]\n",
      "Duplicate marker name found: [ 13:22809710 ]\n",
      "Duplicate marker name found: [ 13:25770484 ]\n",
      "Duplicate marker name found: [ 13:26305285 ]\n",
      "Duplicate marker name found: [ 13:26319063 ]\n",
      "Duplicate marker name found: [ 13:26319063 ]\n",
      "Duplicate marker name found: [ 13:26466718 ]\n",
      "Duplicate marker name found: [ 13:26466718 ]\n",
      "Duplicate marker name found: [ 13:26770773 ]\n",
      "Duplicate marker name found: [ 13:30042722 ]\n",
      "Duplicate marker name found: [ 13:30042722 ]\n",
      "Duplicate marker name found: [ 13:32334210 ]\n",
      "Duplicate marker name found: [ 13:32458883 ]\n",
      "Duplicate marker name found: [ 13:33019387 ]\n",
      "Duplicate marker name found: [ 13:34837723 ]\n",
      "Duplicate marker name found: [ 13:34966624 ]\n",
      "Duplicate marker name found: [ 13:35621338 ]\n",
      "Duplicate marker name found: [ 13:39491217 ]\n",
      "Duplicate marker name found: [ 13:39748795 ]\n",
      "Duplicate marker name found: [ 13:41017341 ]\n",
      "Duplicate marker name found: [ 13:41017341 ]\n",
      "Duplicate marker name found: [ 13:41396252 ]\n",
      "Duplicate marker name found: [ 13:41685156 ]\n",
      "Duplicate marker name found: [ 13:41685156 ]\n",
      "Duplicate marker name found: [ 13:41685156 ]\n",
      "Duplicate marker name found: [ 13:42684723 ]\n",
      "Duplicate marker name found: [ 13:42684723 ]\n",
      "Duplicate marker name found: [ 13:43796516 ]\n",
      "Duplicate marker name found: [ 13:43796516 ]\n",
      "Duplicate marker name found: [ 13:44900871 ]\n",
      "Duplicate marker name found: [ 13:46767703 ]\n",
      "Duplicate marker name found: [ 13:46997864 ]\n",
      "Duplicate marker name found: [ 13:46997864 ]\n",
      "Duplicate marker name found: [ 13:48144659 ]\n",
      "Duplicate marker name found: [ 13:48439236 ]\n",
      "Duplicate marker name found: [ 13:49377453 ]\n",
      "Duplicate marker name found: [ 13:49377453 ]\n",
      "Duplicate marker name found: [ 13:52354314 ]\n",
      "Duplicate marker name found: [ 13:56976354 ]\n",
      "Duplicate marker name found: [ 13:57195291 ]\n",
      "Duplicate marker name found: [ 13:58802781 ]\n",
      "Duplicate marker name found: [ 13:63163730 ]\n",
      "Duplicate marker name found: [ 13:64162776 ]\n",
      "Duplicate marker name found: [ 13:64439587 ]\n",
      "Duplicate marker name found: [ 13:66969731 ]\n",
      "Duplicate marker name found: [ 13:73668747 ]\n",
      "Duplicate marker name found: [ 13:73837485 ]\n",
      "Duplicate marker name found: [ 13:74725789 ]\n",
      "Duplicate marker name found: [ 13:75917639 ]\n",
      "Duplicate marker name found: [ 13:79168450 ]\n",
      "Duplicate marker name found: [ 13:80364907 ]\n",
      "Duplicate marker name found: [ 13:81551123 ]\n",
      "Duplicate marker name found: [ 13:82022111 ]\n",
      "Duplicate marker name found: [ 13:82208288 ]\n",
      "Duplicate marker name found: [ 13:82208288 ]\n",
      "Duplicate marker name found: [ 13:82342107 ]\n",
      "Duplicate marker name found: [ 13:84255065 ]\n",
      "Duplicate marker name found: [ 13:90305168 ]\n",
      "Duplicate marker name found: [ 13:91239539 ]\n",
      "Duplicate marker name found: [ 13:91239539 ]\n",
      "Duplicate marker name found: [ 13:92370353 ]\n",
      "Duplicate marker name found: [ 13:92370353 ]\n",
      "Duplicate marker name found: [ 13:92435508 ]\n",
      "Duplicate marker name found: [ 13:93325541 ]\n",
      "Duplicate marker name found: [ 13:93780001 ]\n",
      "Duplicate marker name found: [ 13:94047047 ]\n",
      "Duplicate marker name found: [ 13:94485314 ]\n",
      "Duplicate marker name found: [ 13:96017655 ]\n",
      "Duplicate marker name found: [ 13:96017655 ]\n",
      "Duplicate marker name found: [ 13:96070486 ]\n",
      "Duplicate marker name found: [ 13:97018702 ]\n",
      "Duplicate marker name found: [ 13:98161701 ]\n",
      "Duplicate marker name found: [ 13:98685671 ]\n",
      "Duplicate marker name found: [ 13:100550199 ]\n",
      "Duplicate marker name found: [ 13:101024606 ]\n",
      "Duplicate marker name found: [ 13:101716911 ]\n",
      "Duplicate marker name found: [ 13:102287291 ]\n",
      "Duplicate marker name found: [ 13:102607049 ]\n",
      "Duplicate marker name found: [ 13:104711993 ]\n",
      "Duplicate marker name found: [ 13:106312457 ]\n",
      "Duplicate marker name found: [ 13:108036601 ]\n",
      "Duplicate marker name found: [ 13:108074821 ]\n",
      "Duplicate marker name found: [ 13:109501873 ]\n",
      "Duplicate marker name found: [ 13:109838957 ]\n",
      "Duplicate marker name found: [ 13:110633687 ]\n",
      "Duplicate marker name found: [ 13:110659316 ]\n",
      "Duplicate marker name found: [ 13:112701053 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1120110 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.793978\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1120110 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr13.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:29:53 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr14.log ]\n",
      "Analysis started: Thu Jan 12 16:29:53 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr14\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr14\n",
      "\n",
      "1027611 (of 1027611) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr14.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr14.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr14.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 14:21011508 ]\n",
      "Duplicate marker name found: [ 14:21910049 ]\n",
      "Duplicate marker name found: [ 14:22975756 ]\n",
      "Duplicate marker name found: [ 14:23427123 ]\n",
      "Duplicate marker name found: [ 14:25080058 ]\n",
      "Duplicate marker name found: [ 14:27660423 ]\n",
      "Duplicate marker name found: [ 14:28207656 ]\n",
      "Duplicate marker name found: [ 14:28498087 ]\n",
      "Duplicate marker name found: [ 14:28498087 ]\n",
      "Duplicate marker name found: [ 14:28931914 ]\n",
      "Duplicate marker name found: [ 14:29956253 ]\n",
      "Duplicate marker name found: [ 14:31305173 ]\n",
      "Duplicate marker name found: [ 14:31766689 ]\n",
      "Duplicate marker name found: [ 14:34106109 ]\n",
      "Duplicate marker name found: [ 14:34993863 ]\n",
      "Duplicate marker name found: [ 14:35161083 ]\n",
      "Duplicate marker name found: [ 14:36262801 ]\n",
      "Duplicate marker name found: [ 14:36322196 ]\n",
      "Duplicate marker name found: [ 14:37265553 ]\n",
      "Duplicate marker name found: [ 14:39749936 ]\n",
      "Duplicate marker name found: [ 14:42039521 ]\n",
      "Duplicate marker name found: [ 14:45316181 ]\n",
      "Duplicate marker name found: [ 14:48079779 ]\n",
      "Duplicate marker name found: [ 14:50017836 ]\n",
      "Duplicate marker name found: [ 14:50017836 ]\n",
      "Duplicate marker name found: [ 14:50096952 ]\n",
      "Duplicate marker name found: [ 14:50096952 ]\n",
      "Duplicate marker name found: [ 14:50416370 ]\n",
      "Duplicate marker name found: [ 14:50416370 ]\n",
      "Duplicate marker name found: [ 14:50457175 ]\n",
      "Duplicate marker name found: [ 14:51564233 ]\n",
      "Duplicate marker name found: [ 14:52816495 ]\n",
      "Duplicate marker name found: [ 14:55330566 ]\n",
      "Duplicate marker name found: [ 14:55906816 ]\n",
      "Duplicate marker name found: [ 14:56376811 ]\n",
      "Duplicate marker name found: [ 14:58332707 ]\n",
      "Duplicate marker name found: [ 14:60141770 ]\n",
      "Duplicate marker name found: [ 14:60293783 ]\n",
      "Duplicate marker name found: [ 14:61510412 ]\n",
      "Duplicate marker name found: [ 14:64049890 ]\n",
      "Duplicate marker name found: [ 14:64049890 ]\n",
      "Duplicate marker name found: [ 14:64280778 ]\n",
      "Duplicate marker name found: [ 14:64280778 ]\n",
      "Duplicate marker name found: [ 14:64280778 ]\n",
      "Duplicate marker name found: [ 14:64840108 ]\n",
      "Duplicate marker name found: [ 14:65783528 ]\n",
      "Duplicate marker name found: [ 14:66786969 ]\n",
      "Duplicate marker name found: [ 14:68907470 ]\n",
      "Duplicate marker name found: [ 14:70381582 ]\n",
      "Duplicate marker name found: [ 14:71206219 ]\n",
      "Duplicate marker name found: [ 14:71725605 ]\n",
      "Duplicate marker name found: [ 14:73510349 ]\n",
      "Duplicate marker name found: [ 14:73632757 ]\n",
      "Duplicate marker name found: [ 14:74165229 ]\n",
      "Duplicate marker name found: [ 14:74768603 ]\n",
      "Duplicate marker name found: [ 14:76223305 ]\n",
      "Duplicate marker name found: [ 14:76223305 ]\n",
      "Duplicate marker name found: [ 14:76301410 ]\n",
      "Duplicate marker name found: [ 14:77421559 ]\n",
      "Duplicate marker name found: [ 14:78054102 ]\n",
      "Duplicate marker name found: [ 14:78054102 ]\n",
      "Duplicate marker name found: [ 14:80784282 ]\n",
      "Duplicate marker name found: [ 14:81099572 ]\n",
      "Duplicate marker name found: [ 14:82480618 ]\n",
      "Duplicate marker name found: [ 14:83674440 ]\n",
      "Duplicate marker name found: [ 14:83822070 ]\n",
      "Duplicate marker name found: [ 14:83960289 ]\n",
      "Duplicate marker name found: [ 14:85035998 ]\n",
      "Duplicate marker name found: [ 14:85347322 ]\n",
      "Duplicate marker name found: [ 14:85347322 ]\n",
      "Duplicate marker name found: [ 14:85347322 ]\n",
      "Duplicate marker name found: [ 14:85347322 ]\n",
      "Duplicate marker name found: [ 14:87759947 ]\n",
      "Duplicate marker name found: [ 14:88597340 ]\n",
      "Duplicate marker name found: [ 14:88597340 ]\n",
      "Duplicate marker name found: [ 14:88597340 ]\n",
      "Duplicate marker name found: [ 14:88678453 ]\n",
      "Duplicate marker name found: [ 14:89193382 ]\n",
      "Duplicate marker name found: [ 14:91389772 ]\n",
      "Duplicate marker name found: [ 14:91389772 ]\n",
      "Duplicate marker name found: [ 14:91648309 ]\n",
      "Duplicate marker name found: [ 14:91648309 ]\n",
      "Duplicate marker name found: [ 14:95213330 ]\n",
      "Duplicate marker name found: [ 14:95213330 ]\n",
      "Duplicate marker name found: [ 14:95235760 ]\n",
      "Duplicate marker name found: [ 14:95516255 ]\n",
      "Duplicate marker name found: [ 14:95516255 ]\n",
      "Duplicate marker name found: [ 14:96563174 ]\n",
      "Duplicate marker name found: [ 14:98480754 ]\n",
      "Duplicate marker name found: [ 14:99554671 ]\n",
      "Duplicate marker name found: [ 14:100687414 ]\n",
      "Duplicate marker name found: [ 14:102745621 ]\n",
      "Duplicate marker name found: [ 14:102745621 ]\n",
      "Duplicate marker name found: [ 14:103689007 ]\n",
      "Duplicate marker name found: [ 14:103689007 ]\n",
      "Duplicate marker name found: [ 14:104177190 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1027611 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.777456\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1027611 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr14.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:30:13 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr15.log ]\n",
      "Analysis started: Thu Jan 12 16:30:13 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr15\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr15\n",
      "\n",
      "951210 (of 951210) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr15.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr15.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr15.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 15:22807775 ]\n",
      "Duplicate marker name found: [ 15:22807775 ]\n",
      "Duplicate marker name found: [ 15:26810315 ]\n",
      "Duplicate marker name found: [ 15:26810315 ]\n",
      "Duplicate marker name found: [ 15:26810315 ]\n",
      "Duplicate marker name found: [ 15:27095912 ]\n",
      "Duplicate marker name found: [ 15:27816594 ]\n",
      "Duplicate marker name found: [ 15:29119505 ]\n",
      "Duplicate marker name found: [ 15:29119505 ]\n",
      "Duplicate marker name found: [ 15:29470496 ]\n",
      "Duplicate marker name found: [ 15:29887457 ]\n",
      "Duplicate marker name found: [ 15:30104061 ]\n",
      "Duplicate marker name found: [ 15:30354174 ]\n",
      "Duplicate marker name found: [ 15:35327252 ]\n",
      "Duplicate marker name found: [ 15:37493847 ]\n",
      "Duplicate marker name found: [ 15:37493847 ]\n",
      "Duplicate marker name found: [ 15:37934563 ]\n",
      "Duplicate marker name found: [ 15:37934563 ]\n",
      "Duplicate marker name found: [ 15:37934563 ]\n",
      "Duplicate marker name found: [ 15:40269333 ]\n",
      "Duplicate marker name found: [ 15:40614428 ]\n",
      "Duplicate marker name found: [ 15:44726269 ]\n",
      "Duplicate marker name found: [ 15:45854076 ]\n",
      "Duplicate marker name found: [ 15:45854076 ]\n",
      "Duplicate marker name found: [ 15:45854076 ]\n",
      "Duplicate marker name found: [ 15:45854076 ]\n",
      "Duplicate marker name found: [ 15:48166692 ]\n",
      "Duplicate marker name found: [ 15:48281549 ]\n",
      "Duplicate marker name found: [ 15:48375426 ]\n",
      "Duplicate marker name found: [ 15:48564615 ]\n",
      "Duplicate marker name found: [ 15:50063562 ]\n",
      "Duplicate marker name found: [ 15:50711635 ]\n",
      "Duplicate marker name found: [ 15:50711635 ]\n",
      "Duplicate marker name found: [ 15:50711635 ]\n",
      "Duplicate marker name found: [ 15:50898576 ]\n",
      "Duplicate marker name found: [ 15:51354474 ]\n",
      "Duplicate marker name found: [ 15:51515455 ]\n",
      "Duplicate marker name found: [ 15:51515455 ]\n",
      "Duplicate marker name found: [ 15:51978878 ]\n",
      "Duplicate marker name found: [ 15:52138514 ]\n",
      "Duplicate marker name found: [ 15:53097576 ]\n",
      "Duplicate marker name found: [ 15:54200769 ]\n",
      "Duplicate marker name found: [ 15:55235101 ]\n",
      "Duplicate marker name found: [ 15:58072195 ]\n",
      "Duplicate marker name found: [ 15:58178906 ]\n",
      "Duplicate marker name found: [ 15:58209743 ]\n",
      "Duplicate marker name found: [ 15:58464033 ]\n",
      "Duplicate marker name found: [ 15:58870279 ]\n",
      "Duplicate marker name found: [ 15:59534120 ]\n",
      "Duplicate marker name found: [ 15:59534120 ]\n",
      "Duplicate marker name found: [ 15:60394623 ]\n",
      "Duplicate marker name found: [ 15:60429910 ]\n",
      "Duplicate marker name found: [ 15:61289238 ]\n",
      "Duplicate marker name found: [ 15:61370965 ]\n",
      "Duplicate marker name found: [ 15:61876495 ]\n",
      "Duplicate marker name found: [ 15:62166158 ]\n",
      "Duplicate marker name found: [ 15:62827666 ]\n",
      "Duplicate marker name found: [ 15:62867804 ]\n",
      "Duplicate marker name found: [ 15:63187458 ]\n",
      "Duplicate marker name found: [ 15:64173006 ]\n",
      "Duplicate marker name found: [ 15:64678818 ]\n",
      "Duplicate marker name found: [ 15:67124498 ]\n",
      "Duplicate marker name found: [ 15:67324719 ]\n",
      "Duplicate marker name found: [ 15:67324719 ]\n",
      "Duplicate marker name found: [ 15:68728803 ]\n",
      "Duplicate marker name found: [ 15:68780044 ]\n",
      "Duplicate marker name found: [ 15:69056489 ]\n",
      "Duplicate marker name found: [ 15:69937135 ]\n",
      "Duplicate marker name found: [ 15:70237932 ]\n",
      "Duplicate marker name found: [ 15:70501516 ]\n",
      "Duplicate marker name found: [ 15:70686107 ]\n",
      "Duplicate marker name found: [ 15:70858271 ]\n",
      "Duplicate marker name found: [ 15:71881703 ]\n",
      "Duplicate marker name found: [ 15:72134151 ]\n",
      "Duplicate marker name found: [ 15:74766405 ]\n",
      "Duplicate marker name found: [ 15:74846144 ]\n",
      "Duplicate marker name found: [ 15:75028233 ]\n",
      "Duplicate marker name found: [ 15:75432355 ]\n",
      "Duplicate marker name found: [ 15:77625484 ]\n",
      "Duplicate marker name found: [ 15:79445570 ]\n",
      "Duplicate marker name found: [ 15:80294998 ]\n",
      "Duplicate marker name found: [ 15:81532112 ]\n",
      "Duplicate marker name found: [ 15:81532112 ]\n",
      "Duplicate marker name found: [ 15:82122684 ]\n",
      "Duplicate marker name found: [ 15:83437346 ]\n",
      "Duplicate marker name found: [ 15:83695721 ]\n",
      "Duplicate marker name found: [ 15:85542782 ]\n",
      "Duplicate marker name found: [ 15:86759460 ]\n",
      "Duplicate marker name found: [ 15:87534671 ]\n",
      "Duplicate marker name found: [ 15:89301420 ]\n",
      "Duplicate marker name found: [ 15:89387599 ]\n",
      "Duplicate marker name found: [ 15:90486163 ]\n",
      "Duplicate marker name found: [ 15:92709280 ]\n",
      "Duplicate marker name found: [ 15:93370706 ]\n",
      "Duplicate marker name found: [ 15:94316963 ]\n",
      "Duplicate marker name found: [ 15:95519604 ]\n",
      "Duplicate marker name found: [ 15:99730386 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 951210 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.7749\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 951210 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr15.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:30:31 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr16.log ]\n",
      "Analysis started: Thu Jan 12 16:30:32 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr16\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr16\n",
      "\n",
      "1043129 (of 1043129) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr16.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr16.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr16.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 16:1349862 ]\n",
      "Duplicate marker name found: [ 16:2545003 ]\n",
      "Duplicate marker name found: [ 16:3234623 ]\n",
      "Duplicate marker name found: [ 16:3528064 ]\n",
      "Duplicate marker name found: [ 16:3528064 ]\n",
      "Duplicate marker name found: [ 16:5606025 ]\n",
      "Duplicate marker name found: [ 16:7622845 ]\n",
      "Duplicate marker name found: [ 16:7622845 ]\n",
      "Duplicate marker name found: [ 16:7987463 ]\n",
      "Duplicate marker name found: [ 16:8033546 ]\n",
      "Duplicate marker name found: [ 16:8033546 ]\n",
      "Duplicate marker name found: [ 16:9062982 ]\n",
      "Duplicate marker name found: [ 16:10306227 ]\n",
      "Duplicate marker name found: [ 16:10323525 ]\n",
      "Duplicate marker name found: [ 16:10945621 ]\n",
      "Duplicate marker name found: [ 16:11566972 ]\n",
      "Duplicate marker name found: [ 16:11953114 ]\n",
      "Duplicate marker name found: [ 16:12329642 ]\n",
      "Duplicate marker name found: [ 16:13379320 ]\n",
      "Duplicate marker name found: [ 16:13657286 ]\n",
      "Duplicate marker name found: [ 16:14574797 ]\n",
      "Duplicate marker name found: [ 16:14574797 ]\n",
      "Duplicate marker name found: [ 16:16013593 ]\n",
      "Duplicate marker name found: [ 16:16019416 ]\n",
      "Duplicate marker name found: [ 16:17266892 ]\n",
      "Duplicate marker name found: [ 16:19074994 ]\n",
      "Duplicate marker name found: [ 16:19074994 ]\n",
      "Duplicate marker name found: [ 16:19392564 ]\n",
      "Duplicate marker name found: [ 16:19392564 ]\n",
      "Duplicate marker name found: [ 16:20036508 ]\n",
      "Duplicate marker name found: [ 16:21304349 ]\n",
      "Duplicate marker name found: [ 16:22730788 ]\n",
      "Duplicate marker name found: [ 16:22869876 ]\n",
      "Duplicate marker name found: [ 16:23027214 ]\n",
      "Duplicate marker name found: [ 16:24139372 ]\n",
      "Duplicate marker name found: [ 16:24139372 ]\n",
      "Duplicate marker name found: [ 16:24230566 ]\n",
      "Duplicate marker name found: [ 16:24314665 ]\n",
      "Duplicate marker name found: [ 16:24412608 ]\n",
      "Duplicate marker name found: [ 16:24672729 ]\n",
      "Duplicate marker name found: [ 16:24672729 ]\n",
      "Duplicate marker name found: [ 16:24904612 ]\n",
      "Duplicate marker name found: [ 16:25010720 ]\n",
      "Duplicate marker name found: [ 16:27321716 ]\n",
      "Duplicate marker name found: [ 16:27450250 ]\n",
      "Duplicate marker name found: [ 16:27450250 ]\n",
      "Duplicate marker name found: [ 16:27576067 ]\n",
      "Duplicate marker name found: [ 16:28147525 ]\n",
      "Duplicate marker name found: [ 16:28515796 ]\n",
      "Duplicate marker name found: [ 16:28999024 ]\n",
      "Duplicate marker name found: [ 16:29008906 ]\n",
      "Duplicate marker name found: [ 16:29915447 ]\n",
      "Duplicate marker name found: [ 16:31003411 ]\n",
      "Duplicate marker name found: [ 16:34254129 ]\n",
      "Duplicate marker name found: [ 16:35075136 ]\n",
      "Duplicate marker name found: [ 16:47299014 ]\n",
      "Duplicate marker name found: [ 16:48550197 ]\n",
      "Duplicate marker name found: [ 16:48670514 ]\n",
      "Duplicate marker name found: [ 16:50283356 ]\n",
      "Duplicate marker name found: [ 16:50283356 ]\n",
      "Duplicate marker name found: [ 16:50722918 ]\n",
      "Duplicate marker name found: [ 16:52935758 ]\n",
      "Duplicate marker name found: [ 16:53744080 ]\n",
      "Duplicate marker name found: [ 16:55802360 ]\n",
      "Duplicate marker name found: [ 16:57338721 ]\n",
      "Duplicate marker name found: [ 16:57338721 ]\n",
      "Duplicate marker name found: [ 16:57471424 ]\n",
      "Duplicate marker name found: [ 16:58107736 ]\n",
      "Duplicate marker name found: [ 16:60284186 ]\n",
      "Duplicate marker name found: [ 16:60284186 ]\n",
      "Duplicate marker name found: [ 16:62657547 ]\n",
      "Duplicate marker name found: [ 16:62755526 ]\n",
      "Duplicate marker name found: [ 16:63322354 ]\n",
      "Duplicate marker name found: [ 16:65132418 ]\n",
      "Duplicate marker name found: [ 16:65248867 ]\n",
      "Duplicate marker name found: [ 16:66170758 ]\n",
      "Duplicate marker name found: [ 16:66324974 ]\n",
      "Duplicate marker name found: [ 16:66539955 ]\n",
      "Duplicate marker name found: [ 16:67353591 ]\n",
      "Duplicate marker name found: [ 16:67751693 ]\n",
      "Duplicate marker name found: [ 16:67751693 ]\n",
      "Duplicate marker name found: [ 16:67751693 ]\n",
      "Duplicate marker name found: [ 16:67751693 ]\n",
      "Duplicate marker name found: [ 16:68363779 ]\n",
      "Duplicate marker name found: [ 16:68785396 ]\n",
      "Duplicate marker name found: [ 16:70670674 ]\n",
      "Duplicate marker name found: [ 16:72349121 ]\n",
      "Duplicate marker name found: [ 16:72349121 ]\n",
      "Duplicate marker name found: [ 16:72349121 ]\n",
      "Duplicate marker name found: [ 16:73656464 ]\n",
      "Duplicate marker name found: [ 16:73905393 ]\n",
      "Duplicate marker name found: [ 16:74208026 ]\n",
      "Duplicate marker name found: [ 16:74710939 ]\n",
      "Duplicate marker name found: [ 16:75559812 ]\n",
      "Duplicate marker name found: [ 16:75766043 ]\n",
      "Duplicate marker name found: [ 16:76006801 ]\n",
      "Duplicate marker name found: [ 16:77464046 ]\n",
      "Duplicate marker name found: [ 16:78678120 ]\n",
      "Duplicate marker name found: [ 16:79876576 ]\n",
      "Duplicate marker name found: [ 16:79920986 ]\n",
      "Duplicate marker name found: [ 16:81261749 ]\n",
      "Duplicate marker name found: [ 16:81628639 ]\n",
      "Duplicate marker name found: [ 16:81687836 ]\n",
      "Duplicate marker name found: [ 16:81687836 ]\n",
      "Duplicate marker name found: [ 16:81868196 ]\n",
      "Duplicate marker name found: [ 16:82072853 ]\n",
      "Duplicate marker name found: [ 16:83463116 ]\n",
      "Duplicate marker name found: [ 16:83887153 ]\n",
      "Duplicate marker name found: [ 16:83887153 ]\n",
      "Duplicate marker name found: [ 16:84621827 ]\n",
      "Duplicate marker name found: [ 16:84650110 ]\n",
      "Duplicate marker name found: [ 16:84650110 ]\n",
      "Duplicate marker name found: [ 16:84667009 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:84811697 ]\n",
      "Duplicate marker name found: [ 16:85380212 ]\n",
      "Duplicate marker name found: [ 16:85421526 ]\n",
      "Duplicate marker name found: [ 16:86236422 ]\n",
      "Duplicate marker name found: [ 16:86708071 ]\n",
      "Duplicate marker name found: [ 16:86708071 ]\n",
      "Duplicate marker name found: [ 16:87503935 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 1043129 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.765768\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 1043129 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr16.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:30:53 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr17.log ]\n",
      "Analysis started: Thu Jan 12 16:30:54 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr17\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr17\n",
      "\n",
      "901318 (of 901318) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr17.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr17.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr17.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 17:1181392 ]\n",
      "Duplicate marker name found: [ 17:1181392 ]\n",
      "Duplicate marker name found: [ 17:1203743 ]\n",
      "Duplicate marker name found: [ 17:1334361 ]\n",
      "Duplicate marker name found: [ 17:2151223 ]\n",
      "Duplicate marker name found: [ 17:2933471 ]\n",
      "Duplicate marker name found: [ 17:3374818 ]\n",
      "Duplicate marker name found: [ 17:3814585 ]\n",
      "Duplicate marker name found: [ 17:4296360 ]\n",
      "Duplicate marker name found: [ 17:4296360 ]\n",
      "Duplicate marker name found: [ 17:5751948 ]\n",
      "Duplicate marker name found: [ 17:6869848 ]\n",
      "Duplicate marker name found: [ 17:6928019 ]\n",
      "Duplicate marker name found: [ 17:6928019 ]\n",
      "Duplicate marker name found: [ 17:7823732 ]\n",
      "Duplicate marker name found: [ 17:8338390 ]\n",
      "Duplicate marker name found: [ 17:10150914 ]\n",
      "Duplicate marker name found: [ 17:11094262 ]\n",
      "Duplicate marker name found: [ 17:11094262 ]\n",
      "Duplicate marker name found: [ 17:11964045 ]\n",
      "Duplicate marker name found: [ 17:12331568 ]\n",
      "Duplicate marker name found: [ 17:12359683 ]\n",
      "Duplicate marker name found: [ 17:12382723 ]\n",
      "Duplicate marker name found: [ 17:12382723 ]\n",
      "Duplicate marker name found: [ 17:12382723 ]\n",
      "Duplicate marker name found: [ 17:12444106 ]\n",
      "Duplicate marker name found: [ 17:12599365 ]\n",
      "Duplicate marker name found: [ 17:13454024 ]\n",
      "Duplicate marker name found: [ 17:14627948 ]\n",
      "Duplicate marker name found: [ 17:17164840 ]\n",
      "Duplicate marker name found: [ 17:17366508 ]\n",
      "Duplicate marker name found: [ 17:17648719 ]\n",
      "Duplicate marker name found: [ 17:18765196 ]\n",
      "Duplicate marker name found: [ 17:18846654 ]\n",
      "Duplicate marker name found: [ 17:19159733 ]\n",
      "Duplicate marker name found: [ 17:20881388 ]\n",
      "Duplicate marker name found: [ 17:25467706 ]\n",
      "Duplicate marker name found: [ 17:25740050 ]\n",
      "Duplicate marker name found: [ 17:27255795 ]\n",
      "Duplicate marker name found: [ 17:28675817 ]\n",
      "Duplicate marker name found: [ 17:29794117 ]\n",
      "Duplicate marker name found: [ 17:30182520 ]\n",
      "Duplicate marker name found: [ 17:30231689 ]\n",
      "Duplicate marker name found: [ 17:30341720 ]\n",
      "Duplicate marker name found: [ 17:33018419 ]\n",
      "Duplicate marker name found: [ 17:33134660 ]\n",
      "Duplicate marker name found: [ 17:35052342 ]\n",
      "Duplicate marker name found: [ 17:35052342 ]\n",
      "Duplicate marker name found: [ 17:35922492 ]\n",
      "Duplicate marker name found: [ 17:37118861 ]\n",
      "Duplicate marker name found: [ 17:37118861 ]\n",
      "Duplicate marker name found: [ 17:37320322 ]\n",
      "Duplicate marker name found: [ 17:37320322 ]\n",
      "Duplicate marker name found: [ 17:37874043 ]\n",
      "Duplicate marker name found: [ 17:38339399 ]\n",
      "Duplicate marker name found: [ 17:38339399 ]\n",
      "Duplicate marker name found: [ 17:40184059 ]\n",
      "Duplicate marker name found: [ 17:43042199 ]\n",
      "Duplicate marker name found: [ 17:43489161 ]\n",
      "Duplicate marker name found: [ 17:45313472 ]\n",
      "Duplicate marker name found: [ 17:46218735 ]\n",
      "Duplicate marker name found: [ 17:47180062 ]\n",
      "Duplicate marker name found: [ 17:47180062 ]\n",
      "Duplicate marker name found: [ 17:47457977 ]\n",
      "Duplicate marker name found: [ 17:47473406 ]\n",
      "Duplicate marker name found: [ 17:47494597 ]\n",
      "Duplicate marker name found: [ 17:48524904 ]\n",
      "Duplicate marker name found: [ 17:48997701 ]\n",
      "Duplicate marker name found: [ 17:49281858 ]\n",
      "Duplicate marker name found: [ 17:49419123 ]\n",
      "Duplicate marker name found: [ 17:49469120 ]\n",
      "Duplicate marker name found: [ 17:49469790 ]\n",
      "Duplicate marker name found: [ 17:50279080 ]\n",
      "Duplicate marker name found: [ 17:51477292 ]\n",
      "Duplicate marker name found: [ 17:51553246 ]\n",
      "Duplicate marker name found: [ 17:52101271 ]\n",
      "Duplicate marker name found: [ 17:52101271 ]\n",
      "Duplicate marker name found: [ 17:52519867 ]\n",
      "Duplicate marker name found: [ 17:52519867 ]\n",
      "Duplicate marker name found: [ 17:52519867 ]\n",
      "Duplicate marker name found: [ 17:52519867 ]\n",
      "Duplicate marker name found: [ 17:54431772 ]\n",
      "Duplicate marker name found: [ 17:55169040 ]\n",
      "Duplicate marker name found: [ 17:55169040 ]\n",
      "Duplicate marker name found: [ 17:55987338 ]\n",
      "Duplicate marker name found: [ 17:57203545 ]\n",
      "Duplicate marker name found: [ 17:57796677 ]\n",
      "Duplicate marker name found: [ 17:58141666 ]\n",
      "Duplicate marker name found: [ 17:59045310 ]\n",
      "Duplicate marker name found: [ 17:59045310 ]\n",
      "Duplicate marker name found: [ 17:59045310 ]\n",
      "Duplicate marker name found: [ 17:59045310 ]\n",
      "Duplicate marker name found: [ 17:59291729 ]\n",
      "Duplicate marker name found: [ 17:60935672 ]\n",
      "Duplicate marker name found: [ 17:60935672 ]\n",
      "Duplicate marker name found: [ 17:60967454 ]\n",
      "Duplicate marker name found: [ 17:61606190 ]\n",
      "Duplicate marker name found: [ 17:61816194 ]\n",
      "Duplicate marker name found: [ 17:61816194 ]\n",
      "Duplicate marker name found: [ 17:62368296 ]\n",
      "Duplicate marker name found: [ 17:65013614 ]\n",
      "Duplicate marker name found: [ 17:66362818 ]\n",
      "Duplicate marker name found: [ 17:66564610 ]\n",
      "Duplicate marker name found: [ 17:68164690 ]\n",
      "Duplicate marker name found: [ 17:68780379 ]\n",
      "Duplicate marker name found: [ 17:70547367 ]\n",
      "Duplicate marker name found: [ 17:70849146 ]\n",
      "Duplicate marker name found: [ 17:70880298 ]\n",
      "Duplicate marker name found: [ 17:71235060 ]\n",
      "Duplicate marker name found: [ 17:71609056 ]\n",
      "Duplicate marker name found: [ 17:71812014 ]\n",
      "Duplicate marker name found: [ 17:73327088 ]\n",
      "Duplicate marker name found: [ 17:73413904 ]\n",
      "Duplicate marker name found: [ 17:73978780 ]\n",
      "Duplicate marker name found: [ 17:74344241 ]\n",
      "Duplicate marker name found: [ 17:74344241 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 901318 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.773643\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 901318 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr17.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:31:11 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr18.log ]\n",
      "Analysis started: Thu Jan 12 16:31:11 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr18\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr18\n",
      "\n",
      "885530 (of 885530) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr18.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr18.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr18.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 18:815442 ]\n",
      "Duplicate marker name found: [ 18:2552190 ]\n",
      "Duplicate marker name found: [ 18:3567040 ]\n",
      "Duplicate marker name found: [ 18:4007654 ]\n",
      "Duplicate marker name found: [ 18:4059713 ]\n",
      "Duplicate marker name found: [ 18:4059713 ]\n",
      "Duplicate marker name found: [ 18:4958223 ]\n",
      "Duplicate marker name found: [ 18:5176440 ]\n",
      "Duplicate marker name found: [ 18:5643799 ]\n",
      "Duplicate marker name found: [ 18:5643799 ]\n",
      "Duplicate marker name found: [ 18:6133332 ]\n",
      "Duplicate marker name found: [ 18:7214194 ]\n",
      "Duplicate marker name found: [ 18:7214194 ]\n",
      "Duplicate marker name found: [ 18:8554395 ]\n",
      "Duplicate marker name found: [ 18:9506480 ]\n",
      "Duplicate marker name found: [ 18:11510832 ]\n",
      "Duplicate marker name found: [ 18:14128292 ]\n",
      "Duplicate marker name found: [ 18:14605112 ]\n",
      "Duplicate marker name found: [ 18:14605112 ]\n",
      "Duplicate marker name found: [ 18:18924364 ]\n",
      "Duplicate marker name found: [ 18:18924364 ]\n",
      "Duplicate marker name found: [ 18:18924364 ]\n",
      "Duplicate marker name found: [ 18:18924364 ]\n",
      "Duplicate marker name found: [ 18:19551290 ]\n",
      "Duplicate marker name found: [ 18:19581427 ]\n",
      "Duplicate marker name found: [ 18:19605268 ]\n",
      "Duplicate marker name found: [ 18:20916966 ]\n",
      "Duplicate marker name found: [ 18:21822257 ]\n",
      "Duplicate marker name found: [ 18:22432730 ]\n",
      "Duplicate marker name found: [ 18:23048793 ]\n",
      "Duplicate marker name found: [ 18:24635901 ]\n",
      "Duplicate marker name found: [ 18:27043335 ]\n",
      "Duplicate marker name found: [ 18:27296464 ]\n",
      "Duplicate marker name found: [ 18:27296464 ]\n",
      "Duplicate marker name found: [ 18:31036881 ]\n",
      "Duplicate marker name found: [ 18:32302570 ]\n",
      "Duplicate marker name found: [ 18:32302570 ]\n",
      "Duplicate marker name found: [ 18:32891360 ]\n",
      "Duplicate marker name found: [ 18:33727370 ]\n",
      "Duplicate marker name found: [ 18:37417480 ]\n",
      "Duplicate marker name found: [ 18:37562524 ]\n",
      "Duplicate marker name found: [ 18:37846715 ]\n",
      "Duplicate marker name found: [ 18:38813430 ]\n",
      "Duplicate marker name found: [ 18:40061620 ]\n",
      "Duplicate marker name found: [ 18:40118819 ]\n",
      "Duplicate marker name found: [ 18:41948525 ]\n",
      "Duplicate marker name found: [ 18:41964907 ]\n",
      "Duplicate marker name found: [ 18:42809367 ]\n",
      "Duplicate marker name found: [ 18:42958728 ]\n",
      "Duplicate marker name found: [ 18:43108542 ]\n",
      "Duplicate marker name found: [ 18:45593476 ]\n",
      "Duplicate marker name found: [ 18:47723603 ]\n",
      "Duplicate marker name found: [ 18:48472480 ]\n",
      "Duplicate marker name found: [ 18:48896816 ]\n",
      "Duplicate marker name found: [ 18:53391038 ]\n",
      "Duplicate marker name found: [ 18:54121978 ]\n",
      "Duplicate marker name found: [ 18:54606039 ]\n",
      "Duplicate marker name found: [ 18:55789727 ]\n",
      "Duplicate marker name found: [ 18:57409597 ]\n",
      "Duplicate marker name found: [ 18:58551749 ]\n",
      "Duplicate marker name found: [ 18:60068673 ]\n",
      "Duplicate marker name found: [ 18:63052540 ]\n",
      "Duplicate marker name found: [ 18:63701276 ]\n",
      "Duplicate marker name found: [ 18:65981254 ]\n",
      "Duplicate marker name found: [ 18:65981254 ]\n",
      "Duplicate marker name found: [ 18:70712153 ]\n",
      "Duplicate marker name found: [ 18:72370060 ]\n",
      "Duplicate marker name found: [ 18:74767541 ]\n",
      "Duplicate marker name found: [ 18:76810340 ]\n",
      "Duplicate marker name found: [ 18:78014365 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 885530 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.788825\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 885530 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr18.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:31:28 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr19.log ]\n",
      "Analysis started: Thu Jan 12 16:31:29 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr19\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr19\n",
      "\n",
      "733122 (of 733122) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr19.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr19.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr19.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 19:769624 ]\n",
      "Duplicate marker name found: [ 19:946796 ]\n",
      "Duplicate marker name found: [ 19:2436379 ]\n",
      "Duplicate marker name found: [ 19:3587268 ]\n",
      "Duplicate marker name found: [ 19:3904850 ]\n",
      "Duplicate marker name found: [ 19:4982602 ]\n",
      "Duplicate marker name found: [ 19:5247205 ]\n",
      "Duplicate marker name found: [ 19:5768331 ]\n",
      "Duplicate marker name found: [ 19:5768331 ]\n",
      "Duplicate marker name found: [ 19:6321615 ]\n",
      "Duplicate marker name found: [ 19:6475048 ]\n",
      "Duplicate marker name found: [ 19:6865205 ]\n",
      "Duplicate marker name found: [ 19:7228991 ]\n",
      "Duplicate marker name found: [ 19:7949029 ]\n",
      "Duplicate marker name found: [ 19:8137604 ]\n",
      "Duplicate marker name found: [ 19:8570490 ]\n",
      "Duplicate marker name found: [ 19:8978542 ]\n",
      "Duplicate marker name found: [ 19:9937315 ]\n",
      "Duplicate marker name found: [ 19:12153951 ]\n",
      "Duplicate marker name found: [ 19:12922731 ]\n",
      "Duplicate marker name found: [ 19:13915283 ]\n",
      "Duplicate marker name found: [ 19:13930468 ]\n",
      "Duplicate marker name found: [ 19:13930468 ]\n",
      "Duplicate marker name found: [ 19:13956870 ]\n",
      "Duplicate marker name found: [ 19:14050681 ]\n",
      "Duplicate marker name found: [ 19:14050681 ]\n",
      "Duplicate marker name found: [ 19:14050681 ]\n",
      "Duplicate marker name found: [ 19:14050681 ]\n",
      "Duplicate marker name found: [ 19:15673651 ]\n",
      "Duplicate marker name found: [ 19:16159449 ]\n",
      "Duplicate marker name found: [ 19:17037114 ]\n",
      "Duplicate marker name found: [ 19:17037114 ]\n",
      "Duplicate marker name found: [ 19:17304981 ]\n",
      "Duplicate marker name found: [ 19:18038438 ]\n",
      "Duplicate marker name found: [ 19:18038438 ]\n",
      "Duplicate marker name found: [ 19:18038438 ]\n",
      "Duplicate marker name found: [ 19:19891298 ]\n",
      "Duplicate marker name found: [ 19:19891298 ]\n",
      "Duplicate marker name found: [ 19:22973483 ]\n",
      "Duplicate marker name found: [ 19:22973483 ]\n",
      "Duplicate marker name found: [ 19:23540039 ]\n",
      "Duplicate marker name found: [ 19:24176935 ]\n",
      "Duplicate marker name found: [ 19:30382409 ]\n",
      "Duplicate marker name found: [ 19:31397096 ]\n",
      "Duplicate marker name found: [ 19:32231190 ]\n",
      "Duplicate marker name found: [ 19:33778576 ]\n",
      "Duplicate marker name found: [ 19:33778576 ]\n",
      "Duplicate marker name found: [ 19:33916203 ]\n",
      "Duplicate marker name found: [ 19:34542215 ]\n",
      "Duplicate marker name found: [ 19:34630728 ]\n",
      "Duplicate marker name found: [ 19:34786420 ]\n",
      "Duplicate marker name found: [ 19:34786420 ]\n",
      "Duplicate marker name found: [ 19:36192381 ]\n",
      "Duplicate marker name found: [ 19:36192381 ]\n",
      "Duplicate marker name found: [ 19:37118583 ]\n",
      "Duplicate marker name found: [ 19:38118580 ]\n",
      "Duplicate marker name found: [ 19:38832856 ]\n",
      "Duplicate marker name found: [ 19:40982932 ]\n",
      "Duplicate marker name found: [ 19:40982932 ]\n",
      "Duplicate marker name found: [ 19:41142882 ]\n",
      "Duplicate marker name found: [ 19:42716189 ]\n",
      "Duplicate marker name found: [ 19:42716189 ]\n",
      "Duplicate marker name found: [ 19:42716189 ]\n",
      "Duplicate marker name found: [ 19:43088355 ]\n",
      "Duplicate marker name found: [ 19:43088355 ]\n",
      "Duplicate marker name found: [ 19:43088355 ]\n",
      "Duplicate marker name found: [ 19:43892182 ]\n",
      "Duplicate marker name found: [ 19:45750854 ]\n",
      "Duplicate marker name found: [ 19:45750854 ]\n",
      "Duplicate marker name found: [ 19:45926018 ]\n",
      "Duplicate marker name found: [ 19:46770841 ]\n",
      "Duplicate marker name found: [ 19:47046891 ]\n",
      "Duplicate marker name found: [ 19:47410735 ]\n",
      "Duplicate marker name found: [ 19:47410735 ]\n",
      "Duplicate marker name found: [ 19:49007046 ]\n",
      "Duplicate marker name found: [ 19:49638292 ]\n",
      "Duplicate marker name found: [ 19:49723542 ]\n",
      "Duplicate marker name found: [ 19:52153979 ]\n",
      "Duplicate marker name found: [ 19:52756242 ]\n",
      "Duplicate marker name found: [ 19:54588401 ]\n",
      "Duplicate marker name found: [ 19:56678868 ]\n",
      "Duplicate marker name found: [ 19:59088116 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 733122 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.77566\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 733122 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr19.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:31:43 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr20.log ]\n",
      "Analysis started: Thu Jan 12 16:31:43 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr20\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr20\n",
      "\n",
      "711997 (of 711997) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr20.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr20.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr20.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 20:343597 ]\n",
      "Duplicate marker name found: [ 20:435294 ]\n",
      "Duplicate marker name found: [ 20:868530 ]\n",
      "Duplicate marker name found: [ 20:868530 ]\n",
      "Duplicate marker name found: [ 20:3616010 ]\n",
      "Duplicate marker name found: [ 20:3842390 ]\n",
      "Duplicate marker name found: [ 20:7301632 ]\n",
      "Duplicate marker name found: [ 20:9379336 ]\n",
      "Duplicate marker name found: [ 20:9955281 ]\n",
      "Duplicate marker name found: [ 20:9955281 ]\n",
      "Duplicate marker name found: [ 20:10040772 ]\n",
      "Duplicate marker name found: [ 20:10219553 ]\n",
      "Duplicate marker name found: [ 20:10219553 ]\n",
      "Duplicate marker name found: [ 20:10219553 ]\n",
      "Duplicate marker name found: [ 20:10219553 ]\n",
      "Duplicate marker name found: [ 20:10339897 ]\n",
      "Duplicate marker name found: [ 20:10832408 ]\n",
      "Duplicate marker name found: [ 20:10852576 ]\n",
      "Duplicate marker name found: [ 20:11582669 ]\n",
      "Duplicate marker name found: [ 20:11582669 ]\n",
      "Duplicate marker name found: [ 20:11774691 ]\n",
      "Duplicate marker name found: [ 20:14915771 ]\n",
      "Duplicate marker name found: [ 20:14915771 ]\n",
      "Duplicate marker name found: [ 20:16358057 ]\n",
      "Duplicate marker name found: [ 20:16429521 ]\n",
      "Duplicate marker name found: [ 20:16429521 ]\n",
      "Duplicate marker name found: [ 20:19743400 ]\n",
      "Duplicate marker name found: [ 20:20572328 ]\n",
      "Duplicate marker name found: [ 20:20840676 ]\n",
      "Duplicate marker name found: [ 20:21258707 ]\n",
      "Duplicate marker name found: [ 20:21640023 ]\n",
      "Duplicate marker name found: [ 20:22398566 ]\n",
      "Duplicate marker name found: [ 20:23496441 ]\n",
      "Duplicate marker name found: [ 20:23496441 ]\n",
      "Duplicate marker name found: [ 20:23502385 ]\n",
      "Duplicate marker name found: [ 20:24745884 ]\n",
      "Duplicate marker name found: [ 20:30100625 ]\n",
      "Duplicate marker name found: [ 20:30167685 ]\n",
      "Duplicate marker name found: [ 20:30167685 ]\n",
      "Duplicate marker name found: [ 20:32138414 ]\n",
      "Duplicate marker name found: [ 20:33280482 ]\n",
      "Duplicate marker name found: [ 20:33937795 ]\n",
      "Duplicate marker name found: [ 20:34184160 ]\n",
      "Duplicate marker name found: [ 20:34595111 ]\n",
      "Duplicate marker name found: [ 20:34595111 ]\n",
      "Duplicate marker name found: [ 20:34718709 ]\n",
      "Duplicate marker name found: [ 20:34984700 ]\n",
      "Duplicate marker name found: [ 20:36384802 ]\n",
      "Duplicate marker name found: [ 20:38272144 ]\n",
      "Duplicate marker name found: [ 20:39326907 ]\n",
      "Duplicate marker name found: [ 20:40510395 ]\n",
      "Duplicate marker name found: [ 20:41311911 ]\n",
      "Duplicate marker name found: [ 20:41311911 ]\n",
      "Duplicate marker name found: [ 20:42020959 ]\n",
      "Duplicate marker name found: [ 20:43359123 ]\n",
      "Duplicate marker name found: [ 20:43634894 ]\n",
      "Duplicate marker name found: [ 20:44813723 ]\n",
      "Duplicate marker name found: [ 20:45045464 ]\n",
      "Duplicate marker name found: [ 20:45513944 ]\n",
      "Duplicate marker name found: [ 20:45979213 ]\n",
      "Duplicate marker name found: [ 20:47227806 ]\n",
      "Duplicate marker name found: [ 20:47298216 ]\n",
      "Duplicate marker name found: [ 20:47298216 ]\n",
      "Duplicate marker name found: [ 20:47944458 ]\n",
      "Duplicate marker name found: [ 20:48164142 ]\n",
      "Duplicate marker name found: [ 20:48164142 ]\n",
      "Duplicate marker name found: [ 20:51817187 ]\n",
      "Duplicate marker name found: [ 20:52228121 ]\n",
      "Duplicate marker name found: [ 20:52228121 ]\n",
      "Duplicate marker name found: [ 20:52959571 ]\n",
      "Duplicate marker name found: [ 20:52959571 ]\n",
      "Duplicate marker name found: [ 20:53109225 ]\n",
      "Duplicate marker name found: [ 20:53109225 ]\n",
      "Duplicate marker name found: [ 20:54723680 ]\n",
      "Duplicate marker name found: [ 20:55388680 ]\n",
      "Duplicate marker name found: [ 20:55682730 ]\n",
      "Duplicate marker name found: [ 20:56355997 ]\n",
      "Duplicate marker name found: [ 20:56898681 ]\n",
      "Duplicate marker name found: [ 20:57060199 ]\n",
      "Duplicate marker name found: [ 20:59337851 ]\n",
      "Duplicate marker name found: [ 20:59337851 ]\n",
      "Duplicate marker name found: [ 20:60686810 ]\n",
      "Duplicate marker name found: [ 20:60758121 ]\n",
      "Duplicate marker name found: [ 20:60823728 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 711997 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.772508\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 711997 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr20.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:31:55 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr21.log ]\n",
      "Analysis started: Thu Jan 12 16:31:55 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr21\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr21\n",
      "\n",
      "471466 (of 471466) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr21.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr21.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr21.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 21:15302167 ]\n",
      "Duplicate marker name found: [ 21:16090569 ]\n",
      "Duplicate marker name found: [ 21:16090569 ]\n",
      "Duplicate marker name found: [ 21:17854185 ]\n",
      "Duplicate marker name found: [ 21:18818199 ]\n",
      "Duplicate marker name found: [ 21:19097405 ]\n",
      "Duplicate marker name found: [ 21:19813598 ]\n",
      "Duplicate marker name found: [ 21:20281034 ]\n",
      "Duplicate marker name found: [ 21:20668172 ]\n",
      "Duplicate marker name found: [ 21:23101329 ]\n",
      "Duplicate marker name found: [ 21:24689547 ]\n",
      "Duplicate marker name found: [ 21:24992971 ]\n",
      "Duplicate marker name found: [ 21:27325057 ]\n",
      "Duplicate marker name found: [ 21:27325057 ]\n",
      "Duplicate marker name found: [ 21:27876543 ]\n",
      "Duplicate marker name found: [ 21:30417069 ]\n",
      "Duplicate marker name found: [ 21:30585931 ]\n",
      "Duplicate marker name found: [ 21:30972110 ]\n",
      "Duplicate marker name found: [ 21:33450689 ]\n",
      "Duplicate marker name found: [ 21:33486549 ]\n",
      "Duplicate marker name found: [ 21:33505771 ]\n",
      "Duplicate marker name found: [ 21:33505771 ]\n",
      "Duplicate marker name found: [ 21:36038270 ]\n",
      "Duplicate marker name found: [ 21:36571330 ]\n",
      "Duplicate marker name found: [ 21:37374886 ]\n",
      "Duplicate marker name found: [ 21:37374886 ]\n",
      "Duplicate marker name found: [ 21:37976009 ]\n",
      "Duplicate marker name found: [ 21:37976009 ]\n",
      "Duplicate marker name found: [ 21:38395892 ]\n",
      "Duplicate marker name found: [ 21:41256726 ]\n",
      "Duplicate marker name found: [ 21:43462438 ]\n",
      "Duplicate marker name found: [ 21:45087097 ]\n",
      "Duplicate marker name found: [ 21:45202407 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 471466 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.740363\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 471466 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr21.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:32:03 2017\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "\n",
      "@----------------------------------------------------------@\n",
      "|        PLINK!       |     v1.07      |   10/Aug/2009     |\n",
      "|----------------------------------------------------------|\n",
      "|  (C) 2009 Shaun Purcell, GNU General Public License, v2  |\n",
      "|----------------------------------------------------------|\n",
      "|  For documentation, citation & bug-report instructions:  |\n",
      "|        http://pngu.mgh.harvard.edu/purcell/plink/        |\n",
      "@----------------------------------------------------------@\n",
      "\n",
      "Skipping web check... [ --noweb ] \n",
      "Writing this text to log file [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr22.log ]\n",
      "Analysis started: Thu Jan 12 16:32:04 2017\n",
      "\n",
      "Options in effect:\n",
      "\t--file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr22\n",
      "\t--noweb\n",
      "\t--recodeAD\n",
      "\t--out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr22\n",
      "\n",
      "447623 (of 447623) markers to be included from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr22.map ]\n",
      "Warning, found 29 individuals with ambiguous sex codes\n",
      "Writing list of these individuals to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr22.nosex ]\n",
      "29 individuals read from [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr22.ped ] \n",
      "0 individuals with nonmissing phenotypes\n",
      "Assuming a disease phenotype (1=unaff, 2=aff, 0=miss)\n",
      "Missing phenotype value is also -9\n",
      "0 cases, 0 controls and 29 missing\n",
      "0 males, 0 females, and 29 of unspecified sex\n",
      " *** WARNING *** DUPLICATE MARKERS FOUND *** \n",
      "Duplicate marker name found: [ 22:18078898 ]\n",
      "Duplicate marker name found: [ 22:18078898 ]\n",
      "Duplicate marker name found: [ 22:18078898 ]\n",
      "Duplicate marker name found: [ 22:18496882 ]\n",
      "Duplicate marker name found: [ 22:18496882 ]\n",
      "Duplicate marker name found: [ 22:19402459 ]\n",
      "Duplicate marker name found: [ 22:19402459 ]\n",
      "Duplicate marker name found: [ 22:19850513 ]\n",
      "Duplicate marker name found: [ 22:23551731 ]\n",
      "Duplicate marker name found: [ 22:24681608 ]\n",
      "Duplicate marker name found: [ 22:28215513 ]\n",
      "Duplicate marker name found: [ 22:28698027 ]\n",
      "Duplicate marker name found: [ 22:28827964 ]\n",
      "Duplicate marker name found: [ 22:28841849 ]\n",
      "Duplicate marker name found: [ 22:29239629 ]\n",
      "Duplicate marker name found: [ 22:31029339 ]\n",
      "Duplicate marker name found: [ 22:32129890 ]\n",
      "Duplicate marker name found: [ 22:33138259 ]\n",
      "Duplicate marker name found: [ 22:33228659 ]\n",
      "Duplicate marker name found: [ 22:33287963 ]\n",
      "Duplicate marker name found: [ 22:33332475 ]\n",
      "Duplicate marker name found: [ 22:33877206 ]\n",
      "Duplicate marker name found: [ 22:33877206 ]\n",
      "Duplicate marker name found: [ 22:35814679 ]\n",
      "Duplicate marker name found: [ 22:35958832 ]\n",
      "Duplicate marker name found: [ 22:35958832 ]\n",
      "Duplicate marker name found: [ 22:36055800 ]\n",
      "Duplicate marker name found: [ 22:36199194 ]\n",
      "Duplicate marker name found: [ 22:36447921 ]\n",
      "Duplicate marker name found: [ 22:36865088 ]\n",
      "Duplicate marker name found: [ 22:37139624 ]\n",
      "Duplicate marker name found: [ 22:37363993 ]\n",
      "Duplicate marker name found: [ 22:37674310 ]\n",
      "Duplicate marker name found: [ 22:37674310 ]\n",
      "Duplicate marker name found: [ 22:38986091 ]\n",
      "Duplicate marker name found: [ 22:38986091 ]\n",
      "Duplicate marker name found: [ 22:39374551 ]\n",
      "Duplicate marker name found: [ 22:40922187 ]\n",
      "Duplicate marker name found: [ 22:41358094 ]\n",
      "Duplicate marker name found: [ 22:44865211 ]\n",
      "Duplicate marker name found: [ 22:44865211 ]\n",
      "Duplicate marker name found: [ 22:44865211 ]\n",
      "Duplicate marker name found: [ 22:47536209 ]\n",
      "Duplicate marker name found: [ 22:47847872 ]\n",
      "Duplicate marker name found: [ 22:47847872 ]\n",
      "Duplicate marker name found: [ 22:48107717 ]\n",
      "Duplicate marker name found: [ 22:49251154 ]\n",
      "\n",
      "Before frequency and genotyping pruning, there are 447623 SNPs\n",
      "29 founders and 0 non-founders found\n",
      "Total genotyping rate in remaining individuals is 0.764917\n",
      "0 SNPs failed missingness test ( GENO > 1 )\n",
      "0 SNPs failed frequency test ( MAF < 0 )\n",
      "After frequency and genotyping pruning, there are 447623 SNPs\n",
      "After filtering, 0 cases, 0 controls and 29 missing\n",
      "After filtering, 0 males, 0 females, and 29 of unspecified sex\n",
      "Writing recoded file to [ /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr22.raw ] \n",
      "\n",
      "Analysis finished: Thu Jan 12 16:32:11 2017\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import subprocess\n",
    "\n",
    "nrs =list(range(0,22))\n",
    "\n",
    "#####Multiple processes \n",
    "\n",
    "for nr in nrs:\n",
    "    print(\"\\n\")\n",
    "    \n",
    "    print(\"export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\")\n",
    "\n",
    "    #tc = (\"vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr\" + str(nr+1) + \".recode.vcf.gz \" \\\n",
    "    #\"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr+1) +\"_imputed_phased_v5a.vcf.gz\" \\\n",
    "    #\"| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz\")\n",
    "    \n",
    "    #print(perl5)\n",
    "    #print(tc)\n",
    "    \n",
    "    #subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT)\n",
    "    #print(subprocess.check_output([perl5, tc], shell=True, stderr=subprocess.STDOUT).decode())\n",
    "\n",
    "    #tc = (\"tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr +1) + \".vcf.gz\")\n",
    "    #print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "    #print(tc)\n",
    "#    \n",
    "#    \n",
    "    #tc = (\"vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1))\n",
    "    #print(tc)\n",
    "    \n",
    "#    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "#    \n",
    "    tc = (\"plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1) + \" --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr\" + str(nr+1))\n",
    "    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "6:63259_0         NaN     NaN      0    NaN    NaN    NaN     NaN     NaN   \n",
      "rs561313667_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs530120680_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "6:69252_G         NaN     NaN    NaN    NaN    NaN      1     NaN     NaN   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "6:63259_0         NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs561313667_0       0       0  ...         0       0       0      0       0   \n",
      "rs530120680_0       0       0  ...         0       0       0      0       0   \n",
      "6:69252_G         NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "6:63259_0        NaN    NaN    NaN    NaN    NaN  \n",
      "rs561313667_0      0      0      0      0      0  \n",
      "rs530120680_0      0      0      0      0      0  \n",
      "6:69252_G        NaN    NaN    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "7:11955_T           1     NaN    NaN    NaN    NaN    NaN     NaN     NaN   \n",
      "7:12051_G           1     NaN    NaN    NaN    NaN    NaN     NaN     NaN   \n",
      "rs555283805_C     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs576737504_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "7:11955_T         NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "7:12051_G         NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs555283805_C       0       0  ...         0       0       0      0       0   \n",
      "rs576737504_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "7:11955_T        NaN    NaN    NaN    NaN    NaN  \n",
      "7:12051_G        NaN    NaN    NaN    NaN    NaN  \n",
      "rs555283805_C      0      0      0      0      0  \n",
      "rs576737504_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "          WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11 imp_P12  \\\n",
      "8:10553_0     NaN       0      0    NaN    NaN    NaN     NaN     NaN     NaN   \n",
      "8:11386_T     NaN     NaN    NaN    NaN    NaN      1     NaN     NaN     NaN   \n",
      "8:11571_A       0       1      1    NaN      0      0     NaN     NaN     NaN   \n",
      "8:11658_G     NaN       1    NaN    NaN    NaN    NaN     NaN     NaN     NaN   \n",
      "\n",
      "          imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31 imp_P4 imp_P5  \\\n",
      "8:10553_0     NaN  ...       NaN     NaN     NaN    NaN     NaN    NaN    NaN   \n",
      "8:11386_T     NaN  ...       NaN     NaN     NaN    NaN     NaN    NaN    NaN   \n",
      "8:11571_A     NaN  ...       NaN     NaN     NaN    NaN     NaN    NaN    NaN   \n",
      "8:11658_G     NaN  ...       NaN     NaN     NaN    NaN     NaN    NaN    NaN   \n",
      "\n",
      "          imp_P6 imp_P7 imp_P8  \n",
      "8:10553_0    NaN    NaN    NaN  \n",
      "8:11386_T    NaN    NaN    NaN  \n",
      "8:11571_A    NaN    NaN    NaN  \n",
      "8:11658_G    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "                                                   WGS_P15 WGS_P17 WGS_P4  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...     NaN     NaN    NaN   \n",
      "9:10293_0                                              NaN     NaN      0   \n",
      "9:10340_T                                              NaN       1    NaN   \n",
      "9:10356_0                                              NaN     NaN      0   \n",
      "\n",
      "                                                   WGS_P5 WGS_P7 WGS_P8  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...    NaN    NaN      1   \n",
      "9:10293_0                                             NaN    NaN    NaN   \n",
      "9:10340_T                                             NaN    NaN    NaN   \n",
      "9:10356_0                                             NaN    NaN    NaN   \n",
      "\n",
      "                                                   imp_P10 imp_P11 imp_P12  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...     NaN     NaN     NaN   \n",
      "9:10293_0                                              NaN     NaN     NaN   \n",
      "9:10340_T                                              NaN     NaN     NaN   \n",
      "9:10356_0                                              NaN     NaN     NaN   \n",
      "\n",
      "                                                   imp_P14  ...   imp_P26  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...     NaN  ...       NaN   \n",
      "9:10293_0                                              NaN  ...       NaN   \n",
      "9:10340_T                                              NaN  ...       NaN   \n",
      "9:10356_0                                              NaN  ...       NaN   \n",
      "\n",
      "                                                   imp_P27 imp_P29 imp_P3  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...     NaN     NaN    NaN   \n",
      "9:10293_0                                              NaN     NaN    NaN   \n",
      "9:10340_T                                              NaN     NaN    NaN   \n",
      "9:10356_0                                              NaN     NaN    NaN   \n",
      "\n",
      "                                                   imp_P31 imp_P4 imp_P5  \\\n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...     NaN    NaN    NaN   \n",
      "9:10293_0                                              NaN    NaN    NaN   \n",
      "9:10340_T                                              NaN    NaN    NaN   \n",
      "9:10356_0                                              NaN    NaN    NaN   \n",
      "\n",
      "                                                   imp_P6 imp_P7 imp_P8  \n",
      "9:10248_TAACCCTAACCCTAACCCTAACCCTAACCCCTAACCCCT...    NaN    NaN    NaN  \n",
      "9:10293_0                                             NaN    NaN    NaN  \n",
      "9:10340_T                                             NaN    NaN    NaN  \n",
      "9:10356_0                                             NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs568182971_G     NaN     NaN    NaN    NaN    NaN    NaN       1       1   \n",
      "rs112920234_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "10:60597_A          1     NaN    NaN    NaN    NaN    NaN     NaN     NaN   \n",
      "10:60637_A        NaN     NaN    NaN    NaN      1    NaN     NaN     NaN   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs568182971_G       0       0  ...         0       0       0      0       0   \n",
      "rs112920234_0       0       0  ...         0       0       0      0       0   \n",
      "10:60597_A        NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "10:60637_A        NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs568182971_G      0      0      0      0      0  \n",
      "rs112920234_0      0      0      0      0      0  \n",
      "10:60597_A       NaN    NaN    NaN    NaN    NaN  \n",
      "10:60637_A       NaN    NaN    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "            WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "11:61837_0      NaN       0      0    NaN    NaN    NaN     NaN     NaN   \n",
      "11:61843_0      NaN       0      0    NaN    NaN    NaN     NaN     NaN   \n",
      "11:70263_G      NaN       1      1    NaN    NaN    NaN     NaN     NaN   \n",
      "11:70701_AG       1       1      1    NaN    NaN    NaN     NaN     NaN   \n",
      "\n",
      "            imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "11:61837_0      NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "11:61843_0      NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "11:70263_G      NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "11:70701_AG     NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "\n",
      "            imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "11:61837_0     NaN    NaN    NaN    NaN    NaN  \n",
      "11:61843_0     NaN    NaN    NaN    NaN    NaN  \n",
      "11:70263_G     NaN    NaN    NaN    NaN    NaN  \n",
      "11:70701_AG    NaN    NaN    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs371828716_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs566046722_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs112712197_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs537376773_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs371828716_0       0       0  ...         0       0       0      0       0   \n",
      "rs566046722_0       0       0  ...         0       0       0      0       0   \n",
      "rs112712197_0       0       0  ...         0       0       0      0       0   \n",
      "rs537376773_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs371828716_0      0      0      0      0      0  \n",
      "rs566046722_0      0      0      0      0      0  \n",
      "rs112712197_0      0      0      0      0      0  \n",
      "rs537376773_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "13:19020145_G       0       0      1      0      0      1     NaN     NaN   \n",
      "rs184229798_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs12018140_0        0       0      0      0      0      0       0       0   \n",
      "rs185910374_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "13:19020145_G     NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs184229798_0       0       0  ...         0       0       0      0       0   \n",
      "rs12018140_0        0       0  ...         0       0       0      0       0   \n",
      "rs185910374_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "13:19020145_G    NaN    NaN    NaN    NaN    NaN  \n",
      "rs184229798_0      0      0      0      0      0  \n",
      "rs12018140_0       0      0      0      0      0  \n",
      "rs185910374_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs28973059_G        1       1    NaN    NaN      1      1       0       0   \n",
      "rs558975054_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs28971053_0      NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs569534039_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs28973059_G        0       0  ...         0       0       0      0       0   \n",
      "rs558975054_0       0       0  ...         0       0       0      0       0   \n",
      "rs28971053_0        0       0  ...         0       0       0      0       0   \n",
      "rs569534039_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs28973059_G       0      0      0      0      0  \n",
      "rs558975054_0      0      0      0      0      0  \n",
      "rs28971053_0       0      0      0      0      0  \n",
      "rs569534039_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs565090028_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs201219124_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs373298708_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs560310552_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs565090028_0       0       0  ...         0       0       0      0       0   \n",
      "rs201219124_0       0       0  ...         0       0       0      0       0   \n",
      "rs373298708_0       0       0  ...         0       0       0      0       0   \n",
      "rs560310552_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs565090028_0      0      0      0      0      0  \n",
      "rs201219124_0      0      0      0      0      0  \n",
      "rs373298708_0      0      0      0      0      0  \n",
      "rs560310552_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "16:60185_C          1       1      1      1      1      1     NaN     NaN   \n",
      "16:60214_A          1       1      1      1      1      1     NaN     NaN   \n",
      "rs375524167_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs188810967_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "16:60185_C        NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "16:60214_A        NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs375524167_0       0       0  ...         0       0       0      0       0   \n",
      "rs188810967_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "16:60185_C       NaN    NaN    NaN    NaN    NaN  \n",
      "16:60214_A       NaN    NaN    NaN    NaN    NaN  \n",
      "rs375524167_0      0      0      0      0      0  \n",
      "rs188810967_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs148170422_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs183779916_A     NaN     NaN    NaN      1    NaN    NaN       0       0   \n",
      "17:302_T            0       1      0      0      1      0     NaN     NaN   \n",
      "rs141990622_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs148170422_0       0       0  ...         0       0       0      0       0   \n",
      "rs183779916_A       0       0  ...         0       0       0      0       0   \n",
      "17:302_T          NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs141990622_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs148170422_0      0      0      0      0      0  \n",
      "rs183779916_A      0      0      0      0      0  \n",
      "17:302_T         NaN    NaN    NaN    NaN    NaN  \n",
      "rs141990622_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "                                                   WGS_P15 WGS_P17 WGS_P4  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...     NaN     NaN    NaN   \n",
      "18:10621_0                                             NaN       0      0   \n",
      "rs115926245_0                                          NaN     NaN    NaN   \n",
      "rs566492364_0                                          NaN     NaN    NaN   \n",
      "\n",
      "                                                   WGS_P5 WGS_P7 WGS_P8  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...    NaN      1    NaN   \n",
      "18:10621_0                                            NaN    NaN    NaN   \n",
      "rs115926245_0                                         NaN    NaN    NaN   \n",
      "rs566492364_0                                         NaN    NaN    NaN   \n",
      "\n",
      "                                                   imp_P10 imp_P11 imp_P12  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...     NaN     NaN     NaN   \n",
      "18:10621_0                                             NaN     NaN     NaN   \n",
      "rs115926245_0                                            0       0       0   \n",
      "rs566492364_0                                            0       0       0   \n",
      "\n",
      "                                                   imp_P14  ...   imp_P26  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...     NaN  ...       NaN   \n",
      "18:10621_0                                             NaN  ...       NaN   \n",
      "rs115926245_0                                            0  ...         0   \n",
      "rs566492364_0                                            0  ...         0   \n",
      "\n",
      "                                                   imp_P27 imp_P29 imp_P3  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...     NaN     NaN    NaN   \n",
      "18:10621_0                                             NaN     NaN    NaN   \n",
      "rs115926245_0                                            0       0      0   \n",
      "rs566492364_0                                            0       0      0   \n",
      "\n",
      "                                                   imp_P31 imp_P4 imp_P5  \\\n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...     NaN    NaN    NaN   \n",
      "18:10621_0                                             NaN    NaN    NaN   \n",
      "rs115926245_0                                            0      0      0   \n",
      "rs566492364_0                                            0      0      0   \n",
      "\n",
      "                                                   imp_P6 imp_P7 imp_P8  \n",
      "18:10503_AACCCTACCCTAACCCTACCCTACCCTAACCCTAACCC...    NaN    NaN    NaN  \n",
      "18:10621_0                                            NaN    NaN    NaN  \n",
      "rs115926245_0                                           0      0      0  \n",
      "rs566492364_0                                           0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "             WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "19:60354_T         1       1      1      1      0      1     NaN     NaN   \n",
      "19:60477_ACC     NaN     NaN      1    NaN    NaN      1     NaN     NaN   \n",
      "19:60542_C       NaN       1      1    NaN      1      1     NaN     NaN   \n",
      "19:60612_A       NaN     NaN    NaN    NaN      1    NaN     NaN     NaN   \n",
      "\n",
      "             imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "19:60354_T       NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "19:60477_ACC     NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "19:60542_C       NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "19:60612_A       NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "\n",
      "             imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "19:60354_T      NaN    NaN    NaN    NaN    NaN  \n",
      "19:60477_ACC    NaN    NaN    NaN    NaN    NaN  \n",
      "19:60542_C      NaN    NaN    NaN    NaN    NaN  \n",
      "19:60612_A      NaN    NaN    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs150241001_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs116145529_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs187713677_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "rs6078030_T       NaN     NaN    NaN      2      2    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs150241001_0       0       0  ...         0       0       0      0       0   \n",
      "rs116145529_0       0       0  ...         0       0       0      0       0   \n",
      "rs187713677_0       0       0  ...         0       0       0      0       0   \n",
      "rs6078030_T         0       0  ...         0       0       0      1       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs150241001_0      0      0      0      0      0  \n",
      "rs116145529_0      0      0      0      0      0  \n",
      "rs187713677_0      0      0      0      0      0  \n",
      "rs6078030_T        0      0      0      0      1  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs550852792_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "21:9411318_C        1     NaN      1    NaN    NaN      1     NaN     NaN   \n",
      "21:9411327_C        1       1      1      1      1      1     NaN     NaN   \n",
      "rs539713234_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs550852792_0       0       0  ...         0       0       0      0       0   \n",
      "21:9411318_C      NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "21:9411327_C      NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs539713234_0       0       0  ...         0       0       0      0       0   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs550852792_0      0      0      0      0      0  \n",
      "21:9411318_C     NaN    NaN    NaN    NaN    NaN  \n",
      "21:9411327_C     NaN    NaN    NaN    NaN    NaN  \n",
      "rs539713234_0      0      0      0      0      0  \n",
      "\n",
      "[4 rows x 29 columns]\n",
      "              WGS_P15 WGS_P17 WGS_P4 WGS_P5 WGS_P7 WGS_P8 imp_P10 imp_P11  \\\n",
      "rs587755077_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "22:16050159_C     NaN       1      1      1      1    NaN     NaN     NaN   \n",
      "rs587654921_0     NaN     NaN    NaN    NaN    NaN    NaN       0       0   \n",
      "22:16050252_A     NaN       1      1      1      1    NaN     NaN     NaN   \n",
      "\n",
      "              imp_P12 imp_P14  ...   imp_P26 imp_P27 imp_P29 imp_P3 imp_P31  \\\n",
      "rs587755077_0       0       0  ...         0       0       0      0       0   \n",
      "22:16050159_C     NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "rs587654921_0       0       0  ...         0       0       0      0       0   \n",
      "22:16050252_A     NaN     NaN  ...       NaN     NaN     NaN    NaN     NaN   \n",
      "\n",
      "              imp_P4 imp_P5 imp_P6 imp_P7 imp_P8  \n",
      "rs587755077_0      0      0      0      0      0  \n",
      "22:16050159_C    NaN    NaN    NaN    NaN    NaN  \n",
      "rs587654921_0      0      0      0      0      0  \n",
      "22:16050252_A    NaN    NaN    NaN    NaN    NaN  \n",
      "\n",
      "[4 rows x 29 columns]\n"
     ]
    }
   ],
   "source": [
    "########Loop through all chromosomes, safe as pickle, create heatmap \n",
    "## \n",
    "import pandas\n",
    "import os \n",
    "import numpy \n",
    "import seaborn \n",
    "import pickle \n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "nrs =list(range(5,22))\n",
    "for nr in nrs:\n",
    "\n",
    "    ### Read-Data \n",
    "    data = pandas.read_table(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr\" + str(nr +1) + \".raw\"), sep = \" \")\n",
    "\n",
    "    ##Transform in numpy object, transform and convert to pandas\n",
    "    table = data.as_matrix()\n",
    "    genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "    pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "    pd_genotype = pd_genotype[6:]\n",
    "\n",
    "    #print(pd_genotype[0:10])\n",
    "    pd_genotype.shape\n",
    "\n",
    "    ##Select only even rows (because they are duplicate)\n",
    "    pd_genotype =pd_genotype[::2]\n",
    "    pd_genotype.shape\n",
    "\n",
    "    print(pd_genotype[1:5])\n",
    "\n",
    "    ##Safe object as pickle \n",
    "    with open(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr\" + str(nr +1) + \"_pickle\"), 'wb') as f:\n",
    "        pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "\n",
    "    #####################################Make correlation heatmap of chr1 \n",
    "    ##Remove NA values \n",
    "    pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "    #print(pd_genotype[1:10])\n",
    "    pd_genotype.shape\n",
    "\n",
    "    ##Calculate correlations \n",
    "    plot_data = pd_genotype.astype(float).corr()\n",
    "    #print(plot_data)\n",
    "\n",
    "    PLOT_CMAP = 'YlOrRd'\n",
    "    VMIN = 0\n",
    "    VMAX = 1\n",
    "    fig_size=(12,12)\n",
    "\n",
    "    ##PLot and write to files\n",
    "\n",
    "    cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "    _=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".pdf\"),  bbox_inches='tight')\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".png\"),  bbox_inches='tight')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
