{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loading libraries "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import re\n",
    "import math, random\n",
    "import numpy,scipy,pandas\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn\n",
    "import itertools\n",
    "import subprocess\n",
    "from pgtools import toolbox, myplots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "ImportError",
     "evalue": "No module named 'rnatools'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-7-ce78b2435f76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0msys\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'/home/dskola/workspace/expression_modeling/'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0;32mimport\u001b[0m \u001b[0mrnatools\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mImportError\u001b[0m: No module named 'rnatools'"
     ]
    }
   ],
   "source": [
    "import sys\n",
    "sys.path.append('/home/dskola/workspace/expression_modeling/')\n",
    "import rnatools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Overview file with FASTQ files used for genotyping and expression:\n",
    "## Depository for all fastq files for genotyping: \n",
    "#/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping\n",
    "\n",
    "##Depository for all fastq files for expresson (Sometimes multiple files per patient, for example, when 2 brain regions) \n",
    "\n",
    "#/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-expression "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "human_h3k27ac_sample_info.csv\n",
      "patient_info.csv\n",
      "human_input_sample_info.csv\n",
      "human_pu_sample_info.csv\n",
      "human_atac_sample_info.csv\n",
      "human_rna_tag_fastq_mapping.csv\n",
      "human_rna_sample_info.csv\n",
      "sample_info.csv\n",
      "human_h3k4me2_sample_info.csv\n"
     ]
    }
   ],
   "source": [
    "######################## \n",
    "##### Copy files from Dylan and rewrite with date to home dir\n",
    "files ={\"human_rna_sample_info.csv\" : \"/home/dskola/human/rna/human_rna_sample_info.csv\", \\\n",
    "        \"human_atac_sample_info.csv\" : \"/home/dskola/human/atac/human_atac_sample_info.csv\", \\\n",
    "        \"patient_info.csv\" :\"/home/dskola/human/patient_info.csv\", \\\n",
    "        \"sample_info.csv\" : \"/home/dskola/human/sample_info.csv\", \\\n",
    "        \"human_rna_tag_fastq_mapping.csv\" : \"/home/dskola/human/dbgap_submission/human_rna_tag_fastq_mapping.csv\", \\\n",
    "        \"human_h3k4me2_sample_info.csv\": \"/home/dskola/human/h3k4me2/human_h3k4me2_sample_info.csv\", \\\n",
    "        \"human_h3k27ac_sample_info.csv\": \"/home/dskola/human/h3k27ac/human_h3k27ac_sample_info.csv\",\\\n",
    "        \"human_input_sample_info.csv\": \"/home/dskola/human/input/human_input_sample_info.csv\", \\\n",
    "        \"human_pu_sample_info.csv\": \"/home/dskola/human/pu1/human_pu1_sample_info.csv\", \\\n",
    "       }\n",
    "      \n",
    "#\"human_epigenomics_sample_name_mapping.csv\": \"/home/dskola/human/human_epigenomics_sample_name_mapping.csv\", \\    \n",
    "\n",
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'    \n",
    "\n",
    "date = \"Feb_20_2017_\"\n",
    "\n",
    "for key,index in files.items():\n",
    "    print(key)\n",
    "    data = pandas.read_csv(index)\n",
    "    data.to_csv(os.path.join(OUTPUT_PATH, date + key))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'list' object has no attribute 'keys'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-21-01b9d19f0769>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mfiles\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkeys\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'list' object has no attribute 'keys'"
     ]
    }
   ],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 330,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'human_rna_tag_fastq_mapping.csv': '/home/iholtman/Human_natural_genetic_variation/rna/Feb_20_2017_human_rna_tag_fastq_mapping.csv', 'human_rna_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/rna/Feb_20_2017_human_rna_sample_info.csv', 'sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_sample_info.csv', 'patient_info.csv': '/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_patient_info.csv'}\n"
     ]
    }
   ],
   "source": [
    "#####################\n",
    "####Load files as basis for master-matrix \n",
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'    \n",
    "date = \"Feb_20_2017_\"\n",
    "files ={\"human_rna_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/rna/Feb_20_2017_human_rna_sample_info.csv\", \\\n",
    "        \"patient_info.csv\" :\"/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_patient_info.csv\", \\\n",
    "        \"sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_sample_info.csv\", \\\n",
    "        \"human_rna_tag_fastq_mapping.csv\" : \"/home/iholtman/Human_natural_genetic_variation/rna/Feb_20_2017_human_rna_tag_fastq_mapping.csv\"}\n",
    "      \n",
    "###\n",
    "human_rna_sample_info = pandas.read_csv(files['human_rna_sample_info.csv'])\n",
    "patient_info = pandas.read_csv(files['patient_info.csv'])\n",
    "sample_info = pandas.read_csv(files['sample_info.csv'])\n",
    "human_rna_tag_fastq_mapping = pandas.read_csv(files['human_rna_tag_fastq_mapping.csv'])\n",
    "   \n",
    "print(files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 331,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "human_rna_sample_info\n",
      "   Unnamed: 0                                             Sample in_vitro  \\\n",
      "0           0  Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_16...    False   \n",
      "1           1  Human_Microglia_Brain_Sorted_RNA_PolyA_P3_Surg...     True   \n",
      "2           2   Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15    False   \n",
      "3           3  Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04    False   \n",
      "4           4  Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7...     True   \n",
      "\n",
      "  wholebrain   mcsf   il34    10d     6h    24h    72h  \\\n",
      "0      False  False  False  False  False  False  False   \n",
      "1      False  False   True  False  False   True  False   \n",
      "2      False  False  False  False  False  False  False   \n",
      "3      False  False  False  False  False  False  False   \n",
      "4      False  False   True  False  False  False  False   \n",
      "\n",
      "                         ...                         patient  acute  \\\n",
      "0                        ...                          HMG003  False   \n",
      "1                        ...                          HMG003  False   \n",
      "2                        ...                          HMG003  False   \n",
      "3                        ...                          HMG015  False   \n",
      "4                        ...                          HMG015  False   \n",
      "\n",
      "  broad_diagnosis                                          pathology   age  \\\n",
      "0           Tumor                                  High grade glioma   4.0   \n",
      "1           Tumor                                  High grade glioma   4.0   \n",
      "2           Tumor                                  High grade glioma   4.0   \n",
      "3           Tumor  Dysembryoblastic neural epithelial tumor, WHO ...  11.0   \n",
      "4           Tumor  Dysembryoblastic neural epithelial tumor, WHO ...  11.0   \n",
      "\n",
      "  sex ethnicity date (mm-dd-yy)               medication regimen  \\\n",
      "0   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "1   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "2   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "3   F         C       1/19/2016                          Keppra    \n",
      "4   F         C       1/19/2016                          Keppra    \n",
      "\n",
      "                                                path  \n",
      "0  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "1  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  \n",
      "2  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "3  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "4  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  \n",
      "\n",
      "[5 rows x 37 columns]\n",
      "\n",
      "-------------------------------------------------------------------------\n",
      "patient_info\n",
      "   Unnamed: 0 Patient   Patient_nr Broad_Diagnosis  \\\n",
      "0           0   HMG001           1           Tumor   \n",
      "1           1   HMG002           2           Tumor   \n",
      "2           2   HMG003           3           Tumor   \n",
      "3           3   HMG004           4        Epilepsy   \n",
      "4           4   HMG005           5        Epilepsy   \n",
      "\n",
      "                      Pathology   Age Sex Ethnicity Date (mm-dd-yy)  \\\n",
      "0                  Glioblastoma  9.00   M         C         3/11/15   \n",
      "1  Atypical rhaboid brain tumor  1.00   F         C         3/25/15   \n",
      "2             High grade glioma  4.00   M       His          4/6/15   \n",
      "3  Cortical dysplasia, grade IA  0.66   F         C         4/20/15   \n",
      "4            Negative pathology  7.00   F       His         5/26/15   \n",
      "\n",
      "                            Medication regimen  \n",
      "0                                         None  \n",
      "1               Steroids, Keppra/Phenobarbitol  \n",
      "2              Temozolomide, Bevaclizumab, XRT  \n",
      "3  Klonopin, Keppra, Trileptal, Phenobarbital   \n",
      "4                       Keppra, Depakote, Onfi  \n",
      "\n",
      "-------------------------------------------------------------------------\n",
      "sample_info\n",
      "   Unnamed: 0 sample_id Patient     Region Tissue resected (in gram)  \\\n",
      "0           0      S001  HMG001  Occipital                       N.A   \n",
      "1           1      S002  HMG002   Temporal                       1.7   \n",
      "2           2      S003  HMG003   Temporal                       3.6   \n",
      "3           3      S004  HMG004     Medial                      14.1   \n",
      "4           4      S005  HMG004     Medial                      14.1   \n",
      "\n",
      "   Cells isolated (in million)       Region Full   tissue_type sample_date  \\\n",
      "0                        0.055  Occipital cortex     Microglia   3/11/2015   \n",
      "1                        1.130   Temporal cortex     Microglia   3/25/2015   \n",
      "2                        0.450   Temporal cortex     Microglia    4/6/2015   \n",
      "3                        5.250     Medial cortex     Microglia   4/20/2015   \n",
      "4                          NaN     Medial cortex  Brain Lysate   4/20/2015   \n",
      "\n",
      "   patient_age  \n",
      "0         9.00  \n",
      "1         1.00  \n",
      "2         4.00  \n",
      "3         0.66  \n",
      "4         0.66  \n",
      "\n",
      "-------------------------------------------------------------------------\n",
      "human_rna_tag_fastq_mapping\n",
      "   Unnamed: 0                                         tag_folder  \\\n",
      "0           0   Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15   \n",
      "1           1   Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15   \n",
      "2           2   Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15   \n",
      "3           3  Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_...   \n",
      "4           4  Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...   \n",
      "\n",
      "                                          fastq_path  \n",
      "0  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "1  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "2  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "3  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "4  /data/archive/15-06-17-glass/Project_David/hg3...  \n"
     ]
    }
   ],
   "source": [
    "print(\"human_rna_sample_info\")\n",
    "print(human_rna_sample_info[0:5])\n",
    "print(\"\\n-------------------------------------------------------------------------\")\n",
    "print(\"patient_info\")\n",
    "print(patient_info[0:5])\n",
    "print(\"\\n-------------------------------------------------------------------------\")\n",
    "print(\"sample_info\")\n",
    "print(sample_info[0:5])\n",
    "print(\"\\n-------------------------------------------------------------------------\")\n",
    "print(\"human_rna_tag_fastq_mapping\")\n",
    "print(human_rna_tag_fastq_mapping[0:5])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 332,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                      cat  patient  patientz\n",
      "fastq                                                                       \n",
      "/data/scratch/iholtman/human_microglia_genotypi...  False       26        26\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  False       27        27\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  False       29        29\n",
      "/data/scratch/iholtman/human_microglia_genotypi...  False        4         0\n"
     ]
    }
   ],
   "source": [
    "############################\n",
    "########Location of fastq-files used for genotyping and add patientnumber as a separate column  \n",
    "import glob\n",
    "import os\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/5-dec/\"\n",
    "dir2= \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/cat_fastq_files_28_nov/\"\n",
    "\n",
    "files= glob.glob(dir +\"*.fastq.gz\") + glob.glob(dir2 +\"*.fastq.gz\")\n",
    "\n",
    "folders = pandas.DataFrame([], index=files)\n",
    "folders.index.name='fastq'\n",
    "\n",
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "folders['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patient\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "#print(folders[\"patient\"])\n",
    "\n",
    "print(folders[1:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 333,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   Unnamed: 0 Patient   Patient_nr Broad_Diagnosis  \\\n",
      "0           0   HMG001           1           Tumor   \n",
      "1           1   HMG002           2           Tumor   \n",
      "2           2   HMG003           3           Tumor   \n",
      "3           3   HMG004           4        Epilepsy   \n",
      "4           4   HMG005           5        Epilepsy   \n",
      "\n",
      "                      Pathology   Age Sex Ethnicity Date (mm-dd-yy)  \\\n",
      "0                  Glioblastoma  9.00   M         C         3/11/15   \n",
      "1  Atypical rhaboid brain tumor  1.00   F         C         3/25/15   \n",
      "2             High grade glioma  4.00   M       His          4/6/15   \n",
      "3  Cortical dysplasia, grade IA  0.66   F         C         4/20/15   \n",
      "4            Negative pathology  7.00   F       His         5/26/15   \n",
      "\n",
      "                            Medication regimen  \\\n",
      "0                                         None   \n",
      "1               Steroids, Keppra/Phenobarbitol   \n",
      "2              Temozolomide, Bevaclizumab, XRT   \n",
      "3  Klonopin, Keppra, Trileptal, Phenobarbital    \n",
      "4                       Keppra, Depakote, Onfi   \n",
      "\n",
      "                                      genotype_fastq  \n",
      "0                                                  0  \n",
      "1  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "2  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "3  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "4  /data/scratch/iholtman/human_microglia_genotyp...  \n"
     ]
    }
   ],
   "source": [
    "#############\n",
    "#Add fastq files used for genotyping to patient info matrix \n",
    "fastqs = []\n",
    "\n",
    "for donor in patient_info[\"Patient_nr\"]:\n",
    "    dir = list(folders.index[folders[\"patient\"] == donor ])\n",
    "    if dir:\n",
    "        fastqs.append(dir[0])\n",
    "    else:\n",
    "        fastqs.append(0)\n",
    "        \n",
    "patient_info[\"genotype_fastq\"] = fastqs\n",
    "\n",
    "print(patient_info[0:5])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 334,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   Unnamed: 0                                             Sample in_vitro  \\\n",
      "0           0  Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_16...    False   \n",
      "1           1  Human_Microglia_Brain_Sorted_RNA_PolyA_P3_Surg...     True   \n",
      "2           2   Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15    False   \n",
      "3           3  Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04    False   \n",
      "4           4  Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7...     True   \n",
      "\n",
      "  wholebrain   mcsf   il34    10d     6h    24h    72h  \\\n",
      "0      False  False  False  False  False  False  False   \n",
      "1      False  False   True  False  False   True  False   \n",
      "2      False  False  False  False  False  False  False   \n",
      "3      False  False  False  False  False  False  False   \n",
      "4      False  False   True  False  False  False  False   \n",
      "\n",
      "                         ...                         patient  acute  \\\n",
      "0                        ...                          HMG003  False   \n",
      "1                        ...                          HMG003  False   \n",
      "2                        ...                          HMG003  False   \n",
      "3                        ...                          HMG015  False   \n",
      "4                        ...                          HMG015  False   \n",
      "\n",
      "  broad_diagnosis                                          pathology   age  \\\n",
      "0           Tumor                                  High grade glioma   4.0   \n",
      "1           Tumor                                  High grade glioma   4.0   \n",
      "2           Tumor                                  High grade glioma   4.0   \n",
      "3           Tumor  Dysembryoblastic neural epithelial tumor, WHO ...  11.0   \n",
      "4           Tumor  Dysembryoblastic neural epithelial tumor, WHO ...  11.0   \n",
      "\n",
      "  sex ethnicity date (mm-dd-yy)               medication regimen  \\\n",
      "0   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "1   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "2   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "3   F         C       1/19/2016                          Keppra    \n",
      "4   F         C       1/19/2016                          Keppra    \n",
      "\n",
      "                                                path  \n",
      "0  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "1  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  \n",
      "2  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "3  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  \n",
      "4  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  \n",
      "\n",
      "[5 rows x 37 columns]\n"
     ]
    }
   ],
   "source": [
    "print(human_rna_sample_info[0:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 335,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Unnamed: 0 Patient   Patient_nr Broad_Diagnosis  \\\n",
      "0            0   HMG001           1           Tumor   \n",
      "1            1   HMG002           2           Tumor   \n",
      "2            2   HMG003           3           Tumor   \n",
      "3            3   HMG004           4        Epilepsy   \n",
      "4            4   HMG005           5        Epilepsy   \n",
      "5            5   HMG006           6           Tumor   \n",
      "6            6   HMG007           7        Epilepsy   \n",
      "7            7   HMG008           8        Epilepsy   \n",
      "8            8   HMG009           9           Tumor   \n",
      "9            9   HMG010          10           Tumor   \n",
      "10          10   HMG011          11        Epilepsy   \n",
      "11          11   HMG012          12           Tumor   \n",
      "12          12   HMG013          13        Epilepsy   \n",
      "13          13   HMG014          14        Epilepsy   \n",
      "14          14   HMG015          15           Tumor   \n",
      "15          15   HMG016          16           Tumor   \n",
      "16          16   HMG017          17        Epilepsy   \n",
      "17          17   HMG018          18           Tumor   \n",
      "18          18   HMG019          19        Epilepsy   \n",
      "19          19   HMG020          20           Tumor   \n",
      "20          20   HMG021          21        Epilepsy   \n",
      "21          21   HMG022          22           Tumor   \n",
      "22          22   HMG023          23        Epilepsy   \n",
      "23          23   HMG024          24           Tumor   \n",
      "24          24   HMG025          25           Tumor   \n",
      "25          25   HMG026          26           Tumor   \n",
      "26          26   HMG027          27           Tumor   \n",
      "27          27   HMG028          28           Tumor   \n",
      "28          28   HMG029          29           Acute   \n",
      "29          29   HMG030          30           Tumor   \n",
      "30          30   HMG031          31        Epilepsy   \n",
      "31          31   HMG032          32           Tumor   \n",
      "32          32   HMG033          33           Tumor   \n",
      "33          33   HMG034          34           Tumor   \n",
      "34          34   HMG035          35           Acute   \n",
      "35          35   HMG036          36           Tumor   \n",
      "36          36   HMG037          37           Tumor   \n",
      "37          37   HMG038          38           Tumor   \n",
      "38          38   HMG039          39           Tumor   \n",
      "39          39   HMG040          40           Tumor   \n",
      "\n",
      "                                            Pathology        Age Sex  \\\n",
      "0                                        Glioblastoma   9.000000   M   \n",
      "1                        Atypical rhaboid brain tumor   1.000000   F   \n",
      "2                                   High grade glioma   4.000000   M   \n",
      "3                        Cortical dysplasia, grade IA   0.660000   F   \n",
      "4                                  Negative pathology   7.000000   F   \n",
      "5              Cerebellar pylotic astrocytoma grade I   3.000000   F   \n",
      "6   Neurocytic ganglioglioma with severe cortical ...  13.000000   F   \n",
      "7      Cortical dysplasia, grade IC, gliosis/scarring  17.000000   M   \n",
      "8               Anaplastic ependymoma - WHO grade III  11.000000   M   \n",
      "9               Anaplastic ependymoma - WHO grade III  13.000000   F   \n",
      "10                                 Negative pathology  15.000000   F   \n",
      "11                       Atypical rhaboid brain tumor   1.166667   M   \n",
      "12                                   Ischemic changes  13.000000   F   \n",
      "13                 Focal cortical dysplasia, type IIB   1.000000   F   \n",
      "14  Dysembryoblastic neural epithelial tumor, WHO ...  11.000000   F   \n",
      "15     Pilocytic astrocytoma with anaplastic features  13.000000   F   \n",
      "16                          Neuronal loss and gliosis   2.000000   F   \n",
      "17                Anaplastic medulloblastoma - WHO IV   5.000000   M   \n",
      "18                                 Negative pathology   1.666667   M   \n",
      "19                              Glioblastoma - WHO IV   5.000000   M   \n",
      "20                 Focal cortical dysplasia, type IIA   9.000000   F   \n",
      "21                                      Pineoblastoma  12.000000   F   \n",
      "22                       Posterior pial angiomatosis    4.000000   M   \n",
      "23         Juvenile Pilocytic Atrocytoma, WHO grade 1   8.000000   M   \n",
      "24         Juvenile Pilocytic Atrocytoma, WHO grade 1   5.000000   M   \n",
      "25  Dysembryoplastic neuroepithelial tumor, WHO gr...  16.000000   M   \n",
      "26                         Embryonic choriocarcinoma    3.000000   M   \n",
      "27                                         Ependymoma  12.000000   M   \n",
      "28                                           Aneurysm  16.000000   F   \n",
      "29                                    Medullablastoma   6.000000   M   \n",
      "30                  Focal cortical dysplasia type Iib   2.000000   M   \n",
      "31                                  High grade glioma   0.416667   M   \n",
      "32                              Vestibular Schwannoma  15.000000   F   \n",
      "33  Medullablastoma - WHO grade IV, anaplastic lar...  10.000000   M   \n",
      "34                    Anoxic ischemic encephalopathy    0.416667   M   \n",
      "35                Pilocytic astrocytome - WHO grade I  13.000000   F   \n",
      "36                              Glioma - WHO grade II  18.000000   M   \n",
      "37              Anaplastic ependymoma - WHO grade III   3.000000   F   \n",
      "38                                            Pending  11.000000   M   \n",
      "39                                            Pending  13.000000   M   \n",
      "\n",
      "    Ethnicity Date (mm-dd-yy)  \\\n",
      "0           C         3/11/15   \n",
      "1           C         3/25/15   \n",
      "2         His          4/6/15   \n",
      "3           C         4/20/15   \n",
      "4         His         5/26/15   \n",
      "5           C         5/28/15   \n",
      "6           C         5/28/15   \n",
      "7         His         6/22/15   \n",
      "8           C         7/20/15   \n",
      "9         His         8/19/15   \n",
      "10        His         8/27/15   \n",
      "11        His        11/25/15   \n",
      "12     Asian         12/14/15   \n",
      "13        His         1/12/16   \n",
      "14          C         1/19/16   \n",
      "15        His         1/25/16   \n",
      "16          C          2/1/16   \n",
      "17        His         2/19/16   \n",
      "18          C         4/26/16   \n",
      "19        His         4/27/16   \n",
      "20          C          6/6/16   \n",
      "21      Asian         7/29/16   \n",
      "22        His          8/8/16   \n",
      "23        His         8/10/16   \n",
      "24          C         8/15/16   \n",
      "25        His         8/15/16   \n",
      "26      AfrAm         8/29/16   \n",
      "27          C         9/18/16   \n",
      "28    MidEast         9/19/16   \n",
      "29        His         9/19/16   \n",
      "30        His         9/27/16   \n",
      "31      Asian        10/14/16   \n",
      "32        His        11/14/16   \n",
      "33        His        11/28/16   \n",
      "34        His        11/20/16   \n",
      "35  Hawaiian         12/16/16   \n",
      "36          C        12/19/16   \n",
      "37          C        12/19/16   \n",
      "38     Indian         1/12/17   \n",
      "39          C         1/12/17   \n",
      "\n",
      "                                   Medication regimen  \\\n",
      "0                                                None   \n",
      "1                      Steroids, Keppra/Phenobarbitol   \n",
      "2                     Temozolomide, Bevaclizumab, XRT   \n",
      "3         Klonopin, Keppra, Trileptal, Phenobarbital    \n",
      "4                              Keppra, Depakote, Onfi   \n",
      "5                                           Steroids    \n",
      "6                                           Phenytoin   \n",
      "7                          Klonopin, Lamictal, Keppra   \n",
      "8                                   Radiation therapy   \n",
      "9                                                None   \n",
      "10                                          Trileptal   \n",
      "11  Methotrexate, Vincristine, Cisplatin, Cyclopho...   \n",
      "12                        Topamax, Vimpat, Zonisamide   \n",
      "13                      Vimpat, Phenobarbital, Keppra   \n",
      "14                                            Keppra    \n",
      "15                                               None   \n",
      "16                                  Clobazam, Topamax   \n",
      "17                                           Decadron   \n",
      "18                                             Keppra   \n",
      "19  Temozolomide, Bevaclizumab, XRT, Pablociclib, ...   \n",
      "20                                    Topamax, Vimpat   \n",
      "21                                           Decadron   \n",
      "22                                     Keppra, Banzal   \n",
      "23                                           Decadron   \n",
      "24                                           Decadron   \n",
      "25                                       Lamotrigine    \n",
      "26                                           Decadron   \n",
      "27                                        Keppra, XRT   \n",
      "28                                               None   \n",
      "29                                           Decadron   \n",
      "30                            Onfi, Keppra, Trileptal   \n",
      "31                                           Decadron   \n",
      "32                                           Decadron   \n",
      "33                                           Decadron   \n",
      "34                                              None    \n",
      "35                                           Decadron   \n",
      "36                                              None    \n",
      "37                                   Decadron, Keppra   \n",
      "38                                             Keppra   \n",
      "39                                             Keppra   \n",
      "\n",
      "                                       genotype_fastq  \\\n",
      "0                                                   0   \n",
      "1   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "2   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "3   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "4   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "5   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "6   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "7   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "8                                                   0   \n",
      "9   /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "10  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "11  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "12                                                  0   \n",
      "13  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "14  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "15  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "16  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "17  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "18  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "19                                                  0   \n",
      "20  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "21                                                  0   \n",
      "22  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "23  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "24                                                  0   \n",
      "25  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "26  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "27                                                  0   \n",
      "28  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "29                                                  0   \n",
      "30  /data/scratch/iholtman/human_microglia_genotyp...   \n",
      "31                                                  0   \n",
      "32                                                  0   \n",
      "33                                                  0   \n",
      "34                                                  0   \n",
      "35                                                  0   \n",
      "36                                                  0   \n",
      "37                                                  0   \n",
      "38                                                  0   \n",
      "39                                                  0   \n",
      "\n",
      "                                          sample1_exp  \\\n",
      "0                                                  Na   \n",
      "1    Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15   \n",
      "2   Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_16...   \n",
      "3   Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG   \n",
      "4    Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15   \n",
      "5    Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21   \n",
      "6    Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15   \n",
      "7   Human_Microglia_RNA_PolyA_P8_150622_Occipital_...   \n",
      "8                                                  Na   \n",
      "9   Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23   \n",
      "10  Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10...   \n",
      "11  Human_Microglia_RNA_PolyA_P12_151125_DGO_15_12_14   \n",
      "12                                                 Na   \n",
      "13  Human_Microglia_RNA_PolyA_P14_exvivo_JoS_16_04_27   \n",
      "14  Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04   \n",
      "15  Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04   \n",
      "16  Human_Microglia_RNA_PolyA_P17_Temporal_160201_...   \n",
      "17  Human_Microglia_RNA_PolyA_P18_exvivo_JoS_16_04_27   \n",
      "18  Human_Microglia_RNA_PolyA_P19_Temporal_160426_...   \n",
      "19                                                 Na   \n",
      "20  Human_Microglia_RNA_PolyA_P21_Cortex_160606_12...   \n",
      "21                                                 Na   \n",
      "22  Human_Microglia_Sorted_RNA_PolyA_P23_Culture_6...   \n",
      "23  Human_Microglia_RNA_PolyA_P24_Tumor_160810_DGO...   \n",
      "24  Human_Microglia_RNA_PolyA_P25_Tumor_160815_DGO...   \n",
      "25  Human_Microglia_RNA_PolyA_P26_Tumor_160815_DGO...   \n",
      "26                                                 Na   \n",
      "27                                                 Na   \n",
      "28  Human_Microglia_RNA_PolyA_P29_Frontal_Aneurysm...   \n",
      "29                                                 Na   \n",
      "30  Human_Microglia_RNA_PolyA_P31_160927_DGO_16_10_28   \n",
      "31                                                 Na   \n",
      "32                                                 Na   \n",
      "33                                                 Na   \n",
      "34  Human_Microglia_RNA_PolyA_P35_AcuteIschemia_16...   \n",
      "35                                                 Na   \n",
      "36                                                 Na   \n",
      "37                                                 Na   \n",
      "38                                                 Na   \n",
      "39                                                 Na   \n",
      "\n",
      "                                          sample2_exp  \\\n",
      "0                                                  Na   \n",
      "1                                                  Na   \n",
      "2   Human_Microglia_Brain_Sorted_RNA_PolyA_P3_Surg...   \n",
      "3                                                  Na   \n",
      "4   Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_...   \n",
      "5                                                  Na   \n",
      "6   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...   \n",
      "7   Human_Microglia_RNA_PolyA_P8_150622_Parietal_D...   \n",
      "8                                                  Na   \n",
      "9                                                  Na   \n",
      "10                                                 Na   \n",
      "11                                                 Na   \n",
      "12                                                 Na   \n",
      "13       P14_brain_RNA_Brain_Grey_JoS_16_04_27_ATTCCT   \n",
      "14  Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7...   \n",
      "15                                                 Na   \n",
      "16  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...   \n",
      "17                                                 Na   \n",
      "18                                                 Na   \n",
      "19                                                 Na   \n",
      "20  Human_Microglia_Sorted_RNA_PolyA_P21_Culture_7...   \n",
      "21                                                 Na   \n",
      "22                                                 Na   \n",
      "23                                                 Na   \n",
      "24  Human_Microglia_Sorted_RNA_PolyA_P25_Culture_7...   \n",
      "25                                                 Na   \n",
      "26                                                 Na   \n",
      "27                                                 Na   \n",
      "28                                                 Na   \n",
      "29                                                 Na   \n",
      "30                                                 Na   \n",
      "31                                                 Na   \n",
      "32                                                 Na   \n",
      "33                                                 Na   \n",
      "34                                                 Na   \n",
      "35                                                 Na   \n",
      "36                                                 Na   \n",
      "37                                                 Na   \n",
      "38                                                 Na   \n",
      "39                                                 Na   \n",
      "\n",
      "                                          sample3_exp  \n",
      "0                                                  Na  \n",
      "1                                                  Na  \n",
      "2    Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15  \n",
      "3                                                  Na  \n",
      "4   Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...  \n",
      "5                                                  Na  \n",
      "6   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...  \n",
      "7   Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_...  \n",
      "8                                                  Na  \n",
      "9                                                  Na  \n",
      "10                                                 Na  \n",
      "11                                                 Na  \n",
      "12                                                 Na  \n",
      "13                                                 Na  \n",
      "14                                                 Na  \n",
      "15                                                 Na  \n",
      "16  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...  \n",
      "17                                                 Na  \n",
      "18                                                 Na  \n",
      "19                                                 Na  \n",
      "20  Human_Microglia_Sorted_RNA_PolyA_P21_Culture_7...  \n",
      "21                                                 Na  \n",
      "22                                                 Na  \n",
      "23                                                 Na  \n",
      "24  Human_Microglia_Sorted_RNA_PolyA_P25_Culture_7...  \n",
      "25                                                 Na  \n",
      "26                                                 Na  \n",
      "27                                                 Na  \n",
      "28                                                 Na  \n",
      "29                                                 Na  \n",
      "30                                                 Na  \n",
      "31                                                 Na  \n",
      "32                                                 Na  \n",
      "33                                                 Na  \n",
      "34                                                 Na  \n",
      "35                                                 Na  \n",
      "36                                                 Na  \n",
      "37                                                 Na  \n",
      "38                                                 Na  \n",
      "39                                                 Na  \n"
     ]
    }
   ],
   "source": [
    "####################\n",
    "### To do: Toevoegen van de geconcetanate bestanden (zie locatie hierboven)\n",
    "### To do: multiple files for expressie analyze (zie locatie hierboven )\n",
    "## Add sample info for each sample: WB, MG, exvivo, concatenated\n",
    "\n",
    "sample1_exp = []\n",
    "sample2_exp = []\n",
    "sample3_exp = []\n",
    "\n",
    "for patient in patient_info['Patient ']:\n",
    "    if ((sum(human_rna_sample_info[\"patient\"] == patient) == 0)):\n",
    "               sample1_exp.append(\"Na\") \n",
    "               sample2_exp.append(\"Na\") \n",
    "               sample3_exp.append(\"Na\") \n",
    "    elif ((sum(human_rna_sample_info[\"patient\"] == patient) == 1)):\n",
    "               sample1_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[0])\n",
    "               sample2_exp.append(\"Na\") \n",
    "               sample3_exp.append(\"Na\") \n",
    "    elif ((sum(human_rna_sample_info[\"patient\"] == patient) == 2)):\n",
    "               sample1_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[0])\n",
    "               sample2_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[1])\n",
    "               sample3_exp.append(\"Na\")\n",
    "    else: \n",
    "               sample1_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[0])\n",
    "               sample2_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[1])\n",
    "               sample3_exp.append(human_rna_sample_info[human_rna_sample_info[\"patient\"] == patient]['Sample'].iloc[2])\n",
    "    \n",
    "patient_info['sample1_exp'] = sample1_exp\n",
    "patient_info['sample2_exp'] = sample2_exp\n",
    "patient_info['sample3_exp'] = sample3_exp\n",
    "\n",
    "print(patient_info)\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 336,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#######Fastq files + Annotation of the file (1st)\n",
    "fastq_path = []\n",
    "\n",
    "for sample in patient_info['sample1_exp']: \n",
    "    if sample == \"Na\":\n",
    "        fastq_path.append(\"Na\")\n",
    "    elif sample: \n",
    "        fastq_path.append(human_rna_tag_fastq_mapping[human_rna_tag_fastq_mapping[\"tag_folder\"] == sample]['fastq_path'].iloc[0])\n",
    "    else: \n",
    "        fastq_path.append(\"Na\")\n",
    "\n",
    "patient_info['sample1_exp_fastq_path'] = fastq_path\n",
    "\n",
    "#for i in patient_info['sample1_exp'].values:\n",
    "#patient_info['sample1_exp'].values.lower()\n",
    "patient_info['sample1_exp_ex_vivo'] = [i.lower().find('ex') > -1 for i in patient_info['sample1_exp']]\n",
    "patient_info['sample1_exp_occipital'] = [i.lower().find('occipital') > -1 for i in patient_info['sample1_exp']]\n",
    "patient_info['sample1_exp_temporal'] = [i.lower().find('temporal') > -1 for i in patient_info['sample1_exp']]\n",
    "patient_info['sample1_exp_in_vitro'] = [i.lower().find('culture') > -1 for i in patient_info['sample1_exp']]\n",
    "patient_info['sample1_exp_microglia'] = [i.lower().find('microglia') > -1 for i in patient_info['sample1_exp']]\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 337,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#######Fastq files + Annotation of the file (2nd)\n",
    "fastq_path = []\n",
    "\n",
    "for sample in patient_info['sample2_exp']: \n",
    "    if sample == \"Na\":\n",
    "        fastq_path.append(\"Na\")\n",
    "    elif sample: \n",
    "        fastq_path.append(human_rna_tag_fastq_mapping[human_rna_tag_fastq_mapping[\"tag_folder\"] == sample][\"fastq_path\"].iloc[0])\n",
    "    else: \n",
    "        fastq_path.append(\"Na\")\n",
    "\n",
    "patient_info['sample2_exp_fastq_path'] = fastq_path\n",
    "\n",
    "fastq_path = []\n",
    "\n",
    "patient_info['sample2_exp_ex_vivo'] = [i.lower().find('ex') > -1 for i in patient_info['sample2_exp']]\n",
    "patient_info['sample2_exp_occipital'] = [i.lower().find('occipital') > -1 for i in patient_info['sample2_exp']]\n",
    "patient_info['sample2_exp_temporal'] = [i.lower().find('temporal') > -1 for i in patient_info['sample2_exp']]\n",
    "patient_info['sample2_exp_in_vitro'] = [i.lower().find('culture') > -1 for i in patient_info['sample2_exp']]\n",
    "patient_info['sample2_exp_microglia'] = [i.lower().find('microglia') > -1 for i in patient_info['sample2_exp']]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 338,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Unnamed: 0 Patient   Patient_nr Broad_Diagnosis  \\\n",
      "0            0   HMG001           1           Tumor   \n",
      "1            1   HMG002           2           Tumor   \n",
      "2            2   HMG003           3           Tumor   \n",
      "3            3   HMG004           4        Epilepsy   \n",
      "4            4   HMG005           5        Epilepsy   \n",
      "5            5   HMG006           6           Tumor   \n",
      "6            6   HMG007           7        Epilepsy   \n",
      "7            7   HMG008           8        Epilepsy   \n",
      "8            8   HMG009           9           Tumor   \n",
      "9            9   HMG010          10           Tumor   \n",
      "10          10   HMG011          11        Epilepsy   \n",
      "11          11   HMG012          12           Tumor   \n",
      "12          12   HMG013          13        Epilepsy   \n",
      "13          13   HMG014          14        Epilepsy   \n",
      "14          14   HMG015          15           Tumor   \n",
      "15          15   HMG016          16           Tumor   \n",
      "16          16   HMG017          17        Epilepsy   \n",
      "17          17   HMG018          18           Tumor   \n",
      "18          18   HMG019          19        Epilepsy   \n",
      "19          19   HMG020          20           Tumor   \n",
      "20          20   HMG021          21        Epilepsy   \n",
      "21          21   HMG022          22           Tumor   \n",
      "22          22   HMG023          23        Epilepsy   \n",
      "23          23   HMG024          24           Tumor   \n",
      "24          24   HMG025          25           Tumor   \n",
      "25          25   HMG026          26           Tumor   \n",
      "26          26   HMG027          27           Tumor   \n",
      "27          27   HMG028          28           Tumor   \n",
      "28          28   HMG029          29           Acute   \n",
      "29          29   HMG030          30           Tumor   \n",
      "30          30   HMG031          31        Epilepsy   \n",
      "31          31   HMG032          32           Tumor   \n",
      "32          32   HMG033          33           Tumor   \n",
      "33          33   HMG034          34           Tumor   \n",
      "34          34   HMG035          35           Acute   \n",
      "35          35   HMG036          36           Tumor   \n",
      "36          36   HMG037          37           Tumor   \n",
      "37          37   HMG038          38           Tumor   \n",
      "38          38   HMG039          39           Tumor   \n",
      "39          39   HMG040          40           Tumor   \n",
      "\n",
      "                                            Pathology        Age Sex  \\\n",
      "0                                        Glioblastoma   9.000000   M   \n",
      "1                        Atypical rhaboid brain tumor   1.000000   F   \n",
      "2                                   High grade glioma   4.000000   M   \n",
      "3                        Cortical dysplasia, grade IA   0.660000   F   \n",
      "4                                  Negative pathology   7.000000   F   \n",
      "5              Cerebellar pylotic astrocytoma grade I   3.000000   F   \n",
      "6   Neurocytic ganglioglioma with severe cortical ...  13.000000   F   \n",
      "7      Cortical dysplasia, grade IC, gliosis/scarring  17.000000   M   \n",
      "8               Anaplastic ependymoma - WHO grade III  11.000000   M   \n",
      "9               Anaplastic ependymoma - WHO grade III  13.000000   F   \n",
      "10                                 Negative pathology  15.000000   F   \n",
      "11                       Atypical rhaboid brain tumor   1.166667   M   \n",
      "12                                   Ischemic changes  13.000000   F   \n",
      "13                 Focal cortical dysplasia, type IIB   1.000000   F   \n",
      "14  Dysembryoblastic neural epithelial tumor, WHO ...  11.000000   F   \n",
      "15     Pilocytic astrocytoma with anaplastic features  13.000000   F   \n",
      "16                          Neuronal loss and gliosis   2.000000   F   \n",
      "17                Anaplastic medulloblastoma - WHO IV   5.000000   M   \n",
      "18                                 Negative pathology   1.666667   M   \n",
      "19                              Glioblastoma - WHO IV   5.000000   M   \n",
      "20                 Focal cortical dysplasia, type IIA   9.000000   F   \n",
      "21                                      Pineoblastoma  12.000000   F   \n",
      "22                       Posterior pial angiomatosis    4.000000   M   \n",
      "23         Juvenile Pilocytic Atrocytoma, WHO grade 1   8.000000   M   \n",
      "24         Juvenile Pilocytic Atrocytoma, WHO grade 1   5.000000   M   \n",
      "25  Dysembryoplastic neuroepithelial tumor, WHO gr...  16.000000   M   \n",
      "26                         Embryonic choriocarcinoma    3.000000   M   \n",
      "27                                         Ependymoma  12.000000   M   \n",
      "28                                           Aneurysm  16.000000   F   \n",
      "29                                    Medullablastoma   6.000000   M   \n",
      "30                  Focal cortical dysplasia type Iib   2.000000   M   \n",
      "31                                  High grade glioma   0.416667   M   \n",
      "32                              Vestibular Schwannoma  15.000000   F   \n",
      "33  Medullablastoma - WHO grade IV, anaplastic lar...  10.000000   M   \n",
      "34                    Anoxic ischemic encephalopathy    0.416667   M   \n",
      "35                Pilocytic astrocytome - WHO grade I  13.000000   F   \n",
      "36                              Glioma - WHO grade II  18.000000   M   \n",
      "37              Anaplastic ependymoma - WHO grade III   3.000000   F   \n",
      "38                                            Pending  11.000000   M   \n",
      "39                                            Pending  13.000000   M   \n",
      "\n",
      "    Ethnicity Date (mm-dd-yy)  \\\n",
      "0           C         3/11/15   \n",
      "1           C         3/25/15   \n",
      "2         His          4/6/15   \n",
      "3           C         4/20/15   \n",
      "4         His         5/26/15   \n",
      "5           C         5/28/15   \n",
      "6           C         5/28/15   \n",
      "7         His         6/22/15   \n",
      "8           C         7/20/15   \n",
      "9         His         8/19/15   \n",
      "10        His         8/27/15   \n",
      "11        His        11/25/15   \n",
      "12     Asian         12/14/15   \n",
      "13        His         1/12/16   \n",
      "14          C         1/19/16   \n",
      "15        His         1/25/16   \n",
      "16          C          2/1/16   \n",
      "17        His         2/19/16   \n",
      "18          C         4/26/16   \n",
      "19        His         4/27/16   \n",
      "20          C          6/6/16   \n",
      "21      Asian         7/29/16   \n",
      "22        His          8/8/16   \n",
      "23        His         8/10/16   \n",
      "24          C         8/15/16   \n",
      "25        His         8/15/16   \n",
      "26      AfrAm         8/29/16   \n",
      "27          C         9/18/16   \n",
      "28    MidEast         9/19/16   \n",
      "29        His         9/19/16   \n",
      "30        His         9/27/16   \n",
      "31      Asian        10/14/16   \n",
      "32        His        11/14/16   \n",
      "33        His        11/28/16   \n",
      "34        His        11/20/16   \n",
      "35  Hawaiian         12/16/16   \n",
      "36          C        12/19/16   \n",
      "37          C        12/19/16   \n",
      "38     Indian         1/12/17   \n",
      "39          C         1/12/17   \n",
      "\n",
      "                                   Medication regimen          ...           \\\n",
      "0                                                None          ...            \n",
      "1                      Steroids, Keppra/Phenobarbitol          ...            \n",
      "2                     Temozolomide, Bevaclizumab, XRT          ...            \n",
      "3         Klonopin, Keppra, Trileptal, Phenobarbital           ...            \n",
      "4                              Keppra, Depakote, Onfi          ...            \n",
      "5                                           Steroids           ...            \n",
      "6                                           Phenytoin          ...            \n",
      "7                          Klonopin, Lamictal, Keppra          ...            \n",
      "8                                   Radiation therapy          ...            \n",
      "9                                                None          ...            \n",
      "10                                          Trileptal          ...            \n",
      "11  Methotrexate, Vincristine, Cisplatin, Cyclopho...          ...            \n",
      "12                        Topamax, Vimpat, Zonisamide          ...            \n",
      "13                      Vimpat, Phenobarbital, Keppra          ...            \n",
      "14                                            Keppra           ...            \n",
      "15                                               None          ...            \n",
      "16                                  Clobazam, Topamax          ...            \n",
      "17                                           Decadron          ...            \n",
      "18                                             Keppra          ...            \n",
      "19  Temozolomide, Bevaclizumab, XRT, Pablociclib, ...          ...            \n",
      "20                                    Topamax, Vimpat          ...            \n",
      "21                                           Decadron          ...            \n",
      "22                                     Keppra, Banzal          ...            \n",
      "23                                           Decadron          ...            \n",
      "24                                           Decadron          ...            \n",
      "25                                       Lamotrigine           ...            \n",
      "26                                           Decadron          ...            \n",
      "27                                        Keppra, XRT          ...            \n",
      "28                                               None          ...            \n",
      "29                                           Decadron          ...            \n",
      "30                            Onfi, Keppra, Trileptal          ...            \n",
      "31                                           Decadron          ...            \n",
      "32                                           Decadron          ...            \n",
      "33                                           Decadron          ...            \n",
      "34                                              None           ...            \n",
      "35                                           Decadron          ...            \n",
      "36                                              None           ...            \n",
      "37                                   Decadron, Keppra          ...            \n",
      "38                                             Keppra          ...            \n",
      "39                                             Keppra          ...            \n",
      "\n",
      "   sample2_exp_occipital sample2_exp_temporal sample2_exp_in_vitro  \\\n",
      "0                  False                False                False   \n",
      "1                  False                False                False   \n",
      "2                  False                False                 True   \n",
      "3                  False                False                False   \n",
      "4                  False                False                 True   \n",
      "5                  False                False                False   \n",
      "6                  False                False                 True   \n",
      "7                  False                False                False   \n",
      "8                  False                False                False   \n",
      "9                  False                False                False   \n",
      "10                 False                False                False   \n",
      "11                 False                False                False   \n",
      "12                 False                False                False   \n",
      "13                 False                False                False   \n",
      "14                 False                False                 True   \n",
      "15                 False                False                False   \n",
      "16                 False                 True                 True   \n",
      "17                 False                False                False   \n",
      "18                 False                False                False   \n",
      "19                 False                False                False   \n",
      "20                 False                False                 True   \n",
      "21                 False                False                False   \n",
      "22                 False                False                False   \n",
      "23                 False                False                False   \n",
      "24                 False                False                 True   \n",
      "25                 False                False                False   \n",
      "26                 False                False                False   \n",
      "27                 False                False                False   \n",
      "28                 False                False                False   \n",
      "29                 False                False                False   \n",
      "30                 False                False                False   \n",
      "31                 False                False                False   \n",
      "32                 False                False                False   \n",
      "33                 False                False                False   \n",
      "34                 False                False                False   \n",
      "35                 False                False                False   \n",
      "36                 False                False                False   \n",
      "37                 False                False                False   \n",
      "38                 False                False                False   \n",
      "39                 False                False                False   \n",
      "\n",
      "   sample2_exp_microglia                             sample3_exp_fastq_path  \\\n",
      "0                  False                                                 Na   \n",
      "1                  False                                                 Na   \n",
      "2                   True  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "3                  False                                                 Na   \n",
      "4                   True  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "5                  False                                                 Na   \n",
      "6                   True  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "7                   True  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "8                  False                                                 Na   \n",
      "9                  False                                                 Na   \n",
      "10                 False                                                 Na   \n",
      "11                 False                                                 Na   \n",
      "12                 False                                                 Na   \n",
      "13                 False                                                 Na   \n",
      "14                  True                                                 Na   \n",
      "15                 False                                                 Na   \n",
      "16                  True  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "17                 False                                                 Na   \n",
      "18                 False                                                 Na   \n",
      "19                 False                                                 Na   \n",
      "20                  True  /data/archive/16-07-27-glass/Project_David/hg3...   \n",
      "21                 False                                                 Na   \n",
      "22                 False                                                 Na   \n",
      "23                 False                                                 Na   \n",
      "24                  True  /data/archive/17-01-13-NextSeq/project_david/H...   \n",
      "25                 False                                                 Na   \n",
      "26                 False                                                 Na   \n",
      "27                 False                                                 Na   \n",
      "28                 False                                                 Na   \n",
      "29                 False                                                 Na   \n",
      "30                 False                                                 Na   \n",
      "31                 False                                                 Na   \n",
      "32                 False                                                 Na   \n",
      "33                 False                                                 Na   \n",
      "34                 False                                                 Na   \n",
      "35                 False                                                 Na   \n",
      "36                 False                                                 Na   \n",
      "37                 False                                                 Na   \n",
      "38                 False                                                 Na   \n",
      "39                 False                                                 Na   \n",
      "\n",
      "   sample3_exp_ex_vivo sample3_exp_occipital sample3_exp_temporal  \\\n",
      "0                False                 False                False   \n",
      "1                False                 False                False   \n",
      "2                False                 False                False   \n",
      "3                False                 False                False   \n",
      "4                False                 False                False   \n",
      "5                False                 False                False   \n",
      "6                False                 False                False   \n",
      "7                False                 False                False   \n",
      "8                False                 False                False   \n",
      "9                False                 False                False   \n",
      "10               False                 False                False   \n",
      "11               False                 False                False   \n",
      "12               False                 False                False   \n",
      "13               False                 False                False   \n",
      "14               False                 False                False   \n",
      "15               False                 False                False   \n",
      "16               False                 False                 True   \n",
      "17               False                 False                False   \n",
      "18               False                 False                False   \n",
      "19               False                 False                False   \n",
      "20               False                 False                False   \n",
      "21               False                 False                False   \n",
      "22               False                 False                False   \n",
      "23               False                 False                False   \n",
      "24               False                 False                False   \n",
      "25               False                 False                False   \n",
      "26               False                 False                False   \n",
      "27               False                 False                False   \n",
      "28               False                 False                False   \n",
      "29               False                 False                False   \n",
      "30               False                 False                False   \n",
      "31               False                 False                False   \n",
      "32               False                 False                False   \n",
      "33               False                 False                False   \n",
      "34               False                 False                False   \n",
      "35               False                 False                False   \n",
      "36               False                 False                False   \n",
      "37               False                 False                False   \n",
      "38               False                 False                False   \n",
      "39               False                 False                False   \n",
      "\n",
      "   sample3_exp_in_vitro sample3_exp_microglia  \n",
      "0                 False                 False  \n",
      "1                 False                 False  \n",
      "2                 False                  True  \n",
      "3                 False                 False  \n",
      "4                  True                  True  \n",
      "5                 False                 False  \n",
      "6                  True                  True  \n",
      "7                  True                  True  \n",
      "8                 False                 False  \n",
      "9                 False                 False  \n",
      "10                False                 False  \n",
      "11                False                 False  \n",
      "12                False                 False  \n",
      "13                False                 False  \n",
      "14                False                 False  \n",
      "15                False                 False  \n",
      "16                 True                  True  \n",
      "17                False                 False  \n",
      "18                False                 False  \n",
      "19                False                 False  \n",
      "20                 True                  True  \n",
      "21                False                 False  \n",
      "22                False                 False  \n",
      "23                False                 False  \n",
      "24                 True                  True  \n",
      "25                False                 False  \n",
      "26                False                 False  \n",
      "27                False                 False  \n",
      "28                False                 False  \n",
      "29                False                 False  \n",
      "30                False                 False  \n",
      "31                False                 False  \n",
      "32                False                 False  \n",
      "33                False                 False  \n",
      "34                False                 False  \n",
      "35                False                 False  \n",
      "36                False                 False  \n",
      "37                False                 False  \n",
      "38                False                 False  \n",
      "39                False                 False  \n",
      "\n",
      "[40 rows x 32 columns]\n"
     ]
    }
   ],
   "source": [
    "#######Fastq files + Annotation of the file (3th)\n",
    "\n",
    "for sample in patient_info['sample3_exp']: \n",
    "    if sample == \"Na\":\n",
    "        fastq_path.append(\"Na\")\n",
    "    elif sample: \n",
    "        fastq_path.append(human_rna_tag_fastq_mapping[human_rna_tag_fastq_mapping[\"tag_folder\"] == sample][\"fastq_path\"].iloc[0])\n",
    "    else: \n",
    "        fastq_path.append(\"Na\")\n",
    "\n",
    "patient_info['sample3_exp_fastq_path'] = fastq_path\n",
    "\n",
    "\n",
    "patient_info['sample3_exp_ex_vivo'] = [i.lower().find('ex') > -1 for i in patient_info['sample3_exp']]\n",
    "patient_info['sample3_exp_occipital'] = [i.lower().find('occipital') > -1 for i in patient_info['sample3_exp']]\n",
    "patient_info['sample3_exp_temporal'] = [i.lower().find('temporal') > -1 for i in patient_info['sample3_exp']]\n",
    "patient_info['sample3_exp_in_vitro'] = [i.lower().find('culture') > -1 for i in patient_info['sample3_exp']]\n",
    "patient_info['sample3_exp_microglia'] = [i.lower().find('microglia') > -1 for i in patient_info['sample3_exp']]\n",
    "\n",
    "\n",
    "\n",
    "#print(patient_info['sample1_exp_ex_vivo'])\n",
    "#print(patient_info.iloc[:,10:15])\n",
    "print(patient_info)\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 342,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   Unnamed: 0 Patient   Patient_nr Broad_Diagnosis  \\\n",
      "1           1   HMG002           2           Tumor   \n",
      "2           2   HMG003           3           Tumor   \n",
      "3           3   HMG004           4        Epilepsy   \n",
      "4           4   HMG005           5        Epilepsy   \n",
      "\n",
      "                      Pathology   Age Sex Ethnicity Date (mm-dd-yy)  \\\n",
      "1  Atypical rhaboid brain tumor  1.00   F         C         3/25/15   \n",
      "2             High grade glioma  4.00   M       His          4/6/15   \n",
      "3  Cortical dysplasia, grade IA  0.66   F         C         4/20/15   \n",
      "4            Negative pathology  7.00   F       His         5/26/15   \n",
      "\n",
      "                            Medication regimen  ...   sample2_exp_temporal  \\\n",
      "1               Steroids, Keppra/Phenobarbitol  ...                  False   \n",
      "2              Temozolomide, Bevaclizumab, XRT  ...                  False   \n",
      "3  Klonopin, Keppra, Trileptal, Phenobarbital   ...                  False   \n",
      "4                       Keppra, Depakote, Onfi  ...                  False   \n",
      "\n",
      "  sample2_exp_in_vitro sample2_exp_microglia  \\\n",
      "1                False                 False   \n",
      "2                 True                  True   \n",
      "3                False                 False   \n",
      "4                 True                  True   \n",
      "\n",
      "                              sample3_exp_fastq_path sample3_exp_ex_vivo  \\\n",
      "1                                                 Na               False   \n",
      "2  /data/archive/15-06-17-glass/Project_David/hg3...               False   \n",
      "3                                                 Na               False   \n",
      "4  /data/archive/15-06-17-glass/Project_David/hg3...               False   \n",
      "\n",
      "  sample3_exp_occipital sample3_exp_temporal sample3_exp_in_vitro  \\\n",
      "1                 False                False                False   \n",
      "2                 False                False                False   \n",
      "3                 False                False                False   \n",
      "4                 False                False                 True   \n",
      "\n",
      "  sample3_exp_microglia    WGS  \n",
      "1                 False  False  \n",
      "2                  True  False  \n",
      "3                 False   True  \n",
      "4                  True   True  \n",
      "\n",
      "[4 rows x 33 columns]\n"
     ]
    }
   ],
   "source": [
    "######## Add cell type - tissue information : WB, MG-exvivo, MG_invitro \n",
    "## Add WGS : 15,17,4,5,7,8 \n",
    "import re                   \n",
    "samples = [15,17,4,5,7,8]\n",
    "WGS = []\n",
    "patient_info['WGS'] = False\n",
    "WGS2 = patient_info['WGS'][:]\n",
    "\n",
    "for sample in samples: \n",
    "    WGS = []\n",
    "    for patient in patient_info[\"Patient_nr\"]:\n",
    "    \n",
    "        if patient == sample:\n",
    "            WGS.append(True)\n",
    "        else: \n",
    "            WGS.append(False)\n",
    "    \n",
    "    WGS2 = WGS + WGS2\n",
    "\n",
    "#print(WGS2)\n",
    "patient_info['WGS'] = WGS2\n",
    "\n",
    "print(patient_info[1:5])\n",
    "\n",
    "\n",
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "patient_info.to_csv(os.path.join(OUTPUT_PATH, 'Master_matrix_RNA_template_20_feb.csv'))\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'human_pu1_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_pu_sample_info.csv', 'human_h3k27ac_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_h3k27ac_sample_info.csv', 'sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_sample_info.csv', 'patient_info.csv': '/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_patient_info.csv', 'human_atac_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_atac_sample_info.csv', 'human_input_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_input_sample_info.csv', 'human_h3k4me2_sample_info.csv': '/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_h3k4me2_sample_info.csv'}\n"
     ]
    }
   ],
   "source": [
    "###########Chip \n",
    "######################## \n",
    "import pandas \n",
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'    \n",
    "date = \"Feb_20_2017_\"\n",
    "files ={\"human_atac_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_atac_sample_info.csv\", \\\n",
    "        \"patient_info.csv\" :\"/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_patient_info.csv\", \\\n",
    "        \"sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/patient_sample/Feb_20_2017_sample_info.csv\", \\\n",
    "        \"human_h3k4me2_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_h3k4me2_sample_info.csv\", \\\n",
    "        \"human_h3k27ac_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_h3k27ac_sample_info.csv\", \\\n",
    "        \"human_input_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_input_sample_info.csv\", \\\n",
    "        \"human_pu1_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/epigenome/Feb_20_2017_human_pu_sample_info.csv\", \\\n",
    "       }\n",
    "      \n",
    "###\n",
    "human_atac_sample_info = pandas.read_csv(files['human_atac_sample_info.csv'])\n",
    "patient_info = pandas.read_csv(files['patient_info.csv'])\n",
    "sample_info = pandas.read_csv(files['sample_info.csv'])\n",
    "human_h3k4me2_sample_info = pandas.read_csv(files['human_h3k4me2_sample_info.csv'])\n",
    "human_h3k27ac_sample_info = pandas.read_csv(files['human_h3k27ac_sample_info.csv'])\n",
    "human_input_sample_info = pandas.read_csv(files['human_input_sample_info.csv'])\n",
    "human_pu1_sample_info = pandas.read_csv(files['human_pu1_sample_info.csv'])\n",
    "   \n",
    "print(files)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "------------------human_atac_sample_info-------------------\n",
      "   Unnamed: 0                                       Unnamed: 0.1   use  \\\n",
      "0           0  Human_Microglia_ATAC_P11_150827_Grey_DGO_15_12_07  True   \n",
      "1           1  Human_Microglia_ATAC_P8_Mix_2_150622_DGO_15_12_11  True   \n",
      "2           2  Human_Microglia_Sorted_ATAC_P21_Culture_7d_IL3...  True   \n",
      "3           3  Human_Microglia_Tumor_ATAC_P24_160810_DGO_16_0...  True   \n",
      "\n",
      "  primary  Patient in_vitro  tumor assay  Broad_Diagnosis  Pathology  Age  \\\n",
      "0    True       11    False  False  atac              NaN        NaN  NaN   \n",
      "1   False        8    False  False  atac              NaN        NaN  NaN   \n",
      "2    True       21     True  False  atac              NaN        NaN  NaN   \n",
      "3   False       24    False   True  atac              NaN        NaN  NaN   \n",
      "\n",
      "   Sex  Ethnicity  Date (mm-dd-yy)  Medication regimen  \n",
      "0  NaN        NaN              NaN                 NaN  \n",
      "1  NaN        NaN              NaN                 NaN  \n",
      "2  NaN        NaN              NaN                 NaN  \n",
      "3  NaN        NaN              NaN                 NaN  \n",
      "------------------human_H3K4me2_sample_info----------------\n",
      "   Unnamed: 0 Unnamed: 0.1   use primary  Patient    assay in_vitro  tumor  \\\n",
      "0           0   H3K4me2_P5  True    True        0  h3k4me2    False  False   \n",
      "1           1   H3K4me2_P7  True    True        0  h3k4me2    False  False   \n",
      "2           2   H3K4me2_P4  True    True        0  h3k4me2    False  False   \n",
      "3           3   H3K4me2_P2  True    True        0  h3k4me2    False  False   \n",
      "\n",
      "   Broad_Diagnosis  Pathology  Age  Sex  Ethnicity  Date (mm-dd-yy)  \\\n",
      "0              NaN        NaN  NaN  NaN        NaN              NaN   \n",
      "1              NaN        NaN  NaN  NaN        NaN              NaN   \n",
      "2              NaN        NaN  NaN  NaN        NaN              NaN   \n",
      "3              NaN        NaN  NaN  NaN        NaN              NaN   \n",
      "\n",
      "   Medication regimen  \n",
      "0                 NaN  \n",
      "1                 NaN  \n",
      "2                 NaN  \n",
      "3                 NaN  \n",
      "------------------human_H3K27Ac_sample_info----------------\n",
      "   Unnamed: 0                                       Unnamed: 0.1   use  \\\n",
      "0           0  Human_Microglia_ChIP_H3K27ac_P15_160119_DGO_16...  True   \n",
      "1           1  Human_Microglia_Sorted_ChIP_H3K27ac_P21_Cultur...  True   \n",
      "2           2  Human_Microglia_Sorted_ChIP_H3K27ac_P21_Cultur...  True   \n",
      "3           3  Human_Microglia_ChIP_H3K27ac_P4_150420_DGO_15_...  True   \n",
      "\n",
      "  primary  Patient    assay in_vitro  tumor  Broad_Diagnosis  Pathology  Age  \\\n",
      "0    True       15  h3k27ac    False  False              NaN        NaN  NaN   \n",
      "1   False       21  h3k27ac     True  False              NaN        NaN  NaN   \n",
      "2    True       21  h3k27ac     True  False              NaN        NaN  NaN   \n",
      "3    True        4  h3k27ac    False  False              NaN        NaN  NaN   \n",
      "\n",
      "   Sex  Ethnicity  Date (mm-dd-yy)  Medication regimen  \n",
      "0  NaN        NaN              NaN                 NaN  \n",
      "1  NaN        NaN              NaN                 NaN  \n",
      "2  NaN        NaN              NaN                 NaN  \n",
      "3  NaN        NaN              NaN                 NaN  \n",
      "------------------human_input_sample_info------------------\n",
      "   Unnamed: 0                                       Unnamed: 0.1   use  \\\n",
      "0           0  Human_Microglia_ChIP_Input_P17_Culture_IL34_7d...  True   \n",
      "1           1                                           Input_P4  True   \n",
      "2           2  Human_Microglia_Sorted_ChIP_Input_P21_Culture_...  True   \n",
      "3           3                                           Input_P7  True   \n",
      "\n",
      "  primary in_vitro  tumor  assay  \n",
      "0    True     True  False  input  \n",
      "1    True    False  False  input  \n",
      "2    True     True  False  input  \n",
      "3    True    False  False  input  \n",
      "------------------human_pu1_sample_info--------------------\n",
      "   Unnamed: 0                                    Unnamed: 0.1   use primary  \\\n",
      "0           0                                          PU1_P7  True    True   \n",
      "1           1                                          PU1_P4  True    True   \n",
      "2           2  Human_Microglia_ChIP_PU1_P17_160201_DGO_160802  True    True   \n",
      "3           3  Human_Microglia_ChIP_PU1_P15_160119_DGO_160802  True    True   \n",
      "\n",
      "  in_vitro  tumor  \n",
      "0    False  False  \n",
      "1    False  False  \n",
      "2    False  False  \n",
      "3    False  False  \n"
     ]
    }
   ],
   "source": [
    "print(\"------------------human_atac_sample_info-------------------\")\n",
    "print(human_atac_sample_info[0:4])\n",
    "\n",
    "print(\"------------------human_H3K4me2_sample_info----------------\")\n",
    "print(human_h3k4me2_sample_info[0:4])\n",
    "                                \n",
    "print(\"------------------human_H3K27Ac_sample_info----------------\")\n",
    "print(human_h3k27ac_sample_info[0:4])\n",
    "                                \n",
    "print(\"------------------human_input_sample_info------------------\")\n",
    "print(human_input_sample_info[0:4])\n",
    "                              \n",
    "print(\"------------------human_pu1_sample_info--------------------\")\n",
    "print(human_pu1_sample_info[0:4])\n",
    "   \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
