{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import re\n",
    "import math, random\n",
    "import numpy,scipy,pandas\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn\n",
    "import itertools\n",
    "import subprocess\n",
    "\n",
    "# from pgtools import myplots\n",
    "from pgtools import toolbox, myplots\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('/home/dskola/workspace/expression_modeling/')\n",
    "import rnatools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz'], ['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz'], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz'], [], [], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz'], ['/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz'], ['/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz'], ['/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H1_DGO_15_10_30_S64_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E3_DGO_15_10_30_S69_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E1_DGO_15_10_30_S67_L006_R1_001.fastq'], ['/data/archive/15-12-27-glass/Project_David/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq'], ['/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04_S51_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04_S54_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_lL34_TGFBR1antagonist_A8301_1uM_24h_rep1_160201_DGO_16_04_04_S47_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_rep1_160201_DGO_16_04_04_S46_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001_2.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_rep1_160201_DGO_16_04_04_S45_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04_S55_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04_S50_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFB1_160201_DGO_16_04_04_S56_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001.fastq.gz'], ['/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_Salk_151209_3_2_DGO_16_05_02_GTGAAA_S7_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_IL34_Salk_1512_09_5_1_DGO_16_05_02_GTTTCG_S9_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_IL34_Salk_1512093_1_DGO_16_05_02_GTCCGC_S6_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_GMCSF_StemPro_IL34_Salk_151209_4_1_DGO_16_05_02_GTGGCC_S8_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz'], ['/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz'], ['/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq'], ['/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq'], ['/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz']]\n"
     ]
    }
   ],
   "source": [
    "import glob\n",
    "import os\n",
    "    \n",
    "fastq_dir = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/fastq_directories.csv')\n",
    "#print(fastq_dir[\"Dirs\"])\n",
    "\n",
    "files = []\n",
    "for dir in fastq_dir[\"Dirs\"]:\n",
    "    #os.listdir(dir + \"*.fastq\" )\n",
    "    dir_fastq = glob.glob(dir +\"*.fastq\")\n",
    "    dir_fastq_gz = glob.glob(dir +\"*.fastq.gz\")\n",
    "    files.append(dir_fastq or dir_fastq_gz)\n",
    "print(files)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz', '/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz', '/data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz', '/data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H1_DGO_15_10_30_S64_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E3_DGO_15_10_30_S69_L006_R1_001.fastq', '/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E1_DGO_15_10_30_S67_L006_R1_001.fastq', '/data/archive/15-12-27-glass/Project_David/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7d_IL34_160119_DGO_16_04_04_S51_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Temporal_160201_DGO_16_04_04_S54_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_lL34_TGFBR1antagonist_A8301_1uM_24h_rep1_160201_DGO_16_04_04_S47_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_rep1_160201_DGO_16_04_04_S46_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001_2.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_rep1_160201_DGO_16_04_04_S45_L003_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_160201_DGO_16_04_04_S55_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04_S50_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFB1_160201_DGO_16_04_04_S56_L004_R1_001.fastq.gz', '/data/archive/16-04-15-glass/Project_David/hg38RNA160415/Human_Microglia_RNA_PolyA_P17_Frontal_160201_DGO_16_04_04_S53_L004_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_Salk_151209_3_2_DGO_16_05_02_GTGAAA_S7_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_IL34_Salk_1512_09_5_1_DGO_16_05_02_GTTTCG_S9_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_StemPro_IL34_Salk_1512093_1_DGO_16_05_02_GTCCGC_S6_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_EC11_GMCSF_StemPro_IL34_Salk_151209_4_1_DGO_16_05_02_GTGGCC_S8_L001_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz', '/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz', '/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq', '/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz', '/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz']\n"
     ]
    }
   ],
   "source": [
    "### One list with all fastq-files  \n",
    "fastqs = []\n",
    "for file in files:\n",
    "    for fastq in file:    \n",
    "        fastqs.append(fastq)\n",
    "\n",
    "print(fastqs)\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: [/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_TGFB1_DGO_15_06_15_ATGAGC_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15_CAACTA_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_DGO_15_06_15_CACCGG_L008_R1_005.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_001.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_002.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_003.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_004.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_007.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_006.fastq.gz, /data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15/Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_TGFB1_DGO_15_06_15_CACGAT_L008_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_007.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21/Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_007.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_15_08_21_GAGTGG_L003_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_002.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_003.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_005.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_008.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_001.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_006.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_004.fastq.gz, /data/archive/15-09-14-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21/Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB1_DGO_15_08_21_GGTAGC_L003_R1_007.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_001.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_003.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_005.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_006.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_004.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_002.fastq.gz, /data/archive/15-10-13-glass/Project_David/RNAhg38/Sample_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23/Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1_007.fastq.gz, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H2_DGO_15_10_30_S65_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_E2_DGO_15_10_30_S68_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq, /data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_IPS_Microglia_RNA_PolyA_H3_DGO_15_10_30_S66_L006_R1_001.fastq, ...]\n",
      "\n",
      "[149 rows x 0 columns]\n"
     ]
    }
   ],
   "source": [
    "folders = pandas.DataFrame([], index=fastqs)\n",
    "folders.index.name='fastq'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "print(folders)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{0, 2, 3, 4, 5, 6, 7, 8, 10, 11, 12, 14, 15, 16, 17, 18, 19, 21, 23, 24, 26, 27, 29, 31}\n",
      "fastq\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_001.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_002.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_003.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_004.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_005.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_006.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_007.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15/Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1_008.fastq.gz                         2\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_001.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_002.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_003.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_004.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_005.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_006.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15/Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1_007.fastq.gz                         3\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_001.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_002.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_003.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_004.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_005.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R1_006.fastq.gz                         5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15/cat_Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15_GGTAGC_L008_R.fastq.gz                          5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_001.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_002.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_003.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_004.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_005.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_006.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_DGO_15_06_15_ATTCCT_L008_R1_007.fastq.gz     5\n",
      "/data/archive/15-06-17-glass/Project_David/hg38-RNA/Sample_Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15/Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_DGO_15_06_15_ACTGAT_L008_R1_001.fastq.gz     5\n",
      "                                                                                                                                                                                                                ..\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_IPS_derived_microglia_RNA_PolyA_H1_StemPro_Salk_1512_09_5_1_DGO_16_05_02_CGTACG_S10_L001_R1_001.fastq.gz                                          0\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_CD64High_RNA_PolyA_P15_160119_DGO_16_05_02_ATGTCA_S20_L002_R1_001.fastq.gz                                                             15\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_24h_160201_DGO_16_05_02_CGTACG_S19_L002_R1_001.fastq.gz       17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_TGFBR1antagonist_A8301_1uM_6h_160201_DGO_16_05_02_GTTTCG_S18_L002_R1_001.fastq.gz        17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_Culture_7d_IL34_Vehicle_DMSO_160201_DGO_16_05_02_GTGGCC_S17_L002_R1_001.fastq.gz                         17\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz                                                                              4\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC_S13_L002_R1_001.fastq.gz                                                                              5\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT_S14_L002_R1_001.fastq.gz                                                                              7\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Occipital_DGO_16_05_02_ATGAGC_S15_L002_R1_001.fastq.gz                                                                    8\n",
      "/data/archive/16-05-06-glass/Project_David/hg38RNA160506/Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_05_02_ATTCCT_S16_L002_R1_001.fastq.gz                                                                     8\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_1d_JoS_16_04_27_CGTACG_S90_L006_R1_001.fastq.gz                                                                 14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_JoS_16_04_27_GAGTGG_S91_L006_R1_001.fastq.gz                                                                 14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_LPS_JoS_16_04_27_GGTAGC_S92_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_7d_TGF_JoS_16_04_27_ACTGAT_S93_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P14_microglia_RNA_MG_exvivo_JoS_16_04_27_GTTTCG_S89_L006_R1_001.fastq.gz                                                             14\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_7d_JoS_16_04_27_CCGTCC_S88_L006_R1_001.fastq.gz                                                                 18\n",
      "/data/archive/16-05-06-glass/Project_Johannes/HuMicrogliaP14and18_2016-5-6/P18_microglia_RNA_MG_exvivo_JoS_16_04_27_ATGTCA_S87_L006_R1_001.fastq.gz                                                             18\n",
      "/data/archive/16-05-26-glass/Project_David/hg38RNA160526/Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_160427_DGO_16_05_12_S90_L007_R1_001.fastq                                                                   3\n",
      "/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq                                                                19\n",
      "/data/archive/16-07-27-glass/Project_David/hg38RNAb/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq                                                                  21\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P23-Temporal-160808-DGO-16-10-28_S1_R1_001.fastq.gz                                                                             23\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P24-Tumor-160810-DGO-16-10-28_S3_R1_001.fastq.gz                                                                                24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz                                                                                26\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz                                                                                27\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz                                                                              29\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-RNA-PolyA-P31-160927-DGO-16-10-28_S9_R1_001.fastq.gz                                                                                      31\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P23-Culture-6h-IL34-160808-DGO-16-10-28_S2_R1_001.fastq.gz                                                               23\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-24h-IL34-160810-DGO-16-10-28_S5_R1_001.fastq.gz                                                              24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P24-Culture-6h-IL34-160810-DGO-16-10-28_S4_R1_001.fastq.gz                                                               24\n",
      "/data/archive/16-10-28-NextSeq/project_david/hg38/rna/Human-Microglia-Sorted-RNA-PolyA-P31-Culture-24h-IL34-160927-DGO-16-10-28_S10_R1_001.fastq.gz                                                             31\n",
      "Name: patient, dtype: int64\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/gpfs/data01/glasslab/home/iholtman/anaconda3/lib/python3.5/site-packages/ipykernel/__main__.py:9: DeprecationWarning: bad escape \\P\n"
     ]
    }
   ],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\P(?P<patient>[0-9]{1,2})\\_\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "    \n",
    "folders['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patient\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "\n",
    "#folders[\"fastq_folder\"] = folders[\"fastq_path\"]\n",
    "print(set(folders[\"patient\"]))\n",
    "print(folders[\"patient\"])\n",
    "\n",
    "#OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "#folders.to_csv(os.path.join(OUTPUT_PATH, 'ALL_fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>cat</th>\n",
       "      <th>patient</th>\n",
       "      <th>patientz</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>fastq</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10_30_S70_L006_R1_001.fastq</th>\n",
       "      <td>False</td>\n",
       "      <td>11</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>/data/archive/15-11-10-glass/Project_David/RNA-hg38-151110/Human_Microglia_RNA_PolyA_P11_White_150827_DGO_15_10_30_S72_L006_R1_001.fastq</th>\n",
       "      <td>False</td>\n",
       "      <td>11</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                      cat  patient  patientz\n",
       "fastq                                                                       \n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11         0\n",
       "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11         0"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "folders[folders[\"patient\"] == 11]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['in_vitro'] = [i.lower().find('in_vitro') > -1 for i in folders.index]\n",
    "folders['tgfb'] = [i.lower().find('tgf') > -1 for i in folders.index]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#fastq_mapping = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/human_rna_tag_fastq_mapping.csv', index_col=0)\n",
    "#print(fastq_mapping[\"fastq_path\"])\n",
    "\n",
    "#OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#tag_mapping = pandas.read_csv('/home/iholtman/Human_natural_genetic_variation/human_rna_tag_fastq_mapping.csv', index_col=0).fastq_path.to_dict()\n",
    "#print(tag_mapping)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#folders = pandas.DataFrame([], index=tag_mapping)\n",
    "#folders.index.name='Sample'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "#print(folders)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Annotate with patient info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "         Broad_Diagnosis                                          Pathology  \\\n",
      "Patient                                                                       \n",
      "1                  Tumor                                       Glioblastoma   \n",
      "2                  Tumor                       Atypical rhaboid brain tumor   \n",
      "3                  Tumor                                  High grade glioma   \n",
      "4               Epilepsy            Epilepsy - cortical dysplasia, grade IA   \n",
      "5               Epilepsy                      Epilepsy - negative pathology   \n",
      "6                  Tumor             Cerebellar pylotic astrocytoma grade I   \n",
      "7               Epilepsy  Epilepsy - neurocytic ganglioglioma with sever...   \n",
      "8               Epilepsy  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "9                  Tumor              Anaplastic ependymoma - WHO grade III   \n",
      "10                 Tumor              Anaplastic ependymoma - WHO grade III   \n",
      "11              Epilepsy                      Epilepsy - negative pathology   \n",
      "12                 Tumor                       Atypical rhaboid brain tumor   \n",
      "13              Epilepsy                        Epilepsy - ischemic changes   \n",
      "14              Epilepsy      Epilepsy - Focal cortical dysplasia, type IIB   \n",
      "15                 Tumor  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "16                 Tumor     Pilocytic astrocytoma with anaplastic features   \n",
      "17              Epilepsy               Epilepsy - neuronal loss and gliosis   \n",
      "18                 Tumor                Anaplastic medulloblastoma - WHO IV   \n",
      "19              Epilepsy                       Epilepsy - pathology pending   \n",
      "20                 Tumor                              Glioblastoma - WHO IV   \n",
      "21              Epilepsy      Epilepsy - focal cortical dysplasia, type IIA   \n",
      "22                 Tumor                                      Pineoblastoma   \n",
      "23              Epilepsy            Epilepsy - posterior pial angiomatosis    \n",
      "24                 Tumor         Juvenile Pilocytic Atrocytoma, WHO grade 1   \n",
      "25                 Tumor         Juvenile Pilocytic Atrocytoma, WHO grade 1   \n",
      "26                 Tumor  Dysembryoplastic neuroepithelial tumor, WHO gr...   \n",
      "27                 Tumor                         Embryonic choriocarcinoma    \n",
      "28                 Tumor                                         Ependymoma   \n",
      "29              Aneurysm                                           Aneurysm   \n",
      "30                 Tumor                                    Medullablastoma   \n",
      "31                   NaN                                                NaN   \n",
      "\n",
      "                Age  Sex Ethnicity Date (mm-dd-yy)  \\\n",
      "Patient                                              \n",
      "1          9.000000    M         C         3/11/15   \n",
      "2          1.000000    F         C         3/25/15   \n",
      "3          4.000000    M       His          4/6/15   \n",
      "4          0.660000    F         C         4/20/15   \n",
      "5          7.000000    F       His         5/26/15   \n",
      "6          3.000000    F         C         5/28/15   \n",
      "7         13.000000    F         C         5/28/15   \n",
      "8         17.000000    M       His         6/22/15   \n",
      "9         11.000000    M         C         7/20/15   \n",
      "10        13.000000    F       His         8/19/15   \n",
      "11        15.000000    F       His         8/27/15   \n",
      "12         1.166667    M       His        11/25/15   \n",
      "13        13.000000    F    Asian         12/14/15   \n",
      "14         1.000000    F       His         1/12/16   \n",
      "15        11.000000    F         C         1/19/16   \n",
      "16        13.000000    F       His         1/25/16   \n",
      "17         2.000000    F         C          2/1/16   \n",
      "18         5.000000    M       His         2/19/16   \n",
      "19         1.666667    M         C         4/26/16   \n",
      "20         5.000000    M       His         4/27/16   \n",
      "21         9.000000    F         C          6/6/16   \n",
      "22        12.000000    F     Asian         7/29/16   \n",
      "23         4.000000    M       His          8/8/16   \n",
      "24         8.000000    M       His         8/10/16   \n",
      "25         5.000000    M         C         8/15/16   \n",
      "26        16.000000    M       His         8/15/16   \n",
      "27         3.000000    M     AfrAm         8/29/16   \n",
      "28              NaN  NaN       NaN         9/18/16   \n",
      "29        16.000000    F       NaN         9/19/16   \n",
      "30              NaN  NaN       NaN         9/19/16   \n",
      "31              NaN  NaN       NaN         9/27/16   \n",
      "\n",
      "                                         Medication regimen  \n",
      "Patient                                                      \n",
      "1                                                      None  \n",
      "2                            Steroids, Keppra/Phenobarbitol  \n",
      "3                           Temozolomide, Bevaclizumab, XRT  \n",
      "4               Klonopin, Keppra, Trileptal, Phenobarbital   \n",
      "5                                    Keppra, Depakote, Onfi  \n",
      "6                                                 Steroids   \n",
      "7                                                 Phenytoin  \n",
      "8                                Klonopin, Lamictal, Keppra  \n",
      "9                                         Radiation therapy  \n",
      "10                                                     None  \n",
      "11                                                Trileptal  \n",
      "12        Methotrexate, Vincristine, Cisplatin, Cyclopho...  \n",
      "13                              Topamax, Vimpat, Zonisamide  \n",
      "14                            Vimpat, Phenobarbital, Keppra  \n",
      "15                                                  Keppra   \n",
      "16                                                     None  \n",
      "17                                        Clobazam, Topamax  \n",
      "18                                                 Decadron  \n",
      "19                                                   Keppra  \n",
      "20        Temozolomide, Bevaclizumab, XRT, Pablociclib, ...  \n",
      "21                                          Topamax, Vimpat  \n",
      "22                                                 Decadron  \n",
      "23                                           Keppra, Banzal  \n",
      "24                                                 Decadron  \n",
      "25                                                 Decadron  \n",
      "26                                             Lamotrigine   \n",
      "27                                                 Decadron  \n",
      "28                                                      NaN  \n",
      "29                                                      NaN  \n",
      "30                                                      NaN  \n",
      "31                                                      NaN  \n"
     ]
    }
   ],
   "source": [
    "patient_table = pandas.read_csv('/gpfs/data01/glasslab/home/iholtman/Human_microglia_python_tutorial/patient_info.csv', index_col=0)\n",
    "print(patient_table)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                                      cat  patient  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        2   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        3   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...  False        3   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   True        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...  False        5   \n",
      "...                                                   ...      ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        8   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        8   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  False       10   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...   True       10   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  False       11   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False       17   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  False        4   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...  False       19   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...  False       21   \n",
      "\n",
      "                                                   fastq_folder  \\\n",
      "fastq                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          NaN   \n",
      "...                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          NaN   \n",
      "\n",
      "                                                   Broad_Diagnosis  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...        Epilepsy   \n",
      "...                                                            ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...           Tumor   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...        Epilepsy   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...           Tumor   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...        Epilepsy   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...        Epilepsy   \n",
      "\n",
      "                                                                                            Pathology  \\\n",
      "fastq                                                                                                   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Atypical rhaboid brain tumor   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                                  High grade glioma   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                      Epilepsy - negative pathology   \n",
      "...                                                                                               ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Epilepsy - cortical dysplasia, grade IC, glios...   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...              Anaplastic ependymoma - WHO grade III   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                      Epilepsy - negative pathology   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                      Epilepsy - negative pathology   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Dysembryoblastic neural epithelial tumor, WHO ...   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...     Pilocytic astrocytoma with anaplastic features   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...               Epilepsy - neuronal loss and gliosis   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...            Epilepsy - cortical dysplasia, grade IA   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       Epilepsy - pathology pending   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...      Epilepsy - focal cortical dysplasia, type IIA   \n",
      "\n",
      "                                                          Age Sex Ethnicity  \\\n",
      "fastq                                                                         \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   1.000000   F         C   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...   4.000000   M       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...   7.000000   F       His   \n",
      "...                                                       ...  ..       ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  17.000000   M       His   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  17.000000   M       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...  13.000000   F       His   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  15.000000   F       His   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...  15.000000   F       His   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  11.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...  13.000000   F       His   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   2.000000   F         C   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...   0.660000   F         C   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...   1.666667   M         C   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...   9.000000   F         C   \n",
      "\n",
      "                                                   Date (mm-dd-yy)  \\\n",
      "fastq                                                                \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         3/25/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...          4/6/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         5/26/15   \n",
      "...                                                            ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         6/22/15   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         6/22/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         8/19/15   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         8/27/15   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         8/27/15   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         1/19/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         1/25/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...          2/1/16   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         4/20/15   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         4/26/16   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...          6/6/16   \n",
      "\n",
      "                                                                             Medication regimen  \\\n",
      "fastq                                                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...               Steroids, Keppra/Phenobarbitol   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...              Temozolomide, Bevaclizumab, XRT   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       Keppra, Depakote, Onfi   \n",
      "...                                                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                   Klonopin, Lamictal, Keppra   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                   Klonopin, Lamictal, Keppra   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                                         None   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                                    Trileptal   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                                    Trileptal   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                                      Keppra    \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                                         None   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                            Clobazam, Topamax   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...  Klonopin, Keppra, Trileptal, Phenobarbital    \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                                       Keppra   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                              Topamax, Vimpat   \n",
      "\n",
      "                                                   sample_id Region  \\\n",
      "fastq                                                                 \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...       NaN    NaN   \n",
      "...                                                      ...    ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...       NaN    NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...       NaN    NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...       NaN    NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...       NaN    NaN   \n",
      "\n",
      "                                                   Tissue resected (in gram)  \\\n",
      "fastq                                                                          \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                       NaN   \n",
      "...                                                                      ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                       NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                       NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                       NaN   \n",
      "\n",
      "                                                    Cells isolated (in million)  \\\n",
      "fastq                                                                             \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...                          NaN   \n",
      "...                                                                         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...                          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                          NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                          NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...                          NaN   \n",
      "\n",
      "                                                   Region Full tissue_type  \\\n",
      "fastq                                                                        \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN         NaN   \n",
      "...                                                        ...         ...   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN         NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN         NaN   \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN         NaN   \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN         NaN   \n",
      "\n",
      "                                                   sample_date  patient_age  \n",
      "fastq                                                                        \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-26-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-06-17-glass/Project_David/hg38...         NaN          NaN  \n",
      "...                                                        ...          ...  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-10-13-glass/Project_David/RNAh...         NaN          NaN  \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN          NaN  \n",
      "/data/archive/15-11-10-glass/Project_David/RNA-...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-04-15-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-05-06-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN          NaN  \n",
      "/data/archive/16-07-27-glass/Project_David/hg38...         NaN          NaN  \n",
      "\n",
      "[124 rows x 18 columns]\n"
     ]
    }
   ],
   "source": [
    "annotated_samples = pandas.merge(folders, patient_table, left_on='patient', right_index=True)\n",
    "\n",
    "sample_info = pandas.read_csv('/home/dskola/human/sample_info.csv')\n",
    "sample_info.drop([8,17,21], axis=0, inplace=True)\n",
    "i = annotated_samples.index\n",
    "annotated_samples = annotated_samples.merge(sample_info, how='left', left_on='patient', right_on='Patient')\n",
    "annotated_samples.drop('Patient', axis=1, inplace=True)\n",
    "annotated_samples.index = i\n",
    "print(annotated_samples)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#annotated_samples = pandas.merge(folders, patient_table, left_on='patient', right_index=True)\n",
    "#print(annotated_samples)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "annotated_samples.to_csv(os.path.join(OUTPUT_PATH, 'fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "annotated_samples.to_csv(os.path.join(OUTPUT_PATH, 'ALL_fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Concetated FASTQ-files processing according to GATK best practices "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Empty DataFrame\n",
      "Columns: []\n",
      "Index: [/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz, /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz]\n"
     ]
    }
   ],
   "source": [
    "\n",
    "import glob\n",
    "import os\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/\"\n",
    "dir2= \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/\"\n",
    "\n",
    "files= glob.glob(dir +\"*.fastq.gz\") + glob.glob(dir2 +\"*.fastq.gz\")\n",
    "\n",
    "folders = pandas.DataFrame([], index=files)\n",
    "folders.index.name='fastq'\n",
    "#folders['in_vitro'] = [True for f in dir_list]\n",
    "print(folders)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "fastq\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz                                          12\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz                                          26\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz                                          27\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz                                        29\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz                                           4\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz                                                                                                    11\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz                                                                                                    14\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz                                                                                                    18\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz                                                                                                    23\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz                                                                                                    24\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz                                                                                                    31\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz                         16\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz    19\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz      21\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz                        2\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz                        3\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz                        6\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz                                                                                   10\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz                                                                                   15\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz                                                                                   17\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz                                                                                     5\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz                                                                                     7\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz                                                                                     8\n",
      "Name: patient, dtype: int64\n"
     ]
    }
   ],
   "source": [
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "folders['patient'] = patients\n",
    "\n",
    "\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patient\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "print(folders[\"patient\"])\n",
    "\n",
    "#OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'\n",
    "#folders.to_csv(os.path.join(OUTPUT_PATH, 'ALL_fastq_human_rna_sample_info.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz 12\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz 26\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz 27\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz 29\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz 4\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz 11\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz 14\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz 18\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz 23\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz 24\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz 31\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz 16\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz 19\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz 21\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz 2\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz 3\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz 6\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz 10\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz 15\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz 17\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz 5\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz 7\n",
      "/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz 8\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 232,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' CreateSequenceDictionary R= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta' O= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.dict'\n"
     ]
    }
   ],
   "source": [
    "##Createdictionary #File is generated \n",
    "\n",
    "#http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/human_g1k_v37.fasta.gz is that the same refernence as used here? \n",
    "\n",
    "#terminal_process = \"java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' CreateSequenceDictionary R= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta' O= '/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.dict'\"\n",
    "\n",
    "#print(terminal_process) \n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##File is generated already! \n",
    "#terminal_process = \"samtools faidx human_g1k_v37.fasta\"\n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Step 1: already happened too! \n",
    "#genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/\"\n",
    "#terminal_process = \"'STAR --runMode genomeGenerate --genomeDir' genome_dir '--genomeFastaFiles human_g1k_v37.fasta --runThreadN 32'\"\n",
    "#print(subprocess.check_output(terminal_process, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P16 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P19 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P21 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P10 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P2 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P3 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P6 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P10.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P10 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P15.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P15 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P17.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P17 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P5.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P5 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P7.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P7 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "STAR --genomeDir /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/cat_fastq_files_28_nov/cat_P8.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step2/P8 --readFilesCommand zcat\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##STEP 2 -first pass Alignment \n",
    "#genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ \"\n",
    "#aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "\n",
    "\n",
    "#for index, row in folders.iterrows():\n",
    "#    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "#    terminal_code = \"STAR --genomeDir \" + genome_dir + \"--readFilesIn \" + index + \" --runThreadN 32 --outFileNamePrefix \" + aligned_dir + \"P\" + str(row['patient']) + \" --readFilesCommand zcat\"\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "#    print(terminal_code)\n",
    "#    print(\"\\n\")\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 291,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P16\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P19\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P21\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23_TTAGGC_L007_R1.fastq.gz P10\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P2\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P3\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P6\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P10.fastq.gz P10\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P10\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P15.fastq.gz P15\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P15\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P17.fastq.gz P17\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P17\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P5.fastq.gz P5\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P5\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P7.fastq.gz P7\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P7\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/cat_P8.fastq.gz P8\n",
      "mkdir /data/scratch/iholtman/human_microglia_genotyping/step3/P8\n"
     ]
    }
   ],
   "source": [
    "#Make sub dirs for step 3\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    genome_dir=\"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "    terminal_code = \"mkdir \" + genome_dir + \"P\" + str(row['patient'])\n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "Nov 29 22:25:52 ..... started STAR run\n",
      "Nov 29 22:25:52 ... starting to generate Genome files\n",
      "Nov 29 22:26:46 ... starting to sort Suffix Array. This may take a long time...\n",
      "Nov 29 22:27:00 ... sorting Suffix Array chunks and saving them to disk...\n",
      "Nov 29 22:34:34 ... loading chunks from disk, packing SA...\n",
      "Nov 29 22:35:29 ... finished generating suffix array\n",
      "Nov 29 22:35:29 ... generating Suffix Array index\n",
      "Nov 29 22:38:27 ... completed Suffix Array index\n",
      "Nov 29 22:38:27 ..... inserting junctions into the genome indices\n",
      "Nov 29 22:39:46 ... writing Genome to disk ...\n",
      "Nov 29 22:39:48 ... writing Suffix Array to disk ...\n",
      "Nov 29 22:40:07 ... writing SAindex to disk\n",
      "Nov 29 22:40:08 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "Nov 29 22:40:08 ..... started STAR run\n",
      "Nov 29 22:40:08 ... starting to generate Genome files\n",
      "Nov 29 22:41:04 ... starting to sort Suffix Array. This may take a long time...\n",
      "Nov 29 22:41:18 ... sorting Suffix Array chunks and saving them to disk...\n",
      "Nov 29 22:49:11 ... loading chunks from disk, packing SA...\n",
      "Nov 29 22:50:06 ... finished generating suffix array\n",
      "Nov 29 22:50:06 ... generating Suffix Array index\n",
      "Nov 29 22:53:03 ... completed Suffix Array index\n",
      "Nov 29 22:53:04 ..... inserting junctions into the genome indices\n",
      "Nov 29 22:54:27 ... writing Genome to disk ...\n",
      "Nov 29 22:54:30 ... writing Suffix Array to disk ...\n",
      "Nov 29 22:54:59 ... writing SAindex to disk\n",
      "Nov 29 22:55:00 ..... finished successfully\n",
      "\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/cat_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n"
     ]
    }
   ],
   "source": [
    "#STEP 3 - update the reference genome\n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "genome_dir=\"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "\n",
    "    terminal_code = \"STAR --runMode genomeGenerate --genomeDir \" + genome_dir + \"P\" + str(row['patient']) + \" --genomeFastaFiles /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "    --sjdbFileChrStartEnd \" + aligned_dir + \"P\" + str(row['patient']) + \"SJ.out.tab --sjdbOverhang 75 --runThreadN 32 --outFileNamePrefix \" \\\n",
    "    + \"P\" + str(row['patient'])\n",
    "                                              \n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")                                       \n",
    "                  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/data/scratch/iholtman/human_microglia_genotyping/step3/P8\n"
     ]
    }
   ],
   "source": [
    "print(reference_dir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz P0\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P0 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P0 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz P0\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P0 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P0 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz P0\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P0 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P0 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz P0\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P0 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P0 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz P4\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P4 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P4 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz P11\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P11 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P11 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz P14\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P14 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P14 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz P18\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P18 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P18 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz P23\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P23 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P23 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz P24\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P24 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P24 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz P31\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P31 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P31 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P16 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P16 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P19 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P19 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P21 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P21 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P2 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P2 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P3 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P3 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P6 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P6 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P10 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P10 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P15 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P15 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P17 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P17 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P5 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P5 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P7 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P7 --readFilesCommand zcat\n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "STAR --genomeDir /data/scratch/iholtman/human_microglia_genotyping/step3/P8 --readFilesIn /data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz --runThreadN 32 --outFileNamePrefix /data/scratch/iholtman/human_microglia_genotyping/step4/P8 --readFilesCommand zcat\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#####STEP 4 realign with updated reference genome \n",
    "final_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step4/\"\n",
    "aligned_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step2/\"\n",
    "reference_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step3/\"\n",
    "#genome_dir=\"/gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/ \"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    reference_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step3/\" + \"P\" + str(row['patient'])\n",
    "    terminal_code = \"STAR --genomeDir \" + reference_dir + \" --readFilesIn \" + index \\\n",
    "    + \" --runThreadN 32 --outFileNamePrefix \" + final_dir + \"P\" + str(row['patient']) + \" --readFilesCommand zcat\"\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    \n",
    "    print(\"\\n\")\n",
    "    \n",
    "    \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz P12\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##Step 5 add readgroups \n",
    "#Add readgroups and markduplicates \n",
    "##77bp read length Next-seq and 50bp is hiseq. For time being everything is set to HISEQ! \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "PICARD=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar\" \n",
    "\n",
    "####\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = PICARD + \" AddOrReplaceReadGroups I=\" + final_dir + \"P\" + str(row['patient']) + \"Aligned.out.sam\" \\\n",
    "    + \" O=\" + step5_dir + \"P\" + str(row['patient']) + \"rg_added_sorted.bam \" + \"SO=coordinate RGID=ILLUMINA RGLB=ILLUMINA RGPL=ILLUMINA RGPU=HISEQ RGSM=\" + \"P\" + str(row['patient']) \n",
    "\n",
    "    #print(terminal_code)\n",
    "    print(\"\\n\")\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz P12\n"
     ]
    }
   ],
   "source": [
    "##Step 5: Markduplicates! \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "PICARD=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar\" \n",
    "\n",
    "####\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = PICARD + \" MarkDuplicates I=\" + step5_dir + \"P\" + str(row['patient']) + \"rg_added_sorted.bam \" + \\\n",
    "    \"O= \" + step5_dir + \"P\" + str(row['patient']) + \"dedupped.bam \" + \"CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT \" + \\\n",
    "    \"M= \" + step5_dir + \"P\" + str(row['patient']) + \"output.metrics\" \n",
    "    \n",
    "\n",
    "    #print(terminal_code)\n",
    "    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "     \n",
    "##\n",
    "##Original \n",
    "##java -jar picard.jar MarkDuplicates I=rg_added_sorted.bam O=dedupped.bam  CREATE_INDEX=true VALIDATION_STRINGENCY=SILENT M=output.metrics "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz P12\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P12dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P12split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz P26\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P26dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P26split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz P27\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P27dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P27split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz P29\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P29dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P29split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz P4\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P4dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P4split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz P11\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P11dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P11split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz P14\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P14dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P14split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz P18\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P18dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P18split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz P23\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P23dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P23split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz P24\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P24dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P24split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz P31\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P31dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P31split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P16dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P19dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P21dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P21split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P2dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P3dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P6dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P10dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P15dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P17dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P5dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P7dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n",
      "working on/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -I /data/scratch/iholtman/human_microglia_genotyping/step5/P8dedupped.bam -o /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###split exons \n",
    "##Question: Is this function referring to the original fasta-file?? \n",
    "step5_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step5/\"\n",
    "step6_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step6/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on\" + index + \" P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T SplitNCigarReads -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "    \" -I \" + step5_dir + \"P\" + str(row['patient']) + \"dedupped.bam\" \\\n",
    "    \" -o \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS \"\n",
    "\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "    \n",
    "###Original \n",
    "##java -jar GenomeAnalysisTK.jar -T SplitNCigarReads -R ref.fasta -I dedupped.bam -o split.bam -rf ReassignOneMappingQuality -RMQF 255 -RMQT 60 -U ALLOW_N_CIGAR_READS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz sample P12\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P12split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P12_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz sample P26\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P26split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P26_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz sample P27\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P27split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P27_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz sample P29\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P29split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P29_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz sample P4\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P4split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P4_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz sample P11\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P11split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P11_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz sample P14\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P14split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P14_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz sample P18\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P18split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P18_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz sample P23\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P23split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P23_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz sample P24\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P24split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P24_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz sample P31\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P31split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P31_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz sample P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz sample P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P21split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0 -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.vcf\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###Variant calling normal VCF !\n",
    "#step7_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step7/\"\n",
    "#GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "\n",
    "#for index, row in folders.iterrows():\n",
    "#    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "#    terminal_code = GATK + \" -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "#    \" -I \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "#    \" -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0\" \\\n",
    "#    \" -o \" + step7_dir + \"P\" + str(row['patient']) + \"_output.vcf\"\n",
    "\n",
    "#    print(terminal_code)\n",
    "#    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "#   print(\"\\n\")\n",
    "\n",
    "\n",
    "##GVCF\n",
    "#java -jar GenomeAnalysisTK.jar \\\n",
    "#    -R reference.fasta \\\n",
    "#     -T HaplotypeCaller \\\n",
    "#     -I sample1.bam \\\n",
    "#     [--dbsnp dbSNP.vcf] \\\n",
    "#     -stand_call_conf 20 \\\n",
    "#     -stand_emit_conf 20 \\\n",
    "#     -o output.raw.snps.indels.vcf\n",
    " \n",
    "    \n",
    "##original \n",
    "##java -jar GenomeAnalysisTK.jar -T HaplotypeCaller -R ref.fasta -I input.bam -dontUseSoftClippedBases \n",
    "#-stand_call_conf 20.0 -stand_emit_conf 20.0 -o output.vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P12-151125-DGO-15-12-14_S89_L008_R1_001.fastq.gz sample P12\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P12split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P12_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P26-Tumor-160815-DGO-16-10-28_S6_R1_001.fastq.gz sample P26\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P26split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P26_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P27-Tumor-160829-DGO-16-10-28_S7_R1_001.fastq.gz sample P27\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P27split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P27_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human-Microglia-RNA-PolyA-P29-Frontal-160919-DGO-16-10-28_S8_R1_001.fastq.gz sample P29\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P29split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P29_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG_S12_L002_R1_001.fastq.gz sample P4\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P4split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P4_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P11.fastq.gz sample P11\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P11split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P11_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P14.fastq.gz sample P14\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P14split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P14_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P18.fastq.gz sample P18\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P18split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P18_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P23.fastq.gz sample P23\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P23split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P23_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P24.fastq.gz sample P24\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P24split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P24_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/5-dec/cat_P31.fastq.gz sample P31\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P31split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P31_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04_S52_L004_R1_001.fastq.gz sample P16\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P16split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P19_Temporal_160426_12cyclesPCR_DGO_16_07_25_S51_L003_R1_001.fastq.gz sample P19\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P19split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/Human_Microglia_RNA_PolyA_P21_Cortex_160606_12cyclesPCR_DGO_16_07_25_S52_L003_R1_001.fastq.gz sample P21\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P21split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15_GAGTGG_L008_R1.fastq.gz sample P2\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P2split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15_CAAAAG_L008_R1.fastq.gz sample P3\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P3split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21_AGTCAA_L002_R1.fastq.gz sample P6\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P6split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P10.fastq.gz sample P10\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P10split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P15.fastq.gz sample P15\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P15split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P17.fastq.gz sample P17\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P17split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P5.fastq.gz sample P5\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P5split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P7.fastq.gz sample P7\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P7split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf\n",
      "\n",
      "\n",
      "working on variant calling of file/data/scratch/iholtman/human_microglia_genotyping/fastq-files/cat_fastq_files_28_nov/cat_P8.fastq.gz sample P8\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta -I /data/scratch/iholtman/human_microglia_genotyping/step6/P8split.bam -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF  -o /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "###Variant calling with G-VCF functionality \n",
    "##Suggesties van NIEK! \n",
    "#Misschien geen cut-offs doen hier? Met of zonder maakt geen verschil! \n",
    "\n",
    "\n",
    "step7_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step7/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "\n",
    "for index, row in folders.iterrows():\n",
    "    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "    terminal_code = GATK + \" -T HaplotypeCaller -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "    \" -I \" + step6_dir + \"P\" + str(row['patient']) + \"split.bam\" \\\n",
    "    \" -dontUseSoftClippedBases -stand_call_conf 20.0 -stand_emit_conf 20.0  --emitRefConfidence GVCF \" \\\n",
    "    \" -o \" + step7_dir + \"P\" + str(row['patient']) + \"_output.g.vcf\"\n",
    "\n",
    "    print(terminal_code)\n",
    "    #\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['/data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P14_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P24_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P4_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P11_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P12_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P3_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P23_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P18_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P31_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P27_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P26_output.g.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step7/P29_output.g.vcf']\n"
     ]
    }
   ],
   "source": [
    "## print files en conversie naar normale format. \n",
    "VCF_Files = glob.glob(step7_dir + \"*.g.vcf\")\n",
    "print(VCF_Files)\n",
    "\n",
    "#voor volgende stap "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "######\n",
    "#Combine all separate g.vcf files into a single VCF file\n",
    "\n",
    "step8_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step8/\"\n",
    "\n",
    "VCF_Files = glob.glob(step7_dir + \"*.g.vcf\")\n",
    "#print(mergeVCF + \" \" + str(VCF_Files) + \" > \" + step1_dir + \"Human_MG_merged_VCF.vcf\")\n",
    "\n",
    "terminal_code = GATK + \" -T GenotypeGVCFs -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -o \" + step8_dir + \"all.sample.vcf -log sample.genotypegvcfs.log  -stand_call_conf 20.0 -stand_emit_conf 20.0 \" \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P19_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P6_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P14_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P24_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P4_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P11_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P17_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P12_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P16_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P10_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P8_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P7_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P5_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_mneicroglia_genotyping/step7/P3_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P23_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P15_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P18_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P31_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P2_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P27_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P21_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P26_output.g.vcf \" + \\\n",
    "\"-V /data/scratch/iholtman/human_microglia_genotyping/step7/P29_output.g.vcf \"\n",
    "    \n",
    "#print(terminal_code)    \n",
    "print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 91,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -Xmx4g -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantRecalibrator -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -input /data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf -resource:hapmap,known=false,training=true,truth=true,prior=15.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/hapmap_3.3.b37.vcf -resource:omni,known=false,training=true,truth=false,prior=12.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_omni2.5.b37.vcf -resource:1000G,known=false,training=true,truth=false,prior=10.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_phase1.snps.high_confidence.b37.vcf -resource:dbsnp,known=true,training=false,truth=false,prior=2.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/dbsnp_138.b37.vcf -an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR -an InbreedingCoeff -mode SNP -recalFile /data/scratch/iholtman/human_microglia_genotyping/step9/output.recal -tranchesFile /data/scratch/iholtman/human_microglia_genotyping/step9/output.tranches -rscriptFile /data/scratch/iholtman/human_microglia_genotyping/step9/output.plots.R \n"
     ]
    }
   ],
   "source": [
    "##### Variant recalibration (VQRS) - hard filtering kan ook! Uitzoeken, of navragen. \n",
    "##VariantRecalibrator - calculate the recalibration model \n",
    "\n",
    "step9_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step9/\"\n",
    "GATKx=\"java -Xmx4g -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "terminal_code = GATKx + \" -T VariantRecalibrator -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -input \" + step8_dir + \"all.sample.vcf \"\\\n",
    "   \"-resource:hapmap,known=false,training=true,truth=true,prior=15.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/hapmap_3.3.b37.vcf \" \\\n",
    "   \"-resource:omni,known=false,training=true,truth=false,prior=12.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_omni2.5.b37.vcf \" \\\n",
    "   \"-resource:1000G,known=false,training=true,truth=false,prior=10.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_phase1.snps.high_confidence.b37.vcf \" \\\n",
    "   \"-resource:dbsnp,known=true,training=false,truth=false,prior=2.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/dbsnp_138.b37.vcf \" \\\n",
    "   \"-an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR -an InbreedingCoeff \" \\\n",
    "   \"-mode SNP \" \\\n",
    "   \"-recalFile \" + step9_dir + \"output.recal \" \\\n",
    "   \"-tranchesFile \" + step9_dir+ \"output.tranches \" \\\n",
    "   \"-rscriptFile \" + step9_dir +\"output.plots.R \"\n",
    "\n",
    "print(terminal_code)    \n",
    "#print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -Xmx4g -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantRecalibrator -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -input /data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf -resource:hapmap,known=false,training=true,truth=true,prior=15.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/hapmap_3.3.b37.vcf -resource:omni,known=false,training=true,truth=false,prior=12.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_omni2.5.b37.vcf -resource:1000G,known=false,training=true,truth=false,prior=10.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_phase1.snps.high_confidence.b37.vcf -resource:dbsnp,known=true,training=false,truth=false,prior=2.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/dbsnp_138.b37.vcf -an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR -an InbreedingCoeff -mode SNP -recalFile /data/scratch/iholtman/human_microglia_genotyping/step9/output_simplified.recal -tranchesFile /data/scratch/iholtman/human_microglia_genotyping/step9/output_simplified.tranches -rscriptFile /data/scratch/iholtman/human_microglia_genotyping/step9/output_simplified.plots.R \n"
     ]
    }
   ],
   "source": [
    "##### Variant recalibration (VQRS) - hard filtering kan ook! Uitzoeken, of navragen. Simplified script\n",
    "##VariantRecalibrator - calculate the recalibration model \n",
    "\n",
    "step9_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step9/\"\n",
    "GATKx=\"java -Xmx4g -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "terminal_code = GATKx + \" -T VariantRecalibrator -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -input \" + step8_dir + \"all.sample.vcf \"\\\n",
    "   \"-resource:hapmap,known=false,training=true,truth=true,prior=15.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/hapmap_3.3.b37.vcf \" \\\n",
    "   \"-resource:omni,known=false,training=true,truth=false,prior=12.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_omni2.5.b37.vcf \" \\\n",
    "   \"-resource:1000G,known=false,training=true,truth=false,prior=10.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/1000G_phase1.snps.high_confidence.b37.vcf \" \\\n",
    "   \"-resource:dbsnp,known=true,training=false,truth=false,prior=2.0 /gpfs/data01/glasslab/home/iholtman/Molgenis/GATK-bundle/dbsnp_138.b37.vcf \" \\\n",
    "   \"-an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR -an InbreedingCoeff \" \\\n",
    "   \"-mode SNP \" \\\n",
    "   \"-recalFile \" + step9_dir + \"output_simplified.recal \" \\\n",
    "   \"-tranchesFile \" + step9_dir+ \"output_simplified.tranches \" \\\n",
    "   \"-rscriptFile \" + step9_dir +\"output_simplified.plots.R \"\n",
    "\n",
    "print(terminal_code)    \n",
    "#print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -V /data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf  -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' --setFilteredGtToNocall -o /data/scratch/iholtman/human_microglia_genotyping/step9/hard_threshold_filtered_snps.vcf\n"
     ]
    }
   ],
   "source": [
    "#####Variant filtering hard-threshold\n",
    "\n",
    "step9_dir = \"/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "terminal_code = GATK + \" -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -V \" + step8_dir + \"all.sample.vcf \"\\\n",
    "\" -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0' --setFilteredGtToNocall\" \\\n",
    "\" -o \" + step9_dir + \"hard_threshold_filtered_snps.vcf\"\n",
    "\n",
    "print(terminal_code)    \n",
    "#print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "##Recommended filter for RNA-seq genotyping: FS > 30.0\" -filterName QD -filter \"QD < 2.0\n",
    "##Filter QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0\n",
    "\n",
    "#java -jar GenomeAnalysisTK.jar \\ \n",
    "#    -T VariantFiltration \\ \n",
    "#    -R reference.fa \\ \n",
    "#    -V raw_snps.vcf \\ \n",
    "#    --filterExpression \"QD < 2.0 || FS > 60.0 || MQ < 40.0 || MQRankSum < -12.5 || ReadPosRankSum < -8.0\" \\ \n",
    "#    --filterName \"my_snp_filter\" \\ \n",
    "#    -o filtered_snps.vcf \n",
    "\n",
    "\n",
    "#######RNA-seq \n",
    "##java -jar GenomeAnalysisTK.jar -T VariantFiltration -R hg_19.fasta -V input.vcf -window 35 \n",
    "##-cluster 3 -filterName FS -filter \"FS > 30.0\" -filterName QD -filter \"QD < 2.0\" -o output.vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar -T ApplyRecalibration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta  -input /data/scratch/iholtman/human_microglia_genotyping/step8/all.sample.vcf  -mode SNP --ts_filter_level 99.0 -recalFile recalibrate_SNP.recal -tranchesFile recalibrate_SNP.tranches -o /data/scratch/iholtman/human_microglia_genotyping/step10/recalibrated_snps_raw_indels.vcf \n"
     ]
    }
   ],
   "source": [
    "##### Variant recalibration (VQRS)\n",
    "##ApplyRecalibration -- apply the recalibration model \n",
    "\n",
    "step10_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step10/\"\n",
    "GATK=\"java -jar /gpfs/data01/glasslab/home/iholtman/Molgenis/GenomeAnalysisTK-3.6/GenomeAnalysisTK.jar\"\n",
    "\n",
    "terminal_code = GATK + \" -T ApplyRecalibration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \" \\\n",
    "\" -input \" + step8_dir + \"all.sample.vcf \" \\\n",
    "    \" -mode SNP\" \\\n",
    "    \" --ts_filter_level 99.5\" \\\n",
    "    \" -recalFile\" + step8_dir + \"recalibrate_SNP.recal\" \\\n",
    "    \" -tranchesFile\" + step8_dir + \"recalibrate_SNP.tranches\" \\\n",
    "    \" -o \" + step10_dir + \"recalibrated_snps_raw_indels.vcf \"\n",
    "\n",
    "print(terminal_code)   \n",
    "#print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 92,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "###Variant filtering \n",
    "#step8_dir = \"/data/scratch/iholtman/human_microglia_genotyping/step8/\"\n",
    "\n",
    "\n",
    "#for index, row in folders.iterrows():\n",
    "#    print(\"working on variant calling of file\" + index + \" sample P\" + str(row['patient']))\n",
    "#    terminal_code = GATK + \" -T VariantFiltration -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta\" \\\n",
    "#    \" -V \" + step7_dir + \"P\" + str(row['patient']) + \"_output.vcf\" + \\\n",
    "#    \" -window 35 -cluster 3 -filterName FS -filter 'FS > 30.0' -filterName QD -filter 'QD < 2.0'\" \\\n",
    "#    \" -o \" + step8_dir + \"P\" + str(row['patient']) + \"filtered_output.vcf\"\n",
    "\n",
    "#    print(terminal_code)\n",
    "#    print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "#    #print(\"\\n\")\n",
    "\n",
    "\n",
    "####original script \n",
    "#java -jar GenomeAnalysisTK.jar -T VariantFiltration -R hg_19.fasta -V input.vcf -window 35 -cluster 3 \n",
    "#-filterName FS -filter \"FS > 30.0\" -filterName QD -filter \"QD < 2.0\" -o output.vcf \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##In de handleiding worden nog een aantal zaken besproken, zoals \"mergeGvcf\" en \"genotypeGvcf\". Als je de \n",
    "#haplotype caller in gvcf modus draait wordt voor alle varianten en reference sites oa de coverage informatie\n",
    "#bewaard. Als je meerdere samples hebt kun je de individuele samples/calls met elkaar mergen, waarbij de \"mergeGvcf\" \n",
    "#stap rekening houdt met de coverage e.d. van de referentie sites. Vervolgens kan de \"genotypeGvcf\" stap deze \n",
    "#informatie gebruiken om optimale genotype calls te maken met meerdere samples. Gedurende deze laatste methode wordt \n",
    "#ook rekening gehouden met de haplotype informatie van de andere samples in je cohort om de callset te verbeteren.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####Base recalibration - recommended but not specified \n",
    "\n",
    "\n",
    "#\\java -jar $GATK \\\n",
    "#    -T BaseRecalibrator \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I $sFILE'split.bam' \\\n",
    "#    -L 20 \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/dbsnp_138.b37.vcf \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/Mills_and_1000G_gold_standard.indels.b37.vcf \\\n",
    "#    -o $sFILE'recal_data.table'\n",
    "\n",
    "#java -jar $GATK \\\n",
    "#    -T BaseRecalibrator \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I $sFILE'split.bam' \\\n",
    "#    -L 20 \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/dbsnp_138.b37.vcf \\\n",
    "#    -knownSites /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/Mills_and_1000G_gold_standard.indels.b37.vcf \\\n",
    "#    -BQSR $sFILE'recal_data.table' \\\n",
    "#    -o $sFILE'post_recal_data.table'\n",
    "\n",
    "    #MAKE plots before and after \n",
    "#java -jar $GATK \\\n",
    "#    -T AnalyzeCovariates \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -L 20 \\\n",
    "#    -before $sFILE'recal_data.table' \\\n",
    "#    -after $sFILE'post_recal_data.table' \\\n",
    "#    -plots $sFILE'recalibration_plots.pdf'\n",
    "\n",
    "#intergate into new BAM-files \n",
    "#java -jar $GATK \\\n",
    "#    -T PrintReads \\\n",
    "#    -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "#    -I split.bam \\\n",
    "#    -L 20 \\\n",
    "##    -BQSR $sFILE'recal_data.table' \\\n",
    "#    -o $sFILE'recal_reads.bam'    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "####TXT conversion files for BEAGLE \n",
    "##https://faculty.washington.edu/browning/beagle_utilities/utilities.html\n",
    "##vcf2beagle.jar\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####################\n",
    "### in Loop without basecalibration\n",
    "#FILES=$(cat /gpfs/data01/glasslab/home/iholtman/Molgenis/second_pass/LIST_FILES)\n",
    "#for P in $FILES\n",
    "##do\n",
    "#java -jar $GATK -T HaplotypeCaller \\\n",
    "# -R /gpfs/data01/glasslab/home/iholtman/Molgenis/hg37/human_g1k_v37.fasta \\\n",
    "# -I $P'_split.bam' \\\n",
    "# -dontUseSoftClippedBases \\\n",
    "## -stand_call_conf 20.0 \\\n",
    "# -stand_emit_conf 20.0 \\\n",
    "#--emitRefConfidence GVCF \\\n",
    "# -o $P'_Genotype_callign_output_no_calibration.g.vcf'\n",
    "#done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##vcf2beagle.jar\n",
    "#https://faculty.washington.edu/browning/beagle_utilities/utilities.html#vcf2beagle explained here \n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##Beagle:: \n",
    "\n",
    "######Step 1: MERGE VCF files into a single VCF\n",
    "## BV door: \n",
    "##https://faculty.washington.edu/browning/beagle_utilities/utilities.html#mergevcf\n",
    "\n",
    "#Step2 Conform-gt to make the VCF files similar to the reference VCF (from 1000G?) \n",
    "## vThe reference and target VCF files are each required to have data for at least 20 individuals so that the \n",
    "#statistical tests performed by the conform-gt program give reasonable results.\n",
    "\n",
    "##/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase1_vcf\n",
    "##chr1.1kg.ref.phase1_release_v3.20101123.vcf.gz       \n",
    "\n",
    "## Step3 BEAGLE imputation and phasing ! \n",
    "\n",
    "#####INformation \n",
    "#https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4263197/ \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/mergevcf.jar ['/data/scratch/iholtman/human_microglia_genotyping/step8/P7filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P19filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P2filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P6filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P3filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P10filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P5filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P16filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P21filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P17filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P8filtered_output.vcf', '/data/scratch/iholtman/human_microglia_genotyping/step8/P15filtered_output.vcf'] > /data/scratch/iholtman/human_microglia_genotyping/imputation/step1/Human_MG_merged_VCF.vcf\n"
     ]
    }
   ],
   "source": [
    "##STEP1 - merging VCF files - has become obsolete by gvcf GATK! \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 135,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase1_vcf/chr1.1kg.ref.phase1_release_v3.20101123.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/step9/hard_threshold_filtered_snps.vcf chrom=1 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr1_conformed match=POS\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr1.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=1 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_1_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr2.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=2 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_2_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr3.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=3 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_3_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr4.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=4 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_4_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr5.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=5 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_5_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr6.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=6 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_6_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr7.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=7 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_7_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr8.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=8 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_8_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr9.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=9 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_9_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr10.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=10 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_10_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr11.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=11 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_11_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr12.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=12 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_12_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr13.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=13 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_13_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr14.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=14 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_14_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr15.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=15 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_15_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr16.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=16 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_16_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr17.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=17 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_17_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr18.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=18 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_18_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr19.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=19 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_19_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr20.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=20 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_20_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr21.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=21 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_21_conformed match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr22.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=22 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_22_conformed match=POS\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#####CONFORM GT for making genotype file comparable to reference genome and perform some QC\n",
    "### loop over chromosomes\n",
    "####Reference genoom is V5 \n",
    "\n",
    "nrs =list(range(22))\n",
    "nnrs=list()\n",
    "\n",
    "for nr in nrs:\n",
    "    nnrs.append(nr+1)\n",
    "\n",
    "###\n",
    "step2_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/\"\n",
    "step9_dir = \"/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/\"\n",
    "\n",
    "##Conform-gt \n",
    "\n",
    "VCF_file = \"/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf\"\n",
    "conform_GT =\"java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar \" \n",
    "\n",
    "for chr in nnrs:\n",
    "    #print(\"working on chr \" + str(chr))\n",
    "    reference = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr\" + str(chr) + \".1kg.phase3.v5.vcf.gz\"\n",
    "    terminal_code = conform_GT + \" ref=\" + reference + \" gt=\" + VCF_file + \" chrom=\" + str(chr) + \" out=\" + step2_dir + \"chr_\"+ str(chr) + \"_conformed\" + \" match=POS\"\n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr1.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=1 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_1_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr2.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=2 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_2_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr3.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=3 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_3_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr4.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=4 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_4_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr5.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=5 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_5_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr6.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=6 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_6_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr7.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=7 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_7_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr8.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=8 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_8_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr9.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=9 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_9_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr10.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=10 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_10_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr11.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=11 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_11_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr12.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=12 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_12_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr13.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=13 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_13_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr14.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=14 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_14_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr15.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=15 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_15_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr16.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=16 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_16_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr17.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=17 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_17_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr18.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=18 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_18_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr19.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=19 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_19_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr20.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=20 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_20_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr21.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=21 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_21_conformed_v5a_ match=POS\n",
      "\n",
      "\n",
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr22.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf chrom=22 out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_22_conformed_v5a_ match=POS\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#####CONFORM GT for making genotype file comparable to reference genome and perform some QC\n",
    "### loop over chromosomes\n",
    "####Reference genoom is V5A (ter vervanging van V5  vanwege chromosoom 17 probleem) \n",
    "\n",
    "nrs =list(range(22))\n",
    "nnrs=list()\n",
    "\n",
    "for nr in nrs:\n",
    "    nnrs.append(nr+1)\n",
    "\n",
    "###\n",
    "step2_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/\"\n",
    "step9_dir = \"/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/\"\n",
    "\n",
    "\n",
    "##Conform-gt \n",
    "\n",
    "VCF_file = \"/data/scratch/iholtman/human_microglia_genotyping/genotyping/step9/hard_threshold_filtered_snps.vcf\"\n",
    "conform_GT =\"java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar \" \n",
    "\n",
    "for chr in nnrs:\n",
    "    #print(\"working on chr \" + str(chr))\n",
    "    reference = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr\" + str(chr) + \".1kg.phase3.v5a.vcf.gz\"\n",
    "    terminal_code = conform_GT + \" ref=\" + reference + \" gt=\" + VCF_file + \" chrom=\" + str(chr) + \" out=\" + step2_dir + \"chr_\"+ str(chr) + \"_conformed_v5a_\" + \" match=POS\" \n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/tools/conform-gt.24May16.cee.jar ref=/gpfs/data01/glasslab/home/iholtman/software/conform-gt/chr20.1kg.ref.phase1_release_v3.20101123.vcf.gz gt=/gpfs/data01/glasslab/home/iholtman/software/conform-gt/chr20.vcf.gz chrom=2 out=/gpfs/data01/glasslab/home/iholtman/software/conform-gtchr2_conformed\n"
     ]
    }
   ],
   "source": [
    "##########\n",
    " Arguments for specifying data\n",
    " gt=[file] specifies a VCF file containing a GT (genotype) format field for each marker. If a genotype contains the phased allele separator, ‘|’, then Beagle will preserve the phase of the genotype during the analysis. If you use the gt argument, all genotypes in the output file will be phased and non-missing.\n",
    " gl=[file] specifies a VCF file containing a GL or PL (genotype likelihood) format field for each marker. Any data in the GT format field will be ignored. If both GL and PL format fields are present for a marker, the GL format will be used.\n",
    " gtgl=[file] specifies a VCF file containing a GT, GL or PL format field for each marker. If a genotype is non-missing, Beagle will ignore the genotype likelihood. If both GL and PL format fields are present for a marker, the GL field will be used.\n",
    " ref=[file] specifies a VCF file containing phased reference genotypes. See the impute parameter.\n",
    " out=[prefix] specifies the output filename prefix. The prefix may be an absolute or relative filename, but it cannot be a directory name.\n",
    " excludesamples=[file] specifies a file containing non-reference samples (one sample per line) to be excluded from the analysis and output files.\n",
    "    \n",
    "Beagle 4.1 Page | 3\n",
    "  excludemarkers=[file] specifies a file containing markers (one marker per line) to be excluded from the analysis and the output files. An excluded marker identifier can either be an identifier from the VCF record’s ID field or a genomic coordinate in the format: CHROM:POS.\n",
    " map=[file] specifies a PLINK format genetic map on the cM scale. HapMap GrCh36 and GrCh37 genetic maps in PLINK format are available for download from the Beagle website. Use of a genetic map is recommended if you are imputing ungenotyped markers. If no genetic map is specified, Beagle will assume a constant recombination rate of 1 cM / Mb.\n",
    "chrom=[chrom:start-end] specifies a chromosome or chromosome interval using a chromosome identifier in the VCF file and the starting and ending positions of the interval. The entire chromosome, the beginning of the chromosome, and the end of a chromosome can be specified by chrom=[chrom], chrom=[chrom:-end], and chrom=[chrom:start-] respectively.\n",
    " maxlr=[number ≥ 1] specifies the maximum likelihood ratio (default: maxlr=5000) at a genotype. If 𝑀 is the maximum of the likelihoods of each possible genotype, any likelihood that is less than (𝑀⁄maxlr) is set to 0.0 to improve computational efficiency.\n",
    "2.2 Other arguments\n",
    " nthreads=[positive integer] specifies the number of threads of execution. If no nthreads parameter is specified, the nthreads parameter will be set equal to the number of CPU cores on the host machine.\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr1.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_1_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr2.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_2_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_2_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr3.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_3_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_3_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr4.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_4_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_4_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr5.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_5_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_5_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr6.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_6_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_6_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr7.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_7_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_7_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr8.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_8_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_8_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr9.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_9_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_9_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr10.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_10_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr11.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_11_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_11_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr12.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_12_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_12_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr13.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_13_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_13_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr14.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_14_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_14_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr15.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_15_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_15_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr16.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_16_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_16_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr17.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_17_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_17_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr18.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_18_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_18_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr19.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_19_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_19_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr20.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_20_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_20_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr21.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_21_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_21_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr22.1kg.phase3.v5a.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_22_conformed_v5a_.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_22_imputed_phased_v5a nthreads=30 lowmem=true\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##########\n",
    "##Step3 \n",
    "##imputation for 1000g reference_v5a_\n",
    "nrs =list(range(22))\n",
    "nnrs=list()\n",
    "\n",
    "for nr in nrs:\n",
    "    nnrs.append(nr+1)\n",
    "\n",
    "#    \n",
    "\n",
    "beagle = \"java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar \" \n",
    "step3_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/\"\n",
    "step2_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/\"\n",
    "\n",
    "\n",
    "###\n",
    "#######\n",
    "for chr in nnrs:\n",
    "    #print(\"working on chr \" + str(chr))\n",
    "    reference = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5a/individual_chromosomes/chr\" + str(chr) + \".1kg.phase3.v5a.vcf.gz\"\n",
    "    terminal_code = beagle + \" ref=\" + reference + \\\n",
    "    \" gt=\" + step2_dir + \"chr_\"+ str(chr) + \"_conformed_v5a_.vcf.gz\" + \\\n",
    "    \" out=\" + step3_dir + \"chr_\"+ str(chr) + \"_imputed_phased_v5a\" \\\n",
    "    \" nthreads=30 lowmem=true\"\n",
    "    \n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr1.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_1_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr2.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_2_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_2_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr3.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_3_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_3_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr4.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_4_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_4_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr5.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_5_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_5_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr6.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_6_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_6_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr7.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_7_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_7_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr8.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_8_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_8_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr9.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_9_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_9_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr10.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_10_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr11.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_11_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_11_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr12.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_12_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_12_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr13.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_13_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_13_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr14.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_14_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_14_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr15.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_15_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_15_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr16.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_16_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_16_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr17.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_17_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_17_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr18.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_18_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_18_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr19.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_19_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_19_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr20.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_20_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_20_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr21.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_21_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_21_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n",
      "java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar  ref=/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr22.1kg.phase3.v5.vcf.gz gt=/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/chr_22_conformed.vcf.gz out=/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_22_imputed_phased nthreads=30 lowmem=true\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "##########\n",
    "##Step3 - start at the second sample! \n",
    "##imputation for normal reference V5 \n",
    "nrs =list(range(22))\n",
    "nnrs=list()\n",
    "\n",
    "for nr in nrs:\n",
    "    nnrs.append(nr+1)\n",
    "\n",
    "#    \n",
    "\n",
    "beagle = \"java -Xmx30g -jar /gpfs/data01/glasslab/home/iholtman/software/beagle/beagle.27Jul16.86a.jar \" \n",
    "step3_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/\"\n",
    "step2_dir = \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step2/\"\n",
    "\n",
    "\n",
    "###\n",
    "#######\n",
    "for chr in nnrs:\n",
    "    #print(\"working on chr \" + str(chr))\n",
    "    reference = \"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr\" + str(chr) + \".1kg.phase3.v5.vcf.gz\"\n",
    "    terminal_code = beagle + \" ref=\" + reference + \\\n",
    "    \" gt=\" + step2_dir + \"chr_\"+ str(chr) + \"_conformed.vcf.gz\" + \\\n",
    "    \" out=\" + step3_dir + \"chr_\"+ str(chr) + \"_imputed_phased\" \\\n",
    "    \" nthreads=30 lowmem=true\"\n",
    "    \n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "    print(\"\\n\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "working on chr 1\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'step2_dir' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-1-44df13daba66>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     19\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"working on chr \"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     20\u001b[0m     \u001b[0mreference\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m\"/gpfs/data01/glasslab/home/iholtman/software/beagle/bochet.gcc.biostat.washington.edu/beagle/1000_Genomes_phase3_v5/chr\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchr\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\".1kg.phase3.v5.vcf.gz\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 21\u001b[0;31m     \u001b[0mterminal_code\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbeagle\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\" ref=\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mreference\u001b[0m \u001b[0;34m+\u001b[0m     \u001b[0;34m\" gt=\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstep2_dir\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"chr_\"\u001b[0m\u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchr\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"_conformed.vcf.gz\"\u001b[0m \u001b[0;34m+\u001b[0m     \u001b[0;34m\" out=\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstep3_dir\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"chr_\"\u001b[0m\u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mchr\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"_imputed_phased\"\u001b[0m     \u001b[0;34m\" nthreads=30 lowmem=true\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     22\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     23\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mterminal_code\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'step2_dir' is not defined"
     ]
    }
   ],
   "source": [
    "######### Concatenate the VCF files per chromosome in one\n",
    "##!Copy andpaste script in terminal!\n",
    "\n",
    "###Activate the perl-scripts: \n",
    "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
    "\n",
    "###\n",
    "#vcf-concat A.vcf.gz B.vcf.gz C.vcf.gz | gzip -c > out.vcf.gz\n",
    "\n",
    "####Concatenate files of different chromosomes \n",
    "vcf-concat \\\n",
    "chr_1_imputed_phased_v5a.vcf.gz \\\n",
    "chr_2_imputed_phased_v5a.vcf.gz \\\n",
    "chr_3_imputed_phased_v5a.vcf.gz \\\n",
    "chr_4_imputed_phased_v5a.vcf.gz \\\n",
    "chr_5_imputed_phased_v5a.vcf.gz \\\n",
    "chr_6_imputed_phased_v5a.vcf.gz \\\n",
    "chr_7_imputed_phased_v5a.vcf.gz \\\n",
    "chr_8_imputed_phased_v5a.vcf.gz \\\n",
    "chr_9_imputed_phased_v5a.vcf.gz \\\n",
    "chr_10_imputed_phased_v5a.vcf.gz \\\n",
    "chr_11_imputed_phased_v5a.vcf.gz \\\n",
    "chr_12_imputed_phased_v5a.vcf.gz \\\n",
    "chr_13_imputed_phased_v5a.vcf.gz \\\n",
    "chr_14_imputed_phased_v5a.vcf.gz \\\n",
    "chr_15_imputed_phased_v5a.vcf.gz \\\n",
    "chr_16_imputed_phased_v5a.vcf.gz \\\n",
    "chr_17_imputed_phased_v5a.vcf.gz \\\n",
    "chr_18_imputed_phased_v5a.vcf.gz \\\n",
    "chr_19_imputed_phased_v5a.vcf.gz \\\n",
    "chr_20_imputed_phased_v5a.vcf.gz \\\n",
    "chr_21_imputed_phased_v5a.vcf.gz \\\n",
    "chr_22_imputed_phased_v5a.vcf.gz \\\n",
    "| bgzip -c > concatenated_out_v5a.vcf.gz\n",
    "\n",
    "######Index the concatenated file  \n",
    "tabix -p vcf concatenated_out_v5a.vcf.gz\n",
    "\n",
    "###Merge files of WGS and concetanated chromosomes\n",
    "#### all -data \n",
    "vcf-merge \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/concatenated_out_v5a.vcf.gz\" \\\n",
    "\"/data/scratch/iholtman/human_microglia_genotyping/WGS/vcf/merged_P15_P17_P4_P5_P7_P8.vcf.gz\" \\\n",
    "| bgzip -c > \"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data.vcf.gz\"\n",
    "\n",
    "\n",
    "########output to plink\n",
    "vcftools --gzvcf merged_RNA_imputed_and_WGS_data.vcf.gz --plink --out concatenated_imputation_and_WGS_output\n",
    "\n",
    "##Plinkt convert to 0,1,2 format \n",
    "plink --file concatenated_imputation_and_WGS_output --noweb --recodeAD --out genotype_converted_to_012_format\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##!Copy andpaste script in terminal!\n",
    "### Same as above but for chromosome but 1 only \n",
    "\n",
    "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
    "\n",
    "######Index the concatenated file  \n",
    "tabix -p vcf chr_1_imputed_phased_v5a.vcf.gz\n",
    "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\n",
    "\n",
    "vcf-merge \"/data/scratch/iholtman/human_microglia_genotyping/WGS/chr1.recode.vcf.gz\" \\\n",
    "\"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased_v5a.vcf.gz\" \\\n",
    "| bgzip -c > \"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\"\n",
    "\n",
    "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr1.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr1 \n",
    "\n",
    "#vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr1.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr1_filtered --remove-filtered-all\n",
    "\n",
    "#######\n",
    "plink --file concatenated_imputation_and_WGS_output_chr1_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr1\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr1.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_1_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr1.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr1\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr1 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr1\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_2_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr2.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_2_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr2.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr2.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr2.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr2\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr2 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr2\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_3_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr3.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_3_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr3.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr3.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr3.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr3\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr3 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr3\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_4_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr4.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_4_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr4.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr4.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr4.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr4\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr4 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr4\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_5_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr5.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_5_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr5.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr5.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr5.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr5\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr5 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr5\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_6_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr6.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_6_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr6.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr6.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr6.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr6\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr6 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr6\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_7_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr7.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_7_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr7.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr7.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr7.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr7\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr7 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr7\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_8_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr8.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_8_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr8.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr8.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr8.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr8\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr8 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr8\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_9_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr9.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_9_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr9.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr9.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr9.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr9\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr9 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr9\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr10.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr10.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr10.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr10.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr10\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr10 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr10\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_11_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr11.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_11_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr11.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr11.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr11.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr11\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr11 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr11\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_12_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr12.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_12_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr12.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr12.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr12.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr12\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr12 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr12\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_13_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr13.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_13_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr13.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr13.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr13.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr13\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr13 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr13\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_14_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr14.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_14_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr14.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr14.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr14.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr14\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr14 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr14\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_15_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr15.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_15_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr15.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr15.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr15.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr15\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr15 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr15\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_16_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr16.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_16_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr16.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr16.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr16.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr16\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr16 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr16\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_17_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr17.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_17_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr17.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr17.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr17.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr17\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr17 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr17\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_18_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr18.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_18_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr18.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr18.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr18.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr18\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr18 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr18\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_19_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr19.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_19_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr19.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr19.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr19.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr19\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr19 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr19\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_20_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr20.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_20_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr20.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr20.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr20.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr20\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr20 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr20\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_21_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr21.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_21_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr21.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr21.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr21.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr21\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr21 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr21\n",
      "\n",
      "\n",
      "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_22_imputed_phased_v5a.vcf.gz\n",
      "vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr22.recode.vcf.gz /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_22_imputed_phased_v5a.vcf.gz| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr22.vcf.gz\n",
      "tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr22.vcf.gz\n",
      "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr22.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr22\n",
      "plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr22 --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr22\n"
     ]
    }
   ],
   "source": [
    "######Each chromosome separetely in a loop\n",
    "\n",
    "######Prepare Each chromosome genotype file separetely: WGS and imputation, index, and merge \n",
    "nrs =list(range(9:23))\n",
    "\n",
    "for nr in nrs:\n",
    "    print(\"\\n\")\n",
    "    print(\"export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\")\n",
    "    print(\"tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr +1) + \"_imputed_phased_v5a.vcf.gz\")\n",
    "    print(\"vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr\" + str(nr+1) + \".recode.vcf.gz \" \\\n",
    "    \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr+1) +\"_imputed_phased_v5a.vcf.gz\" \\\n",
    "    \"| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz\")\n",
    "    print(\"tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr +1) + \".vcf.gz\")\n",
    "    print(\"vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1))\n",
    "    print(\"plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1) + \" --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr\" + str(nr+1))\n",
    "\n",
    "#print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n"
     ]
    },
    {
     "ename": "CalledProcessError",
     "evalue": "Command '\n tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a.vcf.gz' returned non-zero exit status 1",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mCalledProcessError\u001b[0m                        Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-19-7fbecf1692ab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     11\u001b[0m     \u001b[0mtc\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m\"\\n tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\"\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mstr\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnr\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m\"_imputed_phased_v5a.vcf.gz\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0msubprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcheck_output\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtc\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mshell\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0msubprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mSTDOUT\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mdecode\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0;31m#    tc = perl5 + (\"\\n vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr\" + str(nr+1) + \".recode.vcf.gz \" \\\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/gpfs/data01/glasslab/home/iholtman/anaconda3/lib/python3.5/subprocess.py\u001b[0m in \u001b[0;36mcheck_output\u001b[0;34m(timeout, *popenargs, **kwargs)\u001b[0m\n\u001b[1;32m    624\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    625\u001b[0m     return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,\n\u001b[0;32m--> 626\u001b[0;31m                **kwargs).stdout\n\u001b[0m\u001b[1;32m    627\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    628\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/gpfs/data01/glasslab/home/iholtman/anaconda3/lib/python3.5/subprocess.py\u001b[0m in \u001b[0;36mrun\u001b[0;34m(input, timeout, check, *popenargs, **kwargs)\u001b[0m\n\u001b[1;32m    706\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mcheck\u001b[0m \u001b[0;32mand\u001b[0m \u001b[0mretcode\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    707\u001b[0m             raise CalledProcessError(retcode, process.args,\n\u001b[0;32m--> 708\u001b[0;31m                                      output=stdout, stderr=stderr)\n\u001b[0m\u001b[1;32m    709\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mCompletedProcess\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mprocess\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mretcode\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstdout\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mstderr\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    710\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mCalledProcessError\u001b[0m: Command '\n tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_10_imputed_phased_v5a.vcf.gz' returned non-zero exit status 1"
     ]
    }
   ],
   "source": [
    "######Each chromosome separetely in a loop\n",
    "######Prepare Each chromosome genotype file separetely: WGS and imputation, index, and merge \n",
    "###Werkt niet, probleem met perl5 library laden! \n",
    "\n",
    "import subprocess\n",
    "\n",
    "nrs =list(range(9,23))\n",
    "\n",
    "for nr in nrs:\n",
    "    print(\"\\n\")\n",
    "    #tc = (\"\\n tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr +1) + \"_imputed_phased_v5a.vcf.gz\")\n",
    "    \n",
    "    #print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "     \n",
    "#    tc = perl5 + (\"\\n vcf-merge /data/scratch/iholtman/human_microglia_genotyping/WGS/chr\" + str(nr+1) + \".recode.vcf.gz \" \\\n",
    "#    \"/data/scratch/iholtman/human_microglia_genotyping/imputation/step3/chr_\" + str(nr+1) +\"_imputed_phased_v5a.vcf.gz\" \\\n",
    "#    \"| bgzip -c > /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz\")\n",
    "    \n",
    "#    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "#\n",
    "#    tc = perl5 + (\"\\n tabix -p vcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr +1) + \".vcf.gz\")\n",
    "#    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "#    \n",
    "#    \n",
    "#    tc = perl5 + (\"\\n vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/merged_RNA_imputed_and_WGS_data_chr\" + str(nr+1) + \".vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1))\n",
    "#    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "#    \n",
    "#    tc = (\"plink --file /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/concatenated_imputation_and_WGS_output_chr\" + str(nr+1) + \" --noweb --recodeAD --out /data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr\" + str(nr+1))\n",
    "\n",
    "#    print(subprocess.check_output(tc, shell=True, stderr=subprocess.STDOUT).decode())\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "########SNP Chip files:\n",
    "\n",
    "export PERL5LIB=/gpfs/data01/glasslab/home/iholtman/software/vcftools/src/perl\n",
    "\n",
    "vcf-sort /data/scratch/iholtman/human_microglia_genotyping/SNP_CHIP/Phase_5_VCF_conversion_QC_and_liftover/VCF/filtered_human_MG_samples_chr.vcf.gz > sorted_filtered_human_MG_samples_chr.vcf.gz\n",
    "\n",
    "bgzip sorted_filtered_human_MG_samples.vcf\n",
    "tabix -p vcf sorted_filtered_human_MG_samples_chr.vcf.gz\n",
    "\n",
    "tabix -p sorted_filtered_human_MG_samples_chr.vcf.gz\n",
    "\n",
    "/data/scratch/iholtman/human_microglia_genotyping/SNP_CHIP/Phase_5_VCF_conversion_QC_and_liftover/VCF/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz\n",
    "\n",
    "vcf-merge \"/data/scratch/iholtman/human_microglia_genotyping/SNP_CHIP/Phase_5_VCF_conversion_QC_and_liftover/VCF/sorted_filtered_human_MG_samples_chr.vcf.gz\" \\\n",
    "\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz\" \\\n",
    "| bgzip -c > \"/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz\" \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##############Import 0,1,2 genotype file in python and make heatmap\n",
    "####"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import pandas\n",
    "import numpy\n",
    "import seaborn\n",
    "import matplotlib.pyplot as plt\n",
    "import pickle\n",
    "import os "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22]\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "########Loop through all chromosomes, safe as pickle, create heatmap \n",
    "## \n",
    "nrs =list(range(5,22))\n",
    "for nr in nrs:\n",
    "\n",
    "    ### Read-Data \n",
    "    data = pandas.read_table(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/individual_chromosomes/genotype_converted_to_012_format_chr\" + str(nr +1) + \".raw\"), sep = \" \")\n",
    "\n",
    "    ##Transform in numpy object, transform and convert to pandas\n",
    "    table = data.as_matrix()\n",
    "    genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "    pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "    pd_genotype = pd_genotype[6:]\n",
    "\n",
    "    #print(pd_genotype[0:10])\n",
    "    pd_genotype.shape\n",
    "\n",
    "    ##Select only even rows (because they are duplicate)\n",
    "    pd_genotype =pd_genotype[::2]\n",
    "    pd_genotype.shape\n",
    "\n",
    "    print(pd_genotype[1:5])\n",
    "\n",
    "    ##Safe object as pickle \n",
    "    with open(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr\" + str(nr +1) + \"_pickle\"), 'wb') as f:\n",
    "        pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "\n",
    "    #####################################Make correlation heatmap of chr1 \n",
    "    ##Remove NA values \n",
    "    pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "    #print(pd_genotype[1:10])\n",
    "    pd_genotype.shape\n",
    "\n",
    "    ##Calculate correlations \n",
    "    plot_data = pd_genotype.astype(float).corr()\n",
    "    #print(plot_data)\n",
    "\n",
    "    PLOT_CMAP = 'YlOrRd'\n",
    "    VMIN = 0\n",
    "    VMAX = 1\n",
    "    fig_size=(12,12)\n",
    "\n",
    "    ##PLot and write to files\n",
    "\n",
    "    cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "    _=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".pdf\"),  bbox_inches='tight')\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".png\"),  bbox_inches='tight')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "########Chromosome 1 only \n",
    "### Read-Data \n",
    "data = pandas.read_table(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/chr1/genotype_converted_to_012_format_chr1.raw\", sep = \" \")\n",
    "\n",
    "##Transform in numpy object, transform and convert to pandas\n",
    "table = data.as_matrix()\n",
    "genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "pd_genotype = pd_genotype[6:]\n",
    "\n",
    "#print(pd_genotype[0:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Select only even rows (because they are duplicate)\n",
    "pd_genotype =pd_genotype[::2]\n",
    "pd_genotype.shape\n",
    "\n",
    "print(pd_genotype[1:5])\n",
    "\n",
    "##Safe object as pickle \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr1_pickle', 'wb') as f:\n",
    "    pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#####################################Make correlation heatmap of chr1 \n",
    "##Remove NA values \n",
    "pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "#print(pd_genotype[1:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Calculate correlations \n",
    "plot_data = pd_genotype.astype(float).corr()\n",
    "#print(plot_data)\n",
    "\n",
    "PLOT_CMAP = 'YlOrRd'\n",
    "VMIN = 0\n",
    "VMAX = 1\n",
    "fig_size=(12,12)\n",
    "\n",
    "##PLot and write to files\n",
    "\n",
    "cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "_=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS_chr1.pdf',  bbox_inches='tight')\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS_chr1.png',  bbox_inches='tight')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##Load object \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr1_pickle', 'rb') as f:\n",
    "    pd_genotype = pickle.load(f)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##########################################\n",
    "####################Read whole genome! \n",
    "data = pandas.read_table(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/genotype_converted_to_012_format.raw\", sep = \" \")\n",
    "\n",
    "##Transform in numpy object, transform and convert to pandas\n",
    "table = data.as_matrix()\n",
    "genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "pd_genotype = pd_genotype[6:]\n",
    "\n",
    "#print(pd_genotype[0:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Select only even rows (because they are duplicate)\n",
    "pd_genotype =pd_genotype[::2]\n",
    "pd_genotype.shape\n",
    "\n",
    "print(pd_genotype[1:5])\n",
    "\n",
    "##Safe object as pickle \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/whole_genome_pickle', 'wb') as f:\n",
    "    pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#####################################\n",
    "##Remove NA values \n",
    "pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "#print(pd_genotype[1:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "##Calculate correlations \n",
    "plot_data = pd_genotype.astype(float).corr()\n",
    "#print(plot_data)\n",
    "\n",
    "PLOT_CMAP = 'YlOrRd'\n",
    "VMIN = 0\n",
    "VMAX = 1\n",
    "fig_size=(12,12)\n",
    "\n",
    "##PLot and write to files\n",
    "\n",
    "cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "_=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS.pdf',  bbox_inches='tight')\n",
    "plt.savefig('/data/scratch/iholtman/human_microglia_genotyping/jupyter/sample_genotype_correlations_imputed_WGS.png',  bbox_inches='tight')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "##Load object \n",
    "with open('/data/scratch/iholtman/human_microglia_genotyping/jupyter/whole_genome_pickle', 'rb') as f:\n",
    "    pd_genotype = pickle.load(f)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "302598\n"
     ]
    }
   ],
   "source": [
    "###Hoeveel SNPs zijn er, en komen de oorspronkelijke en de gemergde bestanden overeen?? \n",
    "\n",
    "from collections import Counter\n",
    "import pandas\n",
    "pd_genotype.shape\n",
    "#print(pd_genotype)\n",
    "\n",
    "pd_genotype.WGS_P15.value_counts(dropna=False)\n",
    "print(181681 + 94901 + 26016)\n",
    "##dit is het aantal SNPs dat in het ge-mergde bestand terug te vinden is van chr1 \n",
    "\n",
    "####\n",
    "#dir\n",
    "#/data/scratch/iholtman/human_microglia_genotyping/WGS/vcf\n",
    "#vcftools --gzvcf copy_P15.recalibrated.haplotypeCalls.vcf --chr 1 --recode --recode-INFO-all --out chr1_copy_P15.recalibrated.haplotypeCalls.vcf\n",
    "##390464 \n",
    "#Dit is het aantal dat in het oorspronkelijke VCF bestand zat. Er lijken dus 90.000 SNPs verloren te zijn gegaan. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'pd_genotype' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-1-d78fe5ce7b85>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0;32mfor\u001b[0m \u001b[0mcolumn\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mpd_genotype\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mcolumns\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m     \u001b[0;31m#print(column)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0mfunction\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'pd_genotype.'\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mcolumn\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0;34m'.value_counts(dropna=False)'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mfunction\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'pd_genotype' is not defined"
     ]
    }
   ],
   "source": [
    "for column in pd_genotype.columns:\n",
    "    #print(column)\n",
    "    function = 'pd_genotype.' + column + '.value_counts(dropna=False)'\n",
    "    print(function)\n",
    "    \n",
    "    ###Dylan's suggestion: \n",
    "    print(pd_genotype.loc[:,column].value_counts(dropna=False))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "output = pd_genotype.WGS_P4.value_counts(dropna=False) \n",
    "print(output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "NaN     2101020\n",
      " 1.0     197810\n",
      " 0.0      93716\n",
      " 2.0      22861\n",
      "Name: WGS_P5, dtype: int64\n"
     ]
    }
   ],
   "source": [
    "output = pd_genotype.WGS_P5.value_counts(dropna=False) \n",
    "print(output)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#########Concordance tables:\n",
    "#########\n",
    "bcftools gtcheck merged_RNA_imputed_and_WGS_data.vcf.gz -p\n",
    "bcftools gtcheck merged_RNA_imputed_and_WGS_data.vcf.gz -G 1 > bcftools_gtcheck_out\n",
    "\n",
    "bcftools gtcheck merged_RNA_imputed_and_WGS_data.vcf.gz > bcftools_gtcheck_out_2  \n",
    "bcftools stats merged_RNA_imputed_and_WGS_data.vcf.gz > bcftools_stats_out\n",
    "\n",
    "#########\n",
    "vcf-contrast merged_RNA_imputed_and_WGS_data.vcf.gz \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "[Thu Jan 12 15:57:51 PST 2017] picard.vcf.GenotypeConcordance TRUTH_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz CALL_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz OUTPUT=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P15_truth_WGS_P15.vcf TRUTH_SAMPLE=WGS_P15 CALL_SAMPLE=WGS_P15    INTERSECT_INTERVALS=true MIN_GQ=0 MIN_DP=0 OUTPUT_ALL_ROWS=false USE_VCF_INDEX=false MISSING_SITES_HOM_REF=false VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Jan 12 15:57:51 PST 2017] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.3.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2017-01-12 15:57:52\tGenotypeConcordance\tStarting iteration over variants.\n",
      "INFO\t2017-01-12 15:57:52\tGenotypeConcordance\tchecked        10,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:1,120,957\n",
      "INFO\t2017-01-12 15:57:53\tGenotypeConcordance\tchecked        20,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:1,707,743\n",
      "INFO\t2017-01-12 15:57:53\tGenotypeConcordance\tchecked        30,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:2,389,563\n",
      "INFO\t2017-01-12 15:57:54\tGenotypeConcordance\tchecked        40,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,017,290\n",
      "INFO\t2017-01-12 15:57:54\tGenotypeConcordance\tchecked        50,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,627,524\n",
      "INFO\t2017-01-12 15:57:54\tGenotypeConcordance\tchecked        60,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:4,394,403\n",
      "INFO\t2017-01-12 15:57:55\tGenotypeConcordance\tchecked        70,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,073,378\n",
      "INFO\t2017-01-12 15:57:55\tGenotypeConcordance\tchecked        80,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,732,295\n",
      "INFO\t2017-01-12 15:57:55\tGenotypeConcordance\tchecked        90,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:6,438,378\n",
      "INFO\t2017-01-12 15:57:55\tGenotypeConcordance\tchecked       100,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:7,315,516\n",
      "INFO\t2017-01-12 15:57:56\tGenotypeConcordance\tchecked       110,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:8,086,334\n",
      "INFO\t2017-01-12 15:57:56\tGenotypeConcordance\tchecked       120,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:8,968,709\n",
      "INFO\t2017-01-12 15:57:56\tGenotypeConcordance\tchecked       130,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:9,716,019\n",
      "INFO\t2017-01-12 15:57:57\tGenotypeConcordance\tchecked       140,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:10,623,841\n",
      "INFO\t2017-01-12 15:57:57\tGenotypeConcordance\tchecked       150,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:11,403,951\n",
      "INFO\t2017-01-12 15:57:57\tGenotypeConcordance\tchecked       160,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:12,131,083\n",
      "INFO\t2017-01-12 15:57:57\tGenotypeConcordance\tchecked       170,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:12,899,984\n",
      "INFO\t2017-01-12 15:57:58\tGenotypeConcordance\tchecked       180,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:14,034,254\n",
      "INFO\t2017-01-12 15:57:58\tGenotypeConcordance\tchecked       190,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:14,782,380\n",
      "INFO\t2017-01-12 15:57:58\tGenotypeConcordance\tchecked       200,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:15,518,433\n",
      "INFO\t2017-01-12 15:57:58\tGenotypeConcordance\tchecked       210,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:16,317,389\n",
      "INFO\t2017-01-12 15:57:59\tGenotypeConcordance\tchecked       220,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:16,940,194\n",
      "INFO\t2017-01-12 15:57:59\tGenotypeConcordance\tchecked       230,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:17,543,832\n",
      "INFO\t2017-01-12 15:57:59\tGenotypeConcordance\tchecked       240,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:18,280,366\n",
      "INFO\t2017-01-12 15:58:00\tGenotypeConcordance\tchecked       250,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:19,026,057\n",
      "INFO\t2017-01-12 15:58:00\tGenotypeConcordance\tchecked       260,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:19,777,837\n",
      "INFO\t2017-01-12 15:58:00\tGenotypeConcordance\tchecked       270,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:20,592,124\n",
      "INFO\t2017-01-12 15:58:01\tGenotypeConcordance\tchecked       280,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:21,534,929\n",
      "INFO\t2017-01-12 15:58:01\tGenotypeConcordance\tchecked       290,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:22,326,328\n",
      "INFO\t2017-01-12 15:58:01\tGenotypeConcordance\tchecked       300,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:23,116,975\n",
      "INFO\t2017-01-12 15:58:01\tGenotypeConcordance\tchecked       310,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:24,100,808\n",
      "INFO\t2017-01-12 15:58:02\tGenotypeConcordance\tchecked       320,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:25,015,530\n",
      "INFO\t2017-01-12 15:58:02\tGenotypeConcordance\tchecked       330,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:25,907,167\n",
      "INFO\t2017-01-12 15:58:02\tGenotypeConcordance\tchecked       340,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:26,844,752\n",
      "INFO\t2017-01-12 15:58:02\tGenotypeConcordance\tchecked       350,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:27,980,714\n",
      "INFO\t2017-01-12 15:58:03\tGenotypeConcordance\tchecked       360,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:28,981,150\n",
      "INFO\t2017-01-12 15:58:03\tGenotypeConcordance\tchecked       370,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:30,094,819\n",
      "INFO\t2017-01-12 15:58:03\tGenotypeConcordance\tchecked       380,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:30,842,172\n",
      "INFO\t2017-01-12 15:58:03\tGenotypeConcordance\tchecked       390,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:31,587,628\n",
      "INFO\t2017-01-12 15:58:04\tGenotypeConcordance\tchecked       400,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:32,447,963\n",
      "INFO\t2017-01-12 15:58:04\tGenotypeConcordance\tchecked       410,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:33,489,498\n",
      "INFO\t2017-01-12 15:58:04\tGenotypeConcordance\tchecked       420,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:34,445,892\n",
      "INFO\t2017-01-12 15:58:04\tGenotypeConcordance\tchecked       430,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:35,297,017\n",
      "INFO\t2017-01-12 15:58:05\tGenotypeConcordance\tchecked       440,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:36,349,560\n",
      "INFO\t2017-01-12 15:58:05\tGenotypeConcordance\tchecked       450,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:37,300,320\n",
      "INFO\t2017-01-12 15:58:05\tGenotypeConcordance\tchecked       460,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:38,174,415\n",
      "INFO\t2017-01-12 15:58:05\tGenotypeConcordance\tchecked       470,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:39,100,863\n",
      "INFO\t2017-01-12 15:58:06\tGenotypeConcordance\tchecked       480,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:40,075,458\n",
      "INFO\t2017-01-12 15:58:06\tGenotypeConcordance\tchecked       490,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:40,968,201\n",
      "INFO\t2017-01-12 15:58:06\tGenotypeConcordance\tchecked       500,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:41,904,021\n",
      "INFO\t2017-01-12 15:58:07\tGenotypeConcordance\tchecked       510,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:42,843,316\n",
      "INFO\t2017-01-12 15:58:07\tGenotypeConcordance\tchecked       520,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:43,718,158\n",
      "INFO\t2017-01-12 15:58:07\tGenotypeConcordance\tchecked       530,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:44,706,113\n",
      "INFO\t2017-01-12 15:58:08\tGenotypeConcordance\tchecked       540,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:45,691,503\n",
      "INFO\t2017-01-12 15:58:08\tGenotypeConcordance\tchecked       550,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:46,757,990\n",
      "INFO\t2017-01-12 15:58:08\tGenotypeConcordance\tchecked       560,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:47,610,065\n",
      "INFO\t2017-01-12 15:58:09\tGenotypeConcordance\tchecked       570,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:48,529,955\n",
      "INFO\t2017-01-12 15:58:09\tGenotypeConcordance\tchecked       580,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:49,495,669\n",
      "INFO\t2017-01-12 15:58:09\tGenotypeConcordance\tchecked       590,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:50,815,397\n",
      "INFO\t2017-01-12 15:58:09\tGenotypeConcordance\tchecked       600,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:51,923,014\n",
      "INFO\t2017-01-12 15:58:09\tGenotypeConcordance\tchecked       610,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:52,935,423\n",
      "INFO\t2017-01-12 15:58:10\tGenotypeConcordance\tchecked       620,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:53,802,466\n",
      "INFO\t2017-01-12 15:58:10\tGenotypeConcordance\tchecked       630,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:54,708,123\n",
      "INFO\t2017-01-12 15:58:10\tGenotypeConcordance\tchecked       640,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:55,450,942\n",
      "INFO\t2017-01-12 15:58:10\tGenotypeConcordance\tchecked       650,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:56,258,074\n",
      "INFO\t2017-01-12 15:58:11\tGenotypeConcordance\tchecked       660,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:57,130,566\n",
      "INFO\t2017-01-12 15:58:11\tGenotypeConcordance\tchecked       670,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:57,986,610\n",
      "INFO\t2017-01-12 15:58:11\tGenotypeConcordance\tchecked       680,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:58,874,734\n",
      "INFO\t2017-01-12 15:58:12\tGenotypeConcordance\tchecked       690,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:59,704,642\n",
      "INFO\t2017-01-12 15:58:12\tGenotypeConcordance\tchecked       700,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:60,635,069\n",
      "INFO\t2017-01-12 15:58:12\tGenotypeConcordance\tchecked       710,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:61,525,763\n",
      "INFO\t2017-01-12 15:58:12\tGenotypeConcordance\tchecked       720,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:62,435,181\n",
      "INFO\t2017-01-12 15:58:13\tGenotypeConcordance\tchecked       730,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:63,297,853\n",
      "INFO\t2017-01-12 15:58:13\tGenotypeConcordance\tchecked       740,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:64,149,558\n",
      "INFO\t2017-01-12 15:58:13\tGenotypeConcordance\tchecked       750,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:65,061,144\n",
      "INFO\t2017-01-12 15:58:13\tGenotypeConcordance\tchecked       760,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:66,100,450\n",
      "INFO\t2017-01-12 15:58:14\tGenotypeConcordance\tchecked       770,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:67,014,855\n",
      "INFO\t2017-01-12 15:58:14\tGenotypeConcordance\tchecked       780,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:67,947,783\n",
      "INFO\t2017-01-12 15:58:14\tGenotypeConcordance\tchecked       790,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:68,799,916\n",
      "INFO\t2017-01-12 15:58:14\tGenotypeConcordance\tchecked       800,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:69,777,695\n",
      "INFO\t2017-01-12 15:58:15\tGenotypeConcordance\tchecked       810,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:70,764,703\n",
      "INFO\t2017-01-12 15:58:15\tGenotypeConcordance\tchecked       820,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:71,722,094\n",
      "INFO\t2017-01-12 15:58:15\tGenotypeConcordance\tchecked       830,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:72,751,572\n",
      "INFO\t2017-01-12 15:58:15\tGenotypeConcordance\tchecked       840,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:73,686,618\n",
      "INFO\t2017-01-12 15:58:16\tGenotypeConcordance\tchecked       850,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:74,633,797\n",
      "INFO\t2017-01-12 15:58:16\tGenotypeConcordance\tchecked       860,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:75,588,924\n",
      "INFO\t2017-01-12 15:58:16\tGenotypeConcordance\tchecked       870,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:76,479,278\n",
      "INFO\t2017-01-12 15:58:16\tGenotypeConcordance\tchecked       880,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:77,378,755\n",
      "INFO\t2017-01-12 15:58:17\tGenotypeConcordance\tchecked       890,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:78,442,124\n",
      "INFO\t2017-01-12 15:58:17\tGenotypeConcordance\tchecked       900,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:79,343,051\n",
      "INFO\t2017-01-12 15:58:17\tGenotypeConcordance\tchecked       910,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:80,107,710\n",
      "INFO\t2017-01-12 15:58:18\tGenotypeConcordance\tchecked       920,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:80,894,520\n",
      "INFO\t2017-01-12 15:58:18\tGenotypeConcordance\tchecked       930,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:81,661,481\n",
      "INFO\t2017-01-12 15:58:18\tGenotypeConcordance\tchecked       940,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:82,483,449\n",
      "INFO\t2017-01-12 15:58:18\tGenotypeConcordance\tchecked       950,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:83,299,949\n",
      "INFO\t2017-01-12 15:58:18\tGenotypeConcordance\tchecked       960,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:84,203,620\n",
      "INFO\t2017-01-12 15:58:19\tGenotypeConcordance\tchecked       970,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:85,182,774\n",
      "INFO\t2017-01-12 15:58:19\tGenotypeConcordance\tchecked       980,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:86,067,680\n",
      "INFO\t2017-01-12 15:58:19\tGenotypeConcordance\tchecked       990,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:86,962,209\n",
      "INFO\t2017-01-12 15:58:19\tGenotypeConcordance\tchecked     1,000,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:87,883,535\n",
      "INFO\t2017-01-12 15:58:20\tGenotypeConcordance\tchecked     1,010,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:88,852,286\n",
      "INFO\t2017-01-12 15:58:20\tGenotypeConcordance\tchecked     1,020,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:89,803,108\n",
      "INFO\t2017-01-12 15:58:20\tGenotypeConcordance\tchecked     1,030,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:90,776,039\n",
      "INFO\t2017-01-12 15:58:20\tGenotypeConcordance\tchecked     1,040,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:91,725,653\n",
      "INFO\t2017-01-12 15:58:21\tGenotypeConcordance\tchecked     1,050,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:92,586,331\n",
      "INFO\t2017-01-12 15:58:21\tGenotypeConcordance\tchecked     1,060,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:93,572,071\n",
      "INFO\t2017-01-12 15:58:21\tGenotypeConcordance\tchecked     1,070,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:94,567,490\n",
      "INFO\t2017-01-12 15:58:21\tGenotypeConcordance\tchecked     1,080,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:95,464,821\n",
      "INFO\t2017-01-12 15:58:22\tGenotypeConcordance\tchecked     1,090,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:96,459,037\n",
      "INFO\t2017-01-12 15:58:22\tGenotypeConcordance\tchecked     1,100,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:97,413,779\n",
      "INFO\t2017-01-12 15:58:22\tGenotypeConcordance\tchecked     1,110,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:98,403,989\n",
      "INFO\t2017-01-12 15:58:23\tGenotypeConcordance\tchecked     1,120,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:99,373,916\n",
      "INFO\t2017-01-12 15:58:23\tGenotypeConcordance\tchecked     1,130,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:100,226,223\n",
      "INFO\t2017-01-12 15:58:23\tGenotypeConcordance\tchecked     1,140,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:101,205,712\n",
      "INFO\t2017-01-12 15:58:23\tGenotypeConcordance\tchecked     1,150,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:102,069,120\n",
      "INFO\t2017-01-12 15:58:24\tGenotypeConcordance\tchecked     1,160,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:102,856,121\n",
      "INFO\t2017-01-12 15:58:24\tGenotypeConcordance\tchecked     1,170,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:103,594,920\n",
      "INFO\t2017-01-12 15:58:24\tGenotypeConcordance\tchecked     1,180,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:104,787,833\n",
      "INFO\t2017-01-12 15:58:24\tGenotypeConcordance\tchecked     1,190,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:105,436,080\n",
      "INFO\t2017-01-12 15:58:25\tGenotypeConcordance\tchecked     1,200,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:106,102,802\n",
      "INFO\t2017-01-12 15:58:25\tGenotypeConcordance\tchecked     1,210,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:106,860,402\n",
      "INFO\t2017-01-12 15:58:25\tGenotypeConcordance\tchecked     1,220,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:107,709,523\n",
      "INFO\t2017-01-12 15:58:25\tGenotypeConcordance\tchecked     1,230,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:108,611,935\n",
      "INFO\t2017-01-12 15:58:25\tGenotypeConcordance\tchecked     1,240,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:109,661,045\n",
      "INFO\t2017-01-12 15:58:26\tGenotypeConcordance\tchecked     1,250,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:110,640,998\n",
      "INFO\t2017-01-12 15:58:26\tGenotypeConcordance\tchecked     1,260,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:111,585,494\n",
      "INFO\t2017-01-12 15:58:26\tGenotypeConcordance\tchecked     1,270,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:112,468,814\n",
      "INFO\t2017-01-12 15:58:26\tGenotypeConcordance\tchecked     1,280,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:113,310,684\n",
      "INFO\t2017-01-12 15:58:27\tGenotypeConcordance\tchecked     1,290,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:114,333,600\n",
      "INFO\t2017-01-12 15:58:27\tGenotypeConcordance\tchecked     1,300,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:115,277,766\n",
      "INFO\t2017-01-12 15:58:27\tGenotypeConcordance\tchecked     1,310,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:116,173,981\n",
      "INFO\t2017-01-12 15:58:27\tGenotypeConcordance\tchecked     1,320,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:117,088,239\n",
      "INFO\t2017-01-12 15:58:28\tGenotypeConcordance\tchecked     1,330,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:118,014,306\n",
      "INFO\t2017-01-12 15:58:28\tGenotypeConcordance\tchecked     1,340,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:118,986,025\n",
      "INFO\t2017-01-12 15:58:28\tGenotypeConcordance\tchecked     1,350,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:119,895,482\n",
      "INFO\t2017-01-12 15:58:28\tGenotypeConcordance\tchecked     1,360,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:120,855,863\n",
      "INFO\t2017-01-12 15:58:29\tGenotypeConcordance\tchecked     1,370,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:142,560,186\n",
      "INFO\t2017-01-12 15:58:29\tGenotypeConcordance\tchecked     1,380,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:143,134,694\n",
      "INFO\t2017-01-12 15:58:29\tGenotypeConcordance\tchecked     1,390,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:143,518,251\n",
      "INFO\t2017-01-12 15:58:30\tGenotypeConcordance\tchecked     1,400,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:144,956,013\n",
      "INFO\t2017-01-12 15:58:30\tGenotypeConcordance\tchecked     1,410,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:145,706,219\n",
      "INFO\t2017-01-12 15:58:30\tGenotypeConcordance\tchecked     1,420,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:146,994,229\n",
      "INFO\t2017-01-12 15:58:30\tGenotypeConcordance\tchecked     1,430,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:148,558,499\n",
      "INFO\t2017-01-12 15:58:31\tGenotypeConcordance\tchecked     1,440,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:149,287,681\n",
      "INFO\t2017-01-12 15:58:31\tGenotypeConcordance\tchecked     1,450,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:150,398,945\n",
      "INFO\t2017-01-12 15:58:31\tGenotypeConcordance\tchecked     1,460,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:151,459,516\n",
      "INFO\t2017-01-12 15:58:31\tGenotypeConcordance\tchecked     1,470,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:152,433,928\n",
      "INFO\t2017-01-12 15:58:32\tGenotypeConcordance\tchecked     1,480,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:153,175,210\n",
      "INFO\t2017-01-12 15:58:32\tGenotypeConcordance\tchecked     1,490,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:154,116,128\n",
      "INFO\t2017-01-12 15:58:32\tGenotypeConcordance\tchecked     1,500,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:155,169,355\n",
      "INFO\t2017-01-12 15:58:32\tGenotypeConcordance\tchecked     1,510,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:156,316,245\n",
      "INFO\t2017-01-12 15:58:33\tGenotypeConcordance\tchecked     1,520,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:157,227,945\n",
      "INFO\t2017-01-12 15:58:33\tGenotypeConcordance\tchecked     1,530,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:158,040,895\n",
      "INFO\t2017-01-12 15:58:33\tGenotypeConcordance\tchecked     1,540,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:158,772,702\n",
      "INFO\t2017-01-12 15:58:34\tGenotypeConcordance\tchecked     1,550,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:159,678,816\n",
      "INFO\t2017-01-12 15:58:34\tGenotypeConcordance\tchecked     1,560,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:160,585,017\n",
      "INFO\t2017-01-12 15:58:34\tGenotypeConcordance\tchecked     1,570,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:161,463,235\n",
      "INFO\t2017-01-12 15:58:34\tGenotypeConcordance\tchecked     1,580,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:162,347,238\n",
      "INFO\t2017-01-12 15:58:35\tGenotypeConcordance\tchecked     1,590,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:163,202,408\n",
      "INFO\t2017-01-12 15:58:35\tGenotypeConcordance\tchecked     1,600,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:164,043,656\n",
      "INFO\t2017-01-12 15:58:35\tGenotypeConcordance\tchecked     1,610,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:164,898,936\n",
      "INFO\t2017-01-12 15:58:35\tGenotypeConcordance\tchecked     1,620,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:165,684,677\n",
      "INFO\t2017-01-12 15:58:36\tGenotypeConcordance\tchecked     1,630,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:166,521,779\n",
      "INFO\t2017-01-12 15:58:36\tGenotypeConcordance\tchecked     1,640,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:167,398,628\n",
      "INFO\t2017-01-12 15:58:36\tGenotypeConcordance\tchecked     1,650,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:168,184,589\n",
      "INFO\t2017-01-12 15:58:36\tGenotypeConcordance\tchecked     1,660,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:168,994,546\n",
      "INFO\t2017-01-12 15:58:37\tGenotypeConcordance\tchecked     1,670,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:169,850,842\n",
      "INFO\t2017-01-12 15:58:37\tGenotypeConcordance\tchecked     1,680,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:170,782,473\n",
      "INFO\t2017-01-12 15:58:37\tGenotypeConcordance\tchecked     1,690,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:171,615,936\n",
      "INFO\t2017-01-12 15:58:37\tGenotypeConcordance\tchecked     1,700,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:172,529,121\n",
      "INFO\t2017-01-12 15:58:38\tGenotypeConcordance\tchecked     1,710,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:173,534,759\n",
      "INFO\t2017-01-12 15:58:38\tGenotypeConcordance\tchecked     1,720,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:174,599,343\n",
      "INFO\t2017-01-12 15:58:38\tGenotypeConcordance\tchecked     1,730,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:175,533,921\n",
      "INFO\t2017-01-12 15:58:38\tGenotypeConcordance\tchecked     1,740,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:176,392,987\n",
      "INFO\t2017-01-12 15:58:39\tGenotypeConcordance\tchecked     1,750,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:177,478,762\n",
      "INFO\t2017-01-12 15:58:39\tGenotypeConcordance\tchecked     1,760,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:178,444,857\n",
      "INFO\t2017-01-12 15:58:39\tGenotypeConcordance\tchecked     1,770,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:179,335,138\n",
      "INFO\t2017-01-12 15:58:39\tGenotypeConcordance\tchecked     1,780,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:180,187,922\n",
      "INFO\t2017-01-12 15:58:40\tGenotypeConcordance\tchecked     1,790,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:181,078,749\n",
      "INFO\t2017-01-12 15:58:40\tGenotypeConcordance\tchecked     1,800,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:181,947,148\n",
      "INFO\t2017-01-12 15:58:40\tGenotypeConcordance\tchecked     1,810,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:182,845,174\n",
      "INFO\t2017-01-12 15:58:41\tGenotypeConcordance\tchecked     1,820,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:183,791,551\n",
      "INFO\t2017-01-12 15:58:41\tGenotypeConcordance\tchecked     1,830,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:184,717,573\n",
      "INFO\t2017-01-12 15:58:41\tGenotypeConcordance\tchecked     1,840,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:185,649,535\n",
      "INFO\t2017-01-12 15:58:41\tGenotypeConcordance\tchecked     1,850,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:186,580,648\n",
      "INFO\t2017-01-12 15:58:42\tGenotypeConcordance\tchecked     1,860,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:187,490,339\n",
      "INFO\t2017-01-12 15:58:42\tGenotypeConcordance\tchecked     1,870,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:188,255,718\n",
      "INFO\t2017-01-12 15:58:42\tGenotypeConcordance\tchecked     1,880,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:188,965,252\n",
      "INFO\t2017-01-12 15:58:42\tGenotypeConcordance\tchecked     1,890,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:189,713,805\n",
      "INFO\t2017-01-12 15:58:43\tGenotypeConcordance\tchecked     1,900,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:190,541,681\n",
      "INFO\t2017-01-12 15:58:43\tGenotypeConcordance\tchecked     1,910,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:191,390,229\n",
      "INFO\t2017-01-12 15:58:43\tGenotypeConcordance\tchecked     1,920,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:192,341,209\n",
      "INFO\t2017-01-12 15:58:43\tGenotypeConcordance\tchecked     1,930,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:193,353,750\n",
      "INFO\t2017-01-12 15:58:44\tGenotypeConcordance\tchecked     1,940,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:194,158,102\n",
      "INFO\t2017-01-12 15:58:44\tGenotypeConcordance\tchecked     1,950,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:194,936,207\n",
      "INFO\t2017-01-12 15:58:44\tGenotypeConcordance\tchecked     1,960,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:195,802,260\n",
      "INFO\t2017-01-12 15:58:44\tGenotypeConcordance\tchecked     1,970,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:196,705,654\n",
      "INFO\t2017-01-12 15:58:45\tGenotypeConcordance\tchecked     1,980,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:197,591,397\n",
      "INFO\t2017-01-12 15:58:45\tGenotypeConcordance\tchecked     1,990,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:198,546,499\n",
      "INFO\t2017-01-12 15:58:45\tGenotypeConcordance\tchecked     2,000,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:199,523,656\n",
      "INFO\t2017-01-12 15:58:45\tGenotypeConcordance\tchecked     2,010,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:200,392,412\n",
      "INFO\t2017-01-12 15:58:46\tGenotypeConcordance\tchecked     2,020,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:201,243,556\n",
      "INFO\t2017-01-12 15:58:46\tGenotypeConcordance\tchecked     2,030,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:202,089,610\n",
      "INFO\t2017-01-12 15:58:46\tGenotypeConcordance\tchecked     2,040,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:203,062,824\n",
      "INFO\t2017-01-12 15:58:47\tGenotypeConcordance\tchecked     2,050,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:203,890,477\n",
      "INFO\t2017-01-12 15:58:47\tGenotypeConcordance\tchecked     2,060,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:204,741,992\n",
      "INFO\t2017-01-12 15:58:47\tGenotypeConcordance\tchecked     2,070,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:205,615,287\n",
      "INFO\t2017-01-12 15:58:47\tGenotypeConcordance\tchecked     2,080,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:206,882,135\n",
      "INFO\t2017-01-12 15:58:48\tGenotypeConcordance\tchecked     2,090,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:207,908,172\n",
      "INFO\t2017-01-12 15:58:48\tGenotypeConcordance\tchecked     2,100,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:208,782,188\n",
      "INFO\t2017-01-12 15:58:48\tGenotypeConcordance\tchecked     2,110,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:209,634,321\n",
      "INFO\t2017-01-12 15:58:49\tGenotypeConcordance\tchecked     2,120,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:210,528,934\n",
      "INFO\t2017-01-12 15:58:49\tGenotypeConcordance\tchecked     2,130,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:211,348,493\n",
      "INFO\t2017-01-12 15:58:49\tGenotypeConcordance\tchecked     2,140,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:212,233,950\n",
      "INFO\t2017-01-12 15:58:50\tGenotypeConcordance\tchecked     2,150,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:213,102,148\n",
      "INFO\t2017-01-12 15:58:50\tGenotypeConcordance\tchecked     2,160,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:214,041,894\n",
      "INFO\t2017-01-12 15:58:50\tGenotypeConcordance\tchecked     2,170,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:214,885,561\n",
      "INFO\t2017-01-12 15:58:50\tGenotypeConcordance\tchecked     2,180,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:215,832,378\n",
      "INFO\t2017-01-12 15:58:50\tGenotypeConcordance\tchecked     2,190,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:216,662,472\n",
      "INFO\t2017-01-12 15:58:51\tGenotypeConcordance\tchecked     2,200,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:217,442,851\n",
      "INFO\t2017-01-12 15:58:51\tGenotypeConcordance\tchecked     2,210,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:218,264,979\n",
      "INFO\t2017-01-12 15:58:51\tGenotypeConcordance\tchecked     2,220,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:219,125,281\n",
      "INFO\t2017-01-12 15:58:52\tGenotypeConcordance\tchecked     2,230,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:220,050,341\n",
      "INFO\t2017-01-12 15:58:52\tGenotypeConcordance\tchecked     2,240,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:220,962,453\n",
      "INFO\t2017-01-12 15:58:52\tGenotypeConcordance\tchecked     2,250,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:221,845,715\n",
      "INFO\t2017-01-12 15:58:52\tGenotypeConcordance\tchecked     2,260,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:222,741,944\n",
      "INFO\t2017-01-12 15:58:53\tGenotypeConcordance\tchecked     2,270,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:223,647,626\n",
      "INFO\t2017-01-12 15:58:53\tGenotypeConcordance\tchecked     2,280,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:224,520,873\n",
      "INFO\t2017-01-12 15:58:53\tGenotypeConcordance\tchecked     2,290,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:225,434,050\n",
      "INFO\t2017-01-12 15:58:53\tGenotypeConcordance\tchecked     2,300,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:226,371,672\n",
      "INFO\t2017-01-12 15:58:54\tGenotypeConcordance\tchecked     2,310,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:227,320,600\n",
      "INFO\t2017-01-12 15:58:54\tGenotypeConcordance\tchecked     2,320,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:228,129,263\n",
      "INFO\t2017-01-12 15:58:54\tGenotypeConcordance\tchecked     2,330,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:229,027,774\n",
      "INFO\t2017-01-12 15:58:55\tGenotypeConcordance\tchecked     2,340,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:229,841,108\n",
      "INFO\t2017-01-12 15:58:55\tGenotypeConcordance\tchecked     2,350,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:230,622,939\n",
      "INFO\t2017-01-12 15:58:55\tGenotypeConcordance\tchecked     2,360,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:231,468,446\n",
      "INFO\t2017-01-12 15:58:55\tGenotypeConcordance\tchecked     2,370,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:232,321,862\n",
      "INFO\t2017-01-12 15:58:56\tGenotypeConcordance\tchecked     2,380,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:233,153,225\n",
      "INFO\t2017-01-12 15:58:56\tGenotypeConcordance\tchecked     2,390,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:233,956,282\n",
      "INFO\t2017-01-12 15:58:56\tGenotypeConcordance\tchecked     2,400,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:234,728,108\n",
      "INFO\t2017-01-12 15:58:56\tGenotypeConcordance\tchecked     2,410,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:235,631,568\n",
      "INFO\t2017-01-12 15:58:57\tGenotypeConcordance\tchecked     2,420,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:236,469,498\n",
      "INFO\t2017-01-12 15:58:57\tGenotypeConcordance\tchecked     2,430,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:237,172,962\n",
      "INFO\t2017-01-12 15:58:57\tGenotypeConcordance\tchecked     2,440,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:237,962,535\n",
      "INFO\t2017-01-12 15:58:58\tGenotypeConcordance\tchecked     2,450,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:238,728,187\n",
      "INFO\t2017-01-12 15:58:58\tGenotypeConcordance\tchecked     2,460,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:239,511,263\n",
      "INFO\t2017-01-12 15:58:58\tGenotypeConcordance\tchecked     2,470,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:240,311,555\n",
      "INFO\t2017-01-12 15:58:58\tGenotypeConcordance\tchecked     2,480,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:241,083,566\n",
      "INFO\t2017-01-12 15:58:59\tGenotypeConcordance\tchecked     2,490,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:241,845,577\n",
      "INFO\t2017-01-12 15:58:59\tGenotypeConcordance\tchecked     2,500,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:242,532,846\n",
      "INFO\t2017-01-12 15:58:59\tGenotypeConcordance\tchecked     2,510,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:243,261,977\n",
      "INFO\t2017-01-12 15:59:00\tGenotypeConcordance\tchecked     2,520,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:244,311,942\n",
      "INFO\t2017-01-12 15:59:00\tGenotypeConcordance\tchecked     2,530,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:245,077,322\n",
      "INFO\t2017-01-12 15:59:00\tGenotypeConcordance\tchecked     2,540,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:245,800,205\n",
      "INFO\t2017-01-12 15:59:00\tGenotypeConcordance\tchecked     2,550,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:246,454,550\n",
      "INFO\t2017-01-12 15:59:01\tGenotypeConcordance\tchecked     2,560,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:247,179,859\n",
      "INFO\t2017-01-12 15:59:01\tGenotypeConcordance\tchecked     2,570,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:247,891,094\n",
      "INFO\t2017-01-12 15:59:01\tGenotypeConcordance\tchecked     2,580,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:248,544,206\n",
      "INFO\t2017-01-12 15:59:02\tGenotypeConcordance\tchecked     2,590,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 2:94,559\n",
      "INFO\t2017-01-12 15:59:02\tGenotypeConcordance\tchecked     2,600,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 2:837,910\n",
      "INFO\t2017-01-12 15:59:02\tGenotypeConcordance\tchecked     2,610,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 2:1,455,313\n",
      "INFO\t2017-01-12 15:59:02\tGenotypeConcordance\tchecked     2,620,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 2:2,113,647\n",
      "INFO\t2017-01-12 15:59:03\tGenotypeConcordance\tchecked     2,630,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 2:2,833,582\n",
      "INFO\t2017-01-12 15:59:03\tGenotypeConcordance\tchecked     2,640,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 2:3,609,598\n",
      "INFO\t2017-01-12 15:59:03\tGenotypeConcordance\tchecked     2,650,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 2:4,320,485\n",
      "INFO\t2017-01-12 15:59:03\tGenotypeConcordance\tchecked     2,660,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 2:5,168,050\n",
      "INFO\t2017-01-12 15:59:04\tGenotypeConcordance\tchecked     2,670,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 2:5,929,456\n",
      "INFO\t2017-01-12 15:59:04\tGenotypeConcordance\tchecked     2,680,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 2:6,651,520\n",
      "INFO\t2017-01-12 15:59:04\tGenotypeConcordance\tchecked     2,690,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 2:7,475,285\n",
      "INFO\t2017-01-12 15:59:04\tGenotypeConcordance\tchecked     2,700,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 2:8,297,461\n",
      "INFO\t2017-01-12 15:59:05\tGenotypeConcordance\tchecked     2,710,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 2:9,191,332\n",
      "INFO\t2017-01-12 15:59:05\tGenotypeConcordance\tchecked     2,720,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 2:9,995,071\n",
      "INFO\t2017-01-12 15:59:05\tGenotypeConcordance\tchecked     2,730,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 2:10,710,145\n",
      "INFO\t2017-01-12 15:59:06\tGenotypeConcordance\tchecked     2,740,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 2:11,442,442\n",
      "INFO\t2017-01-12 15:59:06\tGenotypeConcordance\tchecked     2,750,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 2:12,208,518\n",
      "INFO\t2017-01-12 15:59:06\tGenotypeConcordance\tchecked     2,760,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 2:13,082,247\n",
      "INFO\t2017-01-12 15:59:06\tGenotypeConcordance\tchecked     2,770,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 2:13,861,615\n",
      "INFO\t2017-01-12 15:59:07\tGenotypeConcordance\tchecked     2,780,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:14,651,618\n",
      "INFO\t2017-01-12 15:59:07\tGenotypeConcordance\tchecked     2,790,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:15,509,146\n",
      "INFO\t2017-01-12 15:59:07\tGenotypeConcordance\tchecked     2,800,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:16,274,500\n",
      "INFO\t2017-01-12 15:59:07\tGenotypeConcordance\tchecked     2,810,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 2:17,168,614\n",
      "INFO\t2017-01-12 15:59:08\tGenotypeConcordance\tchecked     2,820,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:18,102,927\n",
      "INFO\t2017-01-12 15:59:08\tGenotypeConcordance\tchecked     2,830,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:18,909,666\n",
      "INFO\t2017-01-12 15:59:08\tGenotypeConcordance\tchecked     2,840,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 2:19,850,364\n",
      "INFO\t2017-01-12 15:59:08\tGenotypeConcordance\tchecked     2,850,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:20,700,647\n",
      "INFO\t2017-01-12 15:59:09\tGenotypeConcordance\tchecked     2,860,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:21,512,069\n",
      "INFO\t2017-01-12 15:59:09\tGenotypeConcordance\tchecked     2,870,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:22,460,536\n",
      "INFO\t2017-01-12 15:59:09\tGenotypeConcordance\tchecked     2,880,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:23,331,614\n",
      "INFO\t2017-01-12 15:59:09\tGenotypeConcordance\tchecked     2,890,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 2:24,242,812\n",
      "INFO\t2017-01-12 15:59:10\tGenotypeConcordance\tchecked     2,900,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:25,284,933\n",
      "INFO\t2017-01-12 15:59:10\tGenotypeConcordance\tchecked     2,910,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:26,304,419\n",
      "INFO\t2017-01-12 15:59:10\tGenotypeConcordance\tchecked     2,920,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:27,213,937\n",
      "INFO\t2017-01-12 15:59:10\tGenotypeConcordance\tchecked     2,930,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 2:28,384,132\n",
      "INFO\t2017-01-12 15:59:11\tGenotypeConcordance\tchecked     2,940,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:29,271,357\n",
      "INFO\t2017-01-12 15:59:11\tGenotypeConcordance\tchecked     2,950,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:30,051,408\n",
      "INFO\t2017-01-12 15:59:11\tGenotypeConcordance\tchecked     2,960,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 2:30,862,245\n",
      "INFO\t2017-01-12 15:59:11\tGenotypeConcordance\tchecked     2,970,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:31,586,861\n",
      "INFO\t2017-01-12 15:59:12\tGenotypeConcordance\tchecked     2,980,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:32,555,031\n",
      "INFO\t2017-01-12 15:59:12\tGenotypeConcordance\tchecked     2,990,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:33,390,728\n",
      "INFO\t2017-01-12 15:59:12\tGenotypeConcordance\tchecked     3,000,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 2:34,029,312\n",
      "INFO\t2017-01-12 15:59:12\tGenotypeConcordance\tchecked     3,010,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:34,626,685\n",
      "INFO\t2017-01-12 15:59:13\tGenotypeConcordance\tchecked     3,020,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:35,300,605\n",
      "INFO\t2017-01-12 15:59:13\tGenotypeConcordance\tchecked     3,030,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:35,994,111\n",
      "INFO\t2017-01-12 15:59:13\tGenotypeConcordance\tchecked     3,040,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 2:36,699,168\n",
      "INFO\t2017-01-12 15:59:13\tGenotypeConcordance\tchecked     3,050,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:37,453,581\n",
      "INFO\t2017-01-12 15:59:14\tGenotypeConcordance\tchecked     3,060,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:38,024,047\n",
      "INFO\t2017-01-12 15:59:14\tGenotypeConcordance\tchecked     3,070,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:38,691,187\n",
      "INFO\t2017-01-12 15:59:14\tGenotypeConcordance\tchecked     3,080,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 2:39,523,829\n",
      "INFO\t2017-01-12 15:59:14\tGenotypeConcordance\tchecked     3,090,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:40,295,627\n",
      "INFO\t2017-01-12 15:59:15\tGenotypeConcordance\tchecked     3,100,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:41,048,757\n",
      "INFO\t2017-01-12 15:59:15\tGenotypeConcordance\tchecked     3,110,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:41,786,707\n",
      "INFO\t2017-01-12 15:59:15\tGenotypeConcordance\tchecked     3,120,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:42,459,288\n",
      "INFO\t2017-01-12 15:59:16\tGenotypeConcordance\tchecked     3,130,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:43,269,737\n",
      "INFO\t2017-01-12 15:59:16\tGenotypeConcordance\tchecked     3,140,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:44,027,827\n",
      "INFO\t2017-01-12 15:59:16\tGenotypeConcordance\tchecked     3,150,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:44,763,797\n",
      "INFO\t2017-01-12 15:59:16\tGenotypeConcordance\tchecked     3,160,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:45,546,836\n",
      "INFO\t2017-01-12 15:59:17\tGenotypeConcordance\tchecked     3,170,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:46,363,699\n",
      "INFO\t2017-01-12 15:59:17\tGenotypeConcordance\tchecked     3,180,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:47,118,531\n",
      "INFO\t2017-01-12 15:59:17\tGenotypeConcordance\tchecked     3,190,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:47,813,742\n",
      "INFO\t2017-01-12 15:59:17\tGenotypeConcordance\tchecked     3,200,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:48,663,410\n",
      "INFO\t2017-01-12 15:59:18\tGenotypeConcordance\tchecked     3,210,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:49,407,380\n",
      "INFO\t2017-01-12 15:59:18\tGenotypeConcordance\tchecked     3,220,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:50,178,059\n",
      "INFO\t2017-01-12 15:59:18\tGenotypeConcordance\tchecked     3,230,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:50,987,496\n",
      "INFO\t2017-01-12 15:59:18\tGenotypeConcordance\tchecked     3,240,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:51,778,185\n",
      "INFO\t2017-01-12 15:59:19\tGenotypeConcordance\tchecked     3,250,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:52,480,847\n",
      "INFO\t2017-01-12 15:59:19\tGenotypeConcordance\tchecked     3,260,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:53,100,764\n",
      "INFO\t2017-01-12 15:59:19\tGenotypeConcordance\tchecked     3,270,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:53,780,128\n",
      "INFO\t2017-01-12 15:59:20\tGenotypeConcordance\tchecked     3,280,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:54,605,148\n",
      "INFO\t2017-01-12 15:59:20\tGenotypeConcordance\tchecked     3,290,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:55,413,056\n",
      "INFO\t2017-01-12 15:59:20\tGenotypeConcordance\tchecked     3,300,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:56,271,006\n",
      "INFO\t2017-01-12 15:59:21\tGenotypeConcordance\tchecked     3,310,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:56,996,573\n",
      "INFO\t2017-01-12 15:59:21\tGenotypeConcordance\tchecked     3,320,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:57,677,079\n",
      "INFO\t2017-01-12 15:59:21\tGenotypeConcordance\tchecked     3,330,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:58,541,238\n",
      "INFO\t2017-01-12 15:59:21\tGenotypeConcordance\tchecked     3,340,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:59,526,280\n",
      "INFO\t2017-01-12 15:59:22\tGenotypeConcordance\tchecked     3,350,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:60,364,993\n",
      "INFO\t2017-01-12 15:59:22\tGenotypeConcordance\tchecked     3,360,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:61,323,655\n",
      "INFO\t2017-01-12 15:59:22\tGenotypeConcordance\tchecked     3,370,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:62,260,132\n",
      "INFO\t2017-01-12 15:59:22\tGenotypeConcordance\tchecked     3,380,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:63,330,513\n",
      "INFO\t2017-01-12 15:59:23\tGenotypeConcordance\tchecked     3,390,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:64,467,184\n",
      "INFO\t2017-01-12 15:59:23\tGenotypeConcordance\tchecked     3,400,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:65,346,733\n",
      "INFO\t2017-01-12 15:59:23\tGenotypeConcordance\tchecked     3,410,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:66,185,353\n",
      "INFO\t2017-01-12 15:59:24\tGenotypeConcordance\tchecked     3,420,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:67,072,585\n",
      "INFO\t2017-01-12 15:59:24\tGenotypeConcordance\tchecked     3,430,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:67,925,384\n",
      "INFO\t2017-01-12 15:59:24\tGenotypeConcordance\tchecked     3,440,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:68,716,263\n",
      "INFO\t2017-01-12 15:59:24\tGenotypeConcordance\tchecked     3,450,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:69,552,521\n",
      "INFO\t2017-01-12 15:59:25\tGenotypeConcordance\tchecked     3,460,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:70,460,722\n",
      "INFO\t2017-01-12 15:59:25\tGenotypeConcordance\tchecked     3,470,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:71,249,509\n",
      "INFO\t2017-01-12 15:59:25\tGenotypeConcordance\tchecked     3,480,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:72,040,506\n",
      "INFO\t2017-01-12 15:59:26\tGenotypeConcordance\tchecked     3,490,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:73,127,224\n",
      "INFO\t2017-01-12 15:59:26\tGenotypeConcordance\tchecked     3,500,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:74,131,066\n",
      "INFO\t2017-01-12 15:59:26\tGenotypeConcordance\tchecked     3,510,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:75,111,166\n",
      "INFO\t2017-01-12 15:59:27\tGenotypeConcordance\tchecked     3,520,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:75,909,079\n",
      "INFO\t2017-01-12 15:59:27\tGenotypeConcordance\tchecked     3,530,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:76,669,171\n",
      "INFO\t2017-01-12 15:59:27\tGenotypeConcordance\tchecked     3,540,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:77,310,158\n",
      "INFO\t2017-01-12 15:59:28\tGenotypeConcordance\tchecked     3,550,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:78,049,271\n",
      "INFO\t2017-01-12 15:59:28\tGenotypeConcordance\tchecked     3,560,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:78,848,211\n",
      "INFO\t2017-01-12 15:59:28\tGenotypeConcordance\tchecked     3,570,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:79,617,030\n",
      "INFO\t2017-01-12 15:59:29\tGenotypeConcordance\tchecked     3,580,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:80,391,778\n",
      "INFO\t2017-01-12 15:59:29\tGenotypeConcordance\tchecked     3,590,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:81,215,144\n",
      "INFO\t2017-01-12 15:59:29\tGenotypeConcordance\tchecked     3,600,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:82,049,582\n",
      "INFO\t2017-01-12 15:59:30\tGenotypeConcordance\tchecked     3,610,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:82,955,443\n",
      "INFO\t2017-01-12 15:59:30\tGenotypeConcordance\tchecked     3,620,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:83,808,468\n",
      "INFO\t2017-01-12 15:59:30\tGenotypeConcordance\tchecked     3,630,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:84,673,058\n",
      "INFO\t2017-01-12 15:59:30\tGenotypeConcordance\tchecked     3,640,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:85,615,083\n",
      "INFO\t2017-01-12 15:59:31\tGenotypeConcordance\tchecked     3,650,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:86,501,141\n",
      "INFO\t2017-01-12 15:59:31\tGenotypeConcordance\tchecked     3,660,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:87,574,999\n",
      "INFO\t2017-01-12 15:59:31\tGenotypeConcordance\tchecked     3,670,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:88,668,496\n",
      "INFO\t2017-01-12 15:59:31\tGenotypeConcordance\tchecked     3,680,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:89,449,714\n",
      "INFO\t2017-01-12 15:59:32\tGenotypeConcordance\tchecked     3,690,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:91,634,435\n",
      "INFO\t2017-01-12 15:59:32\tGenotypeConcordance\tchecked     3,700,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:92,132,836\n",
      "INFO\t2017-01-12 15:59:32\tGenotypeConcordance\tchecked     3,710,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:95,489,168\n",
      "INFO\t2017-01-12 15:59:33\tGenotypeConcordance\tchecked     3,720,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:96,762,284\n",
      "INFO\t2017-01-12 15:59:33\tGenotypeConcordance\tchecked     3,730,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:97,863,112\n",
      "INFO\t2017-01-12 15:59:33\tGenotypeConcordance\tchecked     3,740,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:99,055,792\n",
      "INFO\t2017-01-12 15:59:34\tGenotypeConcordance\tchecked     3,750,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:100,134,730\n",
      "INFO\t2017-01-12 15:59:34\tGenotypeConcordance\tchecked     3,760,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:101,070,493\n",
      "INFO\t2017-01-12 15:59:34\tGenotypeConcordance\tchecked     3,770,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:101,865,270\n",
      "INFO\t2017-01-12 15:59:34\tGenotypeConcordance\tchecked     3,780,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:102,727,198\n",
      "INFO\t2017-01-12 15:59:35\tGenotypeConcordance\tchecked     3,790,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:103,625,469\n",
      "INFO\t2017-01-12 15:59:35\tGenotypeConcordance\tchecked     3,800,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:104,550,916\n",
      "INFO\t2017-01-12 15:59:35\tGenotypeConcordance\tchecked     3,810,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:105,447,029\n",
      "INFO\t2017-01-12 15:59:35\tGenotypeConcordance\tchecked     3,820,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:106,265,647\n",
      "INFO\t2017-01-12 15:59:36\tGenotypeConcordance\tchecked     3,830,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:107,073,736\n",
      "INFO\t2017-01-12 15:59:36\tGenotypeConcordance\tchecked     3,840,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:107,908,532\n",
      "INFO\t2017-01-12 15:59:36\tGenotypeConcordance\tchecked     3,850,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:108,821,619\n",
      "INFO\t2017-01-12 15:59:36\tGenotypeConcordance\tchecked     3,860,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:109,685,534\n",
      "INFO\t2017-01-12 15:59:37\tGenotypeConcordance\tchecked     3,870,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:111,052,113\n",
      "INFO\t2017-01-12 15:59:37\tGenotypeConcordance\tchecked     3,880,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:112,344,713\n",
      "INFO\t2017-01-12 15:59:37\tGenotypeConcordance\tchecked     3,890,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:113,309,969\n",
      "INFO\t2017-01-12 15:59:37\tGenotypeConcordance\tchecked     3,900,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:114,178,591\n",
      "INFO\t2017-01-12 15:59:38\tGenotypeConcordance\tchecked     3,910,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:115,045,896\n",
      "INFO\t2017-01-12 15:59:38\tGenotypeConcordance\tchecked     3,920,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:115,933,697\n",
      "INFO\t2017-01-12 15:59:38\tGenotypeConcordance\tchecked     3,930,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:116,831,339\n",
      "INFO\t2017-01-12 15:59:38\tGenotypeConcordance\tchecked     3,940,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:117,629,795\n",
      "INFO\t2017-01-12 15:59:39\tGenotypeConcordance\tchecked     3,950,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:118,364,176\n",
      "INFO\t2017-01-12 15:59:39\tGenotypeConcordance\tchecked     3,960,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:119,197,552\n",
      "INFO\t2017-01-12 15:59:39\tGenotypeConcordance\tchecked     3,970,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:120,018,077\n",
      "INFO\t2017-01-12 15:59:39\tGenotypeConcordance\tchecked     3,980,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:121,022,354\n",
      "INFO\t2017-01-12 15:59:40\tGenotypeConcordance\tchecked     3,990,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:121,816,314\n",
      "INFO\t2017-01-12 15:59:40\tGenotypeConcordance\tchecked     4,000,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:122,669,673\n",
      "INFO\t2017-01-12 15:59:40\tGenotypeConcordance\tchecked     4,010,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:123,573,226\n",
      "INFO\t2017-01-12 15:59:40\tGenotypeConcordance\tchecked     4,020,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:124,315,714\n",
      "INFO\t2017-01-12 15:59:41\tGenotypeConcordance\tchecked     4,030,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:125,130,263\n",
      "INFO\t2017-01-12 15:59:41\tGenotypeConcordance\tchecked     4,040,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:125,965,376\n",
      "INFO\t2017-01-12 15:59:41\tGenotypeConcordance\tchecked     4,050,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:126,776,164\n",
      "INFO\t2017-01-12 15:59:41\tGenotypeConcordance\tchecked     4,060,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:127,580,979\n",
      "INFO\t2017-01-12 15:59:42\tGenotypeConcordance\tchecked     4,070,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:128,388,902\n",
      "INFO\t2017-01-12 15:59:42\tGenotypeConcordance\tchecked     4,080,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:129,225,828\n",
      "INFO\t2017-01-12 15:59:42\tGenotypeConcordance\tchecked     4,090,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:130,003,278\n",
      "INFO\t2017-01-12 15:59:42\tGenotypeConcordance\tchecked     4,100,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:130,795,561\n",
      "INFO\t2017-01-12 15:59:43\tGenotypeConcordance\tchecked     4,110,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:131,710,030\n",
      "INFO\t2017-01-12 15:59:43\tGenotypeConcordance\tchecked     4,120,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:132,424,600\n",
      "INFO\t2017-01-12 15:59:43\tGenotypeConcordance\tchecked     4,130,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:132,978,093\n",
      "INFO\t2017-01-12 15:59:43\tGenotypeConcordance\tchecked     4,140,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:133,345,134\n",
      "INFO\t2017-01-12 15:59:44\tGenotypeConcordance\tchecked     4,150,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:134,247,340\n",
      "INFO\t2017-01-12 15:59:44\tGenotypeConcordance\tchecked     4,160,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:135,059,317\n",
      "INFO\t2017-01-12 15:59:44\tGenotypeConcordance\tchecked     4,170,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:136,005,966\n",
      "INFO\t2017-01-12 15:59:44\tGenotypeConcordance\tchecked     4,180,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:137,028,225\n",
      "INFO\t2017-01-12 15:59:45\tGenotypeConcordance\tchecked     4,190,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:137,826,312\n",
      "INFO\t2017-01-12 15:59:45\tGenotypeConcordance\tchecked     4,200,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:138,609,631\n",
      "INFO\t2017-01-12 15:59:45\tGenotypeConcordance\tchecked     4,210,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:139,499,493\n",
      "INFO\t2017-01-12 15:59:45\tGenotypeConcordance\tchecked     4,220,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:140,328,107\n",
      "INFO\t2017-01-12 15:59:46\tGenotypeConcordance\tchecked     4,230,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:141,130,695\n",
      "INFO\t2017-01-12 15:59:46\tGenotypeConcordance\tchecked     4,240,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:141,872,447\n",
      "INFO\t2017-01-12 15:59:46\tGenotypeConcordance\tchecked     4,250,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:142,585,797\n",
      "INFO\t2017-01-12 15:59:46\tGenotypeConcordance\tchecked     4,260,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:143,476,813\n",
      "INFO\t2017-01-12 15:59:47\tGenotypeConcordance\tchecked     4,270,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:144,483,428\n",
      "INFO\t2017-01-12 15:59:47\tGenotypeConcordance\tchecked     4,280,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:145,702,623\n",
      "INFO\t2017-01-12 15:59:47\tGenotypeConcordance\tchecked     4,290,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:146,688,483\n",
      "INFO\t2017-01-12 15:59:47\tGenotypeConcordance\tchecked     4,300,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:147,757,946\n",
      "INFO\t2017-01-12 15:59:47\tGenotypeConcordance\tchecked     4,310,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:148,865,167\n",
      "INFO\t2017-01-12 15:59:48\tGenotypeConcordance\tchecked     4,320,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:149,967,939\n",
      "INFO\t2017-01-12 15:59:48\tGenotypeConcordance\tchecked     4,330,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:150,885,168\n",
      "INFO\t2017-01-12 15:59:48\tGenotypeConcordance\tchecked     4,340,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:151,825,296\n",
      "INFO\t2017-01-12 15:59:48\tGenotypeConcordance\tchecked     4,350,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:152,751,141\n",
      "INFO\t2017-01-12 15:59:49\tGenotypeConcordance\tchecked     4,360,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:153,695,805\n",
      "INFO\t2017-01-12 15:59:49\tGenotypeConcordance\tchecked     4,370,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:154,529,360\n",
      "INFO\t2017-01-12 15:59:49\tGenotypeConcordance\tchecked     4,380,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:155,443,439\n",
      "INFO\t2017-01-12 15:59:49\tGenotypeConcordance\tchecked     4,390,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:156,394,351\n",
      "INFO\t2017-01-12 15:59:50\tGenotypeConcordance\tchecked     4,400,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:157,529,424\n",
      "INFO\t2017-01-12 15:59:50\tGenotypeConcordance\tchecked     4,410,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:158,604,128\n",
      "INFO\t2017-01-12 15:59:50\tGenotypeConcordance\tchecked     4,420,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:159,476,445\n",
      "INFO\t2017-01-12 15:59:50\tGenotypeConcordance\tchecked     4,430,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:160,325,118\n",
      "INFO\t2017-01-12 15:59:51\tGenotypeConcordance\tchecked     4,440,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:161,357,258\n",
      "INFO\t2017-01-12 15:59:51\tGenotypeConcordance\tchecked     4,450,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:162,321,436\n",
      "INFO\t2017-01-12 15:59:51\tGenotypeConcordance\tchecked     4,460,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:163,400,015\n",
      "INFO\t2017-01-12 15:59:51\tGenotypeConcordance\tchecked     4,470,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:164,328,393\n",
      "INFO\t2017-01-12 15:59:52\tGenotypeConcordance\tchecked     4,480,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:165,295,123\n",
      "INFO\t2017-01-12 15:59:52\tGenotypeConcordance\tchecked     4,490,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:166,285,342\n",
      "INFO\t2017-01-12 15:59:52\tGenotypeConcordance\tchecked     4,500,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:167,191,274\n",
      "INFO\t2017-01-12 15:59:52\tGenotypeConcordance\tchecked     4,510,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:167,990,754\n",
      "INFO\t2017-01-12 15:59:53\tGenotypeConcordance\tchecked     4,520,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:168,825,793\n",
      "INFO\t2017-01-12 15:59:53\tGenotypeConcordance\tchecked     4,530,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:169,689,042\n",
      "INFO\t2017-01-12 15:59:53\tGenotypeConcordance\tchecked     4,540,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:170,536,190\n",
      "INFO\t2017-01-12 15:59:53\tGenotypeConcordance\tchecked     4,550,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:171,334,858\n",
      "INFO\t2017-01-12 15:59:54\tGenotypeConcordance\tchecked     4,560,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:172,242,147\n",
      "INFO\t2017-01-12 15:59:54\tGenotypeConcordance\tchecked     4,570,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:173,138,239\n",
      "INFO\t2017-01-12 15:59:54\tGenotypeConcordance\tchecked     4,580,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:173,969,044\n",
      "INFO\t2017-01-12 15:59:54\tGenotypeConcordance\tchecked     4,590,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:174,823,687\n",
      "INFO\t2017-01-12 15:59:55\tGenotypeConcordance\tchecked     4,600,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:175,815,803\n",
      "INFO\t2017-01-12 15:59:55\tGenotypeConcordance\tchecked     4,610,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:176,740,621\n",
      "INFO\t2017-01-12 15:59:55\tGenotypeConcordance\tchecked     4,620,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:177,642,869\n",
      "INFO\t2017-01-12 15:59:55\tGenotypeConcordance\tchecked     4,630,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:178,545,934\n",
      "INFO\t2017-01-12 15:59:56\tGenotypeConcordance\tchecked     4,640,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:179,541,612\n",
      "INFO\t2017-01-12 15:59:56\tGenotypeConcordance\tchecked     4,650,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:180,421,557\n",
      "INFO\t2017-01-12 15:59:56\tGenotypeConcordance\tchecked     4,660,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:181,281,745\n",
      "INFO\t2017-01-12 15:59:56\tGenotypeConcordance\tchecked     4,670,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:182,215,203\n",
      "INFO\t2017-01-12 15:59:57\tGenotypeConcordance\tchecked     4,680,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:183,182,776\n",
      "INFO\t2017-01-12 15:59:57\tGenotypeConcordance\tchecked     4,690,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:184,083,649\n",
      "INFO\t2017-01-12 15:59:57\tGenotypeConcordance\tchecked     4,700,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:184,938,143\n",
      "INFO\t2017-01-12 15:59:58\tGenotypeConcordance\tchecked     4,710,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:185,881,295\n",
      "INFO\t2017-01-12 15:59:58\tGenotypeConcordance\tchecked     4,720,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:186,928,089\n",
      "INFO\t2017-01-12 15:59:58\tGenotypeConcordance\tchecked     4,730,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:187,898,064\n",
      "INFO\t2017-01-12 15:59:58\tGenotypeConcordance\tchecked     4,740,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:188,787,991\n",
      "INFO\t2017-01-12 15:59:59\tGenotypeConcordance\tchecked     4,750,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:189,778,384\n",
      "INFO\t2017-01-12 15:59:59\tGenotypeConcordance\tchecked     4,760,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:190,783,398\n",
      "INFO\t2017-01-12 15:59:59\tGenotypeConcordance\tchecked     4,770,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:191,766,693\n",
      "INFO\t2017-01-12 15:59:59\tGenotypeConcordance\tchecked     4,780,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:192,696,437\n",
      "INFO\t2017-01-12 16:00:00\tGenotypeConcordance\tchecked     4,790,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:193,648,710\n",
      "INFO\t2017-01-12 16:00:00\tGenotypeConcordance\tchecked     4,800,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:194,694,625\n",
      "INFO\t2017-01-12 16:00:00\tGenotypeConcordance\tchecked     4,810,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:195,615,120\n",
      "INFO\t2017-01-12 16:00:00\tGenotypeConcordance\tchecked     4,820,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:196,443,136\n",
      "INFO\t2017-01-12 16:00:01\tGenotypeConcordance\tchecked     4,830,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:197,361,687\n",
      "INFO\t2017-01-12 16:00:01\tGenotypeConcordance\tchecked     4,840,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:198,398,651\n",
      "INFO\t2017-01-12 16:00:01\tGenotypeConcordance\tchecked     4,850,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:199,395,615\n",
      "INFO\t2017-01-12 16:00:01\tGenotypeConcordance\tchecked     4,860,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:200,486,639\n",
      "INFO\t2017-01-12 16:00:02\tGenotypeConcordance\tchecked     4,870,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:201,601,564\n",
      "INFO\t2017-01-12 16:00:02\tGenotypeConcordance\tchecked     4,880,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:202,646,115\n",
      "INFO\t2017-01-12 16:00:02\tGenotypeConcordance\tchecked     4,890,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:203,697,319\n",
      "INFO\t2017-01-12 16:00:02\tGenotypeConcordance\tchecked     4,900,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:204,828,870\n",
      "INFO\t2017-01-12 16:00:03\tGenotypeConcordance\tchecked     4,910,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:205,748,607\n",
      "INFO\t2017-01-12 16:00:03\tGenotypeConcordance\tchecked     4,920,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:206,669,887\n",
      "INFO\t2017-01-12 16:00:03\tGenotypeConcordance\tchecked     4,930,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:207,607,797\n",
      "INFO\t2017-01-12 16:00:03\tGenotypeConcordance\tchecked     4,940,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:208,565,664\n",
      "INFO\t2017-01-12 16:00:04\tGenotypeConcordance\tchecked     4,950,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:209,454,495\n",
      "INFO\t2017-01-12 16:00:04\tGenotypeConcordance\tchecked     4,960,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:210,203,064\n",
      "INFO\t2017-01-12 16:00:04\tGenotypeConcordance\tchecked     4,970,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:211,270,752\n",
      "INFO\t2017-01-12 16:00:04\tGenotypeConcordance\tchecked     4,980,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:212,081,721\n",
      "INFO\t2017-01-12 16:00:05\tGenotypeConcordance\tchecked     4,990,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:212,838,216\n",
      "INFO\t2017-01-12 16:00:05\tGenotypeConcordance\tchecked     5,000,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:213,680,033\n",
      "INFO\t2017-01-12 16:00:05\tGenotypeConcordance\tchecked     5,010,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:214,658,378\n",
      "INFO\t2017-01-12 16:00:05\tGenotypeConcordance\tchecked     5,020,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:215,532,581\n",
      "INFO\t2017-01-12 16:00:06\tGenotypeConcordance\tchecked     5,030,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:216,350,538\n",
      "INFO\t2017-01-12 16:00:06\tGenotypeConcordance\tchecked     5,040,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:217,243,369\n",
      "INFO\t2017-01-12 16:00:06\tGenotypeConcordance\tchecked     5,050,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:218,084,158\n",
      "INFO\t2017-01-12 16:00:06\tGenotypeConcordance\tchecked     5,060,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:218,924,659\n",
      "INFO\t2017-01-12 16:00:07\tGenotypeConcordance\tchecked     5,070,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:219,941,063\n",
      "INFO\t2017-01-12 16:00:07\tGenotypeConcordance\tchecked     5,080,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:220,840,025\n",
      "INFO\t2017-01-12 16:00:07\tGenotypeConcordance\tchecked     5,090,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:221,681,143\n",
      "INFO\t2017-01-12 16:00:07\tGenotypeConcordance\tchecked     5,100,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:222,534,932\n",
      "INFO\t2017-01-12 16:00:08\tGenotypeConcordance\tchecked     5,110,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:223,449,802\n",
      "INFO\t2017-01-12 16:00:08\tGenotypeConcordance\tchecked     5,120,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:224,278,085\n",
      "INFO\t2017-01-12 16:00:08\tGenotypeConcordance\tchecked     5,130,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:225,150,894\n",
      "INFO\t2017-01-12 16:00:08\tGenotypeConcordance\tchecked     5,140,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:226,017,466\n",
      "INFO\t2017-01-12 16:00:09\tGenotypeConcordance\tchecked     5,150,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:226,935,636\n",
      "INFO\t2017-01-12 16:00:09\tGenotypeConcordance\tchecked     5,160,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:227,877,295\n",
      "INFO\t2017-01-12 16:00:09\tGenotypeConcordance\tchecked     5,170,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:228,617,113\n",
      "INFO\t2017-01-12 16:00:09\tGenotypeConcordance\tchecked     5,180,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:229,390,956\n",
      "INFO\t2017-01-12 16:00:10\tGenotypeConcordance\tchecked     5,190,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:230,252,758\n",
      "INFO\t2017-01-12 16:00:10\tGenotypeConcordance\tchecked     5,200,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:231,125,026\n",
      "INFO\t2017-01-12 16:00:10\tGenotypeConcordance\tchecked     5,210,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:231,888,425\n",
      "INFO\t2017-01-12 16:00:10\tGenotypeConcordance\tchecked     5,220,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:232,711,377\n",
      "INFO\t2017-01-12 16:00:10\tGenotypeConcordance\tchecked     5,230,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:233,577,038\n",
      "INFO\t2017-01-12 16:00:11\tGenotypeConcordance\tchecked     5,240,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:234,410,059\n",
      "INFO\t2017-01-12 16:00:11\tGenotypeConcordance\tchecked     5,250,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:235,214,315\n",
      "INFO\t2017-01-12 16:00:11\tGenotypeConcordance\tchecked     5,260,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:235,916,669\n",
      "INFO\t2017-01-12 16:00:11\tGenotypeConcordance\tchecked     5,270,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:236,628,437\n",
      "INFO\t2017-01-12 16:00:12\tGenotypeConcordance\tchecked     5,280,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:237,398,313\n",
      "INFO\t2017-01-12 16:00:12\tGenotypeConcordance\tchecked     5,290,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:238,096,357\n",
      "INFO\t2017-01-12 16:00:12\tGenotypeConcordance\tchecked     5,300,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:238,800,921\n",
      "INFO\t2017-01-12 16:00:12\tGenotypeConcordance\tchecked     5,310,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:239,612,662\n",
      "INFO\t2017-01-12 16:00:13\tGenotypeConcordance\tchecked     5,320,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:240,365,866\n",
      "INFO\t2017-01-12 16:00:13\tGenotypeConcordance\tchecked     5,330,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:241,064,471\n",
      "INFO\t2017-01-12 16:00:13\tGenotypeConcordance\tchecked     5,340,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:241,753,985\n",
      "INFO\t2017-01-12 16:00:13\tGenotypeConcordance\tchecked     5,350,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:242,538,598\n",
      "INFO\t2017-01-12 16:00:14\tGenotypeConcordance\tchecked     5,360,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 3:142,304\n",
      "INFO\t2017-01-12 16:00:14\tGenotypeConcordance\tchecked     5,370,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 3:753,272\n",
      "INFO\t2017-01-12 16:00:14\tGenotypeConcordance\tchecked     5,380,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 3:1,398,990\n",
      "INFO\t2017-01-12 16:00:14\tGenotypeConcordance\tchecked     5,390,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 3:1,921,909\n",
      "INFO\t2017-01-12 16:00:15\tGenotypeConcordance\tchecked     5,400,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 3:2,542,326\n",
      "INFO\t2017-01-12 16:00:15\tGenotypeConcordance\tchecked     5,410,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 3:3,164,512\n",
      "INFO\t2017-01-12 16:00:15\tGenotypeConcordance\tchecked     5,420,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 3:3,786,649\n",
      "INFO\t2017-01-12 16:00:15\tGenotypeConcordance\tchecked     5,430,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 3:4,431,070\n",
      "INFO\t2017-01-12 16:00:16\tGenotypeConcordance\tchecked     5,440,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 3:5,157,196\n",
      "INFO\t2017-01-12 16:00:16\tGenotypeConcordance\tchecked     5,450,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 3:5,767,097\n",
      "INFO\t2017-01-12 16:00:16\tGenotypeConcordance\tchecked     5,460,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 3:6,374,405\n",
      "INFO\t2017-01-12 16:00:16\tGenotypeConcordance\tchecked     5,470,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 3:7,054,619\n",
      "INFO\t2017-01-12 16:00:17\tGenotypeConcordance\tchecked     5,480,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 3:7,744,875\n",
      "INFO\t2017-01-12 16:00:17\tGenotypeConcordance\tchecked     5,490,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 3:8,432,535\n",
      "INFO\t2017-01-12 16:00:17\tGenotypeConcordance\tchecked     5,500,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 3:9,164,409\n",
      "INFO\t2017-01-12 16:00:17\tGenotypeConcordance\tchecked     5,510,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 3:10,058,314\n",
      "INFO\t2017-01-12 16:00:18\tGenotypeConcordance\tchecked     5,520,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 3:10,833,472\n",
      "INFO\t2017-01-12 16:00:18\tGenotypeConcordance\tchecked     5,530,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 3:11,671,483\n",
      "INFO\t2017-01-12 16:00:18\tGenotypeConcordance\tchecked     5,540,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 3:12,522,664\n",
      "INFO\t2017-01-12 16:00:18\tGenotypeConcordance\tchecked     5,550,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 3:13,286,504\n",
      "INFO\t2017-01-12 16:00:19\tGenotypeConcordance\tchecked     5,560,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 3:14,043,326\n",
      "INFO\t2017-01-12 16:00:19\tGenotypeConcordance\tchecked     5,570,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 3:14,841,664\n",
      "INFO\t2017-01-12 16:00:19\tGenotypeConcordance\tchecked     5,580,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 3:15,660,580\n",
      "INFO\t2017-01-12 16:00:19\tGenotypeConcordance\tchecked     5,590,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 3:16,498,472\n",
      "INFO\t2017-01-12 16:00:20\tGenotypeConcordance\tchecked     5,600,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 3:17,382,833\n",
      "INFO\t2017-01-12 16:00:20\tGenotypeConcordance\tchecked     5,610,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 3:18,430,301\n",
      "INFO\t2017-01-12 16:00:20\tGenotypeConcordance\tchecked     5,620,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 3:19,332,760\n",
      "INFO\t2017-01-12 16:00:20\tGenotypeConcordance\tchecked     5,630,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 3:20,227,639\n",
      "INFO\t2017-01-12 16:00:21\tGenotypeConcordance\tchecked     5,640,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 3:21,044,936\n",
      "INFO\t2017-01-12 16:00:21\tGenotypeConcordance\tchecked     5,650,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 3:21,791,930\n",
      "INFO\t2017-01-12 16:00:21\tGenotypeConcordance\tchecked     5,660,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 3:22,325,974\n",
      "INFO\t2017-01-12 16:00:21\tGenotypeConcordance\tchecked     5,670,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 3:23,034,968\n",
      "INFO\t2017-01-12 16:00:22\tGenotypeConcordance\tchecked     5,680,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 3:23,834,246\n",
      "INFO\t2017-01-12 16:00:22\tGenotypeConcordance\tchecked     5,690,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 3:24,648,750\n",
      "INFO\t2017-01-12 16:00:22\tGenotypeConcordance\tchecked     5,700,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 3:25,317,279\n",
      "INFO\t2017-01-12 16:00:22\tGenotypeConcordance\tchecked     5,710,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 3:26,164,092\n",
      "INFO\t2017-01-12 16:00:23\tGenotypeConcordance\tchecked     5,720,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 3:26,927,337\n",
      "INFO\t2017-01-12 16:00:23\tGenotypeConcordance\tchecked     5,730,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 3:27,753,599\n",
      "INFO\t2017-01-12 16:00:23\tGenotypeConcordance\tchecked     5,740,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 3:28,662,924\n",
      "INFO\t2017-01-12 16:00:23\tGenotypeConcordance\tchecked     5,750,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 3:29,472,792\n",
      "INFO\t2017-01-12 16:00:24\tGenotypeConcordance\tchecked     5,760,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 3:30,244,931\n",
      "INFO\t2017-01-12 16:00:24\tGenotypeConcordance\tchecked     5,770,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 3:31,034,548\n",
      "INFO\t2017-01-12 16:00:24\tGenotypeConcordance\tchecked     5,780,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 3:31,892,448\n",
      "INFO\t2017-01-12 16:00:24\tGenotypeConcordance\tchecked     5,790,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 3:32,663,921\n",
      "INFO\t2017-01-12 16:00:25\tGenotypeConcordance\tchecked     5,800,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:33,525,814\n",
      "INFO\t2017-01-12 16:00:25\tGenotypeConcordance\tchecked     5,810,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:34,520,755\n",
      "INFO\t2017-01-12 16:00:25\tGenotypeConcordance\tchecked     5,820,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:35,395,255\n",
      "INFO\t2017-01-12 16:00:25\tGenotypeConcordance\tchecked     5,830,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 3:36,280,768\n",
      "INFO\t2017-01-12 16:00:26\tGenotypeConcordance\tchecked     5,840,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:37,206,578\n",
      "INFO\t2017-01-12 16:00:26\tGenotypeConcordance\tchecked     5,850,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:38,189,828\n",
      "INFO\t2017-01-12 16:00:26\tGenotypeConcordance\tchecked     5,860,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:39,147,202\n",
      "INFO\t2017-01-12 16:00:26\tGenotypeConcordance\tchecked     5,870,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 3:40,005,584\n",
      "INFO\t2017-01-12 16:00:27\tGenotypeConcordance\tchecked     5,880,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:40,977,031\n",
      "INFO\t2017-01-12 16:00:27\tGenotypeConcordance\tchecked     5,890,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:41,817,711\n",
      "INFO\t2017-01-12 16:00:27\tGenotypeConcordance\tchecked     5,900,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:42,586,839\n",
      "INFO\t2017-01-12 16:00:27\tGenotypeConcordance\tchecked     5,910,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 3:43,567,237\n",
      "INFO\t2017-01-12 16:00:27\tGenotypeConcordance\tchecked     5,920,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:44,592,561\n",
      "INFO\t2017-01-12 16:00:28\tGenotypeConcordance\tchecked     5,930,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:45,523,962\n",
      "INFO\t2017-01-12 16:00:28\tGenotypeConcordance\tchecked     5,940,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:46,472,977\n",
      "INFO\t2017-01-12 16:00:28\tGenotypeConcordance\tchecked     5,950,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 3:47,594,105\n",
      "INFO\t2017-01-12 16:00:28\tGenotypeConcordance\tchecked     5,960,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:48,890,681\n",
      "INFO\t2017-01-12 16:00:29\tGenotypeConcordance\tchecked     5,970,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:50,240,933\n",
      "INFO\t2017-01-12 16:00:29\tGenotypeConcordance\tchecked     5,980,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:51,543,025\n",
      "INFO\t2017-01-12 16:00:29\tGenotypeConcordance\tchecked     5,990,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:52,807,018\n",
      "INFO\t2017-01-12 16:00:29\tGenotypeConcordance\tchecked     6,000,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 3:53,745,766\n",
      "INFO\t2017-01-12 16:00:30\tGenotypeConcordance\tchecked     6,010,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:54,552,208\n",
      "INFO\t2017-01-12 16:00:30\tGenotypeConcordance\tchecked     6,020,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:55,346,193\n",
      "INFO\t2017-01-12 16:00:30\tGenotypeConcordance\tchecked     6,030,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 3:56,245,895\n",
      "INFO\t2017-01-12 16:00:30\tGenotypeConcordance\tchecked     6,040,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:57,068,327\n",
      "INFO\t2017-01-12 16:00:31\tGenotypeConcordance\tchecked     6,050,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:58,022,175\n",
      "INFO\t2017-01-12 16:00:31\tGenotypeConcordance\tchecked     6,060,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:58,915,946\n",
      "INFO\t2017-01-12 16:00:31\tGenotypeConcordance\tchecked     6,070,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:59,744,362\n",
      "INFO\t2017-01-12 16:00:31\tGenotypeConcordance\tchecked     6,080,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 3:60,367,056\n",
      "INFO\t2017-01-12 16:00:32\tGenotypeConcordance\tchecked     6,090,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:61,074,082\n",
      "INFO\t2017-01-12 16:00:32\tGenotypeConcordance\tchecked     6,100,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:61,866,807\n",
      "INFO\t2017-01-12 16:00:32\tGenotypeConcordance\tchecked     6,110,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 3:62,711,417\n",
      "INFO\t2017-01-12 16:00:32\tGenotypeConcordance\tchecked     6,120,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:63,449,344\n",
      "INFO\t2017-01-12 16:00:33\tGenotypeConcordance\tchecked     6,130,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:64,345,206\n",
      "INFO\t2017-01-12 16:00:33\tGenotypeConcordance\tchecked     6,140,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:65,099,686\n",
      "INFO\t2017-01-12 16:00:33\tGenotypeConcordance\tchecked     6,150,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 3:65,870,455\n",
      "INFO\t2017-01-12 16:00:33\tGenotypeConcordance\tchecked     6,160,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:66,760,847\n",
      "INFO\t2017-01-12 16:00:34\tGenotypeConcordance\tchecked     6,170,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:67,476,577\n",
      "INFO\t2017-01-12 16:00:34\tGenotypeConcordance\tchecked     6,180,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:68,340,541\n",
      "INFO\t2017-01-12 16:00:34\tGenotypeConcordance\tchecked     6,190,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:69,163,405\n",
      "INFO\t2017-01-12 16:00:34\tGenotypeConcordance\tchecked     6,200,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 3:69,956,607\n",
      "INFO\t2017-01-12 16:00:35\tGenotypeConcordance\tchecked     6,210,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:70,920,041\n",
      "INFO\t2017-01-12 16:00:35\tGenotypeConcordance\tchecked     6,220,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:71,898,456\n",
      "INFO\t2017-01-12 16:00:35\tGenotypeConcordance\tchecked     6,230,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:72,736,441\n",
      "INFO\t2017-01-12 16:00:35\tGenotypeConcordance\tchecked     6,240,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 3:73,520,153\n",
      "INFO\t2017-01-12 16:00:36\tGenotypeConcordance\tchecked     6,250,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:74,323,700\n",
      "INFO\t2017-01-12 16:00:36\tGenotypeConcordance\tchecked     6,260,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:75,158,571\n",
      "INFO\t2017-01-12 16:00:36\tGenotypeConcordance\tchecked     6,270,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:75,694,903\n",
      "INFO\t2017-01-12 16:00:36\tGenotypeConcordance\tchecked     6,280,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 3:76,169,067\n",
      "INFO\t2017-01-12 16:00:37\tGenotypeConcordance\tchecked     6,290,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:76,998,027\n",
      "INFO\t2017-01-12 16:00:37\tGenotypeConcordance\tchecked     6,300,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:77,832,564\n",
      "INFO\t2017-01-12 16:00:37\tGenotypeConcordance\tchecked     6,310,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:78,713,353\n",
      "INFO\t2017-01-12 16:00:37\tGenotypeConcordance\tchecked     6,320,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 3:79,716,784\n",
      "INFO\t2017-01-12 16:00:38\tGenotypeConcordance\tchecked     6,330,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:80,551,644\n",
      "INFO\t2017-01-12 16:00:38\tGenotypeConcordance\tchecked     6,340,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:81,508,145\n",
      "INFO\t2017-01-12 16:00:38\tGenotypeConcordance\tchecked     6,350,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:82,510,638\n",
      "INFO\t2017-01-12 16:00:38\tGenotypeConcordance\tchecked     6,360,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 3:83,454,824\n",
      "INFO\t2017-01-12 16:00:39\tGenotypeConcordance\tchecked     6,370,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:84,375,440\n",
      "INFO\t2017-01-12 16:00:39\tGenotypeConcordance\tchecked     6,380,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:85,236,039\n",
      "INFO\t2017-01-12 16:00:39\tGenotypeConcordance\tchecked     6,390,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:86,119,954\n",
      "INFO\t2017-01-12 16:00:39\tGenotypeConcordance\tchecked     6,400,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 3:86,997,521\n",
      "INFO\t2017-01-12 16:00:40\tGenotypeConcordance\tchecked     6,410,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:87,868,712\n",
      "INFO\t2017-01-12 16:00:40\tGenotypeConcordance\tchecked     6,420,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:88,716,678\n",
      "INFO\t2017-01-12 16:00:40\tGenotypeConcordance\tchecked     6,430,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:89,612,355\n",
      "INFO\t2017-01-12 16:00:40\tGenotypeConcordance\tchecked     6,440,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 3:90,430,115\n",
      "INFO\t2017-01-12 16:00:41\tGenotypeConcordance\tchecked     6,450,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:94,327,307\n",
      "INFO\t2017-01-12 16:00:41\tGenotypeConcordance\tchecked     6,460,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:95,106,745\n",
      "INFO\t2017-01-12 16:00:41\tGenotypeConcordance\tchecked     6,470,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:95,861,359\n",
      "INFO\t2017-01-12 16:00:41\tGenotypeConcordance\tchecked     6,480,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:96,699,514\n",
      "INFO\t2017-01-12 16:00:42\tGenotypeConcordance\tchecked     6,490,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:97,764,431\n",
      "INFO\t2017-01-12 16:00:42\tGenotypeConcordance\tchecked     6,500,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:98,394,062\n",
      "INFO\t2017-01-12 16:00:42\tGenotypeConcordance\tchecked     6,510,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:99,150,890\n",
      "INFO\t2017-01-12 16:00:42\tGenotypeConcordance\tchecked     6,520,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:100,192,167\n",
      "INFO\t2017-01-12 16:00:43\tGenotypeConcordance\tchecked     6,530,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:101,042,354\n",
      "INFO\t2017-01-12 16:00:43\tGenotypeConcordance\tchecked     6,540,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:102,053,029\n",
      "INFO\t2017-01-12 16:00:43\tGenotypeConcordance\tchecked     6,550,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:102,939,521\n",
      "INFO\t2017-01-12 16:00:43\tGenotypeConcordance\tchecked     6,560,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:103,694,562\n",
      "INFO\t2017-01-12 16:00:44\tGenotypeConcordance\tchecked     6,570,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:104,515,697\n",
      "INFO\t2017-01-12 16:00:44\tGenotypeConcordance\tchecked     6,580,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:105,348,141\n",
      "INFO\t2017-01-12 16:00:44\tGenotypeConcordance\tchecked     6,590,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:106,153,752\n",
      "INFO\t2017-01-12 16:00:44\tGenotypeConcordance\tchecked     6,600,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:107,027,393\n",
      "INFO\t2017-01-12 16:00:45\tGenotypeConcordance\tchecked     6,610,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:108,030,714\n",
      "INFO\t2017-01-12 16:00:45\tGenotypeConcordance\tchecked     6,620,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:108,975,423\n",
      "INFO\t2017-01-12 16:00:45\tGenotypeConcordance\tchecked     6,630,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:109,796,515\n",
      "INFO\t2017-01-12 16:00:45\tGenotypeConcordance\tchecked     6,640,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:110,729,247\n",
      "INFO\t2017-01-12 16:00:46\tGenotypeConcordance\tchecked     6,650,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:111,657,902\n",
      "INFO\t2017-01-12 16:00:46\tGenotypeConcordance\tchecked     6,660,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:112,508,344\n",
      "INFO\t2017-01-12 16:00:46\tGenotypeConcordance\tchecked     6,670,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:113,334,752\n",
      "INFO\t2017-01-12 16:00:46\tGenotypeConcordance\tchecked     6,680,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:114,394,131\n",
      "INFO\t2017-01-12 16:00:47\tGenotypeConcordance\tchecked     6,690,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:115,515,376\n",
      "INFO\t2017-01-12 16:00:47\tGenotypeConcordance\tchecked     6,700,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:116,424,880\n",
      "INFO\t2017-01-12 16:00:47\tGenotypeConcordance\tchecked     6,710,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:117,241,077\n",
      "INFO\t2017-01-12 16:00:47\tGenotypeConcordance\tchecked     6,720,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:118,082,137\n",
      "INFO\t2017-01-12 16:00:48\tGenotypeConcordance\tchecked     6,730,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:118,972,605\n",
      "INFO\t2017-01-12 16:00:48\tGenotypeConcordance\tchecked     6,740,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:119,878,441\n",
      "INFO\t2017-01-12 16:00:48\tGenotypeConcordance\tchecked     6,750,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:120,792,187\n",
      "INFO\t2017-01-12 16:00:48\tGenotypeConcordance\tchecked     6,760,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:121,807,420\n",
      "INFO\t2017-01-12 16:00:49\tGenotypeConcordance\tchecked     6,770,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:122,764,377\n",
      "INFO\t2017-01-12 16:00:49\tGenotypeConcordance\tchecked     6,780,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:123,678,926\n",
      "INFO\t2017-01-12 16:00:49\tGenotypeConcordance\tchecked     6,790,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:124,600,003\n",
      "INFO\t2017-01-12 16:00:50\tGenotypeConcordance\tchecked     6,800,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:125,454,681\n",
      "INFO\t2017-01-12 16:00:50\tGenotypeConcordance\tchecked     6,810,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:126,016,956\n",
      "INFO\t2017-01-12 16:00:50\tGenotypeConcordance\tchecked     6,820,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:126,896,517\n",
      "INFO\t2017-01-12 16:00:50\tGenotypeConcordance\tchecked     6,830,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:127,779,944\n",
      "INFO\t2017-01-12 16:00:51\tGenotypeConcordance\tchecked     6,840,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:128,694,269\n",
      "INFO\t2017-01-12 16:00:51\tGenotypeConcordance\tchecked     6,850,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:129,642,923\n",
      "INFO\t2017-01-12 16:00:51\tGenotypeConcordance\tchecked     6,860,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:130,278,086\n",
      "INFO\t2017-01-12 16:00:51\tGenotypeConcordance\tchecked     6,870,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:131,283,892\n",
      "INFO\t2017-01-12 16:00:52\tGenotypeConcordance\tchecked     6,880,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:132,072,419\n",
      "INFO\t2017-01-12 16:00:52\tGenotypeConcordance\tchecked     6,890,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:132,944,181\n",
      "INFO\t2017-01-12 16:00:52\tGenotypeConcordance\tchecked     6,900,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:133,776,063\n",
      "INFO\t2017-01-12 16:00:52\tGenotypeConcordance\tchecked     6,910,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:134,677,613\n",
      "INFO\t2017-01-12 16:00:53\tGenotypeConcordance\tchecked     6,920,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:135,647,261\n",
      "INFO\t2017-01-12 16:00:53\tGenotypeConcordance\tchecked     6,930,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:136,701,538\n",
      "INFO\t2017-01-12 16:00:53\tGenotypeConcordance\tchecked     6,940,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:137,668,783\n",
      "INFO\t2017-01-12 16:00:53\tGenotypeConcordance\tchecked     6,950,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:138,676,755\n",
      "INFO\t2017-01-12 16:00:54\tGenotypeConcordance\tchecked     6,960,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:139,651,255\n",
      "INFO\t2017-01-12 16:00:54\tGenotypeConcordance\tchecked     6,970,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:140,523,699\n",
      "INFO\t2017-01-12 16:00:54\tGenotypeConcordance\tchecked     6,980,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:141,412,883\n",
      "INFO\t2017-01-12 16:00:55\tGenotypeConcordance\tchecked     6,990,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:142,335,410\n",
      "INFO\t2017-01-12 16:00:55\tGenotypeConcordance\tchecked     7,000,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:143,238,874\n",
      "INFO\t2017-01-12 16:00:55\tGenotypeConcordance\tchecked     7,010,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:144,113,799\n",
      "INFO\t2017-01-12 16:00:55\tGenotypeConcordance\tchecked     7,020,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:144,923,706\n",
      "INFO\t2017-01-12 16:00:56\tGenotypeConcordance\tchecked     7,030,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:145,686,843\n",
      "INFO\t2017-01-12 16:00:56\tGenotypeConcordance\tchecked     7,040,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:146,478,254\n",
      "INFO\t2017-01-12 16:00:56\tGenotypeConcordance\tchecked     7,050,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:147,498,662\n",
      "INFO\t2017-01-12 16:00:56\tGenotypeConcordance\tchecked     7,060,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:148,355,413\n",
      "INFO\t2017-01-12 16:00:57\tGenotypeConcordance\tchecked     7,070,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:149,199,064\n",
      "INFO\t2017-01-12 16:00:57\tGenotypeConcordance\tchecked     7,080,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:150,148,092\n",
      "INFO\t2017-01-12 16:00:57\tGenotypeConcordance\tchecked     7,090,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:151,097,654\n",
      "INFO\t2017-01-12 16:00:57\tGenotypeConcordance\tchecked     7,100,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:151,986,427\n",
      "INFO\t2017-01-12 16:00:58\tGenotypeConcordance\tchecked     7,110,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:152,886,681\n",
      "INFO\t2017-01-12 16:00:58\tGenotypeConcordance\tchecked     7,120,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:153,699,976\n",
      "INFO\t2017-01-12 16:00:58\tGenotypeConcordance\tchecked     7,130,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:154,593,619\n",
      "INFO\t2017-01-12 16:00:58\tGenotypeConcordance\tchecked     7,140,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:155,486,703\n",
      "INFO\t2017-01-12 16:00:59\tGenotypeConcordance\tchecked     7,150,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:156,390,308\n",
      "INFO\t2017-01-12 16:00:59\tGenotypeConcordance\tchecked     7,160,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:157,358,048\n",
      "INFO\t2017-01-12 16:00:59\tGenotypeConcordance\tchecked     7,170,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:158,271,690\n",
      "INFO\t2017-01-12 16:00:59\tGenotypeConcordance\tchecked     7,180,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:159,142,234\n",
      "INFO\t2017-01-12 16:01:00\tGenotypeConcordance\tchecked     7,190,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:159,977,751\n",
      "INFO\t2017-01-12 16:01:00\tGenotypeConcordance\tchecked     7,200,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:160,895,762\n",
      "INFO\t2017-01-12 16:01:00\tGenotypeConcordance\tchecked     7,210,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:161,729,127\n",
      "INFO\t2017-01-12 16:01:01\tGenotypeConcordance\tchecked     7,220,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:162,494,864\n",
      "INFO\t2017-01-12 16:01:01\tGenotypeConcordance\tchecked     7,230,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:163,367,841\n",
      "INFO\t2017-01-12 16:01:01\tGenotypeConcordance\tchecked     7,240,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:164,200,352\n",
      "INFO\t2017-01-12 16:01:01\tGenotypeConcordance\tchecked     7,250,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:165,126,400\n",
      "INFO\t2017-01-12 16:01:02\tGenotypeConcordance\tchecked     7,260,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:165,943,024\n",
      "INFO\t2017-01-12 16:01:02\tGenotypeConcordance\tchecked     7,270,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:166,736,793\n",
      "INFO\t2017-01-12 16:01:02\tGenotypeConcordance\tchecked     7,280,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:167,666,443\n",
      "INFO\t2017-01-12 16:01:02\tGenotypeConcordance\tchecked     7,290,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:168,628,196\n",
      "INFO\t2017-01-12 16:01:03\tGenotypeConcordance\tchecked     7,300,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:169,554,129\n",
      "INFO\t2017-01-12 16:01:03\tGenotypeConcordance\tchecked     7,310,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:170,420,463\n",
      "INFO\t2017-01-12 16:01:03\tGenotypeConcordance\tchecked     7,320,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:171,326,315\n",
      "INFO\t2017-01-12 16:01:03\tGenotypeConcordance\tchecked     7,330,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:172,239,988\n",
      "INFO\t2017-01-12 16:01:04\tGenotypeConcordance\tchecked     7,340,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:173,149,550\n",
      "INFO\t2017-01-12 16:01:04\tGenotypeConcordance\tchecked     7,350,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:174,049,118\n",
      "INFO\t2017-01-12 16:01:04\tGenotypeConcordance\tchecked     7,360,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:174,838,151\n",
      "INFO\t2017-01-12 16:01:04\tGenotypeConcordance\tchecked     7,370,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:175,559,562\n",
      "INFO\t2017-01-12 16:01:05\tGenotypeConcordance\tchecked     7,380,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:176,421,790\n",
      "INFO\t2017-01-12 16:01:05\tGenotypeConcordance\tchecked     7,390,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:177,215,858\n",
      "INFO\t2017-01-12 16:01:05\tGenotypeConcordance\tchecked     7,400,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:178,145,346\n",
      "INFO\t2017-01-12 16:01:05\tGenotypeConcordance\tchecked     7,410,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:179,073,852\n",
      "INFO\t2017-01-12 16:01:06\tGenotypeConcordance\tchecked     7,420,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:180,029,935\n",
      "INFO\t2017-01-12 16:01:06\tGenotypeConcordance\tchecked     7,430,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:181,106,125\n",
      "INFO\t2017-01-12 16:01:06\tGenotypeConcordance\tchecked     7,440,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:182,153,831\n",
      "INFO\t2017-01-12 16:01:06\tGenotypeConcordance\tchecked     7,450,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:183,126,506\n",
      "INFO\t2017-01-12 16:01:07\tGenotypeConcordance\tchecked     7,460,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:183,985,669\n",
      "INFO\t2017-01-12 16:01:07\tGenotypeConcordance\tchecked     7,470,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:184,834,329\n",
      "INFO\t2017-01-12 16:01:07\tGenotypeConcordance\tchecked     7,480,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:185,793,273\n",
      "INFO\t2017-01-12 16:01:07\tGenotypeConcordance\tchecked     7,490,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:186,628,163\n",
      "INFO\t2017-01-12 16:01:08\tGenotypeConcordance\tchecked     7,500,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:187,405,595\n",
      "INFO\t2017-01-12 16:01:08\tGenotypeConcordance\tchecked     7,510,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:188,229,424\n",
      "INFO\t2017-01-12 16:01:08\tGenotypeConcordance\tchecked     7,520,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:189,133,827\n",
      "INFO\t2017-01-12 16:01:08\tGenotypeConcordance\tchecked     7,530,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:189,871,351\n",
      "INFO\t2017-01-12 16:01:09\tGenotypeConcordance\tchecked     7,540,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:190,595,591\n",
      "INFO\t2017-01-12 16:01:09\tGenotypeConcordance\tchecked     7,550,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:191,343,146\n",
      "INFO\t2017-01-12 16:01:09\tGenotypeConcordance\tchecked     7,560,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:192,081,284\n",
      "INFO\t2017-01-12 16:01:09\tGenotypeConcordance\tchecked     7,570,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:192,852,448\n",
      "INFO\t2017-01-12 16:01:10\tGenotypeConcordance\tchecked     7,580,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:193,718,465\n",
      "INFO\t2017-01-12 16:01:10\tGenotypeConcordance\tchecked     7,590,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:194,398,645\n",
      "INFO\t2017-01-12 16:01:10\tGenotypeConcordance\tchecked     7,600,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:195,076,877\n",
      "INFO\t2017-01-12 16:01:10\tGenotypeConcordance\tchecked     7,610,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:195,597,503\n",
      "INFO\t2017-01-12 16:01:11\tGenotypeConcordance\tchecked     7,620,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:196,255,340\n",
      "INFO\t2017-01-12 16:01:11\tGenotypeConcordance\tchecked     7,630,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:196,988,678\n",
      "INFO\t2017-01-12 16:01:11\tGenotypeConcordance\tchecked     7,640,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:197,702,371\n",
      "INFO\t2017-01-12 16:01:11\tGenotypeConcordance\tchecked     7,650,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 4:542,898\n",
      "INFO\t2017-01-12 16:01:12\tGenotypeConcordance\tchecked     7,660,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 4:1,221,441\n",
      "INFO\t2017-01-12 16:01:12\tGenotypeConcordance\tchecked     7,670,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 4:1,910,811\n",
      "INFO\t2017-01-12 16:01:12\tGenotypeConcordance\tchecked     7,680,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 4:2,749,343\n",
      "INFO\t2017-01-12 16:01:12\tGenotypeConcordance\tchecked     7,690,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 4:3,514,344\n",
      "INFO\t2017-01-12 16:01:13\tGenotypeConcordance\tchecked     7,700,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 4:3,929,865\n",
      "INFO\t2017-01-12 16:01:13\tGenotypeConcordance\tchecked     7,710,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 4:4,344,979\n",
      "INFO\t2017-01-12 16:01:13\tGenotypeConcordance\tchecked     7,720,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 4:5,089,303\n",
      "INFO\t2017-01-12 16:01:13\tGenotypeConcordance\tchecked     7,730,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 4:5,669,247\n",
      "INFO\t2017-01-12 16:01:14\tGenotypeConcordance\tchecked     7,740,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 4:6,298,705\n",
      "INFO\t2017-01-12 16:01:14\tGenotypeConcordance\tchecked     7,750,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 4:6,957,332\n",
      "INFO\t2017-01-12 16:01:14\tGenotypeConcordance\tchecked     7,760,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 4:7,535,219\n",
      "INFO\t2017-01-12 16:01:14\tGenotypeConcordance\tchecked     7,770,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 4:8,114,300\n",
      "INFO\t2017-01-12 16:01:15\tGenotypeConcordance\tchecked     7,780,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 4:8,722,998\n",
      "INFO\t2017-01-12 16:01:15\tGenotypeConcordance\tchecked     7,790,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 4:9,183,124\n",
      "INFO\t2017-01-12 16:01:15\tGenotypeConcordance\tchecked     7,800,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 4:9,669,609\n",
      "INFO\t2017-01-12 16:01:15\tGenotypeConcordance\tchecked     7,810,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 4:10,210,906\n",
      "INFO\t2017-01-12 16:01:16\tGenotypeConcordance\tchecked     7,820,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 4:10,942,705\n",
      "INFO\t2017-01-12 16:01:16\tGenotypeConcordance\tchecked     7,830,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 4:11,650,710\n",
      "INFO\t2017-01-12 16:01:16\tGenotypeConcordance\tchecked     7,840,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 4:12,318,791\n",
      "INFO\t2017-01-12 16:01:16\tGenotypeConcordance\tchecked     7,850,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 4:13,018,888\n",
      "INFO\t2017-01-12 16:01:17\tGenotypeConcordance\tchecked     7,860,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 4:13,864,746\n",
      "INFO\t2017-01-12 16:01:17\tGenotypeConcordance\tchecked     7,870,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 4:14,624,305\n",
      "INFO\t2017-01-12 16:01:17\tGenotypeConcordance\tchecked     7,880,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 4:15,457,861\n",
      "INFO\t2017-01-12 16:01:17\tGenotypeConcordance\tchecked     7,890,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 4:16,377,284\n",
      "INFO\t2017-01-12 16:01:18\tGenotypeConcordance\tchecked     7,900,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 4:17,240,787\n",
      "INFO\t2017-01-12 16:01:18\tGenotypeConcordance\tchecked     7,910,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 4:18,080,821\n",
      "INFO\t2017-01-12 16:01:18\tGenotypeConcordance\tchecked     7,920,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 4:18,918,814\n",
      "INFO\t2017-01-12 16:01:18\tGenotypeConcordance\tchecked     7,930,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 4:19,736,258\n",
      "INFO\t2017-01-12 16:01:19\tGenotypeConcordance\tchecked     7,940,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 4:20,560,883\n",
      "INFO\t2017-01-12 16:01:19\tGenotypeConcordance\tchecked     7,950,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 4:21,283,211\n",
      "INFO\t2017-01-12 16:01:19\tGenotypeConcordance\tchecked     7,960,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 4:22,057,888\n",
      "INFO\t2017-01-12 16:01:19\tGenotypeConcordance\tchecked     7,970,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 4:22,800,848\n",
      "INFO\t2017-01-12 16:01:20\tGenotypeConcordance\tchecked     7,980,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 4:23,656,138\n",
      "INFO\t2017-01-12 16:01:20\tGenotypeConcordance\tchecked     7,990,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 4:24,464,042\n",
      "INFO\t2017-01-12 16:01:20\tGenotypeConcordance\tchecked     8,000,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 4:25,290,880\n",
      "INFO\t2017-01-12 16:01:20\tGenotypeConcordance\tchecked     8,010,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 4:26,074,011\n",
      "INFO\t2017-01-12 16:01:21\tGenotypeConcordance\tchecked     8,020,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 4:26,996,783\n",
      "INFO\t2017-01-12 16:01:21\tGenotypeConcordance\tchecked     8,030,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 4:27,795,241\n",
      "INFO\t2017-01-12 16:01:21\tGenotypeConcordance\tchecked     8,040,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 4:28,570,198\n",
      "INFO\t2017-01-12 16:01:21\tGenotypeConcordance\tchecked     8,050,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 4:29,369,334\n",
      "INFO\t2017-01-12 16:01:22\tGenotypeConcordance\tchecked     8,060,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 4:30,159,982\n",
      "INFO\t2017-01-12 16:01:22\tGenotypeConcordance\tchecked     8,070,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 4:30,953,996\n",
      "INFO\t2017-01-12 16:01:22\tGenotypeConcordance\tchecked     8,080,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 4:31,734,106\n",
      "INFO\t2017-01-12 16:01:22\tGenotypeConcordance\tchecked     8,090,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 4:32,484,424\n",
      "INFO\t2017-01-12 16:01:23\tGenotypeConcordance\tchecked     8,100,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 4:33,203,384\n",
      "INFO\t2017-01-12 16:01:23\tGenotypeConcordance\tchecked     8,110,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 4:34,051,125\n",
      "INFO\t2017-01-12 16:01:23\tGenotypeConcordance\tchecked     8,120,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 4:34,917,773\n",
      "INFO\t2017-01-12 16:01:23\tGenotypeConcordance\tchecked     8,130,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 4:35,642,436\n",
      "INFO\t2017-01-12 16:01:24\tGenotypeConcordance\tchecked     8,140,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 4:36,438,525\n",
      "INFO\t2017-01-12 16:01:24\tGenotypeConcordance\tchecked     8,150,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 4:37,172,706\n",
      "INFO\t2017-01-12 16:01:24\tGenotypeConcordance\tchecked     8,160,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 4:37,925,177\n",
      "INFO\t2017-01-12 16:01:24\tGenotypeConcordance\tchecked     8,170,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 4:38,750,185\n",
      "INFO\t2017-01-12 16:01:25\tGenotypeConcordance\tchecked     8,180,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 4:39,591,005\n",
      "INFO\t2017-01-12 16:01:25\tGenotypeConcordance\tchecked     8,190,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 4:40,407,664\n",
      "INFO\t2017-01-12 16:01:25\tGenotypeConcordance\tchecked     8,200,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 4:41,208,436\n",
      "INFO\t2017-01-12 16:01:26\tGenotypeConcordance\tchecked     8,210,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 4:42,117,633\n",
      "INFO\t2017-01-12 16:01:26\tGenotypeConcordance\tchecked     8,220,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 4:42,908,795\n",
      "INFO\t2017-01-12 16:01:26\tGenotypeConcordance\tchecked     8,230,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 4:43,683,418\n",
      "INFO\t2017-01-12 16:01:26\tGenotypeConcordance\tchecked     8,240,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 4:44,537,786\n",
      "INFO\t2017-01-12 16:01:26\tGenotypeConcordance\tchecked     8,250,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 4:45,281,592\n",
      "INFO\t2017-01-12 16:01:27\tGenotypeConcordance\tchecked     8,260,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 4:46,140,691\n",
      "INFO\t2017-01-12 16:01:27\tGenotypeConcordance\tchecked     8,270,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 4:47,056,073\n",
      "INFO\t2017-01-12 16:01:27\tGenotypeConcordance\tchecked     8,280,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 4:48,060,169\n",
      "INFO\t2017-01-12 16:01:27\tGenotypeConcordance\tchecked     8,290,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 4:49,095,993\n",
      "INFO\t2017-01-12 16:01:28\tGenotypeConcordance\tchecked     8,300,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 4:49,334,997\n",
      "INFO\t2017-01-12 16:01:28\tGenotypeConcordance\tchecked     8,310,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 4:52,851,830\n",
      "INFO\t2017-01-12 16:01:28\tGenotypeConcordance\tchecked     8,320,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 4:53,712,121\n",
      "INFO\t2017-01-12 16:01:29\tGenotypeConcordance\tchecked     8,330,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 4:54,633,846\n",
      "INFO\t2017-01-12 16:01:29\tGenotypeConcordance\tchecked     8,340,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 4:55,490,518\n",
      "INFO\t2017-01-12 16:01:29\tGenotypeConcordance\tchecked     8,350,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 4:56,300,941\n",
      "INFO\t2017-01-12 16:01:29\tGenotypeConcordance\tchecked     8,360,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 4:57,171,987\n",
      "INFO\t2017-01-12 16:01:30\tGenotypeConcordance\tchecked     8,370,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:57,934,235\n",
      "INFO\t2017-01-12 16:01:30\tGenotypeConcordance\tchecked     8,380,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:58,608,254\n",
      "INFO\t2017-01-12 16:01:30\tGenotypeConcordance\tchecked     8,390,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:59,261,192\n",
      "INFO\t2017-01-12 16:01:30\tGenotypeConcordance\tchecked     8,400,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 4:60,033,417\n",
      "INFO\t2017-01-12 16:01:31\tGenotypeConcordance\tchecked     8,410,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:60,690,523\n",
      "INFO\t2017-01-12 16:01:31\tGenotypeConcordance\tchecked     8,420,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:61,375,698\n",
      "INFO\t2017-01-12 16:01:31\tGenotypeConcordance\tchecked     8,430,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:62,195,940\n",
      "INFO\t2017-01-12 16:01:31\tGenotypeConcordance\tchecked     8,440,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 4:63,135,706\n",
      "INFO\t2017-01-12 16:01:32\tGenotypeConcordance\tchecked     8,450,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:63,808,535\n",
      "INFO\t2017-01-12 16:01:32\tGenotypeConcordance\tchecked     8,460,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:64,456,321\n",
      "INFO\t2017-01-12 16:01:32\tGenotypeConcordance\tchecked     8,470,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:65,227,122\n",
      "INFO\t2017-01-12 16:01:32\tGenotypeConcordance\tchecked     8,480,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 4:65,930,371\n",
      "INFO\t2017-01-12 16:01:33\tGenotypeConcordance\tchecked     8,490,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:66,696,287\n",
      "INFO\t2017-01-12 16:01:33\tGenotypeConcordance\tchecked     8,500,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:67,443,752\n",
      "INFO\t2017-01-12 16:01:33\tGenotypeConcordance\tchecked     8,510,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:68,269,826\n",
      "INFO\t2017-01-12 16:01:33\tGenotypeConcordance\tchecked     8,520,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 4:69,141,149\n",
      "INFO\t2017-01-12 16:01:34\tGenotypeConcordance\tchecked     8,530,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:69,867,584\n",
      "INFO\t2017-01-12 16:01:34\tGenotypeConcordance\tchecked     8,540,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:70,460,328\n",
      "INFO\t2017-01-12 16:01:34\tGenotypeConcordance\tchecked     8,550,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:71,210,727\n",
      "INFO\t2017-01-12 16:01:34\tGenotypeConcordance\tchecked     8,560,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 4:71,994,066\n",
      "INFO\t2017-01-12 16:01:35\tGenotypeConcordance\tchecked     8,570,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:72,868,259\n",
      "INFO\t2017-01-12 16:01:35\tGenotypeConcordance\tchecked     8,580,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:73,633,538\n",
      "INFO\t2017-01-12 16:01:35\tGenotypeConcordance\tchecked     8,590,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:74,539,627\n",
      "INFO\t2017-01-12 16:01:35\tGenotypeConcordance\tchecked     8,600,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 4:75,534,895\n",
      "INFO\t2017-01-12 16:01:36\tGenotypeConcordance\tchecked     8,610,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:76,302,332\n",
      "INFO\t2017-01-12 16:01:36\tGenotypeConcordance\tchecked     8,620,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:77,132,071\n",
      "INFO\t2017-01-12 16:01:36\tGenotypeConcordance\tchecked     8,630,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:78,001,508\n",
      "INFO\t2017-01-12 16:01:36\tGenotypeConcordance\tchecked     8,640,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 4:78,806,713\n",
      "INFO\t2017-01-12 16:01:37\tGenotypeConcordance\tchecked     8,650,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:79,650,815\n",
      "INFO\t2017-01-12 16:01:37\tGenotypeConcordance\tchecked     8,660,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:80,643,067\n",
      "INFO\t2017-01-12 16:01:37\tGenotypeConcordance\tchecked     8,670,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 4:81,589,438\n",
      "INFO\t2017-01-12 16:01:37\tGenotypeConcordance\tchecked     8,680,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:82,495,893\n",
      "INFO\t2017-01-12 16:01:38\tGenotypeConcordance\tchecked     8,690,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:83,369,296\n",
      "INFO\t2017-01-12 16:01:38\tGenotypeConcordance\tchecked     8,700,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:84,240,409\n",
      "INFO\t2017-01-12 16:01:38\tGenotypeConcordance\tchecked     8,710,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 4:85,074,102\n",
      "INFO\t2017-01-12 16:01:38\tGenotypeConcordance\tchecked     8,720,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:86,099,883\n",
      "INFO\t2017-01-12 16:01:39\tGenotypeConcordance\tchecked     8,730,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:86,933,202\n",
      "INFO\t2017-01-12 16:01:39\tGenotypeConcordance\tchecked     8,740,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:87,940,940\n",
      "INFO\t2017-01-12 16:01:39\tGenotypeConcordance\tchecked     8,750,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 4:88,751,096\n",
      "INFO\t2017-01-12 16:01:39\tGenotypeConcordance\tchecked     8,760,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:89,665,891\n",
      "INFO\t2017-01-12 16:01:40\tGenotypeConcordance\tchecked     8,770,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:90,545,192\n",
      "INFO\t2017-01-12 16:01:40\tGenotypeConcordance\tchecked     8,780,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:91,403,851\n",
      "INFO\t2017-01-12 16:01:40\tGenotypeConcordance\tchecked     8,790,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:92,197,449\n",
      "INFO\t2017-01-12 16:01:40\tGenotypeConcordance\tchecked     8,800,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 4:93,013,683\n",
      "INFO\t2017-01-12 16:01:41\tGenotypeConcordance\tchecked     8,810,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:93,999,179\n",
      "INFO\t2017-01-12 16:01:41\tGenotypeConcordance\tchecked     8,820,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:94,932,155\n",
      "INFO\t2017-01-12 16:01:41\tGenotypeConcordance\tchecked     8,830,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:95,783,955\n",
      "INFO\t2017-01-12 16:01:41\tGenotypeConcordance\tchecked     8,840,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 4:96,588,172\n",
      "INFO\t2017-01-12 16:01:42\tGenotypeConcordance\tchecked     8,850,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:97,439,543\n",
      "INFO\t2017-01-12 16:01:42\tGenotypeConcordance\tchecked     8,860,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:98,277,522\n",
      "INFO\t2017-01-12 16:01:42\tGenotypeConcordance\tchecked     8,870,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:99,144,321\n",
      "INFO\t2017-01-12 16:01:42\tGenotypeConcordance\tchecked     8,880,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 4:100,042,242\n",
      "INFO\t2017-01-12 16:01:43\tGenotypeConcordance\tchecked     8,890,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:101,020,755\n",
      "INFO\t2017-01-12 16:01:43\tGenotypeConcordance\tchecked     8,900,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:101,871,546\n",
      "INFO\t2017-01-12 16:01:43\tGenotypeConcordance\tchecked     8,910,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:102,722,852\n",
      "INFO\t2017-01-12 16:01:43\tGenotypeConcordance\tchecked     8,920,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 4:103,633,275\n",
      "INFO\t2017-01-12 16:01:44\tGenotypeConcordance\tchecked     8,930,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:104,514,952\n",
      "INFO\t2017-01-12 16:01:44\tGenotypeConcordance\tchecked     8,940,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:105,414,718\n",
      "INFO\t2017-01-12 16:01:44\tGenotypeConcordance\tchecked     8,950,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:106,367,210\n",
      "INFO\t2017-01-12 16:01:44\tGenotypeConcordance\tchecked     8,960,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 4:107,350,635\n",
      "INFO\t2017-01-12 16:01:45\tGenotypeConcordance\tchecked     8,970,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:108,330,475\n",
      "INFO\t2017-01-12 16:01:45\tGenotypeConcordance\tchecked     8,980,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:109,301,111\n",
      "INFO\t2017-01-12 16:01:45\tGenotypeConcordance\tchecked     8,990,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:110,144,593\n",
      "INFO\t2017-01-12 16:01:45\tGenotypeConcordance\tchecked     9,000,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 4:111,005,409\n",
      "INFO\t2017-01-12 16:01:46\tGenotypeConcordance\tchecked     9,010,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:111,858,679\n",
      "INFO\t2017-01-12 16:01:46\tGenotypeConcordance\tchecked     9,020,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:112,800,228\n",
      "INFO\t2017-01-12 16:01:46\tGenotypeConcordance\tchecked     9,030,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:113,797,055\n",
      "INFO\t2017-01-12 16:01:46\tGenotypeConcordance\tchecked     9,040,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 4:114,704,051\n",
      "INFO\t2017-01-12 16:01:47\tGenotypeConcordance\tchecked     9,050,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:115,488,127\n",
      "INFO\t2017-01-12 16:01:47\tGenotypeConcordance\tchecked     9,060,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:116,329,991\n",
      "INFO\t2017-01-12 16:01:47\tGenotypeConcordance\tchecked     9,070,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:117,110,376\n",
      "INFO\t2017-01-12 16:01:47\tGenotypeConcordance\tchecked     9,080,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 4:117,894,668\n",
      "INFO\t2017-01-12 16:01:48\tGenotypeConcordance\tchecked     9,090,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:118,721,958\n",
      "INFO\t2017-01-12 16:01:48\tGenotypeConcordance\tchecked     9,100,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:119,668,186\n",
      "INFO\t2017-01-12 16:01:48\tGenotypeConcordance\tchecked     9,110,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:120,459,548\n",
      "INFO\t2017-01-12 16:01:48\tGenotypeConcordance\tchecked     9,120,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 4:121,345,846\n",
      "INFO\t2017-01-12 16:01:49\tGenotypeConcordance\tchecked     9,130,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:122,164,895\n",
      "INFO\t2017-01-12 16:01:49\tGenotypeConcordance\tchecked     9,140,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:123,034,210\n",
      "INFO\t2017-01-12 16:01:49\tGenotypeConcordance\tchecked     9,150,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:123,983,264\n",
      "INFO\t2017-01-12 16:01:49\tGenotypeConcordance\tchecked     9,160,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 4:124,821,664\n",
      "INFO\t2017-01-12 16:01:50\tGenotypeConcordance\tchecked     9,170,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:125,713,066\n",
      "INFO\t2017-01-12 16:01:50\tGenotypeConcordance\tchecked     9,180,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:126,556,132\n",
      "INFO\t2017-01-12 16:01:50\tGenotypeConcordance\tchecked     9,190,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:127,370,617\n",
      "INFO\t2017-01-12 16:01:50\tGenotypeConcordance\tchecked     9,200,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 4:128,314,339\n",
      "INFO\t2017-01-12 16:01:51\tGenotypeConcordance\tchecked     9,210,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:129,423,388\n",
      "INFO\t2017-01-12 16:01:51\tGenotypeConcordance\tchecked     9,220,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:130,270,267\n",
      "INFO\t2017-01-12 16:01:51\tGenotypeConcordance\tchecked     9,230,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:131,095,087\n",
      "INFO\t2017-01-12 16:01:51\tGenotypeConcordance\tchecked     9,240,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 4:131,872,320\n",
      "INFO\t2017-01-12 16:01:52\tGenotypeConcordance\tchecked     9,250,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:132,725,160\n",
      "INFO\t2017-01-12 16:01:52\tGenotypeConcordance\tchecked     9,260,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:133,576,536\n",
      "INFO\t2017-01-12 16:01:52\tGenotypeConcordance\tchecked     9,270,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:134,425,481\n",
      "INFO\t2017-01-12 16:01:52\tGenotypeConcordance\tchecked     9,280,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:135,351,753\n",
      "INFO\t2017-01-12 16:01:53\tGenotypeConcordance\tchecked     9,290,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:136,111,663\n",
      "INFO\t2017-01-12 16:01:53\tGenotypeConcordance\tchecked     9,300,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:136,937,648\n",
      "INFO\t2017-01-12 16:01:53\tGenotypeConcordance\tchecked     9,310,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:137,816,744\n",
      "INFO\t2017-01-12 16:01:53\tGenotypeConcordance\tchecked     9,320,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:138,608,230\n",
      "INFO\t2017-01-12 16:01:53\tGenotypeConcordance\tchecked     9,330,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:139,477,559\n",
      "INFO\t2017-01-12 16:01:54\tGenotypeConcordance\tchecked     9,340,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:140,354,073\n",
      "INFO\t2017-01-12 16:01:54\tGenotypeConcordance\tchecked     9,350,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:141,301,420\n",
      "INFO\t2017-01-12 16:01:54\tGenotypeConcordance\tchecked     9,360,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:142,199,705\n",
      "INFO\t2017-01-12 16:01:54\tGenotypeConcordance\tchecked     9,370,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:143,041,723\n",
      "INFO\t2017-01-12 16:01:55\tGenotypeConcordance\tchecked     9,380,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:143,950,762\n",
      "INFO\t2017-01-12 16:01:55\tGenotypeConcordance\tchecked     9,390,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:144,818,822\n",
      "INFO\t2017-01-12 16:01:55\tGenotypeConcordance\tchecked     9,400,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:145,562,738\n",
      "INFO\t2017-01-12 16:01:55\tGenotypeConcordance\tchecked     9,410,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:146,602,653\n",
      "INFO\t2017-01-12 16:01:56\tGenotypeConcordance\tchecked     9,420,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:147,514,783\n",
      "INFO\t2017-01-12 16:01:56\tGenotypeConcordance\tchecked     9,430,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:148,503,146\n",
      "INFO\t2017-01-12 16:01:56\tGenotypeConcordance\tchecked     9,440,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:149,458,917\n",
      "INFO\t2017-01-12 16:01:56\tGenotypeConcordance\tchecked     9,450,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:150,443,928\n",
      "INFO\t2017-01-12 16:01:57\tGenotypeConcordance\tchecked     9,460,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:151,399,246\n",
      "INFO\t2017-01-12 16:01:57\tGenotypeConcordance\tchecked     9,470,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:152,493,890\n",
      "INFO\t2017-01-12 16:01:57\tGenotypeConcordance\tchecked     9,480,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:153,527,808\n",
      "INFO\t2017-01-12 16:01:57\tGenotypeConcordance\tchecked     9,490,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:154,394,503\n",
      "INFO\t2017-01-12 16:01:58\tGenotypeConcordance\tchecked     9,500,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:155,262,227\n",
      "INFO\t2017-01-12 16:01:58\tGenotypeConcordance\tchecked     9,510,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:156,167,667\n",
      "INFO\t2017-01-12 16:01:58\tGenotypeConcordance\tchecked     9,520,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:157,000,262\n",
      "INFO\t2017-01-12 16:01:59\tGenotypeConcordance\tchecked     9,530,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:157,797,969\n",
      "INFO\t2017-01-12 16:01:59\tGenotypeConcordance\tchecked     9,540,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:158,668,868\n",
      "INFO\t2017-01-12 16:01:59\tGenotypeConcordance\tchecked     9,550,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:159,641,244\n",
      "INFO\t2017-01-12 16:01:59\tGenotypeConcordance\tchecked     9,560,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:160,534,399\n",
      "INFO\t2017-01-12 16:02:00\tGenotypeConcordance\tchecked     9,570,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:161,344,495\n",
      "INFO\t2017-01-12 16:02:00\tGenotypeConcordance\tchecked     9,580,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:162,066,509\n",
      "INFO\t2017-01-12 16:02:00\tGenotypeConcordance\tchecked     9,590,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:162,836,241\n",
      "INFO\t2017-01-12 16:02:00\tGenotypeConcordance\tchecked     9,600,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:163,537,134\n",
      "INFO\t2017-01-12 16:02:01\tGenotypeConcordance\tchecked     9,610,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:164,384,855\n",
      "INFO\t2017-01-12 16:02:01\tGenotypeConcordance\tchecked     9,620,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:165,170,243\n",
      "INFO\t2017-01-12 16:02:01\tGenotypeConcordance\tchecked     9,630,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:165,900,979\n",
      "INFO\t2017-01-12 16:02:01\tGenotypeConcordance\tchecked     9,640,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:166,681,527\n",
      "INFO\t2017-01-12 16:02:02\tGenotypeConcordance\tchecked     9,650,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:167,434,458\n",
      "INFO\t2017-01-12 16:02:02\tGenotypeConcordance\tchecked     9,660,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:168,273,946\n",
      "INFO\t2017-01-12 16:02:02\tGenotypeConcordance\tchecked     9,670,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:169,121,588\n",
      "INFO\t2017-01-12 16:02:02\tGenotypeConcordance\tchecked     9,680,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:169,996,302\n",
      "INFO\t2017-01-12 16:02:03\tGenotypeConcordance\tchecked     9,690,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:170,914,622\n",
      "INFO\t2017-01-12 16:02:03\tGenotypeConcordance\tchecked     9,700,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:171,679,226\n",
      "INFO\t2017-01-12 16:02:03\tGenotypeConcordance\tchecked     9,710,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:172,489,942\n",
      "INFO\t2017-01-12 16:02:03\tGenotypeConcordance\tchecked     9,720,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:173,298,069\n",
      "INFO\t2017-01-12 16:02:04\tGenotypeConcordance\tchecked     9,730,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:174,216,526\n",
      "INFO\t2017-01-12 16:02:04\tGenotypeConcordance\tchecked     9,740,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:175,061,519\n",
      "INFO\t2017-01-12 16:02:04\tGenotypeConcordance\tchecked     9,750,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:175,887,670\n",
      "INFO\t2017-01-12 16:02:04\tGenotypeConcordance\tchecked     9,760,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:176,732,480\n",
      "INFO\t2017-01-12 16:02:05\tGenotypeConcordance\tchecked     9,770,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:177,532,499\n",
      "INFO\t2017-01-12 16:02:05\tGenotypeConcordance\tchecked     9,780,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:178,354,599\n",
      "INFO\t2017-01-12 16:02:05\tGenotypeConcordance\tchecked     9,790,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:179,136,091\n",
      "INFO\t2017-01-12 16:02:05\tGenotypeConcordance\tchecked     9,800,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:179,816,709\n",
      "INFO\t2017-01-12 16:02:06\tGenotypeConcordance\tchecked     9,810,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:180,507,487\n",
      "INFO\t2017-01-12 16:02:06\tGenotypeConcordance\tchecked     9,820,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:181,226,698\n",
      "INFO\t2017-01-12 16:02:06\tGenotypeConcordance\tchecked     9,830,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:182,002,228\n",
      "INFO\t2017-01-12 16:02:06\tGenotypeConcordance\tchecked     9,840,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:182,680,609\n",
      "INFO\t2017-01-12 16:02:07\tGenotypeConcordance\tchecked     9,850,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:183,458,715\n",
      "INFO\t2017-01-12 16:02:07\tGenotypeConcordance\tchecked     9,860,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:184,257,215\n",
      "INFO\t2017-01-12 16:02:07\tGenotypeConcordance\tchecked     9,870,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:184,913,872\n",
      "INFO\t2017-01-12 16:02:07\tGenotypeConcordance\tchecked     9,880,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:185,656,390\n",
      "INFO\t2017-01-12 16:02:08\tGenotypeConcordance\tchecked     9,890,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:186,418,492\n",
      "INFO\t2017-01-12 16:02:08\tGenotypeConcordance\tchecked     9,900,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:187,118,832\n",
      "INFO\t2017-01-12 16:02:08\tGenotypeConcordance\tchecked     9,910,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:187,835,603\n",
      "INFO\t2017-01-12 16:02:08\tGenotypeConcordance\tchecked     9,920,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:188,626,838\n",
      "INFO\t2017-01-12 16:02:09\tGenotypeConcordance\tchecked     9,930,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:189,251,223\n",
      "INFO\t2017-01-12 16:02:09\tGenotypeConcordance\tchecked     9,940,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:189,869,211\n",
      "INFO\t2017-01-12 16:02:09\tGenotypeConcordance\tchecked     9,950,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:190,520,824\n",
      "INFO\t2017-01-12 16:02:10\tGenotypeConcordance\tchecked     9,960,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:190,852,643\n",
      "INFO\t2017-01-12 16:02:10\tGenotypeConcordance\tchecked     9,970,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 5:442,904\n",
      "INFO\t2017-01-12 16:02:10\tGenotypeConcordance\tchecked     9,980,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 5:1,044,904\n",
      "INFO\t2017-01-12 16:02:10\tGenotypeConcordance\tchecked     9,990,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 5:1,634,161\n",
      "INFO\t2017-01-12 16:02:11\tGenotypeConcordance\tchecked    10,000,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 5:2,262,683\n",
      "INFO\t2017-01-12 16:02:11\tGenotypeConcordance\tchecked    10,010,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 5:2,886,975\n",
      "INFO\t2017-01-12 16:02:11\tGenotypeConcordance\tchecked    10,020,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 5:3,529,660\n",
      "INFO\t2017-01-12 16:02:12\tGenotypeConcordance\tchecked    10,030,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 5:4,250,000\n",
      "INFO\t2017-01-12 16:02:12\tGenotypeConcordance\tchecked    10,040,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 5:5,018,551\n",
      "INFO\t2017-01-12 16:02:12\tGenotypeConcordance\tchecked    10,050,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 5:5,742,391\n",
      "INFO\t2017-01-12 16:02:12\tGenotypeConcordance\tchecked    10,060,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 5:6,491,204\n",
      "INFO\t2017-01-12 16:02:13\tGenotypeConcordance\tchecked    10,070,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 5:7,260,750\n",
      "INFO\t2017-01-12 16:02:13\tGenotypeConcordance\tchecked    10,080,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 5:8,036,175\n",
      "INFO\t2017-01-12 16:02:13\tGenotypeConcordance\tchecked    10,090,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 5:8,744,233\n",
      "INFO\t2017-01-12 16:02:13\tGenotypeConcordance\tchecked    10,100,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 5:9,551,780\n",
      "INFO\t2017-01-12 16:02:14\tGenotypeConcordance\tchecked    10,110,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 5:10,332,965\n",
      "INFO\t2017-01-12 16:02:14\tGenotypeConcordance\tchecked    10,120,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 5:11,152,355\n",
      "INFO\t2017-01-12 16:02:14\tGenotypeConcordance\tchecked    10,130,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 5:11,996,952\n",
      "INFO\t2017-01-12 16:02:14\tGenotypeConcordance\tchecked    10,140,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 5:12,884,007\n",
      "INFO\t2017-01-12 16:02:15\tGenotypeConcordance\tchecked    10,150,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 5:13,652,373\n",
      "INFO\t2017-01-12 16:02:15\tGenotypeConcordance\tchecked    10,160,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 5:14,483,612\n",
      "INFO\t2017-01-12 16:02:15\tGenotypeConcordance\tchecked    10,170,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 5:15,332,828\n",
      "INFO\t2017-01-12 16:02:15\tGenotypeConcordance\tchecked    10,180,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 5:16,297,063\n",
      "INFO\t2017-01-12 16:02:16\tGenotypeConcordance\tchecked    10,190,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 5:17,033,036\n",
      "INFO\t2017-01-12 16:02:16\tGenotypeConcordance\tchecked    10,200,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 5:17,886,282\n",
      "INFO\t2017-01-12 16:02:16\tGenotypeConcordance\tchecked    10,210,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 5:18,640,846\n",
      "INFO\t2017-01-12 16:02:16\tGenotypeConcordance\tchecked    10,220,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 5:19,470,753\n",
      "INFO\t2017-01-12 16:02:17\tGenotypeConcordance\tchecked    10,230,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 5:20,340,926\n",
      "INFO\t2017-01-12 16:02:17\tGenotypeConcordance\tchecked    10,240,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 5:21,188,773\n",
      "INFO\t2017-01-12 16:02:17\tGenotypeConcordance\tchecked    10,250,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 5:21,934,011\n",
      "INFO\t2017-01-12 16:02:17\tGenotypeConcordance\tchecked    10,260,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 5:22,796,407\n",
      "INFO\t2017-01-12 16:02:17\tGenotypeConcordance\tchecked    10,270,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 5:23,614,535\n",
      "INFO\t2017-01-12 16:02:18\tGenotypeConcordance\tchecked    10,280,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 5:24,402,069\n",
      "INFO\t2017-01-12 16:02:18\tGenotypeConcordance\tchecked    10,290,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 5:25,260,064\n",
      "INFO\t2017-01-12 16:02:18\tGenotypeConcordance\tchecked    10,300,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 5:26,005,870\n",
      "INFO\t2017-01-12 16:02:18\tGenotypeConcordance\tchecked    10,310,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 5:26,737,371\n",
      "INFO\t2017-01-12 16:02:19\tGenotypeConcordance\tchecked    10,320,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 5:27,561,458\n",
      "INFO\t2017-01-12 16:02:19\tGenotypeConcordance\tchecked    10,330,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 5:28,376,064\n",
      "INFO\t2017-01-12 16:02:19\tGenotypeConcordance\tchecked    10,340,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 5:29,221,051\n",
      "INFO\t2017-01-12 16:02:19\tGenotypeConcordance\tchecked    10,350,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 5:29,857,854\n",
      "INFO\t2017-01-12 16:02:20\tGenotypeConcordance\tchecked    10,360,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 5:30,635,630\n",
      "INFO\t2017-01-12 16:02:20\tGenotypeConcordance\tchecked    10,370,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 5:31,486,791\n",
      "INFO\t2017-01-12 16:02:20\tGenotypeConcordance\tchecked    10,380,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 5:32,247,129\n",
      "INFO\t2017-01-12 16:02:20\tGenotypeConcordance\tchecked    10,390,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 5:33,084,945\n",
      "INFO\t2017-01-12 16:02:21\tGenotypeConcordance\tchecked    10,400,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 5:33,869,728\n",
      "INFO\t2017-01-12 16:02:21\tGenotypeConcordance\tchecked    10,410,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 5:34,702,457\n",
      "INFO\t2017-01-12 16:02:21\tGenotypeConcordance\tchecked    10,420,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 5:35,540,135\n",
      "INFO\t2017-01-12 16:02:21\tGenotypeConcordance\tchecked    10,430,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 5:36,414,401\n",
      "INFO\t2017-01-12 16:02:22\tGenotypeConcordance\tchecked    10,440,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 5:37,442,135\n",
      "INFO\t2017-01-12 16:02:22\tGenotypeConcordance\tchecked    10,450,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 5:38,297,409\n",
      "INFO\t2017-01-12 16:02:22\tGenotypeConcordance\tchecked    10,460,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 5:39,155,400\n",
      "INFO\t2017-01-12 16:02:22\tGenotypeConcordance\tchecked    10,470,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 5:39,942,034\n",
      "INFO\t2017-01-12 16:02:23\tGenotypeConcordance\tchecked    10,480,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 5:40,734,183\n",
      "INFO\t2017-01-12 16:02:23\tGenotypeConcordance\tchecked    10,490,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 5:41,527,256\n",
      "INFO\t2017-01-12 16:02:23\tGenotypeConcordance\tchecked    10,500,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 5:42,356,099\n",
      "INFO\t2017-01-12 16:02:23\tGenotypeConcordance\tchecked    10,510,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 5:43,198,241\n",
      "INFO\t2017-01-12 16:02:24\tGenotypeConcordance\tchecked    10,520,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 5:44,041,626\n",
      "INFO\t2017-01-12 16:02:24\tGenotypeConcordance\tchecked    10,530,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 5:44,986,828\n",
      "INFO\t2017-01-12 16:02:24\tGenotypeConcordance\tchecked    10,540,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 5:45,891,794\n",
      "INFO\t2017-01-12 16:02:24\tGenotypeConcordance\tchecked    10,550,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 5:46,363,228\n",
      "INFO\t2017-01-12 16:02:25\tGenotypeConcordance\tchecked    10,560,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 5:49,878,993\n",
      "INFO\t2017-01-12 16:02:25\tGenotypeConcordance\tchecked    10,570,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 5:50,776,123\n",
      "INFO\t2017-01-12 16:02:25\tGenotypeConcordance\tchecked    10,580,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 5:51,592,067\n",
      "INFO\t2017-01-12 16:02:25\tGenotypeConcordance\tchecked    10,590,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 5:52,430,327\n",
      "INFO\t2017-01-12 16:02:26\tGenotypeConcordance\tchecked    10,600,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 5:53,268,199\n",
      "INFO\t2017-01-12 16:02:26\tGenotypeConcordance\tchecked    10,610,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 5:54,083,713\n",
      "INFO\t2017-01-12 16:02:26\tGenotypeConcordance\tchecked    10,620,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 5:54,969,713\n",
      "INFO\t2017-01-12 16:02:26\tGenotypeConcordance\tchecked    10,630,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 5:55,801,586\n",
      "INFO\t2017-01-12 16:02:27\tGenotypeConcordance\tchecked    10,640,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 5:56,717,667\n",
      "INFO\t2017-01-12 16:02:27\tGenotypeConcordance\tchecked    10,650,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 5:57,576,673\n",
      "INFO\t2017-01-12 16:02:27\tGenotypeConcordance\tchecked    10,660,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 5:58,424,097\n",
      "INFO\t2017-01-12 16:02:27\tGenotypeConcordance\tchecked    10,670,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 5:59,399,042\n",
      "INFO\t2017-01-12 16:02:28\tGenotypeConcordance\tchecked    10,680,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 5:60,467,949\n",
      "INFO\t2017-01-12 16:02:28\tGenotypeConcordance\tchecked    10,690,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 5:61,486,338\n",
      "INFO\t2017-01-12 16:02:28\tGenotypeConcordance\tchecked    10,700,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 5:62,380,641\n",
      "INFO\t2017-01-12 16:02:28\tGenotypeConcordance\tchecked    10,710,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 5:63,252,049\n",
      "INFO\t2017-01-12 16:02:29\tGenotypeConcordance\tchecked    10,720,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 5:64,263,903\n",
      "INFO\t2017-01-12 16:02:29\tGenotypeConcordance\tchecked    10,730,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 5:65,223,029\n",
      "INFO\t2017-01-12 16:02:29\tGenotypeConcordance\tchecked    10,740,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 5:66,081,464\n",
      "INFO\t2017-01-12 16:02:29\tGenotypeConcordance\tchecked    10,750,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 5:66,990,048\n",
      "INFO\t2017-01-12 16:02:30\tGenotypeConcordance\tchecked    10,760,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 5:67,866,560\n",
      "INFO\t2017-01-12 16:02:30\tGenotypeConcordance\tchecked    10,770,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 5:68,829,079\n",
      "INFO\t2017-01-12 16:02:30\tGenotypeConcordance\tchecked    10,780,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 5:71,324,354\n",
      "INFO\t2017-01-12 16:02:30\tGenotypeConcordance\tchecked    10,790,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 5:72,356,452\n",
      "INFO\t2017-01-12 16:02:31\tGenotypeConcordance\tchecked    10,800,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 5:73,255,812\n",
      "INFO\t2017-01-12 16:02:31\tGenotypeConcordance\tchecked    10,810,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 5:74,074,589\n",
      "INFO\t2017-01-12 16:02:31\tGenotypeConcordance\tchecked    10,820,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 5:75,057,430\n",
      "INFO\t2017-01-12 16:02:31\tGenotypeConcordance\tchecked    10,830,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 5:75,792,550\n",
      "INFO\t2017-01-12 16:02:31\tGenotypeConcordance\tchecked    10,840,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 5:76,665,184\n",
      "INFO\t2017-01-12 16:02:32\tGenotypeConcordance\tchecked    10,850,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 5:77,570,082\n",
      "INFO\t2017-01-12 16:02:32\tGenotypeConcordance\tchecked    10,860,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 5:78,435,578\n",
      "INFO\t2017-01-12 16:02:32\tGenotypeConcordance\tchecked    10,870,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 5:79,384,952\n",
      "INFO\t2017-01-12 16:02:32\tGenotypeConcordance\tchecked    10,880,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 5:80,183,772\n",
      "INFO\t2017-01-12 16:02:33\tGenotypeConcordance\tchecked    10,890,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 5:81,119,661\n",
      "INFO\t2017-01-12 16:02:33\tGenotypeConcordance\tchecked    10,900,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 5:82,100,752\n",
      "INFO\t2017-01-12 16:02:33\tGenotypeConcordance\tchecked    10,910,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 5:82,988,485\n",
      "INFO\t2017-01-12 16:02:33\tGenotypeConcordance\tchecked    10,920,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 5:83,977,669\n",
      "INFO\t2017-01-12 16:02:34\tGenotypeConcordance\tchecked    10,930,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 5:84,755,439\n",
      "INFO\t2017-01-12 16:02:34\tGenotypeConcordance\tchecked    10,940,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 5:85,562,014\n",
      "INFO\t2017-01-12 16:02:34\tGenotypeConcordance\tchecked    10,950,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 5:86,454,768\n",
      "INFO\t2017-01-12 16:02:34\tGenotypeConcordance\tchecked    10,960,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 5:87,609,719\n",
      "INFO\t2017-01-12 16:02:35\tGenotypeConcordance\tchecked    10,970,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 5:88,875,426\n",
      "INFO\t2017-01-12 16:02:35\tGenotypeConcordance\tchecked    10,980,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 5:89,930,666\n",
      "INFO\t2017-01-12 16:02:35\tGenotypeConcordance\tchecked    10,990,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 5:90,893,636\n",
      "INFO\t2017-01-12 16:02:35\tGenotypeConcordance\tchecked    11,000,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 5:92,005,876\n",
      "INFO\t2017-01-12 16:02:36\tGenotypeConcordance\tchecked    11,010,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:93,183,020\n",
      "INFO\t2017-01-12 16:02:36\tGenotypeConcordance\tchecked    11,020,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:94,366,450\n",
      "INFO\t2017-01-12 16:02:36\tGenotypeConcordance\tchecked    11,030,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:95,348,201\n",
      "INFO\t2017-01-12 16:02:36\tGenotypeConcordance\tchecked    11,040,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 5:96,243,650\n",
      "INFO\t2017-01-12 16:02:37\tGenotypeConcordance\tchecked    11,050,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:97,060,608\n",
      "INFO\t2017-01-12 16:02:37\tGenotypeConcordance\tchecked    11,060,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:97,797,639\n",
      "INFO\t2017-01-12 16:02:37\tGenotypeConcordance\tchecked    11,070,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:98,679,017\n",
      "INFO\t2017-01-12 16:02:37\tGenotypeConcordance\tchecked    11,080,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 5:99,450,876\n",
      "INFO\t2017-01-12 16:02:38\tGenotypeConcordance\tchecked    11,090,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:100,271,134\n",
      "INFO\t2017-01-12 16:02:38\tGenotypeConcordance\tchecked    11,100,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:101,117,693\n",
      "INFO\t2017-01-12 16:02:38\tGenotypeConcordance\tchecked    11,110,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:101,923,665\n",
      "INFO\t2017-01-12 16:02:38\tGenotypeConcordance\tchecked    11,120,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 5:102,855,869\n",
      "INFO\t2017-01-12 16:02:39\tGenotypeConcordance\tchecked    11,130,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:103,673,009\n",
      "INFO\t2017-01-12 16:02:39\tGenotypeConcordance\tchecked    11,140,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:104,490,778\n",
      "INFO\t2017-01-12 16:02:39\tGenotypeConcordance\tchecked    11,150,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:105,387,654\n",
      "INFO\t2017-01-12 16:02:39\tGenotypeConcordance\tchecked    11,160,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 5:106,232,456\n",
      "INFO\t2017-01-12 16:02:40\tGenotypeConcordance\tchecked    11,170,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:107,063,746\n",
      "INFO\t2017-01-12 16:02:40\tGenotypeConcordance\tchecked    11,180,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:107,893,632\n",
      "INFO\t2017-01-12 16:02:40\tGenotypeConcordance\tchecked    11,190,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:108,656,067\n",
      "INFO\t2017-01-12 16:02:40\tGenotypeConcordance\tchecked    11,200,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 5:109,449,361\n",
      "INFO\t2017-01-12 16:02:40\tGenotypeConcordance\tchecked    11,210,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:110,188,988\n",
      "INFO\t2017-01-12 16:02:41\tGenotypeConcordance\tchecked    11,220,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:110,982,398\n",
      "INFO\t2017-01-12 16:02:41\tGenotypeConcordance\tchecked    11,230,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:111,799,267\n",
      "INFO\t2017-01-12 16:02:41\tGenotypeConcordance\tchecked    11,240,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 5:112,500,404\n",
      "INFO\t2017-01-12 16:02:42\tGenotypeConcordance\tchecked    11,250,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:113,282,113\n",
      "INFO\t2017-01-12 16:02:42\tGenotypeConcordance\tchecked    11,260,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:114,043,417\n",
      "INFO\t2017-01-12 16:02:42\tGenotypeConcordance\tchecked    11,270,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:114,747,126\n",
      "INFO\t2017-01-12 16:02:42\tGenotypeConcordance\tchecked    11,280,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 5:115,475,502\n",
      "INFO\t2017-01-12 16:02:43\tGenotypeConcordance\tchecked    11,290,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:116,128,151\n",
      "INFO\t2017-01-12 16:02:43\tGenotypeConcordance\tchecked    11,300,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:116,860,242\n",
      "INFO\t2017-01-12 16:02:43\tGenotypeConcordance\tchecked    11,310,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:117,586,651\n",
      "INFO\t2017-01-12 16:02:43\tGenotypeConcordance\tchecked    11,320,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 5:118,268,018\n",
      "INFO\t2017-01-12 16:02:43\tGenotypeConcordance\tchecked    11,330,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:119,055,044\n",
      "INFO\t2017-01-12 16:02:44\tGenotypeConcordance\tchecked    11,340,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:119,804,529\n",
      "INFO\t2017-01-12 16:02:44\tGenotypeConcordance\tchecked    11,350,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:120,537,042\n",
      "INFO\t2017-01-12 16:02:44\tGenotypeConcordance\tchecked    11,360,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 5:121,238,944\n",
      "INFO\t2017-01-12 16:02:44\tGenotypeConcordance\tchecked    11,370,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:122,087,919\n",
      "INFO\t2017-01-12 16:02:45\tGenotypeConcordance\tchecked    11,380,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:122,999,952\n",
      "INFO\t2017-01-12 16:02:45\tGenotypeConcordance\tchecked    11,390,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:123,816,401\n",
      "INFO\t2017-01-12 16:02:45\tGenotypeConcordance\tchecked    11,400,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 5:124,590,123\n",
      "INFO\t2017-01-12 16:02:45\tGenotypeConcordance\tchecked    11,410,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:125,350,827\n",
      "INFO\t2017-01-12 16:02:46\tGenotypeConcordance\tchecked    11,420,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:126,106,785\n",
      "INFO\t2017-01-12 16:02:46\tGenotypeConcordance\tchecked    11,430,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:126,973,564\n",
      "INFO\t2017-01-12 16:02:46\tGenotypeConcordance\tchecked    11,440,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:127,962,811\n",
      "INFO\t2017-01-12 16:02:46\tGenotypeConcordance\tchecked    11,450,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 5:128,755,827\n",
      "INFO\t2017-01-12 16:02:47\tGenotypeConcordance\tchecked    11,460,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:129,618,090\n",
      "INFO\t2017-01-12 16:02:47\tGenotypeConcordance\tchecked    11,470,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:130,538,004\n",
      "INFO\t2017-01-12 16:02:47\tGenotypeConcordance\tchecked    11,480,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:131,488,880\n",
      "INFO\t2017-01-12 16:02:47\tGenotypeConcordance\tchecked    11,490,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 5:132,444,721\n",
      "INFO\t2017-01-12 16:02:48\tGenotypeConcordance\tchecked    11,500,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:133,310,944\n",
      "INFO\t2017-01-12 16:02:48\tGenotypeConcordance\tchecked    11,510,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:134,343,130\n",
      "INFO\t2017-01-12 16:02:48\tGenotypeConcordance\tchecked    11,520,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:135,213,719\n",
      "INFO\t2017-01-12 16:02:48\tGenotypeConcordance\tchecked    11,530,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 5:136,146,592\n",
      "INFO\t2017-01-12 16:02:49\tGenotypeConcordance\tchecked    11,540,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:137,147,168\n",
      "INFO\t2017-01-12 16:02:49\tGenotypeConcordance\tchecked    11,550,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:138,262,670\n",
      "INFO\t2017-01-12 16:02:49\tGenotypeConcordance\tchecked    11,560,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:139,523,692\n",
      "INFO\t2017-01-12 16:02:49\tGenotypeConcordance\tchecked    11,570,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 5:140,578,250\n",
      "INFO\t2017-01-12 16:02:50\tGenotypeConcordance\tchecked    11,580,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:141,506,555\n",
      "INFO\t2017-01-12 16:02:50\tGenotypeConcordance\tchecked    11,590,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:142,349,030\n",
      "INFO\t2017-01-12 16:02:50\tGenotypeConcordance\tchecked    11,600,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:143,268,544\n",
      "INFO\t2017-01-12 16:02:50\tGenotypeConcordance\tchecked    11,610,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 5:144,098,567\n",
      "INFO\t2017-01-12 16:02:50\tGenotypeConcordance\tchecked    11,620,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:144,968,443\n",
      "INFO\t2017-01-12 16:02:51\tGenotypeConcordance\tchecked    11,630,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:145,792,218\n",
      "INFO\t2017-01-12 16:02:51\tGenotypeConcordance\tchecked    11,640,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:146,672,934\n",
      "INFO\t2017-01-12 16:02:51\tGenotypeConcordance\tchecked    11,650,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 5:147,516,195\n",
      "INFO\t2017-01-12 16:02:51\tGenotypeConcordance\tchecked    11,660,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:148,382,327\n",
      "INFO\t2017-01-12 16:02:52\tGenotypeConcordance\tchecked    11,670,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:149,254,197\n",
      "INFO\t2017-01-12 16:02:52\tGenotypeConcordance\tchecked    11,680,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:150,126,266\n",
      "INFO\t2017-01-12 16:02:52\tGenotypeConcordance\tchecked    11,690,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 5:150,904,341\n",
      "INFO\t2017-01-12 16:02:52\tGenotypeConcordance\tchecked    11,700,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:151,776,870\n",
      "INFO\t2017-01-12 16:02:53\tGenotypeConcordance\tchecked    11,710,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:152,662,803\n",
      "INFO\t2017-01-12 16:02:53\tGenotypeConcordance\tchecked    11,720,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:153,584,135\n",
      "INFO\t2017-01-12 16:02:53\tGenotypeConcordance\tchecked    11,730,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 5:154,491,541\n",
      "INFO\t2017-01-12 16:02:53\tGenotypeConcordance\tchecked    11,740,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:155,395,740\n",
      "INFO\t2017-01-12 16:02:54\tGenotypeConcordance\tchecked    11,750,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:156,258,787\n",
      "INFO\t2017-01-12 16:02:54\tGenotypeConcordance\tchecked    11,760,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:157,082,025\n",
      "INFO\t2017-01-12 16:02:54\tGenotypeConcordance\tchecked    11,770,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:157,906,224\n",
      "INFO\t2017-01-12 16:02:54\tGenotypeConcordance\tchecked    11,780,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 5:158,930,967\n",
      "INFO\t2017-01-12 16:02:55\tGenotypeConcordance\tchecked    11,790,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:159,764,701\n",
      "INFO\t2017-01-12 16:02:55\tGenotypeConcordance\tchecked    11,800,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:160,558,193\n",
      "INFO\t2017-01-12 16:02:55\tGenotypeConcordance\tchecked    11,810,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:161,447,890\n",
      "INFO\t2017-01-12 16:02:55\tGenotypeConcordance\tchecked    11,820,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 5:162,290,454\n",
      "INFO\t2017-01-12 16:02:56\tGenotypeConcordance\tchecked    11,830,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:163,154,418\n",
      "INFO\t2017-01-12 16:02:56\tGenotypeConcordance\tchecked    11,840,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:164,060,480\n",
      "INFO\t2017-01-12 16:02:56\tGenotypeConcordance\tchecked    11,850,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:164,962,496\n",
      "INFO\t2017-01-12 16:02:56\tGenotypeConcordance\tchecked    11,860,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 5:165,828,151\n",
      "INFO\t2017-01-12 16:02:57\tGenotypeConcordance\tchecked    11,870,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:166,698,017\n",
      "INFO\t2017-01-12 16:02:57\tGenotypeConcordance\tchecked    11,880,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:167,624,491\n",
      "INFO\t2017-01-12 16:02:57\tGenotypeConcordance\tchecked    11,890,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:168,444,499\n",
      "INFO\t2017-01-12 16:02:57\tGenotypeConcordance\tchecked    11,900,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 5:169,285,912\n",
      "INFO\t2017-01-12 16:02:58\tGenotypeConcordance\tchecked    11,910,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:170,131,207\n",
      "INFO\t2017-01-12 16:02:58\tGenotypeConcordance\tchecked    11,920,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:170,979,510\n",
      "INFO\t2017-01-12 16:02:58\tGenotypeConcordance\tchecked    11,930,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:171,797,320\n",
      "INFO\t2017-01-12 16:02:58\tGenotypeConcordance\tchecked    11,940,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 5:172,625,758\n",
      "INFO\t2017-01-12 16:02:59\tGenotypeConcordance\tchecked    11,950,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:173,447,445\n",
      "INFO\t2017-01-12 16:02:59\tGenotypeConcordance\tchecked    11,960,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:174,222,149\n",
      "INFO\t2017-01-12 16:02:59\tGenotypeConcordance\tchecked    11,970,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:174,974,131\n",
      "INFO\t2017-01-12 16:02:59\tGenotypeConcordance\tchecked    11,980,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 5:175,879,296\n",
      "INFO\t2017-01-12 16:03:00\tGenotypeConcordance\tchecked    11,990,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:176,742,298\n",
      "INFO\t2017-01-12 16:03:00\tGenotypeConcordance\tchecked    12,000,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:177,618,851\n",
      "INFO\t2017-01-12 16:03:00\tGenotypeConcordance\tchecked    12,010,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:178,260,360\n",
      "INFO\t2017-01-12 16:03:00\tGenotypeConcordance\tchecked    12,020,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 5:178,906,447\n",
      "INFO\t2017-01-12 16:03:01\tGenotypeConcordance\tchecked    12,030,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:179,670,271\n",
      "INFO\t2017-01-12 16:03:01\tGenotypeConcordance\tchecked    12,040,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 5:180,340,541\n",
      "INFO\t2017-01-12 16:03:01\tGenotypeConcordance\tchecked    12,050,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 6:471,112\n",
      "INFO\t2017-01-12 16:03:01\tGenotypeConcordance\tchecked    12,060,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 6:1,216,201\n",
      "INFO\t2017-01-12 16:03:02\tGenotypeConcordance\tchecked    12,070,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 6:1,981,340\n",
      "INFO\t2017-01-12 16:03:02\tGenotypeConcordance\tchecked    12,080,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 6:2,796,358\n",
      "INFO\t2017-01-12 16:03:02\tGenotypeConcordance\tchecked    12,090,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 6:3,464,174\n",
      "INFO\t2017-01-12 16:03:02\tGenotypeConcordance\tchecked    12,100,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 6:4,194,676\n",
      "INFO\t2017-01-12 16:03:03\tGenotypeConcordance\tchecked    12,110,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 6:4,932,311\n",
      "INFO\t2017-01-12 16:03:03\tGenotypeConcordance\tchecked    12,120,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 6:5,684,333\n",
      "INFO\t2017-01-12 16:03:03\tGenotypeConcordance\tchecked    12,130,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 6:6,454,886\n",
      "INFO\t2017-01-12 16:03:03\tGenotypeConcordance\tchecked    12,140,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 6:7,153,646\n",
      "INFO\t2017-01-12 16:03:04\tGenotypeConcordance\tchecked    12,150,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 6:7,938,293\n",
      "INFO\t2017-01-12 16:03:04\tGenotypeConcordance\tchecked    12,160,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 6:8,668,730\n",
      "INFO\t2017-01-12 16:03:04\tGenotypeConcordance\tchecked    12,170,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 6:9,507,852\n",
      "INFO\t2017-01-12 16:03:04\tGenotypeConcordance\tchecked    12,180,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 6:10,343,891\n",
      "INFO\t2017-01-12 16:03:05\tGenotypeConcordance\tchecked    12,190,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 6:11,086,696\n",
      "INFO\t2017-01-12 16:03:05\tGenotypeConcordance\tchecked    12,200,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 6:11,900,987\n",
      "INFO\t2017-01-12 16:03:05\tGenotypeConcordance\tchecked    12,210,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 6:12,756,394\n",
      "INFO\t2017-01-12 16:03:05\tGenotypeConcordance\tchecked    12,220,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 6:13,609,053\n",
      "INFO\t2017-01-12 16:03:06\tGenotypeConcordance\tchecked    12,230,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 6:14,445,739\n",
      "INFO\t2017-01-12 16:03:06\tGenotypeConcordance\tchecked    12,240,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 6:15,240,194\n",
      "INFO\t2017-01-12 16:03:06\tGenotypeConcordance\tchecked    12,250,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 6:16,087,931\n",
      "INFO\t2017-01-12 16:03:06\tGenotypeConcordance\tchecked    12,260,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 6:16,966,904\n",
      "INFO\t2017-01-12 16:03:07\tGenotypeConcordance\tchecked    12,270,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 6:17,801,284\n",
      "INFO\t2017-01-12 16:03:07\tGenotypeConcordance\tchecked    12,280,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 6:18,601,148\n",
      "INFO\t2017-01-12 16:03:07\tGenotypeConcordance\tchecked    12,290,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 6:19,431,136\n",
      "INFO\t2017-01-12 16:03:07\tGenotypeConcordance\tchecked    12,300,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 6:20,212,662\n",
      "INFO\t2017-01-12 16:03:08\tGenotypeConcordance\tchecked    12,310,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 6:21,045,170\n",
      "INFO\t2017-01-12 16:03:08\tGenotypeConcordance\tchecked    12,320,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 6:21,904,782\n",
      "INFO\t2017-01-12 16:03:08\tGenotypeConcordance\tchecked    12,330,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 6:22,668,647\n",
      "INFO\t2017-01-12 16:03:08\tGenotypeConcordance\tchecked    12,340,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 6:23,489,057\n",
      "INFO\t2017-01-12 16:03:09\tGenotypeConcordance\tchecked    12,350,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 6:24,097,567\n",
      "INFO\t2017-01-12 16:03:09\tGenotypeConcordance\tchecked    12,360,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 6:24,872,685\n",
      "INFO\t2017-01-12 16:03:09\tGenotypeConcordance\tchecked    12,370,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 6:25,649,715\n",
      "INFO\t2017-01-12 16:03:09\tGenotypeConcordance\tchecked    12,380,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 6:26,534,616\n",
      "INFO\t2017-01-12 16:03:10\tGenotypeConcordance\tchecked    12,390,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 6:27,470,810\n",
      "INFO\t2017-01-12 16:03:10\tGenotypeConcordance\tchecked    12,400,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 6:28,462,083\n",
      "INFO\t2017-01-12 16:03:10\tGenotypeConcordance\tchecked    12,410,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 6:29,286,670\n",
      "INFO\t2017-01-12 16:03:10\tGenotypeConcordance\tchecked    12,420,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 6:29,811,022\n",
      "INFO\t2017-01-12 16:03:11\tGenotypeConcordance\tchecked    12,430,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 6:30,044,935\n",
      "INFO\t2017-01-12 16:03:11\tGenotypeConcordance\tchecked    12,440,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 6:30,757,813\n",
      "INFO\t2017-01-12 16:03:11\tGenotypeConcordance\tchecked    12,450,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 6:31,234,693\n",
      "INFO\t2017-01-12 16:03:12\tGenotypeConcordance\tchecked    12,460,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 6:31,480,632\n",
      "INFO\t2017-01-12 16:03:12\tGenotypeConcordance\tchecked    12,470,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 6:32,360,464\n",
      "INFO\t2017-01-12 16:03:12\tGenotypeConcordance\tchecked    12,480,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 6:32,505,266\n",
      "INFO\t2017-01-12 16:03:12\tGenotypeConcordance\tchecked    12,490,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 6:32,589,327\n",
      "INFO\t2017-01-12 16:03:13\tGenotypeConcordance\tchecked    12,500,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 6:32,728,842\n",
      "INFO\t2017-01-12 16:03:13\tGenotypeConcordance\tchecked    12,510,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 6:33,226,633\n",
      "INFO\t2017-01-12 16:03:13\tGenotypeConcordance\tchecked    12,520,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 6:33,970,014\n",
      "INFO\t2017-01-12 16:03:14\tGenotypeConcordance\tchecked    12,530,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 6:34,715,186\n",
      "INFO\t2017-01-12 16:03:14\tGenotypeConcordance\tchecked    12,540,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 6:35,664,772\n",
      "INFO\t2017-01-12 16:03:14\tGenotypeConcordance\tchecked    12,550,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 6:36,544,329\n",
      "INFO\t2017-01-12 16:03:14\tGenotypeConcordance\tchecked    12,560,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 6:37,320,844\n",
      "INFO\t2017-01-12 16:03:15\tGenotypeConcordance\tchecked    12,570,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 6:38,169,864\n",
      "INFO\t2017-01-12 16:03:15\tGenotypeConcordance\tchecked    12,580,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 6:39,033,063\n",
      "INFO\t2017-01-12 16:03:15\tGenotypeConcordance\tchecked    12,590,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 6:39,853,946\n",
      "INFO\t2017-01-12 16:03:15\tGenotypeConcordance\tchecked    12,600,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 6:40,640,259\n",
      "INFO\t2017-01-12 16:03:16\tGenotypeConcordance\tchecked    12,610,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 6:41,449,935\n",
      "INFO\t2017-01-12 16:03:16\tGenotypeConcordance\tchecked    12,620,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 6:42,247,863\n",
      "INFO\t2017-01-12 16:03:16\tGenotypeConcordance\tchecked    12,630,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 6:43,186,138\n",
      "INFO\t2017-01-12 16:03:16\tGenotypeConcordance\tchecked    12,640,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 6:44,098,694\n",
      "INFO\t2017-01-12 16:03:17\tGenotypeConcordance\tchecked    12,650,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 6:44,909,518\n",
      "INFO\t2017-01-12 16:03:17\tGenotypeConcordance\tchecked    12,660,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 6:45,768,698\n",
      "INFO\t2017-01-12 16:03:17\tGenotypeConcordance\tchecked    12,670,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 6:46,675,025\n",
      "INFO\t2017-01-12 16:03:17\tGenotypeConcordance\tchecked    12,680,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 6:47,479,657\n",
      "INFO\t2017-01-12 16:03:17\tGenotypeConcordance\tchecked    12,690,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 6:48,305,445\n",
      "INFO\t2017-01-12 16:03:18\tGenotypeConcordance\tchecked    12,700,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 6:49,088,350\n",
      "INFO\t2017-01-12 16:03:18\tGenotypeConcordance\tchecked    12,710,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 6:49,927,623\n",
      "INFO\t2017-01-12 16:03:18\tGenotypeConcordance\tchecked    12,720,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 6:50,844,870\n",
      "INFO\t2017-01-12 16:03:18\tGenotypeConcordance\tchecked    12,730,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 6:51,746,771\n",
      "INFO\t2017-01-12 16:03:19\tGenotypeConcordance\tchecked    12,740,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 6:52,615,360\n",
      "INFO\t2017-01-12 16:03:19\tGenotypeConcordance\tchecked    12,750,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 6:53,412,381\n",
      "INFO\t2017-01-12 16:03:19\tGenotypeConcordance\tchecked    12,760,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 6:54,258,078\n",
      "INFO\t2017-01-12 16:03:19\tGenotypeConcordance\tchecked    12,770,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 6:55,013,303\n",
      "INFO\t2017-01-12 16:03:20\tGenotypeConcordance\tchecked    12,780,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 6:55,786,412\n",
      "INFO\t2017-01-12 16:03:20\tGenotypeConcordance\tchecked    12,790,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 6:56,665,182\n",
      "INFO\t2017-01-12 16:03:20\tGenotypeConcordance\tchecked    12,800,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 6:57,371,946\n",
      "INFO\t2017-01-12 16:03:21\tGenotypeConcordance\tchecked    12,810,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 6:57,588,249\n",
      "INFO\t2017-01-12 16:03:21\tGenotypeConcordance\tchecked    12,820,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 6:58,370,815\n",
      "INFO\t2017-01-12 16:03:21\tGenotypeConcordance\tchecked    12,830,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 6:62,054,774\n",
      "INFO\t2017-01-12 16:03:21\tGenotypeConcordance\tchecked    12,840,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 6:62,838,574\n",
      "INFO\t2017-01-12 16:03:21\tGenotypeConcordance\tchecked    12,850,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 6:63,638,516\n",
      "INFO\t2017-01-12 16:03:22\tGenotypeConcordance\tchecked    12,860,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 6:64,544,537\n",
      "INFO\t2017-01-12 16:03:22\tGenotypeConcordance\tchecked    12,870,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 6:65,351,867\n",
      "INFO\t2017-01-12 16:03:22\tGenotypeConcordance\tchecked    12,880,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 6:66,133,980\n",
      "INFO\t2017-01-12 16:03:22\tGenotypeConcordance\tchecked    12,890,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 6:66,889,399\n",
      "INFO\t2017-01-12 16:03:23\tGenotypeConcordance\tchecked    12,900,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 6:67,550,407\n",
      "INFO\t2017-01-12 16:03:23\tGenotypeConcordance\tchecked    12,910,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 6:68,223,010\n",
      "INFO\t2017-01-12 16:03:23\tGenotypeConcordance\tchecked    12,920,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 6:69,012,763\n",
      "INFO\t2017-01-12 16:03:24\tGenotypeConcordance\tchecked    12,930,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 6:69,849,256\n",
      "INFO\t2017-01-12 16:03:24\tGenotypeConcordance\tchecked    12,940,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 6:70,734,005\n",
      "INFO\t2017-01-12 16:03:24\tGenotypeConcordance\tchecked    12,950,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 6:71,606,550\n",
      "INFO\t2017-01-12 16:03:24\tGenotypeConcordance\tchecked    12,960,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 6:72,399,994\n",
      "INFO\t2017-01-12 16:03:24\tGenotypeConcordance\tchecked    12,970,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 6:73,293,710\n",
      "INFO\t2017-01-12 16:03:25\tGenotypeConcordance\tchecked    12,980,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 6:74,213,881\n",
      "INFO\t2017-01-12 16:03:25\tGenotypeConcordance\tchecked    12,990,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 6:74,971,740\n",
      "INFO\t2017-01-12 16:03:25\tGenotypeConcordance\tchecked    13,000,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 6:75,761,832\n",
      "INFO\t2017-01-12 16:03:25\tGenotypeConcordance\tchecked    13,010,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 6:76,803,226\n",
      "INFO\t2017-01-12 16:03:26\tGenotypeConcordance\tchecked    13,020,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 6:77,503,299\n",
      "INFO\t2017-01-12 16:03:26\tGenotypeConcordance\tchecked    13,030,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 6:78,320,930\n",
      "INFO\t2017-01-12 16:03:26\tGenotypeConcordance\tchecked    13,040,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 6:79,068,557\n",
      "INFO\t2017-01-12 16:03:26\tGenotypeConcordance\tchecked    13,050,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 6:79,779,767\n",
      "INFO\t2017-01-12 16:03:27\tGenotypeConcordance\tchecked    13,060,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 6:80,635,130\n",
      "INFO\t2017-01-12 16:03:27\tGenotypeConcordance\tchecked    13,070,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 6:81,409,273\n",
      "INFO\t2017-01-12 16:03:27\tGenotypeConcordance\tchecked    13,080,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 6:82,095,610\n",
      "INFO\t2017-01-12 16:03:27\tGenotypeConcordance\tchecked    13,090,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 6:82,957,593\n",
      "INFO\t2017-01-12 16:03:28\tGenotypeConcordance\tchecked    13,100,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 6:83,896,510\n",
      "INFO\t2017-01-12 16:03:28\tGenotypeConcordance\tchecked    13,110,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 6:84,871,065\n",
      "INFO\t2017-01-12 16:03:28\tGenotypeConcordance\tchecked    13,120,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 6:85,734,809\n",
      "INFO\t2017-01-12 16:03:29\tGenotypeConcordance\tchecked    13,130,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 6:86,585,114\n",
      "INFO\t2017-01-12 16:03:29\tGenotypeConcordance\tchecked    13,140,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 6:87,317,765\n",
      "INFO\t2017-01-12 16:03:29\tGenotypeConcordance\tchecked    13,150,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 6:88,120,947\n",
      "INFO\t2017-01-12 16:03:29\tGenotypeConcordance\tchecked    13,160,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 6:89,002,133\n",
      "INFO\t2017-01-12 16:03:29\tGenotypeConcordance\tchecked    13,170,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 6:89,839,906\n",
      "INFO\t2017-01-12 16:03:30\tGenotypeConcordance\tchecked    13,180,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 6:90,755,352\n",
      "INFO\t2017-01-12 16:03:30\tGenotypeConcordance\tchecked    13,190,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 6:91,720,488\n",
      "INFO\t2017-01-12 16:03:30\tGenotypeConcordance\tchecked    13,200,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 6:92,584,877\n",
      "INFO\t2017-01-12 16:03:30\tGenotypeConcordance\tchecked    13,210,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 6:93,440,311\n",
      "INFO\t2017-01-12 16:03:31\tGenotypeConcordance\tchecked    13,220,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 6:94,258,257\n",
      "INFO\t2017-01-12 16:03:31\tGenotypeConcordance\tchecked    13,230,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 6:95,081,286\n",
      "INFO\t2017-01-12 16:03:31\tGenotypeConcordance\tchecked    13,240,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 6:96,019,311\n",
      "INFO\t2017-01-12 16:03:31\tGenotypeConcordance\tchecked    13,250,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 6:96,896,078\n",
      "INFO\t2017-01-12 16:03:32\tGenotypeConcordance\tchecked    13,260,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 6:97,869,464\n",
      "INFO\t2017-01-12 16:03:32\tGenotypeConcordance\tchecked    13,270,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 6:98,981,180\n",
      "INFO\t2017-01-12 16:03:32\tGenotypeConcordance\tchecked    13,280,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 6:99,984,803\n",
      "INFO\t2017-01-12 16:03:32\tGenotypeConcordance\tchecked    13,290,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 6:100,886,666\n",
      "INFO\t2017-01-12 16:03:33\tGenotypeConcordance\tchecked    13,300,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 6:101,813,169\n",
      "INFO\t2017-01-12 16:03:33\tGenotypeConcordance\tchecked    13,310,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 6:102,654,831\n",
      "INFO\t2017-01-12 16:03:33\tGenotypeConcordance\tchecked    13,320,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 6:103,468,653\n",
      "INFO\t2017-01-12 16:03:33\tGenotypeConcordance\tchecked    13,330,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 6:104,242,548\n",
      "INFO\t2017-01-12 16:03:34\tGenotypeConcordance\tchecked    13,340,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 6:105,116,200\n",
      "INFO\t2017-01-12 16:03:34\tGenotypeConcordance\tchecked    13,350,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 6:106,070,640\n",
      "INFO\t2017-01-12 16:03:34\tGenotypeConcordance\tchecked    13,360,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 6:106,972,698\n",
      "INFO\t2017-01-12 16:03:34\tGenotypeConcordance\tchecked    13,370,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 6:107,786,703\n",
      "INFO\t2017-01-12 16:03:35\tGenotypeConcordance\tchecked    13,380,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 6:108,721,321\n",
      "INFO\t2017-01-12 16:03:35\tGenotypeConcordance\tchecked    13,390,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 6:109,633,189\n",
      "INFO\t2017-01-12 16:03:35\tGenotypeConcordance\tchecked    13,400,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 6:110,571,347\n",
      "INFO\t2017-01-12 16:03:36\tGenotypeConcordance\tchecked    13,410,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:111,464,713\n",
      "INFO\t2017-01-12 16:03:36\tGenotypeConcordance\tchecked    13,420,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:112,501,468\n",
      "INFO\t2017-01-12 16:03:36\tGenotypeConcordance\tchecked    13,430,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:113,456,066\n",
      "INFO\t2017-01-12 16:03:36\tGenotypeConcordance\tchecked    13,440,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 6:114,326,693\n",
      "INFO\t2017-01-12 16:03:37\tGenotypeConcordance\tchecked    13,450,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:115,295,446\n",
      "INFO\t2017-01-12 16:03:37\tGenotypeConcordance\tchecked    13,460,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:116,100,131\n",
      "INFO\t2017-01-12 16:03:37\tGenotypeConcordance\tchecked    13,470,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:117,081,206\n",
      "INFO\t2017-01-12 16:03:37\tGenotypeConcordance\tchecked    13,480,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 6:118,004,817\n",
      "INFO\t2017-01-12 16:03:38\tGenotypeConcordance\tchecked    13,490,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:118,942,300\n",
      "INFO\t2017-01-12 16:03:38\tGenotypeConcordance\tchecked    13,500,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:119,829,445\n",
      "INFO\t2017-01-12 16:03:38\tGenotypeConcordance\tchecked    13,510,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:120,701,960\n",
      "INFO\t2017-01-12 16:03:38\tGenotypeConcordance\tchecked    13,520,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 6:121,490,824\n",
      "INFO\t2017-01-12 16:03:39\tGenotypeConcordance\tchecked    13,530,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:122,349,231\n",
      "INFO\t2017-01-12 16:03:39\tGenotypeConcordance\tchecked    13,540,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:123,265,483\n",
      "INFO\t2017-01-12 16:03:39\tGenotypeConcordance\tchecked    13,550,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:124,115,011\n",
      "INFO\t2017-01-12 16:03:39\tGenotypeConcordance\tchecked    13,560,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 6:124,972,844\n",
      "INFO\t2017-01-12 16:03:40\tGenotypeConcordance\tchecked    13,570,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:125,885,633\n",
      "INFO\t2017-01-12 16:03:40\tGenotypeConcordance\tchecked    13,580,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:127,043,239\n",
      "INFO\t2017-01-12 16:03:40\tGenotypeConcordance\tchecked    13,590,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:128,081,304\n",
      "INFO\t2017-01-12 16:03:40\tGenotypeConcordance\tchecked    13,600,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 6:129,067,102\n",
      "INFO\t2017-01-12 16:03:41\tGenotypeConcordance\tchecked    13,610,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:129,947,080\n",
      "INFO\t2017-01-12 16:03:41\tGenotypeConcordance\tchecked    13,620,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:130,779,148\n",
      "INFO\t2017-01-12 16:03:41\tGenotypeConcordance\tchecked    13,630,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:131,625,710\n",
      "INFO\t2017-01-12 16:03:41\tGenotypeConcordance\tchecked    13,640,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 6:132,485,007\n",
      "INFO\t2017-01-12 16:03:42\tGenotypeConcordance\tchecked    13,650,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:133,256,704\n",
      "INFO\t2017-01-12 16:03:42\tGenotypeConcordance\tchecked    13,660,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:134,153,208\n",
      "INFO\t2017-01-12 16:03:42\tGenotypeConcordance\tchecked    13,670,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:135,013,843\n",
      "INFO\t2017-01-12 16:03:42\tGenotypeConcordance\tchecked    13,680,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 6:135,935,894\n",
      "INFO\t2017-01-12 16:03:43\tGenotypeConcordance\tchecked    13,690,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:136,923,799\n",
      "INFO\t2017-01-12 16:03:43\tGenotypeConcordance\tchecked    13,700,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:137,801,790\n",
      "INFO\t2017-01-12 16:03:43\tGenotypeConcordance\tchecked    13,710,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:138,616,627\n",
      "INFO\t2017-01-12 16:03:43\tGenotypeConcordance\tchecked    13,720,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 6:139,414,505\n",
      "INFO\t2017-01-12 16:03:44\tGenotypeConcordance\tchecked    13,730,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:140,281,071\n",
      "INFO\t2017-01-12 16:03:44\tGenotypeConcordance\tchecked    13,740,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:141,397,909\n",
      "INFO\t2017-01-12 16:03:44\tGenotypeConcordance\tchecked    13,750,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:142,274,618\n",
      "INFO\t2017-01-12 16:03:44\tGenotypeConcordance\tchecked    13,760,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 6:143,170,566\n",
      "INFO\t2017-01-12 16:03:45\tGenotypeConcordance\tchecked    13,770,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:144,101,343\n",
      "INFO\t2017-01-12 16:03:45\tGenotypeConcordance\tchecked    13,780,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:145,007,723\n",
      "INFO\t2017-01-12 16:03:45\tGenotypeConcordance\tchecked    13,790,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:145,868,680\n",
      "INFO\t2017-01-12 16:03:45\tGenotypeConcordance\tchecked    13,800,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 6:146,881,529\n",
      "INFO\t2017-01-12 16:03:45\tGenotypeConcordance\tchecked    13,810,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:147,852,871\n",
      "INFO\t2017-01-12 16:03:46\tGenotypeConcordance\tchecked    13,820,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:148,620,015\n",
      "INFO\t2017-01-12 16:03:46\tGenotypeConcordance\tchecked    13,830,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:149,407,904\n",
      "INFO\t2017-01-12 16:03:46\tGenotypeConcordance\tchecked    13,840,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 6:150,269,249\n",
      "INFO\t2017-01-12 16:03:46\tGenotypeConcordance\tchecked    13,850,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:150,949,223\n",
      "INFO\t2017-01-12 16:03:47\tGenotypeConcordance\tchecked    13,860,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:151,665,650\n",
      "INFO\t2017-01-12 16:03:47\tGenotypeConcordance\tchecked    13,870,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:152,510,429\n",
      "INFO\t2017-01-12 16:03:47\tGenotypeConcordance\tchecked    13,880,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 6:153,351,923\n",
      "INFO\t2017-01-12 16:03:48\tGenotypeConcordance\tchecked    13,890,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:154,108,443\n",
      "INFO\t2017-01-12 16:03:48\tGenotypeConcordance\tchecked    13,900,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:154,853,986\n",
      "INFO\t2017-01-12 16:03:48\tGenotypeConcordance\tchecked    13,910,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:155,633,969\n",
      "INFO\t2017-01-12 16:03:48\tGenotypeConcordance\tchecked    13,920,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 6:156,463,383\n",
      "INFO\t2017-01-12 16:03:49\tGenotypeConcordance\tchecked    13,930,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:157,314,040\n",
      "INFO\t2017-01-12 16:03:49\tGenotypeConcordance\tchecked    13,940,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:158,239,938\n",
      "INFO\t2017-01-12 16:03:49\tGenotypeConcordance\tchecked    13,950,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:159,029,041\n",
      "INFO\t2017-01-12 16:03:49\tGenotypeConcordance\tchecked    13,960,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 6:159,805,647\n",
      "INFO\t2017-01-12 16:03:50\tGenotypeConcordance\tchecked    13,970,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:160,579,368\n",
      "INFO\t2017-01-12 16:03:50\tGenotypeConcordance\tchecked    13,980,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:161,354,194\n",
      "INFO\t2017-01-12 16:03:50\tGenotypeConcordance\tchecked    13,990,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:162,128,543\n",
      "INFO\t2017-01-12 16:03:50\tGenotypeConcordance\tchecked    14,000,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 6:162,832,167\n",
      "INFO\t2017-01-12 16:03:51\tGenotypeConcordance\tchecked    14,010,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:163,627,243\n",
      "INFO\t2017-01-12 16:03:51\tGenotypeConcordance\tchecked    14,020,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:164,409,147\n",
      "INFO\t2017-01-12 16:03:51\tGenotypeConcordance\tchecked    14,030,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:165,190,113\n",
      "INFO\t2017-01-12 16:03:51\tGenotypeConcordance\tchecked    14,040,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 6:165,944,280\n",
      "INFO\t2017-01-12 16:03:52\tGenotypeConcordance\tchecked    14,050,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:166,693,445\n",
      "INFO\t2017-01-12 16:03:52\tGenotypeConcordance\tchecked    14,060,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:167,378,619\n",
      "INFO\t2017-01-12 16:03:52\tGenotypeConcordance\tchecked    14,070,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:168,086,374\n",
      "INFO\t2017-01-12 16:03:52\tGenotypeConcordance\tchecked    14,080,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 6:168,746,649\n",
      "INFO\t2017-01-12 16:03:53\tGenotypeConcordance\tchecked    14,090,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:169,354,072\n",
      "INFO\t2017-01-12 16:03:53\tGenotypeConcordance\tchecked    14,100,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:170,046,595\n",
      "INFO\t2017-01-12 16:03:53\tGenotypeConcordance\tchecked    14,110,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 6:170,767,224\n",
      "INFO\t2017-01-12 16:03:53\tGenotypeConcordance\tchecked    14,120,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 7:416,852\n",
      "INFO\t2017-01-12 16:03:54\tGenotypeConcordance\tchecked    14,130,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 7:960,050\n",
      "INFO\t2017-01-12 16:03:54\tGenotypeConcordance\tchecked    14,140,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 7:1,514,209\n",
      "INFO\t2017-01-12 16:03:54\tGenotypeConcordance\tchecked    14,150,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 7:2,120,371\n",
      "INFO\t2017-01-12 16:03:54\tGenotypeConcordance\tchecked    14,160,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 7:2,745,489\n",
      "INFO\t2017-01-12 16:03:55\tGenotypeConcordance\tchecked    14,170,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 7:3,292,730\n",
      "INFO\t2017-01-12 16:03:55\tGenotypeConcordance\tchecked    14,180,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 7:3,856,970\n",
      "INFO\t2017-01-12 16:03:55\tGenotypeConcordance\tchecked    14,190,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 7:4,419,504\n",
      "INFO\t2017-01-12 16:03:55\tGenotypeConcordance\tchecked    14,200,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 7:4,986,942\n",
      "INFO\t2017-01-12 16:03:56\tGenotypeConcordance\tchecked    14,210,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 7:5,584,173\n",
      "INFO\t2017-01-12 16:03:56\tGenotypeConcordance\tchecked    14,220,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 7:6,254,993\n",
      "INFO\t2017-01-12 16:03:56\tGenotypeConcordance\tchecked    14,230,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 7:6,924,262\n",
      "INFO\t2017-01-12 16:03:56\tGenotypeConcordance\tchecked    14,240,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 7:7,442,180\n",
      "INFO\t2017-01-12 16:03:57\tGenotypeConcordance\tchecked    14,250,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 7:8,157,016\n",
      "INFO\t2017-01-12 16:03:57\tGenotypeConcordance\tchecked    14,260,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 7:8,827,383\n",
      "INFO\t2017-01-12 16:03:57\tGenotypeConcordance\tchecked    14,270,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 7:9,435,883\n",
      "INFO\t2017-01-12 16:03:57\tGenotypeConcordance\tchecked    14,280,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 7:10,073,714\n",
      "INFO\t2017-01-12 16:03:58\tGenotypeConcordance\tchecked    14,290,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 7:10,759,632\n",
      "INFO\t2017-01-12 16:03:58\tGenotypeConcordance\tchecked    14,300,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 7:11,517,514\n",
      "INFO\t2017-01-12 16:03:58\tGenotypeConcordance\tchecked    14,310,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 7:12,190,315\n",
      "INFO\t2017-01-12 16:03:58\tGenotypeConcordance\tchecked    14,320,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 7:12,799,881\n",
      "INFO\t2017-01-12 16:03:59\tGenotypeConcordance\tchecked    14,330,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 7:13,434,529\n",
      "INFO\t2017-01-12 16:03:59\tGenotypeConcordance\tchecked    14,340,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 7:14,125,432\n",
      "INFO\t2017-01-12 16:03:59\tGenotypeConcordance\tchecked    14,350,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 7:14,847,293\n",
      "INFO\t2017-01-12 16:03:59\tGenotypeConcordance\tchecked    14,360,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 7:15,501,400\n",
      "INFO\t2017-01-12 16:04:00\tGenotypeConcordance\tchecked    14,370,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 7:16,237,550\n",
      "INFO\t2017-01-12 16:04:00\tGenotypeConcordance\tchecked    14,380,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 7:16,956,718\n",
      "INFO\t2017-01-12 16:04:00\tGenotypeConcordance\tchecked    14,390,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 7:17,732,291\n",
      "INFO\t2017-01-12 16:04:00\tGenotypeConcordance\tchecked    14,400,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 7:18,627,588\n",
      "INFO\t2017-01-12 16:04:01\tGenotypeConcordance\tchecked    14,410,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 7:19,379,028\n",
      "INFO\t2017-01-12 16:04:01\tGenotypeConcordance\tchecked    14,420,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 7:20,038,602\n",
      "INFO\t2017-01-12 16:04:01\tGenotypeConcordance\tchecked    14,430,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 7:20,777,031\n",
      "INFO\t2017-01-12 16:04:01\tGenotypeConcordance\tchecked    14,440,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 7:21,582,964\n",
      "INFO\t2017-01-12 16:04:02\tGenotypeConcordance\tchecked    14,450,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 7:22,259,212\n",
      "INFO\t2017-01-12 16:04:02\tGenotypeConcordance\tchecked    14,460,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 7:23,070,387\n",
      "INFO\t2017-01-12 16:04:02\tGenotypeConcordance\tchecked    14,470,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 7:23,915,676\n",
      "INFO\t2017-01-12 16:04:02\tGenotypeConcordance\tchecked    14,480,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 7:24,700,907\n",
      "INFO\t2017-01-12 16:04:03\tGenotypeConcordance\tchecked    14,490,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 7:25,483,161\n",
      "INFO\t2017-01-12 16:04:03\tGenotypeConcordance\tchecked    14,500,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 7:26,381,207\n",
      "INFO\t2017-01-12 16:04:03\tGenotypeConcordance\tchecked    14,510,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 7:27,325,930\n",
      "INFO\t2017-01-12 16:04:03\tGenotypeConcordance\tchecked    14,520,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 7:28,190,727\n",
      "INFO\t2017-01-12 16:04:04\tGenotypeConcordance\tchecked    14,530,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 7:29,024,764\n",
      "INFO\t2017-01-12 16:04:04\tGenotypeConcordance\tchecked    14,540,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 7:29,837,093\n",
      "INFO\t2017-01-12 16:04:04\tGenotypeConcordance\tchecked    14,550,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 7:30,802,303\n",
      "INFO\t2017-01-12 16:04:04\tGenotypeConcordance\tchecked    14,560,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 7:31,654,462\n",
      "INFO\t2017-01-12 16:04:05\tGenotypeConcordance\tchecked    14,570,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 7:32,429,979\n",
      "INFO\t2017-01-12 16:04:05\tGenotypeConcordance\tchecked    14,580,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 7:33,219,952\n",
      "INFO\t2017-01-12 16:04:05\tGenotypeConcordance\tchecked    14,590,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 7:34,117,565\n",
      "INFO\t2017-01-12 16:04:05\tGenotypeConcordance\tchecked    14,600,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 7:34,992,376\n",
      "INFO\t2017-01-12 16:04:06\tGenotypeConcordance\tchecked    14,610,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 7:35,913,034\n",
      "INFO\t2017-01-12 16:04:06\tGenotypeConcordance\tchecked    14,620,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 7:36,742,034\n",
      "INFO\t2017-01-12 16:04:06\tGenotypeConcordance\tchecked    14,630,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 7:37,602,360\n",
      "INFO\t2017-01-12 16:04:06\tGenotypeConcordance\tchecked    14,640,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 7:38,385,712\n",
      "INFO\t2017-01-12 16:04:07\tGenotypeConcordance\tchecked    14,650,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 7:39,249,358\n",
      "INFO\t2017-01-12 16:04:07\tGenotypeConcordance\tchecked    14,660,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 7:40,150,910\n",
      "INFO\t2017-01-12 16:04:07\tGenotypeConcordance\tchecked    14,670,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 7:41,139,004\n",
      "INFO\t2017-01-12 16:04:07\tGenotypeConcordance\tchecked    14,680,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 7:42,001,171\n",
      "INFO\t2017-01-12 16:04:08\tGenotypeConcordance\tchecked    14,690,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 7:42,843,325\n",
      "INFO\t2017-01-12 16:04:08\tGenotypeConcordance\tchecked    14,700,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 7:43,703,622\n",
      "INFO\t2017-01-12 16:04:08\tGenotypeConcordance\tchecked    14,710,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 7:44,664,806\n",
      "INFO\t2017-01-12 16:04:08\tGenotypeConcordance\tchecked    14,720,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 7:45,516,136\n",
      "INFO\t2017-01-12 16:04:09\tGenotypeConcordance\tchecked    14,730,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 7:46,323,938\n",
      "INFO\t2017-01-12 16:04:09\tGenotypeConcordance\tchecked    14,740,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 7:47,144,689\n",
      "INFO\t2017-01-12 16:04:09\tGenotypeConcordance\tchecked    14,750,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 7:47,915,917\n",
      "INFO\t2017-01-12 16:04:09\tGenotypeConcordance\tchecked    14,760,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 7:48,772,811\n",
      "INFO\t2017-01-12 16:04:10\tGenotypeConcordance\tchecked    14,770,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 7:49,547,057\n",
      "INFO\t2017-01-12 16:04:10\tGenotypeConcordance\tchecked    14,780,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 7:50,478,573\n",
      "INFO\t2017-01-12 16:04:10\tGenotypeConcordance\tchecked    14,790,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 7:51,332,935\n",
      "INFO\t2017-01-12 16:04:10\tGenotypeConcordance\tchecked    14,800,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 7:52,054,153\n",
      "INFO\t2017-01-12 16:04:11\tGenotypeConcordance\tchecked    14,810,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 7:52,737,397\n",
      "INFO\t2017-01-12 16:04:11\tGenotypeConcordance\tchecked    14,820,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 7:53,356,363\n",
      "INFO\t2017-01-12 16:04:11\tGenotypeConcordance\tchecked    14,830,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 7:54,109,581\n",
      "INFO\t2017-01-12 16:04:11\tGenotypeConcordance\tchecked    14,840,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 7:54,792,121\n",
      "INFO\t2017-01-12 16:04:12\tGenotypeConcordance\tchecked    14,850,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 7:55,595,270\n",
      "INFO\t2017-01-12 16:04:12\tGenotypeConcordance\tchecked    14,860,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 7:56,389,623\n",
      "INFO\t2017-01-12 16:04:12\tGenotypeConcordance\tchecked    14,870,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 7:57,064,567\n",
      "INFO\t2017-01-12 16:04:12\tGenotypeConcordance\tchecked    14,880,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 7:57,640,840\n",
      "INFO\t2017-01-12 16:04:13\tGenotypeConcordance\tchecked    14,890,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 7:57,927,767\n",
      "INFO\t2017-01-12 16:04:13\tGenotypeConcordance\tchecked    14,900,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 7:61,554,595\n",
      "INFO\t2017-01-12 16:04:13\tGenotypeConcordance\tchecked    14,910,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 7:61,823,904\n",
      "INFO\t2017-01-12 16:04:13\tGenotypeConcordance\tchecked    14,920,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 7:62,189,119\n",
      "INFO\t2017-01-12 16:04:14\tGenotypeConcordance\tchecked    14,930,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 7:62,828,260\n",
      "INFO\t2017-01-12 16:04:14\tGenotypeConcordance\tchecked    14,940,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 7:63,624,322\n",
      "INFO\t2017-01-12 16:04:14\tGenotypeConcordance\tchecked    14,950,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 7:64,427,308\n",
      "INFO\t2017-01-12 16:04:15\tGenotypeConcordance\tchecked    14,960,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 7:65,192,146\n",
      "INFO\t2017-01-12 16:04:15\tGenotypeConcordance\tchecked    14,970,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 7:66,033,033\n",
      "INFO\t2017-01-12 16:04:15\tGenotypeConcordance\tchecked    14,980,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 7:66,825,558\n",
      "INFO\t2017-01-12 16:04:15\tGenotypeConcordance\tchecked    14,990,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 7:67,532,341\n",
      "INFO\t2017-01-12 16:04:16\tGenotypeConcordance\tchecked    15,000,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 7:68,156,564\n",
      "INFO\t2017-01-12 16:04:16\tGenotypeConcordance\tchecked    15,010,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 7:68,889,003\n",
      "INFO\t2017-01-12 16:04:16\tGenotypeConcordance\tchecked    15,020,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 7:70,083,260\n",
      "INFO\t2017-01-12 16:04:16\tGenotypeConcordance\tchecked    15,030,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 7:70,842,337\n",
      "INFO\t2017-01-12 16:04:17\tGenotypeConcordance\tchecked    15,040,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 7:71,592,894\n",
      "INFO\t2017-01-12 16:04:17\tGenotypeConcordance\tchecked    15,050,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 7:72,403,209\n",
      "INFO\t2017-01-12 16:04:17\tGenotypeConcordance\tchecked    15,060,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 7:73,641,497\n",
      "INFO\t2017-01-12 16:04:17\tGenotypeConcordance\tchecked    15,070,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 7:75,243,991\n",
      "INFO\t2017-01-12 16:04:18\tGenotypeConcordance\tchecked    15,080,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 7:76,024,084\n",
      "INFO\t2017-01-12 16:04:18\tGenotypeConcordance\tchecked    15,090,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 7:76,897,826\n",
      "INFO\t2017-01-12 16:04:18\tGenotypeConcordance\tchecked    15,100,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 7:77,825,224\n",
      "INFO\t2017-01-12 16:04:18\tGenotypeConcordance\tchecked    15,110,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 7:78,599,311\n",
      "INFO\t2017-01-12 16:04:19\tGenotypeConcordance\tchecked    15,120,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 7:79,434,835\n",
      "INFO\t2017-01-12 16:04:19\tGenotypeConcordance\tchecked    15,130,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 7:80,311,647\n",
      "INFO\t2017-01-12 16:04:19\tGenotypeConcordance\tchecked    15,140,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 7:81,081,130\n",
      "INFO\t2017-01-12 16:04:19\tGenotypeConcordance\tchecked    15,150,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 7:81,985,187\n",
      "INFO\t2017-01-12 16:04:20\tGenotypeConcordance\tchecked    15,160,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 7:82,869,581\n",
      "INFO\t2017-01-12 16:04:20\tGenotypeConcordance\tchecked    15,170,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 7:83,679,299\n",
      "INFO\t2017-01-12 16:04:20\tGenotypeConcordance\tchecked    15,180,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 7:84,560,389\n",
      "INFO\t2017-01-12 16:04:20\tGenotypeConcordance\tchecked    15,190,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 7:85,456,390\n",
      "INFO\t2017-01-12 16:04:21\tGenotypeConcordance\tchecked    15,200,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 7:86,335,723\n",
      "INFO\t2017-01-12 16:04:21\tGenotypeConcordance\tchecked    15,210,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 7:87,331,322\n",
      "INFO\t2017-01-12 16:04:21\tGenotypeConcordance\tchecked    15,220,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 7:88,354,527\n",
      "INFO\t2017-01-12 16:04:21\tGenotypeConcordance\tchecked    15,230,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 7:89,166,525\n",
      "INFO\t2017-01-12 16:04:22\tGenotypeConcordance\tchecked    15,240,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 7:90,019,464\n",
      "INFO\t2017-01-12 16:04:22\tGenotypeConcordance\tchecked    15,250,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 7:90,969,510\n",
      "INFO\t2017-01-12 16:04:22\tGenotypeConcordance\tchecked    15,260,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 7:92,014,645\n",
      "INFO\t2017-01-12 16:04:22\tGenotypeConcordance\tchecked    15,270,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 7:93,133,758\n",
      "INFO\t2017-01-12 16:04:23\tGenotypeConcordance\tchecked    15,280,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 7:94,119,404\n",
      "INFO\t2017-01-12 16:04:23\tGenotypeConcordance\tchecked    15,290,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 7:95,175,661\n",
      "INFO\t2017-01-12 16:04:23\tGenotypeConcordance\tchecked    15,300,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 7:96,145,105\n",
      "INFO\t2017-01-12 16:04:23\tGenotypeConcordance\tchecked    15,310,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 7:97,060,570\n",
      "INFO\t2017-01-12 16:04:24\tGenotypeConcordance\tchecked    15,320,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 7:97,832,154\n",
      "INFO\t2017-01-12 16:04:24\tGenotypeConcordance\tchecked    15,330,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 7:98,660,529\n",
      "INFO\t2017-01-12 16:04:24\tGenotypeConcordance\tchecked    15,340,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 7:99,705,796\n",
      "INFO\t2017-01-12 16:04:24\tGenotypeConcordance\tchecked    15,350,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 7:100,635,956\n",
      "INFO\t2017-01-12 16:04:25\tGenotypeConcordance\tchecked    15,360,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 7:101,335,614\n",
      "INFO\t2017-01-12 16:04:25\tGenotypeConcordance\tchecked    15,370,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 7:102,296,892\n",
      "INFO\t2017-01-12 16:04:25\tGenotypeConcordance\tchecked    15,380,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 7:103,245,417\n",
      "INFO\t2017-01-12 16:04:25\tGenotypeConcordance\tchecked    15,390,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 7:103,976,997\n",
      "INFO\t2017-01-12 16:04:26\tGenotypeConcordance\tchecked    15,400,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 7:104,845,980\n",
      "INFO\t2017-01-12 16:04:26\tGenotypeConcordance\tchecked    15,410,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 7:105,730,603\n",
      "INFO\t2017-01-12 16:04:26\tGenotypeConcordance\tchecked    15,420,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 7:106,629,378\n",
      "INFO\t2017-01-12 16:04:26\tGenotypeConcordance\tchecked    15,430,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 7:107,551,530\n",
      "INFO\t2017-01-12 16:04:27\tGenotypeConcordance\tchecked    15,440,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 7:108,456,043\n",
      "INFO\t2017-01-12 16:04:27\tGenotypeConcordance\tchecked    15,450,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 7:109,264,452\n",
      "INFO\t2017-01-12 16:04:27\tGenotypeConcordance\tchecked    15,460,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 7:110,063,445\n",
      "INFO\t2017-01-12 16:04:27\tGenotypeConcordance\tchecked    15,470,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 7:111,002,121\n",
      "INFO\t2017-01-12 16:04:28\tGenotypeConcordance\tchecked    15,480,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 7:111,945,260\n",
      "INFO\t2017-01-12 16:04:28\tGenotypeConcordance\tchecked    15,490,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 7:112,942,522\n",
      "INFO\t2017-01-12 16:04:28\tGenotypeConcordance\tchecked    15,500,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 7:113,946,693\n",
      "INFO\t2017-01-12 16:04:28\tGenotypeConcordance\tchecked    15,510,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 7:114,991,272\n",
      "INFO\t2017-01-12 16:04:29\tGenotypeConcordance\tchecked    15,520,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 7:115,993,469\n",
      "INFO\t2017-01-12 16:04:29\tGenotypeConcordance\tchecked    15,530,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 7:117,162,954\n",
      "INFO\t2017-01-12 16:04:29\tGenotypeConcordance\tchecked    15,540,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 7:118,225,978\n",
      "INFO\t2017-01-12 16:04:29\tGenotypeConcordance\tchecked    15,550,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 7:119,161,984\n",
      "INFO\t2017-01-12 16:04:30\tGenotypeConcordance\tchecked    15,560,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 7:120,124,719\n",
      "INFO\t2017-01-12 16:04:30\tGenotypeConcordance\tchecked    15,570,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 7:121,138,156\n",
      "INFO\t2017-01-12 16:04:30\tGenotypeConcordance\tchecked    15,580,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 7:122,155,821\n",
      "INFO\t2017-01-12 16:04:30\tGenotypeConcordance\tchecked    15,590,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 7:123,089,364\n",
      "INFO\t2017-01-12 16:04:31\tGenotypeConcordance\tchecked    15,600,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 7:124,085,851\n",
      "INFO\t2017-01-12 16:04:31\tGenotypeConcordance\tchecked    15,610,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 7:124,959,006\n",
      "INFO\t2017-01-12 16:04:31\tGenotypeConcordance\tchecked    15,620,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 7:125,705,705\n",
      "INFO\t2017-01-12 16:04:31\tGenotypeConcordance\tchecked    15,630,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 7:126,592,772\n",
      "INFO\t2017-01-12 16:04:32\tGenotypeConcordance\tchecked    15,640,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 7:127,644,243\n",
      "INFO\t2017-01-12 16:04:32\tGenotypeConcordance\tchecked    15,650,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 7:128,514,177\n",
      "INFO\t2017-01-12 16:04:32\tGenotypeConcordance\tchecked    15,660,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 7:129,452,834\n",
      "INFO\t2017-01-12 16:04:32\tGenotypeConcordance\tchecked    15,670,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 7:130,495,991\n",
      "INFO\t2017-01-12 16:04:33\tGenotypeConcordance\tchecked    15,680,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 7:131,385,570\n",
      "INFO\t2017-01-12 16:04:33\tGenotypeConcordance\tchecked    15,690,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 7:132,186,073\n",
      "INFO\t2017-01-12 16:04:33\tGenotypeConcordance\tchecked    15,700,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 7:133,237,686\n",
      "INFO\t2017-01-12 16:04:33\tGenotypeConcordance\tchecked    15,710,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 7:134,202,002\n",
      "INFO\t2017-01-12 16:04:34\tGenotypeConcordance\tchecked    15,720,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 7:135,123,402\n",
      "INFO\t2017-01-12 16:04:34\tGenotypeConcordance\tchecked    15,730,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 7:135,979,376\n",
      "INFO\t2017-01-12 16:04:34\tGenotypeConcordance\tchecked    15,740,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 7:136,861,576\n",
      "INFO\t2017-01-12 16:04:34\tGenotypeConcordance\tchecked    15,750,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 7:137,783,240\n",
      "INFO\t2017-01-12 16:04:35\tGenotypeConcordance\tchecked    15,760,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:138,477,526\n",
      "INFO\t2017-01-12 16:04:35\tGenotypeConcordance\tchecked    15,770,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:139,321,855\n",
      "INFO\t2017-01-12 16:04:35\tGenotypeConcordance\tchecked    15,780,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:140,190,337\n",
      "INFO\t2017-01-12 16:04:35\tGenotypeConcordance\tchecked    15,790,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 7:141,169,727\n",
      "INFO\t2017-01-12 16:04:36\tGenotypeConcordance\tchecked    15,800,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:141,992,116\n",
      "INFO\t2017-01-12 16:04:36\tGenotypeConcordance\tchecked    15,810,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:142,474,879\n",
      "INFO\t2017-01-12 16:04:36\tGenotypeConcordance\tchecked    15,820,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:143,632,984\n",
      "INFO\t2017-01-12 16:04:36\tGenotypeConcordance\tchecked    15,830,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 7:144,622,552\n",
      "INFO\t2017-01-12 16:04:37\tGenotypeConcordance\tchecked    15,840,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:145,434,874\n",
      "INFO\t2017-01-12 16:04:37\tGenotypeConcordance\tchecked    15,850,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:146,201,148\n",
      "INFO\t2017-01-12 16:04:37\tGenotypeConcordance\tchecked    15,860,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:146,982,163\n",
      "INFO\t2017-01-12 16:04:37\tGenotypeConcordance\tchecked    15,870,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 7:147,645,125\n",
      "INFO\t2017-01-12 16:04:38\tGenotypeConcordance\tchecked    15,880,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:148,373,150\n",
      "INFO\t2017-01-12 16:04:38\tGenotypeConcordance\tchecked    15,890,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:149,183,459\n",
      "INFO\t2017-01-12 16:04:38\tGenotypeConcordance\tchecked    15,900,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:149,895,465\n",
      "INFO\t2017-01-12 16:04:38\tGenotypeConcordance\tchecked    15,910,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 7:150,602,428\n",
      "INFO\t2017-01-12 16:04:39\tGenotypeConcordance\tchecked    15,920,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:151,425,178\n",
      "INFO\t2017-01-12 16:04:39\tGenotypeConcordance\tchecked    15,930,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:152,113,680\n",
      "INFO\t2017-01-12 16:04:39\tGenotypeConcordance\tchecked    15,940,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:152,801,860\n",
      "INFO\t2017-01-12 16:04:39\tGenotypeConcordance\tchecked    15,950,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 7:153,503,092\n",
      "INFO\t2017-01-12 16:04:40\tGenotypeConcordance\tchecked    15,960,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:154,175,478\n",
      "INFO\t2017-01-12 16:04:40\tGenotypeConcordance\tchecked    15,970,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:154,951,511\n",
      "INFO\t2017-01-12 16:04:40\tGenotypeConcordance\tchecked    15,980,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:155,624,450\n",
      "INFO\t2017-01-12 16:04:40\tGenotypeConcordance\tchecked    15,990,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 7:156,286,725\n",
      "INFO\t2017-01-12 16:04:41\tGenotypeConcordance\tchecked    16,000,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:157,030,216\n",
      "INFO\t2017-01-12 16:04:41\tGenotypeConcordance\tchecked    16,010,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:157,624,989\n",
      "INFO\t2017-01-12 16:04:41\tGenotypeConcordance\tchecked    16,020,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:158,128,863\n",
      "INFO\t2017-01-12 16:04:41\tGenotypeConcordance\tchecked    16,030,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 7:158,765,311\n",
      "INFO\t2017-01-12 16:04:42\tGenotypeConcordance\tchecked    16,040,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 8:307,858\n",
      "INFO\t2017-01-12 16:04:42\tGenotypeConcordance\tchecked    16,050,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 8:855,881\n",
      "INFO\t2017-01-12 16:04:42\tGenotypeConcordance\tchecked    16,060,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 8:1,317,099\n",
      "INFO\t2017-01-12 16:04:42\tGenotypeConcordance\tchecked    16,070,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 8:1,817,602\n",
      "INFO\t2017-01-12 16:04:43\tGenotypeConcordance\tchecked    16,080,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 8:2,253,551\n",
      "INFO\t2017-01-12 16:04:43\tGenotypeConcordance\tchecked    16,090,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 8:2,747,624\n",
      "INFO\t2017-01-12 16:04:43\tGenotypeConcordance\tchecked    16,100,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 8:3,171,301\n",
      "INFO\t2017-01-12 16:04:43\tGenotypeConcordance\tchecked    16,110,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 8:3,511,936\n",
      "INFO\t2017-01-12 16:04:44\tGenotypeConcordance\tchecked    16,120,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 8:3,815,701\n",
      "INFO\t2017-01-12 16:04:44\tGenotypeConcordance\tchecked    16,130,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 8:4,109,451\n",
      "INFO\t2017-01-12 16:04:44\tGenotypeConcordance\tchecked    16,140,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 8:4,448,182\n",
      "INFO\t2017-01-12 16:04:44\tGenotypeConcordance\tchecked    16,150,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 8:4,824,267\n",
      "INFO\t2017-01-12 16:04:45\tGenotypeConcordance\tchecked    16,160,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 8:5,230,310\n",
      "INFO\t2017-01-12 16:04:45\tGenotypeConcordance\tchecked    16,170,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 8:5,629,968\n",
      "INFO\t2017-01-12 16:04:45\tGenotypeConcordance\tchecked    16,180,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 8:6,035,203\n",
      "INFO\t2017-01-12 16:04:45\tGenotypeConcordance\tchecked    16,190,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 8:6,450,224\n",
      "INFO\t2017-01-12 16:04:46\tGenotypeConcordance\tchecked    16,200,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 8:6,933,197\n",
      "INFO\t2017-01-12 16:04:46\tGenotypeConcordance\tchecked    16,210,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 8:7,915,309\n",
      "INFO\t2017-01-12 16:04:46\tGenotypeConcordance\tchecked    16,220,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 8:8,528,908\n",
      "INFO\t2017-01-12 16:04:46\tGenotypeConcordance\tchecked    16,230,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 8:9,103,724\n",
      "INFO\t2017-01-12 16:04:47\tGenotypeConcordance\tchecked    16,240,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 8:9,715,476\n",
      "INFO\t2017-01-12 16:04:47\tGenotypeConcordance\tchecked    16,250,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 8:10,349,672\n",
      "INFO\t2017-01-12 16:04:47\tGenotypeConcordance\tchecked    16,260,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 8:10,957,243\n",
      "INFO\t2017-01-12 16:04:47\tGenotypeConcordance\tchecked    16,270,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 8:11,529,409\n",
      "INFO\t2017-01-12 16:04:48\tGenotypeConcordance\tchecked    16,280,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 8:12,075,226\n",
      "INFO\t2017-01-12 16:04:48\tGenotypeConcordance\tchecked    16,290,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 8:12,675,621\n",
      "INFO\t2017-01-12 16:04:48\tGenotypeConcordance\tchecked    16,300,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 8:13,212,734\n",
      "INFO\t2017-01-12 16:04:48\tGenotypeConcordance\tchecked    16,310,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 8:13,687,093\n",
      "INFO\t2017-01-12 16:04:49\tGenotypeConcordance\tchecked    16,320,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 8:14,169,711\n",
      "INFO\t2017-01-12 16:04:49\tGenotypeConcordance\tchecked    16,330,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 8:14,669,987\n",
      "INFO\t2017-01-12 16:04:49\tGenotypeConcordance\tchecked    16,340,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 8:15,240,776\n",
      "INFO\t2017-01-12 16:04:49\tGenotypeConcordance\tchecked    16,350,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 8:15,745,075\n",
      "INFO\t2017-01-12 16:04:50\tGenotypeConcordance\tchecked    16,360,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 8:16,315,830\n",
      "INFO\t2017-01-12 16:04:50\tGenotypeConcordance\tchecked    16,370,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 8:16,852,283\n",
      "INFO\t2017-01-12 16:04:50\tGenotypeConcordance\tchecked    16,380,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 8:17,469,422\n",
      "INFO\t2017-01-12 16:04:50\tGenotypeConcordance\tchecked    16,390,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 8:18,004,224\n",
      "INFO\t2017-01-12 16:04:51\tGenotypeConcordance\tchecked    16,400,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 8:18,577,420\n",
      "INFO\t2017-01-12 16:04:51\tGenotypeConcordance\tchecked    16,410,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 8:19,170,003\n",
      "INFO\t2017-01-12 16:04:51\tGenotypeConcordance\tchecked    16,420,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 8:19,839,175\n",
      "INFO\t2017-01-12 16:04:52\tGenotypeConcordance\tchecked    16,430,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 8:20,533,362\n",
      "INFO\t2017-01-12 16:04:52\tGenotypeConcordance\tchecked    16,440,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 8:21,223,083\n",
      "INFO\t2017-01-12 16:04:52\tGenotypeConcordance\tchecked    16,450,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 8:22,029,129\n",
      "INFO\t2017-01-12 16:04:52\tGenotypeConcordance\tchecked    16,460,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 8:22,836,272\n",
      "INFO\t2017-01-12 16:04:53\tGenotypeConcordance\tchecked    16,470,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 8:23,540,354\n",
      "INFO\t2017-01-12 16:04:53\tGenotypeConcordance\tchecked    16,480,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 8:24,382,034\n",
      "INFO\t2017-01-12 16:04:53\tGenotypeConcordance\tchecked    16,490,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 8:25,187,933\n",
      "INFO\t2017-01-12 16:04:53\tGenotypeConcordance\tchecked    16,500,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 8:26,024,889\n",
      "INFO\t2017-01-12 16:04:54\tGenotypeConcordance\tchecked    16,510,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 8:26,816,678\n",
      "INFO\t2017-01-12 16:04:54\tGenotypeConcordance\tchecked    16,520,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 8:27,635,009\n",
      "INFO\t2017-01-12 16:04:54\tGenotypeConcordance\tchecked    16,530,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 8:28,461,311\n",
      "INFO\t2017-01-12 16:04:54\tGenotypeConcordance\tchecked    16,540,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 8:29,366,588\n",
      "INFO\t2017-01-12 16:04:55\tGenotypeConcordance\tchecked    16,550,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 8:30,207,421\n",
      "INFO\t2017-01-12 16:04:55\tGenotypeConcordance\tchecked    16,560,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 8:31,123,251\n",
      "INFO\t2017-01-12 16:04:55\tGenotypeConcordance\tchecked    16,570,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 8:31,992,258\n",
      "INFO\t2017-01-12 16:04:55\tGenotypeConcordance\tchecked    16,580,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 8:32,882,217\n",
      "INFO\t2017-01-12 16:04:56\tGenotypeConcordance\tchecked    16,590,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 8:33,780,637\n",
      "INFO\t2017-01-12 16:04:56\tGenotypeConcordance\tchecked    16,600,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 8:34,663,135\n",
      "INFO\t2017-01-12 16:04:56\tGenotypeConcordance\tchecked    16,610,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 8:35,648,562\n",
      "INFO\t2017-01-12 16:04:56\tGenotypeConcordance\tchecked    16,620,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 8:36,675,564\n",
      "INFO\t2017-01-12 16:04:56\tGenotypeConcordance\tchecked    16,630,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 8:37,647,063\n",
      "INFO\t2017-01-12 16:04:57\tGenotypeConcordance\tchecked    16,640,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 8:38,658,984\n",
      "INFO\t2017-01-12 16:04:57\tGenotypeConcordance\tchecked    16,650,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 8:39,627,891\n",
      "INFO\t2017-01-12 16:04:57\tGenotypeConcordance\tchecked    16,660,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 8:40,375,243\n",
      "INFO\t2017-01-12 16:04:57\tGenotypeConcordance\tchecked    16,670,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 8:41,149,507\n",
      "INFO\t2017-01-12 16:04:58\tGenotypeConcordance\tchecked    16,680,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 8:42,013,150\n",
      "INFO\t2017-01-12 16:04:58\tGenotypeConcordance\tchecked    16,690,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 8:43,006,010\n",
      "INFO\t2017-01-12 16:04:58\tGenotypeConcordance\tchecked    16,700,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 8:43,684,593\n",
      "INFO\t2017-01-12 16:04:58\tGenotypeConcordance\tchecked    16,710,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 8:47,287,215\n",
      "INFO\t2017-01-12 16:04:59\tGenotypeConcordance\tchecked    16,720,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 8:48,163,682\n",
      "INFO\t2017-01-12 16:04:59\tGenotypeConcordance\tchecked    16,730,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 8:49,357,151\n",
      "INFO\t2017-01-12 16:04:59\tGenotypeConcordance\tchecked    16,740,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 8:50,307,281\n",
      "INFO\t2017-01-12 16:04:59\tGenotypeConcordance\tchecked    16,750,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 8:51,141,137\n",
      "INFO\t2017-01-12 16:05:00\tGenotypeConcordance\tchecked    16,760,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 8:51,883,407\n",
      "INFO\t2017-01-12 16:05:00\tGenotypeConcordance\tchecked    16,770,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 8:52,626,791\n",
      "INFO\t2017-01-12 16:05:00\tGenotypeConcordance\tchecked    16,780,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 8:53,530,970\n",
      "INFO\t2017-01-12 16:05:00\tGenotypeConcordance\tchecked    16,790,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 8:54,364,075\n",
      "INFO\t2017-01-12 16:05:01\tGenotypeConcordance\tchecked    16,800,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 8:55,217,867\n",
      "INFO\t2017-01-12 16:05:01\tGenotypeConcordance\tchecked    16,810,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 8:56,085,891\n",
      "INFO\t2017-01-12 16:05:01\tGenotypeConcordance\tchecked    16,820,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 8:56,904,454\n",
      "INFO\t2017-01-12 16:05:01\tGenotypeConcordance\tchecked    16,830,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 8:57,723,177\n",
      "INFO\t2017-01-12 16:05:02\tGenotypeConcordance\tchecked    16,840,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 8:58,436,870\n",
      "INFO\t2017-01-12 16:05:02\tGenotypeConcordance\tchecked    16,850,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 8:59,289,763\n",
      "INFO\t2017-01-12 16:05:02\tGenotypeConcordance\tchecked    16,860,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 8:60,198,604\n",
      "INFO\t2017-01-12 16:05:02\tGenotypeConcordance\tchecked    16,870,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 8:61,122,265\n",
      "INFO\t2017-01-12 16:05:02\tGenotypeConcordance\tchecked    16,880,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 8:62,067,486\n",
      "INFO\t2017-01-12 16:05:03\tGenotypeConcordance\tchecked    16,890,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 8:62,924,308\n",
      "INFO\t2017-01-12 16:05:03\tGenotypeConcordance\tchecked    16,900,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 8:63,713,933\n",
      "INFO\t2017-01-12 16:05:03\tGenotypeConcordance\tchecked    16,910,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 8:64,609,056\n",
      "INFO\t2017-01-12 16:05:03\tGenotypeConcordance\tchecked    16,920,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 8:65,738,190\n",
      "INFO\t2017-01-12 16:05:04\tGenotypeConcordance\tchecked    16,930,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 8:66,669,635\n",
      "INFO\t2017-01-12 16:05:04\tGenotypeConcordance\tchecked    16,940,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 8:67,635,455\n",
      "INFO\t2017-01-12 16:05:04\tGenotypeConcordance\tchecked    16,950,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 8:68,673,649\n",
      "INFO\t2017-01-12 16:05:04\tGenotypeConcordance\tchecked    16,960,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 8:69,559,617\n",
      "INFO\t2017-01-12 16:05:05\tGenotypeConcordance\tchecked    16,970,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 8:70,375,937\n",
      "INFO\t2017-01-12 16:05:05\tGenotypeConcordance\tchecked    16,980,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 8:71,248,222\n",
      "INFO\t2017-01-12 16:05:05\tGenotypeConcordance\tchecked    16,990,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 8:72,204,352\n",
      "INFO\t2017-01-12 16:05:05\tGenotypeConcordance\tchecked    17,000,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 8:73,068,636\n",
      "INFO\t2017-01-12 16:05:06\tGenotypeConcordance\tchecked    17,010,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 8:73,929,721\n",
      "INFO\t2017-01-12 16:05:06\tGenotypeConcordance\tchecked    17,020,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 8:74,777,699\n",
      "INFO\t2017-01-12 16:05:06\tGenotypeConcordance\tchecked    17,030,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 8:75,663,682\n",
      "INFO\t2017-01-12 16:05:06\tGenotypeConcordance\tchecked    17,040,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 8:76,499,018\n",
      "INFO\t2017-01-12 16:05:07\tGenotypeConcordance\tchecked    17,050,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 8:77,432,409\n",
      "INFO\t2017-01-12 16:05:07\tGenotypeConcordance\tchecked    17,060,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 8:78,380,536\n",
      "INFO\t2017-01-12 16:05:07\tGenotypeConcordance\tchecked    17,070,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 8:79,277,609\n",
      "INFO\t2017-01-12 16:05:07\tGenotypeConcordance\tchecked    17,080,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 8:80,246,971\n",
      "INFO\t2017-01-12 16:05:08\tGenotypeConcordance\tchecked    17,090,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 8:81,172,159\n",
      "INFO\t2017-01-12 16:05:08\tGenotypeConcordance\tchecked    17,100,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 8:82,116,596\n",
      "INFO\t2017-01-12 16:05:08\tGenotypeConcordance\tchecked    17,110,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 8:82,996,542\n",
      "INFO\t2017-01-12 16:05:08\tGenotypeConcordance\tchecked    17,120,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 8:83,820,378\n",
      "INFO\t2017-01-12 16:05:09\tGenotypeConcordance\tchecked    17,130,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 8:84,599,247\n",
      "INFO\t2017-01-12 16:05:09\tGenotypeConcordance\tchecked    17,140,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 8:85,364,470\n",
      "INFO\t2017-01-12 16:05:09\tGenotypeConcordance\tchecked    17,150,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 8:86,343,937\n",
      "INFO\t2017-01-12 16:05:09\tGenotypeConcordance\tchecked    17,160,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 8:87,518,101\n",
      "INFO\t2017-01-12 16:05:10\tGenotypeConcordance\tchecked    17,170,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 8:88,328,054\n",
      "INFO\t2017-01-12 16:05:10\tGenotypeConcordance\tchecked    17,180,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 8:89,157,257\n",
      "INFO\t2017-01-12 16:05:10\tGenotypeConcordance\tchecked    17,190,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 8:90,035,634\n",
      "INFO\t2017-01-12 16:05:10\tGenotypeConcordance\tchecked    17,200,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 8:90,869,755\n",
      "INFO\t2017-01-12 16:05:11\tGenotypeConcordance\tchecked    17,210,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 8:91,788,105\n",
      "INFO\t2017-01-12 16:05:11\tGenotypeConcordance\tchecked    17,220,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 8:92,842,791\n",
      "INFO\t2017-01-12 16:05:11\tGenotypeConcordance\tchecked    17,230,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 8:93,935,665\n",
      "INFO\t2017-01-12 16:05:11\tGenotypeConcordance\tchecked    17,240,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 8:94,962,637\n",
      "INFO\t2017-01-12 16:05:12\tGenotypeConcordance\tchecked    17,250,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 8:95,818,155\n",
      "INFO\t2017-01-12 16:05:12\tGenotypeConcordance\tchecked    17,260,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 8:96,698,676\n",
      "INFO\t2017-01-12 16:05:12\tGenotypeConcordance\tchecked    17,270,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 8:97,563,575\n",
      "INFO\t2017-01-12 16:05:12\tGenotypeConcordance\tchecked    17,280,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 8:98,498,193\n",
      "INFO\t2017-01-12 16:05:13\tGenotypeConcordance\tchecked    17,290,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 8:99,296,821\n",
      "INFO\t2017-01-12 16:05:13\tGenotypeConcordance\tchecked    17,300,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 8:100,503,115\n",
      "INFO\t2017-01-12 16:05:13\tGenotypeConcordance\tchecked    17,310,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 8:101,577,545\n",
      "INFO\t2017-01-12 16:05:13\tGenotypeConcordance\tchecked    17,320,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 8:102,406,084\n",
      "INFO\t2017-01-12 16:05:14\tGenotypeConcordance\tchecked    17,330,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 8:103,258,448\n",
      "INFO\t2017-01-12 16:05:14\tGenotypeConcordance\tchecked    17,340,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 8:104,124,353\n",
      "INFO\t2017-01-12 16:05:14\tGenotypeConcordance\tchecked    17,350,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 8:105,018,863\n",
      "INFO\t2017-01-12 16:05:14\tGenotypeConcordance\tchecked    17,360,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 8:105,916,115\n",
      "INFO\t2017-01-12 16:05:15\tGenotypeConcordance\tchecked    17,370,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 8:106,883,102\n",
      "INFO\t2017-01-12 16:05:15\tGenotypeConcordance\tchecked    17,380,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 8:107,849,895\n",
      "INFO\t2017-01-12 16:05:15\tGenotypeConcordance\tchecked    17,390,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 8:108,706,623\n",
      "INFO\t2017-01-12 16:05:15\tGenotypeConcordance\tchecked    17,400,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 8:109,608,623\n",
      "INFO\t2017-01-12 16:05:16\tGenotypeConcordance\tchecked    17,410,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 8:110,573,770\n",
      "INFO\t2017-01-12 16:05:16\tGenotypeConcordance\tchecked    17,420,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 8:111,622,857\n",
      "INFO\t2017-01-12 16:05:16\tGenotypeConcordance\tchecked    17,430,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 8:112,537,538\n",
      "INFO\t2017-01-12 16:05:16\tGenotypeConcordance\tchecked    17,440,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 8:113,450,623\n",
      "INFO\t2017-01-12 16:05:17\tGenotypeConcordance\tchecked    17,450,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 8:114,440,909\n",
      "INFO\t2017-01-12 16:05:17\tGenotypeConcordance\tchecked    17,460,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 8:115,245,185\n",
      "INFO\t2017-01-12 16:05:17\tGenotypeConcordance\tchecked    17,470,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 8:116,080,876\n",
      "INFO\t2017-01-12 16:05:17\tGenotypeConcordance\tchecked    17,480,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 8:117,160,596\n",
      "INFO\t2017-01-12 16:05:18\tGenotypeConcordance\tchecked    17,490,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 8:118,146,283\n",
      "INFO\t2017-01-12 16:05:18\tGenotypeConcordance\tchecked    17,500,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 8:119,097,287\n",
      "INFO\t2017-01-12 16:05:18\tGenotypeConcordance\tchecked    17,510,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 8:119,972,696\n",
      "INFO\t2017-01-12 16:05:18\tGenotypeConcordance\tchecked    17,520,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 8:120,860,318\n",
      "INFO\t2017-01-12 16:05:19\tGenotypeConcordance\tchecked    17,530,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 8:121,777,896\n",
      "INFO\t2017-01-12 16:05:19\tGenotypeConcordance\tchecked    17,540,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 8:122,583,991\n",
      "INFO\t2017-01-12 16:05:19\tGenotypeConcordance\tchecked    17,550,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 8:123,466,306\n",
      "INFO\t2017-01-12 16:05:19\tGenotypeConcordance\tchecked    17,560,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 8:124,350,791\n",
      "INFO\t2017-01-12 16:05:20\tGenotypeConcordance\tchecked    17,570,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 8:125,229,684\n",
      "INFO\t2017-01-12 16:05:20\tGenotypeConcordance\tchecked    17,580,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 8:126,009,595\n",
      "INFO\t2017-01-12 16:05:20\tGenotypeConcordance\tchecked    17,590,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 8:126,859,771\n",
      "INFO\t2017-01-12 16:05:20\tGenotypeConcordance\tchecked    17,600,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 8:127,651,334\n",
      "INFO\t2017-01-12 16:05:21\tGenotypeConcordance\tchecked    17,610,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 8:128,446,680\n",
      "INFO\t2017-01-12 16:05:21\tGenotypeConcordance\tchecked    17,620,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 8:129,223,877\n",
      "INFO\t2017-01-12 16:05:21\tGenotypeConcordance\tchecked    17,630,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 8:130,025,628\n",
      "INFO\t2017-01-12 16:05:21\tGenotypeConcordance\tchecked    17,640,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 8:130,877,547\n",
      "INFO\t2017-01-12 16:05:22\tGenotypeConcordance\tchecked    17,650,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 8:131,731,502\n",
      "INFO\t2017-01-12 16:05:22\tGenotypeConcordance\tchecked    17,660,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 8:132,508,007\n",
      "INFO\t2017-01-12 16:05:22\tGenotypeConcordance\tchecked    17,670,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 8:133,264,699\n",
      "INFO\t2017-01-12 16:05:22\tGenotypeConcordance\tchecked    17,680,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 8:134,161,488\n",
      "INFO\t2017-01-12 16:05:23\tGenotypeConcordance\tchecked    17,690,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 8:134,885,329\n",
      "INFO\t2017-01-12 16:05:23\tGenotypeConcordance\tchecked    17,700,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 8:135,641,806\n",
      "INFO\t2017-01-12 16:05:23\tGenotypeConcordance\tchecked    17,710,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 8:136,456,909\n",
      "INFO\t2017-01-12 16:05:23\tGenotypeConcordance\tchecked    17,720,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 8:137,291,187\n",
      "INFO\t2017-01-12 16:05:24\tGenotypeConcordance\tchecked    17,730,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 8:138,078,078\n",
      "INFO\t2017-01-12 16:05:24\tGenotypeConcordance\tchecked    17,740,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 8:138,839,468\n",
      "INFO\t2017-01-12 16:05:24\tGenotypeConcordance\tchecked    17,750,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 8:139,625,123\n",
      "INFO\t2017-01-12 16:05:24\tGenotypeConcordance\tchecked    17,760,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 8:140,335,008\n",
      "INFO\t2017-01-12 16:05:25\tGenotypeConcordance\tchecked    17,770,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 8:141,074,664\n",
      "INFO\t2017-01-12 16:05:25\tGenotypeConcordance\tchecked    17,780,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 8:141,946,507\n",
      "INFO\t2017-01-12 16:05:25\tGenotypeConcordance\tchecked    17,790,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 8:142,636,650\n",
      "INFO\t2017-01-12 16:05:25\tGenotypeConcordance\tchecked    17,800,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 8:143,364,390\n",
      "INFO\t2017-01-12 16:05:26\tGenotypeConcordance\tchecked    17,810,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 8:144,058,475\n",
      "INFO\t2017-01-12 16:05:26\tGenotypeConcordance\tchecked    17,820,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 8:144,717,757\n",
      "INFO\t2017-01-12 16:05:26\tGenotypeConcordance\tchecked    17,830,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 8:145,752,130\n",
      "INFO\t2017-01-12 16:05:26\tGenotypeConcordance\tchecked    17,840,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 9:210,882\n",
      "INFO\t2017-01-12 16:05:27\tGenotypeConcordance\tchecked    17,850,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 9:765,689\n",
      "INFO\t2017-01-12 16:05:27\tGenotypeConcordance\tchecked    17,860,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 9:1,289,862\n",
      "INFO\t2017-01-12 16:05:27\tGenotypeConcordance\tchecked    17,870,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 9:1,806,787\n",
      "INFO\t2017-01-12 16:05:28\tGenotypeConcordance\tchecked    17,880,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 9:2,402,859\n",
      "INFO\t2017-01-12 16:05:28\tGenotypeConcordance\tchecked    17,890,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 9:3,039,034\n",
      "INFO\t2017-01-12 16:05:28\tGenotypeConcordance\tchecked    17,900,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 9:3,780,497\n",
      "INFO\t2017-01-12 16:05:28\tGenotypeConcordance\tchecked    17,910,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 9:4,393,680\n",
      "INFO\t2017-01-12 16:05:29\tGenotypeConcordance\tchecked    17,920,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 9:4,990,416\n",
      "INFO\t2017-01-12 16:05:29\tGenotypeConcordance\tchecked    17,930,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 9:5,720,296\n",
      "INFO\t2017-01-12 16:05:29\tGenotypeConcordance\tchecked    17,940,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 9:6,577,741\n",
      "INFO\t2017-01-12 16:05:29\tGenotypeConcordance\tchecked    17,950,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 9:7,147,976\n",
      "INFO\t2017-01-12 16:05:30\tGenotypeConcordance\tchecked    17,960,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 9:7,727,080\n",
      "INFO\t2017-01-12 16:05:30\tGenotypeConcordance\tchecked    17,970,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 9:8,371,681\n",
      "INFO\t2017-01-12 16:05:30\tGenotypeConcordance\tchecked    17,980,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 9:8,979,212\n",
      "INFO\t2017-01-12 16:05:30\tGenotypeConcordance\tchecked    17,990,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 9:9,634,771\n",
      "INFO\t2017-01-12 16:05:31\tGenotypeConcordance\tchecked    18,000,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 9:10,207,866\n",
      "INFO\t2017-01-12 16:05:31\tGenotypeConcordance\tchecked    18,010,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 9:10,835,964\n",
      "INFO\t2017-01-12 16:05:31\tGenotypeConcordance\tchecked    18,020,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 9:11,426,814\n",
      "INFO\t2017-01-12 16:05:31\tGenotypeConcordance\tchecked    18,030,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 9:12,019,745\n",
      "INFO\t2017-01-12 16:05:32\tGenotypeConcordance\tchecked    18,040,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 9:12,693,777\n",
      "INFO\t2017-01-12 16:05:32\tGenotypeConcordance\tchecked    18,050,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 9:13,505,677\n",
      "INFO\t2017-01-12 16:05:32\tGenotypeConcordance\tchecked    18,060,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 9:14,182,713\n",
      "INFO\t2017-01-12 16:05:32\tGenotypeConcordance\tchecked    18,070,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 9:14,907,332\n",
      "INFO\t2017-01-12 16:05:33\tGenotypeConcordance\tchecked    18,080,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 9:15,578,240\n",
      "INFO\t2017-01-12 16:05:33\tGenotypeConcordance\tchecked    18,090,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 9:16,299,388\n",
      "INFO\t2017-01-12 16:05:33\tGenotypeConcordance\tchecked    18,100,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 9:16,974,010\n",
      "INFO\t2017-01-12 16:05:33\tGenotypeConcordance\tchecked    18,110,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 9:17,652,187\n",
      "INFO\t2017-01-12 16:05:34\tGenotypeConcordance\tchecked    18,120,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 9:18,313,213\n",
      "INFO\t2017-01-12 16:05:34\tGenotypeConcordance\tchecked    18,130,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 9:18,999,766\n",
      "INFO\t2017-01-12 16:05:34\tGenotypeConcordance\tchecked    18,140,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 9:19,706,618\n",
      "INFO\t2017-01-12 16:05:34\tGenotypeConcordance\tchecked    18,150,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 9:20,477,167\n",
      "INFO\t2017-01-12 16:05:35\tGenotypeConcordance\tchecked    18,160,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 9:21,292,255\n",
      "INFO\t2017-01-12 16:05:35\tGenotypeConcordance\tchecked    18,170,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 9:22,088,832\n",
      "INFO\t2017-01-12 16:05:35\tGenotypeConcordance\tchecked    18,180,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 9:22,841,222\n",
      "INFO\t2017-01-12 16:05:35\tGenotypeConcordance\tchecked    18,190,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 9:23,504,185\n",
      "INFO\t2017-01-12 16:05:36\tGenotypeConcordance\tchecked    18,200,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 9:24,301,818\n",
      "INFO\t2017-01-12 16:05:36\tGenotypeConcordance\tchecked    18,210,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 9:25,042,178\n",
      "INFO\t2017-01-12 16:05:36\tGenotypeConcordance\tchecked    18,220,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 9:25,687,819\n",
      "INFO\t2017-01-12 16:05:36\tGenotypeConcordance\tchecked    18,230,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 9:26,465,331\n",
      "INFO\t2017-01-12 16:05:36\tGenotypeConcordance\tchecked    18,240,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 9:27,199,009\n",
      "INFO\t2017-01-12 16:05:37\tGenotypeConcordance\tchecked    18,250,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 9:27,937,370\n",
      "INFO\t2017-01-12 16:05:37\tGenotypeConcordance\tchecked    18,260,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 9:28,639,917\n",
      "INFO\t2017-01-12 16:05:37\tGenotypeConcordance\tchecked    18,270,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 9:29,409,616\n",
      "INFO\t2017-01-12 16:05:37\tGenotypeConcordance\tchecked    18,280,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 9:30,137,652\n",
      "INFO\t2017-01-12 16:05:38\tGenotypeConcordance\tchecked    18,290,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 9:30,938,150\n",
      "INFO\t2017-01-12 16:05:38\tGenotypeConcordance\tchecked    18,300,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 9:31,622,953\n",
      "INFO\t2017-01-12 16:05:38\tGenotypeConcordance\tchecked    18,310,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 9:32,324,971\n",
      "INFO\t2017-01-12 16:05:38\tGenotypeConcordance\tchecked    18,320,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 9:33,170,547\n",
      "INFO\t2017-01-12 16:05:39\tGenotypeConcordance\tchecked    18,330,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 9:34,027,736\n",
      "INFO\t2017-01-12 16:05:39\tGenotypeConcordance\tchecked    18,340,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 9:35,037,954\n",
      "INFO\t2017-01-12 16:05:39\tGenotypeConcordance\tchecked    18,350,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 9:36,104,753\n",
      "INFO\t2017-01-12 16:05:39\tGenotypeConcordance\tchecked    18,360,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 9:36,965,295\n",
      "INFO\t2017-01-12 16:05:40\tGenotypeConcordance\tchecked    18,370,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 9:37,889,110\n",
      "INFO\t2017-01-12 16:05:40\tGenotypeConcordance\tchecked    18,380,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 9:38,676,375\n",
      "INFO\t2017-01-12 16:05:40\tGenotypeConcordance\tchecked    18,390,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 9:40,646,116\n",
      "INFO\t2017-01-12 16:05:41\tGenotypeConcordance\tchecked    18,400,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 9:43,437,884\n",
      "INFO\t2017-01-12 16:05:41\tGenotypeConcordance\tchecked    18,410,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 9:44,761,195\n",
      "INFO\t2017-01-12 16:05:41\tGenotypeConcordance\tchecked    18,420,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 9:46,141,478\n",
      "INFO\t2017-01-12 16:05:41\tGenotypeConcordance\tchecked    18,430,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 9:66,478,266\n",
      "INFO\t2017-01-12 16:05:42\tGenotypeConcordance\tchecked    18,440,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 9:67,302,206\n",
      "INFO\t2017-01-12 16:05:42\tGenotypeConcordance\tchecked    18,450,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 9:68,406,490\n",
      "INFO\t2017-01-12 16:05:42\tGenotypeConcordance\tchecked    18,460,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 9:68,990,079\n",
      "INFO\t2017-01-12 16:05:42\tGenotypeConcordance\tchecked    18,470,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 9:69,784,342\n",
      "INFO\t2017-01-12 16:05:43\tGenotypeConcordance\tchecked    18,480,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 9:70,944,657\n",
      "INFO\t2017-01-12 16:05:43\tGenotypeConcordance\tchecked    18,490,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 9:71,842,391\n",
      "INFO\t2017-01-12 16:05:43\tGenotypeConcordance\tchecked    18,500,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 9:72,743,932\n",
      "INFO\t2017-01-12 16:05:43\tGenotypeConcordance\tchecked    18,510,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 9:73,649,941\n",
      "INFO\t2017-01-12 16:05:44\tGenotypeConcordance\tchecked    18,520,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 9:74,555,136\n",
      "INFO\t2017-01-12 16:05:44\tGenotypeConcordance\tchecked    18,530,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 9:75,492,246\n",
      "INFO\t2017-01-12 16:05:44\tGenotypeConcordance\tchecked    18,540,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 9:76,491,744\n",
      "INFO\t2017-01-12 16:05:44\tGenotypeConcordance\tchecked    18,550,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 9:77,464,267\n",
      "INFO\t2017-01-12 16:05:45\tGenotypeConcordance\tchecked    18,560,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 9:78,294,951\n",
      "INFO\t2017-01-12 16:05:45\tGenotypeConcordance\tchecked    18,570,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 9:79,099,199\n",
      "INFO\t2017-01-12 16:05:45\tGenotypeConcordance\tchecked    18,580,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 9:79,849,874\n",
      "INFO\t2017-01-12 16:05:45\tGenotypeConcordance\tchecked    18,590,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 9:80,741,971\n",
      "INFO\t2017-01-12 16:05:46\tGenotypeConcordance\tchecked    18,600,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 9:81,539,515\n",
      "INFO\t2017-01-12 16:05:46\tGenotypeConcordance\tchecked    18,610,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 9:82,385,885\n",
      "INFO\t2017-01-12 16:05:46\tGenotypeConcordance\tchecked    18,620,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 9:83,179,764\n",
      "INFO\t2017-01-12 16:05:46\tGenotypeConcordance\tchecked    18,630,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 9:83,895,853\n",
      "INFO\t2017-01-12 16:05:47\tGenotypeConcordance\tchecked    18,640,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 9:84,900,687\n",
      "INFO\t2017-01-12 16:05:47\tGenotypeConcordance\tchecked    18,650,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 9:85,847,317\n",
      "INFO\t2017-01-12 16:05:47\tGenotypeConcordance\tchecked    18,660,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 9:86,738,065\n",
      "INFO\t2017-01-12 16:05:47\tGenotypeConcordance\tchecked    18,670,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 9:87,579,283\n",
      "INFO\t2017-01-12 16:05:48\tGenotypeConcordance\tchecked    18,680,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 9:88,487,791\n",
      "INFO\t2017-01-12 16:05:48\tGenotypeConcordance\tchecked    18,690,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 9:89,340,620\n",
      "INFO\t2017-01-12 16:05:48\tGenotypeConcordance\tchecked    18,700,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 9:90,141,514\n",
      "INFO\t2017-01-12 16:05:48\tGenotypeConcordance\tchecked    18,710,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 9:90,871,647\n",
      "INFO\t2017-01-12 16:05:49\tGenotypeConcordance\tchecked    18,720,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 9:91,590,324\n",
      "INFO\t2017-01-12 16:05:49\tGenotypeConcordance\tchecked    18,730,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 9:92,495,150\n",
      "INFO\t2017-01-12 16:05:49\tGenotypeConcordance\tchecked    18,740,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 9:93,451,481\n",
      "INFO\t2017-01-12 16:05:49\tGenotypeConcordance\tchecked    18,750,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 9:94,326,169\n",
      "INFO\t2017-01-12 16:05:50\tGenotypeConcordance\tchecked    18,760,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 9:95,210,570\n",
      "INFO\t2017-01-12 16:05:50\tGenotypeConcordance\tchecked    18,770,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 9:96,114,712\n",
      "INFO\t2017-01-12 16:05:50\tGenotypeConcordance\tchecked    18,780,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 9:97,010,498\n",
      "INFO\t2017-01-12 16:05:50\tGenotypeConcordance\tchecked    18,790,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 9:97,898,118\n",
      "INFO\t2017-01-12 16:05:51\tGenotypeConcordance\tchecked    18,800,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 9:98,839,797\n",
      "INFO\t2017-01-12 16:05:51\tGenotypeConcordance\tchecked    18,810,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 9:99,739,358\n",
      "INFO\t2017-01-12 16:05:51\tGenotypeConcordance\tchecked    18,820,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 9:100,704,638\n",
      "INFO\t2017-01-12 16:05:51\tGenotypeConcordance\tchecked    18,830,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 9:101,574,873\n",
      "INFO\t2017-01-12 16:05:52\tGenotypeConcordance\tchecked    18,840,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 9:102,651,712\n",
      "INFO\t2017-01-12 16:05:52\tGenotypeConcordance\tchecked    18,850,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 9:103,704,965\n",
      "INFO\t2017-01-12 16:05:52\tGenotypeConcordance\tchecked    18,860,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 9:104,605,646\n",
      "INFO\t2017-01-12 16:05:52\tGenotypeConcordance\tchecked    18,870,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 9:105,307,422\n",
      "INFO\t2017-01-12 16:05:53\tGenotypeConcordance\tchecked    18,880,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 9:106,024,523\n",
      "INFO\t2017-01-12 16:05:53\tGenotypeConcordance\tchecked    18,890,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 9:106,760,737\n",
      "INFO\t2017-01-12 16:05:53\tGenotypeConcordance\tchecked    18,900,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 9:107,525,317\n",
      "INFO\t2017-01-12 16:05:53\tGenotypeConcordance\tchecked    18,910,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 9:108,293,284\n",
      "INFO\t2017-01-12 16:05:54\tGenotypeConcordance\tchecked    18,920,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 9:109,106,490\n",
      "INFO\t2017-01-12 16:05:54\tGenotypeConcordance\tchecked    18,930,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 9:110,051,413\n",
      "INFO\t2017-01-12 16:05:54\tGenotypeConcordance\tchecked    18,940,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 9:110,801,732\n",
      "INFO\t2017-01-12 16:05:54\tGenotypeConcordance\tchecked    18,950,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 9:111,541,925\n",
      "INFO\t2017-01-12 16:05:55\tGenotypeConcordance\tchecked    18,960,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 9:112,419,640\n",
      "INFO\t2017-01-12 16:05:55\tGenotypeConcordance\tchecked    18,970,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 9:113,197,163\n",
      "INFO\t2017-01-12 16:05:55\tGenotypeConcordance\tchecked    18,980,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 9:113,996,326\n",
      "INFO\t2017-01-12 16:05:55\tGenotypeConcordance\tchecked    18,990,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 9:114,858,970\n",
      "INFO\t2017-01-12 16:05:56\tGenotypeConcordance\tchecked    19,000,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 9:115,728,200\n",
      "INFO\t2017-01-12 16:05:56\tGenotypeConcordance\tchecked    19,010,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 9:116,692,868\n",
      "INFO\t2017-01-12 16:05:56\tGenotypeConcordance\tchecked    19,020,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 9:117,505,696\n",
      "INFO\t2017-01-12 16:05:56\tGenotypeConcordance\tchecked    19,030,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 9:118,393,276\n",
      "INFO\t2017-01-12 16:05:57\tGenotypeConcordance\tchecked    19,040,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 9:119,383,881\n",
      "INFO\t2017-01-12 16:05:57\tGenotypeConcordance\tchecked    19,050,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 9:120,234,468\n",
      "INFO\t2017-01-12 16:05:57\tGenotypeConcordance\tchecked    19,060,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 9:121,146,490\n",
      "INFO\t2017-01-12 16:05:57\tGenotypeConcordance\tchecked    19,070,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 9:122,051,697\n",
      "INFO\t2017-01-12 16:05:58\tGenotypeConcordance\tchecked    19,080,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 9:123,018,415\n",
      "INFO\t2017-01-12 16:05:58\tGenotypeConcordance\tchecked    19,090,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 9:124,093,813\n",
      "INFO\t2017-01-12 16:05:58\tGenotypeConcordance\tchecked    19,100,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 9:124,977,324\n",
      "INFO\t2017-01-12 16:05:58\tGenotypeConcordance\tchecked    19,110,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 9:125,921,216\n",
      "INFO\t2017-01-12 16:05:59\tGenotypeConcordance\tchecked    19,120,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 9:126,913,488\n",
      "INFO\t2017-01-12 16:05:59\tGenotypeConcordance\tchecked    19,130,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 9:127,809,938\n",
      "INFO\t2017-01-12 16:05:59\tGenotypeConcordance\tchecked    19,140,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 9:128,791,885\n",
      "INFO\t2017-01-12 16:05:59\tGenotypeConcordance\tchecked    19,150,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 9:129,631,774\n",
      "INFO\t2017-01-12 16:06:00\tGenotypeConcordance\tchecked    19,160,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 9:130,680,447\n",
      "INFO\t2017-01-12 16:06:00\tGenotypeConcordance\tchecked    19,170,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 9:131,549,221\n",
      "INFO\t2017-01-12 16:06:00\tGenotypeConcordance\tchecked    19,180,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 9:132,249,683\n",
      "INFO\t2017-01-12 16:06:00\tGenotypeConcordance\tchecked    19,190,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 9:132,983,263\n",
      "INFO\t2017-01-12 16:06:01\tGenotypeConcordance\tchecked    19,200,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 9:133,887,443\n",
      "INFO\t2017-01-12 16:06:01\tGenotypeConcordance\tchecked    19,210,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 9:134,700,370\n",
      "INFO\t2017-01-12 16:06:01\tGenotypeConcordance\tchecked    19,220,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 9:135,558,065\n",
      "INFO\t2017-01-12 16:06:01\tGenotypeConcordance\tchecked    19,230,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 9:136,293,373\n",
      "INFO\t2017-01-12 16:06:02\tGenotypeConcordance\tchecked    19,240,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 9:136,949,180\n",
      "INFO\t2017-01-12 16:06:02\tGenotypeConcordance\tchecked    19,250,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 9:137,627,492\n",
      "INFO\t2017-01-12 16:06:02\tGenotypeConcordance\tchecked    19,260,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 9:138,204,537\n",
      "INFO\t2017-01-12 16:06:02\tGenotypeConcordance\tchecked    19,270,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 9:138,839,570\n",
      "INFO\t2017-01-12 16:06:03\tGenotypeConcordance\tchecked    19,280,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 9:139,567,127\n",
      "INFO\t2017-01-12 16:06:03\tGenotypeConcordance\tchecked    19,290,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 9:140,296,257\n",
      "INFO\t2017-01-12 16:06:03\tGenotypeConcordance\tchecked    19,300,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 9:141,081,573\n",
      "INFO\t2017-01-12 16:06:03\tGenotypeConcordance\tchecked    19,310,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 10:725,681\n",
      "INFO\t2017-01-12 16:06:03\tGenotypeConcordance\tchecked    19,320,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 10:1,415,982\n",
      "INFO\t2017-01-12 16:06:04\tGenotypeConcordance\tchecked    19,330,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 10:2,048,779\n",
      "INFO\t2017-01-12 16:06:04\tGenotypeConcordance\tchecked    19,340,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 10:2,669,744\n",
      "INFO\t2017-01-12 16:06:04\tGenotypeConcordance\tchecked    19,350,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 10:3,237,782\n",
      "INFO\t2017-01-12 16:06:05\tGenotypeConcordance\tchecked    19,360,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 10:3,972,634\n",
      "INFO\t2017-01-12 16:06:05\tGenotypeConcordance\tchecked    19,370,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 10:4,660,761\n",
      "INFO\t2017-01-12 16:06:05\tGenotypeConcordance\tchecked    19,380,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 10:5,265,216\n",
      "INFO\t2017-01-12 16:06:05\tGenotypeConcordance\tchecked    19,390,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 10:5,931,011\n",
      "INFO\t2017-01-12 16:06:06\tGenotypeConcordance\tchecked    19,400,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 10:6,645,899\n",
      "INFO\t2017-01-12 16:06:06\tGenotypeConcordance\tchecked    19,410,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 10:7,286,305\n",
      "INFO\t2017-01-12 16:06:06\tGenotypeConcordance\tchecked    19,420,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 10:8,018,107\n",
      "INFO\t2017-01-12 16:06:06\tGenotypeConcordance\tchecked    19,430,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 10:8,728,517\n",
      "INFO\t2017-01-12 16:06:07\tGenotypeConcordance\tchecked    19,440,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 10:9,561,105\n",
      "INFO\t2017-01-12 16:06:07\tGenotypeConcordance\tchecked    19,450,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 10:10,380,004\n",
      "INFO\t2017-01-12 16:06:07\tGenotypeConcordance\tchecked    19,460,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 10:11,181,950\n",
      "INFO\t2017-01-12 16:06:07\tGenotypeConcordance\tchecked    19,470,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 10:11,933,218\n",
      "INFO\t2017-01-12 16:06:08\tGenotypeConcordance\tchecked    19,480,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 10:12,616,887\n",
      "INFO\t2017-01-12 16:06:08\tGenotypeConcordance\tchecked    19,490,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 10:13,283,711\n",
      "INFO\t2017-01-12 16:06:08\tGenotypeConcordance\tchecked    19,500,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 10:13,966,471\n",
      "INFO\t2017-01-12 16:06:08\tGenotypeConcordance\tchecked    19,510,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 10:14,669,633\n",
      "INFO\t2017-01-12 16:06:09\tGenotypeConcordance\tchecked    19,520,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 10:15,373,886\n",
      "INFO\t2017-01-12 16:06:09\tGenotypeConcordance\tchecked    19,530,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 10:16,116,605\n",
      "INFO\t2017-01-12 16:06:09\tGenotypeConcordance\tchecked    19,540,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 10:16,809,597\n",
      "INFO\t2017-01-12 16:06:09\tGenotypeConcordance\tchecked    19,550,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 10:17,531,922\n",
      "INFO\t2017-01-12 16:06:10\tGenotypeConcordance\tchecked    19,560,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 10:18,586,608\n",
      "INFO\t2017-01-12 16:06:10\tGenotypeConcordance\tchecked    19,570,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 10:19,287,256\n",
      "INFO\t2017-01-12 16:06:10\tGenotypeConcordance\tchecked    19,580,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 10:19,941,090\n",
      "INFO\t2017-01-12 16:06:10\tGenotypeConcordance\tchecked    19,590,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 10:20,677,088\n",
      "INFO\t2017-01-12 16:06:11\tGenotypeConcordance\tchecked    19,600,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 10:21,386,755\n",
      "INFO\t2017-01-12 16:06:11\tGenotypeConcordance\tchecked    19,610,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 10:22,397,086\n",
      "INFO\t2017-01-12 16:06:11\tGenotypeConcordance\tchecked    19,620,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 10:23,293,921\n",
      "INFO\t2017-01-12 16:06:11\tGenotypeConcordance\tchecked    19,630,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 10:24,079,364\n",
      "INFO\t2017-01-12 16:06:12\tGenotypeConcordance\tchecked    19,640,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 10:24,880,195\n",
      "INFO\t2017-01-12 16:06:12\tGenotypeConcordance\tchecked    19,650,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 10:25,728,766\n",
      "INFO\t2017-01-12 16:06:12\tGenotypeConcordance\tchecked    19,660,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 10:26,497,157\n",
      "INFO\t2017-01-12 16:06:12\tGenotypeConcordance\tchecked    19,670,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 10:27,176,737\n",
      "INFO\t2017-01-12 16:06:13\tGenotypeConcordance\tchecked    19,680,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 10:27,899,408\n",
      "INFO\t2017-01-12 16:06:13\tGenotypeConcordance\tchecked    19,690,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 10:28,727,624\n",
      "INFO\t2017-01-12 16:06:13\tGenotypeConcordance\tchecked    19,700,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 10:29,464,255\n",
      "INFO\t2017-01-12 16:06:13\tGenotypeConcordance\tchecked    19,710,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 10:30,175,131\n",
      "INFO\t2017-01-12 16:06:14\tGenotypeConcordance\tchecked    19,720,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 10:30,892,831\n",
      "INFO\t2017-01-12 16:06:14\tGenotypeConcordance\tchecked    19,730,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 10:31,680,578\n",
      "INFO\t2017-01-12 16:06:14\tGenotypeConcordance\tchecked    19,740,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 10:32,538,430\n",
      "INFO\t2017-01-12 16:06:14\tGenotypeConcordance\tchecked    19,750,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 10:33,387,550\n",
      "INFO\t2017-01-12 16:06:15\tGenotypeConcordance\tchecked    19,760,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 10:34,202,005\n",
      "INFO\t2017-01-12 16:06:15\tGenotypeConcordance\tchecked    19,770,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 10:35,061,590\n",
      "INFO\t2017-01-12 16:06:15\tGenotypeConcordance\tchecked    19,780,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 10:35,984,031\n",
      "INFO\t2017-01-12 16:06:15\tGenotypeConcordance\tchecked    19,790,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 10:36,784,026\n",
      "INFO\t2017-01-12 16:06:16\tGenotypeConcordance\tchecked    19,800,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 10:37,586,233\n",
      "INFO\t2017-01-12 16:06:16\tGenotypeConcordance\tchecked    19,810,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 10:38,474,106\n",
      "INFO\t2017-01-12 16:06:16\tGenotypeConcordance\tchecked    19,820,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 10:38,982,460\n",
      "INFO\t2017-01-12 16:06:16\tGenotypeConcordance\tchecked    19,830,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 10:42,383,853\n",
      "INFO\t2017-01-12 16:06:17\tGenotypeConcordance\tchecked    19,840,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 10:42,630,572\n",
      "INFO\t2017-01-12 16:06:17\tGenotypeConcordance\tchecked    19,850,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 10:43,024,580\n",
      "INFO\t2017-01-12 16:06:17\tGenotypeConcordance\tchecked    19,860,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 10:43,820,851\n",
      "INFO\t2017-01-12 16:06:17\tGenotypeConcordance\tchecked    19,870,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 10:44,616,224\n",
      "INFO\t2017-01-12 16:06:18\tGenotypeConcordance\tchecked    19,880,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 10:45,375,110\n",
      "INFO\t2017-01-12 16:06:18\tGenotypeConcordance\tchecked    19,890,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 10:46,186,800\n",
      "INFO\t2017-01-12 16:06:18\tGenotypeConcordance\tchecked    19,900,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 10:47,698,150\n",
      "INFO\t2017-01-12 16:06:18\tGenotypeConcordance\tchecked    19,910,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 10:49,572,482\n",
      "INFO\t2017-01-12 16:06:19\tGenotypeConcordance\tchecked    19,920,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 10:50,385,880\n",
      "INFO\t2017-01-12 16:06:19\tGenotypeConcordance\tchecked    19,930,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 10:51,504,074\n",
      "INFO\t2017-01-12 16:06:19\tGenotypeConcordance\tchecked    19,940,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 10:52,590,708\n",
      "INFO\t2017-01-12 16:06:19\tGenotypeConcordance\tchecked    19,950,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 10:53,444,658\n",
      "INFO\t2017-01-12 16:06:20\tGenotypeConcordance\tchecked    19,960,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 10:54,219,820\n",
      "INFO\t2017-01-12 16:06:20\tGenotypeConcordance\tchecked    19,970,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 10:55,000,909\n",
      "INFO\t2017-01-12 16:06:20\tGenotypeConcordance\tchecked    19,980,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 10:55,728,299\n",
      "INFO\t2017-01-12 16:06:20\tGenotypeConcordance\tchecked    19,990,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 10:56,417,474\n",
      "INFO\t2017-01-12 16:06:21\tGenotypeConcordance\tchecked    20,000,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 10:57,209,676\n",
      "INFO\t2017-01-12 16:06:21\tGenotypeConcordance\tchecked    20,010,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 10:58,005,755\n",
      "INFO\t2017-01-12 16:06:21\tGenotypeConcordance\tchecked    20,020,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 10:58,784,814\n",
      "INFO\t2017-01-12 16:06:21\tGenotypeConcordance\tchecked    20,030,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 10:59,483,570\n",
      "INFO\t2017-01-12 16:06:22\tGenotypeConcordance\tchecked    20,040,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 10:60,314,183\n",
      "INFO\t2017-01-12 16:06:22\tGenotypeConcordance\tchecked    20,050,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 10:61,311,243\n",
      "INFO\t2017-01-12 16:06:22\tGenotypeConcordance\tchecked    20,060,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 10:62,136,212\n",
      "INFO\t2017-01-12 16:06:22\tGenotypeConcordance\tchecked    20,070,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 10:63,002,267\n",
      "INFO\t2017-01-12 16:06:23\tGenotypeConcordance\tchecked    20,080,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 10:63,918,915\n",
      "INFO\t2017-01-12 16:06:23\tGenotypeConcordance\tchecked    20,090,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 10:64,810,727\n",
      "INFO\t2017-01-12 16:06:23\tGenotypeConcordance\tchecked    20,100,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 10:65,743,503\n",
      "INFO\t2017-01-12 16:06:23\tGenotypeConcordance\tchecked    20,110,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 10:66,613,822\n",
      "INFO\t2017-01-12 16:06:24\tGenotypeConcordance\tchecked    20,120,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 10:67,321,688\n",
      "INFO\t2017-01-12 16:06:24\tGenotypeConcordance\tchecked    20,130,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 10:68,085,353\n",
      "INFO\t2017-01-12 16:06:24\tGenotypeConcordance\tchecked    20,140,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 10:68,788,942\n",
      "INFO\t2017-01-12 16:06:24\tGenotypeConcordance\tchecked    20,150,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 10:69,661,777\n",
      "INFO\t2017-01-12 16:06:25\tGenotypeConcordance\tchecked    20,160,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 10:70,491,316\n",
      "INFO\t2017-01-12 16:06:25\tGenotypeConcordance\tchecked    20,170,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 10:71,268,224\n",
      "INFO\t2017-01-12 16:06:25\tGenotypeConcordance\tchecked    20,180,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 10:72,058,768\n",
      "INFO\t2017-01-12 16:06:26\tGenotypeConcordance\tchecked    20,190,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 10:72,788,149\n",
      "INFO\t2017-01-12 16:06:26\tGenotypeConcordance\tchecked    20,200,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 10:73,578,704\n",
      "INFO\t2017-01-12 16:06:26\tGenotypeConcordance\tchecked    20,210,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 10:74,476,968\n",
      "INFO\t2017-01-12 16:06:27\tGenotypeConcordance\tchecked    20,220,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 10:75,568,839\n",
      "INFO\t2017-01-12 16:06:27\tGenotypeConcordance\tchecked    20,230,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 10:76,679,726\n",
      "INFO\t2017-01-12 16:06:27\tGenotypeConcordance\tchecked    20,240,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 10:77,737,701\n",
      "INFO\t2017-01-12 16:06:28\tGenotypeConcordance\tchecked    20,250,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 10:78,671,629\n",
      "INFO\t2017-01-12 16:06:28\tGenotypeConcordance\tchecked    20,260,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 10:79,600,256\n",
      "INFO\t2017-01-12 16:06:28\tGenotypeConcordance\tchecked    20,270,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 10:80,410,447\n",
      "INFO\t2017-01-12 16:06:28\tGenotypeConcordance\tchecked    20,280,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 10:81,229,468\n",
      "INFO\t2017-01-12 16:06:29\tGenotypeConcordance\tchecked    20,290,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 10:81,924,530\n",
      "INFO\t2017-01-12 16:06:29\tGenotypeConcordance\tchecked    20,300,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 10:82,720,794\n",
      "INFO\t2017-01-12 16:06:29\tGenotypeConcordance\tchecked    20,310,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 10:83,496,501\n",
      "INFO\t2017-01-12 16:06:30\tGenotypeConcordance\tchecked    20,320,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 10:84,358,581\n",
      "INFO\t2017-01-12 16:06:30\tGenotypeConcordance\tchecked    20,330,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 10:85,176,752\n",
      "INFO\t2017-01-12 16:06:30\tGenotypeConcordance\tchecked    20,340,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 10:85,947,354\n",
      "INFO\t2017-01-12 16:06:31\tGenotypeConcordance\tchecked    20,350,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 10:86,800,415\n",
      "INFO\t2017-01-12 16:06:31\tGenotypeConcordance\tchecked    20,360,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 10:87,535,013\n",
      "INFO\t2017-01-12 16:06:31\tGenotypeConcordance\tchecked    20,370,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 10:88,409,087\n",
      "INFO\t2017-01-12 16:06:31\tGenotypeConcordance\tchecked    20,380,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 10:89,361,512\n",
      "INFO\t2017-01-12 16:06:32\tGenotypeConcordance\tchecked    20,390,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 10:90,361,234\n",
      "INFO\t2017-01-12 16:06:32\tGenotypeConcordance\tchecked    20,400,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 10:91,291,104\n",
      "INFO\t2017-01-12 16:06:32\tGenotypeConcordance\tchecked    20,410,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 10:92,100,943\n",
      "INFO\t2017-01-12 16:06:33\tGenotypeConcordance\tchecked    20,420,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 10:92,980,926\n",
      "INFO\t2017-01-12 16:06:33\tGenotypeConcordance\tchecked    20,430,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 10:94,056,078\n",
      "INFO\t2017-01-12 16:06:33\tGenotypeConcordance\tchecked    20,440,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 10:95,024,704\n",
      "INFO\t2017-01-12 16:06:34\tGenotypeConcordance\tchecked    20,450,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 10:95,859,119\n",
      "INFO\t2017-01-12 16:06:34\tGenotypeConcordance\tchecked    20,460,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 10:96,752,130\n",
      "INFO\t2017-01-12 16:06:34\tGenotypeConcordance\tchecked    20,470,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 10:97,720,824\n",
      "INFO\t2017-01-12 16:06:34\tGenotypeConcordance\tchecked    20,480,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 10:98,703,175\n",
      "INFO\t2017-01-12 16:06:35\tGenotypeConcordance\tchecked    20,490,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 10:99,636,526\n",
      "INFO\t2017-01-12 16:06:35\tGenotypeConcordance\tchecked    20,500,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 10:100,526,502\n",
      "INFO\t2017-01-12 16:06:35\tGenotypeConcordance\tchecked    20,510,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 10:101,550,981\n",
      "INFO\t2017-01-12 16:06:36\tGenotypeConcordance\tchecked    20,520,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 10:102,454,375\n",
      "INFO\t2017-01-12 16:06:36\tGenotypeConcordance\tchecked    20,530,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 10:103,536,044\n",
      "INFO\t2017-01-12 16:06:36\tGenotypeConcordance\tchecked    20,540,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 10:104,765,420\n",
      "INFO\t2017-01-12 16:06:36\tGenotypeConcordance\tchecked    20,550,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 10:105,816,331\n",
      "INFO\t2017-01-12 16:06:37\tGenotypeConcordance\tchecked    20,560,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 10:106,773,878\n",
      "INFO\t2017-01-12 16:06:37\tGenotypeConcordance\tchecked    20,570,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 10:107,683,576\n",
      "INFO\t2017-01-12 16:06:37\tGenotypeConcordance\tchecked    20,580,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 10:108,541,839\n",
      "INFO\t2017-01-12 16:06:38\tGenotypeConcordance\tchecked    20,590,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 10:109,395,501\n",
      "INFO\t2017-01-12 16:06:38\tGenotypeConcordance\tchecked    20,600,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 10:110,150,313\n",
      "INFO\t2017-01-12 16:06:38\tGenotypeConcordance\tchecked    20,610,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 10:110,892,470\n",
      "INFO\t2017-01-12 16:06:38\tGenotypeConcordance\tchecked    20,620,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 10:111,688,998\n",
      "INFO\t2017-01-12 16:06:39\tGenotypeConcordance\tchecked    20,630,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 10:112,566,850\n",
      "INFO\t2017-01-12 16:06:39\tGenotypeConcordance\tchecked    20,640,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 10:113,370,069\n",
      "INFO\t2017-01-12 16:06:39\tGenotypeConcordance\tchecked    20,650,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 10:114,153,072\n",
      "INFO\t2017-01-12 16:06:40\tGenotypeConcordance\tchecked    20,660,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 10:115,110,707\n",
      "INFO\t2017-01-12 16:06:40\tGenotypeConcordance\tchecked    20,670,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 10:115,958,189\n",
      "INFO\t2017-01-12 16:06:40\tGenotypeConcordance\tchecked    20,680,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 10:116,837,002\n",
      "INFO\t2017-01-12 16:06:40\tGenotypeConcordance\tchecked    20,690,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 10:117,678,091\n",
      "INFO\t2017-01-12 16:06:41\tGenotypeConcordance\tchecked    20,700,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 10:118,540,025\n",
      "INFO\t2017-01-12 16:06:41\tGenotypeConcordance\tchecked    20,710,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 10:119,483,071\n",
      "INFO\t2017-01-12 16:06:41\tGenotypeConcordance\tchecked    20,720,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 10:120,415,610\n",
      "INFO\t2017-01-12 16:06:42\tGenotypeConcordance\tchecked    20,730,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 10:121,225,680\n",
      "INFO\t2017-01-12 16:06:42\tGenotypeConcordance\tchecked    20,740,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 10:122,033,585\n",
      "INFO\t2017-01-12 16:06:42\tGenotypeConcordance\tchecked    20,750,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 10:122,813,199\n",
      "INFO\t2017-01-12 16:06:43\tGenotypeConcordance\tchecked    20,760,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 10:123,582,994\n",
      "INFO\t2017-01-12 16:06:43\tGenotypeConcordance\tchecked    20,770,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 10:124,379,794\n",
      "INFO\t2017-01-12 16:06:43\tGenotypeConcordance\tchecked    20,780,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 10:125,264,951\n",
      "INFO\t2017-01-12 16:06:43\tGenotypeConcordance\tchecked    20,790,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 10:126,042,780\n",
      "INFO\t2017-01-12 16:06:44\tGenotypeConcordance\tchecked    20,800,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 10:126,795,646\n",
      "INFO\t2017-01-12 16:06:44\tGenotypeConcordance\tchecked    20,810,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 10:127,594,582\n",
      "INFO\t2017-01-12 16:06:44\tGenotypeConcordance\tchecked    20,820,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 10:128,338,095\n",
      "INFO\t2017-01-12 16:06:45\tGenotypeConcordance\tchecked    20,830,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 10:129,233,829\n",
      "INFO\t2017-01-12 16:06:45\tGenotypeConcordance\tchecked    20,840,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 10:129,926,565\n",
      "INFO\t2017-01-12 16:06:45\tGenotypeConcordance\tchecked    20,850,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 10:130,678,444\n",
      "INFO\t2017-01-12 16:06:45\tGenotypeConcordance\tchecked    20,860,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 10:131,414,478\n",
      "INFO\t2017-01-12 16:06:46\tGenotypeConcordance\tchecked    20,870,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 10:132,193,733\n",
      "INFO\t2017-01-12 16:06:46\tGenotypeConcordance\tchecked    20,880,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 10:132,832,275\n",
      "INFO\t2017-01-12 16:06:46\tGenotypeConcordance\tchecked    20,890,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 10:133,508,735\n",
      "INFO\t2017-01-12 16:06:47\tGenotypeConcordance\tchecked    20,900,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 10:134,177,217\n",
      "INFO\t2017-01-12 16:06:47\tGenotypeConcordance\tchecked    20,910,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 10:134,818,963\n",
      "INFO\t2017-01-12 16:06:47\tGenotypeConcordance\tchecked    20,920,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 10:135,444,061\n",
      "INFO\t2017-01-12 16:06:48\tGenotypeConcordance\tchecked    20,930,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 11:624,995\n",
      "INFO\t2017-01-12 16:06:48\tGenotypeConcordance\tchecked    20,940,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 11:1,344,447\n",
      "INFO\t2017-01-12 16:06:48\tGenotypeConcordance\tchecked    20,950,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 11:2,026,711\n",
      "INFO\t2017-01-12 16:06:48\tGenotypeConcordance\tchecked    20,960,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 11:2,748,347\n",
      "INFO\t2017-01-12 16:06:49\tGenotypeConcordance\tchecked    20,970,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 11:3,467,305\n",
      "INFO\t2017-01-12 16:06:49\tGenotypeConcordance\tchecked    20,980,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 11:4,122,736\n",
      "INFO\t2017-01-12 16:06:49\tGenotypeConcordance\tchecked    20,990,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 11:4,587,367\n",
      "INFO\t2017-01-12 16:06:50\tGenotypeConcordance\tchecked    21,000,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 11:5,234,024\n",
      "INFO\t2017-01-12 16:06:50\tGenotypeConcordance\tchecked    21,010,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 11:5,826,843\n",
      "INFO\t2017-01-12 16:06:50\tGenotypeConcordance\tchecked    21,020,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 11:6,615,123\n",
      "INFO\t2017-01-12 16:06:51\tGenotypeConcordance\tchecked    21,030,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 11:7,390,425\n",
      "INFO\t2017-01-12 16:06:51\tGenotypeConcordance\tchecked    21,040,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 11:8,045,050\n",
      "INFO\t2017-01-12 16:06:51\tGenotypeConcordance\tchecked    21,050,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 11:8,861,669\n",
      "INFO\t2017-01-12 16:06:51\tGenotypeConcordance\tchecked    21,060,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 11:9,729,626\n",
      "INFO\t2017-01-12 16:06:52\tGenotypeConcordance\tchecked    21,070,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 11:10,567,802\n",
      "INFO\t2017-01-12 16:06:52\tGenotypeConcordance\tchecked    21,080,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 11:11,327,295\n",
      "INFO\t2017-01-12 16:06:52\tGenotypeConcordance\tchecked    21,090,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 11:12,127,181\n",
      "INFO\t2017-01-12 16:06:53\tGenotypeConcordance\tchecked    21,100,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 11:12,931,825\n",
      "INFO\t2017-01-12 16:06:53\tGenotypeConcordance\tchecked    21,110,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 11:13,771,997\n",
      "INFO\t2017-01-12 16:06:53\tGenotypeConcordance\tchecked    21,120,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 11:14,772,334\n",
      "INFO\t2017-01-12 16:06:54\tGenotypeConcordance\tchecked    21,130,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 11:15,733,176\n",
      "INFO\t2017-01-12 16:06:54\tGenotypeConcordance\tchecked    21,140,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 11:16,814,479\n",
      "INFO\t2017-01-12 16:06:54\tGenotypeConcordance\tchecked    21,150,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 11:17,679,578\n",
      "INFO\t2017-01-12 16:06:54\tGenotypeConcordance\tchecked    21,160,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 11:18,604,396\n",
      "INFO\t2017-01-12 16:06:55\tGenotypeConcordance\tchecked    21,170,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 11:19,418,971\n",
      "INFO\t2017-01-12 16:06:55\tGenotypeConcordance\tchecked    21,180,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 11:20,193,304\n",
      "INFO\t2017-01-12 16:06:55\tGenotypeConcordance\tchecked    21,190,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 11:20,971,402\n",
      "INFO\t2017-01-12 16:06:56\tGenotypeConcordance\tchecked    21,200,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 11:21,694,637\n",
      "INFO\t2017-01-12 16:06:56\tGenotypeConcordance\tchecked    21,210,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 11:22,426,734\n",
      "INFO\t2017-01-12 16:06:56\tGenotypeConcordance\tchecked    21,220,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 11:23,165,535\n",
      "INFO\t2017-01-12 16:06:56\tGenotypeConcordance\tchecked    21,230,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 11:23,854,110\n",
      "INFO\t2017-01-12 16:06:57\tGenotypeConcordance\tchecked    21,240,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 11:24,528,308\n",
      "INFO\t2017-01-12 16:06:57\tGenotypeConcordance\tchecked    21,250,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 11:25,194,744\n",
      "INFO\t2017-01-12 16:06:57\tGenotypeConcordance\tchecked    21,260,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 11:25,850,138\n",
      "INFO\t2017-01-12 16:06:58\tGenotypeConcordance\tchecked    21,270,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 11:26,530,867\n",
      "INFO\t2017-01-12 16:06:58\tGenotypeConcordance\tchecked    21,280,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 11:27,426,514\n",
      "INFO\t2017-01-12 16:06:58\tGenotypeConcordance\tchecked    21,290,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 11:28,639,341\n",
      "INFO\t2017-01-12 16:06:59\tGenotypeConcordance\tchecked    21,300,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 11:29,531,037\n",
      "INFO\t2017-01-12 16:06:59\tGenotypeConcordance\tchecked    21,310,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 11:30,400,212\n",
      "INFO\t2017-01-12 16:06:59\tGenotypeConcordance\tchecked    21,320,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 11:31,531,318\n",
      "INFO\t2017-01-12 16:06:59\tGenotypeConcordance\tchecked    21,330,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 11:32,485,030\n",
      "INFO\t2017-01-12 16:07:00\tGenotypeConcordance\tchecked    21,340,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 11:33,451,507\n",
      "INFO\t2017-01-12 16:07:00\tGenotypeConcordance\tchecked    21,350,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 11:34,306,865\n",
      "INFO\t2017-01-12 16:07:00\tGenotypeConcordance\tchecked    21,360,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 11:35,143,574\n",
      "INFO\t2017-01-12 16:07:01\tGenotypeConcordance\tchecked    21,370,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 11:36,008,210\n",
      "INFO\t2017-01-12 16:07:01\tGenotypeConcordance\tchecked    21,380,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 11:36,806,048\n",
      "INFO\t2017-01-12 16:07:01\tGenotypeConcordance\tchecked    21,390,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 11:37,632,706\n",
      "INFO\t2017-01-12 16:07:01\tGenotypeConcordance\tchecked    21,400,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 11:38,411,042\n",
      "INFO\t2017-01-12 16:07:02\tGenotypeConcordance\tchecked    21,410,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 11:39,225,437\n",
      "INFO\t2017-01-12 16:07:02\tGenotypeConcordance\tchecked    21,420,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 11:39,957,989\n",
      "INFO\t2017-01-12 16:07:02\tGenotypeConcordance\tchecked    21,430,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 11:40,812,047\n",
      "INFO\t2017-01-12 16:07:02\tGenotypeConcordance\tchecked    21,440,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 11:41,734,059\n",
      "INFO\t2017-01-12 16:07:03\tGenotypeConcordance\tchecked    21,450,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 11:42,461,347\n",
      "INFO\t2017-01-12 16:07:03\tGenotypeConcordance\tchecked    21,460,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 11:43,167,637\n",
      "INFO\t2017-01-12 16:07:03\tGenotypeConcordance\tchecked    21,470,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 11:44,075,440\n",
      "INFO\t2017-01-12 16:07:04\tGenotypeConcordance\tchecked    21,480,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 11:44,906,045\n",
      "INFO\t2017-01-12 16:07:04\tGenotypeConcordance\tchecked    21,490,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 11:45,781,007\n",
      "INFO\t2017-01-12 16:07:04\tGenotypeConcordance\tchecked    21,500,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 11:46,960,059\n",
      "INFO\t2017-01-12 16:07:04\tGenotypeConcordance\tchecked    21,510,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 11:48,082,064\n",
      "INFO\t2017-01-12 16:07:05\tGenotypeConcordance\tchecked    21,520,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 11:48,625,640\n",
      "INFO\t2017-01-12 16:07:05\tGenotypeConcordance\tchecked    21,530,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 11:49,199,515\n",
      "INFO\t2017-01-12 16:07:05\tGenotypeConcordance\tchecked    21,540,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 11:50,022,940\n",
      "INFO\t2017-01-12 16:07:06\tGenotypeConcordance\tchecked    21,550,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 11:50,640,670\n",
      "INFO\t2017-01-12 16:07:06\tGenotypeConcordance\tchecked    21,560,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 11:51,409,683\n",
      "INFO\t2017-01-12 16:07:06\tGenotypeConcordance\tchecked    21,570,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 11:54,978,599\n",
      "INFO\t2017-01-12 16:07:07\tGenotypeConcordance\tchecked    21,580,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 11:55,451,187\n",
      "INFO\t2017-01-12 16:07:07\tGenotypeConcordance\tchecked    21,590,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 11:56,124,183\n",
      "INFO\t2017-01-12 16:07:07\tGenotypeConcordance\tchecked    21,600,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 11:56,844,128\n",
      "INFO\t2017-01-12 16:07:07\tGenotypeConcordance\tchecked    21,610,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 11:57,873,300\n",
      "INFO\t2017-01-12 16:07:08\tGenotypeConcordance\tchecked    21,620,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 11:58,771,287\n",
      "INFO\t2017-01-12 16:07:08\tGenotypeConcordance\tchecked    21,630,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 11:59,659,528\n",
      "INFO\t2017-01-12 16:07:08\tGenotypeConcordance\tchecked    21,640,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 11:60,481,211\n",
      "INFO\t2017-01-12 16:07:08\tGenotypeConcordance\tchecked    21,650,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 11:61,456,576\n",
      "INFO\t2017-01-12 16:07:09\tGenotypeConcordance\tchecked    21,660,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 11:62,329,292\n",
      "INFO\t2017-01-12 16:07:09\tGenotypeConcordance\tchecked    21,670,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 11:63,182,615\n",
      "INFO\t2017-01-12 16:07:09\tGenotypeConcordance\tchecked    21,680,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 11:64,147,001\n",
      "INFO\t2017-01-12 16:07:10\tGenotypeConcordance\tchecked    21,690,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 11:65,167,904\n",
      "INFO\t2017-01-12 16:07:10\tGenotypeConcordance\tchecked    21,700,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 11:66,202,257\n",
      "INFO\t2017-01-12 16:07:10\tGenotypeConcordance\tchecked    21,710,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 11:67,375,162\n",
      "INFO\t2017-01-12 16:07:10\tGenotypeConcordance\tchecked    21,720,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 11:67,977,596\n",
      "INFO\t2017-01-12 16:07:11\tGenotypeConcordance\tchecked    21,730,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 11:68,825,475\n",
      "INFO\t2017-01-12 16:07:11\tGenotypeConcordance\tchecked    21,740,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 11:69,611,475\n",
      "INFO\t2017-01-12 16:07:11\tGenotypeConcordance\tchecked    21,750,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 11:70,426,352\n",
      "INFO\t2017-01-12 16:07:11\tGenotypeConcordance\tchecked    21,760,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 11:71,175,220\n",
      "INFO\t2017-01-12 16:07:12\tGenotypeConcordance\tchecked    21,770,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 11:71,772,465\n",
      "INFO\t2017-01-12 16:07:12\tGenotypeConcordance\tchecked    21,780,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 11:72,817,255\n",
      "INFO\t2017-01-12 16:07:12\tGenotypeConcordance\tchecked    21,790,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 11:73,735,375\n",
      "INFO\t2017-01-12 16:07:12\tGenotypeConcordance\tchecked    21,800,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 11:74,681,953\n",
      "INFO\t2017-01-12 16:07:13\tGenotypeConcordance\tchecked    21,810,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 11:75,672,886\n",
      "INFO\t2017-01-12 16:07:13\tGenotypeConcordance\tchecked    21,820,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 11:76,541,459\n",
      "INFO\t2017-01-12 16:07:13\tGenotypeConcordance\tchecked    21,830,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 11:77,481,797\n",
      "INFO\t2017-01-12 16:07:14\tGenotypeConcordance\tchecked    21,840,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 11:78,304,659\n",
      "INFO\t2017-01-12 16:07:14\tGenotypeConcordance\tchecked    21,850,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 11:79,144,077\n",
      "INFO\t2017-01-12 16:07:14\tGenotypeConcordance\tchecked    21,860,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 11:79,970,467\n",
      "INFO\t2017-01-12 16:07:14\tGenotypeConcordance\tchecked    21,870,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 11:80,775,922\n",
      "INFO\t2017-01-12 16:07:15\tGenotypeConcordance\tchecked    21,880,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 11:81,487,576\n",
      "INFO\t2017-01-12 16:07:15\tGenotypeConcordance\tchecked    21,890,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 11:82,314,055\n",
      "INFO\t2017-01-12 16:07:15\tGenotypeConcordance\tchecked    21,900,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 11:83,210,558\n",
      "INFO\t2017-01-12 16:07:16\tGenotypeConcordance\tchecked    21,910,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 11:84,029,214\n",
      "INFO\t2017-01-12 16:07:16\tGenotypeConcordance\tchecked    21,920,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 11:84,987,313\n",
      "INFO\t2017-01-12 16:07:16\tGenotypeConcordance\tchecked    21,930,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 11:85,902,035\n",
      "INFO\t2017-01-12 16:07:16\tGenotypeConcordance\tchecked    21,940,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 11:86,729,527\n",
      "INFO\t2017-01-12 16:07:17\tGenotypeConcordance\tchecked    21,950,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 11:87,474,301\n",
      "INFO\t2017-01-12 16:07:17\tGenotypeConcordance\tchecked    21,960,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 11:88,276,892\n",
      "INFO\t2017-01-12 16:07:17\tGenotypeConcordance\tchecked    21,970,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 11:89,087,643\n",
      "INFO\t2017-01-12 16:07:18\tGenotypeConcordance\tchecked    21,980,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 11:90,086,953\n",
      "INFO\t2017-01-12 16:07:18\tGenotypeConcordance\tchecked    21,990,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 11:90,839,732\n",
      "INFO\t2017-01-12 16:07:18\tGenotypeConcordance\tchecked    22,000,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 11:91,582,982\n",
      "INFO\t2017-01-12 16:07:18\tGenotypeConcordance\tchecked    22,010,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 11:92,434,078\n",
      "INFO\t2017-01-12 16:07:19\tGenotypeConcordance\tchecked    22,020,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 11:93,377,875\n",
      "INFO\t2017-01-12 16:07:19\tGenotypeConcordance\tchecked    22,030,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 11:94,158,534\n",
      "INFO\t2017-01-12 16:07:19\tGenotypeConcordance\tchecked    22,040,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 11:95,045,372\n",
      "INFO\t2017-01-12 16:07:20\tGenotypeConcordance\tchecked    22,050,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 11:95,923,127\n",
      "INFO\t2017-01-12 16:07:20\tGenotypeConcordance\tchecked    22,060,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 11:96,851,454\n",
      "INFO\t2017-01-12 16:07:20\tGenotypeConcordance\tchecked    22,070,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 11:97,651,857\n",
      "INFO\t2017-01-12 16:07:20\tGenotypeConcordance\tchecked    22,080,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 11:98,369,187\n",
      "INFO\t2017-01-12 16:07:21\tGenotypeConcordance\tchecked    22,090,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 11:99,042,908\n",
      "INFO\t2017-01-12 16:07:21\tGenotypeConcordance\tchecked    22,100,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 11:99,724,188\n",
      "INFO\t2017-01-12 16:07:21\tGenotypeConcordance\tchecked    22,110,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 11:100,476,938\n",
      "INFO\t2017-01-12 16:07:22\tGenotypeConcordance\tchecked    22,120,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 11:101,298,056\n",
      "INFO\t2017-01-12 16:07:22\tGenotypeConcordance\tchecked    22,130,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 11:102,112,044\n",
      "INFO\t2017-01-12 16:07:22\tGenotypeConcordance\tchecked    22,140,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 11:102,951,503\n",
      "INFO\t2017-01-12 16:07:22\tGenotypeConcordance\tchecked    22,150,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 11:103,766,176\n",
      "INFO\t2017-01-12 16:07:23\tGenotypeConcordance\tchecked    22,160,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 11:104,605,464\n",
      "INFO\t2017-01-12 16:07:23\tGenotypeConcordance\tchecked    22,170,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 11:105,453,453\n",
      "INFO\t2017-01-12 16:07:23\tGenotypeConcordance\tchecked    22,180,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 11:106,460,523\n",
      "INFO\t2017-01-12 16:07:24\tGenotypeConcordance\tchecked    22,190,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 11:107,338,209\n",
      "INFO\t2017-01-12 16:07:24\tGenotypeConcordance\tchecked    22,200,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 11:108,204,808\n",
      "INFO\t2017-01-12 16:07:24\tGenotypeConcordance\tchecked    22,210,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 11:109,098,928\n",
      "INFO\t2017-01-12 16:07:25\tGenotypeConcordance\tchecked    22,220,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 11:110,059,060\n",
      "INFO\t2017-01-12 16:07:25\tGenotypeConcordance\tchecked    22,230,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 11:110,934,323\n",
      "INFO\t2017-01-12 16:07:25\tGenotypeConcordance\tchecked    22,240,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 11:111,928,226\n",
      "INFO\t2017-01-12 16:07:25\tGenotypeConcordance\tchecked    22,250,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 11:112,789,916\n",
      "INFO\t2017-01-12 16:07:26\tGenotypeConcordance\tchecked    22,260,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 11:113,734,365\n",
      "INFO\t2017-01-12 16:07:26\tGenotypeConcordance\tchecked    22,270,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 11:114,509,142\n",
      "INFO\t2017-01-12 16:07:26\tGenotypeConcordance\tchecked    22,280,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 11:115,438,165\n",
      "INFO\t2017-01-12 16:07:27\tGenotypeConcordance\tchecked    22,290,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 11:116,307,582\n",
      "INFO\t2017-01-12 16:07:27\tGenotypeConcordance\tchecked    22,300,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 11:117,095,280\n",
      "INFO\t2017-01-12 16:07:27\tGenotypeConcordance\tchecked    22,310,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 11:117,920,100\n",
      "INFO\t2017-01-12 16:07:27\tGenotypeConcordance\tchecked    22,320,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 11:118,836,281\n",
      "INFO\t2017-01-12 16:07:28\tGenotypeConcordance\tchecked    22,330,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 11:119,664,304\n",
      "INFO\t2017-01-12 16:07:28\tGenotypeConcordance\tchecked    22,340,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 11:120,517,905\n",
      "INFO\t2017-01-12 16:07:28\tGenotypeConcordance\tchecked    22,350,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 11:121,416,589\n",
      "INFO\t2017-01-12 16:07:28\tGenotypeConcordance\tchecked    22,360,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 11:122,367,529\n",
      "INFO\t2017-01-12 16:07:29\tGenotypeConcordance\tchecked    22,370,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 11:123,124,838\n",
      "INFO\t2017-01-12 16:07:29\tGenotypeConcordance\tchecked    22,380,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 11:123,896,733\n",
      "INFO\t2017-01-12 16:07:29\tGenotypeConcordance\tchecked    22,390,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 11:124,713,023\n",
      "INFO\t2017-01-12 16:07:30\tGenotypeConcordance\tchecked    22,400,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 11:125,607,198\n",
      "INFO\t2017-01-12 16:07:30\tGenotypeConcordance\tchecked    22,410,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 11:126,448,571\n",
      "INFO\t2017-01-12 16:07:30\tGenotypeConcordance\tchecked    22,420,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 11:127,318,546\n",
      "INFO\t2017-01-12 16:07:30\tGenotypeConcordance\tchecked    22,430,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 11:128,178,450\n",
      "INFO\t2017-01-12 16:07:31\tGenotypeConcordance\tchecked    22,440,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 11:129,092,082\n",
      "INFO\t2017-01-12 16:07:31\tGenotypeConcordance\tchecked    22,450,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 11:129,895,088\n",
      "INFO\t2017-01-12 16:07:31\tGenotypeConcordance\tchecked    22,460,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 11:130,701,645\n",
      "INFO\t2017-01-12 16:07:32\tGenotypeConcordance\tchecked    22,470,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 11:131,435,048\n",
      "INFO\t2017-01-12 16:07:32\tGenotypeConcordance\tchecked    22,480,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 11:132,194,934\n",
      "INFO\t2017-01-12 16:07:32\tGenotypeConcordance\tchecked    22,490,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 11:132,985,714\n",
      "INFO\t2017-01-12 16:07:32\tGenotypeConcordance\tchecked    22,500,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 11:133,804,152\n",
      "INFO\t2017-01-12 16:07:33\tGenotypeConcordance\tchecked    22,510,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 11:134,541,267\n",
      "INFO\t2017-01-12 16:07:33\tGenotypeConcordance\tchecked    22,520,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 12:313,627\n",
      "INFO\t2017-01-12 16:07:33\tGenotypeConcordance\tchecked    22,530,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 12:1,097,096\n",
      "INFO\t2017-01-12 16:07:34\tGenotypeConcordance\tchecked    22,540,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 12:1,851,862\n",
      "INFO\t2017-01-12 16:07:34\tGenotypeConcordance\tchecked    22,550,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 12:2,680,852\n",
      "INFO\t2017-01-12 16:07:34\tGenotypeConcordance\tchecked    22,560,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 12:3,393,512\n",
      "INFO\t2017-01-12 16:07:34\tGenotypeConcordance\tchecked    22,570,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 12:4,206,830\n",
      "INFO\t2017-01-12 16:07:35\tGenotypeConcordance\tchecked    22,580,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 12:5,061,819\n",
      "INFO\t2017-01-12 16:07:35\tGenotypeConcordance\tchecked    22,590,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 12:5,919,979\n",
      "INFO\t2017-01-12 16:07:35\tGenotypeConcordance\tchecked    22,600,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 12:6,635,274\n",
      "INFO\t2017-01-12 16:07:36\tGenotypeConcordance\tchecked    22,610,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 12:7,572,005\n",
      "INFO\t2017-01-12 16:07:36\tGenotypeConcordance\tchecked    22,620,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 12:8,272,350\n",
      "INFO\t2017-01-12 16:07:36\tGenotypeConcordance\tchecked    22,630,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 12:8,851,166\n",
      "INFO\t2017-01-12 16:07:36\tGenotypeConcordance\tchecked    22,640,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 12:9,609,298\n",
      "INFO\t2017-01-12 16:07:37\tGenotypeConcordance\tchecked    22,650,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 12:10,398,906\n",
      "INFO\t2017-01-12 16:07:37\tGenotypeConcordance\tchecked    22,660,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 12:11,170,074\n",
      "INFO\t2017-01-12 16:07:37\tGenotypeConcordance\tchecked    22,670,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 12:11,717,489\n",
      "INFO\t2017-01-12 16:07:38\tGenotypeConcordance\tchecked    22,680,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 12:12,516,393\n",
      "INFO\t2017-01-12 16:07:38\tGenotypeConcordance\tchecked    22,690,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 12:13,370,829\n",
      "INFO\t2017-01-12 16:07:38\tGenotypeConcordance\tchecked    22,700,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 12:14,231,402\n",
      "INFO\t2017-01-12 16:07:38\tGenotypeConcordance\tchecked    22,710,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 12:15,091,702\n",
      "INFO\t2017-01-12 16:07:39\tGenotypeConcordance\tchecked    22,720,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 12:16,045,476\n",
      "INFO\t2017-01-12 16:07:39\tGenotypeConcordance\tchecked    22,730,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 12:17,041,838\n",
      "INFO\t2017-01-12 16:07:39\tGenotypeConcordance\tchecked    22,740,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 12:17,925,474\n",
      "INFO\t2017-01-12 16:07:39\tGenotypeConcordance\tchecked    22,750,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 12:18,742,458\n",
      "INFO\t2017-01-12 16:07:40\tGenotypeConcordance\tchecked    22,760,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 12:19,581,726\n",
      "INFO\t2017-01-12 16:07:40\tGenotypeConcordance\tchecked    22,770,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 12:20,343,474\n",
      "INFO\t2017-01-12 16:07:40\tGenotypeConcordance\tchecked    22,780,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 12:21,125,365\n",
      "INFO\t2017-01-12 16:07:41\tGenotypeConcordance\tchecked    22,790,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 12:21,909,730\n",
      "INFO\t2017-01-12 16:07:41\tGenotypeConcordance\tchecked    22,800,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 12:22,894,851\n",
      "INFO\t2017-01-12 16:07:41\tGenotypeConcordance\tchecked    22,810,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 12:23,803,265\n",
      "INFO\t2017-01-12 16:07:42\tGenotypeConcordance\tchecked    22,820,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 12:24,673,740\n",
      "INFO\t2017-01-12 16:07:42\tGenotypeConcordance\tchecked    22,830,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 12:25,517,238\n",
      "INFO\t2017-01-12 16:07:42\tGenotypeConcordance\tchecked    22,840,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 12:26,311,517\n",
      "INFO\t2017-01-12 16:07:42\tGenotypeConcordance\tchecked    22,850,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 12:27,206,030\n",
      "INFO\t2017-01-12 16:07:43\tGenotypeConcordance\tchecked    22,860,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 12:28,101,641\n",
      "INFO\t2017-01-12 16:07:43\tGenotypeConcordance\tchecked    22,870,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 12:29,041,646\n",
      "INFO\t2017-01-12 16:07:43\tGenotypeConcordance\tchecked    22,880,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 12:29,854,972\n",
      "INFO\t2017-01-12 16:07:44\tGenotypeConcordance\tchecked    22,890,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 12:30,534,912\n",
      "INFO\t2017-01-12 16:07:44\tGenotypeConcordance\tchecked    22,900,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 12:31,302,583\n",
      "INFO\t2017-01-12 16:07:44\tGenotypeConcordance\tchecked    22,910,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 12:32,062,348\n",
      "INFO\t2017-01-12 16:07:45\tGenotypeConcordance\tchecked    22,920,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 12:32,801,194\n",
      "INFO\t2017-01-12 16:07:45\tGenotypeConcordance\tchecked    22,930,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 12:33,534,368\n",
      "INFO\t2017-01-12 16:07:45\tGenotypeConcordance\tchecked    22,940,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 12:34,307,426\n",
      "INFO\t2017-01-12 16:07:45\tGenotypeConcordance\tchecked    22,950,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 12:34,791,956\n",
      "INFO\t2017-01-12 16:07:46\tGenotypeConcordance\tchecked    22,960,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 12:38,159,731\n",
      "INFO\t2017-01-12 16:07:46\tGenotypeConcordance\tchecked    22,970,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 12:38,595,450\n",
      "INFO\t2017-01-12 16:07:46\tGenotypeConcordance\tchecked    22,980,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 12:39,346,678\n",
      "INFO\t2017-01-12 16:07:46\tGenotypeConcordance\tchecked    22,990,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 12:40,200,736\n",
      "INFO\t2017-01-12 16:07:47\tGenotypeConcordance\tchecked    23,000,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 12:41,034,018\n",
      "INFO\t2017-01-12 16:07:47\tGenotypeConcordance\tchecked    23,010,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 12:41,844,645\n",
      "INFO\t2017-01-12 16:07:47\tGenotypeConcordance\tchecked    23,020,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 12:42,595,907\n",
      "INFO\t2017-01-12 16:07:48\tGenotypeConcordance\tchecked    23,030,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 12:43,439,383\n",
      "INFO\t2017-01-12 16:07:48\tGenotypeConcordance\tchecked    23,040,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 12:44,318,922\n",
      "INFO\t2017-01-12 16:07:48\tGenotypeConcordance\tchecked    23,050,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 12:45,220,052\n",
      "INFO\t2017-01-12 16:07:49\tGenotypeConcordance\tchecked    23,060,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 12:46,088,790\n",
      "INFO\t2017-01-12 16:07:49\tGenotypeConcordance\tchecked    23,070,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 12:47,018,302\n",
      "INFO\t2017-01-12 16:07:49\tGenotypeConcordance\tchecked    23,080,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 12:47,829,588\n",
      "INFO\t2017-01-12 16:07:49\tGenotypeConcordance\tchecked    23,090,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 12:48,658,456\n",
      "INFO\t2017-01-12 16:07:50\tGenotypeConcordance\tchecked    23,100,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 12:49,640,962\n",
      "INFO\t2017-01-12 16:07:50\tGenotypeConcordance\tchecked    23,110,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 12:50,652,083\n",
      "INFO\t2017-01-12 16:07:50\tGenotypeConcordance\tchecked    23,120,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 12:51,659,071\n",
      "INFO\t2017-01-12 16:07:50\tGenotypeConcordance\tchecked    23,130,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 12:52,602,013\n",
      "INFO\t2017-01-12 16:07:51\tGenotypeConcordance\tchecked    23,140,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 12:53,331,970\n",
      "INFO\t2017-01-12 16:07:51\tGenotypeConcordance\tchecked    23,150,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 12:54,302,849\n",
      "INFO\t2017-01-12 16:07:51\tGenotypeConcordance\tchecked    23,160,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 12:55,219,254\n",
      "INFO\t2017-01-12 16:07:51\tGenotypeConcordance\tchecked    23,170,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 12:55,933,792\n",
      "INFO\t2017-01-12 16:07:52\tGenotypeConcordance\tchecked    23,180,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 12:56,962,625\n",
      "INFO\t2017-01-12 16:07:52\tGenotypeConcordance\tchecked    23,190,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 12:58,042,281\n",
      "INFO\t2017-01-12 16:07:52\tGenotypeConcordance\tchecked    23,200,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 12:58,952,712\n",
      "INFO\t2017-01-12 16:07:53\tGenotypeConcordance\tchecked    23,210,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 12:59,842,989\n",
      "INFO\t2017-01-12 16:07:53\tGenotypeConcordance\tchecked    23,220,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 12:60,725,868\n",
      "INFO\t2017-01-12 16:07:53\tGenotypeConcordance\tchecked    23,230,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 12:61,498,477\n",
      "INFO\t2017-01-12 16:07:53\tGenotypeConcordance\tchecked    23,240,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 12:62,310,217\n",
      "INFO\t2017-01-12 16:07:54\tGenotypeConcordance\tchecked    23,250,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 12:63,265,984\n",
      "INFO\t2017-01-12 16:07:54\tGenotypeConcordance\tchecked    23,260,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 12:64,067,829\n",
      "INFO\t2017-01-12 16:07:54\tGenotypeConcordance\tchecked    23,270,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 12:64,998,755\n",
      "INFO\t2017-01-12 16:07:54\tGenotypeConcordance\tchecked    23,280,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 12:66,061,422\n",
      "INFO\t2017-01-12 16:07:55\tGenotypeConcordance\tchecked    23,290,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 12:66,983,046\n",
      "INFO\t2017-01-12 16:07:55\tGenotypeConcordance\tchecked    23,300,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 12:67,878,789\n",
      "INFO\t2017-01-12 16:07:55\tGenotypeConcordance\tchecked    23,310,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 12:68,778,009\n",
      "INFO\t2017-01-12 16:07:56\tGenotypeConcordance\tchecked    23,320,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 12:69,634,706\n",
      "INFO\t2017-01-12 16:07:56\tGenotypeConcordance\tchecked    23,330,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 12:70,481,733\n",
      "INFO\t2017-01-12 16:07:56\tGenotypeConcordance\tchecked    23,340,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 12:71,351,875\n",
      "INFO\t2017-01-12 16:07:56\tGenotypeConcordance\tchecked    23,350,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 12:72,245,731\n",
      "INFO\t2017-01-12 16:07:57\tGenotypeConcordance\tchecked    23,360,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 12:73,218,766\n",
      "INFO\t2017-01-12 16:07:57\tGenotypeConcordance\tchecked    23,370,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 12:74,069,440\n",
      "INFO\t2017-01-12 16:07:57\tGenotypeConcordance\tchecked    23,380,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 12:74,989,531\n",
      "INFO\t2017-01-12 16:07:57\tGenotypeConcordance\tchecked    23,390,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 12:75,880,173\n",
      "INFO\t2017-01-12 16:07:58\tGenotypeConcordance\tchecked    23,400,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 12:76,720,599\n",
      "INFO\t2017-01-12 16:07:58\tGenotypeConcordance\tchecked    23,410,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 12:77,652,904\n",
      "INFO\t2017-01-12 16:07:58\tGenotypeConcordance\tchecked    23,420,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 12:78,531,976\n",
      "INFO\t2017-01-12 16:07:58\tGenotypeConcordance\tchecked    23,430,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 12:79,403,944\n",
      "INFO\t2017-01-12 16:07:59\tGenotypeConcordance\tchecked    23,440,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 12:80,478,326\n",
      "INFO\t2017-01-12 16:07:59\tGenotypeConcordance\tchecked    23,450,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 12:81,409,164\n",
      "INFO\t2017-01-12 16:07:59\tGenotypeConcordance\tchecked    23,460,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 12:82,386,288\n",
      "INFO\t2017-01-12 16:07:59\tGenotypeConcordance\tchecked    23,470,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 12:83,310,584\n",
      "INFO\t2017-01-12 16:08:00\tGenotypeConcordance\tchecked    23,480,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 12:84,095,819\n",
      "INFO\t2017-01-12 16:08:00\tGenotypeConcordance\tchecked    23,490,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 12:84,945,384\n",
      "INFO\t2017-01-12 16:08:00\tGenotypeConcordance\tchecked    23,500,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 12:85,972,453\n",
      "INFO\t2017-01-12 16:08:00\tGenotypeConcordance\tchecked    23,510,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 12:86,849,237\n",
      "INFO\t2017-01-12 16:08:01\tGenotypeConcordance\tchecked    23,520,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 12:87,742,759\n",
      "INFO\t2017-01-12 16:08:01\tGenotypeConcordance\tchecked    23,530,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 12:88,774,181\n",
      "INFO\t2017-01-12 16:08:01\tGenotypeConcordance\tchecked    23,540,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 12:89,904,596\n",
      "INFO\t2017-01-12 16:08:01\tGenotypeConcordance\tchecked    23,550,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 12:90,989,605\n",
      "INFO\t2017-01-12 16:08:02\tGenotypeConcordance\tchecked    23,560,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 12:91,964,219\n",
      "INFO\t2017-01-12 16:08:02\tGenotypeConcordance\tchecked    23,570,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 12:92,862,368\n",
      "INFO\t2017-01-12 16:08:02\tGenotypeConcordance\tchecked    23,580,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 12:93,735,865\n",
      "INFO\t2017-01-12 16:08:03\tGenotypeConcordance\tchecked    23,590,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 12:94,580,465\n",
      "INFO\t2017-01-12 16:08:03\tGenotypeConcordance\tchecked    23,600,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 12:95,398,440\n",
      "INFO\t2017-01-12 16:08:03\tGenotypeConcordance\tchecked    23,610,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 12:96,216,288\n",
      "INFO\t2017-01-12 16:08:03\tGenotypeConcordance\tchecked    23,620,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 12:97,086,351\n",
      "INFO\t2017-01-12 16:08:04\tGenotypeConcordance\tchecked    23,630,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 12:97,956,918\n",
      "INFO\t2017-01-12 16:08:04\tGenotypeConcordance\tchecked    23,640,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 12:98,835,746\n",
      "INFO\t2017-01-12 16:08:04\tGenotypeConcordance\tchecked    23,650,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 12:99,799,739\n",
      "INFO\t2017-01-12 16:08:04\tGenotypeConcordance\tchecked    23,660,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 12:100,671,902\n",
      "INFO\t2017-01-12 16:08:05\tGenotypeConcordance\tchecked    23,670,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 12:101,581,368\n",
      "INFO\t2017-01-12 16:08:05\tGenotypeConcordance\tchecked    23,680,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 12:102,429,347\n",
      "INFO\t2017-01-12 16:08:05\tGenotypeConcordance\tchecked    23,690,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 12:103,419,959\n",
      "INFO\t2017-01-12 16:08:05\tGenotypeConcordance\tchecked    23,700,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 12:104,310,945\n",
      "INFO\t2017-01-12 16:08:06\tGenotypeConcordance\tchecked    23,710,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 12:105,172,880\n",
      "INFO\t2017-01-12 16:08:06\tGenotypeConcordance\tchecked    23,720,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 12:106,030,833\n",
      "INFO\t2017-01-12 16:08:06\tGenotypeConcordance\tchecked    23,730,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 12:106,871,530\n",
      "INFO\t2017-01-12 16:08:06\tGenotypeConcordance\tchecked    23,740,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 12:107,859,928\n",
      "INFO\t2017-01-12 16:08:07\tGenotypeConcordance\tchecked    23,750,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 12:108,704,733\n",
      "INFO\t2017-01-12 16:08:07\tGenotypeConcordance\tchecked    23,760,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 12:109,650,050\n",
      "INFO\t2017-01-12 16:08:07\tGenotypeConcordance\tchecked    23,770,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 12:110,582,833\n",
      "INFO\t2017-01-12 16:08:07\tGenotypeConcordance\tchecked    23,780,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 12:111,636,580\n",
      "INFO\t2017-01-12 16:08:08\tGenotypeConcordance\tchecked    23,790,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 12:112,926,772\n",
      "INFO\t2017-01-12 16:08:08\tGenotypeConcordance\tchecked    23,800,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 12:113,898,522\n",
      "INFO\t2017-01-12 16:08:08\tGenotypeConcordance\tchecked    23,810,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 12:114,655,737\n",
      "INFO\t2017-01-12 16:08:09\tGenotypeConcordance\tchecked    23,820,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 12:115,410,225\n",
      "INFO\t2017-01-12 16:08:09\tGenotypeConcordance\tchecked    23,830,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 12:116,185,637\n",
      "INFO\t2017-01-12 16:08:09\tGenotypeConcordance\tchecked    23,840,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 12:117,089,737\n",
      "INFO\t2017-01-12 16:08:09\tGenotypeConcordance\tchecked    23,850,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 12:117,926,094\n",
      "INFO\t2017-01-12 16:08:10\tGenotypeConcordance\tchecked    23,860,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 12:118,752,878\n",
      "INFO\t2017-01-12 16:08:10\tGenotypeConcordance\tchecked    23,870,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 12:119,577,532\n",
      "INFO\t2017-01-12 16:08:10\tGenotypeConcordance\tchecked    23,880,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 12:120,492,514\n",
      "INFO\t2017-01-12 16:08:10\tGenotypeConcordance\tchecked    23,890,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 12:121,395,940\n",
      "INFO\t2017-01-12 16:08:11\tGenotypeConcordance\tchecked    23,900,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 12:122,225,369\n",
      "INFO\t2017-01-12 16:08:11\tGenotypeConcordance\tchecked    23,910,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 12:123,183,113\n",
      "INFO\t2017-01-12 16:08:11\tGenotypeConcordance\tchecked    23,920,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 12:124,154,840\n",
      "INFO\t2017-01-12 16:08:12\tGenotypeConcordance\tchecked    23,930,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 12:124,965,596\n",
      "INFO\t2017-01-12 16:08:12\tGenotypeConcordance\tchecked    23,940,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 12:125,679,919\n",
      "INFO\t2017-01-12 16:08:12\tGenotypeConcordance\tchecked    23,950,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 12:126,466,142\n",
      "INFO\t2017-01-12 16:08:12\tGenotypeConcordance\tchecked    23,960,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 12:127,177,853\n",
      "INFO\t2017-01-12 16:08:13\tGenotypeConcordance\tchecked    23,970,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 12:127,868,209\n",
      "INFO\t2017-01-12 16:08:13\tGenotypeConcordance\tchecked    23,980,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 12:128,567,404\n",
      "INFO\t2017-01-12 16:08:13\tGenotypeConcordance\tchecked    23,990,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 12:129,219,398\n",
      "INFO\t2017-01-12 16:08:14\tGenotypeConcordance\tchecked    24,000,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 12:129,879,302\n",
      "INFO\t2017-01-12 16:08:14\tGenotypeConcordance\tchecked    24,010,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 12:130,567,576\n",
      "INFO\t2017-01-12 16:08:14\tGenotypeConcordance\tchecked    24,020,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 12:131,228,645\n",
      "INFO\t2017-01-12 16:08:14\tGenotypeConcordance\tchecked    24,030,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 12:131,861,871\n",
      "INFO\t2017-01-12 16:08:15\tGenotypeConcordance\tchecked    24,040,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 12:132,604,096\n",
      "INFO\t2017-01-12 16:08:15\tGenotypeConcordance\tchecked    24,050,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 12:133,251,435\n",
      "INFO\t2017-01-12 16:08:15\tGenotypeConcordance\tchecked    24,060,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 13:19,304,599\n",
      "INFO\t2017-01-12 16:08:15\tGenotypeConcordance\tchecked    24,070,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 13:19,762,694\n",
      "INFO\t2017-01-12 16:08:16\tGenotypeConcordance\tchecked    24,080,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 13:20,474,044\n",
      "INFO\t2017-01-12 16:08:16\tGenotypeConcordance\tchecked    24,090,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 13:21,260,126\n",
      "INFO\t2017-01-12 16:08:16\tGenotypeConcordance\tchecked    24,100,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 13:22,023,707\n",
      "INFO\t2017-01-12 16:08:17\tGenotypeConcordance\tchecked    24,110,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 13:22,778,689\n",
      "INFO\t2017-01-12 16:08:17\tGenotypeConcordance\tchecked    24,120,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 13:23,490,255\n",
      "INFO\t2017-01-12 16:08:17\tGenotypeConcordance\tchecked    24,130,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 13:24,110,002\n",
      "INFO\t2017-01-12 16:08:18\tGenotypeConcordance\tchecked    24,140,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 13:24,796,423\n",
      "INFO\t2017-01-12 16:08:18\tGenotypeConcordance\tchecked    24,150,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 13:25,465,752\n",
      "INFO\t2017-01-12 16:08:18\tGenotypeConcordance\tchecked    24,160,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 13:26,218,409\n",
      "INFO\t2017-01-12 16:08:18\tGenotypeConcordance\tchecked    24,170,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 13:26,976,591\n",
      "INFO\t2017-01-12 16:08:19\tGenotypeConcordance\tchecked    24,180,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 13:27,717,165\n",
      "INFO\t2017-01-12 16:08:19\tGenotypeConcordance\tchecked    24,190,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 13:28,488,883\n",
      "INFO\t2017-01-12 16:08:19\tGenotypeConcordance\tchecked    24,200,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 13:29,306,662\n",
      "INFO\t2017-01-12 16:08:19\tGenotypeConcordance\tchecked    24,210,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 13:30,084,970\n",
      "INFO\t2017-01-12 16:08:20\tGenotypeConcordance\tchecked    24,220,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 13:30,871,177\n",
      "INFO\t2017-01-12 16:08:20\tGenotypeConcordance\tchecked    24,230,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 13:31,714,045\n",
      "INFO\t2017-01-12 16:08:20\tGenotypeConcordance\tchecked    24,240,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 13:32,533,595\n",
      "INFO\t2017-01-12 16:08:21\tGenotypeConcordance\tchecked    24,250,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 13:33,477,112\n",
      "INFO\t2017-01-12 16:08:21\tGenotypeConcordance\tchecked    24,260,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 13:34,290,654\n",
      "INFO\t2017-01-12 16:08:21\tGenotypeConcordance\tchecked    24,270,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 13:35,142,002\n",
      "INFO\t2017-01-12 16:08:21\tGenotypeConcordance\tchecked    24,280,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 13:36,131,705\n",
      "INFO\t2017-01-12 16:08:22\tGenotypeConcordance\tchecked    24,290,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 13:36,999,746\n",
      "INFO\t2017-01-12 16:08:22\tGenotypeConcordance\tchecked    24,300,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 13:37,905,116\n",
      "INFO\t2017-01-12 16:08:22\tGenotypeConcordance\tchecked    24,310,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 13:38,687,006\n",
      "INFO\t2017-01-12 16:08:23\tGenotypeConcordance\tchecked    24,320,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 13:39,515,037\n",
      "INFO\t2017-01-12 16:08:23\tGenotypeConcordance\tchecked    24,330,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 13:40,312,048\n",
      "INFO\t2017-01-12 16:08:23\tGenotypeConcordance\tchecked    24,340,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 13:41,140,781\n",
      "INFO\t2017-01-12 16:08:23\tGenotypeConcordance\tchecked    24,350,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 13:42,029,712\n",
      "INFO\t2017-01-12 16:08:24\tGenotypeConcordance\tchecked    24,360,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 13:42,911,957\n",
      "INFO\t2017-01-12 16:08:24\tGenotypeConcordance\tchecked    24,370,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 13:43,729,296\n",
      "INFO\t2017-01-12 16:08:24\tGenotypeConcordance\tchecked    24,380,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 13:44,669,703\n",
      "INFO\t2017-01-12 16:08:24\tGenotypeConcordance\tchecked    24,390,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 13:45,663,718\n",
      "INFO\t2017-01-12 16:08:25\tGenotypeConcordance\tchecked    24,400,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 13:46,510,533\n",
      "INFO\t2017-01-12 16:08:25\tGenotypeConcordance\tchecked    24,410,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 13:47,405,736\n",
      "INFO\t2017-01-12 16:08:25\tGenotypeConcordance\tchecked    24,420,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 13:48,262,266\n",
      "INFO\t2017-01-12 16:08:26\tGenotypeConcordance\tchecked    24,430,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 13:49,145,424\n",
      "INFO\t2017-01-12 16:08:26\tGenotypeConcordance\tchecked    24,440,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 13:50,088,357\n",
      "INFO\t2017-01-12 16:08:26\tGenotypeConcordance\tchecked    24,450,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 13:51,105,300\n",
      "INFO\t2017-01-12 16:08:26\tGenotypeConcordance\tchecked    24,460,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 13:52,065,039\n",
      "INFO\t2017-01-12 16:08:27\tGenotypeConcordance\tchecked    24,470,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 13:53,059,356\n",
      "INFO\t2017-01-12 16:08:27\tGenotypeConcordance\tchecked    24,480,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 13:53,973,902\n",
      "INFO\t2017-01-12 16:08:27\tGenotypeConcordance\tchecked    24,490,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 13:54,971,013\n",
      "INFO\t2017-01-12 16:08:27\tGenotypeConcordance\tchecked    24,500,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 13:55,867,116\n",
      "INFO\t2017-01-12 16:08:28\tGenotypeConcordance\tchecked    24,510,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 13:56,783,486\n",
      "INFO\t2017-01-12 16:08:28\tGenotypeConcordance\tchecked    24,520,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 13:57,999,028\n",
      "INFO\t2017-01-12 16:08:28\tGenotypeConcordance\tchecked    24,530,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 13:59,108,671\n",
      "INFO\t2017-01-12 16:08:29\tGenotypeConcordance\tchecked    24,540,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 13:60,016,892\n",
      "INFO\t2017-01-12 16:08:29\tGenotypeConcordance\tchecked    24,550,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 13:61,003,976\n",
      "INFO\t2017-01-12 16:08:29\tGenotypeConcordance\tchecked    24,560,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 13:61,802,659\n",
      "INFO\t2017-01-12 16:08:29\tGenotypeConcordance\tchecked    24,570,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 13:62,598,484\n",
      "INFO\t2017-01-12 16:08:30\tGenotypeConcordance\tchecked    24,580,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 13:63,452,736\n",
      "INFO\t2017-01-12 16:08:30\tGenotypeConcordance\tchecked    24,590,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 13:64,125,797\n",
      "INFO\t2017-01-12 16:08:30\tGenotypeConcordance\tchecked    24,600,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 13:64,878,263\n",
      "INFO\t2017-01-12 16:08:30\tGenotypeConcordance\tchecked    24,610,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 13:65,774,822\n",
      "INFO\t2017-01-12 16:08:31\tGenotypeConcordance\tchecked    24,620,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 13:66,668,526\n",
      "INFO\t2017-01-12 16:08:31\tGenotypeConcordance\tchecked    24,630,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 13:67,589,688\n",
      "INFO\t2017-01-12 16:08:31\tGenotypeConcordance\tchecked    24,640,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 13:68,476,463\n",
      "INFO\t2017-01-12 16:08:32\tGenotypeConcordance\tchecked    24,650,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 13:69,344,408\n",
      "INFO\t2017-01-12 16:08:32\tGenotypeConcordance\tchecked    24,660,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 13:70,123,444\n",
      "INFO\t2017-01-12 16:08:32\tGenotypeConcordance\tchecked    24,670,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 13:70,875,105\n",
      "INFO\t2017-01-12 16:08:32\tGenotypeConcordance\tchecked    24,680,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 13:71,646,397\n",
      "INFO\t2017-01-12 16:08:33\tGenotypeConcordance\tchecked    24,690,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 13:72,550,851\n",
      "INFO\t2017-01-12 16:08:33\tGenotypeConcordance\tchecked    24,700,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 13:73,365,211\n",
      "INFO\t2017-01-12 16:08:33\tGenotypeConcordance\tchecked    24,710,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 13:74,216,457\n",
      "INFO\t2017-01-12 16:08:34\tGenotypeConcordance\tchecked    24,720,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 13:75,062,769\n",
      "INFO\t2017-01-12 16:08:34\tGenotypeConcordance\tchecked    24,730,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 13:75,922,295\n",
      "INFO\t2017-01-12 16:08:34\tGenotypeConcordance\tchecked    24,740,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 13:76,749,475\n",
      "INFO\t2017-01-12 16:08:34\tGenotypeConcordance\tchecked    24,750,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 13:77,592,009\n",
      "INFO\t2017-01-12 16:08:35\tGenotypeConcordance\tchecked    24,760,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 13:78,602,863\n",
      "INFO\t2017-01-12 16:08:35\tGenotypeConcordance\tchecked    24,770,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 13:79,430,982\n",
      "INFO\t2017-01-12 16:08:35\tGenotypeConcordance\tchecked    24,780,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 13:80,380,090\n",
      "INFO\t2017-01-12 16:08:35\tGenotypeConcordance\tchecked    24,790,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 13:81,290,711\n",
      "INFO\t2017-01-12 16:08:36\tGenotypeConcordance\tchecked    24,800,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 13:82,174,716\n",
      "INFO\t2017-01-12 16:08:36\tGenotypeConcordance\tchecked    24,810,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 13:83,086,550\n",
      "INFO\t2017-01-12 16:08:36\tGenotypeConcordance\tchecked    24,820,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 13:83,903,522\n",
      "INFO\t2017-01-12 16:08:37\tGenotypeConcordance\tchecked    24,830,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 13:84,728,732\n",
      "INFO\t2017-01-12 16:08:37\tGenotypeConcordance\tchecked    24,840,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 13:85,558,486\n",
      "INFO\t2017-01-12 16:08:37\tGenotypeConcordance\tchecked    24,850,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 13:86,333,194\n",
      "INFO\t2017-01-12 16:08:37\tGenotypeConcordance\tchecked    24,860,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 13:87,268,660\n",
      "INFO\t2017-01-12 16:08:38\tGenotypeConcordance\tchecked    24,870,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 13:88,145,991\n",
      "INFO\t2017-01-12 16:08:38\tGenotypeConcordance\tchecked    24,880,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 13:89,056,014\n",
      "INFO\t2017-01-12 16:08:38\tGenotypeConcordance\tchecked    24,890,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 13:89,983,106\n",
      "INFO\t2017-01-12 16:08:39\tGenotypeConcordance\tchecked    24,900,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 13:90,769,532\n",
      "INFO\t2017-01-12 16:08:39\tGenotypeConcordance\tchecked    24,910,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 13:91,614,449\n",
      "INFO\t2017-01-12 16:08:39\tGenotypeConcordance\tchecked    24,920,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 13:92,416,361\n",
      "INFO\t2017-01-12 16:08:39\tGenotypeConcordance\tchecked    24,930,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 13:93,247,541\n",
      "INFO\t2017-01-12 16:08:40\tGenotypeConcordance\tchecked    24,940,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 13:94,106,204\n",
      "INFO\t2017-01-12 16:08:40\tGenotypeConcordance\tchecked    24,950,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 13:94,992,827\n",
      "INFO\t2017-01-12 16:08:40\tGenotypeConcordance\tchecked    24,960,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 13:95,842,518\n",
      "INFO\t2017-01-12 16:08:41\tGenotypeConcordance\tchecked    24,970,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 13:96,724,928\n",
      "INFO\t2017-01-12 16:08:41\tGenotypeConcordance\tchecked    24,980,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 13:97,744,284\n",
      "INFO\t2017-01-12 16:08:41\tGenotypeConcordance\tchecked    24,990,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 13:98,678,011\n",
      "INFO\t2017-01-12 16:08:41\tGenotypeConcordance\tchecked    25,000,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 13:99,392,176\n",
      "INFO\t2017-01-12 16:08:42\tGenotypeConcordance\tchecked    25,010,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 13:100,239,768\n",
      "INFO\t2017-01-12 16:08:42\tGenotypeConcordance\tchecked    25,020,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 13:101,123,315\n",
      "INFO\t2017-01-12 16:08:42\tGenotypeConcordance\tchecked    25,030,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 13:101,962,466\n",
      "INFO\t2017-01-12 16:08:42\tGenotypeConcordance\tchecked    25,040,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 13:102,781,655\n",
      "INFO\t2017-01-12 16:08:43\tGenotypeConcordance\tchecked    25,050,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 13:103,642,152\n",
      "INFO\t2017-01-12 16:08:43\tGenotypeConcordance\tchecked    25,060,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 13:104,384,890\n",
      "INFO\t2017-01-12 16:08:43\tGenotypeConcordance\tchecked    25,070,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 13:105,118,015\n",
      "INFO\t2017-01-12 16:08:43\tGenotypeConcordance\tchecked    25,080,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 13:105,846,909\n",
      "INFO\t2017-01-12 16:08:44\tGenotypeConcordance\tchecked    25,090,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 13:106,559,339\n",
      "INFO\t2017-01-12 16:08:44\tGenotypeConcordance\tchecked    25,100,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 13:107,386,589\n",
      "INFO\t2017-01-12 16:08:44\tGenotypeConcordance\tchecked    25,110,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 13:108,174,428\n",
      "INFO\t2017-01-12 16:08:44\tGenotypeConcordance\tchecked    25,120,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 13:108,953,991\n",
      "INFO\t2017-01-12 16:08:45\tGenotypeConcordance\tchecked    25,130,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 13:109,669,163\n",
      "INFO\t2017-01-12 16:08:45\tGenotypeConcordance\tchecked    25,140,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 13:110,420,075\n",
      "INFO\t2017-01-12 16:08:45\tGenotypeConcordance\tchecked    25,150,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 13:111,132,820\n",
      "INFO\t2017-01-12 16:08:46\tGenotypeConcordance\tchecked    25,160,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 13:111,873,880\n",
      "INFO\t2017-01-12 16:08:46\tGenotypeConcordance\tchecked    25,170,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 13:112,661,858\n",
      "INFO\t2017-01-12 16:08:46\tGenotypeConcordance\tchecked    25,180,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 13:113,345,185\n",
      "INFO\t2017-01-12 16:08:46\tGenotypeConcordance\tchecked    25,190,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 13:114,011,135\n",
      "INFO\t2017-01-12 16:08:47\tGenotypeConcordance\tchecked    25,200,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 13:114,803,398\n",
      "INFO\t2017-01-12 16:08:47\tGenotypeConcordance\tchecked    25,210,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 14:19,265,633\n",
      "INFO\t2017-01-12 16:08:47\tGenotypeConcordance\tchecked    25,220,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 14:20,118,185\n",
      "INFO\t2017-01-12 16:08:47\tGenotypeConcordance\tchecked    25,230,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 14:20,713,788\n",
      "INFO\t2017-01-12 16:08:48\tGenotypeConcordance\tchecked    25,240,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 14:21,426,155\n",
      "INFO\t2017-01-12 16:08:48\tGenotypeConcordance\tchecked    25,250,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 14:22,226,035\n",
      "INFO\t2017-01-12 16:08:48\tGenotypeConcordance\tchecked    25,260,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 14:22,921,280\n",
      "INFO\t2017-01-12 16:08:49\tGenotypeConcordance\tchecked    25,270,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 14:23,718,040\n",
      "INFO\t2017-01-12 16:08:49\tGenotypeConcordance\tchecked    25,280,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 14:24,525,345\n",
      "INFO\t2017-01-12 16:08:49\tGenotypeConcordance\tchecked    25,290,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 14:25,424,736\n",
      "INFO\t2017-01-12 16:08:49\tGenotypeConcordance\tchecked    25,300,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 14:26,226,897\n",
      "INFO\t2017-01-12 16:08:50\tGenotypeConcordance\tchecked    25,310,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 14:27,142,849\n",
      "INFO\t2017-01-12 16:08:50\tGenotypeConcordance\tchecked    25,320,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 14:28,086,730\n",
      "INFO\t2017-01-12 16:08:50\tGenotypeConcordance\tchecked    25,330,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 14:28,882,960\n",
      "INFO\t2017-01-12 16:08:50\tGenotypeConcordance\tchecked    25,340,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 14:29,843,244\n",
      "INFO\t2017-01-12 16:08:50\tGenotypeConcordance\tchecked    25,350,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 14:30,803,579\n",
      "INFO\t2017-01-12 16:08:51\tGenotypeConcordance\tchecked    25,360,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 14:31,698,980\n",
      "INFO\t2017-01-12 16:08:51\tGenotypeConcordance\tchecked    25,370,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 14:32,508,962\n",
      "INFO\t2017-01-12 16:08:51\tGenotypeConcordance\tchecked    25,380,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 14:33,352,908\n",
      "INFO\t2017-01-12 16:08:51\tGenotypeConcordance\tchecked    25,390,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 14:34,186,012\n",
      "INFO\t2017-01-12 16:08:52\tGenotypeConcordance\tchecked    25,400,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 14:34,953,955\n",
      "INFO\t2017-01-12 16:08:52\tGenotypeConcordance\tchecked    25,410,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 14:35,801,851\n",
      "INFO\t2017-01-12 16:08:52\tGenotypeConcordance\tchecked    25,420,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 14:36,737,697\n",
      "INFO\t2017-01-12 16:08:53\tGenotypeConcordance\tchecked    25,430,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 14:37,651,543\n",
      "INFO\t2017-01-12 16:08:53\tGenotypeConcordance\tchecked    25,440,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 14:38,640,848\n",
      "INFO\t2017-01-12 16:08:53\tGenotypeConcordance\tchecked    25,450,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 14:39,535,695\n",
      "INFO\t2017-01-12 16:08:53\tGenotypeConcordance\tchecked    25,460,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 14:40,367,185\n",
      "INFO\t2017-01-12 16:08:54\tGenotypeConcordance\tchecked    25,470,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 14:41,101,973\n",
      "INFO\t2017-01-12 16:08:54\tGenotypeConcordance\tchecked    25,480,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 14:41,865,711\n",
      "INFO\t2017-01-12 16:08:54\tGenotypeConcordance\tchecked    25,490,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 14:42,694,914\n",
      "INFO\t2017-01-12 16:08:55\tGenotypeConcordance\tchecked    25,500,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 14:43,435,164\n",
      "INFO\t2017-01-12 16:08:55\tGenotypeConcordance\tchecked    25,510,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 14:44,159,974\n",
      "INFO\t2017-01-12 16:08:55\tGenotypeConcordance\tchecked    25,520,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 14:44,993,347\n",
      "INFO\t2017-01-12 16:08:55\tGenotypeConcordance\tchecked    25,530,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 14:45,986,130\n",
      "INFO\t2017-01-12 16:08:56\tGenotypeConcordance\tchecked    25,540,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 14:46,753,003\n",
      "INFO\t2017-01-12 16:08:56\tGenotypeConcordance\tchecked    25,550,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 14:47,608,303\n",
      "INFO\t2017-01-12 16:08:56\tGenotypeConcordance\tchecked    25,560,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 14:48,415,645\n",
      "INFO\t2017-01-12 16:08:57\tGenotypeConcordance\tchecked    25,570,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 14:49,196,486\n",
      "INFO\t2017-01-12 16:08:57\tGenotypeConcordance\tchecked    25,580,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 14:50,051,751\n",
      "INFO\t2017-01-12 16:08:57\tGenotypeConcordance\tchecked    25,590,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 14:50,956,974\n",
      "INFO\t2017-01-12 16:08:57\tGenotypeConcordance\tchecked    25,600,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 14:51,819,420\n",
      "INFO\t2017-01-12 16:08:58\tGenotypeConcordance\tchecked    25,610,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 14:52,626,461\n",
      "INFO\t2017-01-12 16:08:58\tGenotypeConcordance\tchecked    25,620,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 14:53,481,995\n",
      "INFO\t2017-01-12 16:08:58\tGenotypeConcordance\tchecked    25,630,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 14:54,322,781\n",
      "INFO\t2017-01-12 16:08:58\tGenotypeConcordance\tchecked    25,640,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 14:55,256,021\n",
      "INFO\t2017-01-12 16:08:59\tGenotypeConcordance\tchecked    25,650,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 14:56,071,798\n",
      "INFO\t2017-01-12 16:08:59\tGenotypeConcordance\tchecked    25,660,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 14:56,868,003\n",
      "INFO\t2017-01-12 16:08:59\tGenotypeConcordance\tchecked    25,670,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 14:57,692,107\n",
      "INFO\t2017-01-12 16:08:59\tGenotypeConcordance\tchecked    25,680,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 14:58,514,790\n",
      "INFO\t2017-01-12 16:09:00\tGenotypeConcordance\tchecked    25,690,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 14:59,519,143\n",
      "INFO\t2017-01-12 16:09:00\tGenotypeConcordance\tchecked    25,700,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 14:60,460,445\n",
      "INFO\t2017-01-12 16:09:00\tGenotypeConcordance\tchecked    25,710,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 14:61,510,680\n",
      "INFO\t2017-01-12 16:09:00\tGenotypeConcordance\tchecked    25,720,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 14:62,358,047\n",
      "INFO\t2017-01-12 16:09:01\tGenotypeConcordance\tchecked    25,730,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 14:63,140,769\n",
      "INFO\t2017-01-12 16:09:01\tGenotypeConcordance\tchecked    25,740,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 14:64,010,783\n",
      "INFO\t2017-01-12 16:09:01\tGenotypeConcordance\tchecked    25,750,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 14:64,914,537\n",
      "INFO\t2017-01-12 16:09:01\tGenotypeConcordance\tchecked    25,760,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 14:65,804,664\n",
      "INFO\t2017-01-12 16:09:02\tGenotypeConcordance\tchecked    25,770,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 14:66,697,945\n",
      "INFO\t2017-01-12 16:09:02\tGenotypeConcordance\tchecked    25,780,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 14:67,693,839\n",
      "INFO\t2017-01-12 16:09:02\tGenotypeConcordance\tchecked    25,790,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 14:68,685,460\n",
      "INFO\t2017-01-12 16:09:02\tGenotypeConcordance\tchecked    25,800,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 14:69,593,656\n",
      "INFO\t2017-01-12 16:09:03\tGenotypeConcordance\tchecked    25,810,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 14:70,479,078\n",
      "INFO\t2017-01-12 16:09:03\tGenotypeConcordance\tchecked    25,820,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 14:71,301,755\n",
      "INFO\t2017-01-12 16:09:03\tGenotypeConcordance\tchecked    25,830,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 14:72,309,303\n",
      "INFO\t2017-01-12 16:09:03\tGenotypeConcordance\tchecked    25,840,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 14:73,076,504\n",
      "INFO\t2017-01-12 16:09:04\tGenotypeConcordance\tchecked    25,850,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 14:73,967,895\n",
      "INFO\t2017-01-12 16:09:04\tGenotypeConcordance\tchecked    25,860,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 14:74,749,606\n",
      "INFO\t2017-01-12 16:09:04\tGenotypeConcordance\tchecked    25,870,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 14:75,724,473\n",
      "INFO\t2017-01-12 16:09:04\tGenotypeConcordance\tchecked    25,880,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 14:76,659,723\n",
      "INFO\t2017-01-12 16:09:05\tGenotypeConcordance\tchecked    25,890,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 14:77,436,787\n",
      "INFO\t2017-01-12 16:09:05\tGenotypeConcordance\tchecked    25,900,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 14:78,239,259\n",
      "INFO\t2017-01-12 16:09:05\tGenotypeConcordance\tchecked    25,910,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 14:79,104,652\n",
      "INFO\t2017-01-12 16:09:05\tGenotypeConcordance\tchecked    25,920,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 14:80,016,502\n",
      "INFO\t2017-01-12 16:09:06\tGenotypeConcordance\tchecked    25,930,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 14:80,931,119\n",
      "INFO\t2017-01-12 16:09:06\tGenotypeConcordance\tchecked    25,940,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 14:81,836,996\n",
      "INFO\t2017-01-12 16:09:06\tGenotypeConcordance\tchecked    25,950,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 14:82,673,878\n",
      "INFO\t2017-01-12 16:09:06\tGenotypeConcordance\tchecked    25,960,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 14:83,528,049\n",
      "INFO\t2017-01-12 16:09:07\tGenotypeConcordance\tchecked    25,970,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 14:84,402,471\n",
      "INFO\t2017-01-12 16:09:07\tGenotypeConcordance\tchecked    25,980,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 14:85,234,218\n",
      "INFO\t2017-01-12 16:09:07\tGenotypeConcordance\tchecked    25,990,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 14:86,098,183\n",
      "INFO\t2017-01-12 16:09:07\tGenotypeConcordance\tchecked    26,000,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 14:86,839,203\n",
      "INFO\t2017-01-12 16:09:08\tGenotypeConcordance\tchecked    26,010,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 14:87,705,702\n",
      "INFO\t2017-01-12 16:09:08\tGenotypeConcordance\tchecked    26,020,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 14:88,489,630\n",
      "INFO\t2017-01-12 16:09:08\tGenotypeConcordance\tchecked    26,030,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 14:89,384,118\n",
      "INFO\t2017-01-12 16:09:08\tGenotypeConcordance\tchecked    26,040,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 14:90,169,202\n",
      "INFO\t2017-01-12 16:09:09\tGenotypeConcordance\tchecked    26,050,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 14:90,998,817\n",
      "INFO\t2017-01-12 16:09:09\tGenotypeConcordance\tchecked    26,060,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 14:91,845,588\n",
      "INFO\t2017-01-12 16:09:09\tGenotypeConcordance\tchecked    26,070,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 14:92,692,952\n",
      "INFO\t2017-01-12 16:09:09\tGenotypeConcordance\tchecked    26,080,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 14:93,461,185\n",
      "INFO\t2017-01-12 16:09:10\tGenotypeConcordance\tchecked    26,090,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 14:94,338,490\n",
      "INFO\t2017-01-12 16:09:10\tGenotypeConcordance\tchecked    26,100,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 14:95,099,611\n",
      "INFO\t2017-01-12 16:09:10\tGenotypeConcordance\tchecked    26,110,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 14:95,831,101\n",
      "INFO\t2017-01-12 16:09:10\tGenotypeConcordance\tchecked    26,120,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 14:96,525,596\n",
      "INFO\t2017-01-12 16:09:11\tGenotypeConcordance\tchecked    26,130,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 14:97,350,173\n",
      "INFO\t2017-01-12 16:09:11\tGenotypeConcordance\tchecked    26,140,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 14:98,181,180\n",
      "INFO\t2017-01-12 16:09:11\tGenotypeConcordance\tchecked    26,150,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 14:98,901,493\n",
      "INFO\t2017-01-12 16:09:11\tGenotypeConcordance\tchecked    26,160,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 14:99,647,028\n",
      "INFO\t2017-01-12 16:09:12\tGenotypeConcordance\tchecked    26,170,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 14:100,483,129\n",
      "INFO\t2017-01-12 16:09:12\tGenotypeConcordance\tchecked    26,180,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 14:101,306,685\n",
      "INFO\t2017-01-12 16:09:12\tGenotypeConcordance\tchecked    26,190,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 14:102,126,242\n",
      "INFO\t2017-01-12 16:09:12\tGenotypeConcordance\tchecked    26,200,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 14:102,987,286\n",
      "INFO\t2017-01-12 16:09:13\tGenotypeConcordance\tchecked    26,210,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 14:103,772,717\n",
      "INFO\t2017-01-12 16:09:13\tGenotypeConcordance\tchecked    26,220,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 14:104,587,869\n",
      "INFO\t2017-01-12 16:09:13\tGenotypeConcordance\tchecked    26,230,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 14:105,236,672\n",
      "INFO\t2017-01-12 16:09:13\tGenotypeConcordance\tchecked    26,240,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 14:105,978,010\n",
      "INFO\t2017-01-12 16:09:14\tGenotypeConcordance\tchecked    26,250,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 14:106,728,317\n",
      "INFO\t2017-01-12 16:09:14\tGenotypeConcordance\tchecked    26,260,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 15:20,019,922\n",
      "INFO\t2017-01-12 16:09:14\tGenotypeConcordance\tchecked    26,270,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 15:20,492,132\n",
      "INFO\t2017-01-12 16:09:15\tGenotypeConcordance\tchecked    26,280,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 15:21,887,545\n",
      "INFO\t2017-01-12 16:09:15\tGenotypeConcordance\tchecked    26,290,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 15:22,576,643\n",
      "INFO\t2017-01-12 16:09:15\tGenotypeConcordance\tchecked    26,300,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 15:23,506,879\n",
      "INFO\t2017-01-12 16:09:15\tGenotypeConcordance\tchecked    26,310,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 15:24,320,566\n",
      "INFO\t2017-01-12 16:09:16\tGenotypeConcordance\tchecked    26,320,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 15:24,758,125\n",
      "INFO\t2017-01-12 16:09:16\tGenotypeConcordance\tchecked    26,330,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 15:25,391,168\n",
      "INFO\t2017-01-12 16:09:16\tGenotypeConcordance\tchecked    26,340,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 15:26,145,292\n",
      "INFO\t2017-01-12 16:09:16\tGenotypeConcordance\tchecked    26,350,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 15:26,880,157\n",
      "INFO\t2017-01-12 16:09:17\tGenotypeConcordance\tchecked    26,360,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 15:27,682,851\n",
      "INFO\t2017-01-12 16:09:17\tGenotypeConcordance\tchecked    26,370,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 15:28,481,743\n",
      "INFO\t2017-01-12 16:09:17\tGenotypeConcordance\tchecked    26,380,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 15:29,454,669\n",
      "INFO\t2017-01-12 16:09:17\tGenotypeConcordance\tchecked    26,390,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 15:30,269,669\n",
      "INFO\t2017-01-12 16:09:18\tGenotypeConcordance\tchecked    26,400,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 15:31,427,069\n",
      "INFO\t2017-01-12 16:09:18\tGenotypeConcordance\tchecked    26,410,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 15:32,255,382\n",
      "INFO\t2017-01-12 16:09:18\tGenotypeConcordance\tchecked    26,420,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 15:33,259,773\n",
      "INFO\t2017-01-12 16:09:18\tGenotypeConcordance\tchecked    26,430,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 15:33,950,236\n",
      "INFO\t2017-01-12 16:09:19\tGenotypeConcordance\tchecked    26,440,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 15:34,689,957\n",
      "INFO\t2017-01-12 16:09:19\tGenotypeConcordance\tchecked    26,450,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 15:35,516,664\n",
      "INFO\t2017-01-12 16:09:19\tGenotypeConcordance\tchecked    26,460,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 15:36,481,940\n",
      "INFO\t2017-01-12 16:09:20\tGenotypeConcordance\tchecked    26,470,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 15:37,395,212\n",
      "INFO\t2017-01-12 16:09:20\tGenotypeConcordance\tchecked    26,480,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 15:38,231,284\n",
      "INFO\t2017-01-12 16:09:20\tGenotypeConcordance\tchecked    26,490,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 15:39,119,986\n",
      "INFO\t2017-01-12 16:09:20\tGenotypeConcordance\tchecked    26,500,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 15:39,845,432\n",
      "INFO\t2017-01-12 16:09:21\tGenotypeConcordance\tchecked    26,510,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 15:40,747,604\n",
      "INFO\t2017-01-12 16:09:21\tGenotypeConcordance\tchecked    26,520,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 15:41,674,132\n",
      "INFO\t2017-01-12 16:09:21\tGenotypeConcordance\tchecked    26,530,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 15:42,589,729\n",
      "INFO\t2017-01-12 16:09:21\tGenotypeConcordance\tchecked    26,540,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 15:43,677,568\n",
      "INFO\t2017-01-12 16:09:22\tGenotypeConcordance\tchecked    26,550,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 15:44,858,941\n",
      "INFO\t2017-01-12 16:09:22\tGenotypeConcordance\tchecked    26,560,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 15:45,749,030\n",
      "INFO\t2017-01-12 16:09:22\tGenotypeConcordance\tchecked    26,570,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 15:46,574,981\n",
      "INFO\t2017-01-12 16:09:22\tGenotypeConcordance\tchecked    26,580,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 15:47,420,677\n",
      "INFO\t2017-01-12 16:09:23\tGenotypeConcordance\tchecked    26,590,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 15:48,354,993\n",
      "INFO\t2017-01-12 16:09:23\tGenotypeConcordance\tchecked    26,600,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 15:49,460,678\n",
      "INFO\t2017-01-12 16:09:23\tGenotypeConcordance\tchecked    26,610,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 15:50,437,483\n",
      "INFO\t2017-01-12 16:09:23\tGenotypeConcordance\tchecked    26,620,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 15:51,287,916\n",
      "INFO\t2017-01-12 16:09:24\tGenotypeConcordance\tchecked    26,630,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 15:52,259,467\n",
      "INFO\t2017-01-12 16:09:24\tGenotypeConcordance\tchecked    26,640,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 15:53,111,692\n",
      "INFO\t2017-01-12 16:09:24\tGenotypeConcordance\tchecked    26,650,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 15:53,876,637\n",
      "INFO\t2017-01-12 16:09:24\tGenotypeConcordance\tchecked    26,660,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 15:54,578,397\n",
      "INFO\t2017-01-12 16:09:25\tGenotypeConcordance\tchecked    26,670,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 15:55,257,869\n",
      "INFO\t2017-01-12 16:09:25\tGenotypeConcordance\tchecked    26,680,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 15:56,040,341\n",
      "INFO\t2017-01-12 16:09:25\tGenotypeConcordance\tchecked    26,690,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 15:56,866,113\n",
      "INFO\t2017-01-12 16:09:25\tGenotypeConcordance\tchecked    26,700,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 15:57,762,045\n",
      "INFO\t2017-01-12 16:09:26\tGenotypeConcordance\tchecked    26,710,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 15:58,473,395\n",
      "INFO\t2017-01-12 16:09:26\tGenotypeConcordance\tchecked    26,720,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 15:59,270,634\n",
      "INFO\t2017-01-12 16:09:26\tGenotypeConcordance\tchecked    26,730,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 15:60,013,216\n",
      "INFO\t2017-01-12 16:09:26\tGenotypeConcordance\tchecked    26,740,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 15:60,779,014\n",
      "INFO\t2017-01-12 16:09:27\tGenotypeConcordance\tchecked    26,750,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 15:61,639,757\n",
      "INFO\t2017-01-12 16:09:27\tGenotypeConcordance\tchecked    26,760,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 15:62,471,025\n",
      "INFO\t2017-01-12 16:09:27\tGenotypeConcordance\tchecked    26,770,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 15:63,249,167\n",
      "INFO\t2017-01-12 16:09:27\tGenotypeConcordance\tchecked    26,780,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 15:64,141,036\n",
      "INFO\t2017-01-12 16:09:28\tGenotypeConcordance\tchecked    26,790,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 15:65,114,039\n",
      "INFO\t2017-01-12 16:09:28\tGenotypeConcordance\tchecked    26,800,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 15:66,020,432\n",
      "INFO\t2017-01-12 16:09:28\tGenotypeConcordance\tchecked    26,810,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 15:66,861,241\n",
      "INFO\t2017-01-12 16:09:28\tGenotypeConcordance\tchecked    26,820,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 15:67,714,571\n",
      "INFO\t2017-01-12 16:09:29\tGenotypeConcordance\tchecked    26,830,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 15:68,671,717\n",
      "INFO\t2017-01-12 16:09:29\tGenotypeConcordance\tchecked    26,840,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 15:69,559,255\n",
      "INFO\t2017-01-12 16:09:29\tGenotypeConcordance\tchecked    26,850,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 15:70,435,947\n",
      "INFO\t2017-01-12 16:09:30\tGenotypeConcordance\tchecked    26,860,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 15:71,437,333\n",
      "INFO\t2017-01-12 16:09:30\tGenotypeConcordance\tchecked    26,870,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 15:72,287,902\n",
      "INFO\t2017-01-12 16:09:30\tGenotypeConcordance\tchecked    26,880,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 15:73,395,364\n",
      "INFO\t2017-01-12 16:09:30\tGenotypeConcordance\tchecked    26,890,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 15:74,357,244\n",
      "INFO\t2017-01-12 16:09:31\tGenotypeConcordance\tchecked    26,900,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 15:75,449,828\n",
      "INFO\t2017-01-12 16:09:31\tGenotypeConcordance\tchecked    26,910,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 15:76,482,077\n",
      "INFO\t2017-01-12 16:09:31\tGenotypeConcordance\tchecked    26,920,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 15:77,484,477\n",
      "INFO\t2017-01-12 16:09:31\tGenotypeConcordance\tchecked    26,930,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 15:78,275,500\n",
      "INFO\t2017-01-12 16:09:32\tGenotypeConcordance\tchecked    26,940,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 15:79,099,537\n",
      "INFO\t2017-01-12 16:09:32\tGenotypeConcordance\tchecked    26,950,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 15:79,864,072\n",
      "INFO\t2017-01-12 16:09:32\tGenotypeConcordance\tchecked    26,960,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 15:80,624,809\n",
      "INFO\t2017-01-12 16:09:32\tGenotypeConcordance\tchecked    26,970,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 15:81,466,197\n",
      "INFO\t2017-01-12 16:09:33\tGenotypeConcordance\tchecked    26,980,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 15:82,366,461\n",
      "INFO\t2017-01-12 16:09:33\tGenotypeConcordance\tchecked    26,990,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 15:83,940,733\n",
      "INFO\t2017-01-12 16:09:33\tGenotypeConcordance\tchecked    27,000,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 15:84,956,557\n",
      "INFO\t2017-01-12 16:09:34\tGenotypeConcordance\tchecked    27,010,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 15:85,985,755\n",
      "INFO\t2017-01-12 16:09:34\tGenotypeConcordance\tchecked    27,020,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 15:86,721,032\n",
      "INFO\t2017-01-12 16:09:34\tGenotypeConcordance\tchecked    27,030,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 15:87,402,722\n",
      "INFO\t2017-01-12 16:09:34\tGenotypeConcordance\tchecked    27,040,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 15:88,075,564\n",
      "INFO\t2017-01-12 16:09:35\tGenotypeConcordance\tchecked    27,050,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 15:88,878,123\n",
      "INFO\t2017-01-12 16:09:35\tGenotypeConcordance\tchecked    27,060,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 15:89,588,694\n",
      "INFO\t2017-01-12 16:09:35\tGenotypeConcordance\tchecked    27,070,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 15:90,347,814\n",
      "INFO\t2017-01-12 16:09:35\tGenotypeConcordance\tchecked    27,080,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 15:91,122,156\n",
      "INFO\t2017-01-12 16:09:36\tGenotypeConcordance\tchecked    27,090,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 15:91,860,260\n",
      "INFO\t2017-01-12 16:09:36\tGenotypeConcordance\tchecked    27,100,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 15:92,609,246\n",
      "INFO\t2017-01-12 16:09:36\tGenotypeConcordance\tchecked    27,110,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 15:93,310,705\n",
      "INFO\t2017-01-12 16:09:36\tGenotypeConcordance\tchecked    27,120,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 15:93,972,367\n",
      "INFO\t2017-01-12 16:09:37\tGenotypeConcordance\tchecked    27,130,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 15:94,735,128\n",
      "INFO\t2017-01-12 16:09:37\tGenotypeConcordance\tchecked    27,140,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 15:95,504,426\n",
      "INFO\t2017-01-12 16:09:37\tGenotypeConcordance\tchecked    27,150,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 15:96,372,089\n",
      "INFO\t2017-01-12 16:09:37\tGenotypeConcordance\tchecked    27,160,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 15:97,247,665\n",
      "INFO\t2017-01-12 16:09:38\tGenotypeConcordance\tchecked    27,170,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 15:98,033,244\n",
      "INFO\t2017-01-12 16:09:38\tGenotypeConcordance\tchecked    27,180,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 15:98,729,165\n",
      "INFO\t2017-01-12 16:09:38\tGenotypeConcordance\tchecked    27,190,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 15:99,450,259\n",
      "INFO\t2017-01-12 16:09:38\tGenotypeConcordance\tchecked    27,200,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 15:100,260,501\n",
      "INFO\t2017-01-12 16:09:39\tGenotypeConcordance\tchecked    27,210,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 15:100,826,957\n",
      "INFO\t2017-01-12 16:09:39\tGenotypeConcordance\tchecked    27,220,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 15:101,499,504\n",
      "INFO\t2017-01-12 16:09:39\tGenotypeConcordance\tchecked    27,230,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 15:102,117,423\n",
      "INFO\t2017-01-12 16:09:39\tGenotypeConcordance\tchecked    27,240,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 16:370,074\n",
      "INFO\t2017-01-12 16:09:40\tGenotypeConcordance\tchecked    27,250,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 16:941,628\n",
      "INFO\t2017-01-12 16:09:40\tGenotypeConcordance\tchecked    27,260,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 16:1,448,751\n",
      "INFO\t2017-01-12 16:09:40\tGenotypeConcordance\tchecked    27,270,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 16:2,117,260\n",
      "INFO\t2017-01-12 16:09:41\tGenotypeConcordance\tchecked    27,280,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 16:2,970,999\n",
      "INFO\t2017-01-12 16:09:41\tGenotypeConcordance\tchecked    27,290,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 16:3,651,424\n",
      "INFO\t2017-01-12 16:09:41\tGenotypeConcordance\tchecked    27,300,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 16:4,358,039\n",
      "INFO\t2017-01-12 16:09:41\tGenotypeConcordance\tchecked    27,310,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 16:5,035,619\n",
      "INFO\t2017-01-12 16:09:42\tGenotypeConcordance\tchecked    27,320,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 16:5,556,417\n",
      "INFO\t2017-01-12 16:09:42\tGenotypeConcordance\tchecked    27,330,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 16:6,039,092\n",
      "INFO\t2017-01-12 16:09:42\tGenotypeConcordance\tchecked    27,340,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 16:6,565,511\n",
      "INFO\t2017-01-12 16:09:42\tGenotypeConcordance\tchecked    27,350,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 16:6,938,233\n",
      "INFO\t2017-01-12 16:09:43\tGenotypeConcordance\tchecked    27,360,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 16:7,277,983\n",
      "INFO\t2017-01-12 16:09:43\tGenotypeConcordance\tchecked    27,370,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 16:7,774,406\n",
      "INFO\t2017-01-12 16:09:43\tGenotypeConcordance\tchecked    27,380,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 16:8,292,694\n",
      "INFO\t2017-01-12 16:09:44\tGenotypeConcordance\tchecked    27,390,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 16:8,826,064\n",
      "INFO\t2017-01-12 16:09:44\tGenotypeConcordance\tchecked    27,400,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 16:9,385,760\n",
      "INFO\t2017-01-12 16:09:44\tGenotypeConcordance\tchecked    27,410,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 16:10,073,295\n",
      "INFO\t2017-01-12 16:09:44\tGenotypeConcordance\tchecked    27,420,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 16:10,700,928\n",
      "INFO\t2017-01-12 16:09:45\tGenotypeConcordance\tchecked    27,430,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 16:11,369,940\n",
      "INFO\t2017-01-12 16:09:45\tGenotypeConcordance\tchecked    27,440,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 16:11,985,410\n",
      "INFO\t2017-01-12 16:09:45\tGenotypeConcordance\tchecked    27,450,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 16:12,631,768\n",
      "INFO\t2017-01-12 16:09:46\tGenotypeConcordance\tchecked    27,460,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 16:13,091,656\n",
      "INFO\t2017-01-12 16:09:46\tGenotypeConcordance\tchecked    27,470,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 16:13,787,547\n",
      "INFO\t2017-01-12 16:09:46\tGenotypeConcordance\tchecked    27,480,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 16:14,624,301\n",
      "INFO\t2017-01-12 16:09:46\tGenotypeConcordance\tchecked    27,490,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 16:15,823,169\n",
      "INFO\t2017-01-12 16:09:47\tGenotypeConcordance\tchecked    27,500,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 16:16,738,140\n",
      "INFO\t2017-01-12 16:09:47\tGenotypeConcordance\tchecked    27,510,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 16:17,595,028\n",
      "INFO\t2017-01-12 16:09:47\tGenotypeConcordance\tchecked    27,520,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 16:18,383,160\n",
      "INFO\t2017-01-12 16:09:48\tGenotypeConcordance\tchecked    27,530,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 16:19,408,637\n",
      "INFO\t2017-01-12 16:09:48\tGenotypeConcordance\tchecked    27,540,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 16:20,258,451\n",
      "INFO\t2017-01-12 16:09:48\tGenotypeConcordance\tchecked    27,550,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 16:21,163,690\n",
      "INFO\t2017-01-12 16:09:49\tGenotypeConcordance\tchecked    27,560,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 16:22,574,512\n",
      "INFO\t2017-01-12 16:09:49\tGenotypeConcordance\tchecked    27,570,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 16:23,488,031\n",
      "INFO\t2017-01-12 16:09:49\tGenotypeConcordance\tchecked    27,580,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 16:24,351,328\n",
      "INFO\t2017-01-12 16:09:50\tGenotypeConcordance\tchecked    27,590,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 16:25,166,621\n",
      "INFO\t2017-01-12 16:09:50\tGenotypeConcordance\tchecked    27,600,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 16:25,976,419\n",
      "INFO\t2017-01-12 16:09:50\tGenotypeConcordance\tchecked    27,610,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 16:26,731,700\n",
      "INFO\t2017-01-12 16:09:51\tGenotypeConcordance\tchecked    27,620,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 16:27,461,941\n",
      "INFO\t2017-01-12 16:09:51\tGenotypeConcordance\tchecked    27,630,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 16:28,490,200\n",
      "INFO\t2017-01-12 16:09:51\tGenotypeConcordance\tchecked    27,640,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 16:29,589,556\n",
      "INFO\t2017-01-12 16:09:52\tGenotypeConcordance\tchecked    27,650,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 16:30,930,224\n",
      "INFO\t2017-01-12 16:09:52\tGenotypeConcordance\tchecked    27,660,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 16:32,034,627\n",
      "INFO\t2017-01-12 16:09:52\tGenotypeConcordance\tchecked    27,670,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 16:32,491,794\n",
      "INFO\t2017-01-12 16:09:53\tGenotypeConcordance\tchecked    27,680,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 16:33,049,162\n",
      "INFO\t2017-01-12 16:09:53\tGenotypeConcordance\tchecked    27,690,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 16:33,664,589\n",
      "INFO\t2017-01-12 16:09:53\tGenotypeConcordance\tchecked    27,700,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 16:33,986,076\n",
      "INFO\t2017-01-12 16:09:54\tGenotypeConcordance\tchecked    27,710,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 16:34,850,281\n",
      "INFO\t2017-01-12 16:09:54\tGenotypeConcordance\tchecked    27,720,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 16:46,477,222\n",
      "INFO\t2017-01-12 16:09:54\tGenotypeConcordance\tchecked    27,730,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 16:47,784,203\n",
      "INFO\t2017-01-12 16:09:54\tGenotypeConcordance\tchecked    27,740,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 16:48,834,737\n",
      "INFO\t2017-01-12 16:09:55\tGenotypeConcordance\tchecked    27,750,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 16:49,721,942\n",
      "INFO\t2017-01-12 16:09:55\tGenotypeConcordance\tchecked    27,760,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 16:50,636,647\n",
      "INFO\t2017-01-12 16:09:55\tGenotypeConcordance\tchecked    27,770,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 16:51,510,060\n",
      "INFO\t2017-01-12 16:09:56\tGenotypeConcordance\tchecked    27,780,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 16:52,342,550\n",
      "INFO\t2017-01-12 16:09:56\tGenotypeConcordance\tchecked    27,790,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 16:53,166,187\n",
      "INFO\t2017-01-12 16:09:56\tGenotypeConcordance\tchecked    27,800,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 16:54,126,444\n",
      "INFO\t2017-01-12 16:09:57\tGenotypeConcordance\tchecked    27,810,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 16:54,886,126\n",
      "INFO\t2017-01-12 16:09:57\tGenotypeConcordance\tchecked    27,820,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 16:55,658,747\n",
      "INFO\t2017-01-12 16:09:57\tGenotypeConcordance\tchecked    27,830,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 16:56,440,037\n",
      "INFO\t2017-01-12 16:09:58\tGenotypeConcordance\tchecked    27,840,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 16:57,251,503\n",
      "INFO\t2017-01-12 16:09:58\tGenotypeConcordance\tchecked    27,850,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 16:58,044,594\n",
      "INFO\t2017-01-12 16:09:58\tGenotypeConcordance\tchecked    27,860,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 16:58,837,767\n",
      "INFO\t2017-01-12 16:09:59\tGenotypeConcordance\tchecked    27,870,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 16:59,710,112\n",
      "INFO\t2017-01-12 16:09:59\tGenotypeConcordance\tchecked    27,880,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 16:60,598,004\n",
      "INFO\t2017-01-12 16:09:59\tGenotypeConcordance\tchecked    27,890,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 16:61,588,986\n",
      "INFO\t2017-01-12 16:10:00\tGenotypeConcordance\tchecked    27,900,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 16:62,507,747\n",
      "INFO\t2017-01-12 16:10:00\tGenotypeConcordance\tchecked    27,910,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 16:63,375,745\n",
      "INFO\t2017-01-12 16:10:00\tGenotypeConcordance\tchecked    27,920,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 16:64,296,779\n",
      "INFO\t2017-01-12 16:10:01\tGenotypeConcordance\tchecked    27,930,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 16:65,173,715\n",
      "INFO\t2017-01-12 16:10:01\tGenotypeConcordance\tchecked    27,940,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 16:66,016,188\n",
      "INFO\t2017-01-12 16:10:01\tGenotypeConcordance\tchecked    27,950,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 16:67,048,064\n",
      "INFO\t2017-01-12 16:10:01\tGenotypeConcordance\tchecked    27,960,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 16:68,343,793\n",
      "INFO\t2017-01-12 16:10:02\tGenotypeConcordance\tchecked    27,970,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 16:69,356,819\n",
      "INFO\t2017-01-12 16:10:02\tGenotypeConcordance\tchecked    27,980,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 16:70,296,899\n",
      "INFO\t2017-01-12 16:10:02\tGenotypeConcordance\tchecked    27,990,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 16:71,335,258\n",
      "INFO\t2017-01-12 16:10:03\tGenotypeConcordance\tchecked    28,000,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 16:72,355,139\n",
      "INFO\t2017-01-12 16:10:03\tGenotypeConcordance\tchecked    28,010,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 16:73,332,747\n",
      "INFO\t2017-01-12 16:10:03\tGenotypeConcordance\tchecked    28,020,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 16:74,102,050\n",
      "INFO\t2017-01-12 16:10:04\tGenotypeConcordance\tchecked    28,030,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 16:74,877,135\n",
      "INFO\t2017-01-12 16:10:04\tGenotypeConcordance\tchecked    28,040,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 16:75,537,283\n",
      "INFO\t2017-01-12 16:10:04\tGenotypeConcordance\tchecked    28,050,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 16:76,276,902\n",
      "INFO\t2017-01-12 16:10:04\tGenotypeConcordance\tchecked    28,060,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 16:76,930,088\n",
      "INFO\t2017-01-12 16:10:05\tGenotypeConcordance\tchecked    28,070,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 16:77,545,392\n",
      "INFO\t2017-01-12 16:10:05\tGenotypeConcordance\tchecked    28,080,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 16:78,123,591\n",
      "INFO\t2017-01-12 16:10:05\tGenotypeConcordance\tchecked    28,090,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 16:78,581,022\n",
      "INFO\t2017-01-12 16:10:06\tGenotypeConcordance\tchecked    28,100,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 16:78,986,468\n",
      "INFO\t2017-01-12 16:10:06\tGenotypeConcordance\tchecked    28,110,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 16:79,484,853\n",
      "INFO\t2017-01-12 16:10:06\tGenotypeConcordance\tchecked    28,120,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 16:80,027,021\n",
      "INFO\t2017-01-12 16:10:07\tGenotypeConcordance\tchecked    28,130,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 16:80,603,749\n",
      "INFO\t2017-01-12 16:10:07\tGenotypeConcordance\tchecked    28,140,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 16:81,170,121\n",
      "INFO\t2017-01-12 16:10:07\tGenotypeConcordance\tchecked    28,150,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 16:81,751,183\n",
      "INFO\t2017-01-12 16:10:08\tGenotypeConcordance\tchecked    28,160,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 16:82,280,060\n",
      "INFO\t2017-01-12 16:10:08\tGenotypeConcordance\tchecked    28,170,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 16:82,818,499\n",
      "INFO\t2017-01-12 16:10:08\tGenotypeConcordance\tchecked    28,180,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 16:83,322,894\n",
      "INFO\t2017-01-12 16:10:09\tGenotypeConcordance\tchecked    28,190,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 16:83,875,618\n",
      "INFO\t2017-01-12 16:10:09\tGenotypeConcordance\tchecked    28,200,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 16:84,347,646\n",
      "INFO\t2017-01-12 16:10:09\tGenotypeConcordance\tchecked    28,210,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 16:84,838,816\n",
      "INFO\t2017-01-12 16:10:09\tGenotypeConcordance\tchecked    28,220,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 16:85,394,614\n",
      "INFO\t2017-01-12 16:10:10\tGenotypeConcordance\tchecked    28,230,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 16:85,985,535\n",
      "INFO\t2017-01-12 16:10:10\tGenotypeConcordance\tchecked    28,240,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 16:86,487,208\n",
      "INFO\t2017-01-12 16:10:10\tGenotypeConcordance\tchecked    28,250,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 16:87,021,168\n",
      "INFO\t2017-01-12 16:10:11\tGenotypeConcordance\tchecked    28,260,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 16:87,626,154\n",
      "INFO\t2017-01-12 16:10:11\tGenotypeConcordance\tchecked    28,270,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 16:88,131,535\n",
      "INFO\t2017-01-12 16:10:11\tGenotypeConcordance\tchecked    28,280,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 16:88,699,386\n",
      "INFO\t2017-01-12 16:10:12\tGenotypeConcordance\tchecked    28,290,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 16:89,206,841\n",
      "INFO\t2017-01-12 16:10:12\tGenotypeConcordance\tchecked    28,300,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 16:89,801,920\n",
      "INFO\t2017-01-12 16:10:12\tGenotypeConcordance\tchecked    28,310,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 17:168,015\n",
      "INFO\t2017-01-12 16:10:13\tGenotypeConcordance\tchecked    28,320,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 17:853,031\n",
      "INFO\t2017-01-12 16:10:13\tGenotypeConcordance\tchecked    28,330,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 17:1,421,182\n",
      "INFO\t2017-01-12 16:10:14\tGenotypeConcordance\tchecked    28,340,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 17:2,203,408\n",
      "INFO\t2017-01-12 16:10:14\tGenotypeConcordance\tchecked    28,350,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 17:2,946,054\n",
      "INFO\t2017-01-12 16:10:14\tGenotypeConcordance\tchecked    28,360,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 17:3,637,453\n",
      "INFO\t2017-01-12 16:10:15\tGenotypeConcordance\tchecked    28,370,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 17:4,371,113\n",
      "INFO\t2017-01-12 16:10:15\tGenotypeConcordance\tchecked    28,380,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 17:5,140,515\n",
      "INFO\t2017-01-12 16:10:15\tGenotypeConcordance\tchecked    28,390,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 17:5,868,104\n",
      "INFO\t2017-01-12 16:10:16\tGenotypeConcordance\tchecked    28,400,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 17:6,594,343\n",
      "INFO\t2017-01-12 16:10:16\tGenotypeConcordance\tchecked    28,410,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 17:7,394,337\n",
      "INFO\t2017-01-12 16:10:16\tGenotypeConcordance\tchecked    28,420,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 17:8,258,055\n",
      "INFO\t2017-01-12 16:10:17\tGenotypeConcordance\tchecked    28,430,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 17:9,083,200\n",
      "INFO\t2017-01-12 16:10:17\tGenotypeConcordance\tchecked    28,440,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 17:9,796,111\n",
      "INFO\t2017-01-12 16:10:17\tGenotypeConcordance\tchecked    28,450,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 17:10,568,838\n",
      "INFO\t2017-01-12 16:10:18\tGenotypeConcordance\tchecked    28,460,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 17:11,343,068\n",
      "INFO\t2017-01-12 16:10:18\tGenotypeConcordance\tchecked    28,470,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 17:12,170,535\n",
      "INFO\t2017-01-12 16:10:18\tGenotypeConcordance\tchecked    28,480,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 17:12,925,095\n",
      "INFO\t2017-01-12 16:10:19\tGenotypeConcordance\tchecked    28,490,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 17:13,609,487\n",
      "INFO\t2017-01-12 16:10:19\tGenotypeConcordance\tchecked    28,500,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 17:14,344,552\n",
      "INFO\t2017-01-12 16:10:19\tGenotypeConcordance\tchecked    28,510,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 17:14,992,958\n",
      "INFO\t2017-01-12 16:10:20\tGenotypeConcordance\tchecked    28,520,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 17:15,755,824\n",
      "INFO\t2017-01-12 16:10:20\tGenotypeConcordance\tchecked    28,530,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 17:16,621,175\n",
      "INFO\t2017-01-12 16:10:20\tGenotypeConcordance\tchecked    28,540,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 17:17,467,357\n",
      "INFO\t2017-01-12 16:10:21\tGenotypeConcordance\tchecked    28,550,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 17:18,466,336\n",
      "INFO\t2017-01-12 16:10:21\tGenotypeConcordance\tchecked    28,560,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 17:19,626,546\n",
      "INFO\t2017-01-12 16:10:21\tGenotypeConcordance\tchecked    28,570,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 17:20,540,503\n",
      "INFO\t2017-01-12 16:10:22\tGenotypeConcordance\tchecked    28,580,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 17:21,223,950\n",
      "INFO\t2017-01-12 16:10:22\tGenotypeConcordance\tchecked    28,590,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 17:21,666,993\n",
      "INFO\t2017-01-12 16:10:22\tGenotypeConcordance\tchecked    28,600,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 17:25,266,624\n",
      "INFO\t2017-01-12 16:10:23\tGenotypeConcordance\tchecked    28,610,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 17:25,762,907\n",
      "INFO\t2017-01-12 16:10:23\tGenotypeConcordance\tchecked    28,620,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 17:26,821,799\n",
      "INFO\t2017-01-12 16:10:23\tGenotypeConcordance\tchecked    28,630,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 17:27,981,897\n",
      "INFO\t2017-01-12 16:10:24\tGenotypeConcordance\tchecked    28,640,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 17:29,076,862\n",
      "INFO\t2017-01-12 16:10:24\tGenotypeConcordance\tchecked    28,650,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 17:29,989,696\n",
      "INFO\t2017-01-12 16:10:24\tGenotypeConcordance\tchecked    28,660,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 17:30,869,423\n",
      "INFO\t2017-01-12 16:10:24\tGenotypeConcordance\tchecked    28,670,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 17:31,773,020\n",
      "INFO\t2017-01-12 16:10:25\tGenotypeConcordance\tchecked    28,680,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 17:32,580,820\n",
      "INFO\t2017-01-12 16:10:25\tGenotypeConcordance\tchecked    28,690,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 17:33,488,856\n",
      "INFO\t2017-01-12 16:10:25\tGenotypeConcordance\tchecked    28,700,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 17:34,363,703\n",
      "INFO\t2017-01-12 16:10:26\tGenotypeConcordance\tchecked    28,710,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 17:35,372,194\n",
      "INFO\t2017-01-12 16:10:26\tGenotypeConcordance\tchecked    28,720,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 17:36,387,469\n",
      "INFO\t2017-01-12 16:10:26\tGenotypeConcordance\tchecked    28,730,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 17:37,268,133\n",
      "INFO\t2017-01-12 16:10:27\tGenotypeConcordance\tchecked    28,740,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 17:38,235,129\n",
      "INFO\t2017-01-12 16:10:27\tGenotypeConcordance\tchecked    28,750,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 17:39,187,406\n",
      "INFO\t2017-01-12 16:10:27\tGenotypeConcordance\tchecked    28,760,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 17:39,924,576\n",
      "INFO\t2017-01-12 16:10:28\tGenotypeConcordance\tchecked    28,770,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 17:41,028,959\n",
      "INFO\t2017-01-12 16:10:28\tGenotypeConcordance\tchecked    28,780,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 17:41,981,789\n",
      "INFO\t2017-01-12 16:10:28\tGenotypeConcordance\tchecked    28,790,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 17:42,964,475\n",
      "INFO\t2017-01-12 16:10:29\tGenotypeConcordance\tchecked    28,800,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 17:43,868,739\n",
      "INFO\t2017-01-12 16:10:29\tGenotypeConcordance\tchecked    28,810,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 17:44,970,433\n",
      "INFO\t2017-01-12 16:10:30\tGenotypeConcordance\tchecked    28,820,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 17:45,937,923\n",
      "INFO\t2017-01-12 16:10:30\tGenotypeConcordance\tchecked    28,830,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 17:46,977,468\n",
      "INFO\t2017-01-12 16:10:30\tGenotypeConcordance\tchecked    28,840,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 17:47,944,720\n",
      "INFO\t2017-01-12 16:10:31\tGenotypeConcordance\tchecked    28,850,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 17:48,803,520\n",
      "INFO\t2017-01-12 16:10:31\tGenotypeConcordance\tchecked    28,860,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 17:49,713,867\n",
      "INFO\t2017-01-12 16:10:31\tGenotypeConcordance\tchecked    28,870,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 17:50,649,633\n",
      "INFO\t2017-01-12 16:10:32\tGenotypeConcordance\tchecked    28,880,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 17:51,487,974\n",
      "INFO\t2017-01-12 16:10:32\tGenotypeConcordance\tchecked    28,890,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 17:52,341,783\n",
      "INFO\t2017-01-12 16:10:32\tGenotypeConcordance\tchecked    28,900,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 17:53,216,997\n",
      "INFO\t2017-01-12 16:10:33\tGenotypeConcordance\tchecked    28,910,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 17:54,121,113\n",
      "INFO\t2017-01-12 16:10:33\tGenotypeConcordance\tchecked    28,920,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 17:55,063,133\n",
      "INFO\t2017-01-12 16:10:33\tGenotypeConcordance\tchecked    28,930,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 17:55,897,724\n",
      "INFO\t2017-01-12 16:10:34\tGenotypeConcordance\tchecked    28,940,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 17:57,028,805\n",
      "INFO\t2017-01-12 16:10:34\tGenotypeConcordance\tchecked    28,950,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 17:58,212,750\n",
      "INFO\t2017-01-12 16:10:34\tGenotypeConcordance\tchecked    28,960,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 17:59,291,729\n",
      "INFO\t2017-01-12 16:10:35\tGenotypeConcordance\tchecked    28,970,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 17:60,282,418\n",
      "INFO\t2017-01-12 16:10:35\tGenotypeConcordance\tchecked    28,980,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 17:61,244,869\n",
      "INFO\t2017-01-12 16:10:35\tGenotypeConcordance\tchecked    28,990,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 17:62,167,506\n",
      "INFO\t2017-01-12 16:10:36\tGenotypeConcordance\tchecked    29,000,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 17:63,216,128\n",
      "INFO\t2017-01-12 16:10:36\tGenotypeConcordance\tchecked    29,010,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 17:64,023,781\n",
      "INFO\t2017-01-12 16:10:36\tGenotypeConcordance\tchecked    29,020,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 17:64,831,185\n",
      "INFO\t2017-01-12 16:10:37\tGenotypeConcordance\tchecked    29,030,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 17:65,704,455\n",
      "INFO\t2017-01-12 16:10:37\tGenotypeConcordance\tchecked    29,040,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 17:66,485,484\n",
      "INFO\t2017-01-12 16:10:37\tGenotypeConcordance\tchecked    29,050,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 17:67,385,862\n",
      "INFO\t2017-01-12 16:10:38\tGenotypeConcordance\tchecked    29,060,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 17:68,331,568\n",
      "INFO\t2017-01-12 16:10:38\tGenotypeConcordance\tchecked    29,070,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 17:69,174,196\n",
      "INFO\t2017-01-12 16:10:38\tGenotypeConcordance\tchecked    29,080,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 17:69,994,450\n",
      "INFO\t2017-01-12 16:10:39\tGenotypeConcordance\tchecked    29,090,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 17:70,796,942\n",
      "INFO\t2017-01-12 16:10:39\tGenotypeConcordance\tchecked    29,100,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 17:71,524,987\n",
      "INFO\t2017-01-12 16:10:39\tGenotypeConcordance\tchecked    29,110,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 17:72,237,435\n",
      "INFO\t2017-01-12 16:10:39\tGenotypeConcordance\tchecked    29,120,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 17:73,028,078\n",
      "INFO\t2017-01-12 16:10:40\tGenotypeConcordance\tchecked    29,130,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 17:73,857,022\n",
      "INFO\t2017-01-12 16:10:40\tGenotypeConcordance\tchecked    29,140,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 17:74,639,811\n",
      "INFO\t2017-01-12 16:10:40\tGenotypeConcordance\tchecked    29,150,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 17:75,331,648\n",
      "INFO\t2017-01-12 16:10:41\tGenotypeConcordance\tchecked    29,160,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 17:76,015,286\n",
      "INFO\t2017-01-12 16:10:41\tGenotypeConcordance\tchecked    29,170,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 17:76,639,316\n",
      "INFO\t2017-01-12 16:10:41\tGenotypeConcordance\tchecked    29,180,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 17:77,313,109\n",
      "INFO\t2017-01-12 16:10:42\tGenotypeConcordance\tchecked    29,190,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 17:78,104,137\n",
      "INFO\t2017-01-12 16:10:42\tGenotypeConcordance\tchecked    29,200,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 17:78,819,267\n",
      "INFO\t2017-01-12 16:10:43\tGenotypeConcordance\tchecked    29,210,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 17:79,439,361\n",
      "INFO\t2017-01-12 16:10:43\tGenotypeConcordance\tchecked    29,220,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 17:80,220,622\n",
      "INFO\t2017-01-12 16:10:43\tGenotypeConcordance\tchecked    29,230,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 17:80,874,543\n",
      "INFO\t2017-01-12 16:10:43\tGenotypeConcordance\tchecked    29,240,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 18:343,344\n",
      "INFO\t2017-01-12 16:10:44\tGenotypeConcordance\tchecked    29,250,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 18:1,070,098\n",
      "INFO\t2017-01-12 16:10:44\tGenotypeConcordance\tchecked    29,260,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 18:1,865,620\n",
      "INFO\t2017-01-12 16:10:45\tGenotypeConcordance\tchecked    29,270,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 18:2,609,293\n",
      "INFO\t2017-01-12 16:10:45\tGenotypeConcordance\tchecked    29,280,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 18:3,329,118\n",
      "INFO\t2017-01-12 16:10:45\tGenotypeConcordance\tchecked    29,290,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 18:4,089,873\n",
      "INFO\t2017-01-12 16:10:46\tGenotypeConcordance\tchecked    29,300,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 18:4,870,485\n",
      "INFO\t2017-01-12 16:10:46\tGenotypeConcordance\tchecked    29,310,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 18:5,647,385\n",
      "INFO\t2017-01-12 16:10:46\tGenotypeConcordance\tchecked    29,320,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 18:6,472,983\n",
      "INFO\t2017-01-12 16:10:47\tGenotypeConcordance\tchecked    29,330,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 18:7,193,858\n",
      "INFO\t2017-01-12 16:10:47\tGenotypeConcordance\tchecked    29,340,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 18:8,007,349\n",
      "INFO\t2017-01-12 16:10:47\tGenotypeConcordance\tchecked    29,350,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 18:8,753,976\n",
      "INFO\t2017-01-12 16:10:48\tGenotypeConcordance\tchecked    29,360,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 18:9,539,701\n",
      "INFO\t2017-01-12 16:10:48\tGenotypeConcordance\tchecked    29,370,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 18:10,290,966\n",
      "INFO\t2017-01-12 16:10:48\tGenotypeConcordance\tchecked    29,380,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 18:10,963,030\n",
      "INFO\t2017-01-12 16:10:49\tGenotypeConcordance\tchecked    29,390,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 18:11,679,246\n",
      "INFO\t2017-01-12 16:10:49\tGenotypeConcordance\tchecked    29,400,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 18:12,387,659\n",
      "INFO\t2017-01-12 16:10:49\tGenotypeConcordance\tchecked    29,410,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 18:13,217,897\n",
      "INFO\t2017-01-12 16:10:50\tGenotypeConcordance\tchecked    29,420,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 18:14,010,580\n",
      "INFO\t2017-01-12 16:10:50\tGenotypeConcordance\tchecked    29,430,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 18:14,539,793\n",
      "INFO\t2017-01-12 16:10:50\tGenotypeConcordance\tchecked    29,440,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 18:15,203,733\n",
      "INFO\t2017-01-12 16:10:51\tGenotypeConcordance\tchecked    29,450,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 18:18,973,557\n",
      "INFO\t2017-01-12 16:10:51\tGenotypeConcordance\tchecked    29,460,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 18:20,028,670\n",
      "INFO\t2017-01-12 16:10:51\tGenotypeConcordance\tchecked    29,470,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 18:20,946,105\n",
      "INFO\t2017-01-12 16:10:52\tGenotypeConcordance\tchecked    29,480,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 18:21,934,157\n",
      "INFO\t2017-01-12 16:10:52\tGenotypeConcordance\tchecked    29,490,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 18:22,747,289\n",
      "INFO\t2017-01-12 16:10:52\tGenotypeConcordance\tchecked    29,500,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 18:23,633,640\n",
      "INFO\t2017-01-12 16:10:53\tGenotypeConcordance\tchecked    29,510,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 18:24,508,415\n",
      "INFO\t2017-01-12 16:10:53\tGenotypeConcordance\tchecked    29,520,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 18:25,468,124\n",
      "INFO\t2017-01-12 16:10:53\tGenotypeConcordance\tchecked    29,530,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 18:26,487,449\n",
      "INFO\t2017-01-12 16:10:54\tGenotypeConcordance\tchecked    29,540,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 18:27,351,117\n",
      "INFO\t2017-01-12 16:10:54\tGenotypeConcordance\tchecked    29,550,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 18:28,160,788\n",
      "INFO\t2017-01-12 16:10:54\tGenotypeConcordance\tchecked    29,560,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 18:28,984,706\n",
      "INFO\t2017-01-12 16:10:55\tGenotypeConcordance\tchecked    29,570,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 18:29,811,919\n",
      "INFO\t2017-01-12 16:10:55\tGenotypeConcordance\tchecked    29,580,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 18:30,842,774\n",
      "INFO\t2017-01-12 16:10:55\tGenotypeConcordance\tchecked    29,590,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 18:31,891,898\n",
      "INFO\t2017-01-12 16:10:56\tGenotypeConcordance\tchecked    29,600,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 18:32,871,256\n",
      "INFO\t2017-01-12 16:10:56\tGenotypeConcordance\tchecked    29,610,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 18:33,759,114\n",
      "INFO\t2017-01-12 16:10:56\tGenotypeConcordance\tchecked    29,620,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 18:34,809,470\n",
      "INFO\t2017-01-12 16:10:57\tGenotypeConcordance\tchecked    29,630,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 18:35,764,592\n",
      "INFO\t2017-01-12 16:10:57\tGenotypeConcordance\tchecked    29,640,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 18:36,704,958\n",
      "INFO\t2017-01-12 16:10:57\tGenotypeConcordance\tchecked    29,650,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 18:37,727,036\n",
      "INFO\t2017-01-12 16:10:58\tGenotypeConcordance\tchecked    29,660,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 18:38,594,224\n",
      "INFO\t2017-01-12 16:10:58\tGenotypeConcordance\tchecked    29,670,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 18:39,510,120\n",
      "INFO\t2017-01-12 16:10:58\tGenotypeConcordance\tchecked    29,680,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 18:40,425,012\n",
      "INFO\t2017-01-12 16:10:59\tGenotypeConcordance\tchecked    29,690,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 18:41,342,975\n",
      "INFO\t2017-01-12 16:10:59\tGenotypeConcordance\tchecked    29,700,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 18:42,285,729\n",
      "INFO\t2017-01-12 16:10:59\tGenotypeConcordance\tchecked    29,710,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 18:43,228,567\n",
      "INFO\t2017-01-12 16:11:00\tGenotypeConcordance\tchecked    29,720,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 18:44,072,052\n",
      "INFO\t2017-01-12 16:11:00\tGenotypeConcordance\tchecked    29,730,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 18:44,993,364\n",
      "INFO\t2017-01-12 16:11:00\tGenotypeConcordance\tchecked    29,740,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 18:45,858,505\n",
      "INFO\t2017-01-12 16:11:01\tGenotypeConcordance\tchecked    29,750,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 18:46,697,891\n",
      "INFO\t2017-01-12 16:11:01\tGenotypeConcordance\tchecked    29,760,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 18:47,504,347\n",
      "INFO\t2017-01-12 16:11:01\tGenotypeConcordance\tchecked    29,770,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 18:48,306,936\n",
      "INFO\t2017-01-12 16:11:02\tGenotypeConcordance\tchecked    29,780,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 18:49,182,209\n",
      "INFO\t2017-01-12 16:11:02\tGenotypeConcordance\tchecked    29,790,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 18:50,005,954\n",
      "INFO\t2017-01-12 16:11:02\tGenotypeConcordance\tchecked    29,800,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 18:50,738,582\n",
      "INFO\t2017-01-12 16:11:03\tGenotypeConcordance\tchecked    29,810,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 18:51,505,483\n",
      "INFO\t2017-01-12 16:11:03\tGenotypeConcordance\tchecked    29,820,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 18:52,532,162\n",
      "INFO\t2017-01-12 16:11:03\tGenotypeConcordance\tchecked    29,830,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 18:53,642,248\n",
      "INFO\t2017-01-12 16:11:04\tGenotypeConcordance\tchecked    29,840,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 18:54,590,715\n",
      "INFO\t2017-01-12 16:11:04\tGenotypeConcordance\tchecked    29,850,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 18:55,411,554\n",
      "INFO\t2017-01-12 16:11:05\tGenotypeConcordance\tchecked    29,860,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 18:56,203,252\n",
      "INFO\t2017-01-12 16:11:05\tGenotypeConcordance\tchecked    29,870,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 18:57,014,381\n",
      "INFO\t2017-01-12 16:11:05\tGenotypeConcordance\tchecked    29,880,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 18:57,725,641\n",
      "INFO\t2017-01-12 16:11:06\tGenotypeConcordance\tchecked    29,890,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 18:58,511,600\n",
      "INFO\t2017-01-12 16:11:06\tGenotypeConcordance\tchecked    29,900,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 18:59,396,194\n",
      "INFO\t2017-01-12 16:11:06\tGenotypeConcordance\tchecked    29,910,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 18:60,281,207\n",
      "INFO\t2017-01-12 16:11:07\tGenotypeConcordance\tchecked    29,920,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 18:61,218,269\n",
      "INFO\t2017-01-12 16:11:07\tGenotypeConcordance\tchecked    29,930,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 18:62,044,045\n",
      "INFO\t2017-01-12 16:11:07\tGenotypeConcordance\tchecked    29,940,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 18:62,834,950\n",
      "INFO\t2017-01-12 16:11:08\tGenotypeConcordance\tchecked    29,950,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 18:63,613,087\n",
      "INFO\t2017-01-12 16:11:08\tGenotypeConcordance\tchecked    29,960,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 18:64,401,957\n",
      "INFO\t2017-01-12 16:11:08\tGenotypeConcordance\tchecked    29,970,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 18:65,212,889\n",
      "INFO\t2017-01-12 16:11:09\tGenotypeConcordance\tchecked    29,980,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 18:65,938,039\n",
      "INFO\t2017-01-12 16:11:09\tGenotypeConcordance\tchecked    29,990,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 18:66,728,756\n",
      "INFO\t2017-01-12 16:11:09\tGenotypeConcordance\tchecked    30,000,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 18:67,523,260\n",
      "INFO\t2017-01-12 16:11:10\tGenotypeConcordance\tchecked    30,010,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 18:68,372,694\n",
      "INFO\t2017-01-12 16:11:10\tGenotypeConcordance\tchecked    30,020,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 18:69,159,389\n",
      "INFO\t2017-01-12 16:11:11\tGenotypeConcordance\tchecked    30,030,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 18:69,913,590\n",
      "INFO\t2017-01-12 16:11:11\tGenotypeConcordance\tchecked    30,040,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 18:70,641,009\n",
      "INFO\t2017-01-12 16:11:11\tGenotypeConcordance\tchecked    30,050,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 18:71,351,521\n",
      "INFO\t2017-01-12 16:11:12\tGenotypeConcordance\tchecked    30,060,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 18:72,108,494\n",
      "INFO\t2017-01-12 16:11:12\tGenotypeConcordance\tchecked    30,070,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 18:73,031,691\n",
      "INFO\t2017-01-12 16:11:13\tGenotypeConcordance\tchecked    30,080,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 18:73,760,114\n",
      "INFO\t2017-01-12 16:11:13\tGenotypeConcordance\tchecked    30,090,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 18:74,457,620\n",
      "INFO\t2017-01-12 16:11:13\tGenotypeConcordance\tchecked    30,100,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 18:75,134,510\n",
      "INFO\t2017-01-12 16:11:14\tGenotypeConcordance\tchecked    30,110,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 18:75,869,682\n",
      "INFO\t2017-01-12 16:11:14\tGenotypeConcordance\tchecked    30,120,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 18:76,457,627\n",
      "INFO\t2017-01-12 16:11:14\tGenotypeConcordance\tchecked    30,130,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 18:77,169,216\n",
      "INFO\t2017-01-12 16:11:15\tGenotypeConcordance\tchecked    30,140,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 18:77,790,689\n",
      "INFO\t2017-01-12 16:11:15\tGenotypeConcordance\tchecked    30,150,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 19:597,941\n",
      "INFO\t2017-01-12 16:11:15\tGenotypeConcordance\tchecked    30,160,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 19:1,139,469\n",
      "INFO\t2017-01-12 16:11:16\tGenotypeConcordance\tchecked    30,170,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 19:1,819,007\n",
      "INFO\t2017-01-12 16:11:16\tGenotypeConcordance\tchecked    30,180,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 19:2,481,012\n",
      "INFO\t2017-01-12 16:11:16\tGenotypeConcordance\tchecked    30,190,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 19:3,080,333\n",
      "INFO\t2017-01-12 16:11:17\tGenotypeConcordance\tchecked    30,200,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 19:3,742,710\n",
      "INFO\t2017-01-12 16:11:17\tGenotypeConcordance\tchecked    30,210,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 19:4,451,560\n",
      "INFO\t2017-01-12 16:11:17\tGenotypeConcordance\tchecked    30,220,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 19:5,123,070\n",
      "INFO\t2017-01-12 16:11:18\tGenotypeConcordance\tchecked    30,230,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 19:5,858,371\n",
      "INFO\t2017-01-12 16:11:18\tGenotypeConcordance\tchecked    30,240,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 19:6,597,715\n",
      "INFO\t2017-01-12 16:11:19\tGenotypeConcordance\tchecked    30,250,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 19:7,251,423\n",
      "INFO\t2017-01-12 16:11:19\tGenotypeConcordance\tchecked    30,260,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 19:7,999,362\n",
      "INFO\t2017-01-12 16:11:20\tGenotypeConcordance\tchecked    30,270,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 19:8,755,058\n",
      "INFO\t2017-01-12 16:11:20\tGenotypeConcordance\tchecked    30,280,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 19:9,402,550\n",
      "INFO\t2017-01-12 16:11:20\tGenotypeConcordance\tchecked    30,290,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 19:10,158,124\n",
      "INFO\t2017-01-12 16:11:21\tGenotypeConcordance\tchecked    30,300,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 19:11,026,045\n",
      "INFO\t2017-01-12 16:11:21\tGenotypeConcordance\tchecked    30,310,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 19:11,817,808\n",
      "INFO\t2017-01-12 16:11:21\tGenotypeConcordance\tchecked    30,320,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 19:12,669,438\n",
      "INFO\t2017-01-12 16:11:22\tGenotypeConcordance\tchecked    30,330,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 19:13,579,418\n",
      "INFO\t2017-01-12 16:11:22\tGenotypeConcordance\tchecked    30,340,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 19:14,379,965\n",
      "INFO\t2017-01-12 16:11:22\tGenotypeConcordance\tchecked    30,350,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 19:15,084,896\n",
      "INFO\t2017-01-12 16:11:23\tGenotypeConcordance\tchecked    30,360,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 19:15,835,206\n",
      "INFO\t2017-01-12 16:11:23\tGenotypeConcordance\tchecked    30,370,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 19:16,527,917\n",
      "INFO\t2017-01-12 16:11:23\tGenotypeConcordance\tchecked    30,380,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 19:17,324,751\n",
      "INFO\t2017-01-12 16:11:24\tGenotypeConcordance\tchecked    30,390,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 19:18,073,775\n",
      "INFO\t2017-01-12 16:11:24\tGenotypeConcordance\tchecked    30,400,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 19:18,929,561\n",
      "INFO\t2017-01-12 16:11:24\tGenotypeConcordance\tchecked    30,410,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 19:19,891,687\n",
      "INFO\t2017-01-12 16:11:25\tGenotypeConcordance\tchecked    30,420,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 19:20,688,283\n",
      "INFO\t2017-01-12 16:11:25\tGenotypeConcordance\tchecked    30,430,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 19:21,320,316\n",
      "INFO\t2017-01-12 16:11:25\tGenotypeConcordance\tchecked    30,440,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 19:21,957,865\n",
      "INFO\t2017-01-12 16:11:26\tGenotypeConcordance\tchecked    30,450,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 19:22,646,934\n",
      "INFO\t2017-01-12 16:11:26\tGenotypeConcordance\tchecked    30,460,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 19:23,344,209\n",
      "INFO\t2017-01-12 16:11:26\tGenotypeConcordance\tchecked    30,470,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 19:24,030,473\n",
      "INFO\t2017-01-12 16:11:27\tGenotypeConcordance\tchecked    30,480,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 19:27,733,063\n",
      "INFO\t2017-01-12 16:11:27\tGenotypeConcordance\tchecked    30,490,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 19:28,165,191\n",
      "INFO\t2017-01-12 16:11:28\tGenotypeConcordance\tchecked    30,500,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 19:28,897,587\n",
      "INFO\t2017-01-12 16:11:28\tGenotypeConcordance\tchecked    30,510,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 19:29,567,472\n",
      "INFO\t2017-01-12 16:11:28\tGenotypeConcordance\tchecked    30,520,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 19:30,274,191\n",
      "INFO\t2017-01-12 16:11:29\tGenotypeConcordance\tchecked    30,530,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 19:31,140,007\n",
      "INFO\t2017-01-12 16:11:29\tGenotypeConcordance\tchecked    30,540,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 19:32,050,829\n",
      "INFO\t2017-01-12 16:11:29\tGenotypeConcordance\tchecked    30,550,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 19:32,951,841\n",
      "INFO\t2017-01-12 16:11:29\tGenotypeConcordance\tchecked    30,560,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 19:33,728,968\n",
      "INFO\t2017-01-12 16:11:30\tGenotypeConcordance\tchecked    30,570,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 19:34,514,806\n",
      "INFO\t2017-01-12 16:11:30\tGenotypeConcordance\tchecked    30,580,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 19:35,372,415\n",
      "INFO\t2017-01-12 16:11:31\tGenotypeConcordance\tchecked    30,590,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 19:36,188,129\n",
      "INFO\t2017-01-12 16:11:31\tGenotypeConcordance\tchecked    30,600,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 19:37,050,707\n",
      "INFO\t2017-01-12 16:11:31\tGenotypeConcordance\tchecked    30,610,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 19:38,073,098\n",
      "INFO\t2017-01-12 16:11:32\tGenotypeConcordance\tchecked    30,620,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 19:39,000,236\n",
      "INFO\t2017-01-12 16:11:32\tGenotypeConcordance\tchecked    30,630,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 19:39,738,666\n",
      "INFO\t2017-01-12 16:11:32\tGenotypeConcordance\tchecked    30,640,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 19:40,575,008\n",
      "INFO\t2017-01-12 16:11:33\tGenotypeConcordance\tchecked    30,650,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 19:41,438,722\n",
      "INFO\t2017-01-12 16:11:33\tGenotypeConcordance\tchecked    30,660,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 19:42,165,671\n",
      "INFO\t2017-01-12 16:11:34\tGenotypeConcordance\tchecked    30,670,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 19:43,266,640\n",
      "INFO\t2017-01-12 16:11:34\tGenotypeConcordance\tchecked    30,680,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 19:43,697,726\n",
      "INFO\t2017-01-12 16:11:34\tGenotypeConcordance\tchecked    30,690,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 19:44,428,911\n",
      "INFO\t2017-01-12 16:11:34\tGenotypeConcordance\tchecked    30,700,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 19:45,216,026\n",
      "INFO\t2017-01-12 16:11:35\tGenotypeConcordance\tchecked    30,710,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 19:45,974,668\n",
      "INFO\t2017-01-12 16:11:35\tGenotypeConcordance\tchecked    30,720,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 19:46,785,021\n",
      "INFO\t2017-01-12 16:11:35\tGenotypeConcordance\tchecked    30,730,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 19:47,580,641\n",
      "INFO\t2017-01-12 16:11:36\tGenotypeConcordance\tchecked    30,740,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 19:48,381,365\n",
      "INFO\t2017-01-12 16:11:36\tGenotypeConcordance\tchecked    30,750,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 19:49,114,433\n",
      "INFO\t2017-01-12 16:11:36\tGenotypeConcordance\tchecked    30,760,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 19:49,808,849\n",
      "INFO\t2017-01-12 16:11:37\tGenotypeConcordance\tchecked    30,770,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 19:50,593,858\n",
      "INFO\t2017-01-12 16:11:37\tGenotypeConcordance\tchecked    30,780,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 19:51,321,827\n",
      "INFO\t2017-01-12 16:11:37\tGenotypeConcordance\tchecked    30,790,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 19:52,041,801\n",
      "INFO\t2017-01-12 16:11:38\tGenotypeConcordance\tchecked    30,800,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 19:52,798,735\n",
      "INFO\t2017-01-12 16:11:38\tGenotypeConcordance\tchecked    30,810,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 19:53,359,005\n",
      "INFO\t2017-01-12 16:11:38\tGenotypeConcordance\tchecked    30,820,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 19:53,959,745\n",
      "INFO\t2017-01-12 16:11:39\tGenotypeConcordance\tchecked    30,830,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 19:54,536,734\n",
      "INFO\t2017-01-12 16:11:39\tGenotypeConcordance\tchecked    30,840,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 19:55,098,680\n",
      "INFO\t2017-01-12 16:11:39\tGenotypeConcordance\tchecked    30,850,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 19:55,623,635\n",
      "INFO\t2017-01-12 16:11:40\tGenotypeConcordance\tchecked    30,860,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 19:56,298,003\n",
      "INFO\t2017-01-12 16:11:40\tGenotypeConcordance\tchecked    30,870,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 19:56,863,145\n",
      "INFO\t2017-01-12 16:11:40\tGenotypeConcordance\tchecked    30,880,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 19:57,660,692\n",
      "INFO\t2017-01-12 16:11:41\tGenotypeConcordance\tchecked    30,890,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 19:58,503,369\n",
      "INFO\t2017-01-12 16:11:41\tGenotypeConcordance\tchecked    30,900,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 20:223,652\n",
      "INFO\t2017-01-12 16:11:41\tGenotypeConcordance\tchecked    30,910,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 20:923,176\n",
      "INFO\t2017-01-12 16:11:42\tGenotypeConcordance\tchecked    30,920,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 20:1,796,112\n",
      "INFO\t2017-01-12 16:11:42\tGenotypeConcordance\tchecked    30,930,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 20:2,579,107\n",
      "INFO\t2017-01-12 16:11:42\tGenotypeConcordance\tchecked    30,940,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 20:3,409,428\n",
      "INFO\t2017-01-12 16:11:43\tGenotypeConcordance\tchecked    30,950,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 20:4,222,977\n",
      "INFO\t2017-01-12 16:11:43\tGenotypeConcordance\tchecked    30,960,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 20:4,996,506\n",
      "INFO\t2017-01-12 16:11:43\tGenotypeConcordance\tchecked    30,970,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 20:5,730,566\n",
      "INFO\t2017-01-12 16:11:44\tGenotypeConcordance\tchecked    30,980,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 20:6,491,219\n",
      "INFO\t2017-01-12 16:11:44\tGenotypeConcordance\tchecked    30,990,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 20:7,293,914\n",
      "INFO\t2017-01-12 16:11:44\tGenotypeConcordance\tchecked    31,000,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 20:8,106,464\n",
      "INFO\t2017-01-12 16:11:45\tGenotypeConcordance\tchecked    31,010,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 20:8,981,641\n",
      "INFO\t2017-01-12 16:11:45\tGenotypeConcordance\tchecked    31,020,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 20:9,917,409\n",
      "INFO\t2017-01-12 16:11:45\tGenotypeConcordance\tchecked    31,030,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 20:10,823,171\n",
      "INFO\t2017-01-12 16:11:46\tGenotypeConcordance\tchecked    31,040,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 20:11,704,160\n",
      "INFO\t2017-01-12 16:11:46\tGenotypeConcordance\tchecked    31,050,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 20:12,497,212\n",
      "INFO\t2017-01-12 16:11:46\tGenotypeConcordance\tchecked    31,060,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 20:13,294,098\n",
      "INFO\t2017-01-12 16:11:47\tGenotypeConcordance\tchecked    31,070,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 20:14,314,039\n",
      "INFO\t2017-01-12 16:11:47\tGenotypeConcordance\tchecked    31,080,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 20:15,236,000\n",
      "INFO\t2017-01-12 16:11:47\tGenotypeConcordance\tchecked    31,090,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 20:16,034,829\n",
      "INFO\t2017-01-12 16:11:47\tGenotypeConcordance\tchecked    31,100,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 20:16,828,207\n",
      "INFO\t2017-01-12 16:11:48\tGenotypeConcordance\tchecked    31,110,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 20:17,629,161\n",
      "INFO\t2017-01-12 16:11:48\tGenotypeConcordance\tchecked    31,120,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 20:18,408,862\n",
      "INFO\t2017-01-12 16:11:49\tGenotypeConcordance\tchecked    31,130,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 20:19,227,230\n",
      "INFO\t2017-01-12 16:11:49\tGenotypeConcordance\tchecked    31,140,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 20:20,041,868\n",
      "INFO\t2017-01-12 16:11:49\tGenotypeConcordance\tchecked    31,150,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 20:21,016,811\n",
      "INFO\t2017-01-12 16:11:49\tGenotypeConcordance\tchecked    31,160,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 20:22,030,882\n",
      "INFO\t2017-01-12 16:11:50\tGenotypeConcordance\tchecked    31,170,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 20:22,828,996\n",
      "INFO\t2017-01-12 16:11:50\tGenotypeConcordance\tchecked    31,180,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 20:23,627,632\n",
      "INFO\t2017-01-12 16:11:50\tGenotypeConcordance\tchecked    31,190,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 20:24,237,109\n",
      "INFO\t2017-01-12 16:11:51\tGenotypeConcordance\tchecked    31,200,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 20:24,984,699\n",
      "INFO\t2017-01-12 16:11:51\tGenotypeConcordance\tchecked    31,210,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 20:25,764,616\n",
      "INFO\t2017-01-12 16:11:51\tGenotypeConcordance\tchecked    31,220,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 20:26,138,588\n",
      "INFO\t2017-01-12 16:11:52\tGenotypeConcordance\tchecked    31,230,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 20:29,512,482\n",
      "INFO\t2017-01-12 16:11:52\tGenotypeConcordance\tchecked    31,240,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 20:29,913,360\n",
      "INFO\t2017-01-12 16:11:52\tGenotypeConcordance\tchecked    31,250,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 20:30,972,774\n",
      "INFO\t2017-01-12 16:11:53\tGenotypeConcordance\tchecked    31,260,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 20:31,863,599\n",
      "INFO\t2017-01-12 16:11:53\tGenotypeConcordance\tchecked    31,270,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 20:32,938,216\n",
      "INFO\t2017-01-12 16:11:53\tGenotypeConcordance\tchecked    31,280,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 20:34,033,548\n",
      "INFO\t2017-01-12 16:11:54\tGenotypeConcordance\tchecked    31,290,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 20:35,192,121\n",
      "INFO\t2017-01-12 16:11:54\tGenotypeConcordance\tchecked    31,300,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 20:36,185,592\n",
      "INFO\t2017-01-12 16:11:54\tGenotypeConcordance\tchecked    31,310,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 20:37,053,089\n",
      "INFO\t2017-01-12 16:11:55\tGenotypeConcordance\tchecked    31,320,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 20:38,026,027\n",
      "INFO\t2017-01-12 16:11:55\tGenotypeConcordance\tchecked    31,330,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 20:38,994,449\n",
      "INFO\t2017-01-12 16:11:55\tGenotypeConcordance\tchecked    31,340,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 20:39,966,631\n",
      "INFO\t2017-01-12 16:11:56\tGenotypeConcordance\tchecked    31,350,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 20:40,879,061\n",
      "INFO\t2017-01-12 16:11:56\tGenotypeConcordance\tchecked    31,360,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 20:41,702,152\n",
      "INFO\t2017-01-12 16:11:56\tGenotypeConcordance\tchecked    31,370,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 20:42,567,251\n",
      "INFO\t2017-01-12 16:11:57\tGenotypeConcordance\tchecked    31,380,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 20:43,473,176\n",
      "INFO\t2017-01-12 16:11:57\tGenotypeConcordance\tchecked    31,390,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 20:44,361,076\n",
      "INFO\t2017-01-12 16:11:57\tGenotypeConcordance\tchecked    31,400,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 20:45,253,237\n",
      "INFO\t2017-01-12 16:11:58\tGenotypeConcordance\tchecked    31,410,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 20:46,051,930\n",
      "INFO\t2017-01-12 16:11:58\tGenotypeConcordance\tchecked    31,420,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 20:46,833,219\n",
      "INFO\t2017-01-12 16:11:58\tGenotypeConcordance\tchecked    31,430,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 20:47,596,482\n",
      "INFO\t2017-01-12 16:11:59\tGenotypeConcordance\tchecked    31,440,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 20:48,491,474\n",
      "INFO\t2017-01-12 16:11:59\tGenotypeConcordance\tchecked    31,450,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 20:49,333,954\n",
      "INFO\t2017-01-12 16:11:59\tGenotypeConcordance\tchecked    31,460,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 20:50,169,212\n",
      "INFO\t2017-01-12 16:12:00\tGenotypeConcordance\tchecked    31,470,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 20:50,947,016\n",
      "INFO\t2017-01-12 16:12:00\tGenotypeConcordance\tchecked    31,480,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 20:51,863,191\n",
      "INFO\t2017-01-12 16:12:00\tGenotypeConcordance\tchecked    31,490,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 20:52,656,660\n",
      "INFO\t2017-01-12 16:12:01\tGenotypeConcordance\tchecked    31,500,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 20:53,431,548\n",
      "INFO\t2017-01-12 16:12:01\tGenotypeConcordance\tchecked    31,510,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 20:54,194,776\n",
      "INFO\t2017-01-12 16:12:01\tGenotypeConcordance\tchecked    31,520,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 20:54,929,764\n",
      "INFO\t2017-01-12 16:12:01\tGenotypeConcordance\tchecked    31,530,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 20:55,668,324\n",
      "INFO\t2017-01-12 16:12:02\tGenotypeConcordance\tchecked    31,540,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 20:56,432,447\n",
      "INFO\t2017-01-12 16:12:02\tGenotypeConcordance\tchecked    31,550,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 20:57,224,982\n",
      "INFO\t2017-01-12 16:12:02\tGenotypeConcordance\tchecked    31,560,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 20:58,106,502\n",
      "INFO\t2017-01-12 16:12:03\tGenotypeConcordance\tchecked    31,570,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 20:58,889,224\n",
      "INFO\t2017-01-12 16:12:03\tGenotypeConcordance\tchecked    31,580,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 20:59,632,459\n",
      "INFO\t2017-01-12 16:12:03\tGenotypeConcordance\tchecked    31,590,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 20:60,284,329\n",
      "INFO\t2017-01-12 16:12:04\tGenotypeConcordance\tchecked    31,600,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 20:60,939,284\n",
      "INFO\t2017-01-12 16:12:04\tGenotypeConcordance\tchecked    31,610,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 20:61,662,583\n",
      "INFO\t2017-01-12 16:12:04\tGenotypeConcordance\tchecked    31,620,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 20:62,298,541\n",
      "INFO\t2017-01-12 16:12:04\tGenotypeConcordance\tchecked    31,630,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 21:9,481,371\n",
      "INFO\t2017-01-12 16:12:05\tGenotypeConcordance\tchecked    31,640,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 21:9,955,419\n",
      "INFO\t2017-01-12 16:12:05\tGenotypeConcordance\tchecked    31,650,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 21:10,706,664\n",
      "INFO\t2017-01-12 16:12:06\tGenotypeConcordance\tchecked    31,660,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 21:10,868,983\n",
      "INFO\t2017-01-12 16:12:06\tGenotypeConcordance\tchecked    31,670,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 21:14,361,793\n",
      "INFO\t2017-01-12 16:12:06\tGenotypeConcordance\tchecked    31,680,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 21:15,218,151\n",
      "INFO\t2017-01-12 16:12:07\tGenotypeConcordance\tchecked    31,690,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 21:15,738,677\n",
      "INFO\t2017-01-12 16:12:07\tGenotypeConcordance\tchecked    31,700,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 21:16,535,727\n",
      "INFO\t2017-01-12 16:12:07\tGenotypeConcordance\tchecked    31,710,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 21:17,393,193\n",
      "INFO\t2017-01-12 16:12:07\tGenotypeConcordance\tchecked    31,720,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 21:18,190,322\n",
      "INFO\t2017-01-12 16:12:08\tGenotypeConcordance\tchecked    31,730,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 21:18,817,058\n",
      "INFO\t2017-01-12 16:12:08\tGenotypeConcordance\tchecked    31,740,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 21:19,594,702\n",
      "INFO\t2017-01-12 16:12:08\tGenotypeConcordance\tchecked    31,750,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 21:20,348,469\n",
      "INFO\t2017-01-12 16:12:09\tGenotypeConcordance\tchecked    31,760,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 21:21,104,416\n",
      "INFO\t2017-01-12 16:12:09\tGenotypeConcordance\tchecked    31,770,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 21:21,823,904\n",
      "INFO\t2017-01-12 16:12:09\tGenotypeConcordance\tchecked    31,780,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 21:22,478,483\n",
      "INFO\t2017-01-12 16:12:10\tGenotypeConcordance\tchecked    31,790,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 21:23,231,909\n",
      "INFO\t2017-01-12 16:12:10\tGenotypeConcordance\tchecked    31,800,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 21:24,000,969\n",
      "INFO\t2017-01-12 16:12:10\tGenotypeConcordance\tchecked    31,810,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 21:24,713,951\n",
      "INFO\t2017-01-12 16:12:10\tGenotypeConcordance\tchecked    31,820,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 21:25,364,632\n",
      "INFO\t2017-01-12 16:12:11\tGenotypeConcordance\tchecked    31,830,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 21:26,050,834\n",
      "INFO\t2017-01-12 16:12:11\tGenotypeConcordance\tchecked    31,840,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 21:26,930,332\n",
      "INFO\t2017-01-12 16:12:11\tGenotypeConcordance\tchecked    31,850,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 21:27,738,422\n",
      "INFO\t2017-01-12 16:12:12\tGenotypeConcordance\tchecked    31,860,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 21:28,560,262\n",
      "INFO\t2017-01-12 16:12:12\tGenotypeConcordance\tchecked    31,870,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 21:29,329,556\n",
      "INFO\t2017-01-12 16:12:13\tGenotypeConcordance\tchecked    31,880,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 21:30,094,309\n",
      "INFO\t2017-01-12 16:12:13\tGenotypeConcordance\tchecked    31,890,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 21:30,946,049\n",
      "INFO\t2017-01-12 16:12:13\tGenotypeConcordance\tchecked    31,900,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 21:31,740,553\n",
      "INFO\t2017-01-12 16:12:14\tGenotypeConcordance\tchecked    31,910,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 21:32,569,974\n",
      "INFO\t2017-01-12 16:12:14\tGenotypeConcordance\tchecked    31,920,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 21:33,365,595\n",
      "INFO\t2017-01-12 16:12:14\tGenotypeConcordance\tchecked    31,930,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 21:34,237,858\n",
      "INFO\t2017-01-12 16:12:15\tGenotypeConcordance\tchecked    31,940,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 21:35,132,301\n",
      "INFO\t2017-01-12 16:12:15\tGenotypeConcordance\tchecked    31,950,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 21:35,965,934\n",
      "INFO\t2017-01-12 16:12:15\tGenotypeConcordance\tchecked    31,960,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 21:36,777,485\n",
      "INFO\t2017-01-12 16:12:15\tGenotypeConcordance\tchecked    31,970,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 21:37,622,270\n",
      "INFO\t2017-01-12 16:12:16\tGenotypeConcordance\tchecked    31,980,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 21:38,400,659\n",
      "INFO\t2017-01-12 16:12:16\tGenotypeConcordance\tchecked    31,990,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 21:39,250,687\n",
      "INFO\t2017-01-12 16:12:16\tGenotypeConcordance\tchecked    32,000,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 21:40,142,443\n",
      "INFO\t2017-01-12 16:12:17\tGenotypeConcordance\tchecked    32,010,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 21:40,926,096\n",
      "INFO\t2017-01-12 16:12:17\tGenotypeConcordance\tchecked    32,020,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 21:41,637,410\n",
      "INFO\t2017-01-12 16:12:18\tGenotypeConcordance\tchecked    32,030,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 21:42,356,760\n",
      "INFO\t2017-01-12 16:12:18\tGenotypeConcordance\tchecked    32,040,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 21:43,145,704\n",
      "INFO\t2017-01-12 16:12:18\tGenotypeConcordance\tchecked    32,050,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 21:43,798,797\n",
      "INFO\t2017-01-12 16:12:19\tGenotypeConcordance\tchecked    32,060,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 21:44,466,913\n",
      "INFO\t2017-01-12 16:12:19\tGenotypeConcordance\tchecked    32,070,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 21:45,271,831\n",
      "INFO\t2017-01-12 16:12:19\tGenotypeConcordance\tchecked    32,080,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 21:45,959,452\n",
      "INFO\t2017-01-12 16:12:20\tGenotypeConcordance\tchecked    32,090,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 21:46,567,946\n",
      "INFO\t2017-01-12 16:12:20\tGenotypeConcordance\tchecked    32,100,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 21:47,259,186\n",
      "INFO\t2017-01-12 16:12:20\tGenotypeConcordance\tchecked    32,110,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 21:47,920,912\n",
      "INFO\t2017-01-12 16:12:20\tGenotypeConcordance\tchecked    32,120,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 22:16,853,596\n",
      "INFO\t2017-01-12 16:12:21\tGenotypeConcordance\tchecked    32,130,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 22:17,318,864\n",
      "INFO\t2017-01-12 16:12:21\tGenotypeConcordance\tchecked    32,140,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 22:17,958,427\n",
      "INFO\t2017-01-12 16:12:22\tGenotypeConcordance\tchecked    32,150,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 22:18,639,178\n",
      "INFO\t2017-01-12 16:12:22\tGenotypeConcordance\tchecked    32,160,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 22:19,410,846\n",
      "INFO\t2017-01-12 16:12:22\tGenotypeConcordance\tchecked    32,170,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 22:20,177,320\n",
      "INFO\t2017-01-12 16:12:23\tGenotypeConcordance\tchecked    32,180,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 22:21,158,858\n",
      "INFO\t2017-01-12 16:12:23\tGenotypeConcordance\tchecked    32,190,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 22:22,082,603\n",
      "INFO\t2017-01-12 16:12:23\tGenotypeConcordance\tchecked    32,200,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 22:22,714,989\n",
      "INFO\t2017-01-12 16:12:24\tGenotypeConcordance\tchecked    32,210,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 22:23,298,763\n",
      "INFO\t2017-01-12 16:12:24\tGenotypeConcordance\tchecked    32,220,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 22:23,940,670\n",
      "INFO\t2017-01-12 16:12:24\tGenotypeConcordance\tchecked    32,230,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 22:24,702,549\n",
      "INFO\t2017-01-12 16:12:25\tGenotypeConcordance\tchecked    32,240,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 22:25,465,840\n",
      "INFO\t2017-01-12 16:12:25\tGenotypeConcordance\tchecked    32,250,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 22:26,130,523\n",
      "INFO\t2017-01-12 16:12:25\tGenotypeConcordance\tchecked    32,260,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 22:26,904,992\n",
      "INFO\t2017-01-12 16:12:26\tGenotypeConcordance\tchecked    32,270,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 22:27,646,849\n",
      "INFO\t2017-01-12 16:12:26\tGenotypeConcordance\tchecked    32,280,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 22:28,487,368\n",
      "INFO\t2017-01-12 16:12:26\tGenotypeConcordance\tchecked    32,290,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 22:29,552,332\n",
      "INFO\t2017-01-12 16:12:27\tGenotypeConcordance\tchecked    32,300,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 22:30,454,598\n",
      "INFO\t2017-01-12 16:12:27\tGenotypeConcordance\tchecked    32,310,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 22:31,414,524\n",
      "INFO\t2017-01-12 16:12:27\tGenotypeConcordance\tchecked    32,320,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 22:32,373,174\n",
      "INFO\t2017-01-12 16:12:28\tGenotypeConcordance\tchecked    32,330,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 22:33,106,989\n",
      "INFO\t2017-01-12 16:12:28\tGenotypeConcordance\tchecked    32,340,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 22:33,924,594\n",
      "INFO\t2017-01-12 16:12:28\tGenotypeConcordance\tchecked    32,350,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 22:34,726,062\n",
      "INFO\t2017-01-12 16:12:29\tGenotypeConcordance\tchecked    32,360,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 22:35,410,604\n",
      "INFO\t2017-01-12 16:12:29\tGenotypeConcordance\tchecked    32,370,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 22:36,266,441\n",
      "INFO\t2017-01-12 16:12:29\tGenotypeConcordance\tchecked    32,380,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 22:37,087,302\n",
      "INFO\t2017-01-12 16:12:30\tGenotypeConcordance\tchecked    32,390,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 22:37,833,010\n",
      "INFO\t2017-01-12 16:12:30\tGenotypeConcordance\tchecked    32,400,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 22:38,681,603\n",
      "INFO\t2017-01-12 16:12:31\tGenotypeConcordance\tchecked    32,410,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 22:39,538,816\n",
      "INFO\t2017-01-12 16:12:31\tGenotypeConcordance\tchecked    32,420,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 22:40,431,574\n",
      "INFO\t2017-01-12 16:12:31\tGenotypeConcordance\tchecked    32,430,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 22:41,534,436\n",
      "INFO\t2017-01-12 16:12:31\tGenotypeConcordance\tchecked    32,440,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 22:42,541,285\n",
      "INFO\t2017-01-12 16:12:32\tGenotypeConcordance\tchecked    32,450,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 22:43,264,421\n",
      "INFO\t2017-01-12 16:12:32\tGenotypeConcordance\tchecked    32,460,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 22:43,974,763\n",
      "INFO\t2017-01-12 16:12:33\tGenotypeConcordance\tchecked    32,470,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 22:44,739,132\n",
      "INFO\t2017-01-12 16:12:33\tGenotypeConcordance\tchecked    32,480,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 22:45,333,014\n",
      "INFO\t2017-01-12 16:12:33\tGenotypeConcordance\tchecked    32,490,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 22:46,056,236\n",
      "INFO\t2017-01-12 16:12:33\tGenotypeConcordance\tchecked    32,500,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 22:46,789,230\n",
      "INFO\t2017-01-12 16:12:34\tGenotypeConcordance\tchecked    32,510,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 22:47,424,320\n",
      "INFO\t2017-01-12 16:12:34\tGenotypeConcordance\tchecked    32,520,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 22:48,039,140\n",
      "INFO\t2017-01-12 16:12:34\tGenotypeConcordance\tchecked    32,530,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 22:48,719,417\n",
      "INFO\t2017-01-12 16:12:35\tGenotypeConcordance\tchecked    32,540,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 22:49,244,940\n",
      "INFO\t2017-01-12 16:12:35\tGenotypeConcordance\tchecked    32,550,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 22:49,709,319\n",
      "INFO\t2017-01-12 16:12:35\tGenotypeConcordance\tchecked    32,560,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 22:50,325,778\n",
      "INFO\t2017-01-12 16:12:36\tGenotypeConcordance\tchecked    32,570,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 22:51,047,884\n",
      "INFO\t2017-01-12 16:12:36\tGenotypeConcordance\tchecked    32,580,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: X:935,063\n",
      "INFO\t2017-01-12 16:12:37\tGenotypeConcordance\tchecked    32,590,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: X:2,361,712\n",
      "INFO\t2017-01-12 16:12:37\tGenotypeConcordance\tchecked    32,600,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: X:5,125,475\n",
      "INFO\t2017-01-12 16:12:37\tGenotypeConcordance\tchecked    32,610,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: X:9,429,617\n",
      "INFO\t2017-01-12 16:12:38\tGenotypeConcordance\tchecked    32,620,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: X:14,769,205\n",
      "INFO\t2017-01-12 16:12:38\tGenotypeConcordance\tchecked    32,630,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: X:20,705,016\n",
      "INFO\t2017-01-12 16:12:38\tGenotypeConcordance\tchecked    32,640,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: X:25,016,033\n",
      "INFO\t2017-01-12 16:12:39\tGenotypeConcordance\tchecked    32,650,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: X:29,596,864\n",
      "INFO\t2017-01-12 16:12:39\tGenotypeConcordance\tchecked    32,660,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: X:33,419,349\n",
      "INFO\t2017-01-12 16:12:39\tGenotypeConcordance\tchecked    32,670,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: X:38,417,258\n",
      "INFO\t2017-01-12 16:12:40\tGenotypeConcordance\tchecked    32,680,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: X:44,067,038\n",
      "INFO\t2017-01-12 16:12:40\tGenotypeConcordance\tchecked    32,690,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: X:49,731,605\n",
      "INFO\t2017-01-12 16:12:41\tGenotypeConcordance\tchecked    32,700,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: X:56,420,605\n",
      "INFO\t2017-01-12 16:12:41\tGenotypeConcordance\tchecked    32,710,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: X:66,541,098\n",
      "INFO\t2017-01-12 16:12:41\tGenotypeConcordance\tchecked    32,720,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: X:71,953,657\n",
      "INFO\t2017-01-12 16:12:42\tGenotypeConcordance\tchecked    32,730,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: X:77,354,225\n",
      "INFO\t2017-01-12 16:12:42\tGenotypeConcordance\tchecked    32,740,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: X:82,045,482\n",
      "INFO\t2017-01-12 16:12:42\tGenotypeConcordance\tchecked    32,750,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: X:86,374,697\n",
      "INFO\t2017-01-12 16:12:43\tGenotypeConcordance\tchecked    32,760,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: X:89,567,544\n",
      "INFO\t2017-01-12 16:12:43\tGenotypeConcordance\tchecked    32,770,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: X:93,423,195\n",
      "INFO\t2017-01-12 16:12:43\tGenotypeConcordance\tchecked    32,780,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: X:99,281,815\n",
      "INFO\t2017-01-12 16:12:44\tGenotypeConcordance\tchecked    32,790,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: X:106,502,505\n",
      "INFO\t2017-01-12 16:12:44\tGenotypeConcordance\tchecked    32,800,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: X:113,702,053\n",
      "INFO\t2017-01-12 16:12:44\tGenotypeConcordance\tchecked    32,810,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: X:117,536,606\n",
      "INFO\t2017-01-12 16:12:45\tGenotypeConcordance\tchecked    32,820,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: X:122,065,109\n",
      "INFO\t2017-01-12 16:12:45\tGenotypeConcordance\tchecked    32,830,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: X:126,859,678\n",
      "INFO\t2017-01-12 16:12:46\tGenotypeConcordance\tchecked    32,840,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: X:133,271,121\n",
      "INFO\t2017-01-12 16:12:46\tGenotypeConcordance\tchecked    32,850,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: X:139,596,676\n",
      "INFO\t2017-01-12 16:12:46\tGenotypeConcordance\tchecked    32,860,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: X:143,268,766\n",
      "INFO\t2017-01-12 16:12:47\tGenotypeConcordance\tchecked    32,870,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: X:147,221,049\n",
      "INFO\t2017-01-12 16:12:47\tGenotypeConcordance\tchecked    32,880,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: X:152,563,618\n",
      "INFO\t2017-01-12 16:12:47\tGenotypeConcordance\tchecked    32,890,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: Y:9,950,470\n",
      "INFO\t2017-01-12 16:12:48\tGenotypeConcordance\tchecked    32,900,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: Y:13,275,853\n",
      "INFO\t2017-01-12 16:12:48\tGenotypeConcordance\tchecked    32,910,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: Y:13,489,619\n",
      "INFO\t2017-01-12 16:12:48\tGenotypeConcordance\tchecked    32,920,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: Y:28,795,146\n",
      "INFO\t2017-01-12 16:12:49\tGenotypeConcordance\tchecked    32,930,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: GL000235.1:27,259\n",
      "INFO\t2017-01-12 16:12:49\tGenotypeConcordance\tchecked    32,940,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: GL000236.1:20,854\n",
      "INFO\t2017-01-12 16:12:50\tGenotypeConcordance\tchecked    32,950,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: GL000208.1:75,947\n",
      "INFO\t2017-01-12 16:12:50\tGenotypeConcordance\tchecked    32,960,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: GL000209.1:97,874\n",
      "INFO\t2017-01-12 16:12:50\tGenotypeConcordance\tchecked    32,970,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: GL000211.1:121,478\n",
      "INFO\t2017-01-12 16:12:51\tGenotypeConcordance\tchecked    32,980,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: GL000217.1:134,987\n",
      "INFO\t2017-01-12 16:12:51\tGenotypeConcordance\tchecked    32,990,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: GL000216.1:170,739\n",
      "INFO\t2017-01-12 16:12:51\tGenotypeConcordance\tchecked    33,000,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: GL000219.1:126,521\n",
      "INFO\t2017-01-12 16:12:52\tGenotypeConcordance\tchecked    33,010,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: GL000195.1:67,098\n",
      "INFO\t2017-01-12 16:12:52\tGenotypeConcordance\tchecked    33,020,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: GL000193.1:26,272\n",
      "INFO\t2017-01-12 16:12:52\tGenotypeConcordance\tchecked    33,030,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: GL000225.1:33,627\n",
      "INFO\t2017-01-12 16:12:53\tGenotypeConcordance\tchecked    33,040,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: GL000225.1:131,216\n",
      "INFO\t2017-01-12 16:12:53\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MIXED MIXED 5171\n",
      "INFO\t2017-01-12 16:12:53\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: NO_VARIATION NO_VARIATION 27960649\n",
      "INFO\t2017-01-12 16:12:53\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP MNP 9908\n",
      "[Thu Jan 12 16:12:53 PST 2017] picard.vcf.GenotypeConcordance done. Elapsed time: 15.03 minutes.\n",
      "Runtime.totalMemory()=2214068224\n",
      "\n",
      "\n",
      "\n",
      "[Thu Jan 12 16:12:53 PST 2017] picard.vcf.GenotypeConcordance TRUTH_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz CALL_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz OUTPUT=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P15_truth_WGS_P17.vcf TRUTH_SAMPLE=WGS_P17 CALL_SAMPLE=WGS_P15    INTERSECT_INTERVALS=true MIN_GQ=0 MIN_DP=0 OUTPUT_ALL_ROWS=false USE_VCF_INDEX=false MISSING_SITES_HOM_REF=false VERBOSITY=INFO QUIET=false VALIDATION_STRINGENCY=STRICT COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json\n",
      "[Thu Jan 12 16:12:53 PST 2017] Executing as iholtman@epiglass.ucsd.edu on Linux 3.10.0-327.36.3.el7.x86_64 amd64; OpenJDK 64-Bit Server VM 1.8.0_102-b14; Picard version: 2.6.0-SNAPSHOT\n",
      "INFO\t2017-01-12 16:12:54\tGenotypeConcordance\tStarting iteration over variants.\n",
      "INFO\t2017-01-12 16:12:55\tGenotypeConcordance\tchecked        10,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:1,120,957\n",
      "INFO\t2017-01-12 16:12:55\tGenotypeConcordance\tchecked        20,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:1,707,743\n",
      "INFO\t2017-01-12 16:12:55\tGenotypeConcordance\tchecked        30,000 variants.  Elapsed time: 00:00:01s.  Time for last 10,000:    0s.  Last read position: 1:2,389,563\n",
      "INFO\t2017-01-12 16:12:56\tGenotypeConcordance\tchecked        40,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,017,290\n",
      "INFO\t2017-01-12 16:12:56\tGenotypeConcordance\tchecked        50,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:3,627,524\n",
      "INFO\t2017-01-12 16:12:56\tGenotypeConcordance\tchecked        60,000 variants.  Elapsed time: 00:00:02s.  Time for last 10,000:    0s.  Last read position: 1:4,394,403\n",
      "INFO\t2017-01-12 16:12:57\tGenotypeConcordance\tchecked        70,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,073,378\n",
      "INFO\t2017-01-12 16:12:57\tGenotypeConcordance\tchecked        80,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:5,732,295\n",
      "INFO\t2017-01-12 16:12:57\tGenotypeConcordance\tchecked        90,000 variants.  Elapsed time: 00:00:03s.  Time for last 10,000:    0s.  Last read position: 1:6,438,378\n",
      "INFO\t2017-01-12 16:12:58\tGenotypeConcordance\tchecked       100,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:7,315,516\n",
      "INFO\t2017-01-12 16:12:58\tGenotypeConcordance\tchecked       110,000 variants.  Elapsed time: 00:00:04s.  Time for last 10,000:    0s.  Last read position: 1:8,086,334\n",
      "INFO\t2017-01-12 16:12:59\tGenotypeConcordance\tchecked       120,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:8,968,709\n",
      "INFO\t2017-01-12 16:12:59\tGenotypeConcordance\tchecked       130,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:9,716,019\n",
      "INFO\t2017-01-12 16:12:59\tGenotypeConcordance\tchecked       140,000 variants.  Elapsed time: 00:00:05s.  Time for last 10,000:    0s.  Last read position: 1:10,623,841\n",
      "INFO\t2017-01-12 16:13:00\tGenotypeConcordance\tchecked       150,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:11,403,951\n",
      "INFO\t2017-01-12 16:13:00\tGenotypeConcordance\tchecked       160,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:12,131,083\n",
      "INFO\t2017-01-12 16:13:00\tGenotypeConcordance\tchecked       170,000 variants.  Elapsed time: 00:00:06s.  Time for last 10,000:    0s.  Last read position: 1:12,899,984\n",
      "INFO\t2017-01-12 16:13:00\tGenotypeConcordance\tchecked       180,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:14,034,254\n",
      "INFO\t2017-01-12 16:13:01\tGenotypeConcordance\tchecked       190,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:14,782,380\n",
      "INFO\t2017-01-12 16:13:01\tGenotypeConcordance\tchecked       200,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:15,518,433\n",
      "INFO\t2017-01-12 16:13:01\tGenotypeConcordance\tchecked       210,000 variants.  Elapsed time: 00:00:07s.  Time for last 10,000:    0s.  Last read position: 1:16,317,389\n",
      "INFO\t2017-01-12 16:13:02\tGenotypeConcordance\tchecked       220,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:16,940,194\n",
      "INFO\t2017-01-12 16:13:02\tGenotypeConcordance\tchecked       230,000 variants.  Elapsed time: 00:00:08s.  Time for last 10,000:    0s.  Last read position: 1:17,543,832\n",
      "INFO\t2017-01-12 16:13:02\tGenotypeConcordance\tchecked       240,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:18,280,366\n",
      "INFO\t2017-01-12 16:13:03\tGenotypeConcordance\tchecked       250,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:19,026,057\n",
      "INFO\t2017-01-12 16:13:03\tGenotypeConcordance\tchecked       260,000 variants.  Elapsed time: 00:00:09s.  Time for last 10,000:    0s.  Last read position: 1:19,777,837\n",
      "INFO\t2017-01-12 16:13:04\tGenotypeConcordance\tchecked       270,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:20,592,124\n",
      "INFO\t2017-01-12 16:13:04\tGenotypeConcordance\tchecked       280,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:21,534,929\n",
      "INFO\t2017-01-12 16:13:04\tGenotypeConcordance\tchecked       290,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:22,326,328\n",
      "INFO\t2017-01-12 16:13:04\tGenotypeConcordance\tchecked       300,000 variants.  Elapsed time: 00:00:10s.  Time for last 10,000:    0s.  Last read position: 1:23,116,975\n",
      "INFO\t2017-01-12 16:13:05\tGenotypeConcordance\tchecked       310,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:24,100,808\n",
      "INFO\t2017-01-12 16:13:05\tGenotypeConcordance\tchecked       320,000 variants.  Elapsed time: 00:00:11s.  Time for last 10,000:    0s.  Last read position: 1:25,015,530\n",
      "INFO\t2017-01-12 16:13:05\tGenotypeConcordance\tchecked       330,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:25,907,167\n",
      "INFO\t2017-01-12 16:13:06\tGenotypeConcordance\tchecked       340,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:26,844,752\n",
      "INFO\t2017-01-12 16:13:06\tGenotypeConcordance\tchecked       350,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:27,980,714\n",
      "INFO\t2017-01-12 16:13:06\tGenotypeConcordance\tchecked       360,000 variants.  Elapsed time: 00:00:12s.  Time for last 10,000:    0s.  Last read position: 1:28,981,150\n",
      "INFO\t2017-01-12 16:13:07\tGenotypeConcordance\tchecked       370,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:30,094,819\n",
      "INFO\t2017-01-12 16:13:07\tGenotypeConcordance\tchecked       380,000 variants.  Elapsed time: 00:00:13s.  Time for last 10,000:    0s.  Last read position: 1:30,842,172\n",
      "INFO\t2017-01-12 16:13:07\tGenotypeConcordance\tchecked       390,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:31,587,628\n",
      "INFO\t2017-01-12 16:13:08\tGenotypeConcordance\tchecked       400,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:32,447,963\n",
      "INFO\t2017-01-12 16:13:08\tGenotypeConcordance\tchecked       410,000 variants.  Elapsed time: 00:00:14s.  Time for last 10,000:    0s.  Last read position: 1:33,489,498\n",
      "INFO\t2017-01-12 16:13:09\tGenotypeConcordance\tchecked       420,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:34,445,892\n",
      "INFO\t2017-01-12 16:13:09\tGenotypeConcordance\tchecked       430,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:35,297,017\n",
      "INFO\t2017-01-12 16:13:09\tGenotypeConcordance\tchecked       440,000 variants.  Elapsed time: 00:00:15s.  Time for last 10,000:    0s.  Last read position: 1:36,349,560\n",
      "INFO\t2017-01-12 16:13:10\tGenotypeConcordance\tchecked       450,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:37,300,320\n",
      "INFO\t2017-01-12 16:13:10\tGenotypeConcordance\tchecked       460,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:38,174,415\n",
      "INFO\t2017-01-12 16:13:10\tGenotypeConcordance\tchecked       470,000 variants.  Elapsed time: 00:00:16s.  Time for last 10,000:    0s.  Last read position: 1:39,100,863\n",
      "INFO\t2017-01-12 16:13:11\tGenotypeConcordance\tchecked       480,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:40,075,458\n",
      "INFO\t2017-01-12 16:13:11\tGenotypeConcordance\tchecked       490,000 variants.  Elapsed time: 00:00:17s.  Time for last 10,000:    0s.  Last read position: 1:40,968,201\n",
      "INFO\t2017-01-12 16:13:12\tGenotypeConcordance\tchecked       500,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:41,904,021\n",
      "INFO\t2017-01-12 16:13:12\tGenotypeConcordance\tchecked       510,000 variants.  Elapsed time: 00:00:18s.  Time for last 10,000:    0s.  Last read position: 1:42,843,316\n",
      "INFO\t2017-01-12 16:13:13\tGenotypeConcordance\tchecked       520,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:43,718,158\n",
      "INFO\t2017-01-12 16:13:13\tGenotypeConcordance\tchecked       530,000 variants.  Elapsed time: 00:00:19s.  Time for last 10,000:    0s.  Last read position: 1:44,706,113\n",
      "INFO\t2017-01-12 16:13:14\tGenotypeConcordance\tchecked       540,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:45,691,503\n",
      "INFO\t2017-01-12 16:13:14\tGenotypeConcordance\tchecked       550,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:46,757,990\n",
      "INFO\t2017-01-12 16:13:14\tGenotypeConcordance\tchecked       560,000 variants.  Elapsed time: 00:00:20s.  Time for last 10,000:    0s.  Last read position: 1:47,610,065\n",
      "INFO\t2017-01-12 16:13:15\tGenotypeConcordance\tchecked       570,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:48,529,955\n",
      "INFO\t2017-01-12 16:13:15\tGenotypeConcordance\tchecked       580,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:49,495,669\n",
      "INFO\t2017-01-12 16:13:15\tGenotypeConcordance\tchecked       590,000 variants.  Elapsed time: 00:00:21s.  Time for last 10,000:    0s.  Last read position: 1:50,815,397\n",
      "INFO\t2017-01-12 16:13:15\tGenotypeConcordance\tchecked       600,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:51,923,014\n",
      "INFO\t2017-01-12 16:13:16\tGenotypeConcordance\tchecked       610,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:52,935,423\n",
      "INFO\t2017-01-12 16:13:16\tGenotypeConcordance\tchecked       620,000 variants.  Elapsed time: 00:00:22s.  Time for last 10,000:    0s.  Last read position: 1:53,802,466\n",
      "INFO\t2017-01-12 16:13:16\tGenotypeConcordance\tchecked       630,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:54,708,123\n",
      "INFO\t2017-01-12 16:13:17\tGenotypeConcordance\tchecked       640,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:55,450,942\n",
      "INFO\t2017-01-12 16:13:17\tGenotypeConcordance\tchecked       650,000 variants.  Elapsed time: 00:00:23s.  Time for last 10,000:    0s.  Last read position: 1:56,258,074\n",
      "INFO\t2017-01-12 16:13:18\tGenotypeConcordance\tchecked       660,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:57,130,566\n",
      "INFO\t2017-01-12 16:13:18\tGenotypeConcordance\tchecked       670,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:57,986,610\n",
      "INFO\t2017-01-12 16:13:18\tGenotypeConcordance\tchecked       680,000 variants.  Elapsed time: 00:00:24s.  Time for last 10,000:    0s.  Last read position: 1:58,874,734\n",
      "INFO\t2017-01-12 16:13:19\tGenotypeConcordance\tchecked       690,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:59,704,642\n",
      "INFO\t2017-01-12 16:13:19\tGenotypeConcordance\tchecked       700,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:60,635,069\n",
      "INFO\t2017-01-12 16:13:19\tGenotypeConcordance\tchecked       710,000 variants.  Elapsed time: 00:00:25s.  Time for last 10,000:    0s.  Last read position: 1:61,525,763\n",
      "INFO\t2017-01-12 16:13:20\tGenotypeConcordance\tchecked       720,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:62,435,181\n",
      "INFO\t2017-01-12 16:13:20\tGenotypeConcordance\tchecked       730,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:63,297,853\n",
      "INFO\t2017-01-12 16:13:20\tGenotypeConcordance\tchecked       740,000 variants.  Elapsed time: 00:00:26s.  Time for last 10,000:    0s.  Last read position: 1:64,149,558\n",
      "INFO\t2017-01-12 16:13:21\tGenotypeConcordance\tchecked       750,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:65,061,144\n",
      "INFO\t2017-01-12 16:13:21\tGenotypeConcordance\tchecked       760,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:66,100,450\n",
      "INFO\t2017-01-12 16:13:21\tGenotypeConcordance\tchecked       770,000 variants.  Elapsed time: 00:00:27s.  Time for last 10,000:    0s.  Last read position: 1:67,014,855\n",
      "INFO\t2017-01-12 16:13:21\tGenotypeConcordance\tchecked       780,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:67,947,783\n",
      "INFO\t2017-01-12 16:13:22\tGenotypeConcordance\tchecked       790,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:68,799,916\n",
      "INFO\t2017-01-12 16:13:22\tGenotypeConcordance\tchecked       800,000 variants.  Elapsed time: 00:00:28s.  Time for last 10,000:    0s.  Last read position: 1:69,777,695\n",
      "INFO\t2017-01-12 16:13:22\tGenotypeConcordance\tchecked       810,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:70,764,703\n",
      "INFO\t2017-01-12 16:13:23\tGenotypeConcordance\tchecked       820,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:71,722,094\n",
      "INFO\t2017-01-12 16:13:23\tGenotypeConcordance\tchecked       830,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:72,751,572\n",
      "INFO\t2017-01-12 16:13:23\tGenotypeConcordance\tchecked       840,000 variants.  Elapsed time: 00:00:29s.  Time for last 10,000:    0s.  Last read position: 1:73,686,618\n",
      "INFO\t2017-01-12 16:13:24\tGenotypeConcordance\tchecked       850,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:74,633,797\n",
      "INFO\t2017-01-12 16:13:24\tGenotypeConcordance\tchecked       860,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:75,588,924\n",
      "INFO\t2017-01-12 16:13:24\tGenotypeConcordance\tchecked       870,000 variants.  Elapsed time: 00:00:30s.  Time for last 10,000:    0s.  Last read position: 1:76,479,278\n",
      "INFO\t2017-01-12 16:13:25\tGenotypeConcordance\tchecked       880,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:77,378,755\n",
      "INFO\t2017-01-12 16:13:25\tGenotypeConcordance\tchecked       890,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:78,442,124\n",
      "INFO\t2017-01-12 16:13:25\tGenotypeConcordance\tchecked       900,000 variants.  Elapsed time: 00:00:31s.  Time for last 10,000:    0s.  Last read position: 1:79,343,051\n",
      "INFO\t2017-01-12 16:13:26\tGenotypeConcordance\tchecked       910,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:80,107,710\n",
      "INFO\t2017-01-12 16:13:26\tGenotypeConcordance\tchecked       920,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:80,894,520\n",
      "INFO\t2017-01-12 16:13:26\tGenotypeConcordance\tchecked       930,000 variants.  Elapsed time: 00:00:32s.  Time for last 10,000:    0s.  Last read position: 1:81,661,481\n",
      "INFO\t2017-01-12 16:13:27\tGenotypeConcordance\tchecked       940,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:82,483,449\n",
      "INFO\t2017-01-12 16:13:27\tGenotypeConcordance\tchecked       950,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:83,299,949\n",
      "INFO\t2017-01-12 16:13:27\tGenotypeConcordance\tchecked       960,000 variants.  Elapsed time: 00:00:33s.  Time for last 10,000:    0s.  Last read position: 1:84,203,620\n",
      "INFO\t2017-01-12 16:13:28\tGenotypeConcordance\tchecked       970,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:85,182,774\n",
      "INFO\t2017-01-12 16:13:28\tGenotypeConcordance\tchecked       980,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:86,067,680\n",
      "INFO\t2017-01-12 16:13:28\tGenotypeConcordance\tchecked       990,000 variants.  Elapsed time: 00:00:34s.  Time for last 10,000:    0s.  Last read position: 1:86,962,209\n",
      "INFO\t2017-01-12 16:13:28\tGenotypeConcordance\tchecked     1,000,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:87,883,535\n",
      "INFO\t2017-01-12 16:13:29\tGenotypeConcordance\tchecked     1,010,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:88,852,286\n",
      "INFO\t2017-01-12 16:13:29\tGenotypeConcordance\tchecked     1,020,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:89,803,108\n",
      "INFO\t2017-01-12 16:13:29\tGenotypeConcordance\tchecked     1,030,000 variants.  Elapsed time: 00:00:35s.  Time for last 10,000:    0s.  Last read position: 1:90,776,039\n",
      "INFO\t2017-01-12 16:13:30\tGenotypeConcordance\tchecked     1,040,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:91,725,653\n",
      "INFO\t2017-01-12 16:13:30\tGenotypeConcordance\tchecked     1,050,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:92,586,331\n",
      "INFO\t2017-01-12 16:13:30\tGenotypeConcordance\tchecked     1,060,000 variants.  Elapsed time: 00:00:36s.  Time for last 10,000:    0s.  Last read position: 1:93,572,071\n",
      "INFO\t2017-01-12 16:13:31\tGenotypeConcordance\tchecked     1,070,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:94,567,490\n",
      "INFO\t2017-01-12 16:13:31\tGenotypeConcordance\tchecked     1,080,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:95,464,821\n",
      "INFO\t2017-01-12 16:13:31\tGenotypeConcordance\tchecked     1,090,000 variants.  Elapsed time: 00:00:37s.  Time for last 10,000:    0s.  Last read position: 1:96,459,037\n",
      "INFO\t2017-01-12 16:13:31\tGenotypeConcordance\tchecked     1,100,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:97,413,779\n",
      "INFO\t2017-01-12 16:13:32\tGenotypeConcordance\tchecked     1,110,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:98,403,989\n",
      "INFO\t2017-01-12 16:13:32\tGenotypeConcordance\tchecked     1,120,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:99,373,916\n",
      "INFO\t2017-01-12 16:13:32\tGenotypeConcordance\tchecked     1,130,000 variants.  Elapsed time: 00:00:38s.  Time for last 10,000:    0s.  Last read position: 1:100,226,223\n",
      "INFO\t2017-01-12 16:13:33\tGenotypeConcordance\tchecked     1,140,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:101,205,712\n",
      "INFO\t2017-01-12 16:13:33\tGenotypeConcordance\tchecked     1,150,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:102,069,120\n",
      "INFO\t2017-01-12 16:13:33\tGenotypeConcordance\tchecked     1,160,000 variants.  Elapsed time: 00:00:39s.  Time for last 10,000:    0s.  Last read position: 1:102,856,121\n",
      "INFO\t2017-01-12 16:13:34\tGenotypeConcordance\tchecked     1,170,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:103,594,920\n",
      "INFO\t2017-01-12 16:13:34\tGenotypeConcordance\tchecked     1,180,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:104,787,833\n",
      "INFO\t2017-01-12 16:13:34\tGenotypeConcordance\tchecked     1,190,000 variants.  Elapsed time: 00:00:40s.  Time for last 10,000:    0s.  Last read position: 1:105,436,080\n",
      "INFO\t2017-01-12 16:13:34\tGenotypeConcordance\tchecked     1,200,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:106,102,802\n",
      "INFO\t2017-01-12 16:13:35\tGenotypeConcordance\tchecked     1,210,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:106,860,402\n",
      "INFO\t2017-01-12 16:13:35\tGenotypeConcordance\tchecked     1,220,000 variants.  Elapsed time: 00:00:41s.  Time for last 10,000:    0s.  Last read position: 1:107,709,523\n",
      "INFO\t2017-01-12 16:13:35\tGenotypeConcordance\tchecked     1,230,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:108,611,935\n",
      "INFO\t2017-01-12 16:13:36\tGenotypeConcordance\tchecked     1,240,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:109,661,045\n",
      "INFO\t2017-01-12 16:13:36\tGenotypeConcordance\tchecked     1,250,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:110,640,998\n",
      "INFO\t2017-01-12 16:13:36\tGenotypeConcordance\tchecked     1,260,000 variants.  Elapsed time: 00:00:42s.  Time for last 10,000:    0s.  Last read position: 1:111,585,494\n",
      "INFO\t2017-01-12 16:13:37\tGenotypeConcordance\tchecked     1,270,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:112,468,814\n",
      "INFO\t2017-01-12 16:13:37\tGenotypeConcordance\tchecked     1,280,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:113,310,684\n",
      "INFO\t2017-01-12 16:13:37\tGenotypeConcordance\tchecked     1,290,000 variants.  Elapsed time: 00:00:43s.  Time for last 10,000:    0s.  Last read position: 1:114,333,600\n",
      "INFO\t2017-01-12 16:13:38\tGenotypeConcordance\tchecked     1,300,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:115,277,766\n",
      "INFO\t2017-01-12 16:13:38\tGenotypeConcordance\tchecked     1,310,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:116,173,981\n",
      "INFO\t2017-01-12 16:13:38\tGenotypeConcordance\tchecked     1,320,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:117,088,239\n",
      "INFO\t2017-01-12 16:13:38\tGenotypeConcordance\tchecked     1,330,000 variants.  Elapsed time: 00:00:44s.  Time for last 10,000:    0s.  Last read position: 1:118,014,306\n",
      "INFO\t2017-01-12 16:13:39\tGenotypeConcordance\tchecked     1,340,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:118,986,025\n",
      "INFO\t2017-01-12 16:13:39\tGenotypeConcordance\tchecked     1,350,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:119,895,482\n",
      "INFO\t2017-01-12 16:13:39\tGenotypeConcordance\tchecked     1,360,000 variants.  Elapsed time: 00:00:45s.  Time for last 10,000:    0s.  Last read position: 1:120,855,863\n",
      "INFO\t2017-01-12 16:13:40\tGenotypeConcordance\tchecked     1,370,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:142,560,186\n",
      "INFO\t2017-01-12 16:13:40\tGenotypeConcordance\tchecked     1,380,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:143,134,694\n",
      "INFO\t2017-01-12 16:13:40\tGenotypeConcordance\tchecked     1,390,000 variants.  Elapsed time: 00:00:46s.  Time for last 10,000:    0s.  Last read position: 1:143,518,251\n",
      "INFO\t2017-01-12 16:13:41\tGenotypeConcordance\tchecked     1,400,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:144,956,013\n",
      "INFO\t2017-01-12 16:13:41\tGenotypeConcordance\tchecked     1,410,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:145,706,219\n",
      "INFO\t2017-01-12 16:13:41\tGenotypeConcordance\tchecked     1,420,000 variants.  Elapsed time: 00:00:47s.  Time for last 10,000:    0s.  Last read position: 1:146,994,229\n",
      "INFO\t2017-01-12 16:13:42\tGenotypeConcordance\tchecked     1,430,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:148,558,499\n",
      "INFO\t2017-01-12 16:13:42\tGenotypeConcordance\tchecked     1,440,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:149,287,681\n",
      "INFO\t2017-01-12 16:13:42\tGenotypeConcordance\tchecked     1,450,000 variants.  Elapsed time: 00:00:48s.  Time for last 10,000:    0s.  Last read position: 1:150,398,945\n",
      "INFO\t2017-01-12 16:13:43\tGenotypeConcordance\tchecked     1,460,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:151,459,516\n",
      "INFO\t2017-01-12 16:13:43\tGenotypeConcordance\tchecked     1,470,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:152,433,928\n",
      "INFO\t2017-01-12 16:13:43\tGenotypeConcordance\tchecked     1,480,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:153,175,210\n",
      "INFO\t2017-01-12 16:13:43\tGenotypeConcordance\tchecked     1,490,000 variants.  Elapsed time: 00:00:49s.  Time for last 10,000:    0s.  Last read position: 1:154,116,128\n",
      "INFO\t2017-01-12 16:13:44\tGenotypeConcordance\tchecked     1,500,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:155,169,355\n",
      "INFO\t2017-01-12 16:13:44\tGenotypeConcordance\tchecked     1,510,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:156,316,245\n",
      "INFO\t2017-01-12 16:13:44\tGenotypeConcordance\tchecked     1,520,000 variants.  Elapsed time: 00:00:50s.  Time for last 10,000:    0s.  Last read position: 1:157,227,945\n",
      "INFO\t2017-01-12 16:13:45\tGenotypeConcordance\tchecked     1,530,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:158,040,895\n",
      "INFO\t2017-01-12 16:13:45\tGenotypeConcordance\tchecked     1,540,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:158,772,702\n",
      "INFO\t2017-01-12 16:13:45\tGenotypeConcordance\tchecked     1,550,000 variants.  Elapsed time: 00:00:51s.  Time for last 10,000:    0s.  Last read position: 1:159,678,816\n",
      "INFO\t2017-01-12 16:13:46\tGenotypeConcordance\tchecked     1,560,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:160,585,017\n",
      "INFO\t2017-01-12 16:13:46\tGenotypeConcordance\tchecked     1,570,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:161,463,235\n",
      "INFO\t2017-01-12 16:13:46\tGenotypeConcordance\tchecked     1,580,000 variants.  Elapsed time: 00:00:52s.  Time for last 10,000:    0s.  Last read position: 1:162,347,238\n",
      "INFO\t2017-01-12 16:13:46\tGenotypeConcordance\tchecked     1,590,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:163,202,408\n",
      "INFO\t2017-01-12 16:13:47\tGenotypeConcordance\tchecked     1,600,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:164,043,656\n",
      "INFO\t2017-01-12 16:13:47\tGenotypeConcordance\tchecked     1,610,000 variants.  Elapsed time: 00:00:53s.  Time for last 10,000:    0s.  Last read position: 1:164,898,936\n",
      "INFO\t2017-01-12 16:13:47\tGenotypeConcordance\tchecked     1,620,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:165,684,677\n",
      "INFO\t2017-01-12 16:13:48\tGenotypeConcordance\tchecked     1,630,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:166,521,779\n",
      "INFO\t2017-01-12 16:13:48\tGenotypeConcordance\tchecked     1,640,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:167,398,628\n",
      "INFO\t2017-01-12 16:13:48\tGenotypeConcordance\tchecked     1,650,000 variants.  Elapsed time: 00:00:54s.  Time for last 10,000:    0s.  Last read position: 1:168,184,589\n",
      "INFO\t2017-01-12 16:13:49\tGenotypeConcordance\tchecked     1,660,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:168,994,546\n",
      "INFO\t2017-01-12 16:13:49\tGenotypeConcordance\tchecked     1,670,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:169,850,842\n",
      "INFO\t2017-01-12 16:13:49\tGenotypeConcordance\tchecked     1,680,000 variants.  Elapsed time: 00:00:55s.  Time for last 10,000:    0s.  Last read position: 1:170,782,473\n",
      "INFO\t2017-01-12 16:13:50\tGenotypeConcordance\tchecked     1,690,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:171,615,936\n",
      "INFO\t2017-01-12 16:13:50\tGenotypeConcordance\tchecked     1,700,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:172,529,121\n",
      "INFO\t2017-01-12 16:13:50\tGenotypeConcordance\tchecked     1,710,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:173,534,759\n",
      "INFO\t2017-01-12 16:13:50\tGenotypeConcordance\tchecked     1,720,000 variants.  Elapsed time: 00:00:56s.  Time for last 10,000:    0s.  Last read position: 1:174,599,343\n",
      "INFO\t2017-01-12 16:13:51\tGenotypeConcordance\tchecked     1,730,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:175,533,921\n",
      "INFO\t2017-01-12 16:13:51\tGenotypeConcordance\tchecked     1,740,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:176,392,987\n",
      "INFO\t2017-01-12 16:13:51\tGenotypeConcordance\tchecked     1,750,000 variants.  Elapsed time: 00:00:57s.  Time for last 10,000:    0s.  Last read position: 1:177,478,762\n",
      "INFO\t2017-01-12 16:13:51\tGenotypeConcordance\tchecked     1,760,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:178,444,857\n",
      "INFO\t2017-01-12 16:13:52\tGenotypeConcordance\tchecked     1,770,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:179,335,138\n",
      "INFO\t2017-01-12 16:13:52\tGenotypeConcordance\tchecked     1,780,000 variants.  Elapsed time: 00:00:58s.  Time for last 10,000:    0s.  Last read position: 1:180,187,922\n",
      "INFO\t2017-01-12 16:13:52\tGenotypeConcordance\tchecked     1,790,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:181,078,749\n",
      "INFO\t2017-01-12 16:13:53\tGenotypeConcordance\tchecked     1,800,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:181,947,148\n",
      "INFO\t2017-01-12 16:13:53\tGenotypeConcordance\tchecked     1,810,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:182,845,174\n",
      "INFO\t2017-01-12 16:13:53\tGenotypeConcordance\tchecked     1,820,000 variants.  Elapsed time: 00:00:59s.  Time for last 10,000:    0s.  Last read position: 1:183,791,551\n",
      "INFO\t2017-01-12 16:13:54\tGenotypeConcordance\tchecked     1,830,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:184,717,573\n",
      "INFO\t2017-01-12 16:13:54\tGenotypeConcordance\tchecked     1,840,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:185,649,535\n",
      "INFO\t2017-01-12 16:13:54\tGenotypeConcordance\tchecked     1,850,000 variants.  Elapsed time: 00:01:00s.  Time for last 10,000:    0s.  Last read position: 1:186,580,648\n",
      "INFO\t2017-01-12 16:13:54\tGenotypeConcordance\tchecked     1,860,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:187,490,339\n",
      "INFO\t2017-01-12 16:13:55\tGenotypeConcordance\tchecked     1,870,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:188,255,718\n",
      "INFO\t2017-01-12 16:13:55\tGenotypeConcordance\tchecked     1,880,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:188,965,252\n",
      "INFO\t2017-01-12 16:13:55\tGenotypeConcordance\tchecked     1,890,000 variants.  Elapsed time: 00:01:01s.  Time for last 10,000:    0s.  Last read position: 1:189,713,805\n",
      "INFO\t2017-01-12 16:13:56\tGenotypeConcordance\tchecked     1,900,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:190,541,681\n",
      "INFO\t2017-01-12 16:13:56\tGenotypeConcordance\tchecked     1,910,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:191,390,229\n",
      "INFO\t2017-01-12 16:13:56\tGenotypeConcordance\tchecked     1,920,000 variants.  Elapsed time: 00:01:02s.  Time for last 10,000:    0s.  Last read position: 1:192,341,209\n",
      "INFO\t2017-01-12 16:13:57\tGenotypeConcordance\tchecked     1,930,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:193,353,750\n",
      "INFO\t2017-01-12 16:13:57\tGenotypeConcordance\tchecked     1,940,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:194,158,102\n",
      "INFO\t2017-01-12 16:13:57\tGenotypeConcordance\tchecked     1,950,000 variants.  Elapsed time: 00:01:03s.  Time for last 10,000:    0s.  Last read position: 1:194,936,207\n",
      "INFO\t2017-01-12 16:13:58\tGenotypeConcordance\tchecked     1,960,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:195,802,260\n",
      "INFO\t2017-01-12 16:13:58\tGenotypeConcordance\tchecked     1,970,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:196,705,654\n",
      "INFO\t2017-01-12 16:13:58\tGenotypeConcordance\tchecked     1,980,000 variants.  Elapsed time: 00:01:04s.  Time for last 10,000:    0s.  Last read position: 1:197,591,397\n",
      "INFO\t2017-01-12 16:13:58\tGenotypeConcordance\tchecked     1,990,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:198,546,499\n",
      "INFO\t2017-01-12 16:13:59\tGenotypeConcordance\tchecked     2,000,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:199,523,656\n",
      "INFO\t2017-01-12 16:13:59\tGenotypeConcordance\tchecked     2,010,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:200,392,412\n",
      "INFO\t2017-01-12 16:13:59\tGenotypeConcordance\tchecked     2,020,000 variants.  Elapsed time: 00:01:05s.  Time for last 10,000:    0s.  Last read position: 1:201,243,556\n",
      "INFO\t2017-01-12 16:14:00\tGenotypeConcordance\tchecked     2,030,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:202,089,610\n",
      "INFO\t2017-01-12 16:14:00\tGenotypeConcordance\tchecked     2,040,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:203,062,824\n",
      "INFO\t2017-01-12 16:14:00\tGenotypeConcordance\tchecked     2,050,000 variants.  Elapsed time: 00:01:06s.  Time for last 10,000:    0s.  Last read position: 1:203,890,477\n",
      "INFO\t2017-01-12 16:14:01\tGenotypeConcordance\tchecked     2,060,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:204,741,992\n",
      "INFO\t2017-01-12 16:14:01\tGenotypeConcordance\tchecked     2,070,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:205,615,287\n",
      "INFO\t2017-01-12 16:14:01\tGenotypeConcordance\tchecked     2,080,000 variants.  Elapsed time: 00:01:07s.  Time for last 10,000:    0s.  Last read position: 1:206,882,135\n",
      "INFO\t2017-01-12 16:14:01\tGenotypeConcordance\tchecked     2,090,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:207,908,172\n",
      "INFO\t2017-01-12 16:14:02\tGenotypeConcordance\tchecked     2,100,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:208,782,188\n",
      "INFO\t2017-01-12 16:14:02\tGenotypeConcordance\tchecked     2,110,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:209,634,321\n",
      "INFO\t2017-01-12 16:14:02\tGenotypeConcordance\tchecked     2,120,000 variants.  Elapsed time: 00:01:08s.  Time for last 10,000:    0s.  Last read position: 1:210,528,934\n",
      "INFO\t2017-01-12 16:14:03\tGenotypeConcordance\tchecked     2,130,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:211,348,493\n",
      "INFO\t2017-01-12 16:14:03\tGenotypeConcordance\tchecked     2,140,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:212,233,950\n",
      "INFO\t2017-01-12 16:14:03\tGenotypeConcordance\tchecked     2,150,000 variants.  Elapsed time: 00:01:09s.  Time for last 10,000:    0s.  Last read position: 1:213,102,148\n",
      "INFO\t2017-01-12 16:14:04\tGenotypeConcordance\tchecked     2,160,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:214,041,894\n",
      "INFO\t2017-01-12 16:14:04\tGenotypeConcordance\tchecked     2,170,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:214,885,561\n",
      "INFO\t2017-01-12 16:14:04\tGenotypeConcordance\tchecked     2,180,000 variants.  Elapsed time: 00:01:10s.  Time for last 10,000:    0s.  Last read position: 1:215,832,378\n",
      "INFO\t2017-01-12 16:14:05\tGenotypeConcordance\tchecked     2,190,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:216,662,472\n",
      "INFO\t2017-01-12 16:14:05\tGenotypeConcordance\tchecked     2,200,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:217,442,851\n",
      "INFO\t2017-01-12 16:14:05\tGenotypeConcordance\tchecked     2,210,000 variants.  Elapsed time: 00:01:11s.  Time for last 10,000:    0s.  Last read position: 1:218,264,979\n",
      "INFO\t2017-01-12 16:14:05\tGenotypeConcordance\tchecked     2,220,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:219,125,281\n",
      "INFO\t2017-01-12 16:14:06\tGenotypeConcordance\tchecked     2,230,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:220,050,341\n",
      "INFO\t2017-01-12 16:14:06\tGenotypeConcordance\tchecked     2,240,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:220,962,453\n",
      "INFO\t2017-01-12 16:14:06\tGenotypeConcordance\tchecked     2,250,000 variants.  Elapsed time: 00:01:12s.  Time for last 10,000:    0s.  Last read position: 1:221,845,715\n",
      "INFO\t2017-01-12 16:14:07\tGenotypeConcordance\tchecked     2,260,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:222,741,944\n",
      "INFO\t2017-01-12 16:14:07\tGenotypeConcordance\tchecked     2,270,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:223,647,626\n",
      "INFO\t2017-01-12 16:14:07\tGenotypeConcordance\tchecked     2,280,000 variants.  Elapsed time: 00:01:13s.  Time for last 10,000:    0s.  Last read position: 1:224,520,873\n",
      "INFO\t2017-01-12 16:14:08\tGenotypeConcordance\tchecked     2,290,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:225,434,050\n",
      "INFO\t2017-01-12 16:14:08\tGenotypeConcordance\tchecked     2,300,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:226,371,672\n",
      "INFO\t2017-01-12 16:14:08\tGenotypeConcordance\tchecked     2,310,000 variants.  Elapsed time: 00:01:14s.  Time for last 10,000:    0s.  Last read position: 1:227,320,600\n",
      "INFO\t2017-01-12 16:14:09\tGenotypeConcordance\tchecked     2,320,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 1:228,129,263\n",
      "INFO\t2017-01-12 16:14:09\tGenotypeConcordance\tchecked     2,330,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 1:229,027,774\n",
      "INFO\t2017-01-12 16:14:09\tGenotypeConcordance\tchecked     2,340,000 variants.  Elapsed time: 00:01:15s.  Time for last 10,000:    0s.  Last read position: 1:229,841,108\n",
      "INFO\t2017-01-12 16:14:10\tGenotypeConcordance\tchecked     2,350,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 1:230,622,939\n",
      "INFO\t2017-01-12 16:14:10\tGenotypeConcordance\tchecked     2,360,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 1:231,468,446\n",
      "INFO\t2017-01-12 16:14:10\tGenotypeConcordance\tchecked     2,370,000 variants.  Elapsed time: 00:01:16s.  Time for last 10,000:    0s.  Last read position: 1:232,321,862\n",
      "INFO\t2017-01-12 16:14:10\tGenotypeConcordance\tchecked     2,380,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 1:233,153,225\n",
      "INFO\t2017-01-12 16:14:11\tGenotypeConcordance\tchecked     2,390,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 1:233,956,282\n",
      "INFO\t2017-01-12 16:14:11\tGenotypeConcordance\tchecked     2,400,000 variants.  Elapsed time: 00:01:17s.  Time for last 10,000:    0s.  Last read position: 1:234,728,108\n",
      "INFO\t2017-01-12 16:14:11\tGenotypeConcordance\tchecked     2,410,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 1:235,631,568\n",
      "INFO\t2017-01-12 16:14:12\tGenotypeConcordance\tchecked     2,420,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 1:236,469,498\n",
      "INFO\t2017-01-12 16:14:12\tGenotypeConcordance\tchecked     2,430,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 1:237,172,962\n",
      "INFO\t2017-01-12 16:14:12\tGenotypeConcordance\tchecked     2,440,000 variants.  Elapsed time: 00:01:18s.  Time for last 10,000:    0s.  Last read position: 1:237,962,535\n",
      "INFO\t2017-01-12 16:14:13\tGenotypeConcordance\tchecked     2,450,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 1:238,728,187\n",
      "INFO\t2017-01-12 16:14:13\tGenotypeConcordance\tchecked     2,460,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 1:239,511,263\n",
      "INFO\t2017-01-12 16:14:13\tGenotypeConcordance\tchecked     2,470,000 variants.  Elapsed time: 00:01:19s.  Time for last 10,000:    0s.  Last read position: 1:240,311,555\n",
      "INFO\t2017-01-12 16:14:14\tGenotypeConcordance\tchecked     2,480,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 1:241,083,566\n",
      "INFO\t2017-01-12 16:14:14\tGenotypeConcordance\tchecked     2,490,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 1:241,845,577\n",
      "INFO\t2017-01-12 16:14:14\tGenotypeConcordance\tchecked     2,500,000 variants.  Elapsed time: 00:01:20s.  Time for last 10,000:    0s.  Last read position: 1:242,532,846\n",
      "INFO\t2017-01-12 16:14:15\tGenotypeConcordance\tchecked     2,510,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 1:243,261,977\n",
      "INFO\t2017-01-12 16:14:15\tGenotypeConcordance\tchecked     2,520,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 1:244,311,942\n",
      "INFO\t2017-01-12 16:14:15\tGenotypeConcordance\tchecked     2,530,000 variants.  Elapsed time: 00:01:21s.  Time for last 10,000:    0s.  Last read position: 1:245,077,322\n",
      "INFO\t2017-01-12 16:14:15\tGenotypeConcordance\tchecked     2,540,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 1:245,800,205\n",
      "INFO\t2017-01-12 16:14:16\tGenotypeConcordance\tchecked     2,550,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 1:246,454,550\n",
      "INFO\t2017-01-12 16:14:16\tGenotypeConcordance\tchecked     2,560,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 1:247,179,859\n",
      "INFO\t2017-01-12 16:14:16\tGenotypeConcordance\tchecked     2,570,000 variants.  Elapsed time: 00:01:22s.  Time for last 10,000:    0s.  Last read position: 1:247,891,094\n",
      "INFO\t2017-01-12 16:14:17\tGenotypeConcordance\tchecked     2,580,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 1:248,544,206\n",
      "INFO\t2017-01-12 16:14:17\tGenotypeConcordance\tchecked     2,590,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:94,559\n",
      "INFO\t2017-01-12 16:14:17\tGenotypeConcordance\tchecked     2,600,000 variants.  Elapsed time: 00:01:23s.  Time for last 10,000:    0s.  Last read position: 2:837,910\n",
      "INFO\t2017-01-12 16:14:18\tGenotypeConcordance\tchecked     2,610,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:1,455,313\n",
      "INFO\t2017-01-12 16:14:18\tGenotypeConcordance\tchecked     2,620,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:2,113,647\n",
      "INFO\t2017-01-12 16:14:18\tGenotypeConcordance\tchecked     2,630,000 variants.  Elapsed time: 00:01:24s.  Time for last 10,000:    0s.  Last read position: 2:2,833,582\n",
      "INFO\t2017-01-12 16:14:19\tGenotypeConcordance\tchecked     2,640,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:3,609,598\n",
      "INFO\t2017-01-12 16:14:19\tGenotypeConcordance\tchecked     2,650,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:4,320,485\n",
      "INFO\t2017-01-12 16:14:19\tGenotypeConcordance\tchecked     2,660,000 variants.  Elapsed time: 00:01:25s.  Time for last 10,000:    0s.  Last read position: 2:5,168,050\n",
      "INFO\t2017-01-12 16:14:19\tGenotypeConcordance\tchecked     2,670,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:5,929,456\n",
      "INFO\t2017-01-12 16:14:20\tGenotypeConcordance\tchecked     2,680,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:6,651,520\n",
      "INFO\t2017-01-12 16:14:20\tGenotypeConcordance\tchecked     2,690,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:7,475,285\n",
      "INFO\t2017-01-12 16:14:20\tGenotypeConcordance\tchecked     2,700,000 variants.  Elapsed time: 00:01:26s.  Time for last 10,000:    0s.  Last read position: 2:8,297,461\n",
      "INFO\t2017-01-12 16:14:21\tGenotypeConcordance\tchecked     2,710,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:9,191,332\n",
      "INFO\t2017-01-12 16:14:21\tGenotypeConcordance\tchecked     2,720,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:9,995,071\n",
      "INFO\t2017-01-12 16:14:21\tGenotypeConcordance\tchecked     2,730,000 variants.  Elapsed time: 00:01:27s.  Time for last 10,000:    0s.  Last read position: 2:10,710,145\n",
      "INFO\t2017-01-12 16:14:22\tGenotypeConcordance\tchecked     2,740,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:11,442,442\n",
      "INFO\t2017-01-12 16:14:22\tGenotypeConcordance\tchecked     2,750,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:12,208,518\n",
      "INFO\t2017-01-12 16:14:22\tGenotypeConcordance\tchecked     2,760,000 variants.  Elapsed time: 00:01:28s.  Time for last 10,000:    0s.  Last read position: 2:13,082,247\n",
      "INFO\t2017-01-12 16:14:22\tGenotypeConcordance\tchecked     2,770,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:13,861,615\n",
      "INFO\t2017-01-12 16:14:23\tGenotypeConcordance\tchecked     2,780,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:14,651,618\n",
      "INFO\t2017-01-12 16:14:23\tGenotypeConcordance\tchecked     2,790,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:15,509,146\n",
      "INFO\t2017-01-12 16:14:23\tGenotypeConcordance\tchecked     2,800,000 variants.  Elapsed time: 00:01:29s.  Time for last 10,000:    0s.  Last read position: 2:16,274,500\n",
      "INFO\t2017-01-12 16:14:24\tGenotypeConcordance\tchecked     2,810,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:17,168,614\n",
      "INFO\t2017-01-12 16:14:24\tGenotypeConcordance\tchecked     2,820,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:18,102,927\n",
      "INFO\t2017-01-12 16:14:24\tGenotypeConcordance\tchecked     2,830,000 variants.  Elapsed time: 00:01:30s.  Time for last 10,000:    0s.  Last read position: 2:18,909,666\n",
      "INFO\t2017-01-12 16:14:25\tGenotypeConcordance\tchecked     2,840,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:19,850,364\n",
      "INFO\t2017-01-12 16:14:25\tGenotypeConcordance\tchecked     2,850,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:20,700,647\n",
      "INFO\t2017-01-12 16:14:25\tGenotypeConcordance\tchecked     2,860,000 variants.  Elapsed time: 00:01:31s.  Time for last 10,000:    0s.  Last read position: 2:21,512,069\n",
      "INFO\t2017-01-12 16:14:26\tGenotypeConcordance\tchecked     2,870,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:22,460,536\n",
      "INFO\t2017-01-12 16:14:26\tGenotypeConcordance\tchecked     2,880,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:23,331,614\n",
      "INFO\t2017-01-12 16:14:26\tGenotypeConcordance\tchecked     2,890,000 variants.  Elapsed time: 00:01:32s.  Time for last 10,000:    0s.  Last read position: 2:24,242,812\n",
      "INFO\t2017-01-12 16:14:26\tGenotypeConcordance\tchecked     2,900,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:25,284,933\n",
      "INFO\t2017-01-12 16:14:27\tGenotypeConcordance\tchecked     2,910,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:26,304,419\n",
      "INFO\t2017-01-12 16:14:27\tGenotypeConcordance\tchecked     2,920,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:27,213,937\n",
      "INFO\t2017-01-12 16:14:27\tGenotypeConcordance\tchecked     2,930,000 variants.  Elapsed time: 00:01:33s.  Time for last 10,000:    0s.  Last read position: 2:28,384,132\n",
      "INFO\t2017-01-12 16:14:28\tGenotypeConcordance\tchecked     2,940,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:29,271,357\n",
      "INFO\t2017-01-12 16:14:28\tGenotypeConcordance\tchecked     2,950,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:30,051,408\n",
      "INFO\t2017-01-12 16:14:28\tGenotypeConcordance\tchecked     2,960,000 variants.  Elapsed time: 00:01:34s.  Time for last 10,000:    0s.  Last read position: 2:30,862,245\n",
      "INFO\t2017-01-12 16:14:29\tGenotypeConcordance\tchecked     2,970,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:31,586,861\n",
      "INFO\t2017-01-12 16:14:29\tGenotypeConcordance\tchecked     2,980,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:32,555,031\n",
      "INFO\t2017-01-12 16:14:29\tGenotypeConcordance\tchecked     2,990,000 variants.  Elapsed time: 00:01:35s.  Time for last 10,000:    0s.  Last read position: 2:33,390,728\n",
      "INFO\t2017-01-12 16:14:30\tGenotypeConcordance\tchecked     3,000,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:34,029,312\n",
      "INFO\t2017-01-12 16:14:30\tGenotypeConcordance\tchecked     3,010,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:34,626,685\n",
      "INFO\t2017-01-12 16:14:30\tGenotypeConcordance\tchecked     3,020,000 variants.  Elapsed time: 00:01:36s.  Time for last 10,000:    0s.  Last read position: 2:35,300,605\n",
      "INFO\t2017-01-12 16:14:31\tGenotypeConcordance\tchecked     3,030,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:35,994,111\n",
      "INFO\t2017-01-12 16:14:31\tGenotypeConcordance\tchecked     3,040,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:36,699,168\n",
      "INFO\t2017-01-12 16:14:31\tGenotypeConcordance\tchecked     3,050,000 variants.  Elapsed time: 00:01:37s.  Time for last 10,000:    0s.  Last read position: 2:37,453,581\n",
      "INFO\t2017-01-12 16:14:32\tGenotypeConcordance\tchecked     3,060,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:38,024,047\n",
      "INFO\t2017-01-12 16:14:32\tGenotypeConcordance\tchecked     3,070,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:38,691,187\n",
      "INFO\t2017-01-12 16:14:32\tGenotypeConcordance\tchecked     3,080,000 variants.  Elapsed time: 00:01:38s.  Time for last 10,000:    0s.  Last read position: 2:39,523,829\n",
      "INFO\t2017-01-12 16:14:33\tGenotypeConcordance\tchecked     3,090,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:40,295,627\n",
      "INFO\t2017-01-12 16:14:33\tGenotypeConcordance\tchecked     3,100,000 variants.  Elapsed time: 00:01:39s.  Time for last 10,000:    0s.  Last read position: 2:41,048,757\n",
      "INFO\t2017-01-12 16:14:33\tGenotypeConcordance\tchecked     3,110,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:41,786,707\n",
      "INFO\t2017-01-12 16:14:34\tGenotypeConcordance\tchecked     3,120,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:42,459,288\n",
      "INFO\t2017-01-12 16:14:34\tGenotypeConcordance\tchecked     3,130,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:43,269,737\n",
      "INFO\t2017-01-12 16:14:34\tGenotypeConcordance\tchecked     3,140,000 variants.  Elapsed time: 00:01:40s.  Time for last 10,000:    0s.  Last read position: 2:44,027,827\n",
      "INFO\t2017-01-12 16:14:35\tGenotypeConcordance\tchecked     3,150,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:44,763,797\n",
      "INFO\t2017-01-12 16:14:35\tGenotypeConcordance\tchecked     3,160,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:45,546,836\n",
      "INFO\t2017-01-12 16:14:35\tGenotypeConcordance\tchecked     3,170,000 variants.  Elapsed time: 00:01:41s.  Time for last 10,000:    0s.  Last read position: 2:46,363,699\n",
      "INFO\t2017-01-12 16:14:36\tGenotypeConcordance\tchecked     3,180,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:47,118,531\n",
      "INFO\t2017-01-12 16:14:36\tGenotypeConcordance\tchecked     3,190,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:47,813,742\n",
      "INFO\t2017-01-12 16:14:36\tGenotypeConcordance\tchecked     3,200,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:48,663,410\n",
      "INFO\t2017-01-12 16:14:36\tGenotypeConcordance\tchecked     3,210,000 variants.  Elapsed time: 00:01:42s.  Time for last 10,000:    0s.  Last read position: 2:49,407,380\n",
      "INFO\t2017-01-12 16:14:37\tGenotypeConcordance\tchecked     3,220,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:50,178,059\n",
      "INFO\t2017-01-12 16:14:37\tGenotypeConcordance\tchecked     3,230,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:50,987,496\n",
      "INFO\t2017-01-12 16:14:37\tGenotypeConcordance\tchecked     3,240,000 variants.  Elapsed time: 00:01:43s.  Time for last 10,000:    0s.  Last read position: 2:51,778,185\n",
      "INFO\t2017-01-12 16:14:38\tGenotypeConcordance\tchecked     3,250,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:52,480,847\n",
      "INFO\t2017-01-12 16:14:38\tGenotypeConcordance\tchecked     3,260,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:53,100,764\n",
      "INFO\t2017-01-12 16:14:38\tGenotypeConcordance\tchecked     3,270,000 variants.  Elapsed time: 00:01:44s.  Time for last 10,000:    0s.  Last read position: 2:53,780,128\n",
      "INFO\t2017-01-12 16:14:38\tGenotypeConcordance\tchecked     3,280,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:54,605,148\n",
      "INFO\t2017-01-12 16:14:39\tGenotypeConcordance\tchecked     3,290,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:55,413,056\n",
      "INFO\t2017-01-12 16:14:39\tGenotypeConcordance\tchecked     3,300,000 variants.  Elapsed time: 00:01:45s.  Time for last 10,000:    0s.  Last read position: 2:56,271,006\n",
      "INFO\t2017-01-12 16:14:40\tGenotypeConcordance\tchecked     3,310,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:56,996,573\n",
      "INFO\t2017-01-12 16:14:40\tGenotypeConcordance\tchecked     3,320,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:57,677,079\n",
      "INFO\t2017-01-12 16:14:40\tGenotypeConcordance\tchecked     3,330,000 variants.  Elapsed time: 00:01:46s.  Time for last 10,000:    0s.  Last read position: 2:58,541,238\n",
      "INFO\t2017-01-12 16:14:41\tGenotypeConcordance\tchecked     3,340,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:59,526,280\n",
      "INFO\t2017-01-12 16:14:41\tGenotypeConcordance\tchecked     3,350,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:60,364,993\n",
      "INFO\t2017-01-12 16:14:41\tGenotypeConcordance\tchecked     3,360,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:61,323,655\n",
      "INFO\t2017-01-12 16:14:41\tGenotypeConcordance\tchecked     3,370,000 variants.  Elapsed time: 00:01:47s.  Time for last 10,000:    0s.  Last read position: 2:62,260,132\n",
      "INFO\t2017-01-12 16:14:42\tGenotypeConcordance\tchecked     3,380,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:63,330,513\n",
      "INFO\t2017-01-12 16:14:42\tGenotypeConcordance\tchecked     3,390,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:64,467,184\n",
      "INFO\t2017-01-12 16:14:42\tGenotypeConcordance\tchecked     3,400,000 variants.  Elapsed time: 00:01:48s.  Time for last 10,000:    0s.  Last read position: 2:65,346,733\n",
      "INFO\t2017-01-12 16:14:43\tGenotypeConcordance\tchecked     3,410,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:66,185,353\n",
      "INFO\t2017-01-12 16:14:43\tGenotypeConcordance\tchecked     3,420,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:67,072,585\n",
      "INFO\t2017-01-12 16:14:43\tGenotypeConcordance\tchecked     3,430,000 variants.  Elapsed time: 00:01:49s.  Time for last 10,000:    0s.  Last read position: 2:67,925,384\n",
      "INFO\t2017-01-12 16:14:43\tGenotypeConcordance\tchecked     3,440,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:68,716,263\n",
      "INFO\t2017-01-12 16:14:44\tGenotypeConcordance\tchecked     3,450,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:69,552,521\n",
      "INFO\t2017-01-12 16:14:44\tGenotypeConcordance\tchecked     3,460,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:70,460,722\n",
      "INFO\t2017-01-12 16:14:44\tGenotypeConcordance\tchecked     3,470,000 variants.  Elapsed time: 00:01:50s.  Time for last 10,000:    0s.  Last read position: 2:71,249,509\n",
      "INFO\t2017-01-12 16:14:45\tGenotypeConcordance\tchecked     3,480,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:72,040,506\n",
      "INFO\t2017-01-12 16:14:45\tGenotypeConcordance\tchecked     3,490,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:73,127,224\n",
      "INFO\t2017-01-12 16:14:45\tGenotypeConcordance\tchecked     3,500,000 variants.  Elapsed time: 00:01:51s.  Time for last 10,000:    0s.  Last read position: 2:74,131,066\n",
      "INFO\t2017-01-12 16:14:46\tGenotypeConcordance\tchecked     3,510,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:75,111,166\n",
      "INFO\t2017-01-12 16:14:46\tGenotypeConcordance\tchecked     3,520,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:75,909,079\n",
      "INFO\t2017-01-12 16:14:46\tGenotypeConcordance\tchecked     3,530,000 variants.  Elapsed time: 00:01:52s.  Time for last 10,000:    0s.  Last read position: 2:76,669,171\n",
      "INFO\t2017-01-12 16:14:47\tGenotypeConcordance\tchecked     3,540,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:77,310,158\n",
      "INFO\t2017-01-12 16:14:47\tGenotypeConcordance\tchecked     3,550,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:78,049,271\n",
      "INFO\t2017-01-12 16:14:47\tGenotypeConcordance\tchecked     3,560,000 variants.  Elapsed time: 00:01:53s.  Time for last 10,000:    0s.  Last read position: 2:78,848,211\n",
      "INFO\t2017-01-12 16:14:48\tGenotypeConcordance\tchecked     3,570,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:79,617,030\n",
      "INFO\t2017-01-12 16:14:48\tGenotypeConcordance\tchecked     3,580,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:80,391,778\n",
      "INFO\t2017-01-12 16:14:48\tGenotypeConcordance\tchecked     3,590,000 variants.  Elapsed time: 00:01:54s.  Time for last 10,000:    0s.  Last read position: 2:81,215,144\n",
      "INFO\t2017-01-12 16:14:48\tGenotypeConcordance\tchecked     3,600,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:82,049,582\n",
      "INFO\t2017-01-12 16:14:49\tGenotypeConcordance\tchecked     3,610,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:82,955,443\n",
      "INFO\t2017-01-12 16:14:49\tGenotypeConcordance\tchecked     3,620,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:83,808,468\n",
      "INFO\t2017-01-12 16:14:49\tGenotypeConcordance\tchecked     3,630,000 variants.  Elapsed time: 00:01:55s.  Time for last 10,000:    0s.  Last read position: 2:84,673,058\n",
      "INFO\t2017-01-12 16:14:50\tGenotypeConcordance\tchecked     3,640,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:85,615,083\n",
      "INFO\t2017-01-12 16:14:50\tGenotypeConcordance\tchecked     3,650,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:86,501,141\n",
      "INFO\t2017-01-12 16:14:50\tGenotypeConcordance\tchecked     3,660,000 variants.  Elapsed time: 00:01:56s.  Time for last 10,000:    0s.  Last read position: 2:87,574,999\n",
      "INFO\t2017-01-12 16:14:51\tGenotypeConcordance\tchecked     3,670,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:88,668,496\n",
      "INFO\t2017-01-12 16:14:51\tGenotypeConcordance\tchecked     3,680,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:89,449,714\n",
      "INFO\t2017-01-12 16:14:51\tGenotypeConcordance\tchecked     3,690,000 variants.  Elapsed time: 00:01:57s.  Time for last 10,000:    0s.  Last read position: 2:91,634,435\n",
      "INFO\t2017-01-12 16:14:52\tGenotypeConcordance\tchecked     3,700,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:92,132,836\n",
      "INFO\t2017-01-12 16:14:52\tGenotypeConcordance\tchecked     3,710,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:95,489,168\n",
      "INFO\t2017-01-12 16:14:52\tGenotypeConcordance\tchecked     3,720,000 variants.  Elapsed time: 00:01:58s.  Time for last 10,000:    0s.  Last read position: 2:96,762,284\n",
      "INFO\t2017-01-12 16:14:52\tGenotypeConcordance\tchecked     3,730,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:97,863,112\n",
      "INFO\t2017-01-12 16:14:53\tGenotypeConcordance\tchecked     3,740,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:99,055,792\n",
      "INFO\t2017-01-12 16:14:53\tGenotypeConcordance\tchecked     3,750,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:100,134,730\n",
      "INFO\t2017-01-12 16:14:53\tGenotypeConcordance\tchecked     3,760,000 variants.  Elapsed time: 00:01:59s.  Time for last 10,000:    0s.  Last read position: 2:101,070,493\n",
      "INFO\t2017-01-12 16:14:54\tGenotypeConcordance\tchecked     3,770,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:101,865,270\n",
      "INFO\t2017-01-12 16:14:54\tGenotypeConcordance\tchecked     3,780,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:102,727,198\n",
      "INFO\t2017-01-12 16:14:54\tGenotypeConcordance\tchecked     3,790,000 variants.  Elapsed time: 00:02:00s.  Time for last 10,000:    0s.  Last read position: 2:103,625,469\n",
      "INFO\t2017-01-12 16:14:54\tGenotypeConcordance\tchecked     3,800,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:104,550,916\n",
      "INFO\t2017-01-12 16:14:55\tGenotypeConcordance\tchecked     3,810,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:105,447,029\n",
      "INFO\t2017-01-12 16:14:55\tGenotypeConcordance\tchecked     3,820,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:106,265,647\n",
      "INFO\t2017-01-12 16:14:55\tGenotypeConcordance\tchecked     3,830,000 variants.  Elapsed time: 00:02:01s.  Time for last 10,000:    0s.  Last read position: 2:107,073,736\n",
      "INFO\t2017-01-12 16:14:56\tGenotypeConcordance\tchecked     3,840,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:107,908,532\n",
      "INFO\t2017-01-12 16:14:56\tGenotypeConcordance\tchecked     3,850,000 variants.  Elapsed time: 00:02:02s.  Time for last 10,000:    0s.  Last read position: 2:108,821,619\n",
      "INFO\t2017-01-12 16:14:56\tGenotypeConcordance\tchecked     3,860,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:109,685,534\n",
      "INFO\t2017-01-12 16:14:57\tGenotypeConcordance\tchecked     3,870,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:111,052,113\n",
      "INFO\t2017-01-12 16:14:57\tGenotypeConcordance\tchecked     3,880,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:112,344,713\n",
      "INFO\t2017-01-12 16:14:57\tGenotypeConcordance\tchecked     3,890,000 variants.  Elapsed time: 00:02:03s.  Time for last 10,000:    0s.  Last read position: 2:113,309,969\n",
      "INFO\t2017-01-12 16:14:58\tGenotypeConcordance\tchecked     3,900,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:114,178,591\n",
      "INFO\t2017-01-12 16:14:58\tGenotypeConcordance\tchecked     3,910,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:115,045,896\n",
      "INFO\t2017-01-12 16:14:58\tGenotypeConcordance\tchecked     3,920,000 variants.  Elapsed time: 00:02:04s.  Time for last 10,000:    0s.  Last read position: 2:115,933,697\n",
      "INFO\t2017-01-12 16:14:58\tGenotypeConcordance\tchecked     3,930,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:116,831,339\n",
      "INFO\t2017-01-12 16:14:59\tGenotypeConcordance\tchecked     3,940,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:117,629,795\n",
      "INFO\t2017-01-12 16:14:59\tGenotypeConcordance\tchecked     3,950,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:118,364,176\n",
      "INFO\t2017-01-12 16:14:59\tGenotypeConcordance\tchecked     3,960,000 variants.  Elapsed time: 00:02:05s.  Time for last 10,000:    0s.  Last read position: 2:119,197,552\n",
      "INFO\t2017-01-12 16:15:00\tGenotypeConcordance\tchecked     3,970,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:120,018,077\n",
      "INFO\t2017-01-12 16:15:00\tGenotypeConcordance\tchecked     3,980,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:121,022,354\n",
      "INFO\t2017-01-12 16:15:00\tGenotypeConcordance\tchecked     3,990,000 variants.  Elapsed time: 00:02:06s.  Time for last 10,000:    0s.  Last read position: 2:121,816,314\n",
      "INFO\t2017-01-12 16:15:00\tGenotypeConcordance\tchecked     4,000,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:122,669,673\n",
      "INFO\t2017-01-12 16:15:01\tGenotypeConcordance\tchecked     4,010,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:123,573,226\n",
      "INFO\t2017-01-12 16:15:01\tGenotypeConcordance\tchecked     4,020,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:124,315,714\n",
      "INFO\t2017-01-12 16:15:01\tGenotypeConcordance\tchecked     4,030,000 variants.  Elapsed time: 00:02:07s.  Time for last 10,000:    0s.  Last read position: 2:125,130,263\n",
      "INFO\t2017-01-12 16:15:02\tGenotypeConcordance\tchecked     4,040,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:125,965,376\n",
      "INFO\t2017-01-12 16:15:02\tGenotypeConcordance\tchecked     4,050,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:126,776,164\n",
      "INFO\t2017-01-12 16:15:02\tGenotypeConcordance\tchecked     4,060,000 variants.  Elapsed time: 00:02:08s.  Time for last 10,000:    0s.  Last read position: 2:127,580,979\n",
      "INFO\t2017-01-12 16:15:02\tGenotypeConcordance\tchecked     4,070,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:128,388,902\n",
      "INFO\t2017-01-12 16:15:03\tGenotypeConcordance\tchecked     4,080,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:129,225,828\n",
      "INFO\t2017-01-12 16:15:03\tGenotypeConcordance\tchecked     4,090,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:130,003,278\n",
      "INFO\t2017-01-12 16:15:03\tGenotypeConcordance\tchecked     4,100,000 variants.  Elapsed time: 00:02:09s.  Time for last 10,000:    0s.  Last read position: 2:130,795,561\n",
      "INFO\t2017-01-12 16:15:04\tGenotypeConcordance\tchecked     4,110,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:131,710,030\n",
      "INFO\t2017-01-12 16:15:04\tGenotypeConcordance\tchecked     4,120,000 variants.  Elapsed time: 00:02:10s.  Time for last 10,000:    0s.  Last read position: 2:132,424,600\n",
      "INFO\t2017-01-12 16:15:04\tGenotypeConcordance\tchecked     4,130,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:132,978,093\n",
      "INFO\t2017-01-12 16:15:05\tGenotypeConcordance\tchecked     4,140,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:133,345,134\n",
      "INFO\t2017-01-12 16:15:05\tGenotypeConcordance\tchecked     4,150,000 variants.  Elapsed time: 00:02:11s.  Time for last 10,000:    0s.  Last read position: 2:134,247,340\n",
      "INFO\t2017-01-12 16:15:05\tGenotypeConcordance\tchecked     4,160,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:135,059,317\n",
      "INFO\t2017-01-12 16:15:06\tGenotypeConcordance\tchecked     4,170,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:136,005,966\n",
      "INFO\t2017-01-12 16:15:06\tGenotypeConcordance\tchecked     4,180,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:137,028,225\n",
      "INFO\t2017-01-12 16:15:06\tGenotypeConcordance\tchecked     4,190,000 variants.  Elapsed time: 00:02:12s.  Time for last 10,000:    0s.  Last read position: 2:137,826,312\n",
      "INFO\t2017-01-12 16:15:07\tGenotypeConcordance\tchecked     4,200,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:138,609,631\n",
      "INFO\t2017-01-12 16:15:07\tGenotypeConcordance\tchecked     4,210,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:139,499,493\n",
      "INFO\t2017-01-12 16:15:07\tGenotypeConcordance\tchecked     4,220,000 variants.  Elapsed time: 00:02:13s.  Time for last 10,000:    0s.  Last read position: 2:140,328,107\n",
      "INFO\t2017-01-12 16:15:08\tGenotypeConcordance\tchecked     4,230,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:141,130,695\n",
      "INFO\t2017-01-12 16:15:08\tGenotypeConcordance\tchecked     4,240,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:141,872,447\n",
      "INFO\t2017-01-12 16:15:08\tGenotypeConcordance\tchecked     4,250,000 variants.  Elapsed time: 00:02:14s.  Time for last 10,000:    0s.  Last read position: 2:142,585,797\n",
      "INFO\t2017-01-12 16:15:09\tGenotypeConcordance\tchecked     4,260,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:143,476,813\n",
      "INFO\t2017-01-12 16:15:09\tGenotypeConcordance\tchecked     4,270,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:144,483,428\n",
      "INFO\t2017-01-12 16:15:09\tGenotypeConcordance\tchecked     4,280,000 variants.  Elapsed time: 00:02:15s.  Time for last 10,000:    0s.  Last read position: 2:145,702,623\n",
      "INFO\t2017-01-12 16:15:09\tGenotypeConcordance\tchecked     4,290,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:146,688,483\n",
      "INFO\t2017-01-12 16:15:10\tGenotypeConcordance\tchecked     4,300,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:147,757,946\n",
      "INFO\t2017-01-12 16:15:10\tGenotypeConcordance\tchecked     4,310,000 variants.  Elapsed time: 00:02:16s.  Time for last 10,000:    0s.  Last read position: 2:148,865,167\n",
      "INFO\t2017-01-12 16:15:10\tGenotypeConcordance\tchecked     4,320,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:149,967,939\n",
      "INFO\t2017-01-12 16:15:11\tGenotypeConcordance\tchecked     4,330,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:150,885,168\n",
      "INFO\t2017-01-12 16:15:11\tGenotypeConcordance\tchecked     4,340,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:151,825,296\n",
      "INFO\t2017-01-12 16:15:11\tGenotypeConcordance\tchecked     4,350,000 variants.  Elapsed time: 00:02:17s.  Time for last 10,000:    0s.  Last read position: 2:152,751,141\n",
      "INFO\t2017-01-12 16:15:12\tGenotypeConcordance\tchecked     4,360,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:153,695,805\n",
      "INFO\t2017-01-12 16:15:12\tGenotypeConcordance\tchecked     4,370,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:154,529,360\n",
      "INFO\t2017-01-12 16:15:12\tGenotypeConcordance\tchecked     4,380,000 variants.  Elapsed time: 00:02:18s.  Time for last 10,000:    0s.  Last read position: 2:155,443,439\n",
      "INFO\t2017-01-12 16:15:13\tGenotypeConcordance\tchecked     4,390,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:156,394,351\n",
      "INFO\t2017-01-12 16:15:13\tGenotypeConcordance\tchecked     4,400,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:157,529,424\n",
      "INFO\t2017-01-12 16:15:13\tGenotypeConcordance\tchecked     4,410,000 variants.  Elapsed time: 00:02:19s.  Time for last 10,000:    0s.  Last read position: 2:158,604,128\n",
      "INFO\t2017-01-12 16:15:14\tGenotypeConcordance\tchecked     4,420,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:159,476,445\n",
      "INFO\t2017-01-12 16:15:14\tGenotypeConcordance\tchecked     4,430,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:160,325,118\n",
      "INFO\t2017-01-12 16:15:14\tGenotypeConcordance\tchecked     4,440,000 variants.  Elapsed time: 00:02:20s.  Time for last 10,000:    0s.  Last read position: 2:161,357,258\n",
      "INFO\t2017-01-12 16:15:15\tGenotypeConcordance\tchecked     4,450,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:162,321,436\n",
      "INFO\t2017-01-12 16:15:15\tGenotypeConcordance\tchecked     4,460,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:163,400,015\n",
      "INFO\t2017-01-12 16:15:15\tGenotypeConcordance\tchecked     4,470,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:164,328,393\n",
      "INFO\t2017-01-12 16:15:15\tGenotypeConcordance\tchecked     4,480,000 variants.  Elapsed time: 00:02:21s.  Time for last 10,000:    0s.  Last read position: 2:165,295,123\n",
      "INFO\t2017-01-12 16:15:16\tGenotypeConcordance\tchecked     4,490,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:166,285,342\n",
      "INFO\t2017-01-12 16:15:16\tGenotypeConcordance\tchecked     4,500,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:167,191,274\n",
      "INFO\t2017-01-12 16:15:16\tGenotypeConcordance\tchecked     4,510,000 variants.  Elapsed time: 00:02:22s.  Time for last 10,000:    0s.  Last read position: 2:167,990,754\n",
      "INFO\t2017-01-12 16:15:17\tGenotypeConcordance\tchecked     4,520,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:168,825,793\n",
      "INFO\t2017-01-12 16:15:17\tGenotypeConcordance\tchecked     4,530,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:169,689,042\n",
      "INFO\t2017-01-12 16:15:17\tGenotypeConcordance\tchecked     4,540,000 variants.  Elapsed time: 00:02:23s.  Time for last 10,000:    0s.  Last read position: 2:170,536,190\n",
      "INFO\t2017-01-12 16:15:17\tGenotypeConcordance\tchecked     4,550,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:171,334,858\n",
      "INFO\t2017-01-12 16:15:18\tGenotypeConcordance\tchecked     4,560,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:172,242,147\n",
      "INFO\t2017-01-12 16:15:18\tGenotypeConcordance\tchecked     4,570,000 variants.  Elapsed time: 00:02:24s.  Time for last 10,000:    0s.  Last read position: 2:173,138,239\n",
      "INFO\t2017-01-12 16:15:18\tGenotypeConcordance\tchecked     4,580,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:173,969,044\n",
      "INFO\t2017-01-12 16:15:19\tGenotypeConcordance\tchecked     4,590,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:174,823,687\n",
      "INFO\t2017-01-12 16:15:19\tGenotypeConcordance\tchecked     4,600,000 variants.  Elapsed time: 00:02:25s.  Time for last 10,000:    0s.  Last read position: 2:175,815,803\n",
      "INFO\t2017-01-12 16:15:19\tGenotypeConcordance\tchecked     4,610,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:176,740,621\n",
      "INFO\t2017-01-12 16:15:20\tGenotypeConcordance\tchecked     4,620,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:177,642,869\n",
      "INFO\t2017-01-12 16:15:20\tGenotypeConcordance\tchecked     4,630,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:178,545,934\n",
      "INFO\t2017-01-12 16:15:20\tGenotypeConcordance\tchecked     4,640,000 variants.  Elapsed time: 00:02:26s.  Time for last 10,000:    0s.  Last read position: 2:179,541,612\n",
      "INFO\t2017-01-12 16:15:21\tGenotypeConcordance\tchecked     4,650,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:180,421,557\n",
      "INFO\t2017-01-12 16:15:21\tGenotypeConcordance\tchecked     4,660,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:181,281,745\n",
      "INFO\t2017-01-12 16:15:21\tGenotypeConcordance\tchecked     4,670,000 variants.  Elapsed time: 00:02:27s.  Time for last 10,000:    0s.  Last read position: 2:182,215,203\n",
      "INFO\t2017-01-12 16:15:22\tGenotypeConcordance\tchecked     4,680,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:183,182,776\n",
      "INFO\t2017-01-12 16:15:22\tGenotypeConcordance\tchecked     4,690,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:184,083,649\n",
      "INFO\t2017-01-12 16:15:22\tGenotypeConcordance\tchecked     4,700,000 variants.  Elapsed time: 00:02:28s.  Time for last 10,000:    0s.  Last read position: 2:184,938,143\n",
      "INFO\t2017-01-12 16:15:22\tGenotypeConcordance\tchecked     4,710,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:185,881,295\n",
      "INFO\t2017-01-12 16:15:23\tGenotypeConcordance\tchecked     4,720,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:186,928,089\n",
      "INFO\t2017-01-12 16:15:23\tGenotypeConcordance\tchecked     4,730,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:187,898,064\n",
      "INFO\t2017-01-12 16:15:23\tGenotypeConcordance\tchecked     4,740,000 variants.  Elapsed time: 00:02:29s.  Time for last 10,000:    0s.  Last read position: 2:188,787,991\n",
      "INFO\t2017-01-12 16:15:24\tGenotypeConcordance\tchecked     4,750,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:189,778,384\n",
      "INFO\t2017-01-12 16:15:24\tGenotypeConcordance\tchecked     4,760,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:190,783,398\n",
      "INFO\t2017-01-12 16:15:24\tGenotypeConcordance\tchecked     4,770,000 variants.  Elapsed time: 00:02:30s.  Time for last 10,000:    0s.  Last read position: 2:191,766,693\n",
      "INFO\t2017-01-12 16:15:25\tGenotypeConcordance\tchecked     4,780,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:192,696,437\n",
      "INFO\t2017-01-12 16:15:25\tGenotypeConcordance\tchecked     4,790,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:193,648,710\n",
      "INFO\t2017-01-12 16:15:25\tGenotypeConcordance\tchecked     4,800,000 variants.  Elapsed time: 00:02:31s.  Time for last 10,000:    0s.  Last read position: 2:194,694,625\n",
      "INFO\t2017-01-12 16:15:25\tGenotypeConcordance\tchecked     4,810,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:195,615,120\n",
      "INFO\t2017-01-12 16:15:26\tGenotypeConcordance\tchecked     4,820,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:196,443,136\n",
      "INFO\t2017-01-12 16:15:26\tGenotypeConcordance\tchecked     4,830,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:197,361,687\n",
      "INFO\t2017-01-12 16:15:26\tGenotypeConcordance\tchecked     4,840,000 variants.  Elapsed time: 00:02:32s.  Time for last 10,000:    0s.  Last read position: 2:198,398,651\n",
      "INFO\t2017-01-12 16:15:27\tGenotypeConcordance\tchecked     4,850,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 2:199,395,615\n",
      "INFO\t2017-01-12 16:15:27\tGenotypeConcordance\tchecked     4,860,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 2:200,486,639\n",
      "INFO\t2017-01-12 16:15:27\tGenotypeConcordance\tchecked     4,870,000 variants.  Elapsed time: 00:02:33s.  Time for last 10,000:    0s.  Last read position: 2:201,601,564\n",
      "INFO\t2017-01-12 16:15:28\tGenotypeConcordance\tchecked     4,880,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 2:202,646,115\n",
      "INFO\t2017-01-12 16:15:28\tGenotypeConcordance\tchecked     4,890,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 2:203,697,319\n",
      "INFO\t2017-01-12 16:15:28\tGenotypeConcordance\tchecked     4,900,000 variants.  Elapsed time: 00:02:34s.  Time for last 10,000:    0s.  Last read position: 2:204,828,870\n",
      "INFO\t2017-01-12 16:15:29\tGenotypeConcordance\tchecked     4,910,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 2:205,748,607\n",
      "INFO\t2017-01-12 16:15:29\tGenotypeConcordance\tchecked     4,920,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 2:206,669,887\n",
      "INFO\t2017-01-12 16:15:29\tGenotypeConcordance\tchecked     4,930,000 variants.  Elapsed time: 00:02:35s.  Time for last 10,000:    0s.  Last read position: 2:207,607,797\n",
      "INFO\t2017-01-12 16:15:30\tGenotypeConcordance\tchecked     4,940,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 2:208,565,664\n",
      "INFO\t2017-01-12 16:15:30\tGenotypeConcordance\tchecked     4,950,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 2:209,454,495\n",
      "INFO\t2017-01-12 16:15:30\tGenotypeConcordance\tchecked     4,960,000 variants.  Elapsed time: 00:02:36s.  Time for last 10,000:    0s.  Last read position: 2:210,203,064\n",
      "INFO\t2017-01-12 16:15:31\tGenotypeConcordance\tchecked     4,970,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 2:211,270,752\n",
      "INFO\t2017-01-12 16:15:31\tGenotypeConcordance\tchecked     4,980,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 2:212,081,721\n",
      "INFO\t2017-01-12 16:15:31\tGenotypeConcordance\tchecked     4,990,000 variants.  Elapsed time: 00:02:37s.  Time for last 10,000:    0s.  Last read position: 2:212,838,216\n",
      "INFO\t2017-01-12 16:15:32\tGenotypeConcordance\tchecked     5,000,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 2:213,680,033\n",
      "INFO\t2017-01-12 16:15:32\tGenotypeConcordance\tchecked     5,010,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 2:214,658,378\n",
      "INFO\t2017-01-12 16:15:32\tGenotypeConcordance\tchecked     5,020,000 variants.  Elapsed time: 00:02:38s.  Time for last 10,000:    0s.  Last read position: 2:215,532,581\n",
      "INFO\t2017-01-12 16:15:33\tGenotypeConcordance\tchecked     5,030,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 2:216,350,538\n",
      "INFO\t2017-01-12 16:15:33\tGenotypeConcordance\tchecked     5,040,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 2:217,243,369\n",
      "INFO\t2017-01-12 16:15:33\tGenotypeConcordance\tchecked     5,050,000 variants.  Elapsed time: 00:02:39s.  Time for last 10,000:    0s.  Last read position: 2:218,084,158\n",
      "INFO\t2017-01-12 16:15:34\tGenotypeConcordance\tchecked     5,060,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 2:218,924,659\n",
      "INFO\t2017-01-12 16:15:34\tGenotypeConcordance\tchecked     5,070,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 2:219,941,063\n",
      "INFO\t2017-01-12 16:15:34\tGenotypeConcordance\tchecked     5,080,000 variants.  Elapsed time: 00:02:40s.  Time for last 10,000:    0s.  Last read position: 2:220,840,025\n",
      "INFO\t2017-01-12 16:15:34\tGenotypeConcordance\tchecked     5,090,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 2:221,681,143\n",
      "INFO\t2017-01-12 16:15:35\tGenotypeConcordance\tchecked     5,100,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 2:222,534,932\n",
      "INFO\t2017-01-12 16:15:35\tGenotypeConcordance\tchecked     5,110,000 variants.  Elapsed time: 00:02:41s.  Time for last 10,000:    0s.  Last read position: 2:223,449,802\n",
      "INFO\t2017-01-12 16:15:35\tGenotypeConcordance\tchecked     5,120,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 2:224,278,085\n",
      "INFO\t2017-01-12 16:15:36\tGenotypeConcordance\tchecked     5,130,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 2:225,150,894\n",
      "INFO\t2017-01-12 16:15:36\tGenotypeConcordance\tchecked     5,140,000 variants.  Elapsed time: 00:02:42s.  Time for last 10,000:    0s.  Last read position: 2:226,017,466\n",
      "INFO\t2017-01-12 16:15:36\tGenotypeConcordance\tchecked     5,150,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 2:226,935,636\n",
      "INFO\t2017-01-12 16:15:37\tGenotypeConcordance\tchecked     5,160,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 2:227,877,295\n",
      "INFO\t2017-01-12 16:15:37\tGenotypeConcordance\tchecked     5,170,000 variants.  Elapsed time: 00:02:43s.  Time for last 10,000:    0s.  Last read position: 2:228,617,113\n",
      "INFO\t2017-01-12 16:15:37\tGenotypeConcordance\tchecked     5,180,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 2:229,390,956\n",
      "INFO\t2017-01-12 16:15:38\tGenotypeConcordance\tchecked     5,190,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 2:230,252,758\n",
      "INFO\t2017-01-12 16:15:38\tGenotypeConcordance\tchecked     5,200,000 variants.  Elapsed time: 00:02:44s.  Time for last 10,000:    0s.  Last read position: 2:231,125,026\n",
      "INFO\t2017-01-12 16:15:39\tGenotypeConcordance\tchecked     5,210,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 2:231,888,425\n",
      "INFO\t2017-01-12 16:15:39\tGenotypeConcordance\tchecked     5,220,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 2:232,711,377\n",
      "INFO\t2017-01-12 16:15:39\tGenotypeConcordance\tchecked     5,230,000 variants.  Elapsed time: 00:02:45s.  Time for last 10,000:    0s.  Last read position: 2:233,577,038\n",
      "INFO\t2017-01-12 16:15:40\tGenotypeConcordance\tchecked     5,240,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 2:234,410,059\n",
      "INFO\t2017-01-12 16:15:40\tGenotypeConcordance\tchecked     5,250,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 2:235,214,315\n",
      "INFO\t2017-01-12 16:15:40\tGenotypeConcordance\tchecked     5,260,000 variants.  Elapsed time: 00:02:46s.  Time for last 10,000:    0s.  Last read position: 2:235,916,669\n",
      "INFO\t2017-01-12 16:15:41\tGenotypeConcordance\tchecked     5,270,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 2:236,628,437\n",
      "INFO\t2017-01-12 16:15:41\tGenotypeConcordance\tchecked     5,280,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 2:237,398,313\n",
      "INFO\t2017-01-12 16:15:41\tGenotypeConcordance\tchecked     5,290,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 2:238,096,357\n",
      "INFO\t2017-01-12 16:15:41\tGenotypeConcordance\tchecked     5,300,000 variants.  Elapsed time: 00:02:47s.  Time for last 10,000:    0s.  Last read position: 2:238,800,921\n",
      "INFO\t2017-01-12 16:15:42\tGenotypeConcordance\tchecked     5,310,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 2:239,612,662\n",
      "INFO\t2017-01-12 16:15:42\tGenotypeConcordance\tchecked     5,320,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 2:240,365,866\n",
      "INFO\t2017-01-12 16:15:42\tGenotypeConcordance\tchecked     5,330,000 variants.  Elapsed time: 00:02:48s.  Time for last 10,000:    0s.  Last read position: 2:241,064,471\n",
      "INFO\t2017-01-12 16:15:43\tGenotypeConcordance\tchecked     5,340,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 2:241,753,985\n",
      "INFO\t2017-01-12 16:15:43\tGenotypeConcordance\tchecked     5,350,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 2:242,538,598\n",
      "INFO\t2017-01-12 16:15:43\tGenotypeConcordance\tchecked     5,360,000 variants.  Elapsed time: 00:02:49s.  Time for last 10,000:    0s.  Last read position: 3:142,304\n",
      "INFO\t2017-01-12 16:15:44\tGenotypeConcordance\tchecked     5,370,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:753,272\n",
      "INFO\t2017-01-12 16:15:44\tGenotypeConcordance\tchecked     5,380,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:1,398,990\n",
      "INFO\t2017-01-12 16:15:44\tGenotypeConcordance\tchecked     5,390,000 variants.  Elapsed time: 00:02:50s.  Time for last 10,000:    0s.  Last read position: 3:1,921,909\n",
      "INFO\t2017-01-12 16:15:44\tGenotypeConcordance\tchecked     5,400,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:2,542,326\n",
      "INFO\t2017-01-12 16:15:45\tGenotypeConcordance\tchecked     5,410,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:3,164,512\n",
      "INFO\t2017-01-12 16:15:45\tGenotypeConcordance\tchecked     5,420,000 variants.  Elapsed time: 00:02:51s.  Time for last 10,000:    0s.  Last read position: 3:3,786,649\n",
      "INFO\t2017-01-12 16:15:46\tGenotypeConcordance\tchecked     5,430,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:4,431,070\n",
      "INFO\t2017-01-12 16:15:46\tGenotypeConcordance\tchecked     5,440,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:5,157,196\n",
      "INFO\t2017-01-12 16:15:46\tGenotypeConcordance\tchecked     5,450,000 variants.  Elapsed time: 00:02:52s.  Time for last 10,000:    0s.  Last read position: 3:5,767,097\n",
      "INFO\t2017-01-12 16:15:46\tGenotypeConcordance\tchecked     5,460,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:6,374,405\n",
      "INFO\t2017-01-12 16:15:47\tGenotypeConcordance\tchecked     5,470,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:7,054,619\n",
      "INFO\t2017-01-12 16:15:47\tGenotypeConcordance\tchecked     5,480,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:7,744,875\n",
      "INFO\t2017-01-12 16:15:47\tGenotypeConcordance\tchecked     5,490,000 variants.  Elapsed time: 00:02:53s.  Time for last 10,000:    0s.  Last read position: 3:8,432,535\n",
      "INFO\t2017-01-12 16:15:48\tGenotypeConcordance\tchecked     5,500,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:9,164,409\n",
      "INFO\t2017-01-12 16:15:48\tGenotypeConcordance\tchecked     5,510,000 variants.  Elapsed time: 00:02:54s.  Time for last 10,000:    0s.  Last read position: 3:10,058,314\n",
      "INFO\t2017-01-12 16:15:48\tGenotypeConcordance\tchecked     5,520,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:10,833,472\n",
      "INFO\t2017-01-12 16:15:49\tGenotypeConcordance\tchecked     5,530,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:11,671,483\n",
      "INFO\t2017-01-12 16:15:49\tGenotypeConcordance\tchecked     5,540,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:12,522,664\n",
      "INFO\t2017-01-12 16:15:49\tGenotypeConcordance\tchecked     5,550,000 variants.  Elapsed time: 00:02:55s.  Time for last 10,000:    0s.  Last read position: 3:13,286,504\n",
      "INFO\t2017-01-12 16:15:50\tGenotypeConcordance\tchecked     5,560,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:14,043,326\n",
      "INFO\t2017-01-12 16:15:50\tGenotypeConcordance\tchecked     5,570,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:14,841,664\n",
      "INFO\t2017-01-12 16:15:50\tGenotypeConcordance\tchecked     5,580,000 variants.  Elapsed time: 00:02:56s.  Time for last 10,000:    0s.  Last read position: 3:15,660,580\n",
      "INFO\t2017-01-12 16:15:51\tGenotypeConcordance\tchecked     5,590,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:16,498,472\n",
      "INFO\t2017-01-12 16:15:51\tGenotypeConcordance\tchecked     5,600,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:17,382,833\n",
      "INFO\t2017-01-12 16:15:51\tGenotypeConcordance\tchecked     5,610,000 variants.  Elapsed time: 00:02:57s.  Time for last 10,000:    0s.  Last read position: 3:18,430,301\n",
      "INFO\t2017-01-12 16:15:52\tGenotypeConcordance\tchecked     5,620,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:19,332,760\n",
      "INFO\t2017-01-12 16:15:52\tGenotypeConcordance\tchecked     5,630,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:20,227,639\n",
      "INFO\t2017-01-12 16:15:52\tGenotypeConcordance\tchecked     5,640,000 variants.  Elapsed time: 00:02:58s.  Time for last 10,000:    0s.  Last read position: 3:21,044,936\n",
      "INFO\t2017-01-12 16:15:53\tGenotypeConcordance\tchecked     5,650,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:21,791,930\n",
      "INFO\t2017-01-12 16:15:53\tGenotypeConcordance\tchecked     5,660,000 variants.  Elapsed time: 00:02:59s.  Time for last 10,000:    0s.  Last read position: 3:22,325,974\n",
      "INFO\t2017-01-12 16:15:53\tGenotypeConcordance\tchecked     5,670,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:23,034,968\n",
      "INFO\t2017-01-12 16:15:54\tGenotypeConcordance\tchecked     5,680,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:23,834,246\n",
      "INFO\t2017-01-12 16:15:54\tGenotypeConcordance\tchecked     5,690,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:24,648,750\n",
      "INFO\t2017-01-12 16:15:54\tGenotypeConcordance\tchecked     5,700,000 variants.  Elapsed time: 00:03:00s.  Time for last 10,000:    0s.  Last read position: 3:25,317,279\n",
      "INFO\t2017-01-12 16:15:55\tGenotypeConcordance\tchecked     5,710,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:26,164,092\n",
      "INFO\t2017-01-12 16:15:55\tGenotypeConcordance\tchecked     5,720,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:26,927,337\n",
      "INFO\t2017-01-12 16:15:55\tGenotypeConcordance\tchecked     5,730,000 variants.  Elapsed time: 00:03:01s.  Time for last 10,000:    0s.  Last read position: 3:27,753,599\n",
      "INFO\t2017-01-12 16:15:56\tGenotypeConcordance\tchecked     5,740,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:28,662,924\n",
      "INFO\t2017-01-12 16:15:56\tGenotypeConcordance\tchecked     5,750,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:29,472,792\n",
      "INFO\t2017-01-12 16:15:56\tGenotypeConcordance\tchecked     5,760,000 variants.  Elapsed time: 00:03:02s.  Time for last 10,000:    0s.  Last read position: 3:30,244,931\n",
      "INFO\t2017-01-12 16:15:57\tGenotypeConcordance\tchecked     5,770,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:31,034,548\n",
      "INFO\t2017-01-12 16:15:57\tGenotypeConcordance\tchecked     5,780,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:31,892,448\n",
      "INFO\t2017-01-12 16:15:57\tGenotypeConcordance\tchecked     5,790,000 variants.  Elapsed time: 00:03:03s.  Time for last 10,000:    0s.  Last read position: 3:32,663,921\n",
      "INFO\t2017-01-12 16:15:58\tGenotypeConcordance\tchecked     5,800,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:33,525,814\n",
      "INFO\t2017-01-12 16:15:58\tGenotypeConcordance\tchecked     5,810,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:34,520,755\n",
      "INFO\t2017-01-12 16:15:58\tGenotypeConcordance\tchecked     5,820,000 variants.  Elapsed time: 00:03:04s.  Time for last 10,000:    0s.  Last read position: 3:35,395,255\n",
      "INFO\t2017-01-12 16:15:58\tGenotypeConcordance\tchecked     5,830,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:36,280,768\n",
      "INFO\t2017-01-12 16:15:59\tGenotypeConcordance\tchecked     5,840,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:37,206,578\n",
      "INFO\t2017-01-12 16:15:59\tGenotypeConcordance\tchecked     5,850,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:38,189,828\n",
      "INFO\t2017-01-12 16:15:59\tGenotypeConcordance\tchecked     5,860,000 variants.  Elapsed time: 00:03:05s.  Time for last 10,000:    0s.  Last read position: 3:39,147,202\n",
      "INFO\t2017-01-12 16:16:00\tGenotypeConcordance\tchecked     5,870,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:40,005,584\n",
      "INFO\t2017-01-12 16:16:00\tGenotypeConcordance\tchecked     5,880,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:40,977,031\n",
      "INFO\t2017-01-12 16:16:00\tGenotypeConcordance\tchecked     5,890,000 variants.  Elapsed time: 00:03:06s.  Time for last 10,000:    0s.  Last read position: 3:41,817,711\n",
      "INFO\t2017-01-12 16:16:01\tGenotypeConcordance\tchecked     5,900,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:42,586,839\n",
      "INFO\t2017-01-12 16:16:01\tGenotypeConcordance\tchecked     5,910,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:43,567,237\n",
      "INFO\t2017-01-12 16:16:01\tGenotypeConcordance\tchecked     5,920,000 variants.  Elapsed time: 00:03:07s.  Time for last 10,000:    0s.  Last read position: 3:44,592,561\n",
      "INFO\t2017-01-12 16:16:01\tGenotypeConcordance\tchecked     5,930,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:45,523,962\n",
      "INFO\t2017-01-12 16:16:02\tGenotypeConcordance\tchecked     5,940,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:46,472,977\n",
      "INFO\t2017-01-12 16:16:02\tGenotypeConcordance\tchecked     5,950,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:47,594,105\n",
      "INFO\t2017-01-12 16:16:02\tGenotypeConcordance\tchecked     5,960,000 variants.  Elapsed time: 00:03:08s.  Time for last 10,000:    0s.  Last read position: 3:48,890,681\n",
      "INFO\t2017-01-12 16:16:03\tGenotypeConcordance\tchecked     5,970,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:50,240,933\n",
      "INFO\t2017-01-12 16:16:03\tGenotypeConcordance\tchecked     5,980,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:51,543,025\n",
      "INFO\t2017-01-12 16:16:03\tGenotypeConcordance\tchecked     5,990,000 variants.  Elapsed time: 00:03:09s.  Time for last 10,000:    0s.  Last read position: 3:52,807,018\n",
      "INFO\t2017-01-12 16:16:03\tGenotypeConcordance\tchecked     6,000,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:53,745,766\n",
      "INFO\t2017-01-12 16:16:04\tGenotypeConcordance\tchecked     6,010,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:54,552,208\n",
      "INFO\t2017-01-12 16:16:04\tGenotypeConcordance\tchecked     6,020,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:55,346,193\n",
      "INFO\t2017-01-12 16:16:04\tGenotypeConcordance\tchecked     6,030,000 variants.  Elapsed time: 00:03:10s.  Time for last 10,000:    0s.  Last read position: 3:56,245,895\n",
      "INFO\t2017-01-12 16:16:05\tGenotypeConcordance\tchecked     6,040,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:57,068,327\n",
      "INFO\t2017-01-12 16:16:05\tGenotypeConcordance\tchecked     6,050,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:58,022,175\n",
      "INFO\t2017-01-12 16:16:05\tGenotypeConcordance\tchecked     6,060,000 variants.  Elapsed time: 00:03:11s.  Time for last 10,000:    0s.  Last read position: 3:58,915,946\n",
      "INFO\t2017-01-12 16:16:06\tGenotypeConcordance\tchecked     6,070,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:59,744,362\n",
      "INFO\t2017-01-12 16:16:06\tGenotypeConcordance\tchecked     6,080,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:60,367,056\n",
      "INFO\t2017-01-12 16:16:06\tGenotypeConcordance\tchecked     6,090,000 variants.  Elapsed time: 00:03:12s.  Time for last 10,000:    0s.  Last read position: 3:61,074,082\n",
      "INFO\t2017-01-12 16:16:07\tGenotypeConcordance\tchecked     6,100,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:61,866,807\n",
      "INFO\t2017-01-12 16:16:07\tGenotypeConcordance\tchecked     6,110,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:62,711,417\n",
      "INFO\t2017-01-12 16:16:07\tGenotypeConcordance\tchecked     6,120,000 variants.  Elapsed time: 00:03:13s.  Time for last 10,000:    0s.  Last read position: 3:63,449,344\n",
      "INFO\t2017-01-12 16:16:08\tGenotypeConcordance\tchecked     6,130,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:64,345,206\n",
      "INFO\t2017-01-12 16:16:08\tGenotypeConcordance\tchecked     6,140,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:65,099,686\n",
      "INFO\t2017-01-12 16:16:08\tGenotypeConcordance\tchecked     6,150,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:65,870,455\n",
      "INFO\t2017-01-12 16:16:08\tGenotypeConcordance\tchecked     6,160,000 variants.  Elapsed time: 00:03:14s.  Time for last 10,000:    0s.  Last read position: 3:66,760,847\n",
      "INFO\t2017-01-12 16:16:09\tGenotypeConcordance\tchecked     6,170,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:67,476,577\n",
      "INFO\t2017-01-12 16:16:09\tGenotypeConcordance\tchecked     6,180,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:68,340,541\n",
      "INFO\t2017-01-12 16:16:09\tGenotypeConcordance\tchecked     6,190,000 variants.  Elapsed time: 00:03:15s.  Time for last 10,000:    0s.  Last read position: 3:69,163,405\n",
      "INFO\t2017-01-12 16:16:10\tGenotypeConcordance\tchecked     6,200,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:69,956,607\n",
      "INFO\t2017-01-12 16:16:10\tGenotypeConcordance\tchecked     6,210,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:70,920,041\n",
      "INFO\t2017-01-12 16:16:10\tGenotypeConcordance\tchecked     6,220,000 variants.  Elapsed time: 00:03:16s.  Time for last 10,000:    0s.  Last read position: 3:71,898,456\n",
      "INFO\t2017-01-12 16:16:11\tGenotypeConcordance\tchecked     6,230,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:72,736,441\n",
      "INFO\t2017-01-12 16:16:11\tGenotypeConcordance\tchecked     6,240,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:73,520,153\n",
      "INFO\t2017-01-12 16:16:11\tGenotypeConcordance\tchecked     6,250,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:74,323,700\n",
      "INFO\t2017-01-12 16:16:11\tGenotypeConcordance\tchecked     6,260,000 variants.  Elapsed time: 00:03:17s.  Time for last 10,000:    0s.  Last read position: 3:75,158,571\n",
      "INFO\t2017-01-12 16:16:12\tGenotypeConcordance\tchecked     6,270,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:75,694,903\n",
      "INFO\t2017-01-12 16:16:12\tGenotypeConcordance\tchecked     6,280,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:76,169,067\n",
      "INFO\t2017-01-12 16:16:12\tGenotypeConcordance\tchecked     6,290,000 variants.  Elapsed time: 00:03:18s.  Time for last 10,000:    0s.  Last read position: 3:76,998,027\n",
      "INFO\t2017-01-12 16:16:13\tGenotypeConcordance\tchecked     6,300,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:77,832,564\n",
      "INFO\t2017-01-12 16:16:13\tGenotypeConcordance\tchecked     6,310,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:78,713,353\n",
      "INFO\t2017-01-12 16:16:13\tGenotypeConcordance\tchecked     6,320,000 variants.  Elapsed time: 00:03:19s.  Time for last 10,000:    0s.  Last read position: 3:79,716,784\n",
      "INFO\t2017-01-12 16:16:13\tGenotypeConcordance\tchecked     6,330,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:80,551,644\n",
      "INFO\t2017-01-12 16:16:14\tGenotypeConcordance\tchecked     6,340,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:81,508,145\n",
      "INFO\t2017-01-12 16:16:14\tGenotypeConcordance\tchecked     6,350,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:82,510,638\n",
      "INFO\t2017-01-12 16:16:14\tGenotypeConcordance\tchecked     6,360,000 variants.  Elapsed time: 00:03:20s.  Time for last 10,000:    0s.  Last read position: 3:83,454,824\n",
      "INFO\t2017-01-12 16:16:15\tGenotypeConcordance\tchecked     6,370,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:84,375,440\n",
      "INFO\t2017-01-12 16:16:15\tGenotypeConcordance\tchecked     6,380,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:85,236,039\n",
      "INFO\t2017-01-12 16:16:15\tGenotypeConcordance\tchecked     6,390,000 variants.  Elapsed time: 00:03:21s.  Time for last 10,000:    0s.  Last read position: 3:86,119,954\n",
      "INFO\t2017-01-12 16:16:16\tGenotypeConcordance\tchecked     6,400,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:86,997,521\n",
      "INFO\t2017-01-12 16:16:16\tGenotypeConcordance\tchecked     6,410,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:87,868,712\n",
      "INFO\t2017-01-12 16:16:16\tGenotypeConcordance\tchecked     6,420,000 variants.  Elapsed time: 00:03:22s.  Time for last 10,000:    0s.  Last read position: 3:88,716,678\n",
      "INFO\t2017-01-12 16:16:17\tGenotypeConcordance\tchecked     6,430,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:89,612,355\n",
      "INFO\t2017-01-12 16:16:17\tGenotypeConcordance\tchecked     6,440,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:90,430,115\n",
      "INFO\t2017-01-12 16:16:17\tGenotypeConcordance\tchecked     6,450,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:94,327,307\n",
      "INFO\t2017-01-12 16:16:17\tGenotypeConcordance\tchecked     6,460,000 variants.  Elapsed time: 00:03:23s.  Time for last 10,000:    0s.  Last read position: 3:95,106,745\n",
      "INFO\t2017-01-12 16:16:18\tGenotypeConcordance\tchecked     6,470,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:95,861,359\n",
      "INFO\t2017-01-12 16:16:18\tGenotypeConcordance\tchecked     6,480,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:96,699,514\n",
      "INFO\t2017-01-12 16:16:18\tGenotypeConcordance\tchecked     6,490,000 variants.  Elapsed time: 00:03:24s.  Time for last 10,000:    0s.  Last read position: 3:97,764,431\n",
      "INFO\t2017-01-12 16:16:19\tGenotypeConcordance\tchecked     6,500,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:98,394,062\n",
      "INFO\t2017-01-12 16:16:19\tGenotypeConcordance\tchecked     6,510,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:99,150,890\n",
      "INFO\t2017-01-12 16:16:19\tGenotypeConcordance\tchecked     6,520,000 variants.  Elapsed time: 00:03:25s.  Time for last 10,000:    0s.  Last read position: 3:100,192,167\n",
      "INFO\t2017-01-12 16:16:19\tGenotypeConcordance\tchecked     6,530,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:101,042,354\n",
      "INFO\t2017-01-12 16:16:20\tGenotypeConcordance\tchecked     6,540,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:102,053,029\n",
      "INFO\t2017-01-12 16:16:20\tGenotypeConcordance\tchecked     6,550,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:102,939,521\n",
      "INFO\t2017-01-12 16:16:20\tGenotypeConcordance\tchecked     6,560,000 variants.  Elapsed time: 00:03:26s.  Time for last 10,000:    0s.  Last read position: 3:103,694,562\n",
      "INFO\t2017-01-12 16:16:21\tGenotypeConcordance\tchecked     6,570,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:104,515,697\n",
      "INFO\t2017-01-12 16:16:21\tGenotypeConcordance\tchecked     6,580,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:105,348,141\n",
      "INFO\t2017-01-12 16:16:21\tGenotypeConcordance\tchecked     6,590,000 variants.  Elapsed time: 00:03:27s.  Time for last 10,000:    0s.  Last read position: 3:106,153,752\n",
      "INFO\t2017-01-12 16:16:22\tGenotypeConcordance\tchecked     6,600,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:107,027,393\n",
      "INFO\t2017-01-12 16:16:22\tGenotypeConcordance\tchecked     6,610,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:108,030,714\n",
      "INFO\t2017-01-12 16:16:22\tGenotypeConcordance\tchecked     6,620,000 variants.  Elapsed time: 00:03:28s.  Time for last 10,000:    0s.  Last read position: 3:108,975,423\n",
      "INFO\t2017-01-12 16:16:23\tGenotypeConcordance\tchecked     6,630,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:109,796,515\n",
      "INFO\t2017-01-12 16:16:23\tGenotypeConcordance\tchecked     6,640,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:110,729,247\n",
      "INFO\t2017-01-12 16:16:23\tGenotypeConcordance\tchecked     6,650,000 variants.  Elapsed time: 00:03:29s.  Time for last 10,000:    0s.  Last read position: 3:111,657,902\n",
      "INFO\t2017-01-12 16:16:24\tGenotypeConcordance\tchecked     6,660,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:112,508,344\n",
      "INFO\t2017-01-12 16:16:24\tGenotypeConcordance\tchecked     6,670,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:113,334,752\n",
      "INFO\t2017-01-12 16:16:24\tGenotypeConcordance\tchecked     6,680,000 variants.  Elapsed time: 00:03:30s.  Time for last 10,000:    0s.  Last read position: 3:114,394,131\n",
      "INFO\t2017-01-12 16:16:24\tGenotypeConcordance\tchecked     6,690,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:115,515,376\n",
      "INFO\t2017-01-12 16:16:25\tGenotypeConcordance\tchecked     6,700,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:116,424,880\n",
      "INFO\t2017-01-12 16:16:25\tGenotypeConcordance\tchecked     6,710,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:117,241,077\n",
      "INFO\t2017-01-12 16:16:25\tGenotypeConcordance\tchecked     6,720,000 variants.  Elapsed time: 00:03:31s.  Time for last 10,000:    0s.  Last read position: 3:118,082,137\n",
      "INFO\t2017-01-12 16:16:26\tGenotypeConcordance\tchecked     6,730,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:118,972,605\n",
      "INFO\t2017-01-12 16:16:26\tGenotypeConcordance\tchecked     6,740,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:119,878,441\n",
      "INFO\t2017-01-12 16:16:26\tGenotypeConcordance\tchecked     6,750,000 variants.  Elapsed time: 00:03:32s.  Time for last 10,000:    0s.  Last read position: 3:120,792,187\n",
      "INFO\t2017-01-12 16:16:27\tGenotypeConcordance\tchecked     6,760,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:121,807,420\n",
      "INFO\t2017-01-12 16:16:27\tGenotypeConcordance\tchecked     6,770,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:122,764,377\n",
      "INFO\t2017-01-12 16:16:27\tGenotypeConcordance\tchecked     6,780,000 variants.  Elapsed time: 00:03:33s.  Time for last 10,000:    0s.  Last read position: 3:123,678,926\n",
      "INFO\t2017-01-12 16:16:28\tGenotypeConcordance\tchecked     6,790,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:124,600,003\n",
      "INFO\t2017-01-12 16:16:28\tGenotypeConcordance\tchecked     6,800,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:125,454,681\n",
      "INFO\t2017-01-12 16:16:28\tGenotypeConcordance\tchecked     6,810,000 variants.  Elapsed time: 00:03:34s.  Time for last 10,000:    0s.  Last read position: 3:126,016,956\n",
      "INFO\t2017-01-12 16:16:28\tGenotypeConcordance\tchecked     6,820,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:126,896,517\n",
      "INFO\t2017-01-12 16:16:29\tGenotypeConcordance\tchecked     6,830,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:127,779,944\n",
      "INFO\t2017-01-12 16:16:29\tGenotypeConcordance\tchecked     6,840,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:128,694,269\n",
      "INFO\t2017-01-12 16:16:29\tGenotypeConcordance\tchecked     6,850,000 variants.  Elapsed time: 00:03:35s.  Time for last 10,000:    0s.  Last read position: 3:129,642,923\n",
      "INFO\t2017-01-12 16:16:30\tGenotypeConcordance\tchecked     6,860,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:130,278,086\n",
      "INFO\t2017-01-12 16:16:30\tGenotypeConcordance\tchecked     6,870,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:131,283,892\n",
      "INFO\t2017-01-12 16:16:30\tGenotypeConcordance\tchecked     6,880,000 variants.  Elapsed time: 00:03:36s.  Time for last 10,000:    0s.  Last read position: 3:132,072,419\n",
      "INFO\t2017-01-12 16:16:31\tGenotypeConcordance\tchecked     6,890,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 3:132,944,181\n",
      "INFO\t2017-01-12 16:16:31\tGenotypeConcordance\tchecked     6,900,000 variants.  Elapsed time: 00:03:37s.  Time for last 10,000:    0s.  Last read position: 3:133,776,063\n",
      "INFO\t2017-01-12 16:16:31\tGenotypeConcordance\tchecked     6,910,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 3:134,677,613\n",
      "INFO\t2017-01-12 16:16:32\tGenotypeConcordance\tchecked     6,920,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 3:135,647,261\n",
      "INFO\t2017-01-12 16:16:32\tGenotypeConcordance\tchecked     6,930,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 3:136,701,538\n",
      "INFO\t2017-01-12 16:16:32\tGenotypeConcordance\tchecked     6,940,000 variants.  Elapsed time: 00:03:38s.  Time for last 10,000:    0s.  Last read position: 3:137,668,783\n",
      "INFO\t2017-01-12 16:16:33\tGenotypeConcordance\tchecked     6,950,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 3:138,676,755\n",
      "INFO\t2017-01-12 16:16:33\tGenotypeConcordance\tchecked     6,960,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 3:139,651,255\n",
      "INFO\t2017-01-12 16:16:33\tGenotypeConcordance\tchecked     6,970,000 variants.  Elapsed time: 00:03:39s.  Time for last 10,000:    0s.  Last read position: 3:140,523,699\n",
      "INFO\t2017-01-12 16:16:34\tGenotypeConcordance\tchecked     6,980,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 3:141,412,883\n",
      "INFO\t2017-01-12 16:16:34\tGenotypeConcordance\tchecked     6,990,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 3:142,335,410\n",
      "INFO\t2017-01-12 16:16:34\tGenotypeConcordance\tchecked     7,000,000 variants.  Elapsed time: 00:03:40s.  Time for last 10,000:    0s.  Last read position: 3:143,238,874\n",
      "INFO\t2017-01-12 16:16:35\tGenotypeConcordance\tchecked     7,010,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 3:144,113,799\n",
      "INFO\t2017-01-12 16:16:35\tGenotypeConcordance\tchecked     7,020,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 3:144,923,706\n",
      "INFO\t2017-01-12 16:16:35\tGenotypeConcordance\tchecked     7,030,000 variants.  Elapsed time: 00:03:41s.  Time for last 10,000:    0s.  Last read position: 3:145,686,843\n",
      "INFO\t2017-01-12 16:16:35\tGenotypeConcordance\tchecked     7,040,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 3:146,478,254\n",
      "INFO\t2017-01-12 16:16:36\tGenotypeConcordance\tchecked     7,050,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 3:147,498,662\n",
      "INFO\t2017-01-12 16:16:36\tGenotypeConcordance\tchecked     7,060,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 3:148,355,413\n",
      "INFO\t2017-01-12 16:16:36\tGenotypeConcordance\tchecked     7,070,000 variants.  Elapsed time: 00:03:42s.  Time for last 10,000:    0s.  Last read position: 3:149,199,064\n",
      "INFO\t2017-01-12 16:16:37\tGenotypeConcordance\tchecked     7,080,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 3:150,148,092\n",
      "INFO\t2017-01-12 16:16:37\tGenotypeConcordance\tchecked     7,090,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 3:151,097,654\n",
      "INFO\t2017-01-12 16:16:37\tGenotypeConcordance\tchecked     7,100,000 variants.  Elapsed time: 00:03:43s.  Time for last 10,000:    0s.  Last read position: 3:151,986,427\n",
      "INFO\t2017-01-12 16:16:38\tGenotypeConcordance\tchecked     7,110,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 3:152,886,681\n",
      "INFO\t2017-01-12 16:16:38\tGenotypeConcordance\tchecked     7,120,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 3:153,699,976\n",
      "INFO\t2017-01-12 16:16:38\tGenotypeConcordance\tchecked     7,130,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 3:154,593,619\n",
      "INFO\t2017-01-12 16:16:38\tGenotypeConcordance\tchecked     7,140,000 variants.  Elapsed time: 00:03:44s.  Time for last 10,000:    0s.  Last read position: 3:155,486,703\n",
      "INFO\t2017-01-12 16:16:39\tGenotypeConcordance\tchecked     7,150,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 3:156,390,308\n",
      "INFO\t2017-01-12 16:16:39\tGenotypeConcordance\tchecked     7,160,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 3:157,358,048\n",
      "INFO\t2017-01-12 16:16:39\tGenotypeConcordance\tchecked     7,170,000 variants.  Elapsed time: 00:03:45s.  Time for last 10,000:    0s.  Last read position: 3:158,271,690\n",
      "INFO\t2017-01-12 16:16:40\tGenotypeConcordance\tchecked     7,180,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 3:159,142,234\n",
      "INFO\t2017-01-12 16:16:40\tGenotypeConcordance\tchecked     7,190,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 3:159,977,751\n",
      "INFO\t2017-01-12 16:16:40\tGenotypeConcordance\tchecked     7,200,000 variants.  Elapsed time: 00:03:46s.  Time for last 10,000:    0s.  Last read position: 3:160,895,762\n",
      "INFO\t2017-01-12 16:16:41\tGenotypeConcordance\tchecked     7,210,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 3:161,729,127\n",
      "INFO\t2017-01-12 16:16:41\tGenotypeConcordance\tchecked     7,220,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 3:162,494,864\n",
      "INFO\t2017-01-12 16:16:41\tGenotypeConcordance\tchecked     7,230,000 variants.  Elapsed time: 00:03:47s.  Time for last 10,000:    0s.  Last read position: 3:163,367,841\n",
      "INFO\t2017-01-12 16:16:42\tGenotypeConcordance\tchecked     7,240,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 3:164,200,352\n",
      "INFO\t2017-01-12 16:16:42\tGenotypeConcordance\tchecked     7,250,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 3:165,126,400\n",
      "INFO\t2017-01-12 16:16:42\tGenotypeConcordance\tchecked     7,260,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 3:165,943,024\n",
      "INFO\t2017-01-12 16:16:42\tGenotypeConcordance\tchecked     7,270,000 variants.  Elapsed time: 00:03:48s.  Time for last 10,000:    0s.  Last read position: 3:166,736,793\n",
      "INFO\t2017-01-12 16:16:43\tGenotypeConcordance\tchecked     7,280,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 3:167,666,443\n",
      "INFO\t2017-01-12 16:16:43\tGenotypeConcordance\tchecked     7,290,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 3:168,628,196\n",
      "INFO\t2017-01-12 16:16:43\tGenotypeConcordance\tchecked     7,300,000 variants.  Elapsed time: 00:03:49s.  Time for last 10,000:    0s.  Last read position: 3:169,554,129\n",
      "INFO\t2017-01-12 16:16:44\tGenotypeConcordance\tchecked     7,310,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 3:170,420,463\n",
      "INFO\t2017-01-12 16:16:44\tGenotypeConcordance\tchecked     7,320,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 3:171,326,315\n",
      "INFO\t2017-01-12 16:16:44\tGenotypeConcordance\tchecked     7,330,000 variants.  Elapsed time: 00:03:50s.  Time for last 10,000:    0s.  Last read position: 3:172,239,988\n",
      "INFO\t2017-01-12 16:16:44\tGenotypeConcordance\tchecked     7,340,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 3:173,149,550\n",
      "INFO\t2017-01-12 16:16:45\tGenotypeConcordance\tchecked     7,350,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 3:174,049,118\n",
      "INFO\t2017-01-12 16:16:45\tGenotypeConcordance\tchecked     7,360,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 3:174,838,151\n",
      "INFO\t2017-01-12 16:16:45\tGenotypeConcordance\tchecked     7,370,000 variants.  Elapsed time: 00:03:51s.  Time for last 10,000:    0s.  Last read position: 3:175,559,562\n",
      "INFO\t2017-01-12 16:16:46\tGenotypeConcordance\tchecked     7,380,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 3:176,421,790\n",
      "INFO\t2017-01-12 16:16:46\tGenotypeConcordance\tchecked     7,390,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 3:177,215,858\n",
      "INFO\t2017-01-12 16:16:46\tGenotypeConcordance\tchecked     7,400,000 variants.  Elapsed time: 00:03:52s.  Time for last 10,000:    0s.  Last read position: 3:178,145,346\n",
      "INFO\t2017-01-12 16:16:47\tGenotypeConcordance\tchecked     7,410,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 3:179,073,852\n",
      "INFO\t2017-01-12 16:16:47\tGenotypeConcordance\tchecked     7,420,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 3:180,029,935\n",
      "INFO\t2017-01-12 16:16:47\tGenotypeConcordance\tchecked     7,430,000 variants.  Elapsed time: 00:03:53s.  Time for last 10,000:    0s.  Last read position: 3:181,106,125\n",
      "INFO\t2017-01-12 16:16:48\tGenotypeConcordance\tchecked     7,440,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 3:182,153,831\n",
      "INFO\t2017-01-12 16:16:48\tGenotypeConcordance\tchecked     7,450,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 3:183,126,506\n",
      "INFO\t2017-01-12 16:16:48\tGenotypeConcordance\tchecked     7,460,000 variants.  Elapsed time: 00:03:54s.  Time for last 10,000:    0s.  Last read position: 3:183,985,669\n",
      "INFO\t2017-01-12 16:16:48\tGenotypeConcordance\tchecked     7,470,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 3:184,834,329\n",
      "INFO\t2017-01-12 16:16:49\tGenotypeConcordance\tchecked     7,480,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 3:185,793,273\n",
      "INFO\t2017-01-12 16:16:49\tGenotypeConcordance\tchecked     7,490,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 3:186,628,163\n",
      "INFO\t2017-01-12 16:16:49\tGenotypeConcordance\tchecked     7,500,000 variants.  Elapsed time: 00:03:55s.  Time for last 10,000:    0s.  Last read position: 3:187,405,595\n",
      "INFO\t2017-01-12 16:16:50\tGenotypeConcordance\tchecked     7,510,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 3:188,229,424\n",
      "INFO\t2017-01-12 16:16:50\tGenotypeConcordance\tchecked     7,520,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 3:189,133,827\n",
      "INFO\t2017-01-12 16:16:50\tGenotypeConcordance\tchecked     7,530,000 variants.  Elapsed time: 00:03:56s.  Time for last 10,000:    0s.  Last read position: 3:189,871,351\n",
      "INFO\t2017-01-12 16:16:51\tGenotypeConcordance\tchecked     7,540,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 3:190,595,591\n",
      "INFO\t2017-01-12 16:16:51\tGenotypeConcordance\tchecked     7,550,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 3:191,343,146\n",
      "INFO\t2017-01-12 16:16:51\tGenotypeConcordance\tchecked     7,560,000 variants.  Elapsed time: 00:03:57s.  Time for last 10,000:    0s.  Last read position: 3:192,081,284\n",
      "INFO\t2017-01-12 16:16:52\tGenotypeConcordance\tchecked     7,570,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 3:192,852,448\n",
      "INFO\t2017-01-12 16:16:52\tGenotypeConcordance\tchecked     7,580,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 3:193,718,465\n",
      "INFO\t2017-01-12 16:16:52\tGenotypeConcordance\tchecked     7,590,000 variants.  Elapsed time: 00:03:58s.  Time for last 10,000:    0s.  Last read position: 3:194,398,645\n",
      "INFO\t2017-01-12 16:16:53\tGenotypeConcordance\tchecked     7,600,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 3:195,076,877\n",
      "INFO\t2017-01-12 16:16:53\tGenotypeConcordance\tchecked     7,610,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 3:195,597,503\n",
      "INFO\t2017-01-12 16:16:53\tGenotypeConcordance\tchecked     7,620,000 variants.  Elapsed time: 00:03:59s.  Time for last 10,000:    0s.  Last read position: 3:196,255,340\n",
      "INFO\t2017-01-12 16:16:54\tGenotypeConcordance\tchecked     7,630,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 3:196,988,678\n",
      "INFO\t2017-01-12 16:16:54\tGenotypeConcordance\tchecked     7,640,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 3:197,702,371\n",
      "INFO\t2017-01-12 16:16:54\tGenotypeConcordance\tchecked     7,650,000 variants.  Elapsed time: 00:04:00s.  Time for last 10,000:    0s.  Last read position: 4:542,898\n",
      "INFO\t2017-01-12 16:16:55\tGenotypeConcordance\tchecked     7,660,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:1,221,441\n",
      "INFO\t2017-01-12 16:16:55\tGenotypeConcordance\tchecked     7,670,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:1,910,811\n",
      "INFO\t2017-01-12 16:16:55\tGenotypeConcordance\tchecked     7,680,000 variants.  Elapsed time: 00:04:01s.  Time for last 10,000:    0s.  Last read position: 4:2,749,343\n",
      "INFO\t2017-01-12 16:16:55\tGenotypeConcordance\tchecked     7,690,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:3,514,344\n",
      "INFO\t2017-01-12 16:16:56\tGenotypeConcordance\tchecked     7,700,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:3,929,865\n",
      "INFO\t2017-01-12 16:16:56\tGenotypeConcordance\tchecked     7,710,000 variants.  Elapsed time: 00:04:02s.  Time for last 10,000:    0s.  Last read position: 4:4,344,979\n",
      "INFO\t2017-01-12 16:16:56\tGenotypeConcordance\tchecked     7,720,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:5,089,303\n",
      "INFO\t2017-01-12 16:16:57\tGenotypeConcordance\tchecked     7,730,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:5,669,247\n",
      "INFO\t2017-01-12 16:16:57\tGenotypeConcordance\tchecked     7,740,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:6,298,705\n",
      "INFO\t2017-01-12 16:16:57\tGenotypeConcordance\tchecked     7,750,000 variants.  Elapsed time: 00:04:03s.  Time for last 10,000:    0s.  Last read position: 4:6,957,332\n",
      "INFO\t2017-01-12 16:16:58\tGenotypeConcordance\tchecked     7,760,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:7,535,219\n",
      "INFO\t2017-01-12 16:16:58\tGenotypeConcordance\tchecked     7,770,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:8,114,300\n",
      "INFO\t2017-01-12 16:16:58\tGenotypeConcordance\tchecked     7,780,000 variants.  Elapsed time: 00:04:04s.  Time for last 10,000:    0s.  Last read position: 4:8,722,998\n",
      "INFO\t2017-01-12 16:16:59\tGenotypeConcordance\tchecked     7,790,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:9,183,124\n",
      "INFO\t2017-01-12 16:16:59\tGenotypeConcordance\tchecked     7,800,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:9,669,609\n",
      "INFO\t2017-01-12 16:16:59\tGenotypeConcordance\tchecked     7,810,000 variants.  Elapsed time: 00:04:05s.  Time for last 10,000:    0s.  Last read position: 4:10,210,906\n",
      "INFO\t2017-01-12 16:16:59\tGenotypeConcordance\tchecked     7,820,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:10,942,705\n",
      "INFO\t2017-01-12 16:17:00\tGenotypeConcordance\tchecked     7,830,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:11,650,710\n",
      "INFO\t2017-01-12 16:17:00\tGenotypeConcordance\tchecked     7,840,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:12,318,791\n",
      "INFO\t2017-01-12 16:17:00\tGenotypeConcordance\tchecked     7,850,000 variants.  Elapsed time: 00:04:06s.  Time for last 10,000:    0s.  Last read position: 4:13,018,888\n",
      "INFO\t2017-01-12 16:17:01\tGenotypeConcordance\tchecked     7,860,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:13,864,746\n",
      "INFO\t2017-01-12 16:17:01\tGenotypeConcordance\tchecked     7,870,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:14,624,305\n",
      "INFO\t2017-01-12 16:17:01\tGenotypeConcordance\tchecked     7,880,000 variants.  Elapsed time: 00:04:07s.  Time for last 10,000:    0s.  Last read position: 4:15,457,861\n",
      "INFO\t2017-01-12 16:17:02\tGenotypeConcordance\tchecked     7,890,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:16,377,284\n",
      "INFO\t2017-01-12 16:17:02\tGenotypeConcordance\tchecked     7,900,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:17,240,787\n",
      "INFO\t2017-01-12 16:17:02\tGenotypeConcordance\tchecked     7,910,000 variants.  Elapsed time: 00:04:08s.  Time for last 10,000:    0s.  Last read position: 4:18,080,821\n",
      "INFO\t2017-01-12 16:17:03\tGenotypeConcordance\tchecked     7,920,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:18,918,814\n",
      "INFO\t2017-01-12 16:17:03\tGenotypeConcordance\tchecked     7,930,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:19,736,258\n",
      "INFO\t2017-01-12 16:17:03\tGenotypeConcordance\tchecked     7,940,000 variants.  Elapsed time: 00:04:09s.  Time for last 10,000:    0s.  Last read position: 4:20,560,883\n",
      "INFO\t2017-01-12 16:17:04\tGenotypeConcordance\tchecked     7,950,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:21,283,211\n",
      "INFO\t2017-01-12 16:17:04\tGenotypeConcordance\tchecked     7,960,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:22,057,888\n",
      "INFO\t2017-01-12 16:17:04\tGenotypeConcordance\tchecked     7,970,000 variants.  Elapsed time: 00:04:10s.  Time for last 10,000:    0s.  Last read position: 4:22,800,848\n",
      "INFO\t2017-01-12 16:17:04\tGenotypeConcordance\tchecked     7,980,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:23,656,138\n",
      "INFO\t2017-01-12 16:17:05\tGenotypeConcordance\tchecked     7,990,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:24,464,042\n",
      "INFO\t2017-01-12 16:17:05\tGenotypeConcordance\tchecked     8,000,000 variants.  Elapsed time: 00:04:11s.  Time for last 10,000:    0s.  Last read position: 4:25,290,880\n",
      "INFO\t2017-01-12 16:17:05\tGenotypeConcordance\tchecked     8,010,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:26,074,011\n",
      "INFO\t2017-01-12 16:17:06\tGenotypeConcordance\tchecked     8,020,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:26,996,783\n",
      "INFO\t2017-01-12 16:17:06\tGenotypeConcordance\tchecked     8,030,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:27,795,241\n",
      "INFO\t2017-01-12 16:17:06\tGenotypeConcordance\tchecked     8,040,000 variants.  Elapsed time: 00:04:12s.  Time for last 10,000:    0s.  Last read position: 4:28,570,198\n",
      "INFO\t2017-01-12 16:17:07\tGenotypeConcordance\tchecked     8,050,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:29,369,334\n",
      "INFO\t2017-01-12 16:17:07\tGenotypeConcordance\tchecked     8,060,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:30,159,982\n",
      "INFO\t2017-01-12 16:17:07\tGenotypeConcordance\tchecked     8,070,000 variants.  Elapsed time: 00:04:13s.  Time for last 10,000:    0s.  Last read position: 4:30,953,996\n",
      "INFO\t2017-01-12 16:17:08\tGenotypeConcordance\tchecked     8,080,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:31,734,106\n",
      "INFO\t2017-01-12 16:17:08\tGenotypeConcordance\tchecked     8,090,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:32,484,424\n",
      "INFO\t2017-01-12 16:17:08\tGenotypeConcordance\tchecked     8,100,000 variants.  Elapsed time: 00:04:14s.  Time for last 10,000:    0s.  Last read position: 4:33,203,384\n",
      "INFO\t2017-01-12 16:17:09\tGenotypeConcordance\tchecked     8,110,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:34,051,125\n",
      "INFO\t2017-01-12 16:17:09\tGenotypeConcordance\tchecked     8,120,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:34,917,773\n",
      "INFO\t2017-01-12 16:17:09\tGenotypeConcordance\tchecked     8,130,000 variants.  Elapsed time: 00:04:15s.  Time for last 10,000:    0s.  Last read position: 4:35,642,436\n",
      "INFO\t2017-01-12 16:17:09\tGenotypeConcordance\tchecked     8,140,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:36,438,525\n",
      "INFO\t2017-01-12 16:17:10\tGenotypeConcordance\tchecked     8,150,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:37,172,706\n",
      "INFO\t2017-01-12 16:17:10\tGenotypeConcordance\tchecked     8,160,000 variants.  Elapsed time: 00:04:16s.  Time for last 10,000:    0s.  Last read position: 4:37,925,177\n",
      "INFO\t2017-01-12 16:17:10\tGenotypeConcordance\tchecked     8,170,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:38,750,185\n",
      "INFO\t2017-01-12 16:17:11\tGenotypeConcordance\tchecked     8,180,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:39,591,005\n",
      "INFO\t2017-01-12 16:17:11\tGenotypeConcordance\tchecked     8,190,000 variants.  Elapsed time: 00:04:17s.  Time for last 10,000:    0s.  Last read position: 4:40,407,664\n",
      "INFO\t2017-01-12 16:17:11\tGenotypeConcordance\tchecked     8,200,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:41,208,436\n",
      "INFO\t2017-01-12 16:17:12\tGenotypeConcordance\tchecked     8,210,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:42,117,633\n",
      "INFO\t2017-01-12 16:17:12\tGenotypeConcordance\tchecked     8,220,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:42,908,795\n",
      "INFO\t2017-01-12 16:17:12\tGenotypeConcordance\tchecked     8,230,000 variants.  Elapsed time: 00:04:18s.  Time for last 10,000:    0s.  Last read position: 4:43,683,418\n",
      "INFO\t2017-01-12 16:17:13\tGenotypeConcordance\tchecked     8,240,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:44,537,786\n",
      "INFO\t2017-01-12 16:17:13\tGenotypeConcordance\tchecked     8,250,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:45,281,592\n",
      "INFO\t2017-01-12 16:17:13\tGenotypeConcordance\tchecked     8,260,000 variants.  Elapsed time: 00:04:19s.  Time for last 10,000:    0s.  Last read position: 4:46,140,691\n",
      "INFO\t2017-01-12 16:17:14\tGenotypeConcordance\tchecked     8,270,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:47,056,073\n",
      "INFO\t2017-01-12 16:17:14\tGenotypeConcordance\tchecked     8,280,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:48,060,169\n",
      "INFO\t2017-01-12 16:17:14\tGenotypeConcordance\tchecked     8,290,000 variants.  Elapsed time: 00:04:20s.  Time for last 10,000:    0s.  Last read position: 4:49,095,993\n",
      "INFO\t2017-01-12 16:17:14\tGenotypeConcordance\tchecked     8,300,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:49,334,997\n",
      "INFO\t2017-01-12 16:17:15\tGenotypeConcordance\tchecked     8,310,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:52,851,830\n",
      "INFO\t2017-01-12 16:17:15\tGenotypeConcordance\tchecked     8,320,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:53,712,121\n",
      "INFO\t2017-01-12 16:17:15\tGenotypeConcordance\tchecked     8,330,000 variants.  Elapsed time: 00:04:21s.  Time for last 10,000:    0s.  Last read position: 4:54,633,846\n",
      "INFO\t2017-01-12 16:17:16\tGenotypeConcordance\tchecked     8,340,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:55,490,518\n",
      "INFO\t2017-01-12 16:17:16\tGenotypeConcordance\tchecked     8,350,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:56,300,941\n",
      "INFO\t2017-01-12 16:17:16\tGenotypeConcordance\tchecked     8,360,000 variants.  Elapsed time: 00:04:22s.  Time for last 10,000:    0s.  Last read position: 4:57,171,987\n",
      "INFO\t2017-01-12 16:17:17\tGenotypeConcordance\tchecked     8,370,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:57,934,235\n",
      "INFO\t2017-01-12 16:17:17\tGenotypeConcordance\tchecked     8,380,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:58,608,254\n",
      "INFO\t2017-01-12 16:17:17\tGenotypeConcordance\tchecked     8,390,000 variants.  Elapsed time: 00:04:23s.  Time for last 10,000:    0s.  Last read position: 4:59,261,192\n",
      "INFO\t2017-01-12 16:17:18\tGenotypeConcordance\tchecked     8,400,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:60,033,417\n",
      "INFO\t2017-01-12 16:17:18\tGenotypeConcordance\tchecked     8,410,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:60,690,523\n",
      "INFO\t2017-01-12 16:17:18\tGenotypeConcordance\tchecked     8,420,000 variants.  Elapsed time: 00:04:24s.  Time for last 10,000:    0s.  Last read position: 4:61,375,698\n",
      "INFO\t2017-01-12 16:17:19\tGenotypeConcordance\tchecked     8,430,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:62,195,940\n",
      "INFO\t2017-01-12 16:17:19\tGenotypeConcordance\tchecked     8,440,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:63,135,706\n",
      "INFO\t2017-01-12 16:17:19\tGenotypeConcordance\tchecked     8,450,000 variants.  Elapsed time: 00:04:25s.  Time for last 10,000:    0s.  Last read position: 4:63,808,535\n",
      "INFO\t2017-01-12 16:17:19\tGenotypeConcordance\tchecked     8,460,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:64,456,321\n",
      "INFO\t2017-01-12 16:17:20\tGenotypeConcordance\tchecked     8,470,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:65,227,122\n",
      "INFO\t2017-01-12 16:17:20\tGenotypeConcordance\tchecked     8,480,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:65,930,371\n",
      "INFO\t2017-01-12 16:17:20\tGenotypeConcordance\tchecked     8,490,000 variants.  Elapsed time: 00:04:26s.  Time for last 10,000:    0s.  Last read position: 4:66,696,287\n",
      "INFO\t2017-01-12 16:17:21\tGenotypeConcordance\tchecked     8,500,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:67,443,752\n",
      "INFO\t2017-01-12 16:17:21\tGenotypeConcordance\tchecked     8,510,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:68,269,826\n",
      "INFO\t2017-01-12 16:17:21\tGenotypeConcordance\tchecked     8,520,000 variants.  Elapsed time: 00:04:27s.  Time for last 10,000:    0s.  Last read position: 4:69,141,149\n",
      "INFO\t2017-01-12 16:17:22\tGenotypeConcordance\tchecked     8,530,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:69,867,584\n",
      "INFO\t2017-01-12 16:17:22\tGenotypeConcordance\tchecked     8,540,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:70,460,328\n",
      "INFO\t2017-01-12 16:17:22\tGenotypeConcordance\tchecked     8,550,000 variants.  Elapsed time: 00:04:28s.  Time for last 10,000:    0s.  Last read position: 4:71,210,727\n",
      "INFO\t2017-01-12 16:17:23\tGenotypeConcordance\tchecked     8,560,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:71,994,066\n",
      "INFO\t2017-01-12 16:17:23\tGenotypeConcordance\tchecked     8,570,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:72,868,259\n",
      "INFO\t2017-01-12 16:17:23\tGenotypeConcordance\tchecked     8,580,000 variants.  Elapsed time: 00:04:29s.  Time for last 10,000:    0s.  Last read position: 4:73,633,538\n",
      "INFO\t2017-01-12 16:17:24\tGenotypeConcordance\tchecked     8,590,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:74,539,627\n",
      "INFO\t2017-01-12 16:17:24\tGenotypeConcordance\tchecked     8,600,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:75,534,895\n",
      "INFO\t2017-01-12 16:17:24\tGenotypeConcordance\tchecked     8,610,000 variants.  Elapsed time: 00:04:30s.  Time for last 10,000:    0s.  Last read position: 4:76,302,332\n",
      "INFO\t2017-01-12 16:17:25\tGenotypeConcordance\tchecked     8,620,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:77,132,071\n",
      "INFO\t2017-01-12 16:17:25\tGenotypeConcordance\tchecked     8,630,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:78,001,508\n",
      "INFO\t2017-01-12 16:17:25\tGenotypeConcordance\tchecked     8,640,000 variants.  Elapsed time: 00:04:31s.  Time for last 10,000:    0s.  Last read position: 4:78,806,713\n",
      "INFO\t2017-01-12 16:17:26\tGenotypeConcordance\tchecked     8,650,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:79,650,815\n",
      "INFO\t2017-01-12 16:17:26\tGenotypeConcordance\tchecked     8,660,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:80,643,067\n",
      "INFO\t2017-01-12 16:17:26\tGenotypeConcordance\tchecked     8,670,000 variants.  Elapsed time: 00:04:32s.  Time for last 10,000:    0s.  Last read position: 4:81,589,438\n",
      "INFO\t2017-01-12 16:17:26\tGenotypeConcordance\tchecked     8,680,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:82,495,893\n",
      "INFO\t2017-01-12 16:17:27\tGenotypeConcordance\tchecked     8,690,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:83,369,296\n",
      "INFO\t2017-01-12 16:17:27\tGenotypeConcordance\tchecked     8,700,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:84,240,409\n",
      "INFO\t2017-01-12 16:17:27\tGenotypeConcordance\tchecked     8,710,000 variants.  Elapsed time: 00:04:33s.  Time for last 10,000:    0s.  Last read position: 4:85,074,102\n",
      "INFO\t2017-01-12 16:17:28\tGenotypeConcordance\tchecked     8,720,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:86,099,883\n",
      "INFO\t2017-01-12 16:17:28\tGenotypeConcordance\tchecked     8,730,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:86,933,202\n",
      "INFO\t2017-01-12 16:17:28\tGenotypeConcordance\tchecked     8,740,000 variants.  Elapsed time: 00:04:34s.  Time for last 10,000:    0s.  Last read position: 4:87,940,940\n",
      "INFO\t2017-01-12 16:17:29\tGenotypeConcordance\tchecked     8,750,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:88,751,096\n",
      "INFO\t2017-01-12 16:17:29\tGenotypeConcordance\tchecked     8,760,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:89,665,891\n",
      "INFO\t2017-01-12 16:17:29\tGenotypeConcordance\tchecked     8,770,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:90,545,192\n",
      "INFO\t2017-01-12 16:17:29\tGenotypeConcordance\tchecked     8,780,000 variants.  Elapsed time: 00:04:35s.  Time for last 10,000:    0s.  Last read position: 4:91,403,851\n",
      "INFO\t2017-01-12 16:17:30\tGenotypeConcordance\tchecked     8,790,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:92,197,449\n",
      "INFO\t2017-01-12 16:17:30\tGenotypeConcordance\tchecked     8,800,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:93,013,683\n",
      "INFO\t2017-01-12 16:17:30\tGenotypeConcordance\tchecked     8,810,000 variants.  Elapsed time: 00:04:36s.  Time for last 10,000:    0s.  Last read position: 4:93,999,179\n",
      "INFO\t2017-01-12 16:17:31\tGenotypeConcordance\tchecked     8,820,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:94,932,155\n",
      "INFO\t2017-01-12 16:17:31\tGenotypeConcordance\tchecked     8,830,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:95,783,955\n",
      "INFO\t2017-01-12 16:17:31\tGenotypeConcordance\tchecked     8,840,000 variants.  Elapsed time: 00:04:37s.  Time for last 10,000:    0s.  Last read position: 4:96,588,172\n",
      "INFO\t2017-01-12 16:17:31\tGenotypeConcordance\tchecked     8,850,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:97,439,543\n",
      "INFO\t2017-01-12 16:17:32\tGenotypeConcordance\tchecked     8,860,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:98,277,522\n",
      "INFO\t2017-01-12 16:17:32\tGenotypeConcordance\tchecked     8,870,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:99,144,321\n",
      "INFO\t2017-01-12 16:17:32\tGenotypeConcordance\tchecked     8,880,000 variants.  Elapsed time: 00:04:38s.  Time for last 10,000:    0s.  Last read position: 4:100,042,242\n",
      "INFO\t2017-01-12 16:17:33\tGenotypeConcordance\tchecked     8,890,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:101,020,755\n",
      "INFO\t2017-01-12 16:17:33\tGenotypeConcordance\tchecked     8,900,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:101,871,546\n",
      "INFO\t2017-01-12 16:17:33\tGenotypeConcordance\tchecked     8,910,000 variants.  Elapsed time: 00:04:39s.  Time for last 10,000:    0s.  Last read position: 4:102,722,852\n",
      "INFO\t2017-01-12 16:17:34\tGenotypeConcordance\tchecked     8,920,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:103,633,275\n",
      "INFO\t2017-01-12 16:17:34\tGenotypeConcordance\tchecked     8,930,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:104,514,952\n",
      "INFO\t2017-01-12 16:17:34\tGenotypeConcordance\tchecked     8,940,000 variants.  Elapsed time: 00:04:40s.  Time for last 10,000:    0s.  Last read position: 4:105,414,718\n",
      "INFO\t2017-01-12 16:17:34\tGenotypeConcordance\tchecked     8,950,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:106,367,210\n",
      "INFO\t2017-01-12 16:17:35\tGenotypeConcordance\tchecked     8,960,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:107,350,635\n",
      "INFO\t2017-01-12 16:17:35\tGenotypeConcordance\tchecked     8,970,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:108,330,475\n",
      "INFO\t2017-01-12 16:17:35\tGenotypeConcordance\tchecked     8,980,000 variants.  Elapsed time: 00:04:41s.  Time for last 10,000:    0s.  Last read position: 4:109,301,111\n",
      "INFO\t2017-01-12 16:17:36\tGenotypeConcordance\tchecked     8,990,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:110,144,593\n",
      "INFO\t2017-01-12 16:17:36\tGenotypeConcordance\tchecked     9,000,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:111,005,409\n",
      "INFO\t2017-01-12 16:17:36\tGenotypeConcordance\tchecked     9,010,000 variants.  Elapsed time: 00:04:42s.  Time for last 10,000:    0s.  Last read position: 4:111,858,679\n",
      "INFO\t2017-01-12 16:17:37\tGenotypeConcordance\tchecked     9,020,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:112,800,228\n",
      "INFO\t2017-01-12 16:17:37\tGenotypeConcordance\tchecked     9,030,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:113,797,055\n",
      "INFO\t2017-01-12 16:17:37\tGenotypeConcordance\tchecked     9,040,000 variants.  Elapsed time: 00:04:43s.  Time for last 10,000:    0s.  Last read position: 4:114,704,051\n",
      "INFO\t2017-01-12 16:17:38\tGenotypeConcordance\tchecked     9,050,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 4:115,488,127\n",
      "INFO\t2017-01-12 16:17:38\tGenotypeConcordance\tchecked     9,060,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 4:116,329,991\n",
      "INFO\t2017-01-12 16:17:38\tGenotypeConcordance\tchecked     9,070,000 variants.  Elapsed time: 00:04:44s.  Time for last 10,000:    0s.  Last read position: 4:117,110,376\n",
      "INFO\t2017-01-12 16:17:39\tGenotypeConcordance\tchecked     9,080,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 4:117,894,668\n",
      "INFO\t2017-01-12 16:17:39\tGenotypeConcordance\tchecked     9,090,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 4:118,721,958\n",
      "INFO\t2017-01-12 16:17:39\tGenotypeConcordance\tchecked     9,100,000 variants.  Elapsed time: 00:04:45s.  Time for last 10,000:    0s.  Last read position: 4:119,668,186\n",
      "INFO\t2017-01-12 16:17:40\tGenotypeConcordance\tchecked     9,110,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 4:120,459,548\n",
      "INFO\t2017-01-12 16:17:40\tGenotypeConcordance\tchecked     9,120,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 4:121,345,846\n",
      "INFO\t2017-01-12 16:17:40\tGenotypeConcordance\tchecked     9,130,000 variants.  Elapsed time: 00:04:46s.  Time for last 10,000:    0s.  Last read position: 4:122,164,895\n",
      "INFO\t2017-01-12 16:17:40\tGenotypeConcordance\tchecked     9,140,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 4:123,034,210\n",
      "INFO\t2017-01-12 16:17:41\tGenotypeConcordance\tchecked     9,150,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 4:123,983,264\n",
      "INFO\t2017-01-12 16:17:41\tGenotypeConcordance\tchecked     9,160,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 4:124,821,664\n",
      "INFO\t2017-01-12 16:17:41\tGenotypeConcordance\tchecked     9,170,000 variants.  Elapsed time: 00:04:47s.  Time for last 10,000:    0s.  Last read position: 4:125,713,066\n",
      "INFO\t2017-01-12 16:17:42\tGenotypeConcordance\tchecked     9,180,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 4:126,556,132\n",
      "INFO\t2017-01-12 16:17:42\tGenotypeConcordance\tchecked     9,190,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 4:127,370,617\n",
      "INFO\t2017-01-12 16:17:42\tGenotypeConcordance\tchecked     9,200,000 variants.  Elapsed time: 00:04:48s.  Time for last 10,000:    0s.  Last read position: 4:128,314,339\n",
      "INFO\t2017-01-12 16:17:43\tGenotypeConcordance\tchecked     9,210,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 4:129,423,388\n",
      "INFO\t2017-01-12 16:17:43\tGenotypeConcordance\tchecked     9,220,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 4:130,270,267\n",
      "INFO\t2017-01-12 16:17:43\tGenotypeConcordance\tchecked     9,230,000 variants.  Elapsed time: 00:04:49s.  Time for last 10,000:    0s.  Last read position: 4:131,095,087\n",
      "INFO\t2017-01-12 16:17:43\tGenotypeConcordance\tchecked     9,240,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 4:131,872,320\n",
      "INFO\t2017-01-12 16:17:44\tGenotypeConcordance\tchecked     9,250,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 4:132,725,160\n",
      "INFO\t2017-01-12 16:17:44\tGenotypeConcordance\tchecked     9,260,000 variants.  Elapsed time: 00:04:50s.  Time for last 10,000:    0s.  Last read position: 4:133,576,536\n",
      "INFO\t2017-01-12 16:17:44\tGenotypeConcordance\tchecked     9,270,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 4:134,425,481\n",
      "INFO\t2017-01-12 16:17:45\tGenotypeConcordance\tchecked     9,280,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 4:135,351,753\n",
      "INFO\t2017-01-12 16:17:45\tGenotypeConcordance\tchecked     9,290,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 4:136,111,663\n",
      "INFO\t2017-01-12 16:17:45\tGenotypeConcordance\tchecked     9,300,000 variants.  Elapsed time: 00:04:51s.  Time for last 10,000:    0s.  Last read position: 4:136,937,648\n",
      "INFO\t2017-01-12 16:17:46\tGenotypeConcordance\tchecked     9,310,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 4:137,816,744\n",
      "INFO\t2017-01-12 16:17:46\tGenotypeConcordance\tchecked     9,320,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 4:138,608,230\n",
      "INFO\t2017-01-12 16:17:46\tGenotypeConcordance\tchecked     9,330,000 variants.  Elapsed time: 00:04:52s.  Time for last 10,000:    0s.  Last read position: 4:139,477,559\n",
      "INFO\t2017-01-12 16:17:47\tGenotypeConcordance\tchecked     9,340,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 4:140,354,073\n",
      "INFO\t2017-01-12 16:17:47\tGenotypeConcordance\tchecked     9,350,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 4:141,301,420\n",
      "INFO\t2017-01-12 16:17:47\tGenotypeConcordance\tchecked     9,360,000 variants.  Elapsed time: 00:04:53s.  Time for last 10,000:    0s.  Last read position: 4:142,199,705\n",
      "INFO\t2017-01-12 16:17:48\tGenotypeConcordance\tchecked     9,370,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 4:143,041,723\n",
      "INFO\t2017-01-12 16:17:48\tGenotypeConcordance\tchecked     9,380,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 4:143,950,762\n",
      "INFO\t2017-01-12 16:17:48\tGenotypeConcordance\tchecked     9,390,000 variants.  Elapsed time: 00:04:54s.  Time for last 10,000:    0s.  Last read position: 4:144,818,822\n",
      "INFO\t2017-01-12 16:17:49\tGenotypeConcordance\tchecked     9,400,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 4:145,562,738\n",
      "INFO\t2017-01-12 16:17:49\tGenotypeConcordance\tchecked     9,410,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 4:146,602,653\n",
      "INFO\t2017-01-12 16:17:49\tGenotypeConcordance\tchecked     9,420,000 variants.  Elapsed time: 00:04:55s.  Time for last 10,000:    0s.  Last read position: 4:147,514,783\n",
      "INFO\t2017-01-12 16:17:49\tGenotypeConcordance\tchecked     9,430,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 4:148,503,146\n",
      "INFO\t2017-01-12 16:17:50\tGenotypeConcordance\tchecked     9,440,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 4:149,458,917\n",
      "INFO\t2017-01-12 16:17:50\tGenotypeConcordance\tchecked     9,450,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 4:150,443,928\n",
      "INFO\t2017-01-12 16:17:50\tGenotypeConcordance\tchecked     9,460,000 variants.  Elapsed time: 00:04:56s.  Time for last 10,000:    0s.  Last read position: 4:151,399,246\n",
      "INFO\t2017-01-12 16:17:51\tGenotypeConcordance\tchecked     9,470,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 4:152,493,890\n",
      "INFO\t2017-01-12 16:17:51\tGenotypeConcordance\tchecked     9,480,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 4:153,527,808\n",
      "INFO\t2017-01-12 16:17:51\tGenotypeConcordance\tchecked     9,490,000 variants.  Elapsed time: 00:04:57s.  Time for last 10,000:    0s.  Last read position: 4:154,394,503\n",
      "INFO\t2017-01-12 16:17:52\tGenotypeConcordance\tchecked     9,500,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 4:155,262,227\n",
      "INFO\t2017-01-12 16:17:52\tGenotypeConcordance\tchecked     9,510,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 4:156,167,667\n",
      "INFO\t2017-01-12 16:17:52\tGenotypeConcordance\tchecked     9,520,000 variants.  Elapsed time: 00:04:58s.  Time for last 10,000:    0s.  Last read position: 4:157,000,262\n",
      "INFO\t2017-01-12 16:17:53\tGenotypeConcordance\tchecked     9,530,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 4:157,797,969\n",
      "INFO\t2017-01-12 16:17:53\tGenotypeConcordance\tchecked     9,540,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 4:158,668,868\n",
      "INFO\t2017-01-12 16:17:53\tGenotypeConcordance\tchecked     9,550,000 variants.  Elapsed time: 00:04:59s.  Time for last 10,000:    0s.  Last read position: 4:159,641,244\n",
      "INFO\t2017-01-12 16:17:54\tGenotypeConcordance\tchecked     9,560,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 4:160,534,399\n",
      "INFO\t2017-01-12 16:17:54\tGenotypeConcordance\tchecked     9,570,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 4:161,344,495\n",
      "INFO\t2017-01-12 16:17:54\tGenotypeConcordance\tchecked     9,580,000 variants.  Elapsed time: 00:05:00s.  Time for last 10,000:    0s.  Last read position: 4:162,066,509\n",
      "INFO\t2017-01-12 16:17:55\tGenotypeConcordance\tchecked     9,590,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 4:162,836,241\n",
      "INFO\t2017-01-12 16:17:55\tGenotypeConcordance\tchecked     9,600,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 4:163,537,134\n",
      "INFO\t2017-01-12 16:17:55\tGenotypeConcordance\tchecked     9,610,000 variants.  Elapsed time: 00:05:01s.  Time for last 10,000:    0s.  Last read position: 4:164,384,855\n",
      "INFO\t2017-01-12 16:17:55\tGenotypeConcordance\tchecked     9,620,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 4:165,170,243\n",
      "INFO\t2017-01-12 16:17:56\tGenotypeConcordance\tchecked     9,630,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 4:165,900,979\n",
      "INFO\t2017-01-12 16:17:56\tGenotypeConcordance\tchecked     9,640,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 4:166,681,527\n",
      "INFO\t2017-01-12 16:17:56\tGenotypeConcordance\tchecked     9,650,000 variants.  Elapsed time: 00:05:02s.  Time for last 10,000:    0s.  Last read position: 4:167,434,458\n",
      "INFO\t2017-01-12 16:17:57\tGenotypeConcordance\tchecked     9,660,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 4:168,273,946\n",
      "INFO\t2017-01-12 16:17:57\tGenotypeConcordance\tchecked     9,670,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 4:169,121,588\n",
      "INFO\t2017-01-12 16:17:57\tGenotypeConcordance\tchecked     9,680,000 variants.  Elapsed time: 00:05:03s.  Time for last 10,000:    0s.  Last read position: 4:169,996,302\n",
      "INFO\t2017-01-12 16:17:58\tGenotypeConcordance\tchecked     9,690,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 4:170,914,622\n",
      "INFO\t2017-01-12 16:17:58\tGenotypeConcordance\tchecked     9,700,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 4:171,679,226\n",
      "INFO\t2017-01-12 16:17:58\tGenotypeConcordance\tchecked     9,710,000 variants.  Elapsed time: 00:05:04s.  Time for last 10,000:    0s.  Last read position: 4:172,489,942\n",
      "INFO\t2017-01-12 16:17:59\tGenotypeConcordance\tchecked     9,720,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 4:173,298,069\n",
      "INFO\t2017-01-12 16:17:59\tGenotypeConcordance\tchecked     9,730,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 4:174,216,526\n",
      "INFO\t2017-01-12 16:17:59\tGenotypeConcordance\tchecked     9,740,000 variants.  Elapsed time: 00:05:05s.  Time for last 10,000:    0s.  Last read position: 4:175,061,519\n",
      "INFO\t2017-01-12 16:18:00\tGenotypeConcordance\tchecked     9,750,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 4:175,887,670\n",
      "INFO\t2017-01-12 16:18:00\tGenotypeConcordance\tchecked     9,760,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 4:176,732,480\n",
      "INFO\t2017-01-12 16:18:00\tGenotypeConcordance\tchecked     9,770,000 variants.  Elapsed time: 00:05:06s.  Time for last 10,000:    0s.  Last read position: 4:177,532,499\n",
      "INFO\t2017-01-12 16:18:01\tGenotypeConcordance\tchecked     9,780,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 4:178,354,599\n",
      "INFO\t2017-01-12 16:18:01\tGenotypeConcordance\tchecked     9,790,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 4:179,136,091\n",
      "INFO\t2017-01-12 16:18:01\tGenotypeConcordance\tchecked     9,800,000 variants.  Elapsed time: 00:05:07s.  Time for last 10,000:    0s.  Last read position: 4:179,816,709\n",
      "INFO\t2017-01-12 16:18:01\tGenotypeConcordance\tchecked     9,810,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 4:180,507,487\n",
      "INFO\t2017-01-12 16:18:02\tGenotypeConcordance\tchecked     9,820,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 4:181,226,698\n",
      "INFO\t2017-01-12 16:18:02\tGenotypeConcordance\tchecked     9,830,000 variants.  Elapsed time: 00:05:08s.  Time for last 10,000:    0s.  Last read position: 4:182,002,228\n",
      "INFO\t2017-01-12 16:18:02\tGenotypeConcordance\tchecked     9,840,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 4:182,680,609\n",
      "INFO\t2017-01-12 16:18:03\tGenotypeConcordance\tchecked     9,850,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 4:183,458,715\n",
      "INFO\t2017-01-12 16:18:03\tGenotypeConcordance\tchecked     9,860,000 variants.  Elapsed time: 00:05:09s.  Time for last 10,000:    0s.  Last read position: 4:184,257,215\n",
      "INFO\t2017-01-12 16:18:03\tGenotypeConcordance\tchecked     9,870,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 4:184,913,872\n",
      "INFO\t2017-01-12 16:18:04\tGenotypeConcordance\tchecked     9,880,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 4:185,656,390\n",
      "INFO\t2017-01-12 16:18:04\tGenotypeConcordance\tchecked     9,890,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 4:186,418,492\n",
      "INFO\t2017-01-12 16:18:04\tGenotypeConcordance\tchecked     9,900,000 variants.  Elapsed time: 00:05:10s.  Time for last 10,000:    0s.  Last read position: 4:187,118,832\n",
      "INFO\t2017-01-12 16:18:05\tGenotypeConcordance\tchecked     9,910,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 4:187,835,603\n",
      "INFO\t2017-01-12 16:18:05\tGenotypeConcordance\tchecked     9,920,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 4:188,626,838\n",
      "INFO\t2017-01-12 16:18:05\tGenotypeConcordance\tchecked     9,930,000 variants.  Elapsed time: 00:05:11s.  Time for last 10,000:    0s.  Last read position: 4:189,251,223\n",
      "INFO\t2017-01-12 16:18:06\tGenotypeConcordance\tchecked     9,940,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 4:189,869,211\n",
      "INFO\t2017-01-12 16:18:06\tGenotypeConcordance\tchecked     9,950,000 variants.  Elapsed time: 00:05:12s.  Time for last 10,000:    0s.  Last read position: 4:190,520,824\n",
      "INFO\t2017-01-12 16:18:06\tGenotypeConcordance\tchecked     9,960,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 4:190,852,643\n",
      "INFO\t2017-01-12 16:18:07\tGenotypeConcordance\tchecked     9,970,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:442,904\n",
      "INFO\t2017-01-12 16:18:07\tGenotypeConcordance\tchecked     9,980,000 variants.  Elapsed time: 00:05:13s.  Time for last 10,000:    0s.  Last read position: 5:1,044,904\n",
      "INFO\t2017-01-12 16:18:07\tGenotypeConcordance\tchecked     9,990,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:1,634,161\n",
      "INFO\t2017-01-12 16:18:08\tGenotypeConcordance\tchecked    10,000,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:2,262,683\n",
      "INFO\t2017-01-12 16:18:08\tGenotypeConcordance\tchecked    10,010,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:2,886,975\n",
      "INFO\t2017-01-12 16:18:08\tGenotypeConcordance\tchecked    10,020,000 variants.  Elapsed time: 00:05:14s.  Time for last 10,000:    0s.  Last read position: 5:3,529,660\n",
      "INFO\t2017-01-12 16:18:09\tGenotypeConcordance\tchecked    10,030,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:4,250,000\n",
      "INFO\t2017-01-12 16:18:09\tGenotypeConcordance\tchecked    10,040,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:5,018,551\n",
      "INFO\t2017-01-12 16:18:09\tGenotypeConcordance\tchecked    10,050,000 variants.  Elapsed time: 00:05:15s.  Time for last 10,000:    0s.  Last read position: 5:5,742,391\n",
      "INFO\t2017-01-12 16:18:10\tGenotypeConcordance\tchecked    10,060,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:6,491,204\n",
      "INFO\t2017-01-12 16:18:10\tGenotypeConcordance\tchecked    10,070,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:7,260,750\n",
      "INFO\t2017-01-12 16:18:10\tGenotypeConcordance\tchecked    10,080,000 variants.  Elapsed time: 00:05:16s.  Time for last 10,000:    0s.  Last read position: 5:8,036,175\n",
      "INFO\t2017-01-12 16:18:11\tGenotypeConcordance\tchecked    10,090,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:8,744,233\n",
      "INFO\t2017-01-12 16:18:11\tGenotypeConcordance\tchecked    10,100,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:9,551,780\n",
      "INFO\t2017-01-12 16:18:11\tGenotypeConcordance\tchecked    10,110,000 variants.  Elapsed time: 00:05:17s.  Time for last 10,000:    0s.  Last read position: 5:10,332,965\n",
      "INFO\t2017-01-12 16:18:12\tGenotypeConcordance\tchecked    10,120,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 5:11,152,355\n",
      "INFO\t2017-01-12 16:18:12\tGenotypeConcordance\tchecked    10,130,000 variants.  Elapsed time: 00:05:18s.  Time for last 10,000:    0s.  Last read position: 5:11,996,952\n",
      "INFO\t2017-01-12 16:18:12\tGenotypeConcordance\tchecked    10,140,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:12,884,007\n",
      "INFO\t2017-01-12 16:18:13\tGenotypeConcordance\tchecked    10,150,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:13,652,373\n",
      "INFO\t2017-01-12 16:18:13\tGenotypeConcordance\tchecked    10,160,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:14,483,612\n",
      "INFO\t2017-01-12 16:18:13\tGenotypeConcordance\tchecked    10,170,000 variants.  Elapsed time: 00:05:19s.  Time for last 10,000:    0s.  Last read position: 5:15,332,828\n",
      "INFO\t2017-01-12 16:18:14\tGenotypeConcordance\tchecked    10,180,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 5:16,297,063\n",
      "INFO\t2017-01-12 16:18:14\tGenotypeConcordance\tchecked    10,190,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 5:17,033,036\n",
      "INFO\t2017-01-12 16:18:14\tGenotypeConcordance\tchecked    10,200,000 variants.  Elapsed time: 00:05:20s.  Time for last 10,000:    0s.  Last read position: 5:17,886,282\n",
      "INFO\t2017-01-12 16:18:15\tGenotypeConcordance\tchecked    10,210,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:18,640,846\n",
      "INFO\t2017-01-12 16:18:15\tGenotypeConcordance\tchecked    10,220,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:19,470,753\n",
      "INFO\t2017-01-12 16:18:15\tGenotypeConcordance\tchecked    10,230,000 variants.  Elapsed time: 00:05:21s.  Time for last 10,000:    0s.  Last read position: 5:20,340,926\n",
      "INFO\t2017-01-12 16:18:15\tGenotypeConcordance\tchecked    10,240,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:21,188,773\n",
      "INFO\t2017-01-12 16:18:16\tGenotypeConcordance\tchecked    10,250,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:21,934,011\n",
      "INFO\t2017-01-12 16:18:16\tGenotypeConcordance\tchecked    10,260,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:22,796,407\n",
      "INFO\t2017-01-12 16:18:16\tGenotypeConcordance\tchecked    10,270,000 variants.  Elapsed time: 00:05:22s.  Time for last 10,000:    0s.  Last read position: 5:23,614,535\n",
      "INFO\t2017-01-12 16:18:17\tGenotypeConcordance\tchecked    10,280,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:24,402,069\n",
      "INFO\t2017-01-12 16:18:17\tGenotypeConcordance\tchecked    10,290,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:25,260,064\n",
      "INFO\t2017-01-12 16:18:17\tGenotypeConcordance\tchecked    10,300,000 variants.  Elapsed time: 00:05:23s.  Time for last 10,000:    0s.  Last read position: 5:26,005,870\n",
      "INFO\t2017-01-12 16:18:18\tGenotypeConcordance\tchecked    10,310,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:26,737,371\n",
      "INFO\t2017-01-12 16:18:18\tGenotypeConcordance\tchecked    10,320,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:27,561,458\n",
      "INFO\t2017-01-12 16:18:18\tGenotypeConcordance\tchecked    10,330,000 variants.  Elapsed time: 00:05:24s.  Time for last 10,000:    0s.  Last read position: 5:28,376,064\n",
      "INFO\t2017-01-12 16:18:19\tGenotypeConcordance\tchecked    10,340,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:29,221,051\n",
      "INFO\t2017-01-12 16:18:19\tGenotypeConcordance\tchecked    10,350,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:29,857,854\n",
      "INFO\t2017-01-12 16:18:19\tGenotypeConcordance\tchecked    10,360,000 variants.  Elapsed time: 00:05:25s.  Time for last 10,000:    0s.  Last read position: 5:30,635,630\n",
      "INFO\t2017-01-12 16:18:20\tGenotypeConcordance\tchecked    10,370,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:31,486,791\n",
      "INFO\t2017-01-12 16:18:20\tGenotypeConcordance\tchecked    10,380,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:32,247,129\n",
      "INFO\t2017-01-12 16:18:20\tGenotypeConcordance\tchecked    10,390,000 variants.  Elapsed time: 00:05:26s.  Time for last 10,000:    0s.  Last read position: 5:33,084,945\n",
      "INFO\t2017-01-12 16:18:20\tGenotypeConcordance\tchecked    10,400,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:33,869,728\n",
      "INFO\t2017-01-12 16:18:21\tGenotypeConcordance\tchecked    10,410,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:34,702,457\n",
      "INFO\t2017-01-12 16:18:21\tGenotypeConcordance\tchecked    10,420,000 variants.  Elapsed time: 00:05:27s.  Time for last 10,000:    0s.  Last read position: 5:35,540,135\n",
      "INFO\t2017-01-12 16:18:21\tGenotypeConcordance\tchecked    10,430,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:36,414,401\n",
      "INFO\t2017-01-12 16:18:22\tGenotypeConcordance\tchecked    10,440,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:37,442,135\n",
      "INFO\t2017-01-12 16:18:22\tGenotypeConcordance\tchecked    10,450,000 variants.  Elapsed time: 00:05:28s.  Time for last 10,000:    0s.  Last read position: 5:38,297,409\n",
      "INFO\t2017-01-12 16:18:22\tGenotypeConcordance\tchecked    10,460,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:39,155,400\n",
      "INFO\t2017-01-12 16:18:23\tGenotypeConcordance\tchecked    10,470,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:39,942,034\n",
      "INFO\t2017-01-12 16:18:23\tGenotypeConcordance\tchecked    10,480,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:40,734,183\n",
      "INFO\t2017-01-12 16:18:23\tGenotypeConcordance\tchecked    10,490,000 variants.  Elapsed time: 00:05:29s.  Time for last 10,000:    0s.  Last read position: 5:41,527,256\n",
      "INFO\t2017-01-12 16:18:24\tGenotypeConcordance\tchecked    10,500,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:42,356,099\n",
      "INFO\t2017-01-12 16:18:24\tGenotypeConcordance\tchecked    10,510,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:43,198,241\n",
      "INFO\t2017-01-12 16:18:24\tGenotypeConcordance\tchecked    10,520,000 variants.  Elapsed time: 00:05:30s.  Time for last 10,000:    0s.  Last read position: 5:44,041,626\n",
      "INFO\t2017-01-12 16:18:25\tGenotypeConcordance\tchecked    10,530,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:44,986,828\n",
      "INFO\t2017-01-12 16:18:25\tGenotypeConcordance\tchecked    10,540,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:45,891,794\n",
      "INFO\t2017-01-12 16:18:25\tGenotypeConcordance\tchecked    10,550,000 variants.  Elapsed time: 00:05:31s.  Time for last 10,000:    0s.  Last read position: 5:46,363,228\n",
      "INFO\t2017-01-12 16:18:26\tGenotypeConcordance\tchecked    10,560,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:49,878,993\n",
      "INFO\t2017-01-12 16:18:26\tGenotypeConcordance\tchecked    10,570,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:50,776,123\n",
      "INFO\t2017-01-12 16:18:26\tGenotypeConcordance\tchecked    10,580,000 variants.  Elapsed time: 00:05:32s.  Time for last 10,000:    0s.  Last read position: 5:51,592,067\n",
      "INFO\t2017-01-12 16:18:26\tGenotypeConcordance\tchecked    10,590,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:52,430,327\n",
      "INFO\t2017-01-12 16:18:27\tGenotypeConcordance\tchecked    10,600,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:53,268,199\n",
      "INFO\t2017-01-12 16:18:27\tGenotypeConcordance\tchecked    10,610,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:54,083,713\n",
      "INFO\t2017-01-12 16:18:27\tGenotypeConcordance\tchecked    10,620,000 variants.  Elapsed time: 00:05:33s.  Time for last 10,000:    0s.  Last read position: 5:54,969,713\n",
      "INFO\t2017-01-12 16:18:28\tGenotypeConcordance\tchecked    10,630,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:55,801,586\n",
      "INFO\t2017-01-12 16:18:28\tGenotypeConcordance\tchecked    10,640,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:56,717,667\n",
      "INFO\t2017-01-12 16:18:28\tGenotypeConcordance\tchecked    10,650,000 variants.  Elapsed time: 00:05:34s.  Time for last 10,000:    0s.  Last read position: 5:57,576,673\n",
      "INFO\t2017-01-12 16:18:29\tGenotypeConcordance\tchecked    10,660,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:58,424,097\n",
      "INFO\t2017-01-12 16:18:29\tGenotypeConcordance\tchecked    10,670,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:59,399,042\n",
      "INFO\t2017-01-12 16:18:29\tGenotypeConcordance\tchecked    10,680,000 variants.  Elapsed time: 00:05:35s.  Time for last 10,000:    0s.  Last read position: 5:60,467,949\n",
      "INFO\t2017-01-12 16:18:30\tGenotypeConcordance\tchecked    10,690,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:61,486,338\n",
      "INFO\t2017-01-12 16:18:30\tGenotypeConcordance\tchecked    10,700,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:62,380,641\n",
      "INFO\t2017-01-12 16:18:30\tGenotypeConcordance\tchecked    10,710,000 variants.  Elapsed time: 00:05:36s.  Time for last 10,000:    0s.  Last read position: 5:63,252,049\n",
      "INFO\t2017-01-12 16:18:31\tGenotypeConcordance\tchecked    10,720,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:64,263,903\n",
      "INFO\t2017-01-12 16:18:31\tGenotypeConcordance\tchecked    10,730,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:65,223,029\n",
      "INFO\t2017-01-12 16:18:31\tGenotypeConcordance\tchecked    10,740,000 variants.  Elapsed time: 00:05:37s.  Time for last 10,000:    0s.  Last read position: 5:66,081,464\n",
      "INFO\t2017-01-12 16:18:31\tGenotypeConcordance\tchecked    10,750,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:66,990,048\n",
      "INFO\t2017-01-12 16:18:32\tGenotypeConcordance\tchecked    10,760,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:67,866,560\n",
      "INFO\t2017-01-12 16:18:32\tGenotypeConcordance\tchecked    10,770,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:68,829,079\n",
      "INFO\t2017-01-12 16:18:32\tGenotypeConcordance\tchecked    10,780,000 variants.  Elapsed time: 00:05:38s.  Time for last 10,000:    0s.  Last read position: 5:71,324,354\n",
      "INFO\t2017-01-12 16:18:33\tGenotypeConcordance\tchecked    10,790,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:72,356,452\n",
      "INFO\t2017-01-12 16:18:33\tGenotypeConcordance\tchecked    10,800,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:73,255,812\n",
      "INFO\t2017-01-12 16:18:33\tGenotypeConcordance\tchecked    10,810,000 variants.  Elapsed time: 00:05:39s.  Time for last 10,000:    0s.  Last read position: 5:74,074,589\n",
      "INFO\t2017-01-12 16:18:34\tGenotypeConcordance\tchecked    10,820,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:75,057,430\n",
      "INFO\t2017-01-12 16:18:34\tGenotypeConcordance\tchecked    10,830,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:75,792,550\n",
      "INFO\t2017-01-12 16:18:34\tGenotypeConcordance\tchecked    10,840,000 variants.  Elapsed time: 00:05:40s.  Time for last 10,000:    0s.  Last read position: 5:76,665,184\n",
      "INFO\t2017-01-12 16:18:35\tGenotypeConcordance\tchecked    10,850,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:77,570,082\n",
      "INFO\t2017-01-12 16:18:35\tGenotypeConcordance\tchecked    10,860,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:78,435,578\n",
      "INFO\t2017-01-12 16:18:35\tGenotypeConcordance\tchecked    10,870,000 variants.  Elapsed time: 00:05:41s.  Time for last 10,000:    0s.  Last read position: 5:79,384,952\n",
      "INFO\t2017-01-12 16:18:35\tGenotypeConcordance\tchecked    10,880,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:80,183,772\n",
      "INFO\t2017-01-12 16:18:36\tGenotypeConcordance\tchecked    10,890,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:81,119,661\n",
      "INFO\t2017-01-12 16:18:36\tGenotypeConcordance\tchecked    10,900,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:82,100,752\n",
      "INFO\t2017-01-12 16:18:36\tGenotypeConcordance\tchecked    10,910,000 variants.  Elapsed time: 00:05:42s.  Time for last 10,000:    0s.  Last read position: 5:82,988,485\n",
      "INFO\t2017-01-12 16:18:37\tGenotypeConcordance\tchecked    10,920,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:83,977,669\n",
      "INFO\t2017-01-12 16:18:37\tGenotypeConcordance\tchecked    10,930,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:84,755,439\n",
      "INFO\t2017-01-12 16:18:37\tGenotypeConcordance\tchecked    10,940,000 variants.  Elapsed time: 00:05:43s.  Time for last 10,000:    0s.  Last read position: 5:85,562,014\n",
      "INFO\t2017-01-12 16:18:38\tGenotypeConcordance\tchecked    10,950,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 5:86,454,768\n",
      "INFO\t2017-01-12 16:18:38\tGenotypeConcordance\tchecked    10,960,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 5:87,609,719\n",
      "INFO\t2017-01-12 16:18:38\tGenotypeConcordance\tchecked    10,970,000 variants.  Elapsed time: 00:05:44s.  Time for last 10,000:    0s.  Last read position: 5:88,875,426\n",
      "INFO\t2017-01-12 16:18:38\tGenotypeConcordance\tchecked    10,980,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 5:89,930,666\n",
      "INFO\t2017-01-12 16:18:39\tGenotypeConcordance\tchecked    10,990,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 5:90,893,636\n",
      "INFO\t2017-01-12 16:18:39\tGenotypeConcordance\tchecked    11,000,000 variants.  Elapsed time: 00:05:45s.  Time for last 10,000:    0s.  Last read position: 5:92,005,876\n",
      "INFO\t2017-01-12 16:18:39\tGenotypeConcordance\tchecked    11,010,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 5:93,183,020\n",
      "INFO\t2017-01-12 16:18:40\tGenotypeConcordance\tchecked    11,020,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 5:94,366,450\n",
      "INFO\t2017-01-12 16:18:40\tGenotypeConcordance\tchecked    11,030,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 5:95,348,201\n",
      "INFO\t2017-01-12 16:18:40\tGenotypeConcordance\tchecked    11,040,000 variants.  Elapsed time: 00:05:46s.  Time for last 10,000:    0s.  Last read position: 5:96,243,650\n",
      "INFO\t2017-01-12 16:18:41\tGenotypeConcordance\tchecked    11,050,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 5:97,060,608\n",
      "INFO\t2017-01-12 16:18:41\tGenotypeConcordance\tchecked    11,060,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 5:97,797,639\n",
      "INFO\t2017-01-12 16:18:41\tGenotypeConcordance\tchecked    11,070,000 variants.  Elapsed time: 00:05:47s.  Time for last 10,000:    0s.  Last read position: 5:98,679,017\n",
      "INFO\t2017-01-12 16:18:42\tGenotypeConcordance\tchecked    11,080,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 5:99,450,876\n",
      "INFO\t2017-01-12 16:18:42\tGenotypeConcordance\tchecked    11,090,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 5:100,271,134\n",
      "INFO\t2017-01-12 16:18:42\tGenotypeConcordance\tchecked    11,100,000 variants.  Elapsed time: 00:05:48s.  Time for last 10,000:    0s.  Last read position: 5:101,117,693\n",
      "INFO\t2017-01-12 16:18:42\tGenotypeConcordance\tchecked    11,110,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 5:101,923,665\n",
      "INFO\t2017-01-12 16:18:43\tGenotypeConcordance\tchecked    11,120,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 5:102,855,869\n",
      "INFO\t2017-01-12 16:18:43\tGenotypeConcordance\tchecked    11,130,000 variants.  Elapsed time: 00:05:49s.  Time for last 10,000:    0s.  Last read position: 5:103,673,009\n",
      "INFO\t2017-01-12 16:18:43\tGenotypeConcordance\tchecked    11,140,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 5:104,490,778\n",
      "INFO\t2017-01-12 16:18:44\tGenotypeConcordance\tchecked    11,150,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 5:105,387,654\n",
      "INFO\t2017-01-12 16:18:44\tGenotypeConcordance\tchecked    11,160,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 5:106,232,456\n",
      "INFO\t2017-01-12 16:18:44\tGenotypeConcordance\tchecked    11,170,000 variants.  Elapsed time: 00:05:50s.  Time for last 10,000:    0s.  Last read position: 5:107,063,746\n",
      "INFO\t2017-01-12 16:18:45\tGenotypeConcordance\tchecked    11,180,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 5:107,893,632\n",
      "INFO\t2017-01-12 16:18:45\tGenotypeConcordance\tchecked    11,190,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 5:108,656,067\n",
      "INFO\t2017-01-12 16:18:45\tGenotypeConcordance\tchecked    11,200,000 variants.  Elapsed time: 00:05:51s.  Time for last 10,000:    0s.  Last read position: 5:109,449,361\n",
      "INFO\t2017-01-12 16:18:46\tGenotypeConcordance\tchecked    11,210,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 5:110,188,988\n",
      "INFO\t2017-01-12 16:18:46\tGenotypeConcordance\tchecked    11,220,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 5:110,982,398\n",
      "INFO\t2017-01-12 16:18:46\tGenotypeConcordance\tchecked    11,230,000 variants.  Elapsed time: 00:05:52s.  Time for last 10,000:    0s.  Last read position: 5:111,799,267\n",
      "INFO\t2017-01-12 16:18:47\tGenotypeConcordance\tchecked    11,240,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 5:112,500,404\n",
      "INFO\t2017-01-12 16:18:47\tGenotypeConcordance\tchecked    11,250,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 5:113,282,113\n",
      "INFO\t2017-01-12 16:18:47\tGenotypeConcordance\tchecked    11,260,000 variants.  Elapsed time: 00:05:53s.  Time for last 10,000:    0s.  Last read position: 5:114,043,417\n",
      "INFO\t2017-01-12 16:18:47\tGenotypeConcordance\tchecked    11,270,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 5:114,747,126\n",
      "INFO\t2017-01-12 16:18:48\tGenotypeConcordance\tchecked    11,280,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 5:115,475,502\n",
      "INFO\t2017-01-12 16:18:48\tGenotypeConcordance\tchecked    11,290,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 5:116,128,151\n",
      "INFO\t2017-01-12 16:18:48\tGenotypeConcordance\tchecked    11,300,000 variants.  Elapsed time: 00:05:54s.  Time for last 10,000:    0s.  Last read position: 5:116,860,242\n",
      "INFO\t2017-01-12 16:18:49\tGenotypeConcordance\tchecked    11,310,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 5:117,586,651\n",
      "INFO\t2017-01-12 16:18:49\tGenotypeConcordance\tchecked    11,320,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 5:118,268,018\n",
      "INFO\t2017-01-12 16:18:49\tGenotypeConcordance\tchecked    11,330,000 variants.  Elapsed time: 00:05:55s.  Time for last 10,000:    0s.  Last read position: 5:119,055,044\n",
      "INFO\t2017-01-12 16:18:50\tGenotypeConcordance\tchecked    11,340,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 5:119,804,529\n",
      "INFO\t2017-01-12 16:18:50\tGenotypeConcordance\tchecked    11,350,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 5:120,537,042\n",
      "INFO\t2017-01-12 16:18:50\tGenotypeConcordance\tchecked    11,360,000 variants.  Elapsed time: 00:05:56s.  Time for last 10,000:    0s.  Last read position: 5:121,238,944\n",
      "INFO\t2017-01-12 16:18:50\tGenotypeConcordance\tchecked    11,370,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 5:122,087,919\n",
      "INFO\t2017-01-12 16:18:51\tGenotypeConcordance\tchecked    11,380,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 5:122,999,952\n",
      "INFO\t2017-01-12 16:18:51\tGenotypeConcordance\tchecked    11,390,000 variants.  Elapsed time: 00:05:57s.  Time for last 10,000:    0s.  Last read position: 5:123,816,401\n",
      "INFO\t2017-01-12 16:18:51\tGenotypeConcordance\tchecked    11,400,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 5:124,590,123\n",
      "INFO\t2017-01-12 16:18:52\tGenotypeConcordance\tchecked    11,410,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 5:125,350,827\n",
      "INFO\t2017-01-12 16:18:52\tGenotypeConcordance\tchecked    11,420,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 5:126,106,785\n",
      "INFO\t2017-01-12 16:18:52\tGenotypeConcordance\tchecked    11,430,000 variants.  Elapsed time: 00:05:58s.  Time for last 10,000:    0s.  Last read position: 5:126,973,564\n",
      "INFO\t2017-01-12 16:18:53\tGenotypeConcordance\tchecked    11,440,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 5:127,962,811\n",
      "INFO\t2017-01-12 16:18:53\tGenotypeConcordance\tchecked    11,450,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 5:128,755,827\n",
      "INFO\t2017-01-12 16:18:53\tGenotypeConcordance\tchecked    11,460,000 variants.  Elapsed time: 00:05:59s.  Time for last 10,000:    0s.  Last read position: 5:129,618,090\n",
      "INFO\t2017-01-12 16:18:54\tGenotypeConcordance\tchecked    11,470,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 5:130,538,004\n",
      "INFO\t2017-01-12 16:18:54\tGenotypeConcordance\tchecked    11,480,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 5:131,488,880\n",
      "INFO\t2017-01-12 16:18:54\tGenotypeConcordance\tchecked    11,490,000 variants.  Elapsed time: 00:06:00s.  Time for last 10,000:    0s.  Last read position: 5:132,444,721\n",
      "INFO\t2017-01-12 16:18:54\tGenotypeConcordance\tchecked    11,500,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 5:133,310,944\n",
      "INFO\t2017-01-12 16:18:55\tGenotypeConcordance\tchecked    11,510,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 5:134,343,130\n",
      "INFO\t2017-01-12 16:18:55\tGenotypeConcordance\tchecked    11,520,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 5:135,213,719\n",
      "INFO\t2017-01-12 16:18:55\tGenotypeConcordance\tchecked    11,530,000 variants.  Elapsed time: 00:06:01s.  Time for last 10,000:    0s.  Last read position: 5:136,146,592\n",
      "INFO\t2017-01-12 16:18:56\tGenotypeConcordance\tchecked    11,540,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 5:137,147,168\n",
      "INFO\t2017-01-12 16:18:56\tGenotypeConcordance\tchecked    11,550,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 5:138,262,670\n",
      "INFO\t2017-01-12 16:18:56\tGenotypeConcordance\tchecked    11,560,000 variants.  Elapsed time: 00:06:02s.  Time for last 10,000:    0s.  Last read position: 5:139,523,692\n",
      "INFO\t2017-01-12 16:18:56\tGenotypeConcordance\tchecked    11,570,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 5:140,578,250\n",
      "INFO\t2017-01-12 16:18:57\tGenotypeConcordance\tchecked    11,580,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 5:141,506,555\n",
      "INFO\t2017-01-12 16:18:57\tGenotypeConcordance\tchecked    11,590,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 5:142,349,030\n",
      "INFO\t2017-01-12 16:18:57\tGenotypeConcordance\tchecked    11,600,000 variants.  Elapsed time: 00:06:03s.  Time for last 10,000:    0s.  Last read position: 5:143,268,544\n",
      "INFO\t2017-01-12 16:18:58\tGenotypeConcordance\tchecked    11,610,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 5:144,098,567\n",
      "INFO\t2017-01-12 16:18:58\tGenotypeConcordance\tchecked    11,620,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 5:144,968,443\n",
      "INFO\t2017-01-12 16:18:58\tGenotypeConcordance\tchecked    11,630,000 variants.  Elapsed time: 00:06:04s.  Time for last 10,000:    0s.  Last read position: 5:145,792,218\n",
      "INFO\t2017-01-12 16:18:59\tGenotypeConcordance\tchecked    11,640,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 5:146,672,934\n",
      "INFO\t2017-01-12 16:18:59\tGenotypeConcordance\tchecked    11,650,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 5:147,516,195\n",
      "INFO\t2017-01-12 16:18:59\tGenotypeConcordance\tchecked    11,660,000 variants.  Elapsed time: 00:06:05s.  Time for last 10,000:    0s.  Last read position: 5:148,382,327\n",
      "INFO\t2017-01-12 16:18:59\tGenotypeConcordance\tchecked    11,670,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 5:149,254,197\n",
      "INFO\t2017-01-12 16:19:00\tGenotypeConcordance\tchecked    11,680,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 5:150,126,266\n",
      "INFO\t2017-01-12 16:19:00\tGenotypeConcordance\tchecked    11,690,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 5:150,904,341\n",
      "INFO\t2017-01-12 16:19:00\tGenotypeConcordance\tchecked    11,700,000 variants.  Elapsed time: 00:06:06s.  Time for last 10,000:    0s.  Last read position: 5:151,776,870\n",
      "INFO\t2017-01-12 16:19:01\tGenotypeConcordance\tchecked    11,710,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 5:152,662,803\n",
      "INFO\t2017-01-12 16:19:01\tGenotypeConcordance\tchecked    11,720,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 5:153,584,135\n",
      "INFO\t2017-01-12 16:19:01\tGenotypeConcordance\tchecked    11,730,000 variants.  Elapsed time: 00:06:07s.  Time for last 10,000:    0s.  Last read position: 5:154,491,541\n",
      "INFO\t2017-01-12 16:19:02\tGenotypeConcordance\tchecked    11,740,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 5:155,395,740\n",
      "INFO\t2017-01-12 16:19:02\tGenotypeConcordance\tchecked    11,750,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 5:156,258,787\n",
      "INFO\t2017-01-12 16:19:02\tGenotypeConcordance\tchecked    11,760,000 variants.  Elapsed time: 00:06:08s.  Time for last 10,000:    0s.  Last read position: 5:157,082,025\n",
      "INFO\t2017-01-12 16:19:03\tGenotypeConcordance\tchecked    11,770,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 5:157,906,224\n",
      "INFO\t2017-01-12 16:19:03\tGenotypeConcordance\tchecked    11,780,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 5:158,930,967\n",
      "INFO\t2017-01-12 16:19:03\tGenotypeConcordance\tchecked    11,790,000 variants.  Elapsed time: 00:06:09s.  Time for last 10,000:    0s.  Last read position: 5:159,764,701\n",
      "INFO\t2017-01-12 16:19:04\tGenotypeConcordance\tchecked    11,800,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 5:160,558,193\n",
      "INFO\t2017-01-12 16:19:04\tGenotypeConcordance\tchecked    11,810,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 5:161,447,890\n",
      "INFO\t2017-01-12 16:19:04\tGenotypeConcordance\tchecked    11,820,000 variants.  Elapsed time: 00:06:10s.  Time for last 10,000:    0s.  Last read position: 5:162,290,454\n",
      "INFO\t2017-01-12 16:19:04\tGenotypeConcordance\tchecked    11,830,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 5:163,154,418\n",
      "INFO\t2017-01-12 16:19:05\tGenotypeConcordance\tchecked    11,840,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 5:164,060,480\n",
      "INFO\t2017-01-12 16:19:05\tGenotypeConcordance\tchecked    11,850,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 5:164,962,496\n",
      "INFO\t2017-01-12 16:19:05\tGenotypeConcordance\tchecked    11,860,000 variants.  Elapsed time: 00:06:11s.  Time for last 10,000:    0s.  Last read position: 5:165,828,151\n",
      "INFO\t2017-01-12 16:19:06\tGenotypeConcordance\tchecked    11,870,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 5:166,698,017\n",
      "INFO\t2017-01-12 16:19:06\tGenotypeConcordance\tchecked    11,880,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 5:167,624,491\n",
      "INFO\t2017-01-12 16:19:06\tGenotypeConcordance\tchecked    11,890,000 variants.  Elapsed time: 00:06:12s.  Time for last 10,000:    0s.  Last read position: 5:168,444,499\n",
      "INFO\t2017-01-12 16:19:07\tGenotypeConcordance\tchecked    11,900,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 5:169,285,912\n",
      "INFO\t2017-01-12 16:19:07\tGenotypeConcordance\tchecked    11,910,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 5:170,131,207\n",
      "INFO\t2017-01-12 16:19:07\tGenotypeConcordance\tchecked    11,920,000 variants.  Elapsed time: 00:06:13s.  Time for last 10,000:    0s.  Last read position: 5:170,979,510\n",
      "INFO\t2017-01-12 16:19:08\tGenotypeConcordance\tchecked    11,930,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 5:171,797,320\n",
      "INFO\t2017-01-12 16:19:08\tGenotypeConcordance\tchecked    11,940,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 5:172,625,758\n",
      "INFO\t2017-01-12 16:19:08\tGenotypeConcordance\tchecked    11,950,000 variants.  Elapsed time: 00:06:14s.  Time for last 10,000:    0s.  Last read position: 5:173,447,445\n",
      "INFO\t2017-01-12 16:19:08\tGenotypeConcordance\tchecked    11,960,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 5:174,222,149\n",
      "INFO\t2017-01-12 16:19:09\tGenotypeConcordance\tchecked    11,970,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 5:174,974,131\n",
      "INFO\t2017-01-12 16:19:09\tGenotypeConcordance\tchecked    11,980,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 5:175,879,296\n",
      "INFO\t2017-01-12 16:19:09\tGenotypeConcordance\tchecked    11,990,000 variants.  Elapsed time: 00:06:15s.  Time for last 10,000:    0s.  Last read position: 5:176,742,298\n",
      "INFO\t2017-01-12 16:19:10\tGenotypeConcordance\tchecked    12,000,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 5:177,618,851\n",
      "INFO\t2017-01-12 16:19:10\tGenotypeConcordance\tchecked    12,010,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 5:178,260,360\n",
      "INFO\t2017-01-12 16:19:10\tGenotypeConcordance\tchecked    12,020,000 variants.  Elapsed time: 00:06:16s.  Time for last 10,000:    0s.  Last read position: 5:178,906,447\n",
      "INFO\t2017-01-12 16:19:11\tGenotypeConcordance\tchecked    12,030,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 5:179,670,271\n",
      "INFO\t2017-01-12 16:19:11\tGenotypeConcordance\tchecked    12,040,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 5:180,340,541\n",
      "INFO\t2017-01-12 16:19:11\tGenotypeConcordance\tchecked    12,050,000 variants.  Elapsed time: 00:06:17s.  Time for last 10,000:    0s.  Last read position: 6:471,112\n",
      "INFO\t2017-01-12 16:19:11\tGenotypeConcordance\tchecked    12,060,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:1,216,201\n",
      "INFO\t2017-01-12 16:19:12\tGenotypeConcordance\tchecked    12,070,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:1,981,340\n",
      "INFO\t2017-01-12 16:19:12\tGenotypeConcordance\tchecked    12,080,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:2,796,358\n",
      "INFO\t2017-01-12 16:19:12\tGenotypeConcordance\tchecked    12,090,000 variants.  Elapsed time: 00:06:18s.  Time for last 10,000:    0s.  Last read position: 6:3,464,174\n",
      "INFO\t2017-01-12 16:19:13\tGenotypeConcordance\tchecked    12,100,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:4,194,676\n",
      "INFO\t2017-01-12 16:19:13\tGenotypeConcordance\tchecked    12,110,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:4,932,311\n",
      "INFO\t2017-01-12 16:19:13\tGenotypeConcordance\tchecked    12,120,000 variants.  Elapsed time: 00:06:19s.  Time for last 10,000:    0s.  Last read position: 6:5,684,333\n",
      "INFO\t2017-01-12 16:19:14\tGenotypeConcordance\tchecked    12,130,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:6,454,886\n",
      "INFO\t2017-01-12 16:19:14\tGenotypeConcordance\tchecked    12,140,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:7,153,646\n",
      "INFO\t2017-01-12 16:19:14\tGenotypeConcordance\tchecked    12,150,000 variants.  Elapsed time: 00:06:20s.  Time for last 10,000:    0s.  Last read position: 6:7,938,293\n",
      "INFO\t2017-01-12 16:19:15\tGenotypeConcordance\tchecked    12,160,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:8,668,730\n",
      "INFO\t2017-01-12 16:19:15\tGenotypeConcordance\tchecked    12,170,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:9,507,852\n",
      "INFO\t2017-01-12 16:19:15\tGenotypeConcordance\tchecked    12,180,000 variants.  Elapsed time: 00:06:21s.  Time for last 10,000:    0s.  Last read position: 6:10,343,891\n",
      "INFO\t2017-01-12 16:19:15\tGenotypeConcordance\tchecked    12,190,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:11,086,696\n",
      "INFO\t2017-01-12 16:19:16\tGenotypeConcordance\tchecked    12,200,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:11,900,987\n",
      "INFO\t2017-01-12 16:19:16\tGenotypeConcordance\tchecked    12,210,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:12,756,394\n",
      "INFO\t2017-01-12 16:19:16\tGenotypeConcordance\tchecked    12,220,000 variants.  Elapsed time: 00:06:22s.  Time for last 10,000:    0s.  Last read position: 6:13,609,053\n",
      "INFO\t2017-01-12 16:19:17\tGenotypeConcordance\tchecked    12,230,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:14,445,739\n",
      "INFO\t2017-01-12 16:19:17\tGenotypeConcordance\tchecked    12,240,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:15,240,194\n",
      "INFO\t2017-01-12 16:19:17\tGenotypeConcordance\tchecked    12,250,000 variants.  Elapsed time: 00:06:23s.  Time for last 10,000:    0s.  Last read position: 6:16,087,931\n",
      "INFO\t2017-01-12 16:19:18\tGenotypeConcordance\tchecked    12,260,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:16,966,904\n",
      "INFO\t2017-01-12 16:19:18\tGenotypeConcordance\tchecked    12,270,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:17,801,284\n",
      "INFO\t2017-01-12 16:19:18\tGenotypeConcordance\tchecked    12,280,000 variants.  Elapsed time: 00:06:24s.  Time for last 10,000:    0s.  Last read position: 6:18,601,148\n",
      "INFO\t2017-01-12 16:19:19\tGenotypeConcordance\tchecked    12,290,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:19,431,136\n",
      "INFO\t2017-01-12 16:19:19\tGenotypeConcordance\tchecked    12,300,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:20,212,662\n",
      "INFO\t2017-01-12 16:19:19\tGenotypeConcordance\tchecked    12,310,000 variants.  Elapsed time: 00:06:25s.  Time for last 10,000:    0s.  Last read position: 6:21,045,170\n",
      "INFO\t2017-01-12 16:19:19\tGenotypeConcordance\tchecked    12,320,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:21,904,782\n",
      "INFO\t2017-01-12 16:19:20\tGenotypeConcordance\tchecked    12,330,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:22,668,647\n",
      "INFO\t2017-01-12 16:19:20\tGenotypeConcordance\tchecked    12,340,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:23,489,057\n",
      "INFO\t2017-01-12 16:19:20\tGenotypeConcordance\tchecked    12,350,000 variants.  Elapsed time: 00:06:26s.  Time for last 10,000:    0s.  Last read position: 6:24,097,567\n",
      "INFO\t2017-01-12 16:19:21\tGenotypeConcordance\tchecked    12,360,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:24,872,685\n",
      "INFO\t2017-01-12 16:19:21\tGenotypeConcordance\tchecked    12,370,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:25,649,715\n",
      "INFO\t2017-01-12 16:19:21\tGenotypeConcordance\tchecked    12,380,000 variants.  Elapsed time: 00:06:27s.  Time for last 10,000:    0s.  Last read position: 6:26,534,616\n",
      "INFO\t2017-01-12 16:19:22\tGenotypeConcordance\tchecked    12,390,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:27,470,810\n",
      "INFO\t2017-01-12 16:19:22\tGenotypeConcordance\tchecked    12,400,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:28,462,083\n",
      "INFO\t2017-01-12 16:19:22\tGenotypeConcordance\tchecked    12,410,000 variants.  Elapsed time: 00:06:28s.  Time for last 10,000:    0s.  Last read position: 6:29,286,670\n",
      "INFO\t2017-01-12 16:19:23\tGenotypeConcordance\tchecked    12,420,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:29,811,022\n",
      "INFO\t2017-01-12 16:19:23\tGenotypeConcordance\tchecked    12,430,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:30,044,935\n",
      "INFO\t2017-01-12 16:19:23\tGenotypeConcordance\tchecked    12,440,000 variants.  Elapsed time: 00:06:29s.  Time for last 10,000:    0s.  Last read position: 6:30,757,813\n",
      "INFO\t2017-01-12 16:19:24\tGenotypeConcordance\tchecked    12,450,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:31,234,693\n",
      "INFO\t2017-01-12 16:19:24\tGenotypeConcordance\tchecked    12,460,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:31,480,632\n",
      "INFO\t2017-01-12 16:19:24\tGenotypeConcordance\tchecked    12,470,000 variants.  Elapsed time: 00:06:30s.  Time for last 10,000:    0s.  Last read position: 6:32,360,464\n",
      "INFO\t2017-01-12 16:19:25\tGenotypeConcordance\tchecked    12,480,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 6:32,505,266\n",
      "INFO\t2017-01-12 16:19:25\tGenotypeConcordance\tchecked    12,490,000 variants.  Elapsed time: 00:06:31s.  Time for last 10,000:    0s.  Last read position: 6:32,589,327\n",
      "INFO\t2017-01-12 16:19:26\tGenotypeConcordance\tchecked    12,500,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:32,728,842\n",
      "INFO\t2017-01-12 16:19:26\tGenotypeConcordance\tchecked    12,510,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:33,226,633\n",
      "INFO\t2017-01-12 16:19:26\tGenotypeConcordance\tchecked    12,520,000 variants.  Elapsed time: 00:06:32s.  Time for last 10,000:    0s.  Last read position: 6:33,970,014\n",
      "INFO\t2017-01-12 16:19:27\tGenotypeConcordance\tchecked    12,530,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:34,715,186\n",
      "INFO\t2017-01-12 16:19:27\tGenotypeConcordance\tchecked    12,540,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:35,664,772\n",
      "INFO\t2017-01-12 16:19:27\tGenotypeConcordance\tchecked    12,550,000 variants.  Elapsed time: 00:06:33s.  Time for last 10,000:    0s.  Last read position: 6:36,544,329\n",
      "INFO\t2017-01-12 16:19:27\tGenotypeConcordance\tchecked    12,560,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:37,320,844\n",
      "INFO\t2017-01-12 16:19:28\tGenotypeConcordance\tchecked    12,570,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:38,169,864\n",
      "INFO\t2017-01-12 16:19:28\tGenotypeConcordance\tchecked    12,580,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:39,033,063\n",
      "INFO\t2017-01-12 16:19:28\tGenotypeConcordance\tchecked    12,590,000 variants.  Elapsed time: 00:06:34s.  Time for last 10,000:    0s.  Last read position: 6:39,853,946\n",
      "INFO\t2017-01-12 16:19:29\tGenotypeConcordance\tchecked    12,600,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:40,640,259\n",
      "INFO\t2017-01-12 16:19:29\tGenotypeConcordance\tchecked    12,610,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:41,449,935\n",
      "INFO\t2017-01-12 16:19:29\tGenotypeConcordance\tchecked    12,620,000 variants.  Elapsed time: 00:06:35s.  Time for last 10,000:    0s.  Last read position: 6:42,247,863\n",
      "INFO\t2017-01-12 16:19:30\tGenotypeConcordance\tchecked    12,630,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:43,186,138\n",
      "INFO\t2017-01-12 16:19:30\tGenotypeConcordance\tchecked    12,640,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:44,098,694\n",
      "INFO\t2017-01-12 16:19:30\tGenotypeConcordance\tchecked    12,650,000 variants.  Elapsed time: 00:06:36s.  Time for last 10,000:    0s.  Last read position: 6:44,909,518\n",
      "INFO\t2017-01-12 16:19:31\tGenotypeConcordance\tchecked    12,660,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:45,768,698\n",
      "INFO\t2017-01-12 16:19:31\tGenotypeConcordance\tchecked    12,670,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:46,675,025\n",
      "INFO\t2017-01-12 16:19:31\tGenotypeConcordance\tchecked    12,680,000 variants.  Elapsed time: 00:06:37s.  Time for last 10,000:    0s.  Last read position: 6:47,479,657\n",
      "INFO\t2017-01-12 16:19:31\tGenotypeConcordance\tchecked    12,690,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:48,305,445\n",
      "INFO\t2017-01-12 16:19:32\tGenotypeConcordance\tchecked    12,700,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:49,088,350\n",
      "INFO\t2017-01-12 16:19:32\tGenotypeConcordance\tchecked    12,710,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:49,927,623\n",
      "INFO\t2017-01-12 16:19:32\tGenotypeConcordance\tchecked    12,720,000 variants.  Elapsed time: 00:06:38s.  Time for last 10,000:    0s.  Last read position: 6:50,844,870\n",
      "INFO\t2017-01-12 16:19:33\tGenotypeConcordance\tchecked    12,730,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:51,746,771\n",
      "INFO\t2017-01-12 16:19:33\tGenotypeConcordance\tchecked    12,740,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:52,615,360\n",
      "INFO\t2017-01-12 16:19:33\tGenotypeConcordance\tchecked    12,750,000 variants.  Elapsed time: 00:06:39s.  Time for last 10,000:    0s.  Last read position: 6:53,412,381\n",
      "INFO\t2017-01-12 16:19:34\tGenotypeConcordance\tchecked    12,760,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:54,258,078\n",
      "INFO\t2017-01-12 16:19:34\tGenotypeConcordance\tchecked    12,770,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:55,013,303\n",
      "INFO\t2017-01-12 16:19:34\tGenotypeConcordance\tchecked    12,780,000 variants.  Elapsed time: 00:06:40s.  Time for last 10,000:    0s.  Last read position: 6:55,786,412\n",
      "INFO\t2017-01-12 16:19:34\tGenotypeConcordance\tchecked    12,790,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:56,665,182\n",
      "INFO\t2017-01-12 16:19:35\tGenotypeConcordance\tchecked    12,800,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:57,371,946\n",
      "INFO\t2017-01-12 16:19:35\tGenotypeConcordance\tchecked    12,810,000 variants.  Elapsed time: 00:06:41s.  Time for last 10,000:    0s.  Last read position: 6:57,588,249\n",
      "INFO\t2017-01-12 16:19:35\tGenotypeConcordance\tchecked    12,820,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:58,370,815\n",
      "INFO\t2017-01-12 16:19:36\tGenotypeConcordance\tchecked    12,830,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:62,054,774\n",
      "INFO\t2017-01-12 16:19:36\tGenotypeConcordance\tchecked    12,840,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:62,838,574\n",
      "INFO\t2017-01-12 16:19:36\tGenotypeConcordance\tchecked    12,850,000 variants.  Elapsed time: 00:06:42s.  Time for last 10,000:    0s.  Last read position: 6:63,638,516\n",
      "INFO\t2017-01-12 16:19:37\tGenotypeConcordance\tchecked    12,860,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 6:64,544,537\n",
      "INFO\t2017-01-12 16:19:37\tGenotypeConcordance\tchecked    12,870,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 6:65,351,867\n",
      "INFO\t2017-01-12 16:19:37\tGenotypeConcordance\tchecked    12,880,000 variants.  Elapsed time: 00:06:43s.  Time for last 10,000:    0s.  Last read position: 6:66,133,980\n",
      "INFO\t2017-01-12 16:19:37\tGenotypeConcordance\tchecked    12,890,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 6:66,889,399\n",
      "INFO\t2017-01-12 16:19:38\tGenotypeConcordance\tchecked    12,900,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 6:67,550,407\n",
      "INFO\t2017-01-12 16:19:38\tGenotypeConcordance\tchecked    12,910,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 6:68,223,010\n",
      "INFO\t2017-01-12 16:19:38\tGenotypeConcordance\tchecked    12,920,000 variants.  Elapsed time: 00:06:44s.  Time for last 10,000:    0s.  Last read position: 6:69,012,763\n",
      "INFO\t2017-01-12 16:19:39\tGenotypeConcordance\tchecked    12,930,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 6:69,849,256\n",
      "INFO\t2017-01-12 16:19:39\tGenotypeConcordance\tchecked    12,940,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 6:70,734,005\n",
      "INFO\t2017-01-12 16:19:39\tGenotypeConcordance\tchecked    12,950,000 variants.  Elapsed time: 00:06:45s.  Time for last 10,000:    0s.  Last read position: 6:71,606,550\n",
      "INFO\t2017-01-12 16:19:40\tGenotypeConcordance\tchecked    12,960,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 6:72,399,994\n",
      "INFO\t2017-01-12 16:19:40\tGenotypeConcordance\tchecked    12,970,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 6:73,293,710\n",
      "INFO\t2017-01-12 16:19:40\tGenotypeConcordance\tchecked    12,980,000 variants.  Elapsed time: 00:06:46s.  Time for last 10,000:    0s.  Last read position: 6:74,213,881\n",
      "INFO\t2017-01-12 16:19:41\tGenotypeConcordance\tchecked    12,990,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 6:74,971,740\n",
      "INFO\t2017-01-12 16:19:41\tGenotypeConcordance\tchecked    13,000,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 6:75,761,832\n",
      "INFO\t2017-01-12 16:19:41\tGenotypeConcordance\tchecked    13,010,000 variants.  Elapsed time: 00:06:47s.  Time for last 10,000:    0s.  Last read position: 6:76,803,226\n",
      "INFO\t2017-01-12 16:19:41\tGenotypeConcordance\tchecked    13,020,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 6:77,503,299\n",
      "INFO\t2017-01-12 16:19:42\tGenotypeConcordance\tchecked    13,030,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 6:78,320,930\n",
      "INFO\t2017-01-12 16:19:42\tGenotypeConcordance\tchecked    13,040,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 6:79,068,557\n",
      "INFO\t2017-01-12 16:19:42\tGenotypeConcordance\tchecked    13,050,000 variants.  Elapsed time: 00:06:48s.  Time for last 10,000:    0s.  Last read position: 6:79,779,767\n",
      "INFO\t2017-01-12 16:19:43\tGenotypeConcordance\tchecked    13,060,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 6:80,635,130\n",
      "INFO\t2017-01-12 16:19:43\tGenotypeConcordance\tchecked    13,070,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 6:81,409,273\n",
      "INFO\t2017-01-12 16:19:43\tGenotypeConcordance\tchecked    13,080,000 variants.  Elapsed time: 00:06:49s.  Time for last 10,000:    0s.  Last read position: 6:82,095,610\n",
      "INFO\t2017-01-12 16:19:44\tGenotypeConcordance\tchecked    13,090,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 6:82,957,593\n",
      "INFO\t2017-01-12 16:19:44\tGenotypeConcordance\tchecked    13,100,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 6:83,896,510\n",
      "INFO\t2017-01-12 16:19:44\tGenotypeConcordance\tchecked    13,110,000 variants.  Elapsed time: 00:06:50s.  Time for last 10,000:    0s.  Last read position: 6:84,871,065\n",
      "INFO\t2017-01-12 16:19:45\tGenotypeConcordance\tchecked    13,120,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 6:85,734,809\n",
      "INFO\t2017-01-12 16:19:45\tGenotypeConcordance\tchecked    13,130,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 6:86,585,114\n",
      "INFO\t2017-01-12 16:19:45\tGenotypeConcordance\tchecked    13,140,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 6:87,317,765\n",
      "INFO\t2017-01-12 16:19:45\tGenotypeConcordance\tchecked    13,150,000 variants.  Elapsed time: 00:06:51s.  Time for last 10,000:    0s.  Last read position: 6:88,120,947\n",
      "INFO\t2017-01-12 16:19:46\tGenotypeConcordance\tchecked    13,160,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 6:89,002,133\n",
      "INFO\t2017-01-12 16:19:46\tGenotypeConcordance\tchecked    13,170,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 6:89,839,906\n",
      "INFO\t2017-01-12 16:19:46\tGenotypeConcordance\tchecked    13,180,000 variants.  Elapsed time: 00:06:52s.  Time for last 10,000:    0s.  Last read position: 6:90,755,352\n",
      "INFO\t2017-01-12 16:19:47\tGenotypeConcordance\tchecked    13,190,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 6:91,720,488\n",
      "INFO\t2017-01-12 16:19:47\tGenotypeConcordance\tchecked    13,200,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 6:92,584,877\n",
      "INFO\t2017-01-12 16:19:47\tGenotypeConcordance\tchecked    13,210,000 variants.  Elapsed time: 00:06:53s.  Time for last 10,000:    0s.  Last read position: 6:93,440,311\n",
      "INFO\t2017-01-12 16:19:48\tGenotypeConcordance\tchecked    13,220,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 6:94,258,257\n",
      "INFO\t2017-01-12 16:19:48\tGenotypeConcordance\tchecked    13,230,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 6:95,081,286\n",
      "INFO\t2017-01-12 16:19:48\tGenotypeConcordance\tchecked    13,240,000 variants.  Elapsed time: 00:06:54s.  Time for last 10,000:    0s.  Last read position: 6:96,019,311\n",
      "INFO\t2017-01-12 16:19:48\tGenotypeConcordance\tchecked    13,250,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 6:96,896,078\n",
      "INFO\t2017-01-12 16:19:49\tGenotypeConcordance\tchecked    13,260,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 6:97,869,464\n",
      "INFO\t2017-01-12 16:19:49\tGenotypeConcordance\tchecked    13,270,000 variants.  Elapsed time: 00:06:55s.  Time for last 10,000:    0s.  Last read position: 6:98,981,180\n",
      "INFO\t2017-01-12 16:19:49\tGenotypeConcordance\tchecked    13,280,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 6:99,984,803\n",
      "INFO\t2017-01-12 16:19:50\tGenotypeConcordance\tchecked    13,290,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 6:100,886,666\n",
      "INFO\t2017-01-12 16:19:50\tGenotypeConcordance\tchecked    13,300,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 6:101,813,169\n",
      "INFO\t2017-01-12 16:19:50\tGenotypeConcordance\tchecked    13,310,000 variants.  Elapsed time: 00:06:56s.  Time for last 10,000:    0s.  Last read position: 6:102,654,831\n",
      "INFO\t2017-01-12 16:19:51\tGenotypeConcordance\tchecked    13,320,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 6:103,468,653\n",
      "INFO\t2017-01-12 16:19:51\tGenotypeConcordance\tchecked    13,330,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 6:104,242,548\n",
      "INFO\t2017-01-12 16:19:51\tGenotypeConcordance\tchecked    13,340,000 variants.  Elapsed time: 00:06:57s.  Time for last 10,000:    0s.  Last read position: 6:105,116,200\n",
      "INFO\t2017-01-12 16:19:52\tGenotypeConcordance\tchecked    13,350,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 6:106,070,640\n",
      "INFO\t2017-01-12 16:19:52\tGenotypeConcordance\tchecked    13,360,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 6:106,972,698\n",
      "INFO\t2017-01-12 16:19:52\tGenotypeConcordance\tchecked    13,370,000 variants.  Elapsed time: 00:06:58s.  Time for last 10,000:    0s.  Last read position: 6:107,786,703\n",
      "INFO\t2017-01-12 16:19:53\tGenotypeConcordance\tchecked    13,380,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 6:108,721,321\n",
      "INFO\t2017-01-12 16:19:53\tGenotypeConcordance\tchecked    13,390,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 6:109,633,189\n",
      "INFO\t2017-01-12 16:19:53\tGenotypeConcordance\tchecked    13,400,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 6:110,571,347\n",
      "INFO\t2017-01-12 16:19:53\tGenotypeConcordance\tchecked    13,410,000 variants.  Elapsed time: 00:06:59s.  Time for last 10,000:    0s.  Last read position: 6:111,464,713\n",
      "INFO\t2017-01-12 16:19:54\tGenotypeConcordance\tchecked    13,420,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 6:112,501,468\n",
      "INFO\t2017-01-12 16:19:54\tGenotypeConcordance\tchecked    13,430,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 6:113,456,066\n",
      "INFO\t2017-01-12 16:19:54\tGenotypeConcordance\tchecked    13,440,000 variants.  Elapsed time: 00:07:00s.  Time for last 10,000:    0s.  Last read position: 6:114,326,693\n",
      "INFO\t2017-01-12 16:19:55\tGenotypeConcordance\tchecked    13,450,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 6:115,295,446\n",
      "INFO\t2017-01-12 16:19:55\tGenotypeConcordance\tchecked    13,460,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 6:116,100,131\n",
      "INFO\t2017-01-12 16:19:55\tGenotypeConcordance\tchecked    13,470,000 variants.  Elapsed time: 00:07:01s.  Time for last 10,000:    0s.  Last read position: 6:117,081,206\n",
      "INFO\t2017-01-12 16:19:56\tGenotypeConcordance\tchecked    13,480,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 6:118,004,817\n",
      "INFO\t2017-01-12 16:19:56\tGenotypeConcordance\tchecked    13,490,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 6:118,942,300\n",
      "INFO\t2017-01-12 16:19:56\tGenotypeConcordance\tchecked    13,500,000 variants.  Elapsed time: 00:07:02s.  Time for last 10,000:    0s.  Last read position: 6:119,829,445\n",
      "INFO\t2017-01-12 16:19:56\tGenotypeConcordance\tchecked    13,510,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 6:120,701,960\n",
      "INFO\t2017-01-12 16:19:57\tGenotypeConcordance\tchecked    13,520,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 6:121,490,824\n",
      "INFO\t2017-01-12 16:19:57\tGenotypeConcordance\tchecked    13,530,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 6:122,349,231\n",
      "INFO\t2017-01-12 16:19:57\tGenotypeConcordance\tchecked    13,540,000 variants.  Elapsed time: 00:07:03s.  Time for last 10,000:    0s.  Last read position: 6:123,265,483\n",
      "INFO\t2017-01-12 16:19:58\tGenotypeConcordance\tchecked    13,550,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 6:124,115,011\n",
      "INFO\t2017-01-12 16:19:58\tGenotypeConcordance\tchecked    13,560,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 6:124,972,844\n",
      "INFO\t2017-01-12 16:19:58\tGenotypeConcordance\tchecked    13,570,000 variants.  Elapsed time: 00:07:04s.  Time for last 10,000:    0s.  Last read position: 6:125,885,633\n",
      "INFO\t2017-01-12 16:19:58\tGenotypeConcordance\tchecked    13,580,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 6:127,043,239\n",
      "INFO\t2017-01-12 16:19:59\tGenotypeConcordance\tchecked    13,590,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 6:128,081,304\n",
      "INFO\t2017-01-12 16:19:59\tGenotypeConcordance\tchecked    13,600,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 6:129,067,102\n",
      "INFO\t2017-01-12 16:19:59\tGenotypeConcordance\tchecked    13,610,000 variants.  Elapsed time: 00:07:05s.  Time for last 10,000:    0s.  Last read position: 6:129,947,080\n",
      "INFO\t2017-01-12 16:19:59\tGenotypeConcordance\tchecked    13,620,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 6:130,779,148\n",
      "INFO\t2017-01-12 16:20:00\tGenotypeConcordance\tchecked    13,630,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 6:131,625,710\n",
      "INFO\t2017-01-12 16:20:00\tGenotypeConcordance\tchecked    13,640,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 6:132,485,007\n",
      "INFO\t2017-01-12 16:20:00\tGenotypeConcordance\tchecked    13,650,000 variants.  Elapsed time: 00:07:06s.  Time for last 10,000:    0s.  Last read position: 6:133,256,704\n",
      "INFO\t2017-01-12 16:20:01\tGenotypeConcordance\tchecked    13,660,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 6:134,153,208\n",
      "INFO\t2017-01-12 16:20:01\tGenotypeConcordance\tchecked    13,670,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 6:135,013,843\n",
      "INFO\t2017-01-12 16:20:01\tGenotypeConcordance\tchecked    13,680,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 6:135,935,894\n",
      "INFO\t2017-01-12 16:20:01\tGenotypeConcordance\tchecked    13,690,000 variants.  Elapsed time: 00:07:07s.  Time for last 10,000:    0s.  Last read position: 6:136,923,799\n",
      "INFO\t2017-01-12 16:20:02\tGenotypeConcordance\tchecked    13,700,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 6:137,801,790\n",
      "INFO\t2017-01-12 16:20:02\tGenotypeConcordance\tchecked    13,710,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 6:138,616,627\n",
      "INFO\t2017-01-12 16:20:02\tGenotypeConcordance\tchecked    13,720,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 6:139,414,505\n",
      "INFO\t2017-01-12 16:20:02\tGenotypeConcordance\tchecked    13,730,000 variants.  Elapsed time: 00:07:08s.  Time for last 10,000:    0s.  Last read position: 6:140,281,071\n",
      "INFO\t2017-01-12 16:20:03\tGenotypeConcordance\tchecked    13,740,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 6:141,397,909\n",
      "INFO\t2017-01-12 16:20:03\tGenotypeConcordance\tchecked    13,750,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 6:142,274,618\n",
      "INFO\t2017-01-12 16:20:03\tGenotypeConcordance\tchecked    13,760,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 6:143,170,566\n",
      "INFO\t2017-01-12 16:20:03\tGenotypeConcordance\tchecked    13,770,000 variants.  Elapsed time: 00:07:09s.  Time for last 10,000:    0s.  Last read position: 6:144,101,343\n",
      "INFO\t2017-01-12 16:20:04\tGenotypeConcordance\tchecked    13,780,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 6:145,007,723\n",
      "INFO\t2017-01-12 16:20:04\tGenotypeConcordance\tchecked    13,790,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 6:145,868,680\n",
      "INFO\t2017-01-12 16:20:04\tGenotypeConcordance\tchecked    13,800,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 6:146,881,529\n",
      "INFO\t2017-01-12 16:20:04\tGenotypeConcordance\tchecked    13,810,000 variants.  Elapsed time: 00:07:10s.  Time for last 10,000:    0s.  Last read position: 6:147,852,871\n",
      "INFO\t2017-01-12 16:20:05\tGenotypeConcordance\tchecked    13,820,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 6:148,620,015\n",
      "INFO\t2017-01-12 16:20:05\tGenotypeConcordance\tchecked    13,830,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 6:149,407,904\n",
      "INFO\t2017-01-12 16:20:05\tGenotypeConcordance\tchecked    13,840,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 6:150,269,249\n",
      "INFO\t2017-01-12 16:20:05\tGenotypeConcordance\tchecked    13,850,000 variants.  Elapsed time: 00:07:11s.  Time for last 10,000:    0s.  Last read position: 6:150,949,223\n",
      "INFO\t2017-01-12 16:20:06\tGenotypeConcordance\tchecked    13,860,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 6:151,665,650\n",
      "INFO\t2017-01-12 16:20:06\tGenotypeConcordance\tchecked    13,870,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 6:152,510,429\n",
      "INFO\t2017-01-12 16:20:06\tGenotypeConcordance\tchecked    13,880,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 6:153,351,923\n",
      "INFO\t2017-01-12 16:20:06\tGenotypeConcordance\tchecked    13,890,000 variants.  Elapsed time: 00:07:12s.  Time for last 10,000:    0s.  Last read position: 6:154,108,443\n",
      "INFO\t2017-01-12 16:20:07\tGenotypeConcordance\tchecked    13,900,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 6:154,853,986\n",
      "INFO\t2017-01-12 16:20:07\tGenotypeConcordance\tchecked    13,910,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 6:155,633,969\n",
      "INFO\t2017-01-12 16:20:07\tGenotypeConcordance\tchecked    13,920,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 6:156,463,383\n",
      "INFO\t2017-01-12 16:20:07\tGenotypeConcordance\tchecked    13,930,000 variants.  Elapsed time: 00:07:13s.  Time for last 10,000:    0s.  Last read position: 6:157,314,040\n",
      "INFO\t2017-01-12 16:20:08\tGenotypeConcordance\tchecked    13,940,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 6:158,239,938\n",
      "INFO\t2017-01-12 16:20:08\tGenotypeConcordance\tchecked    13,950,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 6:159,029,041\n",
      "INFO\t2017-01-12 16:20:08\tGenotypeConcordance\tchecked    13,960,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 6:159,805,647\n",
      "INFO\t2017-01-12 16:20:08\tGenotypeConcordance\tchecked    13,970,000 variants.  Elapsed time: 00:07:14s.  Time for last 10,000:    0s.  Last read position: 6:160,579,368\n",
      "INFO\t2017-01-12 16:20:09\tGenotypeConcordance\tchecked    13,980,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 6:161,354,194\n",
      "INFO\t2017-01-12 16:20:09\tGenotypeConcordance\tchecked    13,990,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 6:162,128,543\n",
      "INFO\t2017-01-12 16:20:09\tGenotypeConcordance\tchecked    14,000,000 variants.  Elapsed time: 00:07:15s.  Time for last 10,000:    0s.  Last read position: 6:162,832,167\n",
      "INFO\t2017-01-12 16:20:09\tGenotypeConcordance\tchecked    14,010,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 6:163,627,243\n",
      "INFO\t2017-01-12 16:20:10\tGenotypeConcordance\tchecked    14,020,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 6:164,409,147\n",
      "INFO\t2017-01-12 16:20:10\tGenotypeConcordance\tchecked    14,030,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 6:165,190,113\n",
      "INFO\t2017-01-12 16:20:10\tGenotypeConcordance\tchecked    14,040,000 variants.  Elapsed time: 00:07:16s.  Time for last 10,000:    0s.  Last read position: 6:165,944,280\n",
      "INFO\t2017-01-12 16:20:10\tGenotypeConcordance\tchecked    14,050,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 6:166,693,445\n",
      "INFO\t2017-01-12 16:20:11\tGenotypeConcordance\tchecked    14,060,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 6:167,378,619\n",
      "INFO\t2017-01-12 16:20:11\tGenotypeConcordance\tchecked    14,070,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 6:168,086,374\n",
      "INFO\t2017-01-12 16:20:11\tGenotypeConcordance\tchecked    14,080,000 variants.  Elapsed time: 00:07:17s.  Time for last 10,000:    0s.  Last read position: 6:168,746,649\n",
      "INFO\t2017-01-12 16:20:12\tGenotypeConcordance\tchecked    14,090,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 6:169,354,072\n",
      "INFO\t2017-01-12 16:20:12\tGenotypeConcordance\tchecked    14,100,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 6:170,046,595\n",
      "INFO\t2017-01-12 16:20:12\tGenotypeConcordance\tchecked    14,110,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 6:170,767,224\n",
      "INFO\t2017-01-12 16:20:12\tGenotypeConcordance\tchecked    14,120,000 variants.  Elapsed time: 00:07:18s.  Time for last 10,000:    0s.  Last read position: 7:416,852\n",
      "INFO\t2017-01-12 16:20:13\tGenotypeConcordance\tchecked    14,130,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:960,050\n",
      "INFO\t2017-01-12 16:20:13\tGenotypeConcordance\tchecked    14,140,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:1,514,209\n",
      "INFO\t2017-01-12 16:20:13\tGenotypeConcordance\tchecked    14,150,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:2,120,371\n",
      "INFO\t2017-01-12 16:20:13\tGenotypeConcordance\tchecked    14,160,000 variants.  Elapsed time: 00:07:19s.  Time for last 10,000:    0s.  Last read position: 7:2,745,489\n",
      "INFO\t2017-01-12 16:20:14\tGenotypeConcordance\tchecked    14,170,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:3,292,730\n",
      "INFO\t2017-01-12 16:20:14\tGenotypeConcordance\tchecked    14,180,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:3,856,970\n",
      "INFO\t2017-01-12 16:20:14\tGenotypeConcordance\tchecked    14,190,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:4,419,504\n",
      "INFO\t2017-01-12 16:20:14\tGenotypeConcordance\tchecked    14,200,000 variants.  Elapsed time: 00:07:20s.  Time for last 10,000:    0s.  Last read position: 7:4,986,942\n",
      "INFO\t2017-01-12 16:20:15\tGenotypeConcordance\tchecked    14,210,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:5,584,173\n",
      "INFO\t2017-01-12 16:20:15\tGenotypeConcordance\tchecked    14,220,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:6,254,993\n",
      "INFO\t2017-01-12 16:20:15\tGenotypeConcordance\tchecked    14,230,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:6,924,262\n",
      "INFO\t2017-01-12 16:20:15\tGenotypeConcordance\tchecked    14,240,000 variants.  Elapsed time: 00:07:21s.  Time for last 10,000:    0s.  Last read position: 7:7,442,180\n",
      "INFO\t2017-01-12 16:20:16\tGenotypeConcordance\tchecked    14,250,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:8,157,016\n",
      "INFO\t2017-01-12 16:20:16\tGenotypeConcordance\tchecked    14,260,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:8,827,383\n",
      "INFO\t2017-01-12 16:20:16\tGenotypeConcordance\tchecked    14,270,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:9,435,883\n",
      "INFO\t2017-01-12 16:20:16\tGenotypeConcordance\tchecked    14,280,000 variants.  Elapsed time: 00:07:22s.  Time for last 10,000:    0s.  Last read position: 7:10,073,714\n",
      "INFO\t2017-01-12 16:20:17\tGenotypeConcordance\tchecked    14,290,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:10,759,632\n",
      "INFO\t2017-01-12 16:20:17\tGenotypeConcordance\tchecked    14,300,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:11,517,514\n",
      "INFO\t2017-01-12 16:20:17\tGenotypeConcordance\tchecked    14,310,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:12,190,315\n",
      "INFO\t2017-01-12 16:20:17\tGenotypeConcordance\tchecked    14,320,000 variants.  Elapsed time: 00:07:23s.  Time for last 10,000:    0s.  Last read position: 7:12,799,881\n",
      "INFO\t2017-01-12 16:20:18\tGenotypeConcordance\tchecked    14,330,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:13,434,529\n",
      "INFO\t2017-01-12 16:20:18\tGenotypeConcordance\tchecked    14,340,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:14,125,432\n",
      "INFO\t2017-01-12 16:20:18\tGenotypeConcordance\tchecked    14,350,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:14,847,293\n",
      "INFO\t2017-01-12 16:20:18\tGenotypeConcordance\tchecked    14,360,000 variants.  Elapsed time: 00:07:24s.  Time for last 10,000:    0s.  Last read position: 7:15,501,400\n",
      "INFO\t2017-01-12 16:20:19\tGenotypeConcordance\tchecked    14,370,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:16,237,550\n",
      "INFO\t2017-01-12 16:20:19\tGenotypeConcordance\tchecked    14,380,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:16,956,718\n",
      "INFO\t2017-01-12 16:20:19\tGenotypeConcordance\tchecked    14,390,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:17,732,291\n",
      "INFO\t2017-01-12 16:20:19\tGenotypeConcordance\tchecked    14,400,000 variants.  Elapsed time: 00:07:25s.  Time for last 10,000:    0s.  Last read position: 7:18,627,588\n",
      "INFO\t2017-01-12 16:20:20\tGenotypeConcordance\tchecked    14,410,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:19,379,028\n",
      "INFO\t2017-01-12 16:20:20\tGenotypeConcordance\tchecked    14,420,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:20,038,602\n",
      "INFO\t2017-01-12 16:20:20\tGenotypeConcordance\tchecked    14,430,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:20,777,031\n",
      "INFO\t2017-01-12 16:20:20\tGenotypeConcordance\tchecked    14,440,000 variants.  Elapsed time: 00:07:26s.  Time for last 10,000:    0s.  Last read position: 7:21,582,964\n",
      "INFO\t2017-01-12 16:20:21\tGenotypeConcordance\tchecked    14,450,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:22,259,212\n",
      "INFO\t2017-01-12 16:20:21\tGenotypeConcordance\tchecked    14,460,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:23,070,387\n",
      "INFO\t2017-01-12 16:20:21\tGenotypeConcordance\tchecked    14,470,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:23,915,676\n",
      "INFO\t2017-01-12 16:20:21\tGenotypeConcordance\tchecked    14,480,000 variants.  Elapsed time: 00:07:27s.  Time for last 10,000:    0s.  Last read position: 7:24,700,907\n",
      "INFO\t2017-01-12 16:20:22\tGenotypeConcordance\tchecked    14,490,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:25,483,161\n",
      "INFO\t2017-01-12 16:20:22\tGenotypeConcordance\tchecked    14,500,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:26,381,207\n",
      "INFO\t2017-01-12 16:20:22\tGenotypeConcordance\tchecked    14,510,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:27,325,930\n",
      "INFO\t2017-01-12 16:20:22\tGenotypeConcordance\tchecked    14,520,000 variants.  Elapsed time: 00:07:28s.  Time for last 10,000:    0s.  Last read position: 7:28,190,727\n",
      "INFO\t2017-01-12 16:20:23\tGenotypeConcordance\tchecked    14,530,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:29,024,764\n",
      "INFO\t2017-01-12 16:20:23\tGenotypeConcordance\tchecked    14,540,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:29,837,093\n",
      "INFO\t2017-01-12 16:20:23\tGenotypeConcordance\tchecked    14,550,000 variants.  Elapsed time: 00:07:29s.  Time for last 10,000:    0s.  Last read position: 7:30,802,303\n",
      "INFO\t2017-01-12 16:20:23\tGenotypeConcordance\tchecked    14,560,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:31,654,462\n",
      "INFO\t2017-01-12 16:20:24\tGenotypeConcordance\tchecked    14,570,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:32,429,979\n",
      "INFO\t2017-01-12 16:20:24\tGenotypeConcordance\tchecked    14,580,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:33,219,952\n",
      "INFO\t2017-01-12 16:20:24\tGenotypeConcordance\tchecked    14,590,000 variants.  Elapsed time: 00:07:30s.  Time for last 10,000:    0s.  Last read position: 7:34,117,565\n",
      "INFO\t2017-01-12 16:20:24\tGenotypeConcordance\tchecked    14,600,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:34,992,376\n",
      "INFO\t2017-01-12 16:20:25\tGenotypeConcordance\tchecked    14,610,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:35,913,034\n",
      "INFO\t2017-01-12 16:20:25\tGenotypeConcordance\tchecked    14,620,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:36,742,034\n",
      "INFO\t2017-01-12 16:20:25\tGenotypeConcordance\tchecked    14,630,000 variants.  Elapsed time: 00:07:31s.  Time for last 10,000:    0s.  Last read position: 7:37,602,360\n",
      "INFO\t2017-01-12 16:20:25\tGenotypeConcordance\tchecked    14,640,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:38,385,712\n",
      "INFO\t2017-01-12 16:20:26\tGenotypeConcordance\tchecked    14,650,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:39,249,358\n",
      "INFO\t2017-01-12 16:20:26\tGenotypeConcordance\tchecked    14,660,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:40,150,910\n",
      "INFO\t2017-01-12 16:20:26\tGenotypeConcordance\tchecked    14,670,000 variants.  Elapsed time: 00:07:32s.  Time for last 10,000:    0s.  Last read position: 7:41,139,004\n",
      "INFO\t2017-01-12 16:20:26\tGenotypeConcordance\tchecked    14,680,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:42,001,171\n",
      "INFO\t2017-01-12 16:20:27\tGenotypeConcordance\tchecked    14,690,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:42,843,325\n",
      "INFO\t2017-01-12 16:20:27\tGenotypeConcordance\tchecked    14,700,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:43,703,622\n",
      "INFO\t2017-01-12 16:20:27\tGenotypeConcordance\tchecked    14,710,000 variants.  Elapsed time: 00:07:33s.  Time for last 10,000:    0s.  Last read position: 7:44,664,806\n",
      "INFO\t2017-01-12 16:20:27\tGenotypeConcordance\tchecked    14,720,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:45,516,136\n",
      "INFO\t2017-01-12 16:20:28\tGenotypeConcordance\tchecked    14,730,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:46,323,938\n",
      "INFO\t2017-01-12 16:20:28\tGenotypeConcordance\tchecked    14,740,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:47,144,689\n",
      "INFO\t2017-01-12 16:20:28\tGenotypeConcordance\tchecked    14,750,000 variants.  Elapsed time: 00:07:34s.  Time for last 10,000:    0s.  Last read position: 7:47,915,917\n",
      "INFO\t2017-01-12 16:20:28\tGenotypeConcordance\tchecked    14,760,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:48,772,811\n",
      "INFO\t2017-01-12 16:20:29\tGenotypeConcordance\tchecked    14,770,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:49,547,057\n",
      "INFO\t2017-01-12 16:20:29\tGenotypeConcordance\tchecked    14,780,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:50,478,573\n",
      "INFO\t2017-01-12 16:20:29\tGenotypeConcordance\tchecked    14,790,000 variants.  Elapsed time: 00:07:35s.  Time for last 10,000:    0s.  Last read position: 7:51,332,935\n",
      "INFO\t2017-01-12 16:20:30\tGenotypeConcordance\tchecked    14,800,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 7:52,054,153\n",
      "INFO\t2017-01-12 16:20:30\tGenotypeConcordance\tchecked    14,810,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 7:52,737,397\n",
      "INFO\t2017-01-12 16:20:30\tGenotypeConcordance\tchecked    14,820,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 7:53,356,363\n",
      "INFO\t2017-01-12 16:20:30\tGenotypeConcordance\tchecked    14,830,000 variants.  Elapsed time: 00:07:36s.  Time for last 10,000:    0s.  Last read position: 7:54,109,581\n",
      "INFO\t2017-01-12 16:20:31\tGenotypeConcordance\tchecked    14,840,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 7:54,792,121\n",
      "INFO\t2017-01-12 16:20:31\tGenotypeConcordance\tchecked    14,850,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 7:55,595,270\n",
      "INFO\t2017-01-12 16:20:31\tGenotypeConcordance\tchecked    14,860,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 7:56,389,623\n",
      "INFO\t2017-01-12 16:20:31\tGenotypeConcordance\tchecked    14,870,000 variants.  Elapsed time: 00:07:37s.  Time for last 10,000:    0s.  Last read position: 7:57,064,567\n",
      "INFO\t2017-01-12 16:20:32\tGenotypeConcordance\tchecked    14,880,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 7:57,640,840\n",
      "INFO\t2017-01-12 16:20:32\tGenotypeConcordance\tchecked    14,890,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 7:57,927,767\n",
      "INFO\t2017-01-12 16:20:32\tGenotypeConcordance\tchecked    14,900,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 7:61,554,595\n",
      "INFO\t2017-01-12 16:20:32\tGenotypeConcordance\tchecked    14,910,000 variants.  Elapsed time: 00:07:38s.  Time for last 10,000:    0s.  Last read position: 7:61,823,904\n",
      "INFO\t2017-01-12 16:20:33\tGenotypeConcordance\tchecked    14,920,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 7:62,189,119\n",
      "INFO\t2017-01-12 16:20:33\tGenotypeConcordance\tchecked    14,930,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 7:62,828,260\n",
      "INFO\t2017-01-12 16:20:33\tGenotypeConcordance\tchecked    14,940,000 variants.  Elapsed time: 00:07:39s.  Time for last 10,000:    0s.  Last read position: 7:63,624,322\n",
      "INFO\t2017-01-12 16:20:33\tGenotypeConcordance\tchecked    14,950,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 7:64,427,308\n",
      "INFO\t2017-01-12 16:20:34\tGenotypeConcordance\tchecked    14,960,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 7:65,192,146\n",
      "INFO\t2017-01-12 16:20:34\tGenotypeConcordance\tchecked    14,970,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 7:66,033,033\n",
      "INFO\t2017-01-12 16:20:34\tGenotypeConcordance\tchecked    14,980,000 variants.  Elapsed time: 00:07:40s.  Time for last 10,000:    0s.  Last read position: 7:66,825,558\n",
      "INFO\t2017-01-12 16:20:35\tGenotypeConcordance\tchecked    14,990,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 7:67,532,341\n",
      "INFO\t2017-01-12 16:20:35\tGenotypeConcordance\tchecked    15,000,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 7:68,156,564\n",
      "INFO\t2017-01-12 16:20:35\tGenotypeConcordance\tchecked    15,010,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 7:68,889,003\n",
      "INFO\t2017-01-12 16:20:35\tGenotypeConcordance\tchecked    15,020,000 variants.  Elapsed time: 00:07:41s.  Time for last 10,000:    0s.  Last read position: 7:70,083,260\n",
      "INFO\t2017-01-12 16:20:36\tGenotypeConcordance\tchecked    15,030,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 7:70,842,337\n",
      "INFO\t2017-01-12 16:20:36\tGenotypeConcordance\tchecked    15,040,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 7:71,592,894\n",
      "INFO\t2017-01-12 16:20:36\tGenotypeConcordance\tchecked    15,050,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 7:72,403,209\n",
      "INFO\t2017-01-12 16:20:36\tGenotypeConcordance\tchecked    15,060,000 variants.  Elapsed time: 00:07:42s.  Time for last 10,000:    0s.  Last read position: 7:73,641,497\n",
      "INFO\t2017-01-12 16:20:37\tGenotypeConcordance\tchecked    15,070,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 7:75,243,991\n",
      "INFO\t2017-01-12 16:20:37\tGenotypeConcordance\tchecked    15,080,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 7:76,024,084\n",
      "INFO\t2017-01-12 16:20:37\tGenotypeConcordance\tchecked    15,090,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 7:76,897,826\n",
      "INFO\t2017-01-12 16:20:37\tGenotypeConcordance\tchecked    15,100,000 variants.  Elapsed time: 00:07:43s.  Time for last 10,000:    0s.  Last read position: 7:77,825,224\n",
      "INFO\t2017-01-12 16:20:38\tGenotypeConcordance\tchecked    15,110,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 7:78,599,311\n",
      "INFO\t2017-01-12 16:20:38\tGenotypeConcordance\tchecked    15,120,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 7:79,434,835\n",
      "INFO\t2017-01-12 16:20:38\tGenotypeConcordance\tchecked    15,130,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 7:80,311,647\n",
      "INFO\t2017-01-12 16:20:38\tGenotypeConcordance\tchecked    15,140,000 variants.  Elapsed time: 00:07:44s.  Time for last 10,000:    0s.  Last read position: 7:81,081,130\n",
      "INFO\t2017-01-12 16:20:39\tGenotypeConcordance\tchecked    15,150,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 7:81,985,187\n",
      "INFO\t2017-01-12 16:20:39\tGenotypeConcordance\tchecked    15,160,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 7:82,869,581\n",
      "INFO\t2017-01-12 16:20:39\tGenotypeConcordance\tchecked    15,170,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 7:83,679,299\n",
      "INFO\t2017-01-12 16:20:39\tGenotypeConcordance\tchecked    15,180,000 variants.  Elapsed time: 00:07:45s.  Time for last 10,000:    0s.  Last read position: 7:84,560,389\n",
      "INFO\t2017-01-12 16:20:40\tGenotypeConcordance\tchecked    15,190,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 7:85,456,390\n",
      "INFO\t2017-01-12 16:20:40\tGenotypeConcordance\tchecked    15,200,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 7:86,335,723\n",
      "INFO\t2017-01-12 16:20:40\tGenotypeConcordance\tchecked    15,210,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 7:87,331,322\n",
      "INFO\t2017-01-12 16:20:40\tGenotypeConcordance\tchecked    15,220,000 variants.  Elapsed time: 00:07:46s.  Time for last 10,000:    0s.  Last read position: 7:88,354,527\n",
      "INFO\t2017-01-12 16:20:41\tGenotypeConcordance\tchecked    15,230,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 7:89,166,525\n",
      "INFO\t2017-01-12 16:20:41\tGenotypeConcordance\tchecked    15,240,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 7:90,019,464\n",
      "INFO\t2017-01-12 16:20:41\tGenotypeConcordance\tchecked    15,250,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 7:90,969,510\n",
      "INFO\t2017-01-12 16:20:41\tGenotypeConcordance\tchecked    15,260,000 variants.  Elapsed time: 00:07:47s.  Time for last 10,000:    0s.  Last read position: 7:92,014,645\n",
      "INFO\t2017-01-12 16:20:42\tGenotypeConcordance\tchecked    15,270,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 7:93,133,758\n",
      "INFO\t2017-01-12 16:20:42\tGenotypeConcordance\tchecked    15,280,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 7:94,119,404\n",
      "INFO\t2017-01-12 16:20:42\tGenotypeConcordance\tchecked    15,290,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 7:95,175,661\n",
      "INFO\t2017-01-12 16:20:42\tGenotypeConcordance\tchecked    15,300,000 variants.  Elapsed time: 00:07:48s.  Time for last 10,000:    0s.  Last read position: 7:96,145,105\n",
      "INFO\t2017-01-12 16:20:43\tGenotypeConcordance\tchecked    15,310,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 7:97,060,570\n",
      "INFO\t2017-01-12 16:20:43\tGenotypeConcordance\tchecked    15,320,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 7:97,832,154\n",
      "INFO\t2017-01-12 16:20:43\tGenotypeConcordance\tchecked    15,330,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 7:98,660,529\n",
      "INFO\t2017-01-12 16:20:43\tGenotypeConcordance\tchecked    15,340,000 variants.  Elapsed time: 00:07:49s.  Time for last 10,000:    0s.  Last read position: 7:99,705,796\n",
      "INFO\t2017-01-12 16:20:44\tGenotypeConcordance\tchecked    15,350,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 7:100,635,956\n",
      "INFO\t2017-01-12 16:20:44\tGenotypeConcordance\tchecked    15,360,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 7:101,335,614\n",
      "INFO\t2017-01-12 16:20:44\tGenotypeConcordance\tchecked    15,370,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 7:102,296,892\n",
      "INFO\t2017-01-12 16:20:44\tGenotypeConcordance\tchecked    15,380,000 variants.  Elapsed time: 00:07:50s.  Time for last 10,000:    0s.  Last read position: 7:103,245,417\n",
      "INFO\t2017-01-12 16:20:45\tGenotypeConcordance\tchecked    15,390,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 7:103,976,997\n",
      "INFO\t2017-01-12 16:20:45\tGenotypeConcordance\tchecked    15,400,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 7:104,845,980\n",
      "INFO\t2017-01-12 16:20:45\tGenotypeConcordance\tchecked    15,410,000 variants.  Elapsed time: 00:07:51s.  Time for last 10,000:    0s.  Last read position: 7:105,730,603\n",
      "INFO\t2017-01-12 16:20:45\tGenotypeConcordance\tchecked    15,420,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 7:106,629,378\n",
      "INFO\t2017-01-12 16:20:46\tGenotypeConcordance\tchecked    15,430,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 7:107,551,530\n",
      "INFO\t2017-01-12 16:20:46\tGenotypeConcordance\tchecked    15,440,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 7:108,456,043\n",
      "INFO\t2017-01-12 16:20:46\tGenotypeConcordance\tchecked    15,450,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 7:109,264,452\n",
      "INFO\t2017-01-12 16:20:46\tGenotypeConcordance\tchecked    15,460,000 variants.  Elapsed time: 00:07:52s.  Time for last 10,000:    0s.  Last read position: 7:110,063,445\n",
      "INFO\t2017-01-12 16:20:47\tGenotypeConcordance\tchecked    15,470,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 7:111,002,121\n",
      "INFO\t2017-01-12 16:20:47\tGenotypeConcordance\tchecked    15,480,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 7:111,945,260\n",
      "INFO\t2017-01-12 16:20:47\tGenotypeConcordance\tchecked    15,490,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 7:112,942,522\n",
      "INFO\t2017-01-12 16:20:47\tGenotypeConcordance\tchecked    15,500,000 variants.  Elapsed time: 00:07:53s.  Time for last 10,000:    0s.  Last read position: 7:113,946,693\n",
      "INFO\t2017-01-12 16:20:48\tGenotypeConcordance\tchecked    15,510,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 7:114,991,272\n",
      "INFO\t2017-01-12 16:20:48\tGenotypeConcordance\tchecked    15,520,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 7:115,993,469\n",
      "INFO\t2017-01-12 16:20:48\tGenotypeConcordance\tchecked    15,530,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 7:117,162,954\n",
      "INFO\t2017-01-12 16:20:48\tGenotypeConcordance\tchecked    15,540,000 variants.  Elapsed time: 00:07:54s.  Time for last 10,000:    0s.  Last read position: 7:118,225,978\n",
      "INFO\t2017-01-12 16:20:49\tGenotypeConcordance\tchecked    15,550,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 7:119,161,984\n",
      "INFO\t2017-01-12 16:20:49\tGenotypeConcordance\tchecked    15,560,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 7:120,124,719\n",
      "INFO\t2017-01-12 16:20:49\tGenotypeConcordance\tchecked    15,570,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 7:121,138,156\n",
      "INFO\t2017-01-12 16:20:49\tGenotypeConcordance\tchecked    15,580,000 variants.  Elapsed time: 00:07:55s.  Time for last 10,000:    0s.  Last read position: 7:122,155,821\n",
      "INFO\t2017-01-12 16:20:50\tGenotypeConcordance\tchecked    15,590,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 7:123,089,364\n",
      "INFO\t2017-01-12 16:20:50\tGenotypeConcordance\tchecked    15,600,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 7:124,085,851\n",
      "INFO\t2017-01-12 16:20:50\tGenotypeConcordance\tchecked    15,610,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 7:124,959,006\n",
      "INFO\t2017-01-12 16:20:50\tGenotypeConcordance\tchecked    15,620,000 variants.  Elapsed time: 00:07:56s.  Time for last 10,000:    0s.  Last read position: 7:125,705,705\n",
      "INFO\t2017-01-12 16:20:51\tGenotypeConcordance\tchecked    15,630,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 7:126,592,772\n",
      "INFO\t2017-01-12 16:20:51\tGenotypeConcordance\tchecked    15,640,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 7:127,644,243\n",
      "INFO\t2017-01-12 16:20:51\tGenotypeConcordance\tchecked    15,650,000 variants.  Elapsed time: 00:07:57s.  Time for last 10,000:    0s.  Last read position: 7:128,514,177\n",
      "INFO\t2017-01-12 16:20:51\tGenotypeConcordance\tchecked    15,660,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 7:129,452,834\n",
      "INFO\t2017-01-12 16:20:52\tGenotypeConcordance\tchecked    15,670,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 7:130,495,991\n",
      "INFO\t2017-01-12 16:20:52\tGenotypeConcordance\tchecked    15,680,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 7:131,385,570\n",
      "INFO\t2017-01-12 16:20:52\tGenotypeConcordance\tchecked    15,690,000 variants.  Elapsed time: 00:07:58s.  Time for last 10,000:    0s.  Last read position: 7:132,186,073\n",
      "INFO\t2017-01-12 16:20:52\tGenotypeConcordance\tchecked    15,700,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 7:133,237,686\n",
      "INFO\t2017-01-12 16:20:53\tGenotypeConcordance\tchecked    15,710,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 7:134,202,002\n",
      "INFO\t2017-01-12 16:20:53\tGenotypeConcordance\tchecked    15,720,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 7:135,123,402\n",
      "INFO\t2017-01-12 16:20:53\tGenotypeConcordance\tchecked    15,730,000 variants.  Elapsed time: 00:07:59s.  Time for last 10,000:    0s.  Last read position: 7:135,979,376\n",
      "INFO\t2017-01-12 16:20:53\tGenotypeConcordance\tchecked    15,740,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 7:136,861,576\n",
      "INFO\t2017-01-12 16:20:54\tGenotypeConcordance\tchecked    15,750,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 7:137,783,240\n",
      "INFO\t2017-01-12 16:20:54\tGenotypeConcordance\tchecked    15,760,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 7:138,477,526\n",
      "INFO\t2017-01-12 16:20:54\tGenotypeConcordance\tchecked    15,770,000 variants.  Elapsed time: 00:08:00s.  Time for last 10,000:    0s.  Last read position: 7:139,321,855\n",
      "INFO\t2017-01-12 16:20:54\tGenotypeConcordance\tchecked    15,780,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 7:140,190,337\n",
      "INFO\t2017-01-12 16:20:55\tGenotypeConcordance\tchecked    15,790,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 7:141,169,727\n",
      "INFO\t2017-01-12 16:20:55\tGenotypeConcordance\tchecked    15,800,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 7:141,992,116\n",
      "INFO\t2017-01-12 16:20:55\tGenotypeConcordance\tchecked    15,810,000 variants.  Elapsed time: 00:08:01s.  Time for last 10,000:    0s.  Last read position: 7:142,474,879\n",
      "INFO\t2017-01-12 16:20:55\tGenotypeConcordance\tchecked    15,820,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 7:143,632,984\n",
      "INFO\t2017-01-12 16:20:56\tGenotypeConcordance\tchecked    15,830,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 7:144,622,552\n",
      "INFO\t2017-01-12 16:20:56\tGenotypeConcordance\tchecked    15,840,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 7:145,434,874\n",
      "INFO\t2017-01-12 16:20:56\tGenotypeConcordance\tchecked    15,850,000 variants.  Elapsed time: 00:08:02s.  Time for last 10,000:    0s.  Last read position: 7:146,201,148\n",
      "INFO\t2017-01-12 16:20:56\tGenotypeConcordance\tchecked    15,860,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 7:146,982,163\n",
      "INFO\t2017-01-12 16:20:57\tGenotypeConcordance\tchecked    15,870,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 7:147,645,125\n",
      "INFO\t2017-01-12 16:20:57\tGenotypeConcordance\tchecked    15,880,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 7:148,373,150\n",
      "INFO\t2017-01-12 16:20:57\tGenotypeConcordance\tchecked    15,890,000 variants.  Elapsed time: 00:08:03s.  Time for last 10,000:    0s.  Last read position: 7:149,183,459\n",
      "INFO\t2017-01-12 16:20:57\tGenotypeConcordance\tchecked    15,900,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 7:149,895,465\n",
      "INFO\t2017-01-12 16:20:58\tGenotypeConcordance\tchecked    15,910,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 7:150,602,428\n",
      "INFO\t2017-01-12 16:20:58\tGenotypeConcordance\tchecked    15,920,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 7:151,425,178\n",
      "INFO\t2017-01-12 16:20:58\tGenotypeConcordance\tchecked    15,930,000 variants.  Elapsed time: 00:08:04s.  Time for last 10,000:    0s.  Last read position: 7:152,113,680\n",
      "INFO\t2017-01-12 16:20:59\tGenotypeConcordance\tchecked    15,940,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 7:152,801,860\n",
      "INFO\t2017-01-12 16:20:59\tGenotypeConcordance\tchecked    15,950,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 7:153,503,092\n",
      "INFO\t2017-01-12 16:20:59\tGenotypeConcordance\tchecked    15,960,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 7:154,175,478\n",
      "INFO\t2017-01-12 16:20:59\tGenotypeConcordance\tchecked    15,970,000 variants.  Elapsed time: 00:08:05s.  Time for last 10,000:    0s.  Last read position: 7:154,951,511\n",
      "INFO\t2017-01-12 16:21:00\tGenotypeConcordance\tchecked    15,980,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 7:155,624,450\n",
      "INFO\t2017-01-12 16:21:00\tGenotypeConcordance\tchecked    15,990,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 7:156,286,725\n",
      "INFO\t2017-01-12 16:21:00\tGenotypeConcordance\tchecked    16,000,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 7:157,030,216\n",
      "INFO\t2017-01-12 16:21:00\tGenotypeConcordance\tchecked    16,010,000 variants.  Elapsed time: 00:08:06s.  Time for last 10,000:    0s.  Last read position: 7:157,624,989\n",
      "INFO\t2017-01-12 16:21:01\tGenotypeConcordance\tchecked    16,020,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 7:158,128,863\n",
      "INFO\t2017-01-12 16:21:01\tGenotypeConcordance\tchecked    16,030,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 7:158,765,311\n",
      "INFO\t2017-01-12 16:21:01\tGenotypeConcordance\tchecked    16,040,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:307,858\n",
      "INFO\t2017-01-12 16:21:01\tGenotypeConcordance\tchecked    16,050,000 variants.  Elapsed time: 00:08:07s.  Time for last 10,000:    0s.  Last read position: 8:855,881\n",
      "INFO\t2017-01-12 16:21:02\tGenotypeConcordance\tchecked    16,060,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:1,317,099\n",
      "INFO\t2017-01-12 16:21:02\tGenotypeConcordance\tchecked    16,070,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:1,817,602\n",
      "INFO\t2017-01-12 16:21:02\tGenotypeConcordance\tchecked    16,080,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:2,253,551\n",
      "INFO\t2017-01-12 16:21:02\tGenotypeConcordance\tchecked    16,090,000 variants.  Elapsed time: 00:08:08s.  Time for last 10,000:    0s.  Last read position: 8:2,747,624\n",
      "INFO\t2017-01-12 16:21:03\tGenotypeConcordance\tchecked    16,100,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:3,171,301\n",
      "INFO\t2017-01-12 16:21:03\tGenotypeConcordance\tchecked    16,110,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:3,511,936\n",
      "INFO\t2017-01-12 16:21:03\tGenotypeConcordance\tchecked    16,120,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:3,815,701\n",
      "INFO\t2017-01-12 16:21:03\tGenotypeConcordance\tchecked    16,130,000 variants.  Elapsed time: 00:08:09s.  Time for last 10,000:    0s.  Last read position: 8:4,109,451\n",
      "INFO\t2017-01-12 16:21:04\tGenotypeConcordance\tchecked    16,140,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:4,448,182\n",
      "INFO\t2017-01-12 16:21:04\tGenotypeConcordance\tchecked    16,150,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:4,824,267\n",
      "INFO\t2017-01-12 16:21:04\tGenotypeConcordance\tchecked    16,160,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:5,230,310\n",
      "INFO\t2017-01-12 16:21:04\tGenotypeConcordance\tchecked    16,170,000 variants.  Elapsed time: 00:08:10s.  Time for last 10,000:    0s.  Last read position: 8:5,629,968\n",
      "INFO\t2017-01-12 16:21:05\tGenotypeConcordance\tchecked    16,180,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:6,035,203\n",
      "INFO\t2017-01-12 16:21:05\tGenotypeConcordance\tchecked    16,190,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:6,450,224\n",
      "INFO\t2017-01-12 16:21:05\tGenotypeConcordance\tchecked    16,200,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:6,933,197\n",
      "INFO\t2017-01-12 16:21:05\tGenotypeConcordance\tchecked    16,210,000 variants.  Elapsed time: 00:08:11s.  Time for last 10,000:    0s.  Last read position: 8:7,915,309\n",
      "INFO\t2017-01-12 16:21:06\tGenotypeConcordance\tchecked    16,220,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:8,528,908\n",
      "INFO\t2017-01-12 16:21:06\tGenotypeConcordance\tchecked    16,230,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:9,103,724\n",
      "INFO\t2017-01-12 16:21:06\tGenotypeConcordance\tchecked    16,240,000 variants.  Elapsed time: 00:08:12s.  Time for last 10,000:    0s.  Last read position: 8:9,715,476\n",
      "INFO\t2017-01-12 16:21:06\tGenotypeConcordance\tchecked    16,250,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:10,349,672\n",
      "INFO\t2017-01-12 16:21:07\tGenotypeConcordance\tchecked    16,260,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:10,957,243\n",
      "INFO\t2017-01-12 16:21:07\tGenotypeConcordance\tchecked    16,270,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:11,529,409\n",
      "INFO\t2017-01-12 16:21:07\tGenotypeConcordance\tchecked    16,280,000 variants.  Elapsed time: 00:08:13s.  Time for last 10,000:    0s.  Last read position: 8:12,075,226\n",
      "INFO\t2017-01-12 16:21:07\tGenotypeConcordance\tchecked    16,290,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:12,675,621\n",
      "INFO\t2017-01-12 16:21:08\tGenotypeConcordance\tchecked    16,300,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:13,212,734\n",
      "INFO\t2017-01-12 16:21:08\tGenotypeConcordance\tchecked    16,310,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:13,687,093\n",
      "INFO\t2017-01-12 16:21:08\tGenotypeConcordance\tchecked    16,320,000 variants.  Elapsed time: 00:08:14s.  Time for last 10,000:    0s.  Last read position: 8:14,169,711\n",
      "INFO\t2017-01-12 16:21:08\tGenotypeConcordance\tchecked    16,330,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:14,669,987\n",
      "INFO\t2017-01-12 16:21:09\tGenotypeConcordance\tchecked    16,340,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:15,240,776\n",
      "INFO\t2017-01-12 16:21:09\tGenotypeConcordance\tchecked    16,350,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:15,745,075\n",
      "INFO\t2017-01-12 16:21:09\tGenotypeConcordance\tchecked    16,360,000 variants.  Elapsed time: 00:08:15s.  Time for last 10,000:    0s.  Last read position: 8:16,315,830\n",
      "INFO\t2017-01-12 16:21:09\tGenotypeConcordance\tchecked    16,370,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:16,852,283\n",
      "INFO\t2017-01-12 16:21:10\tGenotypeConcordance\tchecked    16,380,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:17,469,422\n",
      "INFO\t2017-01-12 16:21:10\tGenotypeConcordance\tchecked    16,390,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:18,004,224\n",
      "INFO\t2017-01-12 16:21:10\tGenotypeConcordance\tchecked    16,400,000 variants.  Elapsed time: 00:08:16s.  Time for last 10,000:    0s.  Last read position: 8:18,577,420\n",
      "INFO\t2017-01-12 16:21:11\tGenotypeConcordance\tchecked    16,410,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:19,170,003\n",
      "INFO\t2017-01-12 16:21:11\tGenotypeConcordance\tchecked    16,420,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:19,839,175\n",
      "INFO\t2017-01-12 16:21:11\tGenotypeConcordance\tchecked    16,430,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:20,533,362\n",
      "INFO\t2017-01-12 16:21:11\tGenotypeConcordance\tchecked    16,440,000 variants.  Elapsed time: 00:08:17s.  Time for last 10,000:    0s.  Last read position: 8:21,223,083\n",
      "INFO\t2017-01-12 16:21:12\tGenotypeConcordance\tchecked    16,450,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:22,029,129\n",
      "INFO\t2017-01-12 16:21:12\tGenotypeConcordance\tchecked    16,460,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:22,836,272\n",
      "INFO\t2017-01-12 16:21:12\tGenotypeConcordance\tchecked    16,470,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:23,540,354\n",
      "INFO\t2017-01-12 16:21:12\tGenotypeConcordance\tchecked    16,480,000 variants.  Elapsed time: 00:08:18s.  Time for last 10,000:    0s.  Last read position: 8:24,382,034\n",
      "INFO\t2017-01-12 16:21:13\tGenotypeConcordance\tchecked    16,490,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:25,187,933\n",
      "INFO\t2017-01-12 16:21:13\tGenotypeConcordance\tchecked    16,500,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:26,024,889\n",
      "INFO\t2017-01-12 16:21:13\tGenotypeConcordance\tchecked    16,510,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:26,816,678\n",
      "INFO\t2017-01-12 16:21:13\tGenotypeConcordance\tchecked    16,520,000 variants.  Elapsed time: 00:08:19s.  Time for last 10,000:    0s.  Last read position: 8:27,635,009\n",
      "INFO\t2017-01-12 16:21:14\tGenotypeConcordance\tchecked    16,530,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:28,461,311\n",
      "INFO\t2017-01-12 16:21:14\tGenotypeConcordance\tchecked    16,540,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:29,366,588\n",
      "INFO\t2017-01-12 16:21:14\tGenotypeConcordance\tchecked    16,550,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:30,207,421\n",
      "INFO\t2017-01-12 16:21:14\tGenotypeConcordance\tchecked    16,560,000 variants.  Elapsed time: 00:08:20s.  Time for last 10,000:    0s.  Last read position: 8:31,123,251\n",
      "INFO\t2017-01-12 16:21:15\tGenotypeConcordance\tchecked    16,570,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:31,992,258\n",
      "INFO\t2017-01-12 16:21:15\tGenotypeConcordance\tchecked    16,580,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:32,882,217\n",
      "INFO\t2017-01-12 16:21:15\tGenotypeConcordance\tchecked    16,590,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:33,780,637\n",
      "INFO\t2017-01-12 16:21:15\tGenotypeConcordance\tchecked    16,600,000 variants.  Elapsed time: 00:08:21s.  Time for last 10,000:    0s.  Last read position: 8:34,663,135\n",
      "INFO\t2017-01-12 16:21:16\tGenotypeConcordance\tchecked    16,610,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:35,648,562\n",
      "INFO\t2017-01-12 16:21:16\tGenotypeConcordance\tchecked    16,620,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:36,675,564\n",
      "INFO\t2017-01-12 16:21:16\tGenotypeConcordance\tchecked    16,630,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:37,647,063\n",
      "INFO\t2017-01-12 16:21:16\tGenotypeConcordance\tchecked    16,640,000 variants.  Elapsed time: 00:08:22s.  Time for last 10,000:    0s.  Last read position: 8:38,658,984\n",
      "INFO\t2017-01-12 16:21:16\tGenotypeConcordance\tchecked    16,650,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:39,627,891\n",
      "INFO\t2017-01-12 16:21:17\tGenotypeConcordance\tchecked    16,660,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:40,375,243\n",
      "INFO\t2017-01-12 16:21:17\tGenotypeConcordance\tchecked    16,670,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:41,149,507\n",
      "INFO\t2017-01-12 16:21:17\tGenotypeConcordance\tchecked    16,680,000 variants.  Elapsed time: 00:08:23s.  Time for last 10,000:    0s.  Last read position: 8:42,013,150\n",
      "INFO\t2017-01-12 16:21:17\tGenotypeConcordance\tchecked    16,690,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:43,006,010\n",
      "INFO\t2017-01-12 16:21:18\tGenotypeConcordance\tchecked    16,700,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:43,684,593\n",
      "INFO\t2017-01-12 16:21:18\tGenotypeConcordance\tchecked    16,710,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:47,287,215\n",
      "INFO\t2017-01-12 16:21:18\tGenotypeConcordance\tchecked    16,720,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:48,163,682\n",
      "INFO\t2017-01-12 16:21:18\tGenotypeConcordance\tchecked    16,730,000 variants.  Elapsed time: 00:08:24s.  Time for last 10,000:    0s.  Last read position: 8:49,357,151\n",
      "INFO\t2017-01-12 16:21:19\tGenotypeConcordance\tchecked    16,740,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:50,307,281\n",
      "INFO\t2017-01-12 16:21:19\tGenotypeConcordance\tchecked    16,750,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:51,141,137\n",
      "INFO\t2017-01-12 16:21:19\tGenotypeConcordance\tchecked    16,760,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:51,883,407\n",
      "INFO\t2017-01-12 16:21:19\tGenotypeConcordance\tchecked    16,770,000 variants.  Elapsed time: 00:08:25s.  Time for last 10,000:    0s.  Last read position: 8:52,626,791\n",
      "INFO\t2017-01-12 16:21:20\tGenotypeConcordance\tchecked    16,780,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 8:53,530,970\n",
      "INFO\t2017-01-12 16:21:20\tGenotypeConcordance\tchecked    16,790,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 8:54,364,075\n",
      "INFO\t2017-01-12 16:21:20\tGenotypeConcordance\tchecked    16,800,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 8:55,217,867\n",
      "INFO\t2017-01-12 16:21:20\tGenotypeConcordance\tchecked    16,810,000 variants.  Elapsed time: 00:08:26s.  Time for last 10,000:    0s.  Last read position: 8:56,085,891\n",
      "INFO\t2017-01-12 16:21:21\tGenotypeConcordance\tchecked    16,820,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 8:56,904,454\n",
      "INFO\t2017-01-12 16:21:21\tGenotypeConcordance\tchecked    16,830,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 8:57,723,177\n",
      "INFO\t2017-01-12 16:21:21\tGenotypeConcordance\tchecked    16,840,000 variants.  Elapsed time: 00:08:27s.  Time for last 10,000:    0s.  Last read position: 8:58,436,870\n",
      "INFO\t2017-01-12 16:21:21\tGenotypeConcordance\tchecked    16,850,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 8:59,289,763\n",
      "INFO\t2017-01-12 16:21:22\tGenotypeConcordance\tchecked    16,860,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 8:60,198,604\n",
      "INFO\t2017-01-12 16:21:22\tGenotypeConcordance\tchecked    16,870,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 8:61,122,265\n",
      "INFO\t2017-01-12 16:21:22\tGenotypeConcordance\tchecked    16,880,000 variants.  Elapsed time: 00:08:28s.  Time for last 10,000:    0s.  Last read position: 8:62,067,486\n",
      "INFO\t2017-01-12 16:21:22\tGenotypeConcordance\tchecked    16,890,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 8:62,924,308\n",
      "INFO\t2017-01-12 16:21:23\tGenotypeConcordance\tchecked    16,900,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 8:63,713,933\n",
      "INFO\t2017-01-12 16:21:23\tGenotypeConcordance\tchecked    16,910,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 8:64,609,056\n",
      "INFO\t2017-01-12 16:21:23\tGenotypeConcordance\tchecked    16,920,000 variants.  Elapsed time: 00:08:29s.  Time for last 10,000:    0s.  Last read position: 8:65,738,190\n",
      "INFO\t2017-01-12 16:21:23\tGenotypeConcordance\tchecked    16,930,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 8:66,669,635\n",
      "INFO\t2017-01-12 16:21:24\tGenotypeConcordance\tchecked    16,940,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 8:67,635,455\n",
      "INFO\t2017-01-12 16:21:24\tGenotypeConcordance\tchecked    16,950,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 8:68,673,649\n",
      "INFO\t2017-01-12 16:21:24\tGenotypeConcordance\tchecked    16,960,000 variants.  Elapsed time: 00:08:30s.  Time for last 10,000:    0s.  Last read position: 8:69,559,617\n",
      "INFO\t2017-01-12 16:21:24\tGenotypeConcordance\tchecked    16,970,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 8:70,375,937\n",
      "INFO\t2017-01-12 16:21:25\tGenotypeConcordance\tchecked    16,980,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 8:71,248,222\n",
      "INFO\t2017-01-12 16:21:25\tGenotypeConcordance\tchecked    16,990,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 8:72,204,352\n",
      "INFO\t2017-01-12 16:21:25\tGenotypeConcordance\tchecked    17,000,000 variants.  Elapsed time: 00:08:31s.  Time for last 10,000:    0s.  Last read position: 8:73,068,636\n",
      "INFO\t2017-01-12 16:21:25\tGenotypeConcordance\tchecked    17,010,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 8:73,929,721\n",
      "INFO\t2017-01-12 16:21:26\tGenotypeConcordance\tchecked    17,020,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 8:74,777,699\n",
      "INFO\t2017-01-12 16:21:26\tGenotypeConcordance\tchecked    17,030,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 8:75,663,682\n",
      "INFO\t2017-01-12 16:21:26\tGenotypeConcordance\tchecked    17,040,000 variants.  Elapsed time: 00:08:32s.  Time for last 10,000:    0s.  Last read position: 8:76,499,018\n",
      "INFO\t2017-01-12 16:21:26\tGenotypeConcordance\tchecked    17,050,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 8:77,432,409\n",
      "INFO\t2017-01-12 16:21:27\tGenotypeConcordance\tchecked    17,060,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 8:78,380,536\n",
      "INFO\t2017-01-12 16:21:27\tGenotypeConcordance\tchecked    17,070,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 8:79,277,609\n",
      "INFO\t2017-01-12 16:21:27\tGenotypeConcordance\tchecked    17,080,000 variants.  Elapsed time: 00:08:33s.  Time for last 10,000:    0s.  Last read position: 8:80,246,971\n",
      "INFO\t2017-01-12 16:21:27\tGenotypeConcordance\tchecked    17,090,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 8:81,172,159\n",
      "INFO\t2017-01-12 16:21:28\tGenotypeConcordance\tchecked    17,100,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 8:82,116,596\n",
      "INFO\t2017-01-12 16:21:28\tGenotypeConcordance\tchecked    17,110,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 8:82,996,542\n",
      "INFO\t2017-01-12 16:21:28\tGenotypeConcordance\tchecked    17,120,000 variants.  Elapsed time: 00:08:34s.  Time for last 10,000:    0s.  Last read position: 8:83,820,378\n",
      "INFO\t2017-01-12 16:21:29\tGenotypeConcordance\tchecked    17,130,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 8:84,599,247\n",
      "INFO\t2017-01-12 16:21:29\tGenotypeConcordance\tchecked    17,140,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 8:85,364,470\n",
      "INFO\t2017-01-12 16:21:29\tGenotypeConcordance\tchecked    17,150,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 8:86,343,937\n",
      "INFO\t2017-01-12 16:21:29\tGenotypeConcordance\tchecked    17,160,000 variants.  Elapsed time: 00:08:35s.  Time for last 10,000:    0s.  Last read position: 8:87,518,101\n",
      "INFO\t2017-01-12 16:21:30\tGenotypeConcordance\tchecked    17,170,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 8:88,328,054\n",
      "INFO\t2017-01-12 16:21:30\tGenotypeConcordance\tchecked    17,180,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 8:89,157,257\n",
      "INFO\t2017-01-12 16:21:30\tGenotypeConcordance\tchecked    17,190,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 8:90,035,634\n",
      "INFO\t2017-01-12 16:21:30\tGenotypeConcordance\tchecked    17,200,000 variants.  Elapsed time: 00:08:36s.  Time for last 10,000:    0s.  Last read position: 8:90,869,755\n",
      "INFO\t2017-01-12 16:21:31\tGenotypeConcordance\tchecked    17,210,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 8:91,788,105\n",
      "INFO\t2017-01-12 16:21:31\tGenotypeConcordance\tchecked    17,220,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 8:92,842,791\n",
      "INFO\t2017-01-12 16:21:31\tGenotypeConcordance\tchecked    17,230,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 8:93,935,665\n",
      "INFO\t2017-01-12 16:21:31\tGenotypeConcordance\tchecked    17,240,000 variants.  Elapsed time: 00:08:37s.  Time for last 10,000:    0s.  Last read position: 8:94,962,637\n",
      "INFO\t2017-01-12 16:21:32\tGenotypeConcordance\tchecked    17,250,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 8:95,818,155\n",
      "INFO\t2017-01-12 16:21:32\tGenotypeConcordance\tchecked    17,260,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 8:96,698,676\n",
      "INFO\t2017-01-12 16:21:32\tGenotypeConcordance\tchecked    17,270,000 variants.  Elapsed time: 00:08:38s.  Time for last 10,000:    0s.  Last read position: 8:97,563,575\n",
      "INFO\t2017-01-12 16:21:32\tGenotypeConcordance\tchecked    17,280,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 8:98,498,193\n",
      "INFO\t2017-01-12 16:21:33\tGenotypeConcordance\tchecked    17,290,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 8:99,296,821\n",
      "INFO\t2017-01-12 16:21:33\tGenotypeConcordance\tchecked    17,300,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 8:100,503,115\n",
      "INFO\t2017-01-12 16:21:33\tGenotypeConcordance\tchecked    17,310,000 variants.  Elapsed time: 00:08:39s.  Time for last 10,000:    0s.  Last read position: 8:101,577,545\n",
      "INFO\t2017-01-12 16:21:33\tGenotypeConcordance\tchecked    17,320,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 8:102,406,084\n",
      "INFO\t2017-01-12 16:21:34\tGenotypeConcordance\tchecked    17,330,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 8:103,258,448\n",
      "INFO\t2017-01-12 16:21:34\tGenotypeConcordance\tchecked    17,340,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 8:104,124,353\n",
      "INFO\t2017-01-12 16:21:34\tGenotypeConcordance\tchecked    17,350,000 variants.  Elapsed time: 00:08:40s.  Time for last 10,000:    0s.  Last read position: 8:105,018,863\n",
      "INFO\t2017-01-12 16:21:35\tGenotypeConcordance\tchecked    17,360,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 8:105,916,115\n",
      "INFO\t2017-01-12 16:21:35\tGenotypeConcordance\tchecked    17,370,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 8:106,883,102\n",
      "INFO\t2017-01-12 16:21:35\tGenotypeConcordance\tchecked    17,380,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 8:107,849,895\n",
      "INFO\t2017-01-12 16:21:35\tGenotypeConcordance\tchecked    17,390,000 variants.  Elapsed time: 00:08:41s.  Time for last 10,000:    0s.  Last read position: 8:108,706,623\n",
      "INFO\t2017-01-12 16:21:36\tGenotypeConcordance\tchecked    17,400,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 8:109,608,623\n",
      "INFO\t2017-01-12 16:21:36\tGenotypeConcordance\tchecked    17,410,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 8:110,573,770\n",
      "INFO\t2017-01-12 16:21:36\tGenotypeConcordance\tchecked    17,420,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 8:111,622,857\n",
      "INFO\t2017-01-12 16:21:36\tGenotypeConcordance\tchecked    17,430,000 variants.  Elapsed time: 00:08:42s.  Time for last 10,000:    0s.  Last read position: 8:112,537,538\n",
      "INFO\t2017-01-12 16:21:37\tGenotypeConcordance\tchecked    17,440,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 8:113,450,623\n",
      "INFO\t2017-01-12 16:21:37\tGenotypeConcordance\tchecked    17,450,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 8:114,440,909\n",
      "INFO\t2017-01-12 16:21:37\tGenotypeConcordance\tchecked    17,460,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 8:115,245,185\n",
      "INFO\t2017-01-12 16:21:37\tGenotypeConcordance\tchecked    17,470,000 variants.  Elapsed time: 00:08:43s.  Time for last 10,000:    0s.  Last read position: 8:116,080,876\n",
      "INFO\t2017-01-12 16:21:38\tGenotypeConcordance\tchecked    17,480,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 8:117,160,596\n",
      "INFO\t2017-01-12 16:21:38\tGenotypeConcordance\tchecked    17,490,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 8:118,146,283\n",
      "INFO\t2017-01-12 16:21:38\tGenotypeConcordance\tchecked    17,500,000 variants.  Elapsed time: 00:08:44s.  Time for last 10,000:    0s.  Last read position: 8:119,097,287\n",
      "INFO\t2017-01-12 16:21:38\tGenotypeConcordance\tchecked    17,510,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 8:119,972,696\n",
      "INFO\t2017-01-12 16:21:39\tGenotypeConcordance\tchecked    17,520,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 8:120,860,318\n",
      "INFO\t2017-01-12 16:21:39\tGenotypeConcordance\tchecked    17,530,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 8:121,777,896\n",
      "INFO\t2017-01-12 16:21:39\tGenotypeConcordance\tchecked    17,540,000 variants.  Elapsed time: 00:08:45s.  Time for last 10,000:    0s.  Last read position: 8:122,583,991\n",
      "INFO\t2017-01-12 16:21:40\tGenotypeConcordance\tchecked    17,550,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 8:123,466,306\n",
      "INFO\t2017-01-12 16:21:40\tGenotypeConcordance\tchecked    17,560,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 8:124,350,791\n",
      "INFO\t2017-01-12 16:21:40\tGenotypeConcordance\tchecked    17,570,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 8:125,229,684\n",
      "INFO\t2017-01-12 16:21:40\tGenotypeConcordance\tchecked    17,580,000 variants.  Elapsed time: 00:08:46s.  Time for last 10,000:    0s.  Last read position: 8:126,009,595\n",
      "INFO\t2017-01-12 16:21:41\tGenotypeConcordance\tchecked    17,590,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 8:126,859,771\n",
      "INFO\t2017-01-12 16:21:41\tGenotypeConcordance\tchecked    17,600,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 8:127,651,334\n",
      "INFO\t2017-01-12 16:21:41\tGenotypeConcordance\tchecked    17,610,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 8:128,446,680\n",
      "INFO\t2017-01-12 16:21:41\tGenotypeConcordance\tchecked    17,620,000 variants.  Elapsed time: 00:08:47s.  Time for last 10,000:    0s.  Last read position: 8:129,223,877\n",
      "INFO\t2017-01-12 16:21:42\tGenotypeConcordance\tchecked    17,630,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 8:130,025,628\n",
      "INFO\t2017-01-12 16:21:42\tGenotypeConcordance\tchecked    17,640,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 8:130,877,547\n",
      "INFO\t2017-01-12 16:21:42\tGenotypeConcordance\tchecked    17,650,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 8:131,731,502\n",
      "INFO\t2017-01-12 16:21:42\tGenotypeConcordance\tchecked    17,660,000 variants.  Elapsed time: 00:08:48s.  Time for last 10,000:    0s.  Last read position: 8:132,508,007\n",
      "INFO\t2017-01-12 16:21:43\tGenotypeConcordance\tchecked    17,670,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 8:133,264,699\n",
      "INFO\t2017-01-12 16:21:43\tGenotypeConcordance\tchecked    17,680,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 8:134,161,488\n",
      "INFO\t2017-01-12 16:21:43\tGenotypeConcordance\tchecked    17,690,000 variants.  Elapsed time: 00:08:49s.  Time for last 10,000:    0s.  Last read position: 8:134,885,329\n",
      "INFO\t2017-01-12 16:21:43\tGenotypeConcordance\tchecked    17,700,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 8:135,641,806\n",
      "INFO\t2017-01-12 16:21:44\tGenotypeConcordance\tchecked    17,710,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 8:136,456,909\n",
      "INFO\t2017-01-12 16:21:44\tGenotypeConcordance\tchecked    17,720,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 8:137,291,187\n",
      "INFO\t2017-01-12 16:21:44\tGenotypeConcordance\tchecked    17,730,000 variants.  Elapsed time: 00:08:50s.  Time for last 10,000:    0s.  Last read position: 8:138,078,078\n",
      "INFO\t2017-01-12 16:21:44\tGenotypeConcordance\tchecked    17,740,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 8:138,839,468\n",
      "INFO\t2017-01-12 16:21:45\tGenotypeConcordance\tchecked    17,750,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 8:139,625,123\n",
      "INFO\t2017-01-12 16:21:45\tGenotypeConcordance\tchecked    17,760,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 8:140,335,008\n",
      "INFO\t2017-01-12 16:21:45\tGenotypeConcordance\tchecked    17,770,000 variants.  Elapsed time: 00:08:51s.  Time for last 10,000:    0s.  Last read position: 8:141,074,664\n",
      "INFO\t2017-01-12 16:21:46\tGenotypeConcordance\tchecked    17,780,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 8:141,946,507\n",
      "INFO\t2017-01-12 16:21:46\tGenotypeConcordance\tchecked    17,790,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 8:142,636,650\n",
      "INFO\t2017-01-12 16:21:46\tGenotypeConcordance\tchecked    17,800,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 8:143,364,390\n",
      "INFO\t2017-01-12 16:21:46\tGenotypeConcordance\tchecked    17,810,000 variants.  Elapsed time: 00:08:52s.  Time for last 10,000:    0s.  Last read position: 8:144,058,475\n",
      "INFO\t2017-01-12 16:21:47\tGenotypeConcordance\tchecked    17,820,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 8:144,717,757\n",
      "INFO\t2017-01-12 16:21:47\tGenotypeConcordance\tchecked    17,830,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 8:145,752,130\n",
      "INFO\t2017-01-12 16:21:47\tGenotypeConcordance\tchecked    17,840,000 variants.  Elapsed time: 00:08:53s.  Time for last 10,000:    0s.  Last read position: 9:210,882\n",
      "INFO\t2017-01-12 16:21:47\tGenotypeConcordance\tchecked    17,850,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:765,689\n",
      "INFO\t2017-01-12 16:21:48\tGenotypeConcordance\tchecked    17,860,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:1,289,862\n",
      "INFO\t2017-01-12 16:21:48\tGenotypeConcordance\tchecked    17,870,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:1,806,787\n",
      "INFO\t2017-01-12 16:21:48\tGenotypeConcordance\tchecked    17,880,000 variants.  Elapsed time: 00:08:54s.  Time for last 10,000:    0s.  Last read position: 9:2,402,859\n",
      "INFO\t2017-01-12 16:21:48\tGenotypeConcordance\tchecked    17,890,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:3,039,034\n",
      "INFO\t2017-01-12 16:21:49\tGenotypeConcordance\tchecked    17,900,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:3,780,497\n",
      "INFO\t2017-01-12 16:21:49\tGenotypeConcordance\tchecked    17,910,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:4,393,680\n",
      "INFO\t2017-01-12 16:21:49\tGenotypeConcordance\tchecked    17,920,000 variants.  Elapsed time: 00:08:55s.  Time for last 10,000:    0s.  Last read position: 9:4,990,416\n",
      "INFO\t2017-01-12 16:21:50\tGenotypeConcordance\tchecked    17,930,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:5,720,296\n",
      "INFO\t2017-01-12 16:21:50\tGenotypeConcordance\tchecked    17,940,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:6,577,741\n",
      "INFO\t2017-01-12 16:21:50\tGenotypeConcordance\tchecked    17,950,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:7,147,976\n",
      "INFO\t2017-01-12 16:21:50\tGenotypeConcordance\tchecked    17,960,000 variants.  Elapsed time: 00:08:56s.  Time for last 10,000:    0s.  Last read position: 9:7,727,080\n",
      "INFO\t2017-01-12 16:21:51\tGenotypeConcordance\tchecked    17,970,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:8,371,681\n",
      "INFO\t2017-01-12 16:21:51\tGenotypeConcordance\tchecked    17,980,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:8,979,212\n",
      "INFO\t2017-01-12 16:21:51\tGenotypeConcordance\tchecked    17,990,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:9,634,771\n",
      "INFO\t2017-01-12 16:21:51\tGenotypeConcordance\tchecked    18,000,000 variants.  Elapsed time: 00:08:57s.  Time for last 10,000:    0s.  Last read position: 9:10,207,866\n",
      "INFO\t2017-01-12 16:21:52\tGenotypeConcordance\tchecked    18,010,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:10,835,964\n",
      "INFO\t2017-01-12 16:21:52\tGenotypeConcordance\tchecked    18,020,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:11,426,814\n",
      "INFO\t2017-01-12 16:21:52\tGenotypeConcordance\tchecked    18,030,000 variants.  Elapsed time: 00:08:58s.  Time for last 10,000:    0s.  Last read position: 9:12,019,745\n",
      "INFO\t2017-01-12 16:21:52\tGenotypeConcordance\tchecked    18,040,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:12,693,777\n",
      "INFO\t2017-01-12 16:21:53\tGenotypeConcordance\tchecked    18,050,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:13,505,677\n",
      "INFO\t2017-01-12 16:21:53\tGenotypeConcordance\tchecked    18,060,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:14,182,713\n",
      "INFO\t2017-01-12 16:21:53\tGenotypeConcordance\tchecked    18,070,000 variants.  Elapsed time: 00:08:59s.  Time for last 10,000:    0s.  Last read position: 9:14,907,332\n",
      "INFO\t2017-01-12 16:21:54\tGenotypeConcordance\tchecked    18,080,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:15,578,240\n",
      "INFO\t2017-01-12 16:21:54\tGenotypeConcordance\tchecked    18,090,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:16,299,388\n",
      "INFO\t2017-01-12 16:21:54\tGenotypeConcordance\tchecked    18,100,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:16,974,010\n",
      "INFO\t2017-01-12 16:21:54\tGenotypeConcordance\tchecked    18,110,000 variants.  Elapsed time: 00:09:00s.  Time for last 10,000:    0s.  Last read position: 9:17,652,187\n",
      "INFO\t2017-01-12 16:21:55\tGenotypeConcordance\tchecked    18,120,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:18,313,213\n",
      "INFO\t2017-01-12 16:21:55\tGenotypeConcordance\tchecked    18,130,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:18,999,766\n",
      "INFO\t2017-01-12 16:21:55\tGenotypeConcordance\tchecked    18,140,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:19,706,618\n",
      "INFO\t2017-01-12 16:21:55\tGenotypeConcordance\tchecked    18,150,000 variants.  Elapsed time: 00:09:01s.  Time for last 10,000:    0s.  Last read position: 9:20,477,167\n",
      "INFO\t2017-01-12 16:21:56\tGenotypeConcordance\tchecked    18,160,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:21,292,255\n",
      "INFO\t2017-01-12 16:21:56\tGenotypeConcordance\tchecked    18,170,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:22,088,832\n",
      "INFO\t2017-01-12 16:21:56\tGenotypeConcordance\tchecked    18,180,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:22,841,222\n",
      "INFO\t2017-01-12 16:21:56\tGenotypeConcordance\tchecked    18,190,000 variants.  Elapsed time: 00:09:02s.  Time for last 10,000:    0s.  Last read position: 9:23,504,185\n",
      "INFO\t2017-01-12 16:21:57\tGenotypeConcordance\tchecked    18,200,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:24,301,818\n",
      "INFO\t2017-01-12 16:21:57\tGenotypeConcordance\tchecked    18,210,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:25,042,178\n",
      "INFO\t2017-01-12 16:21:57\tGenotypeConcordance\tchecked    18,220,000 variants.  Elapsed time: 00:09:03s.  Time for last 10,000:    0s.  Last read position: 9:25,687,819\n",
      "INFO\t2017-01-12 16:21:57\tGenotypeConcordance\tchecked    18,230,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:26,465,331\n",
      "INFO\t2017-01-12 16:21:58\tGenotypeConcordance\tchecked    18,240,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:27,199,009\n",
      "INFO\t2017-01-12 16:21:58\tGenotypeConcordance\tchecked    18,250,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:27,937,370\n",
      "INFO\t2017-01-12 16:21:58\tGenotypeConcordance\tchecked    18,260,000 variants.  Elapsed time: 00:09:04s.  Time for last 10,000:    0s.  Last read position: 9:28,639,917\n",
      "INFO\t2017-01-12 16:21:59\tGenotypeConcordance\tchecked    18,270,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:29,409,616\n",
      "INFO\t2017-01-12 16:21:59\tGenotypeConcordance\tchecked    18,280,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:30,137,652\n",
      "INFO\t2017-01-12 16:21:59\tGenotypeConcordance\tchecked    18,290,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:30,938,150\n",
      "INFO\t2017-01-12 16:21:59\tGenotypeConcordance\tchecked    18,300,000 variants.  Elapsed time: 00:09:05s.  Time for last 10,000:    0s.  Last read position: 9:31,622,953\n",
      "INFO\t2017-01-12 16:22:00\tGenotypeConcordance\tchecked    18,310,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:32,324,971\n",
      "INFO\t2017-01-12 16:22:00\tGenotypeConcordance\tchecked    18,320,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:33,170,547\n",
      "INFO\t2017-01-12 16:22:00\tGenotypeConcordance\tchecked    18,330,000 variants.  Elapsed time: 00:09:06s.  Time for last 10,000:    0s.  Last read position: 9:34,027,736\n",
      "INFO\t2017-01-12 16:22:00\tGenotypeConcordance\tchecked    18,340,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 9:35,037,954\n",
      "INFO\t2017-01-12 16:22:01\tGenotypeConcordance\tchecked    18,350,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 9:36,104,753\n",
      "INFO\t2017-01-12 16:22:01\tGenotypeConcordance\tchecked    18,360,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 9:36,965,295\n",
      "INFO\t2017-01-12 16:22:01\tGenotypeConcordance\tchecked    18,370,000 variants.  Elapsed time: 00:09:07s.  Time for last 10,000:    0s.  Last read position: 9:37,889,110\n",
      "INFO\t2017-01-12 16:22:02\tGenotypeConcordance\tchecked    18,380,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 9:38,676,375\n",
      "INFO\t2017-01-12 16:22:02\tGenotypeConcordance\tchecked    18,390,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 9:40,646,116\n",
      "INFO\t2017-01-12 16:22:02\tGenotypeConcordance\tchecked    18,400,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 9:43,437,884\n",
      "INFO\t2017-01-12 16:22:02\tGenotypeConcordance\tchecked    18,410,000 variants.  Elapsed time: 00:09:08s.  Time for last 10,000:    0s.  Last read position: 9:44,761,195\n",
      "INFO\t2017-01-12 16:22:03\tGenotypeConcordance\tchecked    18,420,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 9:46,141,478\n",
      "INFO\t2017-01-12 16:22:03\tGenotypeConcordance\tchecked    18,430,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 9:66,478,266\n",
      "INFO\t2017-01-12 16:22:03\tGenotypeConcordance\tchecked    18,440,000 variants.  Elapsed time: 00:09:09s.  Time for last 10,000:    0s.  Last read position: 9:67,302,206\n",
      "INFO\t2017-01-12 16:22:04\tGenotypeConcordance\tchecked    18,450,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 9:68,406,490\n",
      "INFO\t2017-01-12 16:22:04\tGenotypeConcordance\tchecked    18,460,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 9:68,990,079\n",
      "INFO\t2017-01-12 16:22:04\tGenotypeConcordance\tchecked    18,470,000 variants.  Elapsed time: 00:09:10s.  Time for last 10,000:    0s.  Last read position: 9:69,784,342\n",
      "INFO\t2017-01-12 16:22:05\tGenotypeConcordance\tchecked    18,480,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 9:70,944,657\n",
      "INFO\t2017-01-12 16:22:05\tGenotypeConcordance\tchecked    18,490,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 9:71,842,391\n",
      "INFO\t2017-01-12 16:22:05\tGenotypeConcordance\tchecked    18,500,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 9:72,743,932\n",
      "INFO\t2017-01-12 16:22:05\tGenotypeConcordance\tchecked    18,510,000 variants.  Elapsed time: 00:09:11s.  Time for last 10,000:    0s.  Last read position: 9:73,649,941\n",
      "INFO\t2017-01-12 16:22:06\tGenotypeConcordance\tchecked    18,520,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 9:74,555,136\n",
      "INFO\t2017-01-12 16:22:06\tGenotypeConcordance\tchecked    18,530,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 9:75,492,246\n",
      "INFO\t2017-01-12 16:22:06\tGenotypeConcordance\tchecked    18,540,000 variants.  Elapsed time: 00:09:12s.  Time for last 10,000:    0s.  Last read position: 9:76,491,744\n",
      "INFO\t2017-01-12 16:22:06\tGenotypeConcordance\tchecked    18,550,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 9:77,464,267\n",
      "INFO\t2017-01-12 16:22:07\tGenotypeConcordance\tchecked    18,560,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 9:78,294,951\n",
      "INFO\t2017-01-12 16:22:07\tGenotypeConcordance\tchecked    18,570,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 9:79,099,199\n",
      "INFO\t2017-01-12 16:22:07\tGenotypeConcordance\tchecked    18,580,000 variants.  Elapsed time: 00:09:13s.  Time for last 10,000:    0s.  Last read position: 9:79,849,874\n",
      "INFO\t2017-01-12 16:22:08\tGenotypeConcordance\tchecked    18,590,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 9:80,741,971\n",
      "INFO\t2017-01-12 16:22:08\tGenotypeConcordance\tchecked    18,600,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 9:81,539,515\n",
      "INFO\t2017-01-12 16:22:08\tGenotypeConcordance\tchecked    18,610,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 9:82,385,885\n",
      "INFO\t2017-01-12 16:22:08\tGenotypeConcordance\tchecked    18,620,000 variants.  Elapsed time: 00:09:14s.  Time for last 10,000:    0s.  Last read position: 9:83,179,764\n",
      "INFO\t2017-01-12 16:22:09\tGenotypeConcordance\tchecked    18,630,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 9:83,895,853\n",
      "INFO\t2017-01-12 16:22:09\tGenotypeConcordance\tchecked    18,640,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 9:84,900,687\n",
      "INFO\t2017-01-12 16:22:09\tGenotypeConcordance\tchecked    18,650,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 9:85,847,317\n",
      "INFO\t2017-01-12 16:22:09\tGenotypeConcordance\tchecked    18,660,000 variants.  Elapsed time: 00:09:15s.  Time for last 10,000:    0s.  Last read position: 9:86,738,065\n",
      "INFO\t2017-01-12 16:22:10\tGenotypeConcordance\tchecked    18,670,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 9:87,579,283\n",
      "INFO\t2017-01-12 16:22:10\tGenotypeConcordance\tchecked    18,680,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 9:88,487,791\n",
      "INFO\t2017-01-12 16:22:10\tGenotypeConcordance\tchecked    18,690,000 variants.  Elapsed time: 00:09:16s.  Time for last 10,000:    0s.  Last read position: 9:89,340,620\n",
      "INFO\t2017-01-12 16:22:10\tGenotypeConcordance\tchecked    18,700,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 9:90,141,514\n",
      "INFO\t2017-01-12 16:22:11\tGenotypeConcordance\tchecked    18,710,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 9:90,871,647\n",
      "INFO\t2017-01-12 16:22:11\tGenotypeConcordance\tchecked    18,720,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 9:91,590,324\n",
      "INFO\t2017-01-12 16:22:11\tGenotypeConcordance\tchecked    18,730,000 variants.  Elapsed time: 00:09:17s.  Time for last 10,000:    0s.  Last read position: 9:92,495,150\n",
      "INFO\t2017-01-12 16:22:12\tGenotypeConcordance\tchecked    18,740,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 9:93,451,481\n",
      "INFO\t2017-01-12 16:22:12\tGenotypeConcordance\tchecked    18,750,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 9:94,326,169\n",
      "INFO\t2017-01-12 16:22:12\tGenotypeConcordance\tchecked    18,760,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 9:95,210,570\n",
      "INFO\t2017-01-12 16:22:12\tGenotypeConcordance\tchecked    18,770,000 variants.  Elapsed time: 00:09:18s.  Time for last 10,000:    0s.  Last read position: 9:96,114,712\n",
      "INFO\t2017-01-12 16:22:13\tGenotypeConcordance\tchecked    18,780,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 9:97,010,498\n",
      "INFO\t2017-01-12 16:22:13\tGenotypeConcordance\tchecked    18,790,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 9:97,898,118\n",
      "INFO\t2017-01-12 16:22:13\tGenotypeConcordance\tchecked    18,800,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 9:98,839,797\n",
      "INFO\t2017-01-12 16:22:13\tGenotypeConcordance\tchecked    18,810,000 variants.  Elapsed time: 00:09:19s.  Time for last 10,000:    0s.  Last read position: 9:99,739,358\n",
      "INFO\t2017-01-12 16:22:14\tGenotypeConcordance\tchecked    18,820,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 9:100,704,638\n",
      "INFO\t2017-01-12 16:22:14\tGenotypeConcordance\tchecked    18,830,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 9:101,574,873\n",
      "INFO\t2017-01-12 16:22:14\tGenotypeConcordance\tchecked    18,840,000 variants.  Elapsed time: 00:09:20s.  Time for last 10,000:    0s.  Last read position: 9:102,651,712\n",
      "INFO\t2017-01-12 16:22:14\tGenotypeConcordance\tchecked    18,850,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 9:103,704,965\n",
      "INFO\t2017-01-12 16:22:15\tGenotypeConcordance\tchecked    18,860,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 9:104,605,646\n",
      "INFO\t2017-01-12 16:22:15\tGenotypeConcordance\tchecked    18,870,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 9:105,307,422\n",
      "INFO\t2017-01-12 16:22:15\tGenotypeConcordance\tchecked    18,880,000 variants.  Elapsed time: 00:09:21s.  Time for last 10,000:    0s.  Last read position: 9:106,024,523\n",
      "INFO\t2017-01-12 16:22:16\tGenotypeConcordance\tchecked    18,890,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 9:106,760,737\n",
      "INFO\t2017-01-12 16:22:16\tGenotypeConcordance\tchecked    18,900,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 9:107,525,317\n",
      "INFO\t2017-01-12 16:22:16\tGenotypeConcordance\tchecked    18,910,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 9:108,293,284\n",
      "INFO\t2017-01-12 16:22:16\tGenotypeConcordance\tchecked    18,920,000 variants.  Elapsed time: 00:09:22s.  Time for last 10,000:    0s.  Last read position: 9:109,106,490\n",
      "INFO\t2017-01-12 16:22:17\tGenotypeConcordance\tchecked    18,930,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 9:110,051,413\n",
      "INFO\t2017-01-12 16:22:17\tGenotypeConcordance\tchecked    18,940,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 9:110,801,732\n",
      "INFO\t2017-01-12 16:22:17\tGenotypeConcordance\tchecked    18,950,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 9:111,541,925\n",
      "INFO\t2017-01-12 16:22:17\tGenotypeConcordance\tchecked    18,960,000 variants.  Elapsed time: 00:09:23s.  Time for last 10,000:    0s.  Last read position: 9:112,419,640\n",
      "INFO\t2017-01-12 16:22:18\tGenotypeConcordance\tchecked    18,970,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 9:113,197,163\n",
      "INFO\t2017-01-12 16:22:18\tGenotypeConcordance\tchecked    18,980,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 9:113,996,326\n",
      "INFO\t2017-01-12 16:22:18\tGenotypeConcordance\tchecked    18,990,000 variants.  Elapsed time: 00:09:24s.  Time for last 10,000:    0s.  Last read position: 9:114,858,970\n",
      "INFO\t2017-01-12 16:22:18\tGenotypeConcordance\tchecked    19,000,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 9:115,728,200\n",
      "INFO\t2017-01-12 16:22:19\tGenotypeConcordance\tchecked    19,010,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 9:116,692,868\n",
      "INFO\t2017-01-12 16:22:19\tGenotypeConcordance\tchecked    19,020,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 9:117,505,696\n",
      "INFO\t2017-01-12 16:22:19\tGenotypeConcordance\tchecked    19,030,000 variants.  Elapsed time: 00:09:25s.  Time for last 10,000:    0s.  Last read position: 9:118,393,276\n",
      "INFO\t2017-01-12 16:22:20\tGenotypeConcordance\tchecked    19,040,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 9:119,383,881\n",
      "INFO\t2017-01-12 16:22:20\tGenotypeConcordance\tchecked    19,050,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 9:120,234,468\n",
      "INFO\t2017-01-12 16:22:20\tGenotypeConcordance\tchecked    19,060,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 9:121,146,490\n",
      "INFO\t2017-01-12 16:22:20\tGenotypeConcordance\tchecked    19,070,000 variants.  Elapsed time: 00:09:26s.  Time for last 10,000:    0s.  Last read position: 9:122,051,697\n",
      "INFO\t2017-01-12 16:22:21\tGenotypeConcordance\tchecked    19,080,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 9:123,018,415\n",
      "INFO\t2017-01-12 16:22:21\tGenotypeConcordance\tchecked    19,090,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 9:124,093,813\n",
      "INFO\t2017-01-12 16:22:21\tGenotypeConcordance\tchecked    19,100,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 9:124,977,324\n",
      "INFO\t2017-01-12 16:22:21\tGenotypeConcordance\tchecked    19,110,000 variants.  Elapsed time: 00:09:27s.  Time for last 10,000:    0s.  Last read position: 9:125,921,216\n",
      "INFO\t2017-01-12 16:22:22\tGenotypeConcordance\tchecked    19,120,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 9:126,913,488\n",
      "INFO\t2017-01-12 16:22:22\tGenotypeConcordance\tchecked    19,130,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 9:127,809,938\n",
      "INFO\t2017-01-12 16:22:22\tGenotypeConcordance\tchecked    19,140,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 9:128,791,885\n",
      "INFO\t2017-01-12 16:22:22\tGenotypeConcordance\tchecked    19,150,000 variants.  Elapsed time: 00:09:28s.  Time for last 10,000:    0s.  Last read position: 9:129,631,774\n",
      "INFO\t2017-01-12 16:22:23\tGenotypeConcordance\tchecked    19,160,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 9:130,680,447\n",
      "INFO\t2017-01-12 16:22:23\tGenotypeConcordance\tchecked    19,170,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 9:131,549,221\n",
      "INFO\t2017-01-12 16:22:23\tGenotypeConcordance\tchecked    19,180,000 variants.  Elapsed time: 00:09:29s.  Time for last 10,000:    0s.  Last read position: 9:132,249,683\n",
      "INFO\t2017-01-12 16:22:23\tGenotypeConcordance\tchecked    19,190,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 9:132,983,263\n",
      "INFO\t2017-01-12 16:22:24\tGenotypeConcordance\tchecked    19,200,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 9:133,887,443\n",
      "INFO\t2017-01-12 16:22:24\tGenotypeConcordance\tchecked    19,210,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 9:134,700,370\n",
      "INFO\t2017-01-12 16:22:24\tGenotypeConcordance\tchecked    19,220,000 variants.  Elapsed time: 00:09:30s.  Time for last 10,000:    0s.  Last read position: 9:135,558,065\n",
      "INFO\t2017-01-12 16:22:25\tGenotypeConcordance\tchecked    19,230,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 9:136,293,373\n",
      "INFO\t2017-01-12 16:22:25\tGenotypeConcordance\tchecked    19,240,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 9:136,949,180\n",
      "INFO\t2017-01-12 16:22:25\tGenotypeConcordance\tchecked    19,250,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 9:137,627,492\n",
      "INFO\t2017-01-12 16:22:25\tGenotypeConcordance\tchecked    19,260,000 variants.  Elapsed time: 00:09:31s.  Time for last 10,000:    0s.  Last read position: 9:138,204,537\n",
      "INFO\t2017-01-12 16:22:26\tGenotypeConcordance\tchecked    19,270,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 9:138,839,570\n",
      "INFO\t2017-01-12 16:22:26\tGenotypeConcordance\tchecked    19,280,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 9:139,567,127\n",
      "INFO\t2017-01-12 16:22:26\tGenotypeConcordance\tchecked    19,290,000 variants.  Elapsed time: 00:09:32s.  Time for last 10,000:    0s.  Last read position: 9:140,296,257\n",
      "INFO\t2017-01-12 16:22:26\tGenotypeConcordance\tchecked    19,300,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 9:141,081,573\n",
      "INFO\t2017-01-12 16:22:27\tGenotypeConcordance\tchecked    19,310,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:725,681\n",
      "INFO\t2017-01-12 16:22:27\tGenotypeConcordance\tchecked    19,320,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:1,415,982\n",
      "INFO\t2017-01-12 16:22:27\tGenotypeConcordance\tchecked    19,330,000 variants.  Elapsed time: 00:09:33s.  Time for last 10,000:    0s.  Last read position: 10:2,048,779\n",
      "INFO\t2017-01-12 16:22:28\tGenotypeConcordance\tchecked    19,340,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:2,669,744\n",
      "INFO\t2017-01-12 16:22:28\tGenotypeConcordance\tchecked    19,350,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:3,237,782\n",
      "INFO\t2017-01-12 16:22:28\tGenotypeConcordance\tchecked    19,360,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:3,972,634\n",
      "INFO\t2017-01-12 16:22:28\tGenotypeConcordance\tchecked    19,370,000 variants.  Elapsed time: 00:09:34s.  Time for last 10,000:    0s.  Last read position: 10:4,660,761\n",
      "INFO\t2017-01-12 16:22:29\tGenotypeConcordance\tchecked    19,380,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:5,265,216\n",
      "INFO\t2017-01-12 16:22:29\tGenotypeConcordance\tchecked    19,390,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:5,931,011\n",
      "INFO\t2017-01-12 16:22:29\tGenotypeConcordance\tchecked    19,400,000 variants.  Elapsed time: 00:09:35s.  Time for last 10,000:    0s.  Last read position: 10:6,645,899\n",
      "INFO\t2017-01-12 16:22:30\tGenotypeConcordance\tchecked    19,410,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:7,286,305\n",
      "INFO\t2017-01-12 16:22:30\tGenotypeConcordance\tchecked    19,420,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:8,018,107\n",
      "INFO\t2017-01-12 16:22:30\tGenotypeConcordance\tchecked    19,430,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:8,728,517\n",
      "INFO\t2017-01-12 16:22:30\tGenotypeConcordance\tchecked    19,440,000 variants.  Elapsed time: 00:09:36s.  Time for last 10,000:    0s.  Last read position: 10:9,561,105\n",
      "INFO\t2017-01-12 16:22:31\tGenotypeConcordance\tchecked    19,450,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:10,380,004\n",
      "INFO\t2017-01-12 16:22:31\tGenotypeConcordance\tchecked    19,460,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:11,181,950\n",
      "INFO\t2017-01-12 16:22:31\tGenotypeConcordance\tchecked    19,470,000 variants.  Elapsed time: 00:09:37s.  Time for last 10,000:    0s.  Last read position: 10:11,933,218\n",
      "INFO\t2017-01-12 16:22:31\tGenotypeConcordance\tchecked    19,480,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:12,616,887\n",
      "INFO\t2017-01-12 16:22:32\tGenotypeConcordance\tchecked    19,490,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:13,283,711\n",
      "INFO\t2017-01-12 16:22:32\tGenotypeConcordance\tchecked    19,500,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:13,966,471\n",
      "INFO\t2017-01-12 16:22:32\tGenotypeConcordance\tchecked    19,510,000 variants.  Elapsed time: 00:09:38s.  Time for last 10,000:    0s.  Last read position: 10:14,669,633\n",
      "INFO\t2017-01-12 16:22:33\tGenotypeConcordance\tchecked    19,520,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:15,373,886\n",
      "INFO\t2017-01-12 16:22:33\tGenotypeConcordance\tchecked    19,530,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:16,116,605\n",
      "INFO\t2017-01-12 16:22:33\tGenotypeConcordance\tchecked    19,540,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:16,809,597\n",
      "INFO\t2017-01-12 16:22:33\tGenotypeConcordance\tchecked    19,550,000 variants.  Elapsed time: 00:09:39s.  Time for last 10,000:    0s.  Last read position: 10:17,531,922\n",
      "INFO\t2017-01-12 16:22:34\tGenotypeConcordance\tchecked    19,560,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:18,586,608\n",
      "INFO\t2017-01-12 16:22:34\tGenotypeConcordance\tchecked    19,570,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:19,287,256\n",
      "INFO\t2017-01-12 16:22:34\tGenotypeConcordance\tchecked    19,580,000 variants.  Elapsed time: 00:09:40s.  Time for last 10,000:    0s.  Last read position: 10:19,941,090\n",
      "INFO\t2017-01-12 16:22:34\tGenotypeConcordance\tchecked    19,590,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:20,677,088\n",
      "INFO\t2017-01-12 16:22:35\tGenotypeConcordance\tchecked    19,600,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:21,386,755\n",
      "INFO\t2017-01-12 16:22:35\tGenotypeConcordance\tchecked    19,610,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:22,397,086\n",
      "INFO\t2017-01-12 16:22:35\tGenotypeConcordance\tchecked    19,620,000 variants.  Elapsed time: 00:09:41s.  Time for last 10,000:    0s.  Last read position: 10:23,293,921\n",
      "INFO\t2017-01-12 16:22:35\tGenotypeConcordance\tchecked    19,630,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:24,079,364\n",
      "INFO\t2017-01-12 16:22:36\tGenotypeConcordance\tchecked    19,640,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:24,880,195\n",
      "INFO\t2017-01-12 16:22:36\tGenotypeConcordance\tchecked    19,650,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:25,728,766\n",
      "INFO\t2017-01-12 16:22:36\tGenotypeConcordance\tchecked    19,660,000 variants.  Elapsed time: 00:09:42s.  Time for last 10,000:    0s.  Last read position: 10:26,497,157\n",
      "INFO\t2017-01-12 16:22:37\tGenotypeConcordance\tchecked    19,670,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:27,176,737\n",
      "INFO\t2017-01-12 16:22:37\tGenotypeConcordance\tchecked    19,680,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:27,899,408\n",
      "INFO\t2017-01-12 16:22:37\tGenotypeConcordance\tchecked    19,690,000 variants.  Elapsed time: 00:09:43s.  Time for last 10,000:    0s.  Last read position: 10:28,727,624\n",
      "INFO\t2017-01-12 16:22:37\tGenotypeConcordance\tchecked    19,700,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:29,464,255\n",
      "INFO\t2017-01-12 16:22:38\tGenotypeConcordance\tchecked    19,710,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:30,175,131\n",
      "INFO\t2017-01-12 16:22:38\tGenotypeConcordance\tchecked    19,720,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:30,892,831\n",
      "INFO\t2017-01-12 16:22:38\tGenotypeConcordance\tchecked    19,730,000 variants.  Elapsed time: 00:09:44s.  Time for last 10,000:    0s.  Last read position: 10:31,680,578\n",
      "INFO\t2017-01-12 16:22:39\tGenotypeConcordance\tchecked    19,740,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:32,538,430\n",
      "INFO\t2017-01-12 16:22:39\tGenotypeConcordance\tchecked    19,750,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:33,387,550\n",
      "INFO\t2017-01-12 16:22:39\tGenotypeConcordance\tchecked    19,760,000 variants.  Elapsed time: 00:09:45s.  Time for last 10,000:    0s.  Last read position: 10:34,202,005\n",
      "INFO\t2017-01-12 16:22:39\tGenotypeConcordance\tchecked    19,770,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:35,061,590\n",
      "INFO\t2017-01-12 16:22:40\tGenotypeConcordance\tchecked    19,780,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:35,984,031\n",
      "INFO\t2017-01-12 16:22:40\tGenotypeConcordance\tchecked    19,790,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:36,784,026\n",
      "INFO\t2017-01-12 16:22:40\tGenotypeConcordance\tchecked    19,800,000 variants.  Elapsed time: 00:09:46s.  Time for last 10,000:    0s.  Last read position: 10:37,586,233\n",
      "INFO\t2017-01-12 16:22:40\tGenotypeConcordance\tchecked    19,810,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:38,474,106\n",
      "INFO\t2017-01-12 16:22:41\tGenotypeConcordance\tchecked    19,820,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:38,982,460\n",
      "INFO\t2017-01-12 16:22:41\tGenotypeConcordance\tchecked    19,830,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:42,383,853\n",
      "INFO\t2017-01-12 16:22:41\tGenotypeConcordance\tchecked    19,840,000 variants.  Elapsed time: 00:09:47s.  Time for last 10,000:    0s.  Last read position: 10:42,630,572\n",
      "INFO\t2017-01-12 16:22:42\tGenotypeConcordance\tchecked    19,850,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:43,024,580\n",
      "INFO\t2017-01-12 16:22:42\tGenotypeConcordance\tchecked    19,860,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:43,820,851\n",
      "INFO\t2017-01-12 16:22:42\tGenotypeConcordance\tchecked    19,870,000 variants.  Elapsed time: 00:09:48s.  Time for last 10,000:    0s.  Last read position: 10:44,616,224\n",
      "INFO\t2017-01-12 16:22:43\tGenotypeConcordance\tchecked    19,880,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:45,375,110\n",
      "INFO\t2017-01-12 16:22:43\tGenotypeConcordance\tchecked    19,890,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:46,186,800\n",
      "INFO\t2017-01-12 16:22:43\tGenotypeConcordance\tchecked    19,900,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:47,698,150\n",
      "INFO\t2017-01-12 16:22:43\tGenotypeConcordance\tchecked    19,910,000 variants.  Elapsed time: 00:09:49s.  Time for last 10,000:    0s.  Last read position: 10:49,572,482\n",
      "INFO\t2017-01-12 16:22:44\tGenotypeConcordance\tchecked    19,920,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:50,385,880\n",
      "INFO\t2017-01-12 16:22:44\tGenotypeConcordance\tchecked    19,930,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:51,504,074\n",
      "INFO\t2017-01-12 16:22:44\tGenotypeConcordance\tchecked    19,940,000 variants.  Elapsed time: 00:09:50s.  Time for last 10,000:    0s.  Last read position: 10:52,590,708\n",
      "INFO\t2017-01-12 16:22:45\tGenotypeConcordance\tchecked    19,950,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:53,444,658\n",
      "INFO\t2017-01-12 16:22:45\tGenotypeConcordance\tchecked    19,960,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:54,219,820\n",
      "INFO\t2017-01-12 16:22:45\tGenotypeConcordance\tchecked    19,970,000 variants.  Elapsed time: 00:09:51s.  Time for last 10,000:    0s.  Last read position: 10:55,000,909\n",
      "INFO\t2017-01-12 16:22:45\tGenotypeConcordance\tchecked    19,980,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:55,728,299\n",
      "INFO\t2017-01-12 16:22:46\tGenotypeConcordance\tchecked    19,990,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:56,417,474\n",
      "INFO\t2017-01-12 16:22:46\tGenotypeConcordance\tchecked    20,000,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:57,209,676\n",
      "INFO\t2017-01-12 16:22:46\tGenotypeConcordance\tchecked    20,010,000 variants.  Elapsed time: 00:09:52s.  Time for last 10,000:    0s.  Last read position: 10:58,005,755\n",
      "INFO\t2017-01-12 16:22:47\tGenotypeConcordance\tchecked    20,020,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 10:58,784,814\n",
      "INFO\t2017-01-12 16:22:47\tGenotypeConcordance\tchecked    20,030,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 10:59,483,570\n",
      "INFO\t2017-01-12 16:22:47\tGenotypeConcordance\tchecked    20,040,000 variants.  Elapsed time: 00:09:53s.  Time for last 10,000:    0s.  Last read position: 10:60,314,183\n",
      "INFO\t2017-01-12 16:22:47\tGenotypeConcordance\tchecked    20,050,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 10:61,311,243\n",
      "INFO\t2017-01-12 16:22:48\tGenotypeConcordance\tchecked    20,060,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 10:62,136,212\n",
      "INFO\t2017-01-12 16:22:48\tGenotypeConcordance\tchecked    20,070,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 10:63,002,267\n",
      "INFO\t2017-01-12 16:22:48\tGenotypeConcordance\tchecked    20,080,000 variants.  Elapsed time: 00:09:54s.  Time for last 10,000:    0s.  Last read position: 10:63,918,915\n",
      "INFO\t2017-01-12 16:22:49\tGenotypeConcordance\tchecked    20,090,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 10:64,810,727\n",
      "INFO\t2017-01-12 16:22:49\tGenotypeConcordance\tchecked    20,100,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 10:65,743,503\n",
      "INFO\t2017-01-12 16:22:49\tGenotypeConcordance\tchecked    20,110,000 variants.  Elapsed time: 00:09:55s.  Time for last 10,000:    0s.  Last read position: 10:66,613,822\n",
      "INFO\t2017-01-12 16:22:50\tGenotypeConcordance\tchecked    20,120,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 10:67,321,688\n",
      "INFO\t2017-01-12 16:22:50\tGenotypeConcordance\tchecked    20,130,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 10:68,085,353\n",
      "INFO\t2017-01-12 16:22:50\tGenotypeConcordance\tchecked    20,140,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 10:68,788,942\n",
      "INFO\t2017-01-12 16:22:50\tGenotypeConcordance\tchecked    20,150,000 variants.  Elapsed time: 00:09:56s.  Time for last 10,000:    0s.  Last read position: 10:69,661,777\n",
      "INFO\t2017-01-12 16:22:51\tGenotypeConcordance\tchecked    20,160,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 10:70,491,316\n",
      "INFO\t2017-01-12 16:22:51\tGenotypeConcordance\tchecked    20,170,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 10:71,268,224\n",
      "INFO\t2017-01-12 16:22:51\tGenotypeConcordance\tchecked    20,180,000 variants.  Elapsed time: 00:09:57s.  Time for last 10,000:    0s.  Last read position: 10:72,058,768\n",
      "INFO\t2017-01-12 16:22:52\tGenotypeConcordance\tchecked    20,190,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 10:72,788,149\n",
      "INFO\t2017-01-12 16:22:52\tGenotypeConcordance\tchecked    20,200,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 10:73,578,704\n",
      "INFO\t2017-01-12 16:22:52\tGenotypeConcordance\tchecked    20,210,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 10:74,476,968\n",
      "INFO\t2017-01-12 16:22:52\tGenotypeConcordance\tchecked    20,220,000 variants.  Elapsed time: 00:09:58s.  Time for last 10,000:    0s.  Last read position: 10:75,568,839\n",
      "INFO\t2017-01-12 16:22:53\tGenotypeConcordance\tchecked    20,230,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 10:76,679,726\n",
      "INFO\t2017-01-12 16:22:53\tGenotypeConcordance\tchecked    20,240,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 10:77,737,701\n",
      "INFO\t2017-01-12 16:22:53\tGenotypeConcordance\tchecked    20,250,000 variants.  Elapsed time: 00:09:59s.  Time for last 10,000:    0s.  Last read position: 10:78,671,629\n",
      "INFO\t2017-01-12 16:22:53\tGenotypeConcordance\tchecked    20,260,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 10:79,600,256\n",
      "INFO\t2017-01-12 16:22:54\tGenotypeConcordance\tchecked    20,270,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 10:80,410,447\n",
      "INFO\t2017-01-12 16:22:54\tGenotypeConcordance\tchecked    20,280,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 10:81,229,468\n",
      "INFO\t2017-01-12 16:22:54\tGenotypeConcordance\tchecked    20,290,000 variants.  Elapsed time: 00:10:00s.  Time for last 10,000:    0s.  Last read position: 10:81,924,530\n",
      "INFO\t2017-01-12 16:22:55\tGenotypeConcordance\tchecked    20,300,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 10:82,720,794\n",
      "INFO\t2017-01-12 16:22:55\tGenotypeConcordance\tchecked    20,310,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 10:83,496,501\n",
      "INFO\t2017-01-12 16:22:55\tGenotypeConcordance\tchecked    20,320,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 10:84,358,581\n",
      "INFO\t2017-01-12 16:22:55\tGenotypeConcordance\tchecked    20,330,000 variants.  Elapsed time: 00:10:01s.  Time for last 10,000:    0s.  Last read position: 10:85,176,752\n",
      "INFO\t2017-01-12 16:22:56\tGenotypeConcordance\tchecked    20,340,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 10:85,947,354\n",
      "INFO\t2017-01-12 16:22:56\tGenotypeConcordance\tchecked    20,350,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 10:86,800,415\n",
      "INFO\t2017-01-12 16:22:56\tGenotypeConcordance\tchecked    20,360,000 variants.  Elapsed time: 00:10:02s.  Time for last 10,000:    0s.  Last read position: 10:87,535,013\n",
      "INFO\t2017-01-12 16:22:57\tGenotypeConcordance\tchecked    20,370,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 10:88,409,087\n",
      "INFO\t2017-01-12 16:22:57\tGenotypeConcordance\tchecked    20,380,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 10:89,361,512\n",
      "INFO\t2017-01-12 16:22:57\tGenotypeConcordance\tchecked    20,390,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 10:90,361,234\n",
      "INFO\t2017-01-12 16:22:57\tGenotypeConcordance\tchecked    20,400,000 variants.  Elapsed time: 00:10:03s.  Time for last 10,000:    0s.  Last read position: 10:91,291,104\n",
      "INFO\t2017-01-12 16:22:58\tGenotypeConcordance\tchecked    20,410,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 10:92,100,943\n",
      "INFO\t2017-01-12 16:22:58\tGenotypeConcordance\tchecked    20,420,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 10:92,980,926\n",
      "INFO\t2017-01-12 16:22:58\tGenotypeConcordance\tchecked    20,430,000 variants.  Elapsed time: 00:10:04s.  Time for last 10,000:    0s.  Last read position: 10:94,056,078\n",
      "INFO\t2017-01-12 16:22:58\tGenotypeConcordance\tchecked    20,440,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 10:95,024,704\n",
      "INFO\t2017-01-12 16:22:59\tGenotypeConcordance\tchecked    20,450,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 10:95,859,119\n",
      "INFO\t2017-01-12 16:22:59\tGenotypeConcordance\tchecked    20,460,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 10:96,752,130\n",
      "INFO\t2017-01-12 16:22:59\tGenotypeConcordance\tchecked    20,470,000 variants.  Elapsed time: 00:10:05s.  Time for last 10,000:    0s.  Last read position: 10:97,720,824\n",
      "INFO\t2017-01-12 16:23:00\tGenotypeConcordance\tchecked    20,480,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 10:98,703,175\n",
      "INFO\t2017-01-12 16:23:00\tGenotypeConcordance\tchecked    20,490,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 10:99,636,526\n",
      "INFO\t2017-01-12 16:23:00\tGenotypeConcordance\tchecked    20,500,000 variants.  Elapsed time: 00:10:06s.  Time for last 10,000:    0s.  Last read position: 10:100,526,502\n",
      "INFO\t2017-01-12 16:23:00\tGenotypeConcordance\tchecked    20,510,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 10:101,550,981\n",
      "INFO\t2017-01-12 16:23:01\tGenotypeConcordance\tchecked    20,520,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 10:102,454,375\n",
      "INFO\t2017-01-12 16:23:01\tGenotypeConcordance\tchecked    20,530,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 10:103,536,044\n",
      "INFO\t2017-01-12 16:23:01\tGenotypeConcordance\tchecked    20,540,000 variants.  Elapsed time: 00:10:07s.  Time for last 10,000:    0s.  Last read position: 10:104,765,420\n",
      "INFO\t2017-01-12 16:23:02\tGenotypeConcordance\tchecked    20,550,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 10:105,816,331\n",
      "INFO\t2017-01-12 16:23:02\tGenotypeConcordance\tchecked    20,560,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 10:106,773,878\n",
      "INFO\t2017-01-12 16:23:02\tGenotypeConcordance\tchecked    20,570,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 10:107,683,576\n",
      "INFO\t2017-01-12 16:23:02\tGenotypeConcordance\tchecked    20,580,000 variants.  Elapsed time: 00:10:08s.  Time for last 10,000:    0s.  Last read position: 10:108,541,839\n",
      "INFO\t2017-01-12 16:23:03\tGenotypeConcordance\tchecked    20,590,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 10:109,395,501\n",
      "INFO\t2017-01-12 16:23:03\tGenotypeConcordance\tchecked    20,600,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 10:110,150,313\n",
      "INFO\t2017-01-12 16:23:03\tGenotypeConcordance\tchecked    20,610,000 variants.  Elapsed time: 00:10:09s.  Time for last 10,000:    0s.  Last read position: 10:110,892,470\n",
      "INFO\t2017-01-12 16:23:04\tGenotypeConcordance\tchecked    20,620,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 10:111,688,998\n",
      "INFO\t2017-01-12 16:23:04\tGenotypeConcordance\tchecked    20,630,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 10:112,566,850\n",
      "INFO\t2017-01-12 16:23:04\tGenotypeConcordance\tchecked    20,640,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 10:113,370,069\n",
      "INFO\t2017-01-12 16:23:04\tGenotypeConcordance\tchecked    20,650,000 variants.  Elapsed time: 00:10:10s.  Time for last 10,000:    0s.  Last read position: 10:114,153,072\n",
      "INFO\t2017-01-12 16:23:05\tGenotypeConcordance\tchecked    20,660,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 10:115,110,707\n",
      "INFO\t2017-01-12 16:23:05\tGenotypeConcordance\tchecked    20,670,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 10:115,958,189\n",
      "INFO\t2017-01-12 16:23:05\tGenotypeConcordance\tchecked    20,680,000 variants.  Elapsed time: 00:10:11s.  Time for last 10,000:    0s.  Last read position: 10:116,837,002\n",
      "INFO\t2017-01-12 16:23:05\tGenotypeConcordance\tchecked    20,690,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 10:117,678,091\n",
      "INFO\t2017-01-12 16:23:06\tGenotypeConcordance\tchecked    20,700,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 10:118,540,025\n",
      "INFO\t2017-01-12 16:23:06\tGenotypeConcordance\tchecked    20,710,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 10:119,483,071\n",
      "INFO\t2017-01-12 16:23:06\tGenotypeConcordance\tchecked    20,720,000 variants.  Elapsed time: 00:10:12s.  Time for last 10,000:    0s.  Last read position: 10:120,415,610\n",
      "INFO\t2017-01-12 16:23:07\tGenotypeConcordance\tchecked    20,730,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 10:121,225,680\n",
      "INFO\t2017-01-12 16:23:07\tGenotypeConcordance\tchecked    20,740,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 10:122,033,585\n",
      "INFO\t2017-01-12 16:23:07\tGenotypeConcordance\tchecked    20,750,000 variants.  Elapsed time: 00:10:13s.  Time for last 10,000:    0s.  Last read position: 10:122,813,199\n",
      "INFO\t2017-01-12 16:23:07\tGenotypeConcordance\tchecked    20,760,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 10:123,582,994\n",
      "INFO\t2017-01-12 16:23:08\tGenotypeConcordance\tchecked    20,770,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 10:124,379,794\n",
      "INFO\t2017-01-12 16:23:08\tGenotypeConcordance\tchecked    20,780,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 10:125,264,951\n",
      "INFO\t2017-01-12 16:23:08\tGenotypeConcordance\tchecked    20,790,000 variants.  Elapsed time: 00:10:14s.  Time for last 10,000:    0s.  Last read position: 10:126,042,780\n",
      "INFO\t2017-01-12 16:23:09\tGenotypeConcordance\tchecked    20,800,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 10:126,795,646\n",
      "INFO\t2017-01-12 16:23:09\tGenotypeConcordance\tchecked    20,810,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 10:127,594,582\n",
      "INFO\t2017-01-12 16:23:09\tGenotypeConcordance\tchecked    20,820,000 variants.  Elapsed time: 00:10:15s.  Time for last 10,000:    0s.  Last read position: 10:128,338,095\n",
      "INFO\t2017-01-12 16:23:09\tGenotypeConcordance\tchecked    20,830,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 10:129,233,829\n",
      "INFO\t2017-01-12 16:23:10\tGenotypeConcordance\tchecked    20,840,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 10:129,926,565\n",
      "INFO\t2017-01-12 16:23:10\tGenotypeConcordance\tchecked    20,850,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 10:130,678,444\n",
      "INFO\t2017-01-12 16:23:10\tGenotypeConcordance\tchecked    20,860,000 variants.  Elapsed time: 00:10:16s.  Time for last 10,000:    0s.  Last read position: 10:131,414,478\n",
      "INFO\t2017-01-12 16:23:11\tGenotypeConcordance\tchecked    20,870,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 10:132,193,733\n",
      "INFO\t2017-01-12 16:23:11\tGenotypeConcordance\tchecked    20,880,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 10:132,832,275\n",
      "INFO\t2017-01-12 16:23:11\tGenotypeConcordance\tchecked    20,890,000 variants.  Elapsed time: 00:10:17s.  Time for last 10,000:    0s.  Last read position: 10:133,508,735\n",
      "INFO\t2017-01-12 16:23:11\tGenotypeConcordance\tchecked    20,900,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 10:134,177,217\n",
      "INFO\t2017-01-12 16:23:12\tGenotypeConcordance\tchecked    20,910,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 10:134,818,963\n",
      "INFO\t2017-01-12 16:23:12\tGenotypeConcordance\tchecked    20,920,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 10:135,444,061\n",
      "INFO\t2017-01-12 16:23:12\tGenotypeConcordance\tchecked    20,930,000 variants.  Elapsed time: 00:10:18s.  Time for last 10,000:    0s.  Last read position: 11:624,995\n",
      "INFO\t2017-01-12 16:23:13\tGenotypeConcordance\tchecked    20,940,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:1,344,447\n",
      "INFO\t2017-01-12 16:23:13\tGenotypeConcordance\tchecked    20,950,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:2,026,711\n",
      "INFO\t2017-01-12 16:23:13\tGenotypeConcordance\tchecked    20,960,000 variants.  Elapsed time: 00:10:19s.  Time for last 10,000:    0s.  Last read position: 11:2,748,347\n",
      "INFO\t2017-01-12 16:23:13\tGenotypeConcordance\tchecked    20,970,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:3,467,305\n",
      "INFO\t2017-01-12 16:23:14\tGenotypeConcordance\tchecked    20,980,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:4,122,736\n",
      "INFO\t2017-01-12 16:23:14\tGenotypeConcordance\tchecked    20,990,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:4,587,367\n",
      "INFO\t2017-01-12 16:23:14\tGenotypeConcordance\tchecked    21,000,000 variants.  Elapsed time: 00:10:20s.  Time for last 10,000:    0s.  Last read position: 11:5,234,024\n",
      "INFO\t2017-01-12 16:23:15\tGenotypeConcordance\tchecked    21,010,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:5,826,843\n",
      "INFO\t2017-01-12 16:23:15\tGenotypeConcordance\tchecked    21,020,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:6,615,123\n",
      "INFO\t2017-01-12 16:23:15\tGenotypeConcordance\tchecked    21,030,000 variants.  Elapsed time: 00:10:21s.  Time for last 10,000:    0s.  Last read position: 11:7,390,425\n",
      "INFO\t2017-01-12 16:23:16\tGenotypeConcordance\tchecked    21,040,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:8,045,050\n",
      "INFO\t2017-01-12 16:23:16\tGenotypeConcordance\tchecked    21,050,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:8,861,669\n",
      "INFO\t2017-01-12 16:23:16\tGenotypeConcordance\tchecked    21,060,000 variants.  Elapsed time: 00:10:22s.  Time for last 10,000:    0s.  Last read position: 11:9,729,626\n",
      "INFO\t2017-01-12 16:23:16\tGenotypeConcordance\tchecked    21,070,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:10,567,802\n",
      "INFO\t2017-01-12 16:23:17\tGenotypeConcordance\tchecked    21,080,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:11,327,295\n",
      "INFO\t2017-01-12 16:23:17\tGenotypeConcordance\tchecked    21,090,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:12,127,181\n",
      "INFO\t2017-01-12 16:23:17\tGenotypeConcordance\tchecked    21,100,000 variants.  Elapsed time: 00:10:23s.  Time for last 10,000:    0s.  Last read position: 11:12,931,825\n",
      "INFO\t2017-01-12 16:23:18\tGenotypeConcordance\tchecked    21,110,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:13,771,997\n",
      "INFO\t2017-01-12 16:23:18\tGenotypeConcordance\tchecked    21,120,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:14,772,334\n",
      "INFO\t2017-01-12 16:23:18\tGenotypeConcordance\tchecked    21,130,000 variants.  Elapsed time: 00:10:24s.  Time for last 10,000:    0s.  Last read position: 11:15,733,176\n",
      "INFO\t2017-01-12 16:23:18\tGenotypeConcordance\tchecked    21,140,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:16,814,479\n",
      "INFO\t2017-01-12 16:23:19\tGenotypeConcordance\tchecked    21,150,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:17,679,578\n",
      "INFO\t2017-01-12 16:23:19\tGenotypeConcordance\tchecked    21,160,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:18,604,396\n",
      "INFO\t2017-01-12 16:23:19\tGenotypeConcordance\tchecked    21,170,000 variants.  Elapsed time: 00:10:25s.  Time for last 10,000:    0s.  Last read position: 11:19,418,971\n",
      "INFO\t2017-01-12 16:23:20\tGenotypeConcordance\tchecked    21,180,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:20,193,304\n",
      "INFO\t2017-01-12 16:23:20\tGenotypeConcordance\tchecked    21,190,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:20,971,402\n",
      "INFO\t2017-01-12 16:23:20\tGenotypeConcordance\tchecked    21,200,000 variants.  Elapsed time: 00:10:26s.  Time for last 10,000:    0s.  Last read position: 11:21,694,637\n",
      "INFO\t2017-01-12 16:23:20\tGenotypeConcordance\tchecked    21,210,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:22,426,734\n",
      "INFO\t2017-01-12 16:23:21\tGenotypeConcordance\tchecked    21,220,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:23,165,535\n",
      "INFO\t2017-01-12 16:23:21\tGenotypeConcordance\tchecked    21,230,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:23,854,110\n",
      "INFO\t2017-01-12 16:23:21\tGenotypeConcordance\tchecked    21,240,000 variants.  Elapsed time: 00:10:27s.  Time for last 10,000:    0s.  Last read position: 11:24,528,308\n",
      "INFO\t2017-01-12 16:23:22\tGenotypeConcordance\tchecked    21,250,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:25,194,744\n",
      "INFO\t2017-01-12 16:23:22\tGenotypeConcordance\tchecked    21,260,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:25,850,138\n",
      "INFO\t2017-01-12 16:23:22\tGenotypeConcordance\tchecked    21,270,000 variants.  Elapsed time: 00:10:28s.  Time for last 10,000:    0s.  Last read position: 11:26,530,867\n",
      "INFO\t2017-01-12 16:23:23\tGenotypeConcordance\tchecked    21,280,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:27,426,514\n",
      "INFO\t2017-01-12 16:23:23\tGenotypeConcordance\tchecked    21,290,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:28,639,341\n",
      "INFO\t2017-01-12 16:23:23\tGenotypeConcordance\tchecked    21,300,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:29,531,037\n",
      "INFO\t2017-01-12 16:23:23\tGenotypeConcordance\tchecked    21,310,000 variants.  Elapsed time: 00:10:29s.  Time for last 10,000:    0s.  Last read position: 11:30,400,212\n",
      "INFO\t2017-01-12 16:23:24\tGenotypeConcordance\tchecked    21,320,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:31,531,318\n",
      "INFO\t2017-01-12 16:23:24\tGenotypeConcordance\tchecked    21,330,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:32,485,030\n",
      "INFO\t2017-01-12 16:23:24\tGenotypeConcordance\tchecked    21,340,000 variants.  Elapsed time: 00:10:30s.  Time for last 10,000:    0s.  Last read position: 11:33,451,507\n",
      "INFO\t2017-01-12 16:23:24\tGenotypeConcordance\tchecked    21,350,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:34,306,865\n",
      "INFO\t2017-01-12 16:23:25\tGenotypeConcordance\tchecked    21,360,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:35,143,574\n",
      "INFO\t2017-01-12 16:23:25\tGenotypeConcordance\tchecked    21,370,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:36,008,210\n",
      "INFO\t2017-01-12 16:23:25\tGenotypeConcordance\tchecked    21,380,000 variants.  Elapsed time: 00:10:31s.  Time for last 10,000:    0s.  Last read position: 11:36,806,048\n",
      "INFO\t2017-01-12 16:23:26\tGenotypeConcordance\tchecked    21,390,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:37,632,706\n",
      "INFO\t2017-01-12 16:23:26\tGenotypeConcordance\tchecked    21,400,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:38,411,042\n",
      "INFO\t2017-01-12 16:23:26\tGenotypeConcordance\tchecked    21,410,000 variants.  Elapsed time: 00:10:32s.  Time for last 10,000:    0s.  Last read position: 11:39,225,437\n",
      "INFO\t2017-01-12 16:23:26\tGenotypeConcordance\tchecked    21,420,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:39,957,989\n",
      "INFO\t2017-01-12 16:23:27\tGenotypeConcordance\tchecked    21,430,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:40,812,047\n",
      "INFO\t2017-01-12 16:23:27\tGenotypeConcordance\tchecked    21,440,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:41,734,059\n",
      "INFO\t2017-01-12 16:23:27\tGenotypeConcordance\tchecked    21,450,000 variants.  Elapsed time: 00:10:33s.  Time for last 10,000:    0s.  Last read position: 11:42,461,347\n",
      "INFO\t2017-01-12 16:23:28\tGenotypeConcordance\tchecked    21,460,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:43,167,637\n",
      "INFO\t2017-01-12 16:23:28\tGenotypeConcordance\tchecked    21,470,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:44,075,440\n",
      "INFO\t2017-01-12 16:23:28\tGenotypeConcordance\tchecked    21,480,000 variants.  Elapsed time: 00:10:34s.  Time for last 10,000:    0s.  Last read position: 11:44,906,045\n",
      "INFO\t2017-01-12 16:23:29\tGenotypeConcordance\tchecked    21,490,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:45,781,007\n",
      "INFO\t2017-01-12 16:23:29\tGenotypeConcordance\tchecked    21,500,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:46,960,059\n",
      "INFO\t2017-01-12 16:23:29\tGenotypeConcordance\tchecked    21,510,000 variants.  Elapsed time: 00:10:35s.  Time for last 10,000:    0s.  Last read position: 11:48,082,064\n",
      "INFO\t2017-01-12 16:23:29\tGenotypeConcordance\tchecked    21,520,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:48,625,640\n",
      "INFO\t2017-01-12 16:23:30\tGenotypeConcordance\tchecked    21,530,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:49,199,515\n",
      "INFO\t2017-01-12 16:23:30\tGenotypeConcordance\tchecked    21,540,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:50,022,940\n",
      "INFO\t2017-01-12 16:23:30\tGenotypeConcordance\tchecked    21,550,000 variants.  Elapsed time: 00:10:36s.  Time for last 10,000:    0s.  Last read position: 11:50,640,670\n",
      "INFO\t2017-01-12 16:23:31\tGenotypeConcordance\tchecked    21,560,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:51,409,683\n",
      "INFO\t2017-01-12 16:23:31\tGenotypeConcordance\tchecked    21,570,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:54,978,599\n",
      "INFO\t2017-01-12 16:23:31\tGenotypeConcordance\tchecked    21,580,000 variants.  Elapsed time: 00:10:37s.  Time for last 10,000:    0s.  Last read position: 11:55,451,187\n",
      "INFO\t2017-01-12 16:23:32\tGenotypeConcordance\tchecked    21,590,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:56,124,183\n",
      "INFO\t2017-01-12 16:23:32\tGenotypeConcordance\tchecked    21,600,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:56,844,128\n",
      "INFO\t2017-01-12 16:23:32\tGenotypeConcordance\tchecked    21,610,000 variants.  Elapsed time: 00:10:38s.  Time for last 10,000:    0s.  Last read position: 11:57,873,300\n",
      "INFO\t2017-01-12 16:23:33\tGenotypeConcordance\tchecked    21,620,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 11:58,771,287\n",
      "INFO\t2017-01-12 16:23:33\tGenotypeConcordance\tchecked    21,630,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 11:59,659,528\n",
      "INFO\t2017-01-12 16:23:33\tGenotypeConcordance\tchecked    21,640,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 11:60,481,211\n",
      "INFO\t2017-01-12 16:23:33\tGenotypeConcordance\tchecked    21,650,000 variants.  Elapsed time: 00:10:39s.  Time for last 10,000:    0s.  Last read position: 11:61,456,576\n",
      "INFO\t2017-01-12 16:23:34\tGenotypeConcordance\tchecked    21,660,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 11:62,329,292\n",
      "INFO\t2017-01-12 16:23:34\tGenotypeConcordance\tchecked    21,670,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 11:63,182,615\n",
      "INFO\t2017-01-12 16:23:34\tGenotypeConcordance\tchecked    21,680,000 variants.  Elapsed time: 00:10:40s.  Time for last 10,000:    0s.  Last read position: 11:64,147,001\n",
      "INFO\t2017-01-12 16:23:35\tGenotypeConcordance\tchecked    21,690,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 11:65,167,904\n",
      "INFO\t2017-01-12 16:23:35\tGenotypeConcordance\tchecked    21,700,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 11:66,202,257\n",
      "INFO\t2017-01-12 16:23:35\tGenotypeConcordance\tchecked    21,710,000 variants.  Elapsed time: 00:10:41s.  Time for last 10,000:    0s.  Last read position: 11:67,375,162\n",
      "INFO\t2017-01-12 16:23:35\tGenotypeConcordance\tchecked    21,720,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 11:67,977,596\n",
      "INFO\t2017-01-12 16:23:36\tGenotypeConcordance\tchecked    21,730,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 11:68,825,475\n",
      "INFO\t2017-01-12 16:23:36\tGenotypeConcordance\tchecked    21,740,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 11:69,611,475\n",
      "INFO\t2017-01-12 16:23:36\tGenotypeConcordance\tchecked    21,750,000 variants.  Elapsed time: 00:10:42s.  Time for last 10,000:    0s.  Last read position: 11:70,426,352\n",
      "INFO\t2017-01-12 16:23:37\tGenotypeConcordance\tchecked    21,760,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 11:71,175,220\n",
      "INFO\t2017-01-12 16:23:37\tGenotypeConcordance\tchecked    21,770,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 11:71,772,465\n",
      "INFO\t2017-01-12 16:23:37\tGenotypeConcordance\tchecked    21,780,000 variants.  Elapsed time: 00:10:43s.  Time for last 10,000:    0s.  Last read position: 11:72,817,255\n",
      "INFO\t2017-01-12 16:23:38\tGenotypeConcordance\tchecked    21,790,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 11:73,735,375\n",
      "INFO\t2017-01-12 16:23:38\tGenotypeConcordance\tchecked    21,800,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 11:74,681,953\n",
      "INFO\t2017-01-12 16:23:38\tGenotypeConcordance\tchecked    21,810,000 variants.  Elapsed time: 00:10:44s.  Time for last 10,000:    0s.  Last read position: 11:75,672,886\n",
      "INFO\t2017-01-12 16:23:38\tGenotypeConcordance\tchecked    21,820,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 11:76,541,459\n",
      "INFO\t2017-01-12 16:23:39\tGenotypeConcordance\tchecked    21,830,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 11:77,481,797\n",
      "INFO\t2017-01-12 16:23:39\tGenotypeConcordance\tchecked    21,840,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 11:78,304,659\n",
      "INFO\t2017-01-12 16:23:39\tGenotypeConcordance\tchecked    21,850,000 variants.  Elapsed time: 00:10:45s.  Time for last 10,000:    0s.  Last read position: 11:79,144,077\n",
      "INFO\t2017-01-12 16:23:40\tGenotypeConcordance\tchecked    21,860,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 11:79,970,467\n",
      "INFO\t2017-01-12 16:23:40\tGenotypeConcordance\tchecked    21,870,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 11:80,775,922\n",
      "INFO\t2017-01-12 16:23:40\tGenotypeConcordance\tchecked    21,880,000 variants.  Elapsed time: 00:10:46s.  Time for last 10,000:    0s.  Last read position: 11:81,487,576\n",
      "INFO\t2017-01-12 16:23:41\tGenotypeConcordance\tchecked    21,890,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 11:82,314,055\n",
      "INFO\t2017-01-12 16:23:41\tGenotypeConcordance\tchecked    21,900,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 11:83,210,558\n",
      "INFO\t2017-01-12 16:23:41\tGenotypeConcordance\tchecked    21,910,000 variants.  Elapsed time: 00:10:47s.  Time for last 10,000:    0s.  Last read position: 11:84,029,214\n",
      "INFO\t2017-01-12 16:23:41\tGenotypeConcordance\tchecked    21,920,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 11:84,987,313\n",
      "INFO\t2017-01-12 16:23:42\tGenotypeConcordance\tchecked    21,930,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 11:85,902,035\n",
      "INFO\t2017-01-12 16:23:42\tGenotypeConcordance\tchecked    21,940,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 11:86,729,527\n",
      "INFO\t2017-01-12 16:23:42\tGenotypeConcordance\tchecked    21,950,000 variants.  Elapsed time: 00:10:48s.  Time for last 10,000:    0s.  Last read position: 11:87,474,301\n",
      "INFO\t2017-01-12 16:23:43\tGenotypeConcordance\tchecked    21,960,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 11:88,276,892\n",
      "INFO\t2017-01-12 16:23:43\tGenotypeConcordance\tchecked    21,970,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 11:89,087,643\n",
      "INFO\t2017-01-12 16:23:43\tGenotypeConcordance\tchecked    21,980,000 variants.  Elapsed time: 00:10:49s.  Time for last 10,000:    0s.  Last read position: 11:90,086,953\n",
      "INFO\t2017-01-12 16:23:44\tGenotypeConcordance\tchecked    21,990,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 11:90,839,732\n",
      "INFO\t2017-01-12 16:23:44\tGenotypeConcordance\tchecked    22,000,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 11:91,582,982\n",
      "INFO\t2017-01-12 16:23:44\tGenotypeConcordance\tchecked    22,010,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 11:92,434,078\n",
      "INFO\t2017-01-12 16:23:44\tGenotypeConcordance\tchecked    22,020,000 variants.  Elapsed time: 00:10:50s.  Time for last 10,000:    0s.  Last read position: 11:93,377,875\n",
      "INFO\t2017-01-12 16:23:45\tGenotypeConcordance\tchecked    22,030,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 11:94,158,534\n",
      "INFO\t2017-01-12 16:23:45\tGenotypeConcordance\tchecked    22,040,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 11:95,045,372\n",
      "INFO\t2017-01-12 16:23:45\tGenotypeConcordance\tchecked    22,050,000 variants.  Elapsed time: 00:10:51s.  Time for last 10,000:    0s.  Last read position: 11:95,923,127\n",
      "INFO\t2017-01-12 16:23:46\tGenotypeConcordance\tchecked    22,060,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 11:96,851,454\n",
      "INFO\t2017-01-12 16:23:46\tGenotypeConcordance\tchecked    22,070,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 11:97,651,857\n",
      "INFO\t2017-01-12 16:23:46\tGenotypeConcordance\tchecked    22,080,000 variants.  Elapsed time: 00:10:52s.  Time for last 10,000:    0s.  Last read position: 11:98,369,187\n",
      "INFO\t2017-01-12 16:23:46\tGenotypeConcordance\tchecked    22,090,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 11:99,042,908\n",
      "INFO\t2017-01-12 16:23:47\tGenotypeConcordance\tchecked    22,100,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 11:99,724,188\n",
      "INFO\t2017-01-12 16:23:47\tGenotypeConcordance\tchecked    22,110,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 11:100,476,938\n",
      "INFO\t2017-01-12 16:23:47\tGenotypeConcordance\tchecked    22,120,000 variants.  Elapsed time: 00:10:53s.  Time for last 10,000:    0s.  Last read position: 11:101,298,056\n",
      "INFO\t2017-01-12 16:23:48\tGenotypeConcordance\tchecked    22,130,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 11:102,112,044\n",
      "INFO\t2017-01-12 16:23:48\tGenotypeConcordance\tchecked    22,140,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 11:102,951,503\n",
      "INFO\t2017-01-12 16:23:48\tGenotypeConcordance\tchecked    22,150,000 variants.  Elapsed time: 00:10:54s.  Time for last 10,000:    0s.  Last read position: 11:103,766,176\n",
      "INFO\t2017-01-12 16:23:49\tGenotypeConcordance\tchecked    22,160,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 11:104,605,464\n",
      "INFO\t2017-01-12 16:23:49\tGenotypeConcordance\tchecked    22,170,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 11:105,453,453\n",
      "INFO\t2017-01-12 16:23:49\tGenotypeConcordance\tchecked    22,180,000 variants.  Elapsed time: 00:10:55s.  Time for last 10,000:    0s.  Last read position: 11:106,460,523\n",
      "INFO\t2017-01-12 16:23:49\tGenotypeConcordance\tchecked    22,190,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 11:107,338,209\n",
      "INFO\t2017-01-12 16:23:50\tGenotypeConcordance\tchecked    22,200,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 11:108,204,808\n",
      "INFO\t2017-01-12 16:23:50\tGenotypeConcordance\tchecked    22,210,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 11:109,098,928\n",
      "INFO\t2017-01-12 16:23:50\tGenotypeConcordance\tchecked    22,220,000 variants.  Elapsed time: 00:10:56s.  Time for last 10,000:    0s.  Last read position: 11:110,059,060\n",
      "INFO\t2017-01-12 16:23:51\tGenotypeConcordance\tchecked    22,230,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 11:110,934,323\n",
      "INFO\t2017-01-12 16:23:51\tGenotypeConcordance\tchecked    22,240,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 11:111,928,226\n",
      "INFO\t2017-01-12 16:23:51\tGenotypeConcordance\tchecked    22,250,000 variants.  Elapsed time: 00:10:57s.  Time for last 10,000:    0s.  Last read position: 11:112,789,916\n",
      "INFO\t2017-01-12 16:23:52\tGenotypeConcordance\tchecked    22,260,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 11:113,734,365\n",
      "INFO\t2017-01-12 16:23:52\tGenotypeConcordance\tchecked    22,270,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 11:114,509,142\n",
      "INFO\t2017-01-12 16:23:52\tGenotypeConcordance\tchecked    22,280,000 variants.  Elapsed time: 00:10:58s.  Time for last 10,000:    0s.  Last read position: 11:115,438,165\n",
      "INFO\t2017-01-12 16:23:52\tGenotypeConcordance\tchecked    22,290,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 11:116,307,582\n",
      "INFO\t2017-01-12 16:23:53\tGenotypeConcordance\tchecked    22,300,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 11:117,095,280\n",
      "INFO\t2017-01-12 16:23:53\tGenotypeConcordance\tchecked    22,310,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 11:117,920,100\n",
      "INFO\t2017-01-12 16:23:53\tGenotypeConcordance\tchecked    22,320,000 variants.  Elapsed time: 00:10:59s.  Time for last 10,000:    0s.  Last read position: 11:118,836,281\n",
      "INFO\t2017-01-12 16:23:54\tGenotypeConcordance\tchecked    22,330,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 11:119,664,304\n",
      "INFO\t2017-01-12 16:23:54\tGenotypeConcordance\tchecked    22,340,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 11:120,517,905\n",
      "INFO\t2017-01-12 16:23:54\tGenotypeConcordance\tchecked    22,350,000 variants.  Elapsed time: 00:11:00s.  Time for last 10,000:    0s.  Last read position: 11:121,416,589\n",
      "INFO\t2017-01-12 16:23:55\tGenotypeConcordance\tchecked    22,360,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 11:122,367,529\n",
      "INFO\t2017-01-12 16:23:55\tGenotypeConcordance\tchecked    22,370,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 11:123,124,838\n",
      "INFO\t2017-01-12 16:23:55\tGenotypeConcordance\tchecked    22,380,000 variants.  Elapsed time: 00:11:01s.  Time for last 10,000:    0s.  Last read position: 11:123,896,733\n",
      "INFO\t2017-01-12 16:23:55\tGenotypeConcordance\tchecked    22,390,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 11:124,713,023\n",
      "INFO\t2017-01-12 16:23:56\tGenotypeConcordance\tchecked    22,400,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 11:125,607,198\n",
      "INFO\t2017-01-12 16:23:56\tGenotypeConcordance\tchecked    22,410,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 11:126,448,571\n",
      "INFO\t2017-01-12 16:23:56\tGenotypeConcordance\tchecked    22,420,000 variants.  Elapsed time: 00:11:02s.  Time for last 10,000:    0s.  Last read position: 11:127,318,546\n",
      "INFO\t2017-01-12 16:23:57\tGenotypeConcordance\tchecked    22,430,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 11:128,178,450\n",
      "INFO\t2017-01-12 16:23:57\tGenotypeConcordance\tchecked    22,440,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 11:129,092,082\n",
      "INFO\t2017-01-12 16:23:57\tGenotypeConcordance\tchecked    22,450,000 variants.  Elapsed time: 00:11:03s.  Time for last 10,000:    0s.  Last read position: 11:129,895,088\n",
      "INFO\t2017-01-12 16:23:57\tGenotypeConcordance\tchecked    22,460,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 11:130,701,645\n",
      "INFO\t2017-01-12 16:23:58\tGenotypeConcordance\tchecked    22,470,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 11:131,435,048\n",
      "INFO\t2017-01-12 16:23:58\tGenotypeConcordance\tchecked    22,480,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 11:132,194,934\n",
      "INFO\t2017-01-12 16:23:58\tGenotypeConcordance\tchecked    22,490,000 variants.  Elapsed time: 00:11:04s.  Time for last 10,000:    0s.  Last read position: 11:132,985,714\n",
      "INFO\t2017-01-12 16:23:59\tGenotypeConcordance\tchecked    22,500,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 11:133,804,152\n",
      "INFO\t2017-01-12 16:23:59\tGenotypeConcordance\tchecked    22,510,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 11:134,541,267\n",
      "INFO\t2017-01-12 16:23:59\tGenotypeConcordance\tchecked    22,520,000 variants.  Elapsed time: 00:11:05s.  Time for last 10,000:    0s.  Last read position: 12:313,627\n",
      "INFO\t2017-01-12 16:24:00\tGenotypeConcordance\tchecked    22,530,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:1,097,096\n",
      "INFO\t2017-01-12 16:24:00\tGenotypeConcordance\tchecked    22,540,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:1,851,862\n",
      "INFO\t2017-01-12 16:24:00\tGenotypeConcordance\tchecked    22,550,000 variants.  Elapsed time: 00:11:06s.  Time for last 10,000:    0s.  Last read position: 12:2,680,852\n",
      "INFO\t2017-01-12 16:24:00\tGenotypeConcordance\tchecked    22,560,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:3,393,512\n",
      "INFO\t2017-01-12 16:24:01\tGenotypeConcordance\tchecked    22,570,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:4,206,830\n",
      "INFO\t2017-01-12 16:24:01\tGenotypeConcordance\tchecked    22,580,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:5,061,819\n",
      "INFO\t2017-01-12 16:24:01\tGenotypeConcordance\tchecked    22,590,000 variants.  Elapsed time: 00:11:07s.  Time for last 10,000:    0s.  Last read position: 12:5,919,979\n",
      "INFO\t2017-01-12 16:24:02\tGenotypeConcordance\tchecked    22,600,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:6,635,274\n",
      "INFO\t2017-01-12 16:24:02\tGenotypeConcordance\tchecked    22,610,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:7,572,005\n",
      "INFO\t2017-01-12 16:24:02\tGenotypeConcordance\tchecked    22,620,000 variants.  Elapsed time: 00:11:08s.  Time for last 10,000:    0s.  Last read position: 12:8,272,350\n",
      "INFO\t2017-01-12 16:24:03\tGenotypeConcordance\tchecked    22,630,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:8,851,166\n",
      "INFO\t2017-01-12 16:24:03\tGenotypeConcordance\tchecked    22,640,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:9,609,298\n",
      "INFO\t2017-01-12 16:24:03\tGenotypeConcordance\tchecked    22,650,000 variants.  Elapsed time: 00:11:09s.  Time for last 10,000:    0s.  Last read position: 12:10,398,906\n",
      "INFO\t2017-01-12 16:24:03\tGenotypeConcordance\tchecked    22,660,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:11,170,074\n",
      "INFO\t2017-01-12 16:24:04\tGenotypeConcordance\tchecked    22,670,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:11,717,489\n",
      "INFO\t2017-01-12 16:24:04\tGenotypeConcordance\tchecked    22,680,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:12,516,393\n",
      "INFO\t2017-01-12 16:24:04\tGenotypeConcordance\tchecked    22,690,000 variants.  Elapsed time: 00:11:10s.  Time for last 10,000:    0s.  Last read position: 12:13,370,829\n",
      "INFO\t2017-01-12 16:24:05\tGenotypeConcordance\tchecked    22,700,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:14,231,402\n",
      "INFO\t2017-01-12 16:24:05\tGenotypeConcordance\tchecked    22,710,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:15,091,702\n",
      "INFO\t2017-01-12 16:24:05\tGenotypeConcordance\tchecked    22,720,000 variants.  Elapsed time: 00:11:11s.  Time for last 10,000:    0s.  Last read position: 12:16,045,476\n",
      "INFO\t2017-01-12 16:24:05\tGenotypeConcordance\tchecked    22,730,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:17,041,838\n",
      "INFO\t2017-01-12 16:24:06\tGenotypeConcordance\tchecked    22,740,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:17,925,474\n",
      "INFO\t2017-01-12 16:24:06\tGenotypeConcordance\tchecked    22,750,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:18,742,458\n",
      "INFO\t2017-01-12 16:24:06\tGenotypeConcordance\tchecked    22,760,000 variants.  Elapsed time: 00:11:12s.  Time for last 10,000:    0s.  Last read position: 12:19,581,726\n",
      "INFO\t2017-01-12 16:24:07\tGenotypeConcordance\tchecked    22,770,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:20,343,474\n",
      "INFO\t2017-01-12 16:24:07\tGenotypeConcordance\tchecked    22,780,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:21,125,365\n",
      "INFO\t2017-01-12 16:24:07\tGenotypeConcordance\tchecked    22,790,000 variants.  Elapsed time: 00:11:13s.  Time for last 10,000:    0s.  Last read position: 12:21,909,730\n",
      "INFO\t2017-01-12 16:24:08\tGenotypeConcordance\tchecked    22,800,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:22,894,851\n",
      "INFO\t2017-01-12 16:24:08\tGenotypeConcordance\tchecked    22,810,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:23,803,265\n",
      "INFO\t2017-01-12 16:24:08\tGenotypeConcordance\tchecked    22,820,000 variants.  Elapsed time: 00:11:14s.  Time for last 10,000:    0s.  Last read position: 12:24,673,740\n",
      "INFO\t2017-01-12 16:24:08\tGenotypeConcordance\tchecked    22,830,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:25,517,238\n",
      "INFO\t2017-01-12 16:24:09\tGenotypeConcordance\tchecked    22,840,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:26,311,517\n",
      "INFO\t2017-01-12 16:24:09\tGenotypeConcordance\tchecked    22,850,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:27,206,030\n",
      "INFO\t2017-01-12 16:24:09\tGenotypeConcordance\tchecked    22,860,000 variants.  Elapsed time: 00:11:15s.  Time for last 10,000:    0s.  Last read position: 12:28,101,641\n",
      "INFO\t2017-01-12 16:24:10\tGenotypeConcordance\tchecked    22,870,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:29,041,646\n",
      "INFO\t2017-01-12 16:24:10\tGenotypeConcordance\tchecked    22,880,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:29,854,972\n",
      "INFO\t2017-01-12 16:24:10\tGenotypeConcordance\tchecked    22,890,000 variants.  Elapsed time: 00:11:16s.  Time for last 10,000:    0s.  Last read position: 12:30,534,912\n",
      "INFO\t2017-01-12 16:24:11\tGenotypeConcordance\tchecked    22,900,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:31,302,583\n",
      "INFO\t2017-01-12 16:24:11\tGenotypeConcordance\tchecked    22,910,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:32,062,348\n",
      "INFO\t2017-01-12 16:24:11\tGenotypeConcordance\tchecked    22,920,000 variants.  Elapsed time: 00:11:17s.  Time for last 10,000:    0s.  Last read position: 12:32,801,194\n",
      "INFO\t2017-01-12 16:24:12\tGenotypeConcordance\tchecked    22,930,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:33,534,368\n",
      "INFO\t2017-01-12 16:24:12\tGenotypeConcordance\tchecked    22,940,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:34,307,426\n",
      "INFO\t2017-01-12 16:24:12\tGenotypeConcordance\tchecked    22,950,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:34,791,956\n",
      "INFO\t2017-01-12 16:24:12\tGenotypeConcordance\tchecked    22,960,000 variants.  Elapsed time: 00:11:18s.  Time for last 10,000:    0s.  Last read position: 12:38,159,731\n",
      "INFO\t2017-01-12 16:24:13\tGenotypeConcordance\tchecked    22,970,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:38,595,450\n",
      "INFO\t2017-01-12 16:24:13\tGenotypeConcordance\tchecked    22,980,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:39,346,678\n",
      "INFO\t2017-01-12 16:24:13\tGenotypeConcordance\tchecked    22,990,000 variants.  Elapsed time: 00:11:19s.  Time for last 10,000:    0s.  Last read position: 12:40,200,736\n",
      "INFO\t2017-01-12 16:24:14\tGenotypeConcordance\tchecked    23,000,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:41,034,018\n",
      "INFO\t2017-01-12 16:24:14\tGenotypeConcordance\tchecked    23,010,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:41,844,645\n",
      "INFO\t2017-01-12 16:24:14\tGenotypeConcordance\tchecked    23,020,000 variants.  Elapsed time: 00:11:20s.  Time for last 10,000:    0s.  Last read position: 12:42,595,907\n",
      "INFO\t2017-01-12 16:24:14\tGenotypeConcordance\tchecked    23,030,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:43,439,383\n",
      "INFO\t2017-01-12 16:24:15\tGenotypeConcordance\tchecked    23,040,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:44,318,922\n",
      "INFO\t2017-01-12 16:24:15\tGenotypeConcordance\tchecked    23,050,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:45,220,052\n",
      "INFO\t2017-01-12 16:24:15\tGenotypeConcordance\tchecked    23,060,000 variants.  Elapsed time: 00:11:21s.  Time for last 10,000:    0s.  Last read position: 12:46,088,790\n",
      "INFO\t2017-01-12 16:24:16\tGenotypeConcordance\tchecked    23,070,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:47,018,302\n",
      "INFO\t2017-01-12 16:24:16\tGenotypeConcordance\tchecked    23,080,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:47,829,588\n",
      "INFO\t2017-01-12 16:24:16\tGenotypeConcordance\tchecked    23,090,000 variants.  Elapsed time: 00:11:22s.  Time for last 10,000:    0s.  Last read position: 12:48,658,456\n",
      "INFO\t2017-01-12 16:24:16\tGenotypeConcordance\tchecked    23,100,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 12:49,640,962\n",
      "INFO\t2017-01-12 16:24:17\tGenotypeConcordance\tchecked    23,110,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 12:50,652,083\n",
      "INFO\t2017-01-12 16:24:17\tGenotypeConcordance\tchecked    23,120,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 12:51,659,071\n",
      "INFO\t2017-01-12 16:24:17\tGenotypeConcordance\tchecked    23,130,000 variants.  Elapsed time: 00:11:23s.  Time for last 10,000:    0s.  Last read position: 12:52,602,013\n",
      "INFO\t2017-01-12 16:24:18\tGenotypeConcordance\tchecked    23,140,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 12:53,331,970\n",
      "INFO\t2017-01-12 16:24:18\tGenotypeConcordance\tchecked    23,150,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 12:54,302,849\n",
      "INFO\t2017-01-12 16:24:18\tGenotypeConcordance\tchecked    23,160,000 variants.  Elapsed time: 00:11:24s.  Time for last 10,000:    0s.  Last read position: 12:55,219,254\n",
      "INFO\t2017-01-12 16:24:18\tGenotypeConcordance\tchecked    23,170,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 12:55,933,792\n",
      "INFO\t2017-01-12 16:24:19\tGenotypeConcordance\tchecked    23,180,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 12:56,962,625\n",
      "INFO\t2017-01-12 16:24:19\tGenotypeConcordance\tchecked    23,190,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 12:58,042,281\n",
      "INFO\t2017-01-12 16:24:19\tGenotypeConcordance\tchecked    23,200,000 variants.  Elapsed time: 00:11:25s.  Time for last 10,000:    0s.  Last read position: 12:58,952,712\n",
      "INFO\t2017-01-12 16:24:20\tGenotypeConcordance\tchecked    23,210,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 12:59,842,989\n",
      "INFO\t2017-01-12 16:24:20\tGenotypeConcordance\tchecked    23,220,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 12:60,725,868\n",
      "INFO\t2017-01-12 16:24:20\tGenotypeConcordance\tchecked    23,230,000 variants.  Elapsed time: 00:11:26s.  Time for last 10,000:    0s.  Last read position: 12:61,498,477\n",
      "INFO\t2017-01-12 16:24:21\tGenotypeConcordance\tchecked    23,240,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 12:62,310,217\n",
      "INFO\t2017-01-12 16:24:21\tGenotypeConcordance\tchecked    23,250,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 12:63,265,984\n",
      "INFO\t2017-01-12 16:24:21\tGenotypeConcordance\tchecked    23,260,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 12:64,067,829\n",
      "INFO\t2017-01-12 16:24:21\tGenotypeConcordance\tchecked    23,270,000 variants.  Elapsed time: 00:11:27s.  Time for last 10,000:    0s.  Last read position: 12:64,998,755\n",
      "INFO\t2017-01-12 16:24:22\tGenotypeConcordance\tchecked    23,280,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 12:66,061,422\n",
      "INFO\t2017-01-12 16:24:22\tGenotypeConcordance\tchecked    23,290,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 12:66,983,046\n",
      "INFO\t2017-01-12 16:24:22\tGenotypeConcordance\tchecked    23,300,000 variants.  Elapsed time: 00:11:28s.  Time for last 10,000:    0s.  Last read position: 12:67,878,789\n",
      "INFO\t2017-01-12 16:24:23\tGenotypeConcordance\tchecked    23,310,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 12:68,778,009\n",
      "INFO\t2017-01-12 16:24:23\tGenotypeConcordance\tchecked    23,320,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 12:69,634,706\n",
      "INFO\t2017-01-12 16:24:23\tGenotypeConcordance\tchecked    23,330,000 variants.  Elapsed time: 00:11:29s.  Time for last 10,000:    0s.  Last read position: 12:70,481,733\n",
      "INFO\t2017-01-12 16:24:23\tGenotypeConcordance\tchecked    23,340,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 12:71,351,875\n",
      "INFO\t2017-01-12 16:24:24\tGenotypeConcordance\tchecked    23,350,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 12:72,245,731\n",
      "INFO\t2017-01-12 16:24:24\tGenotypeConcordance\tchecked    23,360,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 12:73,218,766\n",
      "INFO\t2017-01-12 16:24:24\tGenotypeConcordance\tchecked    23,370,000 variants.  Elapsed time: 00:11:30s.  Time for last 10,000:    0s.  Last read position: 12:74,069,440\n",
      "INFO\t2017-01-12 16:24:25\tGenotypeConcordance\tchecked    23,380,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 12:74,989,531\n",
      "INFO\t2017-01-12 16:24:25\tGenotypeConcordance\tchecked    23,390,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 12:75,880,173\n",
      "INFO\t2017-01-12 16:24:25\tGenotypeConcordance\tchecked    23,400,000 variants.  Elapsed time: 00:11:31s.  Time for last 10,000:    0s.  Last read position: 12:76,720,599\n",
      "INFO\t2017-01-12 16:24:25\tGenotypeConcordance\tchecked    23,410,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 12:77,652,904\n",
      "INFO\t2017-01-12 16:24:26\tGenotypeConcordance\tchecked    23,420,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 12:78,531,976\n",
      "INFO\t2017-01-12 16:24:26\tGenotypeConcordance\tchecked    23,430,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 12:79,403,944\n",
      "INFO\t2017-01-12 16:24:26\tGenotypeConcordance\tchecked    23,440,000 variants.  Elapsed time: 00:11:32s.  Time for last 10,000:    0s.  Last read position: 12:80,478,326\n",
      "INFO\t2017-01-12 16:24:27\tGenotypeConcordance\tchecked    23,450,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 12:81,409,164\n",
      "INFO\t2017-01-12 16:24:27\tGenotypeConcordance\tchecked    23,460,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 12:82,386,288\n",
      "INFO\t2017-01-12 16:24:27\tGenotypeConcordance\tchecked    23,470,000 variants.  Elapsed time: 00:11:33s.  Time for last 10,000:    0s.  Last read position: 12:83,310,584\n",
      "INFO\t2017-01-12 16:24:27\tGenotypeConcordance\tchecked    23,480,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 12:84,095,819\n",
      "INFO\t2017-01-12 16:24:28\tGenotypeConcordance\tchecked    23,490,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 12:84,945,384\n",
      "INFO\t2017-01-12 16:24:28\tGenotypeConcordance\tchecked    23,500,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 12:85,972,453\n",
      "INFO\t2017-01-12 16:24:28\tGenotypeConcordance\tchecked    23,510,000 variants.  Elapsed time: 00:11:34s.  Time for last 10,000:    0s.  Last read position: 12:86,849,237\n",
      "INFO\t2017-01-12 16:24:29\tGenotypeConcordance\tchecked    23,520,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 12:87,742,759\n",
      "INFO\t2017-01-12 16:24:29\tGenotypeConcordance\tchecked    23,530,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 12:88,774,181\n",
      "INFO\t2017-01-12 16:24:29\tGenotypeConcordance\tchecked    23,540,000 variants.  Elapsed time: 00:11:35s.  Time for last 10,000:    0s.  Last read position: 12:89,904,596\n",
      "INFO\t2017-01-12 16:24:30\tGenotypeConcordance\tchecked    23,550,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 12:90,989,605\n",
      "INFO\t2017-01-12 16:24:30\tGenotypeConcordance\tchecked    23,560,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 12:91,964,219\n",
      "INFO\t2017-01-12 16:24:30\tGenotypeConcordance\tchecked    23,570,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 12:92,862,368\n",
      "INFO\t2017-01-12 16:24:30\tGenotypeConcordance\tchecked    23,580,000 variants.  Elapsed time: 00:11:36s.  Time for last 10,000:    0s.  Last read position: 12:93,735,865\n",
      "INFO\t2017-01-12 16:24:31\tGenotypeConcordance\tchecked    23,590,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 12:94,580,465\n",
      "INFO\t2017-01-12 16:24:31\tGenotypeConcordance\tchecked    23,600,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 12:95,398,440\n",
      "INFO\t2017-01-12 16:24:31\tGenotypeConcordance\tchecked    23,610,000 variants.  Elapsed time: 00:11:37s.  Time for last 10,000:    0s.  Last read position: 12:96,216,288\n",
      "INFO\t2017-01-12 16:24:32\tGenotypeConcordance\tchecked    23,620,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 12:97,086,351\n",
      "INFO\t2017-01-12 16:24:32\tGenotypeConcordance\tchecked    23,630,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 12:97,956,918\n",
      "INFO\t2017-01-12 16:24:32\tGenotypeConcordance\tchecked    23,640,000 variants.  Elapsed time: 00:11:38s.  Time for last 10,000:    0s.  Last read position: 12:98,835,746\n",
      "INFO\t2017-01-12 16:24:32\tGenotypeConcordance\tchecked    23,650,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 12:99,799,739\n",
      "INFO\t2017-01-12 16:24:33\tGenotypeConcordance\tchecked    23,660,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 12:100,671,902\n",
      "INFO\t2017-01-12 16:24:33\tGenotypeConcordance\tchecked    23,670,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 12:101,581,368\n",
      "INFO\t2017-01-12 16:24:33\tGenotypeConcordance\tchecked    23,680,000 variants.  Elapsed time: 00:11:39s.  Time for last 10,000:    0s.  Last read position: 12:102,429,347\n",
      "INFO\t2017-01-12 16:24:34\tGenotypeConcordance\tchecked    23,690,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 12:103,419,959\n",
      "INFO\t2017-01-12 16:24:34\tGenotypeConcordance\tchecked    23,700,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 12:104,310,945\n",
      "INFO\t2017-01-12 16:24:34\tGenotypeConcordance\tchecked    23,710,000 variants.  Elapsed time: 00:11:40s.  Time for last 10,000:    0s.  Last read position: 12:105,172,880\n",
      "INFO\t2017-01-12 16:24:35\tGenotypeConcordance\tchecked    23,720,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 12:106,030,833\n",
      "INFO\t2017-01-12 16:24:35\tGenotypeConcordance\tchecked    23,730,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 12:106,871,530\n",
      "INFO\t2017-01-12 16:24:35\tGenotypeConcordance\tchecked    23,740,000 variants.  Elapsed time: 00:11:41s.  Time for last 10,000:    0s.  Last read position: 12:107,859,928\n",
      "INFO\t2017-01-12 16:24:35\tGenotypeConcordance\tchecked    23,750,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 12:108,704,733\n",
      "INFO\t2017-01-12 16:24:36\tGenotypeConcordance\tchecked    23,760,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 12:109,650,050\n",
      "INFO\t2017-01-12 16:24:36\tGenotypeConcordance\tchecked    23,770,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 12:110,582,833\n",
      "INFO\t2017-01-12 16:24:36\tGenotypeConcordance\tchecked    23,780,000 variants.  Elapsed time: 00:11:42s.  Time for last 10,000:    0s.  Last read position: 12:111,636,580\n",
      "INFO\t2017-01-12 16:24:37\tGenotypeConcordance\tchecked    23,790,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 12:112,926,772\n",
      "INFO\t2017-01-12 16:24:37\tGenotypeConcordance\tchecked    23,800,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 12:113,898,522\n",
      "INFO\t2017-01-12 16:24:37\tGenotypeConcordance\tchecked    23,810,000 variants.  Elapsed time: 00:11:43s.  Time for last 10,000:    0s.  Last read position: 12:114,655,737\n",
      "INFO\t2017-01-12 16:24:37\tGenotypeConcordance\tchecked    23,820,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 12:115,410,225\n",
      "INFO\t2017-01-12 16:24:38\tGenotypeConcordance\tchecked    23,830,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 12:116,185,637\n",
      "INFO\t2017-01-12 16:24:38\tGenotypeConcordance\tchecked    23,840,000 variants.  Elapsed time: 00:11:44s.  Time for last 10,000:    0s.  Last read position: 12:117,089,737\n",
      "INFO\t2017-01-12 16:24:38\tGenotypeConcordance\tchecked    23,850,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 12:117,926,094\n",
      "INFO\t2017-01-12 16:24:39\tGenotypeConcordance\tchecked    23,860,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 12:118,752,878\n",
      "INFO\t2017-01-12 16:24:39\tGenotypeConcordance\tchecked    23,870,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 12:119,577,532\n",
      "INFO\t2017-01-12 16:24:39\tGenotypeConcordance\tchecked    23,880,000 variants.  Elapsed time: 00:11:45s.  Time for last 10,000:    0s.  Last read position: 12:120,492,514\n",
      "INFO\t2017-01-12 16:24:40\tGenotypeConcordance\tchecked    23,890,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 12:121,395,940\n",
      "INFO\t2017-01-12 16:24:40\tGenotypeConcordance\tchecked    23,900,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 12:122,225,369\n",
      "INFO\t2017-01-12 16:24:40\tGenotypeConcordance\tchecked    23,910,000 variants.  Elapsed time: 00:11:46s.  Time for last 10,000:    0s.  Last read position: 12:123,183,113\n",
      "INFO\t2017-01-12 16:24:41\tGenotypeConcordance\tchecked    23,920,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 12:124,154,840\n",
      "INFO\t2017-01-12 16:24:41\tGenotypeConcordance\tchecked    23,930,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 12:124,965,596\n",
      "INFO\t2017-01-12 16:24:41\tGenotypeConcordance\tchecked    23,940,000 variants.  Elapsed time: 00:11:47s.  Time for last 10,000:    0s.  Last read position: 12:125,679,919\n",
      "INFO\t2017-01-12 16:24:41\tGenotypeConcordance\tchecked    23,950,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 12:126,466,142\n",
      "INFO\t2017-01-12 16:24:42\tGenotypeConcordance\tchecked    23,960,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 12:127,177,853\n",
      "INFO\t2017-01-12 16:24:42\tGenotypeConcordance\tchecked    23,970,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 12:127,868,209\n",
      "INFO\t2017-01-12 16:24:42\tGenotypeConcordance\tchecked    23,980,000 variants.  Elapsed time: 00:11:48s.  Time for last 10,000:    0s.  Last read position: 12:128,567,404\n",
      "INFO\t2017-01-12 16:24:43\tGenotypeConcordance\tchecked    23,990,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 12:129,219,398\n",
      "INFO\t2017-01-12 16:24:43\tGenotypeConcordance\tchecked    24,000,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 12:129,879,302\n",
      "INFO\t2017-01-12 16:24:43\tGenotypeConcordance\tchecked    24,010,000 variants.  Elapsed time: 00:11:49s.  Time for last 10,000:    0s.  Last read position: 12:130,567,576\n",
      "INFO\t2017-01-12 16:24:44\tGenotypeConcordance\tchecked    24,020,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 12:131,228,645\n",
      "INFO\t2017-01-12 16:24:44\tGenotypeConcordance\tchecked    24,030,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 12:131,861,871\n",
      "INFO\t2017-01-12 16:24:44\tGenotypeConcordance\tchecked    24,040,000 variants.  Elapsed time: 00:11:50s.  Time for last 10,000:    0s.  Last read position: 12:132,604,096\n",
      "INFO\t2017-01-12 16:24:44\tGenotypeConcordance\tchecked    24,050,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 12:133,251,435\n",
      "INFO\t2017-01-12 16:24:45\tGenotypeConcordance\tchecked    24,060,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:19,304,599\n",
      "INFO\t2017-01-12 16:24:45\tGenotypeConcordance\tchecked    24,070,000 variants.  Elapsed time: 00:11:51s.  Time for last 10,000:    0s.  Last read position: 13:19,762,694\n",
      "INFO\t2017-01-12 16:24:45\tGenotypeConcordance\tchecked    24,080,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:20,474,044\n",
      "INFO\t2017-01-12 16:24:46\tGenotypeConcordance\tchecked    24,090,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:21,260,126\n",
      "INFO\t2017-01-12 16:24:46\tGenotypeConcordance\tchecked    24,100,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:22,023,707\n",
      "INFO\t2017-01-12 16:24:46\tGenotypeConcordance\tchecked    24,110,000 variants.  Elapsed time: 00:11:52s.  Time for last 10,000:    0s.  Last read position: 13:22,778,689\n",
      "INFO\t2017-01-12 16:24:47\tGenotypeConcordance\tchecked    24,120,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:23,490,255\n",
      "INFO\t2017-01-12 16:24:47\tGenotypeConcordance\tchecked    24,130,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:24,110,002\n",
      "INFO\t2017-01-12 16:24:47\tGenotypeConcordance\tchecked    24,140,000 variants.  Elapsed time: 00:11:53s.  Time for last 10,000:    0s.  Last read position: 13:24,796,423\n",
      "INFO\t2017-01-12 16:24:48\tGenotypeConcordance\tchecked    24,150,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:25,465,752\n",
      "INFO\t2017-01-12 16:24:48\tGenotypeConcordance\tchecked    24,160,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:26,218,409\n",
      "INFO\t2017-01-12 16:24:48\tGenotypeConcordance\tchecked    24,170,000 variants.  Elapsed time: 00:11:54s.  Time for last 10,000:    0s.  Last read position: 13:26,976,591\n",
      "INFO\t2017-01-12 16:24:49\tGenotypeConcordance\tchecked    24,180,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:27,717,165\n",
      "INFO\t2017-01-12 16:24:49\tGenotypeConcordance\tchecked    24,190,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:28,488,883\n",
      "INFO\t2017-01-12 16:24:49\tGenotypeConcordance\tchecked    24,200,000 variants.  Elapsed time: 00:11:55s.  Time for last 10,000:    0s.  Last read position: 13:29,306,662\n",
      "INFO\t2017-01-12 16:24:49\tGenotypeConcordance\tchecked    24,210,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:30,084,970\n",
      "INFO\t2017-01-12 16:24:50\tGenotypeConcordance\tchecked    24,220,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:30,871,177\n",
      "INFO\t2017-01-12 16:24:50\tGenotypeConcordance\tchecked    24,230,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:31,714,045\n",
      "INFO\t2017-01-12 16:24:50\tGenotypeConcordance\tchecked    24,240,000 variants.  Elapsed time: 00:11:56s.  Time for last 10,000:    0s.  Last read position: 13:32,533,595\n",
      "INFO\t2017-01-12 16:24:51\tGenotypeConcordance\tchecked    24,250,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 13:33,477,112\n",
      "INFO\t2017-01-12 16:24:51\tGenotypeConcordance\tchecked    24,260,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 13:34,290,654\n",
      "INFO\t2017-01-12 16:24:51\tGenotypeConcordance\tchecked    24,270,000 variants.  Elapsed time: 00:11:57s.  Time for last 10,000:    0s.  Last read position: 13:35,142,002\n",
      "INFO\t2017-01-12 16:24:51\tGenotypeConcordance\tchecked    24,280,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 13:36,131,705\n",
      "INFO\t2017-01-12 16:24:52\tGenotypeConcordance\tchecked    24,290,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 13:36,999,746\n",
      "INFO\t2017-01-12 16:24:52\tGenotypeConcordance\tchecked    24,300,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 13:37,905,116\n",
      "INFO\t2017-01-12 16:24:52\tGenotypeConcordance\tchecked    24,310,000 variants.  Elapsed time: 00:11:58s.  Time for last 10,000:    0s.  Last read position: 13:38,687,006\n",
      "INFO\t2017-01-12 16:24:53\tGenotypeConcordance\tchecked    24,320,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 13:39,515,037\n",
      "INFO\t2017-01-12 16:24:53\tGenotypeConcordance\tchecked    24,330,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 13:40,312,048\n",
      "INFO\t2017-01-12 16:24:53\tGenotypeConcordance\tchecked    24,340,000 variants.  Elapsed time: 00:11:59s.  Time for last 10,000:    0s.  Last read position: 13:41,140,781\n",
      "INFO\t2017-01-12 16:24:54\tGenotypeConcordance\tchecked    24,350,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 13:42,029,712\n",
      "INFO\t2017-01-12 16:24:54\tGenotypeConcordance\tchecked    24,360,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 13:42,911,957\n",
      "INFO\t2017-01-12 16:24:54\tGenotypeConcordance\tchecked    24,370,000 variants.  Elapsed time: 00:12:00s.  Time for last 10,000:    0s.  Last read position: 13:43,729,296\n",
      "INFO\t2017-01-12 16:24:54\tGenotypeConcordance\tchecked    24,380,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 13:44,669,703\n",
      "INFO\t2017-01-12 16:24:55\tGenotypeConcordance\tchecked    24,390,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 13:45,663,718\n",
      "INFO\t2017-01-12 16:24:55\tGenotypeConcordance\tchecked    24,400,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 13:46,510,533\n",
      "INFO\t2017-01-12 16:24:55\tGenotypeConcordance\tchecked    24,410,000 variants.  Elapsed time: 00:12:01s.  Time for last 10,000:    0s.  Last read position: 13:47,405,736\n",
      "INFO\t2017-01-12 16:24:56\tGenotypeConcordance\tchecked    24,420,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 13:48,262,266\n",
      "INFO\t2017-01-12 16:24:56\tGenotypeConcordance\tchecked    24,430,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 13:49,145,424\n",
      "INFO\t2017-01-12 16:24:56\tGenotypeConcordance\tchecked    24,440,000 variants.  Elapsed time: 00:12:02s.  Time for last 10,000:    0s.  Last read position: 13:50,088,357\n",
      "INFO\t2017-01-12 16:24:57\tGenotypeConcordance\tchecked    24,450,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 13:51,105,300\n",
      "INFO\t2017-01-12 16:24:57\tGenotypeConcordance\tchecked    24,460,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 13:52,065,039\n",
      "INFO\t2017-01-12 16:24:57\tGenotypeConcordance\tchecked    24,470,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 13:53,059,356\n",
      "INFO\t2017-01-12 16:24:57\tGenotypeConcordance\tchecked    24,480,000 variants.  Elapsed time: 00:12:03s.  Time for last 10,000:    0s.  Last read position: 13:53,973,902\n",
      "INFO\t2017-01-12 16:24:58\tGenotypeConcordance\tchecked    24,490,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 13:54,971,013\n",
      "INFO\t2017-01-12 16:24:58\tGenotypeConcordance\tchecked    24,500,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 13:55,867,116\n",
      "INFO\t2017-01-12 16:24:58\tGenotypeConcordance\tchecked    24,510,000 variants.  Elapsed time: 00:12:04s.  Time for last 10,000:    0s.  Last read position: 13:56,783,486\n",
      "INFO\t2017-01-12 16:24:59\tGenotypeConcordance\tchecked    24,520,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 13:57,999,028\n",
      "INFO\t2017-01-12 16:24:59\tGenotypeConcordance\tchecked    24,530,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 13:59,108,671\n",
      "INFO\t2017-01-12 16:24:59\tGenotypeConcordance\tchecked    24,540,000 variants.  Elapsed time: 00:12:05s.  Time for last 10,000:    0s.  Last read position: 13:60,016,892\n",
      "INFO\t2017-01-12 16:24:59\tGenotypeConcordance\tchecked    24,550,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 13:61,003,976\n",
      "INFO\t2017-01-12 16:25:00\tGenotypeConcordance\tchecked    24,560,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 13:61,802,659\n",
      "INFO\t2017-01-12 16:25:00\tGenotypeConcordance\tchecked    24,570,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 13:62,598,484\n",
      "INFO\t2017-01-12 16:25:00\tGenotypeConcordance\tchecked    24,580,000 variants.  Elapsed time: 00:12:06s.  Time for last 10,000:    0s.  Last read position: 13:63,452,736\n",
      "INFO\t2017-01-12 16:25:01\tGenotypeConcordance\tchecked    24,590,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 13:64,125,797\n",
      "INFO\t2017-01-12 16:25:01\tGenotypeConcordance\tchecked    24,600,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 13:64,878,263\n",
      "INFO\t2017-01-12 16:25:01\tGenotypeConcordance\tchecked    24,610,000 variants.  Elapsed time: 00:12:07s.  Time for last 10,000:    0s.  Last read position: 13:65,774,822\n",
      "INFO\t2017-01-12 16:25:02\tGenotypeConcordance\tchecked    24,620,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 13:66,668,526\n",
      "INFO\t2017-01-12 16:25:02\tGenotypeConcordance\tchecked    24,630,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 13:67,589,688\n",
      "INFO\t2017-01-12 16:25:02\tGenotypeConcordance\tchecked    24,640,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 13:68,476,463\n",
      "INFO\t2017-01-12 16:25:02\tGenotypeConcordance\tchecked    24,650,000 variants.  Elapsed time: 00:12:08s.  Time for last 10,000:    0s.  Last read position: 13:69,344,408\n",
      "INFO\t2017-01-12 16:25:03\tGenotypeConcordance\tchecked    24,660,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 13:70,123,444\n",
      "INFO\t2017-01-12 16:25:03\tGenotypeConcordance\tchecked    24,670,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 13:70,875,105\n",
      "INFO\t2017-01-12 16:25:03\tGenotypeConcordance\tchecked    24,680,000 variants.  Elapsed time: 00:12:09s.  Time for last 10,000:    0s.  Last read position: 13:71,646,397\n",
      "INFO\t2017-01-12 16:25:04\tGenotypeConcordance\tchecked    24,690,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 13:72,550,851\n",
      "INFO\t2017-01-12 16:25:04\tGenotypeConcordance\tchecked    24,700,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 13:73,365,211\n",
      "INFO\t2017-01-12 16:25:04\tGenotypeConcordance\tchecked    24,710,000 variants.  Elapsed time: 00:12:10s.  Time for last 10,000:    0s.  Last read position: 13:74,216,457\n",
      "INFO\t2017-01-12 16:25:04\tGenotypeConcordance\tchecked    24,720,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 13:75,062,769\n",
      "INFO\t2017-01-12 16:25:05\tGenotypeConcordance\tchecked    24,730,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 13:75,922,295\n",
      "INFO\t2017-01-12 16:25:05\tGenotypeConcordance\tchecked    24,740,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 13:76,749,475\n",
      "INFO\t2017-01-12 16:25:05\tGenotypeConcordance\tchecked    24,750,000 variants.  Elapsed time: 00:12:11s.  Time for last 10,000:    0s.  Last read position: 13:77,592,009\n",
      "INFO\t2017-01-12 16:25:06\tGenotypeConcordance\tchecked    24,760,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 13:78,602,863\n",
      "INFO\t2017-01-12 16:25:06\tGenotypeConcordance\tchecked    24,770,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 13:79,430,982\n",
      "INFO\t2017-01-12 16:25:06\tGenotypeConcordance\tchecked    24,780,000 variants.  Elapsed time: 00:12:12s.  Time for last 10,000:    0s.  Last read position: 13:80,380,090\n",
      "INFO\t2017-01-12 16:25:07\tGenotypeConcordance\tchecked    24,790,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 13:81,290,711\n",
      "INFO\t2017-01-12 16:25:07\tGenotypeConcordance\tchecked    24,800,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 13:82,174,716\n",
      "INFO\t2017-01-12 16:25:07\tGenotypeConcordance\tchecked    24,810,000 variants.  Elapsed time: 00:12:13s.  Time for last 10,000:    0s.  Last read position: 13:83,086,550\n",
      "INFO\t2017-01-12 16:25:07\tGenotypeConcordance\tchecked    24,820,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 13:83,903,522\n",
      "INFO\t2017-01-12 16:25:08\tGenotypeConcordance\tchecked    24,830,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 13:84,728,732\n",
      "INFO\t2017-01-12 16:25:08\tGenotypeConcordance\tchecked    24,840,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 13:85,558,486\n",
      "INFO\t2017-01-12 16:25:08\tGenotypeConcordance\tchecked    24,850,000 variants.  Elapsed time: 00:12:14s.  Time for last 10,000:    0s.  Last read position: 13:86,333,194\n",
      "INFO\t2017-01-12 16:25:09\tGenotypeConcordance\tchecked    24,860,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 13:87,268,660\n",
      "INFO\t2017-01-12 16:25:09\tGenotypeConcordance\tchecked    24,870,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 13:88,145,991\n",
      "INFO\t2017-01-12 16:25:09\tGenotypeConcordance\tchecked    24,880,000 variants.  Elapsed time: 00:12:15s.  Time for last 10,000:    0s.  Last read position: 13:89,056,014\n",
      "INFO\t2017-01-12 16:25:10\tGenotypeConcordance\tchecked    24,890,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 13:89,983,106\n",
      "INFO\t2017-01-12 16:25:10\tGenotypeConcordance\tchecked    24,900,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 13:90,769,532\n",
      "INFO\t2017-01-12 16:25:10\tGenotypeConcordance\tchecked    24,910,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 13:91,614,449\n",
      "INFO\t2017-01-12 16:25:10\tGenotypeConcordance\tchecked    24,920,000 variants.  Elapsed time: 00:12:16s.  Time for last 10,000:    0s.  Last read position: 13:92,416,361\n",
      "INFO\t2017-01-12 16:25:11\tGenotypeConcordance\tchecked    24,930,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 13:93,247,541\n",
      "INFO\t2017-01-12 16:25:11\tGenotypeConcordance\tchecked    24,940,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 13:94,106,204\n",
      "INFO\t2017-01-12 16:25:11\tGenotypeConcordance\tchecked    24,950,000 variants.  Elapsed time: 00:12:17s.  Time for last 10,000:    0s.  Last read position: 13:94,992,827\n",
      "INFO\t2017-01-12 16:25:12\tGenotypeConcordance\tchecked    24,960,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 13:95,842,518\n",
      "INFO\t2017-01-12 16:25:12\tGenotypeConcordance\tchecked    24,970,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 13:96,724,928\n",
      "INFO\t2017-01-12 16:25:12\tGenotypeConcordance\tchecked    24,980,000 variants.  Elapsed time: 00:12:18s.  Time for last 10,000:    0s.  Last read position: 13:97,744,284\n",
      "INFO\t2017-01-12 16:25:13\tGenotypeConcordance\tchecked    24,990,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 13:98,678,011\n",
      "INFO\t2017-01-12 16:25:13\tGenotypeConcordance\tchecked    25,000,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 13:99,392,176\n",
      "INFO\t2017-01-12 16:25:13\tGenotypeConcordance\tchecked    25,010,000 variants.  Elapsed time: 00:12:19s.  Time for last 10,000:    0s.  Last read position: 13:100,239,768\n",
      "INFO\t2017-01-12 16:25:14\tGenotypeConcordance\tchecked    25,020,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 13:101,123,315\n",
      "INFO\t2017-01-12 16:25:14\tGenotypeConcordance\tchecked    25,030,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 13:101,962,466\n",
      "INFO\t2017-01-12 16:25:14\tGenotypeConcordance\tchecked    25,040,000 variants.  Elapsed time: 00:12:20s.  Time for last 10,000:    0s.  Last read position: 13:102,781,655\n",
      "INFO\t2017-01-12 16:25:15\tGenotypeConcordance\tchecked    25,050,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 13:103,642,152\n",
      "INFO\t2017-01-12 16:25:15\tGenotypeConcordance\tchecked    25,060,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 13:104,384,890\n",
      "INFO\t2017-01-12 16:25:15\tGenotypeConcordance\tchecked    25,070,000 variants.  Elapsed time: 00:12:21s.  Time for last 10,000:    0s.  Last read position: 13:105,118,015\n",
      "INFO\t2017-01-12 16:25:15\tGenotypeConcordance\tchecked    25,080,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 13:105,846,909\n",
      "INFO\t2017-01-12 16:25:16\tGenotypeConcordance\tchecked    25,090,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 13:106,559,339\n",
      "INFO\t2017-01-12 16:25:16\tGenotypeConcordance\tchecked    25,100,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 13:107,386,589\n",
      "INFO\t2017-01-12 16:25:16\tGenotypeConcordance\tchecked    25,110,000 variants.  Elapsed time: 00:12:22s.  Time for last 10,000:    0s.  Last read position: 13:108,174,428\n",
      "INFO\t2017-01-12 16:25:17\tGenotypeConcordance\tchecked    25,120,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 13:108,953,991\n",
      "INFO\t2017-01-12 16:25:17\tGenotypeConcordance\tchecked    25,130,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 13:109,669,163\n",
      "INFO\t2017-01-12 16:25:17\tGenotypeConcordance\tchecked    25,140,000 variants.  Elapsed time: 00:12:23s.  Time for last 10,000:    0s.  Last read position: 13:110,420,075\n",
      "INFO\t2017-01-12 16:25:18\tGenotypeConcordance\tchecked    25,150,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 13:111,132,820\n",
      "INFO\t2017-01-12 16:25:18\tGenotypeConcordance\tchecked    25,160,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 13:111,873,880\n",
      "INFO\t2017-01-12 16:25:18\tGenotypeConcordance\tchecked    25,170,000 variants.  Elapsed time: 00:12:24s.  Time for last 10,000:    0s.  Last read position: 13:112,661,858\n",
      "INFO\t2017-01-12 16:25:19\tGenotypeConcordance\tchecked    25,180,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 13:113,345,185\n",
      "INFO\t2017-01-12 16:25:19\tGenotypeConcordance\tchecked    25,190,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 13:114,011,135\n",
      "INFO\t2017-01-12 16:25:19\tGenotypeConcordance\tchecked    25,200,000 variants.  Elapsed time: 00:12:25s.  Time for last 10,000:    0s.  Last read position: 13:114,803,398\n",
      "INFO\t2017-01-12 16:25:20\tGenotypeConcordance\tchecked    25,210,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:19,265,633\n",
      "INFO\t2017-01-12 16:25:20\tGenotypeConcordance\tchecked    25,220,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:20,118,185\n",
      "INFO\t2017-01-12 16:25:20\tGenotypeConcordance\tchecked    25,230,000 variants.  Elapsed time: 00:12:26s.  Time for last 10,000:    0s.  Last read position: 14:20,713,788\n",
      "INFO\t2017-01-12 16:25:20\tGenotypeConcordance\tchecked    25,240,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 14:21,426,155\n",
      "INFO\t2017-01-12 16:25:21\tGenotypeConcordance\tchecked    25,250,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 14:22,226,035\n",
      "INFO\t2017-01-12 16:25:21\tGenotypeConcordance\tchecked    25,260,000 variants.  Elapsed time: 00:12:27s.  Time for last 10,000:    0s.  Last read position: 14:22,921,280\n",
      "INFO\t2017-01-12 16:25:21\tGenotypeConcordance\tchecked    25,270,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 14:23,718,040\n",
      "INFO\t2017-01-12 16:25:22\tGenotypeConcordance\tchecked    25,280,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 14:24,525,345\n",
      "INFO\t2017-01-12 16:25:22\tGenotypeConcordance\tchecked    25,290,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 14:25,424,736\n",
      "INFO\t2017-01-12 16:25:22\tGenotypeConcordance\tchecked    25,300,000 variants.  Elapsed time: 00:12:28s.  Time for last 10,000:    0s.  Last read position: 14:26,226,897\n",
      "INFO\t2017-01-12 16:25:23\tGenotypeConcordance\tchecked    25,310,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 14:27,142,849\n",
      "INFO\t2017-01-12 16:25:23\tGenotypeConcordance\tchecked    25,320,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 14:28,086,730\n",
      "INFO\t2017-01-12 16:25:23\tGenotypeConcordance\tchecked    25,330,000 variants.  Elapsed time: 00:12:29s.  Time for last 10,000:    0s.  Last read position: 14:28,882,960\n",
      "INFO\t2017-01-12 16:25:24\tGenotypeConcordance\tchecked    25,340,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 14:29,843,244\n",
      "INFO\t2017-01-12 16:25:24\tGenotypeConcordance\tchecked    25,350,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 14:30,803,579\n",
      "INFO\t2017-01-12 16:25:24\tGenotypeConcordance\tchecked    25,360,000 variants.  Elapsed time: 00:12:30s.  Time for last 10,000:    0s.  Last read position: 14:31,698,980\n",
      "INFO\t2017-01-12 16:25:24\tGenotypeConcordance\tchecked    25,370,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 14:32,508,962\n",
      "INFO\t2017-01-12 16:25:25\tGenotypeConcordance\tchecked    25,380,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 14:33,352,908\n",
      "INFO\t2017-01-12 16:25:25\tGenotypeConcordance\tchecked    25,390,000 variants.  Elapsed time: 00:12:31s.  Time for last 10,000:    0s.  Last read position: 14:34,186,012\n",
      "INFO\t2017-01-12 16:25:25\tGenotypeConcordance\tchecked    25,400,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 14:34,953,955\n",
      "INFO\t2017-01-12 16:25:26\tGenotypeConcordance\tchecked    25,410,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 14:35,801,851\n",
      "INFO\t2017-01-12 16:25:26\tGenotypeConcordance\tchecked    25,420,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 14:36,737,697\n",
      "INFO\t2017-01-12 16:25:26\tGenotypeConcordance\tchecked    25,430,000 variants.  Elapsed time: 00:12:32s.  Time for last 10,000:    0s.  Last read position: 14:37,651,543\n",
      "INFO\t2017-01-12 16:25:27\tGenotypeConcordance\tchecked    25,440,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 14:38,640,848\n",
      "INFO\t2017-01-12 16:25:27\tGenotypeConcordance\tchecked    25,450,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 14:39,535,695\n",
      "INFO\t2017-01-12 16:25:27\tGenotypeConcordance\tchecked    25,460,000 variants.  Elapsed time: 00:12:33s.  Time for last 10,000:    0s.  Last read position: 14:40,367,185\n",
      "INFO\t2017-01-12 16:25:28\tGenotypeConcordance\tchecked    25,470,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 14:41,101,973\n",
      "INFO\t2017-01-12 16:25:28\tGenotypeConcordance\tchecked    25,480,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 14:41,865,711\n",
      "INFO\t2017-01-12 16:25:28\tGenotypeConcordance\tchecked    25,490,000 variants.  Elapsed time: 00:12:34s.  Time for last 10,000:    0s.  Last read position: 14:42,694,914\n",
      "INFO\t2017-01-12 16:25:29\tGenotypeConcordance\tchecked    25,500,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 14:43,435,164\n",
      "INFO\t2017-01-12 16:25:29\tGenotypeConcordance\tchecked    25,510,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 14:44,159,974\n",
      "INFO\t2017-01-12 16:25:29\tGenotypeConcordance\tchecked    25,520,000 variants.  Elapsed time: 00:12:35s.  Time for last 10,000:    0s.  Last read position: 14:44,993,347\n",
      "INFO\t2017-01-12 16:25:30\tGenotypeConcordance\tchecked    25,530,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 14:45,986,130\n",
      "INFO\t2017-01-12 16:25:30\tGenotypeConcordance\tchecked    25,540,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 14:46,753,003\n",
      "INFO\t2017-01-12 16:25:30\tGenotypeConcordance\tchecked    25,550,000 variants.  Elapsed time: 00:12:36s.  Time for last 10,000:    0s.  Last read position: 14:47,608,303\n",
      "INFO\t2017-01-12 16:25:31\tGenotypeConcordance\tchecked    25,560,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 14:48,415,645\n",
      "INFO\t2017-01-12 16:25:31\tGenotypeConcordance\tchecked    25,570,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 14:49,196,486\n",
      "INFO\t2017-01-12 16:25:31\tGenotypeConcordance\tchecked    25,580,000 variants.  Elapsed time: 00:12:37s.  Time for last 10,000:    0s.  Last read position: 14:50,051,751\n",
      "INFO\t2017-01-12 16:25:31\tGenotypeConcordance\tchecked    25,590,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 14:50,956,974\n",
      "INFO\t2017-01-12 16:25:32\tGenotypeConcordance\tchecked    25,600,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 14:51,819,420\n",
      "INFO\t2017-01-12 16:25:32\tGenotypeConcordance\tchecked    25,610,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 14:52,626,461\n",
      "INFO\t2017-01-12 16:25:32\tGenotypeConcordance\tchecked    25,620,000 variants.  Elapsed time: 00:12:38s.  Time for last 10,000:    0s.  Last read position: 14:53,481,995\n",
      "INFO\t2017-01-12 16:25:33\tGenotypeConcordance\tchecked    25,630,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 14:54,322,781\n",
      "INFO\t2017-01-12 16:25:33\tGenotypeConcordance\tchecked    25,640,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 14:55,256,021\n",
      "INFO\t2017-01-12 16:25:33\tGenotypeConcordance\tchecked    25,650,000 variants.  Elapsed time: 00:12:39s.  Time for last 10,000:    0s.  Last read position: 14:56,071,798\n",
      "INFO\t2017-01-12 16:25:34\tGenotypeConcordance\tchecked    25,660,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 14:56,868,003\n",
      "INFO\t2017-01-12 16:25:34\tGenotypeConcordance\tchecked    25,670,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 14:57,692,107\n",
      "INFO\t2017-01-12 16:25:34\tGenotypeConcordance\tchecked    25,680,000 variants.  Elapsed time: 00:12:40s.  Time for last 10,000:    0s.  Last read position: 14:58,514,790\n",
      "INFO\t2017-01-12 16:25:35\tGenotypeConcordance\tchecked    25,690,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 14:59,519,143\n",
      "INFO\t2017-01-12 16:25:35\tGenotypeConcordance\tchecked    25,700,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 14:60,460,445\n",
      "INFO\t2017-01-12 16:25:35\tGenotypeConcordance\tchecked    25,710,000 variants.  Elapsed time: 00:12:41s.  Time for last 10,000:    0s.  Last read position: 14:61,510,680\n",
      "INFO\t2017-01-12 16:25:35\tGenotypeConcordance\tchecked    25,720,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 14:62,358,047\n",
      "INFO\t2017-01-12 16:25:36\tGenotypeConcordance\tchecked    25,730,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 14:63,140,769\n",
      "INFO\t2017-01-12 16:25:36\tGenotypeConcordance\tchecked    25,740,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 14:64,010,783\n",
      "INFO\t2017-01-12 16:25:36\tGenotypeConcordance\tchecked    25,750,000 variants.  Elapsed time: 00:12:42s.  Time for last 10,000:    0s.  Last read position: 14:64,914,537\n",
      "INFO\t2017-01-12 16:25:37\tGenotypeConcordance\tchecked    25,760,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 14:65,804,664\n",
      "INFO\t2017-01-12 16:25:37\tGenotypeConcordance\tchecked    25,770,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 14:66,697,945\n",
      "INFO\t2017-01-12 16:25:37\tGenotypeConcordance\tchecked    25,780,000 variants.  Elapsed time: 00:12:43s.  Time for last 10,000:    0s.  Last read position: 14:67,693,839\n",
      "INFO\t2017-01-12 16:25:37\tGenotypeConcordance\tchecked    25,790,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 14:68,685,460\n",
      "INFO\t2017-01-12 16:25:38\tGenotypeConcordance\tchecked    25,800,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 14:69,593,656\n",
      "INFO\t2017-01-12 16:25:38\tGenotypeConcordance\tchecked    25,810,000 variants.  Elapsed time: 00:12:44s.  Time for last 10,000:    0s.  Last read position: 14:70,479,078\n",
      "INFO\t2017-01-12 16:25:38\tGenotypeConcordance\tchecked    25,820,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 14:71,301,755\n",
      "INFO\t2017-01-12 16:25:39\tGenotypeConcordance\tchecked    25,830,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 14:72,309,303\n",
      "INFO\t2017-01-12 16:25:39\tGenotypeConcordance\tchecked    25,840,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 14:73,076,504\n",
      "INFO\t2017-01-12 16:25:39\tGenotypeConcordance\tchecked    25,850,000 variants.  Elapsed time: 00:12:45s.  Time for last 10,000:    0s.  Last read position: 14:73,967,895\n",
      "INFO\t2017-01-12 16:25:40\tGenotypeConcordance\tchecked    25,860,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 14:74,749,606\n",
      "INFO\t2017-01-12 16:25:40\tGenotypeConcordance\tchecked    25,870,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 14:75,724,473\n",
      "INFO\t2017-01-12 16:25:40\tGenotypeConcordance\tchecked    25,880,000 variants.  Elapsed time: 00:12:46s.  Time for last 10,000:    0s.  Last read position: 14:76,659,723\n",
      "INFO\t2017-01-12 16:25:41\tGenotypeConcordance\tchecked    25,890,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 14:77,436,787\n",
      "INFO\t2017-01-12 16:25:41\tGenotypeConcordance\tchecked    25,900,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 14:78,239,259\n",
      "INFO\t2017-01-12 16:25:41\tGenotypeConcordance\tchecked    25,910,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 14:79,104,652\n",
      "INFO\t2017-01-12 16:25:41\tGenotypeConcordance\tchecked    25,920,000 variants.  Elapsed time: 00:12:47s.  Time for last 10,000:    0s.  Last read position: 14:80,016,502\n",
      "INFO\t2017-01-12 16:25:42\tGenotypeConcordance\tchecked    25,930,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 14:80,931,119\n",
      "INFO\t2017-01-12 16:25:42\tGenotypeConcordance\tchecked    25,940,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 14:81,836,996\n",
      "INFO\t2017-01-12 16:25:42\tGenotypeConcordance\tchecked    25,950,000 variants.  Elapsed time: 00:12:48s.  Time for last 10,000:    0s.  Last read position: 14:82,673,878\n",
      "INFO\t2017-01-12 16:25:43\tGenotypeConcordance\tchecked    25,960,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 14:83,528,049\n",
      "INFO\t2017-01-12 16:25:43\tGenotypeConcordance\tchecked    25,970,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 14:84,402,471\n",
      "INFO\t2017-01-12 16:25:43\tGenotypeConcordance\tchecked    25,980,000 variants.  Elapsed time: 00:12:49s.  Time for last 10,000:    0s.  Last read position: 14:85,234,218\n",
      "INFO\t2017-01-12 16:25:43\tGenotypeConcordance\tchecked    25,990,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 14:86,098,183\n",
      "INFO\t2017-01-12 16:25:44\tGenotypeConcordance\tchecked    26,000,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 14:86,839,203\n",
      "INFO\t2017-01-12 16:25:44\tGenotypeConcordance\tchecked    26,010,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 14:87,705,702\n",
      "INFO\t2017-01-12 16:25:44\tGenotypeConcordance\tchecked    26,020,000 variants.  Elapsed time: 00:12:50s.  Time for last 10,000:    0s.  Last read position: 14:88,489,630\n",
      "INFO\t2017-01-12 16:25:45\tGenotypeConcordance\tchecked    26,030,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 14:89,384,118\n",
      "INFO\t2017-01-12 16:25:45\tGenotypeConcordance\tchecked    26,040,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 14:90,169,202\n",
      "INFO\t2017-01-12 16:25:45\tGenotypeConcordance\tchecked    26,050,000 variants.  Elapsed time: 00:12:51s.  Time for last 10,000:    0s.  Last read position: 14:90,998,817\n",
      "INFO\t2017-01-12 16:25:45\tGenotypeConcordance\tchecked    26,060,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 14:91,845,588\n",
      "INFO\t2017-01-12 16:25:46\tGenotypeConcordance\tchecked    26,070,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 14:92,692,952\n",
      "INFO\t2017-01-12 16:25:46\tGenotypeConcordance\tchecked    26,080,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 14:93,461,185\n",
      "INFO\t2017-01-12 16:25:46\tGenotypeConcordance\tchecked    26,090,000 variants.  Elapsed time: 00:12:52s.  Time for last 10,000:    0s.  Last read position: 14:94,338,490\n",
      "INFO\t2017-01-12 16:25:47\tGenotypeConcordance\tchecked    26,100,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 14:95,099,611\n",
      "INFO\t2017-01-12 16:25:47\tGenotypeConcordance\tchecked    26,110,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 14:95,831,101\n",
      "INFO\t2017-01-12 16:25:47\tGenotypeConcordance\tchecked    26,120,000 variants.  Elapsed time: 00:12:53s.  Time for last 10,000:    0s.  Last read position: 14:96,525,596\n",
      "INFO\t2017-01-12 16:25:48\tGenotypeConcordance\tchecked    26,130,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 14:97,350,173\n",
      "INFO\t2017-01-12 16:25:48\tGenotypeConcordance\tchecked    26,140,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 14:98,181,180\n",
      "INFO\t2017-01-12 16:25:48\tGenotypeConcordance\tchecked    26,150,000 variants.  Elapsed time: 00:12:54s.  Time for last 10,000:    0s.  Last read position: 14:98,901,493\n",
      "INFO\t2017-01-12 16:25:49\tGenotypeConcordance\tchecked    26,160,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 14:99,647,028\n",
      "INFO\t2017-01-12 16:25:49\tGenotypeConcordance\tchecked    26,170,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 14:100,483,129\n",
      "INFO\t2017-01-12 16:25:49\tGenotypeConcordance\tchecked    26,180,000 variants.  Elapsed time: 00:12:55s.  Time for last 10,000:    0s.  Last read position: 14:101,306,685\n",
      "INFO\t2017-01-12 16:25:50\tGenotypeConcordance\tchecked    26,190,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 14:102,126,242\n",
      "INFO\t2017-01-12 16:25:50\tGenotypeConcordance\tchecked    26,200,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 14:102,987,286\n",
      "INFO\t2017-01-12 16:25:50\tGenotypeConcordance\tchecked    26,210,000 variants.  Elapsed time: 00:12:56s.  Time for last 10,000:    0s.  Last read position: 14:103,772,717\n",
      "INFO\t2017-01-12 16:25:50\tGenotypeConcordance\tchecked    26,220,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 14:104,587,869\n",
      "INFO\t2017-01-12 16:25:51\tGenotypeConcordance\tchecked    26,230,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 14:105,236,672\n",
      "INFO\t2017-01-12 16:25:51\tGenotypeConcordance\tchecked    26,240,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 14:105,978,010\n",
      "INFO\t2017-01-12 16:25:51\tGenotypeConcordance\tchecked    26,250,000 variants.  Elapsed time: 00:12:57s.  Time for last 10,000:    0s.  Last read position: 14:106,728,317\n",
      "INFO\t2017-01-12 16:25:52\tGenotypeConcordance\tchecked    26,260,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 15:20,019,922\n",
      "INFO\t2017-01-12 16:25:52\tGenotypeConcordance\tchecked    26,270,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 15:20,492,132\n",
      "INFO\t2017-01-12 16:25:52\tGenotypeConcordance\tchecked    26,280,000 variants.  Elapsed time: 00:12:58s.  Time for last 10,000:    0s.  Last read position: 15:21,887,545\n",
      "INFO\t2017-01-12 16:25:53\tGenotypeConcordance\tchecked    26,290,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 15:22,576,643\n",
      "INFO\t2017-01-12 16:25:53\tGenotypeConcordance\tchecked    26,300,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 15:23,506,879\n",
      "INFO\t2017-01-12 16:25:53\tGenotypeConcordance\tchecked    26,310,000 variants.  Elapsed time: 00:12:59s.  Time for last 10,000:    0s.  Last read position: 15:24,320,566\n",
      "INFO\t2017-01-12 16:25:54\tGenotypeConcordance\tchecked    26,320,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 15:24,758,125\n",
      "INFO\t2017-01-12 16:25:54\tGenotypeConcordance\tchecked    26,330,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 15:25,391,168\n",
      "INFO\t2017-01-12 16:25:54\tGenotypeConcordance\tchecked    26,340,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 15:26,145,292\n",
      "INFO\t2017-01-12 16:25:54\tGenotypeConcordance\tchecked    26,350,000 variants.  Elapsed time: 00:13:00s.  Time for last 10,000:    0s.  Last read position: 15:26,880,157\n",
      "INFO\t2017-01-12 16:25:55\tGenotypeConcordance\tchecked    26,360,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 15:27,682,851\n",
      "INFO\t2017-01-12 16:25:55\tGenotypeConcordance\tchecked    26,370,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 15:28,481,743\n",
      "INFO\t2017-01-12 16:25:55\tGenotypeConcordance\tchecked    26,380,000 variants.  Elapsed time: 00:13:01s.  Time for last 10,000:    0s.  Last read position: 15:29,454,669\n",
      "INFO\t2017-01-12 16:25:56\tGenotypeConcordance\tchecked    26,390,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 15:30,269,669\n",
      "INFO\t2017-01-12 16:25:56\tGenotypeConcordance\tchecked    26,400,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 15:31,427,069\n",
      "INFO\t2017-01-12 16:25:56\tGenotypeConcordance\tchecked    26,410,000 variants.  Elapsed time: 00:13:02s.  Time for last 10,000:    0s.  Last read position: 15:32,255,382\n",
      "INFO\t2017-01-12 16:25:56\tGenotypeConcordance\tchecked    26,420,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 15:33,259,773\n",
      "INFO\t2017-01-12 16:25:57\tGenotypeConcordance\tchecked    26,430,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 15:33,950,236\n",
      "INFO\t2017-01-12 16:25:57\tGenotypeConcordance\tchecked    26,440,000 variants.  Elapsed time: 00:13:03s.  Time for last 10,000:    0s.  Last read position: 15:34,689,957\n",
      "INFO\t2017-01-12 16:25:57\tGenotypeConcordance\tchecked    26,450,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 15:35,516,664\n",
      "INFO\t2017-01-12 16:25:58\tGenotypeConcordance\tchecked    26,460,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 15:36,481,940\n",
      "INFO\t2017-01-12 16:25:58\tGenotypeConcordance\tchecked    26,470,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 15:37,395,212\n",
      "INFO\t2017-01-12 16:25:58\tGenotypeConcordance\tchecked    26,480,000 variants.  Elapsed time: 00:13:04s.  Time for last 10,000:    0s.  Last read position: 15:38,231,284\n",
      "INFO\t2017-01-12 16:25:59\tGenotypeConcordance\tchecked    26,490,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 15:39,119,986\n",
      "INFO\t2017-01-12 16:25:59\tGenotypeConcordance\tchecked    26,500,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 15:39,845,432\n",
      "INFO\t2017-01-12 16:25:59\tGenotypeConcordance\tchecked    26,510,000 variants.  Elapsed time: 00:13:05s.  Time for last 10,000:    0s.  Last read position: 15:40,747,604\n",
      "INFO\t2017-01-12 16:26:00\tGenotypeConcordance\tchecked    26,520,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 15:41,674,132\n",
      "INFO\t2017-01-12 16:26:00\tGenotypeConcordance\tchecked    26,530,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 15:42,589,729\n",
      "INFO\t2017-01-12 16:26:00\tGenotypeConcordance\tchecked    26,540,000 variants.  Elapsed time: 00:13:06s.  Time for last 10,000:    0s.  Last read position: 15:43,677,568\n",
      "INFO\t2017-01-12 16:26:01\tGenotypeConcordance\tchecked    26,550,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 15:44,858,941\n",
      "INFO\t2017-01-12 16:26:01\tGenotypeConcordance\tchecked    26,560,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 15:45,749,030\n",
      "INFO\t2017-01-12 16:26:01\tGenotypeConcordance\tchecked    26,570,000 variants.  Elapsed time: 00:13:07s.  Time for last 10,000:    0s.  Last read position: 15:46,574,981\n",
      "INFO\t2017-01-12 16:26:02\tGenotypeConcordance\tchecked    26,580,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 15:47,420,677\n",
      "INFO\t2017-01-12 16:26:02\tGenotypeConcordance\tchecked    26,590,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 15:48,354,993\n",
      "INFO\t2017-01-12 16:26:02\tGenotypeConcordance\tchecked    26,600,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 15:49,460,678\n",
      "INFO\t2017-01-12 16:26:02\tGenotypeConcordance\tchecked    26,610,000 variants.  Elapsed time: 00:13:08s.  Time for last 10,000:    0s.  Last read position: 15:50,437,483\n",
      "INFO\t2017-01-12 16:26:03\tGenotypeConcordance\tchecked    26,620,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 15:51,287,916\n",
      "INFO\t2017-01-12 16:26:03\tGenotypeConcordance\tchecked    26,630,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 15:52,259,467\n",
      "INFO\t2017-01-12 16:26:03\tGenotypeConcordance\tchecked    26,640,000 variants.  Elapsed time: 00:13:09s.  Time for last 10,000:    0s.  Last read position: 15:53,111,692\n",
      "INFO\t2017-01-12 16:26:04\tGenotypeConcordance\tchecked    26,650,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 15:53,876,637\n",
      "INFO\t2017-01-12 16:26:04\tGenotypeConcordance\tchecked    26,660,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 15:54,578,397\n",
      "INFO\t2017-01-12 16:26:04\tGenotypeConcordance\tchecked    26,670,000 variants.  Elapsed time: 00:13:10s.  Time for last 10,000:    0s.  Last read position: 15:55,257,869\n",
      "INFO\t2017-01-12 16:26:05\tGenotypeConcordance\tchecked    26,680,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 15:56,040,341\n",
      "INFO\t2017-01-12 16:26:05\tGenotypeConcordance\tchecked    26,690,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 15:56,866,113\n",
      "INFO\t2017-01-12 16:26:05\tGenotypeConcordance\tchecked    26,700,000 variants.  Elapsed time: 00:13:11s.  Time for last 10,000:    0s.  Last read position: 15:57,762,045\n",
      "INFO\t2017-01-12 16:26:05\tGenotypeConcordance\tchecked    26,710,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 15:58,473,395\n",
      "INFO\t2017-01-12 16:26:06\tGenotypeConcordance\tchecked    26,720,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 15:59,270,634\n",
      "INFO\t2017-01-12 16:26:06\tGenotypeConcordance\tchecked    26,730,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 15:60,013,216\n",
      "INFO\t2017-01-12 16:26:06\tGenotypeConcordance\tchecked    26,740,000 variants.  Elapsed time: 00:13:12s.  Time for last 10,000:    0s.  Last read position: 15:60,779,014\n",
      "INFO\t2017-01-12 16:26:07\tGenotypeConcordance\tchecked    26,750,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 15:61,639,757\n",
      "INFO\t2017-01-12 16:26:07\tGenotypeConcordance\tchecked    26,760,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 15:62,471,025\n",
      "INFO\t2017-01-12 16:26:07\tGenotypeConcordance\tchecked    26,770,000 variants.  Elapsed time: 00:13:13s.  Time for last 10,000:    0s.  Last read position: 15:63,249,167\n",
      "INFO\t2017-01-12 16:26:08\tGenotypeConcordance\tchecked    26,780,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 15:64,141,036\n",
      "INFO\t2017-01-12 16:26:08\tGenotypeConcordance\tchecked    26,790,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 15:65,114,039\n",
      "INFO\t2017-01-12 16:26:08\tGenotypeConcordance\tchecked    26,800,000 variants.  Elapsed time: 00:13:14s.  Time for last 10,000:    0s.  Last read position: 15:66,020,432\n",
      "INFO\t2017-01-12 16:26:08\tGenotypeConcordance\tchecked    26,810,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 15:66,861,241\n",
      "INFO\t2017-01-12 16:26:09\tGenotypeConcordance\tchecked    26,820,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 15:67,714,571\n",
      "INFO\t2017-01-12 16:26:09\tGenotypeConcordance\tchecked    26,830,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 15:68,671,717\n",
      "INFO\t2017-01-12 16:26:09\tGenotypeConcordance\tchecked    26,840,000 variants.  Elapsed time: 00:13:15s.  Time for last 10,000:    0s.  Last read position: 15:69,559,255\n",
      "INFO\t2017-01-12 16:26:10\tGenotypeConcordance\tchecked    26,850,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 15:70,435,947\n",
      "INFO\t2017-01-12 16:26:10\tGenotypeConcordance\tchecked    26,860,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 15:71,437,333\n",
      "INFO\t2017-01-12 16:26:10\tGenotypeConcordance\tchecked    26,870,000 variants.  Elapsed time: 00:13:16s.  Time for last 10,000:    0s.  Last read position: 15:72,287,902\n",
      "INFO\t2017-01-12 16:26:11\tGenotypeConcordance\tchecked    26,880,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 15:73,395,364\n",
      "INFO\t2017-01-12 16:26:11\tGenotypeConcordance\tchecked    26,890,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 15:74,357,244\n",
      "INFO\t2017-01-12 16:26:11\tGenotypeConcordance\tchecked    26,900,000 variants.  Elapsed time: 00:13:17s.  Time for last 10,000:    0s.  Last read position: 15:75,449,828\n",
      "INFO\t2017-01-12 16:26:11\tGenotypeConcordance\tchecked    26,910,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 15:76,482,077\n",
      "INFO\t2017-01-12 16:26:12\tGenotypeConcordance\tchecked    26,920,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 15:77,484,477\n",
      "INFO\t2017-01-12 16:26:12\tGenotypeConcordance\tchecked    26,930,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 15:78,275,500\n",
      "INFO\t2017-01-12 16:26:12\tGenotypeConcordance\tchecked    26,940,000 variants.  Elapsed time: 00:13:18s.  Time for last 10,000:    0s.  Last read position: 15:79,099,537\n",
      "INFO\t2017-01-12 16:26:13\tGenotypeConcordance\tchecked    26,950,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 15:79,864,072\n",
      "INFO\t2017-01-12 16:26:13\tGenotypeConcordance\tchecked    26,960,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 15:80,624,809\n",
      "INFO\t2017-01-12 16:26:13\tGenotypeConcordance\tchecked    26,970,000 variants.  Elapsed time: 00:13:19s.  Time for last 10,000:    0s.  Last read position: 15:81,466,197\n",
      "INFO\t2017-01-12 16:26:14\tGenotypeConcordance\tchecked    26,980,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 15:82,366,461\n",
      "INFO\t2017-01-12 16:26:14\tGenotypeConcordance\tchecked    26,990,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 15:83,940,733\n",
      "INFO\t2017-01-12 16:26:14\tGenotypeConcordance\tchecked    27,000,000 variants.  Elapsed time: 00:13:20s.  Time for last 10,000:    0s.  Last read position: 15:84,956,557\n",
      "INFO\t2017-01-12 16:26:14\tGenotypeConcordance\tchecked    27,010,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 15:85,985,755\n",
      "INFO\t2017-01-12 16:26:15\tGenotypeConcordance\tchecked    27,020,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 15:86,721,032\n",
      "INFO\t2017-01-12 16:26:15\tGenotypeConcordance\tchecked    27,030,000 variants.  Elapsed time: 00:13:21s.  Time for last 10,000:    0s.  Last read position: 15:87,402,722\n",
      "INFO\t2017-01-12 16:26:15\tGenotypeConcordance\tchecked    27,040,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 15:88,075,564\n",
      "INFO\t2017-01-12 16:26:16\tGenotypeConcordance\tchecked    27,050,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 15:88,878,123\n",
      "INFO\t2017-01-12 16:26:16\tGenotypeConcordance\tchecked    27,060,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 15:89,588,694\n",
      "INFO\t2017-01-12 16:26:16\tGenotypeConcordance\tchecked    27,070,000 variants.  Elapsed time: 00:13:22s.  Time for last 10,000:    0s.  Last read position: 15:90,347,814\n",
      "INFO\t2017-01-12 16:26:17\tGenotypeConcordance\tchecked    27,080,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 15:91,122,156\n",
      "INFO\t2017-01-12 16:26:17\tGenotypeConcordance\tchecked    27,090,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 15:91,860,260\n",
      "INFO\t2017-01-12 16:26:17\tGenotypeConcordance\tchecked    27,100,000 variants.  Elapsed time: 00:13:23s.  Time for last 10,000:    0s.  Last read position: 15:92,609,246\n",
      "INFO\t2017-01-12 16:26:17\tGenotypeConcordance\tchecked    27,110,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 15:93,310,705\n",
      "INFO\t2017-01-12 16:26:18\tGenotypeConcordance\tchecked    27,120,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 15:93,972,367\n",
      "INFO\t2017-01-12 16:26:18\tGenotypeConcordance\tchecked    27,130,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 15:94,735,128\n",
      "INFO\t2017-01-12 16:26:18\tGenotypeConcordance\tchecked    27,140,000 variants.  Elapsed time: 00:13:24s.  Time for last 10,000:    0s.  Last read position: 15:95,504,426\n",
      "INFO\t2017-01-12 16:26:19\tGenotypeConcordance\tchecked    27,150,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 15:96,372,089\n",
      "INFO\t2017-01-12 16:26:19\tGenotypeConcordance\tchecked    27,160,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 15:97,247,665\n",
      "INFO\t2017-01-12 16:26:19\tGenotypeConcordance\tchecked    27,170,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 15:98,033,244\n",
      "INFO\t2017-01-12 16:26:19\tGenotypeConcordance\tchecked    27,180,000 variants.  Elapsed time: 00:13:25s.  Time for last 10,000:    0s.  Last read position: 15:98,729,165\n",
      "INFO\t2017-01-12 16:26:20\tGenotypeConcordance\tchecked    27,190,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 15:99,450,259\n",
      "INFO\t2017-01-12 16:26:20\tGenotypeConcordance\tchecked    27,200,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 15:100,260,501\n",
      "INFO\t2017-01-12 16:26:20\tGenotypeConcordance\tchecked    27,210,000 variants.  Elapsed time: 00:13:26s.  Time for last 10,000:    0s.  Last read position: 15:100,826,957\n",
      "INFO\t2017-01-12 16:26:20\tGenotypeConcordance\tchecked    27,220,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 15:101,499,504\n",
      "INFO\t2017-01-12 16:26:21\tGenotypeConcordance\tchecked    27,230,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 15:102,117,423\n",
      "INFO\t2017-01-12 16:26:21\tGenotypeConcordance\tchecked    27,240,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 16:370,074\n",
      "INFO\t2017-01-12 16:26:21\tGenotypeConcordance\tchecked    27,250,000 variants.  Elapsed time: 00:13:27s.  Time for last 10,000:    0s.  Last read position: 16:941,628\n",
      "INFO\t2017-01-12 16:26:22\tGenotypeConcordance\tchecked    27,260,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 16:1,448,751\n",
      "INFO\t2017-01-12 16:26:22\tGenotypeConcordance\tchecked    27,270,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 16:2,117,260\n",
      "INFO\t2017-01-12 16:26:22\tGenotypeConcordance\tchecked    27,280,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 16:2,970,999\n",
      "INFO\t2017-01-12 16:26:22\tGenotypeConcordance\tchecked    27,290,000 variants.  Elapsed time: 00:13:28s.  Time for last 10,000:    0s.  Last read position: 16:3,651,424\n",
      "INFO\t2017-01-12 16:26:23\tGenotypeConcordance\tchecked    27,300,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 16:4,358,039\n",
      "INFO\t2017-01-12 16:26:23\tGenotypeConcordance\tchecked    27,310,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 16:5,035,619\n",
      "INFO\t2017-01-12 16:26:23\tGenotypeConcordance\tchecked    27,320,000 variants.  Elapsed time: 00:13:29s.  Time for last 10,000:    0s.  Last read position: 16:5,556,417\n",
      "INFO\t2017-01-12 16:26:23\tGenotypeConcordance\tchecked    27,330,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 16:6,039,092\n",
      "INFO\t2017-01-12 16:26:24\tGenotypeConcordance\tchecked    27,340,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 16:6,565,511\n",
      "INFO\t2017-01-12 16:26:24\tGenotypeConcordance\tchecked    27,350,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 16:6,938,233\n",
      "INFO\t2017-01-12 16:26:24\tGenotypeConcordance\tchecked    27,360,000 variants.  Elapsed time: 00:13:30s.  Time for last 10,000:    0s.  Last read position: 16:7,277,983\n",
      "INFO\t2017-01-12 16:26:25\tGenotypeConcordance\tchecked    27,370,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 16:7,774,406\n",
      "INFO\t2017-01-12 16:26:25\tGenotypeConcordance\tchecked    27,380,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 16:8,292,694\n",
      "INFO\t2017-01-12 16:26:25\tGenotypeConcordance\tchecked    27,390,000 variants.  Elapsed time: 00:13:31s.  Time for last 10,000:    0s.  Last read position: 16:8,826,064\n",
      "INFO\t2017-01-12 16:26:25\tGenotypeConcordance\tchecked    27,400,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 16:9,385,760\n",
      "INFO\t2017-01-12 16:26:26\tGenotypeConcordance\tchecked    27,410,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 16:10,073,295\n",
      "INFO\t2017-01-12 16:26:26\tGenotypeConcordance\tchecked    27,420,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 16:10,700,928\n",
      "INFO\t2017-01-12 16:26:26\tGenotypeConcordance\tchecked    27,430,000 variants.  Elapsed time: 00:13:32s.  Time for last 10,000:    0s.  Last read position: 16:11,369,940\n",
      "INFO\t2017-01-12 16:26:27\tGenotypeConcordance\tchecked    27,440,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 16:11,985,410\n",
      "INFO\t2017-01-12 16:26:27\tGenotypeConcordance\tchecked    27,450,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 16:12,631,768\n",
      "INFO\t2017-01-12 16:26:27\tGenotypeConcordance\tchecked    27,460,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 16:13,091,656\n",
      "INFO\t2017-01-12 16:26:27\tGenotypeConcordance\tchecked    27,470,000 variants.  Elapsed time: 00:13:33s.  Time for last 10,000:    0s.  Last read position: 16:13,787,547\n",
      "INFO\t2017-01-12 16:26:28\tGenotypeConcordance\tchecked    27,480,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 16:14,624,301\n",
      "INFO\t2017-01-12 16:26:28\tGenotypeConcordance\tchecked    27,490,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 16:15,823,169\n",
      "INFO\t2017-01-12 16:26:28\tGenotypeConcordance\tchecked    27,500,000 variants.  Elapsed time: 00:13:34s.  Time for last 10,000:    0s.  Last read position: 16:16,738,140\n",
      "INFO\t2017-01-12 16:26:29\tGenotypeConcordance\tchecked    27,510,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 16:17,595,028\n",
      "INFO\t2017-01-12 16:26:29\tGenotypeConcordance\tchecked    27,520,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 16:18,383,160\n",
      "INFO\t2017-01-12 16:26:29\tGenotypeConcordance\tchecked    27,530,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 16:19,408,637\n",
      "INFO\t2017-01-12 16:26:29\tGenotypeConcordance\tchecked    27,540,000 variants.  Elapsed time: 00:13:35s.  Time for last 10,000:    0s.  Last read position: 16:20,258,451\n",
      "INFO\t2017-01-12 16:26:30\tGenotypeConcordance\tchecked    27,550,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 16:21,163,690\n",
      "INFO\t2017-01-12 16:26:30\tGenotypeConcordance\tchecked    27,560,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 16:22,574,512\n",
      "INFO\t2017-01-12 16:26:30\tGenotypeConcordance\tchecked    27,570,000 variants.  Elapsed time: 00:13:36s.  Time for last 10,000:    0s.  Last read position: 16:23,488,031\n",
      "INFO\t2017-01-12 16:26:30\tGenotypeConcordance\tchecked    27,580,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 16:24,351,328\n",
      "INFO\t2017-01-12 16:26:31\tGenotypeConcordance\tchecked    27,590,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 16:25,166,621\n",
      "INFO\t2017-01-12 16:26:31\tGenotypeConcordance\tchecked    27,600,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 16:25,976,419\n",
      "INFO\t2017-01-12 16:26:31\tGenotypeConcordance\tchecked    27,610,000 variants.  Elapsed time: 00:13:37s.  Time for last 10,000:    0s.  Last read position: 16:26,731,700\n",
      "INFO\t2017-01-12 16:26:32\tGenotypeConcordance\tchecked    27,620,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 16:27,461,941\n",
      "INFO\t2017-01-12 16:26:32\tGenotypeConcordance\tchecked    27,630,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 16:28,490,200\n",
      "INFO\t2017-01-12 16:26:32\tGenotypeConcordance\tchecked    27,640,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 16:29,589,556\n",
      "INFO\t2017-01-12 16:26:32\tGenotypeConcordance\tchecked    27,650,000 variants.  Elapsed time: 00:13:38s.  Time for last 10,000:    0s.  Last read position: 16:30,930,224\n",
      "INFO\t2017-01-12 16:26:33\tGenotypeConcordance\tchecked    27,660,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 16:32,034,627\n",
      "INFO\t2017-01-12 16:26:33\tGenotypeConcordance\tchecked    27,670,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 16:32,491,794\n",
      "INFO\t2017-01-12 16:26:33\tGenotypeConcordance\tchecked    27,680,000 variants.  Elapsed time: 00:13:39s.  Time for last 10,000:    0s.  Last read position: 16:33,049,162\n",
      "INFO\t2017-01-12 16:26:33\tGenotypeConcordance\tchecked    27,690,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 16:33,664,589\n",
      "INFO\t2017-01-12 16:26:34\tGenotypeConcordance\tchecked    27,700,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 16:33,986,076\n",
      "INFO\t2017-01-12 16:26:34\tGenotypeConcordance\tchecked    27,710,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 16:34,850,281\n",
      "INFO\t2017-01-12 16:26:34\tGenotypeConcordance\tchecked    27,720,000 variants.  Elapsed time: 00:13:40s.  Time for last 10,000:    0s.  Last read position: 16:46,477,222\n",
      "INFO\t2017-01-12 16:26:34\tGenotypeConcordance\tchecked    27,730,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 16:47,784,203\n",
      "INFO\t2017-01-12 16:26:35\tGenotypeConcordance\tchecked    27,740,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 16:48,834,737\n",
      "INFO\t2017-01-12 16:26:35\tGenotypeConcordance\tchecked    27,750,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 16:49,721,942\n",
      "INFO\t2017-01-12 16:26:35\tGenotypeConcordance\tchecked    27,760,000 variants.  Elapsed time: 00:13:41s.  Time for last 10,000:    0s.  Last read position: 16:50,636,647\n",
      "INFO\t2017-01-12 16:26:35\tGenotypeConcordance\tchecked    27,770,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 16:51,510,060\n",
      "INFO\t2017-01-12 16:26:36\tGenotypeConcordance\tchecked    27,780,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 16:52,342,550\n",
      "INFO\t2017-01-12 16:26:36\tGenotypeConcordance\tchecked    27,790,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 16:53,166,187\n",
      "INFO\t2017-01-12 16:26:36\tGenotypeConcordance\tchecked    27,800,000 variants.  Elapsed time: 00:13:42s.  Time for last 10,000:    0s.  Last read position: 16:54,126,444\n",
      "INFO\t2017-01-12 16:26:36\tGenotypeConcordance\tchecked    27,810,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 16:54,886,126\n",
      "INFO\t2017-01-12 16:26:37\tGenotypeConcordance\tchecked    27,820,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 16:55,658,747\n",
      "INFO\t2017-01-12 16:26:37\tGenotypeConcordance\tchecked    27,830,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 16:56,440,037\n",
      "INFO\t2017-01-12 16:26:37\tGenotypeConcordance\tchecked    27,840,000 variants.  Elapsed time: 00:13:43s.  Time for last 10,000:    0s.  Last read position: 16:57,251,503\n",
      "INFO\t2017-01-12 16:26:38\tGenotypeConcordance\tchecked    27,850,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 16:58,044,594\n",
      "INFO\t2017-01-12 16:26:38\tGenotypeConcordance\tchecked    27,860,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 16:58,837,767\n",
      "INFO\t2017-01-12 16:26:38\tGenotypeConcordance\tchecked    27,870,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 16:59,710,112\n",
      "INFO\t2017-01-12 16:26:38\tGenotypeConcordance\tchecked    27,880,000 variants.  Elapsed time: 00:13:44s.  Time for last 10,000:    0s.  Last read position: 16:60,598,004\n",
      "INFO\t2017-01-12 16:26:39\tGenotypeConcordance\tchecked    27,890,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 16:61,588,986\n",
      "INFO\t2017-01-12 16:26:39\tGenotypeConcordance\tchecked    27,900,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 16:62,507,747\n",
      "INFO\t2017-01-12 16:26:39\tGenotypeConcordance\tchecked    27,910,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 16:63,375,745\n",
      "INFO\t2017-01-12 16:26:39\tGenotypeConcordance\tchecked    27,920,000 variants.  Elapsed time: 00:13:45s.  Time for last 10,000:    0s.  Last read position: 16:64,296,779\n",
      "INFO\t2017-01-12 16:26:40\tGenotypeConcordance\tchecked    27,930,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 16:65,173,715\n",
      "INFO\t2017-01-12 16:26:40\tGenotypeConcordance\tchecked    27,940,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 16:66,016,188\n",
      "INFO\t2017-01-12 16:26:40\tGenotypeConcordance\tchecked    27,950,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 16:67,048,064\n",
      "INFO\t2017-01-12 16:26:40\tGenotypeConcordance\tchecked    27,960,000 variants.  Elapsed time: 00:13:46s.  Time for last 10,000:    0s.  Last read position: 16:68,343,793\n",
      "INFO\t2017-01-12 16:26:41\tGenotypeConcordance\tchecked    27,970,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 16:69,356,819\n",
      "INFO\t2017-01-12 16:26:41\tGenotypeConcordance\tchecked    27,980,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 16:70,296,899\n",
      "INFO\t2017-01-12 16:26:41\tGenotypeConcordance\tchecked    27,990,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 16:71,335,258\n",
      "INFO\t2017-01-12 16:26:41\tGenotypeConcordance\tchecked    28,000,000 variants.  Elapsed time: 00:13:47s.  Time for last 10,000:    0s.  Last read position: 16:72,355,139\n",
      "INFO\t2017-01-12 16:26:42\tGenotypeConcordance\tchecked    28,010,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 16:73,332,747\n",
      "INFO\t2017-01-12 16:26:42\tGenotypeConcordance\tchecked    28,020,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 16:74,102,050\n",
      "INFO\t2017-01-12 16:26:42\tGenotypeConcordance\tchecked    28,030,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 16:74,877,135\n",
      "INFO\t2017-01-12 16:26:42\tGenotypeConcordance\tchecked    28,040,000 variants.  Elapsed time: 00:13:48s.  Time for last 10,000:    0s.  Last read position: 16:75,537,283\n",
      "INFO\t2017-01-12 16:26:43\tGenotypeConcordance\tchecked    28,050,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 16:76,276,902\n",
      "INFO\t2017-01-12 16:26:43\tGenotypeConcordance\tchecked    28,060,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 16:76,930,088\n",
      "INFO\t2017-01-12 16:26:43\tGenotypeConcordance\tchecked    28,070,000 variants.  Elapsed time: 00:13:49s.  Time for last 10,000:    0s.  Last read position: 16:77,545,392\n",
      "INFO\t2017-01-12 16:26:43\tGenotypeConcordance\tchecked    28,080,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 16:78,123,591\n",
      "INFO\t2017-01-12 16:26:44\tGenotypeConcordance\tchecked    28,090,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 16:78,581,022\n",
      "INFO\t2017-01-12 16:26:44\tGenotypeConcordance\tchecked    28,100,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 16:78,986,468\n",
      "INFO\t2017-01-12 16:26:44\tGenotypeConcordance\tchecked    28,110,000 variants.  Elapsed time: 00:13:50s.  Time for last 10,000:    0s.  Last read position: 16:79,484,853\n",
      "INFO\t2017-01-12 16:26:44\tGenotypeConcordance\tchecked    28,120,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 16:80,027,021\n",
      "INFO\t2017-01-12 16:26:45\tGenotypeConcordance\tchecked    28,130,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 16:80,603,749\n",
      "INFO\t2017-01-12 16:26:45\tGenotypeConcordance\tchecked    28,140,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 16:81,170,121\n",
      "INFO\t2017-01-12 16:26:45\tGenotypeConcordance\tchecked    28,150,000 variants.  Elapsed time: 00:13:51s.  Time for last 10,000:    0s.  Last read position: 16:81,751,183\n",
      "INFO\t2017-01-12 16:26:45\tGenotypeConcordance\tchecked    28,160,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 16:82,280,060\n",
      "INFO\t2017-01-12 16:26:46\tGenotypeConcordance\tchecked    28,170,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 16:82,818,499\n",
      "INFO\t2017-01-12 16:26:46\tGenotypeConcordance\tchecked    28,180,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 16:83,322,894\n",
      "INFO\t2017-01-12 16:26:46\tGenotypeConcordance\tchecked    28,190,000 variants.  Elapsed time: 00:13:52s.  Time for last 10,000:    0s.  Last read position: 16:83,875,618\n",
      "INFO\t2017-01-12 16:26:46\tGenotypeConcordance\tchecked    28,200,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 16:84,347,646\n",
      "INFO\t2017-01-12 16:26:47\tGenotypeConcordance\tchecked    28,210,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 16:84,838,816\n",
      "INFO\t2017-01-12 16:26:47\tGenotypeConcordance\tchecked    28,220,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 16:85,394,614\n",
      "INFO\t2017-01-12 16:26:47\tGenotypeConcordance\tchecked    28,230,000 variants.  Elapsed time: 00:13:53s.  Time for last 10,000:    0s.  Last read position: 16:85,985,535\n",
      "INFO\t2017-01-12 16:26:47\tGenotypeConcordance\tchecked    28,240,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 16:86,487,208\n",
      "INFO\t2017-01-12 16:26:48\tGenotypeConcordance\tchecked    28,250,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 16:87,021,168\n",
      "INFO\t2017-01-12 16:26:48\tGenotypeConcordance\tchecked    28,260,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 16:87,626,154\n",
      "INFO\t2017-01-12 16:26:48\tGenotypeConcordance\tchecked    28,270,000 variants.  Elapsed time: 00:13:54s.  Time for last 10,000:    0s.  Last read position: 16:88,131,535\n",
      "INFO\t2017-01-12 16:26:48\tGenotypeConcordance\tchecked    28,280,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 16:88,699,386\n",
      "INFO\t2017-01-12 16:26:49\tGenotypeConcordance\tchecked    28,290,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 16:89,206,841\n",
      "INFO\t2017-01-12 16:26:49\tGenotypeConcordance\tchecked    28,300,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 16:89,801,920\n",
      "INFO\t2017-01-12 16:26:49\tGenotypeConcordance\tchecked    28,310,000 variants.  Elapsed time: 00:13:55s.  Time for last 10,000:    0s.  Last read position: 17:168,015\n",
      "INFO\t2017-01-12 16:26:49\tGenotypeConcordance\tchecked    28,320,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 17:853,031\n",
      "INFO\t2017-01-12 16:26:50\tGenotypeConcordance\tchecked    28,330,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 17:1,421,182\n",
      "INFO\t2017-01-12 16:26:50\tGenotypeConcordance\tchecked    28,340,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 17:2,203,408\n",
      "INFO\t2017-01-12 16:26:50\tGenotypeConcordance\tchecked    28,350,000 variants.  Elapsed time: 00:13:56s.  Time for last 10,000:    0s.  Last read position: 17:2,946,054\n",
      "INFO\t2017-01-12 16:26:51\tGenotypeConcordance\tchecked    28,360,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 17:3,637,453\n",
      "INFO\t2017-01-12 16:26:51\tGenotypeConcordance\tchecked    28,370,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 17:4,371,113\n",
      "INFO\t2017-01-12 16:26:51\tGenotypeConcordance\tchecked    28,380,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 17:5,140,515\n",
      "INFO\t2017-01-12 16:26:51\tGenotypeConcordance\tchecked    28,390,000 variants.  Elapsed time: 00:13:57s.  Time for last 10,000:    0s.  Last read position: 17:5,868,104\n",
      "INFO\t2017-01-12 16:26:52\tGenotypeConcordance\tchecked    28,400,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 17:6,594,343\n",
      "INFO\t2017-01-12 16:26:52\tGenotypeConcordance\tchecked    28,410,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 17:7,394,337\n",
      "INFO\t2017-01-12 16:26:52\tGenotypeConcordance\tchecked    28,420,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 17:8,258,055\n",
      "INFO\t2017-01-12 16:26:52\tGenotypeConcordance\tchecked    28,430,000 variants.  Elapsed time: 00:13:58s.  Time for last 10,000:    0s.  Last read position: 17:9,083,200\n",
      "INFO\t2017-01-12 16:26:53\tGenotypeConcordance\tchecked    28,440,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 17:9,796,111\n",
      "INFO\t2017-01-12 16:26:53\tGenotypeConcordance\tchecked    28,450,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 17:10,568,838\n",
      "INFO\t2017-01-12 16:26:53\tGenotypeConcordance\tchecked    28,460,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 17:11,343,068\n",
      "INFO\t2017-01-12 16:26:53\tGenotypeConcordance\tchecked    28,470,000 variants.  Elapsed time: 00:13:59s.  Time for last 10,000:    0s.  Last read position: 17:12,170,535\n",
      "INFO\t2017-01-12 16:26:54\tGenotypeConcordance\tchecked    28,480,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 17:12,925,095\n",
      "INFO\t2017-01-12 16:26:54\tGenotypeConcordance\tchecked    28,490,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 17:13,609,487\n",
      "INFO\t2017-01-12 16:26:54\tGenotypeConcordance\tchecked    28,500,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 17:14,344,552\n",
      "INFO\t2017-01-12 16:26:54\tGenotypeConcordance\tchecked    28,510,000 variants.  Elapsed time: 00:14:00s.  Time for last 10,000:    0s.  Last read position: 17:14,992,958\n",
      "INFO\t2017-01-12 16:26:55\tGenotypeConcordance\tchecked    28,520,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 17:15,755,824\n",
      "INFO\t2017-01-12 16:26:55\tGenotypeConcordance\tchecked    28,530,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 17:16,621,175\n",
      "INFO\t2017-01-12 16:26:55\tGenotypeConcordance\tchecked    28,540,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 17:17,467,357\n",
      "INFO\t2017-01-12 16:26:55\tGenotypeConcordance\tchecked    28,550,000 variants.  Elapsed time: 00:14:01s.  Time for last 10,000:    0s.  Last read position: 17:18,466,336\n",
      "INFO\t2017-01-12 16:26:56\tGenotypeConcordance\tchecked    28,560,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 17:19,626,546\n",
      "INFO\t2017-01-12 16:26:56\tGenotypeConcordance\tchecked    28,570,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 17:20,540,503\n",
      "INFO\t2017-01-12 16:26:56\tGenotypeConcordance\tchecked    28,580,000 variants.  Elapsed time: 00:14:02s.  Time for last 10,000:    0s.  Last read position: 17:21,223,950\n",
      "INFO\t2017-01-12 16:26:56\tGenotypeConcordance\tchecked    28,590,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 17:21,666,993\n",
      "INFO\t2017-01-12 16:26:57\tGenotypeConcordance\tchecked    28,600,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 17:25,266,624\n",
      "INFO\t2017-01-12 16:26:57\tGenotypeConcordance\tchecked    28,610,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 17:25,762,907\n",
      "INFO\t2017-01-12 16:26:57\tGenotypeConcordance\tchecked    28,620,000 variants.  Elapsed time: 00:14:03s.  Time for last 10,000:    0s.  Last read position: 17:26,821,799\n",
      "INFO\t2017-01-12 16:26:57\tGenotypeConcordance\tchecked    28,630,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 17:27,981,897\n",
      "INFO\t2017-01-12 16:26:58\tGenotypeConcordance\tchecked    28,640,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 17:29,076,862\n",
      "INFO\t2017-01-12 16:26:58\tGenotypeConcordance\tchecked    28,650,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 17:29,989,696\n",
      "INFO\t2017-01-12 16:26:58\tGenotypeConcordance\tchecked    28,660,000 variants.  Elapsed time: 00:14:04s.  Time for last 10,000:    0s.  Last read position: 17:30,869,423\n",
      "INFO\t2017-01-12 16:26:58\tGenotypeConcordance\tchecked    28,670,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 17:31,773,020\n",
      "INFO\t2017-01-12 16:26:59\tGenotypeConcordance\tchecked    28,680,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 17:32,580,820\n",
      "INFO\t2017-01-12 16:26:59\tGenotypeConcordance\tchecked    28,690,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 17:33,488,856\n",
      "INFO\t2017-01-12 16:26:59\tGenotypeConcordance\tchecked    28,700,000 variants.  Elapsed time: 00:14:05s.  Time for last 10,000:    0s.  Last read position: 17:34,363,703\n",
      "INFO\t2017-01-12 16:26:59\tGenotypeConcordance\tchecked    28,710,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 17:35,372,194\n",
      "INFO\t2017-01-12 16:27:00\tGenotypeConcordance\tchecked    28,720,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 17:36,387,469\n",
      "INFO\t2017-01-12 16:27:00\tGenotypeConcordance\tchecked    28,730,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 17:37,268,133\n",
      "INFO\t2017-01-12 16:27:00\tGenotypeConcordance\tchecked    28,740,000 variants.  Elapsed time: 00:14:06s.  Time for last 10,000:    0s.  Last read position: 17:38,235,129\n",
      "INFO\t2017-01-12 16:27:00\tGenotypeConcordance\tchecked    28,750,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 17:39,187,406\n",
      "INFO\t2017-01-12 16:27:01\tGenotypeConcordance\tchecked    28,760,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 17:39,924,576\n",
      "INFO\t2017-01-12 16:27:01\tGenotypeConcordance\tchecked    28,770,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 17:41,028,959\n",
      "INFO\t2017-01-12 16:27:01\tGenotypeConcordance\tchecked    28,780,000 variants.  Elapsed time: 00:14:07s.  Time for last 10,000:    0s.  Last read position: 17:41,981,789\n",
      "INFO\t2017-01-12 16:27:02\tGenotypeConcordance\tchecked    28,790,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 17:42,964,475\n",
      "INFO\t2017-01-12 16:27:02\tGenotypeConcordance\tchecked    28,800,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 17:43,868,739\n",
      "INFO\t2017-01-12 16:27:02\tGenotypeConcordance\tchecked    28,810,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 17:44,970,433\n",
      "INFO\t2017-01-12 16:27:02\tGenotypeConcordance\tchecked    28,820,000 variants.  Elapsed time: 00:14:08s.  Time for last 10,000:    0s.  Last read position: 17:45,937,923\n",
      "INFO\t2017-01-12 16:27:03\tGenotypeConcordance\tchecked    28,830,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 17:46,977,468\n",
      "INFO\t2017-01-12 16:27:03\tGenotypeConcordance\tchecked    28,840,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 17:47,944,720\n",
      "INFO\t2017-01-12 16:27:03\tGenotypeConcordance\tchecked    28,850,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 17:48,803,520\n",
      "INFO\t2017-01-12 16:27:03\tGenotypeConcordance\tchecked    28,860,000 variants.  Elapsed time: 00:14:09s.  Time for last 10,000:    0s.  Last read position: 17:49,713,867\n",
      "INFO\t2017-01-12 16:27:04\tGenotypeConcordance\tchecked    28,870,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 17:50,649,633\n",
      "INFO\t2017-01-12 16:27:04\tGenotypeConcordance\tchecked    28,880,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 17:51,487,974\n",
      "INFO\t2017-01-12 16:27:04\tGenotypeConcordance\tchecked    28,890,000 variants.  Elapsed time: 00:14:10s.  Time for last 10,000:    0s.  Last read position: 17:52,341,783\n",
      "INFO\t2017-01-12 16:27:04\tGenotypeConcordance\tchecked    28,900,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 17:53,216,997\n",
      "INFO\t2017-01-12 16:27:05\tGenotypeConcordance\tchecked    28,910,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 17:54,121,113\n",
      "INFO\t2017-01-12 16:27:05\tGenotypeConcordance\tchecked    28,920,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 17:55,063,133\n",
      "INFO\t2017-01-12 16:27:05\tGenotypeConcordance\tchecked    28,930,000 variants.  Elapsed time: 00:14:11s.  Time for last 10,000:    0s.  Last read position: 17:55,897,724\n",
      "INFO\t2017-01-12 16:27:05\tGenotypeConcordance\tchecked    28,940,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 17:57,028,805\n",
      "INFO\t2017-01-12 16:27:06\tGenotypeConcordance\tchecked    28,950,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 17:58,212,750\n",
      "INFO\t2017-01-12 16:27:06\tGenotypeConcordance\tchecked    28,960,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 17:59,291,729\n",
      "INFO\t2017-01-12 16:27:06\tGenotypeConcordance\tchecked    28,970,000 variants.  Elapsed time: 00:14:12s.  Time for last 10,000:    0s.  Last read position: 17:60,282,418\n",
      "INFO\t2017-01-12 16:27:07\tGenotypeConcordance\tchecked    28,980,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 17:61,244,869\n",
      "INFO\t2017-01-12 16:27:07\tGenotypeConcordance\tchecked    28,990,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 17:62,167,506\n",
      "INFO\t2017-01-12 16:27:07\tGenotypeConcordance\tchecked    29,000,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 17:63,216,128\n",
      "INFO\t2017-01-12 16:27:07\tGenotypeConcordance\tchecked    29,010,000 variants.  Elapsed time: 00:14:13s.  Time for last 10,000:    0s.  Last read position: 17:64,023,781\n",
      "INFO\t2017-01-12 16:27:08\tGenotypeConcordance\tchecked    29,020,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 17:64,831,185\n",
      "INFO\t2017-01-12 16:27:08\tGenotypeConcordance\tchecked    29,030,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 17:65,704,455\n",
      "INFO\t2017-01-12 16:27:08\tGenotypeConcordance\tchecked    29,040,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 17:66,485,484\n",
      "INFO\t2017-01-12 16:27:08\tGenotypeConcordance\tchecked    29,050,000 variants.  Elapsed time: 00:14:14s.  Time for last 10,000:    0s.  Last read position: 17:67,385,862\n",
      "INFO\t2017-01-12 16:27:09\tGenotypeConcordance\tchecked    29,060,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 17:68,331,568\n",
      "INFO\t2017-01-12 16:27:09\tGenotypeConcordance\tchecked    29,070,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 17:69,174,196\n",
      "INFO\t2017-01-12 16:27:09\tGenotypeConcordance\tchecked    29,080,000 variants.  Elapsed time: 00:14:15s.  Time for last 10,000:    0s.  Last read position: 17:69,994,450\n",
      "INFO\t2017-01-12 16:27:09\tGenotypeConcordance\tchecked    29,090,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 17:70,796,942\n",
      "INFO\t2017-01-12 16:27:10\tGenotypeConcordance\tchecked    29,100,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 17:71,524,987\n",
      "INFO\t2017-01-12 16:27:10\tGenotypeConcordance\tchecked    29,110,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 17:72,237,435\n",
      "INFO\t2017-01-12 16:27:10\tGenotypeConcordance\tchecked    29,120,000 variants.  Elapsed time: 00:14:16s.  Time for last 10,000:    0s.  Last read position: 17:73,028,078\n",
      "INFO\t2017-01-12 16:27:10\tGenotypeConcordance\tchecked    29,130,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 17:73,857,022\n",
      "INFO\t2017-01-12 16:27:11\tGenotypeConcordance\tchecked    29,140,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 17:74,639,811\n",
      "INFO\t2017-01-12 16:27:11\tGenotypeConcordance\tchecked    29,150,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 17:75,331,648\n",
      "INFO\t2017-01-12 16:27:11\tGenotypeConcordance\tchecked    29,160,000 variants.  Elapsed time: 00:14:17s.  Time for last 10,000:    0s.  Last read position: 17:76,015,286\n",
      "INFO\t2017-01-12 16:27:12\tGenotypeConcordance\tchecked    29,170,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 17:76,639,316\n",
      "INFO\t2017-01-12 16:27:12\tGenotypeConcordance\tchecked    29,180,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 17:77,313,109\n",
      "INFO\t2017-01-12 16:27:12\tGenotypeConcordance\tchecked    29,190,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 17:78,104,137\n",
      "INFO\t2017-01-12 16:27:12\tGenotypeConcordance\tchecked    29,200,000 variants.  Elapsed time: 00:14:18s.  Time for last 10,000:    0s.  Last read position: 17:78,819,267\n",
      "INFO\t2017-01-12 16:27:13\tGenotypeConcordance\tchecked    29,210,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 17:79,439,361\n",
      "INFO\t2017-01-12 16:27:13\tGenotypeConcordance\tchecked    29,220,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 17:80,220,622\n",
      "INFO\t2017-01-12 16:27:13\tGenotypeConcordance\tchecked    29,230,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 17:80,874,543\n",
      "INFO\t2017-01-12 16:27:13\tGenotypeConcordance\tchecked    29,240,000 variants.  Elapsed time: 00:14:19s.  Time for last 10,000:    0s.  Last read position: 18:343,344\n",
      "INFO\t2017-01-12 16:27:14\tGenotypeConcordance\tchecked    29,250,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 18:1,070,098\n",
      "INFO\t2017-01-12 16:27:14\tGenotypeConcordance\tchecked    29,260,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 18:1,865,620\n",
      "INFO\t2017-01-12 16:27:14\tGenotypeConcordance\tchecked    29,270,000 variants.  Elapsed time: 00:14:20s.  Time for last 10,000:    0s.  Last read position: 18:2,609,293\n",
      "INFO\t2017-01-12 16:27:14\tGenotypeConcordance\tchecked    29,280,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 18:3,329,118\n",
      "INFO\t2017-01-12 16:27:15\tGenotypeConcordance\tchecked    29,290,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 18:4,089,873\n",
      "INFO\t2017-01-12 16:27:15\tGenotypeConcordance\tchecked    29,300,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 18:4,870,485\n",
      "INFO\t2017-01-12 16:27:15\tGenotypeConcordance\tchecked    29,310,000 variants.  Elapsed time: 00:14:21s.  Time for last 10,000:    0s.  Last read position: 18:5,647,385\n",
      "INFO\t2017-01-12 16:27:16\tGenotypeConcordance\tchecked    29,320,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 18:6,472,983\n",
      "INFO\t2017-01-12 16:27:16\tGenotypeConcordance\tchecked    29,330,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 18:7,193,858\n",
      "INFO\t2017-01-12 16:27:16\tGenotypeConcordance\tchecked    29,340,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 18:8,007,349\n",
      "INFO\t2017-01-12 16:27:16\tGenotypeConcordance\tchecked    29,350,000 variants.  Elapsed time: 00:14:22s.  Time for last 10,000:    0s.  Last read position: 18:8,753,976\n",
      "INFO\t2017-01-12 16:27:17\tGenotypeConcordance\tchecked    29,360,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 18:9,539,701\n",
      "INFO\t2017-01-12 16:27:17\tGenotypeConcordance\tchecked    29,370,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 18:10,290,966\n",
      "INFO\t2017-01-12 16:27:17\tGenotypeConcordance\tchecked    29,380,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 18:10,963,030\n",
      "INFO\t2017-01-12 16:27:17\tGenotypeConcordance\tchecked    29,390,000 variants.  Elapsed time: 00:14:23s.  Time for last 10,000:    0s.  Last read position: 18:11,679,246\n",
      "INFO\t2017-01-12 16:27:18\tGenotypeConcordance\tchecked    29,400,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 18:12,387,659\n",
      "INFO\t2017-01-12 16:27:18\tGenotypeConcordance\tchecked    29,410,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 18:13,217,897\n",
      "INFO\t2017-01-12 16:27:18\tGenotypeConcordance\tchecked    29,420,000 variants.  Elapsed time: 00:14:24s.  Time for last 10,000:    0s.  Last read position: 18:14,010,580\n",
      "INFO\t2017-01-12 16:27:18\tGenotypeConcordance\tchecked    29,430,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 18:14,539,793\n",
      "INFO\t2017-01-12 16:27:19\tGenotypeConcordance\tchecked    29,440,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 18:15,203,733\n",
      "INFO\t2017-01-12 16:27:19\tGenotypeConcordance\tchecked    29,450,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 18:18,973,557\n",
      "INFO\t2017-01-12 16:27:19\tGenotypeConcordance\tchecked    29,460,000 variants.  Elapsed time: 00:14:25s.  Time for last 10,000:    0s.  Last read position: 18:20,028,670\n",
      "INFO\t2017-01-12 16:27:20\tGenotypeConcordance\tchecked    29,470,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 18:20,946,105\n",
      "INFO\t2017-01-12 16:27:20\tGenotypeConcordance\tchecked    29,480,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 18:21,934,157\n",
      "INFO\t2017-01-12 16:27:20\tGenotypeConcordance\tchecked    29,490,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 18:22,747,289\n",
      "INFO\t2017-01-12 16:27:20\tGenotypeConcordance\tchecked    29,500,000 variants.  Elapsed time: 00:14:26s.  Time for last 10,000:    0s.  Last read position: 18:23,633,640\n",
      "INFO\t2017-01-12 16:27:21\tGenotypeConcordance\tchecked    29,510,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 18:24,508,415\n",
      "INFO\t2017-01-12 16:27:21\tGenotypeConcordance\tchecked    29,520,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 18:25,468,124\n",
      "INFO\t2017-01-12 16:27:21\tGenotypeConcordance\tchecked    29,530,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 18:26,487,449\n",
      "INFO\t2017-01-12 16:27:21\tGenotypeConcordance\tchecked    29,540,000 variants.  Elapsed time: 00:14:27s.  Time for last 10,000:    0s.  Last read position: 18:27,351,117\n",
      "INFO\t2017-01-12 16:27:22\tGenotypeConcordance\tchecked    29,550,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 18:28,160,788\n",
      "INFO\t2017-01-12 16:27:22\tGenotypeConcordance\tchecked    29,560,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 18:28,984,706\n",
      "INFO\t2017-01-12 16:27:22\tGenotypeConcordance\tchecked    29,570,000 variants.  Elapsed time: 00:14:28s.  Time for last 10,000:    0s.  Last read position: 18:29,811,919\n",
      "INFO\t2017-01-12 16:27:22\tGenotypeConcordance\tchecked    29,580,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 18:30,842,774\n",
      "INFO\t2017-01-12 16:27:23\tGenotypeConcordance\tchecked    29,590,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 18:31,891,898\n",
      "INFO\t2017-01-12 16:27:23\tGenotypeConcordance\tchecked    29,600,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 18:32,871,256\n",
      "INFO\t2017-01-12 16:27:23\tGenotypeConcordance\tchecked    29,610,000 variants.  Elapsed time: 00:14:29s.  Time for last 10,000:    0s.  Last read position: 18:33,759,114\n",
      "INFO\t2017-01-12 16:27:23\tGenotypeConcordance\tchecked    29,620,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 18:34,809,470\n",
      "INFO\t2017-01-12 16:27:24\tGenotypeConcordance\tchecked    29,630,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 18:35,764,592\n",
      "INFO\t2017-01-12 16:27:24\tGenotypeConcordance\tchecked    29,640,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 18:36,704,958\n",
      "INFO\t2017-01-12 16:27:24\tGenotypeConcordance\tchecked    29,650,000 variants.  Elapsed time: 00:14:30s.  Time for last 10,000:    0s.  Last read position: 18:37,727,036\n",
      "INFO\t2017-01-12 16:27:25\tGenotypeConcordance\tchecked    29,660,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 18:38,594,224\n",
      "INFO\t2017-01-12 16:27:25\tGenotypeConcordance\tchecked    29,670,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 18:39,510,120\n",
      "INFO\t2017-01-12 16:27:25\tGenotypeConcordance\tchecked    29,680,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 18:40,425,012\n",
      "INFO\t2017-01-12 16:27:25\tGenotypeConcordance\tchecked    29,690,000 variants.  Elapsed time: 00:14:31s.  Time for last 10,000:    0s.  Last read position: 18:41,342,975\n",
      "INFO\t2017-01-12 16:27:26\tGenotypeConcordance\tchecked    29,700,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 18:42,285,729\n",
      "INFO\t2017-01-12 16:27:26\tGenotypeConcordance\tchecked    29,710,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 18:43,228,567\n",
      "INFO\t2017-01-12 16:27:26\tGenotypeConcordance\tchecked    29,720,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 18:44,072,052\n",
      "INFO\t2017-01-12 16:27:26\tGenotypeConcordance\tchecked    29,730,000 variants.  Elapsed time: 00:14:32s.  Time for last 10,000:    0s.  Last read position: 18:44,993,364\n",
      "INFO\t2017-01-12 16:27:27\tGenotypeConcordance\tchecked    29,740,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 18:45,858,505\n",
      "INFO\t2017-01-12 16:27:27\tGenotypeConcordance\tchecked    29,750,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 18:46,697,891\n",
      "INFO\t2017-01-12 16:27:27\tGenotypeConcordance\tchecked    29,760,000 variants.  Elapsed time: 00:14:33s.  Time for last 10,000:    0s.  Last read position: 18:47,504,347\n",
      "INFO\t2017-01-12 16:27:27\tGenotypeConcordance\tchecked    29,770,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 18:48,306,936\n",
      "INFO\t2017-01-12 16:27:28\tGenotypeConcordance\tchecked    29,780,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 18:49,182,209\n",
      "INFO\t2017-01-12 16:27:28\tGenotypeConcordance\tchecked    29,790,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 18:50,005,954\n",
      "INFO\t2017-01-12 16:27:28\tGenotypeConcordance\tchecked    29,800,000 variants.  Elapsed time: 00:14:34s.  Time for last 10,000:    0s.  Last read position: 18:50,738,582\n",
      "INFO\t2017-01-12 16:27:29\tGenotypeConcordance\tchecked    29,810,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 18:51,505,483\n",
      "INFO\t2017-01-12 16:27:29\tGenotypeConcordance\tchecked    29,820,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 18:52,532,162\n",
      "INFO\t2017-01-12 16:27:29\tGenotypeConcordance\tchecked    29,830,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 18:53,642,248\n",
      "INFO\t2017-01-12 16:27:29\tGenotypeConcordance\tchecked    29,840,000 variants.  Elapsed time: 00:14:35s.  Time for last 10,000:    0s.  Last read position: 18:54,590,715\n",
      "INFO\t2017-01-12 16:27:30\tGenotypeConcordance\tchecked    29,850,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 18:55,411,554\n",
      "INFO\t2017-01-12 16:27:30\tGenotypeConcordance\tchecked    29,860,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 18:56,203,252\n",
      "INFO\t2017-01-12 16:27:30\tGenotypeConcordance\tchecked    29,870,000 variants.  Elapsed time: 00:14:36s.  Time for last 10,000:    0s.  Last read position: 18:57,014,381\n",
      "INFO\t2017-01-12 16:27:30\tGenotypeConcordance\tchecked    29,880,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 18:57,725,641\n",
      "INFO\t2017-01-12 16:27:31\tGenotypeConcordance\tchecked    29,890,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 18:58,511,600\n",
      "INFO\t2017-01-12 16:27:31\tGenotypeConcordance\tchecked    29,900,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 18:59,396,194\n",
      "INFO\t2017-01-12 16:27:31\tGenotypeConcordance\tchecked    29,910,000 variants.  Elapsed time: 00:14:37s.  Time for last 10,000:    0s.  Last read position: 18:60,281,207\n",
      "INFO\t2017-01-12 16:27:31\tGenotypeConcordance\tchecked    29,920,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 18:61,218,269\n",
      "INFO\t2017-01-12 16:27:32\tGenotypeConcordance\tchecked    29,930,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 18:62,044,045\n",
      "INFO\t2017-01-12 16:27:32\tGenotypeConcordance\tchecked    29,940,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 18:62,834,950\n",
      "INFO\t2017-01-12 16:27:32\tGenotypeConcordance\tchecked    29,950,000 variants.  Elapsed time: 00:14:38s.  Time for last 10,000:    0s.  Last read position: 18:63,613,087\n",
      "INFO\t2017-01-12 16:27:33\tGenotypeConcordance\tchecked    29,960,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 18:64,401,957\n",
      "INFO\t2017-01-12 16:27:33\tGenotypeConcordance\tchecked    29,970,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 18:65,212,889\n",
      "INFO\t2017-01-12 16:27:33\tGenotypeConcordance\tchecked    29,980,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 18:65,938,039\n",
      "INFO\t2017-01-12 16:27:33\tGenotypeConcordance\tchecked    29,990,000 variants.  Elapsed time: 00:14:39s.  Time for last 10,000:    0s.  Last read position: 18:66,728,756\n",
      "INFO\t2017-01-12 16:27:34\tGenotypeConcordance\tchecked    30,000,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 18:67,523,260\n",
      "INFO\t2017-01-12 16:27:34\tGenotypeConcordance\tchecked    30,010,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 18:68,372,694\n",
      "INFO\t2017-01-12 16:27:34\tGenotypeConcordance\tchecked    30,020,000 variants.  Elapsed time: 00:14:40s.  Time for last 10,000:    0s.  Last read position: 18:69,159,389\n",
      "INFO\t2017-01-12 16:27:34\tGenotypeConcordance\tchecked    30,030,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 18:69,913,590\n",
      "INFO\t2017-01-12 16:27:35\tGenotypeConcordance\tchecked    30,040,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 18:70,641,009\n",
      "INFO\t2017-01-12 16:27:35\tGenotypeConcordance\tchecked    30,050,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 18:71,351,521\n",
      "INFO\t2017-01-12 16:27:35\tGenotypeConcordance\tchecked    30,060,000 variants.  Elapsed time: 00:14:41s.  Time for last 10,000:    0s.  Last read position: 18:72,108,494\n",
      "INFO\t2017-01-12 16:27:36\tGenotypeConcordance\tchecked    30,070,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 18:73,031,691\n",
      "INFO\t2017-01-12 16:27:36\tGenotypeConcordance\tchecked    30,080,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 18:73,760,114\n",
      "INFO\t2017-01-12 16:27:36\tGenotypeConcordance\tchecked    30,090,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 18:74,457,620\n",
      "INFO\t2017-01-12 16:27:36\tGenotypeConcordance\tchecked    30,100,000 variants.  Elapsed time: 00:14:42s.  Time for last 10,000:    0s.  Last read position: 18:75,134,510\n",
      "INFO\t2017-01-12 16:27:37\tGenotypeConcordance\tchecked    30,110,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 18:75,869,682\n",
      "INFO\t2017-01-12 16:27:37\tGenotypeConcordance\tchecked    30,120,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 18:76,457,627\n",
      "INFO\t2017-01-12 16:27:37\tGenotypeConcordance\tchecked    30,130,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 18:77,169,216\n",
      "INFO\t2017-01-12 16:27:37\tGenotypeConcordance\tchecked    30,140,000 variants.  Elapsed time: 00:14:43s.  Time for last 10,000:    0s.  Last read position: 18:77,790,689\n",
      "INFO\t2017-01-12 16:27:38\tGenotypeConcordance\tchecked    30,150,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 19:597,941\n",
      "INFO\t2017-01-12 16:27:38\tGenotypeConcordance\tchecked    30,160,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 19:1,139,469\n",
      "INFO\t2017-01-12 16:27:38\tGenotypeConcordance\tchecked    30,170,000 variants.  Elapsed time: 00:14:44s.  Time for last 10,000:    0s.  Last read position: 19:1,819,007\n",
      "INFO\t2017-01-12 16:27:38\tGenotypeConcordance\tchecked    30,180,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 19:2,481,012\n",
      "INFO\t2017-01-12 16:27:39\tGenotypeConcordance\tchecked    30,190,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 19:3,080,333\n",
      "INFO\t2017-01-12 16:27:39\tGenotypeConcordance\tchecked    30,200,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 19:3,742,710\n",
      "INFO\t2017-01-12 16:27:39\tGenotypeConcordance\tchecked    30,210,000 variants.  Elapsed time: 00:14:45s.  Time for last 10,000:    0s.  Last read position: 19:4,451,560\n",
      "INFO\t2017-01-12 16:27:39\tGenotypeConcordance\tchecked    30,220,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 19:5,123,070\n",
      "INFO\t2017-01-12 16:27:40\tGenotypeConcordance\tchecked    30,230,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 19:5,858,371\n",
      "INFO\t2017-01-12 16:27:40\tGenotypeConcordance\tchecked    30,240,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 19:6,597,715\n",
      "INFO\t2017-01-12 16:27:40\tGenotypeConcordance\tchecked    30,250,000 variants.  Elapsed time: 00:14:46s.  Time for last 10,000:    0s.  Last read position: 19:7,251,423\n",
      "INFO\t2017-01-12 16:27:40\tGenotypeConcordance\tchecked    30,260,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 19:7,999,362\n",
      "INFO\t2017-01-12 16:27:41\tGenotypeConcordance\tchecked    30,270,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 19:8,755,058\n",
      "INFO\t2017-01-12 16:27:41\tGenotypeConcordance\tchecked    30,280,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 19:9,402,550\n",
      "INFO\t2017-01-12 16:27:41\tGenotypeConcordance\tchecked    30,290,000 variants.  Elapsed time: 00:14:47s.  Time for last 10,000:    0s.  Last read position: 19:10,158,124\n",
      "INFO\t2017-01-12 16:27:41\tGenotypeConcordance\tchecked    30,300,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 19:11,026,045\n",
      "INFO\t2017-01-12 16:27:42\tGenotypeConcordance\tchecked    30,310,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 19:11,817,808\n",
      "INFO\t2017-01-12 16:27:42\tGenotypeConcordance\tchecked    30,320,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 19:12,669,438\n",
      "INFO\t2017-01-12 16:27:42\tGenotypeConcordance\tchecked    30,330,000 variants.  Elapsed time: 00:14:48s.  Time for last 10,000:    0s.  Last read position: 19:13,579,418\n",
      "INFO\t2017-01-12 16:27:42\tGenotypeConcordance\tchecked    30,340,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 19:14,379,965\n",
      "INFO\t2017-01-12 16:27:43\tGenotypeConcordance\tchecked    30,350,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 19:15,084,896\n",
      "INFO\t2017-01-12 16:27:43\tGenotypeConcordance\tchecked    30,360,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 19:15,835,206\n",
      "INFO\t2017-01-12 16:27:43\tGenotypeConcordance\tchecked    30,370,000 variants.  Elapsed time: 00:14:49s.  Time for last 10,000:    0s.  Last read position: 19:16,527,917\n",
      "INFO\t2017-01-12 16:27:44\tGenotypeConcordance\tchecked    30,380,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 19:17,324,751\n",
      "INFO\t2017-01-12 16:27:44\tGenotypeConcordance\tchecked    30,390,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 19:18,073,775\n",
      "INFO\t2017-01-12 16:27:44\tGenotypeConcordance\tchecked    30,400,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 19:18,929,561\n",
      "INFO\t2017-01-12 16:27:44\tGenotypeConcordance\tchecked    30,410,000 variants.  Elapsed time: 00:14:50s.  Time for last 10,000:    0s.  Last read position: 19:19,891,687\n",
      "INFO\t2017-01-12 16:27:45\tGenotypeConcordance\tchecked    30,420,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 19:20,688,283\n",
      "INFO\t2017-01-12 16:27:45\tGenotypeConcordance\tchecked    30,430,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 19:21,320,316\n",
      "INFO\t2017-01-12 16:27:45\tGenotypeConcordance\tchecked    30,440,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 19:21,957,865\n",
      "INFO\t2017-01-12 16:27:45\tGenotypeConcordance\tchecked    30,450,000 variants.  Elapsed time: 00:14:51s.  Time for last 10,000:    0s.  Last read position: 19:22,646,934\n",
      "INFO\t2017-01-12 16:27:46\tGenotypeConcordance\tchecked    30,460,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 19:23,344,209\n",
      "INFO\t2017-01-12 16:27:46\tGenotypeConcordance\tchecked    30,470,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 19:24,030,473\n",
      "INFO\t2017-01-12 16:27:46\tGenotypeConcordance\tchecked    30,480,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 19:27,733,063\n",
      "INFO\t2017-01-12 16:27:46\tGenotypeConcordance\tchecked    30,490,000 variants.  Elapsed time: 00:14:52s.  Time for last 10,000:    0s.  Last read position: 19:28,165,191\n",
      "INFO\t2017-01-12 16:27:47\tGenotypeConcordance\tchecked    30,500,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 19:28,897,587\n",
      "INFO\t2017-01-12 16:27:47\tGenotypeConcordance\tchecked    30,510,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 19:29,567,472\n",
      "INFO\t2017-01-12 16:27:47\tGenotypeConcordance\tchecked    30,520,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 19:30,274,191\n",
      "INFO\t2017-01-12 16:27:47\tGenotypeConcordance\tchecked    30,530,000 variants.  Elapsed time: 00:14:53s.  Time for last 10,000:    0s.  Last read position: 19:31,140,007\n",
      "INFO\t2017-01-12 16:27:48\tGenotypeConcordance\tchecked    30,540,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 19:32,050,829\n",
      "INFO\t2017-01-12 16:27:48\tGenotypeConcordance\tchecked    30,550,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 19:32,951,841\n",
      "INFO\t2017-01-12 16:27:48\tGenotypeConcordance\tchecked    30,560,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 19:33,728,968\n",
      "INFO\t2017-01-12 16:27:48\tGenotypeConcordance\tchecked    30,570,000 variants.  Elapsed time: 00:14:54s.  Time for last 10,000:    0s.  Last read position: 19:34,514,806\n",
      "INFO\t2017-01-12 16:27:49\tGenotypeConcordance\tchecked    30,580,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 19:35,372,415\n",
      "INFO\t2017-01-12 16:27:49\tGenotypeConcordance\tchecked    30,590,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 19:36,188,129\n",
      "INFO\t2017-01-12 16:27:49\tGenotypeConcordance\tchecked    30,600,000 variants.  Elapsed time: 00:14:55s.  Time for last 10,000:    0s.  Last read position: 19:37,050,707\n",
      "INFO\t2017-01-12 16:27:49\tGenotypeConcordance\tchecked    30,610,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 19:38,073,098\n",
      "INFO\t2017-01-12 16:27:50\tGenotypeConcordance\tchecked    30,620,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 19:39,000,236\n",
      "INFO\t2017-01-12 16:27:50\tGenotypeConcordance\tchecked    30,630,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 19:39,738,666\n",
      "INFO\t2017-01-12 16:27:50\tGenotypeConcordance\tchecked    30,640,000 variants.  Elapsed time: 00:14:56s.  Time for last 10,000:    0s.  Last read position: 19:40,575,008\n",
      "INFO\t2017-01-12 16:27:50\tGenotypeConcordance\tchecked    30,650,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 19:41,438,722\n",
      "INFO\t2017-01-12 16:27:51\tGenotypeConcordance\tchecked    30,660,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 19:42,165,671\n",
      "INFO\t2017-01-12 16:27:51\tGenotypeConcordance\tchecked    30,670,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 19:43,266,640\n",
      "INFO\t2017-01-12 16:27:51\tGenotypeConcordance\tchecked    30,680,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 19:43,697,726\n",
      "INFO\t2017-01-12 16:27:51\tGenotypeConcordance\tchecked    30,690,000 variants.  Elapsed time: 00:14:57s.  Time for last 10,000:    0s.  Last read position: 19:44,428,911\n",
      "INFO\t2017-01-12 16:27:52\tGenotypeConcordance\tchecked    30,700,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 19:45,216,026\n",
      "INFO\t2017-01-12 16:27:52\tGenotypeConcordance\tchecked    30,710,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 19:45,974,668\n",
      "INFO\t2017-01-12 16:27:52\tGenotypeConcordance\tchecked    30,720,000 variants.  Elapsed time: 00:14:58s.  Time for last 10,000:    0s.  Last read position: 19:46,785,021\n",
      "INFO\t2017-01-12 16:27:52\tGenotypeConcordance\tchecked    30,730,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 19:47,580,641\n",
      "INFO\t2017-01-12 16:27:53\tGenotypeConcordance\tchecked    30,740,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 19:48,381,365\n",
      "INFO\t2017-01-12 16:27:53\tGenotypeConcordance\tchecked    30,750,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 19:49,114,433\n",
      "INFO\t2017-01-12 16:27:53\tGenotypeConcordance\tchecked    30,760,000 variants.  Elapsed time: 00:14:59s.  Time for last 10,000:    0s.  Last read position: 19:49,808,849\n",
      "INFO\t2017-01-12 16:27:54\tGenotypeConcordance\tchecked    30,770,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 19:50,593,858\n",
      "INFO\t2017-01-12 16:27:54\tGenotypeConcordance\tchecked    30,780,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 19:51,321,827\n",
      "INFO\t2017-01-12 16:27:54\tGenotypeConcordance\tchecked    30,790,000 variants.  Elapsed time: 00:15:00s.  Time for last 10,000:    0s.  Last read position: 19:52,041,801\n",
      "INFO\t2017-01-12 16:27:55\tGenotypeConcordance\tchecked    30,800,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 19:52,798,735\n",
      "INFO\t2017-01-12 16:27:55\tGenotypeConcordance\tchecked    30,810,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 19:53,359,005\n",
      "INFO\t2017-01-12 16:27:55\tGenotypeConcordance\tchecked    30,820,000 variants.  Elapsed time: 00:15:01s.  Time for last 10,000:    0s.  Last read position: 19:53,959,745\n",
      "INFO\t2017-01-12 16:27:56\tGenotypeConcordance\tchecked    30,830,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 19:54,536,734\n",
      "INFO\t2017-01-12 16:27:56\tGenotypeConcordance\tchecked    30,840,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 19:55,098,680\n",
      "INFO\t2017-01-12 16:27:56\tGenotypeConcordance\tchecked    30,850,000 variants.  Elapsed time: 00:15:02s.  Time for last 10,000:    0s.  Last read position: 19:55,623,635\n",
      "INFO\t2017-01-12 16:27:56\tGenotypeConcordance\tchecked    30,860,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 19:56,298,003\n",
      "INFO\t2017-01-12 16:27:57\tGenotypeConcordance\tchecked    30,870,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 19:56,863,145\n",
      "INFO\t2017-01-12 16:27:57\tGenotypeConcordance\tchecked    30,880,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 19:57,660,692\n",
      "INFO\t2017-01-12 16:27:57\tGenotypeConcordance\tchecked    30,890,000 variants.  Elapsed time: 00:15:03s.  Time for last 10,000:    0s.  Last read position: 19:58,503,369\n",
      "INFO\t2017-01-12 16:27:58\tGenotypeConcordance\tchecked    30,900,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 20:223,652\n",
      "INFO\t2017-01-12 16:27:58\tGenotypeConcordance\tchecked    30,910,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 20:923,176\n",
      "INFO\t2017-01-12 16:27:58\tGenotypeConcordance\tchecked    30,920,000 variants.  Elapsed time: 00:15:04s.  Time for last 10,000:    0s.  Last read position: 20:1,796,112\n",
      "INFO\t2017-01-12 16:27:58\tGenotypeConcordance\tchecked    30,930,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 20:2,579,107\n",
      "INFO\t2017-01-12 16:27:59\tGenotypeConcordance\tchecked    30,940,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 20:3,409,428\n",
      "INFO\t2017-01-12 16:27:59\tGenotypeConcordance\tchecked    30,950,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 20:4,222,977\n",
      "INFO\t2017-01-12 16:27:59\tGenotypeConcordance\tchecked    30,960,000 variants.  Elapsed time: 00:15:05s.  Time for last 10,000:    0s.  Last read position: 20:4,996,506\n",
      "INFO\t2017-01-12 16:28:00\tGenotypeConcordance\tchecked    30,970,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 20:5,730,566\n",
      "INFO\t2017-01-12 16:28:00\tGenotypeConcordance\tchecked    30,980,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 20:6,491,219\n",
      "INFO\t2017-01-12 16:28:00\tGenotypeConcordance\tchecked    30,990,000 variants.  Elapsed time: 00:15:06s.  Time for last 10,000:    0s.  Last read position: 20:7,293,914\n",
      "INFO\t2017-01-12 16:28:01\tGenotypeConcordance\tchecked    31,000,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 20:8,106,464\n",
      "INFO\t2017-01-12 16:28:01\tGenotypeConcordance\tchecked    31,010,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 20:8,981,641\n",
      "INFO\t2017-01-12 16:28:01\tGenotypeConcordance\tchecked    31,020,000 variants.  Elapsed time: 00:15:07s.  Time for last 10,000:    0s.  Last read position: 20:9,917,409\n",
      "INFO\t2017-01-12 16:28:01\tGenotypeConcordance\tchecked    31,030,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 20:10,823,171\n",
      "INFO\t2017-01-12 16:28:02\tGenotypeConcordance\tchecked    31,040,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 20:11,704,160\n",
      "INFO\t2017-01-12 16:28:02\tGenotypeConcordance\tchecked    31,050,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 20:12,497,212\n",
      "INFO\t2017-01-12 16:28:02\tGenotypeConcordance\tchecked    31,060,000 variants.  Elapsed time: 00:15:08s.  Time for last 10,000:    0s.  Last read position: 20:13,294,098\n",
      "INFO\t2017-01-12 16:28:03\tGenotypeConcordance\tchecked    31,070,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 20:14,314,039\n",
      "INFO\t2017-01-12 16:28:03\tGenotypeConcordance\tchecked    31,080,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 20:15,236,000\n",
      "INFO\t2017-01-12 16:28:03\tGenotypeConcordance\tchecked    31,090,000 variants.  Elapsed time: 00:15:09s.  Time for last 10,000:    0s.  Last read position: 20:16,034,829\n",
      "INFO\t2017-01-12 16:28:04\tGenotypeConcordance\tchecked    31,100,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 20:16,828,207\n",
      "INFO\t2017-01-12 16:28:04\tGenotypeConcordance\tchecked    31,110,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 20:17,629,161\n",
      "INFO\t2017-01-12 16:28:04\tGenotypeConcordance\tchecked    31,120,000 variants.  Elapsed time: 00:15:10s.  Time for last 10,000:    0s.  Last read position: 20:18,408,862\n",
      "INFO\t2017-01-12 16:28:05\tGenotypeConcordance\tchecked    31,130,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 20:19,227,230\n",
      "INFO\t2017-01-12 16:28:05\tGenotypeConcordance\tchecked    31,140,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 20:20,041,868\n",
      "INFO\t2017-01-12 16:28:05\tGenotypeConcordance\tchecked    31,150,000 variants.  Elapsed time: 00:15:11s.  Time for last 10,000:    0s.  Last read position: 20:21,016,811\n",
      "INFO\t2017-01-12 16:28:05\tGenotypeConcordance\tchecked    31,160,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 20:22,030,882\n",
      "INFO\t2017-01-12 16:28:06\tGenotypeConcordance\tchecked    31,170,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 20:22,828,996\n",
      "INFO\t2017-01-12 16:28:06\tGenotypeConcordance\tchecked    31,180,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 20:23,627,632\n",
      "INFO\t2017-01-12 16:28:06\tGenotypeConcordance\tchecked    31,190,000 variants.  Elapsed time: 00:15:12s.  Time for last 10,000:    0s.  Last read position: 20:24,237,109\n",
      "INFO\t2017-01-12 16:28:07\tGenotypeConcordance\tchecked    31,200,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 20:24,984,699\n",
      "INFO\t2017-01-12 16:28:07\tGenotypeConcordance\tchecked    31,210,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 20:25,764,616\n",
      "INFO\t2017-01-12 16:28:07\tGenotypeConcordance\tchecked    31,220,000 variants.  Elapsed time: 00:15:13s.  Time for last 10,000:    0s.  Last read position: 20:26,138,588\n",
      "INFO\t2017-01-12 16:28:08\tGenotypeConcordance\tchecked    31,230,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 20:29,512,482\n",
      "INFO\t2017-01-12 16:28:08\tGenotypeConcordance\tchecked    31,240,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 20:29,913,360\n",
      "INFO\t2017-01-12 16:28:08\tGenotypeConcordance\tchecked    31,250,000 variants.  Elapsed time: 00:15:14s.  Time for last 10,000:    0s.  Last read position: 20:30,972,774\n",
      "INFO\t2017-01-12 16:28:09\tGenotypeConcordance\tchecked    31,260,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 20:31,863,599\n",
      "INFO\t2017-01-12 16:28:09\tGenotypeConcordance\tchecked    31,270,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 20:32,938,216\n",
      "INFO\t2017-01-12 16:28:09\tGenotypeConcordance\tchecked    31,280,000 variants.  Elapsed time: 00:15:15s.  Time for last 10,000:    0s.  Last read position: 20:34,033,548\n",
      "INFO\t2017-01-12 16:28:09\tGenotypeConcordance\tchecked    31,290,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 20:35,192,121\n",
      "INFO\t2017-01-12 16:28:10\tGenotypeConcordance\tchecked    31,300,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 20:36,185,592\n",
      "INFO\t2017-01-12 16:28:10\tGenotypeConcordance\tchecked    31,310,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 20:37,053,089\n",
      "INFO\t2017-01-12 16:28:10\tGenotypeConcordance\tchecked    31,320,000 variants.  Elapsed time: 00:15:16s.  Time for last 10,000:    0s.  Last read position: 20:38,026,027\n",
      "INFO\t2017-01-12 16:28:11\tGenotypeConcordance\tchecked    31,330,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 20:38,994,449\n",
      "INFO\t2017-01-12 16:28:11\tGenotypeConcordance\tchecked    31,340,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 20:39,966,631\n",
      "INFO\t2017-01-12 16:28:11\tGenotypeConcordance\tchecked    31,350,000 variants.  Elapsed time: 00:15:17s.  Time for last 10,000:    0s.  Last read position: 20:40,879,061\n",
      "INFO\t2017-01-12 16:28:11\tGenotypeConcordance\tchecked    31,360,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 20:41,702,152\n",
      "INFO\t2017-01-12 16:28:12\tGenotypeConcordance\tchecked    31,370,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 20:42,567,251\n",
      "INFO\t2017-01-12 16:28:12\tGenotypeConcordance\tchecked    31,380,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 20:43,473,176\n",
      "INFO\t2017-01-12 16:28:12\tGenotypeConcordance\tchecked    31,390,000 variants.  Elapsed time: 00:15:18s.  Time for last 10,000:    0s.  Last read position: 20:44,361,076\n",
      "INFO\t2017-01-12 16:28:13\tGenotypeConcordance\tchecked    31,400,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 20:45,253,237\n",
      "INFO\t2017-01-12 16:28:13\tGenotypeConcordance\tchecked    31,410,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 20:46,051,930\n",
      "INFO\t2017-01-12 16:28:13\tGenotypeConcordance\tchecked    31,420,000 variants.  Elapsed time: 00:15:19s.  Time for last 10,000:    0s.  Last read position: 20:46,833,219\n",
      "INFO\t2017-01-12 16:28:14\tGenotypeConcordance\tchecked    31,430,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 20:47,596,482\n",
      "INFO\t2017-01-12 16:28:14\tGenotypeConcordance\tchecked    31,440,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 20:48,491,474\n",
      "INFO\t2017-01-12 16:28:14\tGenotypeConcordance\tchecked    31,450,000 variants.  Elapsed time: 00:15:20s.  Time for last 10,000:    0s.  Last read position: 20:49,333,954\n",
      "INFO\t2017-01-12 16:28:15\tGenotypeConcordance\tchecked    31,460,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 20:50,169,212\n",
      "INFO\t2017-01-12 16:28:15\tGenotypeConcordance\tchecked    31,470,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 20:50,947,016\n",
      "INFO\t2017-01-12 16:28:15\tGenotypeConcordance\tchecked    31,480,000 variants.  Elapsed time: 00:15:21s.  Time for last 10,000:    0s.  Last read position: 20:51,863,191\n",
      "INFO\t2017-01-12 16:28:15\tGenotypeConcordance\tchecked    31,490,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 20:52,656,660\n",
      "INFO\t2017-01-12 16:28:16\tGenotypeConcordance\tchecked    31,500,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 20:53,431,548\n",
      "INFO\t2017-01-12 16:28:16\tGenotypeConcordance\tchecked    31,510,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 20:54,194,776\n",
      "INFO\t2017-01-12 16:28:16\tGenotypeConcordance\tchecked    31,520,000 variants.  Elapsed time: 00:15:22s.  Time for last 10,000:    0s.  Last read position: 20:54,929,764\n",
      "INFO\t2017-01-12 16:28:17\tGenotypeConcordance\tchecked    31,530,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 20:55,668,324\n",
      "INFO\t2017-01-12 16:28:17\tGenotypeConcordance\tchecked    31,540,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 20:56,432,447\n",
      "INFO\t2017-01-12 16:28:17\tGenotypeConcordance\tchecked    31,550,000 variants.  Elapsed time: 00:15:23s.  Time for last 10,000:    0s.  Last read position: 20:57,224,982\n",
      "INFO\t2017-01-12 16:28:18\tGenotypeConcordance\tchecked    31,560,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 20:58,106,502\n",
      "INFO\t2017-01-12 16:28:18\tGenotypeConcordance\tchecked    31,570,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 20:58,889,224\n",
      "INFO\t2017-01-12 16:28:18\tGenotypeConcordance\tchecked    31,580,000 variants.  Elapsed time: 00:15:24s.  Time for last 10,000:    0s.  Last read position: 20:59,632,459\n",
      "INFO\t2017-01-12 16:28:19\tGenotypeConcordance\tchecked    31,590,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 20:60,284,329\n",
      "INFO\t2017-01-12 16:28:19\tGenotypeConcordance\tchecked    31,600,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 20:60,939,284\n",
      "INFO\t2017-01-12 16:28:19\tGenotypeConcordance\tchecked    31,610,000 variants.  Elapsed time: 00:15:25s.  Time for last 10,000:    0s.  Last read position: 20:61,662,583\n",
      "INFO\t2017-01-12 16:28:19\tGenotypeConcordance\tchecked    31,620,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 20:62,298,541\n",
      "INFO\t2017-01-12 16:28:20\tGenotypeConcordance\tchecked    31,630,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 21:9,481,371\n",
      "INFO\t2017-01-12 16:28:20\tGenotypeConcordance\tchecked    31,640,000 variants.  Elapsed time: 00:15:26s.  Time for last 10,000:    0s.  Last read position: 21:9,955,419\n",
      "INFO\t2017-01-12 16:28:20\tGenotypeConcordance\tchecked    31,650,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: 21:10,706,664\n",
      "INFO\t2017-01-12 16:28:21\tGenotypeConcordance\tchecked    31,660,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: 21:10,868,983\n",
      "INFO\t2017-01-12 16:28:21\tGenotypeConcordance\tchecked    31,670,000 variants.  Elapsed time: 00:15:27s.  Time for last 10,000:    0s.  Last read position: 21:14,361,793\n",
      "INFO\t2017-01-12 16:28:22\tGenotypeConcordance\tchecked    31,680,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: 21:15,218,151\n",
      "INFO\t2017-01-12 16:28:22\tGenotypeConcordance\tchecked    31,690,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: 21:15,738,677\n",
      "INFO\t2017-01-12 16:28:22\tGenotypeConcordance\tchecked    31,700,000 variants.  Elapsed time: 00:15:28s.  Time for last 10,000:    0s.  Last read position: 21:16,535,727\n",
      "INFO\t2017-01-12 16:28:22\tGenotypeConcordance\tchecked    31,710,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: 21:17,393,193\n",
      "INFO\t2017-01-12 16:28:23\tGenotypeConcordance\tchecked    31,720,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: 21:18,190,322\n",
      "INFO\t2017-01-12 16:28:23\tGenotypeConcordance\tchecked    31,730,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: 21:18,817,058\n",
      "INFO\t2017-01-12 16:28:23\tGenotypeConcordance\tchecked    31,740,000 variants.  Elapsed time: 00:15:29s.  Time for last 10,000:    0s.  Last read position: 21:19,594,702\n",
      "INFO\t2017-01-12 16:28:24\tGenotypeConcordance\tchecked    31,750,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: 21:20,348,469\n",
      "INFO\t2017-01-12 16:28:24\tGenotypeConcordance\tchecked    31,760,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: 21:21,104,416\n",
      "INFO\t2017-01-12 16:28:24\tGenotypeConcordance\tchecked    31,770,000 variants.  Elapsed time: 00:15:30s.  Time for last 10,000:    0s.  Last read position: 21:21,823,904\n",
      "INFO\t2017-01-12 16:28:25\tGenotypeConcordance\tchecked    31,780,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: 21:22,478,483\n",
      "INFO\t2017-01-12 16:28:25\tGenotypeConcordance\tchecked    31,790,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: 21:23,231,909\n",
      "INFO\t2017-01-12 16:28:25\tGenotypeConcordance\tchecked    31,800,000 variants.  Elapsed time: 00:15:31s.  Time for last 10,000:    0s.  Last read position: 21:24,000,969\n",
      "INFO\t2017-01-12 16:28:26\tGenotypeConcordance\tchecked    31,810,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: 21:24,713,951\n",
      "INFO\t2017-01-12 16:28:26\tGenotypeConcordance\tchecked    31,820,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: 21:25,364,632\n",
      "INFO\t2017-01-12 16:28:26\tGenotypeConcordance\tchecked    31,830,000 variants.  Elapsed time: 00:15:32s.  Time for last 10,000:    0s.  Last read position: 21:26,050,834\n",
      "INFO\t2017-01-12 16:28:26\tGenotypeConcordance\tchecked    31,840,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: 21:26,930,332\n",
      "INFO\t2017-01-12 16:28:27\tGenotypeConcordance\tchecked    31,850,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: 21:27,738,422\n",
      "INFO\t2017-01-12 16:28:27\tGenotypeConcordance\tchecked    31,860,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: 21:28,560,262\n",
      "INFO\t2017-01-12 16:28:27\tGenotypeConcordance\tchecked    31,870,000 variants.  Elapsed time: 00:15:33s.  Time for last 10,000:    0s.  Last read position: 21:29,329,556\n",
      "INFO\t2017-01-12 16:28:28\tGenotypeConcordance\tchecked    31,880,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: 21:30,094,309\n",
      "INFO\t2017-01-12 16:28:28\tGenotypeConcordance\tchecked    31,890,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: 21:30,946,049\n",
      "INFO\t2017-01-12 16:28:28\tGenotypeConcordance\tchecked    31,900,000 variants.  Elapsed time: 00:15:34s.  Time for last 10,000:    0s.  Last read position: 21:31,740,553\n",
      "INFO\t2017-01-12 16:28:29\tGenotypeConcordance\tchecked    31,910,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: 21:32,569,974\n",
      "INFO\t2017-01-12 16:28:29\tGenotypeConcordance\tchecked    31,920,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: 21:33,365,595\n",
      "INFO\t2017-01-12 16:28:29\tGenotypeConcordance\tchecked    31,930,000 variants.  Elapsed time: 00:15:35s.  Time for last 10,000:    0s.  Last read position: 21:34,237,858\n",
      "INFO\t2017-01-12 16:28:29\tGenotypeConcordance\tchecked    31,940,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: 21:35,132,301\n",
      "INFO\t2017-01-12 16:28:30\tGenotypeConcordance\tchecked    31,950,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: 21:35,965,934\n",
      "INFO\t2017-01-12 16:28:30\tGenotypeConcordance\tchecked    31,960,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: 21:36,777,485\n",
      "INFO\t2017-01-12 16:28:30\tGenotypeConcordance\tchecked    31,970,000 variants.  Elapsed time: 00:15:36s.  Time for last 10,000:    0s.  Last read position: 21:37,622,270\n",
      "INFO\t2017-01-12 16:28:31\tGenotypeConcordance\tchecked    31,980,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: 21:38,400,659\n",
      "INFO\t2017-01-12 16:28:31\tGenotypeConcordance\tchecked    31,990,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: 21:39,250,687\n",
      "INFO\t2017-01-12 16:28:31\tGenotypeConcordance\tchecked    32,000,000 variants.  Elapsed time: 00:15:37s.  Time for last 10,000:    0s.  Last read position: 21:40,142,443\n",
      "INFO\t2017-01-12 16:28:32\tGenotypeConcordance\tchecked    32,010,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: 21:40,926,096\n",
      "INFO\t2017-01-12 16:28:32\tGenotypeConcordance\tchecked    32,020,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: 21:41,637,410\n",
      "INFO\t2017-01-12 16:28:32\tGenotypeConcordance\tchecked    32,030,000 variants.  Elapsed time: 00:15:38s.  Time for last 10,000:    0s.  Last read position: 21:42,356,760\n",
      "INFO\t2017-01-12 16:28:33\tGenotypeConcordance\tchecked    32,040,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: 21:43,145,704\n",
      "INFO\t2017-01-12 16:28:33\tGenotypeConcordance\tchecked    32,050,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: 21:43,798,797\n",
      "INFO\t2017-01-12 16:28:33\tGenotypeConcordance\tchecked    32,060,000 variants.  Elapsed time: 00:15:39s.  Time for last 10,000:    0s.  Last read position: 21:44,466,913\n",
      "INFO\t2017-01-12 16:28:34\tGenotypeConcordance\tchecked    32,070,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: 21:45,271,831\n",
      "INFO\t2017-01-12 16:28:34\tGenotypeConcordance\tchecked    32,080,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: 21:45,959,452\n",
      "INFO\t2017-01-12 16:28:34\tGenotypeConcordance\tchecked    32,090,000 variants.  Elapsed time: 00:15:40s.  Time for last 10,000:    0s.  Last read position: 21:46,567,946\n",
      "INFO\t2017-01-12 16:28:34\tGenotypeConcordance\tchecked    32,100,000 variants.  Elapsed time: 00:15:41s.  Time for last 10,000:    0s.  Last read position: 21:47,259,186\n",
      "INFO\t2017-01-12 16:28:35\tGenotypeConcordance\tchecked    32,110,000 variants.  Elapsed time: 00:15:41s.  Time for last 10,000:    0s.  Last read position: 21:47,920,912\n",
      "INFO\t2017-01-12 16:28:35\tGenotypeConcordance\tchecked    32,120,000 variants.  Elapsed time: 00:15:41s.  Time for last 10,000:    0s.  Last read position: 22:16,853,596\n",
      "INFO\t2017-01-12 16:28:35\tGenotypeConcordance\tchecked    32,130,000 variants.  Elapsed time: 00:15:42s.  Time for last 10,000:    0s.  Last read position: 22:17,318,864\n",
      "INFO\t2017-01-12 16:28:36\tGenotypeConcordance\tchecked    32,140,000 variants.  Elapsed time: 00:15:42s.  Time for last 10,000:    0s.  Last read position: 22:17,958,427\n",
      "INFO\t2017-01-12 16:28:36\tGenotypeConcordance\tchecked    32,150,000 variants.  Elapsed time: 00:15:42s.  Time for last 10,000:    0s.  Last read position: 22:18,639,178\n",
      "INFO\t2017-01-12 16:28:36\tGenotypeConcordance\tchecked    32,160,000 variants.  Elapsed time: 00:15:42s.  Time for last 10,000:    0s.  Last read position: 22:19,410,846\n",
      "INFO\t2017-01-12 16:28:37\tGenotypeConcordance\tchecked    32,170,000 variants.  Elapsed time: 00:15:43s.  Time for last 10,000:    0s.  Last read position: 22:20,177,320\n",
      "INFO\t2017-01-12 16:28:37\tGenotypeConcordance\tchecked    32,180,000 variants.  Elapsed time: 00:15:43s.  Time for last 10,000:    0s.  Last read position: 22:21,158,858\n",
      "INFO\t2017-01-12 16:28:37\tGenotypeConcordance\tchecked    32,190,000 variants.  Elapsed time: 00:15:43s.  Time for last 10,000:    0s.  Last read position: 22:22,082,603\n",
      "INFO\t2017-01-12 16:28:38\tGenotypeConcordance\tchecked    32,200,000 variants.  Elapsed time: 00:15:44s.  Time for last 10,000:    0s.  Last read position: 22:22,714,989\n",
      "INFO\t2017-01-12 16:28:38\tGenotypeConcordance\tchecked    32,210,000 variants.  Elapsed time: 00:15:44s.  Time for last 10,000:    0s.  Last read position: 22:23,298,763\n",
      "INFO\t2017-01-12 16:28:38\tGenotypeConcordance\tchecked    32,220,000 variants.  Elapsed time: 00:15:44s.  Time for last 10,000:    0s.  Last read position: 22:23,940,670\n",
      "INFO\t2017-01-12 16:28:38\tGenotypeConcordance\tchecked    32,230,000 variants.  Elapsed time: 00:15:45s.  Time for last 10,000:    0s.  Last read position: 22:24,702,549\n",
      "INFO\t2017-01-12 16:28:39\tGenotypeConcordance\tchecked    32,240,000 variants.  Elapsed time: 00:15:45s.  Time for last 10,000:    0s.  Last read position: 22:25,465,840\n",
      "INFO\t2017-01-12 16:28:39\tGenotypeConcordance\tchecked    32,250,000 variants.  Elapsed time: 00:15:45s.  Time for last 10,000:    0s.  Last read position: 22:26,130,523\n",
      "INFO\t2017-01-12 16:28:39\tGenotypeConcordance\tchecked    32,260,000 variants.  Elapsed time: 00:15:45s.  Time for last 10,000:    0s.  Last read position: 22:26,904,992\n",
      "INFO\t2017-01-12 16:28:40\tGenotypeConcordance\tchecked    32,270,000 variants.  Elapsed time: 00:15:46s.  Time for last 10,000:    0s.  Last read position: 22:27,646,849\n",
      "INFO\t2017-01-12 16:28:40\tGenotypeConcordance\tchecked    32,280,000 variants.  Elapsed time: 00:15:46s.  Time for last 10,000:    0s.  Last read position: 22:28,487,368\n",
      "INFO\t2017-01-12 16:28:40\tGenotypeConcordance\tchecked    32,290,000 variants.  Elapsed time: 00:15:46s.  Time for last 10,000:    0s.  Last read position: 22:29,552,332\n",
      "INFO\t2017-01-12 16:28:41\tGenotypeConcordance\tchecked    32,300,000 variants.  Elapsed time: 00:15:47s.  Time for last 10,000:    0s.  Last read position: 22:30,454,598\n",
      "INFO\t2017-01-12 16:28:41\tGenotypeConcordance\tchecked    32,310,000 variants.  Elapsed time: 00:15:47s.  Time for last 10,000:    0s.  Last read position: 22:31,414,524\n",
      "INFO\t2017-01-12 16:28:41\tGenotypeConcordance\tchecked    32,320,000 variants.  Elapsed time: 00:15:47s.  Time for last 10,000:    0s.  Last read position: 22:32,373,174\n",
      "INFO\t2017-01-12 16:28:41\tGenotypeConcordance\tchecked    32,330,000 variants.  Elapsed time: 00:15:47s.  Time for last 10,000:    0s.  Last read position: 22:33,106,989\n",
      "INFO\t2017-01-12 16:28:42\tGenotypeConcordance\tchecked    32,340,000 variants.  Elapsed time: 00:15:48s.  Time for last 10,000:    0s.  Last read position: 22:33,924,594\n",
      "INFO\t2017-01-12 16:28:42\tGenotypeConcordance\tchecked    32,350,000 variants.  Elapsed time: 00:15:48s.  Time for last 10,000:    0s.  Last read position: 22:34,726,062\n",
      "INFO\t2017-01-12 16:28:42\tGenotypeConcordance\tchecked    32,360,000 variants.  Elapsed time: 00:15:48s.  Time for last 10,000:    0s.  Last read position: 22:35,410,604\n",
      "INFO\t2017-01-12 16:28:43\tGenotypeConcordance\tchecked    32,370,000 variants.  Elapsed time: 00:15:49s.  Time for last 10,000:    0s.  Last read position: 22:36,266,441\n",
      "INFO\t2017-01-12 16:28:43\tGenotypeConcordance\tchecked    32,380,000 variants.  Elapsed time: 00:15:49s.  Time for last 10,000:    0s.  Last read position: 22:37,087,302\n",
      "INFO\t2017-01-12 16:28:43\tGenotypeConcordance\tchecked    32,390,000 variants.  Elapsed time: 00:15:49s.  Time for last 10,000:    0s.  Last read position: 22:37,833,010\n",
      "INFO\t2017-01-12 16:28:44\tGenotypeConcordance\tchecked    32,400,000 variants.  Elapsed time: 00:15:50s.  Time for last 10,000:    0s.  Last read position: 22:38,681,603\n",
      "INFO\t2017-01-12 16:28:44\tGenotypeConcordance\tchecked    32,410,000 variants.  Elapsed time: 00:15:50s.  Time for last 10,000:    0s.  Last read position: 22:39,538,816\n",
      "INFO\t2017-01-12 16:28:44\tGenotypeConcordance\tchecked    32,420,000 variants.  Elapsed time: 00:15:50s.  Time for last 10,000:    0s.  Last read position: 22:40,431,574\n",
      "INFO\t2017-01-12 16:28:44\tGenotypeConcordance\tchecked    32,430,000 variants.  Elapsed time: 00:15:51s.  Time for last 10,000:    0s.  Last read position: 22:41,534,436\n",
      "INFO\t2017-01-12 16:28:45\tGenotypeConcordance\tchecked    32,440,000 variants.  Elapsed time: 00:15:51s.  Time for last 10,000:    0s.  Last read position: 22:42,541,285\n",
      "INFO\t2017-01-12 16:28:45\tGenotypeConcordance\tchecked    32,450,000 variants.  Elapsed time: 00:15:51s.  Time for last 10,000:    0s.  Last read position: 22:43,264,421\n",
      "INFO\t2017-01-12 16:28:45\tGenotypeConcordance\tchecked    32,460,000 variants.  Elapsed time: 00:15:52s.  Time for last 10,000:    0s.  Last read position: 22:43,974,763\n",
      "INFO\t2017-01-12 16:28:46\tGenotypeConcordance\tchecked    32,470,000 variants.  Elapsed time: 00:15:52s.  Time for last 10,000:    0s.  Last read position: 22:44,739,132\n",
      "INFO\t2017-01-12 16:28:46\tGenotypeConcordance\tchecked    32,480,000 variants.  Elapsed time: 00:15:52s.  Time for last 10,000:    0s.  Last read position: 22:45,333,014\n",
      "INFO\t2017-01-12 16:28:46\tGenotypeConcordance\tchecked    32,490,000 variants.  Elapsed time: 00:15:52s.  Time for last 10,000:    0s.  Last read position: 22:46,056,236\n",
      "INFO\t2017-01-12 16:28:47\tGenotypeConcordance\tchecked    32,500,000 variants.  Elapsed time: 00:15:53s.  Time for last 10,000:    0s.  Last read position: 22:46,789,230\n",
      "INFO\t2017-01-12 16:28:47\tGenotypeConcordance\tchecked    32,510,000 variants.  Elapsed time: 00:15:53s.  Time for last 10,000:    0s.  Last read position: 22:47,424,320\n",
      "INFO\t2017-01-12 16:28:47\tGenotypeConcordance\tchecked    32,520,000 variants.  Elapsed time: 00:15:53s.  Time for last 10,000:    0s.  Last read position: 22:48,039,140\n",
      "INFO\t2017-01-12 16:28:48\tGenotypeConcordance\tchecked    32,530,000 variants.  Elapsed time: 00:15:54s.  Time for last 10,000:    0s.  Last read position: 22:48,719,417\n",
      "INFO\t2017-01-12 16:28:48\tGenotypeConcordance\tchecked    32,540,000 variants.  Elapsed time: 00:15:54s.  Time for last 10,000:    0s.  Last read position: 22:49,244,940\n",
      "INFO\t2017-01-12 16:28:48\tGenotypeConcordance\tchecked    32,550,000 variants.  Elapsed time: 00:15:54s.  Time for last 10,000:    0s.  Last read position: 22:49,709,319\n",
      "INFO\t2017-01-12 16:28:49\tGenotypeConcordance\tchecked    32,560,000 variants.  Elapsed time: 00:15:55s.  Time for last 10,000:    0s.  Last read position: 22:50,325,778\n",
      "INFO\t2017-01-12 16:28:49\tGenotypeConcordance\tchecked    32,570,000 variants.  Elapsed time: 00:15:55s.  Time for last 10,000:    0s.  Last read position: 22:51,047,884\n",
      "INFO\t2017-01-12 16:28:49\tGenotypeConcordance\tchecked    32,580,000 variants.  Elapsed time: 00:15:55s.  Time for last 10,000:    0s.  Last read position: X:935,063\n",
      "INFO\t2017-01-12 16:28:50\tGenotypeConcordance\tchecked    32,590,000 variants.  Elapsed time: 00:15:56s.  Time for last 10,000:    0s.  Last read position: X:2,361,712\n",
      "INFO\t2017-01-12 16:28:50\tGenotypeConcordance\tchecked    32,600,000 variants.  Elapsed time: 00:15:56s.  Time for last 10,000:    0s.  Last read position: X:5,125,475\n",
      "INFO\t2017-01-12 16:28:50\tGenotypeConcordance\tchecked    32,610,000 variants.  Elapsed time: 00:15:56s.  Time for last 10,000:    0s.  Last read position: X:9,429,617\n",
      "INFO\t2017-01-12 16:28:51\tGenotypeConcordance\tchecked    32,620,000 variants.  Elapsed time: 00:15:57s.  Time for last 10,000:    0s.  Last read position: X:14,769,205\n",
      "INFO\t2017-01-12 16:28:51\tGenotypeConcordance\tchecked    32,630,000 variants.  Elapsed time: 00:15:57s.  Time for last 10,000:    0s.  Last read position: X:20,705,016\n",
      "INFO\t2017-01-12 16:28:51\tGenotypeConcordance\tchecked    32,640,000 variants.  Elapsed time: 00:15:57s.  Time for last 10,000:    0s.  Last read position: X:25,016,033\n",
      "INFO\t2017-01-12 16:28:51\tGenotypeConcordance\tchecked    32,650,000 variants.  Elapsed time: 00:15:58s.  Time for last 10,000:    0s.  Last read position: X:29,596,864\n",
      "INFO\t2017-01-12 16:28:52\tGenotypeConcordance\tchecked    32,660,000 variants.  Elapsed time: 00:15:58s.  Time for last 10,000:    0s.  Last read position: X:33,419,349\n",
      "INFO\t2017-01-12 16:28:52\tGenotypeConcordance\tchecked    32,670,000 variants.  Elapsed time: 00:15:58s.  Time for last 10,000:    0s.  Last read position: X:38,417,258\n",
      "INFO\t2017-01-12 16:28:52\tGenotypeConcordance\tchecked    32,680,000 variants.  Elapsed time: 00:15:58s.  Time for last 10,000:    0s.  Last read position: X:44,067,038\n",
      "INFO\t2017-01-12 16:28:53\tGenotypeConcordance\tchecked    32,690,000 variants.  Elapsed time: 00:15:59s.  Time for last 10,000:    0s.  Last read position: X:49,731,605\n",
      "INFO\t2017-01-12 16:28:53\tGenotypeConcordance\tchecked    32,700,000 variants.  Elapsed time: 00:15:59s.  Time for last 10,000:    0s.  Last read position: X:56,420,605\n",
      "INFO\t2017-01-12 16:28:53\tGenotypeConcordance\tchecked    32,710,000 variants.  Elapsed time: 00:16:00s.  Time for last 10,000:    0s.  Last read position: X:66,541,098\n",
      "INFO\t2017-01-12 16:28:54\tGenotypeConcordance\tchecked    32,720,000 variants.  Elapsed time: 00:16:00s.  Time for last 10,000:    0s.  Last read position: X:71,953,657\n",
      "INFO\t2017-01-12 16:28:54\tGenotypeConcordance\tchecked    32,730,000 variants.  Elapsed time: 00:16:00s.  Time for last 10,000:    0s.  Last read position: X:77,354,225\n",
      "INFO\t2017-01-12 16:28:54\tGenotypeConcordance\tchecked    32,740,000 variants.  Elapsed time: 00:16:01s.  Time for last 10,000:    0s.  Last read position: X:82,045,482\n",
      "INFO\t2017-01-12 16:28:55\tGenotypeConcordance\tchecked    32,750,000 variants.  Elapsed time: 00:16:01s.  Time for last 10,000:    0s.  Last read position: X:86,374,697\n",
      "INFO\t2017-01-12 16:28:55\tGenotypeConcordance\tchecked    32,760,000 variants.  Elapsed time: 00:16:01s.  Time for last 10,000:    0s.  Last read position: X:89,567,544\n",
      "INFO\t2017-01-12 16:28:55\tGenotypeConcordance\tchecked    32,770,000 variants.  Elapsed time: 00:16:02s.  Time for last 10,000:    0s.  Last read position: X:93,423,195\n",
      "INFO\t2017-01-12 16:28:56\tGenotypeConcordance\tchecked    32,780,000 variants.  Elapsed time: 00:16:02s.  Time for last 10,000:    0s.  Last read position: X:99,281,815\n",
      "INFO\t2017-01-12 16:28:56\tGenotypeConcordance\tchecked    32,790,000 variants.  Elapsed time: 00:16:02s.  Time for last 10,000:    0s.  Last read position: X:106,502,505\n",
      "INFO\t2017-01-12 16:28:56\tGenotypeConcordance\tchecked    32,800,000 variants.  Elapsed time: 00:16:03s.  Time for last 10,000:    0s.  Last read position: X:113,702,053\n",
      "INFO\t2017-01-12 16:28:57\tGenotypeConcordance\tchecked    32,810,000 variants.  Elapsed time: 00:16:03s.  Time for last 10,000:    0s.  Last read position: X:117,536,606\n",
      "INFO\t2017-01-12 16:28:57\tGenotypeConcordance\tchecked    32,820,000 variants.  Elapsed time: 00:16:03s.  Time for last 10,000:    0s.  Last read position: X:122,065,109\n",
      "INFO\t2017-01-12 16:28:58\tGenotypeConcordance\tchecked    32,830,000 variants.  Elapsed time: 00:16:04s.  Time for last 10,000:    0s.  Last read position: X:126,859,678\n",
      "INFO\t2017-01-12 16:28:58\tGenotypeConcordance\tchecked    32,840,000 variants.  Elapsed time: 00:16:04s.  Time for last 10,000:    0s.  Last read position: X:133,271,121\n",
      "INFO\t2017-01-12 16:28:58\tGenotypeConcordance\tchecked    32,850,000 variants.  Elapsed time: 00:16:04s.  Time for last 10,000:    0s.  Last read position: X:139,596,676\n",
      "INFO\t2017-01-12 16:28:58\tGenotypeConcordance\tchecked    32,860,000 variants.  Elapsed time: 00:16:05s.  Time for last 10,000:    0s.  Last read position: X:143,268,766\n",
      "INFO\t2017-01-12 16:28:59\tGenotypeConcordance\tchecked    32,870,000 variants.  Elapsed time: 00:16:05s.  Time for last 10,000:    0s.  Last read position: X:147,221,049\n",
      "INFO\t2017-01-12 16:28:59\tGenotypeConcordance\tchecked    32,880,000 variants.  Elapsed time: 00:16:05s.  Time for last 10,000:    0s.  Last read position: X:152,563,618\n",
      "INFO\t2017-01-12 16:29:00\tGenotypeConcordance\tchecked    32,890,000 variants.  Elapsed time: 00:16:06s.  Time for last 10,000:    0s.  Last read position: Y:9,950,470\n",
      "INFO\t2017-01-12 16:29:00\tGenotypeConcordance\tchecked    32,900,000 variants.  Elapsed time: 00:16:06s.  Time for last 10,000:    0s.  Last read position: Y:13,275,853\n",
      "INFO\t2017-01-12 16:29:00\tGenotypeConcordance\tchecked    32,910,000 variants.  Elapsed time: 00:16:06s.  Time for last 10,000:    0s.  Last read position: Y:13,489,619\n",
      "INFO\t2017-01-12 16:29:01\tGenotypeConcordance\tchecked    32,920,000 variants.  Elapsed time: 00:16:07s.  Time for last 10,000:    0s.  Last read position: Y:28,795,146\n",
      "INFO\t2017-01-12 16:29:01\tGenotypeConcordance\tchecked    32,930,000 variants.  Elapsed time: 00:16:07s.  Time for last 10,000:    0s.  Last read position: GL000235.1:27,259\n",
      "INFO\t2017-01-12 16:29:01\tGenotypeConcordance\tchecked    32,940,000 variants.  Elapsed time: 00:16:07s.  Time for last 10,000:    0s.  Last read position: GL000236.1:20,854\n",
      "INFO\t2017-01-12 16:29:02\tGenotypeConcordance\tchecked    32,950,000 variants.  Elapsed time: 00:16:08s.  Time for last 10,000:    0s.  Last read position: GL000208.1:75,947\n",
      "INFO\t2017-01-12 16:29:02\tGenotypeConcordance\tchecked    32,960,000 variants.  Elapsed time: 00:16:08s.  Time for last 10,000:    0s.  Last read position: GL000209.1:97,874\n",
      "INFO\t2017-01-12 16:29:02\tGenotypeConcordance\tchecked    32,970,000 variants.  Elapsed time: 00:16:08s.  Time for last 10,000:    0s.  Last read position: GL000211.1:121,478\n",
      "INFO\t2017-01-12 16:29:03\tGenotypeConcordance\tchecked    32,980,000 variants.  Elapsed time: 00:16:09s.  Time for last 10,000:    0s.  Last read position: GL000217.1:134,987\n",
      "INFO\t2017-01-12 16:29:03\tGenotypeConcordance\tchecked    32,990,000 variants.  Elapsed time: 00:16:09s.  Time for last 10,000:    0s.  Last read position: GL000216.1:170,739\n",
      "INFO\t2017-01-12 16:29:03\tGenotypeConcordance\tchecked    33,000,000 variants.  Elapsed time: 00:16:09s.  Time for last 10,000:    0s.  Last read position: GL000219.1:126,521\n",
      "INFO\t2017-01-12 16:29:04\tGenotypeConcordance\tchecked    33,010,000 variants.  Elapsed time: 00:16:10s.  Time for last 10,000:    0s.  Last read position: GL000195.1:67,098\n",
      "INFO\t2017-01-12 16:29:04\tGenotypeConcordance\tchecked    33,020,000 variants.  Elapsed time: 00:16:10s.  Time for last 10,000:    0s.  Last read position: GL000193.1:26,272\n",
      "INFO\t2017-01-12 16:29:04\tGenotypeConcordance\tchecked    33,030,000 variants.  Elapsed time: 00:16:10s.  Time for last 10,000:    0s.  Last read position: GL000225.1:33,627\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tchecked    33,040,000 variants.  Elapsed time: 00:16:11s.  Time for last 10,000:    0s.  Last read position: GL000225.1:131,216\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: NO_VARIATION MIXED 859\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MIXED MIXED 1055\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP MNP 4495\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: INDEL MNP 1142\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MIXED MNP 529\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP NO_VARIATION 3927\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP INDEL 1157\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MNP MIXED 541\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: MIXED NO_VARIATION 880\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: SNP INDEL 1736\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: NO_VARIATION NO_VARIATION 26208450\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: NO_VARIATION MNP 3742\n",
      "INFO\t2017-01-12 16:29:05\tGenotypeConcordance\tUncovered truth/call Variant Context Type Counts: INDEL SNP 1720\n",
      "[Thu Jan 12 16:29:05 PST 2017] picard.vcf.GenotypeConcordance done. Elapsed time: 16.19 minutes.\n",
      "Runtime.totalMemory()=2307915776\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "############Concordance table met Picard \n",
    "import subprocess\n",
    "\n",
    "samples = [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"]\n",
    "\n",
    "for file_x in samples:\n",
    "    for file_y in samples:\n",
    "        print(\"\\n\")\n",
    "\n",
    "        terminal_code = (\"java -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance \" + \\\n",
    "        \" CALL_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz \" + \\\n",
    "        \" CALL_SAMPLE= \" + file_x + \\\n",
    "        \" O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_\" + file_x + \"_truth_\" + file_y + \".vcf \" + \\\n",
    "        \" TRUTH_VCF=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/whole_genome/merged_RNA_imputed_and_WGS_data_new_header.vcf.gz \" + \\\n",
    "        \" TRUTH_SAMPLE= \" + file_y )\n",
    "    \n",
    "        #print(terminal_code)\n",
    "        print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "######Combine the picard concordance summary tables in one matrix \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "files = glob.glob(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/*.vcf.genotype_concordance_summary_metrics\")\n",
    "\n",
    "#data = pandas.DataFrame(pandas.read_csv(file, skiprows=6, skipfooter= 7, sep= \"\\t\"))\n",
    "data = pandas.DataFrame()\n",
    "\n",
    "for file in files:\n",
    "   data = data.append(pandas.read_csv(file, skiprows=6, sep= \"\\t\"))\n",
    "\n",
    "data = data[data.VARIANT_TYPE == \"SNP\"]\n",
    "\n",
    "\n",
    "pandas.DataFrame.to_csv(data, \"/data/scratch/iholtman/human_microglia_genotyping/jupyter/concordance_summary_picard_SNP_only.csv\")\n",
    "#print(data.GENOTYPE_CONCORDANCE)\n",
    "#print(data.TRUTH_SAMPLE)\n",
    "#print(data.CALL_SAMPLE)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   CALL_SAMPLE TRUTH_SAMPLE  GENOTYPE_CONCORDANCE\n",
      "0      imp_P11      imp_P10              0.921740\n",
      "0       imp_P8       WGS_P8              0.470092\n",
      "0       WGS_P8       imp_P8              0.470092\n",
      "0       imp_P7       WGS_P7              0.456053\n",
      "0       WGS_P7       imp_P7              0.456053\n",
      "0       imp_P6      imp_P29              0.455230\n",
      "0      imp_P29       imp_P6              0.455230\n",
      "0      imp_P29       imp_P2              0.451618\n",
      "0       imp_P2      imp_P29              0.451618\n",
      "0      imp_P18      imp_P29              0.451343\n",
      "0      imp_P29      imp_P18              0.451343\n",
      "0       imp_P2       imp_P6              0.451267\n",
      "0       imp_P6       imp_P2              0.451267\n",
      "0      imp_P14      imp_P18              0.450490\n",
      "0      imp_P18      imp_P14              0.450490\n",
      "0      imp_P18      imp_P12              0.449436\n",
      "0      imp_P12      imp_P18              0.449436\n",
      "0      imp_P16      imp_P18              0.449011\n",
      "0      imp_P18      imp_P16              0.449011\n",
      "0      WGS_P17      imp_P17              0.449005\n",
      "0      imp_P17      WGS_P17              0.449005\n",
      "0      imp_P14      imp_P29              0.448828\n",
      "0      imp_P29      imp_P14              0.448828\n",
      "0       imp_P6      imp_P21              0.448640\n",
      "0      imp_P21       imp_P6              0.448640\n",
      "0      imp_P29      imp_P19              0.448619\n",
      "0      imp_P19      imp_P29              0.448619\n",
      "0       imp_P3      imp_P18              0.448617\n",
      "0      imp_P18       imp_P3              0.448617\n",
      "0      imp_P15       imp_P6              0.448553\n",
      "..         ...          ...                   ...\n",
      "0      imp_P10      WGS_P15              0.346928\n",
      "0      WGS_P15      imp_P10              0.346928\n",
      "0       WGS_P5      imp_P19              0.346586\n",
      "0      imp_P19       WGS_P5              0.346586\n",
      "0       imp_P8       WGS_P4              0.346491\n",
      "0       WGS_P4       imp_P8              0.346491\n",
      "0      imp_P21       WGS_P8              0.346291\n",
      "0       WGS_P8      imp_P21              0.346291\n",
      "0       imp_P8      WGS_P15              0.346204\n",
      "0      WGS_P15       imp_P8              0.346204\n",
      "0      WGS_P15      imp_P14              0.346129\n",
      "0      imp_P14      WGS_P15              0.346129\n",
      "0      imp_P16       WGS_P4              0.345285\n",
      "0       WGS_P4      imp_P16              0.345285\n",
      "0      imp_P19      WGS_P15              0.345148\n",
      "0      WGS_P15      imp_P19              0.345148\n",
      "0       WGS_P4      imp_P19              0.344132\n",
      "0      imp_P19       WGS_P4              0.344132\n",
      "0      imp_P27       WGS_P7              0.336849\n",
      "0       WGS_P7      imp_P27              0.336849\n",
      "0       WGS_P8      imp_P27              0.336633\n",
      "0      imp_P27       WGS_P8              0.336633\n",
      "0       WGS_P4      imp_P27              0.335580\n",
      "0      imp_P27       WGS_P4              0.335580\n",
      "0      imp_P27       WGS_P5              0.335360\n",
      "0       WGS_P5      imp_P27              0.335360\n",
      "0      WGS_P17      imp_P27              0.335358\n",
      "0      imp_P27      WGS_P17              0.335358\n",
      "0      imp_P27      WGS_P15              0.334348\n",
      "0      WGS_P15      imp_P27              0.334348\n",
      "\n",
      "[811 rows x 3 columns]\n"
     ]
    }
   ],
   "source": [
    "####### Select SNPs only \n",
    "\n",
    "data = data[data.VARIANT_TYPE == \"SNP\"]\n",
    "df1 = data.ix[:, ['CALL_SAMPLE', 'TRUTH_SAMPLE', 'GENOTYPE_CONCORDANCE']]\n",
    "\n",
    "\n",
    "df1 = df1.sort_values(by ='GENOTYPE_CONCORDANCE', ascending = False)\n",
    "df1 = df1[df1.GENOTYPE_CONCORDANCE < 1]\n",
    "\n",
    "print(df1[1:1000])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   VARIANT_TYPE TRUTH_SAMPLE CALL_SAMPLE  TP_COUNT  TN_COUNT  FP_COUNT  \\\n",
      "0           SNP       imp_P6      imp_P8   1641241   1122198   1058687   \n",
      "1         INDEL       imp_P6      imp_P8    248091    155079    153776   \n",
      "0           SNP       WGS_P5     imp_P10   1786384   1586724    489782   \n",
      "1         INDEL       WGS_P5     imp_P10    252631    253874     95643   \n",
      "0           SNP      imp_P18      imp_P6   1671396   1035710    861509   \n",
      "1         INDEL      imp_P18      imp_P6    252052    143023    126144   \n",
      "0           SNP      imp_P26     imp_P10   1706214   1124873    947173   \n",
      "1         INDEL      imp_P26     imp_P10    258394    155376    137099   \n",
      "0           SNP       imp_P2     imp_P16   1670296   1066973    899615   \n",
      "1         INDEL       imp_P2     imp_P16    252759    147831    131353   \n",
      "0           SNP      imp_P17     WGS_P15   1820869   1552574   1662894   \n",
      "1         INDEL      imp_P17     WGS_P15    254028    216440    285250   \n",
      "0           SNP      imp_P19     imp_P17   1705159   1136446    985462   \n",
      "1         INDEL      imp_P19     imp_P17    258993    158763    144939   \n",
      "0           SNP      imp_P12     WGS_P17   1734215   1580561   1750790   \n",
      "1         INDEL      imp_P12     WGS_P17    240318    220635    299484   \n",
      "0           SNP       imp_P3     imp_P29   1723829   1141409    885067   \n",
      "1         INDEL       imp_P3     imp_P29    260987    157837    131732   \n",
      "0           SNP      imp_P14      imp_P8   1672337   1153744   1041773   \n",
      "1         INDEL      imp_P14      imp_P8    253136    159060    150934   \n",
      "0           SNP       imp_P6     imp_P17   1706490   1159069    991523   \n",
      "1         INDEL       imp_P6     imp_P17    257925    162504    147394   \n",
      "0           SNP       imp_P4     imp_P31   1662831   1155813    988668   \n",
      "1         INDEL       imp_P4     imp_P31    252564    160451    144817   \n",
      "0           SNP       WGS_P5     imp_P19   1719534   1593702    482808   \n",
      "1         INDEL       WGS_P5     imp_P19    243788    254356     94335   \n",
      "0           SNP      imp_P19     imp_P18   1668624   1015087    875821   \n",
      "1         INDEL      imp_P19     imp_P18    252735    140064    127719   \n",
      "0           SNP      imp_P26     imp_P11   1709051   1123643    944224   \n",
      "1         INDEL      imp_P26     imp_P11    258551    155254    136849   \n",
      "..          ...          ...         ...       ...       ...       ...   \n",
      "0           SNP      imp_P29      imp_P8   1684396   1195282   1047304   \n",
      "1         INDEL      imp_P29      imp_P8    255902    165152    150689   \n",
      "0           SNP      imp_P10      imp_P5   1684297   1145085    931365   \n",
      "1         INDEL      imp_P10      imp_P5    255071    158358    135540   \n",
      "0           SNP      imp_P19      imp_P6   1661691   1008864    857409   \n",
      "1         INDEL      imp_P19      imp_P6    250872    139287    125303   \n",
      "0           SNP       imp_P3      imp_P8   1678174   1153880   1029347   \n",
      "1         INDEL       imp_P3      imp_P8    254350    158884    148746   \n",
      "0           SNP       imp_P6      imp_P3   1676529   1069301    914709   \n",
      "1         INDEL       imp_P6      imp_P3    252849    147600    134704   \n",
      "0           SNP       imp_P7     imp_P31   1670278   1135580    971010   \n",
      "1         INDEL       imp_P7     imp_P31    253043    158186    142344   \n",
      "0           SNP       imp_P2     imp_P29   1723966   1133071    892857   \n",
      "1         INDEL       imp_P2     imp_P29    261458    156921    132541   \n",
      "0           SNP      imp_P18     imp_P24   1711122   1190671   1005386   \n",
      "1         INDEL      imp_P18     imp_P24    259126    166116    146268   \n",
      "0           SNP      imp_P31      imp_P7   1670278   1135580    967691   \n",
      "1         INDEL      imp_P31      imp_P7    253043    158186    141150   \n",
      "0           SNP       imp_P6     imp_P11   1676206   1084363    975323   \n",
      "1         INDEL       imp_P6     imp_P11    252594    149627    142490   \n",
      "0           SNP      imp_P15      imp_P5   1673305   1118833    935958   \n",
      "1         INDEL      imp_P15      imp_P5    254154    155299    135930   \n",
      "0           SNP      imp_P23      imp_P3   1716602   1091549    875816   \n",
      "1         INDEL      imp_P23      imp_P3    259770    152734    128323   \n",
      "0           SNP      imp_P10     imp_P12   1688271   1084472    892424   \n",
      "1         INDEL      imp_P10     imp_P12    255024    149181    130677   \n",
      "0           SNP      imp_P18     imp_P21   1653108   1010133    865670   \n",
      "1         INDEL      imp_P18     imp_P21    249732    139641    126850   \n",
      "0           SNP       imp_P4     imp_P19   1632797   1081538    936342   \n",
      "1         INDEL       imp_P4     imp_P19    248098    149732    137139   \n",
      "\n",
      "    FN_COUNT  EMPTY_COUNT  \n",
      "0     911868       126798  \n",
      "1     134079        37052  \n",
      "0    1724492      1537217  \n",
      "1     317200       622725  \n",
      "0     877533       119655  \n",
      "1     130279        35653  \n",
      "0     912866       123860  \n",
      "1     135101        37071  \n",
      "0     873954       120484  \n",
      "1     129715        35912  \n",
      "0     458550      1545867  \n",
      "1      90941       553887  \n",
      "0     882605       125538  \n",
      "1     129256        37330  \n",
      "0     479862      1513596  \n",
      "1      93257       550641  \n",
      "0     900161       121685  \n",
      "1     131377        36531  \n",
      "0     925158       126123  \n",
      "1     135597        36933  \n",
      "0     862167       126174  \n",
      "1     126439        37200  \n",
      "0     952931       125385  \n",
      "1     139749        37211  \n",
      "0    1776359      1499822  \n",
      "1     323834       618212  \n",
      "0     881824       119752  \n",
      "1     129181        35704  \n",
      "0     909778       124031  \n",
      "1     134641        37041  \n",
      "..       ...          ...  \n",
      "0     942360       127995  \n",
      "1     140294        37541  \n",
      "0     985804       124895  \n",
      "1     143723        37186  \n",
      "0     884638       119553  \n",
      "1     130691        35700  \n",
      "0     939040       126374  \n",
      "1     137673        37145  \n",
      "0     858875       119891  \n",
      "1     126266        36020  \n",
      "0     967898       126368  \n",
      "1     141724        37410  \n",
      "0     855033       120783  \n",
      "1     126192        36224  \n",
      "0     889172       125431  \n",
      "1     131529        37110  \n",
      "0     971212       126368  \n",
      "1     142861        37410  \n",
      "0     860858       122913  \n",
      "1     127185        36463  \n",
      "0     938731       125918  \n",
      "1     138414        36988  \n",
      "0     964114       123591  \n",
      "1     143034        36964  \n",
      "0     957640       122610  \n",
      "1     139900        36497  \n",
      "0     890686       119967  \n",
      "1     131505        35490  \n",
      "0     960964       123230  \n",
      "1     140372        36722  \n",
      "\n",
      "[1682 rows x 8 columns]\n"
     ]
    }
   ],
   "source": [
    "######Combine the picard concordance contingency tables in one matrix \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "files = glob.glob(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/*.vcf.genotype_concordance_contingency_metrics\")\n",
    "\n",
    "#data = pandas.DataFrame(pandas.read_csv(file, skiprows=6, skipfooter= 7, sep= \"\\t\"))\n",
    "data = pandas.DataFrame()\n",
    "\n",
    "for file in files:\n",
    "   data = data.append(pandas.read_csv(file, skiprows=6, sep= \"\\t\"))\n",
    "\n",
    "\n",
    "print(data)\n",
    "#df1 = data.ix[:, ['CALL_SAMPLE', 'TRUTH_SAMPLE', 'GENOTYPE_CONCORDANCE']]\n",
    "#print(df1)\n",
    "\n",
    "pandas.DataFrame.to_csv(data, \"/data/scratch/iholtman/human_microglia_genotyping/jupyter/concordance_contingency_picard.csv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   VARIANT_TYPE TRUTH_SAMPLE CALL_SAMPLE  TP_COUNT  TN_COUNT  FP_COUNT  \\\n",
      "0           SNP       imp_P6      imp_P8   1641241   1122198   1058687   \n",
      "0           SNP       WGS_P5     imp_P10   1786384   1586724    489782   \n",
      "0           SNP      imp_P18      imp_P6   1671396   1035710    861509   \n",
      "0           SNP      imp_P26     imp_P10   1706214   1124873    947173   \n",
      "0           SNP       imp_P2     imp_P16   1670296   1066973    899615   \n",
      "0           SNP      imp_P17     WGS_P15   1820869   1552574   1662894   \n",
      "0           SNP      imp_P19     imp_P17   1705159   1136446    985462   \n",
      "0           SNP      imp_P12     WGS_P17   1734215   1580561   1750790   \n",
      "0           SNP       imp_P3     imp_P29   1723829   1141409    885067   \n",
      "0           SNP      imp_P14      imp_P8   1672337   1153744   1041773   \n",
      "0           SNP       imp_P6     imp_P17   1706490   1159069    991523   \n",
      "0           SNP       imp_P4     imp_P31   1662831   1155813    988668   \n",
      "0           SNP       WGS_P5     imp_P19   1719534   1593702    482808   \n",
      "0           SNP      imp_P19     imp_P18   1668624   1015087    875821   \n",
      "0           SNP      imp_P26     imp_P11   1709051   1123643    944224   \n",
      "0           SNP       WGS_P4     imp_P17   1814581   1570977    458441   \n",
      "0           SNP       WGS_P5     imp_P26   1775223   1595546    480961   \n",
      "0           SNP      imp_P16     imp_P27   1669438   1244010   1091070   \n",
      "0           SNP      imp_P17     imp_P19   1705159   1136446    882377   \n",
      "0           SNP      imp_P15     imp_P21   1663553   1031711    852905   \n",
      "0           SNP       imp_P5     imp_P18   1659706   1102995    910776   \n",
      "0           SNP      imp_P16     imp_P14   1680721   1069868    888055   \n",
      "0           SNP      imp_P19      imp_P2   1667073   1040993    871300   \n",
      "0           SNP       WGS_P8     imp_P18   1758828   1615148    470805   \n",
      "0           SNP      imp_P23      WGS_P4   1753365   1541142   1711164   \n",
      "0           SNP       WGS_P5     imp_P23   1797288   1568242    508274   \n",
      "0           SNP       imp_P3     imp_P18   1699061   1069866    859680   \n",
      "0           SNP      imp_P10      imp_P4   1651316   1156639    960556   \n",
      "0           SNP       imp_P4      imp_P6   1651445   1100784    901223   \n",
      "0           SNP      imp_P15     imp_P29   1717319   1129337    891544   \n",
      "..          ...          ...         ...       ...       ...       ...   \n",
      "0           SNP      imp_P18      WGS_P5   1741550   1597272   1766659   \n",
      "0           SNP      WGS_P15     imp_P29   1778994   1563164    447462   \n",
      "0           SNP       imp_P3     imp_P10   1698031   1121469    958578   \n",
      "0           SNP      imp_P10     imp_P16   1682228   1089210    886436   \n",
      "0           SNP      imp_P12      imp_P4   1640578   1103994    956812   \n",
      "0           SNP      WGS_P15     imp_P11   1749853   1532386    478247   \n",
      "0           SNP       imp_P6     imp_P29   1720217   1101290    882113   \n",
      "0           SNP       WGS_P8     imp_P14   1756137   1621497    464521   \n",
      "0           SNP       imp_P5     imp_P10   1684297   1145085    985600   \n",
      "0           SNP      imp_P17      imp_P7   1712447   1202286    951876   \n",
      "0           SNP       WGS_P5      imp_P7   1764194   1591364    485135   \n",
      "0           SNP      imp_P14     imp_P21   1652772   1047091    876010   \n",
      "0           SNP       imp_P4     imp_P17   1700364   1221726   1022730   \n",
      "0           SNP      imp_P10      WGS_P4   1748130   1558665   1720945   \n",
      "0           SNP      imp_P14     imp_P27   1690431   1272565   1084084   \n",
      "0           SNP      imp_P29      imp_P8   1684396   1195282   1047304   \n",
      "0           SNP      imp_P10      imp_P5   1684297   1145085    931365   \n",
      "0           SNP      imp_P19      imp_P6   1661691   1008864    857409   \n",
      "0           SNP       imp_P3      imp_P8   1678174   1153880   1029347   \n",
      "0           SNP       imp_P6      imp_P3   1676529   1069301    914709   \n",
      "0           SNP       imp_P7     imp_P31   1670278   1135580    971010   \n",
      "0           SNP       imp_P2     imp_P29   1723966   1133071    892857   \n",
      "0           SNP      imp_P18     imp_P24   1711122   1190671   1005386   \n",
      "0           SNP      imp_P31      imp_P7   1670278   1135580    967691   \n",
      "0           SNP       imp_P6     imp_P11   1676206   1084363    975323   \n",
      "0           SNP      imp_P15      imp_P5   1673305   1118833    935958   \n",
      "0           SNP      imp_P23      imp_P3   1716602   1091549    875816   \n",
      "0           SNP      imp_P10     imp_P12   1688271   1084472    892424   \n",
      "0           SNP      imp_P18     imp_P21   1653108   1010133    865670   \n",
      "0           SNP       imp_P4     imp_P19   1632797   1081538    936342   \n",
      "\n",
      "    FN_COUNT  EMPTY_COUNT  \n",
      "0     911868       126798  \n",
      "0    1724492      1537217  \n",
      "0     877533       119655  \n",
      "0     912866       123860  \n",
      "0     873954       120484  \n",
      "0     458550      1545867  \n",
      "0     882605       125538  \n",
      "0     479862      1513596  \n",
      "0     900161       121685  \n",
      "0     925158       126123  \n",
      "0     862167       126174  \n",
      "0     952931       125385  \n",
      "0    1776359      1499822  \n",
      "0     881824       119752  \n",
      "0     909778       124031  \n",
      "0    1684676      1543638  \n",
      "0    1734441      1517742  \n",
      "0     934128       124601  \n",
      "0     985657       125538  \n",
      "0     918256       121386  \n",
      "0     951117       123695  \n",
      "0     892597       120435  \n",
      "0     893456       118931  \n",
      "0    1783426      1445520  \n",
      "0     488852      1551450  \n",
      "0    1708143      1537100  \n",
      "0     896988       119980  \n",
      "0    1021807       125998  \n",
      "0     949866       123720  \n",
      "0     911565       124157  \n",
      "..       ...          ...  \n",
      "0     479858      1489396  \n",
      "0    1698411      1494381  \n",
      "0     903267       122288  \n",
      "0     963624       122120  \n",
      "0     953273       124466  \n",
      "0    1711874      1554408  \n",
      "0     843937       121445  \n",
      "0    1797161      1473810  \n",
      "0     931563       124895  \n",
      "0     999738       129166  \n",
      "0    1745550      1532280  \n",
      "0     911004       119593  \n",
      "0     941750       128354  \n",
      "0     471375      1545833  \n",
      "0     926039       124353  \n",
      "0     942360       127995  \n",
      "0     985804       124895  \n",
      "0     884638       119553  \n",
      "0     939040       126374  \n",
      "0     858875       119891  \n",
      "0     967898       126368  \n",
      "0     855033       120783  \n",
      "0     889172       125431  \n",
      "0     971212       126368  \n",
      "0     860858       122913  \n",
      "0     938731       125918  \n",
      "0     964114       123591  \n",
      "0     957640       122610  \n",
      "0     890686       119967  \n",
      "0     960964       123230  \n",
      "\n",
      "[841 rows x 8 columns]\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  VARIANT_TYPE TRUTH_SAMPLE CALL_SAMPLE  TP_COUNT  TN_COUNT  FP_COUNT  \\\n",
      "0          SNP      imp_P21     imp_P21   2323688    765912         0   \n",
      "0          SNP      imp_P27     imp_P27   2583458   1069683         0   \n",
      "0          SNP      imp_P31     imp_P31   2438466    882373         0   \n",
      "0          SNP      imp_P26     imp_P26   2422642    859629         0   \n",
      "0          SNP       imp_P3      imp_P3   2402946    857906         0   \n",
      "0          SNP       WGS_P7      WGS_P7   3640025   2273234         0   \n",
      "0          SNP      imp_P16     imp_P16   2368254    806593         0   \n",
      "0          SNP      imp_P18     imp_P18   2361673    816297         0   \n",
      "0          SNP       WGS_P8      WGS_P8   3634080   2206052         0   \n",
      "0          SNP      imp_P15     imp_P15   2400437    839362         0   \n",
      "\n",
      "   FN_COUNT  EMPTY_COUNT     True  False  ratio_T/F  \n",
      "0         0        94594  3089600      0        inf  \n",
      "0         0       100938  3653141      0        inf  \n",
      "0         0        98525  3320839      0        inf  \n",
      "0         0        98848  3282271      0        inf  \n",
      "0         0        96090  3260852      0        inf  \n",
      "0         0       573281  5913259      0        inf  \n",
      "0         0        96315  3174847      0        inf  \n",
      "0         0        95784  3177970      0        inf  \n",
      "0         0       567370  5840132      0        inf  \n",
      "0         0        97899  3239799      0        inf  \n"
     ]
    }
   ],
   "source": [
    "######Combine the picard concordance contingency tables in one matrix \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "files = glob.glob(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/*.vcf.genotype_concordance_contingency_metrics\")\n",
    "\n",
    "#data = pandas.DataFrame(pandas.read_csv(file, skiprows=6, skipfooter= 7, sep= \"\\t\"))\n",
    "data = pandas.DataFrame()\n",
    "\n",
    "for file in files:\n",
    "   data = data.append(pandas.read_csv(file, skiprows=6, sep= \"\\t\"))\n",
    "\n",
    "data = data[data.VARIANT_TYPE == \"SNP\"]\n",
    "\n",
    "data['True'] = data['TP_COUNT'] + data['TN_COUNT'] \n",
    "data['False'] = data['FP_COUNT'] + data['FN_COUNT'] \n",
    "\n",
    "data['ratio_T/F'] = data['True'] / data['False']\n",
    "\n",
    "data = data.sort_values(by ='ratio_T/F', ascending = False)\n",
    "\n",
    "print(data[0:10])\n",
    "\n",
    "#pandas.DataFrame.to_csv(data, \"/data/scratch/iholtman/human_microglia_genotyping/jupyter/concordance_contingency_picard.csv\")\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  TRUTH_SAMPLE CALL_SAMPLE     True   False  ratio_T/F\n",
      "0      WGS_P17     WGS_P15  3142193  785758   3.998932\n",
      "0      WGS_P15     WGS_P17  3142193  785794   3.998749\n",
      "0       WGS_P7     WGS_P15  3159653  790236   3.998366\n",
      "0      WGS_P15      WGS_P7  3159653  790307   3.998007\n",
      "0       WGS_P7      WGS_P5  3171253  794414   3.991940\n",
      "0       WGS_P5      WGS_P7  3171253  794424   3.991890\n",
      "0       WGS_P4      WGS_P7  3158575  793800   3.979056\n",
      "0       WGS_P4      WGS_P5  3133311  787471   3.978954\n",
      "0       WGS_P7      WGS_P4  3158575  793822   3.978946\n",
      "0       WGS_P5      WGS_P4  3133311  787477   3.978924\n",
      "0       WGS_P7     WGS_P17  3174004  800309   3.965973\n",
      "0      WGS_P17      WGS_P7  3174004  800338   3.965829\n",
      "0       WGS_P4     WGS_P15  3136117  791415   3.962671\n",
      "0      WGS_P15      WGS_P4  3136117  791449   3.962500\n",
      "0      WGS_P17      WGS_P5  3158968  799847   3.949465\n",
      "0       WGS_P5     WGS_P17  3158968  799859   3.949406\n",
      "0       WGS_P4     WGS_P17  3111495  789071   3.943238\n",
      "0      WGS_P17      WGS_P4  3111495  789093   3.943128\n",
      "0       WGS_P5     WGS_P15  3127242  794293   3.937139\n",
      "0      WGS_P15      WGS_P5  3127242  794369   3.936762\n"
     ]
    }
   ],
   "source": [
    "#surveys_df[surveys_df.year == 2002]\n",
    "data = data[['TRUTH_SAMPLE', 'CALL_SAMPLE', 'True', 'False','ratio_T/F']]\n",
    "\n",
    "data = data[data['ratio_T/F'] < 10]\n",
    "\n",
    "#s[s > 0]\n",
    "\n",
    "print(data[0:20])\n",
    "#plot(data['ratio_T/F'])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAg0AAAFkCAYAAACjCwibAAAABHNCSVQICAgIfAhkiAAAAAlwSFlz\nAAAPYQAAD2EBqD+naQAAIABJREFUeJzt3XmUZWV57/HvQ89NdxcNAt0MgoCgRAN0EWQQQaAR4aLi\n5aIlSIPRMOiKt7MMDokJcq8goiAhAllOFzVWBhMxDpFBFK4CIXYFiUyNAS4g0DRTd0MPNPR7/3jP\noU4fatin6uyqOru+n7X2qnP22fvs9+1TXfWr53333pFSQpIkaThbjHcDJElSZzA0SJKkQgwNkiSp\nEEODJEkqxNAgSZIKMTRIkqRCDA2SJKkQQ4MkSSrE0CBJkgoxNEiSpEJGFRoi4hMRsSkiLh5mu8Mj\nYllErI+I5RGxZDTHlSRJY2/EoSEi/gD4I+DXw2y3K/BD4KfAPsClwFcjYvFIjy1JksbeiEJDRMwB\nvg18EHh2mM3PAu5PKZ2TUro3pfRl4LvA0pEcW5IkjY+RVhq+DPwgpXRDgW0PBK5vWncNcNAIjy1J\nksbB1FZ3iIj3AvsC+xfcZQGwomndCmBeRMxIKW0Y4BjbAG8DHgTWt9pGSZImsZnArsA1KaWn2vnG\nLYWGiNgJ+BJwVEppYzsb0uRtwN+W+P6SJFXdycB32vmGrVYauoFtgb6IiNq6KcBbIuIjwIyUUmra\n53Fg+6Z12wOrB6oy1DwIMGfOt3nuudfzD/8AO+4IM2e22NoJYunSpVxyySXj3Yy2sT8TV5X6AvZn\nIqtSX6Ba/bn77rs55ZRToPa7tJ1aDQ3XA29sWvd/gLuBzw0QGABuAd7etO7o2vrBrAf40Y9ez7HH\nLuKkk6CrCx58ELbaqsUWTwBdXV0sWrRovJvRNvZn4qpSX8D+TGRV6gtUrz81bR/eb2kiZErp+ZTS\nXY0L8DzwVErpboCIOD8irmrY7Upgt4i4MCL2ioizgROBIa/tADBnDtxyC/zd38GaNXDoofC977XS\nYkmS1C7tuCJkc3VhIbDzyy+m9CBwHHAUcDv5VMs/TCk1n1ExoDe+Ed7zHvjCF+CJJ+Dd785BQpIk\nja1Rh4aU0hEppT9peH56SumIpm1uSil1p5RmpZRem1L6VqvHWboUHn0UdtoJDj4Y/uZvYNOm0bZe\nkiQV1VH3npgyBX78YzjoIDjzzDxc8exwl5aaAHp6esa7CW1lfyauKvUF7M9EVqW+QPX6U5YYeO7i\n+IqIRcCyZcuWDTgxJSX4ylfgjDPgqqvg1FPHvo2SJE1EfX19dHd3A3SnlPra+d4dVWmoi4A/+iNY\ntAiWLIGHHhrvFkmSVH0dGRrqzjsvf122bHzbIUnSZNDRoeHYY2H27Hz9BkmSVK6ODg0RsOuu8MAD\n490SSZKqr6NDA8BrXgOXXWa1QZKksnV8aPiT2hUizjgDNgx2JwtJkjRqHR8ajjgCzjkHrr02n03h\nUIUkSeXo+NAAcOGFcPXVcNddcNNN490aSZKqqRKhAeCd74R58/L9KSRJUvtVJjQAbLcdrFw53q2Q\nJKmaKhcarDRIklSOSoWGbbe10iBJUlkqFRq22y7fBXPFivFuiSRJ1VOp0HDUUfnrddeNbzskSaqi\nSoWGd7wjf52Ad/uWJKnjVSo0ROSvhgZJktqvkqFBkiS1X6VCQ52VBkmS2q9SocHhCUmSymNokCRJ\nhVQyNEiSpParVGios9IgSVL7VSo0ODwhSVJ5DA2SJKmQSoYGSZLUfpUKDXVWGiRJaj9DgyRJKqRy\noSHC0CBJUhkqGRokSVL7VS40gJUGSZLKULnQ4PCEJEnlMDRIkqRCKhkaJElS+1UuNICVBkmSylC5\n0ODwhCRJ5TA0SJKkQioZGiRJUvtVLjSAlQZJkspQudDg8IQkSeVoKTRExJkR8euIWFVbbo6IY4bY\n/rCI2NS0vBQR242+6YMd09AgSVIZpra4/cPAx4H7gABOA74fEfumlO4eZJ8E7AmseXlFSk+03tRi\nnNMgSVI5WgoNKaUfNa3684g4CzgQGCw0AKxMKa1utXEjZaVBkqT2G/GchojYIiLeC8wGbhlqU+D2\niHg0Iq6NiINHesxi7TI0SJJUhlaHJ4iIN5BDwkzykMMJKaV7Btn8MeAM4FfADOBDwM8j4oCU0u0j\na/Jw7TM0SJJUhpZDA3APsA/QBZwIfDMi3jJQcEgpLQeWN6y6NSJ2B5YCS4Y70NKlS+nq6tpsXU9P\nDz09PYPu45wGSdJk0dvbS29v72brVq1aVdrxIo3yz/KIuA74bUrprILbfx44JKV0yBDbLAKWLVu2\njEWLFrXUnq4u+PSn4WMfa2k3SZIqoa+vj+7uboDulFJfO9+7Hddp2II89FDUvuRhi1I4PCFJUjla\nGp6IiPOBfwUeAuYCJwOHAUfXXr8A2CGltKT2/KPAA8Cd5DkQHwLeCixuU/sHaKOhQZKkMrQ6p2E7\n4CpgIbAKuAM4OqV0Q+31BcDODdtPB74I7ACsrW1/ZErpptE0eijOaZAkqRytXqfhg8O8fnrT84uA\ni0bQrlGx0iBJUvt57wlJklSIoUGSJBVSydAgSZLar3KhAaw0SJJUhsqFBocnJEkqh6FBkiQVUsnQ\nIEmS2q9yoQGsNEiSVIbKhQaHJyRJKoehQZIkFVLJ0CBJktqvcqEBrDRIklSGyoUGhyckSSqHoUGS\nJBVSydAgSZLar3KhAaw0SJJUhsqFBocnJEkqh6FBkiQVUsnQIEmS2q9yoQGsNEiSVIbKhQaHJyRJ\nKoehQZIkFVLJ0CBJktqvcqEBrDRIklSGyoUGhyckSSqHoUGSJBVSydAgSZLar3KhAaw0SJJUhsqF\nBocnJEkqh6FBkiQVUsnQIEmS2q9yoQGsNEiSVIbKhQaHJyRJKoehQZIkFVLJ0CBJktqvcqEBrDRI\nklSGyoUGhyckSSqHoUGSJBVSydAgSZLar3KhAaw0SJJUhpZCQ0ScGRG/johVteXmiDhmmH0Oj4hl\nEbE+IpZHxJLRNXm4NhoaJEkqQ6uVhoeBjwOLgG7gBuD7EfH6gTaOiF2BHwI/BfYBLgW+GhGLR9je\nYRkaJEkqx9RWNk4p/ahp1Z9HxFnAgcDdA+xyFnB/Sumc2vN7I+LNwFLgulYbW4RzGiRJKseI5zRE\nxBYR8V5gNnDLIJsdCFzftO4a4KCRHrcIKw2SJLVfS5UGgIh4AzkkzATWACeklO4ZZPMFwIqmdSuA\neRExI6W0odXjD98+Q4MkSWUYSaXhHvL8hAOAK4BvRsTr2tqqUTA0SJJUjpYrDSmlF4H7a0//IyIO\nAD5Knr/Q7HFg+6Z12wOri1QZli5dSldX12brenp66OnpGXQf5zRIkiaL3t5eent7N1u3atWq0o7X\ncmgYwBbAjEFeuwV4e9O6oxl8DsRmLrnkEhYtWtRyg6w0SJImg4H+kO7r66O7u7uU47UUGiLifOBf\ngYeAucDJwGHkIEBEXADskFKqX4vhSuDDEXEh8HXgSOBE4Ni2tH7ANhoaJEkqQ6uVhu2Aq4CFwCrg\nDuDolNINtdcXADvXN04pPRgRxwGXAH8MPAL8YUqp+YyKtjE0SJJUjlav0/DBYV4/fYB1N5EvBDUm\nnNMgSVI5vPeEJEkqpHKhweEJSZLKYWiQJEmFVDI0SJKk9qtcaAArDZIklaFyocHhCUmSymFokCRJ\nhVQyNEiSpParXGgAKw2SJJWhcqHB4QlJksphaJAkSYVUMjRIkqT2q1xoACsNkiSVoXKhweEJSZLK\nYWiQJEmFVDI0SJKk9qtcaAArDZIklaFyocHhCUmSymFokCRJhVQyNEiSpParXGgAKw2SJJWhcqHB\n4QlJksphaJAkSYVUMjRIkqT2q1xoACsNkiSVoXKhweEJSZLKYWiQJEmFVDI0SJKk9qtcaAArDZIk\nlaFyocHhCUmSymFokCRJhVQyNEiSpParXGgAKw2SJJWhcqHB4QlJksphaJAkSYVUMjRIkqT2q1xo\nACsNkiSVoXKhweEJSZLKYWiQJEmFVDI0SJKk9qtcaAArDZIklaGl0BARn4yI2yJidUSsiIjvRcSe\nw+xzWERsalpeiojtRtf0wY5naJAkqQytVhoOBS4D3gQcBUwDro2IWcPsl4DXAgtqy8KU0hMtHrsQ\nQ4MkSeWY2srGKaVjG59HxGnAE0A38Ithdl+ZUlrdUutGwDkNkiSVY7RzGrYiVxGeHma7AG6PiEcj\n4tqIOHiUxx2SlQZJktpvxKEhIgL4EvCLlNJdQ2z6GHAG8N+BdwMPAz+PiH1Heuyh22VokCSpDC0N\nTzS5HNgbOGSojVJKy4HlDatujYjdgaXAkqH2Xbp0KV1dXZut6+npoaenZ9B9DA2SpMmit7eX3t7e\nzdatWrWqtONFGsFv2Ij4a+B44NCU0kMj2P/zwCEppQEDR0QsApYtW7aMRYsWtfTe738/PPQQ3Hhj\nq62SJKnz9fX10d3dDdCdUupr53u3XGmoBYZ3AoeNJDDU7EsetiiFlQZJktqvpdAQEZcDPcA7gOcj\nYvvaS6tSSutr25wP7JhSWlJ7/lHgAeBOYCbwIeCtwOK29OAVbTQ0SJJUhlYrDWeSz5b4edP604Fv\n1h4vBHZueG068EVgB2AtcAdwZErpplYbW4ShQZKkcrR6nYZhz7ZIKZ3e9Pwi4KIW2zViXqdBkqRy\neO8JSZJUSOVCg8MTkiSVw9AgSZIKqWRokCRJ7Ve50ABWGiRJKkPlQoPDE5IklcPQIEmSCqlkaJAk\nSe1XudAAVhokSSpD5UKDwxOSJJXD0CBJkgqpZGiQJEntV7nQAFYaJEkqQ+VCg8MTkiSVw9AgSZIK\nqWRokCRJ7Ve50ABWGiRJKkPlQoPDE5IklcPQIEmSCqlkaJAkSe1XudAAVhokSSpD5UKDwxOSJJXD\n0CBJkgqpZGiQJEntV7nQAFYaJEkqQ+VCg8MTkiSVw9AgSZIKqWRokCRJ7Ve50ABWGiRJKkPlQoPD\nE5IklcPQIEmSCqlkaJAkSe1XudAAVhokSSpD5UKDwxOSJJXD0CBJkgqpZGh48EFYs2a8WyJJUrVU\nLjTstVf++vd/P77tkCSpaioXGk49FebPhyefHO+WSJJULZULDZBDw7PPjncrJEmqlsqGhmeeGe9W\nSJJULZUMDVttZaVBkqR2ayk0RMQnI+K2iFgdESsi4nsRsWeB/Q6PiGURsT4ilkfEkpE3eXhWGiRJ\nar9WKw2HApcBbwKOAqYB10bErMF2iIhdgR8CPwX2AS4FvhoRi0fQ3kKsNEiS1H5TW9k4pXRs4/OI\nOA14AugGfjHIbmcB96eUzqk9vzci3gwsBa5rqbUFzZ8P//7v8MILMH16GUeQJGnyGe2chq2ABDw9\nxDYHAtc3rbsGOGiUxx7U3nvnrx/5iFeHlCSpXUYcGiIigC8Bv0gp3TXEpguAFU3rVgDzImLGSI8/\nlNNOgz/9U/jKV+BrXyvjCJIkTT4tDU80uRzYGzikTW15haVLl9LV1bXZup6eHnp6eobd94IL4OKL\n4b77ymqdJEnjq7e3l97e3s3WrVq1qrTjjSg0RMRfA8cCh6aUHhtm88eB7ZvWbQ+sTiltGGrHSy65\nhEWLFo2kiUyZAvvvDytXjmh3SZImvIH+kO7r66O7u7uU47U8PFELDO8E3ppSeqjALrcARzatO7q2\nvlTbbmtokCSpXVq9TsPlwMnA+4DnI2L72jKzYZvzI+Kqht2uBHaLiAsjYq+IOBs4Ebi4De0fkqFB\nkqT2abXScCYwD/g58GjDclLDNguBnetPUkoPAseRr+twO/lUyz9MKTWfUdF2hgZJktqn1es0DBsy\nUkqnD7DuJvK1HMbUttvC/ffDFVfAkiUwe/ZYt0CSpOqo5L0n6k44ARYvhrPPhnPPHe/WSJLU2Sod\nGnbfHa69Fo49Fm68cbxbI0lSZ6t0aKg7/ni47Ta48srxbokkSZ1rUoSG978/f71rqOtWSpKkIU2K\n0LDllrDffrBx43i3RJKkzjUpQgPAtGnw4ovj3QpJkjrXpAkNU6daaZAkaTQmTWiw0iBJ0uhMmtBg\npUGSpNGZNKHBSoMkSaMzqUKDlQZJkkZu0oQGhyckSRqdSRMaHJ6QJGl0Jk1osNIgSdLoTJrQYKVB\nkqTRmTShwUqDJEmjM2lCg5UGSZJGZ1KFBisNkiSN3KQJDVOnWmmQJGk0Jk1osNIgSdLoTJrQYKVB\nkqTRmTShwUqDJEmjM2lCg5UGSZJGZ9KEBisNkiSNjqFBkiQVMmlCg8MTkiSNzqQJDVYaJEkanUkT\nGuqVhpTGuyWSJHWmSRMapk3LX196aXzbIUlSp5o0oWHq1PzVeQ2SJI3MpAkN9UqD8xokSRqZSRMa\npk/PX9/0JrjyyvFtiyRJnWjShIbDDoNPfAJ23RXOOgtOOglWrBjvVkmS1DkmTWjYemu44AK4+mr4\n1Kfgu9+FnXaCP/7j8W6ZJEmdYdKEhrrp0+Gzn4UHHoDjjoPLLoN3vQu+//3xbpkkSRPb1PFuwHjZ\nZRfo7YWPfxz+6Z9yaDjvPDj0UJg9G2bNgnnzYOHC/vkQkiRNZpM2NEAOBn/1V3D++XDqqTk0DHRK\n5qxZ0NWVhzhe/WrYYw/4vd+DLbfMweINb8ivdXXBFpOudiNJmiwmdWiomzMH/vmf4Zln4MknYe1a\nWLcOnn0WHn8cVq3Kj596Ch56CH7yE/jyl195dckI2GormD8/h4jFi2H33fPjAw+EBQvyNpIkdSJD\nQ4P58/NSREqwYQOsXAnLl+fA8cwz8PTT+euDD8IVV+SwURcBRx0Fxx+fh0e22SYvCxbkKoWBQpI0\nkRkaRigCZs6EnXfOy2BefDGf2nnjjTlIfOMb8LGPwQsvbL7d3Ll56GPuXJgxY/Nl7txcrVi4EPbb\nL1dG5szpr2jMmFFqVyVJAkYQGiLiUOBPgW5gIfCulNK/DLH9YcDPmlYnYGFK6YlWj99ppk6FHXeE\n970vP//Up/L9L1auzMMdTz0Fjz2Whz0eegiefz4Hig0b8rJ6NTzySK5gPPxwHjZpNnt2Dg+DLVtt\nlYPF9Ok5eMyf3z/Rc+ut83qrHJKk4Yyk0rAlcDvwNeCfC+6TgD2BNS+vmASBYTBTpuQhiQULWttv\n7docMJ57Li/14ZCBlocf7n/87LND391z6tQ8qXPOHNh227xMndq/zJ2bL4o1c2b/MmPG5s+LrJ9q\nXUuSOlrLP8ZTSj8BfgIQ0dLfpytTSqtbPZ76zZ6dJ1a2KqVc3diwIYeJNWtg/focJp55Jlc3nnsu\nf/3d7/LEzxdfzMv69bnSccMN+XF9GcmNv6ZM2TxE1KsfzYFj9uxcBZk+PQeNadPyMmcO/P7v958S\nO3Nm3mbKlLxsscXAj2fOzPvOmGFFRZJGY6z+9gvg9oiYCfwGODeldPMYHXvSi+ivGrzude15z3oI\naQwSzc+HW1cfgmne5vnn8/DNxo2bLytX5srJSE2Z0l9RmTUr/3tMmbJ5VWWwdVOn5oAyfXp/2Kkv\n06b171fft/lx/eu0aZvvM9DS+NpgjxuPbxCSNFbGIjQ8BpwB/AqYAXwI+HlEHJBSun0Mjq8STJmS\n/+KfPXvsjvnSS7k60hxAXnoJNm3KX+tL4/P16/uHdOrLunX5tXpFpfHxQOvWrcvzS+ohZ+PGPPfk\nhRfy4/q2Q30ty7Rp/UGmOYw0BqB6xabx8UDLYK9Pn97/mTdXixoD1dy5/Ut9W4ONVA2lh4aU0nJg\necOqWyNid2ApsKTs46s6pkzJp6h2opRykGmsnDQHlY0bB37euL7+uB5Y6pNm61+bg0/z+zW+Z315\n4YVc3Rns9cbt1q7N27Yaguphoz60NGtWf8hprtoM9XzGjLxv4xBV/f2GejxzZv7+kTQ64zU17Tbg\nkOE2Wrp0KV1dXZut6+npoaenp6x2SaWI2HyORaerB5fmysu6dbmSs2ZNrsysW5eDRv2CaY2P6/s0\nVm3Wrs3VpOb19cfr1/e/z8aNrbV56tRik3YblzlzcsWkMXwM9bi+zJ7dP49GKlNvby+9vb2brVu1\nalVpx4s01LT64XaO2MQwp1wOst+1wOqU0omDvL4IWLZs2TIWLVo04vZJqq760NO6dXkZ7PFA82qK\nLPWqynPP9b/f+vWthZVp0/qvwbL99jm4zJ8PO+yQn9cn8zZel6U5gDRWVuoVGod7NJS+vj66u7sB\nulNKfe1875Fcp2FLYA/y5EaA3SJiH+DplNLDEXEBsENKaUlt+48CDwB3AjPJcxreCixuQ/slTVL1\nia1bbjm2x62HlfrSGFQaKyr1isuaNXDffbnysnEj/Pa38Mtf5om99SpNK6ZP7w8P06fnM43q1ZDd\ndsvXYpk7N19ldt48eM1rYO+98/VapNEayfDE/uSLNaXa8sXa+quADwALgMZrJE6vbbMDsBa4Azgy\npXTTCNssSeOm3WGlPtelPtRTDyEDLWvX5vCxfn3/8NDq1TmgrF0Lt9+eT49esyafOt0496R+OvPC\nhfDa1+aQ0Xiac72iUX88dy7stVe+IZ/DLKobyXUabgQGvZdjSun0pucXARe13jRJqr4ttugfmmin\nlHLQ+M1vcnXjqadysFi+PN+Irx4+mkNJfWkcuZ47N1ct6nM85s7NVY358/sva19fdt89X7iufibN\nnDnt7ZfGl9fok6QKishVgwMOyEsrUsoTUu++G+65J18Erj7U8txzuYpx++39z9esyfM/Nm165Xvt\ntlu+y2/jKb3z5+dwUT9DZsaMPJzSOHdj5sxczdl6a+dwTCSGBknSZiLyL/aDD85LESnlCsWdd+ZQ\nsWFDDh4/+EG+0mzjqcOPPAJPFLyRwDbb5DkZRx4J222XQ8bee+fH9Rv3OXwydgwNkqRRi8iVgeaq\nximnDLx943VL6gGjeQ5Hvdrxb/8GF16Y53EMdI2Qrq4cHo44Aj7xCdhjD6sTZTE0SJLGXONcjjlz\nil+4beNG+M//zHMy1qyBJ5/MyyOPwOWXw9e/Dq96Fbz97XDaaXkuxk475crEFoPOxlNRhgZJUseY\nNg0Gu3zPX/wF/OxncPPNcMUV8K1v9b82fXoOE3vsAccd1z9Jc6edYM89Yccdx6b9nc7QIEmqhG22\ngRNPzMu55+YKxDPP5CrEww/na2Nccw189rObXw59+nS49VbYb79xbX5HMDRIkipn3ry8AOSLI2af\n+Uz/43Xrcpg4/HBYvBg+/OF8DYt58+CYY3KY0OYMDZKkSWnWrDw08fnPw1VX5SGNlSvza9/5Dnib\no1dyWogkaVI75RS47jpYsSKfNrrLLtDX1js2VIehQZIk8mmas2bBPvvAF74Ad9wx3i2aeAwNkiQ1\nePe7851I77tvvFsy8TinQZKkBkuW5EWvZKVBkiQVYmiQJEmFGBokSVIhhgZJklSIoUGSJBViaJAk\nSYUYGiRJUiGGBkmSVIihQZIkFWJokCRJhRgaJElSIYYGSZJUiKFBkiQVYmiQJEmFGBokSVIhhgZJ\nklSIoUGSJBViaJAkSYUYGiRJUiGGBkmSVIihQZIkFWJokCRJhRgaJElSIYYGSZJUiKFBkiQVYmiQ\nJEmFGBokSVIhhgZJklSIoWEM9Pb2jncT2sr+TFxV6gvYn4msSn2B6vWnLC2Hhog4NCL+JSJ+FxGb\nIuIdBfY5PCKWRcT6iFgeEUtG1tzOVLVvRvszcVWpL2B/JrIq9QWq15+yjKTSsCVwO3A2kIbbOCJ2\nBX4I/BTYB7gU+GpELB7BsSVJ0jiZ2uoOKaWfAD8BiIgosMtZwP0ppXNqz++NiDcDS4HrWj2+JEka\nH2Mxp+FA4PqmddcAB43BsSVJUpu0XGkYgQXAiqZ1K4B5ETEjpbRhgH1mAtx9991lt21MrFq1ir6+\nvvFuRtvYn4mrSn0B+zORVakvUK3+NPzunNnu946Uhp2WMPjOEZuAd6WU/mWIbe4Fvp5SurBh3dvJ\n8xxmDxQaIuJ9wN+OuGGSJOnklNJ32vmGY1FpeBzYvmnd9sDqQaoMkIcvTgYeBNaX1zRJkipnJrAr\n+XdpW41FaLgFeHvTuqNr6weUUnoKaGs6kiRpErm5jDcdyXUatoyIfSJi39qq3WrPd669fkFEXNWw\ny5W1bS6MiL0i4mzgRODiUbdekiSNmZbnNETEYcDPeOU1Gq5KKX0gIr4B7JJSOqJhn7cAlwB7A48A\n56WUvjWqlkuSpDE1qomQkiRp8vDeE5IkqRBDgyRJKmTChYaI+HBEPBAR6yLi1oj4g/Fu00CK3Lgr\nIs6LiEcjYm1EXBcRezS9PiMivhwRT0bEmoj4bkRsN3a9eLkdn4yI2yJidUSsiIjvRcSeA2w34fsT\nEWdGxK8jYlVtuTkijum0fgwmIj5R+367uGl9R/QpIv6y1v7G5a6mbTqiL7W27BAR36q1ZW3te29R\n0zYd0Z/az93mz2ZTRFzWaX2ptWWLiPhfEXF/rb2/jYg/H2C7juhTRMyJiC9FxIO1tv4iIvZv2qb8\nvqSUJswCvId8XYZTgdcBfwM8DbxqvNs2QFuPAc4D3gm8BLyj6fWP19r+34A3AFcD/wVMb9jmCvK1\nKA4D9iOfIvN/x6EvPwbeD7weeCP5wlsPArM6rT/AcbXPZndgD+B/AxuA13dSPwbp2x8A9wP/AVzc\naZ9NrR1/CdwBbAtsV1u27tC+bAU8AHwV6AZ2AY4CXtOh/dmm4TPZDjiS/LPt0E7rS60tnwKeqP08\neDXwbmA18JEO/Xz+HvhP4BBgt9r/pWeBhWPZlzH/IIf5R7kVuLTheZDPtjhnvNs2TLs38crQ8Ciw\ntOH5PGAdcFLD8w3ACQ3b7FV7rwPGuT+vqrXjzRXpz1PA6Z3cD2AOcC9wBPnspcbQ0DF9qv2g6xvi\n9U7qy+eAG4fZpmP6M0DbvwQs79S+AD8AvtK07rvANzutT+SLNW0Ejmla/yvy2Yhj1pcJMzwREdPI\naf2n9XUp9+p6OuzmVhHxGvI9Nxr7shr4N/r7sj/54lqN29wLPMT493cr8im1T0Pn9qdWnnwvMBu4\nuVP7UfNl4AcppRsaV3Zon14beVjvvyLi29F/jZdO68vxwK8i4h8iD+v1RcQH6y92YH9eVvt5fDLw\ntdrzTuwpad7zAAAEV0lEQVTLzcCREfFagIjYh/xX+o9rzzupT1OBKeRf+o3WAW8ey76MxRUhi3oV\n+R9loJtb7TX2zRmVBeRfugP1ZUHt8fbAC7UPdrBtxlxEBPkvjF+klOpjzR3Vn4h4A/mKozOBNeRk\nfW9EHEQH9aOuFnz2Jf+nb9ZRnw25mngauWqyEDgXuKn2mXVaX3YDzgK+CHwWOAD4q4jYkPJ1aDqt\nP41OALqA+oX6OrEvnyP/dX1PRLxEnsP3Zymlv6u93jF9Sik9FxG3AJ+OiHtqx38f+Zf9fYxhXyZS\naNDEcDn5IlyHjHdDRuEeYB/yD70TgW9GvsBYx4mIncgh7qiU0sbxbs9opZQar4X/m4i4Dfh/wEnk\nz62TbAHcllL6dO35r2vh50yg0y9e9wHgX1NKj493Q0bhPeRfrO8F7iIH70sj4tHUmRcXPAX4OvA7\n4EWgj3y7he6xbMSEGZ4AniRPuhno5lad9o37OHk+xlB9eRyYHhHzhthmTEXEXwPHAoenlB5reKmj\n+pNSejGldH9K6T9SSn8G/Br4KB3Wj5pu8qTBvojYGBEbyZOYPhoRL5D/Sui0Pr0spbQKWE6etNpp\nn89jwN1N6+4mT7qDzusPABHxavKEzq80rO7Evnwe+FxK6R9TSnemlP6WfGXiT9Ze76g+pZQeSCm9\nFdgS2DmldCAwnTw5esz6MmFCQ+2vqGXkGbvAy6XyIynpxhtlSSk9QP4QGvsyD3gT/X1ZRk6Ljdvs\nRf6BM+jNvMpSCwzvBN6aUnqo8bVO7E+TLYAZHdqP68lntOxLrp7sQ5789G1gn5RS/QdGJ/XpZREx\nhxwYHu3Az+eXvHLodC9y5aST/998gBxGf1xf0aF9mU3+Q7TRJmq/9zq0T6SU1qWUVkTEfOBtwNVj\n2pexmv1ZcIboScBaNj/l8ilg2/Fu2wBt3ZL8A3xf8jfi/6w937n2+jm1th9P/qF/NXnsqfH0l8vJ\np2wdTv6L8peMz6k8lwPPAIeSU2d9mdmwTUf0Bzi/1o9dyKcdXVD7j3JEJ/VjmD42nz3RMX0CLgLe\nUvt8DgauI/+C2qYD+7I/eWLaJ8mn+L6PPIfmvZ342dTaEuRT8j47wGud1pdvkCf5HVv7fjuBfArm\n+Z3YJ/Ldod9GvuX1YvKp178EpoxlX8b8gyzwD3N27Zt2HTn97D/ebRqknYeRw8JLTcvXG7Y5l3wa\nzFryfc33aHqPGcBl5KGZNcA/AtuNQ18G6sdLwKlN2034/pDPmb+/9v3zOHAttcDQSf0Ypo830BAa\nOqlPQC/5NOp15B/o36Hhugad1JdaW44lX3diLXAn8IEBtumk/iyu/d/fY5DXO6kvW5LvpvwA8Dz5\nF+hngKmd2CfgfwC/rf3f+R1wKTB3rPviDaskSVIhE2ZOgyRJmtgMDZIkqRBDgyRJKsTQIEmSCjE0\nSJKkQgwNkiSpEEODJEkqxNAgSZIKMTRIkqRCDA2SJKkQQ4MkSSrk/wN4gc4flFKqPQAAAABJRU5E\nrkJggg==\n",
      "text/plain": [
       "<matplotlib.figure.Figure at 0x7f20dd0e47f0>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "plt.plot(list(range(data.shape[0])), data['ratio_T/F'])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(841, 11)\n",
      "  VARIANT_TYPE TRUTH_SAMPLE CALL_SAMPLE  TP_COUNT  TN_COUNT  FP_COUNT  \\\n",
      "0          SNP      WGS_P15     WGS_P17   2355178    787015    388736   \n",
      "0          SNP      WGS_P15      WGS_P4   2357788    778329    393133   \n",
      "0          SNP      WGS_P15      WGS_P5   2348849    778393    386907   \n",
      "0          SNP      WGS_P15      WGS_P7   2368268    791385    387482   \n",
      "0          SNP      WGS_P15      WGS_P8   2284686    728096    403335   \n",
      "0          SNP      WGS_P15     imp_P10   1747977   1531146    479493   \n",
      "0          SNP      WGS_P15     imp_P11   1749853   1532386    478247   \n",
      "0          SNP      WGS_P15     imp_P12   1718454   1534779    475785   \n",
      "0          SNP      WGS_P15     imp_P14   1713602   1553773    456878   \n",
      "\n",
      "   FN_COUNT  EMPTY_COUNT     True    False  ratio_T/F  \n",
      "0    397058      3204507  3142193   785794   3.998749  \n",
      "0    398316      3148603  3136117   791449   3.962500  \n",
      "0    407462      3235835  3127242   794369   3.936762  \n",
      "0    402825      3208304  3159653   790307   3.998007  \n",
      "0    389541      3370142  3012782   792876   3.799815  \n",
      "0   1715152      1554981  3279123  2194645   1.494147  \n",
      "0   1711874      1554408  3282239  2190121   1.498656  \n",
      "0   1729753      1507382  3253233  2205538   1.475029  \n",
      "0   1752793      1521914  3267375  2209671   1.478670  \n"
     ]
    }
   ],
   "source": [
    "##### convert to pairwise matrix \n",
    "######Combine the picard concordance contingency tables in one matrix \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "files = glob.glob(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/*.vcf.genotype_concordance_contingency_metrics\")\n",
    "\n",
    "#data = pandas.DataFrame(pandas.read_csv(file, skiprows=6, skipfooter= 7, sep= \"\\t\"))\n",
    "data = pandas.DataFrame()\n",
    "\n",
    "for file in files:\n",
    "   data = data.append(pandas.read_csv(file, skiprows=6, sep= \"\\t\"))\n",
    "\n",
    "data = data[data.VARIANT_TYPE == \"SNP\"]\n",
    "\n",
    "data['True'] = data['TP_COUNT'] + data['TN_COUNT'] \n",
    "data['False'] = data['FP_COUNT'] + data['FN_COUNT'] \n",
    "\n",
    "data['ratio_T/F'] = data['True'] / data['False']\n",
    "data = data.sort_values(by ='ratio_T/F', ascending = False)\n",
    "\n",
    "data = data.sort_values(by =['TRUTH_SAMPLE', 'CALL_SAMPLE'], )\n",
    "print(data.shape)\n",
    "\n",
    "print(data[1:10])   \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "         WGS_P15  imp_P27   imp_P7  imp_P31   WGS_P5   WGS_P8  imp_P19  \\\n",
      "WGS_P15      inf  3.99875   3.9625  3.93676  3.99801  3.79981  1.49415   \n",
      "imp_P27  3.99893      inf  3.94313  3.94947  3.96583  3.87955  1.51273   \n",
      "imp_P7   3.96267  3.94324      inf  3.97895  3.97906  3.85956  1.50868   \n",
      "imp_P31  3.93714  3.94941  3.97892      inf  3.99189  3.91925  1.52335   \n",
      "WGS_P5   3.99837  3.96597  3.97895  3.99194      inf  3.86579  1.51363   \n",
      "WGS_P8   3.80005   3.8795  3.85942  3.91918  3.86566      inf  1.51277   \n",
      "imp_P19  1.49389  1.51241  1.50835  1.52305  1.51333  1.51246      inf   \n",
      "imp_P18  1.49839   1.5179  1.51121   1.5289  1.51501  1.51865  16.0368   \n",
      "imp_P2   1.47477  1.48601  1.47855  1.48438   1.4948  1.48184  1.49871   \n",
      "imp_P4   1.47842  1.49099  1.49008   1.4905  1.50121  1.49314  1.51675   \n",
      "imp_P3   1.91655  1.52021  1.53212  1.51553  1.52539  1.48266  1.49597   \n",
      "WGS_P7   1.47194  1.47434   1.4706  1.48483  1.48172  1.48036  1.49802   \n",
      "WGS_P4   1.59016  2.01679  1.57943  1.58928  1.59359  1.56216  1.51365   \n",
      "imp_P8   1.48557  1.48767  1.48367  1.48622  1.49604  1.49643  1.50044   \n",
      "imp_P23  1.45675  1.47424  1.45771  1.46629  1.47422  1.46918  1.49028   \n",
      "imp_P12  1.51107  1.50678  1.51729  1.50649  1.50762  1.49037  1.51604   \n",
      "imp_P16  1.47102  1.47541  1.47572  1.47075  1.47289  1.44679  1.48218   \n",
      "imp_P29  1.49436  1.51345  1.49749  1.51816  1.51516  1.55381  1.51107   \n",
      "imp_P26  1.57281  1.58197  1.58392  1.58636  1.58776  1.57739    1.526   \n",
      "imp_P15  1.50818   1.5172  1.51083  1.52121  1.52576  1.51139  1.52206   \n",
      "imp_P6   1.49163  1.49761  1.50451  1.49966  1.50559   1.4948  1.42439   \n",
      "imp_P10  1.55722  1.56837  1.55599  1.55525  1.57131  1.53456  1.53835   \n",
      "imp_P17  1.49169  1.49898  1.49658  1.50706  1.50345   1.5055  1.51436   \n",
      "imp_P14  1.50607  1.51315  1.51383  1.51705  1.51929  1.52164  1.51194   \n",
      "imp_P24  1.52798  1.51711  1.93857  1.51279  1.52525  1.48327  1.41646   \n",
      "imp_P5   1.50829  1.51437  1.51279  1.94336  1.51038  1.49656  1.47582   \n",
      "imp_P21  1.49794  1.49169  1.49414  1.49042  1.50256  1.47208  1.50108   \n",
      "WGS_P17  1.51755  1.51452  1.52189  1.50398  2.00496  1.48584  1.43519   \n",
      "imp_P11  1.50281  1.51629  1.50976    1.522  1.51786  2.09476  1.40436   \n",
      "\n",
      "         imp_P18   imp_P2   imp_P4   ...     imp_P15   imp_P6  imp_P10  \\\n",
      "WGS_P15  1.49866  1.47503  1.47867   ...     1.50845  1.49182  1.55748   \n",
      "imp_P27  1.51822  1.48633  1.49127   ...     1.51749  1.49782  1.56865   \n",
      "imp_P7   1.51154  1.47887  1.49035   ...     1.51114  1.50474  1.55631   \n",
      "imp_P31  1.52921  1.48469  1.49076   ...     1.52152  1.49988  1.55554   \n",
      "WGS_P5   1.51532   1.4951  1.50148   ...     1.52606   1.5058  1.57162   \n",
      "WGS_P8   1.51896  1.48216  1.49342   ...      1.5117  1.49503  1.53486   \n",
      "imp_P19  16.0372  1.49873  1.51673   ...     1.52205  1.42433  1.53833   \n",
      "imp_P18      inf  1.50292  1.51845   ...     1.52787  1.42518  1.54473   \n",
      "imp_P2   1.50292      inf  1.53992   ...     1.52855  1.44253  1.58818   \n",
      "imp_P4   1.51848  1.53998      inf   ...     1.52817  1.47404  1.61935   \n",
      "imp_P3   1.49561  1.49756   1.5071   ...     1.52228  1.42178  1.57875   \n",
      "WGS_P7   1.50613  1.53219  1.54471   ...     1.52955   1.4386  1.58129   \n",
      "WGS_P4   1.51548  1.52125  1.53813   ...      1.5614  1.44589  1.62628   \n",
      "imp_P8   1.50629  1.55992  1.58591   ...     1.55112  1.44841  1.61599   \n",
      "imp_P23  1.49186  1.52707  1.53083   ...     1.50962  1.42718  1.58683   \n",
      "imp_P12  1.51768  1.55034  1.55904   ...     1.53625  1.46199  1.63456   \n",
      "imp_P16  1.48287  1.49997  1.51079   ...     1.50835  1.43617  1.57789   \n",
      "imp_P29  1.51066  1.50915  1.51471   ...     1.51251  1.39884  1.52991   \n",
      "imp_P26   1.5301  1.52139  1.51438   ...     1.55078  1.44347  1.58215   \n",
      "imp_P15  1.52788  1.52859  1.52817   ...         inf  1.43373  1.60313   \n",
      "imp_P6   1.42525  1.44263  1.47409   ...     1.43381      inf   1.5411   \n",
      "imp_P10  1.54476  1.58823  1.61934   ...     1.60315  1.54105      inf   \n",
      "imp_P17  1.52214  1.54081  1.55182   ...     1.53698  1.45815  1.60497   \n",
      "imp_P14  1.51567  1.51928  1.51879   ...     1.53656  1.42708  1.57605   \n",
      "imp_P24  1.42268   1.4369  1.46556   ...     1.45035  1.39796  1.53642   \n",
      "imp_P5   1.47719  1.46109  1.47512   ...     1.48896  1.39831  1.54299   \n",
      "imp_P21  1.50343  1.53425  1.53915   ...     1.53975  1.45075  1.63466   \n",
      "WGS_P17  1.43536  1.44976  1.45149   ...     1.45899  1.37807  1.50967   \n",
      "imp_P11   1.4063  1.40344   1.4368   ...     1.42409  1.35072  1.44732   \n",
      "\n",
      "         imp_P17  imp_P14  imp_P24   imp_P5  imp_P21  WGS_P17  imp_P11  \n",
      "WGS_P15  1.49193  1.50633  1.52825  1.50855   1.4982  1.51782  1.50305  \n",
      "imp_P27  1.49926  1.51345  1.51741  1.51466  1.49198  1.51483  1.51656  \n",
      "imp_P7   1.49687  1.51414  1.93908   1.5131  1.49446  1.52221  1.51005  \n",
      "imp_P31  1.50736  1.51735  1.51309  1.94384   1.4907  1.50427  1.52228  \n",
      "WGS_P5   1.50373  1.51961  1.52556  1.51065  1.50285  2.00551  1.51815  \n",
      "WGS_P8    1.5058  1.52196  1.48357  1.49685  1.47238  1.48614  2.09528  \n",
      "imp_P19  1.51435  1.51194  1.41647  1.47581  1.50108  1.43519  1.40434  \n",
      "imp_P18  1.52212  1.51566  1.42268  1.47718  1.50342  1.43535  1.40628  \n",
      "imp_P2   1.54078  1.51926  1.43688  1.46106  1.53422  1.44974   1.4034  \n",
      "imp_P4   1.55184  1.51882   1.4656  1.47513  1.53916  1.45152   1.4368  \n",
      "imp_P3   1.51209  1.50465  1.46976  1.48939  1.56114   1.4676   1.3956  \n",
      "WGS_P7   1.53151  1.51779  1.43934  1.47607  1.54365  1.43618  1.41381  \n",
      "WGS_P4   1.52141  1.53416  1.48748  1.50897  1.54589   1.4935  1.43228  \n",
      "imp_P8   1.57625  1.54284  1.46516  1.48382  1.55667  1.45586  1.44416  \n",
      "imp_P23  1.53316  1.50888  1.43062   1.4592    1.533  1.43512  1.40678  \n",
      "imp_P12  1.55479  1.52081  1.50304  1.49932  1.58945  1.46986  1.42307  \n",
      "imp_P16  1.51415  1.50642  1.46157  1.46657  1.54159  1.43548  1.38051  \n",
      "imp_P29  1.52623  1.50967  1.39732  1.43891  1.45438  1.41214  1.43937  \n",
      "imp_P26  1.54773  1.53668  1.47215  1.49883  1.53446  1.48738  1.43515  \n",
      "imp_P15  1.53697  1.53656  1.45036  1.48895  1.53975    1.459  1.42408  \n",
      "imp_P6   1.45821  1.42715  1.39804  1.39836  1.45081  1.37814  1.35078  \n",
      "imp_P10  1.60499  1.57608  1.53646    1.543  1.63469  1.50971  1.44732  \n",
      "imp_P17      inf  1.53439  1.45761  1.48824  1.54819  1.45271  1.43877  \n",
      "imp_P14  1.53437      inf  1.45172  1.47129  1.53473  1.44714   1.4275  \n",
      "imp_P24  1.45761  1.45171      inf  1.44629  1.48682  1.45906  1.38166  \n",
      "imp_P5   1.48825  1.47131  1.44629      inf   1.4961  1.45763  1.40553  \n",
      "imp_P21  1.54818  1.53473  1.48682   1.4961      inf  1.48428  1.40237  \n",
      "WGS_P17  1.45271  1.44713  1.45905   1.4576  1.48426      inf  1.37856  \n",
      "imp_P11  1.43879  1.42753  1.38169  1.40555  1.40238  1.37858      inf  \n",
      "\n",
      "[29 rows x 29 columns]\n"
     ]
    }
   ],
   "source": [
    "pairwise = pandas.DataFrame(index = set(data['TRUTH_SAMPLE']), columns = set(data['CALL_SAMPLE']))\n",
    "\n",
    "counter = 0 \n",
    "ratios = data['ratio_T/F']\n",
    "\n",
    "for truth in set(data['TRUTH_SAMPLE']):\n",
    "    for call in set(data['CALL_SAMPLE']):        \n",
    "        pairwise.set_value(truth, call, ratios.iloc[counter])\n",
    "        counter = counter + 1\n",
    "        \n",
    "print(pairwise)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "ratio_T/F    3.998749\n",
       "Name: 0, dtype: float64"
      ]
     },
     "execution_count": 207,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.index\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 173,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "nan"
      ]
     },
     "execution_count": 173,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "truth = 2\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 174,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "        WGS_P15 imp_P27 imp_P7 imp_P31 WGS_P5 WGS_P8 imp_P19 imp_P18 imp_P2  \\\n",
      "WGS_P15     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P27     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P7      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P31     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "WGS_P5      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "WGS_P8      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P19     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P18     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P2      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P4      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P3      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "WGS_P7      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "WGS_P4      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P8      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P23     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P12     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P16     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P29     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P26     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P15     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P6      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P10     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P17     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P14     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P24     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P5      NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P21     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "WGS_P17     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "imp_P11     NaN     NaN    NaN     NaN    NaN    NaN     NaN     NaN    NaN   \n",
      "\n",
      "        imp_P4   ...   imp_P15 imp_P6 imp_P10 imp_P17 imp_P14 imp_P24 imp_P5  \\\n",
      "WGS_P15    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P27    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P7     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P31    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "WGS_P5     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "WGS_P8     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P19    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P18    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P2     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P4     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P3     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "WGS_P7     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "WGS_P4     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P8     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P23    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P12    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P16    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P29    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P26    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P15    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P6     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P10    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P17    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P14    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P24    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P5     NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P21    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "WGS_P17    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "imp_P11    NaN   ...       NaN    NaN     NaN     NaN     NaN     NaN    NaN   \n",
      "\n",
      "        imp_P21 WGS_P17 imp_P11  \n",
      "WGS_P15     NaN     NaN     NaN  \n",
      "imp_P27     NaN     NaN     NaN  \n",
      "imp_P7      NaN     NaN     NaN  \n",
      "imp_P31     NaN     NaN     NaN  \n",
      "WGS_P5      NaN     NaN     NaN  \n",
      "WGS_P8      NaN     NaN     NaN  \n",
      "imp_P19     NaN     NaN     NaN  \n",
      "imp_P18     NaN     NaN     NaN  \n",
      "imp_P2      NaN     NaN     NaN  \n",
      "imp_P4      NaN     NaN     NaN  \n",
      "imp_P3      NaN     NaN     NaN  \n",
      "WGS_P7      NaN     NaN     NaN  \n",
      "WGS_P4      NaN     NaN     NaN  \n",
      "imp_P8      NaN     NaN     NaN  \n",
      "imp_P23     NaN     NaN     NaN  \n",
      "imp_P12     NaN     NaN     NaN  \n",
      "imp_P16     NaN     NaN     NaN  \n",
      "imp_P29     NaN     NaN     NaN  \n",
      "imp_P26     NaN     NaN     NaN  \n",
      "imp_P15     NaN     NaN     NaN  \n",
      "imp_P6      NaN     NaN     NaN  \n",
      "imp_P10     NaN     NaN     NaN  \n",
      "imp_P17     NaN     NaN     NaN  \n",
      "imp_P14     NaN     NaN     NaN  \n",
      "imp_P24     NaN     NaN     NaN  \n",
      "imp_P5      NaN     NaN     NaN  \n",
      "imp_P21     NaN     NaN     NaN  \n",
      "WGS_P17     NaN     NaN     NaN  \n",
      "imp_P11     NaN     NaN     NaN  \n",
      "\n",
      "[29 rows x 29 columns]\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
