{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Loading libraries "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 207,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "import os\n",
    "import re\n",
    "import math, random\n",
    "import numpy,scipy,pandas\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import seaborn\n",
    "import itertools\n",
    "import subprocess\n",
    "from pgtools import toolbox, myplots"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 208,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "ImportError",
     "evalue": "No module named 'rnatools'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mImportError\u001b[0m                               Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-208-ce78b2435f76>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0msys\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpath\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'/home/dskola/workspace/expression_modeling/'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0;32mimport\u001b[0m \u001b[0mrnatools\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mImportError\u001b[0m: No module named 'rnatools'"
     ]
    }
   ],
   "source": [
    "import sys\n",
    "sys.path.append('/home/dskola/workspace/expression_modeling/')\n",
    "#import rnatools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Overview file with FASTQ files used for genotyping and expression:\n",
    "## Depository for all fastq files for genotyping: \n",
    "#/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping\n",
    "\n",
    "##Depository for all fastq files for expresson (Sometimes multiple files per patient, for example, when 2 brain regions) \n",
    "\n",
    "#/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-expression "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 210,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "human_rna_sample_info.csv\n",
      "human_epigenomics_sample_name_mapping.csv\n",
      "human_epigenomics_tag_fastq_mapping.csv\n",
      "patient_info.csv\n",
      "sample_info.csv\n",
      "human_rna_tag_fastq_mapping.csv\n"
     ]
    }
   ],
   "source": [
    "######################## \n",
    "##### Copy files from Dylan and rewrite with date to home dir\n",
    "files ={\"human_rna_sample_info.csv\" : \"/home/dskola/human/dbgap_submission/human_rna_sample_info.csv\", \\\n",
    "        \"patient_info.csv\" :\"/home/dskola/human/patient_info.csv\", \\\n",
    "        \"sample_info.csv\" : \"/home/dskola/human/sample_info.csv\", \\\n",
    "        \"human_rna_tag_fastq_mapping.csv\" : \"/home/dskola/human/dbgap_submission/human_rna_tag_fastq_mapping.csv\", \\\n",
    "        \"human_epigenomics_sample_name_mapping.csv\": \"/home/dskola/human/dbgap_submission/human_epigenomics_sample_name_mapping.csv\", \\\n",
    "        \"human_epigenomics_tag_fastq_mapping.csv\": \"/home/dskola/human/dbgap_submission/human_epigenomics_tag_fastq_mapping.csv\", \\\n",
    "       }\n",
    "      \n",
    "#\"human_epigenomics_sample_name_mapping.csv\": \"/home/dskola/human/human_epigenomics_sample_name_mapping.csv\", \\    \n",
    "\n",
    "OUTPUT_PATH = '/home/iholtman/Human_natural_genetic_variation/'    \n",
    "\n",
    "date = \"March_10_2017_\"\n",
    "\n",
    "for key,index in files.items():\n",
    "    print(key)\n",
    "    data = pandas.read_csv(index)\n",
    "    data.to_csv(os.path.join(OUTPUT_PATH, date + key))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 265,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#####################\n",
    "####Load files as basis for master-matrix \n",
    "files ={\"patient_info.csv\" :\"/home/iholtman/Human_natural_genetic_variation/March_10_2017_patient_info.csv\", \\\n",
    "        \"sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/March_10_2017_sample_info.csv\", \\\n",
    "        \"human_rna_sample_info.csv\" : \"/home/iholtman/Human_natural_genetic_variation/March_10_2017_human_rna_sample_info.csv\", \\\n",
    "        \"human_rna_tag_fastq_mapping.csv\" : \"/home/iholtman/Human_natural_genetic_variation/March_10_2017_human_rna_tag_fastq_mapping.csv\", \\\n",
    "        \"human_epigenomics_sample_name_mapping.csv\": \"/home/iholtman/Human_natural_genetic_variation/March_10_2017_human_epigenomics_sample_name_mapping.csv\", \\\n",
    "        \"human_epigenomics_tag_fastq_mapping.csv\": \"/home/iholtman/Human_natural_genetic_variation/March_10_2017_human_epigenomics_tag_fastq_mapping.csv\", \\\n",
    "       }\n",
    "###\n",
    "patient_info = pandas.read_csv(files['patient_info.csv'])\n",
    "sample_info = pandas.read_csv(files['sample_info.csv'])\n",
    "human_rna_tag_fastq_mapping = pandas.read_csv(files['human_rna_tag_fastq_mapping.csv'])\n",
    "human_rna_sample_info = pandas.read_csv(files['human_rna_sample_info.csv'])\n",
    "human_epigenomics_sample_name_mapping = pandas.read_csv(files['human_epigenomics_sample_name_mapping.csv'])\n",
    "human_epigenomics_tag_fastq_mapping = pandas.read_csv(files['human_epigenomics_tag_fastq_mapping.csv'])\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 269,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "human_epigenomics_tag_fastq_mapping\n",
      "   Unnamed: 0  tag_folder                                         fastq_path\n",
      "0           0  H3K4me2_P2  /data/archive/15-09-14-glass/Project_David/hg3...\n",
      "1           1  H3K4me2_P4  /data/archive/15-09-14-glass/Project_David/hg3...\n",
      "2           2  H3K4me2_P5  /data/archive/15-09-14-glass/Project_David/hg3...\n",
      "(56, 3)\n",
      "\n",
      "-------------------------------------------------------------------------\n",
      "human_epigenomics_sample_name_mapping\n",
      "   Unnamed: 0    filename patient sample_num    assay in_vitro  \\\n",
      "0           0  H3K4me2_P2  HMG002       S002  H3K4me2    False   \n",
      "1           1    Input_P2  HMG002       S002    Input    False   \n",
      "2           2      PU1_P4  HMG004       S004      PU1    False   \n",
      "\n",
      "                             paper_name  \n",
      "0  H3K4me2_HMG002_S002_Microglia_ExVivo  \n",
      "1    Input_HMG002_S002_Microglia_ExVivo  \n",
      "2      PU1_HMG004_S004_Microglia_ExVivo  \n",
      "(43, 7)\n"
     ]
    }
   ],
   "source": [
    "############### Epigenomics data \n",
    "print(\"human_epigenomics_tag_fastq_mapping\")\n",
    "print(human_epigenomics_tag_fastq_mapping[0:3])\n",
    "print(human_epigenomics_tag_fastq_mapping.shape)\n",
    "\n",
    "print(\"\\n-------------------------------------------------------------------------\")\n",
    "print(\"human_epigenomics_sample_name_mapping\")\n",
    "print(human_epigenomics_sample_name_mapping[0:3])\n",
    "print(human_epigenomics_sample_name_mapping.shape)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 270,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "    Unnamed: 0_x                                         tag_folder  \\\n",
      "0              0                                         H3K4me2_P2   \n",
      "1              1                                         H3K4me2_P4   \n",
      "2              2                                         H3K4me2_P5   \n",
      "3              3                                         H3K4me2_P7   \n",
      "4              4                                           Input_P2   \n",
      "5              5                                           Input_P4   \n",
      "6              6                                           Input_P5   \n",
      "7              7                                           Input_P7   \n",
      "8              8                                             PU1_P4   \n",
      "9              9                                             PU1_P5   \n",
      "10            10                                             PU1_P7   \n",
      "11            11  Human_Microglia_ChIP_H3K4me2_P8_Occipital_1506...   \n",
      "12            12  Human_Microglia_ChIP_H3K4me2_P8_Occipital_1506...   \n",
      "13            13  Human_Microglia_ChIP_Input_P8_Occipital_150626...   \n",
      "14            14  Human_Microglia_ChIP_PU1_P8_Occipital_150626_D...   \n",
      "15            15  Human_Microglia_ChIP_H3K4me2_P8_Parietal_15062...   \n",
      "16            16  Human_Microglia_ChIP_PU1_P8_Occipital_150626_D...   \n",
      "17            17  Human_Microglia_ATAC_P11_150827_Grey_DGO_15_12_07   \n",
      "18            18        Human_Microglia_ATAC_P7_150528_DGO_15_12_07   \n",
      "19            19  Human_Microglia_ChIP_H3K4me2_P15_160119_DGO_16...   \n",
      "20            20  Human_Microglia_ChIP_H3K27ac_P15_160119_DGO_16...   \n",
      "21            21  Human_Microglia_ChIP_H3K27ac_P15_160119_DGO_16...   \n",
      "22            22  Human_Microglia_ChIP_H3K4me2_P15_160119_DGO_16...   \n",
      "23            23  Human_Microglia_ChIP_Input_P15_160119_DGO_16_0...   \n",
      "24            24  Human_Microglia_ChIP_PU1_P15_160119_DGO_160802...   \n",
      "\n",
      "                                           fastq_path  Unnamed: 0_y  \\\n",
      "0   /data/archive/15-09-14-glass/Project_David/hg3...           0.0   \n",
      "1   /data/archive/15-09-14-glass/Project_David/hg3...           3.0   \n",
      "2   /data/archive/15-09-14-glass/Project_David/hg3...           8.0   \n",
      "3   /data/archive/15-09-14-glass/Project_David/hg3...          13.0   \n",
      "4   /data/archive/15-09-14-glass/Project_David/hg3...           1.0   \n",
      "5   /data/archive/15-09-14-glass/Project_David/hg3...           5.0   \n",
      "6   /data/archive/15-09-14-glass/Project_David/hg3...           9.0   \n",
      "7   /data/archive/15-09-14-glass/Project_David/hg3...          14.0   \n",
      "8   /data/archive/15-09-14-glass/Project_David/hg3...           2.0   \n",
      "9   /data/archive/15-09-14-glass/Project_David/hg3...           7.0   \n",
      "10  /data/archive/15-09-14-glass/Project_David/hg3...          12.0   \n",
      "11  /data/archive/15-09-17-glass/Project_David/hg3...           NaN   \n",
      "12  /data/archive/15-09-17-glass/Project_David/hg3...           NaN   \n",
      "13  /data/archive/15-09-17-glass/Project_David/hg3...           NaN   \n",
      "14  /data/archive/15-09-17-glass/Project_David/hg3...           NaN   \n",
      "15  /data/archive/15-09-17-glass/Project_David/hg3...          17.0   \n",
      "16  /data/archive/15-09-17-glass/Project_David/hg3...           NaN   \n",
      "17  /data/archive/15-12-07-glass/Project_David/Hum...          19.0   \n",
      "18  /data/archive/15-12-07-glass/Project_David/Hum...          11.0   \n",
      "19  /data/archive/16-02-03-nextseq/Project_David/h...          24.0   \n",
      "20  /data/archive/16-02-03-nextseq/Project_David/h...          25.0   \n",
      "21  /data/archive/16-02-03-nextseq/Project_David/h...           NaN   \n",
      "22  /data/archive/16-02-03-nextseq/Project_David/h...           NaN   \n",
      "23  /data/archive/16-02-03-nextseq/Project_David/h...          26.0   \n",
      "24  /data/archive/16-02-03-nextseq/Project_David/h...           NaN   \n",
      "\n",
      "                                             filename patient sample_num  \\\n",
      "0                                          H3K4me2_P2  HMG002       S002   \n",
      "1                                          H3K4me2_P4  HMG004       S004   \n",
      "2                                          H3K4me2_P5  HMG005       S006   \n",
      "3                                          H3K4me2_P7  HMG007       S009   \n",
      "4                                            Input_P2  HMG002       S002   \n",
      "5                                            Input_P4  HMG004       S004   \n",
      "6                                            Input_P5  HMG005       S006   \n",
      "7                                            Input_P7  HMG007       S009   \n",
      "8                                              PU1_P4  HMG004       S004   \n",
      "9                                              PU1_P5  HMG005       S006   \n",
      "10                                             PU1_P7  HMG007       S009   \n",
      "11                                                NaN     NaN        NaN   \n",
      "12                                                NaN     NaN        NaN   \n",
      "13                                                NaN     NaN        NaN   \n",
      "14                                                NaN     NaN        NaN   \n",
      "15  Human_Microglia_ChIP_H3K4me2_P8_Parietal_15062...  HMG008       S012   \n",
      "16                                                NaN     NaN        NaN   \n",
      "17  Human_Microglia_ATAC_P11_150827_Grey_DGO_15_12_07  HMG011       S017   \n",
      "18        Human_Microglia_ATAC_P7_150528_DGO_15_12_07  HMG007       S009   \n",
      "19  Human_Microglia_ChIP_H3K4me2_P15_160119_DGO_16...  HMG015       S022   \n",
      "20  Human_Microglia_ChIP_H3K27ac_P15_160119_DGO_16...  HMG015       S022   \n",
      "21                                                NaN     NaN        NaN   \n",
      "22                                                NaN     NaN        NaN   \n",
      "23  Human_Microglia_ChIP_Input_P15_160119_DGO_16_0...  HMG015       S022   \n",
      "24                                                NaN     NaN        NaN   \n",
      "\n",
      "      assay in_vitro                            paper_name  \n",
      "0   H3K4me2    False  H3K4me2_HMG002_S002_Microglia_ExVivo  \n",
      "1   H3K4me2    False  H3K4me2_HMG004_S004_Microglia_ExVivo  \n",
      "2   H3K4me2    False  H3K4me2_HMG005_S006_Microglia_ExVivo  \n",
      "3   H3K4me2    False  H3K4me2_HMG007_S009_Microglia_ExVivo  \n",
      "4     Input    False    Input_HMG002_S002_Microglia_ExVivo  \n",
      "5     Input    False    Input_HMG004_S004_Microglia_ExVivo  \n",
      "6     Input    False    Input_HMG005_S006_Microglia_ExVivo  \n",
      "7     Input    False    Input_HMG007_S009_Microglia_ExVivo  \n",
      "8       PU1    False      PU1_HMG004_S004_Microglia_ExVivo  \n",
      "9       PU1    False      PU1_HMG005_S006_Microglia_ExVivo  \n",
      "10      PU1    False      PU1_HMG007_S009_Microglia_ExVivo  \n",
      "11      NaN      NaN                                   NaN  \n",
      "12      NaN      NaN                                   NaN  \n",
      "13      NaN      NaN                                   NaN  \n",
      "14      NaN      NaN                                   NaN  \n",
      "15  H3K4me2    False  H3K4me2_HMG008_S012_Microglia_ExVivo  \n",
      "16      NaN      NaN                                   NaN  \n",
      "17     ATAC    False     ATAC_HMG011_S014_Microglia_ExVivo  \n",
      "18     ATAC    False     ATAC_HMG007_S009_Microglia_ExVivo  \n",
      "19  H3K4me2    False  H3K4me2_HMG015_S015_Microglia_ExVivo  \n",
      "20  H3K27ac    False  H3K27ac_HMG015_S015_Microglia_ExVivo  \n",
      "21      NaN      NaN                                   NaN  \n",
      "22      NaN      NaN                                   NaN  \n",
      "23    Input    False    Input_HMG015_S015_Microglia_ExVivo  \n",
      "24      NaN      NaN                                   NaN  \n",
      "(56, 10)\n"
     ]
    }
   ],
   "source": [
    "###Combined epigenomics files on \n",
    "##human_epigenomics_tag_fastq_mapping[\"Sample\"]\n",
    "##human_epigenomics_sample_name_mapping[\"tag_folder\"]\n",
    "human_epigenomics = human_epigenomics_tag_fastq_mapping.merge(human_epigenomics_sample_name_mapping, how='outer', left_on='tag_folder', right_on='filename')\n",
    "print(human_epigenomics[0:25])\n",
    "\n",
    "print(human_epigenomics.shape)\n",
    "#print(human_epigenomics[\"paper_name\"])\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 271,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "   Unnamed: 0_x  tag_folder  \\\n",
      "0             0  H3K4me2_P2   \n",
      "1             1  H3K4me2_P4   \n",
      "2             2  H3K4me2_P5   \n",
      "\n",
      "                                          fastq_path  Unnamed: 0_y patient  \\\n",
      "0  /data/archive/15-09-14-glass/Project_David/hg3...           0.0  HMG002   \n",
      "1  /data/archive/15-09-14-glass/Project_David/hg3...           3.0  HMG004   \n",
      "2  /data/archive/15-09-14-glass/Project_David/hg3...           8.0  HMG005   \n",
      "\n",
      "  sample_num    assay  \n",
      "0       S002  H3K4me2  \n",
      "1       S004  H3K4me2  \n",
      "2       S006  H3K4me2  \n"
     ]
    }
   ],
   "source": [
    "##Delete surplus columns \n",
    "del human_epigenomics[\"paper_name\"]\n",
    "del human_epigenomics[\"filename\"]\n",
    "\n",
    "## Add Tissue column \n",
    "#human_epigenomics[\"Tissue\"] = \"MG_ex\"\n",
    "#human_epigenomics[\"Tissue\"][human_epigenomics[\"in_vitro\"] == True] = \"MG_iv\"\n",
    "\n",
    "del human_epigenomics[\"in_vitro\"]\n",
    "\n",
    "#human_epigenomics.columns = ['Patient_ID', 'Sample_ID', 'assay',  'fastq_path', 'Tissue']\n",
    "\n",
    "print(human_epigenomics[0:3])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 272,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "human_epigenomics_tag_fastq_mapping\n",
      "   Unnamed: 0                                        tag_folder  \\\n",
      "0           0  Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15   \n",
      "1           1  Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15   \n",
      "2           2  Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15   \n",
      "\n",
      "                                          fastq_path  \n",
      "0  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "1  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "2  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "(54, 3)\n",
      "\n",
      "-------------------------------------------------------------------------\n",
      "human_epigenomics_sample_name_mapping\n",
      "   Unnamed: 0                                             Sample in_vitro  \\\n",
      "0           0  Human_Microglia_Brain_RNA_PolyA_P3_Surgery2_16...    False   \n",
      "1           1  Human_Microglia_Brain_Sorted_RNA_PolyA_P3_Surg...     True   \n",
      "2           2   Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15    False   \n",
      "\n",
      "  wholebrain   mcsf   il34    10d     6h    24h    72h  \\\n",
      "0      False  False  False  False  False  False  False   \n",
      "1      False  False   True  False  False   True  False   \n",
      "2      False  False  False  False  False  False  False   \n",
      "\n",
      "                   ...                   broad_diagnosis          pathology  \\\n",
      "0                  ...                             Tumor  High grade glioma   \n",
      "1                  ...                             Tumor  High grade glioma   \n",
      "2                  ...                             Tumor  High grade glioma   \n",
      "\n",
      "   age sex ethnicity date (mm-dd-yy)               medication regimen  \\\n",
      "0  4.0   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "1  4.0   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "2  4.0   M       His        4/6/2015  Temozolomide, Bevaclizumab, XRT   \n",
      "\n",
      "                                                path sample_num  \\\n",
      "0  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...       S029   \n",
      "1  /home/dskola/glass_data/hg38/Microglia/RNA/InV...       S029   \n",
      "2  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...       S003   \n",
      "\n",
      "                       paper_sample_name  \n",
      "0       RNA_HMG003_S029_Microglia_ExVivo  \n",
      "1  RNA_HMG003_S029_Microglia_InVitro_24h  \n",
      "2       RNA_HMG003_S003_Microglia_ExVivo  \n",
      "\n",
      "[3 rows x 39 columns]\n",
      "(51, 39)\n"
     ]
    }
   ],
   "source": [
    "print(\"human_epigenomics_tag_fastq_mapping\")\n",
    "print(human_rna_tag_fastq_mapping[0:3])\n",
    "print(human_rna_tag_fastq_mapping.shape)\n",
    "print(\"\\n-------------------------------------------------------------------------\")\n",
    "print(\"human_epigenomics_sample_name_mapping\")\n",
    "print(human_rna_sample_info[0:3])\n",
    "print(human_rna_sample_info.shape)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 290,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                         tag_folder  \\\n",
      "0  Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15   \n",
      "1  Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15   \n",
      "2  Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15   \n",
      "\n",
      "                                          fastq_path  \\\n",
      "0  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "1  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "2  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "\n",
      "                                             Sample  tissue patient  \\\n",
      "0  Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15  normal  HMG002   \n",
      "1  Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15  normal  HMG003   \n",
      "2  Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15  normal  HMG005   \n",
      "\n",
      "                                                path sample_num    assay  \n",
      "0  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...       S002  RNA_exp  \n",
      "1  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...       S003  RNA_exp  \n",
      "2  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...       S006  RNA_exp  \n"
     ]
    }
   ],
   "source": [
    "###Combined epigenomics files on \n",
    "\n",
    "human_RNA = human_rna_tag_fastq_mapping.merge(human_rna_sample_info, how='outer', left_on='tag_folder', right_on='Sample')\n",
    "\n",
    "\n",
    "del human_RNA['pathology']\n",
    "del human_RNA['age']\n",
    "del human_RNA['ethnicity']\n",
    "del human_RNA['paper_sample_name']\n",
    "del human_RNA['medication regimen']\n",
    "del human_RNA['date (mm-dd-yy)']\n",
    "del human_RNA['broad_diagnosis']\n",
    "del human_RNA['sex']\n",
    "del human_RNA['in_vitro']\n",
    "del human_RNA['wholebrain']\n",
    "del human_RNA['mcsf']\n",
    "del human_RNA['il34']\n",
    "del human_RNA['10d']\n",
    "del human_RNA['timecourse']\n",
    "del human_RNA['primary']\n",
    "del human_RNA['fetal']\n",
    "del human_RNA['lps']\n",
    "del human_RNA['0h']\n",
    "del human_RNA['72h']\n",
    "del human_RNA['6h']\n",
    "del human_RNA['24h']\n",
    "del human_RNA['stempro']\n",
    "del human_RNA['isoproterenol']\n",
    "del human_RNA['7d']\n",
    "del human_RNA['ncm']\n",
    "del human_RNA['acute']\n",
    "del human_RNA['tgfbr1antagonist']\n",
    "del human_RNA['tgfb']\n",
    "del human_RNA['ribozero']\n",
    "del human_RNA['use']\n",
    "del human_RNA['dmso']\n",
    "del human_RNA['human_serum']\n",
    "del human_RNA['acm']\n",
    "del human_RNA['Unnamed: 0_x']\n",
    "del human_RNA['Unnamed: 0_y']\n",
    "\n",
    "\n",
    "human_RNA[\"assay\"] = \"RNA_exp\"\n",
    "\n",
    "print(human_RNA[0:3])\n",
    "#\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 291,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                                               Sample  Unnamed: 0_x  \\\n",
      "0    Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15           NaN   \n",
      "1    Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15           NaN   \n",
      "2    Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15           NaN   \n",
      "3   Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_...           NaN   \n",
      "4   Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...           NaN   \n",
      "5   Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...           NaN   \n",
      "6    Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15           NaN   \n",
      "7   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...           NaN   \n",
      "8   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...           NaN   \n",
      "9    Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21           NaN   \n",
      "10  Human_Microglia_RNA_PolyA_P8_150622_Occipital_...           NaN   \n",
      "11  Human_Microglia_RNA_PolyA_P8_150622_Parietal_D...           NaN   \n",
      "12  Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_...           NaN   \n",
      "13  Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB...           NaN   \n",
      "14  Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23           NaN   \n",
      "15  Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10...           NaN   \n",
      "16  Human_Microglia_RNA_PolyA_P12_151125_DGO_15_12_14           NaN   \n",
      "17  Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04           NaN   \n",
      "18  Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04           NaN   \n",
      "19  Human_Microglia_RNA_PolyA_P17_Temporal_160201_...           NaN   \n",
      "20  Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7...           NaN   \n",
      "21  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...           NaN   \n",
      "22  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...           NaN   \n",
      "23  Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG           NaN   \n",
      "24  Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC           NaN   \n",
      "25  Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT           NaN   \n",
      "26  Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_...           NaN   \n",
      "27  Human_Microglia_RNA_PolyA_P14_exvivo_JoS_16_04_27           NaN   \n",
      "28  Human_Microglia_RNA_PolyA_P18_exvivo_JoS_16_04_27           NaN   \n",
      "29       P14_brain_RNA_Brain_Grey_JoS_16_04_27_ATTCCT           NaN   \n",
      "..                                                ...           ...   \n",
      "26                                                NaN          26.0   \n",
      "27                                                NaN          27.0   \n",
      "28                                                NaN          28.0   \n",
      "29                                                NaN          29.0   \n",
      "30                                                NaN          30.0   \n",
      "31                                                NaN          31.0   \n",
      "32                                                NaN          32.0   \n",
      "33                                                NaN          33.0   \n",
      "34                                                NaN          44.0   \n",
      "35                                                NaN          34.0   \n",
      "36                                                NaN          35.0   \n",
      "37                                                NaN          36.0   \n",
      "38                                                NaN          37.0   \n",
      "39                                                NaN          38.0   \n",
      "40                                                NaN          39.0   \n",
      "41                                                NaN          40.0   \n",
      "42                                                NaN          41.0   \n",
      "43                                                NaN          42.0   \n",
      "44                                                NaN          43.0   \n",
      "45                                                NaN          45.0   \n",
      "46                                                NaN          46.0   \n",
      "47                                                NaN          47.0   \n",
      "48                                                NaN          48.0   \n",
      "49                                                NaN          49.0   \n",
      "50                                                NaN          50.0   \n",
      "51                                                NaN          51.0   \n",
      "52                                                NaN          52.0   \n",
      "53                                                NaN          53.0   \n",
      "54                                                NaN          54.0   \n",
      "55                                                NaN          55.0   \n",
      "\n",
      "    Unnamed: 0_y    assay                                         fastq_path  \\\n",
      "0            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "1            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "2            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "3            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "4            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "5            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "6            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "7            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "8            NaN  RNA_exp  /data/archive/15-06-17-glass/Project_David/hg3...   \n",
      "9            NaN  RNA_exp  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "10           NaN  RNA_exp  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "11           NaN  RNA_exp  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "12           NaN  RNA_exp  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "13           NaN  RNA_exp  /data/archive/15-09-14-glass/Project_David/hg3...   \n",
      "14           NaN  RNA_exp  /data/archive/15-10-13-glass/Project_David/RNA...   \n",
      "15           NaN  RNA_exp  /data/archive/15-11-10-glass/Project_David/RNA...   \n",
      "16           NaN  RNA_exp  /data/archive/15-12-27-glass/Project_David/Hum...   \n",
      "17           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "18           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "19           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "20           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "21           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "22           NaN  RNA_exp  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "23           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_David/hg3...   \n",
      "24           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_David/hg3...   \n",
      "25           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_David/hg3...   \n",
      "26           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_David/hg3...   \n",
      "27           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_Johannes/...   \n",
      "28           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_Johannes/...   \n",
      "29           NaN  RNA_exp  /data/archive/16-05-06-glass/Project_Johannes/...   \n",
      "..           ...      ...                                                ...   \n",
      "26           NaN      NaN  /data/archive/16-03-15-glass/Project_David/ChI...   \n",
      "27           NaN      NaN  /data/archive/16-03-15-glass/Project_David/ChI...   \n",
      "28          30.0    Input  /data/archive/16-03-15-glass/Project_David/ChI...   \n",
      "29          20.0     ATAC  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "30          21.0     ATAC  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "31          22.0     ATAC  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "32          27.0     ATAC  /data/archive/16-04-15-glass/Project_David/hg3...   \n",
      "33          33.0     ATAC  /data/archive/16-07-27-glass/Project_David/hg3...   \n",
      "34          33.0     ATAC  /data/archive/16-07-27-glass/Project_David/hg3...   \n",
      "35          36.0     ATAC  /data/archive/16-07-27-glass/Project_David/hg3...   \n",
      "36          34.0     ATAC  /data/archive/16-07-27-glass/Project_David/hg3...   \n",
      "37          23.0      PU1  /data/archive/16-08-02-glass/Project_David/hg3...   \n",
      "38          28.0      PU1  /data/archive/16-08-02-glass/Project_David/hg3...   \n",
      "39           NaN      NaN  /data/archive/16-09-03-NextSeq/Project_David/H...   \n",
      "40          37.0  H3K27ac  /data/archive/16-09-21-glass/Project_David/hg3...   \n",
      "41          39.0     ATAC  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "42          40.0     ATAC  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "43          41.0  H3K27ac  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "44           NaN      NaN  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "45          38.0    Input  /data/archive/16-09-03-NextSeq/Project_David/H...   \n",
      "46          42.0    Input  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "47          16.0      PU1  /data/archive/15-09-17-glass/Project_David/hg3...   \n",
      "48          15.0     ATAC  /data/archive/15-12-07-glass/redownload/Human_...   \n",
      "49          10.0     ATAC  /data/archive/15-12-07-glass/redownload/Human_...   \n",
      "50           6.0     ATAC  /data/archive/15-12-07-glass/redownload/Human_...   \n",
      "51           4.0  H3K27ac  /data/archive/15-09-17-glass/Project_David/hg3...   \n",
      "52          31.0    Input  /data/archive/16-03-15-glass/Project_David/ChI...   \n",
      "53          18.0    Input  /data/archive/15-09-17-glass/Project_David/hg3...   \n",
      "54          32.0    Input  /data/archive/16-10-11-NextSeq/Project_David/h...   \n",
      "55          35.0     ATAC  /data/archive/16-09-03-NextSeq/Project_David/H...   \n",
      "\n",
      "                                                 path patient sample_num  \\\n",
      "0   /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG002       S002   \n",
      "1   /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG003       S003   \n",
      "2   /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG005       S006   \n",
      "3   /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG005       S006   \n",
      "4   /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG005       S006   \n",
      "5   /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG005       S006   \n",
      "6   /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG007       S009   \n",
      "7   /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG007       S009   \n",
      "8   /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG007       S009   \n",
      "9   /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG006       S008   \n",
      "10  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG008       S011   \n",
      "11  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG008       S012   \n",
      "12  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG008       S014   \n",
      "13  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG008       S014   \n",
      "14  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG010       S016   \n",
      "15  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG011       S017   \n",
      "16  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG012       S018   \n",
      "17  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG015       S022   \n",
      "18  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG016       S023   \n",
      "19  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG017       S025   \n",
      "20  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG015       S022   \n",
      "21  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG017       S025   \n",
      "22  /home/dskola/glass_data/hg38/Microglia/RNA/InV...  HMG017       S025   \n",
      "23  /home/dskola/glass_data/hg38/Brain/RNA/Human_W...  HMG004       S005   \n",
      "24  /home/dskola/glass_data/hg38/Brain/RNA/Human_W...  HMG005       S007   \n",
      "25  /home/dskola/glass_data/hg38/Brain/RNA/Human_W...  HMG007       S010   \n",
      "26  /home/dskola/glass_data/hg38/Brain/RNA/Human_W...  HMG008       S013   \n",
      "27  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG014       S020   \n",
      "28  /home/dskola/glass_data/hg38/Microglia/RNA/ExV...  HMG018       S026   \n",
      "29  /home/dskola/glass_data/hg38/Brain/RNA/P14_bra...  HMG014       S020   \n",
      "..                                                ...     ...        ...   \n",
      "26                                                NaN     NaN        NaN   \n",
      "27                                                NaN     NaN        NaN   \n",
      "28                                                NaN  HMG017       S025   \n",
      "29                                                NaN  HMG013       S019   \n",
      "30                                                NaN  HMG014       S020   \n",
      "31                                                NaN  HMG015       S022   \n",
      "32                                                NaN  HMG016       S023   \n",
      "33                                                NaN  HMG019       S027   \n",
      "34                                                NaN  HMG019       S027   \n",
      "35                                                NaN  HMG021       S031   \n",
      "36                                                NaN  HMG021       S031   \n",
      "37                                                NaN  HMG015       S022   \n",
      "38                                                NaN  HMG017       S025   \n",
      "39                                                NaN     NaN        NaN   \n",
      "40                                                NaN  HMG021       S031   \n",
      "41                                                NaN  HMG031       S043   \n",
      "42                                                NaN  HMG031       S043   \n",
      "43                                                NaN  HMG031       S043   \n",
      "44                                                NaN     NaN        NaN   \n",
      "45                                                NaN  HMG021       S031   \n",
      "46                                                NaN  HMG031       S043   \n",
      "47                                                NaN  HMG008       S012   \n",
      "48                                                NaN  HMG008       S012   \n",
      "49                                                NaN  HMG006       S008   \n",
      "50                                                NaN  HMG005       S006   \n",
      "51                                                NaN  HMG004       S004   \n",
      "52                                                NaN  HMG017       S025   \n",
      "53                                                NaN  HMG008       S012   \n",
      "54                                                NaN  HMG017       S025   \n",
      "55                                                NaN  HMG023       S033   \n",
      "\n",
      "                                           tag_folder  tissue  \n",
      "0    Human_Microglia_RNA_PolyA_P2_150325_DGO_15_06_15  normal  \n",
      "1    Human_Microglia_RNA_PolyA_P3_150406_DGO_15_06_15  normal  \n",
      "2    Human_Microglia_RNA_PolyA_P5_150528_DGO_15_06_15  normal  \n",
      "3   Human_Microglia_RNA_PolyA_P5_Culture_10d_IL34_...  normal  \n",
      "4   Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...  normal  \n",
      "5   Human_Microglia_RNA_PolyA_P5_Culture_10d_MCSF_...  normal  \n",
      "6    Human_Microglia_RNA_PolyA_P7_150528_DGO_15_06_15  normal  \n",
      "7   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...  normal  \n",
      "8   Human_Microglia_RNA_PolyA_P7_Culture_10d_MCSF_...  normal  \n",
      "9    Human_Microglia_RNA_PolyA_P6_150528_DGO_15_08_21  normal  \n",
      "10  Human_Microglia_RNA_PolyA_P8_150622_Occipital_...  normal  \n",
      "11  Human_Microglia_RNA_PolyA_P8_150622_Parietal_D...  normal  \n",
      "12  Human_Microglia_RNA_PolyA_P8_Culture_IL34_DGO_...  normal  \n",
      "13  Human_Microglia_RNA_PolyA_P8_Culture_IL34_TGFB...  normal  \n",
      "14  Human_Microglia_RNA_PolyA_P10_150819_DGO_15_09_23  normal  \n",
      "15  Human_Microglia_RNA_PolyA_P11_150827_DGO_15_10...  normal  \n",
      "16  Human_Microglia_RNA_PolyA_P12_151125_DGO_15_12_14  normal  \n",
      "17  Human_Microglia_RNA_PolyA_P15_160119_DGO_16_04_04  normal  \n",
      "18  Human_Microglia_RNA_PolyA_P16_160125_DGO_16_04_04  normal  \n",
      "19  Human_Microglia_RNA_PolyA_P17_Temporal_160201_...  normal  \n",
      "20  Human_Microglia_Sorted_RNA_PolyA_P15_Culture_7...  normal  \n",
      "21  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...  normal  \n",
      "22  Human_Microglia_Sorted_RNA_PolyA_P17_Temporal_...  normal  \n",
      "23  Human_WholeBrain_RNA_PolyA_P4_DGO_16_05_02_GAGTGG  normal  \n",
      "24  Human_WholeBrain_RNA_PolyA_P5_DGO_16_05_02_GGTAGC  normal  \n",
      "25  Human_WholeBrain_RNA_PolyA_P7_DGO_16_05_02_ACTGAT  normal  \n",
      "26  Human_WholeBrain_RNA_PolyA_P8_Parietal_DGO_16_...  normal  \n",
      "27  Human_Microglia_RNA_PolyA_P14_exvivo_JoS_16_04_27  normal  \n",
      "28  Human_Microglia_RNA_PolyA_P18_exvivo_JoS_16_04_27  normal  \n",
      "29       P14_brain_RNA_Brain_Grey_JoS_16_04_27_ATTCCT  normal  \n",
      "..                                                ...     ...  \n",
      "26  Human_Microglia_ChIP_H3K27ac_P17_160201_DGO_16...     NaN  \n",
      "27  Human_Microglia_ChIP_PU1_P17_160201_DGO_160802...     NaN  \n",
      "28  Human_Microglia_ChIP_Input_P17_Culture_IL34_7d...     NaN  \n",
      "29       Human_Microglia_ATAC_P13_151214_JOS_16_04_04     NaN  \n",
      "30       Human_Microglia_ATAC_P14_160112_JOS_16_04_04     NaN  \n",
      "31       Human_Microglia_ATAC_P15_160119_DGO_16_04_04     NaN  \n",
      "32       Human_Microglia_ATAC_P16_160125_DGO_16_04_04     NaN  \n",
      "33  Human_Microglia_ATAC_P19_Temporal_160426_DGO_1...     NaN  \n",
      "34  Human_Microglia_ATAC_P19_Temporal_160426_DGO_1...     NaN  \n",
      "35  Human_Microglia_ATAC_P21_Cortex_160606_DGO_16_...     NaN  \n",
      "36  Human_Microglia_Sorted_ATAC_P21_Culture_7d_IL3...     NaN  \n",
      "37     Human_Microglia_ChIP_PU1_P15_160119_DGO_160802     NaN  \n",
      "38     Human_Microglia_ChIP_PU1_P17_160201_DGO_160802     NaN  \n",
      "39  Human_Microglia_Sorted_ChIP_H3K27ac_P21_Cultur...     NaN  \n",
      "40  Human_Microglia_Sorted_ChIP_H3K27ac_P21_Cultur...     NaN  \n",
      "41       Human_Microglia_ATAC_P31_160927_DGO_16_10_11     NaN  \n",
      "42  Human_Microglia_Sorted_ATAC_P31_Culture_IL34_7...     NaN  \n",
      "43  Human_Microglia_Sorted_ChIP_H3K27ac_P31_Cultur...     NaN  \n",
      "44  Human_Microglia_Sorted_ChIP_H3K27ac_P31_Cultur...     NaN  \n",
      "45  Human_Microglia_Sorted_ChIP_Input_P21_Culture_...     NaN  \n",
      "46  Human_Microglia_Sorted_ChIP_Input_P31_Culture_...     NaN  \n",
      "47  Human_Microglia_ChIP_PU1_P8_Parietal_150626_DG...     NaN  \n",
      "48  Human_Microglia_ATAC_P8_150622_Parietal_DGO_15...     NaN  \n",
      "49        Human_Microglia_ATAC_P6_150528_DGO_15_12_07     NaN  \n",
      "50        Human_Microglia_ATAC_P5_150526_DGO_15_12_07     NaN  \n",
      "51  Human_Microglia_ChIP_H3K27ac_P4_150420_DGO_15_...     NaN  \n",
      "52  Human_Microglia_ChIP_Input_P17_160201_DGO_16_0...     NaN  \n",
      "53  Human_Microglia_ChIP_Input_P8_Parietal_150626_...     NaN  \n",
      "54        Human_Microglia_ChIP_Input_P17_DGO_16_10_11     NaN  \n",
      "55  Human_Microglia_Sorted_ATAC_P23_Culture_7d_IL3...     NaN  \n",
      "\n",
      "[111 rows x 10 columns]\n"
     ]
    }
   ],
   "source": [
    "### Combining epigenomic and RNA-data \n",
    "human_RNA_epi = pandas.concat([human_RNA, human_epigenomics])\n",
    "\n",
    "print(human_RNA_epi)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 307,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "######## WGS data \n",
    "import re   \n",
    "import pandas \n",
    "import numpy\n",
    "samples = [\"HMG0015\",\"HMG0017\",\"HMG004\",\"HMG005\",\"HMG007\",\"HMG008\"]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 329,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "'list' object is not callable",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-329-6341c0359fab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0mfiles\u001b[0m\u001b[0;34m=\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir2\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m \u001b[0mfolders\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpandas\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mDataFrame\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindex\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mfiles\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0mfolders\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'fastq'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: 'list' object is not callable"
     ]
    }
   ],
   "source": [
    "############################\n",
    "########Location of fastq-files used for genotyping and add patientnumber as a separate column  \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "import re\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/5-dec/\"\n",
    "dir2= \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/cat_fastq_files_28_nov/\"\n",
    "\n",
    "files= glob.glob(dir +\"*.fastq.gz\") + glob.glob(dir2 +\"*.fastq.gz\")\n",
    "\n",
    "folders = pandas.DataFrame([], index=files)\n",
    "folders.index.name='fastq'\n",
    "\n",
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "folders['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patient\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "#print(folders[\"patient\"])\n",
    "del folders['patientz']\n",
    "\n",
    "print(folders[1:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 324,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "'list' object is not callable",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-324-6341c0359fab>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0mfiles\u001b[0m\u001b[0;34m=\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir2\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     12\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 13\u001b[0;31m \u001b[0mfolders\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpandas\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mDataFrame\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindex\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mfiles\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     14\u001b[0m \u001b[0mfolders\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'fastq'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     15\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: 'list' object is not callable"
     ]
    }
   ],
   "source": [
    "############################\n",
    "########Location of fastq-files used for genotyping and add patientnumber as a separate column  \n",
    "import glob\n",
    "import os\n",
    "import pandas\n",
    "import re\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/5-dec/\"\n",
    "dir2= \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/cat_fastq_files_28_nov/\"\n",
    "\n",
    "files= glob.glob(dir +\"*.fastq.gz\") + glob.glob(dir2 +\"*.fastq.gz\")\n",
    "\n",
    "folders = pandas.DataFrame([], index=files)\n",
    "folders.index.name='fastq'\n",
    "\n",
    "folders.sort_index(inplace=True)\n",
    "\n",
    "folders['cat'] = [i.lower().find('cat') > -1 for i in folders.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "folders['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in folders.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "folders['patientz'] = patientz\n",
    "\n",
    "folders[\"patient\"] = folders[\"patient\"] + folders[\"patientz\"]\n",
    "#print(folders[\"patient\"])\n",
    "del folders['patientz']\n",
    "\n",
    "print(folders[1:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 316,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "'list' object is not callable",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-316-9b6320d67305>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     10\u001b[0m \u001b[0mfiles\u001b[0m\u001b[0;34m=\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mglob\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mglob\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdir2\u001b[0m \u001b[0;34m+\u001b[0m\u001b[0;34m\"*.fastq.gz\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     11\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 12\u001b[0;31m \u001b[0mfiles_genotyping_RNA\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mpandas\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mDataFrame\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mfiles\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     13\u001b[0m \u001b[0mfiles_genotyping_RNA\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'fastq'\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     14\u001b[0m \u001b[0mfiles_genotyping_RNA\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msort_index\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0minplace\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: 'list' object is not callable"
     ]
    }
   ],
   "source": [
    "############################\n",
    "########Location of fastq-files used for genotyping and add patientnumber as a separate column  \n",
    "import glob\n",
    "import os\n",
    "import re\n",
    "\n",
    "dir=  \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/5-dec/\"\n",
    "dir2= \"/data/scratch/iholtman/human_microglia_genotyping/fastq-files/fastq-genotyping/cat_fastq_files_28_nov/\"\n",
    "\n",
    "files= glob.glob(dir +\"*.fastq.gz\") + glob.glob(dir2 +\"*.fastq.gz\")\n",
    "\n",
    "files_genotyping_RNA = pandas.DataFrame(index=files)\n",
    "files_genotyping_RNA.index.name='fastq'\n",
    "files_genotyping_RNA.sort_index(inplace=True)\n",
    "\n",
    "files_genotyping_RNA['cat'] = [i.lower().find('cat') > -1 for i in files_genotyping_RNA.index]\n",
    "\n",
    "#####\n",
    "patients = []\n",
    "for rn in files_genotyping_RNA.index:\n",
    "    patient_match = re.match('\\S*?\\_P(?P<patient>[0-9]{1,2})\\S*', rn)\n",
    "    if patient_match:\n",
    "#         \n",
    "        patients.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patients.append(0)\n",
    "files_genotyping_RNA['patient'] = patients\n",
    "\n",
    "patientz = []\n",
    "for rn in files_genotyping_RNA.index:\n",
    "    patient_match = re.match('\\S*?\\-P(?P<patient>[0-9]{1,2})\\-\\S*', rn)\n",
    "    if patient_match:   \n",
    "        patientz.append(int(patient_match.group('patient')))\n",
    "    else:\n",
    "        patientz.append(0)\n",
    "    \n",
    "files_genotyping_RNA['patientz'] = patientz\n",
    "\n",
    "files_genotyping_RNA[\"patient\"] = files_genotyping_RNA[\"patient\"] + files_genotyping_RNA[\"patientz\"]\n",
    "\n",
    "del files_genotyping_RNA[\"patientz\"]\n",
    "\n",
    "##\n",
    "patient_id = []\n",
    "\n",
    "for patient in files_genotyping_RNA[\"patient\"]:\n",
    "\n",
    "    if patient < 10:\n",
    "        patient_id.append(\"HMG000\"+ str(patient))\n",
    "    else:\n",
    "        patient_id.append(\"HMG00\"+ str(patient))\n",
    "        \n",
    "files_genotyping_RNA[\"Patient_ID\"] = patient_id\n",
    "\n",
    "del files_genotyping_RNA[\"patient\"]\n",
    "\n",
    "files_genotyping_RNA[\"assay\"] = \"RNA-seq_genotyping\"\n",
    "\n",
    " \n",
    "###\n",
    "files_genotyping_RNA[\"Info\"] = \"Na\"\n",
    "concat = files_genotyping_RNA[\"Info\"][:]\n",
    "\n",
    "concat[files_genotyping_RNA[\"cat\"] == True] = \"Concatenation\"\n",
    "concat[files_genotyping_RNA[\"cat\"] == False] = \"single_fastq\"\n",
    "\n",
    "files_genotyping_RNA[\"Info\"] = concat\n",
    "del files_genotyping_RNA[\"cat\"]\n",
    "\n",
    "#####\n",
    "files_genotyping_RNA[\"fastq_path\"] = files_genotyping_RNA.index[:] \n",
    "files_genotyping_RNA[\"Tissue\"] = \"unknown\"\n",
    "files_genotyping_RNA.index = range(100, 123)\n",
    "\n",
    "print(files_genotyping_RNA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fastq_path</th>\n",
       "      <th>assay</th>\n",
       "      <th>Tissue</th>\n",
       "      <th>Patient_ID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>/data/archive/16-05-26-glass/Project_David/hg3...</td>\n",
       "      <td>RNA</td>\n",
       "      <td>MG_ex</td>\n",
       "      <td>HMG003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>/data/archive/16-05-26-glass/Project_David/hg3...</td>\n",
       "      <td>RNA</td>\n",
       "      <td>MG_iv</td>\n",
       "      <td>HMG003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>/data/archive/15-06-17-glass/Project_David/hg3...</td>\n",
       "      <td>RNA</td>\n",
       "      <td>MG_ex</td>\n",
       "      <td>HMG003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>/data/archive/16-04-15-glass/Project_David/hg3...</td>\n",
       "      <td>RNA</td>\n",
       "      <td>MG_ex</td>\n",
       "      <td>HMG015</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>/data/archive/16-04-15-glass/Project_David/hg3...</td>\n",
       "      <td>RNA</td>\n",
       "      <td>MG_iv</td>\n",
       "      <td>HMG015</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                          fastq_path assay Tissue Patient_ID\n",
       "0  /data/archive/16-05-26-glass/Project_David/hg3...   RNA  MG_ex     HMG003\n",
       "1  /data/archive/16-05-26-glass/Project_David/hg3...   RNA  MG_iv     HMG003\n",
       "2  /data/archive/15-06-17-glass/Project_David/hg3...   RNA  MG_ex     HMG003\n",
       "3  /data/archive/16-04-15-glass/Project_David/hg3...   RNA  MG_ex     HMG015\n",
       "4  /data/archive/16-04-15-glass/Project_David/hg3...   RNA  MG_iv     HMG015"
      ]
     },
     "execution_count": 138,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "merged_matrix_rna[0:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "  Info Patient_ID Tissue assay  \\\n",
      "0  NaN     HMG003  MG_ex   RNA   \n",
      "1  NaN     HMG003  MG_iv   RNA   \n",
      "2  NaN     HMG003  MG_ex   RNA   \n",
      "3  NaN     HMG015  MG_ex   RNA   \n",
      "4  NaN     HMG015  MG_iv   RNA   \n",
      "\n",
      "                                          fastq_path  \n",
      "0  /data/archive/16-05-26-glass/Project_David/hg3...  \n",
      "1  /data/archive/16-05-26-glass/Project_David/hg3...  \n",
      "2  /data/archive/15-06-17-glass/Project_David/hg3...  \n",
      "3  /data/archive/16-04-15-glass/Project_David/hg3...  \n",
      "4  /data/archive/16-04-15-glass/Project_David/hg3...  \n"
     ]
    }
   ],
   "source": [
    "merged_matrix_RNA_geno = merged_matrix_rna.append(files_genotyping_RNA)\n",
    "print(merged_matrix_RNA_geno[0:5])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "              Info Patient_ID Sample_ID      Tissue               assay  \\\n",
      "0              NaN     HMG002      S002       MG_ex             H3K4me2   \n",
      "1              NaN     HMG002      S002       MG_ex               Input   \n",
      "2              NaN     HMG004      S004       MG_ex                 PU1   \n",
      "3              NaN     HMG004      S004       MG_ex             H3K4me2   \n",
      "4              NaN     HMG004      S004       MG_ex             H3K27ac   \n",
      "5              NaN     HMG004      S004       MG_ex               Input   \n",
      "6              NaN     HMG005      S006       MG_ex                ATAC   \n",
      "7              NaN     HMG005      S006       MG_ex                 PU1   \n",
      "8              NaN     HMG005      S006       MG_ex             H3K4me2   \n",
      "9              NaN     HMG005      S006       MG_ex               Input   \n",
      "10             NaN     HMG006      S008       MG_ex                ATAC   \n",
      "11             NaN     HMG007      S009       MG_ex                ATAC   \n",
      "12             NaN     HMG007      S009       MG_ex                 PU1   \n",
      "13             NaN     HMG007      S009       MG_ex             H3K4me2   \n",
      "14             NaN     HMG007      S009       MG_ex               Input   \n",
      "15             NaN     HMG008      S012       MG_ex                ATAC   \n",
      "16             NaN     HMG008      S012       MG_ex                 PU1   \n",
      "17             NaN     HMG008      S012       MG_ex             H3K4me2   \n",
      "18             NaN     HMG008      S012       MG_ex               Input   \n",
      "19             NaN     HMG011      S017       MG_ex                ATAC   \n",
      "20             NaN     HMG013      S019       MG_ex                ATAC   \n",
      "21             NaN     HMG014      S020       MG_ex                ATAC   \n",
      "22             NaN     HMG015      S022       MG_ex                ATAC   \n",
      "23             NaN     HMG015      S022       MG_ex                 PU1   \n",
      "24             NaN     HMG015      S022       MG_ex             H3K4me2   \n",
      "25             NaN     HMG015      S022       MG_ex             H3K27ac   \n",
      "26             NaN     HMG015      S022       MG_ex               Input   \n",
      "27             NaN     HMG016      S023       MG_ex                ATAC   \n",
      "28             NaN     HMG017      S025       MG_ex                 PU1   \n",
      "29             NaN     HMG017      S025       MG_ex             H3K27ac   \n",
      "..             ...        ...       ...         ...                 ...   \n",
      "48             NaN     HMG008       NaN       MG_iv                 RNA   \n",
      "49             NaN     HMG008       NaN  WholeBrain                 RNA   \n",
      "50             NaN     HMG004       NaN  WholeBrain                 RNA   \n",
      "51             NaN        NaN       NaN          NA                 RNA   \n",
      "52             NaN        NaN       NaN          NA                 RNA   \n",
      "53             NaN        NaN       NaN          NA                 RNA   \n",
      "54             NaN        NaN       NaN          NA                 RNA   \n",
      "100   single_fastq    HMG0012       NaN     unknown  RNA-seq_genotyping   \n",
      "101   single_fastq    HMG0026       NaN     unknown  RNA-seq_genotyping   \n",
      "102   single_fastq    HMG0027       NaN     unknown  RNA-seq_genotyping   \n",
      "103   single_fastq    HMG0029       NaN     unknown  RNA-seq_genotyping   \n",
      "104   single_fastq    HMG0004       NaN     unknown  RNA-seq_genotyping   \n",
      "105  Concatenation    HMG0011       NaN     unknown  RNA-seq_genotyping   \n",
      "106  Concatenation    HMG0014       NaN     unknown  RNA-seq_genotyping   \n",
      "107  Concatenation    HMG0018       NaN     unknown  RNA-seq_genotyping   \n",
      "108  Concatenation    HMG0023       NaN     unknown  RNA-seq_genotyping   \n",
      "109  Concatenation    HMG0024       NaN     unknown  RNA-seq_genotyping   \n",
      "110  Concatenation    HMG0031       NaN     unknown  RNA-seq_genotyping   \n",
      "111  Concatenation    HMG0016       NaN     unknown  RNA-seq_genotyping   \n",
      "112  Concatenation    HMG0019       NaN     unknown  RNA-seq_genotyping   \n",
      "113  Concatenation    HMG0021       NaN     unknown  RNA-seq_genotyping   \n",
      "114  Concatenation    HMG0002       NaN     unknown  RNA-seq_genotyping   \n",
      "115  Concatenation    HMG0003       NaN     unknown  RNA-seq_genotyping   \n",
      "116  Concatenation    HMG0006       NaN     unknown  RNA-seq_genotyping   \n",
      "117  Concatenation    HMG0010       NaN     unknown  RNA-seq_genotyping   \n",
      "118  Concatenation    HMG0015       NaN     unknown  RNA-seq_genotyping   \n",
      "119  Concatenation    HMG0017       NaN     unknown  RNA-seq_genotyping   \n",
      "120  Concatenation    HMG0005       NaN     unknown  RNA-seq_genotyping   \n",
      "121  Concatenation    HMG0007       NaN     unknown  RNA-seq_genotyping   \n",
      "122  Concatenation    HMG0008       NaN     unknown  RNA-seq_genotyping   \n",
      "\n",
      "                                            fastq_path  \n",
      "0    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "1    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "2    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "3    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "4    /data/archive/15-09-17-glass/Project_David/hg3...  \n",
      "5    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "6    /data/archive/15-12-07-glass/redownload/Human_...  \n",
      "7    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "8    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "9    /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "10   /data/archive/15-12-07-glass/redownload/Human_...  \n",
      "11   /data/archive/15-12-07-glass/Project_David/Hum...  \n",
      "12   /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "13   /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "14   /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "15   /data/archive/15-12-07-glass/redownload/Human_...  \n",
      "16   /data/archive/15-09-17-glass/Project_David/hg3...  \n",
      "17   /data/archive/15-09-17-glass/Project_David/hg3...  \n",
      "18   /data/archive/15-09-17-glass/Project_David/hg3...  \n",
      "19   /data/archive/15-12-07-glass/Project_David/Hum...  \n",
      "20   /data/archive/16-04-15-glass/Project_David/hg3...  \n",
      "21   /data/archive/16-04-15-glass/Project_David/hg3...  \n",
      "22   /data/archive/16-04-15-glass/Project_David/hg3...  \n",
      "23   /data/archive/16-08-02-glass/Project_David/hg3...  \n",
      "24   /data/archive/16-02-03-nextseq/Project_David/h...  \n",
      "25   /data/archive/16-02-03-nextseq/Project_David/h...  \n",
      "26   /data/archive/16-02-03-nextseq/Project_David/h...  \n",
      "27   /data/archive/16-04-15-glass/Project_David/hg3...  \n",
      "28   /data/archive/16-08-02-glass/Project_David/hg3...  \n",
      "29   /data/archive/16-03-15-glass/Project_David/ChI...  \n",
      "..                                                 ...  \n",
      "48   /data/archive/15-09-14-glass/Project_David/hg3...  \n",
      "49   /data/archive/16-05-06-glass/Project_David/hg3...  \n",
      "50   /data/archive/16-05-06-glass/Project_David/hg3...  \n",
      "51   /data/archive/16-10-28-NextSeq/project_david/h...  \n",
      "52   /data/archive/16-10-28-NextSeq/project_david/h...  \n",
      "53   /data/archive/16-10-28-NextSeq/project_david/h...  \n",
      "54   /data/archive/16-05-26-glass/Project_David/hg3...  \n",
      "100  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "101  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "102  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "103  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "104  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "105  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "106  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "107  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "108  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "109  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "110  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "111  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "112  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "113  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "114  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "115  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "116  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "117  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "118  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "119  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "120  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "121  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "122  /data/scratch/iholtman/human_microglia_genotyp...  \n",
      "\n",
      "[134 rows x 6 columns]\n"
     ]
    }
   ],
   "source": [
    "###########Combined RNA-seq & genotyping with epigenomics data \n",
    "merged_matrix_RNA_geno_epi = merged_matrix_epi.append(merged_matrix_RNA_geno)\n",
    "\n",
    "print(merged_matrix_RNA_geno_epi)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [default]",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
