{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###all chromosome in one: \n",
    "\n",
    "        \n",
    "bcftools view  merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz -R annoying_snp.txt -o filtered_merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz\n",
    "\n",
    "vcftools --gzvcf /data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz --plink --out /data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip\n",
    "plink --file merged_RNA_imputed_and_WGS_data_and_SNP_chip --noweb --recodeAD --out Genotype_merged_RNA_imputed_and_WGS_data_and_SNP_chip_converted_to_012_format\n",
    "\n",
    "\n",
    "plink --file merged_RNA_imputed_and_WGS_data_and_SNP_chip --noweb --exclude annoying_snp.txt   \n",
    "\n",
    "bcftools view -i/e\n",
    "\n",
    "\n",
    "vcftools --gzvcf merged_RNA_imputed_and_WGS_data_chr1.vcf.gz --plink --out concatenated_imputation_and_WGS_output_chr1\n",
    "plink --file concatenated_imputation_and_WGS_output_chr1_filtered --noweb --recodeAD --out genotype_converted_to_012_format_chr1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "########One time for all data at once ! \n",
    "## Needs adjustment \n",
    "\n",
    "import pandas\n",
    "import os \n",
    "import numpy \n",
    "import seaborn \n",
    "import pickle \n",
    "import matplotlib.pyplot as plt\n",
    "\n",
    "\n",
    "### Read-Data \n",
    "data = pandas.read_table(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/Genotype_merged_RNA_imputed_and_WGS_data_and_SNP_chip\"), sep = \" \")\n",
    "\n",
    "##Transform in numpy object, transform and convert to pandas\n",
    "table = data.as_matrix()\n",
    "genotype = numpy.matrix.transpose(table)\n",
    "\n",
    "pd_genotype = pandas.DataFrame(genotype, index = data.columns, columns= [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\"])\n",
    "pd_genotype = pd_genotype[6:]\n",
    "\n",
    "#print(pd_genotype[0:10])\n",
    "pd_genotype.shape\n",
    "\n",
    "    ##Select only even rows (because they are duplicate)\n",
    "    pd_genotype =pd_genotype[::2]\n",
    "    pd_genotype.shape\n",
    "\n",
    "    print(pd_genotype[1:5])\n",
    "\n",
    "    ##Safe object as pickle \n",
    "    with open(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/chr\" + str(nr +1) + \"_pickle\"), 'wb') as f:\n",
    "        pickle.dump(pd_genotype, f, pickle.HIGHEST_PROTOCOL)\n",
    "\n",
    "    #####################################Make correlation heatmap of chr1 \n",
    "    ##Remove NA values \n",
    "    pd_genotype = pandas.DataFrame.dropna(pd_genotype)\n",
    "    #print(pd_genotype[1:10])\n",
    "    pd_genotype.shape\n",
    "\n",
    "    ##Calculate correlations \n",
    "    plot_data = pd_genotype.astype(float).corr()\n",
    "    #print(plot_data)\n",
    "\n",
    "    PLOT_CMAP = 'YlOrRd'\n",
    "    VMIN = 0\n",
    "    VMAX = 1\n",
    "    fig_size=(12,12)\n",
    "\n",
    "    ##PLot and write to files\n",
    "\n",
    "    cm = seaborn.clustermap(plot_data, figsize=fig_size, vmin=VMIN, vmax=VMAX, cmap=PLOT_CMAP, annot=False)\n",
    "    _=plt.setp(cm.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)\n",
    "\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".pdf\"),  bbox_inches='tight')\n",
    "    plt.savefig(os.path.join(\"/data/scratch/iholtman/human_microglia_genotyping/jupyter/samples_genotype_correlations_imputed_WGS_chr\" + str(nr+1) + \".png\"),  bbox_inches='tight')\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###Change the header names of the VCF file! \n",
    "        \" CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz \" + \\"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P15 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P15_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P17 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P17_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P4 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P4_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P5 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P5_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P7 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P7_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= WGS_P8 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_WGS_P8_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P10 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P10_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P11 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P11_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P12 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P12_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P14 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P14_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P15 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P15_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P16 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P16_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P17 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P17_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P18 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P18_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P19 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P19_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P2 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P2_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P21 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P21_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P23 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P23_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P24 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P24_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P26 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P26_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P27 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P27_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P29 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P29_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P3 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P3_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P31 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P31_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P4 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P4_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P5 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P5_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P6 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P6_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P7 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P7_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= imp_P8 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_imp_P8_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P5 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P5_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P7 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P7_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P8 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P8_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P12 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P12_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P15 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P15_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P17 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P17_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance  CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  CALL_SAMPLE= Chip_P19 O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_Chip_P19_truth_Chip_P19.vcf  TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz  TRUTH_SAMPLE= Chip_P19\n"
     ]
    }
   ],
   "source": [
    "\n",
    "import subprocess\n",
    "\n",
    "samples = [\"WGS_P15\", \"WGS_P17\", \"WGS_P4\", \"WGS_P5\", \"WGS_P7\", \"WGS_P8\", \"imp_P10\", \"imp_P11\" ,\"imp_P12\", \"imp_P14\", \"imp_P15\", \"imp_P16\", \"imp_P17\", \"imp_P18\", \"imp_P19\", \"imp_P2\" ,\"imp_P21\" ,  \"imp_P23\", \"imp_P24\", \"imp_P26\", \"imp_P27\", \"imp_P29\",\"imp_P3\" ,\"imp_P31\" ,\"imp_P4\",\"imp_P5\" , \"imp_P6\", \"imp_P7\", \"imp_P8\", \"Chip_P5\", \"Chip_P7\", \"Chip_P8\", \"Chip_P12\", \"Chip_P15\", \"Chip_P17\",\"Chip_P19\"]\n",
    " \n",
    "for file_x in samples:\n",
    "    for file_y in samples:\n",
    "        print(\"\\n\")\n",
    "\n",
    "        terminal_code = (\"java -Xmx30g -jar '/gpfs/data01/glasslab/home/iholtman/Molgenis/picard/picard.jar' GenotypeConcordance \" + \\\n",
    "        \" CALL_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz \" + \\\n",
    "        \" CALL_SAMPLE= \" + file_x + \\\n",
    "        \" O=/gpfs/data01/glasslab/data/scratch/iholtman/human_microglia_genotyping/compare_imputation_WGS/concordance_picard/concordance_call_\" + file_x + \"_truth_\" + file_y + \".vcf \" + \\\n",
    "        \" TRUTH_VCF=/data/scratch/iholtman/human_microglia_genotyping/compare_imp_WGS_snp_CHIPS/merged_RNA_imputed_and_WGS_data_and_SNP_chip.vcf.gz \" + \\\n",
    "        \" TRUTH_SAMPLE= \" + file_y )\n",
    "        \n",
    "        # #\" USE_THREADING=TRUE \")\n",
    "    \n",
    "    print(terminal_code)\n",
    "    #print(subprocess.check_output(terminal_code, shell=True, stderr=subprocess.STDOUT).decode()) ##\n",
    "\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [conda root]",
   "language": "python",
   "name": "conda-root-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
